{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "toc": true
   },
   "source": [
    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
    "<div class=\"toc\"><ul class=\"toc-item\"><li><ul class=\"toc-item\"><li><span><a href=\"#Import-Latest-COVID-19-model-from-EMMAA\" data-toc-modified-id=\"Import-Latest-COVID-19-model-from-EMMAA-0.1\"><span class=\"toc-item-num\">0.1&nbsp;&nbsp;</span>Import Latest COVID-19 model from EMMAA</a></span></li><li><span><a href=\"#Find-all-INDRA-statements-associated-with-a-publication\" data-toc-modified-id=\"Find-all-INDRA-statements-associated-with-a-publication-0.2\"><span class=\"toc-item-num\">0.2&nbsp;&nbsp;</span>Find all INDRA statements associated with a publication</a></span></li><li><span><a href=\"#Get-statements-by-subject-and-object\" data-toc-modified-id=\"Get-statements-by-subject-and-object-0.3\"><span class=\"toc-item-num\">0.3&nbsp;&nbsp;</span>Get statements by subject and object</a></span></li><li><span><a href=\"#Print-bibliography\" data-toc-modified-id=\"Print-bibliography-0.4\"><span class=\"toc-item-num\">0.4&nbsp;&nbsp;</span>Print bibliography</a></span></li></ul></li><li><span><a href=\"#Get-causal-claims-from-Logette-et-al-2021\" data-toc-modified-id=\"Get-causal-claims-from-Logette-et-al-2021-1\"><span class=\"toc-item-num\">1&nbsp;&nbsp;</span>Get causal claims from Logette et al 2021</a></span></li><li><span><a href=\"#Get-bibliography-of-Logette-et-al-2021\" data-toc-modified-id=\"Get-bibliography-of-Logette-et-al-2021-2\"><span class=\"toc-item-num\">2&nbsp;&nbsp;</span>Get bibliography of Logette et al 2021</a></span></li><li><span><a href=\"#Get-subset-of-EMMAA-graph-corresponding-to-bibliography\" data-toc-modified-id=\"Get-subset-of-EMMAA-graph-corresponding-to-bibliography-3\"><span class=\"toc-item-num\">3&nbsp;&nbsp;</span>Get subset of EMMAA graph corresponding to bibliography</a></span></li><li><span><a href=\"#Visualize-EMMAA-subgraph-of-bibliography\" data-toc-modified-id=\"Visualize-EMMAA-subgraph-of-bibliography-4\"><span class=\"toc-item-num\">4&nbsp;&nbsp;</span>Visualize EMMAA subgraph of bibliography</a></span></li><li><span><a href=\"#Summarize-EMMAA-graph-of-bibliography\" data-toc-modified-id=\"Summarize-EMMAA-graph-of-bibliography-5\"><span class=\"toc-item-num\">5&nbsp;&nbsp;</span>Summarize EMMAA graph of bibliography</a></span></li><li><span><a href=\"#Causal-BEL-statements-from-each-publication-in-bibliography\" data-toc-modified-id=\"Causal-BEL-statements-from-each-publication-in-bibliography-6\"><span class=\"toc-item-num\">6&nbsp;&nbsp;</span>Causal BEL statements from each publication in bibliography</a></span></li></ul></div>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip indra install pandas pybel requests "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/zuck016/.pyenv/versions/anaconda3-2020.11/lib/python3.8/site-packages/pybel/struct/operations.py:8: TqdmExperimentalWarning: Using `tqdm.autonotebook.tqdm` in notebook mode. Use `tqdm.tqdm` instead to force console mode (e.g. in jupyter console)\n",
      "  from tqdm.autonotebook import tqdm\n"
     ]
    }
   ],
   "source": [
    "from collections import Counter\n",
    "import indra\n",
    "import pandas as pd\n",
    "import pybel\n",
    "import pybel.struct\n",
    "import requests\n",
    "from pybel.io.jupyter import to_jupyter\n",
    "import pathlib\n",
    "from IPython.display import HTML, Markdown, Latex\n",
    "from pybel.struct.filters import build_pmid_inclusion_filter\n",
    "from pybel.struct.mutation.induction import get_subgraph_by_pubmed\n",
    "from indra.assemblers.english.assembler import EnglishAssembler\n",
    "from indra.assemblers.pybel.assembler import PybelAssembler\n",
    "\n",
    "from IPython.display import HTML, Markdown\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.15.4'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.version.get_version()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Import Latest COVID-19 model from EMMAA\n",
    "\n",
    "Use the local nodelink bel graph if it exists. Use downloaded indra statements if they exist\n",
    "or download latest indra statements from EMMAA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\")-derived O-acetylneuraminic acids\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:12962 ! \"D-glucosamine 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28790 ! serotonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6030 ! isoniazide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47867 ! indicator), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'His', 'pos': 4}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16914 ! \"salicylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Ser', 'pos': 1850}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Asp', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28790 ! serotonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53457 ! glucuronoxylomannan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15429 ! hydroxylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4325 ! dapsone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6495 ! lipoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81563 ! \"18(R)-HEPE\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17256 ! \"2'-deoxyadenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27569 ! arabinogalactan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0000785\" ! chromatin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0031012\" ! \"extracellular matrix\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17148 ! putrescine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D042783 ! \"Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0004705\" ! \"JUN kinase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33232 ! application), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:79214 ! \"alpha-N-acetylneuraminyl-(2->8)-alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-(1->4)-beta-D-glucosyl-(1<->1)-ceramide(2-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019426 ! Integrases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C016952 ! N-acetyltryptophanamide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:21500 ! \"N-acetyl-9-O-acetylneuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28790 ! serotonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CNTN2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051557 ! \"Hepatocyte Nuclear Factor 4\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KAT2A(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NPs), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Lys', 'pos': 90}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81563 ! \"18(R)-HEPE\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HLA-DMB(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NPs), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDPK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18367 ! \"phosphate(3-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Lys', 'pos': 375}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78037 ! polyhydroxyalkanoate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010799 ! Phycoerythrin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\")-derived O-acetylneuraminic acids\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6030 ! isoniazide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16337 ! \"phosphatidic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}, {'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32849 ! TEMPO), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:46:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NCp7), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation), activity)\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys18Gly'}]\n",
      "INFO: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22478 ! \"amino alcohol\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydrogencarbonate(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:44] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22260 ! adenosines), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:46:44] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15841 ! polypeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:46:44] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23924 ! \"enzyme inhibitor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:46:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:45] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:45] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:46:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:46] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019895 ! Transposases), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:46] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30314 ! \"(R)-lipoic acid\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:46:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent salicylate(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:47] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019895 ! Transposases), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:46:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent steroid(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:47] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35381 ! monosaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:46:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pyrimidines(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46793 ! \"poly(ethylene glycol)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:46:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47885 ! dinucleotide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:46:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:46:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent methylglyoxal-lysine dimer(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sphingomyelin(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:46:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78505 ! venom), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:46:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNAR(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP2C(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro323Leu'}]\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Arg411Ala'}]\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro88Cys'}]\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro88Cys'}]\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:46:55] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:46:55] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro56Ser'}]\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM38(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM38(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOD3(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STK3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3783 ! \"Coagulation Factor Xa\"), [{'kind': 'hgvs', 'hgvs': 'p.Phe10Ala'}]\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFBR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TJP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kACE2), [{'kind': 'hgvs', 'hgvs': 'p.Thr31Lys'}]\n",
      "WARNING: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mACE2 187\"), [{'kind': 'hgvs', 'hgvs': 'p.His353Lys'}]\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-07-16 15:46:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:00] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:TS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:00] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX58(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:01] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP9(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR4(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:02] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX3X(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFASC(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G3BP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FTH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDH1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro56Ser'}]\n",
      "INFO: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Thr2Asp'}]\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (phosphorylation), (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AQP4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MEF2A(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:08] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDC(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:09] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:10] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:10] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:10] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:11] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ala2Ala'}]\n",
      "WARNING: [2021-07-16 15:47:11] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ala2Ala'}]\n",
      "WARNING: [2021-07-16 15:47:12] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002415 ! Cats), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:15] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:17] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:17] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:18] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:18] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:18] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Relaxin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Severe Acute Respiratory Syndrome(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys61Ala'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys61Ala'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tax(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL83(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Igg-2a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pel2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0016021\" ! \"integral component of membrane\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15843 ! \"arachidonic acid\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'hgvs', 'hgvs': 'p.Asn720Asp'}]\n",
      "INFO: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HKU4), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Kit), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Lys237Glu'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Asn350Lys'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Pro653Leu'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Thr97Ile'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Thr97Ile'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PSEN2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RAS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"RaTG13 S\"), [{'kind': 'hgvs', 'hgvs': 'p.Thr403Arg'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S2GDeltaHR2-presenting), [{'kind': 'hgvs', 'hgvs': 'p.Glu2Pro'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:TYMS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent helicase(mods: (modification), muts: (None, None, None))\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:helicase), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:helicase), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:vBcl2), [{'kind': 'hgvs', 'hgvs': 'p.Ala179Leu'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Cs), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Receptors, Growth Factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase 1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27698 ! \"vanadium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PC-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015032 ! Zinc), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:47:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:49018 ! \"neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:43] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:LYNVEA), [{'kind': 'hgvs', 'hgvs': 'p.Cys3Asp'}]\n",
      "INFO: [2021-07-16 15:47:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:47] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Glu'}]\n",
      "WARNING: [2021-07-16 15:47:47] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:47:47] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys112Ser'}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NAD(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ETC complex I\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73601 ! Lys-Asp), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007523 ! Isocitrates), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C014949 ! acetylhydrazine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85044 ! H3K27Ac), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18257 ! ornithine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDPK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52450 ! 2-O-acetyl-1-O-octadecyl-sn-glycero-3-phosphocholine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17012 ! \"N-acetylneuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent prostaglandin I2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent urea(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent progesterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent testosterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydrogencarbonate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent unsaturated fatty acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent zinc atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent (R)-adrenaline(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent azithromycin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent citric acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent butyric acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent low-density lipoprotein cholesterol(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent antagonist(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipoprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ile-Leu(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 24,25-Dihydroxyvitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AMPK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BLVR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent COX(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent COX(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FSH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemoglobin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL12(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3K_p85(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent plasminogen activation(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNASEL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNF5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RUNX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL24(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SGTA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC2A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLPI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SORT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRP19(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX21(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX21(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFPI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TJP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TM7SF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF1B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TXN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UPP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UPP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XIST(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XIST(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZBTB17(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BANP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC13A3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CAMK4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CAV1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NBAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERMAP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD209(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OTUD7B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL32(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MFN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNBIP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNBIP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POSTN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD74(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD79A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VSIG4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DICER1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIAS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDKN1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEBPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1LG2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDEM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIDEA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HACE1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LINGO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OTUB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCHF8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCHF8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMBIM4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YBX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CELF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM62(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP19A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERFE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12C(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12C(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TSPYL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNMT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent C15orf48(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RCAN3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC6A20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIR23B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DEFA1B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGTR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKR1B1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FST(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FTH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GBF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GIP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GTPBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HCRT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HPSE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI27(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI30(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL13(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL18(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ITGAM(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SMAD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MCL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MDM4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MME(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP13(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ABCG2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MVP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NCL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NEDD4(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OXT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD6IP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PML(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP1R10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTPN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB7A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2L1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent itolizumab(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent itolizumab(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SARS-CoV-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Zinc(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Bronchoalveolar Lavage(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Bronchoalveolar Lavage(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Severe Acute Respiratory Syndrome(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 7a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lexA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YFH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL21A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL21A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Non-structural protein 6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ly6c2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IE1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent avrA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NEC2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 4a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 4a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Lnpep(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lmp-1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BPLF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ompC(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascl1a(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Usp18(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent fni(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3A(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Parp-1 inhibitor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent R-BHB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZAP-L(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent miR-Let-7A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent miR-Let-7A(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:65858 ! \"ergosterol peroxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Lys', 'pos': 562}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV cellular receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0007602\" ! phototransduction), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48545 ! pentapeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:70744 ! Gly-Pro), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7775 ! ononin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Pro'}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35807 ! N-nitrosodimethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15366 ! \"acetic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15366 ! \"acetic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16521 ! lanosterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18723 ! nicotine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16991 ! \"deoxyribonucleic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'pos': 27}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50131 ! \"5-aza-2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6121 ! ketamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17230 ! homocysteine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50131 ! \"5-aza-2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Xlr3c(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28812 ! \"(aminomethyl)phosphonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Xlr3c(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT1-S727), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36916 ! cation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17188 ! \"nucleoside 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 328}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36916 ! cation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46345 ! 5-fluorouracil), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent amino acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent amino acid(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"pBluescript(R) II KS+\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Core protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16474 ! NADPH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"BrSARS-N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BECN1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47885 ! dinucleotide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM21(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27226 ! \"uric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NKX2-1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP1R15A(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17013 ! dTMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDKN2A(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 5}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMBIM4(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPIPB3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP6(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73584 ! Leu-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28260 ! galactose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15373 ! biopterin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Glu', 'pos': 4}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INSR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25351 ! \"mevalonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKBIA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16856 ! glutathione), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35895 ! tetraphosphorus), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60169 ! \"phosphatidylinositol trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT138), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74564 ! Lys-Lys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C068798 ! 4-aminophenylphosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17245 ! \"carbon monoxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25523 ! NADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:8614 ! psilocybin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ganglioneuroma(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 15}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ppp1r15b(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ORF1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18421 ! superoxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER17(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent macro domain(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_alpha(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Adenosine Diphosphate(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Interferon(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent ELK1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RLR(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPAR(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HNRNPK(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Protease(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent METTL3(mods: (sumoylation, K))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent E7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation), muts: (None, None, None))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CEBP(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HDAC1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TBK1(mods: (sumoylation))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2N(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28545 ! valinomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Ala', 'pos': 5}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18361 ! \"diphosphate(4-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK12(mods: (modification), muts: (G, 731, E))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:8753 ! RI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Putative movement protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BPLF1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"HIV20\f",
      "medRxiv preprint doi\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PLWH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48343 ! disulfide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36233 ! disaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52479 ! imidate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4208 ! D-mannopyranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Met', 'pos': 9}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017472 ! \"Receptors, Interleukin-1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:DeltaHbA1c), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0007602\" ! phototransduction), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST6GAL1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ERVK-18), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"GS/BS\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0038023\" ! \"signaling receptor activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CM), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAGT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TUSC3(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016766 ! \"Feline Infectious Peritonitis\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:62084 ! \"N-glycoloyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030246\" ! \"carbohydrate binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"transmembrane protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:51142 ! \"amino groups\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17347 ! testosterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019426 ! Integrases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:62875 ! \"2-methylthio-N(6)-(Delta(2)-isopentenyl)adenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22656 ! \"asparaginyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28940 ! calciol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9367 ! swainsonine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'pos': 29}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D034441 ! \"Heterogeneous-Nuclear Ribonucleoproteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17189 ! \"2,5-dihydroxybenzoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23990 ! \"ethyl ester\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25248 ! \"methyl ester\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33848 ! \"polycyclic arene\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30778 ! \"gallic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30314 ! \"(R)-lipoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 5}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6909 ! metronidazole), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37659 ! \"oleanolic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60027 ! polymer), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35341 ! steroid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:77477 ! nitrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16755 ! \"chenodeoxycholic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 6}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16796 ! melatonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 1}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16796 ! melatonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 4}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 5}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16581 ! pregnenolone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 17}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 17}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28940 ! calciol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16113 ! cholesterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Asp', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17823 ! calcitriol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27300 ! \"vitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Asp', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:86319 ! \"25-hydroxyvitamin D2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:77260 ! \"5-methoxy-2-\\{[(4-methoxy-3,5-dimethylpyridin-2-yl)methyl]sulfinyl\\}-1H-benzimidazole\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50038 ! deoxyhypusine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50038 ! deoxyhypusine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26274 ! \"prolyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28683 ! kynurenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019426 ! Integrases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30762 ! salicylate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76238 ! protopanaxadiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:84403 ! phytoceramides), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16830 ! methylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32875 ! \"methyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:47:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63465 ! marneral), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Gly-Phe zwitterion(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetaldehyde(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetylcholine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15551 ! \"prostaglandin E2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent leukotriene B4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent homogentisate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydroxide(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic GMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glutathione(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent bilirubin IXalpha(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent progesterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent putrescine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent estrone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent testosterone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent triglyceride(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent dinitrogen(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetylcholine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent iron atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent galactosylceramide sulfate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L-thyroxine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hormone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hormone(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent leukotriene(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27300 ! \"vitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent selenium atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent serotonin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phosphatidylinositol(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calciol(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sodium(1+)(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium(1+)(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MPO(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Prtn3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent formic acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent bradykinin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent seaborgium atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent resiquimod(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent fumonisin B1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticotropin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD274(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent (S)-lactic acid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent geneticin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent geneticin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent angiotensin II(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent doxycycline(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosteroid(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL5(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phenyl isocyanate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phenyl isocyanate(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent double-stranded RNA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Val-Ala(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N-benzyloxycarbonyl-L-leucyl-L-leucyl-L-leucinal(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N-benzyloxycarbonyl-L-leucyl-L-leucyl-L-leucinal(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Core protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent polyamine(mods: (modification))\n",
      "WARNING: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD64(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Creatine_kinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Creatine_kinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FSH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GOT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GOT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GST(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G_protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HLA_DR(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemoglobin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL12(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL2R(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LH(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP1LC3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3K(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPAR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPAR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TCR(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VEGF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP1B1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent epithelial to mesenchymal transition(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent epithelial to mesenchymal transition(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNA-directed 5'-3' RNA polymerase activity(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent endoplasmic reticulum(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent integral component of membrane(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNASEL(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BECN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RUNX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL12(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SF3B4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BNIP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST6GAL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST3GAL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC2A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC5A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC5A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BRD2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC7A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC9A1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRPK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRPK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BTN3A3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TDO2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TERT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFEB(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFPI(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFBR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBD(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THY1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR3(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TYK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD40(mods: (modification))\n",
      "INFO: [2021-07-16 15:47:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD40LG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TOMM70(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TXN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP4(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAV1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDAC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VTN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NSD2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZBTB17(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LPIN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent C5AR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC13A3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CANX(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCK(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RHOF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD14(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD209(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB14(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAPGEF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD36(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD36(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GORASP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD47(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL32(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3C(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3C(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD69(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD79A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GORASP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEACAM5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEBPD(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNL3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CFTR(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CREB3L4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CREB3L1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NCR3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX58(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UCHL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYLIP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CGAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CGAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CORO1C(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRH(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACKR3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOOL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD200R1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSK(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NIBAN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP27B1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOO(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RILPL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNF166(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP14(mods: (modification), muts: (None, None, None))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNM1L(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNMT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFASC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CENPV(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1B(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1B(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DPYSL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DPYSL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G3BP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC6A20(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRSS57(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TYMP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIR223(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUXA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUXA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGFR(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF3C(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF3B(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF4E(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGT(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EOMES(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRINS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERN1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIPARP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent F3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent F3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FDPS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF23(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZGLP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKR1B1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FUBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GATA2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GATA3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GH2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GIP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GJA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GRK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GRK5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BNIP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNF4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF8(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RBPJ(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL13(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17D(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17D(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL18(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1B(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (phosphorylation), (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOD(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOD(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JUN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JUNB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KPNA2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KRAS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AR(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AREG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LILRB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LRP5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LTA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MASP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MBL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MBP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MDM2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIITA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIITA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCKSL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MME(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP13(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD200(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOD1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKBIA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOTCH1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NR3C1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NR3C1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP2A2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIGMAR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAG5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRKRA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRSS8(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTBP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGDS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTH(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAE1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCHE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Arginase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Dendritic Cells(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Dendritic Cells(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Fishes(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Major Histocompatibility Complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Major Histocompatibility Complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EPO(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Relaxin(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Renin-Angiotensin System(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNA-Dependent RNA Polymerase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Transferases(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNF4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th1 Cells(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Recombinases(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HESX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nppb(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lexA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent psaA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent armB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pgsA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent E7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tax(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mhc(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 9b(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Non-structural protein 6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Delta-actitoxin-Dar1b(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Igg-2a(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tug1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tug1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent At2g14610(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent flbA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sox11-b(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sox11-b(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Uncharacterized 37.2 kDa protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCD1-1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent yqaT(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gag-pol(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Caspase-12(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Caspase-12(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pel2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rmpA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ceacam10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lanA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lanA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 5b(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent mxa(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ORF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pro-pol(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sfl(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXC chemokine(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENaC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ero1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ero1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PM2.5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RD(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tg(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th17 development.Itolizumab(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th17 development.Itolizumab(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Toll-like receptors (TLRs) 2/4 co-receptor(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YmdB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium-binding(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cyclase(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nsp10(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003165 ! \"Complement System Proteins\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys5Ala'}]\n",
      "WARNING: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-07-16 15:48:02] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-07-16 15:48:04] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:48:04] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:05] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:48:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:05] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:48:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:48:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-07-16 15:48:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-07-16 15:48:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:08] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:48:08] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007473 ! \"Ion Channels\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:09] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33416 ! fullerene), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:48:09] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33416 ! fullerene), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-07-16 15:48:11] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NFAT-GFP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:12] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:48:12] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:48:13] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012194 ! \"RNA-Directed DNA Polymerase\"), [{'kind': 'hgvs', 'hgvs': 'p.Met184Val'}]\n",
      "WARNING: [2021-07-16 15:48:14] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FIP1L1-PDGFRA), [{'kind': 'hgvs', 'hgvs': 'p.Ser601Pro'}]\n",
      "WARNING: [2021-07-16 15:48:14] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FIP1L1-PDGFRA), [{'kind': 'hgvs', 'hgvs': 'p.Thr674Ile'}]\n",
      "WARNING: [2021-07-16 15:48:14] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"oncogenic b-raf\"), [{'kind': 'hgvs', 'hgvs': 'p.Val600Glu'}]\n",
      "WARNING: [2021-07-16 15:48:14] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:p.), [{'kind': 'hgvs', 'hgvs': 'p.Gly12Val'}]\n",
      "WARNING: [2021-07-16 15:48:14] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:AF), [{'kind': 'hgvs', 'hgvs': 'p.Ser678Pro'}]\n",
      "WARNING: [2021-07-16 15:48:15] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:15] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29035 ! \"manganese(2+)\"), [{'kind': 'hgvs', 'hgvs': 'p.Ala0Cys'}]\n",
      "WARNING: [2021-07-16 15:48:15] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63153 ! N-acetyl-D-mannosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-07-16 15:48:16] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:BCR-ABL), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:16] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Bcr-Abl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:16] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:BCR-ABL), [{'kind': 'hgvs', 'hgvs': 'p.Thr315Ile'}]\n",
      "INFO: [2021-07-16 15:48:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ile-Leu(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:17] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78420 ! icariin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:18] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:82128 ! PB1-F2), [{'kind': 'hgvs', 'hgvs': 'p.Asn66Ser'}]\n",
      "WARNING: [2021-07-16 15:48:19] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012208 ! Rhabdomyosarcoma), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:19] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Clathrin(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:19] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:19] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030246\" ! \"carbohydrate binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-07-16 15:48:20] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ROCK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VIM(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent MACROD2(mods: (ribosylation))\n",
      "WARNING: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:22] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKFC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKFC(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NIBAN2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CELF2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERFE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DANCR(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPA1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL16(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ITGAM(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LILRB1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB7A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2L1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-07-16 15:48:26] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C030325 ! \"anthrax toxin\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:27] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "INFO: [2021-07-16 15:48:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ganglioneuroma(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:27] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:27] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "INFO: [2021-07-16 15:48:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009439 ! Neuraminidase), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:30] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:30] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:30] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HESX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nppb(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Lnpep(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APL(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Fc domain\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM proteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:LALAP), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MTase-defective), [{'kind': 'hgvs', 'hgvs': 'p.Asp331Ala'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PC), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PRRSV-nsp1alpha), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent R-BHB(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RAS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Ran-GTP), [{'kind': 'hgvs', 'hgvs': 'p.Gln69Leu'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Ran-GTP), [{'kind': 'hgvs', 'hgvs': 'p.Gln69Leu'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"S50/53E\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"S50/53E\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"S50/53E\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:helicase), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nsp11(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"perforin gene\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"theADRP domains\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16991 ! \"deoxyribonucleic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C405335 ! syncytin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D004275 ! \"DNA, Ribosomal\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016147 ! \"Genes, Tumor Suppressor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22260 ! adenosines), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17790 ! methanol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"p53/TP53\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MS), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73582 ! Leu-Cys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(PUBCHEM:6804 ! 6804), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:80666 ! norbelladine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 1}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:GpppA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Cap), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:58573 ! \"kaempferol oxoanion\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007633 ! Keratins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17622 ! dUMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17478 ! aldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FDA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46923 ! dioxane), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:59050 ! \"dimethyl sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:545959 ! \"homovanillic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:68889 ! hydroxytyrosol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134207 ! methylcytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D049308 ! Dioxygenases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MS), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:31] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73582 ! Leu-Cys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16842 ! formaldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60004 ! mixture), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64276 ! glutaraldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35807 ! N-nitrosodimethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18723 ! nicotine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"DMDA 40926\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asp', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16842 ! formaldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60004 ! mixture), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64276 ! glutaraldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:58573 ! \"kaempferol oxoanion\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007962 ! Leukocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7625 ! \"nordihydroguaiaretic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 4}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16521 ! lanosterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0000785\" ! chromatin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP2C(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'pos': 27}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16991 ! \"deoxyribonucleic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17622 ! dUMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39282 ! iodomethane), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D005796 ! Genes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18361 ! \"diphosphate(4-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17802 ! pseudouridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73622 ! \"beta-hydroxywybutosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 6}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74035 ! \"small nuclear RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000375 ! Aging), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64313 ! 4-demethylwyosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134207 ! methylcytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D049308 ! Dioxygenases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"p53/TP53\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:131444 ! guanidinoacetate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:49886 ! rimantadine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18349 ! \"N(1)-(5-phospho-D-ribosyl)glycinamide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C537847 ! \"Noonan syndrome 3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Arg', 'pos': 467}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"XIST RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017027 ! \"Protein Tyrosine Phosphatases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60816 ! immunogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16793 ! \"mercury(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Cap), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17980 ! \"3'-phospho-5'-adenylyl sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63895 ! \"ionic liquid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(PUBCHEM:6804 ! 6804), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15713 ! UTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48267 ! tubercidin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64313 ! 4-demethylwyosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17802 ! pseudouridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Cap-0 viral mRNAs\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein palmitoylation', 'identifier': '0018345'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RUNX1b), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 787}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3416 ! carfentrazone-ethyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009404 ! \"Nephrotic Syndrome\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 828}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0050126\" ! \"N-carbamoylputrescine amidase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT1-S727), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SPRK1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33838 ! nucleoside), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50266 ! prodrug), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"cycling G-associated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:serine-197), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:parts), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 828}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007074 ! \"Immunoglobulin G\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:increase), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT3/STAT4/STAT5/STAT6\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"type I receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 497}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Val', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:5417 ! glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15354 ! choline), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16919 ! creatine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17087 ! ketone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"NF-jB p50\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"interferon-gamma activated sequence\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16595 ! \"1D-myo-inositol 1,4,5-trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33838 ! nucleoside), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18077 ! dTTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:31536 ! emtricitabine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C087636 ! \"vasodilator-stimulated phosphoprotein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 157}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17013 ! dTMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17992 ! sucrose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 328}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25212 ! metabolite), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27568 ! \"selenium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C113267 ! nephrin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18179 ! phosphoinositide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60948 ! \"alpha-D-Gal-(1->4)-beta-D-Gal-(1->4)-D-Glc\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16474 ! NADPH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18009 ! \"NADP(+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27093 ! \"tricarboxylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28087 ! glycogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74566 ! Lys-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005840\" ! ribosome), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48343 ! disulfide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NRTI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50594 ! \"carbon nanotube\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 2-O-acetyl-1-O-octadecyl-sn-glycero-3-phosphocholine(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60072 ! \"erythrityl tetranitrate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT1/2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17256 ! \"2'-deoxyadenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4705 ! \"double-stranded DNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 680}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C083800 ! \"urea transporter\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ala', 'pos': 1}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26392 ! \"purine nucleoside monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74707 ! Met-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74976 ! \"beta-D-Glcp-(1->3)-[beta-D-Glcp-(1->3)-beta-D-GalpNAc-(1->4)-alpha-D-Glcp-(1->4)]-beta-D-GalpNAc-(1->3)-alpha-D-ManpO[CH2]5NH2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85682 ! \"diacylglycerol 30:1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9765 ! Trypsin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"protein tyrosine kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7421 ! NAC), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"WNK subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 60}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:variety), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:42638 ! \"flufenamic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53490 ! N-formyl-L-methionyl-L-leucyl-L-phenylalanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010750 ! Phosphoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D061307 ! \"Human Umbilical Vein Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6801 ! metformin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006937 ! Hypercholesterolemia), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22984 ! \"calcium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26708 ! \"sodium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D061307 ! \"Human Umbilical Vein Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"recQ subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18022 ! thiocyanate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}, {'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15354 ! choline), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60072 ! \"erythrityl tetranitrate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Asp', 'pos': 1}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 780}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18022 ! thiocyanate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Replicase polyprotein 1ab\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17172 ! \"2'-deoxyguanosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17256 ! \"2'-deoxyadenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:42485 ! \"formyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53490 ! N-formyl-L-methionyl-L-leucyl-L-phenylalanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003713 ! \"Dendritic Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D020033 ! \"Protein Isoforms\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036681 ! \"Fluorescence Recovery After Photobleaching\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48705 ! agonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV Nucleoprotein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019590 ! \"Nucleocapsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SR-rich sequence\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:factors), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72316 ! \"virulence factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification), (phosphorylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT3/STAT4/STAT5/STAT6\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30413 ! heme), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34873 ! N-nitrosodiethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STATs proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 63}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 73}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90695 ! \"anthra[1,9-cd]pyrazol-6(2H)-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000682 ! Amyloid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0004708\" ! \"MAP kinase kinase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"extracellular-regulated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18053 ! \"1-aminocyclopropanecarboxylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28747 ! \"picolinic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017076 ! \"Computer-Aided Design\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Gln690), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Val16), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17076 ! streptomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48705 ! agonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D042783 ! \"Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012269 ! \"Ribosomal Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:phosphatidylinositol-3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18348 ! \"1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37530 ! \"phosphatidylinositol 4-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C020845 ! P-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16749 ! 1-phosphatidyl-1D-myo-inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 133}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 25}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22984 ! \"calcium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36704 ! imiquimod), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002364 ! Caseins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D057167 ! Claudins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:82128 ! PB1-F2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 450}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C087636 ! \"vasodilator-stimulated phosphoprotein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"MLC phosphatase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15918 ! \"2'-deoxycytosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SMAD(mods: (sumoylation))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TGFB(mods: (sumoylation, R, 1))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:8753 ! RI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C069330 ! \"ribosomal protein S6 kinase, 90kDa, polypeptide 3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:147675 ! \"5-iodo-2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60761 ! floxuridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75179 ! trifluridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tryptase(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent mTORC1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent p38(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent p38(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28728 ! \"thromboxane B2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63550 ! TTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010750 ! Phosphoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RNpSTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RIPK3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012194 ! \"RNA-Directed DNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"colorectal carcinoma\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"N SARS-CoV-2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SR-motifs), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SR), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011479 ! Protamines), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019590 ! \"Nucleocapsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75011 ! Val-Glu), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:65769 ! dieckol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 727}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kinase-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29035 ! \"manganese(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification), (phosphorylation))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002347 ! Carps), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"type I receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28744 ! 2-hydroxy-17beta-estradiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16450 ! \"2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:140318 ! tefuryltrione), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:472552 ! \"5-bromo-2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBL4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBL4A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ULK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VIM(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"CD46/CYT-2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SAPK/JNK\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051557 ! \"Hepatocyte Nuclear Factor 4\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Lys-788), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6801 ! metformin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16595 ! \"1D-myo-inositol 1,4,5-trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GCA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GCA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64108 ! \"8-bromo-3',5'-cyclic GMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI44(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15361 ! pyruvate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD9(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008869 ! \"Microtubule-Associated Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Replicase polyprotein 1ab\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:helicase), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEACAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAB1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15918 ! \"2'-deoxycytosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16695 ! \"uridine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17361 ! \"cytidine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17622 ! dUMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNLR1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63112 ! \"fingolimod hydrochloride\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0016887\" ! \"ATPase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIM3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 2204}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHN1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0036387\" ! \"pre-replicative complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"neurofilament heavy chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"neurofilament light chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16016 ! dihydroxyacetone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28260 ! galactose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50699 ! oligosaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23530 ! cytokinin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:31858 ! Mizoribine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63580 ! ribavirin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007511 ! Ischemia), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63580 ! ribavirin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37712 ! O-phosphoserine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"IKKalpha,beta,gamma\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D005399 ! Fishes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Inflammatory Bowel Disease\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"zeta chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D053763 ! Presenilins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0009978\" ! \"allene oxide synthase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:2alpha), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic initiation factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30762 ! salicylate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0009978\" ! \"allene oxide synthase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012208 ! Rhabdomyosarcoma), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 796}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 276}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15373 ! biopterin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017076 ! \"Computer-Aided Design\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036681 ! \"Fluorescence Recovery After Photobleaching\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33709 ! \"amino acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GFAP(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33709 ! \"amino acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP5(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C448021 ! \"glycogen synthase kinase 3 alpha\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2K(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"janus family\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011505 ! \"Protein-Tyrosine Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008780 ! Methyltransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016875 ! \"tau Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K7(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mitogen activated protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15551 ! \"prostaglandin E2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:87103 ! \"2-tert-butyl-9-fluoro-1,6-dihydrobenzo[h]imidazo[4,5-f]isoquinolin-7-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 23}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036022 ! \"Capsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"microtubule-associated protein tau\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63550 ! TTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006360 ! \"Heat-Shock Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RNpSTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:elF4E), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kinase-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25351 ! \"mevalonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34674 ! \"dextran sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16981 ! \"O-(3-O-D-galactosyl-N-acetyl-beta-D-galactosaminyl)-L-serine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23357 ! cofactor), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mitogen activated protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:elongation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C095886 ! anillin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15361 ! pyruvate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32816 ! \"pyruvic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010088 ! Oxidoreductases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mis18 family\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015470 ! \"Leukemia, Myeloid, Acute\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37712 ! O-phosphoserine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C064368 ! \"proto-oncogene protein Spi-1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:eIF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eIF2alpha [\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic initiation factor 2, nor oligoadenylate synthetase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"initiation factor eIF2alpha\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015852 ! \"Eukaryotic Initiation Factor-2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34674 ! \"dextran sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74564 ! Lys-Lys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9516 ! thapsigargin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28757 ! fructose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 787}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cGMP-dependent), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 15}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D050838 ! \"Interferon Regulatory Factor-3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63112 ! \"fingolimod hydrochloride\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:variety), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28747 ! \"picolinic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53407 ! \"poly(lactic acid) macromolecule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63153 ! N-acetyl-D-mannosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017361 ! \"Viral Nonstructural Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:34] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28757 ! fructose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable head completion/stabilization protein(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000070539 ! \"Pleckstrin Homology Domains\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 34}]\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011087 ! \"Polycythemia Vera\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17076 ! streptomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"extracellular-regulated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D050976 ! \"Basic-Leucine Zipper Transcription Factors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D014411 ! \"Neoplastic Stem Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16783 ! \"1D-myo-inositol 1,3,4,5-tetrakisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000080865 ! Formins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER17(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL97(mods: (modification), muts: (None, None, None))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation, R, 42))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, A, 1))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PA(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent C3 exoenzyme(mods: (ribosylation, R))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent GTP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide residue(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP-D-ribose(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphosphate(4-)(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation, C, 3))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent mendelevium atom(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Met-Asp(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Core protein(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent single-stranded DNA(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent AXIN(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent E3_Ub_ligase(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, C))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, R))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s_alpha(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation, R, 42))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RNA binding(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RPL24(mods: (ribosylation, E, 106))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TERF1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SIRT4(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SIRT6(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SH2D3C(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP14(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G3BP1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent AGO2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent FDFT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent GAPDH(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHOA(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent LYZ(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent MBP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NUP214(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ATP6V0C(mods: (ribosylation, Q, 124))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAB5A(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RBFOX2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Transferases(mods: (ribosylation, R))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SCC3(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Large T antigen(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Galphas(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent cysH(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent 13(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide(mods: (ribosylation, H))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide residue(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation, R, 177))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHO(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Death(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SP1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_i_alpha(mods: (ribosylation, C, 351))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHOA(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Galphas(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Death(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_i(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, A, 1))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent KDM2A(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TRPT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP3(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Protease(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SDS(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent FDFT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent IKBKG(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NFAT(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NFASC(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ctbp1(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent C3 exoenzyme(mods: (ribosylation, R))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation, E, 18))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation, E, 19))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation, R, 177))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NAD(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation, 123))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation, R, 41))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAC2(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NPM1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Histone(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RARA(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HIF1A(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MYC(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NCOR(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPARG(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent VHL(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM25(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MAPK7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent METTL3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation, K, 888))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation, K, 96))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation, K, 231))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SUMO3(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 150))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 440))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 60))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent lanA2(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent lanA2(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2I(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CTBP1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SP7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 150))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 440))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 60))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CDK9(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent VHL(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RBFOX2(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent 3D(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RING(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CD55(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NR1H2(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Nsp1alpha(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SAMHD1(mods: (phosphorylation), (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CREBBP(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent U90/U89(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPARG(mods: (sumoylation, K, 107))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DNM1L(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PTPN1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DNM1L(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HDAC3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RANBP2(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM5(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation, K, 452))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent GTF2I(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MRE11(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NBN(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL82(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Daxx protein(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent N(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Ubiquitin(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Histone(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SP100(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent E3_Ub_ligase(mods: (sumoylation))\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE-2 receptor(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF(), nucleoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT1 receptors(), plasma membrane, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT2 receptors(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT2 receptors(), plasma membrane, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATPase(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDCA7L(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E4 protein(), None, inclusion body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERVK-18(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EspB(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F protein(), None, basement membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F protein(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FAK(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GCR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IVNS1ABP(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IVNS1ABP(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCIT:C121977(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCIT:C63927(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPs(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nuclear Factor of Activated T Cells (NFAT)(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBPL proteins(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PABP(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PC2(), None, secretory granule)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRF(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(REV(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIIa(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Rev(), nucleolus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SARS-CoV-2 proteins(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLTRiP protein(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SREBP family(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(V ATPase I(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(V ATPase I(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP40(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP40(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP9(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VacA(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(air(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(effector protein(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(element binding protein(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum-Golgi intermediate compartment (ERGIC)(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gram(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(haemagglutinin(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hairpins(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(kinesin motor(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(microtubule-associated protein 1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(non-structural protein(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nuclear eIF2alpha(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nuclear factor(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(p65 and p50(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pre-miRNA(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pre-miRNA(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(protein tyrosine kinase(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(receptor-activated SMAD(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(replicase(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(replication protein(), None, membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rev(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(short hairpin RNA(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(short hairpin RNA(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(snf7(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(srr7(), cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(synaptobrevin(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(transmembrane protein(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tumour suppressor(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vesicular stomatitis virus G protein(), endoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(viral receptor(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(electron(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(electron(), None, respirasome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(alpha-D-Manp-(1->3)-[alpha-D-Manp-(1->6)]-[beta-D-Xylp-(1->2)]-beta-D-Manp-(1->4)-beta-D-GlcpNAc-(1->4)-D-GlcpNAc-yl group(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(oligomer(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(small nucleolar RNA(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citrate anion(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citrate anion(), mitochondrial matrix, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ile-Thr(), None, trans-Golgi network)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glutamate(1-)(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coenzyme A(), None, mitochondrial matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetyl-CoA(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(choline(), None, neuromuscular junction)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), None, neuromuscular junction)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), axon, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), vesicle, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyruvate(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetic acid(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), host cell, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(protoporphyrin(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation((15Z)-12-oxophyto-10,15-dienoic acid(), chloroplast, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polypeptide(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polypeptide(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arachidonic acid(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTP(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L-glutamic acid(), synapse, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylethanolamine(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cytosine(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), None, organelle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), lysosome, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), virion membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyridine(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(guanine(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hydrogen peroxide(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitrite(), None, chloroplast)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hyaluronic acid(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hyaluronic acid(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipopolysaccharide(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipopolysaccharide(), periplasmic space, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitric oxide(), cell body, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitric oxide(), dendrite, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), None, endoplasmic reticulum membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), phagolysosome, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), phagolysosome, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(1-phosphatidyl-1D-myo-inositol(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADP(), None, nuclear membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(2-oxoglutarate(2-)(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glutathione(), None, cytoplasm)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyrimidine(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(leukotriene C4(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(streptomycin(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, ciliary pocket)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), rough endoplasmic reticulum, coated vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), cytoplasm, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GDP(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L-cysteine(), capsule, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cortisol(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ceramide(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ceramide(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hydrogen chloride(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sucrose(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sucrose(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(thiocyanate(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CMP-N-glycoloyl-beta-neuraminic acid(), cytosol, Golgi lumen)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dopamine(), None, synaptic vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dopamine(), axon, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), cytosol, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidyl-L-serine(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidyl-L-serine(), None, outer membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cadmium atom(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(drug(), virion membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(heparan sulfate proteoglycan(), None, focal adhesion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(proton(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(proton(), None, synaptic vesicle membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ion(), cytoplasm, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neurotransmitter(), None, synaptic cleft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neurotransmitter(), None, synaptic vesicle membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitrites(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), host cellular component, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), sperm plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vitamin D(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(zinc atom(), cytoplasm, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenoxymethylpenicillin(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenoxymethylpenicillin(), vesicle, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acrylamide(), periplasmic space, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(copper atom(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron(3+)(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(copper(2+)(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sodium(1+)(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, endosome)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, replication compartment)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, sarcoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), endolysosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), sarcoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(disulfur(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tunicamycin(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(heme(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(silver atom(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citric acid(), mitochondrial matrix, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citric acid(), mitochondrion, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(bentiromide(), None, periplasmic space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(bithionol(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C-Protein(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(platinum(), None, chloroplast)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(platinum(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cerium(), None, vacuolar membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(noradrenaline(), None, neuromuscular junction)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, apical plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(messenger RNA(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(messenger RNA(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(amino acid(), None, brush border membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(amino acid(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleoside(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(fucose(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(effector(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tetraphosphorus(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sodium taurocholate(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Collagen(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cytochrome(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetyl group(), periplasmic space, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tamoxifen(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-acetyl-beta-neuraminic acid(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded DNA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(disulfide(), lysosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-(3-carboxypropanoyl)-N-hydroxyputrescine(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-(3-carboxypropanoyl)-N-hydroxyputrescine(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dextran(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dextran(), lysosome, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ligand(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(precursor Z(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(4-(ethoxymethylene)-2-phenyloxazol-5-one(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(emeraldine macromolecule(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenyl isocyanate(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(malonaldehyde(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(antigen(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylinositol trisphosphate(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(poly-ADP-D-ribose(), nucleus, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ionomycin(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calicheamicin(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylcholine(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipoprotein(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(1,2-dihexadecanoyl-sn-glycero-3-phosphocholine(), None, lamellar body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(poly(uridylic acid)(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Leu-Thr(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Phe-Gly(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Arg-Val(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Gly-Asn(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(His-Ser(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ile-Ala(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Phe-Glu(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Val-Ile(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NS-5(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), nucleolus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PB1-F2(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ferritin(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(1-hexadecanoyl-2-docosanoyl-sn-glycerol(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RI(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polyamine(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), early endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(24,25-Dihydroxyvitamin D(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Melanin(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(single-stranded DNA(), None, cell periphery)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(single-stranded DNA(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S1(), virion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HEL(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACTN(), None, cytoskeleton)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADCY(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKT(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKT(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(APC_C(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AQP(), cytosol, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARRB(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_III(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_II(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_I(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAPN(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD8(), None, mitochondrial inner-outer membrane contact site)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CK2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNA(), None, nuclear envelope)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cathepsin(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chemokine(), extracellular matrix, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chemokine(), host cellular component, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Clathrin(), None, viral factory)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Clathrin(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cofilin(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DVL(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E3_Ub_ligase(), None, kinase activity)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E3_Ub_ligase(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4G(), None, viral replication complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENO(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERK(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERK(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERM(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETC_complex_II(), endoplasmic reticulum, Golgi apparatus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETC_complex_II(), endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FPR(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GEF(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPase(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPase(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_actin(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin(), None, food vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Histone(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Histone(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKB(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKK_complex(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), None, periplasmic space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JNK(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Kinesin(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, phagocytic vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MKNK(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFkappaB(mods: (modification)), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Na_K_ATPase(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Neuropeptides(), C-fiber, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P2RX(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAK(), cytosol, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3K(), None, kinase activity)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3K_p85(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PKC(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLC(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPAR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP3(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), lysosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB(), endosome, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RASGRF(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAS(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAS(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RLR(), cytosol, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNApo_II(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTK(), None, cell-cell junction)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RXR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT5(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAP(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TGFBR(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TGFB(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TXN(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tubulin(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tubulin(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Wnt(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mTORC1(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(p38(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA binding(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA-directed 5'-3' RNA polymerase activity(), None, viral replication complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coreceptor activity(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(autotransporter activity(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(secretory granule(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NADH dehydrogenase complex(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(macrophage apoptotic process(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mitochondrial outer membrane permeabilization(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHO(), None, transport vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIPK1(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIPK2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNASEL(), cytoskeleton, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL11(), None, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL22(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(mods: (modification)), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), None, phagocytic vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), aggresome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL5(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL9(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPS6KA3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTN2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BID(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SAR1A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCL1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BLVRA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CX3CL1(), cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SDC1(), None, glycocalyx)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SDC2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SDCBP(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SEL1L(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SELP(), membrane, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SELP(), vesicle, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFPQ(), None, paraspeckles)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRSF1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRSF5(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFTPB(), None, multivesicular body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFTPC(), None, multivesicular body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BNIP3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIRT2(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BRAP(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A1(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), cytoplasm, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), membrane, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC9A3(), None, microvillus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD1(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BSG(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SOAT1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SOD3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, cell surface)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), host cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPARC(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPARC(), endosome, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPHK1(), None, phagocytic vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPRY1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRC(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SREBF2(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SREBF2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRPK1(), None, inclusion body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BTN1A1(), None, lipid droplet)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT3(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT4(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT6(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STK17B(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STX17(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SYT1(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SYT1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TALDO1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAZ(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBPL1(), nucleus, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TCF4(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZEB1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TCP1(), nucleus, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TERT(), None, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TF(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TF(), early endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFAM(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFAP2A(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFE3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEB(mods: (phosphorylation, False)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEB(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEC(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFRC(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIA1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TJP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR2(), None, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR4(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR4(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNF(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFRSF1A(), None, extracellular exosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD40(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFSF13B(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, aggresome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53BP2(), cell-cell junction, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSP90B1(), None, extracellular matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRADD(), cytoplasm, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF2(), None, membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF2(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), None, endosome membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF5(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF6(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF6(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIP11(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACP2(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TYK2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2L6(mods: (modification)), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2L6(mods: (modification)), None, ruffle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UGCG(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ULK1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ULK1(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USP10(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USP7(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VCP(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VDR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VDR(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, ruffle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), cytosol, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS41(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VWF(), Weibel-Palade body, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XBP1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XDH(), None, lipid droplet)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XPO1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YY1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZAP70(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM25(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM26(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF112(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CNBP(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZFYVE1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), None, extracellular exosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAK2(), cytoplasm, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARL6(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C4BPA(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GABARAPL2(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GABARAPL2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3B(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C6(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3IP2(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS4A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ASZ1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, early endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), cell surface, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADIPOQ(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CA2(), cytosol, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAG6(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HDAC6(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HTRA2(), mitochondrial intermembrane space, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP1R15A(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKE(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKE(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), None, cytosol)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PINK1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAMP(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CANX(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAPN2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SNX9(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SNX17(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP1(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP3(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAV1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAV2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GOLPH3(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MBTPS1(), None, extracellular matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZNF330(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZNF330(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPINK5(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL37(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBAS(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR7(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR9(), None, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GEMIN4(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNB1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCND3(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNH(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNH(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TSG101(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TSG101(), endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM63(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL33(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHMP4B(), None, organelle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZBP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YAP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD19(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM22(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CARD9(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBPL1A(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, outer membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, protein-containing complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRC4(), None, protein-containing complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB3IP(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD28(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PYCARD(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRDX6(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD4(), None, mitochondrial inner-outer membrane contact site)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD4(), trans-Golgi network, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SH2D3A(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CIB1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, peroxisome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, viral replication complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELL2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD8A(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD9(), None, synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OPTN(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OPTN(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIRAP(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIRAP(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDC14B(), nucleus, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDC25C(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX23(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(APOBEC3G(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARID5A(), nucleus, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COP1(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ESAM(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDH5(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SH3RF1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTBP2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NECTIN3(), None, cell-cell junction)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XPO5(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDK2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDK4(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFRSF13C(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1A(), None, replication compartment)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1B(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1B(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN2A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DROSHA(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CUZD1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NTM(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOSC3(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, late endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI2(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TICAM1(), None, late endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CEBPB(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI4(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI4(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HAVCR2(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIWIL4(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZDHHC3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCSTAMP(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LRRK2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SUGP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, host cell cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), virion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPP5(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CFTR(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFIH1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFIH1(), None, inclusion body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM10(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EDEM1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EDEM1(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCLK2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ALYREF(), cytoplasm, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX58(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX58(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNIP2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBC1D5(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHD3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TERF2IP(), nucleus, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN2L(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), Golgi apparatus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), cell surface, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AAK1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CNKSR1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHUK(), None, mitochondrion)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHUK(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CIRBP(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBP5(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBCB(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(mods: (modification)), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrial intermembrane space, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrial membrane, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrion, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCSK9(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SNX27(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RGCC(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS39(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS39(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIMP2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(WDFY3(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(WRNIP1(), None, replication fork)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG16L1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG16L1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DIABLO(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FIS1(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TMUB1(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CERT1(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CERT1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COPG2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COPG2(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP11(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CP(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC10A3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC10A7(), host cell, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTRAF(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KEAP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CREBBP(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POLDIP3(), Golgi apparatus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CRYAB(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BMF(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YBX3(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LARP4(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CSRP3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOC2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOC2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NMRAL1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YOD1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTLA4(), vesicle, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSB(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSD(), lysosomal lumen, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ANKRD27(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSL(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSS(), trans-Golgi network, late endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYBB(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AFAP1L2(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AMBRA1(), None, focal adhesion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AMBRA1(), cytoskeleton, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MLKL(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DAG1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DAXX(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PARP1(), mitochondrion, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDIT3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX1(), None, cytoplasm)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX17(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX21(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX5(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, endoplasmic reticulum-Golgi intermediate compartment)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), endoplasmic reticulum, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBCEL(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SGMS2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DGKZ(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DGKZ(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MLEC(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GRK2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEKHM1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIP5K1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIP5K1(), mitochondrial membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DLL4(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DTX4(), None, centrosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, peroxisome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), mitochondrial membrane, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VMP1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNASE1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRGM(), None, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNASE1L3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDPN(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPEG1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZCRB1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM1L(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM2(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM2(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRDMT1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFASC(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GNL3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DPEP1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEKHA8(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PBRM1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DPP4(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA6(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RILP(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G3BP2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB2(), mitochondrion, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SCRIB(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SCRN2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPECC1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SCAP(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XRN1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HUWE1(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USO1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFT20(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRSS57(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEA1(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EED(), None, chromosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEF1A1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEF2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGF(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGF(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGFR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGFR(), None, plasma membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGR1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2AK3(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AGO2(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2S1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2S1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), None, viral replication complex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4G2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SERPINB1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELAVL1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELK1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADGRE2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADGRE2(), None, ruffle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTTN(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTTN(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AGT(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENDOG(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENDOG(), mitochondrial intermembrane space, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENPP1(), None, extracellular matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CARD16(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EP300(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EPHX1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ESR1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AHR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETV6(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EZH2(), None, chromosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIF1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F3(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIP(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FBL(), nucleolus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIRE(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FCGR3B(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FGF4(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FGF9(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FKBP1A(), None, centrosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2QL1(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FLNA(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOS(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO3(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKR1B10(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKR1B10(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERVK-21(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MTOR(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FUT2(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FUT2(), None, hyphal tip)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ISG15(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G6PC1(), None, endoplasmic reticulum membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAK(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAPDH(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBF1(), Golgi apparatus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBF1(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GC(), endoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GHR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GHR(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CBLIF(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GJA1(), None, cell surface)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GJA1(), intercalated disc, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GLI1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOP53(), None, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOP53(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZBED1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAP2(), cytoplasm, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CXCL1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CXCL2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA3(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPBP1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GUSB(), host cell, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GYPC(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GZMB(), endosome, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HAND1(), None, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KALRN(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HBZ(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HGF(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HGS(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1A(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1A(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), host cell, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), nucleus, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMG20A(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HNRNPK(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HPSE(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HPSE(), host cell, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSF1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA1A(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, extracellular matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, viral factory)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPB3(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPD1(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNAJB1(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HYAL2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOD2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ANXA1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ICAM1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI16(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI16(), nucleus, multivesicular body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI30(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFITM1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFITM3(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNAR1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNG(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNG(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNGR2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG12(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG12(), None, phagophore assembly site)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, phagophore assembly site)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IK(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKB(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL17A(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL2RA(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL4(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL4(), extracellular exosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ILF2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INPP5D(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INS(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INS(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INSR(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRAK1(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRAK1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF3(mods: (phosphorylation), (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF3(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF5(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF7(mods: (phosphorylation), (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF7(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF9(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ITGA5(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ITGB2(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ITPR1(), endoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAG1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAG2(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAK1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAK2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JUN(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AQP4(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LAMP1(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LCK(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF1(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF1(), None, early endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS1(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, endolysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS8(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS9(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF5(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN1(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHOA(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHOC(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), endosome, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBR1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MARCKS(), plasma membrane, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD2(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD4(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZFYVE9(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAL(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MALT1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP2K7(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K5(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K7(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAPK8(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAPK9(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARL1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD46(), None, membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD46(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MDK(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MDM2(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARNT(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MICA(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MARCKSL1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MME(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP3(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP7(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP9(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP9(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MOV10(), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MT-ATP6(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MT-RNR2(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MVP(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MVP(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYD88(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYD88(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), rough endoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBN(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCAM1(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCL(), nucleolus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NEU1(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFATC1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFATC2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L2(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L2(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKBIA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NMI(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF4(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOS1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOS3(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOTCH1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPM1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPM1(), None, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(), None, viral factory)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR1H4(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NRF1(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YBX1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUMA1(), None, centrosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NXF1(), None, inclusion body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NXF1(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIGMAR1(), None, plasma membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBP(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBP(), None, replication compartment)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSMR(), None, mitochondrial matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OXT(), dendrite, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P4HB(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P4HB(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PABPC1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FURIN(), None, extracellular matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAK1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKN(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKN(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAWR(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCBP2(), nucleus, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP7A(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP7A(), trans-Golgi network, melanosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCSK7(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrial intermembrane space, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrion, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrion, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDK1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDPK1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATR(), None, replication fork)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CFP(), membrane, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SERPINA1(), None, vesicle)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIK3CG(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI4KA(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI4KB(), None, Golgi membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIN1(), None, nuclear body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PKM(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLA2G4A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLA2G4A(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLAUR(), membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLCG2(), cytosol, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEK(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLG(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLSCR1(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLXNB1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PML(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PML(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POLK(), None, replication fork)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIC(), None, extracellular space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPID(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP1CA(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKCA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OLIG2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKCD(), cytosol, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAG3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2AK2(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRNP(), None, extracellular exosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGMN(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAK1(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAK1(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PSTPIP2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTBP1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTEN(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTK2B(), cytosol, mitochondrial matrix)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTPN11(), None, membrane raft)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAVIN1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PEX19(), peroxisome, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PXN(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PEX5(), peroxisome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR1(mods: (modification)), None, cell cortex)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR3(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB1A(), vacuole, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, recycling endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB5A(), late endosome, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAC1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAF1(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAF1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCHE(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCHE(), rough endoplasmic reticulum, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RASAL2(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RB1(), nucleoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBM4(), None, nucleoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBM4(), nucleolus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBFOX2(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBFOX2(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCL2A1(), None, extracellular exosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCL2L11(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(REL(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), None, host)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELB(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RGN(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(anthrax toxin(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein 41(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coronin(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Noonan syndrome 3(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Noonan syndrome 3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cyclophilin D(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Angiotensin-Converting Enzyme 2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SARS-CoV-2(), early endosome, endolysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Alkaline Phosphatase(), None, periplasmic space)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Amnion(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Antigens, Surface(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Capsid(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Capsid(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chlamydia trachomatis(), endosome, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dermatomyositis(), lysosome, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), None, apical plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), rough endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), trans-Golgi network, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hydrogen-Ion Concentration(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ion Channels(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Lipids(), vesicle, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Multiple Sclerosis(), None, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Neuraminidase(), host cellular component, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(mods: (phosphorylation)), None, virion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Oxidoreductases(), endoplasmic reticulum, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protein-Tyrosine Kinases(), None, focal adhesion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Receptors, Antigen, B-Cell(), None, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ribonucleoproteins(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNA-Directed RNA Polymerases(), None, membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Zinc(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Gene Products, tax(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA-Binding Proteins(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Computer-Aided Design(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Viral Nonstructural Proteins(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Guide(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Integrases(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Integrases(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleocapsid Proteins(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Guanine Nucleotide Exchange Factors(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tacrolimus Binding Proteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Small Interfering(), endosome, endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Lectins(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Galectins(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon Regulatory Factor-3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Basic-Leucine Zipper Transcription Factors(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Catenins(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Syntenins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Long Noncoding(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Long Noncoding(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Extracellular Traps(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(orf(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cpyA(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lptD(), periplasmic space, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(washc1(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pgk(), None, chloroplast)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ipaH9.8(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIFamide-related peptide(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG8(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG8(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(egfp(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(US20(), None, lysosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP(), None, multivesicular body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cas9(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Extracellular deoxyribonuclease(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(yopH(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rsgA(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(yopS(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3B(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB1(), cytosol, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB1(), nucleus, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(icd(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cdu1(), None, host cell)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neo(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin subunit beta-(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin subunit beta-(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tax(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tax(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF3b(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, plasma membrane region)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tat(), nucleoplasm, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpx(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpx(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpu(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UL83(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cre(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(apxIIC(), mitochondrion, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acrA(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nfrB(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), cytoplasm, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), cytoplasm, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ken-106(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ken-106(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3a(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(M(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(M(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rep(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gC(), None, host cell nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gC(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(term(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA1(), None, mitochondrial membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Large T antigen(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rep(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hifA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lef(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gag-pol(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P39(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POL(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POL(), None, nuclear body)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(impB(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nit-2(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(4b(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP2(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), cytosol, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ba71V-081(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AN(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XVI(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FASN(), None, replication compartment)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(manC(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cya(), endosome, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Probable integrase/recombinase(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Uncharacterized 9.8 kDa protein(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tsr(), None, outer membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIR3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(B23(), None, nucleolus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFN(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP3(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(smo-1(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(atp2b2(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERV-H1 provirus ancestral Env polyprotein(), None, cell surface)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mtb(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mtb(), host cellular component, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VII(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Stim(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Putative leucine-rich repeat protein PS14(), nucleus, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cyaA(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dif(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(U3(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP3(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP3(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ipaB(), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAL83(), cytoplasm, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(eaeB(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(X(), None, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arm(), None, X chromosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arm(), None, chromosome, centromeric region)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EBNA1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PA(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATJ1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERS1(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARAC3(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ankrd27(), None, endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(staB(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C(), None, endoplasmic reticulum lumen)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(43(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SEC61(), None, vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FPR1(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DOA4(), None, late endosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(crt(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dlg1(), None, immunological synapse)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(chmp4(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nicD(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), plasma membrane, None)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S7(), nucleus, chromosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Trak2(), cytoplasm, mitochondrion)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG9(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HDA5(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dlt(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DEG15(), None, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KRS(), None, extracellular exosome)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Zcrb1(), nucleus, cytoplasm)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sseJ(), None, cytosol)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF4(), None, host cellular component)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(thrb(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Iigp1(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Iigp1(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FAS2(), cytosol, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ctxB(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PERK12(), None, nucleus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, Golgi apparatus)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, rough endoplasmic reticulum)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP5(), None, membrane)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRF(mods: (glycosylation)), None, secretory granule)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP7(muts: (None, None, None)), None, cell surface)\n",
      "INFO: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(transcriptional activator(mods: (modification)), None, nucleus)\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15841 ! polypeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73888 ! Gly-Asn), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D005729 ! Ganglioneuroma), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"transmembrane protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74823 ! Ser-Val), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0036407\" ! \"mycolate outer membrane\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036022 ! \"Capsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015513 ! \"Oncogene Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ARF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 3}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D014166 ! Transferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74072 ! Ile-Leu), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ARF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2N(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP1(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 209}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D048068 ! PPAR-beta), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:USP33c), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2-S protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM52(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK12(mods: (modification), muts: (G, 731, R))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ASCC3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CHD6), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CHD8), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:DICER1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HELLS), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HELQ), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HELZ2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HLTF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SKIV2L), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D020365 ! \"RNA Helicases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D025521 ! \"Tumor Suppressor Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75012 ! Val-Ile), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75012 ! Val-Ile), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030964\" ! \"NADH dehydrogenase complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-07-16 15:48:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    }
   ],
   "source": [
    "emmaa_covid_graph = pybel.from_emmaa('covid19')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "#pybel.to_pickle(emmaa_covid_graph, 'covid19.pickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "emmaa_covid_graph = pybel.from_pickle('covid19.pickle.gz')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------  ------------------------------------\n",
      "Name                   covid19\n",
      "Version                531201a1-697b-4709-bb54-4d094ac31c8f\n",
      "Author                 INDRA\n",
      "Number of Nodes        100263\n",
      "Number of Namespaces   11\n",
      "Number of Edges        802126\n",
      "Number of Annotations  10\n",
      "Number of Citations    114385\n",
      "Number of Authors      0\n",
      "Network Density        7.98E-05\n",
      "Number of Components   742\n",
      "Number of Warnings     0\n",
      "---------------------  ------------------------------------\n",
      "\n",
      "Type (5)             Count  Example\n",
      "-----------------  -------  -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
      "Complex              48834  complex(p(HGNC:2578 ! CYBB), p(HGNC:636 ! AQP3))\n",
      "Protein              21213  p(HGNC:6448 ! KRT31)\n",
      "Abundance            18147  a(TEXT:\"MNK1/2\")\n",
      "BiologicalProcess    11945  bp(MESH:C414354 ! selamectin)\n",
      "Reaction               124  rxn(reactants(a(CHEBI:131170 ! \"(3S)-3-amino-4-[[(2S)-1-[[(2S)-1-[[(2S)-1-[[(2S,3S)-1-[[(2S)-1-[(2S)-2-[[[(1S)-1-carboxy-2-phenylethyl]amino]-oxomethyl]-1-pyrrolidinyl]-3-(1H-imidazol-5-yl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxopentan-2-yl]amino]-3-(4-hydroxyphenyl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxobutan-2-yl]amino]-5-(diaminomethylideneamino)-1-oxopentan-2-yl]amino]-4-oxobutanoic acid\")), products(a(CHEBI:48433 ! angiotensin)))\n",
      "\n",
      "Namespace (11)      Count  Example\n",
      "----------------  -------  ------------------------------------------------------\n",
      "HGNC                68370  p(HGNC:15805 ! GCNA)\n",
      "TEXT                21175  a(TEXT:\"NHMRC Senior 124 Research Fellowship 1155678\")\n",
      "UP                  18162  p(UP:Q05047 ! Q05047)\n",
      "CHEBI               17428  a(CHEBI:81293 ! \"Vinyl ether\")\n",
      "MESH                16473  bp(MESH:D003047 ! Coccidioidomycosis)\n",
      "FPLX                 7676  p(FPLX:\"Creatine_kinase\" ! \"Creatine_kinase\")\n",
      "PFAM                 3734  p(PFAM:PF04279 ! ISPA)\n",
      "GO                   3662  bp(GO:\"GO:0051326\" ! telophase)\n",
      "go                   3148\n",
      "IP                    338  p(IP:IPR008653)\n",
      "PUBCHEM               122  a(PUBCHEM:53481573 ! 53481573)\n",
      "\n",
      "Edge Type (61)                                           Count  Example\n",
      "-----------------------------------------------------  -------  -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
      "Abundance decreases Protein                              94631  a(CHEBI:4917 ! eugenol) decreases p(FPLX:p38 ! p38)\n",
      "Abundance increases Protein                              88917  a(CHEBI:30366 ! imidazolide) increases p(HGNC:2191 ! COL14A1)\n",
      "Protein partOf Complex                                   76189  p(UP:P79741 ! P79741) partOf complex(p(HGNC:8848 ! PES1), p(UP:P79741 ! P79741))\n",
      "BiologicalProcess increases BiologicalProcess            66707  bp(MESH:D002780 ! \"Cholestasis, Intrahepatic\") increases bp(MESH:D002779 ! Cholestasis)\n",
      "Protein increases Protein                                64880  p(UP:O00370 ! O00370) increases p(HGNC:27962 ! STING1)\n",
      "Abundance decreases BiologicalProcess                    60063  a(CHEBI:3347 ! candesartan) decreases bp(MESH:D007031 ! Hypothalamus)\n",
      "Protein increases BiologicalProcess                      51694  p(HGNC:4602 ! CXCL1) increases bp(MESH:D009000 ! Monocytes)\n",
      "Abundance increases BiologicalProcess                    37576  a(CHEBI:16412 ! lipopolysaccharide) increases bp(GO:\"GO:0005623\" ! cell)\n",
      "Protein decreases Protein                                30108  p(FPLX:\"F_actin\" ! \"F_actin\") decreases p(HGNC:1736 ! CDC42)\n",
      "Abundance directlyDecreases Protein                      27709  a(CHEBI:50924 ! sorafenib) directlyDecreases p(HGNC:6873 ! MAPK11)\n",
      "BiologicalProcess decreases BiologicalProcess            25026  bp(MESH:D013010 ! Sonication) decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "Protein decreases BiologicalProcess                      24629  p(HGNC:2432 ! CSF1) decreases bp(MESH:D016256 ! \"Lewis X Antigen\")\n",
      "Abundance partOf Complex                                 20295  a(CHEBI:26708 ! \"sodium atom\") partOf complex(a(CHEBI:26708 ! \"sodium atom\"), a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"))\n",
      "Abundance decreases Abundance                            19761  a(CHEBI:17514 ! cyanide) decreases a(CHEBI:35143 ! hemoglobin)\n",
      "BiologicalProcess increases Protein                      18839  bp(MESH:D007239 ! Infections) increases p(UP:Q9PTJ6 ! Q9PTJ6)\n",
      "Abundance increases Abundance                            18352  a(CHEBI:45409 ! ritonavir) increases a(CHEBI:8212 ! pimozide)\n",
      "Protein increases Abundance                              17898  p(UP:O00370 ! O00370) increases a(CHEBI:26020 ! phosphate)\n",
      "BiologicalProcess increases Abundance                    13239  bp(MESH:D000256 ! Adenoviridae) increases a(TEXT:\"shipyard eye\")\n",
      "Protein decreases Abundance                               8284  p(HGNC:939 ! BAG3) decreases a(TEXT:SD)\n",
      "BiologicalProcess decreases Protein                       8042  bp(MESH:D005334 ! Fever) decreases p(FPLX:LDH ! LDH)\n",
      "BiologicalProcess partOf Complex                          8030  bp(MESH:D009504 ! Neutrophils) partOf complex(bp(MESH:D003713 ! \"Dendritic Cells\"), bp(MESH:D009504 ! Neutrophils))\n",
      "BiologicalProcess decreases Abundance                     5478  bp(MESH:D000740 ! Anemia) decreases a(CHEBI:73817 ! Arg-Met)\n",
      "Protein directlyIncreases Protein                         5209  p(HGNC:6840 ! MAP2K1) directlyIncreases p(FPLX:ERK ! ERK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Protein hasVariant Protein                                3663  p(HGNC:7979 ! NR3C2) hasVariant p(HGNC:7979 ! NR3C2, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Complex increases Protein                                 1178  complex(p(HGNC:12630 ! USP7), p(UP:Q1HVF7 ! Q1HVF7)) increases p(HGNC:11998 ! TP53)\n",
      "Protein directlyDecreases Protein                          837  p(HGNC:12626 ! USP3) directlyDecreases p(HGNC:19102 ! DDX58, pmod(go:0016567 ! \"protein ubiquitination\"))\n",
      "Complex increases BiologicalProcess                        787  complex(p(HGNC:5172 ! HR), p(HGNC:6037 ! ILF2)) increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Abundance directlyIncreases Protein                        699  a(TEXT:EC) directlyIncreases p(HGNC:5344 ! ICAM1)\n",
      "Complex decreases Protein                                  694  complex(a(CHEBI:15939 ! \"glycyrrhizinic acid\"), p(HGNC:4983 ! HMGB1)) decreases p(HGNC:4983 ! HMGB1)\n",
      "Protein directlyIncreases Abundance                        503  p(HGNC:3009 ! DPP4) directlyIncreases a(CHEBI:26536 ! \"retinoic acid\")\n",
      "Complex decreases BiologicalProcess                        376  complex(p(HGNC:12033 ! TRAF3), p(HGNC:22938 ! NLRP12)) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Abundance directlyIncreases Abundance                      287  a(CHEBI:18397 ! 2-phenylethylamine) directlyIncreases a(CHEBI:15949 ! diphthamide)\n",
      "Complex increases Abundance                                240  complex(p(HGNC:5413 ! IFITM2), p(HGNC:5414 ! IFITM3)) increases a(CHEBI:16113 ! cholesterol)\n",
      "Protein directlyIncreases Reaction                         169  p(UP:P07819 ! P07819) directlyIncreases rxn(reactants(a(CHEBI:17992 ! sucrose)), products(a(CHEBI:17234 ! glucose)))\n",
      "BiologicalProcess directlyIncreases Protein                163  bp(MESH:D000650 ! Amnion) directlyIncreases p(FPLX:Interferon ! Interferon)\n",
      "Complex decreases Abundance                                150  complex(a(CHEBI:74056 ! His-Ser), p(HGNC:775 ! SERPINC1)) decreases a(CHEBI:3783 ! \"Coagulation Factor Xa\")\n",
      "Abundance directlyDecreases BiologicalProcess              123  a(CHEBI:61049 ! \"tacrolimus (anhydrous)\") directlyDecreases bp(MESH:D022021 ! \"Tacrolimus Binding Proteins\")\n",
      "Abundance directlyDecreases Abundance                      109  a(CHEBI:52172 ! nilotinib) directlyDecreases a(TEXT:BCR-ABL)\n",
      "Reaction hasReactant Abundance                              99  rxn(reactants(a(CHEBI:27300 ! \"vitamin D\")), products(a(CHEBI:25212 ! metabolite))) hasReactant a(CHEBI:27300 ! \"vitamin D\")\n",
      "Reaction hasProduct Abundance                               98  rxn(reactants(a(CHEBI:15554 ! \"prostaglandin H2\")), products(a(CHEBI:15551 ! \"prostaglandin E2\"))) hasProduct a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "BiologicalProcess directlyDecreases Protein                 90  bp(MESH:C520962 ! obatoclax) directlyDecreases p(HGNC:990 ! BCL2)\n",
      "Protein directlyIncreases BiologicalProcess                 63  p(HGNC:9360 ! PRF1) directlyIncreases bp(GO:\"GO:0003723\" ! \"RNA binding\")\n",
      "Protein directlyDecreases Abundance                         61  p(UP:P94548 ! P94548) directlyDecreases a(CHEBI:17877 ! FADH2)\n",
      "Abundance directlyIncreases BiologicalProcess               31  a(CHEBI:34682 ! diazinon) directlyIncreases bp(MESH:D051379 ! Mice)\n",
      "Reaction hasReactant Protein                                25  rxn(reactants(p(FPLX:JNK ! JNK)), products(p(HGNC:11892 ! TNF))) hasReactant p(FPLX:JNK ! JNK)\n",
      "Reaction hasProduct Protein                                 23  rxn(reactants(a(CHEBI:17026 ! progesterone)), products(p(HGNC:385 ! AKR1C2))) hasProduct p(HGNC:385 ! AKR1C2)\n",
      "Abundance directlyIncreases Reaction                        20  a(TEXT:ARF) directlyIncreases rxn(reactants(a(CHEBI:15354 ! choline)), products(a(CHEBI:16337 ! \"phosphatidic acid\")))\n",
      "BiologicalProcess directlyIncreases Abundance               20  bp(MESH:C015378 ! nas) directlyIncreases a(CHEBI:26667 ! \"sialic acid\")\n",
      "BiologicalProcess directlyIncreases BiologicalProcess       18  bp(MESH:D011494 ! \"Protein Kinases\") directlyIncreases bp(MESH:D011494 ! \"Protein Kinases\")\n",
      "BiologicalProcess directlyIncreases Reaction                12  bp(MESH:D005341 ! Fibrinolysin) directlyIncreases rxn(reactants(p(FPLX:MMP ! MMP)), products(p(FPLX:MMP ! MMP)))\n",
      "BiologicalProcess directlyDecreases Abundance               10  bp(MESH:D006005 ! Phosphorylases) directlyDecreases a(CHEBI:26020 ! phosphate)\n",
      "Abundance directlyDecreases Complex                          7  a(CHEBI:65329 ! LY294002) directlyDecreases complex(p(HGNC:8975 ! PIK3CA), p(HGNC:8979 ! PIK3R1))\n",
      "Protein directlyDecreases BiologicalProcess                  4  p(HGNC:9277 ! PPM1D) directlyDecreases bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\")\n",
      "Reaction hasProduct BiologicalProcess                        3  rxn(reactants(p(HGNC:29233 ! MAVS)), products(bp(MESH:D011328 ! Prions))) hasProduct bp(MESH:D011328 ! Prions)\n",
      "Complex partOf Complex                                       2  complex(a(TEXT:DOMAIN), bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")) partOf complex(complex(a(TEXT:DOMAIN), bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")), p(HGNC:13557 ! ACE2))\n",
      "Abundance decreases Complex                                  1  a(CHEBI:16856 ! glutathione) decreases complex(a(TEXT:Gclc), a(TEXT:Gclm))\n",
      "BiologicalProcess directlyDecreases BiologicalProcess        1  bp(GO:\"GO:0016021\" ! \"integral component of membrane\") directlyDecreases bp(GO:\"GO:0007602\" ! phototransduction)\n",
      "Complex directlyIncreases Protein                            1  complex(p(HGNC:11362 ! STAT1), p(HGNC:6893 ! MAPT)) directlyIncreases p(HGNC:933 ! BACE1)\n",
      "Protein decreases Complex                                    1  p(HGNC:12397 ! TTF1) decreases complex(p(HGNC:6769 ! SMAD3), p(HGNC:6770 ! SMAD4))\n",
      "Complex directlyDecreases Protein                            1  complex(p(HGNC:2347 ! CREB3), p(UP:P04608 ! P04608)) directlyDecreases p(UP:P04608 ! P04608)\n",
      "Reaction hasReactant BiologicalProcess                       1  rxn(reactants(bp(GO:\"GO:0003823\" ! \"antigen binding\")), products(a(TEXT:SPIKE-PROTEIN))) hasReactant bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "\n"
     ]
    }
   ],
   "source": [
    "emmaa_covid_graph.summarize()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Find all INDRA statements associated with a publication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_statements_by_pubmed( pubmed, emmaa_covid_graph=emmaa_covid_graph ):\n",
    "    statements = []\n",
    "    for u, v, data in emmaa_covid_graph.edges(data=True):\n",
    "        if  ('citation' in data) and (\n",
    "             data['citation']['namespace'] == 'pubmed') and (\n",
    "             data['citation']['identifier'] ==  pubmed):\n",
    "                #print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "                if 'evidence' in data:\n",
    "                    #print(data['citation']['namespace'], data['citation']['identifier'])\n",
    "                    #print(data['evidence'])\n",
    "                    statements.append([u,v,data])\n",
    "    return statements\n",
    "        "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Get statements by subject and object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_statements_by_subject_and_object( subject, obj, emmaa_covid_graph):\n",
    "    statements = []\n",
    "    for u, v, data in emmaa_covid_graph.edges(subject, data=True):\n",
    "        if (v in obj) or (u in obj):\n",
    "            #print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "            if 'evidence' in data:\n",
    "                statements.append([u,v,data])\n",
    "    return statements\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Print bibliography"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "import bibtexparser\n",
    "\n",
    "def print_bib( bibfile, graph=emmaa_covid_graph, pmid_graph = {}, pmid_indra = {}, english_statements = {} ):\n",
    "    pmids = get_pmids_of_bibtex(bibfile)\n",
    "    \n",
    "    return print_pmids( pmids, graph, pmid_graph, pmid_indra, english_statements )\n",
    "\n",
    "def get_pmids_of_bibtex(bibfile):\n",
    "    with open(bibfile) as bibtex_file:\n",
    "        bib_database = bibtexparser.load(bibtex_file)\n",
    "    pmids = dict([(entry['pmid'], entry['title'])\n",
    "                    for entry in bib_database.entries\n",
    "                     if 'pmid' in entry])\n",
    "    return pmids\n",
    "\n",
    "\n",
    "def print_pmids( pmids, graph, pmid_graph = {}, pmid_indra = {}, english_statements = {}):    \n",
    "    \n",
    "    for pmid in pmids:\n",
    "        display(Markdown(f'## [{pmids[pmid]}](https://pubmed.ncbi.nlm.nih.gov/{pmid})'))\n",
    "        if pmid not in pmid_graph:\n",
    "            pmid_graph[pmid] = get_subgraph_by_pubmed(graph, pmid)\n",
    "        if pmid not in pmid_indra:\n",
    "            pmid_indra[pmid] = pybel.to_indra_statements(pmid_graph[pmid])\n",
    "        if pmid not in english_statements:\n",
    "            pmid_english = EnglishAssembler(pmid_indra[pmid])\n",
    "            english_model = pmid_english.make_model()\n",
    "            english_statements[pmid] = english_model.split('.')\n",
    "        for i in range(len(pmid_indra[pmid])):\n",
    "            display(Markdown( f'### {english_statements[pmid][i]}\\n'))\n",
    "            for evidence in pmid_indra[pmid][i].evidence:\n",
    "                print(evidence.annotations['bel'])\n",
    "                display(Markdown(f\"{evidence.text}\"))\n",
    "    return pmid_graph, pmid_indra, english_statements\n",
    "\n",
    "def convert_to_bel(claims):\n",
    "    statements_local = []\n",
    "    indra_claims_local =  []\n",
    "    for claim in claims:\n",
    "        if len(claim['statements_local']) >0:\n",
    "            pba = PybelAssembler(claim['statements_local'])\n",
    "            claim['bel_local'] = pba.make_model()\n",
    "            statements_local += claim['statements_local']\n",
    "            indra_claims_local.append(claim)\n",
    "    pba = PybelAssembler(statements_local)\n",
    "    belgraph_local = pba.make_model()\n",
    "    return claims, belgraph_local"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get causal claims from Logette et al 2021"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from indra.sources.reach.api import process_pmc\n",
    "from indra.sources import reach\n",
    "rp  = process_pmc('PMC8356061')\n",
    "logette_bel = PybelAssembler(rp.statements).make_model()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "\n",
       "    /**\n",
       "     * Gets the best name for a node object\n",
       "     * @param {object} d object\n",
       "     * @returns {str} canonical name of the node\n",
       "     */\n",
       "    function getCanonicalName(d) {\n",
       "        if (d.concept && d.concept.name && !(d.variants || d.reactants || d.products || d.members)) {\n",
       "            return d.concept.name\n",
       "        } else if (d.bel) {\n",
       "            return d.bel\n",
       "        } else {\n",
       "            console.log('Undefined node: ' + d);\n",
       "            return 'UNDEFINED'\n",
       "        }\n",
       "    }\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation', 'isA', 'partOf'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return getCanonicalName(d)\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='bkzjetxdromuaycf'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.15.0\", \"document_metadata\": {\"name\": \"indra\", \"version\": \"9453ecff-ad33-4d6f-bf56-8aad3a17fa95\"}, \"namespace_url\": {\"HGNC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/hgnc-human-genes/hgnc-human-genes-20170725.belns\", \"UP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/swissprot/swissprot-20170725.belns\", \"IP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/interpro/interpro-20170731.belns\", \"FPLX\": \"https://raw.githubusercontent.com/sorgerlab/famplex/5f5b573fe26d7405dbccb711ae8e5697b6a3ec7e/export/famplex.belns\", \"CHEBI\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/chebi-ids/chebi-ids-20170725.belns\", \"GO\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go/go-20180109.belns\", \"MESH\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-processes/mesh-processes-20170725.belns\"}, \"namespace_pattern\": {\"PUBCHEM\": \"\\\\d+\"}, \"annotation_url\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}, \"annotation_curie\": [], \"annotation_miriam\": []}, \"nodes\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"7alpha-hydroxycholest-4-en-3-one\", \"identifier\": \"17899\"}, \"id\": \"e1a2b20eb4243f62d62e589911840297\", \"bel\": \"a(CHEBI:17899 ! 7alpha-hydroxycholest-4-en-3-one)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"reactive oxygen species\", \"identifier\": \"26523\"}, \"id\": \"967db55abac7ab8dcfbcc0df2b4c4cd3\", \"bel\": \"a(CHEBI:26523 ! \\\"reactive oxygen species\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"sodium atom\", \"identifier\": \"26708\"}, \"id\": \"01aea6b60f55d7f4d7d27444906b99e9\", \"bel\": \"a(CHEBI:26708 ! \\\"sodium atom\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"2-hydroxy-17beta-estradiol\", \"identifier\": \"28744\"}, \"id\": \"4002bf8c077a5e011977904927e3d03a\", \"bel\": \"a(CHEBI:28744 ! 2-hydroxy-17beta-estradiol)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"sodium(1+)\", \"identifier\": \"29101\"}, \"id\": \"479c4a3e5ee7630eed155bb2a4ce675a\", \"bel\": \"a(CHEBI:29101 ! \\\"sodium(1+)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"metformin\", \"identifier\": \"6801\"}, \"id\": \"6f69bd15b40488139d8325b83eace84d\", \"bel\": \"a(CHEBI:6801 ! metformin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"2-deoxy-D-glucopyranose\", \"identifier\": \"84755\"}, \"id\": \"dc6f40db0b2a653f0bb5da67b10ebff3\", \"bel\": \"a(CHEBI:84755 ! 2-deoxy-D-glucopyranose)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"-defensins\"}, \"id\": \"1a35b585243afac9bd7cfb1ced27aee9\", \"bel\": \"a(TEXT:\\\"-defensins\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"\"}, \"id\": \"556e1a7dfce31fe6bcfef6f0eb657a51\", \"bel\": \"a(TEXT:\\\"\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"Cd147\"}, \"id\": \"7972f9ddb8e70d36e65303bcdf5edb70\", \"bel\": \"a(TEXT:Cd147)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"Glucose\"}, \"id\": \"07ee4fbc1e8da7de0d638d60f6933f39\", \"bel\": \"a(TEXT:Glucose)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"SARS-CoV-2\"}, \"id\": \"bafcec6e270dfd4135015311c9e521df\", \"bel\": \"a(TEXT:SARS-CoV-2)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"antigen\"}, \"id\": \"483ac78ab57419488e8bdf7019930150\", \"bel\": \"a(TEXT:antigen)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"glucose\"}, \"id\": \"0d134e8d451a5aba2be0af2b7bd88fea\", \"bel\": \"a(TEXT:glucose)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"hexokinases\"}, \"id\": \"1f4b7692edd45bcb647c15f328e554aa\", \"bel\": \"a(TEXT:hexokinases)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"lactoferrins\"}, \"id\": \"1fd97c29b2d8c40fff4852c68f516c56\", \"bel\": \"a(TEXT:lactoferrins)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"transport\", \"identifier\": \"GO:0006810\"}, \"id\": \"0f2bdc9751e0b639ef700fccd082c38c\", \"bel\": \"bp(GO:\\\"GO:0006810\\\" ! transport)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"endocytosis\", \"identifier\": \"GO:0006897\"}, \"id\": \"26d00e21f8ac9427236f2e151fad75d5\", \"bel\": \"bp(GO:\\\"GO:0006897\\\" ! endocytosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"inflammatory response\", \"identifier\": \"GO:0006954\"}, \"id\": \"9e3cd403f7bc833ae24bad3f9c2aad48\", \"bel\": \"bp(GO:\\\"GO:0006954\\\" ! \\\"inflammatory response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"immune response\", \"identifier\": \"GO:0006955\"}, \"id\": \"043b1a0262bca90484bcf6c89c1075f9\", \"bel\": \"bp(GO:\\\"GO:0006955\\\" ! \\\"immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"pH reduction\", \"identifier\": \"GO:0045851\"}, \"id\": \"bee490bd47e9c30dea2b457400d7d75e\", \"bel\": \"bp(GO:\\\"GO:0045851\\\" ! \\\"pH reduction\\\")\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"SARS-CoV-2\"}, \"id\": \"bafcec6e270dfd4135015311c9e521df\", \"bel\": \"a(TEXT:SARS-CoV-2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACE2\", \"identifier\": \"13557\"}, \"id\": \"97bb4990b7ce15318e3e41ec4b59055a\", \"bel\": \"p(HGNC:13557 ! ACE2)\"}], \"id\": \"713ebfe77bfabd804f2665d05770518a\", \"bel\": \"complex(a(TEXT:SARS-CoV-2), p(HGNC:13557 ! ACE2))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"HIF1\", \"identifier\": \"HIF1\"}, \"id\": \"5b96516373ba337dfe7941a3edda389c\", \"bel\": \"p(FPLX:HIF1 ! HIF1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Hemoglobin\", \"identifier\": \"Hemoglobin\"}, \"id\": \"7e5005aa2a5cecb5c8d4436e62d121fd\", \"bel\": \"p(FPLX:Hemoglobin ! Hemoglobin)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"LDH\", \"identifier\": \"LDH\"}, \"id\": \"aaef8c653c39cd1a241834d3564a67ae\", \"bel\": \"p(FPLX:LDH ! LDH)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"SLC2A\", \"identifier\": \"SLC2A\"}, \"id\": \"c826e2a2011dfa3cdc8a659db1f9e7f0\", \"bel\": \"p(FPLX:SLC2A ! SLC2A)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACE2\", \"identifier\": \"13557\"}, \"id\": \"97bb4990b7ce15318e3e41ec4b59055a\", \"bel\": \"p(HGNC:13557 ! ACE2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGER\", \"identifier\": \"320\"}, \"id\": \"46efcef764e041272dc7e8cae07361e3\", \"bel\": \"p(HGNC:320 ! AGER)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGT\", \"identifier\": \"333\"}, \"id\": \"1b7929e21a877956819f0408d99da72a\", \"bel\": \"p(HGNC:333 ! AGT)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"HSPA5\", \"identifier\": \"5238\"}, \"id\": \"003f66f7b6a0671bd3de67c7ad4acf59\", \"bel\": \"p(HGNC:5238 ! HSPA5)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"APCS\", \"identifier\": \"584\"}, \"id\": \"aa61a145188e5cd065a4a9d9adb2b578\", \"bel\": \"p(HGNC:584 ! APCS)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ASL\", \"identifier\": \"746\"}, \"id\": \"05c72d1482f0bd1e35bcc13ecf1602fe\", \"bel\": \"p(HGNC:746 ! ASL)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"OCLN\", \"identifier\": \"8104\"}, \"id\": \"6ce6d5287df2105d90ebe176a8dc4ab3\", \"bel\": \"p(HGNC:8104 ! OCLN)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PPY\", \"identifier\": \"9327\"}, \"id\": \"6ce73587589302e595725ef2ba56961e\", \"bel\": \"p(HGNC:9327 ! PPY)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PTPN5\", \"identifier\": \"9657\"}, \"id\": \"698e305e3ed455d0a055b171b396532e\", \"bel\": \"p(HGNC:9657 ! PTPN5)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"IP\", \"name\": \"IPR004724\"}, \"id\": \"6f588d73b93febe86e5f51c3f3342279\", \"bel\": \"p(IP:IPR004724)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"arm\", \"identifier\": \"PF00514\"}, \"id\": \"b268dde639b5990e949c37deeb17e434\", \"bel\": \"p(PFAM:PF00514 ! arm)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"death\", \"identifier\": \"PF00531\"}, \"id\": \"0d9f7065f60028ef0a1ea455e8aa31f8\", \"bel\": \"p(PFAM:PF00531 ! death)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"help\", \"identifier\": \"PF03451\"}, \"id\": \"979b7de392f8b6895ff09431afc6e525\", \"bel\": \"p(PFAM:PF03451 ! help)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"B0C5D4\", \"identifier\": \"B0C5D4\"}, \"id\": \"4742ec727a444b4111c4a87d92ad042e\", \"bel\": \"p(UP:B0C5D4 ! B0C5D4)\"}], \"links\": [{\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"-5952780407095246\": true}, \"uuid\": {\"ba860b98-6de8-4cbf-84dd-1bd6d5ce8d85\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-1159425984281032181\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Glucose normally moves through GLUTs by passive diffusion down a concentration gradient generated by the activity of hexokinases, which phosphorylate intracellular glucose to maintain a low-intracellular concentration of glucose (XREF_BIBR).\", \"source\": 14, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-10743108195344890\": true}, \"uuid\": {\"80276c5d-b7bd-4821-8060-085242ceb3c9\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"7779969998166127912\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"- Elevated blood glucose causes dysregulations in the immune response that facilitates the cytokine storm and acute respiratory distress syndrome (ARDS).\", \"source\": 13, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"29304616440516638\": true}, \"uuid\": {\"e6a33683-757a-4891-a9d3-7fb3192fe57a\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3052150177793613788\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Elevated glucose impairs the cellular immunity in the lung.\", \"source\": 13, \"target\": 19, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15521021061263896\": true}, \"uuid\": {\"0f10c042-8edc-418f-b6e6-57fa89be5304\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1898321142484090073\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"In contrast, transport via SGLT is driven by sodium (Na +) and glucose gradients.\", \"source\": 13, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2410657968820966\": true}, \"uuid\": {\"b73f3fe7-46d0-43fc-a9c6-9e82324026c1\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"7180187475530458854\": true}, \"source_api\": {\"reach\": true}}, \"target\": 15, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"In addition, glucose can indirectly impair -defensins and lactoferrins activities (see below on ASL acidosis).\", \"source\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-1375063654123433\": true}, \"uuid\": {\"77453078-94bd-47f4-8603-e735478d4e76\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"7180187475530458854\": true}, \"source_api\": {\"reach\": true}}, \"target\": 7, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"In addition, glucose can indirectly impair -defensins and lactoferrins activities (see below on ASL acidosis).\", \"source\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-335387520726909\": true}, \"uuid\": {\"70e8bd9f-2792-4419-acd0-e28f06807857\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-4296549616247048292\": true}, \"source_api\": {\"reach\": true}}, \"target\": 31, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Elevated glucose causes acidosis of the ASL.\", \"source\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-335387520726909\": true}, \"uuid\": {\"238acd7b-5ac7-47c8-9b16-f43fb97878a3\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"7473893337020240980\": true}, \"source_api\": {\"reach\": true}}, \"target\": 31, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"We have described how elevated blood glucose can elevate glucose in the ASL and cause acidification of the ASL.\", \"source\": 13, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21235625566682296\": true}, \"uuid\": {\"498dfff0-d32b-4e91-9783-28d05b79bd65\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"7473893337020240980\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"We have described how elevated blood glucose can elevate glucose in the ASL and cause acidification of the ASL.\", \"source\": 13, \"target\": 20, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-1204504242926817\": true}, \"uuid\": {\"13524ea0-f943-4cd6-912a-0ad31513fff2\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-5376618164973923079\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"This analysis is in agreement with a recent study showing that the use of the glucose analog 2-deoxy-D-glucose blocks SARS-CoV-2 replication in Caco-2 cells (XREF_BIBR).\", \"source\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11123275367647113\": true}, \"uuid\": {\"33dab5b2-0018-4b3d-a6a4-72095561754d\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"9191762319741960204\": true}, \"source_api\": {\"reach\": true}, \"hypothesis\": {\"true\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The model suggests that SARS-CoV-2 endocytosis in epithelial cells is substantially increased by high glucose in the ASL in all cases, but with the greatest effect seen for low viral loads (XREF_FIG, blue lines).\", \"source\": 13, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-29392172215236140\": true}, \"uuid\": {\"3296b06f-e1ea-4f20-8251-8366f23c1846\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"9191762319741960204\": true}, \"source_api\": {\"reach\": true}, \"hypothesis\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The model suggests that SARS-CoV-2 endocytosis in epithelial cells is substantially increased by high glucose in the ASL in all cases, but with the greatest effect seen for low viral loads (XREF_FIG, blue lines).\", \"source\": 13, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16980598554768934\": true}, \"uuid\": {\"03a92435-41a2-4468-9997-414396949c1d\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-1933300299424707080\": true}, \"source_api\": {\"reach\": true}}, \"target\": 9, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"This omission is serious because GRP78 is induced by glycemic stress and Cd147 is upregulated by high glucose levels and by AGEs.\", \"source\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"266707767526024\": true}, \"uuid\": {\"f3e7f978-8358-434e-a96f-1731f59cdff3\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7508555493978804116\": true}, \"source_api\": {\"reach\": true}}, \"target\": 34, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The result is overwhelming collective evidence that elevated blood glucose, arising from clinically or subclinical pathology in glucose metabolism or from induced hyperglycemia due to hospitalization, drug treatments and intravenous infusions in ICU should be considered as a biomarker that correlates with and hence is predictive of severity of COVID-19, as well as evidence that elevated glucose can cause an acceleration of virtually every step of the SARS-CoV2 infection.\", \"source\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11100139761953277\": true}, \"uuid\": {\"057de3f7-c0e7-442f-9012-5cbff6384890\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-5116302126960628530\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"We conclude that the literature strongly supports a case for compromised glucose metabolism that causes elevation of glucose levels in extracellular fluids, blood and tissue as a single pathology that can facilitate virtually every step in the life-cycle of SARS-CoV-2, and that induced elevations of glucose by stress during hospitalization, treatment drugs, and in intravenous infusions can contribute to disease severity.\", \"source\": 13, \"target\": 13, \"key\": 0}, {\"relation\": \"partOf\", \"source\": 11, \"target\": 21, \"key\": \"e72de15b9a3339e214847e21ebec073e\"}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34436642981397581\": true}, \"uuid\": {\"769f39dd-0044-4be2-9665-23d8abde2cc9\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-1204362635447449005\": true}, \"source_api\": {\"reach\": true}}, \"target\": 26, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"This exaggerated response is amplified by the SARS-CoV-2 specific inactivation of ACE2 with the resulting local increase in Ang II levels that further aggravates the pro inflammatory phenotype [IL-6 and ROS (XREF_BIBR, XREF_BIBR)].\", \"source\": 11, \"key\": 0}, {\"relation\": \"partOf\", \"source\": 26, \"target\": 21, \"key\": \"4d0cd70e599d6bbb0976c7f2cc543bc4\"}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31180847326022239\": true}, \"uuid\": {\"d17c8a6b-f624-4383-b956-9a34a82bb2f4\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-642272453073810470\": true}, \"source_api\": {\"reach\": true}}, \"target\": 37, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Finally, numerous more recent studies showed that increased FPG is associated with a poor prognosis and increased risk of death from COVID-19, whereas well controlled FPG is associated with a better outcome (XREF_TABLE).\", \"source\": 39, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2866288237065119\": true}, \"uuid\": {\"e5be5fb2-458b-4330-b21f-01b8dded49a8\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"4511858568942120107\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The transcellular transport of glucose is mediated by the facilitative transporter GLUT, expressed at the basolateral membranes, and by GLUT or SGLT1 (sodium-glucose linked transporter) at the apical membrane of the airway and alveoli, respectively (XREF_FIG left panel) and (XREF_BIBR, XREF_BIBR).\", \"source\": 25, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7974076909584955\": true}, \"uuid\": {\"fc85fc27-be73-49f2-a329-fc8f3672931b\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7114963574385319006\": true}, \"source_api\": {\"reach\": true}}, \"target\": 13, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Increased production of ROS caused by cellular damages and hypoxia induces expression of the HIF1  (hypoxia inducible factor 1 ) transcription factor (XREF_BIBR) stimulating the expression of GLUT transporters (XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR) and glycolytic genes (XREF_BIBR, XREF_BIBR), that increase extracellular glucose uptake and glycolytic capacity, amplifying the viral replication (XREF_BIBR  XREF_BIBR).\", \"source\": 25, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-22068788228855068\": true}, \"uuid\": {\"3b61e132-b1b1-49b3-b2ed-1c7812c2ec99\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1898321142484090073\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"In contrast, transport via SGLT is driven by sodium (Na +) and glucose gradients.\", \"source\": 2, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10668862026482131\": true}, \"uuid\": {\"7a7a6b4f-e6d2-4fd9-9237-576482909a05\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1898321142484090073\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"In contrast, transport via SGLT is driven by sodium (Na +) and glucose gradients.\", \"source\": 4, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5057774540857373\": true}, \"uuid\": {\"8ff9d298-a518-44e2-9d7d-435cba610d5f\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"5501678868746353716\": true}, \"source_api\": {\"reach\": true}}, \"target\": 36, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Glucose impairs the humoral arm of lung defenses.\", \"source\": 10, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34798665167207009\": true}, \"uuid\": {\"882c4f6c-12c4-4889-81fa-c37f6d5822dc\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3370546390040986621\": true}, \"source_api\": {\"reach\": true}}, \"target\": 31, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Elevated Glucose Impairs the Primary Non specific Defense of the ASL.\", \"source\": 10, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35404908903817540\": true}, \"uuid\": {\"c68b98b6-9302-43f2-aed3-198a441cc378\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3879724505714114720\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Elevated Glucose Drives the Immune Response Into a Cytokine Storm and ARDS.\", \"source\": 10, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26389739164539841\": true}, \"uuid\": {\"69de6659-7404-4a7b-aad8-7c6fd6561115\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7675604104199895021\": true}, \"source_api\": {\"reach\": true}}, \"target\": 36, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Interestingly, aging and hyperglycemia are also two conditions associated with a decreased number of alveolar macrophages and DCs, with altered function of antigen presenting cells (APCs) (XREF_BIBR  XREF_BIBR), significantly impairing the cellular arm of innate defense.\", \"source\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-2585085001838585\": true}, \"uuid\": {\"9aceae7d-d664-4845-93c0-8e86360d23a8\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7675604104199895021\": true}, \"source_api\": {\"reach\": true}}, \"target\": 36, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Interestingly, aging and hyperglycemia are also two conditions associated with a decreased number of alveolar macrophages and DCs, with altered function of antigen presenting cells (APCs) (XREF_BIBR  XREF_BIBR), significantly impairing the cellular arm of innate defense.\", \"source\": 30, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30272582690913145\": true}, \"uuid\": {\"2eb6e3a9-f9e9-4b65-b0ce-9138bb74d3af\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"8898887964457574878\": true}, \"source_api\": {\"reach\": true}}, \"target\": 33, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Secretion of lactate into airway secretions leads to an acidification of the ASL that inhibits numerous pH dependent antimicrobial agents, such as lysozyme, lactoferrin, -defensin, and LL-37 (XREF_BIBR, XREF_BIBR).\", \"source\": 31, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35842714335343752\": true}, \"uuid\": {\"6b304bef-129d-4e3b-a297-b69d02b39cf5\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"9191762319741960204\": true}, \"source_api\": {\"reach\": true}, \"hypothesis\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The model suggests that SARS-CoV-2 endocytosis in epithelial cells is substantially increased by high glucose in the ASL in all cases, but with the greatest effect seen for low viral loads (XREF_FIG, blue lines).\", \"source\": 31, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22204769849730862\": true}, \"uuid\": {\"6b0b0267-59c3-46c9-9371-c733a9cb7de5\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"9191762319741960204\": true}, \"source_api\": {\"reach\": true}, \"hypothesis\": {\"true\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"The model suggests that SARS-CoV-2 endocytosis in epithelial cells is substantially increased by high glucose in the ASL in all cases, but with the greatest effect seen for low viral loads (XREF_FIG, blue lines).\", \"source\": 31, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-34649905910603521\": true}, \"uuid\": {\"8e92c7b1-5b9f-4e7d-a316-9d6557829c6b\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7006232597205775387\": true}, \"source_api\": {\"reach\": true}}, \"target\": 31, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"However, the accompanying secretion of a HCO 3--rich fluid leads to an imbalance of ions and water, impacting the ASL osmolarity and volume, resulting in increased viscosity of the ASL fluid, diminished beating of the cilia, and reduced mucociliary clearance of waste and pathogens (XREF_BIBR, XREF_BIBR  XREF_BIBR)  direct consequences of hyperglycemia (XREF_BIBR, XREF_BIBR).\", \"source\": 0, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26628157619397592\": true}, \"uuid\": {\"f7fa316b-d359-4674-9cf1-7278390fc6b4\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"3950816178965467840\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"AECII cells also secrete more surfactant proteins to further amplify the local innate defense and help drive the resolution of the inflammation through their immunomodulatory activity and their capacity to stimulate phagocytosis of apoptotic cells (XREF_BIBR, XREF_BIBR).\", \"source\": 38, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11365186275549982\": true}, \"uuid\": {\"967dbabc-1d84-414e-b6de-79da94dbf025\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-5376618164973923079\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"This analysis is in agreement with a recent study showing that the use of the glucose analog 2-deoxy-D-glucose blocks SARS-CoV-2 replication in Caco-2 cells (XREF_BIBR).\", \"source\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28600912019457897\": true}, \"uuid\": {\"51977f6f-e95c-46d4-b747-e52da543ea4b\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1266321137369565644\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Under normal physiological conditions, quiescent endothelial cells (ECs) preserve vascular homeostasis, ensure barrier integrity and function, prevent inflammation, and inhibit coagulation by expressing blood clot lysing enzymes and producing the glycocalyx, a protective layer of glycoproteins and glycolipids with anticoagulant properties.\", \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12376824974242612\": true}, \"uuid\": {\"74ad87aa-8d06-4437-81ee-e84311355ea0\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"863710203497503794\": true}, \"source_api\": {\"reach\": true}}, \"target\": 32, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Firstly, metformin is known to restore the permeability function of tight junctions by increasing the transepithelial electrical resistance as well as the expression of the tight junction proteins claudin 1 and occludin (XREF_BIBR, XREF_BIBR).\", \"source\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"3509601274547776\": true}, \"uuid\": {\"7d280ff3-0a2d-4090-b607-c8df48f11de5\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"2539741572220964406\": true}, \"source_api\": {\"reach\": true}}, \"target\": 23, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Thirdly, metformin decreases the glycation of hemoglobin and presents cardiovascular protective, vasodilative and anti-thrombotic effects (XREF_BIBR  XREF_BIBR).\", \"source\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5456703904455152\": true}, \"uuid\": {\"4959e516-cdd8-4e48-8ae9-0195a34d59f8\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1057916596617540075\": true}, \"source_api\": {\"reach\": true}}, \"target\": 1, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Finally, metformin inhibits formation of AGEs (XREF_BIBR, XREF_BIBR) as well as cytosolic and mitochondrial ROS production induced by AGEs in endothelial and smooth muscle cells (XREF_BIBR).\", \"source\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32804568068160931\": true}, \"uuid\": {\"4640a280-474f-4c66-9a15-879861663f42\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"8102492590622905362\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Metformin not only reduces blood glucose levels and clearance following a bolus of glucose, but also has anti-inflammatory properties as well as cardio-vascular protective effects (i.e., anti-thrombotic).\", \"source\": 5, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11649440761849513\": true}, \"uuid\": {\"d1cc7d6c-d52d-4605-9961-0e46a1476fee\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7114963574385319006\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Increased production of ROS caused by cellular damages and hypoxia induces expression of the HIF1  (hypoxia inducible factor 1 ) transcription factor (XREF_BIBR) stimulating the expression of GLUT transporters (XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR) and glycolytic genes (XREF_BIBR, XREF_BIBR), that increase extracellular glucose uptake and glycolytic capacity, amplifying the viral replication (XREF_BIBR  XREF_BIBR).\", \"source\": 1, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31155280644457031\": true}, \"uuid\": {\"1d90e9e6-9574-4211-a84b-59e7409817ba\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7114963574385319006\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Increased production of ROS caused by cellular damages and hypoxia induces expression of the HIF1  (hypoxia inducible factor 1 ) transcription factor (XREF_BIBR) stimulating the expression of GLUT transporters (XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR) and glycolytic genes (XREF_BIBR, XREF_BIBR), that increase extracellular glucose uptake and glycolytic capacity, amplifying the viral replication (XREF_BIBR  XREF_BIBR).\", \"source\": 1, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16248019358171865\": true}, \"uuid\": {\"9f864c37-7663-471e-b45f-1ab748796359\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1414306872617660026\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Additionally, blockade of both GRP78 and Cd147 is shown to inhibit SARS-CoV-2 viral entry and replication.\", \"source\": 9, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-6586888289516083\": true}, \"uuid\": {\"3074a090-d36f-4acc-8b56-74001aa6b710\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"1414306872617660026\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Additionally, blockade of both GRP78 and Cd147 is shown to inhibit SARS-CoV-2 viral entry and replication.\", \"source\": 29, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5080181045745968\": true}, \"uuid\": {\"fd0c44c6-6657-4bc3-a4e5-78424975936f\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-7114963574385319006\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Increased production of ROS caused by cellular damages and hypoxia induces expression of the HIF1  (hypoxia inducible factor 1 ) transcription factor (XREF_BIBR) stimulating the expression of GLUT transporters (XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR) and glycolytic genes (XREF_BIBR, XREF_BIBR), that increase extracellular glucose uptake and glycolytic capacity, amplifying the viral replication (XREF_BIBR  XREF_BIBR).\", \"source\": 22, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20881577373313582\": true}, \"uuid\": {\"f0d48f6b-2b35-4792-a735-03e6ece6b2c1\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3705373637672401834\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"HIF1  also induces the expression of LDH (XREF_BIBR) causing an increase in the conversion of pyruvate into lactate (a biomarker of a poor COVID-19 prognosis), further decreasing the pH, and further compromising innate defenses.\", \"source\": 22, \"target\": 24, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2048675217688125\": true}, \"uuid\": {\"39e9ce5d-c25f-4570-b7c7-5f562ac61c31\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3705373637672401834\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"HIF1  also induces the expression of LDH (XREF_BIBR) causing an increase in the conversion of pyruvate into lactate (a biomarker of a poor COVID-19 prognosis), further decreasing the pH, and further compromising innate defenses.\", \"source\": 8, \"target\": 24, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"30516691486543486\": true}, \"uuid\": {\"ed973fa9-2a7f-4c76-b78a-1b047fbb9c13\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-124379668534573248\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Second, Ang II was shown to inhibit alveolar fluid clearance, to dysregulate ENaC expression that worsens alveolar edema (XREF_BIBR).\", \"source\": 28, \"target\": 35, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14517489007847264\": true}, \"uuid\": {\"5581c576-34c2-4bf6-9fd9-2618f92fe1ea\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"4835470832999522064\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"34395368\"}, \"evidence\": \"Third, it was shown that Ang II leads to the overexpression of RAGE (XREF_BIBR), the pro inflammatory receptor of AGEs.\", \"source\": 28, \"target\": 27, \"key\": 0}]}, \"#bkzjetxdromuaycf\", 1000, 650, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#98DF8A', 'Composite': '#9467BD', 'Reaction': '#000000', 'Gene': '#FFBB78', 'Abundance': '#AEC7E8', 'RNA': '#FF9896'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "logette_bel = pybel.from_pickle('logette.pickle')\n",
    "#pybel.to_pickle(logette_bel, 'logette.pickle')\n",
    "\n",
    "\n",
    "to_jupyter(logette_bel)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------  ------------------------------------\n",
      "Name                   indra\n",
      "Version                9453ecff-ad33-4d6f-bf56-8aad3a17fa95\n",
      "Number of Nodes        40\n",
      "Number of Namespaces   8\n",
      "Number of Edges        48\n",
      "Number of Annotations  6\n",
      "Number of Citations    1\n",
      "Number of Authors      0\n",
      "Number of Components   4\n",
      "Number of Warnings     0\n",
      "Network Density        3.08E-02\n",
      "---------------------  ------------------------------------\n",
      "\n",
      "Type (4)             Count  Example\n",
      "-----------------  -------  -------------------------------------------------\n",
      "Protein                 18  p(FPLX:SLC2A ! SLC2A)\n",
      "Abundance               16  a(TEXT:\"\")\n",
      "BiologicalProcess        5  bp(GO:\"GO:0006810\" ! transport)\n",
      "Complex                  1  complex(a(TEXT:SARS-CoV-2), p(HGNC:13557 ! ACE2))\n",
      "\n",
      "Prefix (8)    Name                                        Count  Example\n",
      "------------  ----------------------------------------  -------  ----------------------------------------\n",
      "TEXT                                                         10  a(TEXT:glucose)\n",
      "HGNC          HUGO Gene Nomenclature Committee               10  p(HGNC:9657 ! PTPN5)\n",
      "CHEBI         Chemical Entities of Biological Interest        7  a(CHEBI:84755 ! 2-deoxy-D-glucopyranose)\n",
      "GO            Gene Ontology                                   5  bp(GO:\"GO:0006810\" ! transport)\n",
      "FPLX          FamPlex                                         4  p(FPLX:SLC2A ! SLC2A)\n",
      "PFAM          Pfam ID                                         3  p(PFAM:PF00514 ! arm)\n",
      "UP            UniProt protein ID                              1  p(UP:B0C5D4 ! B0C5D4)\n",
      "IP            Protein Domains                                 1  p(IP:IPR004724)\n",
      "\n",
      "Edge Type (13)                           Count  Example\n",
      "-------------------------------------  -------  -----------------------------------------------------------------------------\n",
      "Abundance increases Protein                  8  a(TEXT:glucose) increases p(HGNC:9657 ! PTPN5)\n",
      "Abundance increases BiologicalProcess        7  a(TEXT:glucose) increases bp(GO:\"GO:0006810\" ! transport)\n",
      "Abundance decreases Abundance                6  a(CHEBI:6801 ! metformin) decreases a(TEXT:glucose)\n",
      "Abundance decreases Protein                  5  a(TEXT:SARS-CoV-2) decreases p(HGNC:13557 ! ACE2)\n",
      "Abundance increases Abundance                4  a(TEXT:glucose) increases a(TEXT:glucose)\n",
      "Protein increases Protein                    4  p(FPLX:HIF1 ! HIF1) increases p(FPLX:LDH ! LDH)\n",
      "Protein increases BiologicalProcess          3  p(FPLX:SLC2A ! SLC2A) increases bp(GO:\"GO:0006810\" ! transport)\n",
      "Protein increases Abundance                  3  p(FPLX:SLC2A ! SLC2A) increases a(TEXT:glucose)\n",
      "Protein decreases Protein                    3  p(HGNC:584 ! APCS) decreases p(PFAM:PF00514 ! arm)\n",
      "Abundance decreases BiologicalProcess        2  a(TEXT:glucose) decreases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Abundance directlyIncreases Abundance        1  a(TEXT:hexokinases) directlyIncreases a(TEXT:glucose)\n",
      "Abundance partOf Complex                     1  a(TEXT:SARS-CoV-2) partOf complex(a(TEXT:SARS-CoV-2), p(HGNC:13557 ! ACE2))\n",
      "Protein partOf Complex                       1  p(HGNC:13557 ! ACE2) partOf complex(a(TEXT:SARS-CoV-2), p(HGNC:13557 ! ACE2))\n",
      "\n"
     ]
    }
   ],
   "source": [
    "logette_bel.summarize()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get bibliography of Logette et al 2021"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "pmids = get_pmids_of_bibtex('../data/blood-glucose.bib')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get subset of EMMAA graph corresponding to bibliography"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "glucose_covid = get_subgraph_by_pubmed(emmaa_covid_graph, pmids.keys())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Visualize EMMAA subgraph of bibliography"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "\n",
       "    /**\n",
       "     * Gets the best name for a node object\n",
       "     * @param {object} d object\n",
       "     * @returns {str} canonical name of the node\n",
       "     */\n",
       "    function getCanonicalName(d) {\n",
       "        if (d.concept && d.concept.name && !(d.variants || d.reactants || d.products || d.members)) {\n",
       "            return d.concept.name\n",
       "        } else if (d.bel) {\n",
       "            return d.bel\n",
       "        } else {\n",
       "            console.log('Undefined node: ' + d);\n",
       "            return 'UNDEFINED'\n",
       "        }\n",
       "    }\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation', 'isA', 'partOf'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return getCanonicalName(d)\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='cpfgmhbuxsetqqzd'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.15.4\", \"document_metadata\": {}, \"namespace_url\": {\"HGNC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/hgnc-human-genes/hgnc-human-genes-20170725.belns\", \"UP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/swissprot/swissprot-20170725.belns\", \"IP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/interpro/interpro-20170731.belns\", \"FPLX\": \"https://raw.githubusercontent.com/sorgerlab/famplex/5f5b573fe26d7405dbccb711ae8e5697b6a3ec7e/export/famplex.belns\", \"CHEBI\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/chebi-ids/chebi-ids-20170725.belns\", \"GO\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go/go-20180109.belns\", \"MESH\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-processes/mesh-processes-20170725.belns\"}, \"namespace_pattern\": {\"PUBCHEM\": \"\\\\d+\"}, \"annotation_url\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}, \"annotation_curie\": [], \"annotation_miriam\": []}, \"nodes\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"NAD\", \"identifier\": \"13389\"}, \"id\": \"f5e206fe6cf49d973971c8c8109b0b3b\", \"bel\": \"a(CHEBI:13389 ! NAD)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"camostat\", \"identifier\": \"135632\"}, \"id\": \"f082deac5e277230ef99b72341d40a85\", \"bel\": \"a(CHEBI:135632 ! camostat)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"acetoacetate\", \"identifier\": \"13705\"}, \"id\": \"e6063f52eb20c9d3e05fe9d61e7d5592\", \"bel\": \"a(CHEBI:13705 ! acetoacetate)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"isoxaflutole\", \"identifier\": \"141213\"}, \"id\": \"74e90ec63e45282865184c145cf874fd\", \"bel\": \"a(CHEBI:141213 ! isoxaflutole)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"dioxygen\", \"identifier\": \"15379\"}, \"id\": \"fca982abec2f101c0ebd68f4d863fd37\", \"bel\": \"a(CHEBI:15379 ! dioxygen)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ATP\", \"identifier\": \"15422\"}, \"id\": \"219ce9b7f9e478016facfd573de44d51\", \"bel\": \"a(CHEBI:15422 ! ATP)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"coproporphyrinogen III\", \"identifier\": \"15439\"}, \"id\": \"0ee986f940552d65ab9dc27a992a7789\", \"bel\": \"a(CHEBI:15439 ! \\\"coproporphyrinogen III\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"spermine\", \"identifier\": \"15746\"}, \"id\": \"11f0e158e5ff90c1cf4722f131c43aeb\", \"bel\": \"a(CHEBI:15746 ! spermine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"polypeptide\", \"identifier\": \"15841\"}, \"id\": \"cd3ee78cda19d227a815b1c18d399425\", \"bel\": \"a(CHEBI:15841 ! polypeptide)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"adenosine 5'-monophosphate\", \"identifier\": \"16027\"}, \"id\": \"316680156e69d9f6acb8005f905c5688\", \"bel\": \"a(CHEBI:16027 ! \\\"adenosine 5'-monophosphate\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"hydrogen peroxide\", \"identifier\": \"16240\"}, \"id\": \"01d77d9e7c9cc40943ac48996a7dc6ad\", \"bel\": \"a(CHEBI:16240 ! \\\"hydrogen peroxide\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"lipopolysaccharide\", \"identifier\": \"16412\"}, \"id\": \"5e0255f60babae6a6c72075d574cc5cd\", \"bel\": \"a(CHEBI:16412 ! lipopolysaccharide)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"NADPH\", \"identifier\": \"16474\"}, \"id\": \"1a38630b0ba29bd6775c15f02d92f29b\", \"bel\": \"a(CHEBI:16474 ! NADPH)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"nitric oxide\", \"identifier\": \"16480\"}, \"id\": \"3e9b306d0538fbd25e5f52f3c6943fb3\", \"bel\": \"a(CHEBI:16480 ! \\\"nitric oxide\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"carbon dioxide\", \"identifier\": \"16526\"}, \"id\": \"a8b52d413a5efcddf6edf7225d102caf\", \"bel\": \"a(CHEBI:16526 ! \\\"carbon dioxide\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"spermidine\", \"identifier\": \"16610\"}, \"id\": \"9835a84cc4a644432f963d7f9c67bd91\", \"bel\": \"a(CHEBI:16610 ! spermidine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"peptide\", \"identifier\": \"16670\"}, \"id\": \"a74866a766f5bcf4f52d24ad2d38cf41\", \"bel\": \"a(CHEBI:16670 ! peptide)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ADP\", \"identifier\": \"16761\"}, \"id\": \"4370846822147ab833e5ef5745de338a\", \"bel\": \"a(CHEBI:16761 ! ADP)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"glutathione\", \"identifier\": \"16856\"}, \"id\": \"d5418aef940ead6ee9d1b70790d1e9f5\", \"bel\": \"a(CHEBI:16856 ! glutathione)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"glycoprotein\", \"identifier\": \"17089\"}, \"id\": \"ef0947fe968f77aefd37d5fb3db8f79a\", \"bel\": \"a(CHEBI:17089 ! glycoprotein)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"glucose\", \"identifier\": \"17234\"}, \"id\": \"c58a94372b46d47917dd1bf98c28df4a\", \"bel\": \"a(CHEBI:17234 ! glucose)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"carbon monoxide\", \"identifier\": \"17245\"}, \"id\": \"60a555229b1ac5a23d2b977f939c23d8\", \"bel\": \"a(CHEBI:17245 ! \\\"carbon monoxide\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"testosterone\", \"identifier\": \"17347\"}, \"id\": \"befa99712658509f900a5b65d572f5e5\", \"bel\": \"a(CHEBI:17347 ! testosterone)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"5-aminolevulinic acid\", \"identifier\": \"17549\"}, \"id\": \"5aaa01a313e605f0712b37dce32bef18\", \"bel\": \"a(CHEBI:17549 ! \\\"5-aminolevulinic acid\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ferroheme b\", \"identifier\": \"17627\"}, \"id\": \"3e20040ad19a0d564de73e0cf4afcfe5\", \"bel\": \"a(CHEBI:17627 ! \\\"ferroheme b\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"triglyceride\", \"identifier\": \"17855\"}, \"id\": \"0d59050b9b27d590548bc0ffaa30d3ba\", \"bel\": \"a(CHEBI:17855 ! triglyceride)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"hydrogen chloride\", \"identifier\": \"17883\"}, \"id\": \"43182e06375008909169ac8fefe2d2ef\", \"bel\": \"a(CHEBI:17883 ! \\\"hydrogen chloride\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"chloride\", \"identifier\": \"17996\"}, \"id\": \"f537193ac2d2ecc88675c13c200bb984\", \"bel\": \"a(CHEBI:17996 ! chloride)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"polysaccharide\", \"identifier\": \"18154\"}, \"id\": \"bb8b85047213193fe3069fc7aaa1a6b6\", \"bel\": \"a(CHEBI:18154 ! polysaccharide)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"iron atom\", \"identifier\": \"18248\"}, \"id\": \"1dc05b0df53ed719c80e9b363a6dcfb5\", \"bel\": \"a(CHEBI:18248 ! \\\"iron atom\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"phosphate(3-)\", \"identifier\": \"18367\"}, \"id\": \"22f5589f0e07010f6cf8472a88242497\", \"bel\": \"a(CHEBI:18367 ! \\\"phosphate(3-)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"magnesium(2+)\", \"identifier\": \"18420\"}, \"id\": \"c921a42c94c1be662afbee2f93bfa5ae\", \"bel\": \"a(CHEBI:18420 ! \\\"magnesium(2+)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"superoxide\", \"identifier\": \"18421\"}, \"id\": \"ef9c4e031c1873ed855be08f28303fbb\", \"bel\": \"a(CHEBI:18421 ! superoxide)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"drug\", \"identifier\": \"23888\"}, \"id\": \"cbee8f4b9fbc6ad586b0b8e13509e81e\", \"bel\": \"a(CHEBI:23888 ! drug)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"glucocorticoid\", \"identifier\": \"24261\"}, \"id\": \"cfcd873b836eaa6179e99a18bcb4a349\", \"bel\": \"a(CHEBI:24261 ! glucocorticoid)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"acyclovir\", \"identifier\": \"2453\"}, \"id\": \"874bf1dff9c9300d9034864e2d933f1e\", \"bel\": \"a(CHEBI:2453 ! acyclovir)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"lactate\", \"identifier\": \"24996\"}, \"id\": \"275634f784b14c3bdc66c5e1a43ff722\", \"bel\": \"a(CHEBI:24996 ! lactate)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"magnesium atom\", \"identifier\": \"25107\"}, \"id\": \"b8c0cf379c2150fbd41f3158a68809b3\", \"bel\": \"a(CHEBI:25107 ! \\\"magnesium atom\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"mineralocorticoid\", \"identifier\": \"25354\"}, \"id\": \"881d644f89b942d32f65edbb3923c575\", \"bel\": \"a(CHEBI:25354 ! mineralocorticoid)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"amantadine\", \"identifier\": \"2618\"}, \"id\": \"b6d92e9ca6f1585aca09e75c1309d4a3\", \"bel\": \"a(CHEBI:2618 ! amantadine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"reactive oxygen species\", \"identifier\": \"26523\"}, \"id\": \"967db55abac7ab8dcfbcc0df2b4c4cd3\", \"bel\": \"a(CHEBI:26523 ! \\\"reactive oxygen species\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"sialic acid\", \"identifier\": \"26667\"}, \"id\": \"2c11bb674374fa905c320ab2636a1f71\", \"bel\": \"a(CHEBI:26667 ! \\\"sialic acid\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"sodium atom\", \"identifier\": \"26708\"}, \"id\": \"01aea6b60f55d7f4d7d27444906b99e9\", \"bel\": \"a(CHEBI:26708 ! \\\"sodium atom\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"vitamin D\", \"identifier\": \"27300\"}, \"id\": \"692caf6cc0e729e2d0bc221712dc2aba\", \"bel\": \"a(CHEBI:27300 ! \\\"vitamin D\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"(2R,3R)-2,3-dihydroxy-3-methylpentanoic acid\", \"identifier\": \"27512\"}, \"id\": \"68869887ba155dfd841f59a6fdba667a\", \"bel\": \"a(CHEBI:27512 ! \\\"(2R,3R)-2,3-dihydroxy-3-methylpentanoic acid\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"aldosterone\", \"identifier\": \"27584\"}, \"id\": \"f69b788b1934671031a3be6656999d31\", \"bel\": \"a(CHEBI:27584 ! aldosterone)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"monensin A\", \"identifier\": \"27617\"}, \"id\": \"3c4bc65541ae96738169953f5bb1d9ff\", \"bel\": \"a(CHEBI:27617 ! \\\"monensin A\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"heparin\", \"identifier\": \"28304\"}, \"id\": \"62001639482f8883bfd9bc200a69a372\", \"bel\": \"a(CHEBI:28304 ! heparin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"phosphatidylinositol\", \"identifier\": \"28874\"}, \"id\": \"3b28bba20025d888185b11328c8fe6f4\", \"bel\": \"a(CHEBI:28874 ! phosphatidylinositol)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"potassium(1+)\", \"identifier\": \"29103\"}, \"id\": \"edd6f19b45a0160c9721a936f2d81875\", \"bel\": \"a(CHEBI:29103 ! \\\"potassium(1+)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"calcium(2+)\", \"identifier\": \"29108\"}, \"id\": \"7ca74a2e7ef28cdaef68d1c3077b44f3\", \"bel\": \"a(CHEBI:29108 ! \\\"calcium(2+)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"azanediyl (triplet)\", \"identifier\": \"29339\"}, \"id\": \"98c5073e3e25e103fc0649600403b73b\", \"bel\": \"a(CHEBI:29339 ! \\\"azanediyl (triplet)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"tunicamycin\", \"identifier\": \"29699\"}, \"id\": \"8c13095524d11238d18853fbc1e76d64\", \"bel\": \"a(CHEBI:29699 ! tunicamycin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"photon\", \"identifier\": \"30212\"}, \"id\": \"53f615d482d91e673229cb8d8a8a7d77\", \"bel\": \"a(CHEBI:30212 ! photon)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"heme\", \"identifier\": \"30413\"}, \"id\": \"151e402db005053a74630c5ec936da53\", \"bel\": \"a(CHEBI:30413 ! heme)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"astatine atom\", \"identifier\": \"30415\"}, \"id\": \"473421f22711dee7fae2f43647307e33\", \"bel\": \"a(CHEBI:30415 ! \\\"astatine atom\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"bradykinin\", \"identifier\": \"3165\"}, \"id\": \"0e364eddb5673a79a4718e3c9006bd49\", \"bel\": \"a(CHEBI:3165 ! bradykinin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"pitavastatin\", \"identifier\": \"32020\"}, \"id\": \"32f4d22eee50e9408b76192e94a1df23\", \"bel\": \"a(CHEBI:32020 ! pitavastatin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"catecholamine\", \"identifier\": \"33567\"}, \"id\": \"577eb40be909ed221a22db11d5a236ae\", \"bel\": \"a(CHEBI:33567 ! catecholamine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"captopril\", \"identifier\": \"3380\"}, \"id\": \"58bd06d611a60da84f84673f1feb5c52\", \"bel\": \"a(CHEBI:3380 ! captopril)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"picloram\", \"identifier\": \"34922\"}, \"id\": \"0358da036dce52536f02eee2315599fa\", \"bel\": \"a(CHEBI:34922 ! picloram)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"hemoglobin\", \"identifier\": \"35143\"}, \"id\": \"d3c4b612446fdba3f473e8a071cc45c8\", \"bel\": \"a(CHEBI:35143 ! hemoglobin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"antiviral drug\", \"identifier\": \"36044\"}, \"id\": \"094b43838e083e7e7b9cbe0595939fde\", \"bel\": \"a(CHEBI:36044 ! \\\"antiviral drug\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"chloroquine\", \"identifier\": \"3638\"}, \"id\": \"59c917f3bf8161dcd92b5c67154e7b48\", \"bel\": \"a(CHEBI:3638 ! chloroquine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"mannose\", \"identifier\": \"37684\"}, \"id\": \"059433d3750fa25ccfe15d61416d6005\", \"bel\": \"a(CHEBI:37684 ! mannose)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"Coagulation Factor Xa\", \"identifier\": \"3783\"}, \"id\": \"d8c5ad95a8a94065f8be7a9c9c3efc74\", \"bel\": \"a(CHEBI:3783 ! \\\"Coagulation Factor Xa\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"rosuvastatin\", \"identifier\": \"38545\"}, \"id\": \"208b76c94adabbd186229d42f3e6ecb8\", \"bel\": \"a(CHEBI:38545 ! rosuvastatin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"tamoxifen\", \"identifier\": \"41774\"}, \"id\": \"ff4f75bd7a824b2f5d227d4fa396aa74\", \"bel\": \"a(CHEBI:41774 ! tamoxifen)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"forskolin\", \"identifier\": \"42471\"}, \"id\": \"f125b1ef178c3fa6750867ca46fde85e\", \"bel\": \"a(CHEBI:42471 ! forskolin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"2-O-acetyl-1-O-hexadecyl-sn-glycero-3-phosphocholine\", \"identifier\": \"44811\"}, \"id\": \"0fd14df5e3fbca1aaed60027f20847ba\", \"bel\": \"a(CHEBI:44811 ! 2-O-acetyl-1-O-hexadecyl-sn-glycero-3-phosphocholine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"salicylhydroxamic acid\", \"identifier\": \"45615\"}, \"id\": \"fa24e8f676bef060f89e4f4475c6f905\", \"bel\": \"a(CHEBI:45615 ! \\\"salicylhydroxamic acid\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"N-acetyl-beta-neuraminic acid\", \"identifier\": \"45744\"}, \"id\": \"80cd7882455cd2a7c754a872c9dbe1bb\", \"bel\": \"a(CHEBI:45744 ! \\\"N-acetyl-beta-neuraminic acid\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"high-density lipoprotein cholesterol\", \"identifier\": \"47775\"}, \"id\": \"161489cdf5db3776f302585df7b6a625\", \"bel\": \"a(CHEBI:47775 ! \\\"high-density lipoprotein cholesterol\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"enalapril\", \"identifier\": \"4784\"}, \"id\": \"802f39fedd1b02bea3788290044d0e41\", \"bel\": \"a(CHEBI:4784 ! enalapril)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"indicator\", \"identifier\": \"47867\"}, \"id\": \"2678ec8670f55760c6d8bb1fe9664d47\", \"bel\": \"a(CHEBI:47867 ! indicator)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"angiotensin\", \"identifier\": \"48433\"}, \"id\": \"9e92818ff4dde28e09e2121d9284c4db\", \"bel\": \"a(CHEBI:48433 ! angiotensin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"rosiglitazone\", \"identifier\": \"50122\"}, \"id\": \"261f8d2a7ebe173eaf794b42d7f2096d\", \"bel\": \"a(CHEBI:50122 ! rosiglitazone)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"motexafin gadolinium\", \"identifier\": \"50161\"}, \"id\": \"33f99bd364d2d3ae3289cf843ec722f6\", \"bel\": \"a(CHEBI:50161 ! \\\"motexafin gadolinium\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"N-acetyl-D-glucosamine\", \"identifier\": \"506227\"}, \"id\": \"3aeb53e7950b45982cf50fc42a2b2af0\", \"bel\": \"a(CHEBI:506227 ! N-acetyl-D-glucosamine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"biliverdin(2-)\", \"identifier\": \"57991\"}, \"id\": \"52e93dea9d2b76065963dbe84abbffc7\", \"bel\": \"a(CHEBI:57991 ! \\\"biliverdin(2-)\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"hydroxychloroquine\", \"identifier\": \"5801\"}, \"id\": \"56ccfa4d4c7259cd6cb25436286ca500\", \"bel\": \"a(CHEBI:5801 ! hydroxychloroquine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"N-glycan\", \"identifier\": \"59520\"}, \"id\": \"56d91fb40fcf96fb21e09d91f780b33d\", \"bel\": \"a(CHEBI:59520 ! N-glycan)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"alpha-D-GalpNAc-(1->3)-[alpha-L-Fucp-(1->2)]-beta-D-Galp-(1->4)-beta-D-GlcpNAc\", \"identifier\": \"62486\"}, \"id\": \"65f00950b75410d513287d76127eda48\", \"bel\": \"a(CHEBI:62486 ! \\\"alpha-D-GalpNAc-(1->3)-[alpha-L-Fucp-(1->2)]-beta-D-Galp-(1->4)-beta-D-GlcpNAc\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ribavirin\", \"identifier\": \"63580\"}, \"id\": \"139c18b8524d965e3cab170140faadba\", \"bel\": \"a(CHEBI:63580 ! ribavirin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"estrogen receptor agonist\", \"identifier\": \"63951\"}, \"id\": \"176ce4d96c2a25d160a11f8b1d8a6c0a\", \"bel\": \"a(CHEBI:63951 ! \\\"estrogen receptor agonist\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"tocilizumab\", \"identifier\": \"64360\"}, \"id\": \"c634f41352aabe166c14e23f292d0641\", \"bel\": \"a(CHEBI:64360 ! tocilizumab)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"losartan\", \"identifier\": \"6541\"}, \"id\": \"d75ef2ce6780ec23c552fb8d88aeb006\", \"bel\": \"a(CHEBI:6541 ! losartan)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ruxolitinib\", \"identifier\": \"66919\"}, \"id\": \"22bb433545319c50f8c5aa190eef1400\", \"bel\": \"a(CHEBI:66919 ! ruxolitinib)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"metformin\", \"identifier\": \"6801\"}, \"id\": \"6f69bd15b40488139d8325b83eace84d\", \"bel\": \"a(CHEBI:6801 ! metformin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"alpha-Asp-Leu\", \"identifier\": \"68596\"}, \"id\": \"4f77e12a76109c86c33541e3b943d710\", \"bel\": \"a(CHEBI:68596 ! alpha-Asp-Leu)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"linagliptin\", \"identifier\": \"68610\"}, \"id\": \"94de04ae0939365c0c0db00a54c6a46e\", \"bel\": \"a(CHEBI:68610 ! linagliptin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"dactolisib\", \"identifier\": \"71952\"}, \"id\": \"4a43c470bc8a07d5ee48b12aafec7f6c\", \"bel\": \"a(CHEBI:71952 ! dactolisib)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"Gly-Ile\", \"identifier\": \"73907\"}, \"id\": \"e0a2f9f98e524f38a5a69ed9f77ac5a1\", \"bel\": \"a(CHEBI:73907 ! Gly-Ile)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"Ile-Leu\", \"identifier\": \"74072\"}, \"id\": \"575ed3d8fc294291b09365fafad20a2f\", \"bel\": \"a(CHEBI:74072 ! Ile-Leu)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"peptidoglycan\", \"identifier\": \"8005\"}, \"id\": \"c27c5859fd11b7ceeae6100cb284edd3\", \"bel\": \"a(CHEBI:8005 ! peptidoglycan)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"PB1-F2\", \"identifier\": \"82128\"}, \"id\": \"de92e407a908be140516ddc6ed615684\", \"bel\": \"a(CHEBI:82128 ! PB1-F2)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"Ferritin\", \"identifier\": \"82594\"}, \"id\": \"09c0097762a63cf5b7e22b59baee6ceb\", \"bel\": \"a(CHEBI:82594 ! Ferritin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"24,25-Dihydroxyvitamin D\", \"identifier\": \"89324\"}, \"id\": \"9e9c67a0c79d61b24f74fb652ffbcf86\", \"bel\": \"a(CHEBI:89324 ! \\\"24,25-Dihydroxyvitamin D\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"simvastatin\", \"identifier\": \"9150\"}, \"id\": \"acbe903a1740ab819d544e0cfe1f68bd\", \"bel\": \"a(CHEBI:9150 ! simvastatin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"streptozocin\", \"identifier\": \"9288\"}, \"id\": \"0c1fac939636a98186ffbf543e375248\", \"bel\": \"a(CHEBI:9288 ! streptozocin)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"AGER inhibitors\"}, \"id\": \"b890c9f944148f37c0a8fa3209c81ebe\", \"bel\": \"a(TEXT:\\\"AGER inhibitors\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"HMGB1 inhibitors\"}, \"id\": \"8171be7e1b178cef1d6bd687ad7a549f\", \"bel\": \"a(TEXT:\\\"HMGB1 inhibitors\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"Renal potassium wasting\"}, \"id\": \"d574f322b94c8bbc04728186b7781904\", \"bel\": \"a(TEXT:\\\"Renal potassium wasting\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"SARS-CoV-2 Receptor Binding Domain\"}, \"id\": \"52ded39073778e4a2056ef8bcd574c39\", \"bel\": \"a(TEXT:\\\"SARS-CoV-2 Receptor Binding Domain\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"kidney injury\"}, \"id\": \"ea7f8205b7883320966e502f3d227711\", \"bel\": \"a(TEXT:\\\"kidney injury\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"vit D\"}, \"id\": \"516ce7c77edfe46103a4f0ad63808e8c\", \"bel\": \"a(TEXT:\\\"vit D\\\")\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"Angiotensin-2\"}, \"id\": \"4f02a737416ce95494212bd5210b60ad\", \"bel\": \"a(TEXT:Angiotensin-2)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"COVID-19\"}, \"id\": \"9cb5db56cf8f5a7624f50908cad14233\", \"bel\": \"a(TEXT:COVID-19)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"HC\"}, \"id\": \"e2e758b43ee5d6426433348609840ac6\", \"bel\": \"a(TEXT:HC)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"HCs\"}, \"id\": \"f8017754de2597a129dcf4a1a4bcae81\", \"bel\": \"a(TEXT:HCs)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"HMGB1-AGER\"}, \"id\": \"f1cbfdddaa24d75749ec35f237c71add\", \"bel\": \"a(TEXT:HMGB1-AGER)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"HR\"}, \"id\": \"98d0238749b5a850be0ece4050d5cbaa\", \"bel\": \"a(TEXT:HR)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"IR\"}, \"id\": \"d270a98fe89fcf4ff72048a9ca0ab0fa\", \"bel\": \"a(TEXT:IR)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"LLC-PK1\"}, \"id\": \"a20d1cac0e513ff62dc192fd63cde100\", \"bel\": \"a(TEXT:LLC-PK1)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"SP\"}, \"id\": \"4c4f32163dcce79ac36456bf4155e199\", \"bel\": \"a(TEXT:SP)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"STORM\"}, \"id\": \"9dd5ddbef95e13f002bde88cc483c2e1\", \"bel\": \"a(TEXT:STORM)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"cytokine\"}, \"id\": \"d20336ce10bc46827222d15dbbb2780f\", \"bel\": \"a(TEXT:cytokine)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"deletion\"}, \"id\": \"cede98b687ae10388ef0f85d5af0d1e3\", \"bel\": \"a(TEXT:deletion)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"event\"}, \"id\": \"1945821a76e4c2868f01fb99e50a8425\", \"bel\": \"a(TEXT:event)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"pathogen\"}, \"id\": \"26c4fb2c0621dc396145de08709d4f38\", \"bel\": \"a(TEXT:pathogen)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"rHMGB1\"}, \"id\": \"465ce4472bd2bc756b8e553ba655ced3\", \"bel\": \"a(TEXT:rHMGB1)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"sRAGE\"}, \"id\": \"ef9e5f2fefe67b7b99bb1e1dc4e8ab36\", \"bel\": \"a(TEXT:sRAGE)\"}, {\"function\": \"Abundance\", \"concept\": {\"namespace\": \"TEXT\", \"name\": \"translocation\"}, \"id\": \"6add1f7423781bda35f1d1b7ea82a6a1\", \"bel\": \"a(TEXT:translocation)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"activation of innate immune response\", \"identifier\": \"GO:0002218\"}, \"id\": \"8fba418c5ad0cf78b08146ff4ddd59f4\", \"bel\": \"bp(GO:\\\"GO:0002218\\\" ! \\\"activation of innate immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"adaptive immune response\", \"identifier\": \"GO:0002250\"}, \"id\": \"5abbbafacc0b6f2ffab1650c17e12828\", \"bel\": \"bp(GO:\\\"GO:0002250\\\" ! \\\"adaptive immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"antigen binding\", \"identifier\": \"GO:0003823\"}, \"id\": \"d8bff2c3817d6393e5bcefa0a7a4c374\", \"bel\": \"bp(GO:\\\"GO:0003823\\\" ! \\\"antigen binding\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cell\", \"identifier\": \"GO:0005623\"}, \"id\": \"37116b1ad4fa7aa933254f6aa6d0ed63\", \"bel\": \"bp(GO:\\\"GO:0005623\\\" ! cell)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"endoplasmic reticulum\", \"identifier\": \"GO:0005783\"}, \"id\": \"8bb0c6846888e0e9886b95d851402936\", \"bel\": \"bp(GO:\\\"GO:0005783\\\" ! \\\"endoplasmic reticulum\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"glycolytic process\", \"identifier\": \"GO:0006096\"}, \"id\": \"058b0dfff66b57c9d079c12a53418559\", \"bel\": \"bp(GO:\\\"GO:0006096\\\" ! \\\"glycolytic process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"pentose-phosphate shunt\", \"identifier\": \"GO:0006098\"}, \"id\": \"9dba229598f54d1fd37f9919473601e2\", \"bel\": \"bp(GO:\\\"GO:0006098\\\" ! \\\"pentose-phosphate shunt\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"fatty acid biosynthetic process\", \"identifier\": \"GO:0006633\"}, \"id\": \"9fa50d5be2e856a560195dbaecd26b71\", \"bel\": \"bp(GO:\\\"GO:0006633\\\" ! \\\"fatty acid biosynthetic process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"heme oxidation\", \"identifier\": \"GO:0006788\"}, \"id\": \"0f0b45c48fb09d09e73af3de6548ffc3\", \"bel\": \"bp(GO:\\\"GO:0006788\\\" ! \\\"heme oxidation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"transport\", \"identifier\": \"GO:0006810\"}, \"id\": \"0f2bdc9751e0b639ef700fccd082c38c\", \"bel\": \"bp(GO:\\\"GO:0006810\\\" ! transport)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"endocytosis\", \"identifier\": \"GO:0006897\"}, \"id\": \"26d00e21f8ac9427236f2e151fad75d5\", \"bel\": \"bp(GO:\\\"GO:0006897\\\" ! endocytosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"phagocytosis\", \"identifier\": \"GO:0006909\"}, \"id\": \"f1b249b9df49b6c1c37af059d62604b7\", \"bel\": \"bp(GO:\\\"GO:0006909\\\" ! phagocytosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"apoptotic process\", \"identifier\": \"GO:0006915\"}, \"id\": \"105d57ee25aea2d43968637112956666\", \"bel\": \"bp(GO:\\\"GO:0006915\\\" ! \\\"apoptotic process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"inflammatory response\", \"identifier\": \"GO:0006954\"}, \"id\": \"9e3cd403f7bc833ae24bad3f9c2aad48\", \"bel\": \"bp(GO:\\\"GO:0006954\\\" ! \\\"inflammatory response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"immune response\", \"identifier\": \"GO:0006955\"}, \"id\": \"043b1a0262bca90484bcf6c89c1075f9\", \"bel\": \"bp(GO:\\\"GO:0006955\\\" ! \\\"immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"complement activation\", \"identifier\": \"GO:0006956\"}, \"id\": \"8f82152b7144fca287cb161ce9e0fd77\", \"bel\": \"bp(GO:\\\"GO:0006956\\\" ! \\\"complement activation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\", \"identifier\": \"GO:0007157\"}, \"id\": \"d3468af215628e50c7c81a68c961e1fe\", \"bel\": \"bp(GO:\\\"GO:0007157\\\" ! \\\"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"aging\", \"identifier\": \"GO:0007568\"}, \"id\": \"d57540fcf0d4c042035c0440aed930f7\", \"bel\": \"bp(GO:\\\"GO:0007568\\\" ! aging)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"hemostasis\", \"identifier\": \"GO:0007599\"}, \"id\": \"5158ea41c8d847e10caaf2ef1de37639\", \"bel\": \"bp(GO:\\\"GO:0007599\\\" ! hemostasis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cholesterol metabolic process\", \"identifier\": \"GO:0008203\"}, \"id\": \"3b062a70ab6c43ddcd81a981630b4311\", \"bel\": \"bp(GO:\\\"GO:0008203\\\" ! \\\"cholesterol metabolic process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cell death\", \"identifier\": \"GO:0008219\"}, \"id\": \"3b5791105377b6c5fb50c28c887fe639\", \"bel\": \"bp(GO:\\\"GO:0008219\\\" ! \\\"cell death\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cell population proliferation\", \"identifier\": \"GO:0008283\"}, \"id\": \"642b4368000cd038d0cbfbd0e987ffa2\", \"bel\": \"bp(GO:\\\"GO:0008283\\\" ! \\\"cell population proliferation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"catabolic process\", \"identifier\": \"GO:0009056\"}, \"id\": \"8066dbf6a47b0fc335e9543ac839f69f\", \"bel\": \"bp(GO:\\\"GO:0009056\\\" ! \\\"catabolic process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"pathogenesis\", \"identifier\": \"GO:0009405\"}, \"id\": \"8a3f7d8dcb570d87ef7e0194f4d11a97\", \"bel\": \"bp(GO:\\\"GO:0009405\\\" ! pathogenesis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"viral process\", \"identifier\": \"GO:0016032\"}, \"id\": \"cb0fedcc8247e1f424b632ad92a8f0e5\", \"bel\": \"bp(GO:\\\"GO:0016032\\\" ! \\\"viral process\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cell growth\", \"identifier\": \"GO:0016049\"}, \"id\": \"ed108072ea1fafc64b95e5643b5f04d5\", \"bel\": \"bp(GO:\\\"GO:0016049\\\" ! \\\"cell growth\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"RNA interference\", \"identifier\": \"GO:0016246\"}, \"id\": \"bd566ba817fef6e606290b52d8dff4df\", \"bel\": \"bp(GO:\\\"GO:0016246\\\" ! \\\"RNA interference\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"phosphorylation\", \"identifier\": \"GO:0016310\"}, \"id\": \"d9f3d7769bf252a315bc2c987ba38c3b\", \"bel\": \"bp(GO:\\\"GO:0016310\\\" ! phosphorylation)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"virion\", \"identifier\": \"GO:0019012\"}, \"id\": \"6559f70528e7b8a1f73fb51428b86062\", \"bel\": \"bp(GO:\\\"GO:0019012\\\" ! virion)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"viral life cycle\", \"identifier\": \"GO:0019058\"}, \"id\": \"9d84c10dd273ecb65a450256502f487b\", \"bel\": \"bp(GO:\\\"GO:0019058\\\" ! \\\"viral life cycle\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"viral release from host cell\", \"identifier\": \"GO:0019076\"}, \"id\": \"d6d0e41ce2ae322ae189da5af6d07898\", \"bel\": \"bp(GO:\\\"GO:0019076\\\" ! \\\"viral release from host cell\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"outer membrane\", \"identifier\": \"GO:0019867\"}, \"id\": \"114c5b7f29784245e3f01bd3dea6b11b\", \"bel\": \"bp(GO:\\\"GO:0019867\\\" ! \\\"outer membrane\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"antigen processing and presentation\", \"identifier\": \"GO:0019882\"}, \"id\": \"4121f8be70b94d4348cff14d1327a5bd\", \"bel\": \"bp(GO:\\\"GO:0019882\\\" ! \\\"antigen processing and presentation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"insulin secretion\", \"identifier\": \"GO:0030073\"}, \"id\": \"0060dbbea01c2247d2afa64b2d410676\", \"bel\": \"bp(GO:\\\"GO:0030073\\\" ! \\\"insulin secretion\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"carbohydrate binding\", \"identifier\": \"GO:0030246\"}, \"id\": \"240d2753ca823d06911a3401189d70dd\", \"bel\": \"bp(GO:\\\"GO:0030246\\\" ! \\\"carbohydrate binding\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"neutrophil chemotaxis\", \"identifier\": \"GO:0030593\"}, \"id\": \"d7232b2cd8fea2073e6ce7135c2ec775\", \"bel\": \"bp(GO:\\\"GO:0030593\\\" ! \\\"neutrophil chemotaxis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"type I interferon production\", \"identifier\": \"GO:0032606\"}, \"id\": \"ff4433aeb9bfe173a93992a52907dcf6\", \"bel\": \"bp(GO:\\\"GO:0032606\\\" ! \\\"type I interferon production\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"response to insulin\", \"identifier\": \"GO:0032868\"}, \"id\": \"b63fb7fa523e5417593f9883f1d4f820\", \"bel\": \"bp(GO:\\\"GO:0032868\\\" ! \\\"response to insulin\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"microtubule organizing center attachment site\", \"identifier\": \"GO:0034992\"}, \"id\": \"54796c7dfc2ba1135a909cfa624519f5\", \"bel\": \"bp(GO:\\\"GO:0034992\\\" ! \\\"microtubule organizing center attachment site\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"metaplastic ossification\", \"identifier\": \"GO:0036074\"}, \"id\": \"61ec42362ff2ca7aa49be539b61caad7\", \"bel\": \"bp(GO:\\\"GO:0036074\\\" ! \\\"metaplastic ossification\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"signaling receptor activity\", \"identifier\": \"GO:0038023\"}, \"id\": \"f52f96b499a3e71b6ae9da9e59e91d5e\", \"bel\": \"bp(GO:\\\"GO:0038023\\\" ! \\\"signaling receptor activity\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"viral RNA genome replication\", \"identifier\": \"GO:0039694\"}, \"id\": \"920d93cde41710d77e115074e49b8baa\", \"bel\": \"bp(GO:\\\"GO:0039694\\\" ! \\\"viral RNA genome replication\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"T cell proliferation\", \"identifier\": \"GO:0042098\"}, \"id\": \"2d21f98d156bff0d8d90fb0eda192eb1\", \"bel\": \"bp(GO:\\\"GO:0042098\\\" ! \\\"T cell proliferation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"vasodilation\", \"identifier\": \"GO:0042311\"}, \"id\": \"e01913cc50007078c8c80c6f52c5660a\", \"bel\": \"bp(GO:\\\"GO:0042311\\\" ! vasodilation)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"glucose homeostasis\", \"identifier\": \"GO:0042593\"}, \"id\": \"82313db2b17827cb730347829ead17b6\", \"bel\": \"bp(GO:\\\"GO:0042593\\\" ! \\\"glucose homeostasis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"fibrinolysis\", \"identifier\": \"GO:0042730\"}, \"id\": \"7a93815fc63e28e35febb4792ce66298\", \"bel\": \"bp(GO:\\\"GO:0042730\\\" ! fibrinolysis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"evasion of host immune response\", \"identifier\": \"GO:0042783\"}, \"id\": \"f14ff5dd632a253c19fabd9ccc3815c8\", \"bel\": \"bp(GO:\\\"GO:0042783\\\" ! \\\"evasion of host immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"entry into host\", \"identifier\": \"GO:0044409\"}, \"id\": \"a553c35a16dbf4e738433cd11150fae3\", \"bel\": \"bp(GO:\\\"GO:0044409\\\" ! \\\"entry into host\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"innate immune response\", \"identifier\": \"GO:0045087\"}, \"id\": \"f13bb2fdfa690f98c17a57f105b9a207\", \"bel\": \"bp(GO:\\\"GO:0045087\\\" ! \\\"innate immune response\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"negative regulation of endocytosis\", \"identifier\": \"GO:0045806\"}, \"id\": \"d13f06e44812e0382709f6479b6835f4\", \"bel\": \"bp(GO:\\\"GO:0045806\\\" ! \\\"negative regulation of endocytosis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"glucose import\", \"identifier\": \"GO:0046323\"}, \"id\": \"61ba0311cc4a1609e1d8eea9958fa5c7\", \"bel\": \"bp(GO:\\\"GO:0046323\\\" ! \\\"glucose import\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"coagulation\", \"identifier\": \"GO:0050817\"}, \"id\": \"a8d8479483b23fc78b038cae7e63bb3f\", \"bel\": \"bp(GO:\\\"GO:0050817\\\" ! coagulation)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"positive regulation of NF-kappaB transcription factor activity\", \"identifier\": \"GO:0051092\"}, \"id\": \"72b011ea14520afb21198b943934f268\", \"bel\": \"bp(GO:\\\"GO:0051092\\\" ! \\\"positive regulation of NF-kappaB transcription factor activity\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"defense response to virus\", \"identifier\": \"GO:0051607\"}, \"id\": \"c34ce556400266eb0cd31cfa611aed5c\", \"bel\": \"bp(GO:\\\"GO:0051607\\\" ! \\\"defense response to virus\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"membrane fusion\", \"identifier\": \"GO:0061025\"}, \"id\": \"50019dfde3a57ab54e4df08ee16282d3\", \"bel\": \"bp(GO:\\\"GO:0061025\\\" ! \\\"membrane fusion\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"glycosylation\", \"identifier\": \"GO:0070085\"}, \"id\": \"1aaa0db7ecb35799257aedf1c663f5f0\", \"bel\": \"bp(GO:\\\"GO:0070085\\\" ! glycosylation)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"necrotic cell death\", \"identifier\": \"GO:0070265\"}, \"id\": \"b748fdd7137e5348a2ab86ebe3640ba0\", \"bel\": \"bp(GO:\\\"GO:0070265\\\" ! \\\"necrotic cell death\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"pyroptosis\", \"identifier\": \"GO:0070269\"}, \"id\": \"2d1db3d0459a064ed50253cc7cccb23c\", \"bel\": \"bp(GO:\\\"GO:0070269\\\" ! pyroptosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"platelet aggregation\", \"identifier\": \"GO:0070527\"}, \"id\": \"e523ff8fbd288103b9ae5376f6b289b0\", \"bel\": \"bp(GO:\\\"GO:0070527\\\" ! \\\"platelet aggregation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"clathrin-dependent endocytosis\", \"identifier\": \"GO:0072583\"}, \"id\": \"a0d60d097acf1e10f0c5c2ec61cb8c24\", \"bel\": \"bp(GO:\\\"GO:0072583\\\" ! \\\"clathrin-dependent endocytosis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"mucociliary clearance\", \"identifier\": \"GO:0120197\"}, \"id\": \"f08c06ed1000d0b6695de3198a8eadcb\", \"bel\": \"bp(GO:\\\"GO:0120197\\\" ! \\\"mucociliary clearance\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"cell-cell fusion\", \"identifier\": \"GO:0140253\"}, \"id\": \"47e5c6dc5af573dc9402e6cb89610d40\", \"bel\": \"bp(GO:\\\"GO:0140253\\\" ! \\\"cell-cell fusion\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"positive regulation of leukocyte cell-cell adhesion\", \"identifier\": \"GO:1903039\"}, \"id\": \"18dd4b1d200c954e8b0aad08cdc3bc01\", \"bel\": \"bp(GO:\\\"GO:1903039\\\" ! \\\"positive regulation of leukocyte cell-cell adhesion\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"GO\", \"name\": \"macrophage migration\", \"identifier\": \"GO:1905517\"}, \"id\": \"80bba009e15b7b3c726c1a73b7f49410\", \"bel\": \"bp(GO:\\\"GO:1905517\\\" ! \\\"macrophage migration\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"FPS-ZM1\", \"identifier\": \"C572629\"}, \"id\": \"89455a32bbd879986e7421dc0e5bf796\", \"bel\": \"bp(MESH:C572629 ! FPS-ZM1)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Alarmins\", \"identifier\": \"D000067531\"}, \"id\": \"ec24e79c4208aceb6e6538314ae547c8\", \"bel\": \"bp(MESH:D000067531 ! Alarmins)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Bevacizumab\", \"identifier\": \"D000068258\"}, \"id\": \"859ac52a2dd0cb6ef3d98766045503af\", \"bel\": \"bp(MESH:D000068258 ! Bevacizumab)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Workforce\", \"identifier\": \"D000078329\"}, \"id\": \"a4d551e6c2ebc9af366f6fcc802a5a90\", \"bel\": \"bp(MESH:D000078329 ! Workforce)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Data Analysis\", \"identifier\": \"D000078332\"}, \"id\": \"22766ac965e73b13db150a2c9c034218\", \"bel\": \"bp(MESH:D000078332 ! \\\"Data Analysis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Cytokine Release Syndrome\", \"identifier\": \"D000080424\"}, \"id\": \"0d10539070441eb578ec26183165ef70\", \"bel\": \"bp(MESH:D000080424 ! \\\"Cytokine Release Syndrome\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"COVID-19\", \"identifier\": \"D000086382\"}, \"id\": \"6ff4934f632b2d764b0a1197437068c4\", \"bel\": \"bp(MESH:D000086382 ! COVID-19)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"SARS-CoV-2\", \"identifier\": \"D000086402\"}, \"id\": \"74f15ffd23d837cf0e1c82fa76cdb6ba\", \"bel\": \"bp(MESH:D000086402 ! SARS-CoV-2)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Aging\", \"identifier\": \"D000375\"}, \"id\": \"75858cbb6999662b14f011f96fe7f688\", \"bel\": \"bp(MESH:D000375 ! Aging)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Albuminuria\", \"identifier\": \"D000419\"}, \"id\": \"225b466cb80cdf15cc23d8ec313882b8\", \"bel\": \"bp(MESH:D000419 ! Albuminuria)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Asia\", \"identifier\": \"D001208\"}, \"id\": \"275c00da97eced620317065c75f0faca\", \"bel\": \"bp(MESH:D001208 ! Asia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Asthma\", \"identifier\": \"D001249\"}, \"id\": \"61c21ed353275ff04365ccf64d7aac22\", \"bel\": \"bp(MESH:D001249 ! Asthma)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"B-Lymphocytes\", \"identifier\": \"D001402\"}, \"id\": \"d48ba526a1cdba220bfd34a773540b52\", \"bel\": \"bp(MESH:D001402 ! B-Lymphocytes)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Bacteria\", \"identifier\": \"D001419\"}, \"id\": \"523efb3c55e153822e74fa80e0b6e8c8\", \"bel\": \"bp(MESH:D001419 ! Bacteria)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Bacterial Infections\", \"identifier\": \"D001424\"}, \"id\": \"a44d77c98433ef76c83f454a06c8c8ae\", \"bel\": \"bp(MESH:D001424 ! \\\"Bacterial Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Blood\", \"identifier\": \"D001769\"}, \"id\": \"adcd8ca905df02ec7cae703e0830f4b7\", \"bel\": \"bp(MESH:D001769 ! Blood)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Blood Glucose\", \"identifier\": \"D001786\"}, \"id\": \"978ea3b6d781b710d3eec33187a2a10e\", \"bel\": \"bp(MESH:D001786 ! \\\"Blood Glucose\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Blood Platelets\", \"identifier\": \"D001792\"}, \"id\": \"b77dace474f32a42f1410505be7df060\", \"bel\": \"bp(MESH:D001792 ! \\\"Blood Platelets\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Body Weight\", \"identifier\": \"D001835\"}, \"id\": \"4dc97ca81294337cc2fbfb0c4e05a1ff\", \"bel\": \"bp(MESH:D001835 ! \\\"Body Weight\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Bronchiolitis\", \"identifier\": \"D001988\"}, \"id\": \"bb0ac908bca97614e02280cf8f53a7aa\", \"bel\": \"bp(MESH:D001988 ! Bronchiolitis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Cell Survival\", \"identifier\": \"D002470\"}, \"id\": \"88ccd0b6c5e6c0a3c2e41c3108fb0ff8\", \"bel\": \"bp(MESH:D002470 ! \\\"Cell Survival\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Chiroptera\", \"identifier\": \"D002685\"}, \"id\": \"841a190552fa6a494bf11665cc1dd992\", \"bel\": \"bp(MESH:D002685 ! Chiroptera)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Cystic Fibrosis\", \"identifier\": \"D003550\"}, \"id\": \"9a80b49b0af59541746f815923d7023d\", \"bel\": \"bp(MESH:D003550 ! \\\"Cystic Fibrosis\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Death\", \"identifier\": \"D003643\"}, \"id\": \"b339f2cc47be5b2da4a77ce4b77791fa\", \"bel\": \"bp(MESH:D003643 ! Death)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Dendritic Cells\", \"identifier\": \"D003713\"}, \"id\": \"e81c346b6ab842d76b679da4379b71e7\", \"bel\": \"bp(MESH:D003713 ! \\\"Dendritic Cells\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Dermatomyositis\", \"identifier\": \"D003882\"}, \"id\": \"f2cac7666cf8e7295dcff86a6797192a\", \"bel\": \"bp(MESH:D003882 ! Dermatomyositis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Diabetes Mellitus, Type 2\", \"identifier\": \"D003924\"}, \"id\": \"5a828cfe7669421a988c2c9316559467\", \"bel\": \"bp(MESH:D003924 ! \\\"Diabetes Mellitus, Type 2\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Diabetic Nephropathies\", \"identifier\": \"D003928\"}, \"id\": \"12f88ab1f8b55a6b2953dcfc150efac4\", \"bel\": \"bp(MESH:D003928 ! \\\"Diabetic Nephropathies\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Disease\", \"identifier\": \"D004194\"}, \"id\": \"c4555233ddc096471887fb5a4205a457\", \"bel\": \"bp(MESH:D004194 ! Disease)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Endothelium\", \"identifier\": \"D004727\"}, \"id\": \"fb7b961fcb958b089248f923a23e52b8\", \"bel\": \"bp(MESH:D004727 ! Endothelium)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Endothelium, Vascular\", \"identifier\": \"D004730\"}, \"id\": \"d8288298cb5110e83b094e3f8d9645d3\", \"bel\": \"bp(MESH:D004730 ! \\\"Endothelium, Vascular\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Epithelial Cells\", \"identifier\": \"D004847\"}, \"id\": \"c437a53c02b7e7e20fcb16171a176ae5\", \"bel\": \"bp(MESH:D004847 ! \\\"Epithelial Cells\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Erythrocyte Count\", \"identifier\": \"D004906\"}, \"id\": \"0359e5f24181a1bd0ca23923ff54060a\", \"bel\": \"bp(MESH:D004906 ! \\\"Erythrocyte Count\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Erythrocytes\", \"identifier\": \"D004912\"}, \"id\": \"b30aab19cd2662365040439fd3eaa6cc\", \"bel\": \"bp(MESH:D004912 ! Erythrocytes)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Fibrosis\", \"identifier\": \"D005355\"}, \"id\": \"d1c15a28ccd697a9ef6ff347b1a6a4d7\", \"bel\": \"bp(MESH:D005355 ! Fibrosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Government\", \"identifier\": \"D006076\"}, \"id\": \"dc395ee1658588eb359566c5e274b8ea\", \"bel\": \"bp(MESH:D006076 ! Government)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Heart Failure\", \"identifier\": \"D006333\"}, \"id\": \"bad3183ac05b0dfd18d3aa643a948518\", \"bel\": \"bp(MESH:D006333 ! \\\"Heart Failure\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hemagglutinins\", \"identifier\": \"D006388\"}, \"id\": \"e08fc8963c4cb22940333ab1a09ca414\", \"bel\": \"bp(MESH:D006388 ! Hemagglutinins)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Heparin, Low-Molecular-Weight\", \"identifier\": \"D006495\"}, \"id\": \"8ca37e1013e60580bf1313c091980073\", \"bel\": \"bp(MESH:D006495 ! \\\"Heparin, Low-Molecular-Weight\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hospitalization\", \"identifier\": \"D006760\"}, \"id\": \"9cd6553f74e1379e9a222700e80ddd34\", \"bel\": \"bp(MESH:D006760 ! Hospitalization)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hydrogen-Ion Concentration\", \"identifier\": \"D006863\"}, \"id\": \"63f690c0a49797d1535941149a2a3c23\", \"bel\": \"bp(MESH:D006863 ! \\\"Hydrogen-Ion Concentration\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hyperglycemia\", \"identifier\": \"D006943\"}, \"id\": \"66d42862f7df77f5778d00854b3a1115\", \"bel\": \"bp(MESH:D006943 ! Hyperglycemia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hyperplasia\", \"identifier\": \"D006965\"}, \"id\": \"bf716e6322e4049870db729076c7eb5a\", \"bel\": \"bp(MESH:D006965 ! Hyperplasia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hypertension\", \"identifier\": \"D006973\"}, \"id\": \"2f81af8a0ccd0c4735e3a3e71229f77d\", \"bel\": \"bp(MESH:D006973 ! Hypertension)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hypertrophy\", \"identifier\": \"D006984\"}, \"id\": \"72003954205bba20782d719c5c15e980\", \"bel\": \"bp(MESH:D006984 ! Hypertrophy)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hypoglycemia\", \"identifier\": \"D007003\"}, \"id\": \"11b579d0a70f84e82a427e8c8634aebe\", \"bel\": \"bp(MESH:D007003 ! Hypoglycemia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hypokalemia\", \"identifier\": \"D007008\"}, \"id\": \"d283aeb0fb60537037d189c8c9a46853\", \"bel\": \"bp(MESH:D007008 ! Hypokalemia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Immune System\", \"identifier\": \"D007107\"}, \"id\": \"cd3d5fd109b25d51e62a4bf5df92f369\", \"bel\": \"bp(MESH:D007107 ! \\\"Immune System\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Immunization\", \"identifier\": \"D007114\"}, \"id\": \"80cd53cc3a1e2a4c8bbb7ca2500c13e7\", \"bel\": \"bp(MESH:D007114 ! Immunization)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Immune System Diseases\", \"identifier\": \"D007154\"}, \"id\": \"abda8cde6bff76288216980b12db06c0\", \"bel\": \"bp(MESH:D007154 ! \\\"Immune System Diseases\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Infections\", \"identifier\": \"D007239\"}, \"id\": \"c5ac9f4bb9518f6dd82c4147d9b32389\", \"bel\": \"bp(MESH:D007239 ! Infections)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Influenza, Human\", \"identifier\": \"D007251\"}, \"id\": \"9d6b87d321d3e81e30e5d529f664d694\", \"bel\": \"bp(MESH:D007251 ! \\\"Influenza, Human\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Intensive Care Units\", \"identifier\": \"D007362\"}, \"id\": \"2eba3f068f658257348fbaf61b7aff6a\", \"bel\": \"bp(MESH:D007362 ! \\\"Intensive Care Units\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Ketosis\", \"identifier\": \"D007662\"}, \"id\": \"8fc7e351f1c456b60d82fd7ba36a3c38\", \"bel\": \"bp(MESH:D007662 ! Ketosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Kidney\", \"identifier\": \"D007668\"}, \"id\": \"99fe06fce4707ed032b450bb0ef72e80\", \"bel\": \"bp(MESH:D007668 ! Kidney)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Leukocytes\", \"identifier\": \"D007962\"}, \"id\": \"65c6aee56d235344699897784183e536\", \"bel\": \"bp(MESH:D007962 ! Leukocytes)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lipid Metabolism, Inborn Errors\", \"identifier\": \"D008052\"}, \"id\": \"f37dbb629b8ae6ba0069cb254f24bd9b\", \"bel\": \"bp(MESH:D008052 ! \\\"Lipid Metabolism, Inborn Errors\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lipidoses\", \"identifier\": \"D008064\"}, \"id\": \"2ceed9c0fec6a0a56bbbaf43de5ecd4d\", \"bel\": \"bp(MESH:D008064 ! Lipidoses)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lung\", \"identifier\": \"D008168\"}, \"id\": \"2a3c9faef32d20d7cd1e8b0f07130dd2\", \"bel\": \"bp(MESH:D008168 ! Lung)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lymphopenia\", \"identifier\": \"D008231\"}, \"id\": \"6b837bda4a255c93b962e3c1173e0852\", \"bel\": \"bp(MESH:D008231 ! Lymphopenia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Macrophages\", \"identifier\": \"D008264\"}, \"id\": \"87089445bae1fbb1b8496f5107a67cff\", \"bel\": \"bp(MESH:D008264 ! Macrophages)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Monocytes\", \"identifier\": \"D009000\"}, \"id\": \"2b604d36bc45f1696e73d941c4463fbb\", \"bel\": \"bp(MESH:D009000 ! Monocytes)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Mortality\", \"identifier\": \"D009026\"}, \"id\": \"812f198896c2bd3812e5d23cc5168e83\", \"bel\": \"bp(MESH:D009026 ! Mortality)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Mucus\", \"identifier\": \"D009093\"}, \"id\": \"48aa81ca1dfdf6b2267a563ca8495cc6\", \"bel\": \"bp(MESH:D009093 ! Mucus)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Neuraminidase\", \"identifier\": \"D009439\"}, \"id\": \"ed1cc97e84df460e333f0927698ddc88\", \"bel\": \"bp(MESH:D009439 ! Neuraminidase)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Neutrophils\", \"identifier\": \"D009504\"}, \"id\": \"a4be5f502c04534a7741a30c79c8f4e5\", \"bel\": \"bp(MESH:D009504 ! Neutrophils)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Obesity\", \"identifier\": \"D009765\"}, \"id\": \"c7c94dfe4870e0ed3c0463f6623bf35a\", \"bel\": \"bp(MESH:D009765 ! Obesity)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Oxygen Consumption\", \"identifier\": \"D010101\"}, \"id\": \"b4b7b484c075eb2a8ee9510eca152789\", \"bel\": \"bp(MESH:D010101 ! \\\"Oxygen Consumption\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Permeability\", \"identifier\": \"D010539\"}, \"id\": \"347e47dc1e3a3d7ade4d3109a706177c\", \"bel\": \"bp(MESH:D010539 ! Permeability)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Physiology\", \"identifier\": \"D010827\"}, \"id\": \"9eab84ba4dd82d0edb064e5c43f10df5\", \"bel\": \"bp(MESH:D010827 ! Physiology)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Plasminogen Activators\", \"identifier\": \"D010960\"}, \"id\": \"98435ac7724c35d577ce2341304e394c\", \"bel\": \"bp(MESH:D010960 ! \\\"Plasminogen Activators\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Pneumonia\", \"identifier\": \"D011014\"}, \"id\": \"c638ff76bc3940e9ce8cca496dd329b2\", \"bel\": \"bp(MESH:D011014 ! Pneumonia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Renin-Angiotensin System\", \"identifier\": \"D012084\"}, \"id\": \"59ff1f7a52b4d31e86eb2badba72ba44\", \"bel\": \"bp(MESH:D012084 ! \\\"Renin-Angiotensin System\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Research\", \"identifier\": \"D012106\"}, \"id\": \"6b1a190e5f2871a582aa916e26e15f6b\", \"bel\": \"bp(MESH:D012106 ! Research)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Respiratory Distress Syndrome\", \"identifier\": \"D012128\"}, \"id\": \"a0768dda16757b6f2082b03992c86d47\", \"bel\": \"bp(MESH:D012128 ! \\\"Respiratory Distress Syndrome\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Respiratory Insufficiency\", \"identifier\": \"D012131\"}, \"id\": \"fe5de2b9580e0cc7a16b10dcc9381374\", \"bel\": \"bp(MESH:D012131 ! \\\"Respiratory Insufficiency\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Respiratory Tract Infections\", \"identifier\": \"D012141\"}, \"id\": \"aaa593f64c6955e5a95ed37f500ed694\", \"bel\": \"bp(MESH:D012141 ! \\\"Respiratory Tract Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Rhinovirus\", \"identifier\": \"D012229\"}, \"id\": \"2f5819fbc91bc964f6ec3b8215edc1ff\", \"bel\": \"bp(MESH:D012229 ! Rhinovirus)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"RNA, Viral\", \"identifier\": \"D012367\"}, \"id\": \"23ae43b93dc595d4437a0f29fd240edd\", \"bel\": \"bp(MESH:D012367 ! \\\"RNA, Viral\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Sex\", \"identifier\": \"D012723\"}, \"id\": \"e73dddab992988afafbd1ed40809e450\", \"bel\": \"bp(MESH:D012723 ! Sex)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Sick Sinus Syndrome\", \"identifier\": \"D012804\"}, \"id\": \"d5942a1573abc8dea311758709fe2025\", \"bel\": \"bp(MESH:D012804 ! \\\"Sick Sinus Syndrome\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Smoking\", \"identifier\": \"D012907\"}, \"id\": \"1b717004b5c0eddf5f7967a5718a6809\", \"bel\": \"bp(MESH:D012907 ! Smoking)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Syndrome\", \"identifier\": \"D013577\"}, \"id\": \"d789d72493130dca3a338692c2e86c76\", \"bel\": \"bp(MESH:D013577 ! Syndrome)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"T-Lymphocytes\", \"identifier\": \"D013601\"}, \"id\": \"cf8c2dc4ebbdf1ca716f485e29e60b16\", \"bel\": \"bp(MESH:D013601 ! T-Lymphocytes)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Thrombosis\", \"identifier\": \"D013927\"}, \"id\": \"e5adee2a5a4b75e2913fb6c690bb9978\", \"bel\": \"bp(MESH:D013927 ! Thrombosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Vaccination\", \"identifier\": \"D014611\"}, \"id\": \"bf39dd71bf9793ab7e160b812aaa2736\", \"bel\": \"bp(MESH:D014611 ! Vaccination)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Virulence\", \"identifier\": \"D014774\"}, \"id\": \"2f6f6a33085c264b334d8710b01d05f8\", \"bel\": \"bp(MESH:D014774 ! Virulence)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Virus Diseases\", \"identifier\": \"D014777\"}, \"id\": \"df73b587787f44cc6e277c1ecbcf9662\", \"bel\": \"bp(MESH:D014777 ! \\\"Virus Diseases\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Virus Replication\", \"identifier\": \"D014779\"}, \"id\": \"8dbdc395344b7766a4b0215c50852200\", \"bel\": \"bp(MESH:D014779 ! \\\"Virus Replication\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Viruses\", \"identifier\": \"D014780\"}, \"id\": \"0213f7b9c70aab83280ffdfa71c77843\", \"bel\": \"bp(MESH:D014780 ! Viruses)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Zinc\", \"identifier\": \"D015032\"}, \"id\": \"c72ee1d5d2bd252f7c22a89bb7a1d18f\", \"bel\": \"bp(MESH:D015032 ! Zinc)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Hypertriglyceridemia\", \"identifier\": \"D015228\"}, \"id\": \"1ecd2df2bb7220a97afe6edcef09c598\", \"bel\": \"bp(MESH:D015228 ! Hypertriglyceridemia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"HIV Infections\", \"identifier\": \"D015658\"}, \"id\": \"4fbc7492bcd965cc17ec0c2ee15d88e4\", \"bel\": \"bp(MESH:D015658 ! \\\"HIV Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Comorbidity\", \"identifier\": \"D015897\"}, \"id\": \"4c76f6a62735f8189594aec70522eb5a\", \"bel\": \"bp(MESH:D015897 ! Comorbidity)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Review\", \"identifier\": \"D016454\"}, \"id\": \"279e6e8146c1c93f25ff6a05f77322bd\", \"bel\": \"bp(MESH:D016454 ! Review)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Antigen Presentation\", \"identifier\": \"D017951\"}, \"id\": \"ad3b0555d4d842b3b4b8e1833914b161\", \"bel\": \"bp(MESH:D017951 ! \\\"Antigen Presentation\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Respiratory Syncytial Virus, Human\", \"identifier\": \"D018113\"}, \"id\": \"5539f73a268a8daa8cce4c873215c6a6\", \"bel\": \"bp(MESH:D018113 ! \\\"Respiratory Syncytial Virus, Human\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Arterivirus Infections\", \"identifier\": \"D018174\"}, \"id\": \"506473fa89d068bc9da66ba17373a89f\", \"bel\": \"bp(MESH:D018174 ! \\\"Arterivirus Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Oxidative Stress\", \"identifier\": \"D018384\"}, \"id\": \"61dc9edbaa04628a578be682c15d2f73\", \"bel\": \"bp(MESH:D018384 ! \\\"Oxidative Stress\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Sepsis\", \"identifier\": \"D018805\"}, \"id\": \"aab98a4f0099c58af215a118c50f4300\", \"bel\": \"bp(MESH:D018805 ! Sepsis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Bronchoalveolar Lavage\", \"identifier\": \"D018893\"}, \"id\": \"79ef9c75827bf46768ea2ac552bd7a3a\", \"bel\": \"bp(MESH:D018893 ! \\\"Bronchoalveolar Lavage\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Endotoxemia\", \"identifier\": \"D019446\"}, \"id\": \"2a078a78053622b58c6cfd2f37f588e5\", \"bel\": \"bp(MESH:D019446 ! Endotoxemia)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Goblet Cells\", \"identifier\": \"D020397\"}, \"id\": \"cb7bffb9b6ad351c91cdfe063a1f94d3\", \"bel\": \"bp(MESH:D020397 ! \\\"Goblet Cells\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Collection\", \"identifier\": \"D020471\"}, \"id\": \"2cce4a06efd4a605c4cd35271b7a4dc5\", \"bel\": \"bp(MESH:D020471 ! Collection)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Metabolic Syndrome\", \"identifier\": \"D024821\"}, \"id\": \"b32d18702812612f3af821e69f185bfa\", \"bel\": \"bp(MESH:D024821 ! \\\"Metabolic Syndrome\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Coronavirus OC43, Human\", \"identifier\": \"D028962\"}, \"id\": \"87ad198d96e5adc339a2ea8c096df352\", \"bel\": \"bp(MESH:D028962 ! \\\"Coronavirus OC43, Human\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Herbal Medicine\", \"identifier\": \"D029001\"}, \"id\": \"0f828a61951a33a0d0d7552b7b861f56\", \"bel\": \"bp(MESH:D029001 ! \\\"Herbal Medicine\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Pulmonary Disease, Chronic Obstructive\", \"identifier\": \"D029424\"}, \"id\": \"b5df2753138f04c5429efe763ce2092e\", \"bel\": \"bp(MESH:D029424 ! \\\"Pulmonary Disease, Chronic Obstructive\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Papillomavirus Infections\", \"identifier\": \"D030361\"}, \"id\": \"46826dffa677b31fb6a24a48b2914459\", \"bel\": \"bp(MESH:D030361 ! \\\"Papillomavirus Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Endothelial Cells\", \"identifier\": \"D042783\"}, \"id\": \"41e326e8c38fe71a7efb57fcb2353977\", \"bel\": \"bp(MESH:D042783 ! \\\"Endothelial Cells\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Receptor, PAR-2\", \"identifier\": \"D044464\"}, \"id\": \"f8362ef319f1484fcc826e92687bc563\", \"bel\": \"bp(MESH:D044464 ! \\\"Receptor, PAR-2\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Severe Acute Respiratory Syndrome\", \"identifier\": \"D045169\"}, \"id\": \"87ef41c787bd99565289e3214ab105f7\", \"bel\": \"bp(MESH:D045169 ! \\\"Severe Acute Respiratory Syndrome\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"SARS Virus\", \"identifier\": \"D045473\"}, \"id\": \"d615ecd5a77b71cf5af9dc3ede76cdfd\", \"bel\": \"bp(MESH:D045473 ! \\\"SARS Virus\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Metalloproteases\", \"identifier\": \"D045726\"}, \"id\": \"7d986a7a74a0d7d3490ac4c9f45bc552\", \"bel\": \"bp(MESH:D045726 ! Metalloproteases)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Angiotensin II Type 1 Receptor Blockers\", \"identifier\": \"D047228\"}, \"id\": \"dea16f135ffd62282dcad5cf9690e2cd\", \"bel\": \"bp(MESH:D047228 ! \\\"Angiotensin II Type 1 Receptor Blockers\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Diabetes Complications\", \"identifier\": \"D048909\"}, \"id\": \"647c883c19a26809b1f7b11de64b3cf1\", \"bel\": \"bp(MESH:D048909 ! \\\"Diabetes Complications\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lipogenesis\", \"identifier\": \"D050155\"}, \"id\": \"9197bff8bdbf81aae60079c7af73ec4e\", \"bel\": \"bp(MESH:D050155 ! Lipogenesis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Adipogenesis\", \"identifier\": \"D050156\"}, \"id\": \"780e6c400380c71d0aae3ae938213c5c\", \"bel\": \"bp(MESH:D050156 ! Adipogenesis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Atherosclerosis\", \"identifier\": \"D050197\"}, \"id\": \"35be788d303c7cd6df844ed1d71fbaaa\", \"bel\": \"bp(MESH:D050197 ! Atherosclerosis)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Mice\", \"identifier\": \"D051379\"}, \"id\": \"43c932168e762d394d16c59f477faf01\", \"bel\": \"bp(MESH:D051379 ! Mice)\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lipid Metabolism Disorders\", \"identifier\": \"D052439\"}, \"id\": \"9f3eeaebb9bb44912bf5554a4332bdc6\", \"bel\": \"bp(MESH:D052439 ! \\\"Lipid Metabolism Disorders\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Virus Attachment\", \"identifier\": \"D053585\"}, \"id\": \"1e380605b970e5c820083a4d0b0977fe\", \"bel\": \"bp(MESH:D053585 ! \\\"Virus Attachment\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Virus Internalization\", \"identifier\": \"D053586\"}, \"id\": \"700e9180ae878eb48d5ecaf206ba2c6a\", \"bel\": \"bp(MESH:D053586 ! \\\"Virus Internalization\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Receptors, CCR\", \"identifier\": \"D054388\"}, \"id\": \"5e26ab306b40c18d7647567e0bd9bc1e\", \"bel\": \"bp(MESH:D054388 ! \\\"Receptors, CCR\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Lung Injury\", \"identifier\": \"D055370\"}, \"id\": \"3886e58e54716a79993f2eebe1c03776\", \"bel\": \"bp(MESH:D055370 ! \\\"Lung Injury\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Acute Lung Injury\", \"identifier\": \"D055371\"}, \"id\": \"e9c5b982765fe934418d675e251a7a5d\", \"bel\": \"bp(MESH:D055371 ! \\\"Acute Lung Injury\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Obesity, Abdominal\", \"identifier\": \"D056128\"}, \"id\": \"0661a589bb04bbc911ac7d600b6a8b2b\", \"bel\": \"bp(MESH:D056128 ! \\\"Obesity, Abdominal\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Airway Remodeling\", \"identifier\": \"D056151\"}, \"id\": \"595f0fd84ad346f2cf72cd2f29703702\", \"bel\": \"bp(MESH:D056151 ! \\\"Airway Remodeling\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Adaptive Immunity\", \"identifier\": \"D056704\"}, \"id\": \"61836f1274c6e9c5a0e22bf7c338bdcd\", \"bel\": \"bp(MESH:D056704 ! \\\"Adaptive Immunity\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Plaque, Atherosclerotic\", \"identifier\": \"D058226\"}, \"id\": \"3acd03078f532700dfb7fbcced5d9dc8\", \"bel\": \"bp(MESH:D058226 ! \\\"Plaque, Atherosclerotic\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Asymptomatic Infections\", \"identifier\": \"D058345\"}, \"id\": \"b72403eada3fb81c0528382f59ba4d95\", \"bel\": \"bp(MESH:D058345 ! \\\"Asymptomatic Infections\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Mesenchymal Stem Cells\", \"identifier\": \"D059630\"}, \"id\": \"d7b2e67b562b7e93e147f9b624164004\", \"bel\": \"bp(MESH:D059630 ! \\\"Mesenchymal Stem Cells\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Extracellular Traps\", \"identifier\": \"D065206\"}, \"id\": \"00033bf872480a0014acef7dae885524\", \"bel\": \"bp(MESH:D065206 ! \\\"Extracellular Traps\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Middle East Respiratory Syndrome Coronavirus\", \"identifier\": \"D065207\"}, \"id\": \"d3c788d2e4215cdcbcf5464d331dd480\", \"bel\": \"bp(MESH:D065207 ! \\\"Middle East Respiratory Syndrome Coronavirus\\\")\"}, {\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Cardiotoxicity\", \"identifier\": \"D066126\"}, \"id\": \"7e1b84b017aaeafdd746866452ab577d\", \"bel\": \"bp(MESH:D066126 ! Cardiotoxicity)\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"mannose\", \"identifier\": \"37684\"}, \"id\": \"059433d3750fa25ccfe15d61416d6005\", \"bel\": \"a(CHEBI:37684 ! mannose)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SFTPD\", \"identifier\": \"10803\"}, \"id\": \"904208f32d66c6acf2457f715888b299\", \"bel\": \"p(HGNC:10803 ! SFTPD)\"}], \"id\": \"d2da42c3015c1fe998694382897f51a5\", \"bel\": \"complex(a(CHEBI:37684 ! mannose), p(HGNC:10803 ! SFTPD))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"ligand\", \"identifier\": \"52214\"}, \"id\": \"90f915b7d9d69ddcc9a832257c7b180e\", \"bel\": \"a(CHEBI:52214 ! ligand)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MBL2\", \"identifier\": \"6922\"}, \"id\": \"99ef80d2eb6aed8478400ee0a70b6a13\", \"bel\": \"p(HGNC:6922 ! MBL2)\"}], \"id\": \"39f1b0364fac4fa3fb71016e1be0e23b\", \"bel\": \"complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"N-glycan\", \"identifier\": \"59520\"}, \"id\": \"56d91fb40fcf96fb21e09d91f780b33d\", \"bel\": \"a(CHEBI:59520 ! N-glycan)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LGALS1\", \"identifier\": \"6561\"}, \"id\": \"59e0b7676da1bb33951c5cf85eb417df\", \"bel\": \"p(HGNC:6561 ! LGALS1)\"}], \"id\": \"879e4d2ed94edaa4f91b3c7f7e44cad8\", \"bel\": \"complex(a(CHEBI:59520 ! N-glycan), p(HGNC:6561 ! LGALS1))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"concept\": {\"namespace\": \"CHEBI\", \"name\": \"N-glycan\", \"identifier\": \"59520\"}, \"id\": \"56d91fb40fcf96fb21e09d91f780b33d\", \"bel\": \"a(CHEBI:59520 ! N-glycan)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P84849\", \"identifier\": \"P84849\"}, \"id\": \"a24bcf83fef6b060de08f41c2c07ec0b\", \"bel\": \"p(UP:P84849 ! P84849)\"}], \"id\": \"a63eea3dd87f473f25de72b73ba0e727\", \"bel\": \"complex(a(CHEBI:59520 ! N-glycan), p(UP:P84849 ! P84849))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"BiologicalProcess\", \"concept\": {\"namespace\": \"MESH\", \"name\": \"Extracellular Traps\", \"identifier\": \"D065206\"}, \"id\": \"00033bf872480a0014acef7dae885524\", \"bel\": \"bp(MESH:D065206 ! \\\"Extracellular Traps\\\")\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"VWF\", \"identifier\": \"12726\"}, \"id\": \"0b7135301ae95d3b502556e2eb8491a9\", \"bel\": \"p(HGNC:12726 ! VWF)\"}], \"id\": \"4435a570e7ddae79bda40adb014759bb\", \"bel\": \"complex(bp(MESH:D065206 ! \\\"Extracellular Traps\\\"), p(HGNC:12726 ! VWF))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"TLR\", \"identifier\": \"TLR\"}, \"id\": \"fc5a3de11a23b5758f4c1f0f1269b197\", \"bel\": \"p(FPLX:TLR ! TLR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGER\", \"identifier\": \"320\"}, \"id\": \"46efcef764e041272dc7e8cae07361e3\", \"bel\": \"p(HGNC:320 ! AGER)\"}], \"id\": \"52a5b6a58cf3cf7295dca1dba755a9ab\", \"bel\": \"complex(p(FPLX:TLR ! TLR), p(HGNC:320 ! AGER))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SFTPD\", \"identifier\": \"10803\"}, \"id\": \"904208f32d66c6acf2457f715888b299\", \"bel\": \"p(HGNC:10803 ! SFTPD)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SGCG\", \"identifier\": \"10809\"}, \"id\": \"493b81631e2ec94ca0d8f5d5b78d5953\", \"bel\": \"p(HGNC:10809 ! SGCG)\"}], \"id\": \"fd46199c99fc173db82d53aa15fa1706\", \"bel\": \"complex(p(HGNC:10803 ! SFTPD), p(HGNC:10809 ! SGCG))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SFTPD\", \"identifier\": \"10803\"}, \"id\": \"904208f32d66c6acf2457f715888b299\", \"bel\": \"p(HGNC:10803 ! SFTPD)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"neuraminidase\", \"identifier\": \"PF00064\"}, \"id\": \"43de6b48a4db01438c5effe244e90adc\", \"bel\": \"p(PFAM:PF00064 ! neuraminidase)\"}], \"id\": \"6b40a303bbb4666485da8b121e92693c\", \"bel\": \"complex(p(HGNC:10803 ! SFTPD), p(PFAM:PF00064 ! neuraminidase))\"}, {\"function\": \"Complex\", \"members\": [{\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CDH1\", \"identifier\": \"1748\"}, \"id\": \"bb542da7359289d2170848e94eb9e099\", \"bel\": \"p(HGNC:1748 ! CDH1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"EGFR\", \"identifier\": \"3236\"}, \"id\": \"4221d410040b17a4751ba254380887c0\", \"bel\": \"p(HGNC:3236 ! EGFR)\"}], \"id\": \"c55379dc23bea7eb53364becbc13b2da\", \"bel\": \"complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Chemokine_receptor\", \"identifier\": \"Chemokine_receptor\"}, \"id\": \"8a71081b41e5494751625eb24dc1b8a7\", \"bel\": \"p(FPLX:\\\"Chemokine_receptor\\\" ! \\\"Chemokine_receptor\\\")\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"E3_Ub_ligase\", \"identifier\": \"E3_Ub_ligase\"}, \"id\": \"b9d782e005a5a1ccc06d29bc9a070d40\", \"bel\": \"p(FPLX:\\\"E3_Ub_ligase\\\" ! \\\"E3_Ub_ligase\\\")\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"MAC_1\", \"identifier\": \"MAC_1\"}, \"id\": \"0848c8abb4f8aa2fb3c1aed841e73d09\", \"bel\": \"p(FPLX:\\\"MAC_1\\\" ! \\\"MAC_1\\\")\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"AKT\", \"identifier\": \"AKT\"}, \"id\": \"3844b6ec8ab2fbb869a5ab6c627a90cd\", \"bel\": \"p(FPLX:AKT ! AKT)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"AMPK\", \"identifier\": \"AMPK\"}, \"id\": \"67ff2f20162ead2ec39aed53befe6b0a\", \"bel\": \"p(FPLX:AMPK ! AMPK)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Actin\", \"identifier\": \"Actin\"}, \"id\": \"86eb6a23ee9ff21e5a351f047e8e305c\", \"bel\": \"p(FPLX:Actin ! Actin)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"CD16\", \"identifier\": \"CD16\"}, \"id\": \"93e163ebde2ad7646943e150aaef5a1a\", \"bel\": \"p(FPLX:CD16 ! CD16)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"ELA\", \"identifier\": \"ELA\"}, \"id\": \"eeafce683bb5d0a6fe54bf878cf9d0ba\", \"bel\": \"p(FPLX:ELA ! ELA)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Fibrin\", \"identifier\": \"Fibrin\"}, \"id\": \"fe00a164096d6ee3d27a2d016c4da962\", \"bel\": \"p(FPLX:Fibrin ! Fibrin)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Fibrinogen\", \"identifier\": \"Fibrinogen\"}, \"id\": \"08b0d88f7e49c28c5c43036e1554555b\", \"bel\": \"p(FPLX:Fibrinogen ! Fibrinogen)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"GPX\", \"identifier\": \"GPX\"}, \"id\": \"7ff3621174233f20f25b1e0a05b43da4\", \"bel\": \"p(FPLX:GPX ! GPX)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"GSK3\", \"identifier\": \"GSK3\"}, \"id\": \"add409f2d504cd6b9b4949e4af1d6771\", \"bel\": \"p(FPLX:GSK3 ! GSK3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Glycosyltransferase\", \"identifier\": \"Glycosyltransferase\"}, \"id\": \"bb2b9f8955c2c438cd1ae1b0f44cab45\", \"bel\": \"p(FPLX:Glycosyltransferase ! Glycosyltransferase)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Histone\", \"identifier\": \"Histone\"}, \"id\": \"65f0c40a508da580a3bb1bf0a2d47d5c\", \"bel\": \"p(FPLX:Histone ! Histone)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"IFNA\", \"identifier\": \"IFNA\"}, \"id\": \"abcba84089d4e260a8f86521f99da9c4\", \"bel\": \"p(FPLX:IFNA ! IFNA)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"IFNB\", \"identifier\": \"IFNB\"}, \"id\": \"c8a8eb4875f2b8ce8a75d9e433fbfbb4\", \"bel\": \"p(FPLX:IFNB ! IFNB)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"IL1\", \"identifier\": \"IL1\"}, \"id\": \"baf158b9e03b0f69186c4cb1b79c35d3\", \"bel\": \"p(FPLX:IL1 ! IL1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Interferon\", \"identifier\": \"Interferon\"}, \"id\": \"882a9af6bdf8c1c243dd31a2532a0ab3\", \"bel\": \"p(FPLX:Interferon ! Interferon)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"LDH\", \"identifier\": \"LDH\"}, \"id\": \"aaef8c653c39cd1a241834d3564a67ae\", \"bel\": \"p(FPLX:LDH ! LDH)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"MAPK\", \"identifier\": \"MAPK\"}, \"id\": \"c603f5676335d2312b2e8a35f17f8acb\", \"bel\": \"p(FPLX:MAPK ! MAPK)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"NFkappaB\", \"identifier\": \"NFkappaB\"}, \"id\": \"88d17b6331091fc80738548dc5cbba18\", \"bel\": \"p(FPLX:NFkappaB ! NFkappaB)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"PKC\", \"identifier\": \"PKC\"}, \"id\": \"69303dc315663989113f626d8ece6672\", \"bel\": \"p(FPLX:PKC ! PKC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Protease\", \"identifier\": \"Protease\"}, \"id\": \"7a074bb00b366b7690b5294909525a24\", \"bel\": \"p(FPLX:Protease ! Protease)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"Proteasome\", \"identifier\": \"Proteasome\"}, \"id\": \"73bfbe254590f6b4cff6bb7cc6d255bc\", \"bel\": \"p(FPLX:Proteasome ! Proteasome)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"RAF\", \"identifier\": \"RAF\"}, \"id\": \"52a43bb2dc81248ac6f4052a280f1148\", \"bel\": \"p(FPLX:RAF ! RAF)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"SOD\", \"identifier\": \"SOD\"}, \"id\": \"e4386516ba45b8b245a185bdbd8ffd86\", \"bel\": \"p(FPLX:SOD ! SOD)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"TGFB\", \"identifier\": \"TGFB\"}, \"id\": \"09d99ab676ec73be270afe74c438c059\", \"bel\": \"p(FPLX:TGFB ! TGFB)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"TLR\", \"identifier\": \"TLR\"}, \"id\": \"fc5a3de11a23b5758f4c1f0f1269b197\", \"bel\": \"p(FPLX:TLR ! TLR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"mTORC1\", \"identifier\": \"mTORC1\"}, \"id\": \"9d8fd01470389dfe005960a2f77cdee2\", \"bel\": \"p(FPLX:mTORC1 ! mTORC1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"mTORC2\", \"identifier\": \"mTORC2\"}, \"id\": \"d3e511a83dfa43c6b0a63b15a0747622\", \"bel\": \"p(FPLX:mTORC2 ! mTORC2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"FPLX\", \"name\": \"p38\", \"identifier\": \"p38\"}, \"id\": \"5ff92a93c479ed5b66de9e6ce1a8e22f\", \"bel\": \"p(FPLX:p38 ! p38)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SAT1\", \"identifier\": \"10540\"}, \"id\": \"a10f4fbe57a35882117d11fab35cb7de\", \"bel\": \"p(HGNC:10540 ! SAT1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CCL2\", \"identifier\": \"10618\"}, \"id\": \"3498e108f28145fd55ce54135ad27cd7\", \"bel\": \"p(HGNC:10618 ! CCL2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SELP\", \"identifier\": \"10721\"}, \"id\": \"a06c358b3acc95ba3f7274dc1d78fc7c\", \"bel\": \"p(HGNC:10721 ! SELP)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SELPLG\", \"identifier\": \"10722\"}, \"id\": \"065a702e19fd4f265fe785e33f0392a7\", \"bel\": \"p(HGNC:10722 ! SELPLG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SFTPA2\", \"identifier\": \"10799\"}, \"id\": \"c4566c94d31872f29f6d648ab2a7d3dd\", \"bel\": \"p(HGNC:10799 ! SFTPA2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SFTPD\", \"identifier\": \"10803\"}, \"id\": \"904208f32d66c6acf2457f715888b299\", \"bel\": \"p(HGNC:10803 ! SFTPD)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"GEMIN2\", \"identifier\": \"10884\"}, \"id\": \"5a0a92d7649b114ee9b7b6f2691ca950\", \"bel\": \"p(HGNC:10884 ! GEMIN2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLC2A1\", \"identifier\": \"11005\"}, \"id\": \"a5feafbefa5d920c4f7f2a1e0fac69b3\", \"bel\": \"p(HGNC:11005 ! SLC2A1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLC2A4\", \"identifier\": \"11009\"}, \"id\": \"03b0599148977bd296aa2d8874339732\", \"bel\": \"p(HGNC:11009 ! SLC2A4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLC6A2\", \"identifier\": \"11048\"}, \"id\": \"35e1664ca6c13f29c385a63c57c37c7b\", \"bel\": \"p(HGNC:11048 ! SLC6A2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLC9A1\", \"identifier\": \"11071\"}, \"id\": \"5859c7fc7ff356e4364a984580673769\", \"bel\": \"p(HGNC:11071 ! SLC9A1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLPI\", \"identifier\": \"11092\"}, \"id\": \"ee5531a2a129d831b0d8bf1360a980bc\", \"bel\": \"p(HGNC:11092 ! SLPI)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"BSG\", \"identifier\": \"1116\"}, \"id\": \"588c425e95f98ee0e7a4abd63a1b239c\", \"bel\": \"p(HGNC:1116 ! BSG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SOD1\", \"identifier\": \"11179\"}, \"id\": \"165412f24010694258452de0460afa64\", \"bel\": \"p(HGNC:11179 ! SOD1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SOD2\", \"identifier\": \"11180\"}, \"id\": \"a3ba99ce27d1d29c6b2a828dc9e9a9bf\", \"bel\": \"p(HGNC:11180 ! SOD2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SPP1\", \"identifier\": \"11255\"}, \"id\": \"8ab23b944b3a9343082f899c21fd5e0d\", \"bel\": \"p(HGNC:11255 ! SPP1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TRIM21\", \"identifier\": \"11312\"}, \"id\": \"50825e0b662b87f88f51577f6aeeda96\", \"bel\": \"p(HGNC:11312 ! TRIM21)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACLY\", \"identifier\": \"115\"}, \"id\": \"c9ab08917483a773b508472e0897d493\", \"bel\": \"p(HGNC:115 ! ACLY)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TFPI\", \"identifier\": \"11760\"}, \"id\": \"61e99601c265eb6367db3c47d84fdb10\", \"bel\": \"p(HGNC:11760 ! TFPI)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"THBS1\", \"identifier\": \"11785\"}, \"id\": \"30c70bca5dd08707360c9b7d5c17c6cd\", \"bel\": \"p(HGNC:11785 ! THBS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NKX2-1\", \"identifier\": \"11825\"}, \"id\": \"63cc9b768715c010b59cf2c3bc837638\", \"bel\": \"p(HGNC:11825 ! NKX2-1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TJP1\", \"identifier\": \"11827\"}, \"id\": \"0690c112be9ef023eb2484157d011dcb\", \"bel\": \"p(HGNC:11827 ! TJP1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TLR3\", \"identifier\": \"11849\"}, \"id\": \"750e4998ca45c42291e764bf4fec5f4b\", \"bel\": \"p(HGNC:11849 ! TLR3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TMPRSS2\", \"identifier\": \"11876\"}, \"id\": \"17787f8972c33154e75cae7657634d39\", \"bel\": \"p(HGNC:11876 ! TMPRSS2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TNF\", \"identifier\": \"11892\"}, \"id\": \"f1034233ec212f882bbb4fb6d68cccaa\", \"bel\": \"p(HGNC:11892 ! TNF)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"FASLG\", \"identifier\": \"11936\"}, \"id\": \"99d367083640ceb5c61b6424dc2ae6bd\", \"bel\": \"p(HGNC:11936 ! FASLG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TP53\", \"identifier\": \"11998\"}, \"id\": \"8b066f22dc5b48efae2893fcbbceba83\", \"bel\": \"p(HGNC:11998 ! TP53)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TWIST1\", \"identifier\": \"12428\"}, \"id\": \"8823bc14670684d951d7517e0e836c26\", \"bel\": \"p(HGNC:12428 ! TWIST1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"VHL\", \"identifier\": \"12687\"}, \"id\": \"df6a9c77cd25c9f1b1628dc72841076b\", \"bel\": \"p(HGNC:12687 ! VHL)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"VWF\", \"identifier\": \"12726\"}, \"id\": \"0b7135301ae95d3b502556e2eb8491a9\", \"bel\": \"p(HGNC:12726 ! VWF)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TRIM25\", \"identifier\": \"12932\"}, \"id\": \"5e6558d8981872a4c0da938d4dfad405\", \"bel\": \"p(HGNC:12932 ! TRIM25)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACE2\", \"identifier\": \"13557\"}, \"id\": \"97bb4990b7ce15318e3e41ec4b59055a\", \"bel\": \"p(HGNC:13557 ! ACE2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACE2\", \"identifier\": \"13557\"}, \"variants\": [{\"kind\": \"pmod\", \"concept\": {\"namespace\": \"go\", \"name\": \"protein phosphorylation\", \"identifier\": \"0006468\"}, \"code\": \"Ser\", \"pos\": 680}], \"id\": \"f9e5a145022fcf94a3aa075ffac9f4bb\", \"bel\": \"p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \\\"protein phosphorylation\\\", Ser, 680))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DEGS1\", \"identifier\": \"13709\"}, \"id\": \"69a5b5c8a3ce38cb11700b77fa57a801\", \"bel\": \"p(HGNC:13709 ! DEGS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL25\", \"identifier\": \"13765\"}, \"id\": \"1279d3fb2a4b4f25b6f24813ef6e074c\", \"bel\": \"p(HGNC:13765 ! IL25)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MYO1G\", \"identifier\": \"13880\"}, \"id\": \"ba901b95defe6a1f47c04f84bada4a71\", \"bel\": \"p(HGNC:13880 ! MYO1G)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CALR\", \"identifier\": \"1455\"}, \"id\": \"77e738e5a988158c50849a3ac3e696c3\", \"bel\": \"p(HGNC:1455 ! CALR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CANX\", \"identifier\": \"1473\"}, \"id\": \"963572fc8e947c938653925e147a1fd6\", \"bel\": \"p(HGNC:1473 ! CANX)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SIRT3\", \"identifier\": \"14931\"}, \"id\": \"efefa66cb6331cadb421009902d23460\", \"bel\": \"p(HGNC:14931 ! SIRT3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CASP1\", \"identifier\": \"1499\"}, \"id\": \"1ca299b189a934f6731610714742c708\", \"bel\": \"p(HGNC:1499 ! CASP1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CASP3\", \"identifier\": \"1504\"}, \"id\": \"6f0b259845dd93548cc167edb2514988\", \"bel\": \"p(HGNC:1504 ! CASP3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CAT\", \"identifier\": \"1516\"}, \"id\": \"40e773ee4cc522453b021b34b9f315c9\", \"bel\": \"p(HGNC:1516 ! CAT)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TLR7\", \"identifier\": \"15631\"}, \"id\": \"85eda6249ecf0d2d35b105d22c2cea01\", \"bel\": \"p(HGNC:15631 ! TLR7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL33\", \"identifier\": \"16028\"}, \"id\": \"405ecb6bdf2f6870254210a0b43e5543\", \"bel\": \"p(HGNC:16028 ! IL33)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CCR5\", \"identifier\": \"1606\"}, \"id\": \"0d6ba626eacf2a3955fe66e4134dd51e\", \"bel\": \"p(HGNC:1606 ! CCR5)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL17F\", \"identifier\": \"16404\"}, \"id\": \"53372fdb8d05dfed7d7fe3860acf936e\", \"bel\": \"p(HGNC:16404 ! IL17F)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CD209\", \"identifier\": \"1641\"}, \"id\": \"7095240f0b23748641cb531971211d78\", \"bel\": \"p(HGNC:1641 ! CD209)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CD4\", \"identifier\": \"1678\"}, \"id\": \"f72d88243ec816c074bd6f5aafaf5b24\", \"bel\": \"p(HGNC:1678 ! CD4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SH2D3C\", \"identifier\": \"16884\"}, \"id\": \"6ef46886381cf981119e962de493bb95\", \"bel\": \"p(HGNC:16884 ! SH2D3C)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CD86\", \"identifier\": \"1705\"}, \"id\": \"4babceebd0c1498e9a7d4f959b885823\", \"bel\": \"p(HGNC:1705 ! CD86)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SPDEF\", \"identifier\": \"17257\"}, \"id\": \"fdc3dc0263094125162e8c28df58c4b5\", \"bel\": \"p(HGNC:17257 ! SPDEF)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RCHY1\", \"identifier\": \"17479\"}, \"id\": \"0ce93521122a116d218efc368b75445e\", \"bel\": \"p(HGNC:17479 ! RCHY1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CDH1\", \"identifier\": \"1748\"}, \"id\": \"bb542da7359289d2170848e94eb9e099\", \"bel\": \"p(HGNC:1748 ! CDH1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RNF130\", \"identifier\": \"18280\"}, \"id\": \"2dd0cd71c6e8e491bb3a1f754d6c584c\", \"bel\": \"p(HGNC:18280 ! RNF130)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DHRS2\", \"identifier\": \"18349\"}, \"id\": \"7eba6e7f2aa702c73a205e7668233620\", \"bel\": \"p(HGNC:18349 ! DHRS2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CFTR\", \"identifier\": \"1884\"}, \"id\": \"66c11348e68803577111a9ba04fa1c8d\", \"bel\": \"p(HGNC:1884 ! CFTR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DDX58\", \"identifier\": \"19102\"}, \"id\": \"7cee4324cb2a468e51d341d128d3a7b3\", \"bel\": \"p(HGNC:19102 ! DDX58)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ADAM17\", \"identifier\": \"195\"}, \"id\": \"c04ede5a8a45b41ba56559110fad4834\", \"bel\": \"p(HGNC:195 ! ADAM17)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LYST\", \"identifier\": \"1968\"}, \"id\": \"511ceacc0c7922d55f403a866a0794ce\", \"bel\": \"p(HGNC:1968 ! LYST)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SEPHS1\", \"identifier\": \"19685\"}, \"id\": \"a09e49e2582a0529e742a04a43d28bb4\", \"bel\": \"p(HGNC:19685 ! SEPHS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLCA1\", \"identifier\": \"2015\"}, \"id\": \"2cd1395ae824babbb5e87fb14f4b3da3\", \"bel\": \"p(HGNC:2015 ! CLCA1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN1\", \"identifier\": \"2032\"}, \"id\": \"baed2439066c35adcdc6b4ee86d1c5ed\", \"bel\": \"p(HGNC:2032 ! CLDN1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN18\", \"identifier\": \"2039\"}, \"id\": \"dc52b1f25d63ad8419a59f71db0a36ed\", \"bel\": \"p(HGNC:2039 ! CLDN18)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN2\", \"identifier\": \"2041\"}, \"id\": \"cea10a8b550333794148a2d68bea0b4a\", \"bel\": \"p(HGNC:2041 ! CLDN2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN4\", \"identifier\": \"2046\"}, \"id\": \"fe91f59192ec0b97c6933f162a8f79f3\", \"bel\": \"p(HGNC:2046 ! CLDN4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN5\", \"identifier\": \"2047\"}, \"id\": \"940f2e285e998ed48dc778b550a7551c\", \"bel\": \"p(HGNC:2047 ! CLDN5)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN7\", \"identifier\": \"2049\"}, \"id\": \"5ac092c97fc9026cefb354c743f2c7c7\", \"bel\": \"p(HGNC:2049 ! CLDN7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CLDN8\", \"identifier\": \"2050\"}, \"id\": \"705780fba24bb7c77101e811642d9cc6\", \"bel\": \"p(HGNC:2050 ! CLDN8)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ZUP1\", \"identifier\": \"21224\"}, \"id\": \"69856c267ac495ced583324435681367\", \"bel\": \"p(HGNC:21224 ! ZUP1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CRP\", \"identifier\": \"2367\"}, \"id\": \"fc34a6afb9acbc85f015e9489abc576c\", \"bel\": \"p(HGNC:2367 ! CRP)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CSH1\", \"identifier\": \"2440\"}, \"id\": \"060f36c93e3b272654bc4e9ff1c6f8cb\", \"bel\": \"p(HGNC:2440 ! CSH1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CYBA\", \"identifier\": \"2577\"}, \"id\": \"205de5c710f627cccb327315136cdf72\", \"bel\": \"p(HGNC:2577 ! CYBA)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CYBB\", \"identifier\": \"2578\"}, \"id\": \"6ace7597ec607942e59464d32d9c22d3\", \"bel\": \"p(HGNC:2578 ! CYBB)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ACE\", \"identifier\": \"2707\"}, \"id\": \"eaee11bd9783ee94aff38ebd931d4396\", \"bel\": \"p(HGNC:2707 ! ACE)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DEFB4A\", \"identifier\": \"2767\"}, \"id\": \"ccc1f84b4574ed112cf6e1ab2a11be5d\", \"bel\": \"p(HGNC:2767 ! DEFB4A)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DIANPH\", \"identifier\": \"2875\"}, \"id\": \"49344889272f82475f1bb51e6f75beed\", \"bel\": \"p(HGNC:2875 ! DIANPH)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PATJ\", \"identifier\": \"28881\"}, \"id\": \"572a27408758b2490f7fc80a7ea062e4\", \"bel\": \"p(HGNC:28881 ! PATJ)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RPGRIP1L\", \"identifier\": \"29168\"}, \"id\": \"a953b095f019103ce4c8d9c644c0a368\", \"bel\": \"p(HGNC:29168 ! RPGRIP1L)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MAVS\", \"identifier\": \"29233\"}, \"id\": \"54fa4e6a940ca5801fafeb9cfea2efef\", \"bel\": \"p(HGNC:29233 ! MAVS)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NFASC\", \"identifier\": \"29866\"}, \"id\": \"c8641a4d09075db20f9df92190e3c4f9\", \"bel\": \"p(HGNC:29866 ! NFASC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DPP4\", \"identifier\": \"3009\"}, \"id\": \"0ae15e1f998e5f65035f2892ad4d8fff\", \"bel\": \"p(HGNC:3009 ! DPP4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DEFB4B\", \"identifier\": \"30193\"}, \"id\": \"863544123d748c6130dd30ac3d85cbf4\", \"bel\": \"p(HGNC:30193 ! DEFB4B)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SPECC1\", \"identifier\": \"30615\"}, \"id\": \"d070c8b8b416dca92cfa9ebaeb872fa5\", \"bel\": \"p(HGNC:30615 ! SPECC1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"DUOX1\", \"identifier\": \"3062\"}, \"id\": \"703c218b8b32c1e19aba32ea1dfe5953\", \"bel\": \"p(HGNC:3062 ! DUOX1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TSLP\", \"identifier\": \"30743\"}, \"id\": \"ab7c3bb5c4c7856793ea74e78107b157\", \"bel\": \"p(HGNC:30743 ! TSLP)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGER\", \"identifier\": \"320\"}, \"id\": \"46efcef764e041272dc7e8cae07361e3\", \"bel\": \"p(HGNC:320 ! AGER)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"EGFR\", \"identifier\": \"3236\"}, \"id\": \"4221d410040b17a4751ba254380887c0\", \"bel\": \"p(HGNC:3236 ! EGFR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGT\", \"identifier\": \"333\"}, \"id\": \"1b7929e21a877956819f0408d99da72a\", \"bel\": \"p(HGNC:333 ! AGT)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"AGTR1\", \"identifier\": \"336\"}, \"id\": \"027cdb0020befb24507560a613c995ce\", \"bel\": \"p(HGNC:336 ! AGTR1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"EP300\", \"identifier\": \"3373\"}, \"id\": \"c652cba3e067256e38bec37a59d892f1\", \"bel\": \"p(HGNC:3373 ! EP300)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"F2\", \"identifier\": \"3535\"}, \"id\": \"e66d3bff42810e3f409cbbbf97b3c2c0\", \"bel\": \"p(HGNC:3535 ! F2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"F2R\", \"identifier\": \"3537\"}, \"id\": \"f8a6c58b991cf509836fc85ed3622347\", \"bel\": \"p(HGNC:3537 ! F2R)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"F2RL2\", \"identifier\": \"3539\"}, \"id\": \"531ea7cf8c86a0a2a1e6fb6e88108969\", \"bel\": \"p(HGNC:3539 ! F2RL2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"F3\", \"identifier\": \"3541\"}, \"id\": \"8e23765e705c5a5c8eef2ee373ae2027\", \"bel\": \"p(HGNC:3541 ! F3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"FCN2\", \"identifier\": \"3624\"}, \"id\": \"75456bbecf946d1e050607e4225e0c02\", \"bel\": \"p(HGNC:3624 ! FCN2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"FGF7\", \"identifier\": \"3685\"}, \"id\": \"32987b123a64251c20b7c473f725a9cd\", \"bel\": \"p(HGNC:3685 ! FGF7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MTOR\", \"identifier\": \"3942\"}, \"id\": \"611d96814230dc6acf67fc01f1be12b9\", \"bel\": \"p(HGNC:3942 ! MTOR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"GNL1\", \"identifier\": \"4413\"}, \"id\": \"112b699159fb4ee98c7d3f4fb510e42d\", \"bel\": \"p(HGNC:4413 ! GNL1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"GPR15\", \"identifier\": \"4469\"}, \"id\": \"f0ac98f752118670910543a530c2a887\", \"bel\": \"p(HGNC:4469 ! GPR15)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TECR\", \"identifier\": \"4551\"}, \"id\": \"c9c78f3ae0abcd214457e182a79d4d1b\", \"bel\": \"p(HGNC:4551 ! TECR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"GYPC\", \"identifier\": \"4704\"}, \"id\": \"2bb4df1fb0e28258b4d3814e3906097a\", \"bel\": \"p(HGNC:4704 ! GYPC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ANG\", \"identifier\": \"483\"}, \"id\": \"6fccb287f998ce58b8b1aae56f60d345\", \"bel\": \"p(HGNC:483 ! ANG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"HJV\", \"identifier\": \"4887\"}, \"id\": \"750bc7c3c40ff448cb3e08111813e09e\", \"bel\": \"p(HGNC:4887 ! HJV)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"HIF1A\", \"identifier\": \"4910\"}, \"id\": \"7fe1001daba7651ce967824e7c5f3ed8\", \"bel\": \"p(HGNC:4910 ! HIF1A)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"HMGB1\", \"identifier\": \"4983\"}, \"id\": \"acc73ae0673d459bef26a76a17213076\", \"bel\": \"p(HGNC:4983 ! HMGB1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"HMOX1\", \"identifier\": \"5013\"}, \"id\": \"1bace7a2fc0cbf3c2365b2ba6b1b4d75\", \"bel\": \"p(HGNC:5013 ! HMOX1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"FOXA2\", \"identifier\": \"5022\"}, \"id\": \"5e7ae3eb541d8bb22de6f1013bf0449e\", \"bel\": \"p(HGNC:5022 ! FOXA2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ID1\", \"identifier\": \"5360\"}, \"id\": \"84e112a661a16167bff7d0dabc1ac17a\", \"bel\": \"p(HGNC:5360 ! ID1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IFNG\", \"identifier\": \"5438\"}, \"id\": \"0a92da37a152835b18dffb49c35eb1af\", \"bel\": \"p(HGNC:5438 ! IFNG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IKBKG\", \"identifier\": \"5961\"}, \"id\": \"05b6f46d645b33a66199e35913186ee5\", \"bel\": \"p(HGNC:5961 ! IKBKG)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL13\", \"identifier\": \"5973\"}, \"id\": \"1b3a29b2bc2b0a242c90294776736a7a\", \"bel\": \"p(HGNC:5973 ! IL13)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL15\", \"identifier\": \"5977\"}, \"id\": \"85d5b530b3b87b3e297aaf9b0886c65b\", \"bel\": \"p(HGNC:5977 ! IL15)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL18\", \"identifier\": \"5986\"}, \"id\": \"0e9ca3cdc2eeacdb0dcc7ee0a83005a0\", \"bel\": \"p(HGNC:5986 ! IL18)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL1B\", \"identifier\": \"5992\"}, \"id\": \"55883b821a74dc1f95aeb34a29b0989f\", \"bel\": \"p(HGNC:5992 ! IL1B)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL2\", \"identifier\": \"6001\"}, \"id\": \"6736c4d1d6ede336c466a66fcfd66064\", \"bel\": \"p(HGNC:6001 ! IL2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL4\", \"identifier\": \"6014\"}, \"id\": \"434687c828e4529fa775836452f486f2\", \"bel\": \"p(HGNC:6014 ! IL4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IL6\", \"identifier\": \"6018\"}, \"id\": \"c7b41b06aa9af9b2e98595de2ad6897d\", \"bel\": \"p(HGNC:6018 ! IL6)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"CXCL8\", \"identifier\": \"6025\"}, \"id\": \"77efec19dfa219e943c07ed04f993aa2\", \"bel\": \"p(HGNC:6025 ! CXCL8)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"INS\", \"identifier\": \"6081\"}, \"id\": \"443ee550873ab67fc41b537ba566cd8e\", \"bel\": \"p(HGNC:6081 ! INS)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IRF3\", \"identifier\": \"6118\"}, \"id\": \"c43d1fb4f357d80e945ed7936290e7db\", \"bel\": \"p(HGNC:6118 ! IRF3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"IRF7\", \"identifier\": \"6122\"}, \"id\": \"5677f74e19e2e73165595492af445ae3\", \"bel\": \"p(HGNC:6122 ! IRF7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"KNG1\", \"identifier\": \"6383\"}, \"id\": \"aa205a6174e1f0c714e801d69c323b86\", \"bel\": \"p(HGNC:6383 ! KNG1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LAMP1\", \"identifier\": \"6499\"}, \"id\": \"fcd85d510312ab3e74d3728327f4e906\", \"bel\": \"p(HGNC:6499 ! LAMP1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LGALS1\", \"identifier\": \"6561\"}, \"id\": \"59e0b7676da1bb33951c5cf85eb417df\", \"bel\": \"p(HGNC:6561 ! LGALS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LTF\", \"identifier\": \"6720\"}, \"id\": \"1fe042effb1c114c6b556794e254dd5e\", \"bel\": \"p(HGNC:6720 ! LTF)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"LYZ\", \"identifier\": \"6740\"}, \"id\": \"781a79ea0396ce111994ea26d863dc91\", \"bel\": \"p(HGNC:6740 ! LYZ)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MAS1\", \"identifier\": \"6899\"}, \"id\": \"711b4cb7c5758a9e78683bfd8e39dcba\", \"bel\": \"p(HGNC:6899 ! MAS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MBL2\", \"identifier\": \"6922\"}, \"id\": \"99ef80d2eb6aed8478400ee0a70b6a13\", \"bel\": \"p(HGNC:6922 ! MBL2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MCC\", \"identifier\": \"6935\"}, \"id\": \"10a6c412d764b85b9e2de517396958e3\", \"bel\": \"p(HGNC:6935 ! MCC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MGAT1\", \"identifier\": \"7044\"}, \"id\": \"20b2a2ba1eec2043899b4d96a87f0c41\", \"bel\": \"p(HGNC:7044 ! MGAT1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ASL\", \"identifier\": \"746\"}, \"id\": \"05c72d1482f0bd1e35bcc13ecf1602fe\", \"bel\": \"p(HGNC:746 ! ASL)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MUC5AC\", \"identifier\": \"7515\"}, \"id\": \"ca7288c60c06c534a13a3578638c6349\", \"bel\": \"p(HGNC:7515 ! MUC5AC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MYC\", \"identifier\": \"7553\"}, \"id\": \"8f8c191633855c82503987045e06ffca\", \"bel\": \"p(HGNC:7553 ! MYC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"MYOM2\", \"identifier\": \"7614\"}, \"variants\": [{\"kind\": \"pmod\", \"concept\": {\"namespace\": \"go\", \"name\": \"protein glycosylation\", \"identifier\": \"0006486\"}}], \"id\": \"dc0f6c5b2dab2df4e51407421ea4a710\", \"bel\": \"p(HGNC:7614 ! MYOM2, pmod(go:0006486 ! \\\"protein glycosylation\\\"))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NF1\", \"identifier\": \"7765\"}, \"id\": \"c01afece16e8144dc370654cbeac543f\", \"bel\": \"p(HGNC:7765 ! NF1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NMI\", \"identifier\": \"7854\"}, \"id\": \"9b50bca0c316e23787cc1b47da6e7f5e\", \"bel\": \"p(HGNC:7854 ! NMI)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NOS1\", \"identifier\": \"7872\"}, \"id\": \"d2108f6a894d716a7768072b57681a5c\", \"bel\": \"p(HGNC:7872 ! NOS1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NOS2\", \"identifier\": \"7873\"}, \"id\": \"2cfc6141c45b8c1290a77cea29ee224f\", \"bel\": \"p(HGNC:7873 ! NOS2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NOS3\", \"identifier\": \"7876\"}, \"id\": \"9fc0061b28f7cacc807a3908e2a8e595\", \"bel\": \"p(HGNC:7876 ! NOS3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NOX1\", \"identifier\": \"7889\"}, \"id\": \"aa847ff0b0f5c9827f38734d0398dc8d\", \"bel\": \"p(HGNC:7889 ! NOX1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NOX4\", \"identifier\": \"7891\"}, \"id\": \"77ba888a66b8f633aae5e530f99dc72f\", \"bel\": \"p(HGNC:7891 ! NOX4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NR1I2\", \"identifier\": \"7968\"}, \"id\": \"4fb38749342a28bce9acd57ffd297fe0\", \"bel\": \"p(HGNC:7968 ! NR1I2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NR3C2\", \"identifier\": \"7979\"}, \"id\": \"6b387de5281cce0b48cca2f7545246ad\", \"bel\": \"p(HGNC:7979 ! NR3C2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"NUP214\", \"identifier\": \"8064\"}, \"id\": \"8a2652ce8881ee6cbe459445256da1d1\", \"bel\": \"p(HGNC:8064 ! NUP214)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"OSM\", \"identifier\": \"8506\"}, \"id\": \"cc679b177a469fa46592a64437e3400b\", \"bel\": \"p(HGNC:8506 ! OSM)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"FURIN\", \"identifier\": \"8568\"}, \"id\": \"0cfc2a7fb33e508247d545b35ce888ab\", \"bel\": \"p(HGNC:8568 ! FURIN)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SERPINE1\", \"identifier\": \"8583\"}, \"id\": \"9fc00d18b3d888e2df6f544310d2e709\", \"bel\": \"p(HGNC:8583 ! SERPINE1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PDC\", \"identifier\": \"8759\"}, \"id\": \"65e27c00074ab5463fdaa23dfef3d676\", \"bel\": \"p(HGNC:8759 ! PDC)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PDCD1\", \"identifier\": \"8760\"}, \"id\": \"3adc41a972ddfa11c3de81eb6e16309f\", \"bel\": \"p(HGNC:8760 ! PDCD1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PDPK1\", \"identifier\": \"8816\"}, \"id\": \"204cb5af4537ebb9ad9d0a9dcac00d34\", \"bel\": \"p(HGNC:8816 ! PDPK1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PDPK1\", \"identifier\": \"8816\"}, \"variants\": [{\"kind\": \"pmod\", \"concept\": {\"namespace\": \"go\", \"name\": \"protein phosphorylation\", \"identifier\": \"0006468\"}}], \"id\": \"4f642f6e9d707318a4a9ae301727f495\", \"bel\": \"p(HGNC:8816 ! PDPK1, pmod(go:0006468 ! \\\"protein phosphorylation\\\"))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"ATR\", \"identifier\": \"882\"}, \"id\": \"57259bed2c07b882f04621341b81e0e2\", \"bel\": \"p(HGNC:882 ! ATR)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PI3\", \"identifier\": \"8947\"}, \"id\": \"4398a32410b897766bae8691d777276d\", \"bel\": \"p(HGNC:8947 ! PI3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"SLC33A1\", \"identifier\": \"95\"}, \"id\": \"a72285d5ecf42e63e9790fb3e0cf1876\", \"bel\": \"p(HGNC:95 ! SLC33A1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PTGER4\", \"identifier\": \"9596\"}, \"id\": \"24f84f9e3093a6d8dc005e7814ab2bcf\", \"bel\": \"p(HGNC:9596 ! PTGER4)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"PTGS2\", \"identifier\": \"9605\"}, \"id\": \"dd9d910036ae1395a5969a5660371939\", \"bel\": \"p(HGNC:9605 ! PTGS2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RAB5B\", \"identifier\": \"9784\"}, \"id\": \"e905506992cc06ea57c10fde52079bd1\", \"bel\": \"p(HGNC:9784 ! RAB5B)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RAB5C\", \"identifier\": \"9785\"}, \"id\": \"842b8dc7b37398cd73e2adc15d02dc7b\", \"bel\": \"p(HGNC:9785 ! RAB5C)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"RAF1\", \"identifier\": \"9829\"}, \"id\": \"9d129714758157ced75c34ee98b34d8b\", \"bel\": \"p(HGNC:9829 ! RAF1)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"REN\", \"identifier\": \"9958\"}, \"id\": \"e8b7fe4a8b0bcbcc35d8c856e6fa211c\", \"bel\": \"p(HGNC:9958 ! REN)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"HGNC\", \"name\": \"TRIM27\", \"identifier\": \"9975\"}, \"id\": \"17aeaa28d3c7ca315d5c3f9609354795\", \"bel\": \"p(HGNC:9975 ! TRIM27)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"IP\", \"name\": \"IPR002959\"}, \"id\": \"64f292a1a70eb33f945a11df4bb950f0\", \"bel\": \"p(IP:IPR002959)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"IP\", \"name\": \"IPR004724\"}, \"id\": \"6f588d73b93febe86e5f51c3f3342279\", \"bel\": \"p(IP:IPR004724)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"IP\", \"name\": \"IPR007702\"}, \"id\": \"cbc9ec8d714d2cab1320fbdd4fa59618\", \"bel\": \"p(IP:IPR007702)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"furin-like\", \"identifier\": \"PF00757\"}, \"id\": \"941001cde5036826bb748b66e29cbc35\", \"bel\": \"p(PFAM:PF00757 ! furin-like)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"branch\", \"identifier\": \"PF02485\"}, \"id\": \"71b673b3119304b885b03a813c063eb6\", \"bel\": \"p(PFAM:PF02485 ! branch)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"gram\", \"identifier\": \"PF02893\"}, \"id\": \"c57c952adc9d9c03d505deb0531eb4aa\", \"bel\": \"p(PFAM:PF02893 ! gram)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"octapeptide\", \"identifier\": \"PF03373\"}, \"id\": \"98bd67f130cc7212b4520f272aa63d67\", \"bel\": \"p(PFAM:PF03373 ! octapeptide)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"PFAM\", \"name\": \"mucin-like\", \"identifier\": \"PF16058\"}, \"id\": \"33b018fb9869be17350c9cb2da895cfb\", \"bel\": \"p(PFAM:PF16058 ! mucin-like)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"A0A5C1RGL7\", \"identifier\": \"A0A5C1RGL7\"}, \"id\": \"79d2108c4ca81cc5fafef0d81bfb0c20\", \"bel\": \"p(UP:A0A5C1RGL7 ! A0A5C1RGL7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"A3M775\", \"identifier\": \"A3M775\"}, \"id\": \"ecdfc2f3d2c5f50a4fc52f0117f557de\", \"bel\": \"p(UP:A3M775 ! A3M775)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"F4KFT7\", \"identifier\": \"F4KFT7\"}, \"id\": \"6316d42e155b7b9164304f1f9e351873\", \"bel\": \"p(UP:F4KFT7 ! F4KFT7)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"H2AM12\", \"identifier\": \"H2AM12\"}, \"id\": \"91204a326f149c04c2deafbe862b4544\", \"bel\": \"p(UP:H2AM12 ! H2AM12)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"O09029\", \"identifier\": \"O09029\"}, \"id\": \"d19b880e7eb7d97a043ad148b3848fef\", \"bel\": \"p(UP:O09029 ! O09029)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"O29445\", \"identifier\": \"O29445\"}, \"id\": \"a888e50bfae47422bdf1bb0980ab7085\", \"bel\": \"p(UP:O29445 ! O29445)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"O64399\", \"identifier\": \"O64399\"}, \"id\": \"b76386e01b00b1c5061397939cfd9526\", \"bel\": \"p(UP:O64399 ! O64399)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P00814\", \"identifier\": \"P00814\"}, \"id\": \"300fce7b0aa90974abf9368a55a0c7cf\", \"bel\": \"p(UP:P00814 ! P00814)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P02080\", \"identifier\": \"P02080\"}, \"id\": \"718f768332c51dc96caa0f3b70acd628\", \"bel\": \"p(UP:P02080 ! P02080)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P03709\", \"identifier\": \"P03709\"}, \"id\": \"b92abbe91b36ecbf86479c3cff6eb764\", \"bel\": \"p(UP:P03709 ! P03709)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P04608\", \"identifier\": \"P04608\"}, \"id\": \"f0f5f3f299f4b9e11c10080e572cc500\", \"bel\": \"p(UP:P04608 ! P04608)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P07052\", \"identifier\": \"P07052\"}, \"id\": \"fc0b3e7160b7bd71acdff723929b10b1\", \"bel\": \"p(UP:P07052 ! P07052)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P08707\", \"identifier\": \"P08707\"}, \"id\": \"9ced09e2d88f45eb07d33eaebde989ae\", \"bel\": \"p(UP:P08707 ! P08707)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0C710\", \"identifier\": \"P0C710\"}, \"id\": \"5a56194b0063c600c0bdd4c76b468633\", \"bel\": \"p(UP:P0C710 ! P0C710)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0C797\", \"identifier\": \"P0C797\"}, \"id\": \"64ae1768f808b9052e29ae78314479e5\", \"bel\": \"p(UP:P0C797 ! P0C797)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0CW71\", \"identifier\": \"P0CW71\"}, \"id\": \"1c91e3bb951853ac0a65fb5c455d7579\", \"bel\": \"p(UP:P0CW71 ! P0CW71)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0DTC2\", \"identifier\": \"P0DTC2\"}, \"id\": \"4116883724f3ca1e591b677e35dbea95\", \"bel\": \"p(UP:P0DTC2 ! P0DTC2)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0DTC2\", \"identifier\": \"P0DTC2\"}, \"variants\": [{\"kind\": \"hgvs\", \"hgvs\": \"p.?\"}], \"id\": \"adb3ef168a8533a268a5e0980972063b\", \"bel\": \"p(UP:P0DTC2 ! P0DTC2, var(\\\"p.?\\\"))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P0DTC9\", \"identifier\": \"P0DTC9\"}, \"variants\": [{\"kind\": \"pmod\", \"concept\": {\"namespace\": \"go\", \"name\": \"protein phosphorylation\", \"identifier\": \"0006468\"}}], \"id\": \"002f5250cbee2bf144e03f51b6d4b21e\", \"bel\": \"p(UP:P0DTC9 ! P0DTC9, pmod(go:0006468 ! \\\"protein phosphorylation\\\"))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P13917\", \"identifier\": \"P13917\"}, \"id\": \"58664d7f8a754acdf071319cc52933e8\", \"bel\": \"p(UP:P13917 ! P13917)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"P84849\", \"identifier\": \"P84849\"}, \"id\": \"a24bcf83fef6b060de08f41c2c07ec0b\", \"bel\": \"p(UP:P84849 ! P84849)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q24174\", \"identifier\": \"Q24174\"}, \"id\": \"af0bd31f813b5185440713cb50755b0d\", \"bel\": \"p(UP:Q24174 ! Q24174)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q38424\", \"identifier\": \"Q38424\"}, \"id\": \"2024fb0db9012ed3185575e4bded1732\", \"bel\": \"p(UP:Q38424 ! Q38424)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q55C99\", \"identifier\": \"Q55C99\"}, \"id\": \"b3189c103eda02ebd429b70ce696d5cb\", \"bel\": \"p(UP:Q55C99 ! Q55C99)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q80H93\", \"identifier\": \"Q80H93\"}, \"variants\": [{\"kind\": \"pmod\", \"concept\": {\"namespace\": \"go\", \"name\": \"protein ubiquitination\", \"identifier\": \"0016567\"}}], \"id\": \"4989fa6fa830a47b6364ad36bbb33b58\", \"bel\": \"p(UP:Q80H93 ! Q80H93, pmod(go:0016567 ! \\\"protein ubiquitination\\\"))\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q869C3\", \"identifier\": \"Q869C3\"}, \"id\": \"e292a728bdd3b16ae0e82e18079e9a50\", \"bel\": \"p(UP:Q869C3 ! Q869C3)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q9T1T5\", \"identifier\": \"Q9T1T5\"}, \"id\": \"c53d96ef3cb0376bbbbf196247d17544\", \"bel\": \"p(UP:Q9T1T5 ! Q9T1T5)\"}, {\"function\": \"Protein\", \"concept\": {\"namespace\": \"UP\", \"name\": \"Q9W4S7\", \"identifier\": \"Q9W4S7\"}, \"id\": \"ef52c6936b6c33247a0069476172325e\", \"bel\": \"p(UP:Q9W4S7 ! Q9W4S7)\"}], \"links\": [{\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23103802047133572\": true}, \"uuid\": {\"4f92915e-dd76-43b1-b3f5-ef78dc13b202\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7336170101838134299\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 339, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"96 Histones activate platelets via TLR-2 and TLR-4 and enhance fibrin formation , while the contact activation system , via FXII activation , is initiated by negatively charged DNA .\", \"source\": 344, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10072123627412039\": true}, \"uuid\": {\"ce0adc73-aea3-40d1-a7ad-b39fbe03ce38\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"-995941319787512474\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"96 Histones activate platelets via TLR-2 and TLR-4 and enhance fibrin formation , while the contact activation system , via FXII activation , is initiated by negatively charged DNA .\", \"source\": 344, \"target\": 204, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22384480168304024\": true}, \"uuid\": {\"90f4c79c-1e73-4e1a-8b50-689316fcf28c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5187463990965369953\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"88 , 89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production .\", \"source\": 204, \"target\": 125, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-18076332435128244\": true}, \"uuid\": {\"133810ea-b3af-466a-ab50-dde298b618df\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"832931734227946725\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"The importance of platelets in mediating T-cell recruitment and trafficking is underscored by data demonstrating that platelets play a fundamental role in the accumulation of cytotoxic T cells , and virus-induced organ damage , in a murine model of acute viral hepatitis.87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution.88 ,89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production .\", \"source\": 204, \"target\": 125, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24392982139337930\": true}, \"uuid\": {\"93d307c6-5982-4084-91b3-3392d8047907\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 204, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24392982139337930\": true}, \"uuid\": {\"93d307c6-5982-4084-91b3-3392d8047907\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 204, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24392982139337930\": true}, \"uuid\": {\"93d307c6-5982-4084-91b3-3392d8047907\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 204, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17331045903166563\": true}, \"uuid\": {\"a0490203-fc05-498a-95ba-cf5053796524\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"7920281624656773409\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"For example , platelets express major histocompatibility complex class 1 molecules and can , therefore , directly activate T cells in an major histocompatibility complex class 1 dependent manner ,84 while platelet-derived PF-4 and serotonin can regulate Th17 expansion85 and naive T-cell activation and proliferation ,86 respectively .\", \"source\": 204, \"target\": 270, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21751405736558678\": true}, \"uuid\": {\"7aa82a0d-38eb-41f6-8356-4b9d8c9b3ce0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2503607577164908016\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"likely\": true, \"previously\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"These interactions are likely important since platelets have previously been demonstrated to be key modulators of adaptive immunity in the context of a range of inflammatory disorders .\", \"source\": 204, \"target\": 315, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21751405736558678\": true}, \"uuid\": {\"7aa82a0d-38eb-41f6-8356-4b9d8c9b3ce0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2503607577164908016\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"likely\": true, \"previously\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Nevertheless , delineating how platelets modulate the adaptive immune response to SARS-CoV-2 and the role adaptive immunity plays in mediating thrombosis and organ injury will be crucial in uncovering novel therapeutic strategies .\", \"source\": 204, \"target\": 315, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-32726273277996891\": true}, \"uuid\": {\"1a18ac68-0a95-4167-96d2-4481f5f5bc4c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 204, \"target\": 139, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-32726273277996891\": true}, \"uuid\": {\"1a18ac68-0a95-4167-96d2-4481f5f5bc4c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 204, \"target\": 139, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-32726273277996891\": true}, \"uuid\": {\"1a18ac68-0a95-4167-96d2-4481f5f5bc4c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 204, \"target\": 139, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27658615892536388\": true}, \"uuid\": {\"8f54294e-5dba-4487-a801-bb878a800843\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4370558968734663725\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Levels of vWF, fibrinogen, and factor VIII were elevated in COVID-19 (83) .\", \"source\": 340, \"target\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5686232411627413\": true}, \"uuid\": {\"614ee111-93ce-4a11-b1ef-9ccab09d121b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2093508689490039097\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"previously\": true}}, \"target\": 339, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis .\", \"source\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11911252902806362\": true}, \"uuid\": {\"7355403a-8736-47b9-9c65-6a86013f6de6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2093508689490039097\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"previously\": true}}, \"target\": 452, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis .\", \"source\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9841882630527885\": true}, \"uuid\": {\"ec6c45f0-154f-4ed1-81d5-6fb77ecedd90\": true}, \"belief\": {\"0.99746355\": true}, \"source_hash\": {\"-3796604975999605268\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32244852\"}, \"evidence\": \"ACE2 could be a novel adhesion molecule for SARS-CoV-2 causing Covid-19 and a potential therapeutic target for the prevention of fatal microbial infections, and therefore it should be fast tracked and prioritized for research and investigation.\", \"source\": 193, \"target\": 194, \"key\": 35}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15734269076288934\": true}, \"uuid\": {\"4d8aa2c6-10fd-4f23-bc8e-aa547065b282\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"4508326262597746344\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}, \"negated\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis .\", \"source\": 193, \"target\": 261, \"key\": 97}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25034681553914186\": true}, \"uuid\": {\"d656e4c0-28a7-415e-b3fd-0abd11e7db13\": true}, \"belief\": {\"0.9499999999895397\": true}, \"source_hash\": {\"-5174575179727369723\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32567404\"}, \"evidence\": \"The Sepsis-Induced Coagulopathy ( SIC ) Score , able to predicts likelihood of sepsis-induced coagulopathy , has been evaluated in the COVID-19 pandemic given concerns that COVID-19 may cause thrombosis .\", \"source\": 193, \"target\": 271, \"key\": 15}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-16992349688174560\": true}, \"uuid\": {\"7da897d1-0e38-4cb9-8d65-6587dd124364\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6196998427393000566\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response.\", \"source\": 388, \"target\": 123, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28195142794092511\": true}, \"uuid\": {\"f97e007c-7cd2-4446-936e-db6e047c9da0\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-3739584405159490278\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Cellular p53 inhibits replication of SARS-CoV and HCoV-NL63, presumably by activating genes involved in innate immunity.\", \"source\": 388, \"target\": 299, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21109669863503139\": true}, \"uuid\": {\"679d007c-724f-49f3-9075-31fe6a4926c9\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"-6369665858226714411\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 53, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma .\", \"source\": 299, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5353055752865414\": true}, \"uuid\": {\"f7fc6131-75fb-44d1-840d-2ac7c1d75d90\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"-7739628755725909434\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"indicate\": true}}, \"target\": 345, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma .\", \"source\": 299, \"key\": 5}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28884009792519108\": true}, \"uuid\": {\"1af2ed7f-ceab-41f6-a4f6-0e41c4aa8154\": true}, \"belief\": {\"0.993\": true}, \"source_hash\": {\"2329577592897858409\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"suggest\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32463365\"}, \"evidence\": \"The question of whether coronaviruses have evolved to exploit such a network may be relevant to pursue , particularly given that downregulation of ACE2 by SARS-CoV has been reported previously and is associated with poor clinical outcomes ( Glowacka et al ., 2010 ; Kuba et al ., 2010 ) .\", \"source\": 299, \"key\": 26}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15526709899718757\": true}, \"uuid\": {\"79de7c43-ce7d-4f27-bed0-0384e326a54f\": true}, \"belief\": {\"0.9974999999999987\": true}, \"source_hash\": {\"-933943039142404084\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"target\": 13, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Endothelial cells also synthesise Ch-S; the enzyme endothelial nitric oxide synthase (eNOS) is traditionally thought to mediate synthesis of nitric oxide (NO); however, it has been established that when eNOS is membrane bound, it is no longer able to synthesise NO.\", \"source\": 500, \"key\": 11}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-2755617017968116\": true}, \"uuid\": {\"6cb5c571-caba-4485-8df5-51b7a3d48ab4\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-3385695780620174958\": true}, \"source_api\": {\"reach\": true}}, \"target\": 32, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"This leads to eNOS uncoupling, where eNOS produces superoxide instead of NO [XREF_BIBR].\", \"source\": 500, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11009699051275995\": true}, \"uuid\": {\"75ad6fc6-cd99-4e33-9cda-162990689fcc\": true}, \"belief\": {\"0.9499999999999152\": true}, \"source_hash\": {\"580255531883654927\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"NO, on the other hand, inhibits viral replication via changes in the structure of viral proteins [34] .\", \"source\": 13, \"target\": 164, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12288036541925991\": true}, \"uuid\": {\"585ef47e-5403-4286-8a27-205c4f634e06\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7648319993566736089\": true}, \"source_api\": {\"reach\": true}}, \"target\": 498, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Three NO synthases contribute to production of NO in airway epithelia: the constitutively expressed NOS1 and NOS3, and inducible NOS2.\", \"source\": 13, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12288036541925991\": true}, \"uuid\": {\"585ef47e-5403-4286-8a27-205c4f634e06\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7648319993566736089\": true}, \"source_api\": {\"reach\": true}}, \"target\": 498, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Three NO synthases contribute to production of NO in airway epithelia : the constitutively expressed NOS1 and NOS3, and inducible NOS2.\", \"source\": 13, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23373107113446601\": true}, \"uuid\": {\"89774c59-9ea1-48f6-a287-d5540389fe7c\": true}, \"belief\": {\"0.999824999830846\": true}, \"source_hash\": {\"-4036621356676210066\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 499, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Three NO synthases contribute to production of NO in airway epithelia: the constitutively expressed NOS1 and NOS3, and inducible NOS2.\", \"source\": 13, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23373107113446601\": true}, \"uuid\": {\"89774c59-9ea1-48f6-a287-d5540389fe7c\": true}, \"belief\": {\"0.999824999830846\": true}, \"source_hash\": {\"-4036621356676210066\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 499, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Three NO synthases contribute to production of NO in airway epithelia : the constitutively expressed NOS1 and NOS3, and inducible NOS2.\", \"source\": 13, \"key\": 12}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14891576988648505\": true}, \"uuid\": {\"7781c093-dd07-4517-8ff1-abafeb579f73\": true}, \"belief\": {\"0.949980317\": true}, \"source_hash\": {\"5127777704857601077\": true}, \"source_api\": {\"reach\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32173574\"}, \"evidence\": \"For instance, diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators, especially IL-1beta and TNFalpha, which promote systemic insulin resistance and beta-cell damage.\", \"source\": 386, \"key\": 4}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-25283140673854367\": true}, \"uuid\": {\"fd6e7841-1533-4477-8b89-d28cbaff389d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1229623821476541186\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Indeed, increased TNF-alpha levels attenuate cldn5 expression in a mouse model of acute lung inflammation [XREF_BIBR].\", \"source\": 386, \"target\": 427, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2819732159409997\": true}, \"uuid\": {\"684fba46-9510-44b7-8b88-9364088c6cb8\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"-5497739048982210580\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32173574\"}, \"evidence\": \"For instance , diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators , especially , IL-1beta and TNFalpha , which promotes systemic insulin resistance and beta-cell damage 17 .\", \"source\": 386, \"target\": 199, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16514861048975347\": true}, \"uuid\": {\"e9673808-a747-4cb3-8ac6-e92aeb8a0a75\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1904433110637014887\": true}, \"source_api\": {\"reach\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Ang-II signaling cascades inhibit insulin release and decrease insulin sensitivity.\", \"source\": 480, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-30742844141080344\": true}, \"uuid\": {\"9a0e4637-c153-4866-8e93-69fa686608b2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7524152733547168555\": true}, \"source_api\": {\"reach\": true}}, \"target\": 525, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"18 In animal models, insulin augments janus kinase-signal transducer and activator of transcription (JAK and STAT) signaling to inhibit viral replication.\", \"source\": 480, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15931642812743522\": true}, \"uuid\": {\"355f5427-bede-4877-85df-90752fc51a2e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"786403992382173459\": true}, \"source_api\": {\"reach\": true}}, \"target\": 4, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Coupled with the glucose fuelling effect on the cellular redox system, insulin increases mitochondrial production of reactive oxygen species (ROS) via generation of ceramides.\", \"source\": 480, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22232409947768872\": true}, \"uuid\": {\"080efd53-4f07-458f-9839-5f6f7900af9f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4440085798221893602\": true}, \"source_api\": {\"reach\": true}}, \"target\": 30, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release.\", \"source\": 480, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2075102701558147\": true}, \"uuid\": {\"f83525a0-488e-4554-9846-10f604c52178\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-4440085798221893602\": true}, \"source_api\": {\"reach\": true}}, \"target\": 50, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release.\", \"source\": 480, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2075102701558147\": true}, \"uuid\": {\"f83525a0-488e-4554-9846-10f604c52178\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-4440085798221893602\": true}, \"source_api\": {\"reach\": true}}, \"target\": 50, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release.\", \"source\": 480, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30042553597472207\": true}, \"uuid\": {\"fe1e4443-1b01-4938-acc8-090302f1b3d8\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-2626175376076233939\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"This study clearly demonstrates that loss of the Mas receptor leads to metabolic dysfunction and implicates the ACE2/Ang-(17)/Mas receptor axis in the development of insulin resistance.Obesity and insulin resistance are associated with impaired endothelium-dependent and insulin-mediated vasodilation (Laakso et al., 1990, Steinberg et al., 1996, Williams et al., 1996).\", \"source\": 480, \"target\": 166, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"1033312619701859\": true}, \"uuid\": {\"97030b2e-e4e9-42ba-8046-5ed9a5ac607f\": true}, \"belief\": {\"0.9499999996125795\": true}, \"source_hash\": {\"-8100800120932652912\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Insulin administration decreases ACE2 protein expression in the lungs of diabetic mice (96) .\", \"source\": 480, \"target\": 393, \"key\": 29}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-1768407395274132\": true}, \"uuid\": {\"6769e800-a42b-4cbb-82cb-cb7d347a84d3\": true}, \"belief\": {\"0.9973784916409645\": true}, \"source_hash\": {\"5014531921600983310\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"suggest\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32228322\"}, \"evidence\": \"Insulin administration attenuates ACE2 expression, while hypoglycemic agents such as glucagon like peptide-1 (GLP-1) agonists (liraglutide) and thiazolidinediones (TZDs; pioglitazone), antihypertensives such as ACE inhibitors, and statins upregulate ACE2.\", \"source\": 480, \"key\": 54}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6418513221016837\": true}, \"uuid\": {\"65181804-63a7-43ce-93c6-8cf2b9926fa0\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-1718100081218382388\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"39 Insulin mediates de novo lipogenesis and adipogenesis .\", \"source\": 480, \"target\": 303, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24145885802330203\": true}, \"uuid\": {\"c3c7e726-2acc-48ff-9bdf-bf3db5e7ac69\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3039193815437244324\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"39 Insulin mediates de novo lipogenesis and adipogenesis .\", \"source\": 480, \"target\": 304, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"19103934801551825\": true}, \"uuid\": {\"9de6dbd3-2662-4665-b9a6-bef1191c4a3c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1560815448935115743\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"An encouragement to chase this track more in depth comes from the observation that in non-COVID-19 critically ill hyperglycemic patients admitted at ICU, plasma levels of sRAGE are higher in DM vs. non DM patients, and reduced by intensive insulin therapy only in DM patients (41).\", \"source\": 480, \"target\": 121, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34046683031309473\": true}, \"uuid\": {\"f92d163c-5787-4499-8f15-1efe3fdbe4b6\": true}, \"belief\": {\"0.993\": true}, \"source_hash\": {\"-5973795755221734307\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32444458\"}, \"evidence\": \"Typically, this has been managed by a combination of continuous intravenous insulin infusion and frequent glucose testing to improve glycemic control and reduce glucose variability to the minimum in order to reduce mortality in ICU (4).\", \"source\": 480, \"key\": 25}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34046683031309473\": true}, \"uuid\": {\"f92d163c-5787-4499-8f15-1efe3fdbe4b6\": true}, \"belief\": {\"0.993\": true}, \"source_hash\": {\"-5973795755221734307\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32444458\"}, \"evidence\": \"The main aims of a safe and effective insulin regimen are to reduce contact frequency, reduce glucose variability, minimize risk of severe hypoglycemia, and improve overall glycemic control.\", \"source\": 480, \"key\": 26}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34046683031309473\": true}, \"uuid\": {\"f92d163c-5787-4499-8f15-1efe3fdbe4b6\": true}, \"belief\": {\"0.993\": true}, \"source_hash\": {\"-5973795755221734307\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"Optimal glucose control in intensive care is limited by manual sampling of glucose and intravenous insulin adjustment, as well as increased nursing workload and the need of protective equipment.\", \"source\": 480, \"key\": 54}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26188341534533534\": true}, \"uuid\": {\"5c96ab91-a972-4c41-8087-6db5159bfdbc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"959335178432775057\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Hyperinsulinaemia causes increased renal excretion of magnesium, and insulin resistance (IR) reduces intracellular magnesium levels.\", \"source\": 480, \"target\": 31, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-10850071252237573\": true}, \"uuid\": {\"7563548d-30ce-4de3-b56f-169183d27d6b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5183146057317442619\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In non-diabetic rats, acute Ang-II infusion prevented the first phase insulin secretion in response to glucose (Carlsson et al., 1998), and in healthy human subjects, pressor doses of Ang-II suppressed basal, pulsatile and glucose-stimulated insulin release (Fliser et al., 1997).\", \"source\": 480, \"target\": 156, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1052375199047433\": true}, \"uuid\": {\"29e3289c-5ce9-4227-b72c-bc50b5ffc3e1\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7078694086227832995\": true}, \"source_api\": {\"reach\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32173574\"}, \"evidence\": \"For instance, diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators, especially IL-1beta and TNFalpha, which promote systemic insulin resistance and beta-cell damage.\", \"source\": 475, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"8662473568909839\": true}, \"uuid\": {\"208b5e91-7cb3-46b9-82c7-1d30d0266cea\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-5497739048982210580\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32173574\"}, \"evidence\": \"For instance , diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators , especially , IL-1beta and TNFalpha , which promotes systemic insulin resistance and beta-cell damage 17 .\", \"source\": 475, \"target\": 199, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-29129398976472959\": true}, \"uuid\": {\"e7f299a2-7171-407f-95d2-5a114683adeb\": true}, \"belief\": {\"0.9929997519942\": true}, \"source_hash\": {\"-2081913717546445243\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"However, due to the long-term cardiovascular benefits, in vitro evidence of suppression of IL-6-induced CRP expression by statins, and the emerging epidemiologic data on lower odds of mortality from COVID-19 among statin users, therapy with statins can be continued during COVID-19 illness (129, 155) .\", \"source\": 478, \"target\": 431, \"key\": 13}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23666051005342888\": true}, \"uuid\": {\"df44a064-6e95-49db-963f-63d9ee938e0c\": true}, \"belief\": {\"0.9824999999989017\": true}, \"source_hash\": {\"710776032178934743\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"The mechanisms for lymphopenia are not known, but elevated levels of IL-6 and TNF- may cause T cell apoptosis(66,69).\", \"source\": 478, \"target\": 135, \"key\": 21}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34952691398578401\": true}, \"uuid\": {\"75d5db08-dad5-4d5a-b205-3fe6bd244094\": true}, \"belief\": {\"0.9804\": true}, \"source_hash\": {\"-8011915834940869521\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"suggest\": true}}, \"target\": 389, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"Studies have shown that tocilizumab, an IL-6 receptor antibody, could reverse CRS in the setting of chimeric antigen receptor T-cell therapy for cancer [36].\", \"source\": 478, \"key\": 15}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9686862843009560\": true}, \"uuid\": {\"1ba7ded8-48a2-4642-878e-2357ad50f30d\": true}, \"belief\": {\"0.98249981399565\": true}, \"source_hash\": {\"-4767523093900814469\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression.\", \"source\": 478, \"target\": 455, \"key\": 22}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9686862843009560\": true}, \"uuid\": {\"1ba7ded8-48a2-4642-878e-2357ad50f30d\": true}, \"belief\": {\"0.98249981399565\": true}, \"source_hash\": {\"-4767523093900814469\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression.\", \"source\": 478, \"target\": 455, \"key\": 23}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9594015846885040\": true}, \"uuid\": {\"f22913f3-3965-48c8-b1f4-84c06e9d5c90\": true}, \"belief\": {\"0.99596289\": true}, \"source_hash\": {\"2561253971139201809\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"risk\": true}, \"bel\": \"p(HGNC:6018 ! IL6) increases p(HGNC:8583 ! SERPINE1)\"}, \"target\": 508, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase , SOD and catalase and decreased production of glutathione , further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins , membranes and haem.68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1.64 70 Hyperinsulinaemia increases production of PAI-1 , coupled with MgD-induced increased production of PAI-1 that inhibits fibrinolysis , together they compound coagulation risk .\", \"source\": 478, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9594015846885040\": true}, \"uuid\": {\"f22913f3-3965-48c8-b1f4-84c06e9d5c90\": true}, \"belief\": {\"0.99596289\": true}, \"source_hash\": {\"2561253971139201809\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"risk\": true}, \"bel\": \"p(HGNC:6018 ! IL6) increases p(HGNC:8583 ! SERPINE1)\"}, \"target\": 508, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase , SOD and catalase and decreased production of glutathione , further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins , membranes and haem.68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1.64 70 Hyperinsulinaemia increases production of PAI-1 , coupled with MgD-induced increased production of PAI-1 that inhibits fibrinolysis , together they compound coagulation risk .\", \"source\": 478, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7248679661820881\": true}, \"uuid\": {\"cdf021f6-370f-45cf-9727-c27f0f7a802d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-506014373413407189\": true}, \"source_api\": {\"reach\": true}}, \"target\": 523, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"CRP can stimulate monocytes to release proinflammatory cytokines such as interleukin-6 (IL-6), interleukin-1 beta (IL-1beta), and tumour necrosis factor alpha (TNF-alpha) [XREF_BIBR] and also endothelial cells to express intracellular adhesion molecule (ICAM) -1 and vascular cell adhesion molecule (VCAM) -1 [XREF_BIBR], effects which will further promote inflammation.\", \"source\": 431, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32309207610809759\": true}, \"uuid\": {\"003a36a8-73bd-4e61-91d5-895d376a841f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-506014373413407189\": true}, \"source_api\": {\"reach\": true}}, \"target\": 475, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"CRP can stimulate monocytes to release proinflammatory cytokines such as interleukin-6 (IL-6), interleukin-1 beta (IL-1beta), and tumour necrosis factor alpha (TNF-alpha) [XREF_BIBR] and also endothelial cells to express intracellular adhesion molecule (ICAM) -1 and vascular cell adhesion molecule (VCAM) -1 [XREF_BIBR], effects which will further promote inflammation.\", \"source\": 431, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19615999250251214\": true}, \"uuid\": {\"3e40f0be-fad0-451f-93a6-462ef6226e19\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-596602143964714353\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Inhibiting NF-kappaB signaling in renal tubular epithelial lead to reduce tubular injury , apoptosis , necrosis , and accumulation of interstitial inflammatory cells , which consequently results in ameliorated kidney damage after ischemic AKI51 .\", \"source\": 135, \"target\": 241, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"18297593839824962\": true}, \"uuid\": {\"1d12e10e-bd85-43aa-af40-5f63d4585779\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"604553855472311321\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The combined reduction in -cell apoptosis and increase in islet blood flow could cause an increase in insulin secretion and preservation of islet function in diabetes.\", \"source\": 135, \"target\": 156, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"18297593839824962\": true}, \"uuid\": {\"1d12e10e-bd85-43aa-af40-5f63d4585779\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"604553855472311321\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The combined reduction in -cell apoptosis and increase in islet blood flow could cause an increase in insulin secretion and preservation of islet function in diabetes.\", \"source\": 135, \"target\": 156, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21996016281332008\": true}, \"uuid\": {\"6184954a-cb63-4413-8969-cd985243b800\": true}, \"belief\": {\"0.9991246555475\": true}, \"source_hash\": {\"1691424234695049800\": true}, \"source_api\": {\"reach\": true}}, \"target\": 452, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32647915\"}, \"evidence\": \"Coagulopathy was defined as prothrombin time prolonged by 3 s or activated partial thromboplastin time prolonged by 5 s [21].\", \"source\": 455, \"key\": 22}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21996016281332008\": true}, \"uuid\": {\"6184954a-cb63-4413-8969-cd985243b800\": true}, \"belief\": {\"0.9991246555475\": true}, \"source_hash\": {\"1691424234695049800\": true}, \"source_api\": {\"reach\": true}}, \"target\": 452, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32554532\"}, \"evidence\": \"Tissue factor, usually hidden on the subendothelium, is upregulated on platelets, leucocytes and EC during inflammation, leading to activation of both the extrinsic and intrinsic coagulation pathways to make thrombin.\", \"source\": 455, \"key\": 25}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19180985868162148\": true}, \"uuid\": {\"1aa6a2f9-7842-4e79-ae8b-e031d48d4cd5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"6825724686211961094\": true}, \"source_api\": {\"reach\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Differences in cell type, the extent of SP-D multimerization, or differences in size or organization of bacterial aggregates could account for some of the apparent inconsistencies.Although LPS mediates the binding of SP-D to at least some Gram-negative bacteria, SP-D can also bind to specific forms of soluble LPS, particularly `rough' or O-antigen-deficient forms [35,41,44].\", \"source\": 11, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19180985868162148\": true}, \"uuid\": {\"1aa6a2f9-7842-4e79-ae8b-e031d48d4cd5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"6825724686211961094\": true}, \"source_api\": {\"reach\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Although LPS mediates the binding of SP-D to at least some Gram negative bacteria, SP-D can also bind to specific forms of soluble LPS, particularly ` rough ' or O-antigen-deficient forms [XREF_BIBR, XREF_BIBR, XREF_BIBR].\", \"source\": 11, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19180985868162148\": true}, \"uuid\": {\"1aa6a2f9-7842-4e79-ae8b-e031d48d4cd5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"6825724686211961094\": true}, \"source_api\": {\"reach\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Differences in cell type, the extent of SP-D multimerization, or differences in size or organization of bacterial aggregates could account for some of the apparent inconsistencies.Although LPS mediates the binding of SP-D to at least some Gram-negative bacteria, SP-D can also bind to spe- In the latter study the authors suggested that fungal aggregation inhibits phagocytosis.\", \"source\": 11, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23387476518985017\": true}, \"uuid\": {\"76a65981-dd39-4adc-b58a-dd72382070dd\": true}, \"belief\": {\"0.9499940951\": true}, \"source_hash\": {\"-2125468750508666067\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"In contrast, low-dose lipopolysaccharides (LPS) failed to induce ACE2 mRNA expression in Calu-3 cells, suggesting that the activity of rHMGB1 induced ACE2 expression does not depend on endotoxin contamination.\", \"source\": 11, \"target\": 393, \"key\": 32}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22885540058345120\": true}, \"uuid\": {\"9a900677-4a5d-4fb6-9c0d-b88dc3550a8a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5360745888470836392\": true}, \"source_api\": {\"reach\": true}}, \"target\": 436, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR - XREF_BIBR While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-kappaB activation and by IL-1.\", \"source\": 11, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9527340621587867\": true}, \"uuid\": {\"e26b9bcc-177f-458f-abb8-566370a90e45\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3516651434542474699\": true}, \"source_api\": {\"reach\": true}}, \"target\": 443, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"[141] [142] [143] While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-B activation and by IL-1.\", \"source\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"30082883186485362\": true}, \"uuid\": {\"89c5f98e-7607-4924-a6e8-0fe4bdbb2098\": true}, \"belief\": {\"0.992969382\": true}, \"source_hash\": {\"1756523515465160634\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"LPS treatment caused decreased cell viability in the PTECs from both WT and KO mice; and treated KO PTECs exhibited lower viability compared to treated WT PTECs (Fig. 8B).\", \"source\": 11, \"target\": 207, \"key\": 10}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31713712609396788\": true}, \"uuid\": {\"cd7f39b2-96bf-47ab-8605-4dcc3e8fa66e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4179698419426217256\": true}, \"source_api\": {\"reach\": true}}, \"target\": 528, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30293076\"}, \"evidence\": \"With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37].\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33343686949768930\": true}, \"uuid\": {\"a895446c-be54-4e84-b7df-7108bf196f18\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"897406505870257708\": true}, \"source_api\": {\"reach\": true}}, \"target\": 10, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"SP-D also potentiates IAV-induced hydrogen peroxide responses in neutrophils, and protects neutrophils from IAV-induced inhibition of respiratory burst responses.\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13253510853271293\": true}, \"uuid\": {\"adabf29d-0a80-4f1b-8fee-1d5d239afb99\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3499292497447331218\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Human SP-A, but not SP-D, increases the uptake and deacylation of tritiated E. coli LPS by isolated alveolar macrophages [52], but the biological significance of this finding is uncertain.SP-D binds in a CRD-dependent manner to gpA (gp-140), a heavily mannosylated glycoprotein associated with the trophozoites and cysts of Pneumocystis carinii [53].\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13253510853271293\": true}, \"uuid\": {\"adabf29d-0a80-4f1b-8fee-1d5d239afb99\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3499292497447331218\": true}, \"source_api\": {\"reach\": true}}, \"target\": 11, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Human SP-A, but not SP-D, increases the uptake and deacylation of tritiated E. coli LPS by isolated alveolar macrophages[52], but the biological significance of this finding is uncertain.Pneumocystis SP-D binds in a CRD-dependent manner to gpA (gp-140), a heavily mannosylated glycoprotein associated with the trophozoites and cysts of Pneumocystis carinii [53].\", \"source\": 367, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31917977424806800\": true}, \"uuid\": {\"51c0149f-61ba-4768-9872-bf5c321d0270\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"422115551004187757\": true}, \"source_api\": {\"reach\": true}}, \"target\": 48, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"For example, Poulain et al have shown that SP-D can mediate the formation of characteristic tubular configurations of lipid mixtures containing PI [XREF_BIBR].\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9708194613378357\": true}, \"uuid\": {\"6eeb2c32-5223-4301-ad2e-156eb2c6584f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4179698419426217256\": true}, \"source_api\": {\"reach\": true}}, \"target\": 50, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30293076\"}, \"evidence\": \"With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37].\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19141207586114327\": true}, \"uuid\": {\"39001305-8d0d-47c5-bfd8-e3ef3785e351\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-46049251677988734\": true}, \"source_api\": {\"reach\": true}}, \"target\": 361, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"When infection occurs, the exposed collagenous tail of SP-D adheres to the calreticulin/CD91 receptor complex and initiates a pro-inflammatory response by stimulating p38 MAPK phyosphorylation and NF-B activation.\", \"source\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12848473928981499\": true}, \"uuid\": {\"40241258-f45a-4e8f-af25-882ea19563f4\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"2568828765844583583\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"These findings from in vivo study demonstrated that lack of SP-D in lung and/or kidney promotes severe renal injury, tubular cell apoptosis, NF-B activation and production of pro-inflammatory cytokines, which consequently results in aggravated lung-kidney crosstalk during pneumonia-induced AKI.\", \"source\": 367, \"target\": 135, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12848473928981499\": true}, \"uuid\": {\"40241258-f45a-4e8f-af25-882ea19563f4\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"2568828765844583583\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"We found that renal tubular necrosis/apoptosis and inflammatory responses are important mechanisms involving in bacterial pneumonia-induced AKI, in which both pulmonary and renal SP-D participates in the lung-kidney crosstalk and directly or indirectly modulate renal tubular apoptosis and its inflammatory responses.\", \"source\": 367, \"target\": 135, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2898540271646475\": true}, \"uuid\": {\"8a26e6d9-1401-48a8-be4b-fa78ae8ff220\": true}, \"belief\": {\"0.9824770365\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.Published: xx xx xxxx OPEN www.nature.com/scientificreports/AKI is a serious complication affecting more than half of all ICU patients 1 .\", \"source\": 367, \"target\": 135, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2898540271646475\": true}, \"uuid\": {\"8a26e6d9-1401-48a8-be4b-fa78ae8ff220\": true}, \"belief\": {\"0.9824770365\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"SP-D deficiency increases renal tubular cell apoptosis and caspase-3 activation.\", \"source\": 367, \"target\": 135, \"key\": 7}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2898540271646475\": true}, \"uuid\": {\"8a26e6d9-1401-48a8-be4b-fa78ae8ff220\": true}, \"belief\": {\"0.9824770365\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-kappaB signaling.\", \"source\": 367, \"target\": 135, \"key\": 8}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2898540271646475\": true}, \"uuid\": {\"8a26e6d9-1401-48a8-be4b-fa78ae8ff220\": true}, \"belief\": {\"0.9824770365\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"To our knowledge, this study is the first report to demonstrate that both pulmonary and renal SP-D protects from pneumonia-induced AKI through modulating lung-kidney crosstalk and attenuating renal tubular apoptosis, NF-B activation and pro-inflammatory cytokine production.Many clinical and experimental studies demonstrated mechanical ventilation as an important mediator of AKI in ALI and ARDS36,37.\", \"source\": 367, \"target\": 135, \"key\": 10}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2898540271646475\": true}, \"uuid\": {\"8a26e6d9-1401-48a8-be4b-fa78ae8ff220\": true}, \"belief\": {\"0.9824770365\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.\", \"source\": 367, \"target\": 135, \"key\": 11}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15946754346973587\": true}, \"uuid\": {\"7375ec3f-80ed-4bbf-8f9d-56144a1e7ff2\": true}, \"belief\": {\"0.9961499877237129\": true}, \"source_hash\": {\"-46049251677988734\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"bel\": \"p(HGNC:10803 ! SFTPD) increases bp(GO:\\\"GO:0006954\\\" ! \\\"inflammatory response\\\")\"}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"20036966\"}, \"evidence\": \"In addition to direct antiviral activities, the SP-A and SP-D modulate innate and adaptive immunity and inflammation associated with infection.\", \"source\": 367, \"target\": 136, \"key\": 11}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15946754346973587\": true}, \"uuid\": {\"7375ec3f-80ed-4bbf-8f9d-56144a1e7ff2\": true}, \"belief\": {\"0.9961499877237129\": true}, \"source_hash\": {\"-46049251677988734\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"bel\": \"p(HGNC:10803 ! SFTPD) increases bp(GO:\\\"GO:0006954\\\" ! \\\"inflammatory response\\\")\"}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"When infection occurs, the exposed collagenous tail of SP-D adheres to the calreticulin/CD91 receptor complex and initiates a pro-inflammatory response by stimulating p38 MAPK phyosphorylation and NF-B activation.\", \"source\": 367, \"target\": 136, \"key\": 15}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17295884124947285\": true}, \"uuid\": {\"a1a35264-7fe8-4920-81d7-fb3ba877c732\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5380862463071672097\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [9] .\", \"source\": 367, \"target\": 155, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17295884124947285\": true}, \"uuid\": {\"a1a35264-7fe8-4920-81d7-fb3ba877c732\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5380862463071672097\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"SP-D is recognized to modulate the antigen presentation to specific major histocompatibility complex class II T-cell hybridomas in vitro in a dose-dependent fashion.\", \"source\": 367, \"target\": 155, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-7470167996696153\": true}, \"uuid\": {\"9c9d7af0-424c-4860-ac7e-66b0f16f0067\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5537783239577327453\": true}, \"source_api\": {\"reach\": true}}, \"target\": 402, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"SP-D deficiency increases renal tubular cell apoptosis and caspase-3 activation.\", \"source\": 367, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-16359939086956041\": true}, \"uuid\": {\"c0d08611-be98-4056-b71f-dd83e01b7882\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"3605943661160015577\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Notably , human bronchoalveolar lavage ( BAL ) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro , and depletion of SP-D from BAL reduces its HA inhibitory activity [ 61 ] .\", \"source\": 367, \"target\": 224, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-16359939086956041\": true}, \"uuid\": {\"c0d08611-be98-4056-b71f-dd83e01b7882\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"3605943661160015577\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [XREF_BIBR].\", \"source\": 367, \"target\": 224, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-900810055929715\": true}, \"uuid\": {\"ed6c333d-c1f7-450c-b849-4a6f8e6af7cf\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4147769329169529666\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Pulmonary SP-D deficiency leads to Decreased bacterial clearance and deteriorated lung injury in pneumonia model.\", \"source\": 367, \"target\": 258, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31058115641331605\": true}, \"uuid\": {\"04d65878-be44-4d1b-8592-f3d6cdc02d87\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8487226968937687682\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"SP-D is recognized to modulate the antigen presentation to specific major histocompatibility complex class II T-cell hybridomas in vitro in a dose-dependent fashion .\", \"source\": 367, \"target\": 282, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9923588567946431\": true}, \"uuid\": {\"7e545830-e6b9-459d-8c2a-9d90b7fa6749\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4147769329169529666\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Pulmonary SP-D deficiency leads to Decreased bacterial clearance and deteriorated lung injury in pneumonia model.\", \"source\": 367, \"target\": 311, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11342553235450213\": true}, \"uuid\": {\"8a169082-616e-4cc2-8a47-ef9c122b91fa\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6571794615333519518\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 104, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis , inflammation and NF-kappaB signaling .\", \"source\": 367, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11342553235450213\": true}, \"uuid\": {\"8a169082-616e-4cc2-8a47-ef9c122b91fa\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6571794615333519518\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 104, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis , inflammation and NF-kappaB signaling OPEN .\", \"source\": 367, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11752193968928824\": true}, \"uuid\": {\"357f8839-72be-42ca-9649-cb410da3b695\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"4971151349772318241\": true}, \"source_api\": {\"reach\": true}}, \"target\": 351, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Figure 7 : SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney .\", \"source\": 367, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11752193968928824\": true}, \"uuid\": {\"357f8839-72be-42ca-9649-cb410da3b695\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"4971151349772318241\": true}, \"source_api\": {\"reach\": true}}, \"target\": 351, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney .\", \"source\": 367, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11752193968928824\": true}, \"uuid\": {\"357f8839-72be-42ca-9649-cb410da3b695\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"4971151349772318241\": true}, \"source_api\": {\"reach\": true}}, \"target\": 351, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-kappaB signaling.\", \"source\": 367, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-12877185710729354\": true}, \"uuid\": {\"52aee159-fa43-4407-971d-f6c4ed4ec3d7\": true}, \"belief\": {\"0.992999173314\": true}, \"source_hash\": {\"4299232301609147038\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"bel\": \"p(HGNC:10803 ! SFTPD) decreases bp(GO:\\\"GO:0006909\\\" ! phagocytosis)\"}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Furthermore, rat SP-D inhibits the phagocytosis of Candida albicans by these cells [57].\", \"source\": 367, \"target\": 134, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"84480173854834\": true}, \"uuid\": {\"3a6dab2e-1fa7-4738-8ddb-98ca1e8bd199\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8638150641548722338\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"For example, SP-D can decrease interleukin-2-dependent T-lymphocyte proliferation [95].\", \"source\": 367, \"target\": 165, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-22624613601731544\": true}, \"uuid\": {\"2a261aa8-57bd-411f-ad45-9e50282965a7\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"446707300337361949\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Figure 7 : SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney .\", \"source\": 367, \"target\": 241, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-22624613601731544\": true}, \"uuid\": {\"2a261aa8-57bd-411f-ad45-9e50282965a7\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"446707300337361949\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney .\", \"source\": 367, \"target\": 241, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-15620747982771875\": true}, \"uuid\": {\"a49aa54f-ec63-4e61-b17d-3e9829db5238\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-6732477509169708622\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Although the overexpression of wild-type SP-D in type II pneumocytes with the SP-D-deficient mice can prevent the lipidosis and inflammatory changes [94], the ability of overexpressed wild-type SP-D or exogenous SP-D to ameliorate these abnormalities has not yet been described.\", \"source\": 367, \"target\": 244, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23305150793009416\": true}, \"uuid\": {\"90c45373-58a4-42bc-89c6-0f7622a57615\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.Published: xx xx xxxx OPEN www.nature.com/scientificreports/AKI is a serious complication affecting more than half of all ICU patients 1 .\", \"source\": 367, \"target\": 286, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23305150793009416\": true}, \"uuid\": {\"90c45373-58a4-42bc-89c6-0f7622a57615\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1427251683479620420\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.\", \"source\": 367, \"target\": 286, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10354989232126673\": true}, \"uuid\": {\"5efbfec7-ed8d-4632-a35b-f876a79b93c6\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4724133515130203385\": true}, \"source_api\": {\"reach\": true}}, \"target\": 99, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Similarly, in streptozotocin-induced diabetic mice, a T1DM model, ACE2 protein expression is elevated in the early (Wysocki et al., 2006) and decreased in the late (Tikellis et al., 2003) phase of diabetic nephropathy.\", \"source\": 393, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10354989232126673\": true}, \"uuid\": {\"5efbfec7-ed8d-4632-a35b-f876a79b93c6\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4724133515130203385\": true}, \"source_api\": {\"reach\": true}}, \"target\": 99, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Similarly, in streptozotocin-induced diabetic mice, a T1DM model, ACE2 protein expression is elevated in the early and decreased in the late (Tikellis et al., 2003) phase of diabetic nephropathy.\", \"source\": 393, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28110951017181848\": true}, \"uuid\": {\"9432f3f2-6b5f-4e91-b64b-cab4aeb40411\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5200589895390077186\": true}, \"source_api\": {\"reach\": true}}, \"target\": 99, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity.\", \"source\": 393, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7689219948275611\": true}, \"uuid\": {\"018c9b08-4810-4e46-97b4-c4b5cd3e7c51\": true}, \"belief\": {\"0.9499999956953279\": true}, \"source_hash\": {\"2063445003748420189\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis.\", \"source\": 393, \"target\": 135, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7689219948275611\": true}, \"uuid\": {\"018c9b08-4810-4e46-97b4-c4b5cd3e7c51\": true}, \"belief\": {\"0.9499999956953279\": true}, \"source_hash\": {\"2063445003748420189\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis.\", \"source\": 393, \"target\": 135, \"key\": 12}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26599410028295207\": true}, \"uuid\": {\"ae8b55c5-0e51-45d1-aa57-6c3bd2885d2b\": true}, \"belief\": {\"0.9974999114265\": true}, \"source_hash\": {\"-1650041487186134197\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true, \"may\": true, \"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"ACE2 plays an important role in the reninangiotensin system (RAS), and the imbalance between ACE/Ang II/AT1R pathway and ACE2/Ang (17)/Mas receptor pathway in the RAS system will lead to multi-system inflammation [4].\", \"source\": 393, \"target\": 136, \"key\": 22}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26599410028295207\": true}, \"uuid\": {\"ae8b55c5-0e51-45d1-aa57-6c3bd2885d2b\": true}, \"belief\": {\"0.9974999114265\": true}, \"source_hash\": {\"-1650041487186134197\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true, \"may\": true, \"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"ACE2 plays an important role in the renin-angiotensin system (RAS), and the imbalance between ACE and Ang II/AT1R pathway and ACE2 and Ang (1-7)/Mas receptor pathway in the RAS system will lead to multi-system inflammation [XREF_BIBR].\", \"source\": 393, \"target\": 136, \"key\": 23}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-2233124740665336\": true}, \"uuid\": {\"60827fa8-2902-4d00-ac0c-956725b499bd\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"2159145329239424735\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In the islets, ACE2 may modulate blood flow and morphology in order to maintain insulin secretion (Fig. 3).\", \"source\": 393, \"target\": 156, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"29249197072688277\": true}, \"uuid\": {\"b8596c32-8ba8-46da-a826-fc3cd88cd0da\": true}, \"belief\": {\"0.94999995217031\": true}, \"source_hash\": {\"-7298177518244109876\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"The viral spike protein of SARS-Cov2 seems to induce a TNF--converting enzyme (TACE)-dependent alteration of ACE-2, which allows virus penetration into host cells, and the more TNF- levels are increased the more this pathway is facilitated [43].\", \"source\": 393, \"target\": 170, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-21266046860969280\": true}, \"uuid\": {\"ab298671-ab2c-4959-8235-cfa15eac886d\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1609765601049195028\": true}, \"source_api\": {\"reach\": true}}, \"target\": 463, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII.\", \"source\": 393, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25312692021440862\": true}, \"uuid\": {\"f4eab102-7670-4835-9b71-5b075ff79bf0\": true}, \"belief\": {\"0.9975\": true}, \"source_hash\": {\"-2286467406343614465\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true, \"hypothesize\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 393, \"target\": 193, \"key\": 52}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25312692021440862\": true}, \"uuid\": {\"f4eab102-7670-4835-9b71-5b075ff79bf0\": true}, \"belief\": {\"0.9975\": true}, \"source_hash\": {\"-2286467406343614465\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true, \"hypothesize\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 393, \"target\": 193, \"key\": 55}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35000116904293300\": true}, \"uuid\": {\"6c7c4dc3-6cde-4cd6-a604-a1b17bbb36be\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"561754501282917108\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ACE2, a new component of the RAS, is elevated in diabetes and diabetic nephropathy and may be involved in a compensatory mechanism opposing the ACE/Ang-II/AT1-receptor axis.\", \"source\": 393, \"target\": 214, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28901764041630522\": true}, \"uuid\": {\"20b876f4-1b07-46e2-b467-89879430ec4b\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-1286808115612273956\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Finally in two models of T1DM (streptozotocin and Akita mouse models), ACE2 gene deletion accelerated the development of diabetic kidney disease (Wong et al., 2007, Tikellis et al., 2008).\", \"source\": 393, \"target\": 214, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6537756584957430\": true}, \"uuid\": {\"83f6786f-f9dd-4b39-b214-af4b4d0a4c02\": true}, \"belief\": {\"0.9973784974922894\": true}, \"source_hash\": {\"8462008536023836879\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis.\", \"source\": 393, \"target\": 221, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6537756584957430\": true}, \"uuid\": {\"83f6786f-f9dd-4b39-b214-af4b4d0a4c02\": true}, \"belief\": {\"0.9973784974922894\": true}, \"source_hash\": {\"8462008536023836879\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis.\", \"source\": 393, \"target\": 221, \"key\": 12}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26373033894925085\": true}, \"uuid\": {\"11c02bba-4af3-47b1-892e-d9006ff858c0\": true}, \"belief\": {\"0.9998689203560914\": true}, \"source_hash\": {\"1986432523719170787\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"However, it might also attenuate the inhibition of RAS by ACE2 in infected organs.\", \"source\": 393, \"target\": 259, \"key\": 98}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26373033894925085\": true}, \"uuid\": {\"11c02bba-4af3-47b1-892e-d9006ff858c0\": true}, \"belief\": {\"0.9998689203560914\": true}, \"source_hash\": {\"1986432523719170787\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"However , it might also attenuate the inhibition of RAS by ACE2 in infected organs .\", \"source\": 393, \"target\": 259, \"key\": 148}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31191593303241843\": true}, \"uuid\": {\"a58db00a-7e82-48e9-84aa-07ce2a04b39d\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"-3701504269372903359\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Deficiency in ACE-2 leads to increased formation of atherosclerotic plaques and blocking of ACE-2 results in prevention of atherosclerosis 20 ( Fig. 1 ) .\", \"source\": 393, \"target\": 305, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-6976707069426111\": true}, \"uuid\": {\"cb8b4c6f-35d3-4217-afab-3d033f733d6e\": true}, \"belief\": {\"0.95\": true}, \"source_hash\": {\"7418677185325683355\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In 8-week-old db/db mice, a model of early T2DM, ACE2 expression is elevated while ACE expression is decreased in both glomeruli and cortex (Ye et al., 2004) prior to the development of diabetic nephropathy.\", \"source\": 393, \"target\": 306, \"key\": 14}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"20416062921491507\": true}, \"uuid\": {\"23ed3030-405c-41ca-88ce-d66b3a8027bc\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-1452839003075639252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Deficiency in ACE-2 leads to increased formation of atherosclerotic plaques and blocking of ACE-2 results in prevention of atherosclerosis 20 ( Fig. 1 ) .\", \"source\": 393, \"target\": 316, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"20416062921491507\": true}, \"uuid\": {\"23ed3030-405c-41ca-88ce-d66b3a8027bc\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-1452839003075639252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Deficiency in ACE-2 leads toincreased formation of atherosclerotic plaques and blocking of ACE-2 results inprevention of atherosclerosis20(Fig. 1).\", \"source\": 393, \"target\": 316, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4651094153485196\": true}, \"uuid\": {\"f01cbdf7-6285-4393-966c-526e2adf11c5\": true}, \"belief\": {\"0.349125\": true}, \"source_hash\": {\"8992705003985303283\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"negated\": {\"true\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32660650\"}, \"evidence\": \"ACE2 catalyzes angiotensin II conversion to angiotensin-(1-7), and the ACE2/angiotensin-(1-7)/MAS axis counteracts the negative effects of the renin-angiotensin system (RAS), which plays important roles in maintaining the physiological and pathophysiological balance of the body.\", \"source\": 393, \"key\": 63}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4651094153485196\": true}, \"uuid\": {\"f01cbdf7-6285-4393-966c-526e2adf11c5\": true}, \"belief\": {\"0.349125\": true}, \"source_hash\": {\"8992705003985303283\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"negated\": {\"true\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1 and ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2 mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1 generated ANGII.\", \"source\": 393, \"key\": 66}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4651094153485196\": true}, \"uuid\": {\"f01cbdf7-6285-4393-966c-526e2adf11c5\": true}, \"belief\": {\"0.349125\": true}, \"source_hash\": {\"8992705003985303283\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}, \"negated\": {\"true\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII.\", \"source\": 393, \"key\": 92}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"23690377257933350\": true}, \"uuid\": {\"4c9e0a1c-aec5-4f42-b0c6-f0414724d50b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1291268755310223628\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"In the heart, ACE-2 prevents progressivecardiac fibrosis associated with aging and/or cardiac pressure overloadand is beneficial in heart failure1920.\", \"source\": 393, \"target\": 140, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"9254964375465689\": true}, \"uuid\": {\"c8c5a719-e9ae-4029-b1eb-618e9bf06117\": true}, \"belief\": {\"0.949271\": true}, \"source_hash\": {\"3023617629629695993\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 437, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Indeed , several studies demonstrated that loss or inhibition of ACE2 exacerbates diabetic nephropathy Soler et al ., 2007 ; Wong et al ., 2007 ; Tikellis et al ., 2008 ) .\", \"source\": 393, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"9254964375465689\": true}, \"uuid\": {\"c8c5a719-e9ae-4029-b1eb-618e9bf06117\": true}, \"belief\": {\"0.949271\": true}, \"source_hash\": {\"3023617629629695993\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 437, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"This compensatory effect of ACE2 in diabetes has already been suggested in the kidney by studies demonstrating that ACE2 inhibition worsens diabetic nephropathy .\", \"source\": 393, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7693060271668066\": true}, \"uuid\": {\"7c9f704a-43f4-47b3-90a5-8a8fb4d9054f\": true}, \"belief\": {\"0.0\": true}, \"source_hash\": {\"8779262782798026620\": true}, \"source_api\": {\"eidos\": true}, \"negated\": {\"true\": true}}, \"target\": 77, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"64 MgD has been implicated in perturbations of pancreatic beta-cell function , hyperinsulinaemia , IR and CVD due to its diverse and essential roles in an extensive list of cellular metabolic pathways , not least counting ATP transport , DNA repair capacity and cell viability .\", \"source\": 207, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33518452083724185\": true}, \"uuid\": {\"a6c3743e-225a-4e7e-ab51-25d71fba8024\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6242934954666260444\": true}, \"source_api\": {\"reach\": true}}, \"target\": 553, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19333547\"}, \"evidence\": \"However, significant elevation of ALT, an indicator of liver damage, was not associated with higher mortality.\", \"source\": 74, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1386966968726740\": true}, \"uuid\": {\"2ff21ec8-9cb1-4da2-9fb9-b0aec6071358\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"-6992120230558240189\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 447, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"Furthermore , activation of RAGE causes not only an inflammatory gene expression profile but also a positive feed-forward loop , in which inflammatory stimuli activate NF-kappaB , which induces RAGE expression , followed by a sustained NF-kappaB activation [ 18 ] .\", \"source\": 351, \"key\": 1}, {\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"23584417585668779\": true}, \"uuid\": {\"d7b96b49-0ed6-4dec-9b14-f4359ef2f091\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4664198718280115281\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"Furthermore, activation of RAGE causes not only an inflammatory gene expression profile but also a positive feed-forward loop, in which inflammatory stimuli activate NF-kappaB, which induces RAGE expression, followed by a sustained NF-kappaB activation [XREF_BIBR].\", \"source\": 351, \"target\": 447, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23963044191759368\": true}, \"uuid\": {\"c02e7499-c4a9-471c-b44c-17a35899debe\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5576804373287924013\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Even in the absence of inflammation, inhibition of constitutive basal NFkappaB activity in human airway epithelial cells causes an up-regulation of cldn5 expression, disturbs TJ organisation and increases paracellular permeabilty [XREF_BIBR].\", \"source\": 351, \"target\": 427, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23963044191759368\": true}, \"uuid\": {\"c02e7499-c4a9-471c-b44c-17a35899debe\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5576804373287924013\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Inhibition of basal NFkappaB activity in airway epithelial cells increases cldn5 expression in the absence of inflammation [XREF_BIBR].\", \"source\": 351, \"target\": 427, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11527240298439642\": true}, \"uuid\": {\"df117d1f-ac06-4a45-9a75-28bba970028a\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-1889233530757485385\": true}, \"source_api\": {\"reach\": true}}, \"target\": 427, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Overall, NFkappaB dependent reduction of cldn5 seems to improve epithelial barrier function during lung inflammation and seems to be beneficial with respect to epithelial function.\", \"source\": 351, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20924132852373770\": true}, \"uuid\": {\"937580ec-5c39-4978-8759-aa5f7e50be9e\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5819746239313574874\": true}, \"source_api\": {\"reach\": true}}, \"target\": 441, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32375153\"}, \"evidence\": \"It binds to chromosomal DNA but also to Toll-like receptor 3 (TLR3), TLR4, and the receptor for advanced glycation end products (RAGE) that activates nuclear factor (NF)-B ( Fig. 1a) , which mediate the upregulation of leukocyte adhesion molecules as well as the production of proinflammatory cytokines and angiogenic factors that promote inflammation.\", \"source\": 447, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"19701749986649897\": true}, \"uuid\": {\"1a7b579c-de11-4b78-ad34-35f145c0e738\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5221922205023663789\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 447, \"target\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"19701749986649897\": true}, \"uuid\": {\"1a7b579c-de11-4b78-ad34-35f145c0e738\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5221922205023663789\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 447, \"target\": 193, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10062175476674025\": true}, \"uuid\": {\"0099868f-68bc-4d3c-9e74-5130e62e5e3d\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"7527317255325707226\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"In other cells, including endothelial cells, airway smooth muscle cells, vascular smooth muscle cells, neurons, and immune cells, RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation.\", \"source\": 447, \"target\": 447, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16316264278891794\": true}, \"uuid\": {\"6300a7a8-77a1-4d9d-86ec-69829e2f3d57\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7737394362393587147\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"In addition to the well-known inflammatory activity of HMGB1, we found that the exogenous human HMGB1 protein (\\\" rHMGB1 \\\", purity :> 95%; endotoxin level : < 0.10 EU per 1 mug) time-dependently induced the upregulation of ACE2 mRNA in Calu-3 cells, a human lung epithelial cell line.\", \"source\": 466, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27255258905333720\": true}, \"uuid\": {\"3479f353-c81e-41fa-97b6-d965ae8c2719\": true}, \"belief\": {\"0.9989788027355\": true}, \"source_hash\": {\"-1237438280828349886\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Figure2 : Recombinant human HMGB1 induces ACE2 expression.\", \"source\": 466, \"target\": 393, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27255258905333720\": true}, \"uuid\": {\"3479f353-c81e-41fa-97b6-d965ae8c2719\": true}, \"belief\": {\"0.9989788027355\": true}, \"source_hash\": {\"-1237438280828349886\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Exogenous HMGB1 induces the expression of SARS-CoV-2 entry receptor ACE2 in alveolar epithelial cells in an AGER dependent manner.\", \"source\": 466, \"target\": 393, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27255258905333720\": true}, \"uuid\": {\"3479f353-c81e-41fa-97b6-d965ae8c2719\": true}, \"belief\": {\"0.9989788027355\": true}, \"source_hash\": {\"-1237438280828349886\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"These findings indicate that AGER is required for HMGB1-induced ACE2 expression in lung epithelial cells.\", \"source\": 466, \"target\": 393, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27255258905333720\": true}, \"uuid\": {\"3479f353-c81e-41fa-97b6-d965ae8c2719\": true}, \"belief\": {\"0.9989788027355\": true}, \"source_hash\": {\"-1237438280828349886\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Figure3: AGER is required for HMGB1-induced ACE2 expression in lung epithelial cells. (A) ELISA analysis of TNF release in indicated human monocytes (THP1) following treatment with human HMGB1 protein (200 ng/ml) for 24 h (n = 3, t test). (B) Western blot analysis of protein expression in human lung epithelial cells (Calu-3) after siRNA-mediated depletion of TLR4 or AGER. (C) Q-PCR analysis of ACE2 mRNA in indicated Calu-3 cells following treatment with human HMGB1 protein (200 ng/ml) for 24 h (n = 3, t test).\", \"source\": 466, \"target\": 393, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27255258905333720\": true}, \"uuid\": {\"3479f353-c81e-41fa-97b6-d965ae8c2719\": true}, \"belief\": {\"0.9989788027355\": true}, \"source_hash\": {\"-1237438280828349886\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Recombinant human HMGB1 induces ACE2 expression.\", \"source\": 466, \"target\": 393, \"key\": 13}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7109037409786789\": true}, \"uuid\": {\"30cee953-f93b-46ba-acf4-8bab3401d020\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5221922205023663789\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 466, \"target\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7109037409786789\": true}, \"uuid\": {\"30cee953-f93b-46ba-acf4-8bab3401d020\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5221922205023663789\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 .\", \"source\": 466, \"target\": 193, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8857422784528723\": true}, \"uuid\": {\"a52f9303-b930-4368-8818-d66fcfe65840\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3516838941337507276\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"In particular , HMGB1 directly stimulates platelets through TLR4 , and RAGE , while S100 proteins released from neutrophils and platelets facilitate thrombus formation through RAGE activation ( 26 ) .\", \"source\": 466, \"target\": 204, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8857422784528723\": true}, \"uuid\": {\"a52f9303-b930-4368-8818-d66fcfe65840\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3516838941337507276\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"In particular , HMGB1 directly stimulates platelets through TLR4 , and RAGE , while S100 proteins released from neutrophils and platelets facilitate thrombus formation through RAGE activation ( 26 ) .\", \"source\": 466, \"target\": 204, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"30981930784378452\": true}, \"uuid\": {\"15512f9b-df85-4878-8af0-12dd07828049\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation.\", \"source\": 15, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33679743681569305\": true}, \"uuid\": {\"0ba98cf6-f853-4568-b916-f5ad6866fc41\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationFig.\", \"source\": 15, \"target\": 238, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33679743681569305\": true}, \"uuid\": {\"0ba98cf6-f853-4568-b916-f5ad6866fc41\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation.\", \"source\": 15, \"target\": 238, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33760396524600780\": true}, \"uuid\": {\"a1344ba5-d1b4-4f56-853a-4e8c1d3ed2aa\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5558083028487357300\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116].\", \"source\": 348, \"target\": 173, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26021325730148398\": true}, \"uuid\": {\"65481b1e-44ea-482f-a606-433c6301ff25\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-8206585929295683782\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 362, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs , lowering the levels of spermidine and spermine and thereby dampening influenza replication Role of IFN in IDO activation .\", \"source\": 348, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35066410670911470\": true}, \"uuid\": {\"ec679a59-9094-4e96-b51e-ae95a626d839\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-750671858054908370\": true}, \"source_api\": {\"reach\": true}}, \"target\": 473, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"They do however appear to be closely involved in later stages of the inflammatory response, either through direct activation using the interferon receptor pathway (see below) or through type I IFN mediated trans-presentation of IL-15 by dendritic cells.\", \"source\": 348, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35066410670911470\": true}, \"uuid\": {\"ec679a59-9094-4e96-b51e-ae95a626d839\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-750671858054908370\": true}, \"source_api\": {\"reach\": true}}, \"target\": 473, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"They do however appear to be closely involved in later stages of the inflammatory response, either through direct activation using the interferon receptor pathway (see below) or through type I IFN-mediated trans-presentation of IL-15 by dendritic cells (reviewed by Hwang and colleagues [47]).\", \"source\": 348, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28443240646898058\": true}, \"uuid\": {\"5c0f7a7f-b4b0-40fa-a35e-e31dfa628540\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3532794508678703033\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It has also been demonstrated that type I IFN can stimulate oxygen consumption in a range of cells , including conventional dendritic cells ( DCs ) , keratinocytes , and memory T cells [ 117 ] .\", \"source\": 348, \"target\": 254, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26613633736140084\": true}, \"uuid\": {\"dd771eab-8b79-49fa-accd-6c5badb532eb\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8070012689609223479\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"Severe Acute Respiratory Syndrome Coronavirus Elicits a Weak Interferon Response Compared to Traditional Interferon-Inducing Viruses.Severe acute respiratory syndrome (SARS) is a newly identified infectious disease and its causative agent has been convincingly identified as a new member of the Coronaviridae family called SARS-CoV [1].\", \"source\": 348, \"target\": 269, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6382340187331536\": true}, \"uuid\": {\"102e4762-fcc8-4f51-93f6-dd1e0d75873a\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4343953383924740739\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The IFN response induces greatly increased levels of iNOS in the lungs of infected mice, leading to the production of a significant amount of NO and peroxynitrite species, which are among the most important pathogenic factors in influenza virus-induced pneumonia in mice [147].\", \"source\": 348, \"target\": 499, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"23947362717466799\": true}, \"uuid\": {\"10cb8cba-2c32-4b13-ad71-2fbfe8cc602d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8521308540096144985\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 516, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"IFNs affect the lipid metabolism through downregulation of SREBP-2 , leading to higher cholesterol biosynthesis , and suppressed host cell apoptosis and PGE-2 production .\", \"source\": 348, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28350191618239927\": true}, \"uuid\": {\"08ea9500-4773-42f0-bb13-e341a57ff0e8\": true}, \"belief\": {\"0.991866\": true}, \"source_hash\": {\"1897457681817436658\": true}, \"source_api\": {\"reach\": true}}, \"target\": 40, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It is also reported that influenza infection significantly increases ROS production by inducing Nox4 , and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [ 16 ] .\", \"source\": 238, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28350191618239927\": true}, \"uuid\": {\"08ea9500-4773-42f0-bb13-e341a57ff0e8\": true}, \"belief\": {\"0.991866\": true}, \"source_hash\": {\"1897457681817436658\": true}, \"source_api\": {\"reach\": true}}, \"target\": 40, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Xanthine oxidase is also an ROS-producing enzyme that is induced by influenza infection [23, 24] , and its inhibition can hinder ROS increase in the cell.On the other hand, increased expression of SOD1 reduces influenza virus titers within the cell [25] .\", \"source\": 238, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28350191618239927\": true}, \"uuid\": {\"08ea9500-4773-42f0-bb13-e341a57ff0e8\": true}, \"belief\": {\"0.991866\": true}, \"source_hash\": {\"1897457681817436658\": true}, \"source_api\": {\"reach\": true}}, \"target\": 40, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It is also reported that influenza infection significantly increases ROS production by inducing Nox4, and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [16] .\", \"source\": 238, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32281869837969739\": true}, \"uuid\": {\"9fe33944-9455-4a47-ba3d-4ed7fc4d618b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5885501480695363850\": true}, \"source_api\": {\"reach\": true}}, \"target\": 359, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Activation of mTORC1&2 signaling and downstream factors by influenza infection may have an essential role in the upregulation of these metabolic processes.\", \"source\": 238, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32281869837969739\": true}, \"uuid\": {\"9fe33944-9455-4a47-ba3d-4ed7fc4d618b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5885501480695363850\": true}, \"source_api\": {\"reach\": true}}, \"target\": 359, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"On the other hand, influenza M2 protein is capable of down-regulation of the mTORC1 inhibitor REDD1, thereby enhancing the mTORC1 activation [69].\", \"source\": 238, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12896211113207335\": true}, \"uuid\": {\"a6c081e5-b4d8-4557-99cc-c4227068996f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7633049643904067338\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism.\", \"source\": 238, \"target\": 128, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4024059557402157\": true}, \"uuid\": {\"d1fe0e45-a10b-4dc5-a24a-b76d79ae7f50\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"-5451089972493558032\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The pentose phosphate shunt, as another glucose-consuming pathway, is enhanced by influenza infection to help produce more nucleotides, especially ATP.\", \"source\": 238, \"target\": 129, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4024059557402157\": true}, \"uuid\": {\"d1fe0e45-a10b-4dc5-a24a-b76d79ae7f50\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"-5451089972493558032\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The pentose phosphate shunt , as another glucose-consuming pathway , is enhanced by influenza infection to help produce more nucleotides , especially ATP .\", \"source\": 238, \"target\": 129, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23231366597679230\": true}, \"uuid\": {\"bb1314b0-82d9-452b-b1cb-dffc7e6a0921\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1602753586112551156\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In this regard, the results of a study revealed that influenza infection could induce fatty acid biosynthesis and cholesterol metabolism in human lung basal epithelial tumor cells [87].\", \"source\": 238, \"target\": 130, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35800235033647489\": true}, \"uuid\": {\"ef5f35c2-783b-4582-ada8-f625783406c2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1602753586112551156\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In this regard, the results of a study revealed that influenza infection could induce fatty acid biosynthesis and cholesterol metabolism in human lung basal epithelial tumor cells [87].\", \"source\": 238, \"target\": 142, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7185196589667620\": true}, \"uuid\": {\"991479a9-666d-4fbe-abc7-fced1af4822a\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"target\": 164, \"key\": 8}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7185196589667620\": true}, \"uuid\": {\"991479a9-666d-4fbe-abc7-fced1af4822a\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"target\": 164, \"key\": 11}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17294560584825954\": true}, \"uuid\": {\"e7131011-333e-4cac-a535-470f84695c44\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"4471618604597702014\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"It should be noted that the 2008 H1N1 virus also has a stop codon at 220, thus suggesting that it behaves similar to H7N1 with the six amino acid deletion.Finally, since there is evidence that severe influenza caused by certain influenza viruses such as H5N1 trigger an exaggerated innate immune response, are there therapeutic options in addition to antiviral therapy that may be utilized to \\\"buy time\\\" before the acquired immune response takes over?\", \"source\": 238, \"target\": 171, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34005892238248828\": true}, \"uuid\": {\"53b35e1a-4cd4-482d-89b6-492aa9dc2b5e\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"target\": 386, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34005892238248828\": true}, \"uuid\": {\"53b35e1a-4cd4-482d-89b6-492aa9dc2b5e\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"target\": 386, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-30220514113405214\": true}, \"uuid\": {\"89a42c73-c947-4fbb-9a1d-546f8a0cf133\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3732301610755248089\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"previously\": true}}, \"target\": 465, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"As mentioned previously , the influenza infection induces upregulation of HIF-1alpha .\", \"source\": 238, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28882375092116453\": true}, \"uuid\": {\"168055e4-681f-4e9f-bee1-feed9c4f4f69\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"key\": 15}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28882375092116453\": true}, \"uuid\": {\"168055e4-681f-4e9f-bee1-feed9c4f4f69\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1384156195328026626\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation.\", \"source\": 238, \"key\": 17}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"202977271735235\": true}, \"uuid\": {\"2214cb95-00b3-48be-8633-9a2e9e551a7a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4078807772300343233\": true}, \"source_api\": {\"reach\": true}}, \"target\": 502, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It is also reported that influenza infection significantly increases ROS production by inducing Nox4, and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [16] .\", \"source\": 238, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2246857647965\": true}, \"uuid\": {\"66b8dd97-4532-4156-8394-10aecf54b009\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"2742564170424315670\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 509, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism .\", \"source\": 238, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2164865314393129\": true}, \"uuid\": {\"4e40ab7b-da6a-4651-9ec6-4953e4bade53\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7027283022700331020\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"As situated above, both influenza A and B viruses display hemagglutinin proteins on their surface, which bind to sialic acids displayed on the host cell surface and mediate pHdependent fusion of the viral membrane with the host cell membrane (Skehel & Wiley, 2000; Lamb & Krug, 2001; Wang et al., 2007b; Wang et al., 2008a; Harrison, 2008; Gamblin & Skehel, 2010).\", \"source\": 238, \"target\": 227, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-30398327221948933\": true}, \"uuid\": {\"cbae67da-30de-42ee-bfd1-3391435e723c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-373015584286651401\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In addition to impaired metabolism in mitochondria, influenza infection induces severe peroxisomal lipid metabolism disorders, which can be inferred from abnormal levels of several specific long-chain fatty acids [101] (Fig. 1).\", \"source\": 238, \"target\": 307, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30370352543575284\": true}, \"uuid\": {\"ac4feb1c-5d0e-465b-a43d-dcf044ae9315\": true}, \"belief\": {\"0.9824993799855\": true}, \"source_hash\": {\"-8846887404481152839\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"With regard to the extacellular compartment the role of NA in cleaving the sialylated mucin and glycosylation of the HA has already been discussed.Within the cell it has been found for a long time that influenza has evolved a strategy to evade the powerful IFN defence mechanism [73] but in 1998 Garcia-Sastre and colleagues showed that the production of recombinant influenza lacking a gene called non-structural protein (NS1) was able to suppress this IFN pathway [74].\", \"source\": 238, \"key\": 12}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25244789628790120\": true}, \"uuid\": {\"b60ee56d-c84f-45b1-8ca2-9f520cc21c6b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation.\", \"source\": 7, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7257015975157854\": true}, \"uuid\": {\"1b69c32c-6fe7-4a8e-bf46-698099de1f9f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationFig.\", \"source\": 7, \"target\": 238, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7257015975157854\": true}, \"uuid\": {\"1b69c32c-6fe7-4a8e-bf46-698099de1f9f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2163151375925698289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation.\", \"source\": 7, \"target\": 238, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28982042356491829\": true}, \"uuid\": {\"1e382fd7-26a9-4fab-94bf-34fc44f1ae6b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1319341727771515445\": true}, \"source_api\": {\"reach\": true}}, \"target\": 2, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Glucose oxidation consumes four nicotinamide adenine dinucleotide (NAD+) to produce two acetyl moieties, whereas beta oxidation consumes two, ketolysis one and acetoacetate none (XREF_SUPPLEMENTARY); consequently, glucose oxidation depletes the NAD+ intracellularpool more than the other three substrates combined, thereby decreasing NAD+ availability for mitochondrial deacetylase sirtuin 3 (SIRT3) activity.\", \"source\": 20, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28173404751544197\": true}, \"uuid\": {\"ed1ad3f9-b242-4526-97e1-df835d26e676\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8447332272834679012\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In a study by Kohio and Adamson, a dose-specific increase in influenza infection was associated with higher glucose levels, whereas the treatment of cells with glycolysis inhibitors remarkably suppressed the viral replication.\", \"source\": 20, \"target\": 164, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1710426607458109\": true}, \"uuid\": {\"51729d91-755e-42c1-8f3e-960967c15514\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1117654818606650911\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Furthermore, some hydroxylation of inactive to active vitamin D occurs intracellularly within other tissues and may be inhibited by excessive mitochondrial ROS and NAD+ glucose metabolism depletion.\", \"source\": 20, \"target\": 97, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15880147849414506\": true}, \"uuid\": {\"7310c10c-c1bd-47b5-812e-ca21b51b82af\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-913302861349556363\": true}, \"source_api\": {\"reach\": true}}, \"target\": 527, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan.\", \"source\": 78, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15912073298086723\": true}, \"uuid\": {\"cf73696c-c97d-49f0-b496-3aaeae2442e3\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-913302861349556363\": true}, \"source_api\": {\"reach\": true}}, \"target\": 81, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan.\", \"source\": 78, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32775278412457205\": true}, \"uuid\": {\"13daaf6b-20ce-44f1-adae-0e43b8d53575\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7906119930501508434\": true}, \"source_api\": {\"reach\": true}}, \"target\": 432, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Delputte PL & Nauwynck HJ (2004) Porcine arterivirus infection of alveolar macrophages is mediated by sialic acid on the virus.\", \"source\": 41, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-6246018225447974\": true}, \"uuid\": {\"f3b831b8-b4ec-419e-81c5-7667fa0d2fa0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-9019739462130699330\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Delputte PL & Nauwynck HJ (2004) Porcine arterivirus infection of alveolar macrophages is mediated by sialic acid on the virus.\", \"source\": 41, \"target\": 284, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35124073742055043\": true}, \"uuid\": {\"c785fcaf-5860-4fbe-978f-f2b0c8d16dd5\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5990416385963911003\": true}, \"source_api\": {\"reach\": true}}, \"target\": 97, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Thus, the association with lower vitamin D may be both a marker and maker of poor health due to hyperinsulinaemia inducing low vitamin D status and availability, as a result of vitamin D sequestration into adipocytes and prevention of hydroxylation by mitochondrial ROS sensitive 1alpha-25OHD-hydroxylase.\", \"source\": 40, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3533076354195680\": true}, \"uuid\": {\"3cc91771-d7b7-4beb-9bd3-eeaa24232c20\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1117654818606650911\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Furthermore, some hydroxylation of inactive to active vitamin D occurs intracellularly within other tissues and may be inhibited by excessive mitochondrial ROS and NAD+ glucose metabolism depletion.\", \"source\": 40, \"target\": 97, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12751175263875363\": true}, \"uuid\": {\"21bb863e-ba27-4d28-b92a-aa5804918316\": true}, \"belief\": {\"0.982423455\": true}, \"source_hash\": {\"-7707237897567325438\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32228322\"}, \"evidence\": \"AngII, through the activation of Ang II type 1a receptors induces vasoconstriction and proliferation, whereas Ang 1-7 stimulates vasodilatation and suppresses cell growth (XREF_FIG).\", \"source\": 449, \"target\": 144, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1491099230967129\": true}, \"uuid\": {\"40f127ce-3e0a-4f36-a7e1-99f89d4f4b9b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1081981234178637093\": true}, \"source_api\": {\"reach\": true}}, \"target\": 501, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"NOX activation is triggered by interaction of Ang-II with the AT1 receptor and requires the assembly of the cytosolic subunits p47phox, p40phox and p67phox on the membrane bound scaffold proteins p22phox and Nox2/4.\", \"source\": 449, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23734119708958166\": true}, \"uuid\": {\"d42b5ee8-99d8-4e5f-8e42-f1f74f34c9d5\": true}, \"belief\": {\"0.95\": true}, \"source_hash\": {\"-9021326961095184337\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Also , activation of macrophages by IFNgamma induces expression of the ATP-citrate lyase enzyme ( ACLY ) , and blockage of ACLY activity reduces the production of ROS and nitric oxide [ 120 ] .\", \"source\": 470, \"target\": 247, \"key\": 48}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31546050066204542\": true}, \"uuid\": {\"9710aa2c-1eb5-41c4-97c7-d7bafa8efa76\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3330453286283531136\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"IFN-gamma enhances cldn7 expression and the transepithelial electrical resistance in submandibular glands [XREF_BIBR].\", \"source\": 470, \"target\": 428, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31587987929653800\": true}, \"uuid\": {\"d0660d4d-8bca-42c7-abef-67547c9794e0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5134636651840275727\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"target\": 36, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32942131\"}, \"evidence\": \"Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production .\", \"source\": 247, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14795217042229381\": true}, \"uuid\": {\"86396dce-4128-4659-83c0-adb54b22fad9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1340480226631380510\": true}, \"source_api\": {\"reach\": true}}, \"target\": 428, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Phosphorylation of cldn7 within its C-terminal, intracellular domain via WNK4 kinase modulates cldn7 permeability.\", \"source\": 428, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-10638066408814471\": true}, \"uuid\": {\"c9a066e6-2efa-482c-9b20-51f115980961\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1176022098009464946\": true}, \"source_api\": {\"reach\": true}}, \"target\": 27, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Silencing of cldn7 expression in LLC-PK1 reduces transepithelial resistance and increases paracellular permselectivity for Na + over Cl - [XREF_BIBR].\", \"source\": 428, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20932380897038526\": true}, \"uuid\": {\"af6f9f54-b5bc-4fb6-8129-5e06b3e9e786\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-421409972400661301\": true}, \"source_api\": {\"reach\": true}}, \"target\": 18, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem.\", \"source\": 403, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35215417452828276\": true}, \"uuid\": {\"26890249-dc6e-48ce-a461-b4c3c33db8e9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5726246861465007516\": true}, \"source_api\": {\"reach\": true}}, \"target\": 409, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"For instance, it has been shown that multivalent mannosecontaining molecules or mannose-based glycomimetics can compromise binding of HIV-1 gp120 with DC-SIGN (Wang et al., 2008b; Luallen et al., 2009; Martinez-Avila et al., 2009b; Becer et al., 2010) and inhibit DC-SIGN-mediated trans-infection of CD4 + T lymphocytes (Martinez-Avila et al., 2009a; Sattin et al., 2010; Berzi et al., 2012) .\", \"source\": 64, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-29077287704643725\": true}, \"uuid\": {\"2d2bc668-1c91-4d89-ac4b-8cbea7c97334\": true}, \"belief\": {\"0.3324999998644028\": true}, \"source_hash\": {\"7504302399780364241\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"if\": true, \"can\": true}, \"negated\": {\"true\": true}}, \"target\": 116, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"Next , we looked at cytokines that CD4 + T cells produce .\", \"source\": 409, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"3234812795832705\": true}, \"uuid\": {\"35ce5639-33af-4896-ba94-85b1706cd532\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-792586659688719795\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"If and how CD4 + T cells contribute to MERS-CoV pathogenesis is not clear, but we found alterations in the CD4 + T cell response to MERS-CoV infection in diabetic mice although to a lesser extent than we found differences in the IMM response.\", \"source\": 409, \"target\": 146, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3424560313106747\": true}, \"uuid\": {\"e892d0ab-a68e-4483-9578-5338f606cac9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7826385146933890099\": true}, \"source_api\": {\"reach\": true}}, \"target\": 460, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"BOB is supported by Ong Tiong Tat Professorship.\", \"source\": 541, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-19243500054824934\": true}, \"uuid\": {\"41a92cc8-1c9e-4711-ada9-644ed63b97b6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6220278726923779622\": true}, \"source_api\": {\"reach\": true}}, \"target\": 539, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"These NA-inhibitors are most commonly N-Acetylneuraminic acid (Neu5Ac) derivatives, which competitively inhibit NA activity [104].\", \"source\": 71, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33001816086018404\": true}, \"uuid\": {\"306f5051-7e02-4ab3-b86b-a89a04e54cd5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-6039830897329989994\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Rosiglitazone, a thiazolidinedione (TZD), upregulates vascular ACE2 protein expression in hypertensive rats (98).\", \"source\": 76, \"target\": 393, \"key\": 8}, {\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"-519760512378577\": true}, \"uuid\": {\"a0ae56a4-3399-40f9-b7a4-748756cb8c00\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-8857932317587267001\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"AMPK itself has been shown to increase ACE2 stability by phosphorylating ACE2 Ser 680 in human umbilical vein endothelial cells and humanstorm or vascular damage and finally lead to reduction in mortality rate.\", \"source\": 335, \"target\": 394, \"key\": 15}, {\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"-519760512378577\": true}, \"uuid\": {\"a0ae56a4-3399-40f9-b7a4-748756cb8c00\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"-8857932317587267001\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"AMPK itself has been shown to increase ACE2 stability by phosphorylating ACE2 Ser 680 in human umbilical vein endothelial cells and human embryonic kidney 293cells.\", \"source\": 335, \"target\": 394, \"key\": 16}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20897183679452725\": true}, \"uuid\": {\"09cd85d1-df4b-461d-8c72-5843461f8328\": true}, \"belief\": {\"0.9961331601\": true}, \"source_hash\": {\"4743951378660910246\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2 , CCL5 and CD11b + cells and decrease IFNgamma secretion but did not affect IFNalpha , CCL2 , IL6 or TNFalpha , thus showing that endothelial cells are able to actually modulate the inflammatory response [ 71 ] .\", \"source\": 296, \"target\": 136, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3112751348420056\": true}, \"uuid\": {\"26e76899-d270-4ba8-a7e3-43787fc09238\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8376246687766322493\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO .\", \"source\": 296, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33872418244677354\": true}, \"uuid\": {\"dad44ab6-0215-412c-bf0a-ed8e3d7abf1f\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4254182676319009054\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"if\": true, \"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32554532\"}, \"evidence\": \"This , in turn , is likely to relate to reduced fibrinolysis due to increased production of PAI-1 , again as a result of inflammation .\", \"source\": 136, \"target\": 168, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33872418244677354\": true}, \"uuid\": {\"dad44ab6-0215-412c-bf0a-ed8e3d7abf1f\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4254182676319009054\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"if\": true, \"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32554532\"}, \"evidence\": \"This , in turn , is likely to relate to reduced fibrinolysis due to increased production of plasminogen activator inhibitor-1 ( PAI-1 ) , again as a result of inflammation .\", \"source\": 136, \"target\": 168, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5393003454443282\": true}, \"uuid\": {\"90365b53-f108-4c0b-be9e-bf1081778a6e\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1189533806944834199\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 447, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"In other cells , including endothelial cells , airway smooth muscle cells , vascular smooth muscle cells , neurons , and immune cells , RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation .\", \"source\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5393003454443282\": true}, \"uuid\": {\"90365b53-f108-4c0b-be9e-bf1081778a6e\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1189533806944834199\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 447, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"In other cells , including endothelial cells , airway smooth muscle cells , vascular smooth muscle cells , neurons , and immune cells , RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation .\", \"source\": 136, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2464253844654186\": true}, \"uuid\": {\"35b7d11c-609c-4f96-994d-d710aeb0d0e5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7701934601149921302\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"hypothesize\": true}}, \"target\": 508, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32554532\"}, \"evidence\": \"This , in turn , is likely to relate to reduced fibrinolysis due to increased production of PAI-1 , again as a result of inflammation .\", \"source\": 136, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33161774442535248\": true}, \"uuid\": {\"a76013e8-d119-476b-90ea-d8a36bd22c89\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6705437197214125685\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"likely\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32554532\"}, \"evidence\": \"This , in turn , is likely to relate to reduced fibrinolysis due to increased production of plasminogen activator inhibitor-1 ( PAI-1 ) , again as a result of inflammation .\", \"source\": 136, \"target\": 257, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23337022309612234\": true}, \"uuid\": {\"0a8273d5-abd0-4c8c-a1ab-305e33e68df0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5440520121713919373\": true}, \"source_api\": {\"reach\": true}}, \"target\": 13, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Also, activation of macrophages by IFN induces expression of the ATP-citrate lyase enzyme (ACLY), and blockage of ACLY activity reduces the production of ROS and nitric oxide [120].\", \"source\": 379, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"3414576628556833\": true}, \"uuid\": {\"856b3b94-62df-4aad-835d-4967a2702a4b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5440520121713919373\": true}, \"source_api\": {\"reach\": true}}, \"target\": 40, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Also, activation of macrophages by IFN induces expression of the ATP-citrate lyase enzyme (ACLY), and blockage of ACLY activity reduces the production of ROS and nitric oxide [120].\", \"source\": 379, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9917690942816242\": true}, \"uuid\": {\"e18acac7-98cf-460d-9da0-6d2bd6c1131a\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"8354399958692409467\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Mas -/- mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression.\", \"source\": 370, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9917690942816242\": true}, \"uuid\": {\"e18acac7-98cf-460d-9da0-6d2bd6c1131a\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"8354399958692409467\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Mas / mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression .\", \"source\": 370, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13395509290781188\": true}, \"uuid\": {\"02079c56-6805-4427-9a4a-f66f9deed544\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"8354399958692409467\": true}, \"source_api\": {\"reach\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Mas / mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression .\", \"source\": 370, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-12478295378008302\": true}, \"uuid\": {\"1e3ace8b-cd38-4284-b95d-442bd88f2fed\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5372835587252800237\": true}, \"source_api\": {\"reach\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Mas -/- mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression.\", \"source\": 370, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-913311396940347\": true}, \"uuid\": {\"317b5008-d0ca-485a-96b9-104e183f22dd\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6054317991921816856\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"This hypothesis suggests several approaches to treat COVID-19 by restoring ACE1/ACE2 balance: 1) ANG II receptor blockers (ARBs); 2) ACE1 inhibitors (ACEIs); 3) Agonists of receptors activated by ACE2-derived peptides [e.g., ANG (1-7), which activates MAS1]; 4) Recombinant human ACE2 or ACE2 peptides as decoys for the virus.\", \"source\": 463, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-913311396940347\": true}, \"uuid\": {\"317b5008-d0ca-485a-96b9-104e183f22dd\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6054317991921816856\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"This hypothesis suggests several approaches to treat COVID-19 by restoring ACE1 and ACE2 balance : 1) ANG II receptor blockers (ARBs); 2) ACE1 inhibitors (ACEIs); 3) Agonists of receptors activated by ACE2 derived peptides [e.g., ANG (1-7), which activates MAS1]; 4) Recombinant human ACE2 or ACE2 peptides as decoys for the virus.\", \"source\": 463, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"1648942110252040\": true}, \"uuid\": {\"98d287a3-390a-455b-aee6-f63fbd348eda\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"427735016823643123\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006; Tikellis et al., 2004a,b) , (2) loss of insulin secretion (Lupi et al., 2006; Shao et al., 2006; Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006; Tikellis et al., 2004b; Nakayama et al., 2005) , independently of changes in plasma glucose levels.\", \"source\": 221, \"target\": 156, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"1648942110252040\": true}, \"uuid\": {\"98d287a3-390a-455b-aee6-f63fbd348eda\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"427735016823643123\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006, Tikellis et al., 2004a, Tikellis et al., 2004b), (2) loss of insulin secretion (Lupi et al., 2006, Shao et al., 2006, Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006, Tikellis et al., 2004b, Nakayama et al., 2005), independently of changes in plasma glucose levels.\", \"source\": 221, \"target\": 156, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"1648942110252040\": true}, \"uuid\": {\"98d287a3-390a-455b-aee6-f63fbd348eda\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"427735016823643123\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006; Tikellis et al., 2004a,b) , (2) loss of insulin secretion (Lupi et al., 2006; Shao et al., 2006; Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006; Tikellis et al., 2004b; Nakayama et al., 2005) , independently of changes in plasma glucose levels.\", \"source\": 221, \"target\": 156, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-35003309509174966\": true}, \"uuid\": {\"b523dc40-158f-45cf-85a6-5cf40cb248cb\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8113192872861330686\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991).\", \"source\": 259, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-24281034667983636\": true}, \"uuid\": {\"6f6cb9fd-8702-4419-af16-d480d2e54665\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"413356804322069354\": true}, \"source_api\": {\"reach\": true}}, \"target\": 25, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"RAS blockade prevents hepatic triglyceride accumulation in obese Zucker rats, a common feature of insulin resistant and diabetic states (Toblli et al., 2007).\", \"source\": 259, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9855991476729151\": true}, \"uuid\": {\"621d0f26-8ece-4108-a468-0c9b64b3635d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-8154933589063097662\": true}, \"source_api\": {\"reach\": true}}, \"target\": 45, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"This imbalance hasbeen postulated as a potential cause of diabetic nephropathy.In the adrenal glands, the local secretory RAS stimulates aldosterone productionand serves as an amplification system for circulating Ang II. Importantly theregulation of the secretory adrenal RAS is independent of the circulatory RASand the activity of the adrenal RAS correlates with aldosterone production andregulation of potassium serum concentrations22.\", \"source\": 259, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21885756152080608\": true}, \"uuid\": {\"849cbda5-4b71-4152-9de2-c852cab6fa9d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4367081444959245262\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 357, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand , RAS blockade decreases TGF-beta expression in the islet and is associated with improved islet morphology and function ( Tikellis et al ., 2004b ) .\", \"source\": 259, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24917261441971939\": true}, \"uuid\": {\"ea1df410-55cc-43ea-adc3-7bb67d0c6c28\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8959415991729429647\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand, RAS blockade decreases TGF-beta expression in the islet and is associated with improved islet morphology and function.\", \"source\": 259, \"target\": 357, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24917261441971939\": true}, \"uuid\": {\"ea1df410-55cc-43ea-adc3-7bb67d0c6c28\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8959415991729429647\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand, RAS blockade decreases TGF- expression in the islet and is associated with improved islet morphology and function (Tikellis et al., 2004b).\", \"source\": 259, \"target\": 357, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15189316580657788\": true}, \"uuid\": {\"f44d62d8-3004-418c-b192-b88e40554ad4\": true}, \"belief\": {\"0.99596289\": true}, \"source_hash\": {\"3846149621592322928\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32715618\"}, \"evidence\": \"ACE2 is a carboxypeptidase negatively regulating the renin-angiotensin system ( RAS ) , which can induce vasodilation by cleaving angiotensin II.\", \"source\": 259, \"target\": 166, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15189316580657788\": true}, \"uuid\": {\"f44d62d8-3004-418c-b192-b88e40554ad4\": true}, \"belief\": {\"0.99596289\": true}, \"source_hash\": {\"3846149621592322928\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32715618\"}, \"evidence\": \"ACE2 is a carboxypeptidase negatively regulating the renin-angiotensin system ( RAS ) , which can induce vasodilation by cleaving angiotensin II.\", \"source\": 259, \"target\": 166, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23432676953112728\": true}, \"uuid\": {\"00c9b966-ae66-4b5e-884e-337b9272a9c2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"468960436410674531\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology.\", \"source\": 259, \"target\": 433, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1485448789858101\": true}, \"uuid\": {\"8b9d71fd-3b41-4213-97ea-de35d799148e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"468960436410674531\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology.\", \"source\": 259, \"target\": 434, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11666766647557407\": true}, \"uuid\": {\"61971590-2400-4d72-b192-27ff023dcfdf\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"468960436410674531\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology.\", \"source\": 259, \"target\": 501, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7370866213603532\": true}, \"uuid\": {\"eeacf689-ccd0-451f-ae66-3a980a487139\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7790914817860114080\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"RAS blockade improves glucose homeostasis by reducing  - cell death and improving insulin secretion and end-organ insulin sensitivity .\", \"source\": 259, \"target\": 167, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7370866213603532\": true}, \"uuid\": {\"eeacf689-ccd0-451f-ae66-3a980a487139\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7790914817860114080\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"RAS blockade improves glucose homeostasis by reducing beta-cell death and improving insulin secretion and end-organ insulin sensitivity .\", \"source\": 259, \"target\": 167, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30883505877609783\": true}, \"uuid\": {\"d964588a-fb4a-4e37-864f-f1941420b943\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5200589895390077186\": true}, \"source_api\": {\"reach\": true}}, \"target\": 99, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity.\", \"source\": 306, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"30163255024160698\": true}, \"uuid\": {\"34af190a-98bb-45c6-85a6-0be1d7a157e6\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8782827904518730437\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"We found that diabetic male mice had significantly reduced levels of Ccl2 expression at days 2 (P = 0.0084) and 7 (P = 0.0014) after infection ( Figure 6A ).\", \"source\": 306, \"target\": 363, \"key\": 11}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"21039657111397506\": true}, \"uuid\": {\"b2b9d9f2-58fc-437f-a92e-e3a332479b0f\": true}, \"belief\": {\"0.9499940951\": true}, \"source_hash\": {\"5009811013585337679\": true}, \"source_api\": {\"reach\": true}}, \"target\": 386, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"Diabetic male mice had reduced upregulation of Tnfa at 2 (P = 0.0145) and 7 (P = 0.0041) days after infection ( Figure 6B ) and reduced upregulation of Il6 2 days after infection ( Figure 6C) ; however, Nos2 expression was upregulated similarly in the lungs of male diabetic and control mice during MERS-CoV infection ( Figure 6E ).\", \"source\": 306, \"key\": 24}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-11826426099429590\": true}, \"uuid\": {\"e6b424ab-8e25-407c-8e33-03d10407731a\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"5696026435753959574\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity.\", \"source\": 306, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33773106570062992\": true}, \"uuid\": {\"a5805534-1444-4ee0-9860-d5b64dae7069\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8600700473001587916\": true}, \"source_api\": {\"reach\": true}}, \"target\": 435, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32375153\"}, \"evidence\": \"Kuba et al. [18] showed in mice that SARS-CoV downregulated ACE II protein, contributing to severe lung injury.\", \"source\": 306, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5757582715306240\": true}, \"uuid\": {\"a7b0ff8c-f237-48e8-8602-d7b3cf5b8bec\": true}, \"belief\": {\"0.9824999832596085\": true}, \"source_hash\": {\"5170133787940482409\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"Diabetic male mice had reduced upregulation of Tnfa at 2 (P = 0.0145) and 7 (P = 0.0041) days after infection ( Figure 6B ) and reduced upregulation of Il6 2 days after infection ( Figure 6C) ; however, Nos2 expression was upregulated similarly in the lungs of male diabetic and control mice during MERS-CoV infection ( Figure 6E ).\", \"source\": 306, \"key\": 10}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12030030467072521\": true}, \"uuid\": {\"25c11192-a9da-4802-a95f-2230a84125eb\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"431545271358347755\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"Male C57BL/6 mice are more susceptible than females to diet-induced obesity and the subsequent development of metabolic disease (40, 54) .\", \"source\": 306, \"target\": 253, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-1527688738325885\": true}, \"uuid\": {\"8e1cb91e-1b2e-48cd-a79b-53a0c955fb2a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-827404244985451201\": true}, \"source_api\": {\"reach\": true}}, \"target\": 426, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86].\", \"source\": 306, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"3429852087261815\": true}, \"uuid\": {\"2b5beff5-7908-4f5e-9dc6-8047dd27f13b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-827404244985451201\": true}, \"source_api\": {\"reach\": true}}, \"target\": 383, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86].\", \"source\": 306, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-7820788456319245\": true}, \"uuid\": {\"f5ad68f2-1f86-4159-b492-1279cb476576\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-827404244985451201\": true}, \"source_api\": {\"reach\": true}}, \"target\": 425, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86].\", \"source\": 306, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31261878140120801\": true}, \"uuid\": {\"38ad5b72-b22e-416e-b174-298ce4f560a4\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-827404244985451201\": true}, \"source_api\": {\"reach\": true}}, \"target\": 427, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86].\", \"source\": 306, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22394352919415299\": true}, \"uuid\": {\"9e375d9e-47d7-4c21-9f8d-29b59d16d4c2\": true}, \"belief\": {\"0.9989832139771575\": true}, \"source_hash\": {\"7958114039575565262\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 450, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"Activation of the angiotensin II receptor type 1 (AT1R) by Ang II leads to the induction of NF-B [9, 10], and subsequent inflammation through pathways distinct from those mediating classical Gq-induced signaling [11].\", \"source\": 106, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22394352919415299\": true}, \"uuid\": {\"9e375d9e-47d7-4c21-9f8d-29b59d16d4c2\": true}, \"belief\": {\"0.9989832139771575\": true}, \"source_hash\": {\"7958114039575565262\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 450, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"Activation of the angiotensin II receptor type 1 (AT1R) by Ang II leads to the induction of NF-B [ xref , xref ], and subsequent inflammation through pathways distinct from those mediating classical Gq-induced signaling [ xref ].\", \"source\": 106, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17562749274348156\": true}, \"uuid\": {\"5baeedc5-5970-47c0-bd57-f6587ddbec84\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"665988265701996580\": true}, \"source_api\": {\"reach\": true}}, \"target\": 537, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II-mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE/And II/ ATR1 pathway.A.\", \"source\": 106, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17562749274348156\": true}, \"uuid\": {\"5baeedc5-5970-47c0-bd57-f6587ddbec84\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"665988265701996580\": true}, \"source_api\": {\"reach\": true}}, \"target\": 537, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE and And II/ATR1 pathway.\", \"source\": 106, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"34540120074544096\": true}, \"uuid\": {\"de3cc604-f0fc-4bc2-927c-3d842ec055ee\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"3955002955392843954\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Ang II mediates vasoconstriction as well as aldosterone release from the adrenal gland, resulting in sodium retention and an increase in blood pressure through the angiotensin 1 receptor (AT 1 R).\", \"source\": 106, \"target\": 45, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12259545310173256\": true}, \"uuid\": {\"b8335e47-4b79-41d7-a856-e50daf4c7e97\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"9059213448851616740\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 376, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperglycaemia increases haemoglobin glycation damage. xref A study published in Diabetes Research and Clinical Practice analysed 132 patients with COVID-19, classified into three groups based on haemoglobin A1c (HbA1c) levels; results found a significant linear negative correlation between haemoglobin oxygen saturation (SaO 2 )and HbA1c, p=0.01. xref Hyperinsulinaemia inhibits beta-oxidation and ketolysis, thereby enforcing cellular ATP generation from glucose substrate. xref Glucose oxidation consumes four nicotinamide adenine dinucleotide (NAD+) to produce two acetyl moieties, whereas beta-oxidation consumes two, ketolysis one and acetoacetate none ( xref ); consequently, glucose oxidation depletes the NAD+ intracellularpool more than the other three substrates combined, thereby decreasing NAD+ availability for mitochondrial deacetylase sirtuin 3 (SIRT3) activity. xref NAD+ dependent SIRT3 activates manganese superoxide dismutase 2 (MnSOD2) via acetylation of lysine residue 68 (K68) and increases the production of NADPH via isocitrate dehydrogenase (IDH2) to drive the reduction of oxidised glutathione (GSSG) to reduced glutathione (GSH).\", \"source\": 400, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12259545310173256\": true}, \"uuid\": {\"b8335e47-4b79-41d7-a856-e50daf4c7e97\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"9059213448851616740\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 376, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"18 19 NAD+ dependent SIRT3 activates manganese superoxide dismutase 2 (MnSOD2) via acetylation of lysine residue 68 (K68) and increases the production of NADPH via isocitrate dehydrogenase (IDH2) to drive the reduction of oxidised glutathione (GSSG) to reduced glutathione (GSH).\", \"source\": 400, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27431459820425234\": true}, \"uuid\": {\"fc2a973a-f3f6-4a4f-a1e8-d2259a86024a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1684478035106486018\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"(C) The membrane bound C-type lectin receptor, DC-SIGN recognises oligomannose-type glycans via a Ca2+ ion and initiates phagocytosis of the pathogen (PDB ID: 1SL4) [216].\", \"source\": 408, \"target\": 134, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24634956852243735\": true}, \"uuid\": {\"fdb4e22f-aec5-4048-b647-f31e69accdf4\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1684478035106486018\": true}, \"source_api\": {\"reach\": true}}, \"target\": 469, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"(C) The membrane bound C-type lectin receptor, DC-SIGN recognises oligomannose-type glycans via a Ca2+ ion and initiates phagocytosis of the pathogen (PDB ID: 1SL4) [216].\", \"source\": 408, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21532635532788861\": true}, \"uuid\": {\"2ae42ebf-885d-4803-9a48-f287eeeec3b3\": true}, \"belief\": {\"0.9990821138071085\": true}, \"source_hash\": {\"488026841088968481\": true}, \"source_api\": {\"reach\": true}}, \"target\": 520, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 .\", \"source\": 408, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21532635532788861\": true}, \"uuid\": {\"2ae42ebf-885d-4803-9a48-f287eeeec3b3\": true}, \"belief\": {\"0.9990821138071085\": true}, \"source_hash\": {\"488026841088968481\": true}, \"source_api\": {\"reach\": true}}, \"target\": 520, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"(2007, 2010) showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll like receptor (TLR)-elicited signals to induce synthesis of full-length HIV transcripts as well as production of the immunosuppressive cytokine IL-10 (Gringhuis etal., (2007, 2010).\", \"source\": 408, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31906055353062865\": true}, \"uuid\": {\"989879b9-1165-4648-85e1-3b18a4bf9cd9\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7398196391285565455\": true}, \"source_api\": {\"reach\": true}}, \"target\": 358, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 .\", \"source\": 520, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31906055353062865\": true}, \"uuid\": {\"989879b9-1165-4648-85e1-3b18a4bf9cd9\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7398196391285565455\": true}, \"source_api\": {\"reach\": true}}, \"target\": 358, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 .\", \"source\": 520, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31906055353062865\": true}, \"uuid\": {\"989879b9-1165-4648-85e1-3b18a4bf9cd9\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7398196391285565455\": true}, \"source_api\": {\"reach\": true}}, \"target\": 358, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"(2007, 2010) showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll like receptor (TLR)-elicited signals to induce synthesis of full-length HIV transcripts as well as production of the immunosuppressive cytokine IL-10 (Gringhuis etal., (2007, 2010).\", \"source\": 520, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"369375717728683\": true}, \"uuid\": {\"0896064f-bba5-42ac-aa91-65b027aff5f7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5558083028487357300\": true}, \"source_api\": {\"reach\": true}}, \"target\": 5, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116].\", \"source\": 334, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11691578131327070\": true}, \"uuid\": {\"1e448876-4eaa-4924-a712-b206764a8376\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1170660298992588169\": true}, \"source_api\": {\"reach\": true}}, \"target\": 494, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Moreover, AKT-dependent inactivation of FoxOs can increase glycolysis [72, 73] by removing the suppressive force of c-Myc [7476].\", \"source\": 334, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11691578131327070\": true}, \"uuid\": {\"1e448876-4eaa-4924-a712-b206764a8376\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1170660298992588169\": true}, \"source_api\": {\"reach\": true}}, \"target\": 494, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Moreover, AKT-dependent inactivation of FoxOs can increase glycolysis [72, 73] by removing the suppressive force of c-Myc [74] [75] [76] .\", \"source\": 334, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-6469580263537367\": true}, \"uuid\": {\"12c5f527-4e43-4016-b070-79040467fc90\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8965669779407196648\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Recent studies have implicated the Akt pathway in Ang-(17) mediated NO release (Sampaio et al., 2007).\", \"source\": 334, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"19901350621211055\": true}, \"uuid\": {\"7438025e-da0d-4c2f-9ea1-63d6f49f641e\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"321615686604344760\": true}, \"source_api\": {\"reach\": true}}, \"target\": 26, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 5, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3992846055924133\": true}, \"uuid\": {\"c9c08854-2f85-477f-acda-ef6c2c433546\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 5, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3992846055924133\": true}, \"uuid\": {\"c9c08854-2f85-477f-acda-ef6c2c433546\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 5, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3992846055924133\": true}, \"uuid\": {\"c9c08854-2f85-477f-acda-ef6c2c433546\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 5, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26382344204871011\": true}, \"uuid\": {\"ebaf8b01-b427-4b09-9f67-4938c062ae10\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1233446694793484896\": true}, \"source_api\": {\"reach\": true}}, \"target\": 492, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 5, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26382344204871011\": true}, \"uuid\": {\"ebaf8b01-b427-4b09-9f67-4938c062ae10\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1233446694793484896\": true}, \"source_api\": {\"reach\": true}}, \"target\": 492, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 5, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23899878846506327\": true}, \"uuid\": {\"137e5892-2ef8-4987-95c4-e12f19c23d04\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4297136352077009536\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"This study revealed that enhancing vacuolar-type ATPase (a proton pump essential for influenza uncoating) via increasing glucose metabolism and, as a result, higher available ATP resources, augments the virus infection [50].\", \"source\": 5, \"target\": 274, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15991592878714539\": true}, \"uuid\": {\"59f82342-6925-42f3-b4f7-67893dc3c6bf\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 532, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-4101374892431454\": true}, \"uuid\": {\"97db005e-4e9f-4114-a38d-a205d019a3fa\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1597643647299577184\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In addition, high ATP consumption and reduced -oxidation, as well as glucose oxidation by influenza infection, contribute to the ATP crisis and hence influenza-related multi-organ failureFig.\", \"source\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-4101374892431454\": true}, \"uuid\": {\"97db005e-4e9f-4114-a38d-a205d019a3fa\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1597643647299577184\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"In addition, high ATP consumption and reduced -oxidation, as well as glucose oxidation by influenza infection, contribute to the ATP crisis and hence influenza-related multi-organ failure [96] .\", \"source\": 5, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17236544232796270\": true}, \"uuid\": {\"50d70f75-c23a-4aa0-b47d-c70b707d432c\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"4240924903409549253\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78].\", \"source\": 494, \"target\": 128, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17236544232796270\": true}, \"uuid\": {\"50d70f75-c23a-4aa0-b47d-c70b707d432c\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"4240924903409549253\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1 , glycolytic genes , and lactate dehydrogenase ( LDH ) , as the converter of pyruvate to lactate [ 77 , 78 ] .\", \"source\": 494, \"target\": 128, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23467133714874616\": true}, \"uuid\": {\"e5af4a43-5d46-4a2a-9bed-a7f43943a4ba\": true}, \"belief\": {\"0.98922\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 481, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23467133714874616\": true}, \"uuid\": {\"e5af4a43-5d46-4a2a-9bed-a7f43943a4ba\": true}, \"belief\": {\"0.98922\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection .\", \"source\": 481, \"key\": 5}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23467133714874616\": true}, \"uuid\": {\"e5af4a43-5d46-4a2a-9bed-a7f43943a4ba\": true}, \"belief\": {\"0.98922\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 481, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3512012524820312\": true}, \"uuid\": {\"475bf6ed-9e18-462d-8baa-0687caa630b2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 496, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 481, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-4925432307953790\": true}, \"uuid\": {\"2c278565-239d-4b02-ac3e-083c55deb6b5\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6638407516331381119\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Similarly, the accessory protein 6 of SARS-CoV was shown to interact with the IFN-signaling pathway-mediating protein Nmi and promote its ubiquitin-dependent proteasomal degradation, thereby potentially modulating the virus-induced innate immune response [200].\", \"source\": 497, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"997425151401358\": true}, \"uuid\": {\"46106249-6748-4aa8-9b35-f949917a4574\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4287957897742287075\": true}, \"source_api\": {\"reach\": true}}, \"target\": 448, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"In Calu3 epithelia, IL-4 is reported to activate an EGFR-dependent MAPK/ERK1/2 pathway [100], whereas in human bronchial cell derived epithelia IL-4 as well as IL-13 act via the Janus kinase/signal transducer and activator of transcription (JAK/STAT) pathway [107], most likely via binding to the same receptor, namely heteromeric IL-13R/IL-4R receptor [139].\", \"source\": 477, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"2454656286656659\": true}, \"uuid\": {\"420c0348-c37c-4bb8-9e07-be7967818014\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3994671532708179892\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia, and submucosal gland hypertrophy resulting in obstruction of airways.Goblet cell metaplasia and hyperplasia are also observed in patients with other chronic airway diseases such as asthma and chronic obstructive pulmonary disease (COPD), and is induced by the coordinated action of EGFR and IL-13.\", \"source\": 448, \"target\": 231, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1356617172154088\": true}, \"uuid\": {\"9a36d14c-b3d3-4990-be24-e5c4329771ab\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"3190240448742959641\": true}, \"source_api\": {\"reach\": true}}, \"target\": 334, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The viral hemagglutinin (HA) protein, along with virus replication, can upregulate PDPK1-mediated phosphorylation and activate AKT, which is required for induction of the mTORC1 signaling pathway by the influenza virus.\", \"source\": 511, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"33016197450309792\": true}, \"uuid\": {\"e91b00a4-e1ac-49ee-b01b-e5dd61230d26\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6182487766039948083\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"previously\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [ 9 ] .\", \"source\": 211, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33125371722838112\": true}, \"uuid\": {\"0b348487-0a7f-4250-8121-f93ecaa064f1\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"6182487766039948083\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"previously\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [ 9 ] .\", \"source\": 211, \"target\": 282, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17792427571709124\": true}, \"uuid\": {\"6ad07a8e-bdce-4e4d-a58c-549b7254d302\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6258165971303256232\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Experiments in 293 TRex cells using an inducible DCSIGN expression system have shown that high surface expression levels of DCSIGN correlate with optimal binding of HIV1 particles, and that lowering the DCSIGN expression levels can significantly reduce the efficiency of HIV1 binding (Pohlmann et al., 2001).\", \"source\": 211, \"target\": 211, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21232241190604808\": true}, \"uuid\": {\"af9f907c-566f-49b0-8970-3ef10766ac41\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"8420014764705681483\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Activation of NFB also induces a pro-inflammatory response, and thus, it induces the production and release of a variety of proinflammatory factors, which potentially interferes with TJ integrity.\", \"source\": 441, \"target\": 136, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8121003815775150\": true}, \"uuid\": {\"fe92b4f8-4be1-44f5-b99d-7b2d9d50a3d7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4017990929021931271\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32375153\"}, \"evidence\": \"It binds to chromosomal DNA but also to Toll-like receptor 3 (TLR3), TLR4, and the receptor for advanced glycation end products (RAGE) that activates nuclear factor (NF)-B ( Fig. 1a) , which mediate the upregulation of leukocyte adhesion molecules as well as the production of proinflammatory cytokines and angiogenic factors that promote inflammation.\", \"source\": 441, \"target\": 185, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35055207159167310\": true}, \"uuid\": {\"79181def-a3a7-498d-a74c-853f0ba09bee\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6712617664237363203\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Inhibition of basal NFB activity in airway epithelial cells increases cldn5 expression in the absence of inflammation [134].\", \"source\": 441, \"target\": 427, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35055207159167310\": true}, \"uuid\": {\"79181def-a3a7-498d-a74c-853f0ba09bee\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6712617664237363203\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Even in the absence of inflammation, inhibition of constitutive basal NFB activity in human airway epithelial cells causes an up-regulation of cldn5 expression, disturbs TJ organisation and increases paracellular permeabilty [134].\", \"source\": 441, \"target\": 427, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35368056320264640\": true}, \"uuid\": {\"16f4be71-5887-4747-81eb-fca97a0e5313\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1833559340732002697\": true}, \"source_api\": {\"reach\": true}}, \"target\": 427, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Overall, NFB-dependent reduction of cldn5 seems to improve epithelial barrier function during lung inflammation and seems to be beneficial with respect to epithelial function.\", \"source\": 441, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10761544296856007\": true}, \"uuid\": {\"ed1453e3-11c5-4876-8742-d5a016f530e3\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7817445556355559493\": true}, \"source_api\": {\"reach\": true}}, \"target\": 411, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"For example, Hodges et al. (2007) reported that binding of HIV1 to DCSIGN compromises DC maturation and primes these cells for transinfection: Upregulation of CD86 and MHCII is suppressed, whereas synapse formation between DCs and CD4+ T cells is promoted (Hodges et al., 2007).\", \"source\": 237, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13347343379566104\": true}, \"uuid\": {\"2113bfa9-0f74-484a-8436-ad309750593d\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-6054283929832492204\": true}, \"source_api\": {\"reach\": true}}, \"target\": 443, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"144, 145 hBD2 is induced by P. aeruginosa infection in normal but not in cystic fibrosis airway epithelia.\", \"source\": 237, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30404322185134341\": true}, \"uuid\": {\"c85d021b-7ef8-4826-a559-0481fa445ed3\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2153473492512848186\": true}, \"source_api\": {\"reach\": true}}, \"target\": 506, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"For example, IFV infection was found to induce oncostatin M (OSM) which causes tight junction opening (Pothoven et al., 2015; Tian et al., 2018).\", \"source\": 237, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26419407671079995\": true}, \"uuid\": {\"04d36cb6-7751-4d54-874d-7415f04ac341\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7543870953870730602\": true}, \"source_api\": {\"reach\": true}}, \"target\": 82, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"The increased inflammation, in turn, worsens the symptoms of airway diseases.Additionally, in patients with asthma and patients with CRS with nasal polyp (CRSwNP), viral infections such as RV and RSV promote a Type 2-biased immune response (Becker, 2006; Jackson et al., 2014; Jurak et al., 2018).\", \"source\": 274, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20344013025090026\": true}, \"uuid\": {\"d16aea2a-a26e-4f5f-a4b3-7ee831ae3c72\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7175315360142992379\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"Viral infections in such conditions will then cause increase neutrophilic activation; worsening the symptoms and airway remodeling in the airway thereby exacerbating COPD, CRSsNP and even CRSwNP in certain cases (Wang et al., 2009; Tacon et al., 2010; Linden et al., 2019).\", \"source\": 274, \"target\": 314, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22855859471151968\": true}, \"uuid\": {\"5068b382-7fc1-4605-b3e2-4026b3358b77\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5881444001244155995\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 116, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"28797733\"}, \"evidence\": \"Of note , the low cytokine induction by HCoV-OC43 was also confirmed at these higher inoculum doses ( data not shown ) .\", \"source\": 292, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30946113029025236\": true}, \"uuid\": {\"67ff6435-997e-4b00-8a83-50edf9d278e0\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-8314429920870072609\": true}, \"source_api\": {\"reach\": true}}, \"target\": 529, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"ACE catalyzes the conversion of angiotensin I to the octapeptide, angiotensin II (AngII), whereas ACE2 converts Ang II to angiotensin 1-7.\", \"source\": 435, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"33621275426159678\": true}, \"uuid\": {\"d0b1ddd5-a85f-4429-bcc5-721749435031\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1737409411173660356\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"The catalytic activity of ACE results in increased Ang II levels and increased catabolism of Ang-(1-7) whereas the catalytic activity of ACE-2 is predominantly on Ang I and Ang II and leads to formation of Ang-(1-7).\", \"source\": 435, \"target\": 145, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30032912109135884\": true}, \"uuid\": {\"da6d5e14-3c60-4878-a8b0-0f92343d93c2\": true}, \"belief\": {\"0.991866\": true}, \"source_hash\": {\"7681605979109788833\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Modulation of this system by ACE-inhibitors or AT 1 -Receptor blockers is now considered as the first-line therapy as well as for prevention and management of vascular complications.In this regard, the questions arise if (i) differences in ACE-2 may explain the exacerbated course of disease in patients with metabolic diseases and (ii) if ACE modulation in COVID-19 patients is neutral, beneficial, or harmful.\", \"source\": 435, \"target\": 193, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-105127793864548\": true}, \"uuid\": {\"5242908f-e534-4638-ae72-e8a5defde7d2\": true}, \"belief\": {\"0.997499704754953\": true}, \"source_hash\": {\"-3241061682399519926\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Supporting evidence include : ( 1 ) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels ( Carlsson et al ., 1998 ) ; ( 2 ) acute Ang-II infusion decreases islet blood flow in healthy non-diabetic Sprague-Dawley rats ( Carlsson et al ., 1998 ) ; and ( 3 ) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels ( Carlsson et al ., 1998 ) .\", \"source\": 435, \"target\": 259, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27214742334729005\": true}, \"uuid\": {\"d045b8a7-431b-49ef-a808-6019c4bc7481\": true}, \"belief\": {\"0.07680749999999996\": true}, \"source_hash\": {\"-7727184777488362626\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}, \"negated\": {\"true\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"The catalytic activity of ACE to activate Ang II occurs most extensively in the lung.\", \"source\": 435, \"key\": 122}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11479437898330275\": true}, \"uuid\": {\"42823447-69f1-43f3-91ff-d92d6f636c2b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5963551819131909987\": true}, \"source_api\": {\"reach\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32375153\"}, \"evidence\": \"Interestingly, ACE II overexpression has been reported to reduce HMGB1, besides reducing apoptosis in the myocardium postinfarction, in a rat model [19] .\", \"source\": 435, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-8675898844143792\": true}, \"uuid\": {\"492e4300-76eb-4773-8623-b372e42ef8b4\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8104880351190937056\": true}, \"source_api\": {\"reach\": true}}, \"target\": 501, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In addition, RAS blockade has been associated with a decrease in islet AT1 and ACE expression (Tikellis et al., 2004a,b) , which would result in decreased NOX activation.\", \"source\": 435, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-8675898844143792\": true}, \"uuid\": {\"492e4300-76eb-4773-8623-b372e42ef8b4\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8104880351190937056\": true}, \"source_api\": {\"reach\": true}}, \"target\": 501, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In addition, RAS blockade has been associated with a decrease in islet AT1 and ACE expression (Tikellis et al., 2004a, Tikellis et al., 2004b), which would result in decreased NOX activation.\", \"source\": 435, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"32791701806873133\": true}, \"uuid\": {\"64dc7167-bb41-4b8b-873c-c5e19cd1b2ae\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4299373981841112440\": true}, \"source_api\": {\"reach\": true}}, \"target\": 111, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"In a prospective observational cohort study using survey data from hospitalized patients (n = 16,749), obesity increased fatality risk (adjusted for age and gender) (HR = 1.37, 95% CI = 1.16-1.63) (13) .\", \"source\": 253, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20237404515458859\": true}, \"uuid\": {\"dba3b9e0-13b9-4709-9041-0e848485925b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4299373981841112440\": true}, \"source_api\": {\"reach\": true}}, \"target\": 543, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"In a prospective observational cohort study using survey data from hospitalized patients (n = 16,749), obesity increased fatality risk (adjusted for age and gender) (HR = 1.37, 95% CI = 1.16-1.63) (13) .\", \"source\": 253, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"30385714619761056\": true}, \"uuid\": {\"80a6d898-3803-465f-a675-5e3f04464a3d\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-180306211246190898\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"MERS-CoV, which contains a minimal FCS, is inhibited by furin inhibitor (FI) but not by the pan-lysosomal protease inhibitor (CPI).\", \"source\": 540, \"target\": 320, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14595010530949511\": true}, \"uuid\": {\"e5233a77-ef6b-479d-ab06-ab90ddab0219\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4384491510276064712\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27578435\"}, \"evidence\": \"Interestingly , despite recognizing the same receptor DPP4 , MERS-CoV and HKU4 spikes differ in their activities to mediate virus entry : HKU4 spike mediates virus entry into bat cells but not human cells , whereas MERS-CoV spike mediates virus entry into both bat and human cells ( 152 ) .\", \"source\": 320, \"target\": 309, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1868471236209268\": true}, \"uuid\": {\"8b5d5dd2-e02f-4aa8-9dac-b76bb3657118\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3074056124652932697\": true}, \"source_api\": {\"reach\": true}}, \"target\": 423, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Phosphorylation at its N-terminal domain by MAP kinases enhances cldn1s sealing properties [37].\", \"source\": 350, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"19149588540432514\": true}, \"uuid\": {\"25241857-8093-4a81-ba29-cd2298e3ea75\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-806641410445368289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"DM inhibits neutrophil chemotaxis, phagocytosis, and intracellular killing of microbes.\", \"source\": 558, \"target\": 134, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30566926543970249\": true}, \"uuid\": {\"63d898e8-21a1-42e9-8cf0-e8669a8afe15\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-806641410445368289\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"DM inhibits neutrophil chemotaxis, phagocytosis, and intracellular killing of microbes.\", \"source\": 558, \"target\": 158, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27092086281856932\": true}, \"uuid\": {\"766f879c-0b92-4b69-b797-9a9cb0646942\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1319834119825936957\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"These findings from in vivo study demonstrated that lack of SP-D in lung and / or kidney promotes severe renal injury , tubular cell apoptosis , NF-kappaB activation and production of pro-inflammatory cytokines , which consequently results in aggravated lung-kidney crosstalk during pneumonia-induced AKI .\", \"source\": 241, \"target\": 175, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27679159330137276\": true}, \"uuid\": {\"9cda87bf-1b14-45bc-afd1-2fb540533e0a\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"1212153085856531787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 26, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 50, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27679159330137276\": true}, \"uuid\": {\"9cda87bf-1b14-45bc-afd1-2fb540533e0a\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"1212153085856531787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 26, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 50, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11828778716099766\": true}, \"uuid\": {\"cf8014bc-1426-4b74-b2d4-91b02482d94e\": true}, \"belief\": {\"0.9499999999651322\": true}, \"source_hash\": {\"5564819523158179204\": true}, \"source_api\": {\"reach\": true}}, \"target\": 27, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration.\", \"source\": 50, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20621828950113203\": true}, \"uuid\": {\"5fd390db-a639-4714-8df2-619510136727\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 422, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 50, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-2423359241326799\": true}, \"uuid\": {\"6d7283b5-f4a5-4c1d-98a6-d11731bbb32a\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 532, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 50, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"27493966371136686\": true}, \"uuid\": {\"5ee50d90-6209-40f7-8139-d1071fcc70ea\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-69884066552089156\": true}, \"source_api\": {\"reach\": true}}, \"target\": 105, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Even with supplementation or adequate sun exposure, a mismatched low vitamin K2 status needed to move Ca 2+ into bones would further result in increased plasma Ca 2+ inhibition of vit D activation.\", \"source\": 50, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-24908339700546823\": true}, \"uuid\": {\"59f914c8-3a10-432b-886a-5e2c31908bcc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1783051300808249252\": true}, \"source_api\": {\"reach\": true}}, \"target\": 19, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold.\", \"source\": 127, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23470445169833346\": true}, \"uuid\": {\"52484a0a-1aa7-4388-8a3a-9f65ff32660c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2436610330607393645\": true}, \"source_api\": {\"reach\": true}}, \"target\": 419, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"The viral spike protein of SARS-Cov2 seems to induce a TNF--converting enzyme (TACE)-dependent alteration of ACE-2, which allows virus penetration into host cells, and the more TNF- levels are increased the more this pathway is facilitated [43].\", \"source\": 194, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17049607048910648\": true}, \"uuid\": {\"b14828d8-1f6a-4115-99d2-ed5fef3ed98f\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-1898470261445394113\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true}}, \"target\": 480, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33529600\"}, \"evidence\": \"Indeed , the development of more classical T1D has been temporally associated with exposure to viral infection ( Vehik et al ., 2019 ) ; hence , it seems reasonable to query mechanisms whereby SARS-CoV-2 might trigger insulin deficiency and T1D .\", \"source\": 194, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28034724849882447\": true}, \"uuid\": {\"e754a124-f40d-445e-bb41-5747eb2e2fd1\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"-4443191155604475014\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na +, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28034724849882447\": true}, \"uuid\": {\"e754a124-f40d-445e-bb41-5747eb2e2fd1\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"-4443191155604475014\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na + , an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28034724849882447\": true}, \"uuid\": {\"e754a124-f40d-445e-bb41-5747eb2e2fd1\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"-4443191155604475014\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-29859222360910630\": true}, \"uuid\": {\"f2648a06-8c32-4367-b60c-d1f34a7294cc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na +, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 15}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-29859222360910630\": true}, \"uuid\": {\"f2648a06-8c32-4367-b60c-d1f34a7294cc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na + , an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 16}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-29859222360910630\": true}, \"uuid\": {\"f2648a06-8c32-4367-b60c-d1f34a7294cc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 417, \"key\": 17}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"22924328993332880\": true}, \"uuid\": {\"82247000-aa6e-4c00-a428-106fc7c4f0b6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8100400780734035787\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 492, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-495385756651960\": true}, \"uuid\": {\"77eb6d64-2fe1-4dad-916b-0d97b455d5ce\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5360745888470836392\": true}, \"source_api\": {\"reach\": true}}, \"target\": 436, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR - XREF_BIBR While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-kappaB activation and by IL-1.\", \"source\": 347, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-524365016427491\": true}, \"uuid\": {\"cefdf1ca-08d9-4bcc-8fe6-fbb23a04ec5e\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3516651434542474699\": true}, \"source_api\": {\"reach\": true}}, \"target\": 443, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"[141] [142] [143] While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-B activation and by IL-1.\", \"source\": 347, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1076830852183187\": true}, \"uuid\": {\"a305f3cd-6dad-4e07-9a94-6f89de14eafa\": true}, \"belief\": {\"0.9499999956953279\": true}, \"source_hash\": {\"7785224756722598135\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"risk\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32416116\"}, \"evidence\": \"2 In late January 2020 , the United States had its first Covid-19 case in Washington state ,3 followed by massive growth of infections in New York , the current epicenter.2 , 4 As had been observed in China and Italy , the disease resulted in a large number of hospitalizations , respiratory failure and intensive care unit ( ICU ) admissions.5 , 6 The New York-Metropolitan area began to see a rapid increase in Covid-19 cases in February , 2020 .\", \"source\": 215, \"target\": 226, \"key\": 8}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1076830852183187\": true}, \"uuid\": {\"a305f3cd-6dad-4e07-9a94-6f89de14eafa\": true}, \"belief\": {\"0.9499999956953279\": true}, \"source_hash\": {\"7785224756722598135\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"risk\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32416116\"}, \"evidence\": \"2 , 4 As had been observed in China and Italy , the disease resulted in a large number of hospitalizations , respiratory failure and intensive care unit ( ICU ) admissions .\", \"source\": 215, \"target\": 226, \"key\": 9}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1311604127184249\": true}, \"uuid\": {\"4ee1bee2-f035-4b8d-8e30-a6d668404e0e\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8872667050125692881\": true}, \"source_api\": {\"reach\": true}}, \"target\": 397, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The surface glycoprotein HA exists as a trimer of hetero-dimers that is synthesised as a single polypeptide (HA0) that is subsequently cleaved into the receptor binding HA1 domain and the membrane-fusion mediating HA2 subunit by host cell proteases.\", \"source\": 353, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20660447416641164\": true}, \"uuid\": {\"72f7d1a8-3608-4d9c-b225-9a42b288c8ff\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"2639121273280643064\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"Inhibitors of cathepsin proteases have also been shown to inhibit entry of SARS-CoV and feline CoVs [23], [37], [38], while treatment of cell-bound virus particles with different proteases was shown to enhance virus entry and/or cell-cell fusion [27], [34], [39][45].\", \"source\": 353, \"target\": 299, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9381113147211037\": true}, \"uuid\": {\"0e9979df-908e-4c49-8c6e-cbe69790e3e5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-7811648439862175558\": true}, \"source_api\": {\"reach\": true}}, \"target\": 551, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"Once MBL binds to viral glycoproteins, it recruits mannose-binding lectin-associated serine proteases (MASPs), which trigger the lectin pathway of the complement system [[277], [278], [279]].\", \"source\": 353, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32721110537337032\": true}, \"uuid\": {\"a7f7631d-6321-47a9-9b7b-06785d14bc7d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"492497809504051904\": true}, \"source_api\": {\"reach\": true}}, \"target\": 373, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"In COPD patients, levels of SLPI and lysozyme were shown to decrease with bacterial infection, while lactoferrin levels remain unchanged. xref This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen.\", \"source\": 353, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32721110537337032\": true}, \"uuid\": {\"a7f7631d-6321-47a9-9b7b-06785d14bc7d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"492497809504051904\": true}, \"source_api\": {\"reach\": true}}, \"target\": 373, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen.\", \"source\": 353, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32721110537337032\": true}, \"uuid\": {\"a7f7631d-6321-47a9-9b7b-06785d14bc7d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"492497809504051904\": true}, \"source_api\": {\"reach\": true}}, \"target\": 373, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen.\", \"source\": 353, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32332745993440017\": true}, \"uuid\": {\"7aae8c87-714b-48a1-9a91-6d895ca68ced\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4243241728207718767\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Moreover, galectin1 binding impedes endocytosis and maturation of the NiVF precursor NiVF0 in infected cells (Garner et al., 2010), further illustrating the capacity of this lectin to block different stages of the viral infection/replication process.In sum, soluble host lectins can counteract viral infection in various ways: they may directly neutralize virus by destabilizing or aggregating virions, interfere with crucial steps in the viral infection process (e.g.\", \"source\": 551, \"target\": 147, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"14009742947603919\": true}, \"uuid\": {\"a5b0570a-9663-4930-8f07-453b21d29878\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3463300552221131583\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen.\", \"source\": 373, \"target\": 338, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16606046011468918\": true}, \"uuid\": {\"2e23e75a-cac7-4eed-9a21-934fb0a69411\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"492497809504051904\": true}, \"source_api\": {\"reach\": true}}, \"target\": 373, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen.\", \"source\": 373, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28495711274608713\": true}, \"uuid\": {\"6a80d28f-838c-4982-99f6-ecbaeff422a9\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3463300552221131583\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen.\", \"source\": 373, \"target\": 479, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"33037280350895711\": true}, \"uuid\": {\"d7b26025-f101-4c4e-81aa-76c2d60f6878\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3463300552221131583\": true}, \"source_api\": {\"reach\": true}}, \"target\": 479, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen.\", \"source\": 373, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-10468666881688478\": true}, \"uuid\": {\"d210aeae-6f8f-4467-b732-1931688aa4b8\": true}, \"belief\": {\"0.9996499998995576\": true}, \"source_hash\": {\"7359174736852498929\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"if\": true, \"expect\": true}}, \"target\": 474, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5).\", \"source\": 401, \"key\": 52}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-10468666881688478\": true}, \"uuid\": {\"d210aeae-6f8f-4467-b732-1931688aa4b8\": true}, \"belief\": {\"0.9996499998995576\": true}, \"source_hash\": {\"7359174736852498929\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"if\": true, \"expect\": true}}, \"target\": 474, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5) .\", \"source\": 401, \"key\": 53}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23836089500517478\": true}, \"uuid\": {\"2b8d8949-20e5-4106-ba68-a3eb57654d02\": true}, \"belief\": {\"0.9998547499583165\": true}, \"source_hash\": {\"2516624590594796391\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"whether\": true}}, \"target\": 475, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5).\", \"source\": 401, \"key\": 47}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23836089500517478\": true}, \"uuid\": {\"2b8d8949-20e5-4106-ba68-a3eb57654d02\": true}, \"belief\": {\"0.9998547499583165\": true}, \"source_hash\": {\"2516624590594796391\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"whether\": true}}, \"target\": 475, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5) .\", \"source\": 401, \"key\": 48}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9289020305098062\": true}, \"uuid\": {\"1cbac98e-ec95-4edb-80eb-50e34c2bb793\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"-8432140085964565085\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"Viruses that lacked glycosylation of HA were able to cause more disease in mice than viruses that were glycosylated.Three classes of lectin have been demonstrated to show anti-influenza activity - the C-type lectins, S-type lectins and pentraxins (reviewed in Ng and colleagues [34].\", \"source\": 224, \"target\": 215, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17274212573946077\": true}, \"uuid\": {\"0a0ad751-93d5-4151-9290-b6c08cb19797\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8086142832247148267\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The viral hemagglutinin (HA) protein, along with virus replication, can upregulate PDPK1-mediated phosphorylation and activate AKT, which is required for induction of the mTORC1 signaling pathway by the influenza virus.\", \"source\": 224, \"target\": 512, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2606779783845546\": true}, \"uuid\": {\"1672b4d0-6b4a-45c7-96ee-40d0b88109e3\": true}, \"belief\": {\"0.9499999999999993\": true}, \"source_hash\": {\"3738736145076980525\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"Further, SARS CoV spike protein has been found to reduce ACE-2 expression and to augment pulmonary injury [16] .\", \"source\": 298, \"target\": 393, \"key\": 26}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2606779783845546\": true}, \"uuid\": {\"1672b4d0-6b4a-45c7-96ee-40d0b88109e3\": true}, \"belief\": {\"0.9499999999999993\": true}, \"source_hash\": {\"3738736145076980525\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"In an animal model study, it has been reported that infusion of SARS CoV spike protein reduced ACE-2 expression and augmented pulmonary injury.\", \"source\": 298, \"target\": 393, \"key\": 27}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2606779783845546\": true}, \"uuid\": {\"1672b4d0-6b4a-45c7-96ee-40d0b88109e3\": true}, \"belief\": {\"0.9499999999999993\": true}, \"source_hash\": {\"3738736145076980525\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"It hasbeen observed that ACE-2 membrane expression and plasma levels are reduced afterinfection with SARS coronavirus16.Further, SARS CoV spike protein has been found to reduce ACE-2 expression and toaugment pulmonary injury16.\", \"source\": 298, \"target\": 393, \"key\": 38}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2606779783845546\": true}, \"uuid\": {\"1672b4d0-6b4a-45c7-96ee-40d0b88109e3\": true}, \"belief\": {\"0.9499999999999993\": true}, \"source_hash\": {\"3738736145076980525\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"It ispossible that an ACE/ACE-2 imbalance is one of the potential mechanismsexplaining why patients with cardio-metabolic problems are at higher risk forrespiratory failure20.In an animal model study, it has been reported that infusion of SARS CoV spikeprotein reduced ACE-2 expression and augmented pulmonary injury.\", \"source\": 298, \"target\": 393, \"key\": 39}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17571538921002562\": true}, \"uuid\": {\"33967255-4ef0-4396-9016-0a0c98ec6487\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-596602143964714353\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Inhibiting NF-kappaB signaling in renal tubular epithelial lead to reduce tubular injury , apoptosis , necrosis , and accumulation of interstitial inflammatory cells , which consequently results in ameliorated kidney damage after ischemic AKI51 .\", \"source\": 179, \"target\": 241, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22466945131208761\": true}, \"uuid\": {\"55af1e77-e025-4e85-b023-e54ccf040160\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"-6463105608164642445\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"AGEs induced CD147 glycosylation in endothelial cells may increase the activity of metalloproteinases (MMPs) and loosen tight junctions.\", \"source\": 374, \"target\": 300, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17947640519941572\": true}, \"uuid\": {\"fec89d7b-4ce8-43fc-a1e0-b86c05f63502\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4310453302841031618\": true}, \"source_api\": {\"reach\": true}}, \"target\": 490, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"28797733\"}, \"evidence\": \"To conclude, our results shed light on the complex interplay between viruses and the host tissue response and contribute to the understanding of the mechanism behind chronic respiratory tract infections in immunocompromised patients.Key messagesRV-B48 and HCoV-OC43 induce low toxicity in respiratory tissues because they affect neither MCC nor tissue integrity.\", \"source\": 263, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31749685644224828\": true}, \"uuid\": {\"bf19ac1c-7d6f-43ec-802f-f9debbefba2d\": true}, \"belief\": {\"0.95\": true}, \"source_hash\": {\"7706762924568934658\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"These viruses all have a bullet-shaped morphology and enter cells through receptor-mediated endocytosis followed by membrane fusion induced by low pH.\", \"source\": 227, \"target\": 177, \"key\": 46}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-22194222563546471\": true}, \"uuid\": {\"dc7b75cb-f829-40f6-ab23-23014b388992\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8089201604221639167\": true}, \"source_api\": {\"reach\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Conditions of mildly acidic pH, as might be found in endocytic compartments, are predicted to disrupt the lectin dependent activities of SP-D [XREF_BIBR].\", \"source\": 227, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"15845147506598733\": true}, \"uuid\": {\"36a06867-53f4-4dda-92f3-19d0bf5ba7e7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5421036128599752704\": true}, \"source_api\": {\"reach\": true}}, \"target\": 462, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"In the case of Lassa virus, the associated lowered pH induces a conformational change of the GPC that leads to binding to lysosomal-associated membrane protein 1 (LAMP1) [201,264].\", \"source\": 227, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"33444687128011178\": true}, \"uuid\": {\"8a4e290f-0b81-4085-8edb-b6a0eb0ae6ad\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5421036128599752704\": true}, \"source_api\": {\"reach\": true}}, \"target\": 484, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"In the case of Lassa virus, the associated lowered pH induces a conformational change of the GPC that leads to binding to lysosomal-associated membrane protein 1 (LAMP1) [201,264].\", \"source\": 462, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4851926933703746\": true}, \"uuid\": {\"2476c6d2-9e3a-4f8a-bcbb-1cc4f08461d1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4763595561213487949\": true}, \"source_api\": {\"reach\": true}}, \"target\": 4, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32422233\"}, \"evidence\": \"Age as a continuous variable was significantly associated with mortality (OR : 1.03; 95% CI : 1.00-1.07; p = 0.041) (XREF_TABLE panel A), and increasing oxygen requirements (OR : 1.02; 95% CI : 1.00-1.05; p = 0.044) (XREF_TABLE panel B), but not with intubation, although a trend was noticed (OR : 1.03; 95% CI : 1.00-1.05; p = 0.071) (XREF_TABLE panel C) in the multivariate analysis (model 4).\", \"source\": 195, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26126147087873566\": true}, \"uuid\": {\"4f6ac9fa-106c-4243-b315-72704eab8b13\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"5352394224930655761\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Supporting evidence include : (1) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels; (2) acute Ang-II infusion decreases islet blood flow in healthy non diabetic Sprague-Dawley rats; and (3) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels.\", \"source\": 515, \"target\": 259, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26126147087873566\": true}, \"uuid\": {\"4f6ac9fa-106c-4243-b315-72704eab8b13\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"5352394224930655761\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Supporting evidence include:(1) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels (Carlsson et al., 1998);  (2) acute Ang-II infusion decreases islet blood flow in healthy nondiabetic Sprague-Dawley rats (Carlsson et al., 1998) ; and (3) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels (Carlsson et al., 1998) .\", \"source\": 515, \"target\": 259, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30286861936524033\": true}, \"uuid\": {\"ce5c8017-5a79-4342-a032-d4b9256fabb7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3091706909919831099\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"16007097\"}, \"evidence\": \"For AT1 inhibition of Spike-Fc-mediated acute lung injury, we treated the Spike(S1190)-Fc-challenged mice with the AT1 inhibitor losartan (15 mg/kg).\", \"source\": 515, \"target\": 312, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"12163492922545586\": true}, \"uuid\": {\"c44e3da3-e0a5-4cd2-9e5b-182f5f327123\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8180925928817135335\": true}, \"source_api\": {\"reach\": true}}, \"target\": 548, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"Consistently, infections with MHV carrying wild type or mutant S protein were equally blocked by inhibitors of clathrin mediated endocytosis whereas the virus with the mutant S (MHV-S2 ' FCS) was much less sensitive to inhibitors of endosomal maturation, including BafA1, or to perturbants of the actin cytoskeleton.\", \"source\": 133, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"30262287551468329\": true}, \"uuid\": {\"695d4f8d-79c1-477e-a174-e24edb268247\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"4797556757158405669\": true}, \"source_api\": {\"reach\": true}}, \"target\": 49, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"This imbalance has been postulated as a potential cause of diabetic nephropathy.In the adrenal glands, the local secretory RAS stimulates aldosterone production and serves as an amplification system for circulating Ang II. Importantly the regulation of the secretory adrenal RAS is independent of the circulatory RAS and the activity of the adrenal RAS correlates with aldosterone production and regulation of potassium serum concentrations [22] .\", \"source\": 45, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26857662041697613\": true}, \"uuid\": {\"93dccda1-ef8d-4730-80e3-4bcacd52e593\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7728093515263944252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"In the heart, aldosterone can increase vascular expression of ICAM-1, MCP-1, osteopontin, and COX-2, which can be blocked by the MR antagonist, eplerenone [XREF_BIBR], indicating involvement of the MR. Proinflammatory effects of aldosterone have also been reported to occur in the kidney, where aldosterone and salt treatment caused MR dependent leukocyte infiltration and elevation of osteopontin, IL-6, IL-1beta, and MCP-1 levels [XREF_BIBR].\", \"source\": 45, \"target\": 377, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35074704505285570\": true}, \"uuid\": {\"a675763b-f2ff-4ff2-ab6d-c6da1c64d3d4\": true}, \"belief\": {\"0.9999937368199937\": true}, \"source_hash\": {\"-8743912235067232095\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 504, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"Aldosterone activates the MR in the distal renal tubule of the kidney to increase sodium and water retention, and potassium excretion, leading to an increase in blood volume and thus blood pressure [XREF_BIBR].\", \"source\": 45, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35074704505285570\": true}, \"uuid\": {\"a675763b-f2ff-4ff2-ab6d-c6da1c64d3d4\": true}, \"belief\": {\"0.9999937368199937\": true}, \"source_hash\": {\"-8743912235067232095\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 504, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"Aldosterone activates the MR in the distal renal tubule of the kidney to increase sodium and water retention, and potassium excretion, leading to an increase in blood volume and thus blood pressure [ xref ].\", \"source\": 45, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9422165649150267\": true}, \"uuid\": {\"080fa084-164e-4419-ae97-d41ab96f40b8\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7728093515263944252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"In the heart, aldosterone can increase vascular expression of ICAM-1, MCP-1, osteopontin, and COX-2, which can be blocked by the MR antagonist, eplerenone [XREF_BIBR], indicating involvement of the MR. Proinflammatory effects of aldosterone have also been reported to occur in the kidney, where aldosterone and salt treatment caused MR dependent leukocyte infiltration and elevation of osteopontin, IL-6, IL-1beta, and MCP-1 levels [XREF_BIBR].\", \"source\": 45, \"target\": 517, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"29299377260448343\": true}, \"uuid\": {\"3f73dbf4-4682-4129-9f93-0d90e95fd85c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6576322422241762910\": true}, \"source_api\": {\"reach\": true}}, \"target\": 97, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion, reduced intracellular levels, lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition.\", \"source\": 31, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5534220562654434\": true}, \"uuid\": {\"f2ff3a5e-283d-4c9c-89e1-9a13efe46fc9\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8074531563301899319\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Mg2 + is required to transport vitamin D around in the blood , and activation of vitamin D to its active form via hepatic and renal hydroxylation is Mg2 + dependent.65 Clinical manifestations of MgD include , but are not limited to , hyperinsulinaemia 's metabolic pathologies such as T2DM , osteoporosis , vitamin D metabolism disorders , CVD and hyperglycaemia.66 67 There is a strong relationship between MgD and increased oxidative stress .\", \"source\": 31, \"target\": 202, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5534220562654434\": true}, \"uuid\": {\"f2ff3a5e-283d-4c9c-89e1-9a13efe46fc9\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8074531563301899319\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Mg2 + is required to transport vitamin D around in the blood , and activation of vitamin D to its active form via hepatic and renal hydroxylation is Mg2 + dependent.65 Clinical manifestations of MgD include , but are not limited to , hyperinsulinaemia 's metabolic pathologies such as T2DM , osteoporosis , vitamin D metabolism disorders , CVD and hyperglycaemia.66 67 There is a strong relationship between MgD and increased oxidative stress .\", \"source\": 31, \"target\": 202, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23253900399412764\": true}, \"uuid\": {\"4dc52705-2a2f-4734-b720-0db2785e4bed\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6833555483860646477\": true}, \"source_api\": {\"reach\": true}}, \"target\": 69, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin.\", \"source\": 452, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5612514967909023\": true}, \"uuid\": {\"76556fd6-4de2-46c5-9c4f-9255792dcd82\": true}, \"belief\": {\"0.999874453249027\": true}, \"source_hash\": {\"-6011839295001733189\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"can\": true}}, \"target\": 453, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin efficiently cleaves and activates PAR-1, -3, and -4.131,132 Thus, in addition to being a potent platelet agonist, thrombin also activates immune effector and endothelial cells.\", \"source\": 452, \"key\": 14}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5612514967909023\": true}, \"uuid\": {\"76556fd6-4de2-46c5-9c4f-9255792dcd82\": true}, \"belief\": {\"0.999874453249027\": true}, \"source_hash\": {\"-6011839295001733189\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"can\": true}}, \"target\": 453, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin efficiently cleaves and activates PAR-1, -3, and -4.\", \"source\": 452, \"key\": 16}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5612514967909023\": true}, \"uuid\": {\"76556fd6-4de2-46c5-9c4f-9255792dcd82\": true}, \"belief\": {\"0.999874453249027\": true}, \"source_hash\": {\"-6011839295001733189\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"can\": true}}, \"target\": 453, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Similarly, monocyte and T-cell functions are enhanced by thrombin activation of monocyte and T-cell expressed PAR-1 and PAR-3.\", \"source\": 452, \"key\": 21}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"29725209886686286\": true}, \"uuid\": {\"bb889b93-6002-443e-9cc3-2a86e9e74caf\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"3232062563775997749\": true}, \"source_api\": {\"reach\": true}}, \"target\": 454, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Similarly, monocyte and T-cell functions are enhanced by thrombin activation of monocyte and T-cell expressed PAR-1 and PAR-3.\", \"source\": 452, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31472109003967311\": true}, \"uuid\": {\"baabbdd2-79d9-4b76-9675-c26ee4988a01\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-9214782991718794805\": true}, \"source_api\": {\"reach\": true}}, \"target\": 479, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin.\", \"source\": 452, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17621459422135372\": true}, \"uuid\": {\"7617a91f-91a3-49d6-a751-94c3a04531bb\": true}, \"belief\": {\"0.9499999987085984\": true}, \"source_hash\": {\"-1672225026429919272\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 .\", \"source\": 452, \"target\": 204, \"key\": 8}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17621459422135372\": true}, \"uuid\": {\"7617a91f-91a3-49d6-a751-94c3a04531bb\": true}, \"belief\": {\"0.9499999987085984\": true}, \"source_hash\": {\"-1672225026429919272\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"The levels of MPO-DNA and citrullinated histone H3 were higher in patients receiving mechanical ventilation compared with those not requiring any supplemental oxygen suggesting that levels of NETs correlate with disease severity.109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 .\", \"source\": 452, \"target\": 204, \"key\": 9}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-24250067262227\": true}, \"uuid\": {\"04fc4d19-4f1e-460d-aae6-3b31d8f3fa94\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"3844891431479328198\": true}, \"source_api\": {\"sparser\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"133,134 Indeed, thrombin-induced activation of leukocytes results in the production of several cytokines and growth factors while also upregulating leukocyte adhesive function.\", \"source\": 452, \"target\": 242, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24180556033135587\": true}, \"uuid\": {\"f1885e1e-6d47-41bc-9ff6-18ea0ec2bb2f\": true}, \"belief\": {\"0.997091188059\": true}, \"source_hash\": {\"-8194469299719347080\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"131 , 132 Thus , in addition to being a potent platelet agonist , thrombin also activates immune effector and endothelial cells .\", \"source\": 452, \"target\": 296, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24180556033135587\": true}, \"uuid\": {\"f1885e1e-6d47-41bc-9ff6-18ea0ec2bb2f\": true}, \"belief\": {\"0.997091188059\": true}, \"source_hash\": {\"-8194469299719347080\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin activates endothelial cells via the endothelial PAR-1 receptor , leading to upregulation of IL-6 , IL-8 , PAF ( platelet-activating factor ) , and MCP ( monocyte chemoattractant protein ) -1 in addition to the adhesion molecules P-selectin , E-selectin and ICAM ( intercellular adhesion molecule ) -1 , all of which serve to increase leukocyte recruitment and activation .\", \"source\": 452, \"target\": 296, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23970211973997556\": true}, \"uuid\": {\"2b4d943d-3ef2-4669-80f4-8e763b6e598d\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6833555483860646477\": true}, \"source_api\": {\"reach\": true}}, \"target\": 546, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin.\", \"source\": 452, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"8721599838927747\": true}, \"uuid\": {\"780968cd-a966-4351-acfc-509fd3a9a4b8\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6413492722101596146\": true}, \"source_api\": {\"reach\": true}}, \"target\": 55, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"149 Importantly, Heparins, including unfractionated heparin (UFH) and low molecular weight heparin (LMWH), bind antithrombin (AT), and potentiate the inhibitory effect of AT on coagulation factors Xa and thrombin.\", \"source\": 452, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"18697850716937212\": true}, \"uuid\": {\"e1ebf9bd-7a52-4ba3-a632-0699d97a546a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6057129107289598472\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution .\", \"source\": 270, \"target\": 181, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"29961953266667862\": true}, \"uuid\": {\"ec778013-7e66-4506-b2ea-2847592ebc82\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"832931734227946725\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"The importance of platelets in mediating T-cell recruitment and trafficking is underscored by data demonstrating that platelets play a fundamental role in the accumulation of cytotoxic T cells , and virus-induced organ damage , in a murine model of acute viral hepatitis.87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution.88 ,89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production .\", \"source\": 270, \"target\": 181, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-16009256628393867\": true}, \"uuid\": {\"f45bac53-3aaf-450f-a9ed-a25c7f3376fc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8322799347938832679\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"65 Inhibition of aldose reductase or SERPINb3a, both of which are induced in asthma reduced SPDEF expression, attenuates development of goblet cell hyperplasia.\", \"source\": 198, \"target\": 412, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5020395646224015\": true}, \"uuid\": {\"e8e97e67-3a49-487c-a379-176f4e1dcbc1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5351206591375827261\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"target\": 162, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25340113968842749\": true}, \"uuid\": {\"09d42fe1-02fb-4f20-a90b-6d40e60cb284\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5351206591375827261\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"target\": 162, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1032724568253927\": true}, \"uuid\": {\"6c7e8b5b-af7d-49fb-b1d6-e1a191d775b2\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2257292651389704609\": true}, \"source_api\": {\"reach\": true}}, \"target\": 382, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1032724568253927\": true}, \"uuid\": {\"6c7e8b5b-af7d-49fb-b1d6-e1a191d775b2\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2257292651389704609\": true}, \"source_api\": {\"reach\": true}}, \"target\": 382, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR While both TTF-1 and FOXA2 repress goblet cell metaplasia, XREF_BIBR, XREF_BIBR SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20925098470424089\": true}, \"uuid\": {\"260429eb-5848-4002-a46c-d18781853419\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2257292651389704609\": true}, \"source_api\": {\"reach\": true}}, \"target\": 468, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20925098470424089\": true}, \"uuid\": {\"260429eb-5848-4002-a46c-d18781853419\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2257292651389704609\": true}, \"source_api\": {\"reach\": true}}, \"target\": 468, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR While both TTF-1 and FOXA2 repress goblet cell metaplasia, XREF_BIBR, XREF_BIBR SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 412, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12401695384859696\": true}, \"uuid\": {\"dd8735f4-bafb-4ede-bcc5-9553d6ebafad\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2322437381107684572\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"mTORC1 signaling, in turn, promotes c-Myc expression at the translational level [70].\", \"source\": 359, \"target\": 494, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22781383053771568\": true}, \"uuid\": {\"ade78502-c64a-4daa-a053-dcee087486df\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5134636651840275727\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32942131\"}, \"evidence\": \"Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production .\", \"source\": 128, \"target\": 247, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"610646333884139\": true}, \"uuid\": {\"6c91cac6-c4a3-4a15-a6d9-07bbde7e7942\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8452542832135570552\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"On the other hand, the pentose phosphate pathway (PPP), as another glucose-consuming pathway reported to be enhanced by IVI [61], contributes to a higher yield of nucleotides and ATP for viral replication [62].\", \"source\": 129, \"target\": 164, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27085011674696526\": true}, \"uuid\": {\"9f30a6e3-761a-4fff-a6e6-fd58b848b3ac\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"2742564170424315670\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism .\", \"source\": 509, \"target\": 128, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25277829317670045\": true}, \"uuid\": {\"0d3dc738-8d69-4d92-a197-d5c3f81aa79d\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"665988265701996580\": true}, \"source_api\": {\"reach\": true}}, \"target\": 447, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II-mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE/And II/ ATR1 pathway.A.\", \"source\": 537, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25277829317670045\": true}, \"uuid\": {\"0d3dc738-8d69-4d92-a197-d5c3f81aa79d\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"665988265701996580\": true}, \"source_api\": {\"reach\": true}}, \"target\": 447, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE and And II/ATR1 pathway.\", \"source\": 537, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32258825916089898\": true}, \"uuid\": {\"c987f7a7-4bdf-4a45-b4bb-bd3f44fcf4a0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7198567501208071399\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"RAS blockade improves glucose homeostasis by reducing  - cell death and improving insulin secretion and end-organ insulin sensitivity .\", \"source\": 143, \"target\": 167, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-16378081455828101\": true}, \"uuid\": {\"60d56887-a7a5-4d1d-8536-c63b66310c74\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 440, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 .\", \"source\": 294, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-16378081455828101\": true}, \"uuid\": {\"60d56887-a7a5-4d1d-8536-c63b66310c74\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 440, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017).\", \"source\": 294, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-35073142689361139\": true}, \"uuid\": {\"1dc319b0-3008-40b2-9162-75d6ed638d34\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 451, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 .\", \"source\": 294, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-35073142689361139\": true}, \"uuid\": {\"1dc319b0-3008-40b2-9162-75d6ed638d34\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 451, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017).\", \"source\": 294, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31316159295816643\": true}, \"uuid\": {\"a3da55d2-36d1-4e2c-b323-e7c7ea7994f4\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 .\", \"source\": 294, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31316159295816643\": true}, \"uuid\": {\"a3da55d2-36d1-4e2c-b323-e7c7ea7994f4\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6333057144222468793\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017).\", \"source\": 294, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8769628183990021\": true}, \"uuid\": {\"8f25ce2b-c280-4ae5-a275-8b2b976b0ddb\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3405433921107868349\": true}, \"source_api\": {\"reach\": true}}, \"target\": 523, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1.\", \"source\": 77, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35446414693419527\": true}, \"uuid\": {\"56b82214-5f58-4b35-bcec-8504c8976df6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5080324565153497757\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 116, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1 .\", \"source\": 77, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"17900482395107875\": true}, \"uuid\": {\"9d1142be-4831-42ce-862e-488dfd12b5fc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3405433921107868349\": true}, \"source_api\": {\"reach\": true}}, \"target\": 475, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1.\", \"source\": 77, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12575744561683655\": true}, \"uuid\": {\"b218688c-78ab-4ada-9f94-b261ff7c91c4\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3405433921107868349\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1.\", \"source\": 77, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15160381320444325\": true}, \"uuid\": {\"554b29a1-6c64-4041-8d7d-cfb8ca83c044\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3405433921107868349\": true}, \"source_api\": {\"reach\": true}}, \"target\": 508, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1.\", \"source\": 77, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"19779264264527844\": true}, \"uuid\": {\"9e2b7a97-ba68-4a4c-b47c-86d12177c15b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4814773833882463163\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem.\", \"source\": 77, \"target\": 341, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33776447423342523\": true}, \"uuid\": {\"1971125f-1333-4627-bb86-02571b7ccec6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4814773833882463163\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem.\", \"source\": 77, \"target\": 356, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-23343227356109460\": true}, \"uuid\": {\"fd1fec34-cdfd-4ffb-8702-a78897b05863\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4814773833882463163\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem.\", \"source\": 77, \"target\": 403, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7878467113191330\": true}, \"uuid\": {\"8639bc9a-3eae-406c-b2c5-0c8b4fc40ec7\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"-5411344347802270828\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"possible\": true}}, \"target\": 43, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition .\", \"source\": 37, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7878467113191330\": true}, \"uuid\": {\"8639bc9a-3eae-406c-b2c5-0c8b4fc40ec7\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"-5411344347802270828\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"possible\": true}}, \"target\": 43, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition .\", \"source\": 37, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7878467113191330\": true}, \"uuid\": {\"8639bc9a-3eae-406c-b2c5-0c8b4fc40ec7\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"-5411344347802270828\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"possible\": true}}, \"target\": 43, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition .\", \"source\": 37, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-34434832513284580\": true}, \"uuid\": {\"615cca01-e432-478f-b566-e7078df9f568\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8560532218787949360\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion.\", \"source\": 59, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"6053765084580216\": true}, \"uuid\": {\"1256acf3-6bd4-4960-a792-b4b9c1904b1e\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-8560532218787949360\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion.\", \"source\": 59, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"6053765084580216\": true}, \"uuid\": {\"1256acf3-6bd4-4960-a792-b4b9c1904b1e\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-8560532218787949360\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991).\", \"source\": 59, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20367117324364850\": true}, \"uuid\": {\"036f0060-4d00-4ae9-ae65-b9c6b2e7d97e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8113192872861330686\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991).\", \"source\": 59, \"target\": 160, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1868399795832672\": true}, \"uuid\": {\"6edd95c9-78e1-43fb-846e-0d931fd202a6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8113192872861330686\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991).\", \"source\": 59, \"target\": 173, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-28505300314468975\": true}, \"uuid\": {\"8c4628aa-0362-43c3-8f7c-892fc2f50269\": true}, \"belief\": {\"0.0\": true}, \"source_hash\": {\"5471794119506503986\": true}, \"source_api\": {\"eidos\": true}, \"negated\": {\"true\": true}}, \"target\": 354, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"Unlike many other results of ubiqutination , the change in RIG-I does not lead to degradation in the proteasome but actually is a critical step for subsequent interaction of RIG-I with MAVS though the exact mechanism how this is facilitated is not clearly understood [ 61 ] .\", \"source\": 418, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32240402876783065\": true}, \"uuid\": {\"0f91cd1d-71b2-4ed1-998a-214f23f4a7c3\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7788123676700703245\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32291542\"}, \"evidence\": \"Studies of recombinant human angiotensin converting enzyme 2 (rhACE2), mesenchymal stem cell, PD-1 blocking antibody, bevacizumab injection, and immunoglobulin of cured patients are registered in the website of Clinical Trial [XREF_BIBR], and some are recruiting patients.\", \"source\": 510, \"target\": 189, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7893995130880593\": true}, \"uuid\": {\"92b4c690-1423-4cc8-95c9-e8e527dd28ee\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3897447882761148860\": true}, \"source_api\": {\"reach\": true}}, \"target\": 108, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"This review discussed an overview on the role of ACE-2 and furin in the binding of CoV with HC biomembrane mediated by SP.\", \"source\": 114, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"7893995130880593\": true}, \"uuid\": {\"92b4c690-1423-4cc8-95c9-e8e527dd28ee\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3897447882761148860\": true}, \"source_api\": {\"reach\": true}}, \"target\": 108, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"The SP mediates CoV entry and conducts the interaction of CoV with receptor (ACE-2) on the HCs as well as mediating the fusion of HC biomembrane and viral envelope.\", \"source\": 114, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20472656592226039\": true}, \"uuid\": {\"cdb489d8-d67b-49f0-bad6-2ed47c3f3008\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1659805886517933029\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"19439011\"}, \"evidence\": \"SPD is recognized to modulate the antigen presentation to specific major histocompatibility complex class II Tcell hybridomas in vitro in a dosedependent fashion.\", \"source\": 114, \"target\": 155, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"19144470707361569\": true}, \"uuid\": {\"a67798ac-8ac9-4a78-9abd-bcea230d5839\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6956560016573157181\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Both SPA and SPD inhibit the viral hemagglutinating activity that is required for IAV attachment to target cells (Hartshorn et al., 1994; Malhotra et al., 1994; Benne et al., 1995; Hartshorn et al., 1996, 1997, 2000; van Eijk et al., 2002, 2003, 2004; Mikerov et al., 2008; Hillaire et al., 2011) and SPD was reported to inhibit the viral neuraminidase (Reading et al., 1997; Hillaire et al., 2011).\", \"source\": 114, \"target\": 251, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7616237893032184\": true}, \"uuid\": {\"a7d2c098-99aa-48dd-8fcd-07be61b2c692\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1709976018579862637\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Importantly , however , the neuraminidase activity can prevent virions from aggregating via hemagglutinin-sialic acid interactions , promotes efficient release of newly formed virions from ( sialic acid-carrying ) infected cells , and provides a counterweight to the interaction of hemagglutinin molecules with nontarget cell-associated glycans : neuraminidase-mediated removal of sialic acids from decoy receptors prevents virions from establishing high-avidity interactions with these glycoconjugates and may even provide an escape route for virions after hemagglutinin-mediated binding to nontarget cell-associated glycans ( Colman ,1994 ; Suzukiet al ., 1994 ; Gimsaet al ., 1996 ; Barrereet al ., 1997 ; Gamblin & Skehel ,2010 ) .\", \"source\": 251, \"target\": 151, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35608651790749444\": true}, \"uuid\": {\"e45fe899-bc10-41c9-8823-5df466980ef8\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6196998427393000566\": true}, \"source_api\": {\"reach\": true}}, \"target\": 332, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response.\", \"source\": 410, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17037504069448705\": true}, \"uuid\": {\"67863d52-d81b-4aee-9ff4-971ab81d66fe\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6196998427393000566\": true}, \"source_api\": {\"reach\": true}}, \"target\": 413, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response.\", \"source\": 410, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25466268744247176\": true}, \"uuid\": {\"2ddcc889-3f5b-4da1-a33d-34491c594ae1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6196998427393000566\": true}, \"source_api\": {\"reach\": true}}, \"target\": 388, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response.\", \"source\": 410, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-12966301631135326\": true}, \"uuid\": {\"b9b90f82-ec24-4827-af7a-a96a8c76cffa\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7426274763355285599\": true}, \"source_api\": {\"reach\": true}}, \"target\": 526, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"For example, a drug that blocks the receptor may make CoV entry into cells difficult.Furin and furin-like proteases belong to the group of proprotein convertases.\", \"source\": 33, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21759159768253064\": true}, \"uuid\": {\"d09c87db-c774-49a1-adf0-85923185bc05\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3720125285214726150\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29579213\"}, \"evidence\": \"Mutation of Nglycosylation sites in the globular head of glycoprotein G increased cell-cell fusion, whereas removal of a glycosite in the stalk region of Hendra glycoprotein G abolished cell-cell fusion (Bradel-Tretheway et al. 2015) .\", \"source\": 381, \"target\": 184, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-25695374765508665\": true}, \"uuid\": {\"f0e95925-1594-42f9-9a9e-16deb3010c25\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2351298319170187590\": true}, \"source_api\": {\"reach\": true}}, \"target\": 367, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"These data further demonstrated that the lack of pulmonary and/or renal SP-D enhanced renal NF-B activation and inflammatory cytokine expression in the kidney in pneumonia-induced AKI.To study the protective effect of SP-D on tubular apoptosis, PTECs were isolated from the kidneys of WT and KO mice.\", \"source\": 258, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"24394934792364357\": true}, \"uuid\": {\"b0ea863e-b164-4c09-8949-facca9248034\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-37541597023880909\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Interestingly, sera from individuals with COVID-19 triggered NET release from control neutrophils in A c c e p t e d M a n u s c r i p t vitro (82) .\", \"source\": 536, \"target\": 371, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27829017247061947\": true}, \"uuid\": {\"9cd81e33-e2f7-4b94-9387-84490808e6dc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3994671532708179892\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia, and submucosal gland hypertrophy resulting in obstruction of airways.Goblet cell metaplasia and hyperplasia are also observed in patients with other chronic airway diseases such as asthma and chronic obstructive pulmonary disease (COPD), and is induced by the coordinated action of EGFR and IL-13.\", \"source\": 472, \"target\": 231, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31214527009606447\": true}, \"uuid\": {\"232258e0-0805-41bf-b4b8-ce688f26a112\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1449772811470652261\": true}, \"source_api\": {\"reach\": true}}, \"target\": 424, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"In mouse, lung IL-13 reduces cldn18 while increasing cldn4 expression [XREF_BIBR].\", \"source\": 472, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31214527009606447\": true}, \"uuid\": {\"232258e0-0805-41bf-b4b8-ce688f26a112\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1449772811470652261\": true}, \"source_api\": {\"reach\": true}}, \"target\": 424, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"In mouse, lung IL-13 reduces cldn18 while increasing cldn4 expression [118].\", \"source\": 472, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"24279925825586556\": true}, \"uuid\": {\"78f851a3-93e7-44be-82c1-7b31c66a2431\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3605943661160015577\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [61].\", \"source\": 287, \"target\": 224, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"24279925825586556\": true}, \"uuid\": {\"78f851a3-93e7-44be-82c1-7b31c66a2431\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"3605943661160015577\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [XREF_BIBR].\", \"source\": 287, \"target\": 224, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35165392806255116\": true}, \"uuid\": {\"a58449f7-6704-4bf3-9051-0ac4f4a6d6df\": true}, \"belief\": {\"0.9804\": true}, \"source_hash\": {\"9198592615245131963\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 435, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Estradiol decreases, while testosterone increases ACE activity in the lung (120) .\", \"source\": 22, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11324141315041200\": true}, \"uuid\": {\"21e66954-94b3-4e16-8da9-010e2be83116\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"206641558973096474\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"A phase 2 trial is currently evaluating leronlimab , a humanized monoclonal antibody inhibiting CCR5 which is already under study in HIV infection [ 46 ] , to determine whether blockage of this pathway is able to reduce mortality at 28 days in severe forms of COVID-19 ( NCT04347239 ) .\", \"source\": 60, \"target\": 249, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13849926917255472\": true}, \"uuid\": {\"b930ebb2-6296-4aa3-b3ff-39ad7c4cac8f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3448448340839535513\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 119, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"These viruses are classified as bio-safety level 4 pathogens due to their high mortality rates and scarcity of approved treatments or cures .\", \"source\": 249, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14974584609526463\": true}, \"uuid\": {\"1f25b4b7-6134-43f2-9291-1c1ca309bc0f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2133761071558953756\": true}, \"source_api\": {\"reach\": true}}, \"target\": 524, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na + , an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis.\", \"source\": 209, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27025663005505368\": true}, \"uuid\": {\"5fac3850-6e0f-4a0d-9b11-7c7242cc09fb\": true}, \"belief\": {\"0.949271\": true}, \"source_hash\": {\"2864888135401569735\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true, \"indicate\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32220611\"}, \"evidence\": \"In vitro exposure of pulmonary epithelial cells to elevated glucose concentrations significantly increased influenza virus infection and replication , suggesting that hyperglycemia may increase viral replication in vivo .\", \"source\": 228, \"target\": 152, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-716671906467218\": true}, \"uuid\": {\"72291e09-0a2c-4266-a72b-242804d7117f\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-6437069736512415702\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32469464\"}, \"evidence\": \"19 Acute hyperglycemia has been shown to upregulate ACE2 expression on This article is protected by copyright.\", \"source\": 228, \"target\": 393, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9111357397791116\": true}, \"uuid\": {\"5cef730a-a067-4f8d-b3ff-80fb50806958\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7809975192936923067\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"HMGB1 is also upregulated by hyperglycemia and appears to play a major role in lung inflammation especially in the context of DM .\", \"source\": 228, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9111357397791116\": true}, \"uuid\": {\"5cef730a-a067-4f8d-b3ff-80fb50806958\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7809975192936923067\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"HMGB1 is also upregulated by hyperglycemia and appears to play a major role in lung inflammation especially in the context of DM .\", \"source\": 228, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31877616359925176\": true}, \"uuid\": {\"453ff581-9523-45e7-bbe1-848c82acac4f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1504493312042362812\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"The red arrows emphasize hyperglycemia-induced changes in the islet RAS and its consequences.\", \"source\": 228, \"target\": 259, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28045608523835880\": true}, \"uuid\": {\"f2f528eb-3acd-4b63-8529-59e1c5d1d294\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3293596024275827712\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"In vivo and in vitro studies reveal the mechanisms by which hyperglycemia increases virulence and how glucose control and insulin reduce it .\", \"source\": 228, \"target\": 273, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21987418435527225\": true}, \"uuid\": {\"5a13e98b-a850-4f5a-8494-f340cb709a04\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"7422646625418389465\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Viral GPs then contribute to multiple viral functions, including viral entry and immune evasion (Merry and Astrautsova 2010) .\", \"source\": 415, \"target\": 169, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9591226845990657\": true}, \"uuid\": {\"d0517408-11a3-4eec-a7b6-9ae6f46dad97\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-437433830666826529\": true}, \"source_api\": {\"reach\": true}}, \"target\": 96, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 349, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-25127253877778896\": true}, \"uuid\": {\"2431deb7-a6a2-4981-94ac-51db9fee6ea7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1366912351931562243\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78].\", \"source\": 349, \"target\": 128, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1217874546258454\": true}, \"uuid\": {\"d69b5fbb-9aeb-48bb-a399-5a9b3e0d5d74\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"2304228491589390248\": true}, \"source_api\": {\"reach\": true}}, \"target\": 431, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 349, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33481806616057698\": true}, \"uuid\": {\"f9b5b7de-4c44-493b-ac90-e5ebb7f58b2c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8149760404012331683\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 372, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"The actin- and macropinocytosis-affecting drug EIPA, which inhibits the Na + /H + exchanger NHE1, led to reduced luciferase expression both when added prior to and after entry of MHV at 2 hpi.\", \"source\": 336, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-451275228199988\": true}, \"uuid\": {\"5bc57b85-8307-4b9e-9a58-c076923e3853\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6413492722101596146\": true}, \"source_api\": {\"reach\": true}}, \"target\": 55, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"149 Importantly, Heparins, including unfractionated heparin (UFH) and low molecular weight heparin (LMWH), bind antithrombin (AT), and potentiate the inhibitory effect of AT on coagulation factors Xa and thrombin.\", \"source\": 65, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11507357229726941\": true}, \"uuid\": {\"be2cc18e-b66e-4cee-8ab1-23550502a3a1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"401643414877310907\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 429, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"In the airway epithelium , cldn8 is up-regulated by glucocorticoids but not by mineralocorticoids [ 66 ] .\", \"source\": 34, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33905360414222616\": true}, \"uuid\": {\"827a8c09-1e20-4ee0-97f0-74ed3d0f2bd2\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-1290188827113080726\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32179126\"}, \"evidence\": \"Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications .\", \"source\": 34, \"target\": 302, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21730041722780653\": true}, \"uuid\": {\"c9164d53-8642-4fde-b051-c1eafe00df61\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-171662295973029873\": true}, \"source_api\": {\"reach\": true}}, \"target\": 26, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Therefore, cldn8 increases permselectivity of TJ for Cl.\", \"source\": 429, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19714054781066524\": true}, \"uuid\": {\"b8c5a08c-d764-458d-a39c-4f572c191e10\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-936961767248133553\": true}, \"source_api\": {\"reach\": true}}, \"target\": 42, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Cldn8 mediated sealing of TJ to Na + parallels Na + absorption in human colon cells [XREF_BIBR], and hence, it is proposed that cldn8 augments sodium resorption by preventing paracellular leakage of Na +.\", \"source\": 429, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26750390506720715\": true}, \"uuid\": {\"4b6acc0b-8cde-409a-ac21-b4f0e64bcca8\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6050514574641082078\": true}, \"source_api\": {\"reach\": true}}, \"target\": 355, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Other work by Gringhuis et al. (2007, 2010) showed that binding of HIV1 to DCSIGN triggers Raf1 dependent signaling, which modulates tolllike receptor (TLR)elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL10 (Gringhuis et al., (2007, 2010).\", \"source\": 93, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23584766227905898\": true}, \"uuid\": {\"04d8c096-416b-4683-be30-1fb9062fb88b\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-4549290121756549843\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"predict\": true}}, \"target\": 13, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Indeed, ACE inhibitors improve skeletal muscle glucose uptake through a bradykinin mediated increase in NO.\", \"source\": 483, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1205447871563743\": true}, \"uuid\": {\"fc3bc9f0-6498-4056-be90-2e6db555703f\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-4798493131047169468\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 219, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1205447871563743\": true}, \"uuid\": {\"fc3bc9f0-6498-4056-be90-2e6db555703f\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-4798493131047169468\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 219, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1205447871563743\": true}, \"uuid\": {\"fc3bc9f0-6498-4056-be90-2e6db555703f\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-4798493131047169468\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 219, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1205447871563743\": true}, \"uuid\": {\"fc3bc9f0-6498-4056-be90-2e6db555703f\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-4798493131047169468\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells (RBCs) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge, thus preventing thrombi and agglutination via maintaining electrorepulsion driven dispersion.\", \"source\": 219, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34768098771478134\": true}, \"uuid\": {\"f4b99a86-2f59-4b84-baf7-652ccbed2e63\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 219, \"target\": 139, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34768098771478134\": true}, \"uuid\": {\"f4b99a86-2f59-4b84-baf7-652ccbed2e63\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 219, \"target\": 139, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34768098771478134\": true}, \"uuid\": {\"f4b99a86-2f59-4b84-baf7-652ccbed2e63\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 219, \"target\": 139, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35131718761979397\": true}, \"uuid\": {\"2d5e3e16-75d1-4099-a3ec-592cc9652a27\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8764249334378087391\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Massive agglutination of organisms could contribute to lung host defense by promoting airway mucociliary clearance, but it might also influence the internalization by phagocytic cells.SP-D enhances the binding and uptake of IAV by neutrophils [33,61,63], and these effects are highly dependent on the extent of multimerization of SP-D and its ability to agglutinate the viral particles [18,30,33].\", \"source\": 139, \"target\": 183, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"29859287511622326\": true}, \"uuid\": {\"7d7e5cfe-d557-4a87-a43e-9d95453eb26b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-421409972400661301\": true}, \"source_api\": {\"reach\": true}}, \"target\": 18, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem.\", \"source\": 356, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17039078421898032\": true}, \"uuid\": {\"4e30ff3f-d72d-471d-9a3e-a7e1c700a6ed\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1918181136093837831\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"Actually, it has been recently demonstrated that HCs infected by several kinds of viruses stimulate an interferon-based activity to block the enzymatic activity of furin-like enzymes (Lodermeyer et al., 2013).\", \"source\": 109, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17039078421898032\": true}, \"uuid\": {\"4e30ff3f-d72d-471d-9a3e-a7e1c700a6ed\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"1918181136093837831\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"Actually, it has been recently demonstrated that HCs infected by several kinds of viruses stimulate an interferon based activity to block the enzymatic activity of furin like enzymes.\", \"source\": 109, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-34094830888277625\": true}, \"uuid\": {\"4f3a8fab-5b31-4139-b3af-d9027dee3045\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"217467305111889477\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Enoxaparin may also elicit similar effects on HMGB1 ( 46 ) .\", \"source\": 47, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-34094830888277625\": true}, \"uuid\": {\"4f3a8fab-5b31-4139-b3af-d9027dee3045\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"217467305111889477\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Enoxaparin may also elicit similar effects on HMGB1 ( 46 ) .\", \"source\": 47, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"16792051294611736\": true}, \"uuid\": {\"e89c53e6-2ae7-4d72-b5f8-6028835e4cf3\": true}, \"belief\": {\"0.9926598\": true}, \"source_hash\": {\"6367977285759204737\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"142, 143 Indeed, heparin, acting as a decoy receptor for naturally expressed heparan sulfate, has been shown to inhibit the binding of the SARS coronavirus to cells in vitro.\", \"source\": 47, \"target\": 298, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20347035140785113\": true}, \"uuid\": {\"a8652675-3ae5-4372-b464-43993915ddae\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4114814556447855763\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"21 Interestingly , however , studies evaluating Chinese and White populations have demonstrated that abnormal coagulation parameters are associated with severe In vascular homeostasis , the endothelium possesses anti-inflammatory and antithrombotic properties due to the expression of CD39 , nitric oxide ( NO ) , and prostacyclin in addition to the natural anticoagulants , TFPI ( tissue factor pathway inhibitor ) , activated protein C , and thrombomodulin .\", \"source\": 380, \"target\": 216, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20810695862211085\": true}, \"uuid\": {\"472f7782-8d51-4adb-9361-a28d35a2332c\": true}, \"belief\": {\"0.9824993799855\": true}, \"source_hash\": {\"6328102033858926248\": true}, \"source_api\": {\"reach\": true}}, \"target\": 21, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 467, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20810695862211085\": true}, \"uuid\": {\"472f7782-8d51-4adb-9361-a28d35a2332c\": true}, \"belief\": {\"0.9824993799855\": true}, \"source_hash\": {\"6328102033858926248\": true}, \"source_api\": {\"reach\": true}}, \"target\": 21, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 467, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-23866396659666751\": true}, \"uuid\": {\"d81aaa30-d441-49a6-a25e-706c51f92048\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4524677770031509034\": true}, \"source_api\": {\"reach\": true}}, \"target\": 23, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem-oxygenase (HO) induces hepatic aminolevulinic acid synthase (ALAS1) (enzyme in the first step of haem synthesis) activity.\", \"source\": 467, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"18075805173579363\": true}, \"uuid\": {\"b41c6ead-517a-467a-938c-e64152983d76\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"6328102033858926248\": true}, \"source_api\": {\"reach\": true}}, \"target\": 29, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 467, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14735261599319613\": true}, \"uuid\": {\"c307ba8a-8fdd-4cc3-8c8d-3c4e164fd5d3\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"6328102033858926248\": true}, \"source_api\": {\"reach\": true}}, \"target\": 79, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 467, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20086851824930679\": true}, \"uuid\": {\"73893d19-e9ed-45d4-9b82-a2aef432c49b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5003427663175487120\": true}, \"source_api\": {\"isi\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression.\", \"source\": 80, \"target\": 393, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25400346744468390\": true}, \"uuid\": {\"03f0ed91-c90c-49f8-a5bd-10c0ad9ef013\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1115522893515788135\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"In vitro data suggest that hydroxychloroquine can reduce glycosylation of the ACE2 receptor and also can lower blood glucose concentrations , both which could be possible mechanisms of antiviral activity .\", \"source\": 80, \"target\": 203, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25400346744468390\": true}, \"uuid\": {\"03f0ed91-c90c-49f8-a5bd-10c0ad9ef013\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"1115522893515788135\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true, \"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"In vitro data suggest that hydroxychloroquine can reduce glycosylation of the ACE2 receptor and also can lower blood glucose concentrations , both which could be possible mechanisms of antiviral activity .\", \"source\": 80, \"target\": 203, \"key\": 4}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35450574673175604\": true}, \"uuid\": {\"6fedad9d-e74f-463f-ab6a-4024d07c6c30\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5003427663175487120\": true}, \"source_api\": {\"isi\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression.\", \"source\": 80, \"target\": 110, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28156989979898945\": true}, \"uuid\": {\"49ae8d23-d9e4-4a86-b51a-17ef963de30b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3973598293395746202\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression.\", \"source\": 110, \"target\": 393, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-14122116205838603\": true}, \"uuid\": {\"cc30eef5-1a10-41f2-8ccd-90148a3cafb0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"959335178432775057\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"XREF_BIBR Hyperinsulinaemia causes increased renal excretion of magnesium, and insulin resistance (IR) reduces intracellular magnesium levels.\", \"source\": 112, \"target\": 31, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-13866309478312853\": true}, \"uuid\": {\"344fb325-efac-45f7-917b-38c3c0964b1a\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3373179886467589763\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"Inhibition of endosomal maturation by NH 4 Cl, BafA1 and Chloq ( Figure 3B , deep sky blue) or of clathrin-mediated endocytosis by Chlopro, Mon, and Dynasore ( Figure 3B , dark blue) severely inhibited MHV fusion.\", \"source\": 51, \"target\": 182, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14966902030929574\": true}, \"uuid\": {\"b34c45a2-465e-4235-8555-3ca6c3041a12\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-4791203231932235156\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"RNAi-mediated downregulation of endocytosis-associated proteins affects MHV infection.\", \"source\": 149, \"target\": 172, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20346968873865507\": true}, \"uuid\": {\"d218a99e-ab78-4cd6-8a3d-76e9e22fe139\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2630288415978593412\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"On one hand, these glycans can trigger a sterilising innate immune response, for example by the activation of the complement cascade.\", \"source\": 138, \"target\": 171, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"32608895331904425\": true}, \"uuid\": {\"9d333b55-f143-43e3-b0af-b6b8f0066f8f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2838676830283100976\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 455, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Finally , complement activation leads to the recruitment of leukocytes and upregulates TF expression , amplifies platelet activation and upregulates endothelial expression of proinflammatory cytokines , including IL ( interleukin ) -1 , IL-6 , IL-8 , and MCP ( monocyte chemoattractant protein ) -1 .\", \"source\": 138, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32303202189393044\": true}, \"uuid\": {\"6ac47dd3-f731-446e-b081-4a9b92c99082\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2838676830283100976\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Finally , complement activation leads to the recruitment of leukocytes and upregulates TF expression , amplifies platelet activation and upregulates endothelial expression of proinflammatory cytokines , including IL ( interleukin ) -1 , IL-6 , IL-8 , and MCP ( monocyte chemoattractant protein ) -1 .\", \"source\": 138, \"target\": 204, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-6787572469025547\": true}, \"uuid\": {\"335f5615-9129-4add-ad7d-c45dbb74ef75\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"-7276091608288579997\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"127 Previous data have highlighted that a major mechanism by which viral infection can induce a procoagulant phenotype pertains to the upregulation of TF expression by virus infected endothelial cells and endothelial TLR-3 signaling in response to virus derived pathogen associated molecular patterns.\", \"source\": 384, \"target\": 455, \"key\": 12}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13466023971755260\": true}, \"uuid\": {\"02703358-ba83-4ee6-b3b8-b23255122239\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"3679903822198429218\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"At present, there are 2 clinical trials (ChiCTR2000030170 and ChiCTR2000029580) the results of which will hopefully provide further insights into JAK and AAK1 inhibition effects in COVID-19 patients.C-C chemokine receptor type 5 (CCR5) is expressed on the surface of white blood cells, especially T-CD4+ cells, and mediates macrophage migration into areas of inflammation, favoring the release of inflammatory cytokines and amplification of the immune response [46].\", \"source\": 406, \"target\": 137, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28597358905238204\": true}, \"uuid\": {\"13249d80-13fc-4a57-adda-6415cc4e4ad5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8959490296759092194\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] .\", \"source\": 406, \"target\": 186, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28597358905238204\": true}, \"uuid\": {\"13249d80-13fc-4a57-adda-6415cc4e4ad5\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8959490296759092194\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"C-C Chemokine Receptor Type 5 Inhibition C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] .\", \"source\": 406, \"target\": 186, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5337843954969320\": true}, \"uuid\": {\"2337dc7a-6f6b-43c8-ba97-87b3a6fc2c17\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4460677682141442227\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33024307\"}, \"evidence\": \"Extensive surveillance of SARS-CoV-2-related viruses in China, Southeast Asia and other regions targeting bats, wild and captured pangolins and other wildlife species will help us to better understand the zoonotic origin of SARS-CoV-2.\", \"source\": 197, \"target\": 208, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13545113987071362\": true}, \"uuid\": {\"0ca401db-d258-4815-953d-e89559db5dca\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-2041021156470522365\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32422233\"}, \"evidence\": \"In the multivariate analysis (model 3), male sex (OR: 2.77; 95% CI: 1.48-5.19; p = 0.001), increasing age analyzed in quartiles (OR: 1.38; 95% CI: 1.panel A).\", \"source\": 266, \"target\": 195, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28998980374258241\": true}, \"uuid\": {\"246a123e-638c-4e7c-bd40-1aabea79154a\": true}, \"belief\": {\"0.98249981399565\": true}, \"source_hash\": {\"-5596021276895866335\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}}, \"target\": 29, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"132 Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration.\", \"source\": 486, \"key\": 5}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28998980374258241\": true}, \"uuid\": {\"246a123e-638c-4e7c-bd40-1aabea79154a\": true}, \"belief\": {\"0.98249981399565\": true}, \"source_hash\": {\"-5596021276895866335\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}}, \"target\": 29, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration.\", \"source\": 486, \"key\": 13}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-24762626725797668\": true}, \"uuid\": {\"785e0607-05bc-43cf-a64f-6560fa2a0755\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"8511000432474559056\": true}, \"source_api\": {\"reach\": true}}, \"target\": 557, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin, which disrupts the outer membrane, allowing lysozyme to gain access to the peptidoglycan layer.\", \"source\": 486, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-7852162285311371\": true}, \"uuid\": {\"099b6355-4276-4e54-aae9-ea334e24d8f3\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1926783695190202708\": true}, \"source_api\": {\"reach\": true}}, \"target\": 94, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin, which disrupts the outer membrane, allowing lysozyme to gain access to the peptidoglycan layer.\", \"source\": 486, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-27777379549788703\": true}, \"uuid\": {\"93dd4aa8-e624-439f-95a1-8ef38870aaa5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8484957420515354582\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer .\", \"source\": 486, \"target\": 154, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-27777379549788703\": true}, \"uuid\": {\"93dd4aa8-e624-439f-95a1-8ef38870aaa5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8484957420515354582\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Lysozyme , an enzyme found in airway epithelial secretions , exerts antimicrobial effect against a wide range of gram-positive bacteria by degrading their peptidoglycan layer.131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer.132 Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration.133 Lactoferrin also displays antiviral activity against both RNA and DNA viruses by either inhibiting binding of the virus to host cells or by binding to the virus itself.134 ,135 Lactoferrin levels increase in response to bacterial and viral infections .\", \"source\": 486, \"target\": 154, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"32512995447185266\": true}, \"uuid\": {\"d8d2528d-f1a1-44a2-bcde-1aa8cd8588b1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5558083028487357300\": true}, \"source_api\": {\"reach\": true}}, \"target\": 5, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116].\", \"source\": 514, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-21230328500049809\": true}, \"uuid\": {\"3a823077-21ff-4d4b-97d3-0948d846b676\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1560815448935115743\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"An encouragement to chase this track more in depth comes from the observation that in non-COVID-19 critically ill hyperglycemic patients admitted at ICU, plasma levels of sRAGE are higher in DM vs. non DM patients, and reduced by intensive insulin therapy only in DM patients (41).\", \"source\": 212, \"target\": 121, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"8778823180626014\": true}, \"uuid\": {\"1827bfb6-aaab-44f3-a633-1632724a053f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4497407204963047821\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 5, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33063540\"}, \"evidence\": \"Under anaerobic conditions , glucose ferments into lactate , which produces a limited amount of adenosine triphosphate ( ATP ) .\", \"source\": 36, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-2098613882031230\": true}, \"uuid\": {\"b65deede-86c6-4704-9037-8e0092e0dc7d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8912175683335367936\": true}, \"source_api\": {\"reach\": true}}, \"target\": 96, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 36, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"18622990453591958\": true}, \"uuid\": {\"d755058d-f47a-452b-be0b-ac7cb30c3cd4\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"-5134636651840275727\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32942131\"}, \"evidence\": \"Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production .\", \"source\": 36, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6085224834592248\": true}, \"uuid\": {\"22ebfaa3-6441-4110-b34f-a0c12b86868a\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"5643023122211539455\": true}, \"source_api\": {\"reach\": true}}, \"target\": 431, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 36, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28587262591953537\": true}, \"uuid\": {\"724a6bb5-abc2-462a-a48c-979b78510d54\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"8857824203905508662\": true}, \"source_api\": {\"reach\": true}}, \"target\": 12, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"18 19 NAD + dependent SIRT3 activates manganese superoxide dismutase 2 ( MnSOD2 ) via acetylation of lysine residue 68 ( K68 ) and increases the production of NADPH via isocitrate dehydrogenase ( IDH2 ) to drive the reduction of oxidised glutathione ( GSSG ) to reduced glutathione ( GSH ) .\", \"source\": 0, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12881042511935175\": true}, \"uuid\": {\"cc96b4dc-2406-4a56-9829-821ac1a4c977\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5130039314594903235\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 455, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"100 Activated leukocytes , principally monocytes , and neutrophils , upregulate TF in response to proinflammatory cytokines , predominantly IL-6 , and subsequently act to enhance coagulation .\", \"source\": 252, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-16970332896020476\": true}, \"uuid\": {\"71972a92-ffd1-4acd-942e-bc79c57c7486\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8711763079269116814\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 5, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"For instance , IFNbeta has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3 / AKT-dependent manner , thereby increasing ATP production [ 116 ] .\", \"source\": 346, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5356350581609109\": true}, \"uuid\": {\"5f979ed1-2180-4f2a-afb4-9c7877881401\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"267974333188792678\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Keratinocyte growth factor (KGF) increases SP-D expression and protein production in association with pneumocyte hyperplasia and after injury caused by bleomycin [105].\", \"source\": 457, \"target\": 367, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5356350581609109\": true}, \"uuid\": {\"5f979ed1-2180-4f2a-afb4-9c7877881401\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"267974333188792678\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Keratinocyte growth factor (KGF) increases SP-D expression and protein production in association with pneumocyte hyperplasia and after injury caused by bleomycin [XREF_BIBR].\", \"source\": 457, \"target\": 367, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"29868350343653611\": true}, \"uuid\": {\"93e3ae97-1fcc-4740-8c55-153436309f27\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4056421924790190900\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"112 Moreover, platelet P-selectin induces monocyte TF expression and upregulates the expression of monocyte Mac-1 (alpha M beta 2, CD11b and CD18).\", \"source\": 364, \"target\": 333, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"16902590516424133\": true}, \"uuid\": {\"ea2a171f-b70b-4aea-8cd9-ed2f5316bedd\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4056421924790190900\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"112 Moreover, platelet P-selectin induces monocyte TF expression and upregulates the expression of monocyte Mac-1 (alpha M beta 2, CD11b and CD18).\", \"source\": 364, \"target\": 455, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"637728159072654\": true}, \"uuid\": {\"56c802d7-c823-4ed6-b635-cd5e47e0acb1\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4636109177624199794\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"This response is similar to what is observed after bacterial infection in which macrophages from diabetic mice or humans have been shown to have reduced phagocytic function and expression of TNF-, IL-6, CCL2, and IL-12 (65) (66) (67) .\", \"source\": 201, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-12879112366386087\": true}, \"uuid\": {\"9b9d5970-30f5-4428-9768-e45935aacdf5\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4636109177624199794\": true}, \"source_api\": {\"reach\": true}}, \"target\": 363, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"This response is similar to what is observed after bacterial infection in which macrophages from diabetic mice or humans have been shown to have reduced phagocytic function and expression of TNF-, IL-6, CCL2, and IL-12 (65) (66) (67) .\", \"source\": 201, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6348079141058706\": true}, \"uuid\": {\"0d3c4d45-80b6-49b8-8115-29fbd3661079\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6257108394171638638\": true}, \"source_api\": {\"reach\": true}}, \"target\": 111, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"However, after adjusting for other confounders, the presence of hypertension increased the risk slightly (HR =1.07, 95% CI = 1.00-1.15) (14) .\", \"source\": 230, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6348079141058706\": true}, \"uuid\": {\"0d3c4d45-80b6-49b8-8115-29fbd3661079\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6257108394171638638\": true}, \"source_api\": {\"reach\": true}}, \"target\": 111, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"In a large study from the UK, accounting for age and sex, hypertension increases the risk of in-hospital death (HR = 1.22, 95% CI =1.15-1.30).\", \"source\": 230, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3401594388405438\": true}, \"uuid\": {\"b64c5f73-3fc5-44ba-a8be-d3cccad67cc3\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"5747800940286111385\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"risk\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Deep vein thrombosis occurrence increases significantly with elevated carbon monoxide concentrations , increasing the risk of pulmonary emboli ( PE ) and acute coronary syndrome.23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk .\", \"source\": 230, \"target\": 131, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3401594388405438\": true}, \"uuid\": {\"b64c5f73-3fc5-44ba-a8be-d3cccad67cc3\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"5747800940286111385\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"risk\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk .\", \"source\": 230, \"target\": 131, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3401594388405438\": true}, \"uuid\": {\"b64c5f73-3fc5-44ba-a8be-d3cccad67cc3\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"5747800940286111385\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"risk\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Deep vein thrombosis occurrence increases significantly with elevated carbon monoxide concentrations , increasing the risk of pulmonary emboli ( PE ) and acute coronary syndrome.23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk .\", \"source\": 230, \"target\": 131, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"574852354930872\": true}, \"uuid\": {\"3c3aeb0c-67b0-4fad-91ab-da99f9cecd81\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8363541067322718705\": true}, \"source_api\": {\"reach\": true}}, \"target\": 543, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"However, after adjusting for other confounders, the presence of hypertension increased the risk slightly (HR =1.07, 95% CI = 1.00-1.15) (14) .\", \"source\": 230, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"1135366909342241\": true}, \"uuid\": {\"fbfa2e74-7644-4ac7-9506-90ed92a3672c\": true}, \"belief\": {\"0.979665\": true}, \"source_hash\": {\"-1100350664990952163\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis, inflammation and NF-B signaling OPEN.pneumonia-induced sepsis.\", \"source\": 286, \"target\": 135, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-15919320202667609\": true}, \"uuid\": {\"4a32ca9f-f0e0-44a8-a293-e507f62455d9\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"7839234538021613990\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis, inflammation and NF-B signaling OPEN.pneumonia-induced sepsis.\", \"source\": 286, \"target\": 136, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31831557589908613\": true}, \"uuid\": {\"a59c8a6e-fda7-4266-bec7-ee364bba2359\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5654731676882551931\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia , and submucosal gland hypertrophy resulting in obstruction of airways .\", \"source\": 229, \"target\": 250, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26881292599562012\": true}, \"uuid\": {\"be5136c6-d7df-4f2d-933a-d0ea3b043e15\": true}, \"belief\": {\"0.9816495\": true}, \"source_hash\": {\"-8538859823506274175\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 96, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 246, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1927139420683844\": true}, \"uuid\": {\"c70f9b29-5049-492a-ba13-02b5bd1eb230\": true}, \"belief\": {\"0.99708229353\": true}, \"source_hash\": {\"-80057021992063723\": true}, \"source_api\": {\"reach\": true}}, \"target\": 431, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients.\", \"source\": 246, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35105136078016014\": true}, \"uuid\": {\"2e33591d-f6e8-49c3-9fcf-04fd2b28663f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-489842241526344206\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"At last, using a constitutively active phosphor-mimetic IRF3, it was recently shown that the DUB activity of PLPro also inhibited IRF3 at a postactivation step [196].\", \"source\": 430, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13271804993398704\": true}, \"uuid\": {\"208b1382-1469-44c7-8c07-1830041e02b7\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"9104209993257731948\": true}, \"source_api\": {\"reach\": true}}, \"target\": 421, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"The furin induces the cleavage of SPs at the RNTR761EV site (Bergeron et al., 2005) and results in the production of two segments: S1 and S2 (Figure 2b); each segment shows its own biological activity.\", \"source\": 507, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13271804993398704\": true}, \"uuid\": {\"208b1382-1469-44c7-8c07-1830041e02b7\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"9104209993257731948\": true}, \"source_api\": {\"reach\": true}}, \"target\": 421, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"The furin induces the cleavage of SPs at the RNTR761vEV site and results in the production of two segments : S1 and S2 (XREF_FIG); each segment shows its own biological activity.\", \"source\": 507, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"3137193870736731\": true}, \"uuid\": {\"6676df22-89d5-451c-896a-76dadb6e12a3\": true}, \"belief\": {\"0.9974998848535086\": true}, \"source_hash\": {\"-3190507349569191444\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"MERS-CoV , which contains a minimal FCS , is inhibited by furin inhibitor ( FI ) but not by the pan-lysosomal protease inhibitor ( CPI ) .\", \"source\": 507, \"target\": 320, \"key\": 13}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5932785887071917\": true}, \"uuid\": {\"8e0b62a7-892c-4830-afdb-44fbb2a6f127\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-9098918993761077103\": true}, \"source_api\": {\"reach\": true}}, \"target\": 397, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The surface glycoprotein HA exists as a trimer of hetero-dimers that is synthesised as a single polypeptide (HA0) that is subsequently cleaved into the receptor binding HA1 domain and the membrane-fusion mediating HA2 subunit by host cell proteases.\", \"source\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26021155500054272\": true}, \"uuid\": {\"378932be-776c-4005-8545-60557890511d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6295217941457638469\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"101, 102 Depletion of PALS1 leads to the loss of PATJ, disruption of cell polarity, decreased TER, and altered trafficking of E-cadherin.\", \"source\": 461, \"target\": 438, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25187925905698925\": true}, \"uuid\": {\"cc367c59-8269-4c1e-868d-095e8314774a\": true}, \"belief\": {\"0.949271\": true}, \"source_hash\": {\"993859597859123648\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32372149\"}, \"evidence\": \"ACE2 plays an important role in the renin-angiotensin system (RAS), and the imbalance between ACE/Ang II/AT1R pathway and ACE2/Ang (1-7)/ Mas receptor pathway in the RAS system will lead to multisystem inflammation [4] .\", \"source\": 488, \"target\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15558827484106353\": true}, \"uuid\": {\"431d80ed-170e-4360-a3e3-f1013686f609\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8188441235124157144\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Mas receptor expression has also been identified in the liver and is elevated in biliary fibrosis (Herath et al., 2007).\", \"source\": 488, \"target\": 221, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33703141384934955\": true}, \"uuid\": {\"74e1ed4d-ad31-4cc8-a215-171471ef34d6\": true}, \"belief\": {\"0.9499999999990586\": true}, \"source_hash\": {\"9187916143977181738\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Additionally, paramyxoviral GP-induced membrane fusion does not require a drop in pH, as many other viral families do.\", \"source\": 534, \"target\": 177, \"key\": 14}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33703141384934955\": true}, \"uuid\": {\"74e1ed4d-ad31-4cc8-a215-171471ef34d6\": true}, \"belief\": {\"0.9499999999990586\": true}, \"source_hash\": {\"9187916143977181738\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Binding of the attachment GP (HN, H or G) to a cell surface receptor triggers the fusion GP (F) to insert itself into the host cell membrane, facilitating viral-cell membrane fusion (Iorio et al. 2009; Chang and Dutch 2012) .\", \"source\": 534, \"target\": 177, \"key\": 15}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-30502537953339668\": true}, \"uuid\": {\"97476716-6eb2-4694-b79f-a02194ff878f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6295217941457638469\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"101, 102 Depletion of PALS1 leads to the loss of PATJ, disruption of cell polarity, decreased TER, and altered trafficking of E-cadherin.\", \"source\": 414, \"target\": 438, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-10676966401234684\": true}, \"uuid\": {\"a88de857-c9bd-4083-9fb6-81b480772b1e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1874598850483302026\": true}, \"source_api\": {\"reach\": true}}, \"target\": 6, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"This may contribute to respiration independent increased carbon dioxide levels, as extramitochondrial haem synthesis step 5, produces four carbon dioxide molecules via the decarboxylation of uroporphyrinogen III to coproporphyrinogen III, further taxing the external respiration system.\", \"source\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-4663318369181314\": true}, \"uuid\": {\"1fc74c64-b476-47c7-beb4-164891125608\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 378, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-20760708490921242\": true}, \"uuid\": {\"81318612-6e55-4917-b4bf-6c1f88759264\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 378, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"12910136755106180\": true}, \"uuid\": {\"34c9080c-928e-4a8e-b405-5d79dcabec82\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 496, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 378, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-2752452007797226\": true}, \"uuid\": {\"4ebda165-f9c0-4a56-b3f9-402b1be9ac72\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6737299411419827340\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Of clinical relevance, blockade of the reninangiotensin system prevents new-onset diabetes and reduces the risk of diabetic complications.\", \"source\": 521, \"target\": 302, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"34098015961451693\": true}, \"uuid\": {\"7765cdbd-6eda-482f-b18e-a7d0afba8e86\": true}, \"belief\": {\"0.9973750600677\": true}, \"source_hash\": {\"-5793623738344183379\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"Cellular infection and viral replication cause activation of the inflammasome in the host cell, leading to the release of proinflammatory cytokines and cell death by pyroptosis with ensuing release of a damage-associated molecular pattern, further amplifying the inflammatory response [7, 8, 9].\", \"source\": 180, \"target\": 143, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5894972213903825\": true}, \"uuid\": {\"ec3556c1-79d1-4e1e-b42f-9198036840ca\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-6616129564876427800\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"SP-D KO mice also showed increased inflammation and lung injury caused by endotoxemia 17 .\", \"source\": 288, \"target\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5894972213903825\": true}, \"uuid\": {\"ec3556c1-79d1-4e1e-b42f-9198036840ca\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"-6616129564876427800\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"However, endotoxemia-induced indirect lung injury showed increased pulmonary inflammation but with a paradoxical decrease in mortality in SP-D KO mice when compared with WT mice17.\", \"source\": 288, \"target\": 136, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1070912317366832\": true}, \"uuid\": {\"e67e603e-9637-4198-9a0f-0a3e9df27b51\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7047129327418342347\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) .\", \"source\": 83, \"target\": 267, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1070912317366832\": true}, \"uuid\": {\"e67e603e-9637-4198-9a0f-0a3e9df27b51\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7047129327418342347\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) .\", \"source\": 83, \"target\": 267, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1070912317366832\": true}, \"uuid\": {\"e67e603e-9637-4198-9a0f-0a3e9df27b51\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-7047129327418342347\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) .\", \"source\": 83, \"target\": 267, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11049281037639997\": true}, \"uuid\": {\"1f714f53-6a6c-465d-8dee-f4077e2cdcae\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"788127187935824179\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 369, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Deletion of all three N-glycans completely blocked CSE .\", \"source\": 117, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-4349923190244333\": true}, \"uuid\": {\"2547bdad-25b2-48ad-ad27-f11e80d2dc2d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1366912351931562243\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78].\", \"source\": 369, \"target\": 128, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20463018722379383\": true}, \"uuid\": {\"9baadd26-7a20-4b01-b420-e8b69332d720\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1856462107741649160\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29579213\"}, \"evidence\": \"Conversely, the deletion of the Ebola virus mucin-like domain decreased the immune response in mice, although it did not have an effect on immunogenicity in vitro (Dowling et al. 2007).\", \"source\": 530, \"target\": 306, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12401760367222338\": true}, \"uuid\": {\"71259494-d1bc-45c0-afe7-29997ff1593e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4662201417729592355\": true}, \"source_api\": {\"reach\": true}}, \"target\": 352, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Increased expression of influenza M2 protein can activate protein kinase C and increase reactive oxygen species (ROS) production [10] .\", \"source\": 505, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12161625017927270\": true}, \"uuid\": {\"26e3c370-dda2-40a7-8f01-38de1b4e6294\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7402027533243552185\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Similarly, rHMGB1 (but not low-dose LPS) also induced the upregulation of ACE2 mRNA in HepG2 (a human liver cell line), Caco2 (a human intestine cell line), and RT4 (a human urinary bladder cell line) cells.\", \"source\": 120, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"8383038928348764\": true}, \"uuid\": {\"d80d3af9-5d8f-4d35-8df1-518bf02513c5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"912743691949735686\": true}, \"source_api\": {\"reach\": true}}, \"target\": 386, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"We consistently found that rHMGB1 induced TNF release was blocked in TLR4-/- THP1 cell (a human monocytic cell line).\", \"source\": 120, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"8383038928348764\": true}, \"uuid\": {\"d80d3af9-5d8f-4d35-8df1-518bf02513c5\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"912743691949735686\": true}, \"source_api\": {\"reach\": true}}, \"target\": 386, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"We consistently found that rHMGB1 induced TNF release was blocked in TLR4 A/A THP1 cell (a human monocytic cell line).\", \"source\": 120, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"4811726090351493\": true}, \"uuid\": {\"a855f3d5-f4db-4fae-8891-38a4b15f90d2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8573078188862412247\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Tunicamycin treatment inhibits viral release, likely by trapping viral GPs in the cytoplasm of the infected cell.\", \"source\": 52, \"target\": 153, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-32140406486790311\": true}, \"uuid\": {\"6d7aa359-76f0-4754-89e4-dc37e070161f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8683905135949325626\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Additionally, tunicamycin treatment inhibited SeV viral replication (Nakamura et al. 1982) .\", \"source\": 52, \"target\": 164, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"31531948140570773\": true}, \"uuid\": {\"2b5de13e-70f1-4fba-ae48-63e424040557\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8573078188862412247\": true}, \"source_api\": {\"reach\": true}}, \"target\": 415, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Tunicamycin treatment inhibits viral release, likely by trapping viral GPs in the cytoplasm of the infected cell.\", \"source\": 52, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12069606483766498\": true}, \"uuid\": {\"48d63b40-81d7-4498-83e3-95410b9f3c68\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"8766110901861728666\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"may\": true, \"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32271993\"}, \"evidence\": \"Moreover , obesity and the metabolic syndrome are thought to increase type 2 inflammation , which may have effects on the lung parenchyma and bronchi ( 8) .\", \"source\": 291, \"target\": 136, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14700215358097096\": true}, \"uuid\": {\"32d52867-1a2e-4007-9bb1-da348501a885\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-145998067916886353\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 407, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"In diabetic humans infected with M. tuberculosis , CD8 + T cell numbers in circulation were similar to healthy patients , but a higher frequency of these cells produced IFN , IL-2 , and IL-17F along with having decreased antigen-specific expression of cytotoxic markers ( 58 ) .\", \"source\": 126, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26777647337085364\": true}, \"uuid\": {\"9835f888-ffd7-4faf-b523-8e13d51321ba\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"-8795846753999398942\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 476, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"In diabetic humans infected with M. tuberculosis , CD8 + T cell numbers in circulation were similar to healthy patients , but a higher frequency of these cells produced IFN , IL-2 , and IL-17F along with having decreased antigen-specific expression of cytotoxic markers ( 58 ) .\", \"source\": 126, \"key\": 5}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3938219864781809\": true}, \"uuid\": {\"492a331c-c908-43a1-9381-ebd98f9c8ca3\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"8712264991226635271\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Ang II, through the activation of Ang II type 1a receptors, induces vasoconstriction and proliferation, but angiotensin 1-7 stimulates vasodilatation and suppresses cell growth.\", \"source\": 75, \"target\": 148, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"32917678287072914\": true}, \"uuid\": {\"fb9e6bdf-32ed-4981-9d77-92edfeecb25d\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"2420811417397093084\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 455, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"100 Activated leukocytes , principally monocytes , and neutrophils , upregulate TF in response to proinflammatory cytokines , predominantly IL-6 , and subsequently act to enhance coagulation .\", \"source\": 248, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"22062125597434302\": true}, \"uuid\": {\"4b2d104c-d9e6-41fe-bcfc-996d36711cd3\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"8992679658034904653\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"expect\": true}}, \"target\": 385, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32537478\"}, \"evidence\": \"It is worth mentioning that serine protease inhibitors such as 274 camostat mesylate which blocks TMPRSS2 activity could be used for treatment of SARS-275CoV-2-infected patients, 6, 26 or as a preventive approach especially for patients identified as 276 high risk based on ACE2 nasal screening.\", \"source\": 1, \"key\": 46}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8808082855643020\": true}, \"uuid\": {\"d3abd505-c352-4938-8cfc-02457cbfe969\": true}, \"belief\": {\"0.991866\": true}, \"source_hash\": {\"-2747538331993190250\": true}, \"source_api\": {\"reach\": true}}, \"target\": 13, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Indeed, ACE inhibitors improve skeletal muscle glucose uptake through a bradykinin-mediated increase in NO (Henriksen et al., 1999).\", \"source\": 56, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19857802335323063\": true}, \"uuid\": {\"ef9e5d52-3ec6-49ad-96c2-192912c94025\": true}, \"belief\": {\"0.9974990151322722\": true}, \"source_hash\": {\"-2851987500157838699\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"It is well known that Ang-(17) potentiates bradykinin-mediated vasodilation (Paula et al., 1995, Lima et al., 1997).\", \"source\": 56, \"target\": 166, \"key\": 10}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17533624946767864\": true}, \"uuid\": {\"f774272e-dd52-4aae-be8a-c5973b8c5d88\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"9171435980758853165\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"In both T1DM and T2DM, loss of Ang-(17) potentiation of bradykinin-mediated vasodilation is rescued by chronic but not acute insulin (Oliveira et al., 2002, Rastelli et al., 2007).\", \"source\": 56, \"target\": 166, \"key\": 24}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27256361616910242\": true}, \"uuid\": {\"4ef013b4-e2b3-46d2-af99-bcf1a3396742\": true}, \"belief\": {\"0.94999995217031\": true}, \"source_hash\": {\"6791170434154724436\": true}, \"source_api\": {\"reach\": true}}, \"target\": 9, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"First, metformin works by inhibiting the ATP synthesis to increase the AMP or ADP levels that in turn indirectly induce the activation of the AMP activated protein kinase (AMPK) in hepatocytes.\", \"source\": 88, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10036642598619563\": true}, \"uuid\": {\"03891932-f7d7-4a0a-80b5-550e8db11b43\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"9162696058733270147\": true}, \"source_api\": {\"reach\": true}}, \"target\": 37, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"A reduction in gastric acidity production by PPI drugs decrease Mg 2+ solubility, thus decreasing absorption potential in the gastrointestinal (GI) tract; metformin causes GI Mg 2+ loss, while diuretics and hyperglycaemia decrease renal tubular reabsorption of Mg 2+.\", \"source\": 88, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4051603025043013\": true}, \"uuid\": {\"30a1830b-b9f3-4138-932d-cd6d938d6775\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"9162696058733270147\": true}, \"source_api\": {\"reach\": true}}, \"target\": 92, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"A reduction in gastric acidity production by PPI drugs decrease Mg 2+ solubility, thus decreasing absorption potential in the gastrointestinal (GI) tract; metformin causes GI Mg 2+ loss, while diuretics and hyperglycaemia decrease renal tubular reabsorption of Mg 2+.\", \"source\": 88, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20022981514247206\": true}, \"uuid\": {\"0433ad3b-bbbc-4013-982e-aae1388e5cb7\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"1613936963366379427\": true}, \"source_api\": {\"reach\": true}}, \"target\": 458, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"[9, 10] Second, metformin inhibits the mammalian target of rapamycin (mTOR) signaling via liver kinase B1 (LKB1).\", \"source\": 88, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20022981514247206\": true}, \"uuid\": {\"0433ad3b-bbbc-4013-982e-aae1388e5cb7\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"1613936963366379427\": true}, \"source_api\": {\"reach\": true}}, \"target\": 458, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"The PI3K/AKT/mTOR pathway plays a major role in MERS-CoV infection, which may also be implicated in the pathogenesis of SARS-CoV-2 infection, given many similarities between these two viruses.Therefore, inhibition of mTOR signaling by metformin may result in a reduction of SARS-CoV-2 infectivity and mortality from COVID-19 infection.\", \"source\": 88, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20022981514247206\": true}, \"uuid\": {\"0433ad3b-bbbc-4013-982e-aae1388e5cb7\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"1613936963366379427\": true}, \"source_api\": {\"reach\": true}}, \"target\": 458, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"Therefore, inhibition of mTOR signaling by metformin may result in a reduction of SARS-CoV-2 infectivity and mortality from COVID-19 infection.\", \"source\": 88, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20022981514247206\": true}, \"uuid\": {\"0433ad3b-bbbc-4013-982e-aae1388e5cb7\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"1613936963366379427\": true}, \"source_api\": {\"reach\": true}}, \"target\": 458, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"Second, metformin inhibits the mammalian target of rapamycin (mTOR) signaling via liver kinase B1 (LKB1).\", \"source\": 88, \"key\": 9}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28944980460718309\": true}, \"uuid\": {\"dddac482-6abb-438a-aae5-06d785631ff4\": true}, \"belief\": {\"0.999974967195\": true}, \"source_hash\": {\"-5791459418776781677\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"could\": true}}, \"target\": 112, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"26021280\"}, \"evidence\": \"Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR].\", \"source\": 88, \"key\": 30}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28944980460718309\": true}, \"uuid\": {\"dddac482-6abb-438a-aae5-06d785631ff4\": true}, \"belief\": {\"0.999974967195\": true}, \"source_hash\": {\"-5791459418776781677\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"could\": true}}, \"target\": 112, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"26021280\"}, \"evidence\": \"Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR].\", \"source\": 88, \"key\": 56}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28944980460718309\": true}, \"uuid\": {\"dddac482-6abb-438a-aae5-06d785631ff4\": true}, \"belief\": {\"0.999974967195\": true}, \"source_hash\": {\"-5791459418776781677\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"could\": true}}, \"target\": 112, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"26021280\"}, \"evidence\": \"Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR].\", \"source\": 88, \"key\": 89}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-8969855582340497\": true}, \"uuid\": {\"2be431d1-2772-4fc6-a0fe-88779d83124d\": true}, \"belief\": {\"0.05518229353000004\": true}, \"source_hash\": {\"-8859890793154507335\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"if\": true, \"whether\": true}, \"negated\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"[ 11 ] Finally , through activation of AMPK , metformin can inhibit the inflammatory response that could potentially contribute to mortality through mechanism such as cytokine storm and vascular damage.10 , 12 Therefore , metformin use will decrease the inflammatory response , reducing the incidence of cytokine storm or vascular damage and finally lead to reduction in mortality rate .\", \"source\": 88, \"target\": 249, \"key\": 9}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-25532367439496652\": true}, \"uuid\": {\"ddfcd9e4-433f-448f-9ada-321be12a8daf\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5699811072152272753\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 423, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Activation of protease-activated receptor 2 ( PAR2 ) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [ 95 ] .\", \"source\": 503, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"260506042973872\": true}, \"uuid\": {\"1ea2e56a-b635-4c0a-8c7e-8d56c6d9ea74\": true}, \"belief\": {\"0.9499999999968619\": true}, \"source_hash\": {\"-2009954255710980895\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"consider\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Therefore , the lung and kidney SP-D still plays the role of inhibiting inflammation by blocking Toll-like receptor 4 through the CRD54 to some extent even though they were dramatically reduced in infected WT mice , which resulted in the attenuated lung-kidney crosstalk than in KO or hTG mice .\", \"source\": 163, \"target\": 136, \"key\": 11}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33608561620647864\": true}, \"uuid\": {\"9dfe8c4e-b405-48ed-b415-edaceda9328a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2565963295239507055\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"if\": true, \"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"17398101\"}, \"evidence\": \"If carbohydrate is positioned close to the HA cleavage site , the glycan can prevent protease access and virus entry .\", \"source\": 28, \"target\": 309, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15497754100328031\": true}, \"uuid\": {\"ed5cfafa-257d-4138-b942-d622d940bbb3\": true}, \"belief\": {\"0.95\": true}, \"source_hash\": {\"-9102873696415741496\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33024307\"}, \"evidence\": \"Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs60 ,61 ) .\", \"source\": 192, \"target\": 261, \"key\": 18}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15497754100328031\": true}, \"uuid\": {\"ed5cfafa-257d-4138-b942-d622d940bbb3\": true}, \"belief\": {\"0.95\": true}, \"source_hash\": {\"-9102873696415741496\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33024307\"}, \"evidence\": \"Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs 60,61 ) .\", \"source\": 192, \"target\": 261, \"key\": 19}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-10614391718634934\": true}, \"uuid\": {\"89772edf-735b-4f77-828a-04f1ecd25ab7\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"4328716184730300581\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33024307\"}, \"evidence\": \"Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs60 ,61 ) .\", \"source\": 192, \"target\": 262, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-10614391718634934\": true}, \"uuid\": {\"89772edf-735b-4f77-828a-04f1ecd25ab7\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"4328716184730300581\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"might\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33024307\"}, \"evidence\": \"Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs 60,61 ) .\", \"source\": 192, \"target\": 262, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-2576795783465982\": true}, \"uuid\": {\"f57d134d-c911-4c05-92f8-4a79503a34cf\": true}, \"belief\": {\"0.0\": true}, \"source_hash\": {\"669569813816027678\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"suggest\": true}, \"negated\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31550243\"}, \"evidence\": \"These data suggest that prolonged disease following MERS-CoV infection is not due to increased virus replication , a delay in virus clearance , or increased extrapulmonary dissemination of the virus .\", \"source\": 275, \"target\": 215, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27699863846148654\": true}, \"uuid\": {\"e7bff77d-8882-493b-b27a-87272468debc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-5685221942939849142\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 80, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"However , some studies have shown that hydroxychloroquine is ineffective in some patients , partly due to its cardiotoxicity [ 31 ] , although more clinical trials are ongoing .\", \"source\": 321, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"27699863846148654\": true}, \"uuid\": {\"e7bff77d-8882-493b-b27a-87272468debc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-5685221942939849142\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 80, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"However , some studies have shown that hydroxychloroquine is ineffective in some patients , partly due to its cardiotoxicity [ 31 ] , although more clinical trials are ongoing .\", \"source\": 321, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"4539879569389452\": true}, \"uuid\": {\"74c6fe82-175d-47f1-bd46-22a6cdf7c344\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-938611972429656515\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"However, galectin-1 has also been shown to promote HIV infection of T cells by cross-linking the virus to host cells [298,299].\", \"source\": 485, \"target\": 279, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"9337520278055066\": true}, \"uuid\": {\"31920c83-beb9-4013-aba1-6e108d31b39a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8350310908433689874\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"Galectin-1 has been shown to bind to the envelope fusion glycoprotein (F) of Nipah [[294], [295], [296]], Hendra [294], and HA of influenza [297] viruses and inhibit membrane fusion.\", \"source\": 485, \"target\": 177, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"28076029637512279\": true}, \"uuid\": {\"acbf50e2-df9c-4adc-b068-5afc81d8b5b5\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7175103377123225507\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" ADP-ribosylated N protein can be detected in coronavirus-infected cells and in purified virions.\", \"source\": 17, \"target\": 505, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20903898713914806\": true}, \"uuid\": {\"8c00e0ae-45ac-45cb-9edb-0e918aa32849\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3267835160193813294\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"To investigate the antiinflammatory effects of statins, a phase 2 RCT is evaluating a combination of the Janus kinase (JAK) inhibitor, ruxolitinib and simvastatin in the prevention and treatment of respiratory failure in COVID-19 (ClinicalTrials.gov Identifier: NCT04348695).\", \"source\": 98, \"target\": 193, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1463575731519120\": true}, \"uuid\": {\"210df09c-2c94-4d7f-8f3e-f263a6e2678d\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"-4622530130208513259\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 .\", \"source\": 319, \"target\": 204, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"1463575731519120\": true}, \"uuid\": {\"210df09c-2c94-4d7f-8f3e-f263a6e2678d\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"-4622530130208513259\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"The levels of MPO-DNA and citrullinated histone H3 were higher in patients receiving mechanical ventilation compared with those not requiring any supplemental oxygen suggesting that levels of NETs correlate with disease severity.109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 .\", \"source\": 319, \"target\": 204, \"key\": 6}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5967374504037936\": true}, \"uuid\": {\"a678548d-b3f5-4a39-aca5-d64a3651de3e\": true}, \"belief\": {\"0.9974999999924453\": true}, \"source_hash\": {\"-4045176205671634601\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"Several in vivo studies in animal models and ex vivo human lung models have proven that MSC can prevent lung injury , reduce inflammation , regulate immune responses , and foster alveolar fluid clearance [ 47 ] .\", \"source\": 318, \"target\": 136, \"key\": 22}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-17017073157970410\": true}, \"uuid\": {\"fc27fd30-3732-4277-af63-257aefe79529\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"6950664131732659610\": true}, \"source_api\": {\"reach\": true}}, \"target\": 533, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos).\", \"source\": 405, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23086943486589876\": true}, \"uuid\": {\"8e112187-f65d-4f40-9145-6454cf77ff15\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2450385320562646191\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 487, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer .\", \"source\": 487, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33556400074601591\": true}, \"uuid\": {\"b2417c86-017d-44af-8850-898a85996b87\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Hyperinsulinaemia and hyperglycaemia increases oxidised haem which increases haem catabolism, resulting in chronic increased CO production.\", \"source\": 54, \"target\": 145, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33556400074601591\": true}, \"uuid\": {\"b2417c86-017d-44af-8850-898a85996b87\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 54, \"target\": 145, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-101142162716651\": true}, \"uuid\": {\"e14540f2-9381-422a-ab4c-2edb273f9b8f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"target\": 97, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 54, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15639315560845537\": true}, \"uuid\": {\"95b35587-b428-42a7-93f8-e1f7bde238b7\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4815649184594005115\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"These life-threatening syndromes are caused by increased permeability of the alveolar and airway epithelium and exudate formation .\", \"source\": 255, \"target\": 269, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"36004597877608747\": true}, \"uuid\": {\"c78c634c-a602-45a3-88e0-39f18712fa6f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-47027461813382191\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"In addition, Kopecky-Bromberg et al. [24] found that SARS-CoV open reading frame (ORF) 3b, ORF 6, and N proteins inhibit the expression of IRF-3, suggesting that, like other highly pathogenic viruses, SARS-CoV genome encodes more than one protein able to inhibit IFN.Therefore, altogether these results may help to explain the rapid rise in virus titers during the initial phase of SARS disease.\", \"source\": 531, \"target\": 481, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"32347565099556704\": true}, \"uuid\": {\"637431b7-43d6-4d90-b2c8-feac3ff80cd3\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2735963416130193884\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"27 , 28 Hypoglycemia also increases mortality 29 and may go undetected with infrequent glucose determinations .\", \"source\": 232, \"target\": 249, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"23358648761479775\": true}, \"uuid\": {\"14a27452-dcba-4724-a400-6c3835426776\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"401643414877310907\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 429, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"In the airway epithelium , cldn8 is up-regulated by glucocorticoids but not by mineralocorticoids [ 66 ] .\", \"source\": 38, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"annotations\": {\"stmt_hash\": {\"-28791596420152451\": true}, \"uuid\": {\"4afb8d31-df15-4d3d-baf4-d9bfc3daa4c4\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3144926386375614639\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29579213\"}, \"evidence\": \"Most types of O-glycosylation are carried out in the secretory pathway, but cytosolic and nuclear proteins can also be glycosylated with N-acetylglucosamine (GlcNAc) by a cytosolic glycosyltransferase (Moremen et al. 2012; Yang and Qian 2017).\", \"source\": 343, \"target\": 78, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22028237537878280\": true}, \"uuid\": {\"0bdb6468-b9c4-40d5-886b-a692d7e062e6\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"310665815426129990\": true}, \"source_api\": {\"reach\": true}}, \"target\": 16, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32274964\"}, \"evidence\": \"(c) CD signals of QSARS-4 peptide after incubation with different concentrations of recombinant furin (Basak et al., 2007).\", \"source\": 538, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"487340363336876\": true}, \"uuid\": {\"e1b79dd9-d260-472b-b620-fa000cb9fd08\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8615139638761651277\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"20036966\"}, \"evidence\": \"In addition to direct antiviral activities, the SP-A and SP-D modulate innate and adaptive immunity and inflammation associated with infection.\", \"source\": 387, \"target\": 124, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30492846845919691\": true}, \"uuid\": {\"88104c26-c5e6-4699-9f67-86c3958b0304\": true}, \"belief\": {\"0.98285\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 522, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30492846845919691\": true}, \"uuid\": {\"88104c26-c5e6-4699-9f67-86c3958b0304\": true}, \"belief\": {\"0.98285\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection .\", \"source\": 522, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30492846845919691\": true}, \"uuid\": {\"88104c26-c5e6-4699-9f67-86c3958b0304\": true}, \"belief\": {\"0.98285\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 522, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17165884426082545\": true}, \"uuid\": {\"1aa8c5b8-d161-49bc-95d1-a338fe33421d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 522, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17165884426082545\": true}, \"uuid\": {\"1aa8c5b8-d161-49bc-95d1-a338fe33421d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection .\", \"source\": 522, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17165884426082545\": true}, \"uuid\": {\"1aa8c5b8-d161-49bc-95d1-a338fe33421d\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 482, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 522, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"21654924087179002\": true}, \"uuid\": {\"b08a5d33-9ccf-4337-956a-584cd141e398\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8240613420591144196\": true}, \"source_api\": {\"reach\": true}}, \"target\": 496, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection.\", \"source\": 522, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-1692447096197785\": true}, \"uuid\": {\"69ebf300-dc27-4a48-9c52-24745663bc36\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3402588927684383400\": true}, \"source_api\": {\"reach\": true}}, \"target\": 72, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Clinically, metabolic syndrome is defined as the presence of three or more of the following factors: increased waist circumference (population and country-specific cutoff), hypertriglyceridemia (>150 mg/dL or on treatment for hypertriglyceridemia), elevated blood pressure (systolic  130 and/or diastolic  85 mm Hg or with a history of hypertension on treatment), reduced high-density lipoprotein cholesterol (<40 mg/dL in males; <50 mg/dL in females), and dysglycemia ( 100 mg/dL or on treatment for hyperglycemia) (2).\", \"source\": 278, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"240139848778815\": true}, \"uuid\": {\"095ae9f8-ffee-4298-ae7e-19a42c453afb\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3402588927684383400\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Clinically, metabolic syndrome is defined as the presence of three or more of the following factors: increased waist circumference (population and country-specific cutoff), hypertriglyceridemia (>150 mg/dL or on treatment for hypertriglyceridemia), elevated blood pressure (systolic  130 and/or diastolic  85 mm Hg or with a history of hypertension on treatment), reduced high-density lipoprotein cholesterol (<40 mg/dL in males; <50 mg/dL in females), and dysglycemia ( 100 mg/dL or on treatment for hyperglycemia) (2).\", \"source\": 278, \"target\": 228, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34355763419181757\": true}, \"uuid\": {\"198e7b93-e444-4eee-87ee-38cf80fc4499\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-109172356155407790\": true}, \"source_api\": {\"reach\": true}}, \"target\": 39, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32220035\"}, \"evidence\": \"The modes of action of approved antivirals include inhibition of virus entry or fusion (enfuvirtide), inhibition of uncoating (amantadine), inhibition of nucleic acid synthesis (acyclovir, ribavirin), maturation inhibitors (lopinavir, ritonavir) and inhibitors of virus release (oseltamivir).\", \"source\": 35, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1918529973210317\": true}, \"uuid\": {\"d4f5ea63-15d2-43bf-bc29-9511d70fd4be\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2201343995621321079\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Sham = Sham infection, Pneu = Pneumonia.SP-D deficiency increases NF-B activation and renal inflammation in the kidney.\", \"source\": 70, \"target\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13073750591957833\": true}, \"uuid\": {\"65434aea-d129-42b6-8090-d0448fa4e79e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-2077445203627985156\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30337682\"}, \"evidence\": \"Sham = Sham infection, Pneu = Pneumonia.Figure 8: Primary proximal tubular epithelial cells (PTECs) from KO mice increase apoptosis and apoptosis-related caspase-3 expression after LPS treatment compared to those from WT mice.\", \"source\": 70, \"target\": 402, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"31023019578144377\": true}, \"uuid\": {\"aa694535-44c4-418a-8bd4-98454d92c009\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3292056227616648405\": true}, \"source_api\": {\"reach\": true}}, \"target\": 533, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos).\", \"source\": 396, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5222721481044498\": true}, \"uuid\": {\"ecf0f3b3-bac1-49b0-bd8f-cc2b8e179184\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8246786178201165613\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 122, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Additionally , SARS-CoV N protein was shown to translocate to cytoplasmic stress granules in response to cellular stress , while phosphorylation in the serine-arginine rich region inhibited this translocation [ 133 ] .\", \"source\": 150, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3536476931991917\": true}, \"uuid\": {\"9bd0872a-bcc8-4f2e-9db8-99ec8766b943\": true}, \"belief\": {\"0.9926598\": true}, \"source_hash\": {\"-4971714610315652007\": true}, \"source_api\": {\"reach\": true}}, \"target\": 10, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"123 Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2.\", \"source\": 445, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3536476931991917\": true}, \"uuid\": {\"9bd0872a-bcc8-4f2e-9db8-99ec8766b943\": true}, \"belief\": {\"0.9926598\": true}, \"source_hash\": {\"-4971714610315652007\": true}, \"source_api\": {\"reach\": true}}, \"target\": 10, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Viral infections and pro-inflammatory cytokines induce expression of NOS2 and defective NOS2 expression is responsible for increased viral replication in cystic fibrosis, while overexpression of NOS2 provides protection against viral infection.150,151 Recently we demonstrated that COPD airway epithelial cells show a trend in decreased expression of NOS2 and this was associated with impaired clearance of rhinovirus.123 Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2.\", \"source\": 445, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-3536476931991917\": true}, \"uuid\": {\"9bd0872a-bcc8-4f2e-9db8-99ec8766b943\": true}, \"belief\": {\"0.9926598\": true}, \"source_hash\": {\"-4971714610315652007\": true}, \"source_api\": {\"reach\": true}}, \"target\": 10, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"XREF_BIBR Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2.\", \"source\": 445, \"key\": 5}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20809107549003530\": true}, \"uuid\": {\"d928442c-5ffd-43b5-b3c7-bbf8d1b6281a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4179698419426217256\": true}, \"source_api\": {\"reach\": true}}, \"target\": 528, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30293076\"}, \"evidence\": \"With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37].\", \"source\": 366, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-15507406493871859\": true}, \"uuid\": {\"274fc0b8-a5ed-4a1d-a632-511216b1d941\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4179698419426217256\": true}, \"source_api\": {\"reach\": true}}, \"target\": 50, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30293076\"}, \"evidence\": \"With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37].\", \"source\": 366, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3927848590111677\": true}, \"uuid\": {\"43eb0717-46a0-40ac-b7de-52410ec8c6fc\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6850062552220512619\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"17398101\"}, \"evidence\": \"Similarly, SP-A and SP-D can both interact with influenza to prevent the association of the HA with its receptor.\", \"source\": 366, \"target\": 224, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21772423473804843\": true}, \"uuid\": {\"83683acd-8ec3-4e60-b932-46a21a5b609f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6136138729970713850\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JH , and YY contributed to data analysis .\", \"source\": 89, \"target\": 191, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21772423473804843\": true}, \"uuid\": {\"83683acd-8ec3-4e60-b932-46a21a5b609f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"6136138729970713850\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JH , and YY contributed to data analysis .\", \"source\": 89, \"target\": 191, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-22324991321942579\": true}, \"uuid\": {\"8aded083-7900-47a3-a6af-8701532307cb\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5602704121651221012\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JL , and YZ contributed to data collection .\", \"source\": 89, \"target\": 290, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-22324991321942579\": true}, \"uuid\": {\"8aded083-7900-47a3-a6af-8701532307cb\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5602704121651221012\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JL , and YZ contributed to data collection .\", \"source\": 89, \"target\": 290, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"18270822532801512\": true}, \"uuid\": {\"2db1849d-037a-4c4c-bb15-d9c71f14c2ef\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"target\": 54, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 61, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-31570795512537743\": true}, \"uuid\": {\"73a668c5-18eb-466f-acbf-dba58dedd367\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 61, \"target\": 145, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33392233279305606\": true}, \"uuid\": {\"893ce1ef-3b8f-43f9-b96c-53e6edc343c4\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"target\": 467, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 61, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-25930590018038126\": true}, \"uuid\": {\"fff5a203-267d-4b16-8cd0-66aa8511c4e0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-371794646661860878\": true}, \"source_api\": {\"reach\": true}}, \"target\": 97, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation.\", \"source\": 61, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"2907457820477088\": true}, \"uuid\": {\"ee7ecdc8-36f9-4c02-989d-ee50b365bb5a\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1969130739583424762\": true}, \"source_api\": {\"trips\": true}}, \"target\": 115, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32942131\"}, \"evidence\": \"Induction of ketosis as a potential therapeutic option to limit hyperglycemia and prevent cytokine storm in COVID-19.\", \"source\": 240, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28841584086462317\": true}, \"uuid\": {\"678a847b-2107-4c31-8ba8-50d61e9ba1ef\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1969130739583424762\": true}, \"source_api\": {\"trips\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32942131\"}, \"evidence\": \"Induction of ketosis as a potential therapeutic option to limit hyperglycemia and prevent cytokine storm in COVID-19.\", \"source\": 240, \"target\": 228, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3362765041691094\": true}, \"uuid\": {\"5ff403ca-0446-469b-bf45-bb22e21d2240\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1783051300808249252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold.\", \"source\": 398, \"target\": 127, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"15125720946431384\": true}, \"uuid\": {\"49f362c3-d10b-401e-bbbf-0069dbff1925\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1783051300808249252\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold.\", \"source\": 399, \"target\": 127, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-18092176590427630\": true}, \"uuid\": {\"61370fdb-92c8-481f-aba2-83dbf532e1e0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8783387814893972482\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32469464\"}, \"evidence\": \"An animal model shows that the complication of type 2 diabetes causes an immune disorder and enhances disease severity following MERS-CoV infection.\", \"source\": 213, \"target\": 215, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20710859493215399\": true}, \"uuid\": {\"77913254-d62b-4845-9f6c-0bc85e1142af\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8783387814893972482\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32469464\"}, \"evidence\": \"An animal model shows that the complication of type 2 diabetes causes an immune disorder and enhances disease severity following MERS-CoV infection.\", \"source\": 213, \"target\": 236, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5412341228226538\": true}, \"uuid\": {\"4fc467c4-d7a0-4052-8860-6dc57f135ca7\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 220, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5412341228226538\": true}, \"uuid\": {\"4fc467c4-d7a0-4052-8860-6dc57f135ca7\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 220, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5412341228226538\": true}, \"uuid\": {\"4fc467c4-d7a0-4052-8860-6dc57f135ca7\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"target\": 420, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 220, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34368164446934567\": true}, \"uuid\": {\"87fe8257-7166-418f-8e6a-e74c3cb963d4\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 220, \"target\": 139, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34368164446934567\": true}, \"uuid\": {\"87fe8257-7166-418f-8e6a-e74c3cb963d4\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion .\", \"source\": 220, \"target\": 139, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34368164446934567\": true}, \"uuid\": {\"87fe8257-7166-418f-8e6a-e74c3cb963d4\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-4579454018606615742\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"think\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme .\", \"source\": 220, \"target\": 139, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28267197098550779\": true}, \"uuid\": {\"990ce249-8feb-41a4-bbcd-1f2cc2ec9269\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3986287486659339700\": true}, \"source_api\": {\"reach\": true}}, \"target\": 21, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 24, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-32788075953327744\": true}, \"uuid\": {\"66218b8c-8fdb-46df-951c-3f3324eca648\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3986287486659339700\": true}, \"source_api\": {\"reach\": true}}, \"target\": 29, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 24, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28642861022583415\": true}, \"uuid\": {\"42176161-e587-4567-876b-c4b4f6110930\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3986287486659339700\": true}, \"source_api\": {\"reach\": true}}, \"target\": 79, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin.\", \"source\": 24, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"6809002446421610\": true}, \"uuid\": {\"bb928eae-b4ec-4585-a828-d0632ed15dce\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7739523029985678237\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 353, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"137 , 138 Epithelial cells produce protease inhibitors , such as secretory leukoprotease inhibitor ( SLPI ) , elastase inhibitor , alpha1-antiprotease , and antichymotrypsin .\", \"source\": 218, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-19100002306802130\": true}, \"uuid\": {\"5473d3df-39e7-4f0a-9b8b-f5ed9ae90fa9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5818360035712509125\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Tamoxifen, an estrogen receptor agonist, inhibits SARS-CoV-2 in vitro (93) .\", \"source\": 67, \"target\": 194, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13205489580662193\": true}, \"uuid\": {\"f3548f65-ce6a-4eae-99f4-58f714d15bcc\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4407569707119757397\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Levels of vWF, fibrinogen, and factor VIII were elevated in COVID-19 (83) .\", \"source\": 391, \"target\": 193, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"6400482992148365\": true}, \"uuid\": {\"62869303-f80a-446f-b46f-89329d88633e\": true}, \"belief\": {\"0.9995764999635395\": true}, \"source_hash\": {\"-1314222136921279188\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 466, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Additional FDA approved drugs are known to disrupt RAGE signaling, thus suggesting that a drug repositioning effort could provide a fast track for controlling at least some of the negative outcomes associated with severe ARDS.However, it should be noted that chloroquine, currently used in COVID-19 therapy, inhibits HMGB1 release especially in DM patients (43).\", \"source\": 63, \"key\": 24}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-14063472054452187\": true}, \"uuid\": {\"41c14beb-ee02-4cad-9162-096fdc015ab0\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-1290188827113080726\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32179126\"}, \"evidence\": \"Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications .\", \"source\": 58, \"target\": 203, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5220911447638017\": true}, \"uuid\": {\"b7a67922-9998-4ffb-9068-2fbbf5b9a03e\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-1290188827113080726\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32179126\"}, \"evidence\": \"Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications .\", \"source\": 58, \"target\": 302, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"34664975971577453\": true}, \"uuid\": {\"b59109d9-3459-4ae5-8bcd-0e2390ef9b4b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2233707475720190031\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"These events may ultimately lead to pyroptosis , which determines the release of further mediators from dying cells and the propagation of the initial damage ( 10 , 22 ) .\", \"source\": 118, \"target\": 180, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"34664975971577453\": true}, \"uuid\": {\"b59109d9-3459-4ae5-8bcd-0e2390ef9b4b\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2233707475720190031\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"These events may ultimately lead to pyroptosis , which determines the release of further mediators from dying cells and the propagation of the initial damage ( 10 , 22 ) .\", \"source\": 118, \"target\": 180, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3021910274983909\": true}, \"uuid\": {\"aaef0242-e91a-489e-8ada-25bca65cad41\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1609765601049195028\": true}, \"source_api\": {\"reach\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1 and ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2 mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1 generated ANGII.\", \"source\": 556, \"key\": 13}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-3021910274983909\": true}, \"uuid\": {\"aaef0242-e91a-489e-8ada-25bca65cad41\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1609765601049195028\": true}, \"source_api\": {\"reach\": true}}, \"target\": 106, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII.\", \"source\": 556, \"key\": 14}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"24677734108618849\": true}, \"uuid\": {\"3a74b223-59b0-495c-817f-37a9701187b8\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1609765601049195028\": true}, \"source_api\": {\"reach\": true}}, \"target\": 463, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32333398\"}, \"evidence\": \"ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII.\", \"source\": 556, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30333974471926193\": true}, \"uuid\": {\"e5a71d55-30da-4c98-887d-0bf4d2851092\": true}, \"belief\": {\"0.9499940951\": true}, \"source_hash\": {\"8521836595300522138\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) .\", \"source\": 217, \"target\": 174, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30333974471926193\": true}, \"uuid\": {\"e5a71d55-30da-4c98-887d-0bf4d2851092\": true}, \"belief\": {\"0.9499940951\": true}, \"source_hash\": {\"8521836595300522138\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) .\", \"source\": 217, \"target\": 174, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11112718255224262\": true}, \"uuid\": {\"725b444a-bd50-449a-a247-1440540dff59\": true}, \"belief\": {\"0.949980317\": true}, \"source_hash\": {\"2376335047144079915\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) .\", \"source\": 217, \"target\": 136, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-11112718255224262\": true}, \"uuid\": {\"725b444a-bd50-449a-a247-1440540dff59\": true}, \"belief\": {\"0.949980317\": true}, \"source_hash\": {\"2376335047144079915\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) .\", \"source\": 217, \"target\": 136, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-21130168753563612\": true}, \"uuid\": {\"af1bb8b9-2dbe-4700-9bb5-3ed5053f0b28\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4630435875479959596\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 436, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"135 Herr et al. showed that hBD2 is significantly reduced in the pharyngeal wash and sputum of current or former smokers compared with non-smokers , and exposure of airway epithelium to cigarette smoke in vitro inhibited induction of HBD2 by bacteria .\", \"source\": 200, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"25744859194518165\": true}, \"uuid\": {\"7cda26d2-0aa1-46d5-977a-421544c4bd8d\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-2828302159359323754\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"whether\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33301246\"}, \"evidence\": \"These data do not address whether vaccination prevents asymptomatic infection ; a serologic end point that can detect a history of infection regardless of whether symptoms were present ( SARS-CoV-2 N-binding antibody ) will be reported later .\", \"source\": 272, \"target\": 317, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"15196386433954957\": true}, \"uuid\": {\"2dbfbe1f-3b1c-4991-a657-0b60bfbf581f\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-8973405960339778804\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"17398101\"}, \"evidence\": \"Second , changes in glycosylation can affect interaction with receptors and cause a virus to be more recognizable by the innate factors of host immune cells and less recognized by antibody , thus impacting viral replication and infectivity .\", \"source\": 178, \"target\": 276, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7621178187117257\": true}, \"uuid\": {\"47d71ac2-5741-49ab-a143-9952ed924285\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4230310607035009562\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"17398101\"}, \"evidence\": \"Interestingly , in the case of Nipah virus , glycosylation has a dual role : glycosylation leads to enhanced resistance to antibody neutralization but alternatively causes a reduction in membrane fusion and viral entry [ 42 ] .\", \"source\": 178, \"target\": 177, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"10308807512605939\": true}, \"uuid\": {\"799900ec-8503-4380-9335-03b4d6ba344b\": true}, \"belief\": {\"0.34387427664975\": true}, \"source_hash\": {\"5373432414984234716\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"suggest\": true}, \"negated\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos).\", \"source\": 446, \"target\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-9638854656330537\": true}, \"uuid\": {\"d9940fad-a871-4296-93b6-ba8fdb18b9d6\": true}, \"belief\": {\"0.949271\": true}, \"source_hash\": {\"6063618837213512316\": true}, \"source_api\": {\"reach\": true}}, \"target\": 533, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos).\", \"source\": 446, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-17768287840912038\": true}, \"uuid\": {\"87d08527-9f69-4e56-908c-d9f8e1a5feb9\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"-3378092635838334526\": true}, \"source_api\": {\"reach\": true}}, \"target\": 392, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199].\", \"source\": 545, \"key\": 7}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35967257165841899\": true}, \"uuid\": {\"fac84e64-cb9a-45d2-9595-34cf2fa74dcb\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-3378092635838334526\": true}, \"source_api\": {\"reach\": true}}, \"target\": 418, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199].\", \"source\": 545, \"key\": 3}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"35010703892452780\": true}, \"uuid\": {\"96709541-e4f4-4e71-ab32-cc4e629a577a\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-3378092635838334526\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199].\", \"source\": 392, \"target\": 159, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25346947633690088\": true}, \"uuid\": {\"4be34118-4da7-4166-afbf-87d2df65723e\": true}, \"belief\": {\"0.9998749998771973\": true}, \"source_hash\": {\"-7542043233753122399\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"might\": true, \"hypothesize\": true}}, \"target\": 418, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"It will be remembered that TRIM25 is required to ubiqutinate and activate RIG-I leading to subsequent IFN production.\", \"source\": 392, \"key\": 10}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"25346947633690088\": true}, \"uuid\": {\"4be34118-4da7-4166-afbf-87d2df65723e\": true}, \"belief\": {\"0.9998749998771973\": true}, \"source_hash\": {\"-7542043233753122399\": true}, \"source_api\": {\"sparser\": true}, \"hedgings\": {\"might\": true, \"hypothesize\": true}}, \"target\": 418, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199].\", \"source\": 392, \"key\": 21}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14443741616224222\": true}, \"uuid\": {\"ed07aa8d-d625-4f8f-9a7a-293cc6e85b54\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-1110801508074742736\": true}, \"source_api\": {\"reach\": true}}, \"target\": 534, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"In addition, there are N-glycans within the MLD that may partially contribute to the phenotypes observed.The GP 2 subunit contains two N-glycosylation sites conserved among all five EBOV species (Jeffers et al. 2002; Ritchie et al. 2010 ).\", \"source\": 395, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33101450588160698\": true}, \"uuid\": {\"0d251eb1-cb64-4b8d-aaf1-a51f93b1ed3b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5773093601496436984\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"These comorbidities accelerate atherosclerosis , which leads to cardiovascular complications and may increase the severity and mortality of COVID-19 .\", \"source\": 280, \"target\": 305, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5003083026019430\": true}, \"uuid\": {\"6acbfb0b-91bc-4b67-a982-548c5a01d77c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1362320550829124312\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse .\", \"source\": 256, \"target\": 245, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5003083026019430\": true}, \"uuid\": {\"6acbfb0b-91bc-4b67-a982-548c5a01d77c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1362320550829124312\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse .\", \"source\": 256, \"target\": 245, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5003083026019430\": true}, \"uuid\": {\"6acbfb0b-91bc-4b67-a982-548c5a01d77c\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1362320550829124312\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse .\", \"source\": 256, \"target\": 245, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"29065685132039653\": true}, \"uuid\": {\"543d12d9-a685-4abe-9f8c-551a40a73c64\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"8422614985902208673\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Treatment of the Golgi transport blocker monensin inhibited the transport of MHV S protein from trans-Golgi network to the cell surface [21].\", \"source\": 46, \"target\": 132, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21958803190794201\": true}, \"uuid\": {\"4334da7b-5fd8-4d2e-9a13-dbfa44d0f5d9\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"307867701868199899\": true}, \"source_api\": {\"reach\": true}}, \"target\": 547, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Treatment of the Golgi transport blocker monensin inhibited the transport of MHV S protein from trans-Golgi network to the cell surface [21].\", \"source\": 46, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-20008860349584904\": true}, \"uuid\": {\"dc40d5bf-c190-4ea8-9970-ebd7cfc73572\": true}, \"belief\": {\"0.949999468559\": true}, \"source_hash\": {\"-1568063520140897209\": true}, \"source_api\": {\"sparser\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"However, recombinant IBV harboring alanine substitutions at all four putative phosphorylation sites (S190A/S192A/T378A/S379A) could still be recovered and grew at a similar growth rate as wild-type IBV, suggesting that ATR-dependent phosphorylation of N protein is not essential for IBV replication in vitro [119].\", \"source\": 513, \"target\": 549, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"14576156541765320\": true}, \"uuid\": {\"ead76e5f-dba3-4843-a977-d0ff1f72243f\": true}, \"belief\": {\"0.9999395\": true}, \"source_hash\": {\"-6295881615163116053\": true}, \"source_api\": {\"ctd\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24124542\"}, \"evidence\": \"The forskolin induced CFTR dependent inhibition of Ps.\", \"source\": 68, \"key\": 41}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"14576156541765320\": true}, \"uuid\": {\"ead76e5f-dba3-4843-a977-d0ff1f72243f\": true}, \"belief\": {\"0.9999395\": true}, \"source_hash\": {\"-6295881615163116053\": true}, \"source_api\": {\"ctd\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24124542\"}, \"evidence\": \"The forskolin induced CFTR dependent inhibition of Ps.\", \"source\": 68, \"key\": 58}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"14576156541765320\": true}, \"uuid\": {\"ead76e5f-dba3-4843-a977-d0ff1f72243f\": true}, \"belief\": {\"0.9999395\": true}, \"source_hash\": {\"-6295881615163116053\": true}, \"source_api\": {\"ctd\": true}}, \"target\": 417, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24124542\"}, \"evidence\": \"The forskolin induced CFTR dependent inhibition of Ps.\", \"source\": 68, \"key\": 73}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-20665272733289255\": true}, \"uuid\": {\"3f356565-fa51-41ee-aaaa-811e06ac38a0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5351206591375827261\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 382, \"target\": 162, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27972629466844162\": true}, \"uuid\": {\"2ce7d0e0-5684-45fb-97aa-5f24a088fb81\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1399783371464040255\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 500, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting.58 Following eNOS sulfate synthesis , sulfotransferase enzyme SULT2B1b catalyses cholesterol sulfurylation , producing Ch-S .\", \"source\": 277, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27972629466844162\": true}, \"uuid\": {\"2ce7d0e0-5684-45fb-97aa-5f24a088fb81\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1399783371464040255\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 500, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting .\", \"source\": 277, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27972629466844162\": true}, \"uuid\": {\"2ce7d0e0-5684-45fb-97aa-5f24a088fb81\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"1399783371464040255\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 500, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32938758\"}, \"evidence\": \"57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting.58 Following eNOS sulfate synthesis , sulfotransferase enzyme SULT2B1b catalyses cholesterol sulfurylation , producing Ch-S .\", \"source\": 277, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28206943042885155\": true}, \"uuid\": {\"4c30ac2b-e3d5-45cd-b925-8c830270ed4d\": true}, \"belief\": {\"0.9499999987085984\": true}, \"source_hash\": {\"332208803678294331\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"believe\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Therefore , this review will hopefully spark further interest and research , and importantly raise awareness of the importance of anticoagulation in patients with COVID-19 for both the prevention and treatment of thrombosis .\", \"source\": 281, \"target\": 260, \"key\": 6}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-34261086878398747\": true}, \"uuid\": {\"a0f5bf54-8263-49a3-bebe-195088e22a47\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-299418866202643888\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 3, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32660650\"}, \"evidence\": \"Restoring the balance between the RAS and ACE2 / angiotensin - ( 1-7 ) / MAS may help attenuate organ injuries .\", \"source\": 161, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6449131215149637\": true}, \"uuid\": {\"78a541ae-2ca4-4339-b1ae-1d7dddb6cedd\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"762730482492435574\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Moreover, inhibition of GSK-3 by kenpaullone significantly reduced the phosphorylation level of N protein, as well as the supernatant virus titer and cytopathic effects on VeroE6 cell-infected SARS-CoV [131].\", \"source\": 342, \"target\": 545, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-26038891269165159\": true}, \"uuid\": {\"6c730c13-c323-4ce6-9b92-d165c1c490cb\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-8103528893389759498\": true}, \"source_api\": {\"sparser\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"In fact, a recent study showed that phosphorylation of the MHV-JHM N protein by GSK-3 allowed the recruitment of RNA helicase DDX1 to facilitate template read-through, enabling the synthesis of genomic RNA and longer sgRNAs [132].\", \"source\": 342, \"target\": 549, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-21390841818370192\": true}, \"uuid\": {\"9d596e14-e3ae-407e-906e-7e37795b40b9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1361735882851267448\": true}, \"source_api\": {\"reach\": true}}, \"target\": 20, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32369736\"}, \"evidence\": \"Elevated BG level has been reported to increase the glucose concentration in airway epithelial secretion (Philips et al., 2003), which may disrupt the defensive capacity of airway epithelia.\", \"source\": 550, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"11816493708983206\": true}, \"uuid\": {\"80566574-cb0e-47ef-882e-e0c6e84fd585\": true}, \"belief\": {\"1\": true}, \"source_hash\": {\"4558541420499315309\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"Pitavastatin has been shown to reduce IL-6 and IL-8 secretions and mRNA expression of IL-6, IL-8, and granulocyte macrophage colony stimulating factor (GM-CSF) in LPS stimulated human bronchial epithelium cells.\", \"source\": 57, \"target\": 478, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"5751197602542423\": true}, \"uuid\": {\"acd78ed3-0542-4b8e-b1c6-2592a25dafb8\": true}, \"belief\": {\"0.9497812999999999\": true}, \"source_hash\": {\"5517519744671232077\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"target\": 404, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"The activation of platelet TLR-7 by viral RNA can evoke a range of functional responses which support thrombus formation , including platelet degranulation , the formation of platelet-leukocyte aggregates and platelet C3 release which in turn stimulates NETosis .\", \"source\": 265, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14261276548036834\": true}, \"uuid\": {\"0f833644-168f-4513-a136-5dbf99b0b04e\": true}, \"belief\": {\"0.33249999999999996\": true}, \"source_hash\": {\"-115528823111332437\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"whether\": true}, \"negated\": {\"true\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33378609\"}, \"evidence\": \"Thirty participants in the trial had severe Covid-19 ; all 30 were in the placebo group ( indicating vaccine efficacy of 100 % [ 95 % CI , could not be estimated to 1.0 ] ) , and one death among these participants was attributed to Covid-19 ( Table S16 ) .\", \"source\": 107, \"target\": 210, \"key\": 42}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-24528185997982447\": true}, \"uuid\": {\"804e0fa1-3dee-45be-b17a-56ff9d2cf4a3\": true}, \"belief\": {\"0.9816495\": true}, \"source_hash\": {\"-3009790864098874584\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"suggest\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32483488\"}, \"evidence\": \"Angiotensin-converting enzyme ( ACE ) inhibitor and angiotensin-receptor blocker ( ARB ) have been hypothetically suggested to exacerbate COVID-19 due to increase in angiotensin II level [ 31 ] .\", \"source\": 301, \"target\": 193, \"key\": 5}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-34991513043650165\": true}, \"uuid\": {\"3319f9ad-807b-44ec-8f15-bf20d3b6d5f0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-163774356692421132\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"ARB therapy did however reduce albuminuria in Akita-ACE2/y mice (Wong et al., 2007).\", \"source\": 301, \"target\": 196, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6846978724375486\": true}, \"uuid\": {\"ee27cebc-d4e1-451c-b535-8ddda5fc401d\": true}, \"belief\": {\"0.94999995217031\": true}, \"source_hash\": {\"717354136665718223\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) .\", \"source\": 62, \"target\": 223, \"key\": 7}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6846978724375486\": true}, \"uuid\": {\"ee27cebc-d4e1-451c-b535-8ddda5fc401d\": true}, \"belief\": {\"0.94999995217031\": true}, \"source_hash\": {\"717354136665718223\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) .\", \"source\": 62, \"target\": 223, \"key\": 8}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"6846978724375486\": true}, \"uuid\": {\"ee27cebc-d4e1-451c-b535-8ddda5fc401d\": true}, \"belief\": {\"0.94999995217031\": true}, \"source_hash\": {\"717354136665718223\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32850890\"}, \"evidence\": \"Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) .\", \"source\": 62, \"target\": 223, \"key\": 9}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"22709097547050960\": true}, \"uuid\": {\"e152be93-c7fb-4edc-a83f-36651b1f5939\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-213132849019644341\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 107, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33301246\"}, \"evidence\": \"The results demonstrate that Covid-19 can be prevented by immunization , provide proof of concept that RNA-based vaccines are a promising new approach for protecting humans against infectious diseases , and demonstrate the speed with which an RNA-based vaccine can be developed with a sufficient investment of resources .\", \"source\": 235, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13259166555554798\": true}, \"uuid\": {\"bbbef77e-f86b-4cf4-be73-fe4dbdee6519\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"6136138729970713850\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JH , and YY contributed to data analysis .\", \"source\": 464, \"target\": 191, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13259166555554798\": true}, \"uuid\": {\"bbbef77e-f86b-4cf4-be73-fe4dbdee6519\": true}, \"belief\": {\"0.94757\": true}, \"source_hash\": {\"6136138729970713850\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33042873\"}, \"evidence\": \"DL , JH , and YY contributed to data analysis .\", \"source\": 464, \"target\": 191, \"key\": 4}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"34866089693579067\": true}, \"uuid\": {\"ffca9f0b-ee7c-46b3-8666-ddd773bb328d\": true}, \"belief\": {\"0.9615\": true}, \"source_hash\": {\"-3300326278658408342\": true}, \"source_api\": {\"ctd\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25136585\"}, \"evidence\": \"Even in participants without hyperlipidemia, rosuvastatin has been shown to reduce CRP levels in association with a lower incidence of cardiovascular events [XREF_BIBR].\", \"source\": 66, \"target\": 431, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1382415908259021\": true}, \"uuid\": {\"d3d36db8-a24f-4f08-b8e8-003bacf35638\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3267835160193813294\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"To investigate the antiinflammatory effects of statins, a phase 2 RCT is evaluating a combination of the Janus kinase (JAK) inhibitor, ruxolitinib and simvastatin in the prevention and treatment of respiratory failure in COVID-19 (ClinicalTrials.gov Identifier: NCT04348695).\", \"source\": 87, \"target\": 193, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-4636467573837435\": true}, \"uuid\": {\"de4ef471-d6eb-4ae2-b4d8-1ddfc77d608e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"3017636662794832179\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"For instance , carbohydrate binding agents that directly interact with glycans on the virion surface may be able to suppress virus attachment and entry .\", \"source\": 157, \"target\": 308, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-8174170615193612\": true}, \"uuid\": {\"b7b72016-f072-4946-9cc3-4f3112344d37\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1845211531675091391\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] .\", \"source\": 310, \"target\": 186, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33227414877785926\": true}, \"uuid\": {\"26143518-d205-4e32-93df-cfc9a206da5f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"1197906976600697748\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18948167\"}, \"evidence\": \"Additionally, enalapril prevents fibrosis and oxidative stress in the liver, heart and kidney of rats with T1DM (de Cavanagh et al., 2001).\", \"source\": 73, \"target\": 221, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-22103414400134971\": true}, \"uuid\": {\"4efa8adb-e04f-4b53-8c33-8fa44c87e0f8\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"7735931485475816678\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"Infection of macrophages or epithelial cells in the in vitro setting pointed to the latter scenario [52, 53], and transcription analysis showed that H5N1 was more potent than seasonal influenza in the production of inflammatory cytokines and chemokines.Once influenza encounters the cell there are pattern recognition receptors (PRR) that are stimulated to initiate an antiviral response leading to the production of interferon and cytokines and chemokines.\", \"source\": 542, \"target\": 176, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7925807987531959\": true}, \"uuid\": {\"c46da021-03cd-42f7-849e-a600d91e5c40\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2172757642327609912\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression.\", \"source\": 542, \"target\": 455, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-7925807987531959\": true}, \"uuid\": {\"c46da021-03cd-42f7-849e-a600d91e5c40\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2172757642327609912\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression.\", \"source\": 542, \"target\": 455, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-12031750307563056\": true}, \"uuid\": {\"26bf9f60-3c2e-4500-886b-5664cd8e1874\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6111275131335036851\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010).\", \"source\": 323, \"target\": 138, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"26919385615205406\": true}, \"uuid\": {\"70b547fa-f9f9-4ee1-8449-60c543ddfac0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6111275131335036851\": true}, \"source_api\": {\"reach\": true}}, \"target\": 489, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010).\", \"source\": 323, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-35637860807647825\": true}, \"uuid\": {\"afd6d1e3-dab4-4070-a27e-af644855035b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6111275131335036851\": true}, \"source_api\": {\"reach\": true}}, \"target\": 551, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24188132\"}, \"evidence\": \"Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010).\", \"source\": 323, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-26663721519210703\": true}, \"uuid\": {\"6f3aabb8-67e1-407c-9278-8e3f13acdc47\": true}, \"belief\": {\"0.89\": true}, \"source_hash\": {\"2044666902448916691\": true}, \"source_api\": {\"ctd\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32220112\"}, \"evidence\": \"No evidence text.\", \"source\": 225, \"target\": 193, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-33840723800486838\": true}, \"uuid\": {\"4ce22e1f-f328-43ab-9c3a-1173da59a63d\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"7337627505938353707\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}}, \"target\": 389, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"Studies have shown that tocilizumab , an IL-6 receptor antibody , could reverse CRS in the setting of chimeric antigen receptor T-cell therapy for cancer [ 36 ] .\", \"source\": 85, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-12480822168826323\": true}, \"uuid\": {\"9f579c11-43a4-4da4-87f7-83e41823748e\": true}, \"belief\": {\"0.34973049973049997\": true}, \"source_hash\": {\"8921468446930613081\": true}, \"source_api\": {\"reach\": true}, \"hedgings\": {\"can\": true, \"indicate\": true}, \"negated\": {\"true\": true}}, \"target\": 449, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32220035\"}, \"evidence\": \"36, 37 Angiotensin I receptor blockers such as losartan are thought to reduce lung injury in SARS cases and can potentially also be used in treatment of COVID-19.\", \"source\": 86, \"key\": 74}, {\"relation\": \"directlyDecreases\", \"annotations\": {\"stmt_hash\": {\"-25097400324426231\": true}, \"uuid\": {\"638b9a64-2c9e-4397-aef0-fc0115993e35\": true}, \"belief\": {\"0.9999999697464276\": true}, \"source_hash\": {\"6287590896332340847\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}, \"source_id\": {\"patent:US-9321791-B2\": true}}, \"target\": 442, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29459239\"}, \"evidence\": \"XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status.\", \"source\": 90, \"key\": 48}, {\"relation\": \"directlyDecreases\", \"annotations\": {\"stmt_hash\": {\"-25097400324426231\": true}, \"uuid\": {\"638b9a64-2c9e-4397-aef0-fc0115993e35\": true}, \"belief\": {\"0.9999999697464276\": true}, \"source_hash\": {\"6287590896332340847\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}, \"source_id\": {\"patent:US-9321791-B2\": true}}, \"target\": 442, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29459239\"}, \"evidence\": \"XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status.\", \"source\": 90, \"key\": 87}, {\"relation\": \"directlyDecreases\", \"annotations\": {\"stmt_hash\": {\"-25097400324426231\": true}, \"uuid\": {\"638b9a64-2c9e-4397-aef0-fc0115993e35\": true}, \"belief\": {\"0.9999999697464276\": true}, \"source_hash\": {\"6287590896332340847\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"may\": true}, \"source_id\": {\"patent:US-9321791-B2\": true}}, \"target\": 442, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29459239\"}, \"evidence\": \"XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status.\", \"source\": 90, \"key\": 179}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35450807960795338\": true}, \"uuid\": {\"69ce0e5c-c220-414f-930d-7b52b6757f57\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"4042712712363746342\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Indeed, BEZ235 induced hemostasis in the PI3K/mTOR pathway via phosphorylation of p85 and 4E-BP1 and through reconstitution of metabolic status, which was already altered by the virus [9].\", \"source\": 91, \"target\": 141, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"35450807960795338\": true}, \"uuid\": {\"69ce0e5c-c220-414f-930d-7b52b6757f57\": true}, \"belief\": {\"0.8775000000000001\": true}, \"source_hash\": {\"4042712712363746342\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Indeed , BEZ235 induced hemostasis in the PI3K / mTOR pathway via phosphorylation of p85 and 4E-BP1 and through reconstitution of metabolic status , which was already altered by the virus [ 9 ] .\", \"source\": 91, \"target\": 141, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-35309894497012588\": true}, \"uuid\": {\"22d575a7-f67c-421f-8c76-ad5b3cc2f9bc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6548232888517425846\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"should\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32244852\"}, \"evidence\": \"Governments should act to reduce smoking rates in all countries in accordance with the WHO Framework Convention on Tobacco Control ( FCTC ) , and initiate a stimulus package for health , as they have done for business , at the time of this outbreak / pandemic including all communicable pulmonary diseases and Covid-19 , as it is possible that smoking exacerbates contraction , transmission , and mortality .\", \"source\": 222, \"target\": 268, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-13911424478057792\": true}, \"uuid\": {\"a40f6534-886e-49a0-8cd2-22ff96119c81\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"7281677294829612940\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"On the other hand, PB1-F2 decreases superoxide anion dismutase 1 (SOD1) expression and consequently disrupts the ROS scavenging process [11] .\", \"source\": 95, \"target\": 375, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-29080007861724359\": true}, \"uuid\": {\"57e48969-0951-480e-b3a4-6e61a0e7cc5a\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6313171670767078965\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"The pandemic of COVID-19 has caused significant disruption in the intensive care unit ( ICU ) due to inadequate staffing , protective equipment , ventilators , and bed capacity.1 Demographics reveal that patients at greatest mortality risk are those with diabetes and / or hyperglycemia .\", \"source\": 190, \"target\": 239, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-29080007861724359\": true}, \"uuid\": {\"57e48969-0951-480e-b3a4-6e61a0e7cc5a\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-6313171670767078965\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33282306\"}, \"evidence\": \"The pandemic of COVID-19 has caused significant disruption in the intensive care unit ( ICU ) due to inadequate staffing , protective equipment , ventilators , and bed capacity .\", \"source\": 190, \"target\": 239, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-25147119890873656\": true}, \"uuid\": {\"2c531e6f-063e-439a-ad18-2a6e82258750\": true}, \"belief\": {\"0.94993439\": true}, \"source_hash\": {\"8696613803013549763\": true}, \"source_api\": {\"reach\": true}}, \"target\": 478, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"104 This is perhaps best highlighted by the observation that patients with severe COVID-19 in ICU exhibit a significant expansion of CD14 + CD16 + monocytes producing IL-6 compared with nonhospitalized patients with COVID-19.\", \"source\": 337, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21267018938271564\": true}, \"uuid\": {\"21dfad7c-350e-4d5d-b4fd-5cc51dffd39d\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-8452219553835199352\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 493, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"40 MUC5AC is mainly produced by goblet cells , while MUC5B is predominantly produced by submucosal glands .\", \"source\": 289, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-1805128238121826\": true}, \"uuid\": {\"3ef30303-e9d3-4adc-a7ac-4f6a8fb9c053\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"4013293364195278674\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Human respiratory syncytial virus ( HRSV , also referred to as RSV ) causes severe bronchiolitis in infants , accounting for over 50 % of childhood bronchiolitis cases requiring hospitalization ( Rodriguez and Ramilo 2014 ) .\", \"source\": 283, \"target\": 206, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13134416781647431\": true}, \"uuid\": {\"40a2188e-e5ac-4801-90f8-af404f56f376\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2743963733042845672\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 122, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Coxsackievirus and adenovirus bind to CAR , inducing disassembly of the tight junction and reduction in transepithelial resistance ( TER ) .109,110 Previously , we demonstrated that rhinovirus ( RV ) dissociates ZO-1 , occludin , and claudin 4 from the tight junctions and increases bacterial association and translocation across polarized airway epithelium.93 ,111,112 However , our studies have not shown binding of RV to tight junction proteins .\", \"source\": 264, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-13134416781647431\": true}, \"uuid\": {\"40a2188e-e5ac-4801-90f8-af404f56f376\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2743963733042845672\": true}, \"source_api\": {\"eidos\": true}}, \"target\": 122, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"109 , 110 Previously , we demonstrated that rhinovirus ( RV ) dissociates ZO-1 , occludin , and claudin 4 from the tight junctions and increases bacterial association and translocation across polarized airway epithelium .\", \"source\": 264, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-14248314059554096\": true}, \"uuid\": {\"ede2c5cf-960f-4c18-97b0-55868d8ab1a0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-7877761571099698863\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Activation of protease-activated receptor 2 (PAR2) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [95].\", \"source\": 297, \"target\": 423, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-14248314059554096\": true}, \"uuid\": {\"ede2c5cf-960f-4c18-97b0-55868d8ab1a0\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-7877761571099698863\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Activation of protease activated receptor 2 (PAR2) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [XREF_BIBR].\", \"source\": 297, \"target\": 423, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"18016847893913167\": true}, \"uuid\": {\"bc4bbe28-e93a-4159-8d05-2d9a03536735\": true}, \"belief\": {\"0.9499998405677\": true}, \"source_hash\": {\"-8153419324866742580\": true}, \"source_api\": {\"reach\": true}}, \"target\": 471, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \" PDCoV nsp5 mediates cleavage of NEMO and STAT2 to suppress production and signaling of type I interferon.This work is licensed under the Creative Commons Attribution4.0 License.\", \"source\": 444, \"key\": 9}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22420059528365430\": true}, \"uuid\": {\"72ca2b9e-170a-4f52-9d9b-c7a6fbe22f5e\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"7909617762717269569\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 494, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Additionally, the NS1 protein can effectively promote the activity of mTORC2, which, in turn, upregulates c-Myc through FoxO inhibition [71].\", \"source\": 360, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"22420059528365430\": true}, \"uuid\": {\"72ca2b9e-170a-4f52-9d9b-c7a6fbe22f5e\": true}, \"belief\": {\"0.97305\": true}, \"source_hash\": {\"7909617762717269569\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"can\": true}}, \"target\": 494, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"Additionally , the NS1 protein can effectively promote the activity of mTORC2 , which , in turn , upregulates c-Myc through FoxO inhibition [ 71 ] .\", \"source\": 360, \"key\": 3}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19526705585142670\": true}, \"uuid\": {\"5d72d558-ba52-43fb-a81a-73a73a51745f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2172811382311689917\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32760352\"}, \"evidence\": \"Both classes of proteins contribute to the inflammatory response by binding to RAGE and members of the Toll like Receptors (TLRs) family.In particular, RAGE activation triggers the NF-kappabeta-mediated transcription of inflammatory genes, together with the activation of the NRP3 inflammasome.\", \"source\": 327, \"target\": 136, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13275526954706515\": true}, \"uuid\": {\"aece87b2-3182-4734-8be6-9eacb357b92f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3963487972235200640\": true}, \"source_api\": {\"reach\": true}}, \"target\": 364, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32586214\"}, \"evidence\": \"Activated platelets express P-selectin which allows leukocyte tethering via leukocyte expressed PSGL-1 (P-selectin glycoprotein ligand 1), while GP-Ib facilitates stable adhesion via binding to the leukocyte integrin Mac-1.\", \"source\": 365, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-5327094738994391\": true}, \"uuid\": {\"74b2744d-5c7f-4ac9-b351-75e3b334f208\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4462452898363040944\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (XREF_FIG).\", \"source\": 322, \"target\": 224, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32813948971971089\": true}, \"uuid\": {\"36336354-fa52-47db-9a2f-4bac30eaaa4c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8140943301777416613\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2).\", \"source\": 322, \"target\": 224, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-5246401600242066\": true}, \"uuid\": {\"0f4af9fc-b82e-4054-b670-aba23519d14d\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2411814577979680718\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2, CCL5 and CD11b+ cells and decrease IFN secretion but did not affect IFN, CCL2, IL6 or TNF, thus showing that endothelial cells are able to actually modulate the inflammatory response [71].\", \"source\": 368, \"target\": 348, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-19850859663055951\": true}, \"uuid\": {\"111b449d-7bf5-4867-91c8-65c6d0c6b299\": true}, \"belief\": {\"0.9419\": true}, \"source_hash\": {\"-4914821488742287070\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"Overexpression of VHL promoted the ubiquitin-proteasomal degradation of SARS-CoV nsp16, while knockdown of VHL increased the protein stability of nsp16 [158].\", \"source\": 390, \"target\": 299, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"890844560215028\": true}, \"uuid\": {\"087c68c5-744b-49f2-9772-fe1529c65064\": true}, \"belief\": {\"0.951\": true}, \"source_hash\": {\"38143018277219563\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"believe\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161756\"}, \"evidence\": \"Therefore, the role of HPV infection in causing chronic inflammation in the airway and their association to exacerbations of chronic airway inflammatory diseases, which is scarcely explored, should be investigated in the future.\", \"source\": 295, \"target\": 136, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"9435965887699602\": true}, \"uuid\": {\"f4d3d64e-1dce-4831-b895-02ec634cef57\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5351206591375827261\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1.\", \"source\": 468, \"target\": 162, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-30958633233248363\": true}, \"uuid\": {\"327c210c-b572-41d0-acd8-7c2d60284bd0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-8110061693393049183\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"Restrepo C, Dong Q, Savov J, Mariencheck W, Wright JR: Surfactant  Protein D stimulates phagocytosis of Pseudomonas aeruginosa by alveolar macrophages.\", \"source\": 416, \"target\": 134, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-25048483566275298\": true}, \"uuid\": {\"f696858d-bf3e-48e4-8ab7-45b89cc6accf\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"2511776393631121736\": true}, \"source_api\": {\"sparser\": true}}, \"target\": 331, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32599589\"}, \"evidence\": \"C-C Chemokine Receptor Type 5 Inhibition.\", \"source\": 439, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-22662973410838563\": true}, \"uuid\": {\"de4bd2ea-03b8-4101-a50b-ef677cc3c84f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-3178391972037894589\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"30293076\"}, \"evidence\": \"L-ficolin has been reported to bind both HA and neuraminidase glycoproteins and inhibit IAV infection in vitro and in vivo [111].\", \"source\": 456, \"target\": 237, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"8928792035692038\": true}, \"uuid\": {\"8a88a431-eea7-4149-892e-9e4bc2a11b93\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-913302861349556363\": true}, \"source_api\": {\"reach\": true}}, \"target\": 527, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan.\", \"source\": 491, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-33877068116950423\": true}, \"uuid\": {\"7d693370-747a-4e7e-b10e-f2871f6504df\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-913302861349556363\": true}, \"source_api\": {\"reach\": true}}, \"target\": 81, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan.\", \"source\": 491, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14983002020757512\": true}, \"uuid\": {\"be6eccf0-0c02-4f35-8050-dca3f9242528\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5567892262531239159\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"A better understanding of SARS-CoV strategies to alter IFN- subtype expression may elucidate the mechanism underlying virusinduced airway disease.In summary, our data, together with previous results, demonstrate that SARS-CoV (HSR-1 strain) induces a weak IFN response in comparison with well-established IFN-inducing viruses, such as those for VSV and NDV.\", \"source\": 459, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"14983002020757512\": true}, \"uuid\": {\"be6eccf0-0c02-4f35-8050-dca3f9242528\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"5567892262531239159\": true}, \"source_api\": {\"reach\": true}}, \"target\": 348, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"18781076\"}, \"evidence\": \"A better understanding of SARS-CoV strategies to alter IFN- subtype expression may elucidate the mechanism underlying virus-induced airway disease.In summary, our data, together with previous results, demonstrate that SARS-CoV (HSR-1 strain) induces a weak IFN response in comparison with well-established IFN-inducing viruses, such as those for VSV and NDV.\", \"source\": 459, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"13706207459945397\": true}, \"uuid\": {\"7c7611ee-341e-41c8-a7d4-7037573096ed\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5390234243203537321\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Damage-associated molecular pattern molecules ( DAMPs ) are endogenous ' alarmin ' molecules that are released from dead or damaged cells , activating the immune system by interacting with several pattern recognition receptors , including toll-like receptors ( TLR2 , TLR4 , and TLR9 ) and the advanced glycosylation end-product specific receptor ( AGER , also known as RAGE ) [ 8, 9 ] .\", \"source\": 188, \"target\": 234, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"13706207459945397\": true}, \"uuid\": {\"7c7611ee-341e-41c8-a7d4-7037573096ed\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"-5390234243203537321\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Damage-associated molecular pattern molecules ( DAMPs ) are endogenous ' alarmin ' molecules that are released from dead or damaged cells , activating the immune system by interacting with several pattern recognition receptors , including toll-like receptors ( TLR2 , TLR4 , and TLR9 ) and the advanced glycosylation end-product specific receptor ( AGER , also known as RAGE ) [ 8, 9 ] .\", \"source\": 188, \"target\": 234, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28928715773946319\": true}, \"uuid\": {\"e4dcdc82-00ab-4185-911e-fdcad98a61a9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"6766993604298104410\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"However, it was found that recombinant MHV containing N-linked glycosylated M protein induced a higher level of type I interferon compared with the wild-type MHV with O-linked glycosylated M protein, whereas MHV with nonglycosylated M protein was a poor interferon inducer in cell culture [104].\", \"source\": 495, \"target\": 348, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"20451602843867971\": true}, \"uuid\": {\"5e62dad6-c6e1-4954-9754-3c6f46d7f2a9\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"6291547120202459770\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32340044\"}, \"evidence\": \"In a recent preprint study , hypokalemia due to renal potassium wasting has been reported with 93 % of the critically ill patients having hypokalemia independent of gastrointestinal symptoms 33 .\", \"source\": 102, \"target\": 233, \"key\": 1}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-4439428174266294\": true}, \"uuid\": {\"b1b7d63e-c074-43b8-a37e-2c2955686973\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-1607071984598129669\": true}, \"source_api\": {\"eidos\": true}, \"hedgings\": {\"could\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32161622\"}, \"evidence\": \"The results showed that this herbal medicine could ameliorate weight loss and inflammatory mediators in infected mice through the regulation of amino acid , fatty acid , and arachidonic acid pathways [ 162 ] .\", \"source\": 293, \"target\": 205, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"13572428196049986\": true}, \"uuid\": {\"a77dab16-857b-4fcd-b437-e0bbeae00b9b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-688625178005487957\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] .\", \"source\": 313, \"target\": 230, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"9441920834512646\": true}, \"uuid\": {\"e81ab4a5-d947-4b1e-b9d1-250b32c56fde\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-688625178005487957\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] .\", \"source\": 313, \"target\": 243, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-28188624193694466\": true}, \"uuid\": {\"b144b007-a50b-4389-9ab6-d12f775837d2\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-688625178005487957\": true}, \"source_api\": {\"eidos\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32737423\"}, \"evidence\": \"Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] .\", \"source\": 313, \"target\": 285, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"17106388087571273\": true}, \"uuid\": {\"5eef0fa2-a74a-4dcd-9008-446e345e9b0e\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-6637732114571725586\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32271993\"}, \"evidence\": \"Furthermore, the abnormal secretion of adipokines and cytokines like TNF-alpha and interferon characterise a chronic low-grade inflammation characteristic of abdominal obesity, which may impair immune response (7) and have effects on the lung parenchyma and bronchi (8) .\", \"source\": 313, \"target\": 137, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"12848805905391295\": true}, \"uuid\": {\"fd8f038e-7d1f-4680-905e-14b1b43e3ec0\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"414768866411928565\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) .\", \"source\": 326, \"target\": 174, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-19464320786564622\": true}, \"uuid\": {\"6fcb15b0-a97a-4b0f-a412-5759dbaedbdc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"414768866411928565\": true}, \"source_api\": {\"reach\": true}}, \"target\": 455, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) .\", \"source\": 326, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"21465816005005320\": true}, \"uuid\": {\"19032424-37a6-4821-a87a-38501757cba9\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5782454906889656812\": true}, \"source_api\": {\"reach\": true}}, \"target\": 552, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32463365\"}, \"evidence\": \"Figure 2 -- figure supplement 8. : UMAP based dimensionality reduction plot denoting cell types (A, C, E) and corresponding expression levels of Ace2, Dpp4, Anpep and Tmprss2 (B, D, F) from multiple studies : (A-B) adipose stromovascular fraction from mice treated with PBS or a beta-3 adrenergic agonist (Rajbhandari and Arneson, 2019), (C-D) murine adipose stromovascular fraction from Tabula Muris (Tabula Muris Consortium et al., 2018), and (E-F) adipose tissue from mice subjected to 24 hr of cold shock and processed for single nucleus RNA sequencing (Rajbhandari and Arneson, 2019).\", \"source\": 535, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"23106300922675558\": true}, \"uuid\": {\"1af81562-0f14-46f9-aa10-25a733a7fb14\": true}, \"belief\": {\"0.86\": true}, \"source_hash\": {\"2370953221708076363\": true}, \"source_api\": {\"reach\": true}}, \"target\": 44, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"In fact, co-expression of SARS-CoV nsp3, nsp4 and nsp6 induced DMV formation in the transfected cells [143].\", \"source\": 544, \"key\": 1}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"28708671625316299\": true}, \"uuid\": {\"3bf4f330-e398-4da0-8abd-598ddf6c7b90\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7811648439862175558\": true}, \"source_api\": {\"reach\": true}}, \"target\": 551, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"31121217\"}, \"evidence\": \"Once MBL binds to viral glycoproteins, it recruits mannose-binding lectin-associated serine proteases (MASPs), which trigger the lectin pathway of the complement system [[277], [278], [279]].\", \"source\": 554, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"32185721552648479\": true}, \"uuid\": {\"6dc33aec-f93a-4405-b105-3517a3c6a005\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5977079311855894879\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32844132\"}, \"evidence\": \"These changes can lead to decreased binding between ACE2 receptor and SARS-CoV-2 Receptor Binding Domain (RBD) due to steric hindrance by the addition of a large sized PO 4 3- molecule, thus reducing SARS-CoV-2 infectivity.\", \"source\": 103, \"target\": 194, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-8524788385108914\": true}, \"uuid\": {\"2058d340-f44f-4c29-91c1-37271931119f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7589493521730749552\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera.\", \"source\": 324, \"target\": 184, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"36002792007654028\": true}, \"uuid\": {\"a5794e30-51a1-44f9-af17-8179037e4daa\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-7589493521730749552\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"29878112\"}, \"evidence\": \"Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera.\", \"source\": 325, \"target\": 184, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"5604186360371221\": true}, \"uuid\": {\"feeef791-4e1f-427a-a38e-fe6540bd530c\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"809237932800621923\": true}, \"source_api\": {\"reach\": true}}, \"target\": 414, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands.\", \"source\": 330, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-19825856484832954\": true}, \"uuid\": {\"934fcf3b-7e29-4146-9af8-47ec9c94f817\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"809237932800621923\": true}, \"source_api\": {\"reach\": true}}, \"target\": 448, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24665407\"}, \"evidence\": \"Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands.\", \"source\": 330, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-2772781744968753\": true}, \"uuid\": {\"bfe8be6d-cfd1-4159-8694-45f67efba224\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-3551239808576757505\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"On the other hand, infection with MHV-S2FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 (Figure 9, blue bars).\", \"source\": 518, \"target\": 237, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-2772781744968753\": true}, \"uuid\": {\"bfe8be6d-cfd1-4159-8694-45f67efba224\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-3551239808576757505\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"On the other hand, infection with MHV-S29FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 ( Figure 9 , blue bars).\", \"source\": 518, \"target\": 237, \"key\": 2}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"33083963941842673\": true}, \"uuid\": {\"2be1dbcd-9cbb-4003-8c34-54da44d858e6\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-3551239808576757505\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"On the other hand, infection with MHV-S2FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 (Figure 9, blue bars).\", \"source\": 519, \"target\": 237, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"33083963941842673\": true}, \"uuid\": {\"2be1dbcd-9cbb-4003-8c34-54da44d858e6\": true}, \"belief\": {\"0.923\": true}, \"source_hash\": {\"-3551239808576757505\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"25375324\"}, \"evidence\": \"On the other hand, infection with MHV-S29FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 ( Figure 9 , blue bars).\", \"source\": 519, \"target\": 237, \"key\": 2}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-27062653562713331\": true}, \"uuid\": {\"c5e01b03-964f-4353-93be-7377a9c41b67\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"4478844896361947093\": true}, \"source_api\": {\"sparser\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"27921210\"}, \"evidence\": \"Overexpressing cldn7 in the porcine kidney, epithelial cell line LLC-PK1 resulted in an increased transepithelial electrical resistance via reducing paracellular Cl permeability while forming a Na+ pore.\", \"source\": 113, \"target\": 428, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"11820602656568957\": true}, \"uuid\": {\"7fd9fd31-48f6-4272-80ed-a3561d29b4fc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5003427663175487120\": true}, \"source_api\": {\"isi\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression.\", \"source\": 187, \"target\": 110, \"key\": 0}, {\"relation\": \"increases\", \"annotations\": {\"stmt_hash\": {\"-4960725745785770\": true}, \"uuid\": {\"151f25de-3707-4e98-8733-7d3040be5f3b\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5003427663175487120\": true}, \"source_api\": {\"isi\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression.\", \"source\": 187, \"target\": 393, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"24371585351225002\": true}, \"uuid\": {\"0b917b2b-8c1f-451a-b6ef-3b5a926fba70\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5416990426291207605\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2.\", \"source\": 100, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"7737107930805370\": true}, \"uuid\": {\"8f9603c0-c658-4572-881f-fde83899b835\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"5416990426291207605\": true}, \"source_api\": {\"reach\": true}}, \"target\": 393, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"33313438\"}, \"evidence\": \"Importantly, our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2.\", \"source\": 101, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-27549278664887479\": true}, \"uuid\": {\"b2013388-42f2-4a0d-8521-495b59252e58\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-5818360035712509125\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32603424\"}, \"evidence\": \"Tamoxifen, an estrogen receptor agonist, inhibits SARS-CoV-2 in vitro (93) .\", \"source\": 84, \"target\": 194, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"28534111997254488\": true}, \"uuid\": {\"ac956659-db03-4cd5-8dae-8f4da3c0a001\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"594790985643328734\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"11667972\"}, \"evidence\": \"SP-D also binds to the neuraminidase of IAV and inhibits neuraminidase enzyme activity [62].\", \"source\": 329, \"target\": 251, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-30123591237431449\": true}, \"uuid\": {\"6e3c132f-27a2-464f-8cd1-89d09f28204f\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"8140943301777416613\": true}, \"source_api\": {\"reach\": true}}, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"24265946\"}, \"evidence\": \"In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2).\", \"source\": 328, \"target\": 224, \"key\": 0}, {\"relation\": \"decreases\", \"annotations\": {\"stmt_hash\": {\"-32993470052003416\": true}, \"uuid\": {\"c58830f1-c964-457d-ab76-68f3acd474bc\": true}, \"belief\": {\"0.65\": true}, \"source_hash\": {\"-4530008574924260329\": true}, \"source_api\": {\"reach\": true}}, \"target\": 481, \"citation\": {\"namespace\": \"pubmed\", \"identifier\": \"32201497\"}, \"evidence\": \"The ubiquitinated 8b and 8ab may mediate rapid degradation of IRF3 and regulate host antiviral innate immunity [169].\", \"source\": 555, \"key\": 0}]}, \"#cpfgmhbuxsetqqzd\", 1000, 650, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#98DF8A', 'Composite': '#9467BD', 'Reaction': '#000000', 'Gene': '#FFBB78', 'Abundance': '#AEC7E8', 'RNA': '#FF9896'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(glucose_covid)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summarize EMMAA graph of bibliography"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------  ---------\n",
      "Name\n",
      "Version\n",
      "Number of Nodes        559\n",
      "Number of Namespaces    10\n",
      "Number of Edges        815\n",
      "Number of Annotations    9\n",
      "Number of Citations     72\n",
      "Number of Authors        0\n",
      "Network Density          0.00261\n",
      "Number of Components    65\n",
      "Number of Warnings       0\n",
      "---------------------  ---------\n",
      "\n",
      "Type (4)             Count  Example\n",
      "-----------------  -------  ----------------------------------------------------------------------\n",
      "Protein                228  p(HGNC:29168 ! RPGRIP1L)\n",
      "BiologicalProcess      199  bp(GO:\"GO:0030246\" ! \"carbohydrate binding\")\n",
      "Abundance              123  a(CHEBI:59520 ! N-glycan)\n",
      "Complex                  9  complex(bp(MESH:D065206 ! \"Extracellular Traps\"), p(HGNC:12726 ! VWF))\n",
      "\n",
      "Namespace (10)      Count  Example\n",
      "----------------  -------  ---------------------------------------------\n",
      "HGNC                  171  p(HGNC:882 ! ATR)\n",
      "MESH                  136  bp(MESH:D009765 ! Obesity)\n",
      "CHEBI                 104  a(CHEBI:2453 ! acyclovir)\n",
      "GO                     64  bp(GO:\"GO:0006956\" ! \"complement activation\")\n",
      "FPLX                   32  p(FPLX:NFkappaB ! NFkappaB)\n",
      "UP                     29  p(UP:P04608 ! P04608)\n",
      "TEXT                   23  a(TEXT:STORM)\n",
      "PFAM                    6  p(PFAM:PF03373 ! octapeptide)\n",
      "go                      5\n",
      "IP                      3  p(IP:IPR002959)\n",
      "\n",
      "Edge Type (26)                                   Count  Example\n",
      "---------------------------------------------  -------  -------------------------------------------------------------------------------------------------------------\n",
      "Protein increases Protein                          113  p(HGNC:1641 ! CD209) increases p(HGNC:9829 ! RAF1)\n",
      "Protein increases BiologicalProcess                 86  p(HGNC:5973 ! IL13) increases bp(MESH:D006984 ! Hypertrophy)\n",
      "BiologicalProcess increases BiologicalProcess       68  bp(MESH:D006973 ! Hypertension) increases bp(GO:\"GO:0006788\" ! \"heme oxidation\")\n",
      "Protein decreases Protein                           63  p(FPLX:NFkappaB ! NFkappaB) decreases p(HGNC:2047 ! CLDN5)\n",
      "BiologicalProcess increases Protein                 56  bp(MESH:D004912 ! Erythrocytes) increases p(HGNC:1968 ! LYST)\n",
      "BiologicalProcess decreases BiologicalProcess       55  bp(MESH:D006076 ! Government) decreases bp(MESH:D012907 ! Smoking)\n",
      "Abundance increases Protein                         54  a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:10803 ! SFTPD)\n",
      "Protein increases Abundance                         46  p(FPLX:AKT ! AKT) increases a(CHEBI:15422 ! ATP)\n",
      "Protein decreases BiologicalProcess                 45  p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "BiologicalProcess decreases Protein                 34  bp(MESH:D001249 ! Asthma) decreases p(HGNC:17257 ! SPDEF)\n",
      "Abundance increases BiologicalProcess               34  a(CHEBI:17234 ! glucose) increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Abundance increases Abundance                       29  a(CHEBI:29108 ! \"calcium(2+)\") increases a(CHEBI:17996 ! chloride)\n",
      "Abundance decreases Protein                         26  a(CHEBI:24996 ! lactate) decreases p(FPLX:Interferon ! Interferon)\n",
      "Abundance decreases BiologicalProcess               23  a(CHEBI:29699 ! tunicamycin) decreases bp(GO:\"GO:0019076\" ! \"viral release from host cell\")\n",
      "BiologicalProcess increases Abundance               22  bp(MESH:D066126 ! Cardiotoxicity) increases a(CHEBI:5801 ! hydroxychloroquine)\n",
      "Abundance decreases Abundance                       16  a(CHEBI:26523 ! \"reactive oxygen species\") decreases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n",
      "Protein decreases Abundance                         15  p(HGNC:10803 ! SFTPD) decreases a(TEXT:\"kidney injury\")\n",
      "BiologicalProcess decreases Abundance                8  bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\") decreases a(CHEBI:17089 ! glycoprotein)\n",
      "Complex decreases BiologicalProcess                  6  complex(p(HGNC:10803 ! SFTPD), p(PFAM:PF00064 ! neuraminidase)) decreases bp(MESH:D009439 ! Neuraminidase)\n",
      "Protein directlyIncreases Protein                    3  p(FPLX:NFkappaB ! NFkappaB) directlyIncreases p(HGNC:320 ! AGER)\n",
      "Complex increases BiologicalProcess                  3  complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n",
      "Complex increases Protein                            3  complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases p(HGNC:6922 ! MBL2)\n",
      "Abundance directlyDecreases Protein                  3  a(CHEBI:68610 ! linagliptin) directlyDecreases p(HGNC:3009 ! DPP4)\n",
      "Complex decreases Protein                            2  complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR)) decreases p(HGNC:1748 ! CDH1)\n",
      "Abundance directlyIncreases Protein                  1  a(CHEBI:16761 ! ADP) directlyIncreases p(HGNC:8064 ! NUP214)\n",
      "Protein directlyIncreases Abundance                  1  p(FPLX:Glycosyltransferase ! Glycosyltransferase) directlyIncreases a(CHEBI:506227 ! N-acetyl-D-glucosamine)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "glucose_covid.summarize()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Causal BEL statements from each publication in bibliography"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pickle\n",
    "\n",
    "with open('../data/pmid_graph.pickle', 'rb') as p1:\n",
    "    pmid_graph = pickle.load(p1)\n",
    "with open('../data/pmid_indra.pickle', 'rb') as p2:\n",
    "    pmid_indra = pickle.load(p2)\n",
    "with open('../data/english_statements.pickle', 'rb') as p3:\n",
    "    english_statements = pickle.load( p3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "## [Type 2 alveolar cells are stem cells in adult lung.](https://pubmed.ncbi.nlm.nih.gov/23921127)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The intersection between aging and cardiovascular disease.](https://pubmed.ncbi.nlm.nih.gov/22499900)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Gammadelta T cells are reduced and rendered unresponsive by hyperglycemia and chronic {TNFalpha} in mouse models of obesity and metabolic disease.](https://pubmed.ncbi.nlm.nih.gov/20625397)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metabolic effects of influenza virus infection in cultured animal cells: Intra- and extracellular metabolite profiling.](https://pubmed.ncbi.nlm.nih.gov/20465796)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Angiotensin {II}-induced production of mitochondrial reactive oxygen species: potential mechanisms and relevance for cardiovascular disease.](https://pubmed.ncbi.nlm.nih.gov/22443458)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glucose metabolism in patients with essential hypertension.](https://pubmed.ncbi.nlm.nih.gov/16564774)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Blood pressure and incidence of twelve cardiovascular diseases: lifetime risks, healthy life-years lost, and age-specific associations in 125 million people.](https://pubmed.ncbi.nlm.nih.gov/24881994)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Intracellular functions of N-linked glycans.](https://pubmed.ncbi.nlm.nih.gov/11269317)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Alveolar macrophage-derived type I interferons orchestrate innate immunity to {RSV} through recruitment of antiviral monocytes.](https://pubmed.ncbi.nlm.nih.gov/25897172)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Angiotensin {II} increases the permeability and {PV}-1 expression of endothelial cells.](https://pubmed.ncbi.nlm.nih.gov/22012329)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Intensive insulin therapy in the medical {ICU}.](https://pubmed.ncbi.nlm.nih.gov/16452557)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Effects of airway surface liquid {pH} on host defense in cystic fibrosis.](https://pubmed.ncbi.nlm.nih.gov/24560894)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{pH} modulates the activity and synergism of the airway surface liquid antimicrobials \\beta-defensin-3 and {LL}-37.](https://pubmed.ncbi.nlm.nih.gov/25512526)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Reduced airway surface {pH} impairs bacterial killing in the porcine cystic fibrosis lung.](https://pubmed.ncbi.nlm.nih.gov/22763554)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Influence of external {pH} on ciliary beat frequency in human bronchi and bronchioles.](https://pubmed.ncbi.nlm.nih.gov/9551733)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The receptor for advanced glycation end products impairs host defense in pneumococcal pneumonia.](https://pubmed.ncbi.nlm.nih.gov/19299735)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High glucose disrupts oligosaccharide recognition function via competitive inhibition: a potential mechanism for immune dysregulation in diabetes mellitus.](https://pubmed.ncbi.nlm.nih.gov/20674073)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Crystal structure of the trimeric alpha-helical coiled-coil and the three lectin domains of human lung surfactant protein D.](https://pubmed.ncbi.nlm.nih.gov/10368295)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Vertebrate protein glycosylation: diversity, synthesis and function.](https://pubmed.ncbi.nlm.nih.gov/22722607)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypertension caused by angiotensin {II} infusion involves increased superoxide production in the central nervous system.](https://pubmed.ncbi.nlm.nih.gov/15192025)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Why do cancers have high aerobic glycolysis?](https://pubmed.ncbi.nlm.nih.gov/15516961)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycemic control in non-diabetic critically ill patients.](https://pubmed.ncbi.nlm.nih.gov/21925080)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [On the frequency of protein glycosylation, as deduced from analysis of the {SWI\\SS}-{PROT} database.](https://pubmed.ncbi.nlm.nih.gov/10580125)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The warburg effect: how does it benefit cancer cells?](https://pubmed.ncbi.nlm.nih.gov/26778478)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Humoral and mucosal immunity in protection from natural respiratory syncytial virus infection in adults.](https://pubmed.ncbi.nlm.nih.gov/15216475)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Immunoregulatory functions of surfactant proteins.](https://pubmed.ncbi.nlm.nih.gov/15630429)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Reactive metabolites and {AGE}/{RAGE}-mediated cellular dysfunction affect the aging process: a mini-review.](https://pubmed.ncbi.nlm.nih.gov/20962515)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia and cystic fibrosis alter respiratory fluid glucose concentrations estimated by breath condensate analysis.](https://pubmed.ncbi.nlm.nih.gov/17303703)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Intensive insulin therapy in critically ill patients.](https://pubmed.ncbi.nlm.nih.gov/11794168)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Roles of inflammation, oxidative stress, and vascular dysfunction in hypertension.](https://pubmed.ncbi.nlm.nih.gov/25136585)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### NOS3 increases the amount of superoxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7876 ! NOS3) increases a(CHEBI:18421 ! superoxide)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This leads to eNOS uncoupling, where eNOS produces superoxide instead of NO [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Aldosterone increases the amount of SPP1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27584 ! aldosterone) increases p(HGNC:11255 ! SPP1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the heart, aldosterone can increase vascular expression of ICAM-1, MCP-1, osteopontin, and COX-2, which can be blocked by the MR antagonist, eplerenone [XREF_BIBR], indicating involvement of the MR. Proinflammatory effects of aldosterone have also been reported to occur in the kidney, where aldosterone and salt treatment caused MR dependent leukocyte infiltration and elevation of osteopontin, IL-6, IL-1beta, and MCP-1 levels [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Aldosterone increases the amount of NR3C2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27584 ! aldosterone) increases p(HGNC:7979 ! NR3C2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Aldosterone activates the MR in the distal renal tubule of the kidney to increase sodium and water retention, and potassium excretion, leading to an increase in blood volume and thus blood pressure [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Aldosterone increases the amount of NR3C2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27584 ! aldosterone) increases p(HGNC:7979 ! NR3C2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Aldosterone activates the MR in the distal renal tubule of the kidney to increase sodium and water retention, and potassium excretion, leading to an increase in blood volume and thus blood pressure [ xref ]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Aldosterone increases the amount of PTGS2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27584 ! aldosterone) increases p(HGNC:9605 ! PTGS2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the heart, aldosterone can increase vascular expression of ICAM-1, MCP-1, osteopontin, and COX-2, which can be blocked by the MR antagonist, eplerenone [XREF_BIBR], indicating involvement of the MR. Proinflammatory effects of aldosterone have also been reported to occur in the kidney, where aldosterone and salt treatment caused MR dependent leukocyte infiltration and elevation of osteopontin, IL-6, IL-1beta, and MCP-1 levels [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CRP increases the amount of IL1B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2367 ! CRP) increases p(HGNC:5992 ! IL1B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "CRP can stimulate monocytes to release proinflammatory cytokines such as interleukin-6 (IL-6), interleukin-1 beta (IL-1beta), and tumour necrosis factor alpha (TNF-alpha) [XREF_BIBR] and also endothelial cells to express intracellular adhesion molecule (ICAM) -1 and vascular cell adhesion molecule (VCAM) -1 [XREF_BIBR], effects which will further promote inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pitavastatin decreases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:32020 ! pitavastatin) decreases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Pitavastatin has been shown to reduce IL-6 and IL-8 secretions and mRNA expression of IL-6, IL-8, and granulocyte macrophage colony stimulating factor (GM-CSF) in LPS stimulated human bronchial epithelium cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Rosuvastatin decreases the amount of CRP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:38545 ! rosuvastatin) decreases p(HGNC:2367 ! CRP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Even in participants without hyperlipidemia, rosuvastatin has been shown to reduce CRP levels in association with a lower incidence of cardiovascular events [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia stimulates coagulation, whereas hyperinsulinemia impairs fibrinolysis in healthy humans.](https://pubmed.ncbi.nlm.nih.gov/16731846)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Trimeric {HIV}-1-Env Structures Define Glycan Shields from Clades A, B, and G.](https://pubmed.ncbi.nlm.nih.gov/27114034)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Evolution of the receptor binding properties of the influenza A({H3N2}) hemagglutinin.](https://pubmed.ncbi.nlm.nih.gov/23236176)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Influenza virus-induced lung injury: pathogenesis and implications for treatment.](https://pubmed.ncbi.nlm.nih.gov/25792631)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia inhibits complement-mediated immunological control of S. aureus in a rat model of peritonitis.](https://pubmed.ncbi.nlm.nih.gov/25610878)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Elevated plasma surfactant protein D ({SP}-D) levels and a direct correlation with anti-severe acute respiratory syndrome coronavirus-specific {IgG} antibody in {SARS} patients.](https://pubmed.ncbi.nlm.nih.gov/19439011)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Dendritic Cells increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003713 ! \"Dendritic Cells\") increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [ 9 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Dendritic Cells activates Antigen Presentation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003713 ! \"Dendritic Cells\") increases bp(MESH:D017951 ! \"Antigen Presentation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [ 9 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates antigen processing and presentation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0019882\" ! \"antigen processing and presentation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Another simple explanation includes SP-D augmentation of antigen presentation by dendritic cells as previously demonstrated for bone marrow-derived dendritic cells in vitro [9] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates antigen processing and presentation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0019882\" ! \"antigen processing and presentation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D is recognized to modulate the antigen presentation to specific major histocompatibility complex class II T-cell hybridomas in vitro in a dose-dependent fashion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates Antigen Presentation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(MESH:D017951 ! \"Antigen Presentation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D is recognized to modulate the antigen presentation to specific major histocompatibility complex class II T-cell hybridomas in vitro in a dose-dependent fashion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SP activates antigen processing and presentation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:SP) increases bp(GO:\"GO:0019882\" ! \"antigen processing and presentation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SPD is recognized to modulate the antigen presentation to specific major histocompatibility complex class II Tcell hybridomas in vitro in a dosedependent fashion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Comorbidity is a better predictor of impaired immunity than chronological age in older adults.](https://pubmed.ncbi.nlm.nih.gov/16137288)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advanced glycoxidation and lipoxidation end products ({AGEs} and {ALEs}): an overview of their mechanisms of formation.](https://pubmed.ncbi.nlm.nih.gov/23767955)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Modulation of glucose metabolism by the renin-angiotensin-aldosterone system.](https://pubmed.ncbi.nlm.nih.gov/25564475)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycolytic control of vacuolar-type {ATPase} activity: a mechanism to regulate influenza viral infection.](https://pubmed.ncbi.nlm.nih.gov/23876457)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The sweeter side of {ACE2}: physiological evidence for a role in diabetes.](https://pubmed.ncbi.nlm.nih.gov/18948167)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### AGT increases the amount of NOX1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:333 ! AGT) increases p(HGNC:7889 ! NOX1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NOX activation is triggered by interaction of Ang-II with the AT1 receptor and requires the assembly of the cytosolic subunits p47phox, p40phox and p67phox on the membrane bound scaffold proteins p22phox and Nox2/4."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC2A4 increases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11009 ! SLC2A4) increases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mas -/- mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC2A4 increases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11009 ! SLC2A4) increases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mas / mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC2A4 increases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11009 ! SLC2A4) increases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mas / mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC2A4 decreases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11009 ! SLC2A4) decreases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mas -/- mice also exhibit impaired insulin sensitivity and glucose tolerance, reduced adipose glucose uptake and impaired glucose transporter 4 (GLUT4) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS increases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ang-II signaling cascades inhibit insulin release and decrease insulin sensitivity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS activates vasodilation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases bp(GO:\"GO:0042311\" ! vasodilation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This study clearly demonstrates that loss of the Mas receptor leads to metabolic dysfunction and implicates the ACE2/Ang-(17)/Mas receptor axis in the development of insulin resistance.Obesity and insulin resistance are associated with impaired endothelium-dependent and insulin-mediated vasodilation (Laakso et al., 1990, Steinberg et al., 1996, Williams et al., 1996)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In non-diabetic rats, acute Ang-II infusion prevented the first phase insulin secretion in response to glucose (Carlsson et al., 1998), and in healthy human subjects, pressor doses of Ang-II suppressed basal, pulsatile and glucose-stimulated insulin release (Fliser et al., 1997)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 increases the amount of streptozocin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases a(CHEBI:9288 ! streptozocin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, in streptozotocin-induced diabetic mice, a T1DM model, ACE2 protein expression is elevated in the early (Wysocki et al., 2006) and decreased in the late (Tikellis et al., 2003) phase of diabetic nephropathy."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 increases the amount of streptozocin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases a(CHEBI:9288 ! streptozocin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, in streptozotocin-induced diabetic mice, a T1DM model, ACE2 protein expression is elevated in the early and decreased in the late (Tikellis et al., 2003) phase of diabetic nephropathy."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 decreases the amount of streptozocin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases a(CHEBI:9288 ! streptozocin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the islets, ACE2 may modulate blood flow and morphology in order to maintain insulin secretion (Fig. 3)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates Diabetic Nephropathies\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D003928 ! \"Diabetic Nephropathies\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2, a new component of the RAS, is elevated in diabetes and diabetic nephropathy and may be involved in a compensatory mechanism opposing the ACE/Ang-II/AT1-receptor axis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 inhibits Diabetic Nephropathies\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(MESH:D003928 ! \"Diabetic Nephropathies\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Finally in two models of T1DM (streptozotocin and Akita mouse models), ACE2 gene deletion accelerated the development of diabetic kidney disease (Wong et al., 2007, Tikellis et al., 2008)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates Fibrosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D005355 ! Fibrosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates Fibrosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D005355 ! Fibrosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 through TGF- inhibition may reduce amyloid deposition, islet fibrosis and subsequent -cell apoptosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates Mice\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D051379 ! Mice)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In 8-week-old db/db mice, a model of early T2DM, ACE2 expression is elevated while ACE expression is decreased in both glomeruli and cortex (Ye et al., 2004) prior to the development of diabetic nephropathy."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 decreases the amount of DIANPH\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases p(HGNC:2875 ! DIANPH)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed , several studies demonstrated that loss or inhibition of ACE2 exacerbates diabetic nephropathy Soler et al ., 2007 ; Wong et al ., 2007 ; Tikellis et al ., 2008 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 decreases the amount of DIANPH\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases p(HGNC:2875 ! DIANPH)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This compensatory effect of ACE2 in diabetes has already been suggested in the kidney by studies demonstrating that ACE2 inhibition worsens diabetic nephropathy ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Apoptotic process inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006915\" ! \"apoptotic process\") decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The combined reduction in -cell apoptosis and increase in islet blood flow could cause an increase in insulin secretion and preservation of islet function in diabetes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Apoptotic process inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006915\" ! \"apoptotic process\") decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The combined reduction in -cell apoptosis and increase in islet blood flow could cause an increase in insulin secretion and preservation of islet function in diabetes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Fibrosis inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D005355 ! Fibrosis) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006; Tikellis et al., 2004a,b) , (2) loss of insulin secretion (Lupi et al., 2006; Shao et al., 2006; Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006; Tikellis et al., 2004b; Nakayama et al., 2005) , independently of changes in plasma glucose levels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Fibrosis inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D005355 ! Fibrosis) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006, Tikellis et al., 2004a, Tikellis et al., 2004b), (2) loss of insulin secretion (Lupi et al., 2006, Shao et al., 2006, Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006, Tikellis et al., 2004b, Nakayama et al., 2005), independently of changes in plasma glucose levels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Fibrosis inhibits insulin secretion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D005355 ! Fibrosis) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This was demonstrated by prevention of (1) islet fibrosis (Shao et al., 2006; Tikellis et al., 2004a,b) , (2) loss of insulin secretion (Lupi et al., 2006; Shao et al., 2006; Tikellis et al., 2004b) and (3) loss of -cells and reduction in islet insulin content (Shao et al., 2006; Tikellis et al., 2004b; Nakayama et al., 2005) , independently of changes in plasma glucose levels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice increases the amount of streptozocin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) increases a(CHEBI:9288 ! streptozocin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, in streptozotocin-induced diabetic ACE2/y mice, ACE2 gene deletion inhibited the beneficial effects of ACE inhibitor (Tikellis et al., 2008) in diabetic nephropathy, suggesting that ACE inhibition potentiates ACE2 activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AKT increases the amount of nitric oxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AKT ! AKT) increases a(CHEBI:16480 ! \"nitric oxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Recent studies have implicated the Akt pathway in Ang-(17) mediated NO release (Sampaio et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC33A1 activates Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:95 ! SLC33A1) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Supporting evidence include : (1) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels; (2) acute Ang-II infusion decreases islet blood flow in healthy non diabetic Sprague-Dawley rats; and (3) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC33A1 activates Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:95 ! SLC33A1) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Supporting evidence include:(1) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels (Carlsson et al., 1998);  (2) acute Ang-II infusion decreases islet blood flow in healthy nondiabetic Sprague-Dawley rats (Carlsson et al., 1998) ; and (3) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels (Carlsson et al., 1998) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of triglyceride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases a(CHEBI:17855 ! triglyceride)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "RAS blockade prevents hepatic triglyceride accumulation in obese Zucker rats, a common feature of insulin resistant and diabetic states (Toblli et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of TGFB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(FPLX:TGFB ! TGFB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand , RAS blockade decreases TGF-beta expression in the islet and is associated with improved islet morphology and function ( Tikellis et al ., 2004b ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of TGFB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(FPLX:TGFB ! TGFB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, RAS blockade decreases TGF-beta expression in the islet and is associated with improved islet morphology and function."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of TGFB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(FPLX:TGFB ! TGFB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, RAS blockade decreases TGF- expression in the islet and is associated with improved islet morphology and function (Tikellis et al., 2004b)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of CYBA\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(HGNC:2577 ! CYBA)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of CYBB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(HGNC:2578 ! CYBB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of NOX1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases p(HGNC:7889 ! NOX1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, RAS blockade-mediated decreases in the expression of the NOX components p22phox and Nox2 (Shao et al., 2006), and nitrotyrosine levels (Shao et al., 2006, Tikellis et al., 2004b) in the islet were associated with prevention of islet dysfunction and deleterious changes in islet morphology."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System inhibits glucose homeostasis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") decreases bp(GO:\"GO:0042593\" ! \"glucose homeostasis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "RAS blockade improves glucose homeostasis by reducing  - cell death and improving insulin secretion and end-organ insulin sensitivity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System inhibits glucose homeostasis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") decreases bp(GO:\"GO:0042593\" ! \"glucose homeostasis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "RAS blockade improves glucose homeostasis by reducing beta-cell death and improving insulin secretion and end-organ insulin sensitivity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cell death inhibits glucose homeostasis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0008219\" ! \"cell death\") decreases bp(GO:\"GO:0042593\" ! \"glucose homeostasis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "RAS blockade improves glucose homeostasis by reducing  - cell death and improving insulin secretion and end-organ insulin sensitivity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Captopril increases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3380 ! captopril) increases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Captopril decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3380 ! captopril) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Captopril decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3380 ! captopril) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Captopril activates response to insulin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3380 ! captopril) increases bp(GO:\"GO:0032868\" ! \"response to insulin\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Captopril activates glucose import\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3380 ! captopril) increases bp(GO:\"GO:0046323\" ! \"glucose import\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The most remarkable evidence for the role of the RAS in diabetes is the demonstration that captopril suppressed hepatic glucose output and increased hepatic glucose uptake in response to insulin infusion (Torlone et al., 1991)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hyperglycemia activates Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The red arrows emphasize hyperglycemia-induced changes in the islet RAS and its consequences."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  KNG1 increases the amount of nitric oxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6383 ! KNG1) increases a(CHEBI:16480 ! \"nitric oxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed, ACE inhibitors improve skeletal muscle glucose uptake through a bradykinin mediated increase in NO."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE activates Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Supporting evidence include : ( 1 ) greater Ang-II levels in islet microvessels compared to exocrine pancreas microvessels ( Carlsson et al ., 1998 ) ; ( 2 ) acute Ang-II infusion decreases islet blood flow in healthy non-diabetic Sprague-Dawley rats ( Carlsson et al ., 1998 ) ; and ( 3 ) RAS blockade by ACE inhibitors or AT1 receptor blockers preferentially increased blood flow to islet microvessels ( Carlsson et al ., 1998 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE decreases the amount of NOX1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) decreases p(HGNC:7889 ! NOX1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, RAS blockade has been associated with a decrease in islet AT1 and ACE expression (Tikellis et al., 2004a,b) , which would result in decreased NOX activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE decreases the amount of NOX1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) decreases p(HGNC:7889 ! NOX1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, RAS blockade has been associated with a decrease in islet AT1 and ACE expression (Tikellis et al., 2004a, Tikellis et al., 2004b), which would result in decreased NOX activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MAS1 activates Fibrosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6899 ! MAS1) increases bp(MESH:D005355 ! Fibrosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mas receptor expression has also been identified in the liver and is elevated in biliary fibrosis (Herath et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  REN activates Diabetes Complications\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9958 ! REN) increases bp(MESH:D048909 ! \"Diabetes Complications\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Of clinical relevance, blockade of the reninangiotensin system prevents new-onset diabetes and reduces the risk of diabetic complications."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bradykinin increases the amount of nitric oxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3165 ! bradykinin) increases a(CHEBI:16480 ! \"nitric oxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed, ACE inhibitors improve skeletal muscle glucose uptake through a bradykinin-mediated increase in NO (Henriksen et al., 1999)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bradykinin activates vasodilation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3165 ! bradykinin) increases bp(GO:\"GO:0042311\" ! vasodilation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It is well known that Ang-(17) potentiates bradykinin-mediated vasodilation (Paula et al., 1995, Lima et al., 1997)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bradykinin inhibits vasodilation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3165 ! bradykinin) decreases bp(GO:\"GO:0042311\" ! vasodilation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In both T1DM and T2DM, loss of Ang-(17) potentiation of bradykinin-mediated vasodilation is rescued by chronic but not acute insulin (Oliveira et al., 2002, Rastelli et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin II Type 1 Receptor Blockers inhibits Albuminuria\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D047228 ! \"Angiotensin II Type 1 Receptor Blockers\") decreases bp(MESH:D000419 ! Albuminuria)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ARB therapy did however reduce albuminuria in Akita-ACE2/y mice (Wong et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Enalapril inhibits Fibrosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:4784 ! enalapril) decreases bp(MESH:D005355 ! Fibrosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, enalapril prevents fibrosis and oxidative stress in the liver, heart and kidney of rats with T1DM (de Cavanagh et al., 2001)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Biochemistry and molecular cell biology of diabetic complications.](https://pubmed.ncbi.nlm.nih.gov/11742414)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Coronavirus spike proteins in viral entry and pathogenesis.](https://pubmed.ncbi.nlm.nih.gov/11162792)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Alveolar epithelial cells: master regulators of lung homeostasis.](https://pubmed.ncbi.nlm.nih.gov/23988571)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Inflammation and angiotensin {II}.](https://pubmed.ncbi.nlm.nih.gov/12676174)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin Uniquely Prevents Thrombosis by Inhibiting Platelet Activation and {mtDNA} Release.](https://pubmed.ncbi.nlm.nih.gov/27805009)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Cilia and mucociliary clearance.](https://pubmed.ncbi.nlm.nih.gov/27864314)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Regulation of intracellular {pH}.](https://pubmed.ncbi.nlm.nih.gov/15545345)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advanced glycation end products and diabetic complications.](https://pubmed.ncbi.nlm.nih.gov/24634591)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Aging, diabetes, and hypertension are associated with decreased nasal mucociliary clearance.](https://pubmed.ncbi.nlm.nih.gov/23100111)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Thin and strong! The bioengineering dilemma in the structural and functional design of the blood-gas barrier.](https://pubmed.ncbi.nlm.nih.gov/15987796)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity-induced insulin resistance and hyperglycemia: etiologic factors and molecular mechanisms.](https://pubmed.ncbi.nlm.nih.gov/18580184)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advanced glycation end-products: a review.](https://pubmed.ncbi.nlm.nih.gov/11270668)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Virus glycosylation: role in virulence and immune interactions.](https://pubmed.ncbi.nlm.nih.gov/17398101)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Polysaccharide inhibits Virus Internalization\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:18154 ! polysaccharide) decreases bp(MESH:D053586 ! \"Virus Internalization\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "If carbohydrate is positioned close to the HA cleavage site , the glycan can prevent protease access and virus entry ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPA2 inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10799 ! SFTPA2) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, SP-A and SP-D can both interact with influenza to prevent the association of the HA with its receptor."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glycosylation activates Viruses\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0070085\" ! glycosylation) increases bp(MESH:D014780 ! Viruses)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Second , changes in glycosylation can affect interaction with receptors and cause a virus to be more recognizable by the innate factors of host immune cells and less recognized by antibody , thus impacting viral replication and infectivity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glycosylation inhibits membrane fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0070085\" ! glycosylation) decreases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Interestingly , in the case of Nipah virus , glycosylation has a dual role : glycosylation leads to enhanced resistance to antibody neutralization but alternatively causes a reduction in membrane fusion and viral entry [ 42 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Fasting glucose and {HbA1c} levels as risk factors for the development of hypertension in Japanese individuals: Toranomon hospital health management center study 16 ({TOPICS} 16).](https://pubmed.ncbi.nlm.nih.gov/25231510)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [All the \"{RAGE}\" in lung disease: The receptor for advanced glycation endproducts ({RAGE}) is a major mediator of pulmonary inflammatory responses.](https://pubmed.ncbi.nlm.nih.gov/28416135)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Macrophages in tuberculosis: friend or foe.](https://pubmed.ncbi.nlm.nih.gov/23864058)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Viral activation of cellular metabolism.](https://pubmed.ncbi.nlm.nih.gov/25812764)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Acute respiratory distress syndrome.](https://pubmed.ncbi.nlm.nih.gov/28792873)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Virus entry: what has {pH} got to do with it?](https://pubmed.ncbi.nlm.nih.gov/23377025)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The relative roles of passive surface forces and active ion transport in the modulation of airway surface liquid volume and composition.](https://pubmed.ncbi.nlm.nih.gov/11479349)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Short-term safety, tolerability and efficacy of a very low-calorie-ketogenic diet interventional weight loss program versus hypocaloric diet in patients with type 2 diabetes mellitus.](https://pubmed.ncbi.nlm.nih.gov/27643725)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Bitter-sweet symphony: glycan-lectin interactions in virus biology.](https://pubmed.ncbi.nlm.nih.gov/24188132)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Sialic acid increases the amount of CSH1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:26667 ! \"sialic acid\") increases p(HGNC:2440 ! CSH1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Delputte PL & Nauwynck HJ (2004) Porcine arterivirus infection of alveolar macrophages is mediated by sialic acid on the virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Sialic acid activates Arterivirus Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:26667 ! \"sialic acid\") increases bp(MESH:D018174 ! \"Arterivirus Infections\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Delputte PL & Nauwynck HJ (2004) Porcine arterivirus infection of alveolar macrophages is mediated by sialic acid on the virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mannose decreases the amount of CD4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:37684 ! mannose) decreases p(HGNC:1678 ! CD4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, it has been shown that multivalent mannosecontaining molecules or mannose-based glycomimetics can compromise binding of HIV-1 gp120 with DC-SIGN (Wang et al., 2008b; Luallen et al., 2009; Martinez-Avila et al., 2009b; Becer et al., 2010) and inhibit DC-SIGN-mediated trans-infection of CD4 + T lymphocytes (Martinez-Avila et al., 2009a; Sattin et al., 2010; Berzi et al., 2012) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD209 increases the amount of RAF1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1641 ! CD209) increases p(HGNC:9829 ! RAF1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD209 increases the amount of RAF1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1641 ! CD209) increases p(HGNC:9829 ! RAF1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "(2007, 2010) showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll like receptor (TLR)-elicited signals to induce synthesis of full-length HIV transcripts as well as production of the immunosuppressive cytokine IL-10 (Gringhuis etal., (2007, 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAF1 increases the amount of TLR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9829 ! RAF1) increases p(FPLX:TLR ! TLR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAF1 increases the amount of TLR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9829 ! RAF1) increases p(FPLX:TLR ! TLR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Other work by Gringhuis et al. (2007 Gringhuis et al. ( , 2010 showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll-like receptor (TLR)-elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL-10 ( Gringhuis et al., (2007 Gringhuis et al., ( , 2010 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAF1 increases the amount of TLR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9829 ! RAF1) increases p(FPLX:TLR ! TLR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "(2007, 2010) showed that binding of HIV-1 to DC-SIGN triggers Raf-1 dependent signaling, which modulates toll like receptor (TLR)-elicited signals to induce synthesis of full-length HIV transcripts as well as production of the immunosuppressive cytokine IL-10 (Gringhuis etal., (2007, 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Dendritic Cells inhibits Dendritic Cells\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003713 ! \"Dendritic Cells\") decreases bp(MESH:D003713 ! \"Dendritic Cells\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Experiments in 293 TRex cells using an inducible DCSIGN expression system have shown that high surface expression levels of DCSIGN correlate with optimal binding of HIV1 particles, and that lowering the DCSIGN expression levels can significantly reduce the efficiency of HIV1 binding (Pohlmann et al., 2001)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Infections increases the amount of CD86\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007239 ! Infections) increases p(HGNC:1705 ! CD86)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example, Hodges et al. (2007) reported that binding of HIV1 to DCSIGN compromises DC maturation and primes these cells for transinfection: Upregulation of CD86 and MHCII is suppressed, whereas synapse formation between DCs and CD4+ T cells is promoted (Hodges et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates Hydrogen-Ion Concentration\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(MESH:D006863 ! \"Hydrogen-Ion Concentration\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As situated above, both influenza A and B viruses display hemagglutinin proteins on their surface, which bind to sialic acids displayed on the host cell surface and mediate pHdependent fusion of the viral membrane with the host cell membrane (Skehel & Wiley, 2000; Lamb & Krug, 2001; Wang et al., 2007b; Wang et al., 2008a; Harrison, 2008; Gamblin & Skehel, 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SP inhibits Neuraminidase\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:SP) decreases bp(MESH:D009439 ! Neuraminidase)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Both SPA and SPD inhibit the viral hemagglutinating activity that is required for IAV attachment to target cells (Hartshorn et al., 1994; Malhotra et al., 1994; Benne et al., 1995; Hartshorn et al., 1996, 1997, 2000; van Eijk et al., 2002, 2003, 2004; Mikerov et al., 2008; Hillaire et al., 2011) and SPD was reported to inhibit the viral neuraminidase (Reading et al., 1997; Hillaire et al., 2011)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Neuraminidase inhibits virion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009439 ! Neuraminidase) decreases bp(GO:\"GO:0019012\" ! virion)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , however , the neuraminidase activity can prevent virions from aggregating via hemagglutinin-sialic acid interactions , promotes efficient release of newly formed virions from ( sialic acid-carrying ) infected cells , and provides a counterweight to the interaction of hemagglutinin molecules with nontarget cell-associated glycans : neuraminidase-mediated removal of sialic acids from decoy receptors prevents virions from establishing high-avidity interactions with these glycoconjugates and may even provide an escape route for virions after hemagglutinin-mediated binding to nontarget cell-associated glycans ( Colman ,1994 ; Suzukiet al ., 1994 ; Gimsaet al ., 1996 ; Barrereet al ., 1997 ; Gamblin & Skehel ,2010 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ile-Leu increases the amount of RAF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:74072 ! Ile-Leu) increases p(FPLX:RAF ! RAF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Other work by Gringhuis et al. (2007, 2010) showed that binding of HIV1 to DCSIGN triggers Raf1 dependent signaling, which modulates tolllike receptor (TLR)elicited signals to induce synthesis of fulllength HIV transcripts as well as production of the immunosuppressive cytokine IL10 (Gringhuis et al., (2007, 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P84849 inhibits viral process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P84849 ! P84849) decreases bp(GO:\"GO:0016032\" ! \"viral process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, galectin1 binding impedes endocytosis and maturation of the NiVF precursor NiVF0 in infected cells (Garner et al., 2010), further illustrating the capacity of this lectin to block different stages of the viral infection/replication process.In sum, soluble host lectins can counteract viral infection in various ways: they may directly neutralize virus by destabilizing or aggregating virions, interfere with crucial steps in the viral infection process (e.g."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MBL2 binds ligand\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ligand bound to MBL2 activates complement activation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MBL2 binds ligand\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases p(HGNC:6922 ! MBL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ligand bound to MBL2 increases the amount of MBL2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases p(HGNC:6922 ! MBL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MBL2 binds ligand\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases p(UP:P84849 ! P84849)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ligand bound to MBL2 increases the amount of P84849\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:52214 ! ligand), p(HGNC:6922 ! MBL2)) increases p(UP:P84849 ! P84849)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ligand binding by MBL (or alternatively ficolins) can lead to activation of MBLassociated serine proteases (MASPs) and initiate the complement cascade via the socalled lectin pathway (Blue et al., 2004; Bottazzi et al., 2010)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Management of diabetes and hyperglycemia in hospitals.](https://pubmed.ncbi.nlm.nih.gov/14747243)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Angiotensin {II} revisited: new roles in inflammation, immunology and aging.](https://pubmed.ncbi.nlm.nih.gov/20597104)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Aging Impairs Alveolar Macrophage Phagocytosis and Increases Influenza-Induced Mortality in Mice.](https://pubmed.ncbi.nlm.nih.gov/28646038)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Ketogenic diet for treatment of intractable epilepsy in adults: A meta-analysis of observational studies.](https://pubmed.ncbi.nlm.nih.gov/29588983)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role and importance of club cells (Clara cells) in the pathogenesis of some respiratory diseases.](https://pubmed.ncbi.nlm.nih.gov/27212975)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glucose depletion in the airway surface liquid is essential for sterility of the airways.](https://pubmed.ncbi.nlm.nih.gov/21311590)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advantages and pitfalls of fructosamine and glycated albumin in the diagnosis and treatment of diabetes.](https://pubmed.ncbi.nlm.nih.gov/25591856)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Measurement of monovalent and polyvalent carbohydrate-lectin binding by back-scattering interferometry.](https://pubmed.ncbi.nlm.nih.gov/19462965)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Infection as a comorbidity of {COPD}.](https://pubmed.ncbi.nlm.nih.gov/20513910)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Postprandial blood glucose as a risk factor for cardiovascular disease in Type {II} diabetes: the epidemiological evidence.](https://pubmed.ncbi.nlm.nih.gov/11793012)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin: from mechanisms of action to therapies.](https://pubmed.ncbi.nlm.nih.gov/25456737)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin reduces airway glucose permeability and hyperglycaemia-induced Staphylococcus aureus load independently of effects on blood glucose.](https://pubmed.ncbi.nlm.nih.gov/23709760)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Age-Related Changes in Glucose Metabolism, Hyperglycemia, and Cardiovascular Risk.](https://pubmed.ncbi.nlm.nih.gov/30355075)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia and endothelial dysfunction in atherosclerosis: lessons from type 1 diabetes.](https://pubmed.ncbi.nlm.nih.gov/22489274)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Drug-induced hyperglycemia.](https://pubmed.ncbi.nlm.nih.gov/11667913)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The {ENGases}: versatile biocatalysts for the production of homogeneous N-linked glycopeptides and glycoproteins.](https://pubmed.ncbi.nlm.nih.gov/28681051)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Multi-timescale modeling of activity-dependent metabolic coupling in the neuron-glia-vasculature ensemble.](https://pubmed.ncbi.nlm.nih.gov/25719367)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [N-glycan microheterogeneity regulates interactions of plasma proteins.](https://pubmed.ncbi.nlm.nih.gov/30111543)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis, inflammation and {NF}-{\\kappaB} signaling.](https://pubmed.ncbi.nlm.nih.gov/30337682)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Lipopolysaccharide inhibits Cell Survival\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) decreases bp(MESH:D002470 ! \"Cell Survival\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "LPS treatment caused decreased cell viability in the PTECs from both WT and KO mice; and treated KO PTECs exhibited lower viability compared to treated WT PTECs (Fig. 8B)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Apoptotic process activates Kidney\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006915\" ! \"apoptotic process\") increases bp(MESH:D007668 ! Kidney)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibiting NF-kappaB signaling in renal tubular epithelial lead to reduce tubular injury , apoptosis , necrosis , and accumulation of interstitial inflammatory cells , which consequently results in ameliorated kidney damage after ischemic AKI51 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Kidney activates positive regulation of NF-kappaB transcription factor activity\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007668 ! Kidney) increases bp(GO:\"GO:0051092\" ! \"positive regulation of NF-kappaB transcription factor activity\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These findings from in vivo study demonstrated that lack of SP-D in lung and / or kidney promotes severe renal injury , tubular cell apoptosis , NF-kappaB activation and production of pro-inflammatory cytokines , which consequently results in aggravated lung-kidney crosstalk during pneumonia-induced AKI ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Necrotic cell death activates Kidney\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0070265\" ! \"necrotic cell death\") increases bp(MESH:D007668 ! Kidney)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibiting NF-kappaB signaling in renal tubular epithelial lead to reduce tubular injury , apoptosis , necrosis , and accumulation of interstitial inflammatory cells , which consequently results in ameliorated kidney damage after ischemic AKI51 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pneumonia increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D011014 ! Pneumonia) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These data further demonstrated that the lack of pulmonary and/or renal SP-D enhanced renal NF-B activation and inflammatory cytokine expression in the kidney in pneumonia-induced AKI.To study the protective effect of SP-D on tubular apoptosis, PTECs were isolated from the kidneys of WT and KO mice."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of p38\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases p(FPLX:p38 ! p38)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "When infection occurs, the exposed collagenous tail of SP-D adheres to the calreticulin/CD91 receptor complex and initiates a pro-inflammatory response by stimulating p38 MAPK phyosphorylation and NF-B activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These findings from in vivo study demonstrated that lack of SP-D in lung and/or kidney promotes severe renal injury, tubular cell apoptosis, NF-B activation and production of pro-inflammatory cytokines, which consequently results in aggravated lung-kidney crosstalk during pneumonia-induced AKI."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "We found that renal tubular necrosis/apoptosis and inflammatory responses are important mechanisms involving in bacterial pneumonia-induced AKI, in which both pulmonary and renal SP-D participates in the lung-kidney crosstalk and directly or indirectly modulate renal tubular apoptosis and its inflammatory responses."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.Published: xx xx xxxx OPEN www.nature.com/scientificreports/AKI is a serious complication affecting more than half of all ICU patients 1 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D deficiency increases renal tubular cell apoptosis and caspase-3 activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-kappaB signaling."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "To our knowledge, this study is the first report to demonstrate that both pulmonary and renal SP-D protects from pneumonia-induced AKI through modulating lung-kidney crosstalk and attenuating renal tubular apoptosis, NF-B activation and pro-inflammatory cytokine production.Many clinical and experimental studies demonstrated mechanical ventilation as an important mediator of AKI in ALI and ARDS36,37."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "When infection occurs, the exposed collagenous tail of SP-D adheres to the calreticulin/CD91 receptor complex and initiates a pro-inflammatory response by stimulating p38 MAPK phyosphorylation and NF-B activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of CASP3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases p(HGNC:1504 ! CASP3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D deficiency increases renal tubular cell apoptosis and caspase-3 activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates Pneumonia\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(MESH:D011014 ! Pneumonia)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Pulmonary SP-D deficiency leads to Decreased bacterial clearance and deteriorated lung injury in pneumonia model."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates Lung Injury\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(MESH:D055370 ! \"Lung Injury\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Pulmonary SP-D deficiency leads to Decreased bacterial clearance and deteriorated lung injury in pneumonia model."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of kidney injury\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases a(TEXT:\"kidney injury\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis , inflammation and NF-kappaB signaling ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of kidney injury\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases a(TEXT:\"kidney injury\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis , inflammation and NF-kappaB signaling OPEN ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of NFkappaB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases p(FPLX:NFkappaB ! NFkappaB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Figure 7 : SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of NFkappaB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases p(FPLX:NFkappaB ! NFkappaB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD decreases the amount of NFkappaB\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases p(FPLX:NFkappaB ! NFkappaB)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-kappaB signaling."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits Kidney\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(MESH:D007668 ! Kidney)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Figure 7 : SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits Kidney\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(MESH:D007668 ! Kidney)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D deficiency increases NF-kappaB activation and renal inflammation in the kidney ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits Sepsis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(MESH:D018805 ! Sepsis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling.Published: xx xx xxxx OPEN www.nature.com/scientificreports/AKI is a serious complication affecting more than half of all ICU patients 1 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits Sepsis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(MESH:D018805 ! Sepsis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Collectively, SP-D attenuates AKI in the sepsis by modulating renal apoptosis, inflammation and NF-B signaling."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Sepsis inhibits apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D018805 ! Sepsis) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis, inflammation and NF-B signaling OPEN.pneumonia-induced sepsis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Sepsis inhibits inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D018805 ! Sepsis) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Surfactant protein D attenuates acute lung and kidney injuries in pneumonia-induced sepsis through modulating apoptosis, inflammation and NF-B signaling OPEN.pneumonia-induced sepsis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endotoxemia activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D019446 ! Endotoxemia) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D KO mice also showed increased inflammation and lung injury caused by endotoxemia 17 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endotoxemia activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D019446 ! Endotoxemia) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, endotoxemia-induced indirect lung injury showed increased pulmonary inflammation but with a paradoxical decrease in mortality in SP-D KO mice when compared with WT mice17."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Signaling receptor activity activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0038023\" ! \"signaling receptor activity\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Therefore , the lung and kidney SP-D still plays the role of inhibiting inflammation by blocking Toll-like receptor 4 through the CRD54 to some extent even though they were dramatically reduced in infected WT mice , which resulted in the attenuated lung-kidney crosstalk than in KO or hTG mice ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Salicylhydroxamic acid activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:45615 ! \"salicylhydroxamic acid\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Sham = Sham infection, Pneu = Pneumonia.SP-D deficiency increases NF-B activation and renal inflammation in the kidney."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Salicylhydroxamic acid increases the amount of CASP3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:45615 ! \"salicylhydroxamic acid\") increases p(HGNC:1504 ! CASP3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Sham = Sham infection, Pneu = Pneumonia.Figure 8: Primary proximal tubular epithelial cells (PTECs) from KO mice increase apoptosis and apoptosis-related caspase-3 expression after LPS treatment compared to those from WT mice."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{AMPK} agonists ameliorate sodium and fluid transport and inflammation in cystic fibrosis airway epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/19617399)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Alternative biomarkers for assessing glycemic control in diabetes: fructosamine, glycated albumin, and 1,5-anhydroglucitol.](https://pubmed.ncbi.nlm.nih.gov/26191510)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Slippery when wet: airway surface liquid homeostasis and mucus hydration.](https://pubmed.ncbi.nlm.nih.gov/30243435)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycaemia and pulmonary infection.](https://pubmed.ncbi.nlm.nih.gov/16923307)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Global aspects of viral glycosylation.](https://pubmed.ncbi.nlm.nih.gov/29579213)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### THBS1 activates cell-cell fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11785 ! THBS1) increases bp(GO:\"GO:0140253\" ! \"cell-cell fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mutation of Nglycosylation sites in the globular head of glycoprotein G increased cell-cell fusion, whereas removal of a glycosite in the stalk region of Hendra glycoprotein G abolished cell-cell fusion (Bradel-Tretheway et al. 2015) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mucin-like activates Mice\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(PFAM:PF16058 ! mucin-like) increases bp(MESH:D051379 ! Mice)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Conversely, the deletion of the Ebola virus mucin-like domain decreased the immune response in mice, although it did not have an effect on immunogenicity in vitro (Dowling et al. 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glycosyltransferase increases the amount of N-acetyl-D-glucosamine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Glycosyltransferase ! Glycosyltransferase) directlyIncreases a(CHEBI:506227 ! N-acetyl-D-glucosamine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Most types of O-glycosylation are carried out in the secretory pathway, but cytosolic and nuclear proteins can also be glycosylated with N-acetylglucosamine (GlcNAc) by a cytosolic glycosyltransferase (Moremen et al. 2012; Yang and Qian 2017)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Low-Carb and Ketogenic Diets in Type 1 and Type 2 Diabetes.](https://pubmed.ncbi.nlm.nih.gov/31035514)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Management of hyperglycemia during enteral and parenteral nutrition therapy.](https://pubmed.ncbi.nlm.nih.gov/23065369)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Exploitation of glycosylation in enveloped virus pathobiology.](https://pubmed.ncbi.nlm.nih.gov/31121217)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### N-acetyl-D-glucosamine increases the amount of branch\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:506227 ! N-acetyl-D-glucosamine) increases p(PFAM:PF02485 ! branch)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N-acetyl-D-glucosamine increases the amount of N-glycan\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:506227 ! N-acetyl-D-glucosamine) increases a(CHEBI:59520 ! N-glycan)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N-acetyl-beta-neuraminic acid decreases the amount of P02080\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\") decreases p(UP:P02080 ! P02080)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These NA-inhibitors are most commonly N-Acetylneuraminic acid (Neu5Ac) derivatives, which competitively inhibit NA activity [104]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD209 activates phagocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1641 ! CD209) increases bp(GO:\"GO:0006909\" ! phagocytosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "(C) The membrane bound C-type lectin receptor, DC-SIGN recognises oligomannose-type glycans via a Ca2+ ion and initiates phagocytosis of the pathogen (PDB ID: 1SL4) [216]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD209 increases the amount of ID1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1641 ! CD209) increases p(HGNC:5360 ! ID1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "(C) The membrane bound C-type lectin receptor, DC-SIGN recognises oligomannose-type glycans via a Ca2+ ion and initiates phagocytosis of the pathogen (PDB ID: 1SL4) [216]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endoplasmic reticulum decreases the amount of glycoprotein\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\") decreases a(CHEBI:17089 ! glycoprotein)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease increases the amount of MYO1G\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) increases p(HGNC:13880 ! MYO1G)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The surface glycoprotein HA exists as a trimer of hetero-dimers that is synthesised as a single polypeptide (HA0) that is subsequently cleaved into the receptor binding HA1 domain and the membrane-fusion mediating HA2 subunit by host cell proteases."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease increases the amount of P84849\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) increases p(UP:P84849 ! P84849)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Once MBL binds to viral glycoproteins, it recruits mannose-binding lectin-associated serine proteases (MASPs), which trigger the lectin pathway of the complement system [[277], [278], [279]]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydrogen-Ion Concentration decreases the amount of GYPC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006863 ! \"Hydrogen-Ion Concentration\") decreases p(HGNC:4704 ! GYPC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the case of Lassa virus, the associated lowered pH induces a conformational change of the GPC that leads to binding to lysosomal-associated membrane protein 1 (LAMP1) [201,264]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GYPC increases the amount of LAMP1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4704 ! GYPC) increases p(HGNC:6499 ! LAMP1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the case of Lassa virus, the associated lowered pH induces a conformational change of the GPC that leads to binding to lysosomal-associated membrane protein 1 (LAMP1) [201,264]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Complement activation activates innate immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006956\" ! \"complement activation\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On one hand, these glycans can trigger a sterilising innate immune response, for example by the activation of the complement cascade."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Polypeptide increases the amount of MYO1G\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15841 ! polypeptide) increases p(HGNC:13880 ! MYO1G)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The surface glycoprotein HA exists as a trimer of hetero-dimers that is synthesised as a single polypeptide (HA0) that is subsequently cleaved into the receptor binding HA1 domain and the membrane-fusion mediating HA2 subunit by host cell proteases."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LGALS1 activates HIV Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6561 ! LGALS1) increases bp(MESH:D015658 ! \"HIV Infections\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, galectin-1 has also been shown to promote HIV infection of T cells by cross-linking the virus to host cells [298,299]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LGALS1 inhibits membrane fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6561 ! LGALS1) decreases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Galectin-1 has been shown to bind to the envelope fusion glycoprotein (F) of Nipah [[294], [295], [296]], Hendra [294], and HA of influenza [297] viruses and inhibit membrane fusion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CALR inhibits endoplasmic reticulum\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1455 ! CALR) decreases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CANX inhibits endoplasmic reticulum\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1473 ! CANX) decreases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ER chaperones, calnexin and calreticulin, bind to this substrate glycan to sequester the glycoprotein in the ER such that the protein is able to properly fold."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MGAT1 increases the amount of branch\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7044 ! MGAT1) increases p(PFAM:PF02485 ! branch)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MGAT1 increases the amount of N-glycan\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7044 ! MGAT1) increases a(CHEBI:59520 ! N-glycan)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The action of GlcNAc transferase-I (GlcNAcT-I) initiates the first branch of the N-glycan."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MaspS increases the amount of P84849\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q55C99 ! Q55C99) increases p(UP:P84849 ! P84849)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Once MBL binds to viral glycoproteins, it recruits mannose-binding lectin-associated serine proteases (MASPs), which trigger the lectin pathway of the complement system [[277], [278], [279]]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypoxia upregulates activity and expression of the glucose transporter {GLUT1} in alveolar epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/10572068)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Lactate is a natural suppressor of {RLR} signaling by targeting {MAVS}.](https://pubmed.ncbi.nlm.nih.gov/31155231)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The renin-angiotensin-aldosterone system and glucose homeostasis.](https://pubmed.ncbi.nlm.nih.gov/21880378)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High glucose activates the alternative {ACE2}/Ang-(1-7)/Mas and {APN}/Ang {IV}/{IRAP} {RAS} axes in pancreatic \\beta-cells.](https://pubmed.ncbi.nlm.nih.gov/23942780)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role of macrophages in the pathogenesis of {ALI}/{ARDS}.](https://pubmed.ncbi.nlm.nih.gov/29950923)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Rhinovirus induces an anabolic reprogramming in host cell metabolism essential for viral replication.](https://pubmed.ncbi.nlm.nih.gov/29987044)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Sodium and chloride concentrations, {pH}, and depth of airway surface liquid in distal airways.](https://pubmed.ncbi.nlm.nih.gov/14557401)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Ketogenic diets potentially reverse Type {II} diabetes and ameliorate clinical depression: A case study.](https://pubmed.ncbi.nlm.nih.gov/31336509)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Tight junctions in pulmonary epithelia during lung inflammation.](https://pubmed.ncbi.nlm.nih.gov/27921210)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### TNF decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11892 ! TNF) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed, increased TNF-alpha levels attenuate cldn5 expression in a mouse model of acute lung inflammation [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFkappaB decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:NFkappaB ! NFkappaB) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Even in the absence of inflammation, inhibition of constitutive basal NFkappaB activity in human airway epithelial cells causes an up-regulation of cldn5 expression, disturbs TJ organisation and increases paracellular permeabilty [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFkappaB decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:NFkappaB ! NFkappaB) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibition of basal NFkappaB activity in airway epithelial cells increases cldn5 expression in the absence of inflammation [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFkappaB decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:NFkappaB ! NFkappaB) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Overall, NFkappaB dependent reduction of cldn5 seems to improve epithelial barrier function during lung inflammation and seems to be beneficial with respect to epithelial function."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IFNG increases the amount of CLDN7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5438 ! IFNG) increases p(HGNC:2049 ! CLDN7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFN-gamma enhances cldn7 expression and the transepithelial electrical resistance in submandibular glands [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CLDN7 increases the amount of CLDN7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2049 ! CLDN7) increases p(HGNC:2049 ! CLDN7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Phosphorylation of cldn7 within its C-terminal, intracellular domain via WNK4 kinase modulates cldn7 permeability."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CLDN7 decreases the amount of chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2049 ! CLDN7) decreases a(CHEBI:17996 ! chloride)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Silencing of cldn7 expression in LLC-PK1 reduces transepithelial resistance and increases paracellular permselectivity for Na + over Cl - [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL4 increases the amount of EGFR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6014 ! IL4) increases p(HGNC:3236 ! EGFR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In Calu3 epithelia, IL-4 is reported to activate an EGFR-dependent MAPK/ERK1/2 pathway [100], whereas in human bronchial cell derived epithelia IL-4 as well as IL-13 act via the Janus kinase/signal transducer and activator of transcription (JAK/STAT) pathway [107], most likely via binding to the same receptor, namely heteromeric IL-13R/IL-4R receptor [139]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFASC activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29866 ! NFASC) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of NFB also induces a pro-inflammatory response, and thus, it induces the production and release of a variety of proinflammatory factors, which potentially interferes with TJ integrity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFASC decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29866 ! NFASC) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibition of basal NFB activity in airway epithelial cells increases cldn5 expression in the absence of inflammation [134]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFASC decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29866 ! NFASC) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Even in the absence of inflammation, inhibition of constitutive basal NFB activity in human airway epithelial cells causes an up-regulation of cldn5 expression, disturbs TJ organisation and increases paracellular permeabilty [134]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFASC decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29866 ! NFASC) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Overall, NFB-dependent reduction of cldn5 seems to improve epithelial barrier function during lung inflammation and seems to be beneficial with respect to epithelial function."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of CLDN4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:2046 ! CLDN4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of TJP1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:11827 ! TJP1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of CLDN2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:2041 ! CLDN2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of CLDN5\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:2047 ! CLDN5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The genetic modulation of TNF- in mice revealed that TNF- negatively regulates tight junction proteins, namely cldn2, cldn4, cldn5 and ZO-1 in the lung, which results in increased alveolar permeability [86]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MAPK increases the amount of CLDN1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:MAPK ! MAPK) increases p(HGNC:2032 ! CLDN1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Phosphorylation at its N-terminal domain by MAP kinases enhances cldn1s sealing properties [37]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL13 decreases the amount of CLDN18\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5973 ! IL13) decreases p(HGNC:2039 ! CLDN18)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In mouse, lung IL-13 reduces cldn18 while increasing cldn4 expression [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL13 decreases the amount of CLDN18\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5973 ! IL13) decreases p(HGNC:2039 ! CLDN18)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In mouse, lung IL-13 reduces cldn18 while increasing cldn4 expression [118]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glucocorticoid increases the amount of CLDN8\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24261 ! glucocorticoid) increases p(HGNC:2050 ! CLDN8)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the airway epithelium , cldn8 is up-regulated by glucocorticoids but not by mineralocorticoids [ 66 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CLDN8 increases the amount of hydrogen chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2050 ! CLDN8) increases a(CHEBI:17883 ! \"hydrogen chloride\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Therefore, cldn8 increases permselectivity of TJ for Cl."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CLDN8 increases the amount of sodium atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2050 ! CLDN8) increases a(CHEBI:26708 ! \"sodium atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cldn8 mediated sealing of TJ to Na + parallels Na + absorption in human colon cells [XREF_BIBR], and hence, it is proposed that cldn8 augments sodium resorption by preventing paracellular leakage of Na +."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NR1I2 decreases the amount of CLDN1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7968 ! NR1I2) decreases p(HGNC:2032 ! CLDN1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of protease-activated receptor 2 ( PAR2 ) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [ 95 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL33 increases the amount of TH2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:16028 ! IL33) increases p(UP:F4KFT7 ! F4KFT7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Permeability activates Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D010539 ! Permeability) increases bp(MESH:D013577 ! Syndrome)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These life-threatening syndromes are caused by increased permeability of the alveolar and airway epithelium and exudate formation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mineralocorticoid decreases the amount of CLDN8\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:25354 ! mineralocorticoid) decreases p(HGNC:2050 ! CLDN8)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the airway epithelium , cldn8 is up-regulated by glucocorticoids but not by mineralocorticoids [ 66 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL25 increases the amount of TH2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13765 ! IL25) increases p(UP:F4KFT7 ! F4KFT7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TSLP activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:30743 ! TSLP) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TSLP increases the amount of TH2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:30743 ! TSLP) increases p(UP:F4KFT7 ! F4KFT7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "TSLP, IL-25 and IL-33 activate T helper cell type 2 (TH2)-driven inflammation, which is dominated by eosinophils (eos)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Receptor, PAR-2 decreases the amount of CLDN1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D044464 ! \"Receptor, PAR-2\") decreases p(HGNC:2032 ! CLDN1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of protease-activated receptor 2 (PAR2) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [95]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Receptor, PAR-2 decreases the amount of CLDN1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D044464 ! \"Receptor, PAR-2\") decreases p(HGNC:2032 ! CLDN1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of protease activated receptor 2 (PAR2) transiently down-regulates cldn1 expression and decreases epithelial permeability with a similar time course [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LLC-PK1 increases the amount of CLDN7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:LLC-PK1) increases p(HGNC:2049 ! CLDN7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Overexpressing cldn7 in the porcine kidney, epithelial cell line LLC-PK1 resulted in an increased transepithelial electrical resistance via reducing paracellular Cl permeability while forming a Na+ pore."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Effect of television on obesity and excess of weight and consequences of health.](https://pubmed.ncbi.nlm.nih.gov/26274965)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Expression of {MUC5AC} and {MUC5B} mucins in normal and cystic fibrosis lung.](https://pubmed.ncbi.nlm.nih.gov/11860173)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Sweet talk: insights into the nature and importance of glucose transport in lung epithelium.](https://pubmed.ncbi.nlm.nih.gov/22878875)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Lactate and regulation of lung glycolytic rate.](https://pubmed.ncbi.nlm.nih.gov/6720945)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Immunological Impacts of Diabetes on the Susceptibility of Mycobacterium tuberculosis.](https://pubmed.ncbi.nlm.nih.gov/31583258)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity and clotting: Body mass index independently contributes to hypercoagulability after injury.](https://pubmed.ncbi.nlm.nih.gov/25539200)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia: a prothrombotic factor?](https://pubmed.ncbi.nlm.nih.gov/20492456)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hospitalized cases of influenza A({H1N1})pdm09 in the French territories of the Americas, July 2009-March 2010.](https://pubmed.ncbi.nlm.nih.gov/23099873)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Ketogenic diet activates protective \\gamma\\delta T cell responses against influenza virus infection.](https://pubmed.ncbi.nlm.nih.gov/31732517)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The Ketogenic Diet: Evidence for Optimism but High-Quality Research Needed.](https://pubmed.ncbi.nlm.nih.gov/31825066)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Consequences of hypoxia for the pulmonary alveolar epithelial cell innate immune response.](https://pubmed.ncbi.nlm.nih.gov/30381478)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Macrophage polarization in chronic inflammatory diseases: killers or builders?](https://pubmed.ncbi.nlm.nih.gov/29507865)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Structure, function, and evolution of coronavirus spike proteins.](https://pubmed.ncbi.nlm.nih.gov/27578435)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Middle East Respiratory Syndrome Coronavirus activates Virus Internalization\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\") increases bp(MESH:D053586 ! \"Virus Internalization\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Interestingly , despite recognizing the same receptor DPP4 , MERS-CoV and HKU4 spikes differ in their activities to mediate virus entry : HKU4 spike mediates virus entry into bat cells but not human cells , whereas MERS-CoV spike mediates virus entry into both bat and human cells ( 152 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role of respiratory epithelium in host defence against influenza virus infection.](https://pubmed.ncbi.nlm.nih.gov/30348265)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Viral hijacking of cellular metabolism.](https://pubmed.ncbi.nlm.nih.gov/31319842)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Trajectories of {BMI} change impact glucose and insulin metabolism.](https://pubmed.ncbi.nlm.nih.gov/29361342)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Circulating surfactant protein-D and the risk of cardiovascular morbidity and mortality.](https://pubmed.ncbi.nlm.nih.gov/21653561)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China.](https://pubmed.ncbi.nlm.nih.gov/32031570)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High risk of coagulopathy among Type-2 Diabetes Mellitus clients at a municipal hospital in Ghana.](https://pubmed.ncbi.nlm.nih.gov/29622820)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Staphylococcus aureus and Influenza A Virus: Partners in Coinfection.](https://pubmed.ncbi.nlm.nih.gov/27965455)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Ketogenic diet and epilepsy: what we know so far.](https://pubmed.ncbi.nlm.nih.gov/30760973)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Ketogenic diet in alzheimer's disease.](https://pubmed.ncbi.nlm.nih.gov/31405021)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [A crucial role of angiotensin converting enzyme 2 ({ACE2}) in {SARS} coronavirus-induced lung injury.](https://pubmed.ncbi.nlm.nih.gov/16007097)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SLC33A1 inhibits Acute Lung Injury\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:95 ! SLC33A1) decreases bp(MESH:D055371 ! \"Acute Lung Injury\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For AT1 inhibition of Spike-Fc-mediated acute lung injury, we treated the Spike(S1190)-Fc-challenged mice with the AT1 inhibitor losartan (15 mg/kg)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The global epidemiology of hypertension.](https://pubmed.ncbi.nlm.nih.gov/32024986)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Global trends in diabetes complications: a review of current evidence.](https://pubmed.ncbi.nlm.nih.gov/30171279)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role of collectins and galectins in lung innate immune defense.](https://pubmed.ncbi.nlm.nih.gov/30233589)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Clinical predictors of mortality due to {COVID}-19 based on an analysis of data of 150 patients from Wuhan, China.](https://pubmed.ncbi.nlm.nih.gov/32125452)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity is Associated with Higher Blood Pressure and Higher Levels of Angiotensin {II} but Lower Angiotensin-(1-7) in Adolescents Born Preterm.](https://pubmed.ncbi.nlm.nih.gov/30404738)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impact of obesity on influenza A virus pathogenesis, immune response, and evolution.](https://pubmed.ncbi.nlm.nih.gov/31134099)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Influenza virus and glycemic variability in diabetes: A killer combination?](https://pubmed.ncbi.nlm.nih.gov/28588558)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypoxia signaling in human diseases and therapeutic targets.](https://pubmed.ncbi.nlm.nih.gov/31221962)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Coronavirus cell entry occurs through the endo-/lysosomal pathway in a proteolysis-dependent manner.](https://pubmed.ncbi.nlm.nih.gov/25375324)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Fi inhibits Middle East Respiratory Syndrome Coronavirus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P03709 ! P03709) decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MERS-CoV, which contains a minimal FCS, is inhibited by furin inhibitor (FI) but not by the pan-lysosomal protease inhibitor (CPI)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease activates SARS Virus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) increases bp(MESH:D045473 ! \"SARS Virus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibitors of cathepsin proteases have also been shown to inhibit entry of SARS-CoV and feline CoVs [23], [37], [38], while treatment of cell-bound virus particles with different proteases was shown to enhance virus entry and/or cell-cell fusion [27], [34], [39][45]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endocytosis decreases the amount of S\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006897\" ! endocytosis) decreases p(UP:P0DTC2 ! P0DTC2, var(\"p.?\"))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Consistently, infections with MHV carrying wild type or mutant S protein were equally blocked by inhibitors of clathrin mediated endocytosis whereas the virus with the mutant S (MHV-S2 ' FCS) was much less sensitive to inhibitors of endosomal maturation, including BafA1, or to perturbants of the actin cytoskeleton."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Actin decreases the amount of SLC9A1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Actin ! Actin) decreases p(HGNC:11071 ! SLC9A1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The actin- and macropinocytosis-affecting drug EIPA, which inhibits the Na + /H + exchanger NHE1, led to reduced luciferase expression both when added prior to and after entry of MHV at 2 hpi."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Azanediyl (triplet) inhibits clathrin-dependent endocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29339 ! \"azanediyl (triplet)\") decreases bp(GO:\"GO:0072583\" ! \"clathrin-dependent endocytosis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Inhibition of endosomal maturation by NH 4 Cl, BafA1 and Chloq ( Figure 3B , deep sky blue) or of clathrin-mediated endocytosis by Chlopro, Mon, and Dynasore ( Figure 3B , dark blue) severely inhibited MHV fusion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RNA interference activates negative regulation of endocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0016246\" ! \"RNA interference\") increases bp(GO:\"GO:0045806\" ! \"negative regulation of endocytosis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "RNAi-mediated downregulation of endocytosis-associated proteins affects MHV infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FURIN activates Middle East Respiratory Syndrome Coronavirus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8568 ! FURIN) increases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MERS-CoV , which contains a minimal FCS , is inhibited by furin inhibitor ( FI ) but not by the pan-lysosomal protease inhibitor ( CPI ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAB5B inhibits Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9784 ! RAB5B) decreases bp(MESH:D007239 ! Infections)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, infection with MHV-S2FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 (Figure 9, blue bars)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAB5B inhibits Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9784 ! RAB5B) decreases bp(MESH:D007239 ! Infections)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, infection with MHV-S29FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 ( Figure 9 , blue bars)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAB5C inhibits Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9785 ! RAB5C) decreases bp(MESH:D007239 ! Infections)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, infection with MHV-S2FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 (Figure 9, blue bars)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RAB5C inhibits Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9785 ! RAB5C) decreases bp(MESH:D007239 ! Infections)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, infection with MHV-S29FCS was significantly diminished by downregulation of the early endosomal proteins RAB5B and RAB5C, but not of the late endosomal proteins RAB7A and RAB7B or the HOPS complex components VPS11 and VPS41 ( Figure 9 , blue bars)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Prevalence of comorbidities and its effects in patients infected with {SARS}-{CoV}-2: a systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/32173574)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### TNF increases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11892 ! TNF) increases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators, especially IL-1beta and TNFalpha, which promote systemic insulin resistance and beta-cell damage."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TNF inhibits B-Lymphocytes\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11892 ! TNF) decreases bp(MESH:D001402 ! B-Lymphocytes)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance , diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators , especially , IL-1beta and TNFalpha , which promotes systemic insulin resistance and beta-cell damage 17 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL1B increases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5992 ! IL1B) increases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators, especially IL-1beta and TNFalpha, which promote systemic insulin resistance and beta-cell damage."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL1B inhibits B-Lymphocytes\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5992 ! IL1B) decreases bp(MESH:D001402 ! B-Lymphocytes)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance , diabetes occurs in part because the accumulation of activated innate immune cells in metabolic tissues leads to the release of inflammatory mediators , especially , IL-1beta and TNFalpha , which promotes systemic insulin resistance and beta-cell damage 17 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Prominent changes in blood coagulation of patients with {SARS}-{CoV}-2 infection.](https://pubmed.ncbi.nlm.nih.gov/32172226)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Barrier function of airway tract epithelium.](https://pubmed.ncbi.nlm.nih.gov/24665407)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Lipopolysaccharide increases the amount of DEFB4A\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:2767 ! DEFB4A)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR - XREF_BIBR While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-kappaB activation and by IL-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lipopolysaccharide increases the amount of DEFB4B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:30193 ! DEFB4B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "[141] [142] [143] While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-B activation and by IL-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Infections increases the amount of DEFB4B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007239 ! Infections) increases p(HGNC:30193 ! DEFB4B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "144, 145 hBD2 is induced by P. aeruginosa infection in normal but not in cystic fibrosis airway epithelia."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Nitric oxide increases the amount of NOS1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16480 ! \"nitric oxide\") increases p(HGNC:7872 ! NOS1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Three NO synthases contribute to production of NO in airway epithelia: the constitutively expressed NOS1 and NOS3, and inducible NOS2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Nitric oxide increases the amount of NOS1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16480 ! \"nitric oxide\") increases p(HGNC:7872 ! NOS1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Three NO synthases contribute to production of NO in airway epithelia : the constitutively expressed NOS1 and NOS3, and inducible NOS2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Nitric oxide increases the amount of NOS2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16480 ! \"nitric oxide\") increases p(HGNC:7873 ! NOS2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Three NO synthases contribute to production of NO in airway epithelia: the constitutively expressed NOS1 and NOS3, and inducible NOS2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Nitric oxide increases the amount of NOS2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16480 ! \"nitric oxide\") increases p(HGNC:7873 ! NOS2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Three NO synthases contribute to production of NO in airway epithelia : the constitutively expressed NOS1 and NOS3, and inducible NOS2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) increases the amount of hydrogen chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") increases a(CHEBI:17883 ! \"hydrogen chloride\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) increases the amount of hydrogen chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") increases a(CHEBI:17883 ! \"hydrogen chloride\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) increases the amount of chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") increases a(CHEBI:17996 ! chloride)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) increases the amount of CLCA1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") increases p(HGNC:2015 ! CLCA1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) increases the amount of ubiC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") increases p(UP:A3M775 ! A3M775)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of hydrogen chloride\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases a(CHEBI:17883 ! \"hydrogen chloride\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of ASL\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(HGNC:746 ! ASL)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium activated Cl - channel (CaCC), outwardly rectifying Cl - channel (ORCC), Cl - channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of ASL\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(HGNC:746 ! ASL)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl  channel (CaCC), outwardly rectifying Cl  channel (ORCC), Cl  channel 2 (CLC2), and epithelial Na + channel (ENaC) regulate the ASL hydration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP increases the amount of ubiC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases p(UP:A3M775 ! A3M775)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In normal airways, the coordinated functioning of ATP-activated cystic fibrosis transmembrane conductance regulator (CFTR), calcium-activated Cl channel (CaCC), outwardly rectifying Cl- channel (ORCC), Cl- channel 2 (CLC2), and epithelial Na+ channel (ENaC) regulate the ASL hydration.54 CFTR negatively regulates ENac and therefore absent or dsyfunctional CFTR increases ENaC activity leading to hyperabsorption of Na+, an increased driving force for fluid reabsorption resulting in reduced ASL depth and impaired mucociliary clearance as observed in the chronic airway disease cystic fibrosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL1 increases the amount of DEFB4A\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:IL1 ! IL1) increases p(HGNC:2767 ! DEFB4A)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR - XREF_BIBR While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-kappaB activation and by IL-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL1 increases the amount of DEFB4B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:IL1 ! IL1) increases p(HGNC:30193 ! DEFB4B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "[141] [142] [143] While hBD1 is constitutively expressed, hBD2 to hBD4 expression is induced by LPS via NF-B activation and by IL-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease decreases the amount of SLPI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) decreases p(HGNC:11092 ! SLPI)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In COPD patients, levels of SLPI and lysozyme were shown to decrease with bacterial infection, while lactoferrin levels remain unchanged. xref This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease decreases the amount of SLPI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) decreases p(HGNC:11092 ! SLPI)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Protease decreases the amount of SLPI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Protease ! Protease) decreases p(HGNC:11092 ! SLPI)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLPI decreases the amount of ELA\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11092 ! SLPI) decreases p(FPLX:ELA ! ELA)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLPI increases the amount of SLPI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11092 ! SLPI) increases p(HGNC:11092 ! SLPI)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "139 This could be due to inactivation of SLPI by proteases or decreased expression of SLPI, but either way the end result under these conditions is an imbalance in the ratio of antiproteases to proteases in the airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLPI decreases the amount of CXCL8\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11092 ! SLPI) decreases p(HGNC:6025 ! CXCL8)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLPI decreases the amount of CXCL8\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11092 ! SLPI) decreases p(HGNC:6025 ! CXCL8)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Accordingly, administration of SLPI decreased the levels of IL-8 and elastase activity in airway secretion of cystic fibrosis patients, who also have reduced SLPI in their airway lumen."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Asthma decreases the amount of SPDEF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001249 ! Asthma) decreases p(HGNC:17257 ! SPDEF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "65 Inhibition of aldose reductase or SERPINb3a, both of which are induced in asthma reduced SPDEF expression, attenuates development of goblet cell hyperplasia."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF activates metaplastic ossification\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) increases bp(GO:\"GO:0036074\" ! \"metaplastic ossification\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF inhibits metaplastic ossification\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) decreases bp(GO:\"GO:0036074\" ! \"metaplastic ossification\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF increases the amount of NKX2-1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) increases p(HGNC:11825 ! NKX2-1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF increases the amount of NKX2-1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) increases p(HGNC:11825 ! NKX2-1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR While both TTF-1 and FOXA2 repress goblet cell metaplasia, XREF_BIBR, XREF_BIBR SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF increases the amount of FOXA2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) increases p(HGNC:5022 ! FOXA2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPDEF increases the amount of FOXA2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:17257 ! SPDEF) increases p(HGNC:5022 ! FOXA2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR While both TTF-1 and FOXA2 repress goblet cell metaplasia, XREF_BIBR, XREF_BIBR SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL13 activates Hypertrophy\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5973 ! IL13) increases bp(MESH:D006984 ! Hypertrophy)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia, and submucosal gland hypertrophy resulting in obstruction of airways.Goblet cell metaplasia and hyperplasia are also observed in patients with other chronic airway diseases such as asthma and chronic obstructive pulmonary disease (COPD), and is induced by the coordinated action of EGFR and IL-13."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  EGFR activates Hypertrophy\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3236 ! EGFR) increases bp(MESH:D006984 ! Hypertrophy)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia, and submucosal gland hypertrophy resulting in obstruction of airways.Goblet cell metaplasia and hyperplasia are also observed in patients with other chronic airway diseases such as asthma and chronic obstructive pulmonary disease (COPD), and is induced by the coordinated action of EGFR and IL-13."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF decreases the amount of iron atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases a(CHEBI:18248 ! \"iron atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "132 Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF decreases the amount of iron atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases a(CHEBI:18248 ! \"iron atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF decreases the amount of 13\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases p(UP:Q9T1T5 ! Q9T1T5)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin, which disrupts the outer membrane, allowing lysozyme to gain access to the peptidoglycan layer."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF decreases the amount of peptidoglycan\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases a(CHEBI:8005 ! peptidoglycan)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin, which disrupts the outer membrane, allowing lysozyme to gain access to the peptidoglycan layer."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF inhibits outer membrane\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases bp(GO:\"GO:0019867\" ! \"outer membrane\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LTF inhibits outer membrane\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6720 ! LTF) decreases bp(GO:\"GO:0019867\" ! \"outer membrane\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lysozyme , an enzyme found in airway epithelial secretions , exerts antimicrobial effect against a wide range of gram-positive bacteria by degrading their peptidoglycan layer.131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer.132 Lactoferrin is an iron-chelator and inhibits microbial growth by sequestering iron which is essential for microbial respiration.133 Lactoferrin also displays antiviral activity against both RNA and DNA viruses by either inhibiting binding of the virus to host cells or by binding to the virus itself.134 ,135 Lactoferrin levels increase in response to bacterial and viral infections ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hyperplasia activates Mucus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006965 ! Hyperplasia) increases bp(MESH:D009093 ! Mucus)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "54 In cystic fibrosis patients this condition is further exacerbated by excessive mucus production due to goblet cell metaplasia and hyperplasia , and submucosal gland hypertrophy resulting in obstruction of airways ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TECR decreases the amount of PATJ\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4551 ! TECR) decreases p(HGNC:28881 ! PATJ)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "101, 102 Depletion of PALS1 leads to the loss of PATJ, disruption of cell polarity, decreased TER, and altered trafficking of E-cadherin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CDH1 increases the amount of PATJ\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1748 ! CDH1) increases p(HGNC:28881 ! PATJ)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "101, 102 Depletion of PALS1 leads to the loss of PATJ, disruption of cell polarity, decreased TER, and altered trafficking of E-cadherin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LYZ increases the amount of LYZ\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6740 ! LYZ) increases p(HGNC:6740 ! LYZ)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "131 Lysozyme is also effective against gram-negative bacteria in the presence of lactoferrin , which disrupts the outer membrane , allowing lysozyme to gain access to the peptidoglycan layer ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DUOX1 increases the amount of hydrogen peroxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3062 ! DUOX1) increases a(CHEBI:16240 ! \"hydrogen peroxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "123 Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DUOX1 increases the amount of hydrogen peroxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3062 ! DUOX1) increases a(CHEBI:16240 ! \"hydrogen peroxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Viral infections and pro-inflammatory cytokines induce expression of NOS2 and defective NOS2 expression is responsible for increased viral replication in cystic fibrosis, while overexpression of NOS2 provides protection against viral infection.150,151 Recently we demonstrated that COPD airway epithelial cells show a trend in decreased expression of NOS2 and this was associated with impaired clearance of rhinovirus.123 Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DUOX1 increases the amount of hydrogen peroxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3062 ! DUOX1) increases a(CHEBI:16240 ! \"hydrogen peroxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Extracellular hydrogen peroxide is produced by dual oxidase 1 and 2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Epithelial Cells decreases the amount of Protease\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004847 ! \"Epithelial Cells\") decreases p(FPLX:Protease ! Protease)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "137 , 138 Epithelial cells produce protease inhibitors , such as secretory leukoprotease inhibitor ( SLPI ) , elastase inhibitor , alpha1-antiprotease , and antichymotrypsin ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bacteria decreases the amount of DEFB4A\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001419 ! Bacteria) decreases p(HGNC:2767 ! DEFB4A)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "135 Herr et al. showed that hBD2 is significantly reduced in the pharyngeal wash and sputum of current or former smokers compared with non-smokers , and exposure of airway epithelium to cigarette smoke in vitro inhibited induction of HBD2 by bacteria ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NKX2-1 inhibits metaplastic ossification\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11825 ! NKX2-1) decreases bp(GO:\"GO:0036074\" ! \"metaplastic ossification\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Goblet Cells increases the amount of MUC5AC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D020397 ! \"Goblet Cells\") increases p(HGNC:7515 ! MUC5AC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "40 MUC5AC is mainly produced by goblet cells , while MUC5B is predominantly produced by submucosal glands ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Rhinovirus increases the amount of translocation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012229 ! Rhinovirus) increases a(TEXT:translocation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Coxsackievirus and adenovirus bind to CAR , inducing disassembly of the tight junction and reduction in transepithelial resistance ( TER ) .109,110 Previously , we demonstrated that rhinovirus ( RV ) dissociates ZO-1 , occludin , and claudin 4 from the tight junctions and increases bacterial association and translocation across polarized airway epithelium.93 ,111,112 However , our studies have not shown binding of RV to tight junction proteins ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Rhinovirus increases the amount of translocation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012229 ! Rhinovirus) increases a(TEXT:translocation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "109 , 110 Previously , we demonstrated that rhinovirus ( RV ) dissociates ZO-1 , occludin , and claudin 4 from the tight junctions and increases bacterial association and translocation across polarized airway epithelium ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FOXA2 inhibits metaplastic ossification\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5022 ! FOXA2) decreases bp(GO:\"GO:0036074\" ! \"metaplastic ossification\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "60 While both TTF-1 and FOXA2 repress goblet cell metaplasia, 61,62 SPDEF promotes goblet cell metaplasia by downregulating FOXA2 and TTF-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  EGFR binds CDH1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR)) decreases p(HGNC:1748 ! CDH1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CDH1 bound to EGFR decreases the amount of CDH1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR)) decreases p(HGNC:1748 ! CDH1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  EGFR binds CDH1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR)) decreases p(HGNC:3236 ! EGFR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CDH1 bound to EGFR decreases the amount of EGFR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:1748 ! CDH1), p(HGNC:3236 ! EGFR)) decreases p(HGNC:3236 ! EGFR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Under normal conditions, E-cadherin interacts and retains EGFR in adherens junctions of airway epithelium,116 thus preventing EGFR activation.117-119 Dissociation of E-cadherin from the adherens junction complex or reduced expression of E-cadherin, each of which may occur during epithelial to mesenchymal transition, may cause uncoupling and redistribution of EGFR from the adherens junction to the apical cell surface,120 where it is readily activated by EGF ligands."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{SARS}-{CoV}-2: What do we know so far?](https://pubmed.ncbi.nlm.nih.gov/32220035)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Acyclovir decreases the amount of amantadine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:2453 ! acyclovir) decreases a(CHEBI:2618 ! amantadine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The modes of action of approved antivirals include inhibition of virus entry or fusion (enfuvirtide), inhibition of uncoating (amantadine), inhibition of nucleic acid synthesis (acyclovir, ribavirin), maturation inhibitors (lopinavir, ritonavir) and inhibitors of virus release (oseltamivir)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Losartan decreases the amount of AGT\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6541 ! losartan) decreases p(HGNC:333 ! AGT)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "36, 37 Angiotensin I receptor blockers such as losartan are thought to reduce lung injury in SARS cases and can potentially also be used in treatment of COVID-19."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy.](https://pubmed.ncbi.nlm.nih.gov/16873249)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The Effect of Short-Term Hyperglycemia on the Innate Immune System.](https://pubmed.ncbi.nlm.nih.gov/26897277)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Anticoagulant treatment is associated with decreased mortality in severe coronavirus disease 2019 patients with coagulopathy.](https://pubmed.ncbi.nlm.nih.gov/32220112)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Heparin, Low-Molecular-Weight inhibits COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006495 ! \"Heparin, Low-Molecular-Weight\") decreases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "No evidence text."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{SARS}-{CoV}-2 ({COVID}-19) by the numbers.](https://pubmed.ncbi.nlm.nih.gov/32228860)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 pandemic, coronaviruses, and diabetes mellitus.](https://pubmed.ncbi.nlm.nih.gov/32228322)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### AGT activates cell population proliferation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:333 ! AGT) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "AngII, through the activation of Ang II type 1a receptors induces vasoconstriction and proliferation, whereas Ang 1-7 stimulates vasodilatation and suppresses cell growth (XREF_FIG)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Insulin administration attenuates ACE2 expression, while hypoglycemic agents such as glucagon like peptide-1 (GLP-1) agonists (liraglutide) and thiazolidinediones (TZDs; pioglitazone), antihypertensives such as ACE inhibitors, and statins upregulate ACE2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Comorbid diabetes results in immune dysregulation and enhanced disease severity following {MERS}-{CoV} infection.](https://pubmed.ncbi.nlm.nih.gov/31550243)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Mice decreases the amount of CCL2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:10618 ! CCL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "We found that diabetic male mice had significantly reduced levels of Ccl2 expression at days 2 (P = 0.0084) and 7 (P = 0.0014) after infection ( Figure 6A )."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of TNF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:11892 ! TNF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Diabetic male mice had reduced upregulation of Tnfa at 2 (P = 0.0145) and 7 (P = 0.0041) days after infection ( Figure 6B ) and reduced upregulation of Il6 2 days after infection ( Figure 6C) ; however, Nos2 expression was upregulated similarly in the lungs of male diabetic and control mice during MERS-CoV infection ( Figure 6E )."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice increases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) increases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Diabetic male mice had reduced upregulation of Tnfa at 2 (P = 0.0145) and 7 (P = 0.0041) days after infection ( Figure 6B ) and reduced upregulation of Il6 2 days after infection ( Figure 6C) ; however, Nos2 expression was upregulated similarly in the lungs of male diabetic and control mice during MERS-CoV infection ( Figure 6E )."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice activates Obesity\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) increases bp(MESH:D009765 ! Obesity)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Male C57BL/6 mice are more susceptible than females to diet-induced obesity and the subsequent development of metabolic disease (40, 54) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD4 increases the amount of cytokine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1678 ! CD4) increases a(TEXT:cytokine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Next , we looked at cytokines that CD4 + T cells produce ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD4 activates pathogenesis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1678 ! CD4) increases bp(GO:\"GO:0009405\" ! pathogenesis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "If and how CD4 + T cells contribute to MERS-CoV pathogenesis is not clear, but we found alterations in the CD4 + T cell response to MERS-CoV infection in diabetic mice although to a lesser extent than we found differences in the IMM response."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bacterial Infections decreases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001424 ! \"Bacterial Infections\") decreases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This response is similar to what is observed after bacterial infection in which macrophages from diabetic mice or humans have been shown to have reduced phagocytic function and expression of TNF-, IL-6, CCL2, and IL-12 (65) (66) (67) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bacterial Infections decreases the amount of CCL2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001424 ! \"Bacterial Infections\") decreases p(HGNC:10618 ! CCL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This response is similar to what is observed after bacterial infection in which macrophages from diabetic mice or humans have been shown to have reduced phagocytic function and expression of TNF-, IL-6, CCL2, and IL-12 (65) (66) (67) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cell increases the amount of IL17F\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0005623\" ! cell) increases p(HGNC:16404 ! IL17F)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In diabetic humans infected with M. tuberculosis , CD8 + T cell numbers in circulation were similar to healthy patients , but a higher frequency of these cells produced IFN , IL-2 , and IL-17F along with having decreased antigen-specific expression of cytotoxic markers ( 58 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cell increases the amount of IL2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0005623\" ! cell) increases p(HGNC:6001 ! IL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In diabetic humans infected with M. tuberculosis , CD8 + T cell numbers in circulation were similar to healthy patients , but a higher frequency of these cells produced IFN , IL-2 , and IL-17F along with having decreased antigen-specific expression of cytotoxic markers ( 58 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Virus Replication inhibits Disease\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D014779 ! \"Virus Replication\") decreases bp(MESH:D004194 ! Disease)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These data suggest that prolonged disease following MERS-CoV infection is not due to increased virus replication , a delay in virus clearance , or increased extrapulmonary dissemination of the virus ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Timely blood glucose management for the outbreak of 2019 novel coronavirus disease ({COVID}-19) is urgently needed.](https://pubmed.ncbi.nlm.nih.gov/32179126)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Glucocorticoid activates Diabetes Complications\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24261 ! glucocorticoid) increases bp(MESH:D048909 ! \"Diabetes Complications\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Catecholamine activates Blood Glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:33567 ! catecholamine) increases bp(MESH:D001786 ! \"Blood Glucose\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Catecholamine activates Diabetes Complications\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:33567 ! catecholamine) increases bp(MESH:D048909 ! \"Diabetes Complications\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Infection of SARS-CoV-2 with diabetes might trigger stress condition and increased secretion of hyperglycemic hormones , such as glucocorticoid and catecholamines , which results in elevated blood glucose , abnormal glucose variability and diabetic complications ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Post-translational modifications of coronavirus proteins: roles and function.](https://pubmed.ncbi.nlm.nih.gov/32201497)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### TP53 inhibits activation of innate immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11998 ! TP53) decreases bp(GO:\"GO:0002218\" ! \"activation of innate immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TP53 inhibits SARS Virus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11998 ! TP53) decreases bp(MESH:D045473 ! \"SARS Virus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cellular p53 inhibits replication of SARS-CoV and HCoV-NL63, presumably by activating genes involved in innate immunity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NMI increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7854 ! NMI) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, the accessory protein 6 of SARS-CoV was shown to interact with the IFN-signaling pathway-mediating protein Nmi and promote its ubiquitin-dependent proteasomal degradation, thereby potentially modulating the virus-induced innate immune response [200]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SH2D3C increases the amount of E3_Ub_ligase\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:16884 ! SH2D3C) increases p(FPLX:\"E3_Ub_ligase\" ! \"E3_Ub_ligase\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SH2D3C increases the amount of RCHY1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:16884 ! SH2D3C) increases p(HGNC:17479 ! RCHY1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SH2D3C decreases the amount of TP53\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:16884 ! SH2D3C) decreases p(HGNC:11998 ! TP53)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " The SARS-unique domain in the SARS-CoV nsp3 enhances the E3 ligase activity of RCHY1 and promotes proteasomal degradation of p53, thereby modulating the activation of innate immune response."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ZUP1 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:21224 ! ZUP1) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "At last, using a constitutively active phosphor-mimetic IRF3, it was recently shown that the DUB activity of PLPro also inhibited IRF3 at a postactivation step [196]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ADP increases the amount of NUP214\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16761 ! ADP) directlyIncreases p(HGNC:8064 ! NUP214)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " ADP-ribosylated N protein can be detected in coronavirus-infected cells and in purified virions."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Phosphorylation decreases the amount of translocation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0016310\" ! phosphorylation) decreases a(TEXT:translocation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , SARS-CoV N protein was shown to translocate to cytoplasmic stress granules in response to cellular stress , while phosphorylation in the serine-arginine rich region inhibited this translocation [ 133 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N decreases the amount of TRIM25\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P0C797 ! P0C797) decreases p(HGNC:12932 ! TRIM25)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N decreases the amount of DDX58\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P0C797 ! P0C797) decreases p(HGNC:19102 ! DDX58)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM25 inhibits type I interferon production\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:12932 ! TRIM25) decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM25 increases the amount of DDX58\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:12932 ! TRIM25) increases p(HGNC:19102 ! DDX58)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The N protein of SARS-CoV was found to bind to the SPRY domain of TRIM25 and inhibit TRIM25-dependent RIG-I activation, thereby suppressing the type I IFN production induced by poly(I:C) or Sendai virus [199]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Monensin A activates transport\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27617 ! \"monensin A\") increases bp(GO:\"GO:0006810\" ! transport)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Treatment of the Golgi transport blocker monensin inhibited the transport of MHV S protein from trans-Golgi network to the cell surface [21]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Monensin A increases the amount of S\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27617 ! \"monensin A\") increases p(UP:P0DTC2 ! P0DTC2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Treatment of the Golgi transport blocker monensin inhibited the transport of MHV S protein from trans-Golgi network to the cell surface [21]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATR leads to the phosphorylation of N\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:882 ! ATR) increases p(UP:P0DTC9 ! P0DTC9, pmod(go:0006468 ! \"protein phosphorylation\"))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, recombinant IBV harboring alanine substitutions at all four putative phosphorylation sites (S190A/S192A/T378A/S379A) could still be recovered and grew at a similar growth rate as wild-type IBV, suggesting that ATR-dependent phosphorylation of N protein is not essential for IBV replication in vitro [119]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GSK3 increases the amount of N\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:GSK3 ! GSK3) increases p(UP:P0C797 ! P0C797)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, inhibition of GSK-3 by kenpaullone significantly reduced the phosphorylation level of N protein, as well as the supernatant virus titer and cytopathic effects on VeroE6 cell-infected SARS-CoV [131]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GSK3 leads to the phosphorylation of N\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:GSK3 ! GSK3) increases p(UP:P0DTC9 ! P0DTC9, pmod(go:0006468 ! \"protein phosphorylation\"))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In fact, a recent study showed that phosphorylation of the MHV-JHM N protein by GSK-3 allowed the recruitment of RNA helicase DDX1 to facilitate template read-through, enabling the synthesis of genomic RNA and longer sgRNAs [132]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Carbohydrate binding inhibits Virus Attachment\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0030246\" ! \"carbohydrate binding\") decreases bp(MESH:D053585 ! \"Virus Attachment\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance , carbohydrate binding agents that directly interact with glycans on the virion surface may be able to suppress virus attachment and entry ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SPECC1 increases the amount of IKBKG\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:30615 ! SPECC1) increases p(HGNC:5961 ! IKBKG)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       " PDCoV nsp5 mediates cleavage of NEMO and STAT2 to suppress production and signaling of type I interferon.This work is licensed under the Creative Commons Attribution4.0 License."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  VHL inhibits SARS Virus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:12687 ! VHL) decreases bp(MESH:D045473 ! \"SARS Virus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Overexpression of VHL promoted the ubiquitin-proteasomal degradation of SARS-CoV nsp16, while knockdown of VHL increased the protein stability of nsp16 [158]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glycosylated MYOM2 increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7614 ! MYOM2, pmod(go:0006486 ! \"protein glycosylation\")) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, it was found that recombinant MHV containing N-linked glycosylated M protein induced a higher level of type I interferon compared with the wild-type MHV with O-linked glycosylated M protein, whereas MHV with nonglycosylated M protein was a poor interferon inducer in cell culture [104]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P0C710 increases the amount of (2R,3R)-2,3-dihydroxy-3-methylpentanoic acid\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P0C710 ! P0C710) increases a(CHEBI:27512 ! \"(2R,3R)-2,3-dihydroxy-3-methylpentanoic acid\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In fact, co-expression of SARS-CoV nsp3, nsp4 and nsp6 induced DMV formation in the transfected cells [143]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ubiquitinated Q80H93 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q80H93 ! Q80H93, pmod(go:0016567 ! \"protein ubiquitination\")) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The ubiquitinated 8b and 8ab may mediate rapid degradation of IRF3 and regulate host antiviral innate immunity [169]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Nutritional ketosis as an intervention to relieve astrogliosis: Possible therapeutic applications in the treatment of neurodegenerative and neuroprogressive disorders.](https://pubmed.ncbi.nlm.nih.gov/32093791)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia and the novel Covid-19 infection: Possible pathophysiologic mechanisms.](https://pubmed.ncbi.nlm.nih.gov/32240876)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Intensive care management of coronavirus disease 2019 ({COVID}-19): challenges and recommendations.](https://pubmed.ncbi.nlm.nih.gov/32272080)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Smoking Upregulates Angiotensin-Converting Enzyme-2 Receptor: A Potential Adhesion Site for Novel Coronavirus {SARS}-{CoV}-2 (Covid-19).](https://pubmed.ncbi.nlm.nih.gov/32244852)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### COVID-19 activates SARS-CoV-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086382 ! COVID-19) increases bp(MESH:D000086402 ! SARS-CoV-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 could be a novel adhesion molecule for SARS-CoV-2 causing Covid-19 and a potential therapeutic target for the prevention of fatal microbial infections, and therefore it should be fast tracked and prioritized for research and investigation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Government inhibits Smoking\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006076 ! Government) decreases bp(MESH:D012907 ! Smoking)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Governments should act to reduce smoking rates in all countries in accordance with the WHO Framework Convention on Tobacco Control ( FCTC ) , and initiate a stimulus package for health , as they have done for business , at the time of this outbreak / pandemic including all communicable pulmonary diseases and Covid-19 , as it is possible that smoking exacerbates contraction , transmission , and mortality ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High Prevalence of Obesity in Severe Acute Respiratory Syndrome Coronavirus-2 ({SARS}-{CoV}-2) Requiring Invasive Mechanical Ventilation.](https://pubmed.ncbi.nlm.nih.gov/32271993)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Metabolic Syndrome activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D024821 ! \"Metabolic Syndrome\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover , obesity and the metabolic syndrome are thought to increase type 2 inflammation , which may have effects on the lung parenchyma and bronchi ( 8) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity, Abdominal inhibits immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D056128 ! \"Obesity, Abdominal\") decreases bp(GO:\"GO:0006955\" ! \"immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore, the abnormal secretion of adipokines and cytokines like TNF-alpha and interferon characterise a chronic low-grade inflammation characteristic of abdominal obesity, which may impair immune response (7) and have effects on the lung parenchyma and bronchi (8) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity in Patients Younger Than 60 Years Is a Risk Factor for {COVID}-19 Hospital Admission.](https://pubmed.ncbi.nlm.nih.gov/32271368)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Incidence of thrombotic complications in critically ill {ICU} patients with {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32291094)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Monocarboxylate transporter 4 ({MCT4}) is a high affinity transporter capable of exporting lactate in high-lactate microenvironments.](https://pubmed.ncbi.nlm.nih.gov/31719150)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Plasma glucose levels and diabetes are independent predictors for mortality and morbidity in patients with {SARS}.](https://pubmed.ncbi.nlm.nih.gov/16759303)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Diabetes and the severity of pandemic influenza A ({H1N1}) infection.](https://pubmed.ncbi.nlm.nih.gov/20587722)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The effect of body mass index and fasting glucose on the relationship between blood pressure and incident diabetes mellitus: a 5-year follow-up study.](https://pubmed.ncbi.nlm.nih.gov/21753771)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The procoagulant pattern of patients with {COVID}-19 acute respiratory distress syndrome.](https://pubmed.ncbi.nlm.nih.gov/32302448)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Recent progress and challenges in drug development against {COVID}-19 coronavirus ({SARS}-{CoV}-2) - an update on the status.](https://pubmed.ncbi.nlm.nih.gov/32320825)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Acute Pulmonary Embolism Associated with {COVID}-19 Pneumonia Detected with Pulmonary {CT} Angiography.](https://pubmed.ncbi.nlm.nih.gov/32324103)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The pivotal link between {ACE2} deficiency and {SARS}-{CoV}-2 infection.](https://pubmed.ncbi.nlm.nih.gov/32336612)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The epidemiology and clinical information about {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32291542)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### PDCD1 inhibits Bevacizumab\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8760 ! PDCD1) decreases bp(MESH:D000068258 ! Bevacizumab)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Studies of recombinant human angiotensin converting enzyme 2 (rhACE2), mesenchymal stem cell, PD-1 blocking antibody, bevacizumab injection, and immunoglobulin of cured patients are registered in the website of Clinical Trial [XREF_BIBR], and some are recruiting patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Association of chemosensory dysfunction and {COVID}-19 in patients presenting with influenza-like symptoms.](https://pubmed.ncbi.nlm.nih.gov/32279441)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Pulmonary embolism in patients with {COVID}-19: Time to change the paradigm of computed tomography.](https://pubmed.ncbi.nlm.nih.gov/32302782)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [A review on the cleavage priming of the spike protein on coronavirus by angiotensin-converting enzyme-2 and furin.](https://pubmed.ncbi.nlm.nih.gov/32274964)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SP increases the amount of HC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:SP) increases a(TEXT:HC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This review discussed an overview on the role of ACE-2 and furin in the binding of CoV with HC biomembrane mediated by SP."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SP increases the amount of HC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:SP) increases a(TEXT:HC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The SP mediates CoV entry and conducts the interaction of CoV with receptor (ACE-2) on the HCs as well as mediating the fusion of HC biomembrane and viral envelope."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Drug decreases the amount of furin-like\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:23888 ! drug) decreases p(PFAM:PF00757 ! furin-like)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example, a drug that blocks the receptor may make CoV entry into cells difficult.Furin and furin-like proteases belong to the group of proprotein convertases."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HCs increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:HCs) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Actually, it has been recently demonstrated that HCs infected by several kinds of viruses stimulate an interferon-based activity to block the enzymatic activity of furin-like enzymes (Lodermeyer et al., 2013)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HCs increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:HCs) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Actually, it has been recently demonstrated that HCs infected by several kinds of viruses stimulate an interferon based activity to block the enzymatic activity of furin like enzymes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FURIN increases the amount of SEPHS1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8568 ! FURIN) increases p(HGNC:19685 ! SEPHS1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The furin induces the cleavage of SPs at the RNTR761EV site (Bergeron et al., 2005) and results in the production of two segments: S1 and S2 (Figure 2b); each segment shows its own biological activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FURIN increases the amount of SEPHS1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8568 ! FURIN) increases p(HGNC:19685 ! SEPHS1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The furin induces the cleavage of SPs at the RNTR761vEV site and results in the production of two segments : S1 and S2 (XREF_FIG); each segment shows its own biological activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD increases the amount of peptide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P00814 ! P00814) increases a(CHEBI:16670 ! peptide)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "(c) CD signals of QSARS-4 peptide after incubation with different concentrations of recombinant furin (Basak et al., 2007)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Commentary: {COVID}-19 in patients with diabetes.](https://pubmed.ncbi.nlm.nih.gov/32220611)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Hyperglycemia activates viral life cycle\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases bp(GO:\"GO:0019058\" ! \"viral life cycle\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In vitro exposure of pulmonary epithelial cells to elevated glucose concentrations significantly increased influenza virus infection and replication , suggesting that hyperglycemia may increase viral replication in vivo ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and its implications for thrombosis and anticoagulation.](https://pubmed.ncbi.nlm.nih.gov/32339221)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Pulmonary Embolism in Patients With {COVID}-19: Awareness of an Increased Prevalence.](https://pubmed.ncbi.nlm.nih.gov/32330083)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Type I {IFN} immunoprofiling in {COVID}-19 patients.](https://pubmed.ncbi.nlm.nih.gov/32360285)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity and impaired metabolic health in patients with {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32327737)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Factors determining the appearance of glucose in upper and lower respiratory tract secretions.](https://pubmed.ncbi.nlm.nih.gov/14647890)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Expression of the {SARS}-{CoV}-2 cell receptor gene {ACE2} in a wide variety of human tissues.](https://pubmed.ncbi.nlm.nih.gov/32345362)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The micromechanics of lung alveoli: structure and function of surfactant and tissue components.](https://pubmed.ncbi.nlm.nih.gov/30390118)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and Multiorgan Response.](https://pubmed.ncbi.nlm.nih.gov/32439197)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Association of Blood Glucose Control and Outcomes in Patients with {COVID}-19 and Pre-existing Type 2 Diabetes.](https://pubmed.ncbi.nlm.nih.gov/32369736)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### BG increases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P13917 ! P13917) increases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Elevated BG level has been reported to increase the glucose concentration in airway epithelial secretion (Philips et al., 2003), which may disrupt the defensive capacity of airway epithelia."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia management in hospitalized patients with {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32371556)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{ACE2} activators for the treatment of {COVID} 19 patients.](https://pubmed.ncbi.nlm.nih.gov/32379346)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Acute respiratory failure in {COVID}-19: is it \"typical\" {ARDS}?](https://pubmed.ncbi.nlm.nih.gov/32375845)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Tobacco smoke is a source of toxic reactive glycation products.](https://pubmed.ncbi.nlm.nih.gov/9391127)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Should we stimulate or suppress immune responses in {COVID}-19? Cytokine and anti-cytokine interventions.](https://pubmed.ncbi.nlm.nih.gov/32376392)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin in patients with and without diabetes: a paradigm shift in cardiovascular disease management.](https://pubmed.ncbi.nlm.nih.gov/31029144)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycosylation of the viral attachment protein of avian coronavirus is essential for host cell and receptor binding.](https://pubmed.ncbi.nlm.nih.gov/30902814)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Acute kidney injury in patients hospitalized with{COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32416116)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Disease activates Hospitalization\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004194 ! Disease) increases bp(MESH:D006760 ! Hospitalization)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "2 In late January 2020 , the United States had its first Covid-19 case in Washington state ,3 followed by massive growth of infections in New York , the current epicenter.2 , 4 As had been observed in China and Italy , the disease resulted in a large number of hospitalizations , respiratory failure and intensive care unit ( ICU ) admissions.5 , 6 The New York-Metropolitan area began to see a rapid increase in Covid-19 cases in February , 2020 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Disease activates Hospitalization\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004194 ! Disease) increases bp(MESH:D006760 ! Hospitalization)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "2 , 4 As had been observed in China and Italy , the disease resulted in a large number of hospitalizations , respiratory failure and intensive care unit ( ICU ) admissions ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Severe obesity, increasing age and male sex are independently associated with worse in-hospital outcomes, and higher in-hospital mortality, in a cohort of patients with {COVID}-19 in the Bronx, New York.](https://pubmed.ncbi.nlm.nih.gov/32422233)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Aging increases the amount of dioxygen\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000375 ! Aging) increases a(CHEBI:15379 ! dioxygen)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Age as a continuous variable was significantly associated with mortality (OR : 1.03; 95% CI : 1.00-1.07; p = 0.041) (XREF_TABLE panel A), and increasing oxygen requirements (OR : 1.02; 95% CI : 1.00-1.05; p = 0.044) (XREF_TABLE panel B), but not with intubation, although a trend was noticed (OR : 1.03; 95% CI : 1.00-1.05; p = 0.071) (XREF_TABLE panel C) in the multivariate analysis (model 4)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Sex activates Aging\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012723 ! Sex) increases bp(MESH:D000375 ! Aging)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the multivariate analysis (model 3), male sex (OR: 2.77; 95% CI: 1.48-5.19; p = 0.001), increasing age analyzed in quartiles (OR: 1.38; 95% CI: 1.panel A)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Obesity Is a Risk Factor for Greater {COVID}-19 Severity.](https://pubmed.ncbi.nlm.nih.gov/32409499)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The ketogenic diet for super-refractory status epilepticus patients in intensive care units.](https://pubmed.ncbi.nlm.nih.gov/30638692)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia in critically ill patients: management and prognosis.](https://pubmed.ncbi.nlm.nih.gov/26261382)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The {ACE}-2 in {COVID}-19: Foe or Friend?](https://pubmed.ncbi.nlm.nih.gov/32340044)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Tat increases the amount of GPR15\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P04608 ! P04608) increases p(HGNC:4469 ! GPR15)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "BOB is supported by Ong Tiong Tat Professorship."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates Atherosclerosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D050197 ! Atherosclerosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deficiency in ACE-2 leads to increased formation of atherosclerotic plaques and blocking of ACE-2 results in prevention of atherosclerosis 20 ( Fig. 1 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 inhibits Plaque, Atherosclerotic\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(MESH:D058226 ! \"Plaque, Atherosclerotic\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deficiency in ACE-2 leads to increased formation of atherosclerotic plaques and blocking of ACE-2 results in prevention of atherosclerosis 20 ( Fig. 1 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 inhibits Plaque, Atherosclerotic\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(MESH:D058226 ! \"Plaque, Atherosclerotic\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deficiency in ACE-2 leads toincreased formation of atherosclerotic plaques and blocking of ACE-2 results inprevention of atherosclerosis20(Fig. 1)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 inhibits aging\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(GO:\"GO:0007568\" ! aging)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In the heart, ACE-2 prevents progressivecardiac fibrosis associated with aging and/or cardiac pressure overloadand is beneficial in heart failure1920."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Severe Acute Respiratory Syndrome decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Further, SARS CoV spike protein has been found to reduce ACE-2 expression and to augment pulmonary injury [16] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Severe Acute Respiratory Syndrome decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In an animal model study, it has been reported that infusion of SARS CoV spike protein reduced ACE-2 expression and augmented pulmonary injury."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Severe Acute Respiratory Syndrome decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It hasbeen observed that ACE-2 membrane expression and plasma levels are reduced afterinfection with SARS coronavirus16.Further, SARS CoV spike protein has been found to reduce ACE-2 expression and toaugment pulmonary injury16."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Severe Acute Respiratory Syndrome decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It ispossible that an ACE/ACE-2 imbalance is one of the potential mechanismsexplaining why patients with cardio-metabolic problems are at higher risk forrespiratory failure20.In an animal model study, it has been reported that infusion of SARS CoV spikeprotein reduced ACE-2 expression and augmented pulmonary injury."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Aldosterone increases the amount of potassium(1+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:27584 ! aldosterone) increases a(CHEBI:29103 ! \"potassium(1+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This imbalance has been postulated as a potential cause of diabetic nephropathy.In the adrenal glands, the local secretory RAS stimulates aldosterone production and serves as an amplification system for circulating Ang II. Importantly the regulation of the secretory adrenal RAS is independent of the circulatory RAS and the activity of the adrenal RAS correlates with aldosterone production and regulation of potassium serum concentrations [22] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin-2 increases the amount of aldosterone\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:Angiotensin-2) increases a(CHEBI:27584 ! aldosterone)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ang II mediates vasoconstriction as well as aldosterone release from the adrenal gland, resulting in sodium retention and an increase in blood pressure through the angiotensin 1 receptor (AT 1 R)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System increases the amount of aldosterone\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases a(CHEBI:27584 ! aldosterone)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This imbalance hasbeen postulated as a potential cause of diabetic nephropathy.In the adrenal glands, the local secretory RAS stimulates aldosterone productionand serves as an amplification system for circulating Ang II. Importantly theregulation of the secretory adrenal RAS is independent of the circulatory RASand the activity of the adrenal RAS correlates with aldosterone production andregulation of potassium serum concentrations22."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE activates catabolic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) increases bp(GO:\"GO:0009056\" ! \"catabolic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The catalytic activity of ACE results in increased Ang II levels and increased catabolism of Ang-(1-7) whereas the catalytic activity of ACE-2 is predominantly on Ang I and Ang II and leads to formation of Ang-(1-7)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Modulation of this system by ACE-inhibitors or AT 1 -Receptor blockers is now considered as the first-line therapy as well as for prevention and management of vascular complications.In this regard, the questions arise if (i) differences in ACE-2 may explain the exacerbated course of disease in patients with metabolic diseases and (ii) if ACE modulation in COVID-19 patients is neutral, beneficial, or harmful."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE increases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) increases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The catalytic activity of ACE to activate Ang II occurs most extensively in the lung."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renal potassium wasting activates Hypokalemia\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:\"Renal potassium wasting\") increases bp(MESH:D007008 ! Hypokalemia)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In a recent preprint study , hypokalemia due to renal potassium wasting has been reported with 93 % of the critically ill patients having hypokalemia independent of gastrointestinal symptoms 33 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycemic Characteristics and Clinical Outcomes of {COVID}-19 Patients Hospitalized in the United States.](https://pubmed.ncbi.nlm.nih.gov/32389027)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [A hypothesis for pathobiology and treatment of {COVID}-19: The centrality of {ACE1}/{ACE2} imbalance.](https://pubmed.ncbi.nlm.nih.gov/32333398)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### ACE2 decreases the amount of ANG\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases p(HGNC:483 ! ANG)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 increases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1 and ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2 mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1 generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 increases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ANG increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:483 ! ANG) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This hypothesis suggests several approaches to treat COVID-19 by restoring ACE1/ACE2 balance: 1) ANG II receptor blockers (ARBs); 2) ACE1 inhibitors (ACEIs); 3) Agonists of receptors activated by ACE2-derived peptides [e.g., ANG (1-7), which activates MAS1]; 4) Recombinant human ACE2 or ACE2 peptides as decoys for the virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ANG increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:483 ! ANG) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This hypothesis suggests several approaches to treat COVID-19 by restoring ACE1 and ACE2 balance : 1) ANG II receptor blockers (ARBs); 2) ACE1 inhibitors (ACEIs); 3) Agonists of receptors activated by ACE2 derived peptides [e.g., ANG (1-7), which activates MAS1]; 4) Recombinant human ACE2 or ACE2 peptides as decoys for the virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ace decreases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q869C3 ! Q869C3) decreases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1 and ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2 mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1 generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ace decreases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q869C3 ! Q869C3) decreases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ace decreases the amount of ANG\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q869C3 ! Q869C3) decreases p(HGNC:483 ! ANG)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE1/ACE2 imbalance occurs due to the binding of SARS-CoV-2 to ACE2, reducing ACE2-mediated conversion of ANG II to ANG peptides that counteract pathophysiological effects of ACE1-generated ANGII."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycemic variability in diabetes increases the severity of influenza.](https://pubmed.ncbi.nlm.nih.gov/32209691)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Binding of {SARS} coronavirus to its receptor damages islets and causes acute diabetes.](https://pubmed.ncbi.nlm.nih.gov/19333547)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Indicator increases the amount of alt\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:47867 ! indicator) increases p(UP:Q38424 ! Q38424)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, significant elevation of ALT, an indicator of liver damage, was not associated with higher mortality."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Elevation of blood glucose level predicts worse outcomes in hospitalized patients with {COVID}-19: a retrospective cohort study.](https://pubmed.ncbi.nlm.nih.gov/32503812)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Diabetes, hypertension, and cardiovascular disease: clinical insights and vascular mechanisms.](https://pubmed.ncbi.nlm.nih.gov/29459239)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Linagliptin decreases the amount of DPP4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68610 ! linagliptin) directlyDecreases p(HGNC:3009 ! DPP4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Linagliptin decreases the amount of DPP4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68610 ! linagliptin) directlyDecreases p(HGNC:3009 ! DPP4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Linagliptin decreases the amount of DPP4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68610 ! linagliptin) directlyDecreases p(HGNC:3009 ! DPP4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Another example includes inhibition of dipeptidyl peptidase-4 by linagliptin, which reduces obesity related insulin resistance and inflammation by regulating M1/M2 macrophage status."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and the Digestive System.](https://pubmed.ncbi.nlm.nih.gov/32618648)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Six months of coronavirus: the mysteries scientists are still racing to solve.](https://pubmed.ncbi.nlm.nih.gov/32620885)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impaired glucose metabolism in patients with diabetes, prediabetes, and obesity is associated with severe {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32589756)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Assessment of risk, severity, mortality, glycemic control and antidiabetic agents in patients with diabetes and {COVID}-19: A narrative review.](https://pubmed.ncbi.nlm.nih.gov/32533989)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Airways Expression of {SARS}-{CoV}-2 Receptor, {ACE2}, and {TMPR\\SS2} Is Lower in Children Than Adults and Increases with Smoking and {COPD}.](https://pubmed.ncbi.nlm.nih.gov/32537478)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Camostat decreases the amount of TMPRSS2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:135632 ! camostat) decreases p(HGNC:11876 ! TMPRSS2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It is worth mentioning that serine protease inhibitors such as 274 camostat mesylate which blocks TMPRSS2 activity could be used for treatment of SARS-275CoV-2-infected patients, 6, 26 or as a preventive approach especially for patients identified as 276 high risk based on ACE2 nasal screening."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metabolic Syndrome and {COVID} 19: Endocrine-Immune-Vascular Interactions Shapes Clinical Course.](https://pubmed.ncbi.nlm.nih.gov/32603424)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Fibrinogen activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Fibrinogen ! Fibrinogen) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Levels of vWF, fibrinogen, and factor VIII were elevated in COVID-19 (83) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 increases the amount of CRP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases p(HGNC:2367 ! CRP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, due to the long-term cardiovascular benefits, in vitro evidence of suppression of IL-6-induced CRP expression by statins, and the emerging epidemiologic data on lower odds of mortality from COVID-19 among statin users, therapy with statins can be continued during COVID-19 illness (129, 155) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 activates apoptotic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The mechanisms for lymphopenia are not known, but elevated levels of IL-6 and TNF- may cause T cell apoptosis(66,69)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Rosiglitazone increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50122 ! rosiglitazone) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Rosiglitazone, a thiazolidinedione (TZD), upregulates vascular ACE2 protein expression in hypertensive rats (98)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Myc inhibits phagocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q9W4S7 ! Q9W4S7) decreases bp(GO:\"GO:0006909\" ! phagocytosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DM inhibits neutrophil chemotaxis, phagocytosis, and intracellular killing of microbes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Myc inhibits neutrophil chemotaxis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:Q9W4S7 ! Q9W4S7) decreases bp(GO:\"GO:0030593\" ! \"neutrophil chemotaxis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DM inhibits neutrophil chemotaxis, phagocytosis, and intracellular killing of microbes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Insulin administration decreases ACE2 protein expression in the lungs of diabetic mice (96) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SerA increases the amount of SLC6A2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:O29445 ! O29445) increases p(HGNC:11048 ! SLC6A2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Interestingly, sera from individuals with COVID-19 triggered NET release from control neutrophils in A c c e p t e d M a n u s c r i p t vitro (82) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Testosterone increases the amount of ACE\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17347 ! testosterone) increases p(HGNC:2707 ! ACE)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Estradiol decreases, while testosterone increases ACE activity in the lung (120) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE increases the amount of octapeptide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) increases p(PFAM:PF03373 ! octapeptide)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE catalyzes the conversion of angiotensin I to the octapeptide, angiotensin II (AngII), whereas ACE2 converts Ang II to angiotensin 1-7."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LDH increases the amount of Ferritin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:LDH ! LDH) increases a(CHEBI:82594 ! Ferritin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LDH increases the amount of CRP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:LDH ! LDH) increases p(HGNC:2367 ! CRP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity increases the amount of HR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009765 ! Obesity) increases a(TEXT:HR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In a prospective observational cohort study using survey data from hospitalized patients (n = 16,749), obesity increased fatality risk (adjusted for age and gender) (HR = 1.37, 95% CI = 1.16-1.63) (13) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity increases the amount of CI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009765 ! Obesity) increases p(UP:P08707 ! P08707)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In a prospective observational cohort study using survey data from hospitalized patients (n = 16,749), obesity increased fatality risk (adjusted for age and gender) (HR = 1.37, 95% CI = 1.16-1.63) (13) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lactate increases the amount of Ferritin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24996 ! lactate) increases a(CHEBI:82594 ! Ferritin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lactate increases the amount of CRP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24996 ! lactate) increases p(HGNC:2367 ! CRP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension increases the amount of HR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases a(TEXT:HR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, after adjusting for other confounders, the presence of hypertension increased the risk slightly (HR =1.07, 95% CI = 1.00-1.15) (14) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension increases the amount of HR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases a(TEXT:HR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In a large study from the UK, accounting for age and sex, hypertension increases the risk of in-hospital death (HR = 1.22, 95% CI =1.15-1.30)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension increases the amount of CI\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases p(UP:P08707 ! P08707)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, after adjusting for other confounders, the presence of hypertension increased the risk slightly (HR =1.07, 95% CI = 1.00-1.15) (14) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lymphopenia increases the amount of Ferritin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D008231 ! Lymphopenia) increases a(CHEBI:82594 ! Ferritin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lymphopenia increases the amount of CRP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D008231 ! Lymphopenia) increases p(HGNC:2367 ! CRP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Lymphopenia, elevated lactate dehydrogenase (LDH), neutrophilia, increased C-reactive protein (CRP), mild increases in liver enzymes (alanine aminotransferase (ALT) and aspartate aminotransferase (AST)), higher levels of D-dimer, ferritin, and pro-calcitonin are often observed in these patients."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin inhibits cell growth\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:48433 ! angiotensin) decreases bp(GO:\"GO:0016049\" ! \"cell growth\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ang II, through the activation of Ang II type 1a receptors, induces vasoconstriction and proliferation, but angiotensin 1-7 stimulates vasodilatation and suppresses cell growth."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Simvastatin activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:9150 ! simvastatin) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "To investigate the antiinflammatory effects of statins, a phase 2 RCT is evaluating a combination of the Janus kinase (JAK) inhibitor, ruxolitinib and simvastatin in the prevention and treatment of respiratory failure in COVID-19 (ClinicalTrials.gov Identifier: NCT04348695)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertriglyceridemia decreases the amount of high-density lipoprotein cholesterol\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015228 ! Hypertriglyceridemia) decreases a(CHEBI:47775 ! \"high-density lipoprotein cholesterol\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Clinically, metabolic syndrome is defined as the presence of three or more of the following factors: increased waist circumference (population and country-specific cutoff), hypertriglyceridemia (>150 mg/dL or on treatment for hypertriglyceridemia), elevated blood pressure (systolic  130 and/or diastolic  85 mm Hg or with a history of hypertension on treatment), reduced high-density lipoprotein cholesterol (<40 mg/dL in males; <50 mg/dL in females), and dysglycemia ( 100 mg/dL or on treatment for hyperglycemia) (2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertriglyceridemia inhibits Hyperglycemia\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015228 ! Hypertriglyceridemia) decreases bp(MESH:D006943 ! Hyperglycemia)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Clinically, metabolic syndrome is defined as the presence of three or more of the following factors: increased waist circumference (population and country-specific cutoff), hypertriglyceridemia (>150 mg/dL or on treatment for hypertriglyceridemia), elevated blood pressure (systolic  130 and/or diastolic  85 mm Hg or with a history of hypertension on treatment), reduced high-density lipoprotein cholesterol (<40 mg/dL in males; <50 mg/dL in females), and dysglycemia ( 100 mg/dL or on treatment for hyperglycemia) (2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Tamoxifen inhibits SARS-CoV-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:41774 ! tamoxifen) decreases bp(MESH:D000086402 ! SARS-CoV-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Tamoxifen, an estrogen receptor agonist, inhibits SARS-CoV-2 in vitro (93) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  VWF activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:12726 ! VWF) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Levels of vWF, fibrinogen, and factor VIII were elevated in COVID-19 (83) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ruxolitinib activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:66919 ! ruxolitinib) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "To investigate the antiinflammatory effects of statins, a phase 2 RCT is evaluating a combination of the Janus kinase (JAK) inhibitor, ruxolitinib and simvastatin in the prevention and treatment of respiratory failure in COVID-19 (ClinicalTrials.gov Identifier: NCT04348695)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P07052 increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P07052 ! P07052) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P07052 increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P07052 ! P07052) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "PRR activation pathway and IL-6 stimulate monocytes tissue factor (TF) expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  VWF binds Extracellular Traps\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(bp(MESH:D065206 ! \"Extracellular Traps\"), p(HGNC:12726 ! VWF)) increases bp(GO:\"GO:0050817\" ! coagulation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Extracellular Traps bound to VWF activates coagulation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(bp(MESH:D065206 ! \"Extracellular Traps\"), p(HGNC:12726 ! VWF)) increases bp(GO:\"GO:0050817\" ! coagulation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  VWF binds Extracellular Traps\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(bp(MESH:D065206 ! \"Extracellular Traps\"), p(HGNC:12726 ! VWF)) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Extracellular Traps bound to VWF increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(bp(MESH:D065206 ! \"Extracellular Traps\"), p(HGNC:12726 ! VWF)) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NETs recruit platelets by binding to von Willebrand factor (vWF) and activate factor XII and TF to trigger the contact (intrinsic) and extrinsic coagulation pathways, respectively (81) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Estrogen receptor agonist inhibits SARS-CoV-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:63951 ! \"estrogen receptor agonist\") decreases bp(MESH:D000086402 ! SARS-CoV-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Tamoxifen, an estrogen receptor agonist, inhibits SARS-CoV-2 in vitro (93) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Lymphopenia in severe coronavirus disease-2019 ({COVID}-19): systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/32483488)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Angiotensin II Type 1 Receptor Blockers activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D047228 ! \"Angiotensin II Type 1 Receptor Blockers\") increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Angiotensin-converting enzyme ( ACE ) inhibitor and angiotensin-receptor blocker ( ARB ) have been hypothetically suggested to exacerbate COVID-19 due to increase in angiotensin II level [ 31 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Propagation of respiratory viruses in human airway epithelia reveals persistent virus-specific signatures.](https://pubmed.ncbi.nlm.nih.gov/28797733)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Coronavirus OC43, Human increases the amount of cytokine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D028962 ! \"Coronavirus OC43, Human\") increases a(TEXT:cytokine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Of note , the low cytokine induction by HCoV-OC43 was also confirmed at these higher inoculum doses ( data not shown ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Respiratory Tract Infections increases the amount of MCC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012141 ! \"Respiratory Tract Infections\") increases p(HGNC:6935 ! MCC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "To conclude, our results shed light on the complex interplay between viruses and the host tissue response and contribute to the understanding of the mechanism behind chronic respiratory tract infections in immunocompromised patients.Key messagesRV-B48 and HCoV-OC43 induce low toxicity in respiratory tissues because they affect neither MCC nor tissue integrity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Thrombosis and {COVID}-19 pneumonia: the clot thickens!](https://pubmed.ncbi.nlm.nih.gov/32554532)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Inflammatory response inhibits fibrinolysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") decreases bp(GO:\"GO:0042730\" ! fibrinolysis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This , in turn , is likely to relate to reduced fibrinolysis due to increased production of PAI-1 , again as a result of inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Inflammatory response inhibits fibrinolysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") decreases bp(GO:\"GO:0042730\" ! fibrinolysis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This , in turn , is likely to relate to reduced fibrinolysis due to increased production of plasminogen activator inhibitor-1 ( PAI-1 ) , again as a result of inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Inflammatory response increases the amount of SERPINE1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") increases p(HGNC:8583 ! SERPINE1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This , in turn , is likely to relate to reduced fibrinolysis due to increased production of PAI-1 , again as a result of inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Inflammatory response inhibits Plasminogen Activators\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") decreases bp(MESH:D010960 ! \"Plasminogen Activators\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This , in turn , is likely to relate to reduced fibrinolysis due to increased production of plasminogen activator inhibitor-1 ( PAI-1 ) , again as a result of inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F3 increases the amount of F2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3541 ! F3) increases p(HGNC:3535 ! F2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Tissue factor, usually hidden on the subendothelium, is upregulated on platelets, leucocytes and EC during inflammation, leading to activation of both the extrinsic and intrinsic coagulation pathways to make thrombin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and anosmia: A review based on up-to-date knowledge.](https://pubmed.ncbi.nlm.nih.gov/32563019)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Persistent Symptoms in Patients After Acute {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32644129)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role of angiotensin-converting enzyme 2 in the pathogenesis of {COVID}-19: the villain or the hero?](https://pubmed.ncbi.nlm.nih.gov/32597377)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 acute respiratory distress syndrome ({ARDS}): clinical features and differences from typical pre-{COVID}-19 {ARDS}.](https://pubmed.ncbi.nlm.nih.gov/32572965)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Newly diagnosed diabetes is associated with a higher risk of mortality than known diabetes in hospitalized patients with {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32469464)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Hyperglycemia increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "19 Acute hyperglycemia has been shown to upregulate ACE2 expression on This article is protected by copyright."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Diabetes Mellitus, Type 2 activates Disease\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003924 ! \"Diabetes Mellitus, Type 2\") increases bp(MESH:D004194 ! Disease)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "An animal model shows that the complication of type 2 diabetes causes an immune disorder and enhances disease severity following MERS-CoV infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Diabetes Mellitus, Type 2 activates Immune System Diseases\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003924 ! \"Diabetes Mellitus, Type 2\") increases bp(MESH:D007154 ! \"Immune System Diseases\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "An animal model shows that the complication of type 2 diabetes causes an immune disorder and enhances disease severity following MERS-CoV infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The vascular endothelium: the cornerstone of organ dysfunction in severe {SARS}-{CoV}-2 infection.](https://pubmed.ncbi.nlm.nih.gov/32546188)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia and the worse prognosis of {COVID}-19. Why a fast blood glucose control should be mandatory.](https://pubmed.ncbi.nlm.nih.gov/32360400)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{RAGE} receptor: May be a potential inflammatory mediator for {SARS}-{COV}-2 infection?](https://pubmed.ncbi.nlm.nih.gov/32531537)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Vulnerabilities in coronavirus glycan shields despite extensive glycosylation.](https://pubmed.ncbi.nlm.nih.gov/32461612)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{HMGB1}: A Possible Crucial Therapeutic Target for {COVID}-19?](https://pubmed.ncbi.nlm.nih.gov/32375153)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### NFASC activates positive regulation of leukocyte cell-cell adhesion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29866 ! NFASC) increases bp(GO:\"GO:1903039\" ! \"positive regulation of leukocyte cell-cell adhesion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It binds to chromosomal DNA but also to Toll-like receptor 3 (TLR3), TLR4, and the receptor for advanced glycation end products (RAGE) that activates nuclear factor (NF)-B ( Fig. 1a) , which mediate the upregulation of leukocyte adhesion molecules as well as the production of proinflammatory cytokines and angiogenic factors that promote inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mice decreases the amount of ACE\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D051379 ! Mice) decreases p(HGNC:2707 ! ACE)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Kuba et al. [18] showed in mice that SARS-CoV downregulated ACE II protein, contributing to severe lung injury."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE decreases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:2707 ! ACE) decreases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Interestingly, ACE II overexpression has been reported to reduce HMGB1, besides reducing apoptosis in the myocardium postinfarction, in a rat model [19] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER increases the amount of NFASC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:320 ! AGER) increases p(HGNC:29866 ! NFASC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It binds to chromosomal DNA but also to Toll-like receptor 3 (TLR3), TLR4, and the receptor for advanced glycation end products (RAGE) that activates nuclear factor (NF)-B ( Fig. 1a) , which mediate the upregulation of leukocyte adhesion molecules as well as the production of proinflammatory cytokines and angiogenic factors that promote inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Respiratory viral infections in exacerbation of chronic airway inflammatory diseases: novel mechanisms and insights from the upper airway epithelium.](https://pubmed.ncbi.nlm.nih.gov/32161756)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Infections increases the amount of OSM\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007239 ! Infections) increases p(HGNC:8506 ! OSM)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example, IFV infection was found to induce oncostatin M (OSM) which causes tight junction opening (Pothoven et al., 2015; Tian et al., 2018)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Virus Diseases increases the amount of alpha-D-GalpNAc-(1->3)-[alpha-L-Fucp-(1->2)]-beta-D-Galp-(1->4)-beta-D-GlcpNAc\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D014777 ! \"Virus Diseases\") increases a(CHEBI:62486 ! \"alpha-D-GalpNAc-(1->3)-[alpha-L-Fucp-(1->2)]-beta-D-Galp-(1->4)-beta-D-GlcpNAc\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The increased inflammation, in turn, worsens the symptoms of airway diseases.Additionally, in patients with asthma and patients with CRS with nasal polyp (CRSwNP), viral infections such as RV and RSV promote a Type 2-biased immune response (Becker, 2006; Jackson et al., 2014; Jurak et al., 2018)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Virus Diseases activates Airway Remodeling\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D014777 ! \"Virus Diseases\") increases bp(MESH:D056151 ! \"Airway Remodeling\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Viral infections in such conditions will then cause increase neutrophilic activation; worsening the symptoms and airway remodeling in the airway thereby exacerbating COPD, CRSsNP and even CRSwNP in certain cases (Wang et al., 2009; Tacon et al., 2010; Linden et al., 2019)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of MAVS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:29233 ! MAVS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of MAVS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:29233 ! MAVS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of EP300\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:3373 ! EP300)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of EP300\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:3373 ! EP300)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pulmonary Disease, Chronic Obstructive decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\") decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016, 2017)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Papillomavirus Infections activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D030361 ! \"Papillomavirus Infections\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Therefore, the role of HPV infection in causing chronic inflammation in the airway and their association to exacerbations of chronic airway inflammatory diseases, which is scarcely explored, should be investigated in the future."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{SARS}: prognosis, outcome and sequelae.](https://pubmed.ncbi.nlm.nih.gov/15018132)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Fasting blood glucose at admission is an independent predictor for 28-day mortality in patients with {COVID}-19 without previous diagnosis of diabetes: a multi-centre retrospective study.](https://pubmed.ncbi.nlm.nih.gov/32647915)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### F3 increases the amount of F2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3541 ! F3) increases p(HGNC:3535 ! F2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Coagulopathy was defined as prothrombin time prolonged by 3 s or activated partial thromboplastin time prolonged by 5 s [21]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The Emerging Threat of (Micro)Thrombosis in {COVID}-19 and Its Therapeutic Implications.](https://pubmed.ncbi.nlm.nih.gov/32586214)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Histone increases the amount of Fibrin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Histone ! Histone) increases p(FPLX:Fibrin ! Fibrin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "96 Histones activate platelets via TLR-2 and TLR-4 and enhance fibrin formation , while the contact activation system , via FXII activation , is initiated by negatively charged DNA ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Histone activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Histone ! Histone) increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "96 Histones activate platelets via TLR-2 and TLR-4 and enhance fibrin formation , while the contact activation system , via FXII activation , is initiated by negatively charged DNA ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets activates antigen binding\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "88 , 89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets inhibits antigen binding\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") decreases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The importance of platelets in mediating T-cell recruitment and trafficking is underscored by data demonstrating that platelets play a fundamental role in the accumulation of cytotoxic T cells , and virus-induced organ damage , in a murine model of acute viral hepatitis.87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution.88 ,89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets activates T-Lymphocytes\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases bp(MESH:D013601 ! T-Lymphocytes)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example , platelets express major histocompatibility complex class 1 molecules and can , therefore , directly activate T cells in an major histocompatibility complex class 1 dependent manner ,84 while platelet-derived PF-4 and serotonin can regulate Th17 expansion85 and naive T-cell activation and proliferation ,86 respectively ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets activates Adaptive Immunity\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases bp(MESH:D056704 ! \"Adaptive Immunity\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These interactions are likely important since platelets have previously been demonstrated to be key modulators of adaptive immunity in the context of a range of inflammatory disorders ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets activates Adaptive Immunity\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases bp(MESH:D056704 ! \"Adaptive Immunity\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Nevertheless , delineating how platelets modulate the adaptive immune response to SARS-CoV-2 and the role adaptive immunity plays in mediating thrombosis and organ injury will be crucial in uncovering novel therapeutic strategies ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  COVID-19 increases the amount of Fibrin\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086382 ! COVID-19) increases p(FPLX:Fibrin ! Fibrin)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  COVID-19 increases the amount of F2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086382 ! COVID-19) increases p(HGNC:3535 ! F2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  COVID-19 activates Respiratory Distress Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086382 ! COVID-19) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As discussed previously , severe COVID-19 can result in ARDS , which promotes mass upregulation of inflammatory cytokines and leukocytes , and can result in extensive thrombin generation , fibrin formation , and microvascular thrombosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of 2-O-acetyl-1-O-hexadecyl-sn-glycero-3-phosphocholine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases a(CHEBI:44811 ! 2-O-acetyl-1-O-hexadecyl-sn-glycero-3-phosphocholine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of F2R\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(HGNC:3537 ! F2R)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin efficiently cleaves and activates PAR-1, -3, and -4.131,132 Thus, in addition to being a potent platelet agonist, thrombin also activates immune effector and endothelial cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of F2R\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(HGNC:3537 ! F2R)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin efficiently cleaves and activates PAR-1, -3, and -4."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of F2R\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(HGNC:3537 ! F2R)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, monocyte and T-cell functions are enhanced by thrombin activation of monocyte and T-cell expressed PAR-1 and PAR-3."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of F2RL2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(HGNC:3539 ! F2RL2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, monocyte and T-cell functions are enhanced by thrombin activation of monocyte and T-cell expressed PAR-1 and PAR-3."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of CXCL8\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(HGNC:6025 ! CXCL8)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The levels of MPO-DNA and citrullinated histone H3 were higher in patients receiving mechanical ventilation compared with those not requiring any supplemental oxygen suggesting that levels of NETs correlate with disease severity.109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 activates Leukocytes\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases bp(MESH:D007962 ! Leukocytes)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "133,134 Indeed, thrombin-induced activation of leukocytes results in the production of several cytokines and growth factors while also upregulating leukocyte adhesive function."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 activates Endothelial Cells\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases bp(MESH:D042783 ! \"Endothelial Cells\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "131 , 132 Thus , in addition to being a potent platelet agonist , thrombin also activates immune effector and endothelial cells ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 activates Endothelial Cells\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases bp(MESH:D042783 ! \"Endothelial Cells\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin activates endothelial cells via the endothelial PAR-1 receptor , leading to upregulation of IL-6 , IL-8 , PAF ( platelet-activating factor ) , and MCP ( monocyte chemoattractant protein ) -1 in addition to the adhesion molecules P-selectin , E-selectin and ICAM ( intercellular adhesion molecule ) -1 , all of which serve to increase leukocyte recruitment and activation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 increases the amount of P0CW71\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) increases p(UP:P0CW71 ! P0CW71)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thrombin induced endothelial activation, via PAR-1, induces the production of a range of proinflammatory cytokines and chemokines, including IL-6, IL-8, PAF (platelet activating factor), and MCP-1, while also triggering the expression of the proadhesive molecules ICAM-1, E-and P-selectin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  F2 decreases the amount of astatine atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3535 ! F2) decreases a(CHEBI:30415 ! \"astatine atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "149 Importantly, Heparins, including unfractionated heparin (UFH) and low molecular weight heparin (LMWH), bind antithrombin (AT), and potentiate the inhibitory effect of AT on coagulation factors Xa and thrombin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  T-Lymphocytes activates platelet aggregation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D013601 ! T-Lymphocytes) increases bp(GO:\"GO:0070527\" ! \"platelet aggregation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  T-Lymphocytes inhibits platelet aggregation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D013601 ! T-Lymphocytes) decreases bp(GO:\"GO:0070527\" ! \"platelet aggregation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The importance of platelets in mediating T-cell recruitment and trafficking is underscored by data demonstrating that platelets play a fundamental role in the accumulation of cytotoxic T cells , and virus-induced organ damage , in a murine model of acute viral hepatitis.87 Akin to the bidirectional relationship between platelets and other leukocyte subsets , the activation of T cells amplifies platelet aggregation with more recent data highlighting an important role for T cells recruited to sites of venous thrombosis in orchestrating the inflammatory response and subsequent thrombus resolution.88 ,89 In addition to mediating T-cell functions , platelets can increase B-cell immunoglobulin ( Ig ) G1 , IgG2 , and IgG3 production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Coagulation Factor Xa decreases the amount of astatine atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3783 ! \"Coagulation Factor Xa\") decreases a(CHEBI:30415 ! \"astatine atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "149 Importantly, Heparins, including unfractionated heparin (UFH) and low molecular weight heparin (LMWH), bind antithrombin (AT), and potentiate the inhibitory effect of AT on coagulation factors Xa and thrombin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heparin inhibits Severe Acute Respiratory Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:28304 ! heparin) decreases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "142, 143 Indeed, heparin, acting as a decoy receptor for naturally expressed heparan sulfate, has been shown to inhibit the binding of the SARS coronavirus to cells in vitro."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TFPI activates Endothelium\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11760 ! TFPI) increases bp(MESH:D004727 ! Endothelium)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "21 Interestingly , however , studies evaluating Chinese and White populations have demonstrated that abnormal coagulation parameters are associated with severe In vascular homeostasis , the endothelium possesses anti-inflammatory and antithrombotic properties due to the expression of CD39 , nitric oxide ( NO ) , and prostacyclin in addition to the natural anticoagulants , TFPI ( tissue factor pathway inhibitor ) , activated protein C , and thrombomodulin ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Complement activation increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006956\" ! \"complement activation\") increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Finally , complement activation leads to the recruitment of leukocytes and upregulates TF expression , amplifies platelet activation and upregulates endothelial expression of proinflammatory cytokines , including IL ( interleukin ) -1 , IL-6 , IL-8 , and MCP ( monocyte chemoattractant protein ) -1 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Complement activation activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006956\" ! \"complement activation\") increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Finally , complement activation leads to the recruitment of leukocytes and upregulates TF expression , amplifies platelet activation and upregulates endothelial expression of proinflammatory cytokines , including IL ( interleukin ) -1 , IL-6 , IL-8 , and MCP ( monocyte chemoattractant protein ) -1 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TLR3 increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11849 ! TLR3) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "127 Previous data have highlighted that a major mechanism by which viral infection can induce a procoagulant phenotype pertains to the upregulation of TF expression by virus infected endothelial cells and endothelial TLR-3 signaling in response to virus derived pathogen associated molecular patterns."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Neutrophils increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009504 ! Neutrophils) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "100 Activated leukocytes , principally monocytes , and neutrophils , upregulate TF in response to proinflammatory cytokines , predominantly IL-6 , and subsequently act to enhance coagulation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SELP increases the amount of MAC_1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10721 ! SELP) increases p(FPLX:\"MAC_1\" ! \"MAC_1\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "112 Moreover, platelet P-selectin induces monocyte TF expression and upregulates the expression of monocyte Mac-1 (alpha M beta 2, CD11b and CD18)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SELP increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10721 ! SELP) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "112 Moreover, platelet P-selectin induces monocyte TF expression and upregulates the expression of monocyte Mac-1 (alpha M beta 2, CD11b and CD18)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Monocytes increases the amount of F3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009000 ! Monocytes) increases p(HGNC:3541 ! F3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "100 Activated leukocytes , principally monocytes , and neutrophils , upregulate TF in response to proinflammatory cytokines , predominantly IL-6 , and subsequently act to enhance coagulation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Extracellular Traps activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D065206 ! \"Extracellular Traps\") increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Extracellular Traps activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D065206 ! \"Extracellular Traps\") increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The levels of MPO-DNA and citrullinated histone H3 were higher in patients receiving mechanical ventilation compared with those not requiring any supplemental oxygen suggesting that levels of NETs correlate with disease severity.109 Interestingly , the finding that the level of NETs correlated with elevations in D-dimer and sera from patients with COVID-19 could induce NETosis in purified neutrophils suggests that NETs , by way of their ability to generate thrombin , via activation of FXII , and activate platelets , may play an important role in mediating the prothrombotic phenotype observed in COVID-19 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Review activates Research\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D016454 ! Review) increases bp(MESH:D012106 ! Research)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Therefore , this review will hopefully spark further interest and research , and importantly raise awareness of the importance of anticoagulation in patients with COVID-19 for both the prevention and treatment of thrombosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RNA, Viral increases the amount of TLR7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012367 ! \"RNA, Viral\") increases p(HGNC:15631 ! TLR7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The activation of platelet TLR-7 by viral RNA can evoke a range of functional responses which support thrombus formation , including platelet degranulation , the formation of platelet-leukocyte aggregates and platelet C3 release which in turn stimulates NETosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CD16 increases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:CD16 ! CD16) increases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "104 This is perhaps best highlighted by the observation that patients with severe COVID-19 in ICU exhibit a significant expansion of CD14 + CD16 + monocytes producing IL-6 compared with nonhospitalized patients with COVID-19."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SELPLG increases the amount of SELP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10722 ! SELPLG) increases p(HGNC:10721 ! SELP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activated platelets express P-selectin which allows leukocyte tethering via leukocyte expressed PSGL-1 (P-selectin glycoprotein ligand 1), while GP-Ib facilitates stable adhesion via binding to the leukocyte integrin Mac-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The Impact of Mutations in {SARS}-{CoV}-2 Spike on Viral Infectivity and Antigenicity.](https://pubmed.ncbi.nlm.nih.gov/32730807)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Bacterial co-infection and secondary infection in patients with {COVID}-19: a living rapid review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/32711058)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The protein expression profile of {ACE2} in human tissues.](https://pubmed.ncbi.nlm.nih.gov/32715618)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Renin-Angiotensin System activates vasodilation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases bp(GO:\"GO:0042311\" ! vasodilation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 is a carboxypeptidase negatively regulating the renin-angiotensin system ( RAS ) , which can induce vasodilation by cleaving angiotensin II."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Renin-Angiotensin System activates vasodilation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D012084 ! \"Renin-Angiotensin System\") increases bp(GO:\"GO:0042311\" ! vasodilation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 is a carboxypeptidase negatively regulating the renin-angiotensin system ( RAS ) , which can induce vasodilation by cleaving angiotensin II."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Evaluation of the {mRNA}-1273 Vaccine against {SARS}-{CoV}-2 in Nonhuman Primates.](https://pubmed.ncbi.nlm.nih.gov/32722908)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Role of advanced glycation end products in mobility and considerations in possible dietary and nutritional intervention strategies.](https://pubmed.ncbi.nlm.nih.gov/30337945)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19: The Immune Responses and Clinical Therapy Candidates.](https://pubmed.ncbi.nlm.nih.gov/32756480)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Haemoglobin A1c is a predictor of {COVID}-19 severity in patients with diabetes.](https://pubmed.ncbi.nlm.nih.gov/32852883)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Association Between Administration of Systemic Corticosteroids and Mortality Among Critically Ill Patients With {COVID}-19: A Meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/32876694)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High glucose levels increase influenza-associated damage to the pulmonary epithelial-endothelial barrier.](https://pubmed.ncbi.nlm.nih.gov/32697191)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Deducing the N- and O-glycosylation profile of the spike protein of novel coronavirus {SARS}-{CoV}-2.](https://pubmed.ncbi.nlm.nih.gov/32363391)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Molecular Architecture of the {SARS}-{CoV}-2 Virus.](https://pubmed.ncbi.nlm.nih.gov/32979942)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Evaluation of bacterial co-infections of the respiratory tract in {COVID}-19 patients admitted to {ICU}.](https://pubmed.ncbi.nlm.nih.gov/32873235)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{SARS}-{CoV}-2 vaccines in development.](https://pubmed.ncbi.nlm.nih.gov/32967006)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Beyond Shielding: The Roles of Glycans in the {SARS}-{CoV}-2 Spike Protein.](https://pubmed.ncbi.nlm.nih.gov/33140034)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Characteristics of {SARS}-{CoV}-2 and {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/33024307)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Asia activates Chiroptera\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001208 ! Asia) increases bp(MESH:D002685 ! Chiroptera)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Extensive surveillance of SARS-CoV-2-related viruses in China, Southeast Asia and other regions targeting bats, wild and captured pangolins and other wildlife species will help us to better understand the zoonotic origin of SARS-CoV-2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cytokine Release Syndrome activates Respiratory Distress Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000080424 ! \"Cytokine Release Syndrome\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs60 ,61 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cytokine Release Syndrome activates Respiratory Distress Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000080424 ! \"Cytokine Release Syndrome\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs 60,61 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cytokine Release Syndrome inhibits Respiratory Insufficiency\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000080424 ! \"Cytokine Release Syndrome\") decreases bp(MESH:D012131 ! \"Respiratory Insufficiency\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs60 ,61 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cytokine Release Syndrome inhibits Respiratory Insufficiency\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000080424 ! \"Cytokine Release Syndrome\") decreases bp(MESH:D012131 ! \"Respiratory Insufficiency\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cytokine storm syndrome causes acute respiratory distress syndrome and respiratory failure , which is considered the main cause of death in patients with COVID-19 ( refs 60,61 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Provocative question: should ketogenic metabolic therapy become the standard of care for glioblastoma?](https://pubmed.ncbi.nlm.nih.gov/31025151)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Single-Cell {RNA} Expression Profiling of {ACE2}, the Receptor of {SARS}-{CoV}-2.](https://pubmed.ncbi.nlm.nih.gov/32663409)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Biomarkers and outcomes of {COVID}-19 hospitalisations: systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/32934000)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impaired fasting glucose and diabetes are related to higher risks of complications and mortality among patients with coronavirus disease 2019.](https://pubmed.ncbi.nlm.nih.gov/32754119)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Surfactant protein-D and pulmonary host defense.](https://pubmed.ncbi.nlm.nih.gov/11667972)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Lipopolysaccharide increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Differences in cell type, the extent of SP-D multimerization, or differences in size or organization of bacterial aggregates could account for some of the apparent inconsistencies.Although LPS mediates the binding of SP-D to at least some Gram-negative bacteria, SP-D can also bind to specific forms of soluble LPS, particularly `rough' or O-antigen-deficient forms [35,41,44]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lipopolysaccharide increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Although LPS mediates the binding of SP-D to at least some Gram negative bacteria, SP-D can also bind to specific forms of soluble LPS, particularly ` rough ' or O-antigen-deficient forms [XREF_BIBR, XREF_BIBR, XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lipopolysaccharide increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Differences in cell type, the extent of SP-D multimerization, or differences in size or organization of bacterial aggregates could account for some of the apparent inconsistencies.Although LPS mediates the binding of SP-D to at least some Gram-negative bacteria, SP-D can also bind to spe- In the latter study the authors suggested that fungal aggregation inhibits phagocytosis."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of hydrogen peroxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases a(CHEBI:16240 ! \"hydrogen peroxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D also potentiates IAV-induced hydrogen peroxide responses in neutrophils, and protects neutrophils from IAV-induced inhibition of respiratory burst responses."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of lipopolysaccharide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases a(CHEBI:16412 ! lipopolysaccharide)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Human SP-A, but not SP-D, increases the uptake and deacylation of tritiated E. coli LPS by isolated alveolar macrophages [52], but the biological significance of this finding is uncertain.SP-D binds in a CRD-dependent manner to gpA (gp-140), a heavily mannosylated glycoprotein associated with the trophozoites and cysts of Pneumocystis carinii [53]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of lipopolysaccharide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases a(CHEBI:16412 ! lipopolysaccharide)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Human SP-A, but not SP-D, increases the uptake and deacylation of tritiated E. coli LPS by isolated alveolar macrophages[52], but the biological significance of this finding is uncertain.Pneumocystis SP-D binds in a CRD-dependent manner to gpA (gp-140), a heavily mannosylated glycoprotein associated with the trophozoites and cysts of Pneumocystis carinii [53]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of phosphatidylinositol\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases a(CHEBI:28874 ! phosphatidylinositol)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example, Poulain et al have shown that SP-D can mediate the formation of characteristic tubular configurations of lipid mixtures containing PI [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Notably , human bronchoalveolar lavage ( BAL ) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro , and depletion of SP-D from BAL reduces its HA inhibitory activity [ 61 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD activates Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits phagocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0006909\" ! phagocytosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore, rat SP-D inhibits the phagocytosis of Candida albicans by these cells [57]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits T cell proliferation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(GO:\"GO:0042098\" ! \"T cell proliferation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For example, SP-D can decrease interleukin-2-dependent T-lymphocyte proliferation [95]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD inhibits Lipidoses\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) decreases bp(MESH:D008064 ! Lipidoses)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Although the overexpression of wild-type SP-D in type II pneumocytes with the SP-D-deficient mice can prevent the lipidosis and inflammatory changes [94], the ability of overexpressed wild-type SP-D or exogenous SP-D to ameliorate these abnormalities has not yet been described."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydrogen-Ion Concentration decreases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006863 ! \"Hydrogen-Ion Concentration\") decreases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Conditions of mildly acidic pH, as might be found in endocytic compartments, are predicted to disrupt the lectin dependent activities of SP-D [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bronchoalveolar Lavage inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D018893 ! \"Bronchoalveolar Lavage\") decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [61]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Bronchoalveolar Lavage inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D018893 ! \"Bronchoalveolar Lavage\") decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Notably, human bronchoalveolar lavage (BAL) contains sufficient concentrations of SP-D to inhibit IAV HA activity in vitro, and depletion of SP-D from BAL reduces its HA inhibitory activity [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules activates mucociliary clearance\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\") increases bp(GO:\"GO:0120197\" ! \"mucociliary clearance\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Massive agglutination of organisms could contribute to lung host defense by promoting airway mucociliary clearance, but it might also influence the internalization by phagocytic cells.SP-D enhances the binding and uptake of IAV by neutrophils [33,61,63], and these effects are highly dependent on the extent of multimerization of SP-D and its ability to agglutinate the viral particles [18,30,33]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FGF7 increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3685 ! FGF7) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Keratinocyte growth factor (KGF) increases SP-D expression and protein production in association with pneumocyte hyperplasia and after injury caused by bleomycin [105]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FGF7 increases the amount of SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3685 ! FGF7) increases p(HGNC:10803 ! SFTPD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Keratinocyte growth factor (KGF) increases SP-D expression and protein production in association with pneumocyte hyperplasia and after injury caused by bleomycin [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DHRS2 activates phagocytosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:18349 ! DHRS2) increases bp(GO:\"GO:0006909\" ! phagocytosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Restrepo C, Dong Q, Savov J, Mariencheck W, Wright JR: Surfactant  Protein D stimulates phagocytosis of Pseudomonas aeruginosa by alveolar macrophages."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Neuraminidase binds SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:10803 ! SFTPD), p(PFAM:PF00064 ! neuraminidase)) decreases bp(MESH:D009439 ! Neuraminidase)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D also binds to the neuraminidase of IAV and inhibits neuraminidase enzyme activity [62]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD bound to neuraminidase inhibits Neuraminidase\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:10803 ! SFTPD), p(PFAM:PF00064 ! neuraminidase)) decreases bp(MESH:D009439 ! Neuraminidase)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SP-D also binds to the neuraminidase of IAV and inhibits neuraminidase enzyme activity [62]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Carbohydrate recognition domain of surfactant protein D mediates interactions with Pneumocystis carinii glycoprotein A.](https://pubmed.ncbi.nlm.nih.gov/11306442)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Increased susceptibility of diabetic mice to influenza virus infection: compromise of collectin-mediated host defense of the lung by glucose?](https://pubmed.ncbi.nlm.nih.gov/9658139)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impact of {COVID}-19 and comorbidities on health and economics: Focus on developing countries and India.](https://pubmed.ncbi.nlm.nih.gov/32891011)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Evolution of Antibody Immunity to {SARS}-{CoV}-2.](https://pubmed.ncbi.nlm.nih.gov/33173867)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Prevalence of underlying diseases in died cases of {COVID}-19: A systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/33095835)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Relationships between hyperinsulinaemia, magnesium, vitamin D, thrombosis and {COVID}-19: rationale for clinical management.](https://pubmed.ncbi.nlm.nih.gov/32938758)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### NOS3 increases the amount of nitric oxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7876 ! NOS3) increases a(CHEBI:16480 ! \"nitric oxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Endothelial cells also synthesise Ch-S; the enzyme endothelial nitric oxide synthase (eNOS) is traditionally thought to mediate synthesis of nitric oxide (NO); however, it has been established that when eNOS is membrane bound, it is no longer able to synthesise NO."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 increases the amount of SERPINE1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases p(HGNC:8583 ! SERPINE1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase , SOD and catalase and decreased production of glutathione , further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins , membranes and haem.68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1.64 70 Hyperinsulinaemia increases production of PAI-1 , coupled with MgD-induced increased production of PAI-1 that inhibits fibrinolysis , together they compound coagulation risk ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IL6 increases the amount of SERPINE1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) increases p(HGNC:8583 ! SERPINE1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase , SOD and catalase and decreased production of glutathione , further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins , membranes and haem.68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1.64 70 Hyperinsulinaemia increases production of PAI-1 , coupled with MgD-induced increased production of PAI-1 that inhibits fibrinolysis , together they compound coagulation risk ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glucose increases the amount of acetoacetate\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17234 ! glucose) increases a(CHEBI:13705 ! acetoacetate)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Glucose oxidation consumes four nicotinamide adenine dinucleotide (NAD+) to produce two acetyl moieties, whereas beta oxidation consumes two, ketolysis one and acetoacetate none (XREF_SUPPLEMENTARY); consequently, glucose oxidation depletes the NAD+ intracellularpool more than the other three substrates combined, thereby decreasing NAD+ availability for mitochondrial deacetylase sirtuin 3 (SIRT3) activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glucose increases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17234 ! glucose) increases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore, some hydroxylation of inactive to active vitamin D occurs intracellularly within other tissues and may be inhibited by excessive mitochondrial ROS and NAD+ glucose metabolism depletion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Reactive oxygen species increases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:26523 ! \"reactive oxygen species\") increases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thus, the association with lower vitamin D may be both a marker and maker of poor health due to hyperinsulinaemia inducing low vitamin D status and availability, as a result of vitamin D sequestration into adipocytes and prevention of hydroxylation by mitochondrial ROS sensitive 1alpha-25OHD-hydroxylase."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Reactive oxygen species decreases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:26523 ! \"reactive oxygen species\") decreases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore, some hydroxylation of inactive to active vitamin D occurs intracellularly within other tissues and may be inhibited by excessive mitochondrial ROS and NAD+ glucose metabolism depletion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CAT increases the amount of glutathione\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1516 ! CAT) increases a(CHEBI:16856 ! glutathione)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endothelial Cells increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SIRT3 increases the amount of SOD2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:14931 ! SIRT3) increases p(HGNC:11180 ! SOD2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperglycaemia increases haemoglobin glycation damage. xref A study published in Diabetes Research and Clinical Practice analysed 132 patients with COVID-19, classified into three groups based on haemoglobin A1c (HbA1c) levels; results found a significant linear negative correlation between haemoglobin oxygen saturation (SaO 2 )and HbA1c, p=0.01. xref Hyperinsulinaemia inhibits beta-oxidation and ketolysis, thereby enforcing cellular ATP generation from glucose substrate. xref Glucose oxidation consumes four nicotinamide adenine dinucleotide (NAD+) to produce two acetyl moieties, whereas beta-oxidation consumes two, ketolysis one and acetoacetate none ( xref ); consequently, glucose oxidation depletes the NAD+ intracellularpool more than the other three substrates combined, thereby decreasing NAD+ availability for mitochondrial deacetylase sirtuin 3 (SIRT3) activity. xref NAD+ dependent SIRT3 activates manganese superoxide dismutase 2 (MnSOD2) via acetylation of lysine residue 68 (K68) and increases the production of NADPH via isocitrate dehydrogenase (IDH2) to drive the reduction of oxidised glutathione (GSSG) to reduced glutathione (GSH)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SIRT3 increases the amount of SOD2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:14931 ! SIRT3) increases p(HGNC:11180 ! SOD2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "18 19 NAD+ dependent SIRT3 activates manganese superoxide dismutase 2 (MnSOD2) via acetylation of lysine residue 68 (K68) and increases the production of NADPH via isocitrate dehydrogenase (IDH2) to drive the reduction of oxidised glutathione (GSSG) to reduced glutathione (GSH)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Calcium(2+) decreases the amount of vit D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29108 ! \"calcium(2+)\") decreases a(TEXT:\"vit D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Even with supplementation or adequate sun exposure, a mismatched low vitamin K2 status needed to move Ca 2+ into bones would further result in increased plasma Ca 2+ inhibition of vit D activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS increases the amount of dioxygen\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases a(CHEBI:15379 ! dioxygen)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Coupled with the glucose fuelling effect on the cellular redox system, insulin increases mitochondrial production of reactive oxygen species (ROS) via generation of ceramides."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS increases the amount of phosphate(3-)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases a(CHEBI:18367 ! \"phosphate(3-)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS increases the amount of calcium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases a(CHEBI:29108 ! \"calcium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS increases the amount of calcium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases a(CHEBI:29108 ! \"calcium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Insulin stimulates bone resorption and calcium (Ca 2+) and phosphate release."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS activates Lipogenesis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases bp(MESH:D050155 ! Lipogenesis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "39 Insulin mediates de novo lipogenesis and adipogenesis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS activates Adipogenesis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) increases bp(MESH:D050156 ! Adipogenesis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "39 Insulin mediates de novo lipogenesis and adipogenesis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS decreases the amount of magnesium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases a(CHEBI:18420 ! \"magnesium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Hyperinsulinaemia causes increased renal excretion of magnesium, and insulin resistance (IR) reduces intracellular magnesium levels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium(2+) decreases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:18420 ! \"magnesium(2+)\") decreases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion, reduced intracellular levels, lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium(2+) inhibits Blood\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:18420 ! \"magnesium(2+)\") decreases bp(MESH:D001769 ! Blood)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mg2 + is required to transport vitamin D around in the blood , and activation of vitamin D to its active form via hepatic and renal hydroxylation is Mg2 + dependent.65 Clinical manifestations of MgD include , but are not limited to , hyperinsulinaemia 's metabolic pathologies such as T2DM , osteoporosis , vitamin D metabolism disorders , CVD and hyperglycaemia.66 67 There is a strong relationship between MgD and increased oxidative stress ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium(2+) inhibits Blood\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:18420 ! \"magnesium(2+)\") decreases bp(MESH:D001769 ! Blood)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Mg2 + is required to transport vitamin D around in the blood , and activation of vitamin D to its active form via hepatic and renal hydroxylation is Mg2 + dependent.65 Clinical manifestations of MgD include , but are not limited to , hyperinsulinaemia 's metabolic pathologies such as T2DM , osteoporosis , vitamin D metabolism disorders , CVD and hyperglycaemia.66 67 There is a strong relationship between MgD and increased oxidative stress ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cell Survival increases the amount of motexafin gadolinium\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D002470 ! \"Cell Survival\") increases a(CHEBI:50161 ! \"motexafin gadolinium\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "64 MgD has been implicated in perturbations of pancreatic beta-cell function , hyperinsulinaemia , IR and CVD due to its diverse and essential roles in an extensive list of cellular metabolic pathways , not least counting ATP transport , DNA repair capacity and cell viability ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium increases the amount of cytokine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") increases a(TEXT:cytokine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "68 69 MgD increases the production of cytokines : IL-1beta , IL-6 , tumour necrosis factor alpha and PAI-1 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium increases the amount of IL1B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") increases p(HGNC:5992 ! IL1B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium increases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") increases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium increases the amount of SERPINE1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") increases p(HGNC:8583 ! SERPINE1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR MgD increases the production of cytokines : IL-1beta, IL-6, tumour necrosis factor alpha and PAI-1."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium decreases the amount of GPX\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") decreases p(FPLX:GPX ! GPX)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium decreases the amount of SOD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") decreases p(FPLX:SOD ! SOD)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Motexafin gadolinium decreases the amount of CAT\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:50161 ! \"motexafin gadolinium\") decreases p(HGNC:1516 ! CAT)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium atom increases the amount of vitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:25107 ! \"magnesium atom\") increases a(CHEBI:27300 ! \"vitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium atom increases the amount of vitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:25107 ! \"magnesium atom\") increases a(CHEBI:27300 ! \"vitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Magnesium atom increases the amount of vitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:25107 ! \"magnesium atom\") increases a(CHEBI:27300 ! \"vitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperinsulinaemia promotes : magnesium depletion via increased renal excretion , reduced intracellular levels , lowers vitamin D status via sequestration into adipocytes and hydroxylation activation inhibition ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells (RBCs) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge, thus preventing thrombi and agglutination via maintaining electrorepulsion driven dispersion."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocyte Count inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004906 ! \"Erythrocyte Count\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SOD increases the amount of glutathione\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:SOD ! SOD) increases a(CHEBI:16856 ! glutathione)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "MgD drives decreased expression and activity of antioxidant defence enzymes glutathione peroxidase, SOD and catalase and decreased production of glutathione, further taxing mitochondrial health and subsequent increased ROS production and consequent oxidative damage to proteins, membranes and haem."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMOX1 increases the amount of carbon monoxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5013 ! HMOX1) increases a(CHEBI:17245 ! \"carbon monoxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMOX1 increases the amount of carbon monoxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5013 ! HMOX1) increases a(CHEBI:17245 ! \"carbon monoxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMOX1 increases the amount of 5-aminolevulinic acid\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5013 ! HMOX1) increases a(CHEBI:17549 ! \"5-aminolevulinic acid\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem-oxygenase (HO) induces hepatic aminolevulinic acid synthase (ALAS1) (enzyme in the first step of haem synthesis) activity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMOX1 increases the amount of iron atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5013 ! HMOX1) increases a(CHEBI:18248 ! \"iron atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMOX1 increases the amount of biliverdin(2-)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5013 ! HMOX1) increases a(CHEBI:57991 ! \"biliverdin(2-)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IR decreases the amount of magnesium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:IR) decreases a(CHEBI:18420 ! \"magnesium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "XREF_BIBR Hyperinsulinaemia causes increased renal excretion of magnesium, and insulin resistance (IR) reduces intracellular magnesium levels."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NAD increases the amount of NADPH\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:13389 ! NAD) increases a(CHEBI:16474 ! NADPH)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "18 19 NAD + dependent SIRT3 activates manganese superoxide dismutase 2 ( MnSOD2 ) via acetylation of lysine residue 68 ( K68 ) and increases the production of NADPH via isocitrate dehydrogenase ( IDH2 ) to drive the reduction of oxidised glutathione ( GSSG ) to reduced glutathione ( GSH ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension activates heme oxidation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases bp(GO:\"GO:0006788\" ! \"heme oxidation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deep vein thrombosis occurrence increases significantly with elevated carbon monoxide concentrations , increasing the risk of pulmonary emboli ( PE ) and acute coronary syndrome.23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension activates heme oxidation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases bp(GO:\"GO:0006788\" ! \"heme oxidation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypertension activates heme oxidation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006973 ! Hypertension) increases bp(GO:\"GO:0006788\" ! \"heme oxidation\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deep vein thrombosis occurrence increases significantly with elevated carbon monoxide concentrations , increasing the risk of pulmonary emboli ( PE ) and acute coronary syndrome.23 24 Hyperinsulinaemia drives the pathogenesis of obesity , CVD , T2DM , hypertension , increased haem oxidation , haem breakdown , endogenous carbon monoxide production and resultant increased thrombi risk ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Blood Platelets inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D001792 ! \"Blood Platelets\") decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Carbon dioxide increases the amount of coproporphyrinogen III\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16526 ! \"carbon dioxide\") increases a(CHEBI:15439 ! \"coproporphyrinogen III\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This may contribute to respiration independent increased carbon dioxide levels, as extramitochondrial haem synthesis step 5, produces four carbon dioxide molecules via the decarboxylation of uroporphyrinogen III to coproporphyrinogen III, further taxing the external respiration system."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of magnesium atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases a(CHEBI:25107 ! \"magnesium atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "A reduction in gastric acidity production by PPI drugs decrease Mg 2+ solubility, thus decreasing absorption potential in the gastrointestinal (GI) tract; metformin causes GI Mg 2+ loss, while diuretics and hyperglycaemia decrease renal tubular reabsorption of Mg 2+."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of Gly-Ile\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases a(CHEBI:73907 ! Gly-Ile)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "A reduction in gastric acidity production by PPI drugs decrease Mg 2+ solubility, thus decreasing absorption potential in the gastrointestinal (GI) tract; metformin causes GI Mg 2+ loss, while diuretics and hyperglycaemia decrease renal tubular reabsorption of Mg 2+."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heme activates catabolic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:30413 ! heme) increases bp(GO:\"GO:0009056\" ! \"catabolic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Hyperinsulinaemia and hyperglycaemia increases oxidised haem which increases haem catabolism, resulting in chronic increased CO production."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heme activates catabolic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:30413 ! heme) increases bp(GO:\"GO:0009056\" ! \"catabolic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heme decreases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:30413 ! heme) decreases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemoglobin increases the amount of heme\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:35143 ! hemoglobin) increases a(CHEBI:30413 ! heme)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemoglobin activates catabolic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:35143 ! hemoglobin) increases bp(GO:\"GO:0009056\" ! \"catabolic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemoglobin increases the amount of HMOX1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:35143 ! hemoglobin) increases p(HGNC:5013 ! HMOX1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemoglobin decreases the amount of 24,25-Dihydroxyvitamin D\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:35143 ! hemoglobin) decreases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Years of hyperinsulinaemia that would manifest overt pathologies such as obesity, CVD, hypertension and cancer would come with an already heavy-risk burden list, which includes : increased haemoglobin glycation damage, intracellular haem oxidation with reduced antioxidative capacity, increased haem-oxygenase haem catabolism thus producing increased endogenous carbon monoxide production, leading to increased risk of DVT and subsequent PE and decreased mitochondrial vitamin D hydroxylase activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes increases the amount of LYST\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) increases p(HGNC:1968 ! LYST)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Erythrocytes inhibits heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004912 ! Erythrocytes) decreases bp(GO:\"GO:0007157\" ! \"heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Red blood cells ( RBCs ) and platelets produce Ch-S that aids in maintaining their extracellular side plasma membrane negative charge , thus preventing thrombi and agglutination via maintaining electrorepulsion-driven dispersion.50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO.51-54 Membrane-bound eNOS lacks association with intracellular calmodulin binding ; this results in a closed conformation of the heterodimer enzyme ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ferroheme b increases the amount of carbon monoxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17627 ! \"ferroheme b\") increases a(CHEBI:17245 ! \"carbon monoxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ferroheme b increases the amount of iron atom\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17627 ! \"ferroheme b\") increases a(CHEBI:18248 ! \"iron atom\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ferroheme b increases the amount of biliverdin(2-)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17627 ! \"ferroheme b\") increases a(CHEBI:57991 ! \"biliverdin(2-)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Haem is catabolised and degraded by HO (gene Hmox1 and Hmox2), producing ferrous iron, biliverdin and carbon monoxide, resulting in increased plasma ferritin and bilirubin."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Zinc increases the amount of NOS3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015032 ! Zinc) increases p(HGNC:7876 ! NOS3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting.58 Following eNOS sulfate synthesis , sulfotransferase enzyme SULT2B1b catalyses cholesterol sulfurylation , producing Ch-S ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Zinc increases the amount of NOS3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015032 ! Zinc) increases p(HGNC:7876 ! NOS3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Zinc increases the amount of NOS3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015032 ! Zinc) increases p(HGNC:7876 ! NOS3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "57 The catalytic activity of eNOS is zinc dependent , a deficiency in zinc would result in inhibition of eNOS sulfate synthesis and inhibition of eNOS may result in increased clotting.58 Following eNOS sulfate synthesis , sulfotransferase enzyme SULT2B1b catalyses cholesterol sulfurylation , producing Ch-S ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Admission hyperglycaemia as a predictor of mortality in patients hospitalized with {COVID}-19 regardless of diabetes status: data from the Spanish {SEMI}-{COVID}-19 Registry.](https://pubmed.ncbi.nlm.nih.gov/33063540)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Lactate decreases the amount of ATP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24996 ! lactate) decreases a(CHEBI:15422 ! ATP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Under anaerobic conditions , glucose ferments into lactate , which produces a limited amount of adenosine triphosphate ( ATP ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Safety, tolerability, and immunogenicity of an inactivated {SARS}-{CoV}-2 vaccine in healthy adults aged 18-59 years: a randomised, double-blind, placebo-controlled, phase 1/2 clinical trial.](https://pubmed.ncbi.nlm.nih.gov/33217362)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Admission fasting blood glucose predicts 30-day poor outcome in patients hospitalized for {COVID}-19 pneumonia.](https://pubmed.ncbi.nlm.nih.gov/32627338)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and cardiovascular consequences: Is the endothelial dysfunction the hardest challenge?](https://pubmed.ncbi.nlm.nih.gov/32871306)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19: A review of therapeutic strategies and vaccine candidates.](https://pubmed.ncbi.nlm.nih.gov/33217545)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin and {COVID}-19: From cellular mechanisms to reduced mortality.](https://pubmed.ncbi.nlm.nih.gov/32750451)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Safety and efficacy of the {ChAdOx1} {nCoV}-19 vaccine ({AZD1222}) against {SARS}-{CoV}-2: an interim analysis of four randomised controlled trials in Brazil, South Africa, and the {UK}.](https://pubmed.ncbi.nlm.nih.gov/33306989)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Safety and Efficacy of the {BNT162b2} {mRNA} Covid-19 Vaccine.](https://pubmed.ncbi.nlm.nih.gov/33301246)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Vaccination inhibits Asymptomatic Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D014611 ! Vaccination) decreases bp(MESH:D058345 ! \"Asymptomatic Infections\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These data do not address whether vaccination prevents asymptomatic infection ; a serologic end point that can detect a history of infection regardless of whether symptoms were present ( SARS-CoV-2 N-binding antibody ) will be reported later ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Immunization decreases the amount of COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007114 ! Immunization) decreases a(TEXT:COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The results demonstrate that Covid-19 can be prevented by immunization , provide proof of concept that RNA-based vaccines are a promising new approach for protecting humans against infectious diseases , and demonstrate the speed with which an RNA-based vaccine can be developed with a sufficient investment of resources ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Risk of Metformin in Patients With Type 2 Diabetes With {COVID}-19: A Preliminary Retrospective Report.](https://pubmed.ncbi.nlm.nih.gov/32955785)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin Is Associated with Higher Incidence of Acidosis, but Not Mortality, in Individuals with {COVID}-19 and Pre-existing Type 2 Diabetes.](https://pubmed.ncbi.nlm.nih.gov/32861268)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin use is associated with reduced mortality rate from coronavirus disease 2019 ({COVID}-19) infection.](https://pubmed.ncbi.nlm.nih.gov/32844132)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### AMPK phosphorylates ACE2 on S680\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AMPK ! AMPK) directlyIncreases p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 680))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "AMPK itself has been shown to increase ACE2 stability by phosphorylating ACE2 Ser 680 in human umbilical vein endothelial cells and humanstorm or vascular damage and finally lead to reduction in mortality rate."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AMPK phosphorylates ACE2 on S680\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AMPK ! AMPK) directlyIncreases p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 680))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "AMPK itself has been shown to increase ACE2 stability by phosphorylating ACE2 Ser 680 in human umbilical vein endothelial cells and human embryonic kidney 293cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of adenosine 5'-monophosphate\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases a(CHEBI:16027 ! \"adenosine 5'-monophosphate\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "First, metformin works by inhibiting the ATP synthesis to increase the AMP or ADP levels that in turn indirectly induce the activation of the AMP activated protein kinase (AMPK) in hepatocytes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of MTOR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases p(HGNC:3942 ! MTOR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "[9, 10] Second, metformin inhibits the mammalian target of rapamycin (mTOR) signaling via liver kinase B1 (LKB1)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of MTOR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases p(HGNC:3942 ! MTOR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The PI3K/AKT/mTOR pathway plays a major role in MERS-CoV infection, which may also be implicated in the pathogenesis of SARS-CoV-2 infection, given many similarities between these two viruses.Therefore, inhibition of mTOR signaling by metformin may result in a reduction of SARS-CoV-2 infectivity and mortality from COVID-19 infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of MTOR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases p(HGNC:3942 ! MTOR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Therefore, inhibition of mTOR signaling by metformin may result in a reduction of SARS-CoV-2 infectivity and mortality from COVID-19 infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin increases the amount of MTOR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) increases p(HGNC:3942 ! MTOR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Second, metformin inhibits the mammalian target of rapamycin (mTOR) signaling via liver kinase B1 (LKB1)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin inhibits Mortality\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) decreases bp(MESH:D009026 ! Mortality)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "[ 11 ] Finally , through activation of AMPK , metformin can inhibit the inflammatory response that could potentially contribute to mortality through mechanism such as cytokine storm and vascular damage.10 , 12 Therefore , metformin use will decrease the inflammatory response , reducing the incidence of cytokine storm or vascular damage and finally lead to reduction in mortality rate ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SARS-CoV-2 Receptor Binding Domain inhibits SARS-CoV-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:\"SARS-CoV-2 Receptor Binding Domain\") decreases bp(MESH:D000086402 ! SARS-CoV-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These changes can lead to decreased binding between ACE2 receptor and SARS-CoV-2 Receptor Binding Domain (RBD) due to steric hindrance by the addition of a large sized PO 4 3- molecule, thus reducing SARS-CoV-2 infectivity."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Role of angiotensin-converting enzyme 2 ({ACE2}) in {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32660650)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### ACE2 increases the amount of Angiotensin-2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases a(TEXT:Angiotensin-2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 catalyzes angiotensin II conversion to angiotensin-(1-7), and the ACE2/angiotensin-(1-7)/MAS axis counteracts the negative effects of the renin-angiotensin system (RAS), which plays important roles in maintaining the physiological and pathophysiological balance of the body."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Microtubule organizing center attachment site increases the amount of isoxaflutole\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0034992\" ! \"microtubule organizing center attachment site\") increases a(CHEBI:141213 ! isoxaflutole)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Restoring the balance between the RAS and ACE2 / angiotensin - ( 1-7 ) / MAS may help attenuate organ injuries ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Escape from neutralizing antibodies by {SARS}-{CoV}-2 spike protein variants.](https://pubmed.ncbi.nlm.nih.gov/33112236)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Efficacy and Safety of the {mRNA}-1273 {SARS}-{CoV}-2 Vaccine.](https://pubmed.ncbi.nlm.nih.gov/33378609)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### COVID-19 activates Death\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:COVID-19) increases bp(MESH:D003643 ! Death)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Thirty participants in the trial had severe Covid-19 ; all 30 were in the placebo group ( indicating vaccine efficacy of 100 % [ 95 % CI , could not be estimated to 1.0 ] ) , and one death among these participants was attributed to Covid-19 ( Table S16 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Virus-Receptor Interactions of Glycosylated {SARS}-{CoV}-2 Spike and Human {ACE2} Receptor.](https://pubmed.ncbi.nlm.nih.gov/32841605)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The {SARS}-{CoV}-2 Spike Glycoprotein as a Drug and Vaccine Target: Structural Insights into Its Complexes with {ACE2} and Antibodies.](https://pubmed.ncbi.nlm.nih.gov/33105869)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Genetic Variants of {SARS}-{CoV}-2-What Do They Mean?](https://pubmed.ncbi.nlm.nih.gov/33404586)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycaemia in critically ill patients: marker or mediator of mortality?](https://pubmed.ncbi.nlm.nih.gov/16834760)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Safety and efficacy of an {rAd26} and {rAd5} vector-based heterologous prime-boost {COVID}-19 vaccine: an interim analysis of a randomised controlled phase 3 trial in Russia.](https://pubmed.ncbi.nlm.nih.gov/33545094)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Diabetes, obesity, metabolism, and {SARS}-{CoV}-2 infection: the end of the beginning.](https://pubmed.ncbi.nlm.nih.gov/33529600)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SARS-CoV-2 decreases the amount of INS\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086402 ! SARS-CoV-2) decreases p(HGNC:6081 ! INS)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed , the development of more classical T1D has been temporally associated with exposure to viral infection ( Vehik et al ., 2019 ) ; hence , it seems reasonable to query mechanisms whereby SARS-CoV-2 might trigger insulin deficiency and T1D ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Quantifying asymptomatic infection and transmission of {COVID}-19 in New York City using observed cases, serology, and testing capacity.](https://pubmed.ncbi.nlm.nih.gov/33571106)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Global pandemics interconnected - obesity, impaired metabolic health and {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/33479538)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Predictors of hospital discharge and mortality in patients with diabetes and {COVID}-19: updated results from the nationwide {CORONADO} study.](https://pubmed.ncbi.nlm.nih.gov/33599800)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Diabetes and altered glucose metabolism with aging.](https://pubmed.ncbi.nlm.nih.gov/23702405)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Knowledge synthesis of 100 million biomedical documents augments the deep expression profiling of coronavirus receptors.](https://pubmed.ncbi.nlm.nih.gov/32463365)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SARS Virus decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The question of whether coronaviruses have evolved to exploit such a network may be relevant to pursue , particularly given that downregulation of ACE2 by SARS-CoV has been reported previously and is associated with poor clinical outcomes ( Glowacka et al ., 2010 ; Kuba et al ., 2010 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  BHLHE22 increases the amount of ab\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:O09029 ! O09029) increases p(UP:Q24174 ! Q24174)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Figure 2 -- figure supplement 8. : UMAP based dimensionality reduction plot denoting cell types (A, C, E) and corresponding expression levels of Ace2, Dpp4, Anpep and Tmprss2 (B, D, F) from multiple studies : (A-B) adipose stromovascular fraction from mice treated with PBS or a beta-3 adrenergic agonist (Rajbhandari and Arneson, 2019), (C-D) murine adipose stromovascular fraction from Tabula Muris (Tabula Muris Consortium et al., 2018), and (E-F) adipose tissue from mice subjected to 24 hr of cold shock and processed for single nucleus RNA sequencing (Rajbhandari and Arneson, 2019)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The avian coronavirus infectious bronchitis virus undergoes direct low-{pH}-dependent fusion activation during entry into host cells.](https://pubmed.ncbi.nlm.nih.gov/16537586)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Anticoagulant treatment in {COVID}-19: a narrative review.](https://pubmed.ncbi.nlm.nih.gov/32809158)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Looking beyond {COVID}-19 vaccine phase 3 trials.](https://pubmed.ncbi.nlm.nih.gov/33469205)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Airway glucose concentrations and effect on growth of respiratory pathogens in cystic fibrosis.](https://pubmed.ncbi.nlm.nih.gov/16844431)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Therapeutic strategies for glucose transporter 1 deficiency syndrome.](https://pubmed.ncbi.nlm.nih.gov/31464092)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Dual nature of human {ACE2} glycosylation in binding to {SARS}-{CoV}-2 spike.](https://pubmed.ncbi.nlm.nih.gov/33903171)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypertension and related diseases in the era of {COVID}-19: a report from the Japanese Society of Hypertension Task Force on {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32737423)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### ACE2 inhibits Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, it might also attenuate the inhibition of RAS by ACE2 in infected organs."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 inhibits Renin-Angiotensin System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However , it might also attenuate the inhibition of RAS by ACE2 in infected organs ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Comorbidity activates Atherosclerosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D015897 ! Comorbidity) increases bp(MESH:D050197 ! Atherosclerosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These comorbidities accelerate atherosclerosis , which leads to cardiovascular complications and may increase the severity and mortality of COVID-19 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity, Abdominal activates Hypertension\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D056128 ! \"Obesity, Abdominal\") increases bp(MESH:D006973 ! Hypertension)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity, Abdominal activates Lipid Metabolism, Inborn Errors\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D056128 ! \"Obesity, Abdominal\") increases bp(MESH:D008052 ! \"Lipid Metabolism, Inborn Errors\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Obesity, Abdominal activates Oxidative Stress\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D056128 ! \"Obesity, Abdominal\") increases bp(MESH:D018384 ! \"Oxidative Stress\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore , obesity , especially abdominal obesity , increases inflammatory cytokines and oxidative stress and causes hypertension , diabetes , and dyslipidemia [ 21 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Comprehensive characterization of N- and O- glycosylation of {SARS}-{CoV}-2 human receptor angiotensin converting enzyme 2.](https://pubmed.ncbi.nlm.nih.gov/33135055)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metabolic host response and therapeutic approaches to influenza infection.](https://pubmed.ncbi.nlm.nih.gov/32161622)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Spermidine decreases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16610 ! spermidine) decreases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Spermidine inhibits Influenza, Human\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16610 ! spermidine) decreases bp(MESH:D007251 ! \"Influenza, Human\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationFig."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Spermidine inhibits Influenza, Human\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16610 ! spermidine) decreases bp(MESH:D007251 ! \"Influenza, Human\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon activates glucose import\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases bp(GO:\"GO:0046323\" ! \"glucose import\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon increases the amount of SAT1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases p(HGNC:10540 ! SAT1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs , lowering the levels of spermidine and spermine and thereby dampening influenza replication Role of IFN in IDO activation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon activates Oxygen Consumption\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases bp(MESH:D010101 ! \"Oxygen Consumption\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It has also been demonstrated that type I IFN can stimulate oxygen consumption in a range of cells , including conventional dendritic cells ( DCs ) , keratinocytes , and memory T cells [ 117 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon increases the amount of NOS2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases p(HGNC:7873 ! NOS2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The IFN response induces greatly increased levels of iNOS in the lungs of infected mice, leading to the production of a significant amount of NO and peroxynitrite species, which are among the most important pathogenic factors in influenza virus-induced pneumonia in mice [147]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon decreases the amount of PTGER4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) decreases p(HGNC:9596 ! PTGER4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "IFNs affect the lipid metabolism through downregulation of SREBP-2 , leading to higher cholesterol biosynthesis , and suppressed host cell apoptosis and PGE-2 production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of reactive oxygen species\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It is also reported that influenza infection significantly increases ROS production by inducing Nox4 , and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [ 16 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of reactive oxygen species\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Xanthine oxidase is also an ROS-producing enzyme that is induced by influenza infection [23, 24] , and its inhibition can hinder ROS increase in the cell.On the other hand, increased expression of SOD1 reduces influenza virus titers within the cell [25] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of reactive oxygen species\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It is also reported that influenza infection significantly increases ROS production by inducing Nox4, and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [16] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of mTORC1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases p(FPLX:mTORC1 ! mTORC1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of mTORC1&2 signaling and downstream factors by influenza infection may have an essential role in the upregulation of these metabolic processes."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of mTORC1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases p(FPLX:mTORC1 ! mTORC1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, influenza M2 protein is capable of down-regulation of the mTORC1 inhibitor REDD1, thereby enhancing the mTORC1 activation [69]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates pentose-phosphate shunt\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0006098\" ! \"pentose-phosphate shunt\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The pentose phosphate shunt, as another glucose-consuming pathway, is enhanced by influenza infection to help produce more nucleotides, especially ATP."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates pentose-phosphate shunt\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0006098\" ! \"pentose-phosphate shunt\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The pentose phosphate shunt , as another glucose-consuming pathway , is enhanced by influenza infection to help produce more nucleotides , especially ATP ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates fatty acid biosynthetic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0006633\" ! \"fatty acid biosynthetic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this regard, the results of a study revealed that influenza infection could induce fatty acid biosynthesis and cholesterol metabolism in human lung basal epithelial tumor cells [87]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates cholesterol metabolic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0008203\" ! \"cholesterol metabolic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this regard, the results of a study revealed that influenza infection could induce fatty acid biosynthesis and cholesterol metabolism in human lung basal epithelial tumor cells [87]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of HIF1A\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases p(HGNC:4910 ! HIF1A)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned previously , the influenza infection induces upregulation of HIF-1alpha ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of NOX4\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases p(HGNC:7891 ! NOX4)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It is also reported that influenza infection significantly increases ROS production by inducing Nox4, and the proliferation of this virus in lung epithelial cells is dependent on redox-sensitive pathways activated by Nox4-derived ROS [16] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human increases the amount of PDC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases p(HGNC:8759 ! PDC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates Lipid Metabolism Disorders\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(MESH:D052439 ! \"Lipid Metabolism Disorders\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition to impaired metabolism in mitochondria, influenza infection induces severe peroxisomal lipid metabolism disorders, which can be inferred from abnormal levels of several specific long-chain fatty acids [101] (Fig. 1)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Spermine decreases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15746 ! spermine) decreases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Spermine inhibits Influenza, Human\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15746 ! spermine) decreases bp(MESH:D007251 ! \"Influenza, Human\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationFig."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Spermine inhibits Influenza, Human\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15746 ! spermine) decreases bp(MESH:D007251 ! \"Influenza, Human\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "SAT-1 can be upregulated by IFNs, lowering the levels of spermidine and spermine and thereby dampening influenza replicationRole of IFN in IDO activation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Glucose activates viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:17234 ! glucose) increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In a study by Kohio and Adamson, a dose-specific increase in influenza infection was associated with higher glucose levels, whereas the treatment of cells with glycolysis inhibitors remarkably suppressed the viral replication."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IFNG activates Macrophages\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:5438 ! IFNG) increases bp(MESH:D008264 ! Macrophages)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Also , activation of macrophages by IFNgamma induces expression of the ATP-citrate lyase enzyme ( ACLY ) , and blockage of ACLY activity reduces the production of ROS and nitric oxide [ 120 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACLY increases the amount of nitric oxide\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:115 ! ACLY) increases a(CHEBI:16480 ! \"nitric oxide\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Also, activation of macrophages by IFN induces expression of the ATP-citrate lyase enzyme (ACLY), and blockage of ACLY activity reduces the production of ROS and nitric oxide [120]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACLY increases the amount of reactive oxygen species\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:115 ! ACLY) increases a(CHEBI:26523 ! \"reactive oxygen species\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Also, activation of macrophages by IFN induces expression of the ATP-citrate lyase enzyme (ACLY), and blockage of ACLY activity reduces the production of ROS and nitric oxide [120]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Nitric oxide inhibits viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16480 ! \"nitric oxide\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "NO, on the other hand, inhibits viral replication via changes in the structure of viral proteins [34] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AKT increases the amount of ATP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AKT ! AKT) increases a(CHEBI:15422 ! ATP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AKT decreases the amount of MYC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AKT ! AKT) decreases p(HGNC:7553 ! MYC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, AKT-dependent inactivation of FoxOs can increase glycolysis [72, 73] by removing the suppressive force of c-Myc [7476]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AKT decreases the amount of MYC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:AKT ! AKT) decreases p(HGNC:7553 ! MYC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Moreover, AKT-dependent inactivation of FoxOs can increase glycolysis [72, 73] by removing the suppressive force of c-Myc [74] [75] [76] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP activates Virus Diseases\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) increases bp(MESH:D014777 ! \"Virus Diseases\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "This study revealed that enhancing vacuolar-type ATPase (a proton pump essential for influenza uncoating) via increasing glucose metabolism and, as a result, higher available ATP resources, augments the virus infection [50]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, high ATP consumption and reduced -oxidation, as well as glucose oxidation by influenza infection, contribute to the ATP crisis and hence influenza-related multi-organ failureFig."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ATP decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:15422 ! ATP) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, high ATP consumption and reduced -oxidation, as well as glucose oxidation by influenza infection, contribute to the ATP crisis and hence influenza-related multi-organ failure [96] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MYC activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7553 ! MYC) increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MYC activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:7553 ! MYC) increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1 , glycolytic genes , and lactate dehydrogenase ( LDH ) , as the converter of pyruvate to lactate [ 77 , 78 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  PDPK1 increases the amount of AKT\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8816 ! PDPK1) increases p(FPLX:AKT ! AKT)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The viral hemagglutinin (HA) protein, along with virus replication, can upregulate PDPK1-mediated phosphorylation and activate AKT, which is required for induction of the mTORC1 signaling pathway by the influenza virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemagglutinins leads to the phosphorylation of PDPK1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006388 ! Hemagglutinins) increases p(HGNC:8816 ! PDPK1, pmod(go:0006468 ! \"protein phosphorylation\"))\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The viral hemagglutinin (HA) protein, along with virus replication, can upregulate PDPK1-mediated phosphorylation and activate AKT, which is required for induction of the mTORC1 signaling pathway by the influenza virus."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MTORC1 increases the amount of MYC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:mTORC1 ! mTORC1) increases p(HGNC:7553 ! MYC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "mTORC1 signaling, in turn, promotes c-Myc expression at the translational level [70]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pentose-phosphate shunt activates viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006098\" ! \"pentose-phosphate shunt\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, the pentose phosphate pathway (PPP), as another glucose-consuming pathway reported to be enhanced by IVI [61], contributes to a higher yield of nucleotides and ATP for viral replication [62]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  PDC activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8759 ! PDC) increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It has also been shown that influenza infection stimulates pDCs to enhance their glycolysis and develop a Warburg-like remodeling of energy metabolism ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LDH activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:LDH ! LDH) increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  PI3 increases the amount of ATP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8947 ! PI3) increases a(CHEBI:15422 ! ATP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance, IFN has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3/AKT-dependent manner, thereby increasing ATP production [116]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IFNB increases the amount of ATP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:IFNB ! IFNB) increases a(CHEBI:15422 ! ATP)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "For instance , IFNbeta has been shown to induce the glucose uptake of embryonic fibroblasts in a PI3 / AKT-dependent manner , thereby increasing ATP production [ 116 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NUP214 increases the amount of PKC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:8064 ! NUP214) increases p(FPLX:PKC ! PKC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Increased expression of influenza M2 protein can activate protein kinase C and increase reactive oxygen species (ROS) production [10] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Dactolisib activates hemostasis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:71952 ! dactolisib) increases bp(GO:\"GO:0007599\" ! hemostasis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed, BEZ235 induced hemostasis in the PI3K/mTOR pathway via phosphorylation of p85 and 4E-BP1 and through reconstitution of metabolic status, which was already altered by the virus [9]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Dactolisib activates hemostasis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:71952 ! dactolisib) increases bp(GO:\"GO:0007599\" ! hemostasis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Indeed , BEZ235 induced hemostasis in the PI3K / mTOR pathway via phosphorylation of p85 and 4E-BP1 and through reconstitution of metabolic status , which was already altered by the virus [ 9 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  PB1-F2 decreases the amount of SOD1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:82128 ! PB1-F2) decreases p(HGNC:11179 ! SOD1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "On the other hand, PB1-F2 decreases superoxide anion dismutase 1 (SOD1) expression and consequently disrupts the ROS scavenging process [11] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SLC2A1 activates glycolytic process\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11005 ! SLC2A1) increases bp(GO:\"GO:0006096\" ! \"glycolytic process\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Myc enhances glycolysis by upregulating expression of the glucose transporter GLUT1, glycolytic genes, and lactate dehydrogenase (LDH), as the converter of pyruvate to lactate [77, 78]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MTORC2 increases the amount of MYC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:mTORC2 ! mTORC2) increases p(HGNC:7553 ! MYC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, the NS1 protein can effectively promote the activity of mTORC2, which, in turn, upregulates c-Myc through FoxO inhibition [71]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MTORC2 increases the amount of MYC\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:mTORC2 ! mTORC2) increases p(HGNC:7553 ! MYC)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , the NS1 protein can effectively promote the activity of mTORC2 , which , in turn , upregulates c-Myc through FoxO inhibition [ 71 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Herbal Medicine inhibits Body Weight\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D029001 ! \"Herbal Medicine\") decreases bp(MESH:D001835 ! \"Body Weight\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The results showed that this herbal medicine could ameliorate weight loss and inflammatory mediators in infected mice through the regulation of amino acid , fatty acid , and arachidonic acid pathways [ 162 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Analysis of the {SARS}-{CoV}-2 spike protein glycan shield reveals implications for immune recognition.](https://pubmed.ncbi.nlm.nih.gov/32929138)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Multiple organ dysfunction in {SARS}-{CoV}-2: {MODS}-{CoV}-2.](https://pubmed.ncbi.nlm.nih.gov/32567404)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### COVID-19 activates Thrombosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086382 ! COVID-19) increases bp(MESH:D013927 ! Thrombosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The Sepsis-Induced Coagulopathy ( SIC ) Score , able to predicts likelihood of sepsis-induced coagulopathy , has been evaluated in the COVID-19 pandemic given concerns that COVID-19 may cause thrombosis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Angiotensin {II}-mediated microvascular thrombosis.](https://pubmed.ncbi.nlm.nih.gov/20975035)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [A Machine-Generated View of the Role of Blood Glucose Levels in the Severity of {COVID}-19](https://pubmed.ncbi.nlm.nih.gov/34395368)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Population-based incidence estimates of influenza-associated respiratory failure hospitalizations, 2003 to 2009.](https://pubmed.ncbi.nlm.nih.gov/23855650)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Proinflammatory mediators disrupt glucose homeostasis in airway surface liquid.](https://pubmed.ncbi.nlm.nih.gov/22623330)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Regulation of alveolar fluid clearance and {ENaC} expression in lung by exogenous angiotensin {II}.](https://pubmed.ncbi.nlm.nih.gov/22138610)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{ACE2} and diabetes: {ACE} of {ACEs}?](https://pubmed.ncbi.nlm.nih.gov/21115782)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{ACE2} deficiency shifts energy metabolism towards glucose utilization.](https://pubmed.ncbi.nlm.nih.gov/25484288)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role of serum surfactant protein D as a biomarker of exacerbation of chronic obstructive pulmonary disease.](https://pubmed.ncbi.nlm.nih.gov/26942111)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Higher blood glucose level associated with body mass index and gut microbiota in elderly people.](https://pubmed.ncbi.nlm.nih.gov/24936169)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The role and molecular mechanism of action of surfactant protein D in innate host defense against influenza A virus.](https://pubmed.ncbi.nlm.nih.gov/29951070)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Collectins and cationic antimicrobial peptides of the respiratory epithelia.](https://pubmed.ncbi.nlm.nih.gov/16966437)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia is associated with poor in-hospital outcome in elderly patients with acute ischemic stroke.](https://pubmed.ncbi.nlm.nih.gov/31374068)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Effective glucose metabolism maintains low intracellular glucose in airway epithelial cells after exposure to hyperglycemia.](https://pubmed.ncbi.nlm.nih.gov/31433692)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Early Observation and Mitigation of Challenges in Diabetes Management of {COVID}-19 Patients in Critical Care Units.](https://pubmed.ncbi.nlm.nih.gov/32444458)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### INS decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Typically, this has been managed by a combination of continuous intravenous insulin infusion and frequent glucose testing to improve glycemic control and reduce glucose variability to the minimum in order to reduce mortality in ICU (4)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The main aims of a safe and effective insulin regimen are to reduce contact frequency, reduce glucose variability, minimize risk of severe hypoglycemia, and improve overall glycemic control."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Response: commentary: consumer reports of \"keto flu\" associated with the ketogenic diet.](https://pubmed.ncbi.nlm.nih.gov/33392235)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The battle between influenza and the innate immune response in the human respiratory tract.](https://pubmed.ncbi.nlm.nih.gov/24265946)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Endothelial Cells activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2 , CCL5 and CD11b + cells and decrease IFNgamma secretion but did not affect IFNalpha , CCL2 , IL6 or TNFalpha , thus showing that endothelial cells are able to actually modulate the inflammatory response [ 71 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IRF3 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6118 ! IRF3) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IRF3 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6118 ! IRF3) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IRF3 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6118 ! IRF3) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  IRF3 decreases the amount of NF1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6118 ! IRF3) decreases p(HGNC:7765 ! NF1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon increases the amount of IL15\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases p(HGNC:5977 ! IL15)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "They do however appear to be closely involved in later stages of the inflammatory response, either through direct activation using the interferon receptor pathway (see below) or through type I IFN mediated trans-presentation of IL-15 by dendritic cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Interferon increases the amount of IL15\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases p(HGNC:5977 ! IL15)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "They do however appear to be closely involved in later stages of the inflammatory response, either through direct activation using the interferon receptor pathway (see below) or through type I IFN-mediated trans-presentation of IL-15 by dendritic cells (reviewed by Hwang and colleagues [47])."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hemagglutinins activates Disease\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006388 ! Hemagglutinins) increases bp(MESH:D004194 ! Disease)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Viruses that lacked glycosylation of HA were able to cause more disease in mice than viruses that were glycosylated.Three classes of lectin have been demonstrated to show anti-influenza activity - the C-type lectins, S-type lectins and pentraxins (reviewed in Ng and colleagues [34]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human inhibits viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human inhibits viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human activates innate immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It should be noted that the 2008 H1N1 virus also has a stop codon at 220, thus suggesting that it behaves similar to H7N1 with the six amino acid deletion.Finally, since there is evidence that severe influenza caused by certain influenza viruses such as H5N1 trigger an exaggerated innate immune response, are there therapeutic options in addition to antiviral therapy that may be utilized to \"buy time\" before the acquired immune response takes over?"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human decreases the amount of TNF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases p(HGNC:11892 ! TNF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human decreases the amount of TNF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases p(HGNC:11892 ! TNF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human decreases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human decreases the amount of IL6\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases p(HGNC:6018 ! IL6)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In influenza infection the type 1 interferons (IFN- and IFN-) are the major cytokines that limit viral replication with TNF, IL-1 and IL-6 recruiting immune cells to the sites of infection and producing inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Influenza, Human decreases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007251 ! \"Influenza, Human\") decreases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "With regard to the extacellular compartment the role of NA in cleaving the sialylated mucin and glycosylation of the HA has already been discussed.Within the cell it has been found for a long time that influenza has evolved a strategy to evade the powerful IFN defence mechanism [73] but in 1998 Garcia-Sastre and colleagues showed that the production of recombinant influenza lacking a gene called non-structural protein (NS1) was able to suppress this IFN pathway [74]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DDX58 decreases the amount of Proteasome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:19102 ! DDX58) decreases p(FPLX:Proteasome ! Proteasome)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Unlike many other results of ubiqutination , the change in RIG-I does not lead to degradation in the proteasome but actually is a critical step for subsequent interaction of RIG-I with MAVS though the exact mechanism how this is facilitated is not clearly understood [ 61 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM21 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11312 ! TRIM21) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM21 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11312 ! TRIM21) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM21 decreases the amount of NF1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11312 ! TRIM21) decreases p(HGNC:7765 ! NF1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30alpha inhibits NF1kappabeta, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1 , TRIM30alpha inhibits NF1kappabeta , TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1kappabeta , IRF3 and IRF7 , thus affecting the downstream activation pathway in viral infection ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of IRF7\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:6122 ! IRF7)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM27 decreases the amount of NF1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:9975 ! TRIM27) decreases p(HGNC:7765 ! NF1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "As mentioned above TRIM25 ubiquitinates and activates RIG-1, TRIM30 inhibits NF1, TRIM21 stabilizes IRF3 and TRIM27 inhibits NF1, IRF3 and IRF7, thus affecting the downstream activation pathway in viral infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TRIM25 increases the amount of DDX58\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:12932 ! TRIM25) increases p(HGNC:19102 ! DDX58)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "It will be remembered that TRIM25 is required to ubiqutinate and activate RIG-I leading to subsequent IFN production."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P07052 activates defense response to virus\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:P07052 ! P07052) increases bp(GO:\"GO:0051607\" ! \"defense response to virus\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Infection of macrophages or epithelial cells in the in vitro setting pointed to the latter scenario [52, 53], and transcription analysis showed that H5N1 was more potent than seasonal influenza in the production of inflammatory cytokines and chemokines.Once influenza encounters the cell there are pattern recognition receptors (PRR) that are stimulated to initiate an antiviral response leading to the production of interferon and cytokines and chemokines."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD binds mannose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:37684 ! mannose), p(HGNC:10803 ! SFTPD)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (XREF_FIG)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mannose bound to SFTPD inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:37684 ! mannose), p(HGNC:10803 ! SFTPD)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (XREF_FIG)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD binds mannose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:37684 ! mannose), p(HGNC:10803 ! SFTPD)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mannose bound to SFTPD inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:37684 ! mannose), p(HGNC:10803 ! SFTPD)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GEMIN2 increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10884 ! GEMIN2) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2, CCL5 and CD11b+ cells and decrease IFN secretion but did not affect IFN, CCL2, IL6 or TNF, thus showing that endothelial cells are able to actually modulate the inflammatory response [71]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SGCG binds SFTPD\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:10803 ! SFTPD), p(HGNC:10809 ! SGCG)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD bound to SGCG inhibits Hemagglutinins\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(HGNC:10803 ! SFTPD), p(HGNC:10809 ! SGCG)) decreases bp(MESH:D006388 ! Hemagglutinins)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast SP-D binds mannose type oligosaccharides and has been demonstrated to interact with influenza virus HA involved in haemagglutination inhibition, and is also able to mediate aggregation of viral particles and inhibit HA activity through interaction of the viral HA or NA (Fig. 2)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Generation of reactive oxygen species by endothelial and smooth muscle cells: influence of hyperglycemia and metformin.](https://pubmed.ncbi.nlm.nih.gov/17111300)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Effects of hypoxia on the alveolar epithelium.](https://pubmed.ncbi.nlm.nih.gov/16222038)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypocaloric support in the critically ill.](https://pubmed.ncbi.nlm.nih.gov/10227923)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Antibacterial activity of lysozyme and lactoferrin is inhibited by binding of advanced glycation-modified proteins to a conserved motif.](https://pubmed.ncbi.nlm.nih.gov/7489363)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The influence of {pH} on surface properties of lung surfactants.](https://pubmed.ncbi.nlm.nih.gov/7564483)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Evidence for a protective role of pulmonary surfactant protein D ({SP}-D) against influenza A viruses.](https://pubmed.ncbi.nlm.nih.gov/8040272)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Human blood dendritic cell subsets exhibit discriminative pattern recognition receptor profiles.](https://pubmed.ncbi.nlm.nih.gov/24444310)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glucose in bronchial aspirates increases the risk of respiratory {MRSA} in intubated patients.](https://pubmed.ncbi.nlm.nih.gov/16135681)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [High glucose induces human endothelial cell apoptosis through a phosphoinositide 3-kinase-regulated cyclooxygenase-2 pathway.](https://pubmed.ncbi.nlm.nih.gov/15653566)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Dengue virus induces and requires glycolysis for optimal replication.](https://pubmed.ncbi.nlm.nih.gov/25505078)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Airway glucose homeostasis: A new target in the prevention and treatment of pulmonary infection.](https://pubmed.ncbi.nlm.nih.gov/28610911)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The feasibility, safety and effectiveness of a ketogenic diet for refractory status epilepticus in adults in the intensive care unit.](https://pubmed.ncbi.nlm.nih.gov/30484010)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Survey of immune-related, mannose/fucose-binding C-type lectin receptors reveals widely divergent sugar-binding specificities.](https://pubmed.ncbi.nlm.nih.gov/21112966)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Airway macrophage and dendritic cell subsets in the resting human lung.](https://pubmed.ncbi.nlm.nih.gov/30806245)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glucose metabolism affects coagulation factors: The {NEO} study.](https://pubmed.ncbi.nlm.nih.gov/31325222)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Regulation of airway surface liquid volume and mucus transport by active ion transport.](https://pubmed.ncbi.nlm.nih.gov/16113411)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Severe acute respiratory syndrome coronavirus elicits a weak interferon response compared to traditional interferon-inducing viruses.](https://pubmed.ncbi.nlm.nih.gov/18781076)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Interferon activates Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:Interferon ! Interferon) increases bp(MESH:D013577 ! Syndrome)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Severe Acute Respiratory Syndrome Coronavirus Elicits a Weak Interferon Response Compared to Traditional Interferon-Inducing Viruses.Severe acute respiratory syndrome (SARS) is a newly identified infectious disease and its causative agent has been convincingly identified as a new member of the Coronaviridae family called SARS-CoV [1]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SARS Virus increases the amount of photon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SARS Virus increases the amount of IFNA\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  A0A5C1RGL7 decreases the amount of IRF3\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:A0A5C1RGL7 ! A0A5C1RGL7) decreases p(HGNC:6118 ! IRF3)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, Kopecky-Bromberg et al. [24] found that SARS-CoV open reading frame (ORF) 3b, ORF 6, and N proteins inhibit the expression of IRF-3, suggesting that, like other highly pathogenic viruses, SARS-CoV genome encodes more than one protein able to inhibit IFN.Therefore, altogether these results may help to explain the rapid rise in virus titers during the initial phase of SARS disease."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GNL1 increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4413 ! GNL1) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "A better understanding of SARS-CoV strategies to alter IFN- subtype expression may elucidate the mechanism underlying virusinduced airway disease.In summary, our data, together with previous results, demonstrate that SARS-CoV (HSR-1 strain) induces a weak IFN response in comparison with well-established IFN-inducing viruses, such as those for VSV and NDV."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GNL1 increases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4413 ! GNL1) increases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "A better understanding of SARS-CoV strategies to alter IFN- subtype expression may elucidate the mechanism underlying virus-induced airway disease.In summary, our data, together with previous results, demonstrate that SARS-CoV (HSR-1 strain) induces a weak IFN response in comparison with well-established IFN-inducing viruses, such as those for VSV and NDV."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The diabetic lung: An easy target for {SARS}-{CoV}-2?](https://pubmed.ncbi.nlm.nih.gov/32426928)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycemia is a strong predictor of poor prognosis in {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32712122)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Managing Hyperglycemia in the {COVID}-19 Inflammatory Storm.](https://pubmed.ncbi.nlm.nih.gov/32778570)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Profiling and characterization of {SARS}-{CoV}-2 mutants' infectivity and antigenicity.](https://pubmed.ncbi.nlm.nih.gov/32883949)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Induction of ketosis as a potential therapeutic option to limit hyperglycemia and prevent cytokine storm in {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/32942131)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Glycolytic process activates Macrophages\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006096\" ! \"glycolytic process\") increases bp(MESH:D008264 ! Macrophages)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Macrophages decreases the amount of lactate\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D008264 ! Macrophages) decreases a(CHEBI:24996 ! lactate)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Lactate decreases the amount of Interferon\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:24996 ! lactate) decreases p(FPLX:Interferon ! Interferon)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Third , an EKD , which could reduce high lactate formation by macrophages due to glycolysis , could favor the production of interferon type I , which are inhibited by excessive lactate production ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ketosis decreases the amount of STORM\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007662 ! Ketosis) decreases a(TEXT:STORM)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Induction of ketosis as a potential therapeutic option to limit hyperglycemia and prevent cytokine storm in COVID-19."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ketosis inhibits Hyperglycemia\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007662 ! Ketosis) decreases bp(MESH:D006943 ! Hyperglycemia)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Induction of ketosis as a potential therapeutic option to limit hyperglycemia and prevent cytokine storm in COVID-19."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Analysis of Risk Factors for 24 Patients With {COVID}-19 Developing From Moderate to Severe Condition.](https://pubmed.ncbi.nlm.nih.gov/33042873)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Alpha-Asp-Leu activates Data Analysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68596 ! alpha-Asp-Leu) increases bp(MESH:D000078332 ! \"Data Analysis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JH , and YY contributed to data analysis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Alpha-Asp-Leu activates Data Analysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68596 ! alpha-Asp-Leu) increases bp(MESH:D000078332 ! \"Data Analysis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JH , and YY contributed to data analysis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Alpha-Asp-Leu activates Collection\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68596 ! alpha-Asp-Leu) increases bp(MESH:D020471 ! Collection)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JL , and YZ contributed to data collection ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Alpha-Asp-Leu activates Collection\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:68596 ! alpha-Asp-Leu) increases bp(MESH:D020471 ! Collection)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JL , and YZ contributed to data collection ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HJV activates Data Analysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4887 ! HJV) increases bp(MESH:D000078332 ! \"Data Analysis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JH , and YY contributed to data analysis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HJV activates Data Analysis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4887 ! HJV) increases bp(MESH:D000078332 ! \"Data Analysis\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "DL , JH , and YY contributed to data analysis ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impaired glucose metabolism in hypertensive patients with/without the metabolic syndrome.](https://pubmed.ncbi.nlm.nih.gov/24832774)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hemostasis and ageing.](https://pubmed.ncbi.nlm.nih.gov/18947391)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Overview and utilization of the {NCI} thesaurus.](https://pubmed.ncbi.nlm.nih.gov/18629178)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hypertension and the prothrombotic state.](https://pubmed.ncbi.nlm.nih.gov/11095159)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Activity and responsiveness of the renin-angiotensin system in the aging rat.](https://pubmed.ncbi.nlm.nih.gov/11049862)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Effect of antihypertensive drugs on insulin, glucose, and lipid metabolism.](https://pubmed.ncbi.nlm.nih.gov/1828417)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Elevated paracellular glucose flux across cystic fibrosis airway epithelial monolayers is an important factor for Pseudomonas aeruginosa growth.](https://pubmed.ncbi.nlm.nih.gov/24124542)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Forskolin decreases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:42471 ! forskolin) decreases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The forskolin induced CFTR dependent inhibition of Ps."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Forskolin decreases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:42471 ! forskolin) decreases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The forskolin induced CFTR dependent inhibition of Ps."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Forskolin decreases the amount of CFTR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:42471 ! forskolin) decreases p(HGNC:1884 ! CFTR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The forskolin induced CFTR dependent inhibition of Ps."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glucose homeostasis across human airway epithelial cell monolayers: role of diffusion, transport and metabolism.](https://pubmed.ncbi.nlm.nih.gov/18781323)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Mucociliary clearance--a critical upper airway host defense mechanism and methods of assessment.](https://pubmed.ncbi.nlm.nih.gov/17218804)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [A review of low-carbohydrate ketogenic diets.](https://pubmed.ncbi.nlm.nih.gov/14525681)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Glycan clustering stabilizes the mannose patch of {HIV}-1 and preserves vulnerability to broadly neutralizing antibodies.](https://pubmed.ncbi.nlm.nih.gov/26105115)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Corticosteroids for the treatment of human infection with influenza virus: a systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/26123860)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The impact of aging on epithelial barriers.](https://pubmed.ncbi.nlm.nih.gov/28686506)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Cell based therapy for type 1 diabetes: should we take hyperglycemia into account?](https://pubmed.ncbi.nlm.nih.gov/30804929)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The Dual Role of Surfactant Protein-D in Vascular Inflammation and Development of Cardiovascular Disease.](https://pubmed.ncbi.nlm.nih.gov/31616435)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{SARS}-{CoV}-2-mediated inflammatory response in lungs: should we look at {RAGE}?](https://pubmed.ncbi.nlm.nih.gov/32372149)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### NFkappaB increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:NFkappaB ! NFkappaB) increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore , activation of RAGE causes not only an inflammatory gene expression profile but also a positive feed-forward loop , in which inflammatory stimuli activate NF-kappaB , which induces RAGE expression , followed by a sustained NF-kappaB activation [ 18 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  NFkappaB increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(FPLX:NFkappaB ! NFkappaB) directlyIncreases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Furthermore, activation of RAGE causes not only an inflammatory gene expression profile but also a positive feed-forward loop, in which inflammatory stimuli activate NF-kappaB, which induces RAGE expression, followed by a sustained NF-kappaB activation [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 plays an important role in the reninangiotensin system (RAS), and the imbalance between ACE/Ang II/AT1R pathway and ACE2/Ang (17)/Mas receptor pathway in the RAS system will lead to multi-system inflammation [4]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 plays an important role in the renin-angiotensin system (RAS), and the imbalance between ACE and Ang II/AT1R pathway and ACE2 and Ang (1-7)/Mas receptor pathway in the RAS system will lead to multi-system inflammation [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin-2 increases the amount of AGTR1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:Angiotensin-2) increases p(HGNC:336 ! AGTR1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of the angiotensin II receptor type 1 (AT1R) by Ang II leads to the induction of NF-B [9, 10], and subsequent inflammation through pathways distinct from those mediating classical Gq-induced signaling [11]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin-2 increases the amount of AGTR1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:Angiotensin-2) increases p(HGNC:336 ! AGTR1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Activation of the angiotensin II receptor type 1 (AT1R) by Ang II leads to the induction of NF-B [ xref , xref ], and subsequent inflammation through pathways distinct from those mediating classical Gq-induced signaling [ xref ]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin-2 increases the amount of MYB34\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:Angiotensin-2) increases p(UP:O64399 ! O64399)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II-mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE/And II/ ATR1 pathway.A."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Angiotensin-2 increases the amount of MYB34\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:Angiotensin-2) increases p(UP:O64399 ! O64399)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE and And II/ATR1 pathway."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MYB34 increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:O64399 ! O64399) increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II-mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE/And II/ ATR1 pathway.A."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MYB34 increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:O64399 ! O64399) increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In this commentary, based on a very recent research report, we intend to highlight how a new mechanism describing the RAGE transactivation produced by Ang II mediated ATR1 activation can run continuously and thus, reinforcing a sustained inflammation in lungs, due to the SARS-Cov-2-mediated imbalance of the ACE and And II/ATR1 pathway."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  MAS1 activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6899 ! MAS1) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "ACE2 plays an important role in the renin-angiotensin system (RAS), and the imbalance between ACE/Ang II/AT1R pathway and ACE2/Ang (1-7)/ Mas receptor pathway in the RAS system will lead to multisystem inflammation [4] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Morphogenesis and cytopathic effect of {SARS}-{CoV}-2 infection in human airway epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/32764693)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{COVID}-19 and Diabetes: The Importance of Controlling {RAGE}.](https://pubmed.ncbi.nlm.nih.gov/32760352)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### HMGB1 activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In particular , HMGB1 directly stimulates platelets through TLR4 , and RAGE , while S100 proteins released from neutrophils and platelets facilitate thrombus formation through RAGE activation ( 26 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 activates Blood Platelets\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases bp(MESH:D001792 ! \"Blood Platelets\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In particular , HMGB1 directly stimulates platelets through TLR4 , and RAGE , while S100 proteins released from neutrophils and platelets facilitate thrombus formation through RAGE activation ( 26 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Inflammatory response increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In other cells , including endothelial cells , airway smooth muscle cells , vascular smooth muscle cells , neurons , and immune cells , RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Inflammatory response increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0006954\" ! \"inflammatory response\") increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In other cells , including endothelial cells , airway smooth muscle cells , vascular smooth muscle cells , neurons , and immune cells , RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER increases the amount of AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:320 ! AGER) increases p(HGNC:320 ! AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In other cells, including endothelial cells, airway smooth muscle cells, vascular smooth muscle cells, neurons, and immune cells, RAGE expression is induced by inflammation and by local expression of RAGE ligands in a feed-forward loop that perpetuates inflammation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CASP1 increases the amount of IL18\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1499 ! CASP1) increases p(HGNC:5986 ! IL18)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CASP1 increases the amount of IL18\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1499 ! CASP1) increases p(HGNC:5986 ! IL18)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CASP1 increases the amount of IL1B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1499 ! CASP1) increases p(HGNC:5992 ! IL1B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CASP1 increases the amount of IL1B\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1499 ! CASP1) increases p(HGNC:5992 ! IL1B)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However, the cytopathic effect of the virus at the level of alveolar cells and vascular endothelium may induce massive pyroptosis, an inflammatory form of cell death linked to caspase 1 dependent activation of proinflammatory cytokines IL-1 and IL-18 by the NLR family pyrin domain containing 3 (NLRP3) inflammasome (5) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  BSG activates Metalloproteases\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1116 ! BSG) increases bp(MESH:D045726 ! Metalloproteases)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "AGEs induced CD147 glycosylation in endothelial cells may increase the activity of metalloproteinases (MMPs) and loosen tight junctions."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  INS decreases the amount of sRAGE\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases a(TEXT:sRAGE)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "An encouragement to chase this track more in depth comes from the observation that in non-COVID-19 critically ill hyperglycemic patients admitted at ICU, plasma levels of sRAGE are higher in DM vs. non DM patients, and reduced by intensive insulin therapy only in DM patients (41)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hyperglycemia increases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "HMGB1 is also upregulated by hyperglycemia and appears to play a major role in lung inflammation especially in the context of DM ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hyperglycemia increases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "HMGB1 is also upregulated by hyperglycemia and appears to play a major role in lung inflammation especially in the context of DM ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heparin increases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:28304 ! heparin) increases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Enoxaparin may also elicit similar effects on HMGB1 ( 46 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Heparin increases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:28304 ! heparin) increases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Enoxaparin may also elicit similar effects on HMGB1 ( 46 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Dermatomyositis decreases the amount of sRAGE\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D003882 ! Dermatomyositis) decreases a(TEXT:sRAGE)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "An encouragement to chase this track more in depth comes from the observation that in non-COVID-19 critically ill hyperglycemic patients admitted at ICU, plasma levels of sRAGE are higher in DM vs. non DM patients, and reduced by intensive insulin therapy only in DM patients (41)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Chloroquine decreases the amount of HMGB1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:3638 ! chloroquine) decreases p(HGNC:4983 ! HMGB1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additional FDA approved drugs are known to disrupt RAGE signaling, thus suggesting that a drug repositioning effort could provide a fast track for controlling at least some of the negative outcomes associated with severe ARDS.However, it should be noted that chloroquine, currently used in COVID-19 therapy, inhibits HMGB1 release especially in DM patients (43)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Event activates pyroptosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:event) increases bp(GO:\"GO:0070269\" ! pyroptosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These events may ultimately lead to pyroptosis , which determines the release of further mediators from dying cells and the propagation of the initial damage ( 10 , 22 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Event activates pyroptosis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:event) increases bp(GO:\"GO:0070269\" ! pyroptosis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These events may ultimately lead to pyroptosis , which determines the release of further mediators from dying cells and the propagation of the initial damage ( 10 , 22 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endothelium, Vascular inhibits coagulation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004730 ! \"Endothelium, Vascular\") decreases bp(GO:\"GO:0050817\" ! coagulation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endothelium, Vascular inhibits coagulation\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004730 ! \"Endothelium, Vascular\") decreases bp(GO:\"GO:0050817\" ! coagulation)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endothelium, Vascular inhibits inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004730 ! \"Endothelium, Vascular\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Endothelium, Vascular inhibits inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D004730 ! \"Endothelium, Vascular\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally , thrombomodulin ( TM ) a protein expressed by the vascular endothelium that interacts with various proteins and inhibits coagulation and inflammation , has been shown to bind and block HMGB1 binding to RAGE ( 48 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER binds TLR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(FPLX:TLR ! TLR), p(HGNC:320 ! AGER)) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Both classes of proteins contribute to the inflammatory response by binding to RAGE and members of the Toll like Receptors (TLRs) family.In particular, RAGE activation triggers the NF-kappabeta-mediated transcription of inflammatory genes, together with the activation of the NRP3 inflammasome."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  TLR bound to AGER activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(p(FPLX:TLR ! TLR), p(HGNC:320 ! AGER)) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Both classes of proteins contribute to the inflammatory response by binding to RAGE and members of the Toll like Receptors (TLRs) family.In particular, RAGE activation triggers the NF-kappabeta-mediated transcription of inflammatory genes, together with the activation of the NRP3 inflammasome."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Site-specific N-glycosylation Characterization of Recombinant {SARS}-{CoV}-2 Spike Proteins.](https://pubmed.ncbi.nlm.nih.gov/33077685)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [{HMGB1} as a potential biomarker and therapeutic target for severe {COVID}-19.](https://pubmed.ncbi.nlm.nih.gov/33313438)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Lipopolysaccharide increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In contrast, low-dose lipopolysaccharides (LPS) failed to induce ACE2 mRNA expression in Calu-3 cells, suggesting that the activity of rHMGB1 induced ACE2 expression does not depend on endotoxin contamination."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition to the well-known inflammatory activity of HMGB1, we found that the exogenous human HMGB1 protein (\" rHMGB1 \", purity :> 95%; endotoxin level : < 0.10 EU per 1 mug) time-dependently induced the upregulation of ACE2 mRNA in Calu-3 cells, a human lung epithelial cell line."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Figure2 : Recombinant human HMGB1 induces ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Exogenous HMGB1 induces the expression of SARS-CoV-2 entry receptor ACE2 in alveolar epithelial cells in an AGER dependent manner."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These findings indicate that AGER is required for HMGB1-induced ACE2 expression in lung epithelial cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Figure3: AGER is required for HMGB1-induced ACE2 expression in lung epithelial cells. (A) ELISA analysis of TNF release in indicated human monocytes (THP1) following treatment with human HMGB1 protein (200 ng/ml) for 24 h (n = 3, t test). (B) Western blot analysis of protein expression in human lung epithelial cells (Calu-3) after siRNA-mediated depletion of TLR4 or AGER. (C) Q-PCR analysis of ACE2 mRNA in indicated Calu-3 cells following treatment with human HMGB1 protein (200 ng/ml) for 24 h (n = 3, t test)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Recombinant human HMGB1 induces ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:4983 ! HMGB1) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydroxychloroquine increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:5801 ! hydroxychloroquine) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydroxychloroquine increases the amount of HMGB1-AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:5801 ! hydroxychloroquine) increases a(TEXT:HMGB1-AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1-AGER increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:HMGB1-AGER) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RHMGB1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:rHMGB1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Similarly, rHMGB1 (but not low-dose LPS) also induced the upregulation of ACE2 mRNA in HepG2 (a human liver cell line), Caco2 (a human intestine cell line), and RT4 (a human urinary bladder cell line) cells."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RHMGB1 decreases the amount of TNF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:rHMGB1) decreases p(HGNC:11892 ! TNF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "We consistently found that rHMGB1 induced TNF release was blocked in TLR4-/- THP1 cell (a human monocytic cell line)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RHMGB1 decreases the amount of TNF\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:rHMGB1) decreases p(HGNC:11892 ! TNF)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "We consistently found that rHMGB1 induced TNF release was blocked in TLR4 A/A THP1 cell (a human monocytic cell line)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cardiotoxicity increases the amount of hydroxychloroquine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D066126 ! Cardiotoxicity) increases a(CHEBI:5801 ! hydroxychloroquine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However , some studies have shown that hydroxychloroquine is ineffective in some patients , partly due to its cardiotoxicity [ 31 ] , although more clinical trials are ongoing ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Cardiotoxicity increases the amount of hydroxychloroquine\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D066126 ! Cardiotoxicity) increases a(CHEBI:5801 ! hydroxychloroquine)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "However , some studies have shown that hydroxychloroquine is ineffective in some patients , partly due to its cardiotoxicity [ 31 ] , although more clinical trials are ongoing ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:320 ! AGER) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER activates COVID-19\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:320 ! AGER) increases bp(MESH:D000086382 ! COVID-19)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly , our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2 ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Alarmins inhibits Immune System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000067531 ! Alarmins) decreases bp(MESH:D007107 ! \"Immune System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Damage-associated molecular pattern molecules ( DAMPs ) are endogenous ' alarmin ' molecules that are released from dead or damaged cells , activating the immune system by interacting with several pattern recognition receptors , including toll-like receptors ( TLR2 , TLR4 , and TLR9 ) and the advanced glycosylation end-product specific receptor ( AGER , also known as RAGE ) [ 8, 9 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Alarmins inhibits Immune System\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000067531 ! Alarmins) decreases bp(MESH:D007107 ! \"Immune System\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Damage-associated molecular pattern molecules ( DAMPs ) are endogenous ' alarmin ' molecules that are released from dead or damaged cells , activating the immune system by interacting with several pattern recognition receptors , including toll-like receptors ( TLR2 , TLR4 , and TLR9 ) and the advanced glycosylation end-product specific receptor ( AGER , also known as RAGE ) [ 8, 9 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FPS-ZM1 increases the amount of HMGB1-AGER\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:C572629 ! FPS-ZM1) increases a(TEXT:HMGB1-AGER)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FPS-ZM1 increases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:C572629 ! FPS-ZM1) increases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, genetic (using AGER siRNA) or pharmacological (using glycyrrhizin, chloroquine, hydroxychloroquine, and FPS-ZM1) inhibition of the HMGB1-AGER pathway blocks ACE2 expression."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  AGER inhibitors decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:\"AGER inhibitors\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  HMGB1 inhibitors decreases the amount of ACE2\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:\"HMGB1 inhibitors\") decreases p(HGNC:13557 ! ACE2)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Importantly, our study highlights that HMGB1 inhibitors or AGER inhibitors can prevent SARS-CoV-2 infection by limiting the upregulation of ACE2."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Thermodynamics of monosaccharide binding to concanavalin A, pea (Pisum sativum) lectin, and lentil (Lens culinaris) lectin.](https://pubmed.ncbi.nlm.nih.gov/8463297)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin inhibition of glycation processes.](https://pubmed.ncbi.nlm.nih.gov/14502106)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advanced Glycation End Products Enhance Macrophages Polarization into M1 Phenotype through Activating {RAGE}/{NF}-{\\kappaB} Pathway.](https://pubmed.ncbi.nlm.nih.gov/26114112)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Role of surfactant protein A and D ({SP}-A and {SP}-D) in human antiviral host defense.](https://pubmed.ncbi.nlm.nih.gov/20036966)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SFTPD activates inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition to direct antiviral activities, the SP-A and SP-D modulate innate and adaptive immunity and inflammation associated with infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FASLG activates adaptive immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:11936 ! FASLG) increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition to direct antiviral activities, the SP-A and SP-D modulate innate and adaptive immunity and inflammation associated with infection."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Possible role of amino acids, peptides, and sugar transporter in protein removal and innate lung defense.](https://pubmed.ncbi.nlm.nih.gov/14612156)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Advanced glycation end products, oxidant stress and vascular lesions.](https://pubmed.ncbi.nlm.nih.gov/9061302)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Surfactant protein A and surfactant protein D variation in pulmonary disease.](https://pubmed.ncbi.nlm.nih.gov/17544823)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Relationship between hyperglycemia and infection in critically ill patients.](https://pubmed.ncbi.nlm.nih.gov/16006275)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Quantifying the effects of diuretics and \\beta-adrenoceptor blockers on glycaemic control in diabetes mellitus - a systematic review and meta-analysis.](https://pubmed.ncbi.nlm.nih.gov/25377481)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Metformin: an old drug for the treatment of diabetes but a new drug for the protection of the endothelium.](https://pubmed.ncbi.nlm.nih.gov/26021280)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Metformin decreases the amount of IR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) decreases a(TEXT:IR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin decreases the amount of IR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) decreases a(TEXT:IR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Metformin decreases the amount of IR\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:6801 ! metformin) decreases a(TEXT:IR)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Metformin has been shown to reduce hepatic IR by correcting the activity of two key enzymes, glucose-6-phosphatase (G6pc) and glucokinase [XREF_BIBR]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Hyperglycaemia and Pseudomonas aeruginosa acidify cystic fibrosis airway surface liquid by elevating epithelial monocarboxylate transporter 2 dependent lactate-H+ secretion.](https://pubmed.ncbi.nlm.nih.gov/27897253)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Addicted to sugar: roles of glycans in the order Mononegavirales.](https://pubmed.ncbi.nlm.nih.gov/29878112)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Hydrogen-Ion Concentration activates membrane fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006863 ! \"Hydrogen-Ion Concentration\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These viruses all have a bullet-shaped morphology and enter cells through receptor-mediated endocytosis followed by membrane fusion induced by low pH."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RNF130 activates evasion of host immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:18280 ! RNF130) increases bp(GO:\"GO:0042783\" ! \"evasion of host immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Viral GPs then contribute to multiple viral functions, including viral entry and immune evasion (Merry and Astrautsova 2010) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GP activates membrane fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:H2AM12 ! H2AM12) increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, paramyxoviral GP-induced membrane fusion does not require a drop in pH, as many other viral families do."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  GP activates membrane fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(UP:H2AM12 ! H2AM12) increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Binding of the attachment GP (HN, H or G) to a cell surface receptor triggers the fusion GP (F) to insert itself into the host cell membrane, facilitating viral-cell membrane fusion (Iorio et al. 2009; Chang and Dutch 2012) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Deletion decreases the amount of SLC2A1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:deletion) decreases p(HGNC:11005 ! SLC2A1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Deletion of all three N-glycans completely blocked CSE ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Tunicamycin inhibits viral release from host cell\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29699 ! tunicamycin) decreases bp(GO:\"GO:0019076\" ! \"viral release from host cell\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Tunicamycin treatment inhibits viral release, likely by trapping viral GPs in the cytoplasm of the infected cell."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Tunicamycin inhibits viral RNA genome replication\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29699 ! tunicamycin) decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, tunicamycin treatment inhibited SeV viral replication (Nakamura et al. 1982) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Tunicamycin decreases the amount of RNF130\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:29699 ! tunicamycin) decreases p(HGNC:18280 ! RNF130)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Tunicamycin treatment inhibits viral release, likely by trapping viral GPs in the cytoplasm of the infected cell."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mortality increases the amount of pathogen\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D009026 ! Mortality) increases a(TEXT:pathogen)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "These viruses are classified as bio-safety level 4 pathogens due to their high mortality rates and scarcity of approved treatments or cures ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  DEGS1 increases the amount of GP\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13709 ! DEGS1) increases p(UP:H2AM12 ! H2AM12)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In addition, there are N-glycans within the MLD that may partially contribute to the phenotypes observed.The GP 2 subunit contains two N-glycosylation sites conserved among all five EBOV species (Jeffers et al. 2002; Ritchie et al. 2010 )."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Respiratory Syncytial Virus, Human activates Bronchiolitis\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D018113 ! \"Respiratory Syncytial Virus, Human\") increases bp(MESH:D001988 ! Bronchiolitis)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Human respiratory syncytial virus ( HRSV , also referred to as RSV ) causes severe bronchiolitis in infants , accounting for over 50 % of childhood bronchiolitis cases requiring hospitalization ( Rodriguez and Ramilo 2014 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  LGALS1 binds N-glycan\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:59520 ! N-glycan), p(HGNC:6561 ! LGALS1)) decreases bp(GO:\"GO:0140253\" ! \"cell-cell fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N-glycan bound to LGALS1 inhibits cell-cell fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:59520 ! N-glycan), p(HGNC:6561 ! LGALS1)) decreases bp(GO:\"GO:0140253\" ! \"cell-cell fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  P84849 binds N-glycan\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:59520 ! N-glycan), p(UP:P84849 ! P84849)) decreases bp(GO:\"GO:0140253\" ! \"cell-cell fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  N-glycan bound to P84849 inhibits cell-cell fusion\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "complex(a(CHEBI:59520 ! N-glycan), p(UP:P84849 ! P84849)) decreases bp(GO:\"GO:0140253\" ! \"cell-cell fusion\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Additionally, Galectin-1, a lectin that binds primarily a specific N-glycan in NiV F, inhibited NiV cell-cell fusion if added post-infection but enhanced viral entry if present at the time of infection by increasing viral attachment to target cellsMany glycan functions are conserved among viral genera."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Blood glucose control in the {ICU}: how tight?](https://pubmed.ncbi.nlm.nih.gov/28275621)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Immunomodulatory roles of surfactant proteins A and D: implications in lung disease.](https://pubmed.ncbi.nlm.nih.gov/17607008)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Surfactant Proteins A and D: Trimerized Innate Immunity Proteins with an Affinity for Viral Fusion Proteins.](https://pubmed.ncbi.nlm.nih.gov/30293076)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### SFTPD increases the amount of gram\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases p(PFAM:PF02893 ! gram)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPD increases the amount of calcium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10803 ! SFTPD) increases a(CHEBI:29108 ! \"calcium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPA2 increases the amount of gram\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10799 ! SFTPA2) increases p(PFAM:PF02893 ! gram)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SFTPA2 increases the amount of calcium(2+)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:10799 ! SFTPA2) increases a(CHEBI:29108 ! \"calcium(2+)\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "With the use of flow cytometric and fluo rescent-microscopic assays, SP-A and SP-D have been shown to increase the calcium-dependent neutrophil uptake of both gram-negative and gram-positive bacteria such as Escherichia coli, Streptococcus pneumoniae, and Staphylococcus aureus [37]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  FCN2 inhibits Infections\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:3624 ! FCN2) decreases bp(MESH:D007239 ! Infections)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "L-ficolin has been reported to bind both HA and neuraminidase glycoproteins and inhibit IAV infection in vitro and in vivo [111]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Weathering the Cytokine Storm in {COVID}-19: Therapeutic Implications.](https://pubmed.ncbi.nlm.nih.gov/32599589)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### IL6 decreases the amount of TWIST1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6018 ! IL6) decreases p(HGNC:12428 ! TWIST1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Studies have shown that tocilizumab, an IL-6 receptor antibody, could reverse CRS in the setting of chimeric antigen receptor T-cell therapy for cancer [36]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  ACE2 activates entry into host\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2) increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The viral spike protein of SARS-Cov2 seems to induce a TNF--converting enzyme (TACE)-dependent alteration of ACE-2, which allows virus penetration into host cells, and the more TNF- levels are increased the more this pathway is facilitated [43]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  SARS-CoV-2 increases the amount of ADAM17\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000086402 ! SARS-CoV-2) increases p(HGNC:195 ! ADAM17)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The viral spike protein of SARS-Cov2 seems to induce a TNF--converting enzyme (TACE)-dependent alteration of ACE-2, which allows virus penetration into host cells, and the more TNF- levels are increased the more this pathway is facilitated [43]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Picloram activates Mortality\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:34922 ! picloram) increases bp(MESH:D009026 ! Mortality)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "A phase 2 trial is currently evaluating leronlimab , a humanized monoclonal antibody inhibiting CCR5 which is already under study in HIV infection [ 46 ] , to determine whether blockage of this pathway is able to reduce mortality at 28 days in severe forms of COVID-19 ( NCT04347239 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CCR5 activates immune response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1606 ! CCR5) increases bp(GO:\"GO:0006955\" ! \"immune response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "At present, there are 2 clinical trials (ChiCTR2000030170 and ChiCTR2000029580) the results of which will hopefully provide further insights into JAK and AAK1 inhibition effects in COVID-19 patients.C-C chemokine receptor type 5 (CCR5) is expressed on the surface of white blood cells, especially T-CD4+ cells, and mediates macrophage migration into areas of inflammation, favoring the release of inflammatory cytokines and amplification of the immune response [46]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CCR5 activates macrophage migration\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1606 ! CCR5) increases bp(GO:\"GO:1905517\" ! \"macrophage migration\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  CCR5 activates macrophage migration\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:1606 ! CCR5) increases bp(GO:\"GO:1905517\" ! \"macrophage migration\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "C-C Chemokine Receptor Type 5 Inhibition C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Pyroptosis activates cell death\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(GO:\"GO:0070269\" ! pyroptosis) increases bp(GO:\"GO:0008219\" ! \"cell death\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Cellular infection and viral replication cause activation of the inflammasome in the host cell, leading to the release of proinflammatory cytokines and cell death by pyroptosis with ensuing release of a damage-associated molecular pattern, further amplifying the inflammatory response [7, 8, 9]."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Mesenchymal Stem Cells inhibits inflammatory response\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D059630 ! \"Mesenchymal Stem Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Several in vivo studies in animal models and ex vivo human lung models have proven that MSC can prevent lung injury , reduce inflammation , regulate immune responses , and foster alveolar fluid clearance [ 47 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Receptors, CCR activates macrophage migration\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D054388 ! \"Receptors, CCR\") increases bp(GO:\"GO:1905517\" ! \"macrophage migration\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "C-C chemokine receptor type 5 ( CCR5 ) is expressed on the surface of white blood cells , especially T-CD4 + cells , and mediates macrophage migration into areas of inflammation , favoring the release of inflammatory cytokines and amplification of the immune response [ 46 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Tocilizumab decreases the amount of TWIST1\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:64360 ! tocilizumab) decreases p(HGNC:12428 ! TWIST1)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Studies have shown that tocilizumab , an IL-6 receptor antibody , could reverse CRS in the setting of chimeric antigen receptor T-cell therapy for cancer [ 36 ] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  RPGRIP1L decreases the amount of Chemokine_receptor\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:29168 ! RPGRIP1L) decreases p(FPLX:\"Chemokine_receptor\" ! \"Chemokine_receptor\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "C-C Chemokine Receptor Type 5 Inhibition."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Single virus targeting multiple organs: what we know and where we are heading?](https://pubmed.ncbi.nlm.nih.gov/32850890)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Ribavirin activates Sick Sinus Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:63580 ! ribavirin) increases bp(MESH:D012804 ! \"Sick Sinus Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ribavirin activates Sick Sinus Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:63580 ! ribavirin) increases bp(MESH:D012804 ! \"Sick Sinus Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Ribavirin activates Sick Sinus Syndrome\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:63580 ! ribavirin) increases bp(MESH:D012804 ! \"Sick Sinus Syndrome\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Ribavirin , which is being used for treatment ( 36 ) , is reported to induce sick sinus syndrome ( 37 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Physiology inhibits Lung\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D010827 ! Physiology) decreases bp(MESH:D008168 ! Lung)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Physiology inhibits Lung\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D010827 ! Physiology) decreases bp(MESH:D008168 ! Lung)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Physiology inhibits Lung\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D010827 ! Physiology) decreases bp(MESH:D008168 ! Lung)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Virus entry into these cells affects their normal physiology and associated processes , which may lead to lung collapse ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Antiviral drug activates Heart Failure\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:36044 ! \"antiviral drug\") increases bp(MESH:D006333 ! \"Heart Failure\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Antiviral drug activates Heart Failure\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:36044 ! \"antiviral drug\") increases bp(MESH:D006333 ! \"Heart Failure\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Antiviral drug activates Heart Failure\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:36044 ! \"antiviral drug\") increases bp(MESH:D006333 ! \"Heart Failure\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Antiviral drugs have a very high risk potential for the heart and may lead to cardiac insufficiency , toxicity , and arrhythmia ( 35 ) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [The phenotypic changes of \\gamma\\delta T cells in {COVID}-19 patients.](https://pubmed.ncbi.nlm.nih.gov/32864865)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Identifying and quantifying robust risk factors for mortality in critically ill patients with {COVID}-19 using quantile regression.](https://pubmed.ncbi.nlm.nih.gov/33046291)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Managing hyperglycemia during the {COVID}-19 pandemic: Improving outcomes using new technologies in intensive care.](https://pubmed.ncbi.nlm.nih.gov/33282306)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### INS decreases the amount of glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:6081 ! INS) decreases a(CHEBI:17234 ! glucose)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Optimal glucose control in intensive care is limited by manual sampling of glucose and intravenous insulin adjustment, as well as increased nursing workload and the need of protective equipment."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hyperglycemia activates Virulence\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D006943 ! Hyperglycemia) increases bp(MESH:D014774 ! Virulence)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In vivo and in vitro studies reveal the mechanisms by which hyperglycemia increases virulence and how glucose control and insulin reduce it ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydroxychloroquine inhibits Blood Glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:5801 ! hydroxychloroquine) decreases bp(MESH:D001786 ! \"Blood Glucose\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In vitro data suggest that hydroxychloroquine can reduce glycosylation of the ACE2 receptor and also can lower blood glucose concentrations , both which could be possible mechanisms of antiviral activity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hydroxychloroquine inhibits Blood Glucose\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(CHEBI:5801 ! hydroxychloroquine) decreases bp(MESH:D001786 ! \"Blood Glucose\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "In vitro data suggest that hydroxychloroquine can reduce glycosylation of the ACE2 receptor and also can lower blood glucose concentrations , both which could be possible mechanisms of antiviral activity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Hypoglycemia activates Mortality\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D007003 ! Hypoglycemia) increases bp(MESH:D009026 ! Mortality)\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "27 , 28 Hypoglycemia also increases mortality 29 and may go undetected with infrequent glucose determinations ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Workforce inhibits Intensive Care Units\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000078329 ! Workforce) decreases bp(MESH:D007362 ! \"Intensive Care Units\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The pandemic of COVID-19 has caused significant disruption in the intensive care unit ( ICU ) due to inadequate staffing , protective equipment , ventilators , and bed capacity.1 Demographics reveal that patients at greatest mortality risk are those with diabetes and / or hyperglycemia ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "###  Workforce inhibits Intensive Care Units\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bp(MESH:D000078329 ! Workforce) decreases bp(MESH:D007362 ! \"Intensive Care Units\")\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "The pandemic of COVID-19 has caused significant disruption in the intensive care unit ( ICU ) due to inadequate staffing , protective equipment , ventilators , and bed capacity ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Lung surfactant protein D ({SP}-D) response and regulation during acute and chronic lung injury.](https://pubmed.ncbi.nlm.nih.gov/23435873)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Impaired fasting glucose, blood pressure and cardiovascular disease mortality.](https://pubmed.ncbi.nlm.nih.gov/12364347)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "## [Age as independent determinant of glucose tolerance.](https://pubmed.ncbi.nlm.nih.gov/2015973)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pmid_graph, pmid_indra,english_statements = print_bib('../data/blood-glucose.bib',\n",
    "                                                      emmaa_covid_graph,\n",
    "                                                      pmid_graph,\n",
    "                                                     pmid_indra,\n",
    "                                                     english_statements)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "anaconda3-2020.11",
   "language": "python",
   "name": "anaconda3-2020.11"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": false,
   "autoclose": false,
   "autocomplete": false,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "ctrl-q"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": true,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {
    "height": "calc(100% - 180px)",
    "left": "10px",
    "top": "150px",
    "width": "290.797px"
   },
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
