{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "toc": true
   },
   "source": [
    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
    "<div class=\"toc\"><ul class=\"toc-item\"><li><span><a href=\"#Import-Latest-COVID-19-model-from-EMMAA\" data-toc-modified-id=\"Import-Latest-COVID-19-model-from-EMMAA-1\"><span class=\"toc-item-num\">1&nbsp;&nbsp;</span>Import Latest COVID-19 model from EMMAA</a></span></li><li><span><a href=\"#Find-relations-between-ACE2-and-Interferon\" data-toc-modified-id=\"Find-relations-between-ACE2-and-Interferon-2\"><span class=\"toc-item-num\">2&nbsp;&nbsp;</span>Find relations between ACE2 and Interferon</a></span><ul class=\"toc-item\"><li><span><a href=\"#Find-all-INDRA-statements-associated-with-a-publication\" data-toc-modified-id=\"Find-all-INDRA-statements-associated-with-a-publication-2.1\"><span class=\"toc-item-num\">2.1&nbsp;&nbsp;</span>Find all INDRA statements associated with a publication</a></span></li></ul></li></ul></div>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip indra install pandas pybel requests pybel_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:18:06] pybel.config - no configuration found, using default sqlite connection sqlite:////Users/zuck016/.data/pybel/pybel_0.14.0_cache.db\n"
     ]
    }
   ],
   "source": [
    "from collections import Counter\n",
    "import indra\n",
    "import pandas as pd\n",
    "import pybel\n",
    "import pybel.struct\n",
    "import requests\n",
    "import pybel_tools.mutation\n",
    "from pybel.io.jupyter import to_jupyter"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Import Latest COVID-19 model from EMMAA\n",
    "\n",
    "Use the local nodelink bel graph if it exists. Use downloaded indra statements if they exist\n",
    "or download latest indra statements from EMMAA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\")-derived O-acetylneuraminic acids\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:12962 ! \"D-glucosamine 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6030 ! isoniazide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47867 ! indicator), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'His', 'pos': 4}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16914 ! \"salicylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Ser', 'pos': 1850}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Asp', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:40356 ! N-acetyl-alpha-D-galactosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53457 ! glucuronoxylomannan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15429 ! hydroxylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4325 ! dapsone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6495 ! lipoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81563 ! \"18(R)-HEPE\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27569 ! arabinogalactan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0000785\" ! chromatin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0031012\" ! \"extracellular matrix\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17148 ! putrescine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:EC), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0004705\" ! \"JUN kinase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:32] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33232 ! application), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:79214 ! \"alpha-N-acetylneuraminyl-(2->8)-alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-(1->4)-beta-D-glucosyl-(1<->1)-ceramide(2-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:21500 ! \"N-acetyl-9-O-acetylneuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28790 ! serotonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CNTN2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051557 ! \"Hepatocyte Nuclear Factor 4\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KAT2A(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}, 'code': 'Lys', 'pos': 90}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81559 ! \"resolvin E1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81563 ! \"18(R)-HEPE\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HLA-DMB(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDPK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18367 ! \"phosphate(3-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15746 ! spermine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16610 ! spermidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78037 ! polyhydroxyalkanoate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010799 ! Phycoerythrin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\")-derived O-acetylneuraminic acids\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6030 ! isoniazide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16337 ! \"phosphatidic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}, {'kind': 'hgvs', 'hgvs': 'p.?'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32849 ! TEMPO), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:27:33] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NCp7), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:34] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation), activity)\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys18Gly'}]\n",
      "INFO: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22478 ! \"amino alcohol\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:35] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydrogencarbonate(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22260 ! adenosines), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:27:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15841 ! polypeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:27:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23924 ! \"enzyme inhibitor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:27:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019895 ! Transposases), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30314 ! \"(R)-lipoic acid\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent salicylate(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019895 ! Transposases), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent steroid(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35381 ! monosaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:27:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pyrimidines(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:41] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46793 ! \"poly(ethylene glycol)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:27:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47885 ! dinucleotide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:27:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:27:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent methylglyoxal-lysine dimer(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sphingomyelin(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:27:44] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78505 ! venom), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:27:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNAR(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP2C(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:47] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro323Leu'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Arg411Ala'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro88Cys'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro88Cys'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:27:48] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro56Ser'}]\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM38(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM38(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOD3(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STK3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3783 ! \"Coagulation Factor Xa\"), [{'kind': 'hgvs', 'hgvs': 'p.Phe10Ala'}]\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFBR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TJP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kACE2), [{'kind': 'hgvs', 'hgvs': 'p.Thr31Lys'}]\n",
      "WARNING: [2021-03-24 09:27:52] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mACE2 187\"), [{'kind': 'hgvs', 'hgvs': 'p.His353Lys'}]\n",
      "INFO: [2021-03-24 09:27:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP6(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:54] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:TS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:27:54] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX58(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDHX(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP9(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:56] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR4(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:56] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX3X(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFASC(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "INFO: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G3BP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:57] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FTH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:27:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:27:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDH1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:27:59] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.Pro56Ser'}]\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Thr2Asp'}]\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (phosphorylation), (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AQP4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MEF2A(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28515 ! acetoacetamide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:02] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:03] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:04] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ala2Ala'}]\n",
      "WARNING: [2021-03-24 09:28:04] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ala2Ala'}]\n",
      "WARNING: [2021-03-24 09:28:06] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002415 ! Cats), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:08] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:08] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:08] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Pro'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:09] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008459 ! \"Measles virus\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:10] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:10] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:11] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:11] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:11] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Relaxin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Severe Acute Respiratory Syndrome(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys61Ala'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys61Ala'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tax(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL83(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:16] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Igg-2a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:17] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pel2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0016021\" ! \"integral component of membrane\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15843 ! \"arachidonic acid\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'hgvs', 'hgvs': 'p.Asn720Asp'}]\n",
      "INFO: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:HKU4), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Kit), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Lys237Glu'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Leu336Met'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Asn350Lys'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Pro653Leu'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Thr97Ile'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'hgvs', 'hgvs': 'p.Thr97Ile'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PSEN2), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RAS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent helicase(mods: (modification), muts: (None, None, None))\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:vBcl2), [{'kind': 'hgvs', 'hgvs': 'p.Ala179Leu'}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Receptors, Growth Factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase 1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27698 ! \"vanadium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PC-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:18] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015032 ! Zinc), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:28:19] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:19] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:49018 ! \"neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:20] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:20] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:21] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:21] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:21] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:21] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:LYNVEA), [{'kind': 'hgvs', 'hgvs': 'p.Cys3Asp'}]\n",
      "INFO: [2021-03-24 09:28:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Ser1Glu'}]\n",
      "WARNING: [2021-03-24 09:28:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:28:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys112Ser'}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NAD(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60027 ! polymer), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ETC complex I\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16087 ! \"isocitrate(3-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73601 ! Lys-Asp), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C014949 ! acetylhydrazine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60027 ! polymer), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17029 ! chitin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85044 ! H3K27Ac), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76279 ! \"2''-O-acetyl-ADP-D-ribose\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18257 ! ornithine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDPK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52450 ! 2-O-acetyl-1-O-octadecyl-sn-glycero-3-phosphocholine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26667 ! \"sialic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17012 ! \"N-acetylneuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent prostaglandin I2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:43] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent urea(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent progesterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent testosterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydrogencarbonate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent unsaturated fatty acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:44] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent zinc atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent (R)-adrenaline(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent azithromycin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent citric acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent butyric acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:45] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent low-density lipoprotein cholesterol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:46] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent antagonist(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipoprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ile-Leu(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 24,25-Dihydroxyvitamin D(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AMPK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BLVR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent COX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent COX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FSH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemoglobin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL12(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3K_p85(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent plasminogen activation(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNASEL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNF5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RUNX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL24(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPA2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SGTA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC2A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLPI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SORT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRP19(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STUB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX21(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX21(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFPI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBS1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TJP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TM7SF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF1B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TXN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UPP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UPP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:47] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XIST(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XIST(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZBTB17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BANP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CA8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC13A3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CAMK4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CAV1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NBAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERMAP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD209(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OTUD7B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL32(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MFN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNBIP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNBIP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POSTN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD74(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD79A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VSIG4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DICER1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIAS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDKN1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEBPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1LG2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDEM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIDEA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HACE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LINGO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OTUB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCHF8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCHF8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMBIM4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YBX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAMD4B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CELF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM62(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP19A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERFE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12C(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZC3H12C(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TSPYL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNMT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent C15orf48(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RCAN3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC6A20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIR23B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DEFA1B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGTR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKR1B1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FST(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FTH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ALB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GBF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GIP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCR3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GTPBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HCRT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HPSE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI27(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI30(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL13(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL18(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1R1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IDO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ITGAM(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KDELR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLRC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SMAD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MCL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MDM4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MME(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP13(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ABCG2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MVP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NCL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NEDD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPTX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OXT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD6IP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLAT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PML(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP1R10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTPN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB7A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2L1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent itolizumab(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent itolizumab(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SARS-CoV-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:48] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tissue Plasminogen Activator(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Zinc(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Bronchoalveolar Lavage(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Bronchoalveolar Lavage(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Severe Acute Respiratory Syndrome(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 7a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lexA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YFH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL21A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL21A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Non-structural protein 6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ly6c2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent avrA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NEC2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 4a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 4a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Lnpep(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lmp-1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BPLF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ompC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascl1a(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Usp18(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent fni(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Parp-1 inhibitor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent R-BHB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZAP-L(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent miR-Let-7A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent miR-Let-7A(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:65858 ! \"ergosterol peroxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Lys', 'pos': 562}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0007602\" ! phototransduction), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV cellular receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48545 ! pentapeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:70744 ! Gly-Pro), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7775 ! ononin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Pro'}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35807 ! N-nitrosodimethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15366 ! \"acetic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15366 ! \"acetic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18154 ! polysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16521 ! lanosterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18723 ! nicotine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16991 ! \"deoxyribonucleic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'pos': 27}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50131 ! \"5-aza-2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6121 ! ketamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17230 ! homocysteine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50131 ! \"5-aza-2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Xlr3c(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28812 ! \"(aminomethyl)phosphonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Xlr3c(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT1-S727), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36916 ! cation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17188 ! \"nucleoside 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 328}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36916 ! cation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46345 ! 5-fluorouracil), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent amino acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent amino acid(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:49] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"pBluescript(R) II KS+\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Core protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cyclin(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16474 ! NADPH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"BrSARS-N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BECN1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47885 ! dinucleotide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM21(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27226 ! \"uric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NKX2-1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP1R15A(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17013 ! dTMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM11(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDKN2A(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 5}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMBIM4(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPIPB3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP6(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73584 ! Leu-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28260 ! galactose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15373 ! biopterin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Glu', 'pos': 4}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INSR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25351 ! \"mevalonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKBIA(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16856 ! glutathione), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35895 ! tetraphosphorus), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60169 ! \"phosphatidylinositol trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT138), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74564 ! Lys-Lys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C068798 ! 4-aminophenylphosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17245 ! \"carbon monoxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25523 ! NADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ganglioneuroma(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37788 ! \"O(4)-phospho-L-tyrosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ppp1r15b(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S7(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ORF1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18421 ! superoxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent macro domain(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_alpha(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Adenosine Diphosphate(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Interferon(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent ELK1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RLR(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPAR(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HNRNPK(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Protease(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent METTL3(mods: (sumoylation, K))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent E7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation), muts: (None, None, None))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CEBP(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HDAC1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TBK1(mods: (sumoylation))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2N(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28545 ! valinomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Ala', 'pos': 5}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP28(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18361 ! \"diphosphate(4-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK12(mods: (modification), muts: (G, 731, E))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:8753 ! RI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Putative movement protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BPLF1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:S1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"HIV20\f",
      "medRxiv preprint doi\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PLWH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48343 ! disulfide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36233 ! disaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0007602\" ! phototransduction), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52479 ! imidate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4208 ! D-mannopyranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Met', 'pos': 9}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017472 ! \"Receptors, Interleukin-1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:506227 ! N-acetyl-D-glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:DeltaHbA1c), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "INFO: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST6GAL1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:50] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"GS/BS\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0038023\" ! \"signaling receptor activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CM), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAGT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TUSC3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016766 ! \"Feline Infectious Peritonitis\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:62084 ! \"N-glycoloyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Asn', 'pos': 386}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:tip), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Trp', 'pos': 100}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030246\" ! \"carbohydrate binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:51142 ! \"amino groups\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019532 ! Map), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17347 ! testosterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009077 ! Mucins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019426 ! Integrases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:62875 ! \"2-methylthio-N(6)-(Delta(2)-isopentenyl)adenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22656 ! \"asparaginyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28940 ! calciol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9367 ! swainsonine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'pos': 29}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D034441 ! \"Heterogeneous-Nuclear Ribonucleoproteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17189 ! \"2,5-dihydroxybenzoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23990 ! \"ethyl ester\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25248 ! \"methyl ester\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30778 ! \"gallic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30314 ! \"(R)-lipoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 5}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6909 ! metronidazole), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37659 ! \"oleanolic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60027 ! polymer), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35341 ! steroid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:77477 ! nitrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16755 ! \"chenodeoxycholic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 6}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16796 ! melatonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 1}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16796 ! melatonin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 4}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 5}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72544 ! flavonoids), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Cys', 'pos': 6}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16581 ! pregnenolone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 17}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17026 ! progesterone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Ala', 'pos': 17}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28940 ! calciol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16113 ! cholesterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Asp', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27300 ! \"vitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}, 'code': 'Asp', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:86319 ! \"25-hydroxyvitamin D2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:77260 ! \"5-methoxy-2-\\{[(4-methoxy-3,5-dimethylpyridin-2-yl)methyl]sulfinyl\\}-1H-benzimidazole\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50038 ! deoxyhypusine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50038 ! deoxyhypusine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26274 ! \"prolyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28683 ! kynurenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019426 ! Integrases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30762 ! salicylate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:76238 ! protopanaxadiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:84403 ! phytoceramides), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16830 ! methylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32875 ! \"methyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63465 ! marneral), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Gly-Phe zwitterion(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetaldehyde(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetylcholine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15551 ! \"prostaglandin E2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent leukotriene B4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cholesterol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent homogentisate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:51] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hydroxide(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic GMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16412 ! lipopolysaccharide), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NADPH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatinine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glutathione(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent creatine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent bilirubin IXalpha(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent progesterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent putrescine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent estrone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent testosterone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cortisol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent triglyceride(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent dinitrogen(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent acetylcholine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent iron atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent galactosylceramide sulfate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L-thyroxine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:52] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ascorbic acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hormone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent hormone(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lactate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent leukotriene(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27300 ! \"vitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:53] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent selenium atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent serotonin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phosphatidylinositol(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calciol(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sodium(1+)(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium(1+)(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MPO(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Prtn3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent formic acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent bradykinin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin E(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent seaborgium atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent resiquimod(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent fumonisin B1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:54] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticotropin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD274(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent (S)-lactic acid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent geneticin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:55] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent geneticin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent angiotensin II(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent doxycycline(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent corticosteroid(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL5(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phenyl isocyanate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent phenyl isocyanate(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:56] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent malonaldehyde(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent double-stranded RNA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Val-Ala(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N-benzyloxycarbonyl-L-leucyl-L-leucyl-L-leucinal(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N-benzyloxycarbonyl-L-leucyl-L-leucyl-L-leucinal(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Core protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent polyamine(mods: (modification))\n",
      "WARNING: [2021-03-24 09:28:57] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Activin_A(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD64(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD8(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Creatine_kinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Creatine_kinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EDN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FSH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GNRH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GOT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GOT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GST(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G_protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HLA_DR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemoglobin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL12(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL2R(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP1LC3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAPK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFkappaB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Na_K_ATPase(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PI3K(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPAR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPAR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TCR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VEGF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP1B1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent epithelial to mesenchymal transition(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent epithelial to mesenchymal transition(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNA-directed 5'-3' RNA polymerase activity(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent endoplasmic reticulum(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent integral component of membrane(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNASEL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BECN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RUNX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL12(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SF3B4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFRP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRSF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SFTPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BNIP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST6GAL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ST3GAL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC2A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC5A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC5A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BRD2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC7A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC9A1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOX2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRPK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SRPK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BTN3A3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TDO2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TERT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFEB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TFPI(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFBR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THBS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent THY1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STAT2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TYK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEMIP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TMPRSS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:58] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNF(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFRSF11B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD40(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFSF13B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD40LG(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TOMM70(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TXN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VAV1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VDAC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VTN(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NSD2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent XBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZBTB17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LPIN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent C5AR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CLEC4M(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADIPOQ(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENDOU(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC13A3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CANX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZEB2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT6(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIRT7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CASP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HAMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TLR7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RHOF(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL33(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCR4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD14(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD209(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB14(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAPGEF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD36(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD36(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GORASP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD47(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL32(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3C(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3C(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD69(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD79A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GORASP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TREM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEACAM5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEBPD(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNL3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CFTR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CREB3L4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CREB3L1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NCR3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDX58(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SOCS3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ADAM17(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UCHL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYLIP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GRK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GRK5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDHX(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CGAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CGAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CORO1C(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MORC4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRH(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACKR3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOOL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD200R1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CSK(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTLA4(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CTNNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NIBAN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARHGAP24(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYP27B1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DDIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOO(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RILPL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNF166(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP14(mods: (modification), muts: (None, None, None))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNM1L(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DNMT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFASC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CENPV(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DPYSL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DPYSL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent G3BP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SLC6A20(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRSS57(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TYMP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIR223(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUXA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUXA(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGFR(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF3C(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF3B(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF4E(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ELAVL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AGT(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EOMES(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG7(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRINS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERN1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIPARP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent F3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent F3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FDPS(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGF23(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ZGLP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AKR1B1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FUBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GATA2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GATA3(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GH2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GIP(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GJA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GLI1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP5(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GYPC(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GZMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:28:59] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BNIP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HINT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNF4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF8(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RBPJ(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATG5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL13(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17D(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL17D(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL18(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1B(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL7(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (phosphorylation), (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF7(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOD(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APOD(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG20(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF9(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JUN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JUNB(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KLF5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KPNA2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent KRAS(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AR(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LAMP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent AREG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LEP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LILRB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LRP5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LTA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MASP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MBL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MBP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MDM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARNTL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIITA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CIITA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MIF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATXN3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MARCKSL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MME(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP13(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MMP9(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD200(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ARSF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOD1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFKBIA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOTCH1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATF6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NR3C1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NR3C1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent OAS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP2A2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SIGMAR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SERPINE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent POU5F1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPARGC1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAG5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EIF2AK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRKRA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRSS8(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTBP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGDS(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BAX(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTH(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTX3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAE1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCHE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Chemokine(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent REN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Arginase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Dendritic Cells(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Dendritic Cells(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Fishes(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:00] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ferritin(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Major Histocompatibility Complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Major Histocompatibility Complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EPO(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Relaxin(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Renin-Angiotensin System(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RNA-Dependent RNA Polymerase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Transferases(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOS(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNF4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th1 Cells(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MicroRNAs(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Recombinases(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mice(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent reactive oxygen species(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HESX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nppb(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lexA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent psaA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FOXP3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent armB(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pgsA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent E7(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TK(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tax(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tat(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mhc(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 9b(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 1a(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Non-structural protein 6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Delta-actitoxin-Dar1b(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Igg-2a(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tug1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tug1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent At2g14610(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent flbA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sox11-b(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sox11-b(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Uncharacterized 37.2 kDa protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable integrase/recombinase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CYCD1-1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent yqaT(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gag-pol(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Caspase-12(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Caspase-12(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent gE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent X(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pel2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rmpA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ceacam10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lanA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lanA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Genome polyprotein(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 5b(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent mxa(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ORF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent pro-pol(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent sfl(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXC chemokine(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Cap(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ENaC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ero1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ero1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FLAG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PM2.5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RD(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SH2D3A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th17 development.Itolizumab(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Th17 development.Itolizumab(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Toll-like receptors (TLRs) 2/4 co-receptor(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent YmdB(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium-binding(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent cyclase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nsp10(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n",
      "WARNING: [2021-03-24 09:29:01] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003165 ! \"Complement System Proteins\"), [{'kind': 'hgvs', 'hgvs': 'p.Cys5Ala'}]\n",
      "WARNING: [2021-03-24 09:29:04] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16243 ! quercetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:29:04] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:05] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:29:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glycoprotein(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:05] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:29:05] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent glucose(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:06] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 3',5'-cyclic AMP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:29:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "WARNING: [2021-03-24 09:29:06] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18152 ! myricetin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein glycosylation', 'identifier': '0006486'}}]\n",
      "INFO: [2021-03-24 09:29:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:07] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vitamin D(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:09] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28424 ! \"alpha-Neu5NAc-(2->8)-alpha-Neu5NAc-(2->3)-beta-D-Gal-(1->4)-beta-D-Glc-(1<->1')-Cer\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:29:09] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:10] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007473 ! \"Ion Channels\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:10] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33416 ! fullerene), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:29:10] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33416 ! fullerene), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein hydroxylation', 'identifier': '0018126'}}]\n",
      "WARNING: [2021-03-24 09:29:12] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NFAT-GFP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:13] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:29:13] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:45744 ! \"N-acetyl-beta-neuraminic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:29:14] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012194 ! \"RNA-Directed DNA Polymerase\"), [{'kind': 'hgvs', 'hgvs': 'p.Met184Val'}]\n",
      "WARNING: [2021-03-24 09:29:15] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FIP1L1-PDGFRA), [{'kind': 'hgvs', 'hgvs': 'p.Ser601Pro'}]\n",
      "WARNING: [2021-03-24 09:29:15] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FIP1L1-PDGFRA), [{'kind': 'hgvs', 'hgvs': 'p.Thr674Ile'}]\n",
      "WARNING: [2021-03-24 09:29:15] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"oncogenic b-raf\"), [{'kind': 'hgvs', 'hgvs': 'p.Val600Glu'}]\n",
      "WARNING: [2021-03-24 09:29:15] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:p.), [{'kind': 'hgvs', 'hgvs': 'p.Gly12Val'}]\n",
      "WARNING: [2021-03-24 09:29:16] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:AF), [{'kind': 'hgvs', 'hgvs': 'p.Ser678Pro'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:16] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:16] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29035 ! \"manganese(2+)\"), [{'kind': 'hgvs', 'hgvs': 'p.Ala0Cys'}]\n",
      "WARNING: [2021-03-24 09:29:16] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63153 ! N-acetyl-D-mannosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "WARNING: [2021-03-24 09:29:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:BCR-ABL), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Bcr-Abl), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:18] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:BCR-ABL), [{'kind': 'hgvs', 'hgvs': 'p.Thr315Ile'}]\n",
      "INFO: [2021-03-24 09:29:18] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ile-Leu(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:19] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:78420 ! icariin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:19] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:82128 ! PB1-F2), [{'kind': 'hgvs', 'hgvs': 'p.Asn66Ser'}]\n",
      "WARNING: [2021-03-24 09:29:21] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012208 ! Rhabdomyosarcoma), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Clathrin(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:21] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDX(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Troponin(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:22] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:23] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030246\" ! \"carbohydrate binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein acetylation', 'identifier': '0006473'}}]\n",
      "INFO: [2021-03-24 09:29:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ROCK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCL5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:23] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SELE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TALDO1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.Cys3His'}]\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TP53(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP11(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VIM(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VWF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:24] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HDAC6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent MACROD2(mods: (ribosylation))\n",
      "WARNING: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY12(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:25] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CRP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKFC(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TKFC(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CCN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NIBAN2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CELF2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SHFL(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TNFAIP8L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ERFE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PARP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:26] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DANCR(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GPNMB(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:27] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY13(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ANPEP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPA1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HNRNPF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ID2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFIT1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL10(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL16(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:28] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent INS(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ITGAM(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent LILRB1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYOM2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NFE2L2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NOS3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent NPC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:29] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent P2RY2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PDCD4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PLP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PRDM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTEN(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RAB7A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BCL2L1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'hgvs', 'hgvs': 'p.Asp614Gly'}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:30] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C030325 ! \"anthrax toxin\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:32] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Ganglioneuroma(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:32] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006388 ! Hemagglutinins), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:33] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Mucins(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:33] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009439 ! Neuraminidase), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HESX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent L(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent env(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpu(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:35] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent rep(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent N(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nahG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nppb(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Lnpep(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent APL(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Fc domain\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFITM proteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IVNS1ABP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:LALAP), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MTase-defective), [{'kind': 'hgvs', 'hgvs': 'p.Asp331Ala'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PC), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PRRSV-nsp1alpha), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent R-BHB(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RAS), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Ran-GTP), [{'kind': 'hgvs', 'hgvs': 'p.Gln69Leu'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Ran-GTP), [{'kind': 'hgvs', 'hgvs': 'p.Gln69Leu'}]\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TIF(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nsp11(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"perforin gene\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"theADRP domains\"), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C405335 ! syncytin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D004275 ! \"DNA, Ribosomal\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016147 ! \"Genes, Tumor Suppressor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22260 ! adenosines), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17790 ! methanol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MS), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73582 ! Leu-Cys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(PUBCHEM:6804 ! 6804), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:36] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 1}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:GpppA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29256 ! thiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Cap), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:58573 ! \"kaempferol oxoanion\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007633 ! Keratins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17622 ! dUMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17478 ! aldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:FDA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:46923 ! dioxane), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:59050 ! \"dimethyl sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:545959 ! \"homovanillic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:68889 ! hydroxytyrosol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134207 ! methylcytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D049308 ! Dioxygenases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:MS), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73582 ! Leu-Cys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16842 ! formaldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60004 ! mixture), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64276 ! glutaraldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35807 ! N-nitrosodimethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18723 ! nicotine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"DMDA 40926\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asp', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16842 ! formaldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60004 ! mixture), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64276 ! glutaraldehyde), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:58573 ! \"kaempferol oxoanion\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16708 ! adenine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85083 ! sofosbuvir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007962 ! Leukocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7625 ! \"nordihydroguaiaretic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 4}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16521 ! lanosterol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0000785\" ! chromatin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16040 ! cytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PPP2C(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Lys'}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85045 ! H3K27Me3), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'pos': 27}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16991 ! \"deoxyribonucleic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39282 ! iodomethane), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64344 ! \"2',4',6'-trihydroxyacetophenone\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D005796 ! Genes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18361 ! \"diphosphate(4-)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17802 ! pseudouridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73622 ! \"beta-hydroxywybutosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16335 ! adenosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 6}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16261 ! chitosan), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74035 ! \"small nuclear RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000375 ! Aging), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16234 ! hydroxide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33942 ! ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15864 ! luteolin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64313 ! 4-demethylwyosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134207 ! methylcytosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D049308 ! Dioxygenases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:47013 ! D-ribofuranose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:131444 ! guanidinoacetate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:49886 ! rimantadine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18349 ! \"N(1)-(5-phospho-D-ribosyl)glycinamide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C537847 ! \"Noonan syndrome 3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Arg', 'pos': 467}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:80666 ! norbelladine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"XIST RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017027 ! \"Protein Tyrosine Phosphatases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60816 ! immunogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16793 ! \"mercury(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16704 ! uridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Cap), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17980 ! \"3'-phospho-5'-adenylyl sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16235 ! guanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 7}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63895 ! \"ionic liquid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(PUBCHEM:6804 ! 6804), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15713 ! UTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16497 ! dGTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48267 ! tubercidin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64313 ! 4-demethylwyosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}, 'code': 'Asn', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17802 ! pseudouridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Cap-0 viral mRNAs\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein methylation', 'identifier': '0006479'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein palmitoylation', 'identifier': '0018345'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RUNX1b), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 787}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009404 ! \"Nephrotic Syndrome\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 828}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0050126\" ! \"N-carbamoylputrescine amidase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:STAT1-S727), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SPRK1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33838 ! nucleoside), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50266 ! prodrug), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"cycling G-associated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000086402 ! SARS-CoV-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:serine-197), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:parts), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 828}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:increase), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT3/STAT4/STAT5/STAT6\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"type I receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 497}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Val', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:5417 ! glucosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15354 ! choline), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16919 ! creatine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17087 ! ketone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent lipopolysaccharide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"NF-jB p50\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"interferon-gamma activated sequence\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent nitric oxide(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16595 ! \"1D-myo-inositol 1,4,5-trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33838 ! nucleoside), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18077 ! dTTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:31536 ! emtricitabine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:134722 ! favipiravir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C087636 ! \"vasodilator-stimulated phosphoprotein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 157}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17013 ! dTMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17992 ! sucrose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 328}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent potassium atom(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16027 ! \"adenosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:37] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C113267 ! nephrin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18179 ! phosphoinositide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent calcium(2+)(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60948 ! \"alpha-D-Gal-(1->4)-beta-D-Gal-(1->4)-D-Glc\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent heme(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16474 ! NADPH), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18009 ! \"NADP(+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:27093 ! \"tricarboxylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28087 ! glycogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74566 ! Lys-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005840\" ! ribosome), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48343 ! disulfide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NRTI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50594 ! \"carbon nanotube\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent 2-O-acetyl-1-O-octadecyl-sn-glycero-3-phosphocholine(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60072 ! \"erythrityl tetranitrate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT1/2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17256 ! \"2'-deoxyadenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4705 ! \"double-stranded DNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE-2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 680}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C083800 ! \"urea transporter\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ala', 'pos': 1}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26392 ! \"purine nucleoside monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74707 ! Met-Met), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74976 ! \"beta-D-Glcp-(1->3)-[beta-D-Glcp-(1->3)-beta-D-GalpNAc-(1->4)-alpha-D-Glcp-(1->4)]-beta-D-GalpNAc-(1->3)-alpha-D-ManpO[CH2]5NH2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:85682 ! \"diacylglycerol 30:1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9765 ! Trypsin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"protein tyrosine kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:7421 ! NAC), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"WNK subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 60}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:42638 ! \"flufenamic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48706 ! antagonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52214 ! ligand), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53490 ! N-formyl-L-methionyl-L-leucyl-L-phenylalanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:38] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010750 ! Phosphoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D029322 ! Norovirus), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D061307 ! \"Human Umbilical Vein Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15351 ! acetyl-CoA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6801 ! metformin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006937 ! Hypercholesterolemia), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22984 ! \"calcium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D061307 ! \"Human Umbilical Vein Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"recQ subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18022 ! thiocyanate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}, {'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15354 ! choline), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60072 ! \"erythrityl tetranitrate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15996 ! GTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Asp', 'pos': 1}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 780}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18022 ! thiocyanate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17172 ! \"2'-deoxyguanosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17256 ! \"2'-deoxyadenosine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90682 ! \"torin 2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:42485 ! \"formyl group\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53490 ! N-formyl-L-methionyl-L-leucyl-L-phenylalanine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003713 ! \"Dendritic Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D020033 ! \"Protein Isoforms\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036681 ! \"Fluorescence Recovery After Photobleaching\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48705 ! agonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D019590 ! \"Nucleocapsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SR-rich sequence\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:factors), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKK_complex(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:72316 ! \"virulence factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification), (phosphorylation))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STAT3/STAT4/STAT5/STAT6\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30413 ! heme), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34873 ! N-nitrosodiethylamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"STATs proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 63}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'pos': 73}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29339 ! \"azanediyl (triplet)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:90695 ! \"anthra[1,9-cd]pyrazol-6(2H)-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000682 ! Amyloid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"extracellular-regulated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18053 ! \"1-aminocyclopropanecarboxylic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28747 ! \"picolinic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017076 ! \"Computer-Aided Design\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Gln690), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Val16), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17076 ! streptomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:48705 ! agonist), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D042783 ! \"Endothelial Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:IkB), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012269 ! \"Ribosomal Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:18348 ! \"1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37530 ! \"phosphatidylinositol 4-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C020845 ! P-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16749 ! 1-phosphatidyl-1D-myo-inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D003643 ! Death), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 133}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17489 ! \"3',5'-cyclic AMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 25}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:22984 ! \"calcium atom\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36704 ! imiquimod), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002364 ! Caseins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D057167 ! Claudins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:subunits), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:82128 ! PB1-F2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 450}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C087636 ! \"vasodilator-stimulated phosphoprotein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"MLC phosphatase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser'}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15918 ! \"2'-deoxycytosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SMAD(mods: (sumoylation))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"receptor-activated SMAD\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TGFB(mods: (sumoylation, R, 1))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:3815 ! Collagen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:8753 ! RI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C069330 ! \"ribosomal protein S6 kinase, 90kDa, polypeptide 3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:147675 ! \"5-iodo-2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:465284 ! ganciclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:60761 ! floxuridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75179 ! trifluridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Tryptase(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent mTORC1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent p38(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent p38(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29312 ! sulfanyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010770 ! Phosphotransferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D054323 ! \"Shiga-Toxigenic Escherichia coli\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010750 ! Phosphoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RNpSTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RIPK3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent BSG(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012194 ! \"RNA-Directed DNA Polymerase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"colorectal carcinoma\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"N SARS-CoV-2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SR-motifs), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SR), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SARS-CoV-2 N protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75011 ! Val-Glu), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:65769 ! dieckol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 727}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kinase-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29035 ! \"manganese(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TBK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRF3(mods: (modification), (phosphorylation))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TGFB1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"type I receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28744 ! 2-hydroxy-17beta-estradiol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16450 ! \"2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17748 ! thymidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:140318 ! tefuryltrione), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16750 ! guanosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:472552 ! \"5-bromo-2'-deoxyuridine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 1177}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53076 ! \"4-(ethoxymethylene)-2-phenyloxazol-5-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017667 ! Adipocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRAF2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cytokine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBL4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBL4A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ULK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16898 ! pyrimidine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP18(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP25(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent VIM(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"SAPK/JNK\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051557 ! \"Hepatocyte Nuclear Factor 4\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ACE2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Lys-788), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:6801 ! metformin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35366 ! \"fatty acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IKBKE(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16595 ! \"1D-myo-inositol 1,4,5-trisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SPINK5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GCA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GCA(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:64108 ! \"8-bromo-3',5'-cyclic GMP\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFI44(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15361 ! pyruvate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CD9(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008869 ! \"Microtubule-Associated Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CEACAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TAB1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15918 ! \"2'-deoxycytosine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16695 ! \"uridine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17361 ! \"cytidine 5'-monophosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17622 ! dUMP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:CXXC1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TICAM1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IFNLR1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29191 ! hydroxyl), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63112 ! \"fingolimod hydrochloride\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0016887\" ! \"ATPase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PIM3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 2204}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CHN1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0036387\" ! \"pre-replicative complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"neurofilament heavy chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:39] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"neurofilament light chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16016 ! dihydroxyacetone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28260 ! galactose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:4170 ! \"D-glucopyranose 6-phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:50699 ! oligosaccharide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23530 ! cytokinin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:31858 ! Mizoribine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63580 ! ribavirin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37684 ! mannose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15698 ! \"2'-deoxycytidine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26020 ! phosphate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:39949 ! \"triphosphoric acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63580 ! ribavirin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent STING1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37712 ! O-phosphoserine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAVS(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"IKKalpha,beta,gamma\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"Inflammatory Bowel Disease\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"zeta chain\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:10110 ! zidovudine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent DUSP1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D053763 ! Presenilins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent EGR1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0009978\" ! \"allene oxide synthase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:2alpha), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic initiation factor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:30762 ! salicylate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0009978\" ! \"allene oxide synthase activity\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16960 ! ADP-D-ribose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D012208 ! Rhabdomyosarcoma), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 796}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FGL2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent FKBP4(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15373 ! biopterin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017076 ! \"Computer-Aided Design\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036681 ! \"Fluorescence Recovery After Photobleaching\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33709 ! \"amino acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81639 ! \"Core protein\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GFAP(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:33709 ! \"amino acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008190 ! Lyases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent GP5(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17552 ! GDP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C448021 ! \"glycogen synthase kinase 3 alpha\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HIF1A(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2K(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HMOX1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent HSPA5(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ATF4-CHOP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IGF1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"janus family\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011505 ! \"Protein-Tyrosine Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IL6(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CXCL8(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D022781 ! Hepatocytes), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent IRS1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent JAK1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D016875 ! \"tau Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011947 ! \"Receptors, Antigen, B-Cell\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K4(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MAP2K7(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mitogen activated protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15551 ! \"prostaglandin E2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ala', 'pos': 2}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:52290 ! mitogen), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:87103 ! \"2-tert-butyl-9-fluoro-1,6-dihydrobenzo[h]imidazo[4,5-f]isoquinolin-7-one\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Thr', 'pos': 23}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036022 ! \"Capsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"microtubule-associated protein tau\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63550 ! TTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D006360 ! \"Heat-Shock Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RNpSTP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:elF4E), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:kinase-1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:25351 ! \"mevalonic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent MYD88(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009504 ! Neutrophils), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34674 ! \"dextran sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16981 ! \"O-(3-O-D-galactosyl-N-acetyl-beta-D-galactosaminyl)-L-serine\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:23357 ! cofactor), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mitogen activated protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ATP1B3(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:elongation), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C095886 ! anillin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15361 ! pyruvate), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:32816 ! \"pyruvic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D010088 ! Oxidoreductases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16480 ! \"nitric oxide\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26523 ! \"reactive oxygen species\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28874 ! phosphatidylinositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0003723\" ! \"RNA binding\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"mis18 family\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015470 ! \"Leukemia, Myeloid, Acute\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:37712 ! O-phosphoserine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:C064368 ! \"proto-oncogene protein Spi-1\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:eIF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eIF2alpha [\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic initiation factor 2, nor oligoadenylate synthetase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"initiation factor eIF2alpha\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine-protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73497 ! Glu-Ile-Phe), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005850\" ! \"eukaryotic translation initiation factor 2 complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015852 ! \"Eukaryotic Initiation Factor-2\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D007074 ! \"Immunoglobulin G\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PI), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28765 ! \"phosphatidylinositol phosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PTGS2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34674 ! \"dextran sulfate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D064448 ! Filamins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74564 ! Lys-Lys), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RELA(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:9516 ! thapsigargin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28757 ! fructose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"ACE2 receptor\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 787}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16393 ! sphingosine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Nucleoproteins(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:cGMP-dependent), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"eukaryotic subfamily\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:132554 ! oligomer), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16227 ! pyridine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Tyr'}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D050838 ! \"Interferon Regulatory Factor-3\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63112 ! \"fingolimod hydrochloride\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Angiotensin-2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:Angiotensin-2), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:variety), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28747 ! \"picolinic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:53407 ! \"poly(lactic acid) macromolecule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:63153 ! N-acetyl-D-mannosamine), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:29108 ! \"calcium(2+)\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011494 ! \"Protein Kinases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D017361 ! \"Viral Nonstructural Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent vpr(mods: (modification))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:28757 ! fructose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Probable head completion/stabilization protein(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent tars-1(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:24848 ! inositol), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000070539 ! \"Pleckstrin Homology Domains\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}, 'code': 'Ser', 'pos': 34}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ICP0(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:NP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16761 ! ADP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D011087 ! \"Polycythemia Vera\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17076 ! streptomycin), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"extracellular-regulated kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:2453 ! acyclovir), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:SD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D050976 ! \"Basic-Leucine Zipper Transcription Factors\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D014411 ! \"Neoplastic Stem Cells\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent S9(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17234 ! glucose), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16783 ! \"1D-myo-inositol 1,3,4,5-tetrakisphosphate\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "WARNING: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D000080865 ! Formins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein phosphorylation', 'identifier': '0006468'}}]\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent PER17(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL97(mods: (modification), muts: (None, None, None))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation, R, 42))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, A, 1))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PA(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:40] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent C3 exoenzyme(mods: (ribosylation, R))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent GTP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide residue(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP-D-ribose(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphosphate(4-)(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation, C, 3))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent mendelevium atom(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Met-Asp(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Core protein(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent single-stranded DNA(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent AXIN(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent E3_Ub_ligase(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, C))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, R))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s_alpha(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation, R, 42))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RNA binding(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RPL24(mods: (ribosylation, E, 106))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TERF1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SIRT4(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SIRT6(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SH2D3C(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP14(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G3BP1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent AGO2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent FDFT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent GAPDH(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHOA(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent LYZ(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent MBP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NUP214(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ATP6V0C(mods: (ribosylation, Q, 124))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAB5A(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RBFOX2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Transferases(mods: (ribosylation, R))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SCC3(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Large T antigen(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Galphas(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent cysH(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent 13(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide(mods: (ribosylation, H))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent diphthamide residue(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation, R, 177))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHO(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Death(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SP1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Histone_H2B(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_i_alpha(mods: (ribosylation, C, 351))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RHOA(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Galphas(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Death(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_i(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_protein(mods: (ribosylation, A, 1))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TRPT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP3(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Protease(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent SDS(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent FDFT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent IKBKG(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NFAT(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NFASC(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ampicillin(mods: (ribosylation, A, 4))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent STAT1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent PARP10(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent TBK1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ctbp1(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent C3 exoenzyme(mods: (ribosylation, R))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Actin(mods: (ribosylation, R, 177))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent N(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent NAD(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent toxin(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent G_s(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation, 123))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation, R, 41))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent EEF2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent ADP(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent Ubiquitin(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAS(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type ribosylation on agent RAC2(mods: (ribosylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NPM1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Histone(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RARA(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HIF1A(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MYC(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NCOR(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPARG(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent VHL(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Interferon(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STAT1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MAPK7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent METTL3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation, K, 888))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation, K, 96))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation, K, 231))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent SUMO3(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 150))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 440))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 60))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent lanA2(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent lanA2(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2I(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CTBP1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SP7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 150))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 440))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent EIF2AK2(mods: (sumoylation, K, 60))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent gC(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM28(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CDK9(mods: (sumoylation))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent VHL(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DDX58(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RBFOX2(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent 3D(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RING(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CGAS(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent STING1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NFKBIA(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NLRP3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Nsp1alpha(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent CREBBP(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IE1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent U90/U89(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PPARG(mods: (sumoylation, K, 107))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DNM1L(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PTPN1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DNM1L(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UIMC1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent HDAC3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent RANBP2(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TRIM5(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IFIH1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent UBE2I(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation, K, 452))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent GTF2I(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent MRE11(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent NBN(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UL82(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Daxx protein(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent BZLF1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent DAXX(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent N(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent alpha synuclein measurement(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Ubiquitin(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent PML(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF3(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent TP53(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent RB1(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent Histone(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent SP100(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent IRF7(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Skipping modification of type sumoylation on agent E3_Ub_ligase(mods: (sumoylation))\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, plasma membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3A(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE-2 receptor(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF(), nucleoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT1 receptors(), plasma membrane, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT2 receptors(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AT2 receptors(), plasma membrane, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATPase(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDCA7L(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E4 protein(), None, inclusion body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EspB(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F protein(), None, basement membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F protein(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FAK(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IVNS1ABP(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IVNS1ABP(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCIT:C121977(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCIT:C63927(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPs(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nuclear Factor of Activated T Cells (NFAT)(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBPL proteins(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PABP(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRF(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(REV(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIIa(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Rev(), nucleolus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SARS-CoV-2 proteins(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLTRiP protein(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SREBP family(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(V ATPase I(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(V ATPase I(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP40(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP40(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP9(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VacA(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(air(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(effector protein(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum-Golgi intermediate compartment (ERGIC)(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gram(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(haemagglutinin(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hairpins(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(kinesin motor(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(microtubule-associated protein 1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nuclear eIF2alpha(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nuclear factor(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pre-miRNA(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pre-miRNA(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(protein tyrosine kinase(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(receptor-activated SMAD(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(replicase(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(replication protein(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rev(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(short hairpin RNA(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(short hairpin RNA(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(snf7(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(srr7(), cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(synaptobrevin(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(transmembrane protein(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tumour suppressor(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vesicular stomatitis virus G protein(), endoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(viral receptor(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(electron(), None, mitochondrion)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(electron(), None, respirasome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(alpha-D-Manp-(1->3)-[alpha-D-Manp-(1->6)]-[beta-D-Xylp-(1->2)]-beta-D-Manp-(1->4)-beta-D-GlcpNAc-(1->4)-D-GlcpNAc-yl group(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(oligomer(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(small nucleolar RNA(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ile-Thr(), None, trans-Golgi network)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glutamate(1-)(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coenzyme A(), None, mitochondrial matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetyl-CoA(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(choline(), None, neuromuscular junction)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), None, neuromuscular junction)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), axon, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetylcholine(), vesicle, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyruvate(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetic acid(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP(), host cell, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(protoporphyrin(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polypeptide(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polypeptide(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arachidonic acid(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTP(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L-glutamic acid(), synapse, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylethanolamine(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cytosine(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), None, organelle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), lysosome, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cholesterol(), virion membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyridine(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(guanine(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hydrogen peroxide(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitrite(), None, chloroplast)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hyaluronic acid(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hyaluronic acid(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipopolysaccharide(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipopolysaccharide(), periplasmic space, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitric oxide(), cell body, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitric oxide(), dendrite, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), None, endoplasmic reticulum membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), phagolysosome, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(peptide(), phagolysosome, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(1-phosphatidyl-1D-myo-inositol(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADP(), None, nuclear membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(2-oxoglutarate(2-)(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glutathione(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pyrimidine(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(leukotriene C4(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(streptomycin(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, ciliary pocket)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein(), rough endoplasmic reticulum, coated vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glucose(), cytoplasm, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, cell surface)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(estrone(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GDP(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L-cysteine(), capsule, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cortisol(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ceramide(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hydrogen chloride(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sucrose(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sucrose(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(thiocyanate(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CMP-N-glycoloyl-beta-neuraminic acid(), cytosol, Golgi lumen)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dopamine(), None, synaptic vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dopamine(), axon, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), cytosol, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron atom(), endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidyl-L-serine(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidyl-L-serine(), None, outer membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cadmium atom(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(drug(), virion membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(heparan sulfate proteoglycan(), None, focal adhesion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(proton(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(proton(), None, synaptic vesicle membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ion(), cytoplasm, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neurotransmitter(), None, synaptic cleft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neurotransmitter(), None, synaptic vesicle membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nitrites(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(reactive oxygen species(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), host cellular component, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sialic acid(), sperm plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(toxin(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vitamin D(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(zinc atom(), cytoplasm, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenoxymethylpenicillin(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenoxymethylpenicillin(), vesicle, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(copper atom(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(iron(3+)(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(copper(2+)(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sodium(1+)(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, replication compartment)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), None, sarcoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), endolysosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calcium(2+)(), sarcoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(disulfur(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tunicamycin(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(heme(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(silver atom(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citric acid(), mitochondrial matrix, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(citric acid(), mitochondrion, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(bentiromide(), None, periplasmic space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(bithionol(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C-Protein(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(platinum(), None, chloroplast)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(platinum(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cerium(), None, vacuolar membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(noradrenaline(), None, neuromuscular junction)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, apical plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleic acid(), endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(messenger RNA(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(messenger RNA(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(amino acid(), None, brush border membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(amino acid(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nucleoside(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(fucose(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(effector(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tetraphosphorus(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sodium taurocholate(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Collagen(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cytochrome(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acetyl group(), periplasmic space, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-acetyl-beta-neuraminic acid(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded DNA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(disulfide(), lysosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-(3-carboxypropanoyl)-N-hydroxyputrescine(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-(3-carboxypropanoyl)-N-hydroxyputrescine(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dextran(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dextran(), lysosome, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ligand(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(precursor Z(), endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(4-(ethoxymethylene)-2-phenyloxazol-5-one(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(emeraldine macromolecule(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phenyl isocyanate(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(malonaldehyde(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(antigen(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polymer(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylinositol trisphosphate(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(poly-ADP-D-ribose(), nucleus, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ionomycin(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(calicheamicin(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(phosphatidylcholine(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lipoprotein(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(double-stranded RNA(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(1,2-dihexadecanoyl-sn-glycero-3-phosphocholine(), None, lamellar body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(poly(uridylic acid)(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Leu-Thr(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Phe-Gly(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Arg-Val(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Gly-Asn(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(His-Ser(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ile-Ala(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Val-Ile(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N-oleoylsphingosine(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NS-5(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Core protein(), nucleolus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PB1-F2(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RI(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(polyamine(), endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), None, cytosol)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), early endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(24,25-Dihydroxyvitamin D(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Melanin(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(single-stranded DNA(), None, cell periphery)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(single-stranded DNA(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S1(), virion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HEL(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACTN(), None, cytoskeleton)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADCY(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKT(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKT(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AP1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(APC_C(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARRB(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Actin(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_III(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_II(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Adaptor_protein_I(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAPN(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD8(), None, mitochondrial inner-outer membrane contact site)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CK2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cathepsin(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chemokine(), extracellular matrix, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chemokine(), host cellular component, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Clathrin(), None, viral factory)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Clathrin(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cofilin(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DVL(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E3_Ub_ligase(), None, kinase activity)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(E3_Ub_ligase(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4G(), None, viral replication complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENO(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERK(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERK(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERM(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETC_complex_II(), endoplasmic reticulum, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETC_complex_II(), endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FNR(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FPR(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GEF(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPase(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPase(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_actin(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G_protein(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin(), None, food vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Histone(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKB(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKK_complex(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon(), nucleus, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JNK(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Kinesin(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LDH(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, phagocytic vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MKNK(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFkappaB(mods: (modification)), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Na_K_ATPase(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Neuropeptides(), C-fiber, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P2RX(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAK(), cytosol, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3K(), None, kinase activity)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3K_p85(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PKC(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLC(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPAR(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP3(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), lysosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protease(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB(), endosome, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RASGRF(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAS(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAS(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RLR(), cytosol, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNApo_II(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTK(), None, cell-cell junction)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RXR(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT5(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAP(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TGFBR(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TGFB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TXN(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Wnt(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mTORC1(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(p38(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA binding(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA-directed 5'-3' RNA polymerase activity(), None, viral replication complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(galactose binding(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(endoplasmic reticulum(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coreceptor activity(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(autotransporter activity(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(secretory granule(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NADH dehydrogenase complex(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(macrophage apoptotic process(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mitochondrial outer membrane permeabilization(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHO(), None, transport vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIPK1(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RIPK2(), None, membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNASEL(), cytoskeleton, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL11(), None, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL22(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(mods: (modification)), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), None, phagocytic vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BECN1(), aggresome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL5(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPL9(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RPS6KA3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTN2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BID(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SAR1A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCL1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BLVRA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CX3CL1(), cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SDC2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SDCBP(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SEL1L(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SELP(), membrane, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SELP(), vesicle, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFPQ(), None, paraspeckles)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRSF5(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFTPB(), None, multivesicular body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SFTPC(), None, multivesicular body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIRT2(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BRAP(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A1(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), cytoplasm, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC2A4(), membrane, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC9A3(), None, microvillus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD1(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BSG(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SOAT1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SOD3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BST2(), host cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPARC(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPARC(), endosome, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPHK1(), None, phagocytic vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPRY1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SQSTM1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRC(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SREBF2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SRPK1(), None, inclusion body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BTN1A1(), None, lipid droplet)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(mods: (phosphorylation)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT1(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT3(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT4(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STAT6(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STIP1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STK17B(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STX17(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SYT1(mods: (phosphorylation)), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SYT1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TALDO1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAZ(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBK1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBPL1(), nucleus, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TCF4(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZEB1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TCP1(), nucleus, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TERT(), None, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TF(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TF(), early endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFAM(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFAP2A(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEB(mods: (phosphorylation, False)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEB(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFEC(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TFRC(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIA1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIAM1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TJP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR2(), None, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR4(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR4(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNF(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFRSF1A(), None, extracellular exosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD40(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFSF13B(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, aggresome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TP53BP2(), cell-cell junction, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSP90B1(), None, extracellular matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRADD(), cytoplasm, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF2(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), None, endosome membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF3(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF5(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF6(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAF6(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIP11(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACP2(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TYK2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2L6(mods: (modification)), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2L6(mods: (modification)), None, ruffle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACP5(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ULK1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ULK1(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USP10(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USP7(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VCP(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VDR(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VDR(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), None, ruffle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VIM(), cytosol, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VWF(), Weibel-Palade body, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCN4(), membrane, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XBP1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XDH(), None, lipid droplet)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XPO1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XRCC1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YY1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZAP70(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM25(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM26(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF112(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CNBP(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZFYVE1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), None, extracellular exosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AICDA(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRAK2(), cytoplasm, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARL6(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C4BPA(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GABARAPL2(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GABARAPL2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3B(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3C(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C6(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS4A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ASZ1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, early endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), cell surface, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE2(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADIPOQ(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CA2(), cytosol, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAG6(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HDAC6(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMPD3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HTRA2(), mitochondrial intermembrane space, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP1R15A(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKE(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKE(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALR(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PINK1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAMP(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CANX(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAPN2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SNX9(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SNX17(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP1(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP3(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CASP8(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAV1(), None, envelope)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAV1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAV2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GOLPH3(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MBTPS1(), None, extracellular matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZNF330(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZNF330(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SPINK5(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CBX4(), None, secretory granule)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL37(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBAS(), None, endosome)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR7(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TLR9(), None, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GEMIN4(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNB1(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCND3(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNH(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CCNH(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GDAP1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TSG101(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TSG101(), endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM63(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL33(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHMP4B(), None, organelle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZBP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YAP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRIM22(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CARD9(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBPL1A(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, outer membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP3(), None, protein-containing complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRC4(), None, protein-containing complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB3IP(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD28(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PYCARD(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRDX6(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD4(), None, mitochondrial inner-outer membrane contact site)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD4(), trans-Golgi network, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SH2D3A(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CIB1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, peroxisome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD74(), None, viral replication complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELL2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD8A(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD9(), None, synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OPTN(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OPTN(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIRAP(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TIRAP(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDC14B(), nucleus, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDC25C(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX23(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(APOBEC3G(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARID5A(), nucleus, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COP1(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ESAM(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDH5(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SH3RF1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTBP2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NECTIN3(), None, cell-cell junction)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XPO5(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDK2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDK4(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNFRSF13C(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1A(), None, replication compartment)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1B(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN1B(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CDKN2A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DROSHA(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CUZD1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NTM(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOSC3(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, late endosome)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNF130(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ASXL1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI2(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TICAM1(), None, late endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CEBPB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI4(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PADI4(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HAVCR2(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIWIL4(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZDHHC3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCSTAMP(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LRRK2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SUGP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, host cell cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNPC3(), virion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPP5(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CFTR(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFIH1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFIH1(), None, inclusion body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM10(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EDEM1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EDEM1(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DCLK2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ALYREF(), cytoplasm, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX58(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX58(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TNIP2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBC1D5(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHD3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TERF2IP(), nucleus, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN2L(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHN1(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), Golgi apparatus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADAM17(), cell surface, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AAK1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CNKSR1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHUK(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CHUK(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CIRBP(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBP5(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBCB(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(mods: (modification)), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrial intermembrane space, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrial membrane, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYCS(), mitochondrion, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCSK9(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RGCC(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS39(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VPS39(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIMP2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(WDFY3(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(WRNIP1(), None, replication fork)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG16L1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG16L1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FIS1(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TMUB1(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CERT1(), None, Golgi apparatus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CERT1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COPG2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(COPG2(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NLRP11(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CP(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC10A3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SLC10A7(), host cell, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RTRAF(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KEAP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CREBBP(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CRYAB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BMF(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YBX3(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LARP4(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CSRP3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOC2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EXOC2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NMRAL1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YOD1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTLA4(), vesicle, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTNNB1(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSB(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSD(), lysosomal lumen, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ANKRD27(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSL(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTSS(), trans-Golgi network, late endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CYBB(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AFAP1L2(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AMBRA1(), None, focal adhesion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AMBRA1(), cytoskeleton, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MLKL(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DAG1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DAXX(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ACE(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PARP1(), mitochondrion, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDB1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDIT3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX17(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX21(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DDX5(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, endoplasmic reticulum-Golgi intermediate compartment)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), endoplasmic reticulum, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(STING1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TBCEL(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SGMS2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DGKZ(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DGKZ(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MLEC(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GRK2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEKHM1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIP5K1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIP5K1(), mitochondrial membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DLL4(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DTX4(), None, centrosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), None, peroxisome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAVS(), mitochondrial membrane, cytosol)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VMP1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNASE1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRGM(), None, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNASE1L3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDPN(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPEG1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZCRB1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GNPTAB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM1L(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM2(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNM2(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TRDMT1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFASC(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CALCOCO2(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DPEP1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEKHA8(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PBRM1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DPP4(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA6(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RILP(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G3BP2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB2(), mitochondrion, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SCRIB(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SCRN2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XRN1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HUWE1(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(USO1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFT20(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRSS57(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEA1(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EED(), None, chromosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEF1A1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EEF2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGF(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGF(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGFR(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGFR(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EGR1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2AK3(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AGO2(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2S1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2S1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), None, viral replication complex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4E(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF4G2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELANE(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SERPINB1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELAVL1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ELK1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADGRE2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ADGRE2(), None, ruffle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTTN(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CTTN(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AGT(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENDOG(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENDOG(), mitochondrial intermembrane space, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ENPP1(), None, extracellular matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CARD16(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EP300(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EPHX1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AHR(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ETV6(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EZH2(), None, chromosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIF1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(F3(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIP(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FBL(), nucleolus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIRE(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FCGR3B(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FGF4(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FGF9(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FKBP1A(), None, centrosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UBE2QL1(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FLNA(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOS(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO3(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FOXO3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKR1B10(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AKR1B10(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERVK-18(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERVK-21(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MTOR(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FUT2(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FUT2(), None, hyphal tip)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ISG15(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(G6PC1(), None, endoplasmic reticulum membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAK(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAPDH(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBF1(), Golgi apparatus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GBF1(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GC(), endoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GHR(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GHR(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CBLIF(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GJA1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GJA1(), intercalated disc, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GLI1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOP53(), None, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOP53(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP5(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZBED1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(TAP2(), cytoplasm, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CXCL1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CXCL2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA3(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDIA3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GSR(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GTPBP1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GUSB(), host cell, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GYPC(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GZMB(), endosome, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HAND1(), None, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KALRN(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HGF(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HGS(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1A(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIF1A(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), host cell, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMGB1(), nucleus, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HMG20A(), nucleus, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HNRNPK(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HPSE(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HPSE(), host cell, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSF1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA1A(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, extracellular matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA5(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPA8(), None, viral factory)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPB3(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HSPD1(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNAJB1(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HYAL2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOD2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ANXA1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ICAM1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI16(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI16(), nucleus, multivesicular body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFI30(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFITM1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFITM3(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNAR1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNG(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNG(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFNGR2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG12(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG12(), None, phagophore assembly site)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG5(), None, phagophore assembly site)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IK(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IKBKB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL17A(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL2RA(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL4(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IL4(), extracellular exosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ILF2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INS(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(INS(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRAK1(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRAK1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IREB2(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF3(mods: (phosphorylation), (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF3(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF5(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF7(mods: (phosphorylation), (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF7(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IRF9(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ITGB2(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ITPR1(), endoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAG1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAG2(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAK1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JAK2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JUN(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(JUN(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AQP4(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LAMP1(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LCK(), None, immunological synapse)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF1(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF1(), None, early endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS1(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, endolysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS8(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGALS9(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARF5(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LMAN1(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHOA(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RHOC(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LTF(), endosome, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBR1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD2(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SMAD4(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ZFYVE9(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAL(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MALT1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP2K7(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K5(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP3K7(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAPK8(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAPK8IP1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAPK9(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARL1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD46(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CD46(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MDK(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MDM2(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MICA(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MARCKSL1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MME(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP3(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP7(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP9(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MMP9(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MOV10(), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MPO(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MT-ATP6(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MT-RNR2(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MVP(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MVP(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYD88(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYD88(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYL3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MYOM2(), rough endoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NBN(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCAM1(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NCL(), nucleolus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NEU1(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFATC1(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFATC2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L2(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFE2L2(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NFKB2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF2(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NMI(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF4(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOS1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOS3(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NOTCH1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPM1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NPM1(), None, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATF6(), None, viral factory)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR1H4(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NR3C1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NRF1(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(YBX1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUMA1(), None, centrosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NUP214(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NXF1(), None, inclusion body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(NXF1(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIGMAR1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBP(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSBP(), None, replication compartment)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OSMR(), None, mitochondrial matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OXT(), dendrite, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P4HB(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P4HB(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PABPC1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FURIN(), None, extracellular matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAK1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKN(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKN(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PAWR(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCBP2(), nucleus, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP7A(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATP7A(), trans-Golgi network, melanosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PCSK7(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrial intermembrane space, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrion, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AIFM1(), mitochondrion, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDK1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PDPK1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATR(), None, replication fork)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CFP(), membrane, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SERPINA1(), None, vesicle)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIK3CG(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI4KA(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PI4KB(), None, Golgi membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PIN1(), None, nuclear body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PKM(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLA2G4A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLA2G4A(), cytosol, None)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLAUR(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLCG2(), cytosol, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLEK(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLG(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLSCR1(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PLXNB1(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PMAIP1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PML(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PML(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POLK(), None, replication fork)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPIC(), None, extracellular space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPID(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PPP1CA(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKCA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(OLIG2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRKCD(), cytosol, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAG3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EIF2AK2(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRNP(), None, extracellular exosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(LGMN(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAK1(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAK1(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PSTPIP2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTBP1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTEN(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), cytosol, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BAX(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTK2B(), cytosol, mitochondrial matrix)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PTPN11(), None, membrane raft)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(CAVIN1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PEX19(), peroxisome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PXN(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR1(mods: (modification)), None, cell cortex)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR3(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCAR3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB1A(), vacuole, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB11A(), None, recycling endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAB5A(), late endosome, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAC1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAF1(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAF1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCHE(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCHE(), rough endoplasmic reticulum, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RAMP1(), membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RB1(), nucleoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBM4(), None, nucleoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBM4(), nucleolus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBFOX2(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RBFOX2(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCL2A1(), None, extracellular exosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(BCL2L11(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(REL(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), None, host)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELA(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RELB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RGN(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(anthrax toxin(), None, lysosome)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(glycoprotein 41(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(coronin(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Noonan syndrome 3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Cyclophilin D(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Angiotensin-Converting Enzyme 2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Alkaline Phosphatase(), None, periplasmic space)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Amnion(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Antigens, Surface(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Capsid(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Capsid(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Chlamydia trachomatis(), endosome, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dermatomyositis(), lysosome, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), None, apical plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), rough endoplasmic reticulum, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemagglutinins(), trans-Golgi network, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hydrogen-Ion Concentration(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ion Channels(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Multiple Sclerosis(), None, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Neuraminidase(), host cellular component, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(mods: (phosphorylation)), None, virion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleoproteins(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Oxidoreductases(), endoplasmic reticulum, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Protein-Tyrosine Kinases(), None, focal adhesion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Receptors, Antigen, B-Cell(), None, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ribonucleoproteins(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DNA-Directed RNA Polymerases(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Zinc(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Gene Products, tax(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA-Binding Proteins(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Computer-Aided Design(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Viral Nonstructural Proteins(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Guide(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Integrases(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Integrases(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Nucleocapsid Proteins(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Guanine Nucleotide Exchange Factors(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Coat Protein Complex I(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tacrolimus Binding Proteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Small Interfering(), endosome, endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Lectins(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Interferon Regulatory Factor-3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Basic-Leucine Zipper Transcription Factors(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Catenins(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Syntenins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Long Noncoding(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA, Long Noncoding(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Extracellular Traps(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(orf(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cpyA(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lptD(), periplasmic space, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(pgk(), None, chloroplast)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ipaH9.8(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SIFamide-related peptide(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG8(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG8(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(egfp(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(US20(), None, lysosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP(), None, multivesicular body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GP(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cas9(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Extracellular deoxyribonuclease(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(yopH(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rsgA(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(yopS(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(MAP1LC3B(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB1(), cytosol, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PHB1(), nucleus, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(icd(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cdu1(), None, host cell)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(neo(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin subunit beta-(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Hemoglobin subunit beta-(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ricin(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tax(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Tax(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF3b(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, plasma membrane region)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tat(), nucleoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpx(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpx(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(vpu(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(UL83(mods: (modification)), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cre(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(apxIIC(), mitochondrion, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(acrA(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nfrB(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), cytoplasm, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), cytoplasm, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ken-106(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ken-106(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(3a(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(M(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(M(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rep(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gC(), None, host cell nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gC(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(RNA1(), None, mitochondrial membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Large T antigen(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(rep(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(hifA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(lef(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(gag-pol(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(P39(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POL(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(POL(), None, nuclear body)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(impB(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nit-2(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, nucleus)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(4b(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP2(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AR1(), cytosol, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ba71V-081(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(AN(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(N(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(XVI(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Genome polyprotein(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FASN(), None, replication compartment)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(manC(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cya(), endosome, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Probable integrase/recombinase(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Uncharacterized 9.8 kDa protein(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(tsr(), None, outer membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HIR3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(B23(), None, nucleolus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(IFN(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP3(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(smo-1(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(atp2b2(), cytoplasm, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERV-H1 provirus ancestral Env polyprotein(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mtb(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(mtb(), host cellular component, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VII(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Stim(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Putative leucine-rich repeat protein PS14(), nucleus, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(cyaA(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dif(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(U3(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP3(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRP3(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ipaB(), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(GAL83(), cytoplasm, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(eaeB(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(X(), None, mitochondrion)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arm(), None, X chromosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(arm(), None, chromosome, centromeric region)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(EBNA1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PA(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATJ1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ERS1(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ARAC3(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Ankrd27(), None, endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(staB(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(C(), None, endoplasmic reticulum lumen)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(43(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(SEC61(), None, vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FPR1(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DOA4(), None, late endosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(crt(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Dlg1(), None, immunological synapse)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(chmp4(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(nicD(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(5a(), plasma membrane, None)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(L(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(S7(), nucleus, chromosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Trak2(), cytoplasm, mitochondrion)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ATG9(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(HDA5(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(dlt(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(DEG15(), None, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(KRS(), None, extracellular exosome)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Zcrb1(), nucleus, cytoplasm)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(sseJ(), None, cytosol)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ORF4(), None, host cellular component)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Iigp1(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(Iigp1(), None, symbiont-containing vacuole)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(FAS2(), cytosol, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(ctxB(), None, membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PERK12(), None, nucleus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, Golgi apparatus)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(env(), None, rough endoplasmic reticulum)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(envelope glycoproteins(), None, plasma membrane)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(PRF(mods: (glycosylation)), None, secretory granule)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(VP7(muts: (None, None, None)), None, cell surface)\n",
      "INFO: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Unhandled statement: Translocation(transcriptional activator(mods: (modification)), None, nucleus)\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15841 ! polypeptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:73888 ! Gly-Asn), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D005729 ! Ganglioneuroma), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:41] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74823 ! Ser-Val), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0036407\" ! \"mycolate outer membrane\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 3}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D036022 ! \"Capsid Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:35224 ! effector), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D015513 ! \"Oncogene Proteins\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ARF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 3}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D014166 ! Transferases), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:74072 ! Ile-Leu), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:ARF), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent UBE2N(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP1(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent USP7(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM25(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:RD), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:26536 ! \"retinoic acid\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"serine/threonine protein kinase\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}, 'code': 'Lys', 'pos': 209}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:34826 ! M1), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D048068 ! PPAR-beta), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:USP33c), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D002213 ! Capsid), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D009698 ! Nucleoproteins), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent WWP2(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17089 ! glycoprotein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent TRIM52(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:67208 ! \"double-stranded RNA\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:PA), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent CDK12(mods: (modification), muts: (G, 731, R))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D020365 ! \"RNA Helicases\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent ISG15(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:15422 ! ATP), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:81035 ! NS-5), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:5B), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75012 ! Val-Ile), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(TEXT:\"alpha synuclein measurement\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:75012 ! Val-Ile), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "INFO: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Skipping modification of type modification on agent Hemagglutinins(mods: (modification))\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:36080 ! protein), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:16670 ! peptide), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D008285 ! \"Major Histocompatibility Complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(MESH:D051379 ! Mice), [{'kind': 'hgvs', 'hgvs': 'p.?'}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: bp(GO:\"GO:0030964\" ! \"NADH dehydrogenase complex\"), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n",
      "WARNING: [2021-03-24 09:29:42] indra.assemblers.pybel.assembler - Node should not have variants: a(CHEBI:17263 ! estrone), [{'kind': 'pmod', 'concept': {'namespace': 'go', 'name': 'protein ubiquitination', 'identifier': '0016567'}}]\n"
     ]
    }
   ],
   "source": [
    "emmaa_covid_graph = pybel.from_emmaa('covid19')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: [2021-03-24 09:19:05] indra.statements.io - Error creating statement: 'str' object has no attribute 'get'\n",
      "WARNING: [2021-03-24 09:19:05] indra.statements.io - Error creating statement: 'str' object has no attribute 'get'\n",
      "WARNING: [2021-03-24 09:19:05] indra.statements.io - Error creating statement: 'str' object has no attribute 'get'\n",
      "WARNING: [2021-03-24 09:19:05] indra.statements.io - Error creating statement: 'str' object has no attribute 'get'\n",
      "WARNING: [2021-03-24 09:19:05] indra.statements.io - Error creating statement: 'str' object has no attribute 'get'\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "local_covid_indra_statements = '../data/emmaa/covid19-indra-grounded.latest.statements.json'\n",
    "local_covid_indra_nodelink = '../data/emmaa/covid19-indra-grounded.latest.bel.nodelink.json'\n",
    "if os.path.exists(local_covid_indra_nodelink):\n",
    "    emmaa_covid_graph = pybel.from_indra_statements_json_file(local_covid_indra_nodelink)\n",
    "elif os.path.exists(local_covid_indra_statements):\n",
    "    pybel.to_nodelink_file(emmaa_covid_graph, local_covid_indra_nodelink)\n",
    "else:\n",
    "    emmaa_covid_graph = pybel.from_emmaa('covid19')\n",
    "    pybel.to_nodelink_file(emmaa_covid_graph, local_covid_indra_nodelink)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.path.exists(local_covid_indra_nodelink)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../data/emmaa/covid19-indra-grounded.latest.bel.nodelink.json\n"
     ]
    }
   ],
   "source": [
    "!ls ../data/emmaa/covid19-indra-grounded.latest.bel.nodelink.json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'emmaa_covid_graph' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-85d1433e1960>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mpybel\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mto_nodelink_file\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0memmaa_covid_graph\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m'../data/emmaa/covid19-indra-grounded.latest.bel.nodelink.json'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'emmaa_covid_graph' is not defined"
     ]
    }
   ],
   "source": [
    "pybel.to_nodelink_file(emmaa_covid_graph,'../data/emmaa/covid19-indra-grounded.latest.bel.nodelink.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2021-03-12 14:55:42--  https://emmaa.s3.amazonaws.com/assembled/covid19/statements_2021-03-12-18-00-52.gz\n",
      "Resolving emmaa.s3.amazonaws.com (emmaa.s3.amazonaws.com)... 52.217.0.131\n",
      "Connecting to emmaa.s3.amazonaws.com (emmaa.s3.amazonaws.com)|52.217.0.131|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 184601796 (176M) [binary/octet-stream]\n",
      "Saving to: ‘statements_2021-03-12-18-00-52.gz’\n",
      "\n",
      "statements_2021-03- 100%[===================>] 176.05M  2.17MB/s    in 20s     \n",
      "\n",
      "2021-03-12 14:56:03 (8.66 MB/s) - ‘statements_2021-03-12-18-00-52.gz’ saved [184601796/184601796]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget -c https://emmaa.s3.amazonaws.com/assembled/covid19/statements_2021-03-12-18-00-52.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------  ------------------------------------\n",
      "Name                   covid19\n",
      "Version                927b1878-47ce-4bf1-a1ec-f5634f3818b6\n",
      "Author                 INDRA\n",
      "Number of Nodes        95991\n",
      "Number of Namespaces   11\n",
      "Number of Edges        756991\n",
      "Number of Annotations  9\n",
      "Number of Citations    109140\n",
      "Number of Authors      0\n",
      "Network Density        8.22E-05\n",
      "Number of Components   681\n",
      "Number of Warnings     0\n",
      "---------------------  ------------------------------------\n",
      "\n",
      "Type (5)             Count  Example\n",
      "-----------------  -------  ----------------------------------------------------------------------------------\n",
      "Complex              46489  complex(p(FPLX:\"G_i\" ! \"G_i\"), p(HGNC:1331 ! C5))\n",
      "Protein              20867  p(HGNC:7014 ! MEOX2)\n",
      "Abundance            16981  a(CHEBI:29852 ! \"methylarsonic acid\")\n",
      "BiologicalProcess    11530  bp(MESH:D011013 ! Pneumonectomy)\n",
      "Reaction               124  rxn(reactants(p(UP:SL-0427 ! SL-0427)), products(p(FPLX:Interferon ! Interferon)))\n",
      "\n",
      "Namespace (11)      Count  Example\n",
      "----------------  -------  --------------------------------------------------------------------\n",
      "HGNC                66239  p(HGNC:11240 ! SPHK1)\n",
      "TEXT                19048  a(TEXT:ein)\n",
      "UP                  17459  p(UP:Q38710 ! Q38710)\n",
      "CHEBI               16882  a(CHEBI:584020 ! methysergide)\n",
      "MESH                15556  bp(MESH:D004548 ! Elasticity)\n",
      "FPLX                 7255  p(FPLX:FPR ! FPR, pmod(go:0006468 ! \"protein phosphorylation\", 303))\n",
      "PFAM                 3655  p(PFAM:PF15392 ! Joubert)\n",
      "GO                   3549  bp(GO:\"GO:0070508\" ! \"cholesterol import\")\n",
      "go                   3103\n",
      "IP                    250  p(IP:IPR015433)\n",
      "PUBCHEM               119  a(PUBCHEM:22833565 ! 22833565)\n",
      "\n",
      "Edge Type (60)                                           Count  Example\n",
      "-----------------------------------------------------  -------  --------------------------------------------------------------------------------------------------------------------------------------------\n",
      "Abundance increases Protein                              87907  a(CHEBI:15422 ! ATP) increases p(HGNC:18873 ! IFIH1)\n",
      "Abundance decreases Protein                              84422  a(CHEBI:31638 ! fulvestrant) decreases p(HGNC:4603 ! CXCL2)\n",
      "Protein partOf Complex                                   73208  p(HGNC:15532 ! JAM3) partOf complex(p(FPLX:Integrins ! Integrins), p(HGNC:14685 ! F11R), p(HGNC:15532 ! JAM3))\n",
      "Protein increases Protein                                64027  p(UP:Q8JH68 ! Q8JH68) increases p(UP:Q9W4S7 ! Q9W4S7)\n",
      "BiologicalProcess increases BiologicalProcess            61945  bp(MESH:D007694 ! \"Killer Cells, Natural\") increases bp(MESH:D001921 ! Brain)\n",
      "Abundance decreases BiologicalProcess                    54498  a(CHEBI:28304 ! heparin) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Protein increases BiologicalProcess                      50798  p(HGNC:3009 ! DPP4) increases bp(MESH:D000086382 ! COVID-19)\n",
      "Abundance increases BiologicalProcess                    35224  a(CHEBI:44915 ! propofol) increases bp(MESH:D007022 ! Hypotension)\n",
      "Protein decreases Protein                                29721  p(HGNC:8974 ! PIK3C3) decreases p(HGNC:12611 ! USP13)\n",
      "Protein decreases BiologicalProcess                      24128  p(HGNC:8763 ! PDCD4) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "BiologicalProcess decreases BiologicalProcess            23016  bp(MESH:D005334 ! Fever) decreases bp(MESH:D009325 ! Nausea)\n",
      "Abundance directlyDecreases Protein                      22513  a(CHEBI:39867 ! \"valproic acid\") directlyDecreases p(HGNC:4616 ! GSK3A)\n",
      "Abundance partOf Complex                                 18799  a(TEXT:importin) partOf complex(a(TEXT:importin), p(UP:Q4L2X2 ! Q4L2X2))\n",
      "Abundance decreases Abundance                            18672  a(CHEBI:16670 ! peptide) decreases a(TEXT:dose)\n",
      "BiologicalProcess increases Protein                      18017  bp(GO:\"GO:0004709\" ! \"MAP kinase kinase kinase activity\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Protein increases Abundance                              17053  p(HGNC:2014 ! CLC) increases a(CHEBI:73509 ! Glu-Ser)\n",
      "Abundance increases Abundance                            17049  a(TEXT:\"survival probability\") increases a(TEXT:\"survival probability\")\n",
      "BiologicalProcess increases Abundance                    12230  bp(MESH:C023826 ! \"poly(dC)\") increases a(TEXT:\"nsp13 helicase\")\n",
      "Protein decreases Abundance                               8137  p(HGNC:2596 ! CYP1A2) decreases a(CHEBI:8050 ! phenacetin)\n",
      "BiologicalProcess decreases Protein                       7739  bp(MESH:D034741 ! \"RNA, Small Interfering\") decreases p(HGNC:9236 ! PPARG)\n",
      "BiologicalProcess partOf Complex                          7435  bp(MESH:D006238 ! Haploidy) partOf complex(bp(MESH:D006238 ! Haploidy), p(PFAM:PF00424 ! REV))\n",
      "Protein directlyIncreases Protein                         5136  p(HGNC:3236 ! EGFR) directlyIncreases p(HGNC:3236 ! EGFR, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "BiologicalProcess decreases Abundance                     5050  bp(MESH:D019895 ! Transposases) decreases a(CHEBI:49867 ! \"antimony(3+)\")\n",
      "Protein hasVariant Protein                                3613  p(HGNC:2561 ! CXCR4) hasVariant p(HGNC:2561 ! CXCR4, var(\"p.?\"))\n",
      "Complex increases Protein                                 1148  complex(p(HGNC:583 ! APC), p(HGNC:9452 ! PROCR)) increases p(HGNC:3537 ! F2R)\n",
      "Protein directlyDecreases Protein                          832  p(HGNC:6990 ! MECP2) directlyDecreases p(HGNC:4235 ! GFAP)\n",
      "Complex increases BiologicalProcess                        771  complex(p(FPLX:\"LFA_1\" ! \"LFA_1\"), p(HGNC:584 ! APCS)) increases bp(GO:\"GO:0042110\" ! \"T cell activation\")\n",
      "Complex decreases Protein                                  684  complex(bp(MESH:D000086402 ! SARS-CoV-2), p(HGNC:13557 ! ACE2)) decreases p(HGNC:13557 ! ACE2)\n",
      "Abundance directlyIncreases Protein                        676  a(TEXT:TRIM25alpha) directlyIncreases p(HGNC:19102 ! DDX58, pmod(go:0016567 ! \"protein ubiquitination\"))\n",
      "Protein directlyIncreases Abundance                        483  p(HGNC:11305 ! SRPK1) directlyIncreases a(TEXT:\"N SARS-CoV-2\")\n",
      "Complex decreases BiologicalProcess                        368  complex(p(HGNC:9253 ! PPIA), p(UP:P21942 ! P21942)) decreases bp(GO:\"GO:0071897\" ! \"DNA biosynthetic process\")\n",
      "Abundance directlyIncreases Abundance                      277  a(CHEBI:16761 ! ADP) directlyIncreases a(CHEBI:16960 ! ADP-D-ribose)\n",
      "Complex increases Abundance                                226  complex(p(HGNC:37 ! ABCA7), p(HGNC:600 ! APOA1)) increases a(CHEBI:16247 ! phospholipid)\n",
      "Protein directlyIncreases Reaction                         171  p(PFAM:PF00619 ! CARD) directlyIncreases rxn(reactants(p(HGNC:29233 ! MAVS)), products(bp(MESH:D011328 ! Prions)))\n",
      "BiologicalProcess directlyIncreases Protein                151  bp(MESH:C003614 ! \"4-toluenesulfonyl fluoride\") directlyIncreases p(FPLX:AMPK ! AMPK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Complex decreases Abundance                                143  complex(a(CHEBI:16704 ! uridine), a(CHEBI:90879 ! tipiracil)) decreases a(TEXT:ERN1)\n",
      "Reaction hasProduct Abundance                               99  rxn(reactants(a(CHEBI:2718 ! \"angiotensin I\")), products(a(TEXT:Angiotensin-2))) hasProduct a(TEXT:Angiotensin-2)\n",
      "Reaction hasReactant Abundance                              97  rxn(reactants(a(CHEBI:17552 ! GDP)), products(a(CHEBI:15996 ! GTP))) hasReactant a(CHEBI:17552 ! GDP)\n",
      "Abundance directlyDecreases Abundance                       90  a(CHEBI:16261 ! chitosan) directlyDecreases a(CHEBI:17029 ! chitin)\n",
      "BiologicalProcess directlyDecreases Protein                 72  bp(MESH:C520962 ! obatoclax) directlyDecreases p(HGNC:990 ! BCL2)\n",
      "Abundance directlyDecreases BiologicalProcess               69  a(CHEBI:16243 ! quercetin) directlyDecreases bp(MESH:D006388 ! Hemagglutinins)\n",
      "Protein directlyDecreases Abundance                         60  p(HGNC:3363 ! ENTPD1) directlyDecreases a(CHEBI:15422 ! ATP)\n",
      "Protein directlyIncreases BiologicalProcess                 59  p(FPLX:AKT ! AKT) directlyIncreases bp(MESH:D029322 ! Norovirus)\n",
      "Abundance directlyIncreases BiologicalProcess               33  a(CHEBI:6495 ! lipoprotein) directlyIncreases bp(MESH:D008285 ! \"Major Histocompatibility Complex\")\n",
      "Reaction hasReactant Protein                                26  rxn(reactants(p(HGNC:402 ! ALDH1A1)), products(a(CHEBI:144078 ! \"dihydroartemisinic acid\"))) hasReactant p(HGNC:402 ! ALDH1A1)\n",
      "Reaction hasProduct Protein                                 23  rxn(reactants(a(TEXT:Angiotensin-2)), products(p(HGNC:11863 ! TM7SF2))) hasProduct p(HGNC:11863 ! TM7SF2)\n",
      "BiologicalProcess directlyIncreases Abundance               21  bp(MESH:D002490 ! \"Central Nervous System\") directlyIncreases a(CHEBI:35224 ! effector)\n",
      "Abundance directlyIncreases Reaction                        19  a(TEXT:ARF) directlyIncreases rxn(reactants(a(CHEBI:15354 ! choline)), products(a(CHEBI:16337 ! \"phosphatidic acid\")))\n",
      "BiologicalProcess directlyIncreases BiologicalProcess       16  bp(MESH:D011494 ! \"Protein Kinases\") directlyIncreases bp(MESH:D011494 ! \"Protein Kinases\")\n",
      "BiologicalProcess directlyIncreases Reaction                12  bp(MESH:D014166 ! Transferases) directlyIncreases rxn(reactants(a(CHEBI:18361 ! \"diphosphate(4-)\")), products(a(CHEBI:15422 ! ATP)))\n",
      "BiologicalProcess directlyDecreases Abundance                9  bp(MESH:D006005 ! Phosphorylases) directlyDecreases a(CHEBI:39949 ! \"triphosphoric acid\")\n",
      "Abundance directlyDecreases Complex                          7  a(CHEBI:71952 ! dactolisib) directlyDecreases complex(p(HGNC:8975 ! PIK3CA), p(HGNC:8979 ! PIK3R1))\n",
      "Protein directlyDecreases BiologicalProcess                  4  p(FPLX:Phosphatase ! Phosphatase) directlyDecreases bp(GO:\"GO:0042769\" ! \"DNA damage response, detection of DNA damage\")\n",
      "Reaction hasProduct BiologicalProcess                        2  rxn(reactants(p(HGNC:29233 ! MAVS)), products(bp(MESH:D011328 ! Prions))) hasProduct bp(MESH:D011328 ! Prions)\n",
      "Abundance decreases Complex                                  1  a(CHEBI:16856 ! glutathione) decreases complex(a(TEXT:Gclc), a(TEXT:Gclm))\n",
      "Complex partOf Complex                                       1  complex(a(TEXT:JUNCTION), p(UP:O49835 ! O49835)) partOf complex(a(CHEBI:36080 ! protein), complex(a(TEXT:JUNCTION), p(UP:O49835 ! O49835)))\n",
      "Complex directlyIncreases Protein                            1  complex(p(HGNC:11362 ! STAT1), p(HGNC:6893 ! MAPT)) directlyIncreases p(HGNC:933 ! BACE1)\n",
      "Protein decreases Complex                                    1  p(HGNC:12397 ! TTF1) decreases complex(p(HGNC:6769 ! SMAD3), p(HGNC:6770 ! SMAD4))\n",
      "Complex directlyDecreases Protein                            1  complex(p(HGNC:2347 ! CREB3), p(UP:P04608 ! P04608)) directlyDecreases p(UP:P04608 ! P04608)\n",
      "Reaction hasReactant BiologicalProcess                       1  rxn(reactants(bp(GO:\"GO:0003823\" ! \"antigen binding\")), products(a(TEXT:SPIKE-PROTEIN))) hasReactant bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "\n"
     ]
    }
   ],
   "source": [
    "emmaa_covid_graph.summarize()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "a(TEXT:BHB)\n",
      "p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"protein acetylation\"))\n",
      "p(FPLX:Histone ! Histone)\n",
      "a(TEXT:MS)\n",
      "a(TEXT:\")-derived O-acetylneuraminic acids\")\n"
     ]
    }
   ],
   "source": [
    "# look at the first 5 nodes in the graph\n",
    "for i, node in zip(range(5), emmaa_covid_graph.nodes()):\n",
    "    print(node)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Check a protein is in the graph\n",
    "histone_h3_family = pybel.dsl.Protein(namespace='FPLX', identifier='Histone_H3', name='Histone_H3')\n",
    "histone_h3_family in emmaa_covid_graph"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{<BEL p(HGNC:13557 ! ACE2)>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 680))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 787))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006479 ! \"protein methylation\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006486 ! \"protein glycosylation\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0006486 ! \"protein glycosylation\", Ser))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0016567 ! \"protein ubiquitination\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, pmod(go:0016567 ! \"protein ubiquitination\", Lys, 788))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, var(\"p.?\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, var(\"p.Asp30Ala\"))>,\n",
       " <BEL p(HGNC:13557 ! ACE2, var(\"p.Phe83Tyr\"))>}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Get all proteins whose names are ACE2 - might have modifications, too!\n",
    "ace2s = {\n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.Protein) # This line means check if the node has a name\n",
    "    if node.name == 'ACE2'\n",
    "}\n",
    "\n",
    "ace2s"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<BEL bp(MESH:D042783 ! \"Endothelial Cells\")>,\n",
       " <BEL a(TEXT:\"endothelial dysfunction\")>,\n",
       " <BEL bp(GO:\"GO:0072577\" ! \"endothelial cell apoptotic process\")>,\n",
       " <BEL bp(GO:\"GO:0042118\" ! \"endothelial cell activation\")>,\n",
       " <BEL bp(GO:\"GO:0043542\" ! \"endothelial cell migration\")>,\n",
       " <BEL bp(MESH:D061307 ! \"Human Umbilical Vein Endothelial Cells\")>,\n",
       " <BEL bp(GO:\"GO:0001935\" ! \"endothelial cell proliferation\")>,\n",
       " <BEL bp(GO:\"GO:0140074\" ! \"cardiac endothelial to mesenchymal transition\")>,\n",
       " <BEL a(TEXT:\"extrahepatic endothelial anticoagulant protein S\")>,\n",
       " <BEL bp(GO:\"GO:0038084\" ! \"vascular endothelial growth factor signaling pathway\")>,\n",
       " <BEL bp(MESH:D066026 ! \"Endothelial Progenitor Cells\")>,\n",
       " <BEL bp(GO:\"GO:0010573\" ! \"vascular endothelial growth factor production\")>,\n",
       " <BEL bp(GO:\"GO:0101023\" ! \"vascular endothelial cell proliferation\")>,\n",
       " <BEL a(TEXT:\"lung capillary endothelial cell\")>,\n",
       " <BEL a(TEXT:\"endothelial HUVEC\")>,\n",
       " <BEL a(TEXT:\"endothelial Tie-2 receptors\")>,\n",
       " <BEL bp(GO:\"GO:0035767\" ! \"endothelial cell chemotaxis\")>,\n",
       " <BEL bp(GO:\"GO:0038085\" ! \"vascular endothelial growth factor binding\")>,\n",
       " <BEL bp(GO:\"GO:0045446\" ! \"endothelial cell differentiation\")>,\n",
       " <BEL bp(MESH:C087589 ! \"anti-endothelial cell antibody\")>,\n",
       " <BEL bp(GO:\"GO:0001885\" ! \"endothelial cell development\")>,\n",
       " <BEL bp(MESH:D057129 ! \"Iridocorneal Endothelial Syndrome\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial\")>,\n",
       " <BEL a(TEXT:\"endothelial P-\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial growth factor 165\")>,\n",
       " <BEL bp(MESH:D016228 ! \"Endothelial Growth Factors\")>,\n",
       " <BEL a(TEXT:\"endothelial barrier\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial growth factor receptors (VEGFRs) 1\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial growth factor receptors (VEGFRs) 1, 2 and 3\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial growth factor receptors 1, 2, and 3\")>,\n",
       " <BEL a(TEXT:\"vascular endothelial growth factor receptors 2 and 3 [\")>,\n",
       " <BEL a(CHEBI:65207 ! \"vascular endothelial growth factor receptor antagonist\")>]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get all proteins named angiotensin, turns out there's only 1\n",
    "endothelial = [ \n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.BaseConcept)\n",
    "    if 'endothelial' in node.name.lower()  \n",
    "]\n",
    "endothelial"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<BEL p(FPLX:IFNA ! IFNA)>,\n",
       " <BEL a(TEXT:\"IFNalpha/beta\")>,\n",
       " <BEL p(FPLX:IFNAR ! IFNAR)>,\n",
       " <BEL p(HGNC:5419 ! IFNA13)>,\n",
       " <BEL a(TEXT:\"anti-Ifnar1 mAb\")>,\n",
       " <BEL p(HGNC:5417 ! IFNA1)>,\n",
       " <BEL p(HGNC:5425 ! IFNA4)>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, var(\"p.Ser526Ala\"))>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1)>,\n",
       " <BEL p(HGNC:5423 ! IFNA2)>,\n",
       " <BEL p(HGNC:5426 ! IFNA5)>,\n",
       " <BEL p(HGNC:5433 ! IFNAR2)>,\n",
       " <BEL a(TEXT:\"IFNAR receptor\")>,\n",
       " <BEL p(HGNC:5420 ! IFNA14)>,\n",
       " <BEL p(HGNC:5429 ! IFNA8)>,\n",
       " <BEL p(FPLX:IFNAR ! IFNAR, pmod(go:0006468 ! \"protein phosphorylation\"))>,\n",
       " <BEL a(TEXT:IFNAR1-IFNAR2)>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0016567 ! \"protein ubiquitination\"))>,\n",
       " <BEL p(HGNC:5427 ! IFNA6)>,\n",
       " <BEL a(TEXT:\"anti-IFNalpha autoantibodies\")>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0006468 ! \"protein phosphorylation\", Tyr, 466))>,\n",
       " <BEL p(HGNC:5422 ! IFNA17)>,\n",
       " <BEL a(TEXT:\"Lupus anti-IFNalpha autoantibodies\")>,\n",
       " <BEL a(TEXT:\"anti-IFNAR1 monoclonal antibodies\")>,\n",
       " <BEL a(TEXT:\"IFNA receptor\")>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0006468 ! \"protein phosphorylation\"))>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0006468 ! \"protein phosphorylation\", Tyr))>,\n",
       " <BEL p(FPLX:IFNA ! IFNA, pmod(go:0006468 ! \"protein phosphorylation\", Tyr))>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 532))>,\n",
       " <BEL p(FPLX:IFNA ! IFNA, pmod(go:0006468 ! \"protein phosphorylation\"))>,\n",
       " <BEL p(HGNC:5432 ! IFNAR1, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 535))>,\n",
       " <BEL p(HGNC:5433 ! IFNAR2, pmod(go:0006468 ! \"protein phosphorylation\"))>,\n",
       " <BEL p(FPLX:IFNAR ! IFNAR, pmod(go:0006468 ! \"protein phosphorylation\", Tyr))>]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ifn = [ \n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.BaseConcept)\n",
    "    if 'ifna' in node.name.lower()  \n",
    "]\n",
    "ifn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<BEL bp(MESH:D000086402 ! SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 isolate Australia/VIC01/2020\")>,\n",
       " <BEL bp(MESH:D045473 ! \"SARS Virus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 binding sites\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 intracellular entry\")>,\n",
       " <BEL a(TEXT:\"Architect SARS-CoV-2 IgG\")>,\n",
       " <BEL a(TEXT:\"LIAISON SARS-CoV-2 S1/S2 IgG\")>,\n",
       " <BEL a(TEXT:SARS-CoV-229)>,\n",
       " <BEL a(TEXT:\"CC-BY-NC-ND 4.0 International license.91 Results92 SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"CC-BY-ND 4.0 International license.25 Summary 26 SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N gene\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral RNA\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-S)>,\n",
       " <BEL a(TEXT:VSV-eGFP-SARS-CoV-2)>,\n",
       " <BEL a(TEXT:ChAd-SARS-CoV-2-S)>,\n",
       " <BEL a(TEXT:\"ChAdOx1 encoded SARS-CoV-2 antigens\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 swab\")>,\n",
       " <BEL a(TEXT:\"Engineered SARS-CoV-2 receptor binding domain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 antigens\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA\")>,\n",
       " <BEL a(TEXT:\"investigation.ParameterValuesSARS-COV-2 ProteomesBinding Threshold HLA allele\")>,\n",
       " <BEL a(TEXT:\"Therapy.The SARS-CoV-2 virus outbreak\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 IgG antibody\")>,\n",
       " <BEL a(TEXT:INTRODUCTIONSARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 neutralizing antibodies\")>,\n",
       " <BEL a(TEXT:IntroductionSARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S gene\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 genes\")>,\n",
       " <BEL a(TEXT:\"LightMix(R) Modular SARSCoV-2 RT-PCR kit\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV PL2 pro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 PR-1\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2PR-1)>,\n",
       " <BEL a(TEXT:\"field isolate SARS-CoV-2 PR-1\")>,\n",
       " <BEL a(TEXT:\"N SARS-CoV-2 LLPS\")>,\n",
       " <BEL a(TEXT:SARS-CoV-226)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 35\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2.8\n",
       " bioRxiv preprint doi\")>,\n",
       " <BEL a(TEXT:\"anti-SARS-CoV-2 neutralizing antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-Cov-2 mRNA\")>,\n",
       " <BEL a(TEXT:\"QIAstat-Dx Respiratory SARS-CoV-2 panel\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 specific neutralizing antibodies\")>,\n",
       " <BEL a(TEXT:\"RdRp_SARSr-P2\")>,\n",
       " <BEL a(TEXT:\"Results108 Purified SARS-CoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS131 CoV-2\")>,\n",
       " <BEL a(TEXT:SARS-CoV-1)>,\n",
       " <BEL a(TEXT:\"SARS CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS nsp3\")>,\n",
       " <BEL a(TEXT:\"SARS- Co-V 2\")>,\n",
       " <BEL a(TEXT:\"SARS-163 CoV-2\")>,\n",
       " <BEL a(TEXT:SARS-176CoV-2)>,\n",
       " <BEL a(TEXT:SARS-2-S)>,\n",
       " <BEL a(TEXT:SARS-2-S-HDL)>,\n",
       " <BEL a(TEXT:\"SARS-280 CoV-2\")>,\n",
       " <BEL a(TEXT:SARS-COV-19)>,\n",
       " <BEL a(TEXT:\"SARS-COV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV 9b protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV Mpro mutation\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV N protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV Nsp15\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 homolog [\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV PLpro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV accessory protein 8b\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nucleocapsid protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV open reading frame (ORF) 8b protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV papain-like protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-1 ORF3b\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-1 S\")>,\n",
       " <BEL a(TEXT:SARS-CoV-19)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 (\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 -a coronavirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 154\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 191\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 424\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 432 miRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 AP66\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Assay\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 CD4+\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 GZ69\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 IgG antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 IgG proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 MIRA55\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N geneGAACTTCTCCTGCTAGAATGGCDAMP-crRNA\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 UAAUUUCUACUAAGUGUAGAUCCCUACUGCUGCCUGGAGUUGN geneAASynthetic SARS-CoV-2 N DNA fragmentGCCATTCGAGCAGGAGAATTTCCCCTACTGCTGCCAGGAGTTGA ATTTCTTGAATTACCGCGACTACGTGnCOV_N1 RT-qPCR forward primer 1 GACCCCAAAATCAGCGAAATnCOV_N1 RT-qPCR reverse primer 1 TCTGGTTACTGCCAGTTGAATCTGnCOV_N1 RT-qPCR probe 1FAM-ACCCCGCATTACGTTTGGTGGACC-BHQ1AIOD-CRISPR\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF3\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF3a\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Orf9b\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RBM\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA [8]\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA elements\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RT-PCR\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RT-ddPCR\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RUO kit\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-glycoprotein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S1/S2 site\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 VSV-GFP pseudovirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Vaccine Epitopes\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 [\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 a (+) single-stranded RNA virus fasten similar receptor\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 coronavirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 encoded 443 miRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 encoded miRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 exploits receptor\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 icELISA\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection.Main\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection.SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 like SARS-CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 merits\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 miRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 neutralizing NbsTo\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 neutralizing antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pandemic\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pathogen\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 peptide ORF1ab 4211:4252\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 peptide ORF1ab 882 4211:4252\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 peptides\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pneumonia\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 protein NSP12\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudovirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 reactive IgG antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 receptors\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 recombinant antigensThe in-planta\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 specific antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 specific protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 specific siRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 surface proteins Spike (S)\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 transmission\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 transmission.Effects\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral load\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virulence factors\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virus 19\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virus spike protein\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2.COVID-19)>,\n",
       " <BEL a(TEXT:SARS-CoV-22)>,\n",
       " <BEL a(TEXT:SARS-CoV-2infection33)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-33 2\")>,\n",
       " <BEL a(TEXT:SARS-CoV-4)>,\n",
       " <BEL a(TEXT:SARS-CoV-ACE2)>,\n",
       " <BEL a(TEXT:SARS-CoV-ST)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-g/mL 1 g/mL 2.5 g/mL 5 g/mL 10 g/mL\")>,\n",
       " <BEL a(TEXT:SARS-CoV-cpsR-19)>,\n",
       " <BEL a(TEXT:\"SARS-CoV157 2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 Vaccine Epitopes\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV368 2\")>,\n",
       " <BEL a(TEXT:\"SARS-N mAbs\")>,\n",
       " <BEL a(TEXT:\"SARS-S protein\")>,\n",
       " <BEL a(TEXT:SARS-S)>,\n",
       " <BEL a(TEXT:\"SARS104 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS157 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS164 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS175 CoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS203 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS221 CoV-2 S proteins\")>,\n",
       " <BEL a(TEXT:\"SARS257\t CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS281 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS288 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS61 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS9 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 pseudovirus\")>,\n",
       " <BEL a(TEXT:SARSr-Rp-BatCoV)>,\n",
       " <BEL a(TEXT:\"SARS‐CoV‐2)\")>,\n",
       " <BEL a(TEXT:\"Spike protein of SARS-CoV-1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV 2 ORF1ab\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2S proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N, S, and ORF1ab genes\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleoprotein gene\")>,\n",
       " <BEL a(TEXT:VSV-SARS-CoV-2-S)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 WK-521 RNA\")>,\n",
       " <BEL a(TEXT:\"Vitros Immunodiagnostic Anti-SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 genomic RNA fragment 21571-25392\")>,\n",
       " <BEL a(TEXT:Zanamivir.SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"aCC-BY-NC-ND 4.0 International license.43 Author Summary 44 SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"anti-SARS-CoV-2 antibodies\")>,\n",
       " <BEL a(TEXT:\"anti-SARS-CoV-2 non-neutralizing antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleocapsid\")>,\n",
       " <BEL a(TEXT:\"coronavirus SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:died.SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection40\")>,\n",
       " <BEL a(TEXT:\"human-to-human transmission ofSARS-CoV-2\")>,\n",
       " <BEL a(TEXT:immunocytochemistry.SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 receptor binding domains\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleocapsid protein\")>,\n",
       " <BEL a(TEXT:n-SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"new SARS-CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp16 2'-O-MTase\")>,\n",
       " <BEL a(TEXT:\"permission.Figure 2 SARS-CoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"permission.Figure S2.Fig.S2 SARS-Cov2 RBD-Fc Vacc\")>,\n",
       " <BEL a(TEXT:permission.SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 susceptibility\")>,\n",
       " <BEL a(TEXT:SARS-CoV29)>,\n",
       " <BEL a(CHEBI:147285 ! \"EC 3.4.22.69 (SARS coronavirus main proteinase) inhibitor\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virus titer\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N gene fragment\")>,\n",
       " <BEL a(TEXT:rSARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"the.SARS CoV-1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 5' UTR\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 StrepORF9c\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mpro\")>,\n",
       " <BEL a(TEXT:anti-SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 network\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Inactivation Coupon\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 involvement.A\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 proteases\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA clearance\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 tropism\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 633\")>,\n",
       " <BEL bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARs Cov2\")>,\n",
       " <BEL a(TEXT:SARS-2-Spp)>,\n",
       " <BEL a(TEXT:\"pSARS141 CoV-2\")>,\n",
       " <BEL a(TEXT:genesSARS-CoV-2)>,\n",
       " <BEL p(HGNC:10537 ! SARS1)>,\n",
       " <BEL a(TEXT:\"BrSARS-N protein\")>,\n",
       " <BEL a(TEXT:\"BrSARS-N proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-N protein\")>,\n",
       " <BEL a(TEXT:\"shuttle SARS-CoV N protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Spikes\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S trimer\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 disease\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV construct\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV S\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry (7)\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry.ACE2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry.There\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry22\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection1,2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudoparticles\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudovirus particle 182\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV60 2\")>,\n",
       " <BEL a(TEXT:SARS2pp)>,\n",
       " <BEL a(TEXT:\"SARS516 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS535 CoV-2 viruses\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 associated hypercoagulation.To\")>,\n",
       " <BEL a(TEXT:\"anti-SARS-CoV-2 antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 RNA\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2pp)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 [152]\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 antigens 201\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 open reading frame (ORF) proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 mRNA\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 model mRNAsThe sequence\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 5' UTR\")>,\n",
       " <BEL p(HGNC:17697 ! SARS2)>,\n",
       " <BEL a(TEXT:SARS-CoV-2-RBD)>,\n",
       " <BEL a(TEXT:\"IgG anti-S1-SARS-CoV-2 antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA yield\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S pseudovirus\")>,\n",
       " <BEL a(TEXT:\"SARS-2-S activating TTSPs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 enhancement\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV192 2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Related Proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 related proteins\")>,\n",
       " <BEL a(TEXT:HIV-SARS-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 transcriptome\")>,\n",
       " <BEL a(TEXT:\"DEHGs COVID-19  SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"DEHGs COVID-19 SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ssRNA\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 S.Overall\")>,\n",
       " <BEL bp(MESH:C000706209 ! \"3C-like proteinase, SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudovirus neutralizing antibody\")>,\n",
       " <BEL a(TEXT:pSARS-PLpro)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 structural protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 16\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-reactive IgG antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 replication.INTRODUCTIONThe coronavirus disease 2019\")>,\n",
       " <BEL a(TEXT:gRNAsSARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"anti-SARS-CoV-2 IgG antibodies\")>,\n",
       " <BEL a(TEXT:SARS-CoV1)>,\n",
       " <BEL a(TEXT:\"SARS-CoV685 2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 perseveres\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV 2\")>,\n",
       " <BEL a(TEXT:\"SARS–CoV–2)\")>,\n",
       " <BEL a(TEXT:\"receptor-binding site of274 SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 sequences\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudoviruses\")>,\n",
       " <BEL a(TEXT:SARS-CoV-212)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 variants\")>,\n",
       " <BEL a(TEXT:\"virus (MLV) pseudotyped SARS-CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 epitopes\")>,\n",
       " <BEL a(TEXT:SARS-CoV-1-Spike-RBD)>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 N protein (50 muM)\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Nsp15\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2S protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 antigens-spike (S) protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2Receptor Binding Domain\")>,\n",
       " <BEL a(TEXT:\"Recombinant SARS-CoV-2 RBM\")>,\n",
       " <BEL a(TEXT:SARS-Co-2)>,\n",
       " <BEL a(TEXT:SARS-CoC-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV (S) protein\")>,\n",
       " <BEL a(TEXT:\"c SARS-CoV-2 S\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-spike protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 peripheral Spike-Fc protein\")>,\n",
       " <BEL a(TEXT:\"SARS24\t CoV-2 S trimer\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV S 171\t protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RBD-419 S\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RBD420 S\")>,\n",
       " <BEL a(TEXT:SARS-CoV-RBD)>,\n",
       " <BEL a(TEXT:anti-SARS-CoV)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 5\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-spreading)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S 473 trimer\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 PLPro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-RBD\")>,\n",
       " <BEL a(TEXT:\"mutant SARS-CoV-2 RBDs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 15-mer epitopes\")>,\n",
       " <BEL a(TEXT:\"full-length SARS-CoV-2 ectodomain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CLpro\")>,\n",
       " <BEL a(TEXT:\"SARS-COV-2 reference genome.Three binding site hsa-miR-770-5p\")>,\n",
       " <BEL a(TEXT:SARS-RBA)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N-NTD\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP12\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP12NSP7\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Nsp12\")>,\n",
       " <BEL a(TEXT:\"SARS-\n",
       " bioRxiv preprint doi\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-N)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleocapsid (N)protein\")>,\n",
       " <BEL a(TEXT:\"SARS- CoV-2\")>,\n",
       " <BEL a(TEXT:SARS-2-S1)>,\n",
       " <BEL a(TEXT:SARS-2-S2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12 917 polymerase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12 polymerase 637\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12 polymerase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp16\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV specific antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RBDs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 66\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 82\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S 72 protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-614G\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 SStrep\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp12\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp16\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-SP)>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 PLpro\")>,\n",
       " <BEL a(TEXT:SARS-Cov2-E)>,\n",
       " <BEL a(TEXT:SARS-Cov2-E-CDC)>,\n",
       " <BEL a(TEXT:\"recombinant SARS-CoV-2 S\")>,\n",
       " <BEL a(TEXT:\"sars-cov nsp12 polymerase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 polymerase complex\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp14\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mac1 domain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 mAbs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP14\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RdRp-complex\")>,\n",
       " <BEL a(TEXT:\"[SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2's Spike protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Nsp14\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 protein sequence\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 protein sequences\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mac1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP15\")>,\n",
       " <BEL a(TEXT:\"N SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:SARS-nCoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV ORF7a\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-E)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-Proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 463 S trimer\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2.27)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 receptor binding domain\")>,\n",
       " <BEL a(TEXT:\"SARS CoV2\")>,\n",
       " <BEL a(TEXT:\"SARS-COV-RBD proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV S structures\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 41\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 42\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Pseudotyped Virus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RDB\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S Glycoproteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S1-RBD protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 gains\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 spikes\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2-S-RBD 181\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-S-RBD)>,\n",
       " <BEL a(TEXT:SARS-CoV-2-spike-glycoprotein)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2547 S\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2RBD)>,\n",
       " <BEL a(TEXT:SARS-CoV-S)>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 S\")>,\n",
       " <BEL a(TEXT:SARS-CoV2-SP)>,\n",
       " <BEL a(TEXT:\"SARS-CoV24\t 2\")>,\n",
       " <BEL a(TEXT:\"SARS100 CoV-2-S-RBD protein\")>,\n",
       " <BEL a(TEXT:\"SARS103 CoV\")>,\n",
       " <BEL a(TEXT:SARSCoV-1)>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 RBD-Fc\")>,\n",
       " <BEL a(TEXT:\"Sars-Cov-2 Receptor Binding Domain\")>,\n",
       " <BEL a(TEXT:\"recombinant SARS-CoV-2 RBD219-N1C1 protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2's S proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 PLpro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 receptor-binding domain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-215\n",
       " bioRxiv preprint doi\")>,\n",
       " <BEL a(TEXT:SARS-CoV2-RBD)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Nucleocapsid Protein\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2nsp10)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA sequences\")>,\n",
       " <BEL a(TEXT:\"Table 2 Multiple SARS-CoV-2 bait proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 special proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 sequence\")>,\n",
       " <BEL a(TEXT:\"glycoproteins S of SARS-CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 variant\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 protein Nsp6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV NP-1\")>,\n",
       " <BEL a(TEXT:ACE2-to-SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV Antibody\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry associated gene\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2; high-titer-specific antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV Mac2-Mac3-DPUP\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 IgM\")>,\n",
       " <BEL a(TEXT:\"recombinant SARS-CoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"recombinant SARSCoV-2 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mpro.48-60,62-68,74\")>,\n",
       " <BEL a(TEXT:SARS-CoV2-cell)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP6 protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Nsp6 protein\")>,\n",
       " <BEL a(TEXT:sars3a)>,\n",
       " <BEL a(TEXT:sars9b)>,\n",
       " <BEL a(TEXT:\"SARS-CoV S 120 protein 53\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV S 121 protein53\")>,\n",
       " <BEL a(TEXT:\"WT SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS842 CoV-2 S trimer\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 open reading frame (ORF) 3b, ORF 6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S RNA\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RNA.44\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 5' UTR mScarlet reporter\")>,\n",
       " <BEL a(TEXT:\"full-length SARS-CoV-2 mRNAs\")>,\n",
       " <BEL a(TEXT:\"full-length SARS-CoV-2 RNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV cellular receptor\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV PL2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 83\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 84\")>,\n",
       " <BEL a(TEXT:\"COVID-19pos patients.SARS-CoV2\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 antigens\")>,\n",
       " <BEL a(TEXT:\"SARS-114 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV 3b protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV encoding proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV unique domain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 COVID-19 4 5\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 MNA vaccines\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 (1*106\")>,\n",
       " <BEL a(TEXT:\"SARS-Cov-2 antigens\")>,\n",
       " <BEL a(TEXT:\"SARS26 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS435 CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-specific IgG1\")>,\n",
       " <BEL a(TEXT:\"SARS-coronavirus nsp12\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-3CLprotease)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 entry receptors\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 receptor\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 159\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 strain 2019-nCoV/USA-WA1/2020\")>,\n",
       " <BEL a(TEXT:\"COVID-19 virus SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2-N-nsp12 467\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pneumonia 394\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2-GFP)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 652\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV63 2\")>,\n",
       " <BEL a(TEXT:\"SARS62 CoV-2 contact transmission (8)\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV 3CLpro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Infection.The pandemic\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 WK-521\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2WK-521)>,\n",
       " <BEL a(TEXT:\"S-mediated 293T/SARS-CoV-2/EGFP\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 211 S\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pneumonia [\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2.6)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 onset outbreak\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudovirus entry\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 5' UTR-luciferase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF3b variant\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CLpro protease10\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 growth\")>,\n",
       " <BEL a(TEXT:\"PLproof SARS-CoV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleoprotein (NCP) transcript 444\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 PsV\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nucleocapsid genes\")>,\n",
       " <BEL a(TEXT:\"SARS HRC\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV ORF6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV accessory protein ORF6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV field isolate Frankfurt-1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV protein 16\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-1 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 (2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 -ACE2 complex2,33\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 122 transmission\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 CIP\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mpro activity.Lopinavir\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV Mpro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mpro inhibitor cinanserin\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 NSP6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF3b protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF6 protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF6 proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF9b\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 ORF9c\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 PLP\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Receptor Binding Domain\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S murine polyclonal antibodies\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 accessory proteins\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 cases\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 cause\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 gene\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nonstructural protein 1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pathobiology.ACE2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 proteins ORF6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 replicon\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral infection17\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2, b) control proin-flammatory cytokines\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-217\n",
       " bioRxiv preprint doi\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2nsp1)>,\n",
       " <BEL a(TEXT:\"SARS486 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS78 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 virus isolate SARS-CoV-2-Jan\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 mutants\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Entry 483\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12 RNA polymerases\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV nsp12/7/8 RNA polymerase complex\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp12/7/8 complex\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp12/7/8\")>,\n",
       " <BEL a(TEXT:\"VSV260 eGFP-SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:rAOaV-1-SARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CLPro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2.131 2\")>,\n",
       " <BEL a(TEXT:ppSARS-CoV-2)>,\n",
       " <BEL a(TEXT:\"SARS514 CoV-2\")>,\n",
       " <BEL a(TEXT:ppSARS-2)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 pseudovirions64,65\")>,\n",
       " <BEL a(TEXT:SARS-CoV-215)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CL-Protease\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CLprotease\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral 133\t infection17\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 3CL-protease\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection20\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 IgM False Positives\")>,\n",
       " <BEL a(TEXT:\"SARS142 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Coronavirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Entry\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 252\")>,\n",
       " <BEL a(TEXT:\"SARS-141 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 Virus <em>\")>,\n",
       " <BEL a(TEXT:\"SARS-320 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2  71\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 Mpro\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 load\")>,\n",
       " <BEL a(TEXT:SARS-Cov1)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp3.The\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Replication\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 RdRps\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV MTase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV N7-MTase\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 VeroE6\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Mpro We\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection Liu et al. 2020b\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 replication [2]\")>,\n",
       " <BEL a(TEXT:SARS-CoV-2.94)>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 [124]\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 Spike-ACE2 Protein-Protein Interaction-a Mechanism\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 VLP-uptake\")>,\n",
       " <BEL a(TEXT:\"SARS192 CoV-2-Nluc\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 replication\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 polymerase\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 viral load pneumonia\")>,\n",
       " <BEL a(TEXT:\"pSARS-CoV-2 142\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S-Mediated Virus Entry\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 recognition\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-21\n",
       " bioRxiv preprint doi\")>,\n",
       " <BEL a(TEXT:\"VSV-SARS-CoV414\t 2\")>,\n",
       " <BEL a(TEXT:\"replication995 competent VSV-SARS-CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 messenger RNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 model mRNAs\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 protein\")>,\n",
       " <BEL a(TEXT:\"Sars-Cov-2 mental health sequelae\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 transmission 150 (2, 3\")>,\n",
       " <BEL a(TEXT:\"SARS S protein\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virus [\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 luciferase reporter\")>,\n",
       " <BEL a(TEXT:\"SARSr-CoVs WIV1\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 virus attachment\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 infection.bioRxiv 2020.07.02.183764\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 S+ pseudovirus\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 460 pneumonia\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV2 pneumonia\")>,\n",
       " <BEL a(TEXT:\"SARSCoV-2 pneumonia\")>,\n",
       " <BEL a(TEXT:\"SARS-S pseudotyped virus (\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 replication 378 (A) Small\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nanoLuc\")>,\n",
       " <BEL a(TEXT:\"SARS413 CoV-2\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2-S protein (4 mg/mL\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV replication\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 N gene:5'- GUCUAGAGGACAGAAUUUUUCAACGGGUGUGCCAAUGGCCACUUUCCAGGUGG CAAAGCCCGUUGAGCUUCUCAAAUCUGAGAAGUGGCACCGAAGAACGCUGAAGC GCUG-3'\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 DeltaSL1 5' UTR mScarlet reporter\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV genomeat S protein-\")>,\n",
       " <BEL a(TEXT:\"SARS-CoV-2 nsp16-nsp10 2'-O-methyltransferase complex\")>,\n",
       " <BEL p(HGNC:10537 ! SARS1, pmod(go:0006468 ! \"protein phosphorylation\"))>]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get all proteins named angiotensin, turns out there's only 1\n",
    "sars2 = [ \n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.BaseConcept)\n",
    "    if 'sars' in node.name.lower()  \n",
    "]\n",
    "sars2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<BEL a(CHEBI:48433 ! angiotensin)>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get all proteins named angiotensin, turns out there's only 1\n",
    "angiotensin = [ \n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.BaseConcept)\n",
    "    if node.name.lower() == 'angiotensin'\n",
    "][0]\n",
    "angiotensin"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Find relations between ACE2 and Interferon"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[<BEL bp(MESH:D000086402 ! SARS-CoV-2)>,\n",
       "  <BEL bp(GO:\"GO:0006897\" ! endocytosis)>,\n",
       "  {'relation': 'decreases',\n",
       "   'annotations': {'stmt_hash': {7661834816203743: True},\n",
       "    'uuid': {'67df9a85-3625-4448-95db-a3bef5bc6c7a': True},\n",
       "    'belief': {0.65: True},\n",
       "    'source_hash': {2282832247182912510: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32429572'},\n",
       "   'evidence': 'A diagram illustrating the life cycle of SARS-COV2 and potential lipid modifying drugs that can used as broad-spectrum antiviral drugs to inhibit viral entry, membrane fusion, or endocytosis as well as inhibition of fatty acid and cholesterol synthesis.A diagram illustrating the life cycle of SARS-COV2 and potential lipid modifying drugs that can used as broad-spectrum antiviral drugs to inhibit viral entry, membrane fusion, or endocytosis as well as inhibition of fatty acid and cholesterol synthesis.Contributions: M.A.-F., T.A.T., M.G.Q., A.H., J.A., and F.A.-M.'}],\n",
       " [<BEL bp(MESH:D045473 ! \"SARS Virus\")>,\n",
       "  <BEL bp(GO:\"GO:0006897\" ! endocytosis)>,\n",
       "  {'relation': 'decreases',\n",
       "   'annotations': {'stmt_hash': {-24067367279423937: True},\n",
       "    'uuid': {'61b022f1-466f-4f09-ad97-75336b5450da': True},\n",
       "    'belief': {0.65: True},\n",
       "    'source_hash': {-3057465467105383500: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32603829'},\n",
       "   'evidence': 'T20 (enfuvirtide) which is approved by USFDA against AIDS may significantly interact with the HR2 region of the SARS-CoV by inhibiting endocytosis into the host cell.'}],\n",
       " [<BEL a(TEXT:\"SARS-CoV-2 exploits receptor\")>,\n",
       "  <BEL bp(GO:\"GO:0006897\" ! endocytosis)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-14222858553920401: True},\n",
       "    'uuid': {'a73aaf6b-d223-44a7-9db9-a6dcf831ec68': True},\n",
       "    'belief': {0.65: True},\n",
       "    'source_hash': {5470633076183941300: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '33713620'},\n",
       "   'evidence': 'Further investigation revealed that SARS-CoV-2 exploits receptor mediated endocytosis through interaction between its spike with sACE2 or sACE2-vasopressin via AT1 or AVPR1B, respectively.'}]]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def get_statements_by_subject_and_object( subject, obj, emmaa_covid_graph):\n",
    "    statements = []\n",
    "    for u, v, data in emmaa_covid_graph.edges(subject, data=True):\n",
    "        if v in obj:\n",
    "            #print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "            if 'evidence' in data:\n",
    "                statements.append([u,v,data])\n",
    "    return statements\n",
    "sars2_endocytosis = get_statements_by_subject_and_object( sars2, endocytosis, emmaa_covid_graph )\n",
    "sars2_endocytosis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "p(HGNC:13557 ! ACE2, var(\"p.?\")) decreases p(UP:P0DTC2 ! P0DTC2)\n"
     ]
    }
   ],
   "source": [
    "print(emmaa_covid_graph.edge_to_bel(u, v, data))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Find all INDRA statements associated with a publication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_statements_by_pubmed( pubmed, emmaa_covid_graph=emmaa_covid_graph ):\n",
    "    statements = []\n",
    "    for u, v, data in emmaa_covid_graph.edges(data=True):\n",
    "        if  ('citation' in data) and (\n",
    "             data['citation']['namespace'] == 'pubmed') and (\n",
    "             data['citation']['identifier'] ==  pubmed):\n",
    "                #print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "                if 'evidence' in data:\n",
    "                    #print(data['citation']['namespace'], data['citation']['identifier'])\n",
    "                    #print(data['evidence'])\n",
    "                    statements.append([u,v,data])\n",
    "    return statements\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[<BEL a(TEXT:\"SARS-CoV-2 exploits receptor\")>,\n",
       "  <BEL bp(GO:\"GO:0006897\" ! endocytosis)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-14222858553920401: True},\n",
       "    'uuid': {'a73aaf6b-d223-44a7-9db9-a6dcf831ec68': True},\n",
       "    'belief': {0.65: True},\n",
       "    'source_hash': {5470633076183941300: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '33713620'},\n",
       "   'evidence': 'Further investigation revealed that SARS-CoV-2 exploits receptor mediated endocytosis through interaction between its spike with sACE2 or sACE2-vasopressin via AT1 or AVPR1B, respectively.'}]]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sACE2 = get_statements_by_pubmed(\"33713620\", emmaa_covid_graph=emmaa_covid_graph)\n",
    "sACE2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clathrin_mediated_endocytosis_pubmed1 = \"32428379\"\n",
    "clathrin_mediated_endocytosis_statements1 = get_statements_by_pubmed(clathrin_mediated_endocytosis_pubmed1)\n",
    "clathrin_mediated_endocytosis_statements1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clathrin_mediated_endocytosis_pubmed2 = \"33476648\"\n",
    "clathrin_mediated_endocytosis_statements2 = get_statements_by_pubmed(clathrin_mediated_endocytosis_pubmed2)\n",
    "clathrin_mediated_endocytosis_statements2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<BEL bp(GO:\"GO:0006897\" ! endocytosis)>]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "endocytosis = [ \n",
    "    node\n",
    "    for node in emmaa_covid_graph\n",
    "    if isinstance(node, pybel.dsl.BaseConcept)\n",
    "    if node.name.lower() == 'endocytosis'\n",
    "]\n",
    "endocytosis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "complex(sACE2,SARS-CoV-2) increases rxn(reactant(extracellular sARS-CoV-2), product( intracellular SARS-CoV-2))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[<BEL p(FPLX:Protease ! Protease)>,\n",
       "  <BEL p(UP:P0DTC2 ! P0DTC2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-19306595894560024: True},\n",
       "    'uuid': {'f9c895ff-f0d0-4507-b3d7-f3e420b93dfb': True},\n",
       "    'belief': {0.999874835975: True},\n",
       "    'source_hash': {6012065514567406677: True},\n",
       "    'source_api': {'sparser': True},\n",
       "    'hedgings': {'can': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'The main host protease that mediates S protein activation on primary target cells and initial viral entry is the type II transmembrane serine protease TMPRSS2 (Glowacka et al., 2011, Hoffmann et al., 2020, Iwata-Yoshikawa et al., 2019, Matsuyama et al., 2010, Shulla et al., 2011, Walls et al., 2020).'}],\n",
       " [<BEL p(FPLX:Protease ! Protease)>,\n",
       "  <BEL p(UP:P0DTC2 ! P0DTC2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-19306595894560024: True},\n",
       "    'uuid': {'f9c895ff-f0d0-4507-b3d7-f3e420b93dfb': True},\n",
       "    'belief': {0.999874835975: True},\n",
       "    'source_hash': {6012065514567406677: True},\n",
       "    'source_api': {'sparser': True},\n",
       "    'hedgings': {'can': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Importantly, rapidly evolving literature has identified that SARS-CoV-2-S might have a furin cleavage site, leading to a broader set of host proteases that could mediate S protein activation (Bugge et al., 2009, Coutard et al., 2020, Walls et al., 2020).'}],\n",
       " [<BEL p(FPLX:Protease ! Protease)>,\n",
       "  <BEL p(UP:P0DTC2 ! P0DTC2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-19306595894560024: True},\n",
       "    'uuid': {'f9c895ff-f0d0-4507-b3d7-f3e420b93dfb': True},\n",
       "    'belief': {0.999874835975: True},\n",
       "    'source_hash': {6012065514567406677: True},\n",
       "    'source_api': {'sparser': True},\n",
       "    'hedgings': {'can': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Whether ACE2 and TMPRSS2 are needed on the same cell or soluble proteases can activate SARS-CoV-2 S protein to invade ACE2 single-positive cells is an area of active inquiry (Coutard et al., 2020, Letko et al., 2020).'}],\n",
       " [<BEL p(HGNC:13557 ! ACE2)>,\n",
       "  <BEL p(FPLX:IFNA ! IFNA)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {17431060127177171: True},\n",
       "    'uuid': {'4d1944df-4e30-4ff1-b948-0f10967e54ef': True},\n",
       "    'belief': {0.86: True},\n",
       "    'source_hash': {1241874760967516269: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Across several studies of human and NHP tissues, we found ISGs upregulated in ACE2-expressing cells.Strikingly, by treating primary human upper airway basal cells with distinct types of inflammatory cytokines, we demonstrate that IFN-α drives ACE2 expression.'}],\n",
       " [<BEL bp(MESH:D007251 ! \"Influenza, Human\")>,\n",
       "  <BEL p(HGNC:13557 ! ACE2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {14403465990584053: True},\n",
       "    'uuid': {'9af31335-f61a-4436-98e1-da800e9a847b': True},\n",
       "    'belief': {0.65: True},\n",
       "    'source_hash': {1351938624458273461: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Human influenza infection also induces broader expression of ACE2 in upper airway epithelial cells and is corroborated by publicly available databases.'}],\n",
       " [<BEL p(HGNC:8568 ! FURIN)>,\n",
       "  <BEL p(FPLX:Protease ! Protease)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {35990399066667168: True},\n",
       "    'uuid': {'eacdc1f6-21f7-416f-8fd1-d4dea87cfe3d': True},\n",
       "    'belief': {0.991866: True},\n",
       "    'source_hash': {420691777208417172: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Conversely, we found that IFN-γ, relative to IFN-α2, induced potent upregulation of GBP5, a GTPase-like protein thought to act as a viral restriction factor through inhibiting furin-mediated protease activity, which could limit viral processing from infected cells, whereas IFN-α2 more robustly induced IFITM1 (Figure S3G–S3K) (Braun and Sauter, 2019).'}],\n",
       " [<BEL a(CHEBI:48432 ! \"angiotensin II\")>,\n",
       "  <BEL bp(MESH:D055371 ! \"Acute Lung Injury\")>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {24607568772017343: True},\n",
       "    'uuid': {'c1c2813e-94f9-4037-bbdb-8e144070fc33': True},\n",
       "    'belief': {0.923: True},\n",
       "    'source_hash': {-326329275939217676: True},\n",
       "    'source_api': {'reach': True}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Angiotensin II then acts to drive acute lung injury through various mechanisms, including increased vascular permeability (Imai et al., 2005).'}],\n",
       " [<BEL a(CHEBI:30220 ! helion)>,\n",
       "  <BEL p(HGNC:13557 ! ACE2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-1665838370328589: True},\n",
       "    'uuid': {'edcc3361-87fe-4dd5-8c9b-edb9b1388ceb': True},\n",
       "    'belief': {0.86: True},\n",
       "    'source_hash': {944359984491862484: True},\n",
       "    'source_api': {'eidos': True},\n",
       "    'hedgings': {'whether': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Next , we asked whether a 12 h stimulation with IFN-alpha would upregulate Ace2 in vivo .'}],\n",
       " [<BEL a(CHEBI:30220 ! helion)>,\n",
       "  <BEL p(HGNC:13557 ! ACE2)>,\n",
       "  {'relation': 'increases',\n",
       "   'annotations': {'stmt_hash': {-1665838370328589: True},\n",
       "    'uuid': {'edcc3361-87fe-4dd5-8c9b-edb9b1388ceb': True},\n",
       "    'belief': {0.86: True},\n",
       "    'source_hash': {944359984491862484: True},\n",
       "    'source_api': {'eidos': True},\n",
       "    'hedgings': {'whether': True}},\n",
       "   'target': {'modifier': 'Activity',\n",
       "    'effect': {'namespace': 'go',\n",
       "     'name': 'molecular function',\n",
       "     'identifier': '0003674'}},\n",
       "   'citation': {'namespace': 'pubmed', 'identifier': '32413319'},\n",
       "   'evidence': 'Next , we asked whether a 12 h stimulation with IFN-a would upregulate Ace2 in vivo .'}]]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ace2_isg = \"32413319\"\n",
    "ace2_isg_statements= get_statements_by_pubmed(ace2_isg, emmaa_covid_graph)\n",
    "ace2_isg_statements"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "Pro-inflammatory cytokines released by effector T cells , mostly Th1 and Th17 , are key players in BBB alteration , via the increase of adhesion molecules on endothelial cells that enables lymphocytes adhesion and infiltration into the CNS [ 102 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "The septic stress not only disrupts crucial cellular and tissue functions, but also exacerbates endoplasmic reticulum stress and leads to the release of damage-associated molecular pattern (DAMP) molecules and inflammatory mediators, which cause leukocyte and vascular endothelial cell (EC) activation, increased capillary permeability, neutrophil infiltration, and tissue injury12,13.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:132153 ! hyaluronate)\n",
      "Altered flow dynamics cause further endothelial cell damage and glycocalyx shedding [ 47 ] Inflammatory conditions with high concentrations of extracellular histones and cytokines damage the endothelial glycocalyx accompanied by its degradation shedding of syndecan-1 , heparan sulfate , and hyaluronan [ 47 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:138157 ! \"chemokine ligand 10\")\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:138157 ! \"chemokine ligand 10\")\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:140399 ! \"specialised pro-resolving mediator\")\n",
      "These SPMs are synthesized by macrophages , neutrophiles and partially by endothelial cells and are crucial for ending neutrophile influx , macrophage switching , and clearance of apoptotic cells thereby actively contribute to resolution of inflammation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "IL-1 ( endogenous pyrogen ) induces fever and hypermetabolism , stimulates fibroblasts and endothelial cells to produce GM-CSF and PGE 2 , enhances PGE2 synthesis in the hypothalamus , increases skeletal muscle proteolysis , has a priming effect on macrophages , is important for maintaining the gut-mucosal barrier and turns on the liver synthesis of acute phase proteins ( for review see [ 53 , 54 ] ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "Alternatively , prostaglandin E2 ( PGE2 ) and other prostaglandins may be produced by the endothelial cells which , in turn , induce a neurotransmitter-like substance , such as cAMP , that acts to raise the set-point [ 25 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds.55 Prostacyclin binds to its receptor ( a G-protein coupled receptor ) found on the surface of vascular smooth muscle and platelets , activates cyclic adenosine monophosphate ( cAMP ) , and results in the most potent endogenous inhibitor of platelet aggregation , vascular smooth muscle relaxation and vasodilation of the pulmonary arteries .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced downstream of cyclooxygenase by prostacyclin synthase in EC and, like NO, is also a potent vasodilator and inhibitor of platelet aggregation acting primarily through cyclic AMP as opposed to NO, which predominantly activates cyclic GMP production (Egan and FitzGerald, 2006) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced downstream of cyclooxygenase by prostacyclin synthase in EC and, like NO, is also a potent vasodilator and inhibitor of platelet aggregation acting primarily through cyclic AMP as opposed to NO, which predominantly activates cyclic GMP production.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds.55 Prostacyclin binds to its receptor ( a G-protein coupled receptor ) found on the surface of vascular smooth muscle and platelets , activates cyclic adenosine monophosphate ( cAMP ) , and results in the most potent endogenous inhibitor of platelet aggregation , vascular smooth muscle relaxation and vasodilation of the pulmonary arteries .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The increase of intracellular pH causes a selective change in the presentation of proper antigens ; Blockage of T-cell response and reduction of proinflammatory cytokine production , including INF-gamma , TNF , IL-1 and IL-6 ; Blockage of Toll-like receptors 7 and 9 , especially in plasmacytoid dendritic cell with inhibition of INF-alpha , which plays an important role in the pathophysiology of SLE ; cGAS-STING signaling inhibition ; Inhibition of phospholipase A2 activity ; Stimulation of nitric oxide production by endothelial cells with antiproliferative effect ; Antithrombotic effect through the inhibition of platelet aggregation in a dose-dependent manner , decreased production of arachidonic acid by activated platelets and action on antiphospholipid antibodies .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Stimulation of nitric oxide production by endothelial cells with antiproliferative effect ; Inhibition of phospholipase A2 activity ; cGAS-STING signaling inhibition ; Blockage of Toll-like receptors 7 and 9 , especially in plasmacytoid dendritic cell with inhibition of INF-alpha , which plays an important role in the pathophysiology of SLE ; Blockage of T-cell response and reduction of pro-inflammatory cytokine production , including INF-gamma , TNF , IL-1 and IL-6 ; Alkalinization of lysosomes and other intracellular acid compartments with interference in phagocytosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Nitric oxide ( NO ) is generated by endothelial cells , platelets , neutrophils , macrophages , and neurons ( Schemann and Schaff , 1995 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Endothelial cells produce NO through the enzyme NO-synthase ( NOS ) , which transforms L-arginine amino acid into citrulline [ 27 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "In a healthy condition , angiotensin-converting enzyme 2 ( ACE2 ) converts angiotensin II to angiotensin 1-7 which stimulates endothelial cells to produce nitric oxide ( NO ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "In 1987 , Palmer et al. remarkably discovered that endothelial cells produce NO , the critical endothelium-derived relaxing factor for proper vascular function [ 70 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Although the antiviral effect of NO was previously described [ 66,86,87 ] , we were surprised that BK inhibited NO production in infected HBMECs because B2R signaling classically upregulates NO production by normal endothelial cells via activation of different isoforms of NO synthase ( eNOS ) [ 88,89,90,91 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "These results suggest that BK , otherwise a classical inducer of NO production by healthy endothelial cells , suppresses NO production in DENV-2-infected HBMECs , ultimately rendering the stressed endothelial cells increasingly susceptible to viral replication .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "It is known that ACE converts angiotensin I ( AngI ) into the pro-inflammatory peptide angiotensin II ( AngII ) , and ACE2 metabolizes AngII to produce angiotensin - ( 1-7 ) ( ( Ang - ( 1-7 ) ) ) and Ang - ( 1-7 ) promotes endothelial cells to produce NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "It is known that ACE converts angiotensin I ( AngI ) into the pro-inflammatory peptide angiotensin II ( AngII ) , and ACE2 metabolizes AngII to produce angiotensin - ( 1-7 ) ( ( Ang - ( 1-7 ) ) ) and Ang - ( 1-7 ) promotes endothelial cells to produce NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Healthy endothelial cells synthesize nitric oxide ( NO ) by conversion of l-arginine to l-citrulline by nitric oxide synthase .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Healthy endothelial cells synthesize nitric oxide ( NO ) by conversion of l-arginine to l-citrulline by nitric oxide synthase .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Pretreatment of lung capillary EC with heparinase III completely abolished the pressure induced increase in Lp and reduced intracellular NO detected by a fluorescent dye, suggesting that cell-surface HS directly participates in mechanotransduction resulting in NO production and increased Lp.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "We conclude that : ( 1 ) mitochondrial H 2 S oxidation plays an important role in the regulation of intrarenal H 2 S , ( 2 ) H 2 S concentration is higher in the renal medulla most likely due to physiological hypoxia in this tissue , ( 3 ) nephroprotective effect of statins demonstrated in many studies may be mediated by elevation of H 2 S. N-Ethyl-L-glutamine enhances the synthesis of nitric oxide by endothelial cells Results of recent studies indicate that enhanced production of glucosamine from glutamine and fructose-6-phosphate in endothelial cells ( EC ) results in reduced synthesis of nitric oxide ( NO ) from L-arginine by NO synthase ( NOS ) , leading to impaired relaxation of blood vessels .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "30 muM tetrandrine inhibited NO production by the endothelial cells through blockade of Ca2 + release-activated Ca2 + channels ( Kwan et al.2001 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "30 lM tetrandrine inhibited NO production by the endothelial cells through blockade of Ca 2 ?\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:18421 ! superoxide)\n",
      "The underlying mechanism of ALI induced by oleic acid is associated with cytokines releases such as TNF - , which stimulates monocytes to produce IL-1.As the core of the cytokine-network , TNF -  and IL-1 play important roles not only in the production of other inflammatory cytokines , but also in the migration and adherence of neutrophils to endothelial cells ( Yoshimi et al ., 2000 ) , which together with endothelial cell injury by cytokines result in the production of superoxide radicals and subsequently injure alveolar epithelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:18421 ! superoxide)\n",
      "The underlying mechanism of ALI induced by oleic acid is associated with cytokines releases such as TNF-alpha , which stimulates monocytes to produce IL-1.As the core of the cytokine-network , TNF-alpha and IL-1 play important roles not only in the production of other inflammatory cytokines , but also in the migration and adherence of neutrophils to endothelial cells ( Yoshimi et al ., 2000 ) , which together with endothelial cell injury by cytokines result in the production of superoxide radicals and subsequently injure alveolar epithelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:27601 ! pteridine)\n",
      "Neopterin or 2-amino-4-hydroxy-6 - ( D-erythro-1 ' ,2 ' ,3 ' - trihydroxypropyl ) - pteridine is produced from guanosine triphosphate via guanosine triphosphate cyclohydrolase I ( GTPCH I ) by activated monocytes , macrophages , dendritic cells , and endothelial cells and to a lesser extent in renal epithelial cells , fibroblasts , and vascular smooth muscle cells upon stimulation mainly by interferon gamma and to a lesser extent by interferon alpha and beta with its release being enhanced by tumor necrosis factor [ 2 , 3 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:28815 ! \"heparan sulfate\")\n",
      "33 , W-4400 MiJnster , Germany 154 EFFECT OF LMW HEPARINS AND RELATED GLYCOSAMINOGLYCANS ON THE SYNTHESIS OF HEPARAN SULFATE BY ENDOTHELIAL CELLS W. Jeske , H. Nader , C.P .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:30512 ! \"silver atom\")\n",
      "Since acute phase proteins are produced by liver cells , whereas vWF : Ag is synthesized by endothelial cells , we postulate that this coagulation factor is highly sensitive in reflecting the inflammatory process in the intestine .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:64007 ! EET)\n",
      "Indeed , CYP2C enzymes are induced by hypoxia , and endothelial cells are a major source of EETs during inflammation and angiogenesis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-alpha and IL-6 are mostly derived from lymphocytes and macrophages , but it was produced at a lower concentration with pre-probiotics ; perhaps this type of cytokines is also produced by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-a and IL-6 are mostly derived from lymphocytes and macrophages , but it was produced at a lower concentration with pre-probiotics ; perhaps this type of cytokines is also produced by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "However , previous literature has demonstrated that endothelial cells are a source of cytokines / chemokines in the lung during influenza infection , and that S1PR1 signaling in the endothelium provides a mechanism for inhibiting cytokine storms and blunting the accumulation of innate inflammatory infiltrates characterized as macrophages / monocytes , neutrophils , and NK cells 3 , 7 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-alpha can also activate endothelial cells to upregulate other cytokines and adhesion molecule expression , thus enhancing vascular permeability , cellular adhesion , and procoagulant activity [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial infection by Chlamydia pneumoniae also activates endothelial cells to produce cytokines and adhesion molecules and become procoagulant [ 21 ] [ 22 ] [ 23 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "It is well-known that DIC starts when endothelial cells and monocytes are stimulated to produce cytokines after a serious damage , with abnormal production of von Willebrand factor and Tissue Factor ( TF ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Several studies indicated that endothelial cells might be the source of the cytokines and involved in lung injury during influenza virus infection [ 29 ] [ 30 ] [ 31 ] [ 32 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Studies also described elevation of dimer D , troponin I , serum IL-6 and C-reactive protein levels.3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial cells are activated by cytokines and viral particles and produce monocyte chemoattractants and adhesion molecules .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial cells indeed produce inflammatory cytokines , including IL-6 , and upregulate adhesion molecules , further promoting leukocytes ' recruitment and capillary leakage [ 37 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Moreover , endothelial cells produce various cytokines and adhesion molecules [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "TNF-alpha can also activate endothelial cells to upregulate other cytokines and adhesion molecule expression , thus enhancing vascular permeability , cellular adhesion , and procoagulant activity [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Additionally , ANGPT2 stimulation induced adhesion molecule expression by endothelial cells , which may further promote the myeloid cell trafficking to the metastases .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial infection by Chlamydia pneumoniae also activates endothelial cells to produce cytokines and adhesion molecules and become procoagulant [ 21 ] [ 22 ] [ 23 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "61 C5a directly activates endothelial cells to upregulate adhesion molecules such as P-selectin .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:malignant)\n",
      "Alveolar macrophages , neutrophils , lymphocytes , fibroblasts , and endothelial cells all mediated release of cytokines , chemokines , and growth factors may act to promote epithelial dysfunction and malignant progression .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"endothelial dysfunction\")\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(TEXT:\"endothelial dysfunction\")\n",
      "Moreover , complexes of presumably NET-borne DNA stimulate vascular plasmacytoid dendritic cells , with a strong interferon type 1 response , promoting atherogenesis.41 NETs might also directly cause endothelial dysfunction by activation and induction of damage to endothelial cells , illustrated by the establishment of NETs in atherosclerotic lesions and arterial thrombi in humans .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(TEXT:\"endothelial dysfunction\")\n",
      "Moreover , complexes of presumably NET-borne DNA stimulate vascular plasmacytoid dendritic cells , with a strong interferon type 1 response , promoting atherogenesis.41 NETs might also directly cause endothelial dysfunction by activation and induction of damage to endothelial cells , illustrated by the establishment of NETs in atherosclerotic lesions and arterial thrombi in humans.42 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:Fibrin ! Fibrin)\n",
      "In case of damage , endothelial cells trigger fibrin formation , as well as platelet adhesion and aggregation [ 11 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:Protease ! Protease)\n",
      "However , endothelial cells and perivascular cells such as pericytes produce proteolytic enzymes [ 13 ] , which may enhance fibrinolysis once the fibrin clots have been extruded .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:VEGF ! VEGF)\n",
      "For example , during tumorigenesis , IL-1 from recruited myeloid cells causes endothelial cells to upregulate VEGF and other proangiogenic factors , which promotes an inflammatory microenvironment that supports tumor angiogenesis ( 172 , 173 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "Endothelial cells are activated by growth factors ( VEGF , hepatocyte growth factor ( HGF ) and fibroblast growth factors ( FGFs ) ) to initiate angiogenesis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0005604\" ! \"basement membrane\")\n",
      "Endothelial cells produce basement membrane , and are capable of contraction .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "The higher EC 50 value of 100nM measured for HRV A1 indicates a less critical role of eIF4A activity in HRV A1 IRES mediated translation initiation.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006897\" ! endocytosis)\n",
      "Pathogenic HCPS-causing hantaviruses such as SNV or ANDV will first bind to beta3-Integrin receptors on endothelial cells which will mediate endocytosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "This result reveals that apocynin may protect EC from ER stress induced apoptosis.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Both endothelial cells and VSMCs secrete pro-inflammatory cytokines when either damaged or undergoing apoptosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Thrombin , FXa and FVIIa , produced by the coagulation cascade , augment the inflammatory response by activating platelets , endothelial cells , monocytes , neutrophils , NKT and T cells through PARs .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "[ 52 ] [ 53 ] [ 54 ] suggest that SARS-CoV-2 infects endothelial cells to cause inflammation ( endothelialitis ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Thus, the study demonstrates a novel mechanism underlying HS-augmented lung inflammation, namely that induction of increased TLR2 surface expression in lung endothelial cells, which is induced by HS/R and mediated by HMGB1 activation of TLR4 signaling, is an important mechanism responsible for EC-mediated inflammation and organ injury following HS (122).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "74 The local effects of INF-alpha collectively function to increase inflammation within the tissue by activating endothelial cells , thereby increasing vascular permeability and allowing the passage of immune cells , immunoglobulin , and complement into the tissue .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been determined that SARS-CoV2 binds to ACE2 receptors on the endothelial cells which causes a massive inflammatory response increasing blood vessel constriction , leading to end-organ damage and stroke ; ACE2 recombinant therapy is a potential treatment for COVID-19-related stroke [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been determined that SARS-CoV2 binds to ACE2 receptors on the endothelial cells which causes a massive inflammatory response increasing blood vessel constriction , leading to end-organ damage and stroke ; ACE2 recombinant therapy is a potential treatment for COVID-19-related stroke [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2 , CCL5 and CD11b + cells and decrease IFNgamma secretion but did not affect IFNalpha , CCL2 , IL6 or TNFalpha , thus showing that endothelial cells are able to actually modulate the inflammatory response [ 71 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These endothelial cells prevent inflammation by inhibiting their interaction with immune cells and platelets and by expressing coagulation inhibitors and fibrinolytic enzymes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "This evokes cytokine and chemokine release from defense cells with non-cytopathic effects , and can lead to endothelial damage as well as an intense inflammatory response .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Still , umpteen experimental trials are required to assess direct interaction of virus-platelets or the message sent by damaged endothelial cells that led to inflammation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "The EC activation encountered during PE exacerbates systemic inflammation and increases the expression of leukocyte adhesion molecules such as intercellular adhesion molecule-1 (ICAM-1), vascular cell adhesion molecule-1 (VCAM-1), P-selectin (SELP), and E-selectin (SELE) [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "Endothelial cells interact with platelets and leucocytes , prompting their recruitment , adhesion , and interaction on thrombogenic surfaces , at sites of vascular injury driven by inflammation or infection , or at lesion-prone sites such as the carotid bifurcation [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Functional-blocking or shRNA-mediated depletion of integrin β3 in EC cells dramatically decreased the rate of CSFV proliferation.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Further studies will be required to determine the relative importance of plasma VWF/FVIII levels and EC activation in the pathogenesis underlying Covid‐induced coagulopathy and pulmonary microvascular occlusion.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0016032\" ! \"viral process\")\n",
      "This finding is consistent with the characteristics of the so called `` primary cytokine '' storm induced by viral infection which were mainly produced by alveolar macrophages , epithelial cells and endothelial cells , rather than those observed in `` secondary cytokine '' storm induced by different subsets of activated T lymphocytes in late stage of viral infection or a complication of T cell-engaging therapies [ 15,16 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0030112\" ! glycocalyx)\n",
      "Endothelial cells also produce a glycocalyx , a protective layer of glycoproteins and glycolipids with anticoagulation properties [ 96 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "119,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants-tissue factor ( TF ) to initiate the blood clotting cascade , 121 , 122 von Willebrand factor ( vWF ) to enhance platelet adhesion , 123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "119,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants-tissue factor ( TF ) to initiate the blood clotting cascade , 121 , 122 von Willebrand factor ( vWF ) to enhance platelet adhesion , 123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "In response to activation by IL-1beta and TNF , endothelial cells promote coagulation by expressing P-selectin , von Willebrand factor and fibrinogen , leading to massive platelet binding113 ( Fig. 3d ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "For cardiovascular effects , it is suggested that individuals carrying the A antigen have a higher prevalence of thromboembolism due its association with high levels of von Willebrand factor [ 67 ] , a glycoprotein synthesized and secreted by endothelial cells and megakaryocytes that stimulates coagulation [ 68 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:1990266\" ! \"neutrophil migration\")\n",
      "Ligation of EC adhesion molecules by neutrophil binding triggers downstream signaling pathways in EC that induce cytoskeletal contraction and junctional opening to allow neutrophil migration (Hordijk, 2006).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:1990742\" ! microvesicle)\n",
      "Endothelial cells produce microvesicles in response to inflammatory conditions and inflammatory mediators , including cytokines , thrombin and complement 5a [ 34 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10618 ! CCL2)\n",
      "Similarly , CCL2 is produced in the bone marrow by endothelial cells following TLR signaling in infected lungs , which is crucial for inflammatory monocyte release into the bloodstream .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10618 ! CCL2)\n",
      "The experimental model of vasculitis reportedly underlies the interaction between the CCR2 expressed in monocytes and the CCL2 induced by the NOD1-dependent signaling in EC (78).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "Endothelial cell , astrocytes and microglia are a prominent source of CXCL10 and CXCL9 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10672 ! CXCL12)\n",
      "Much of the work studying inflammatory-induced CXCL12 has been done in the context of autoimmune neuroinflammation , such as MS and its murine model experimental autoimmune encephalomyelitis ( EAE ) , during which endothelial cells , neurons , microglia , and astrocytes have all been shown to increase their expression of CXCL12 ( Ambrosini et al. 2005 ; Calderon et al. 2006 ; Krumbholz et al. 2006 ; McCandless et al. 2006 ; McCandless et al. 2008a ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10803 ! SFTPD)\n",
      "SP-D is synthesized by Club cells , type II alveolar cells , and endothelial cells , and the levels of SP-D multimers and trimers in the serum are highly genetically determined .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:11892 ! TNF)\n",
      "TNF-a is produced by macrophages , monocytes , endothelial cells , neutrophils , smooth muscle cells , activated lymphocytes , astrocytes and adipocytes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12663 ! VCAM1)\n",
      "These may stimulate endothelial cells to produce a vascular cell adhesion molecule that results in further buildup of cells and shrinkage of the arterial diameter ( Zapolska-Downar et al ., 2001 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12663 ! VCAM1)\n",
      "Incubation of EC with IgG, from patients positive for aPL, increased the expression of VCAM-1, E-selectin, and MCP-1 as previously reported for AECA [125] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12726 ! VWF)\n",
      "VWF is synthesized by endothelial cells as a 2,813 amino acid-long protein that is processed intracellularly to form VWF antigen 2 ( VWFAg2 ) and VWF multimer ( Haberichter , 2015 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12726 ! VWF)\n",
      "VWF is synthesized by endothelial cells as a 2,813 amino acid-long protein that is processed intracellularly to form VWF antigen 2 ( VWFAg2 ) and VWF multimer ( Haberichter , 2015 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Endothelial cells that also produce S1P can work in synergy with pericytes to elicit thymocyte egress .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Platelets , RBCs and endothelial cells have been proven to be the main sources of S1P [ 18 ] [ 19 ] [ 20 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Platelets , RBCs and endothelial cells have been proven to be the main sources of S1P [ 18-20 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15532 ! JAM3)\n",
      "Moreover , the upregulation of JAM2 and JAM3 by endothelial cells and de novo synthesis of JAM3 by hepatic stellate cells trigger autoimmune-mediated liver fibrosis in mice [ 24 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:16028 ! IL33)\n",
      "In addition to keratinocytes , 52 endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:18454 ! PROK1)\n",
      "Prokineticin-1 and -2 as well as PKR1 are synthesized in the human myocardium by endothelial cells and cardiomyocytes [ 416 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:19232 ! CXCL17)\n",
      "Other cells may amplify CXCL17 signaling from the epithelium , such as endothelial cells which are also producers of CXCL17 [ 24 ] , suggesting that this chemokine facilitates the exit of APCs from the vasculature .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:1968 ! LYST)\n",
      "50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "55 In addition , studies also reported that endothelial cells can produce GM-CSF in response to pro-inflammatory cytokine stimulation.56-58 In vitro , GM-CSF can regulate proliferation and / or activation of myeloid cells , namely monocytes , macrophages , DCs , neutrophils and eosinophils .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "IL-17D does not stimulate the proliferation of immune cells of its own , but , in response to stress , it induces endothelial cells to produce IL-6 , IL-8 and GM-CSF ( 96 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "Other cell types like macrophages , endothelial cells , fibroblasts and T cells also produce GM-CSF , but AECs were shown to be the only CD45 - cell population in the distal lung parenchyma that upregulates GM-CSF upon IV infection , and produces high levels of GM-CSF in the alveolar lining fluid [ 28 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2545 ! CTSS)\n",
      "It has come forth that apart from tumor cells , tumor-associated macrophages and endothelial cells can produce cathepsin S , which promotes neovascularization and tumor growth [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2707 ! ACE)\n",
      "In contrary , endothelial cells are a major source for angiotensin-converting enzyme 2 ( ACE2 ) , which downregulates angiotensin 2 , and increased serum levels of ACE2 after IAV infection correlates closely with epithelial cell protection and improved outcome after human H5N1 and H7N9 infection .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2707 ! ACE)\n",
      "Very recently, an ACE inhibitor-induced dimerization, phosphorylation, and signaling of ACE has been demonstrated in EC that enhanced endothelial ACE expression [43] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3541 ! F3)\n",
      "Collectively , these results suggest that COVID-19 PRP-induced NETs may amplify the TF / thrombin axis by activating endothelial cells to express TF .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3541 ! F3)\n",
      "The activation of EC and monocytes upregulates the production of tissue factor leading to a procoagulant state [XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases p(HGNC:3541 ! F3)\n",
      "Damage to the endothelial cells activates TF which serves as the primary trigger for the extrinsic coagulation cascade .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3676 ! FGF2)\n",
      "CONCLUSIONS After ischemic stroke , optogenetic inhibition of GABAergic neurons upregulated bFGF expression by endothelial cells and promoted neurobehavioral recovery , possibly orchestrated by astrocytes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3676 ! FGF2)\n",
      "Indeed , endothelial cells produce FGF2 that modulates cell proliferation and migration , as well as the production of proteinases and their receptors .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3818 ! FOXM1)\n",
      "Using EC restricted disruption of Foxm1 in mice and primary human lung microvascular ECs, Mirza et al. demonstrated that FoxM1 is required for endothelial AJ recovery via its transcriptional control of beta-catenin 7.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:4524 ! PROKR1)\n",
      "Prokineticin-1 and -2 as well as PKR1 are synthesized in the human myocardium by endothelial cells and cardiomyocytes [ 416 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:484 ! ANGPT1)\n",
      "Angiopoietin-1 , stromal-derived factor-1alpha , and vascular endothelial growth factor are synthesized by endothelial cells to release neurogenic factors , such as neurotrophins and BDNF .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5164 ! HPSE)\n",
      "This class of derivatives (OSK5), along with new preparations of O sulfated sulfaminoheparosans (NSOSK5), were also tested as heparanase inhibitors in a translational project \" Heparanase \" supported by the EC, which recognized the enzyme as a potential therapeutic target for cancer.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5164 ! HPSE)\n",
      "Once in the blood , the virus may induce endothelial glycocalyx degradation by activating leukocytes , platelets , and endothelial cells to secrete MMPs and heparanase that target glycocalyx components ( 39 , 48 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5438 ! IFNG)\n",
      "Furthermore , NEUs can be triggered by virus-related inflammatory factors , such as IL-6 , IL-8 , TNF-alpha , and granulocyte colony stimulating factor , and interferon gamma factors , which are produced by LYM and endothelial cells [ 27 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is produced by activated leukocytes and endothelial cells and acts on multiple cells and tissues .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Vascular injury and inflammation stimulate IL-6 synthesis by endothelial cells that in turn are activated by IL-6 [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Fibroblasts , endothelial cells , activated B cells , monocytes , and helper T cells produce IL6 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Monocytes , endothelial cells , fibroblasts , and activated Th2 cells produce IL-6 [ 16 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Upon activation by infectious agents like SARS-COV-2 , IL-6 is produced by macrophages , endothelial cells , and T cells and activate many tissue cells , including endothelial cells and parenchymal cells to produce effector molecules of inflammation 17 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Upon activation by infectious agents like SARS-COV-2 , IL-6 is produced by macrophages , endothelial cells , and T cells and activate many tissue cells , including endothelial cells and parenchymal cells to produce effector molecules of inflammation.17 IL-10 is mainly produced by monocytes and T cells and the most potent anti-inflammatory cytokine , which reduces inflammatory processes and thus limits collateral inflammation-induced tissue damage.18 , 19 On one hand , SARS-COV-2 invasion activated T cell-mediated immunity which resulted in increasing production of inflammatory cytokine .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "IL-17D does not stimulate the proliferation of immune cells of its own , but , in response to stress , it induces endothelial cells to produce IL-6 , IL-8 and GM-CSF ( 96 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:613 ! APOE)\n",
      "46 ,  60  Interestingly , in the vascular endothelium , although endothelial cells do not produce ApoE , its local expression by macrophages has a paracrine effect on these cells , inhibiting VCAM-1 , stimulating NO synthesis , suppressing endothelial activation , and decreasing adhesion of monocytes to the endothelium , whereas ApoE4 was found to antagonize these anti-inflammatory effects .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7098 ! CXCL9)\n",
      "Endothelial cell , astrocytes and microglia are a prominent source of CXCL10 and CXCL9 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7876 ! NOS3)\n",
      "OxPAPC treatment of EC induced a dose- and time dependent activation of eNOS, and NOS inhibitor, N-nitro-L-arginine-methyl ester (L-NAME), significantly inhibited SREBP activation and IL-8 synthesis by OxPAPC [XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7876 ! NOS3)\n",
      "Although the antiviral effect of NO was previously described [ 66,86,87 ] , we were surprised that BK inhibited NO production in infected HBMECs because B2R signaling classically upregulates NO production by normal endothelial cells via activation of different isoforms of NO synthase ( eNOS ) [ 88,89,90,91 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7883 ! NOTCH3)\n",
      "EC-derived Notch ligand DLL4 activates Notch3 receptors on brain α-SMA positive pericytes and reduces pericyte migration and angiogenic function.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:8760 ! PDCD1)\n",
      "Surprisingly , endothelial cells upregulate PD-1 and secrete high levels of IL-10 , TGF-alpha , TGF-beta and IFN-gamma , thereby promoting recruitment of immunosuppressive cells such as pDCs which interacts with Tregs , to the lung , and hence enhance the immunosuppressive abeyants of activated Tregs [ 222 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acutephase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "Endotoxin and TNF-alpha also produce multiple abnormalities in coagulation and fibrinolysis , leading to microvascular clotting and diffuse intravascular coagulation.They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acutephase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acute-phase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9253 ! PPIA)\n",
      "Second, CyPA increases EC activation and inflammation by increasing vascular cell adhesion molecule 1 (VCAM-1) expression.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9253 ! PPIA)\n",
      "Second, CyPA increases EC activation and inflammation by increasing vascular cell Table 1 Major CyPA functions in protein folding and trafficking, T-cell activation and cell signaling   CyPA  functions   CyPA targets  CyPA effects  References   Folding  HIV-1 Gag  Promotion of both the formation and the infectivity of virions of HIV-1   5,6 Homo-oligomeric a7 neuronal nicotinic receptor (nAChR) and type 3 serotonin receptor (5HT3R)Maturation of the homo-oligomeric receptor 7, 8 Collagen Acceleration of the folding of the procollagen I 9 TransferrinAcceleration of an initial stage in transferrin folding 10 Drosophila rhodopsin Folding and stability of R1-6 rhodopsin by the NinaA-encoded protein (an eye-specific CyP) 11, 12 Trafficking Apoptosis-inducing factor (AIF) Co-translocation to the neuronal nuclei to induce cell death after cerebral hypoxia-ischemia and ALS 13, 14 Heterogeneous nuclear ribonucleoprotein A2 (hnRNP A2)CXCR4-mediated nuclear export of hnRNP A2, nuclear translocation of ERK1/2, and chemotactic cell migration 15 Transport to the plasma membrane 16 Asialoglycoprotein receptor (ASGPR) Distribution of ASGPR between the plasma membrane and the endosomal pool 17 Fructose-1,6-bisphosphatase (FBPase) Import of FBPase into intermediate transport vesicles for vacuole delivery 18 Zinc-finger protein 1 (Zpr1) Promotion of nuclear export adhesion molecule 1 (VCAM-1) expression.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9367 ! PRH2)\n",
      "Thrombin binding to thrombomodulin and the endothelial protein C receptor (EPCR) on the surface of EC releases activated protein C (APC), which degrades thrombin and elicits cytoprotective effects, intriguingly through PAR ligation (Esmon, 2004 (Esmon, , 2006 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9367 ! PRH2)\n",
      "Thrombin binding to thrombomodulin and the endothelial protein C receptor (EPCR) on the surface of EC releases activated protein C (APC), which degrades thrombin and elicits cytoprotective effects, intriguingly through PAR ligation (Esmon, 2004, Esmon, 2006, Looney and Matthay, 2006).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Other cell types , particularly macrophages , dendritic cells ( DCs ) , and endothelial cells ( ECs ) , produce PTX3 in a gene expression-dependent fashion [ 4 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Thus , circulating and lung myelomonocytic cells and endothelial cells are a major source of PTX3 , and PTX3 plasma concentration can serve as an independent strong prognostic indicator of short-term mortality in COVID-19 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Thus , circulating and lung myelomonocytic cells and endothelial cells are a major source of PTX3 , and PTX3 plasma concentration can serve as an independent strong prognostic indicator of short-term mortality in COVID-19 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "In these tumors , infiltrating macrophages and endothelial cells were the major source of PTX3 in response to locally produced IL-1 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D002630 ! \"Chemotactic Factors\")\n",
      "Hence , viral-induced damage 372 exposes tissue factors on endothelial cells which are activated by persistent cytokines to produce 373 chemoattractants in recruiting leukocytes to endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D003142 ! Communication)\n",
      "Transcytosis in the endothelial cells ( van Deurs et al ., 2003 ; Navarro et al ., 2004 ) is one of the fundamental processes by which the endothelial cells contribute to the communication between blood and tissues .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "Complement proteins can also be produced by epithelial cells ( 59 ) , endothelial cells ( 60 ) , and circulating immune cells such as dendritic cells , granulocytes , macrophages , and monocytes ( 61 , 62 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "The formation of NETs by neutrophils are aided by activated platelets associated with damaged endothelial cells that further activate the complement , fuelling the coagulation cascade and thrombi formation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D004211 ! \"Disseminated Intravascular Coagulation\")\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D004777 ! Environment)\n",
      "As the first responders to sterile inflammatory injury , it is important to understand how endothelial cells , neutrophils and platelets help create this environment .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005334 ! Fever)\n",
      "It further recruits inflammatory cells to the site of infection and stimulates the synthesis of prostaglandin E2 from endothelial cells , which binds to the receptor EP3 in neural cells to generate fever .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005347 ! Fibroblasts)\n",
      "Alveolar macrophages are also thought to produce increased amounts of interleukin-8 , a powerful chemokine for neutrophils which are regularly found in the BAL of patients with scleroderma Mastocytes and their mediators are also thought to play a role in the pathogenesis of the disease , together with endothelin-1 , a factor produced by endothelial cells which directly stimulates the fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005355 ! Fibrosis)\n",
      "It is suggested that endothelial cells may contribute to fibrosis in two ways : by becoming dysfunctional and by differentiating into highly fibrogenic mesenchymal cells referred to as myofibroblasts [ 11 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005355 ! Fibrosis)\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D006470 ! Hemorrhage)\n",
      "Its pathophysiology involves endothelial dysfunction and angiogenesis related to ACE-2 receptor , the host-cell receptor for SARS-CoV-2 , expressed by endothelial cells , which may lead to thrombosis or hemorrhage [ 1 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D006973 ! Hypertension)\n",
      "A probable mechanism might be viral effects on the ACE-2 receptors on the endothelial cells and platelets that can lead to hypertension and hyper-coagulable states respectively .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D007109 ! Immunity)\n",
      "These vessels are lined with endothelial cells , which sit at the interface between blood and tissues and contribute to haemostasis , immunity and inflammatory reactions10 ,11 ( Fig. 1 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009369 ! Neoplasms)\n",
      "In these tumors , infiltrating macrophages and endothelial cells were the major source of PTX3 in response to locally produced IL-1 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009389 ! \"Neovascularization, Pathologic\")\n",
      "Our experiment in vivo demonstrated that the increased protein expression of VEGF and its receptor Flt-1 as well as neovascularization in atherosclerotic plaques were reduced by TACE gene silencing , which lent support to a recent study that pathological neovascularization was attenuated by inactivation of TACE in endothelial cells 36 , 37 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009504 ! Neutrophils)\n",
      "As a result , endothelial cells stimulate neutrophils to form NETs .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D010959 ! \"Tissue Plasminogen Activator\")\n",
      "Tissue plasminogen activator ( tPA ) is produced by endothelial cells and , as its name suggests , activates plasminogen .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D010959 ! \"Tissue Plasminogen Activator\")\n",
      "Endothelial cells can also synthesize and release tissue plasminogen activator that converts plasminogen to plasmin , thereby activating fibrinolysis and clot dissolution221-223 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Its pathophysiology involves endothelial dysfunction and angiogenesis related to ACE-2 receptor , the host-cell receptor for SARS-CoV-2 , expressed by endothelial cells , which may lead to thrombosis or hemorrhage [ 1 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Recent studies have indicated that annexin A2 may be a target for autoantibodies in SLE , SS , RA , and systemic vasculitis , and it has been hypothesized that anti-annexin A2 antibodies can induce thrombosis by activating endothelial cells and inhibiting plasmin generation323334 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Thus , the complement system can also induce thrombosis by inducing endothelial cells to release von Willebrand factor and express P-selectin ( Table 1 , Fig. 1 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(MESH:D013927 ! Thrombosis)\n",
      "EC dysfunction disrupts the balance between vasoconstriction and vasodilation and initiates a number of events that trigger EC activation and predispose the vessel wall to increased endothelial permeability, leukocyte adherence, endothelial proliferation, pro-oxidation and thrombosis.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D014024 ! Tissues)\n",
      "In particular , C5a promotes procoagulant activity through several mechanisms , including the induction of tissue factor by endothelial cells and neutrophils , as well as the upregulation of plasminogen activator inhibitor-1 in mast cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015603 ! Keratinocytes)\n",
      "In addition to keratinocytes , 52 endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015603 ! Keratinocytes)\n",
      "In addition to keratinocytes , endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33.12 , 14 IL-33 is stored preformed in the nucleus of these cells and produced readily to exert its inflammatory effects .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "This in turn activates the endothelial cells to upregulate cell adhesion molecules ( see Ch .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "Endothelial cells upregulate cell adhesion molecules when exposed to a number of mediators , including leukotriene B 4 , as well as cytokines such as tumor necrosis factor and interleukins 1e8 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "Endothelial cells upregulate cell adhesion molecules when exposed to a number of mediators , including leukotriene B4 , as well as cytokines such as tumor necrosis factor and interleukins 1-8 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D016766 ! \"Feline Infectious Peritonitis\")\n",
      "Therefore, the present work focused on the expression of the EC adhesion molecules and possible triggers of EC activation during the development of FIP.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "As COVID-19 is not a hemorrhagic disease , it is probable that ADE , if present , is not mediated by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019281 ! Dimerization)\n",
      "It is estimated that endothelial cells can cleave b 2 GPI monomers and promote their dimerization .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019281 ! Dimerization)\n",
      "It is estimated that endothelial cells can cleave beta2GPI monomers and promote their dimerization .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "Restoration of EC specific FoxM1 expression in Pik3cg global knockout mice promoted EC regeneration and normalized the repair process.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "MCP-1 is also produced by macrophages , fibroblasts , epithelial cells and endothelial cells [ 73 ] , so IL-1beta and MCP-1 was rarely detected in Jurkat cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Incubation of EC with IgG, from patients positive for aPL, increased the expression of VCAM-1, E-selectin, and MCP-1 as previously reported for AECA [125] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") partOf complex(a(CHEBI:53571 ! \"poly(acrylonitrile) macromolecule\"), bp(MESH:D042783 ! \"Endothelial Cells\"))\n"
     ]
    },
    {
     "ename": "KeyError",
     "evalue": "'evidence'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-10-06775ad83515>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0memmaa_covid_graph\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0medge_to_bel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mu\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mv\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'evidence'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m: 'evidence'"
     ]
    }
   ],
   "source": [
    "for u, v, data in emmaa_covid_graph.edges(endothelial, data=True):\n",
    "        print()\n",
    "        print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "        print(data['evidence'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "Pro-inflammatory cytokines released by effector T cells , mostly Th1 and Th17 , are key players in BBB alteration , via the increase of adhesion molecules on endothelial cells that enables lymphocytes adhesion and infiltration into the CNS [ 102 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "The septic stress not only disrupts crucial cellular and tissue functions, but also exacerbates endoplasmic reticulum stress and leads to the release of damage-associated molecular pattern (DAMP) molecules and inflammatory mediators, which cause leukocyte and vascular endothelial cell (EC) activation, increased capillary permeability, neutrophil infiltration, and tissue injury12,13.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:132153 ! hyaluronate)\n",
      "Altered flow dynamics cause further endothelial cell damage and glycocalyx shedding [ 47 ] Inflammatory conditions with high concentrations of extracellular histones and cytokines damage the endothelial glycocalyx accompanied by its degradation shedding of syndecan-1 , heparan sulfate , and hyaluronan [ 47 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:138157 ! \"chemokine ligand 10\")\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:138157 ! \"chemokine ligand 10\")\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:140399 ! \"specialised pro-resolving mediator\")\n",
      "These SPMs are synthesized by macrophages , neutrophiles and partially by endothelial cells and are crucial for ending neutrophile influx , macrophage switching , and clearance of apoptotic cells thereby actively contribute to resolution of inflammation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "IL-1 ( endogenous pyrogen ) induces fever and hypermetabolism , stimulates fibroblasts and endothelial cells to produce GM-CSF and PGE 2 , enhances PGE2 synthesis in the hypothalamus , increases skeletal muscle proteolysis , has a priming effect on macrophages , is important for maintaining the gut-mucosal barrier and turns on the liver synthesis of acute phase proteins ( for review see [ 53 , 54 ] ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "Alternatively , prostaglandin E2 ( PGE2 ) and other prostaglandins may be produced by the endothelial cells which , in turn , induce a neurotransmitter-like substance , such as cAMP , that acts to raise the set-point [ 25 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds.55 Prostacyclin binds to its receptor ( a G-protein coupled receptor ) found on the surface of vascular smooth muscle and platelets , activates cyclic adenosine monophosphate ( cAMP ) , and results in the most potent endogenous inhibitor of platelet aggregation , vascular smooth muscle relaxation and vasodilation of the pulmonary arteries .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced downstream of cyclooxygenase by prostacyclin synthase in EC and, like NO, is also a potent vasodilator and inhibitor of platelet aggregation acting primarily through cyclic AMP as opposed to NO, which predominantly activates cyclic GMP production (Egan and FitzGerald, 2006) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced downstream of cyclooxygenase by prostacyclin synthase in EC and, like NO, is also a potent vasodilator and inhibitor of platelet aggregation acting primarily through cyclic AMP as opposed to NO, which predominantly activates cyclic GMP production.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:15552 ! \"prostaglandin I2\")\n",
      "Prostacyclin is produced predominantly by endothelial cells and induces potent vasodilation of all vascular beds.55 Prostacyclin binds to its receptor ( a G-protein coupled receptor ) found on the surface of vascular smooth muscle and platelets , activates cyclic adenosine monophosphate ( cAMP ) , and results in the most potent endogenous inhibitor of platelet aggregation , vascular smooth muscle relaxation and vasodilation of the pulmonary arteries .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The increase of intracellular pH causes a selective change in the presentation of proper antigens ; Blockage of T-cell response and reduction of proinflammatory cytokine production , including INF-gamma , TNF , IL-1 and IL-6 ; Blockage of Toll-like receptors 7 and 9 , especially in plasmacytoid dendritic cell with inhibition of INF-alpha , which plays an important role in the pathophysiology of SLE ; cGAS-STING signaling inhibition ; Inhibition of phospholipase A2 activity ; Stimulation of nitric oxide production by endothelial cells with antiproliferative effect ; Antithrombotic effect through the inhibition of platelet aggregation in a dose-dependent manner , decreased production of arachidonic acid by activated platelets and action on antiphospholipid antibodies .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Stimulation of nitric oxide production by endothelial cells with antiproliferative effect ; Inhibition of phospholipase A2 activity ; cGAS-STING signaling inhibition ; Blockage of Toll-like receptors 7 and 9 , especially in plasmacytoid dendritic cell with inhibition of INF-alpha , which plays an important role in the pathophysiology of SLE ; Blockage of T-cell response and reduction of pro-inflammatory cytokine production , including INF-gamma , TNF , IL-1 and IL-6 ; Alkalinization of lysosomes and other intracellular acid compartments with interference in phagocytosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Nitric oxide ( NO ) is generated by endothelial cells , platelets , neutrophils , macrophages , and neurons ( Schemann and Schaff , 1995 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Endothelial cells produce NO through the enzyme NO-synthase ( NOS ) , which transforms L-arginine amino acid into citrulline [ 27 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "The action partially results from the enhanced synthesis and release of nitric oxide by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "In a healthy condition , angiotensin-converting enzyme 2 ( ACE2 ) converts angiotensin II to angiotensin 1-7 which stimulates endothelial cells to produce nitric oxide ( NO ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "In 1987 , Palmer et al. remarkably discovered that endothelial cells produce NO , the critical endothelium-derived relaxing factor for proper vascular function [ 70 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Although the antiviral effect of NO was previously described [ 66,86,87 ] , we were surprised that BK inhibited NO production in infected HBMECs because B2R signaling classically upregulates NO production by normal endothelial cells via activation of different isoforms of NO synthase ( eNOS ) [ 88,89,90,91 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "These results suggest that BK , otherwise a classical inducer of NO production by healthy endothelial cells , suppresses NO production in DENV-2-infected HBMECs , ultimately rendering the stressed endothelial cells increasingly susceptible to viral replication .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "It is known that ACE converts angiotensin I ( AngI ) into the pro-inflammatory peptide angiotensin II ( AngII ) , and ACE2 metabolizes AngII to produce angiotensin - ( 1-7 ) ( ( Ang - ( 1-7 ) ) ) and Ang - ( 1-7 ) promotes endothelial cells to produce NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "It is known that ACE converts angiotensin I ( AngI ) into the pro-inflammatory peptide angiotensin II ( AngII ) , and ACE2 metabolizes AngII to produce angiotensin - ( 1-7 ) ( ( Ang - ( 1-7 ) ) ) and Ang - ( 1-7 ) promotes endothelial cells to produce NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Healthy endothelial cells synthesize nitric oxide ( NO ) by conversion of l-arginine to l-citrulline by nitric oxide synthase .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Healthy endothelial cells synthesize nitric oxide ( NO ) by conversion of l-arginine to l-citrulline by nitric oxide synthase .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Because SARS-CoV-2 infects endothelial cells , which are a source of NO , carrying NO from nanoparticles may be an alternative for NO replacement , as well as a response to the viral attack on endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Other antithrombotic mechanism of aspirin involves upregulation of nitric oxide ( NO ) metabolic production by endothelial cells , through inhibition prostacyclin synthesis , which leads to platelet inactivation ( 26 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "Pretreatment of lung capillary EC with heparinase III completely abolished the pressure induced increase in Lp and reduced intracellular NO detected by a fluorescent dye, suggesting that cell-surface HS directly participates in mechanotransduction resulting in NO production and increased Lp.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "We conclude that : ( 1 ) mitochondrial H 2 S oxidation plays an important role in the regulation of intrarenal H 2 S , ( 2 ) H 2 S concentration is higher in the renal medulla most likely due to physiological hypoxia in this tissue , ( 3 ) nephroprotective effect of statins demonstrated in many studies may be mediated by elevation of H 2 S. N-Ethyl-L-glutamine enhances the synthesis of nitric oxide by endothelial cells Results of recent studies indicate that enhanced production of glucosamine from glutamine and fructose-6-phosphate in endothelial cells ( EC ) results in reduced synthesis of nitric oxide ( NO ) from L-arginine by NO synthase ( NOS ) , leading to impaired relaxation of blood vessels .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "30 muM tetrandrine inhibited NO production by the endothelial cells through blockade of Ca2 + release-activated Ca2 + channels ( Kwan et al.2001 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(CHEBI:16480 ! \"nitric oxide\")\n",
      "30 lM tetrandrine inhibited NO production by the endothelial cells through blockade of Ca 2 ?\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:18421 ! superoxide)\n",
      "The underlying mechanism of ALI induced by oleic acid is associated with cytokines releases such as TNF - , which stimulates monocytes to produce IL-1.As the core of the cytokine-network , TNF -  and IL-1 play important roles not only in the production of other inflammatory cytokines , but also in the migration and adherence of neutrophils to endothelial cells ( Yoshimi et al ., 2000 ) , which together with endothelial cell injury by cytokines result in the production of superoxide radicals and subsequently injure alveolar epithelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:18421 ! superoxide)\n",
      "The underlying mechanism of ALI induced by oleic acid is associated with cytokines releases such as TNF-alpha , which stimulates monocytes to produce IL-1.As the core of the cytokine-network , TNF-alpha and IL-1 play important roles not only in the production of other inflammatory cytokines , but also in the migration and adherence of neutrophils to endothelial cells ( Yoshimi et al ., 2000 ) , which together with endothelial cell injury by cytokines result in the production of superoxide radicals and subsequently injure alveolar epithelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:27601 ! pteridine)\n",
      "Neopterin or 2-amino-4-hydroxy-6 - ( D-erythro-1 ' ,2 ' ,3 ' - trihydroxypropyl ) - pteridine is produced from guanosine triphosphate via guanosine triphosphate cyclohydrolase I ( GTPCH I ) by activated monocytes , macrophages , dendritic cells , and endothelial cells and to a lesser extent in renal epithelial cells , fibroblasts , and vascular smooth muscle cells upon stimulation mainly by interferon gamma and to a lesser extent by interferon alpha and beta with its release being enhanced by tumor necrosis factor [ 2 , 3 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:28815 ! \"heparan sulfate\")\n",
      "33 , W-4400 MiJnster , Germany 154 EFFECT OF LMW HEPARINS AND RELATED GLYCOSAMINOGLYCANS ON THE SYNTHESIS OF HEPARAN SULFATE BY ENDOTHELIAL CELLS W. Jeske , H. Nader , C.P .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:30512 ! \"silver atom\")\n",
      "Since acute phase proteins are produced by liver cells , whereas vWF : Ag is synthesized by endothelial cells , we postulate that this coagulation factor is highly sensitive in reflecting the inflammatory process in the intestine .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:37537 ! \"phorbol 13-acetate 12-myristate\")\n",
      "The major source of these high levels of tPA among COVID-19 patients is likely endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(CHEBI:64007 ! EET)\n",
      "Indeed , CYP2C enzymes are induced by hypoxia , and endothelial cells are a major source of EETs during inflammation and angiogenesis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-alpha and IL-6 are mostly derived from lymphocytes and macrophages , but it was produced at a lower concentration with pre-probiotics ; perhaps this type of cytokines is also produced by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-a and IL-6 are mostly derived from lymphocytes and macrophages , but it was produced at a lower concentration with pre-probiotics ; perhaps this type of cytokines is also produced by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "However , previous literature has demonstrated that endothelial cells are a source of cytokines / chemokines in the lung during influenza infection , and that S1PR1 signaling in the endothelium provides a mechanism for inhibiting cytokine storms and blunting the accumulation of innate inflammatory infiltrates characterized as macrophages / monocytes , neutrophils , and NK cells 3 , 7 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "TNF-alpha can also activate endothelial cells to upregulate other cytokines and adhesion molecule expression , thus enhancing vascular permeability , cellular adhesion , and procoagulant activity [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial cells also produce the cytokines , and mitochondria are involved in the regulation of cytokine production .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Endothelial infection by Chlamydia pneumoniae also activates endothelial cells to produce cytokines and adhesion molecules and become procoagulant [ 21 ] [ 22 ] [ 23 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "It is well-known that DIC starts when endothelial cells and monocytes are stimulated to produce cytokines after a serious damage , with abnormal production of von Willebrand factor and Tissue Factor ( TF ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Several studies indicated that endothelial cells might be the source of the cytokines and involved in lung injury during influenza virus infection [ 29 ] [ 30 ] [ 31 ] [ 32 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "Studies also described elevation of dimer D , troponin I , serum IL-6 and C-reactive protein levels.3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:cytokine)\n",
      "3 , 5 , 7 , 9 , 14 As for IL-6 , this cytokine is synthesised by macrophages , T cells , endothelial cells and fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial cells are activated by cytokines and viral particles and produce monocyte chemoattractants and adhesion molecules .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial cells indeed produce inflammatory cytokines , including IL-6 , and upregulate adhesion molecules , further promoting leukocytes ' recruitment and capillary leakage [ 37 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Moreover , endothelial cells produce various cytokines and adhesion molecules [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "TNF-alpha can also activate endothelial cells to upregulate other cytokines and adhesion molecule expression , thus enhancing vascular permeability , cellular adhesion , and procoagulant activity [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Additionally , ANGPT2 stimulation induced adhesion molecule expression by endothelial cells , which may further promote the myeloid cell trafficking to the metastases .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "Endothelial infection by Chlamydia pneumoniae also activates endothelial cells to produce cytokines and adhesion molecules and become procoagulant [ 21 ] [ 22 ] [ 23 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"adhesion factor\")\n",
      "61 C5a directly activates endothelial cells to upregulate adhesion molecules such as P-selectin .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:malignant)\n",
      "Alveolar macrophages , neutrophils , lymphocytes , fibroblasts , and endothelial cells all mediated release of cytokines , chemokines , and growth factors may act to promote epithelial dysfunction and malignant progression .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases a(TEXT:\"endothelial dysfunction\")\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(TEXT:\"endothelial dysfunction\")\n",
      "Moreover , complexes of presumably NET-borne DNA stimulate vascular plasmacytoid dendritic cells , with a strong interferon type 1 response , promoting atherogenesis.41 NETs might also directly cause endothelial dysfunction by activation and induction of damage to endothelial cells , illustrated by the establishment of NETs in atherosclerotic lesions and arterial thrombi in humans .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases a(TEXT:\"endothelial dysfunction\")\n",
      "Moreover , complexes of presumably NET-borne DNA stimulate vascular plasmacytoid dendritic cells , with a strong interferon type 1 response , promoting atherogenesis.41 NETs might also directly cause endothelial dysfunction by activation and induction of damage to endothelial cells , illustrated by the establishment of NETs in atherosclerotic lesions and arterial thrombi in humans.42 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:Fibrin ! Fibrin)\n",
      "In case of damage , endothelial cells trigger fibrin formation , as well as platelet adhesion and aggregation [ 11 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:Protease ! Protease)\n",
      "However , endothelial cells and perivascular cells such as pericytes produce proteolytic enzymes [ 13 ] , which may enhance fibrinolysis once the fibrin clots have been extruded .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(FPLX:VEGF ! VEGF)\n",
      "For example , during tumorigenesis , IL-1 from recruited myeloid cells causes endothelial cells to upregulate VEGF and other proangiogenic factors , which promotes an inflammatory microenvironment that supports tumor angiogenesis ( 172 , 173 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "The activation of both of these receptors on endothelial cells contributes to angiogenesis ( Ghebrehiwet et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "Endothelial cells are activated by growth factors ( VEGF , hepatocyte growth factor ( HGF ) and fibroblast growth factors ( FGFs ) ) to initiate angiogenesis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0005604\" ! \"basement membrane\")\n",
      "Endothelial cells produce basement membrane , and are capable of contraction .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "The higher EC 50 value of 100nM measured for HRV A1 indicates a less critical role of eIF4A activity in HRV A1 IRES mediated translation initiation.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006897\" ! endocytosis)\n",
      "Pathogenic HCPS-causing hantaviruses such as SNV or ANDV will first bind to beta3-Integrin receptors on endothelial cells which will mediate endocytosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "This result reveals that apocynin may protect EC from ER stress induced apoptosis.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Both endothelial cells and VSMCs secrete pro-inflammatory cytokines when either damaged or undergoing apoptosis .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Thrombin , FXa and FVIIa , produced by the coagulation cascade , augment the inflammatory response by activating platelets , endothelial cells , monocytes , neutrophils , NKT and T cells through PARs .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "[ 52 ] [ 53 ] [ 54 ] suggest that SARS-CoV-2 infects endothelial cells to cause inflammation ( endothelialitis ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Thus, the study demonstrates a novel mechanism underlying HS-augmented lung inflammation, namely that induction of increased TLR2 surface expression in lung endothelial cells, which is induced by HS/R and mediated by HMGB1 activation of TLR4 signaling, is an important mechanism responsible for EC-mediated inflammation and organ injury following HS (122).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "74 The local effects of INF-alpha collectively function to increase inflammation within the tissue by activating endothelial cells , thereby increasing vascular permeability and allowing the passage of immune cells , immunoglobulin , and complement into the tissue .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been determined that SARS-CoV2 binds to ACE2 receptors on the endothelial cells which causes a massive inflammatory response increasing blood vessel constriction , leading to end-organ damage and stroke ; ACE2 recombinant therapy is a potential treatment for COVID-19-related stroke [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been determined that SARS-CoV2 binds to ACE2 receptors on the endothelial cells which causes a massive inflammatory response increasing blood vessel constriction , leading to end-organ damage and stroke ; ACE2 recombinant therapy is a potential treatment for COVID-19-related stroke [ 9 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Further work demonstrated a level of SIP1 expression on lymphatic and vascular endothelial cells and that treatment with an agonist of SIP1 was able to reduce the production of chemokines CCL2 , CCL5 and CD11b + cells and decrease IFNgamma secretion but did not affect IFNalpha , CCL2 , IL6 or TNFalpha , thus showing that endothelial cells are able to actually modulate the inflammatory response [ 71 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These endothelial cells prevent inflammation by inhibiting their interaction with immune cells and platelets and by expressing coagulation inhibitors and fibrinolytic enzymes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "This evokes cytokine and chemokine release from defense cells with non-cytopathic effects , and can lead to endothelial damage as well as an intense inflammatory response .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Still , umpteen experimental trials are required to assess direct interaction of virus-platelets or the message sent by damaged endothelial cells that led to inflammation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "The EC activation encountered during PE exacerbates systemic inflammation and increases the expression of leukocyte adhesion molecules such as intercellular adhesion molecule-1 (ICAM-1), vascular cell adhesion molecule-1 (VCAM-1), P-selectin (SELP), and E-selectin (SELE) [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "Endothelial cells interact with platelets and leucocytes , prompting their recruitment , adhesion , and interaction on thrombogenic surfaces , at sites of vascular injury driven by inflammation or infection , or at lesion-prone sites such as the carotid bifurcation [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Functional-blocking or shRNA-mediated depletion of integrin β3 in EC cells dramatically decreased the rate of CSFV proliferation.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Further studies will be required to determine the relative importance of plasma VWF/FVIII levels and EC activation in the pathogenesis underlying Covid‐induced coagulopathy and pulmonary microvascular occlusion.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0016032\" ! \"viral process\")\n",
      "This finding is consistent with the characteristics of the so called `` primary cytokine '' storm induced by viral infection which were mainly produced by alveolar macrophages , epithelial cells and endothelial cells , rather than those observed in `` secondary cytokine '' storm induced by different subsets of activated T lymphocytes in late stage of viral infection or a complication of T cell-engaging therapies [ 15,16 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0030112\" ! glycocalyx)\n",
      "Endothelial cells also produce a glycocalyx , a protective layer of glycoproteins and glycolipids with anticoagulation properties [ 96 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "119,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants-tissue factor ( TF ) to initiate the blood clotting cascade , 121 , 122 von Willebrand factor ( vWF ) to enhance platelet adhesion , 123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "119,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants-tissue factor ( TF ) to initiate the blood clotting cascade , 121 , 122 von Willebrand factor ( vWF ) to enhance platelet adhesion , 123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "In response to activation by IL-1beta and TNF , endothelial cells promote coagulation by expressing P-selectin , von Willebrand factor and fibrinogen , leading to massive platelet binding113 ( Fig. 3d ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "For cardiovascular effects , it is suggested that individuals carrying the A antigen have a higher prevalence of thromboembolism due its association with high levels of von Willebrand factor [ 67 ] , a glycoprotein synthesized and secreted by endothelial cells and megakaryocytes that stimulates coagulation [ 68 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0050817\" ! coagulation)\n",
      "AT1 also promotes increased expression of adhesion molecules such as VCAM-1 and E-Selectin that recruit inflammatory cells to the injury site , and inflammation synergistically promotes coagulation.119 ,120 AII prompts endothelial cells to promote coagulation , by new expression of procoagulants - - tissue factor ( TF ) to initiate the blood clotting cascade ,121,122 von Willebrand factor ( vWF ) to enhance platelet adhesion ,123 and plasminogen activator inhibitor ( PAI ) -1 to blunt fibrinolytic degradation of blood clots.37 ,42,43,119 Activation of these processes would then be the drivers of stroke in COVID-19-induced RAS imbalance , with overactive AII sabotaging the anti-coagulant actions of otherwise healthy endothelium .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:0090195\" ! \"chemokine secretion\")\n",
      "In many cases , infection promotes chemokine secretion by endothelial cells , which increases vascular permeability and permits viruses to bypass the first layer of the BBB .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:1990266\" ! \"neutrophil migration\")\n",
      "Ligation of EC adhesion molecules by neutrophil binding triggers downstream signaling pathways in EC that induce cytoskeletal contraction and junctional opening to allow neutrophil migration (Hordijk, 2006).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(GO:\"GO:1990742\" ! microvesicle)\n",
      "Endothelial cells produce microvesicles in response to inflammatory conditions and inflammatory mediators , including cytokines , thrombin and complement 5a [ 34 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10618 ! CCL2)\n",
      "Similarly , CCL2 is produced in the bone marrow by endothelial cells following TLR signaling in infected lungs , which is crucial for inflammatory monocyte release into the bloodstream .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10618 ! CCL2)\n",
      "The experimental model of vasculitis reportedly underlies the interaction between the CCR2 expressed in monocytes and the CCL2 induced by the NOD1-dependent signaling in EC (78).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "Endothelial cell , astrocytes and microglia are a prominent source of CXCL10 and CXCL9 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10637 ! CXCL10)\n",
      "IP-10 is also known as CXCL10 or interferon gamma-induced protein 10 that is produced by monocytes , endothelial cells and fibroblasts and serves as a chemoattractant for immune cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10672 ! CXCL12)\n",
      "Much of the work studying inflammatory-induced CXCL12 has been done in the context of autoimmune neuroinflammation , such as MS and its murine model experimental autoimmune encephalomyelitis ( EAE ) , during which endothelial cells , neurons , microglia , and astrocytes have all been shown to increase their expression of CXCL12 ( Ambrosini et al. 2005 ; Calderon et al. 2006 ; Krumbholz et al. 2006 ; McCandless et al. 2006 ; McCandless et al. 2008a ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:10803 ! SFTPD)\n",
      "SP-D is synthesized by Club cells , type II alveolar cells , and endothelial cells , and the levels of SP-D multimers and trimers in the serum are highly genetically determined .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:11892 ! TNF)\n",
      "TNF-a is produced by macrophages , monocytes , endothelial cells , neutrophils , smooth muscle cells , activated lymphocytes , astrocytes and adipocytes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12663 ! VCAM1)\n",
      "These may stimulate endothelial cells to produce a vascular cell adhesion molecule that results in further buildup of cells and shrinkage of the arterial diameter ( Zapolska-Downar et al ., 2001 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12663 ! VCAM1)\n",
      "Incubation of EC with IgG, from patients positive for aPL, increased the expression of VCAM-1, E-selectin, and MCP-1 as previously reported for AECA [125] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12726 ! VWF)\n",
      "VWF is synthesized by endothelial cells as a 2,813 amino acid-long protein that is processed intracellularly to form VWF antigen 2 ( VWFAg2 ) and VWF multimer ( Haberichter , 2015 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:12726 ! VWF)\n",
      "VWF is synthesized by endothelial cells as a 2,813 amino acid-long protein that is processed intracellularly to form VWF antigen 2 ( VWFAg2 ) and VWF multimer ( Haberichter , 2015 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Endothelial cells that also produce S1P can work in synergy with pericytes to elicit thymocyte egress .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Platelets , RBCs and endothelial cells have been proven to be the main sources of S1P [ 18 ] [ 19 ] [ 20 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15456 ! MBTPS1)\n",
      "Platelets , RBCs and endothelial cells have been proven to be the main sources of S1P [ 18-20 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:15532 ! JAM3)\n",
      "Moreover , the upregulation of JAM2 and JAM3 by endothelial cells and de novo synthesis of JAM3 by hepatic stellate cells trigger autoimmune-mediated liver fibrosis in mice [ 24 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:16028 ! IL33)\n",
      "In addition to keratinocytes , 52 endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:18454 ! PROK1)\n",
      "Prokineticin-1 and -2 as well as PKR1 are synthesized in the human myocardium by endothelial cells and cardiomyocytes [ 416 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:19232 ! CXCL17)\n",
      "Other cells may amplify CXCL17 signaling from the epithelium , such as endothelial cells which are also producers of CXCL17 [ 24 ] , suggesting that this chemokine facilitates the exit of APCs from the vasculature .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:1968 ! LYST)\n",
      "50 Endothelial cells also synthesise Ch-S ; the enzyme endothelial nitric oxide synthase ( eNOS ) is traditionally thought to mediate synthesis of nitric oxide ( NO ) ; however , it has been established that when eNOS is membrane bound , it is no longer able to synthesise NO .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "55 In addition , studies also reported that endothelial cells can produce GM-CSF in response to pro-inflammatory cytokine stimulation.56-58 In vitro , GM-CSF can regulate proliferation and / or activation of myeloid cells , namely monocytes , macrophages , DCs , neutrophils and eosinophils .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "IL-17D does not stimulate the proliferation of immune cells of its own , but , in response to stress , it induces endothelial cells to produce IL-6 , IL-8 and GM-CSF ( 96 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2434 ! CSF2)\n",
      "Other cell types like macrophages , endothelial cells , fibroblasts and T cells also produce GM-CSF , but AECs were shown to be the only CD45 - cell population in the distal lung parenchyma that upregulates GM-CSF upon IV infection , and produces high levels of GM-CSF in the alveolar lining fluid [ 28 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2545 ! CTSS)\n",
      "It has come forth that apart from tumor cells , tumor-associated macrophages and endothelial cells can produce cathepsin S , which promotes neovascularization and tumor growth [ 39 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2707 ! ACE)\n",
      "In contrary , endothelial cells are a major source for angiotensin-converting enzyme 2 ( ACE2 ) , which downregulates angiotensin 2 , and increased serum levels of ACE2 after IAV infection correlates closely with epithelial cell protection and improved outcome after human H5N1 and H7N9 infection .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:2707 ! ACE)\n",
      "Very recently, an ACE inhibitor-induced dimerization, phosphorylation, and signaling of ACE has been demonstrated in EC that enhanced endothelial ACE expression [43] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3541 ! F3)\n",
      "Collectively , these results suggest that COVID-19 PRP-induced NETs may amplify the TF / thrombin axis by activating endothelial cells to express TF .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3541 ! F3)\n",
      "The activation of EC and monocytes upregulates the production of tissue factor leading to a procoagulant state [XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases p(HGNC:3541 ! F3)\n",
      "Damage to the endothelial cells activates TF which serves as the primary trigger for the extrinsic coagulation cascade .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3676 ! FGF2)\n",
      "CONCLUSIONS After ischemic stroke , optogenetic inhibition of GABAergic neurons upregulated bFGF expression by endothelial cells and promoted neurobehavioral recovery , possibly orchestrated by astrocytes .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3676 ! FGF2)\n",
      "Indeed , endothelial cells produce FGF2 that modulates cell proliferation and migration , as well as the production of proteinases and their receptors .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:3818 ! FOXM1)\n",
      "Using EC restricted disruption of Foxm1 in mice and primary human lung microvascular ECs, Mirza et al. demonstrated that FoxM1 is required for endothelial AJ recovery via its transcriptional control of beta-catenin 7.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:4524 ! PROKR1)\n",
      "Prokineticin-1 and -2 as well as PKR1 are synthesized in the human myocardium by endothelial cells and cardiomyocytes [ 416 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:484 ! ANGPT1)\n",
      "Angiopoietin-1 , stromal-derived factor-1alpha , and vascular endothelial growth factor are synthesized by endothelial cells to release neurogenic factors , such as neurotrophins and BDNF .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5164 ! HPSE)\n",
      "This class of derivatives (OSK5), along with new preparations of O sulfated sulfaminoheparosans (NSOSK5), were also tested as heparanase inhibitors in a translational project \" Heparanase \" supported by the EC, which recognized the enzyme as a potential therapeutic target for cancer.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5164 ! HPSE)\n",
      "Once in the blood , the virus may induce endothelial glycocalyx degradation by activating leukocytes , platelets , and endothelial cells to secrete MMPs and heparanase that target glycocalyx components ( 39 , 48 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:5438 ! IFNG)\n",
      "Furthermore , NEUs can be triggered by virus-related inflammatory factors , such as IL-6 , IL-8 , TNF-alpha , and granulocyte colony stimulating factor , and interferon gamma factors , which are produced by LYM and endothelial cells [ 27 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is produced by activated leukocytes and endothelial cells and acts on multiple cells and tissues .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Vascular injury and inflammation stimulate IL-6 synthesis by endothelial cells that in turn are activated by IL-6 [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Fibroblasts , endothelial cells , activated B cells , monocytes , and helper T cells produce IL6 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Monocytes , endothelial cells , fibroblasts , and activated Th2 cells produce IL-6 [ 16 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "IL-6 is generated by monocytes , endothelial cells , macrophages and fibroblasts in response to diverse stimuli ( TNF-alpha , IL-1 and IL-17 ) during systemic inflammation [ 7 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Upon activation by infectious agents like SARS-COV-2 , IL-6 is produced by macrophages , endothelial cells , and T cells and activate many tissue cells , including endothelial cells and parenchymal cells to produce effector molecules of inflammation 17 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6018 ! IL6)\n",
      "Upon activation by infectious agents like SARS-COV-2 , IL-6 is produced by macrophages , endothelial cells , and T cells and activate many tissue cells , including endothelial cells and parenchymal cells to produce effector molecules of inflammation.17 IL-10 is mainly produced by monocytes and T cells and the most potent anti-inflammatory cytokine , which reduces inflammatory processes and thus limits collateral inflammation-induced tissue damage.18 , 19 On one hand , SARS-COV-2 invasion activated T cell-mediated immunity which resulted in increasing production of inflammatory cytokine .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "IL-17D does not stimulate the proliferation of immune cells of its own , but , in response to stress , it induces endothelial cells to produce IL-6 , IL-8 and GM-CSF ( 96 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:6025 ! CXCL8)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:613 ! APOE)\n",
      "46 ,  60  Interestingly , in the vascular endothelium , although endothelial cells do not produce ApoE , its local expression by macrophages has a paracrine effect on these cells , inhibiting VCAM-1 , stimulating NO synthesis , suppressing endothelial activation , and decreasing adhesion of monocytes to the endothelium , whereas ApoE4 was found to antagonize these anti-inflammatory effects .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7098 ! CXCL9)\n",
      "Endothelial cell , astrocytes and microglia are a prominent source of CXCL10 and CXCL9 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7876 ! NOS3)\n",
      "OxPAPC treatment of EC induced a dose- and time dependent activation of eNOS, and NOS inhibitor, N-nitro-L-arginine-methyl ester (L-NAME), significantly inhibited SREBP activation and IL-8 synthesis by OxPAPC [XREF_BIBR].\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7876 ! NOS3)\n",
      "Although the antiviral effect of NO was previously described [ 66,86,87 ] , we were surprised that BK inhibited NO production in infected HBMECs because B2R signaling classically upregulates NO production by normal endothelial cells via activation of different isoforms of NO synthase ( eNOS ) [ 88,89,90,91 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:7883 ! NOTCH3)\n",
      "EC-derived Notch ligand DLL4 activates Notch3 receptors on brain α-SMA positive pericytes and reduces pericyte migration and angiogenic function.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:8760 ! PDCD1)\n",
      "Surprisingly , endothelial cells upregulate PD-1 and secrete high levels of IL-10 , TGF-alpha , TGF-beta and IFN-gamma , thereby promoting recruitment of immunosuppressive cells such as pDCs which interacts with Tregs , to the lung , and hence enhance the immunosuppressive abeyants of activated Tregs [ 222 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acutephase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "Endotoxin and TNF-alpha also produce multiple abnormalities in coagulation and fibrinolysis , leading to microvascular clotting and diffuse intravascular coagulation.They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acutephase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9071 ! PLG)\n",
      "They also induce endothelial cells to produce plasminogen activator and IL-6 , which is an important modulator of the production of acute-phase proteins by the liver .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9253 ! PPIA)\n",
      "Second, CyPA increases EC activation and inflammation by increasing vascular cell adhesion molecule 1 (VCAM-1) expression.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9253 ! PPIA)\n",
      "Second, CyPA increases EC activation and inflammation by increasing vascular cell Table 1 Major CyPA functions in protein folding and trafficking, T-cell activation and cell signaling   CyPA  functions   CyPA targets  CyPA effects  References   Folding  HIV-1 Gag  Promotion of both the formation and the infectivity of virions of HIV-1   5,6 Homo-oligomeric a7 neuronal nicotinic receptor (nAChR) and type 3 serotonin receptor (5HT3R)Maturation of the homo-oligomeric receptor 7, 8 Collagen Acceleration of the folding of the procollagen I 9 TransferrinAcceleration of an initial stage in transferrin folding 10 Drosophila rhodopsin Folding and stability of R1-6 rhodopsin by the NinaA-encoded protein (an eye-specific CyP) 11, 12 Trafficking Apoptosis-inducing factor (AIF) Co-translocation to the neuronal nuclei to induce cell death after cerebral hypoxia-ischemia and ALS 13, 14 Heterogeneous nuclear ribonucleoprotein A2 (hnRNP A2)CXCR4-mediated nuclear export of hnRNP A2, nuclear translocation of ERK1/2, and chemotactic cell migration 15 Transport to the plasma membrane 16 Asialoglycoprotein receptor (ASGPR) Distribution of ASGPR between the plasma membrane and the endosomal pool 17 Fructose-1,6-bisphosphatase (FBPase) Import of FBPase into intermediate transport vesicles for vacuole delivery 18 Zinc-finger protein 1 (Zpr1) Promotion of nuclear export adhesion molecule 1 (VCAM-1) expression.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9367 ! PRH2)\n",
      "Thrombin binding to thrombomodulin and the endothelial protein C receptor (EPCR) on the surface of EC releases activated protein C (APC), which degrades thrombin and elicits cytoprotective effects, intriguingly through PAR ligation (Esmon, 2004 (Esmon, , 2006 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9367 ! PRH2)\n",
      "Thrombin binding to thrombomodulin and the endothelial protein C receptor (EPCR) on the surface of EC releases activated protein C (APC), which degrades thrombin and elicits cytoprotective effects, intriguingly through PAR ligation (Esmon, 2004, Esmon, 2006, Looney and Matthay, 2006).\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Other cell types , particularly macrophages , dendritic cells ( DCs ) , and endothelial cells ( ECs ) , produce PTX3 in a gene expression-dependent fashion [ 4 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Thus , circulating and lung myelomonocytic cells and endothelial cells are a major source of PTX3 , and PTX3 plasma concentration can serve as an independent strong prognostic indicator of short-term mortality in COVID-19 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "Thus , circulating and lung myelomonocytic cells and endothelial cells are a major source of PTX3 , and PTX3 plasma concentration can serve as an independent strong prognostic indicator of short-term mortality in COVID-19 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(HGNC:9692 ! PTX3)\n",
      "In these tumors , infiltrating macrophages and endothelial cells were the major source of PTX3 in response to locally produced IL-1 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D002630 ! \"Chemotactic Factors\")\n",
      "Hence , viral-induced damage 372 exposes tissue factors on endothelial cells which are activated by persistent cytokines to produce 373 chemoattractants in recruiting leukocytes to endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D003142 ! Communication)\n",
      "Transcytosis in the endothelial cells ( van Deurs et al ., 2003 ; Navarro et al ., 2004 ) is one of the fundamental processes by which the endothelial cells contribute to the communication between blood and tissues .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "Complement proteins can also be produced by epithelial cells ( 59 ) , endothelial cells ( 60 ) , and circulating immune cells such as dendritic cells , granulocytes , macrophages , and monocytes ( 61 , 62 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "The formation of NETs by neutrophils are aided by activated platelets associated with damaged endothelial cells that further activate the complement , fuelling the coagulation cascade and thrombi formation .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D004211 ! \"Disseminated Intravascular Coagulation\")\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D004777 ! Environment)\n",
      "As the first responders to sterile inflammatory injury , it is important to understand how endothelial cells , neutrophils and platelets help create this environment .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005334 ! Fever)\n",
      "It further recruits inflammatory cells to the site of infection and stimulates the synthesis of prostaglandin E2 from endothelial cells , which binds to the receptor EP3 in neural cells to generate fever .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005347 ! Fibroblasts)\n",
      "Alveolar macrophages are also thought to produce increased amounts of interleukin-8 , a powerful chemokine for neutrophils which are regularly found in the BAL of patients with scleroderma Mastocytes and their mediators are also thought to play a role in the pathogenesis of the disease , together with endothelin-1 , a factor produced by endothelial cells which directly stimulates the fibroblasts .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005355 ! Fibrosis)\n",
      "It is suggested that endothelial cells may contribute to fibrosis in two ways : by becoming dysfunctional and by differentiating into highly fibrogenic mesenchymal cells referred to as myofibroblasts [ 11 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D005355 ! Fibrosis)\n",
      "Trans-signaling by endothelial cells may result in increased vasculopathy , fibrosis , endothelial dysfunction , vascular permeability and disseminated intravascular coagulation [ 28 ] , [ 33 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D006470 ! Hemorrhage)\n",
      "Its pathophysiology involves endothelial dysfunction and angiogenesis related to ACE-2 receptor , the host-cell receptor for SARS-CoV-2 , expressed by endothelial cells , which may lead to thrombosis or hemorrhage [ 1 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D006973 ! Hypertension)\n",
      "A probable mechanism might be viral effects on the ACE-2 receptors on the endothelial cells and platelets that can lead to hypertension and hyper-coagulable states respectively .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D007109 ! Immunity)\n",
      "These vessels are lined with endothelial cells , which sit at the interface between blood and tissues and contribute to haemostasis , immunity and inflammatory reactions10 ,11 ( Fig. 1 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009369 ! Neoplasms)\n",
      "In these tumors , infiltrating macrophages and endothelial cells were the major source of PTX3 in response to locally produced IL-1 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009389 ! \"Neovascularization, Pathologic\")\n",
      "Our experiment in vivo demonstrated that the increased protein expression of VEGF and its receptor Flt-1 as well as neovascularization in atherosclerotic plaques were reduced by TACE gene silencing , which lent support to a recent study that pathological neovascularization was attenuated by inactivation of TACE in endothelial cells 36 , 37 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D009504 ! Neutrophils)\n",
      "As a result , endothelial cells stimulate neutrophils to form NETs .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D010959 ! \"Tissue Plasminogen Activator\")\n",
      "Tissue plasminogen activator ( tPA ) is produced by endothelial cells and , as its name suggests , activates plasminogen .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D010959 ! \"Tissue Plasminogen Activator\")\n",
      "Endothelial cells can also synthesize and release tissue plasminogen activator that converts plasminogen to plasmin , thereby activating fibrinolysis and clot dissolution221-223 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Its pathophysiology involves endothelial dysfunction and angiogenesis related to ACE-2 receptor , the host-cell receptor for SARS-CoV-2 , expressed by endothelial cells , which may lead to thrombosis or hemorrhage [ 1 ] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Recent studies have indicated that annexin A2 may be a target for autoantibodies in SLE , SS , RA , and systemic vasculitis , and it has been hypothesized that anti-annexin A2 antibodies can induce thrombosis by activating endothelial cells and inhibiting plasmin generation323334 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Thus , the complement system can also induce thrombosis by inducing endothelial cells to release von Willebrand factor and express P-selectin ( Table 1 , Fig. 1 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") decreases bp(MESH:D013927 ! Thrombosis)\n",
      "EC dysfunction disrupts the balance between vasoconstriction and vasodilation and initiates a number of events that trigger EC activation and predispose the vessel wall to increased endothelial permeability, leukocyte adherence, endothelial proliferation, pro-oxidation and thrombosis.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D014024 ! Tissues)\n",
      "In particular , C5a promotes procoagulant activity through several mechanisms , including the induction of tissue factor by endothelial cells and neutrophils , as well as the upregulation of plasminogen activator inhibitor-1 in mast cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015603 ! Keratinocytes)\n",
      "In addition to keratinocytes , 52 endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015603 ! Keratinocytes)\n",
      "In addition to keratinocytes , endothelial cells , macrophages , mast cells and basophils are other cellular sources of IL-33.12 , 14 IL-33 is stored preformed in the nucleus of these cells and produced readily to exert its inflammatory effects .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "This in turn activates the endothelial cells to upregulate cell adhesion molecules ( see Ch .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "Endothelial cells upregulate cell adhesion molecules when exposed to a number of mediators , including leukotriene B 4 , as well as cytokines such as tumor necrosis factor and interleukins 1e8 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D015815 ! \"Cell Adhesion Molecules\")\n",
      "Endothelial cells upregulate cell adhesion molecules when exposed to a number of mediators , including leukotriene B4 , as well as cytokines such as tumor necrosis factor and interleukins 1-8 .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D016766 ! \"Feline Infectious Peritonitis\")\n",
      "Therefore, the present work focused on the expression of the EC adhesion molecules and possible triggers of EC activation during the development of FIP.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "As COVID-19 is not a hemorrhagic disease , it is probable that ADE , if present , is not mediated by endothelial cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019281 ! Dimerization)\n",
      "It is estimated that endothelial cells can cleave b 2 GPI monomers and promote their dimerization .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D019281 ! Dimerization)\n",
      "It is estimated that endothelial cells can cleave beta2GPI monomers and promote their dimerization .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "Restoration of EC specific FoxM1 expression in Pik3cg global knockout mice promoted EC regeneration and normalized the repair process.\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "MCP-1 is also produced by macrophages , fibroblasts , epithelial cells and endothelial cells [ 73 ] , so IL-1beta and MCP-1 was rarely detected in Jurkat cells .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Endothelial cells also produce IL-8 , as well as MCP-1 , in response to IL-6 complexed to the soluble receptor ( sIL-6Ra ) , which binds the cell surface receptor , gp130 ( Romano et al ., 1997 ) .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") increases p(UP:P0CW71 ! P0CW71)\n",
      "Incubation of EC with IgG, from patients positive for aPL, increased the expression of VCAM-1, E-selectin, and MCP-1 as previously reported for AECA [125] .\n",
      "\n",
      "bp(MESH:D042783 ! \"Endothelial Cells\") partOf complex(a(CHEBI:53571 ! \"poly(acrylonitrile) macromolecule\"), bp(MESH:D042783 ! \"Endothelial Cells\"))\n"
     ]
    },
    {
     "ename": "KeyError",
     "evalue": "'evidence'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-10-06775ad83515>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0memmaa_covid_graph\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0medge_to_bel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mu\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mv\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'evidence'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m: 'evidence'"
     ]
    }
   ],
   "source": [
    "for u, v, data in emmaa_covid_graph.edges(endothelial, data=True):\n",
    "        print()\n",
    "        print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "        print(data['evidence'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11876 ! TMPRSS2)\n",
      "More recently, a specific SARS-CoV-2 A2a subtype mediated TMPRSS2 and MX1 genes up-regulation has been described in European and North American populations (22).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11876 ! TMPRSS2)\n",
      "Given the fact that SARS-CoV-2 infection does not increase expressions of ACE2 and TMPRSS2, medications induced upregulation of these receptors in tissues with low baseline expression, such as nerve tissue, could increase exposure of these sites to viral infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11876 ! TMPRSS2)\n",
      "In fact, a recent study elegantly demonstrated that host cell 174 entry of SARS-CoV-2 can be blocked by a clinically proven inhibitor of TMPRSS2 indicating the critical 175 importance of TMPRSS2 in determining SARS-CoV-2 infectivity 2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11876 ! TMPRSS2)\n",
      "In fact, a recent study elegantly demonstrated that host cell 173 entry of SARS-CoV-2 can be blocked by a clinically proven inhibitor of TMPRSS2 indicating the critical 174 importance of TMPRSS2 in determining SARS-CoV-2 infectivity 2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:7532 ! MX1)\n",
      "More recently, a specific SARS-CoV-2 A2a subtype mediated TMPRSS2 and MX1 genes up-regulation has been described in European and North American populations (22).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACE receptors\")\n",
      "It is still unclear if SARS-CoV-2 can block the entry of DENV by inhibiting the ACE receptors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACE2 receptor\")\n",
      "Indeed, SARS-CoV-2 viral entry might trigger cleavage of ACE2 receptor ectodomain, affecting ACE2 function and systemic release.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACE2 receptor\")\n",
      "SARS-CoV2 has been shown to mediate the downregulation of the ACE2 receptor, with resultant associated overactivation of the renin-angiotensin-aldosterone system (RAAS) [4].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE2 receptor\")\n",
      "SARS-CoV-2 enters the cell by membrane diffusion and slows down the regulation of ACE2 receptor.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACE2 receptors\")\n",
      "Similarly, SARS-CoV-2 also targets ACE2 receptors on human respiratory epithelial cells, suggesting a similar RBD structure to SARS-CoV-1 [5, 11].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACE2 receptors\")\n",
      "The potential ocular presence of the SARS-CoV-2 in the eye of a patient can target ACE2 receptors in the endothelium of the conjunctival vessels and use the lacrimal sac a potential space to evade immune detection and clinical isolation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE2 receptors\")\n",
      "However, the entry of SARS-CoV-2 in the cells by membrane fusion down-regulates ACE2 receptors, thus SARS-CoV-2 seems to entry into the cell with the membrane receptor, which is functionally detached from the membrane external site.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE2 receptors\")\n",
      "Similarly, it could be hypothesized that SARS-CoV-2 may downregulate ACE2 receptors and thereby leading to an over-production of AngII, which may be another possible explanation of cytokine storm in COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE2 receptors\")\n",
      "(64) Moreover, down-regulation of ACE2 receptors by SARS-CoV-2 leads to inflammation and multi-organ failure.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE2 receptors\")\n",
      "[64] Moreover, down-regulation of ACE2 receptors by SARS-CoV-2 leads to inflammation and multi-organ failure.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:ACE2-ATII)\n",
      "As a result, a chronic low-grade inflammation is present, which excellerates due to SARS-CoV-2 infection induced disbalance of ACE2-ATII.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ACUTE NASAL DRYNESS IN COVID-1926\")\n",
      "Running head : ACUTE NASAL DRYNESS IN COVID-1926 Introduction 27 In 2020, a new disease, known as COVID-19, caused by an infection of SARS-CoV-2, 28 has spread globally.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:BBB)\n",
      "Although it is too early to elucidate the long-term side effects of SARS-CoV-2 infection, the background obtained with other respiratory viruses suggests that SARS-CoV-2 might induce permanent sequelae in the CNS through damage to the BBB, including dementia in predisposed patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:BNT162b2)\n",
      "Nevertheless, the binding and structural analyses indicate 103 that the BNT162b2 RNA sequence encodes a recombinant P2 S that authentically presents the 104 ACE2 binding site and other epitopes targeted by SARS-CoV-2 neutralising antibodies.105 To characterise BNT162b2 elicited B- and T-cell responses, BALB/c mice were immunized 106 intramuscularly (IM) once with 0.2, 1, or 5 microg BNT162b2 or received a buffer control.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CC-BY-NC 4.0 International license.27 Abstract28\")\n",
      "It is made available under a CC-BY-NC 4.0 International license.27 Abstract28 The COVID-19 pandemic caused by the SARS-CoV-2 requires a fast development of 29 antiviral drugs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CC-BY-NC-ND 4.0 International license .2 BackgroundCoronavirus Disease-2019\")\n",
      "The copyright holder for this preprint (which was not certified by peer review) is the author and funder, who has granted medRxiv a license to display the preprint in perpetuity.It is made available under a CC-BY-NC-ND 4.0 International license.2 BackgroundCoronavirus Disease-2019 (COVID-19) is caused by the novel betacoronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CC-BY-NC-ND 4.0 International license .2 BackgroundCoronavirus Disease-2019\")\n",
      "The copyright holder for this preprint (which was not certified by peer review) is the author and funder, who has granted medRxiv a license to display the preprint in perpetuity.It is made available under a CC-BY-NC-ND 4.0 International license.2 BackgroundCoronavirus Disease-2019 (COVID-19) is caused by the novel betacoronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "The SARS-CoV-2 90 infection induces antiviral CD4+ T cells, helping the production of neutralizing and blocking antibodies and 91 the formation of effector IFN-gamma-producing CD4+ T cells and cytotoxic CD8+ T cells, all arms of immunity 92 critical in reducing viral load in the majority of asymptomatic and convalescence patients 26, 27, 28, 29, 30, 93 31.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "Broad and strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in 415 UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "A trend has been observed where, SARS-CoV-2 infection rapidly activates CD4+ T lymphocytes thereby forming pathogenic T helper (Th1) cells and release various cytokines including GM-CSF.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "Broad and strong memory CD4+ and CD8+ T 389 cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "No reuse allowed without permission.141 strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent 142 COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD4+\")\n",
      "Ho, Oxford Immunology Network Covid-19 Response T cell Consortium, 16 ISARIC4C Investigators, R. J. Cornall, C. P. Conlon, P. Klenerman, G. R. Screaton, J. 17 Mongkolsapaya, A. McMichael, J. C. Knight, G. Ogg, T. Dong, Broad and strong memory 18 CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following 19 COVID-19, Nat Immunol 21, 1336-1345 (2020).20 26.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"CD4+\")\n",
      "It has been shown that SARS-CoV-2 infection causes overall reduction in the CD4+ and CD8+ T cell counts in peripheral blood (3, 4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"CD4+\")\n",
      "The question arises whether SARS-CoV-2, like HIV, can directly decrease CD4+ T cell count.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD8+\")\n",
      "Broad and strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in 415 UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD8+\")\n",
      "Broad and strong memory CD4+ and CD8+ T 389 cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD8+\")\n",
      "No reuse allowed without permission.141 strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent 142 COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"CD8+\")\n",
      "Ho, Oxford Immunology Network Covid-19 Response T cell Consortium, 16 ISARIC4C Investigators, R. J. Cornall, C. P. Conlon, P. Klenerman, G. R. Screaton, J. 17 Mongkolsapaya, A. McMichael, J. C. Knight, G. Ogg, T. Dong, Broad and strong memory 18 CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following 19 COVID-19, Nat Immunol 21, 1336-1345 (2020).20 26.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"CD8+\")\n",
      "It has been shown that SARS-CoV-2 infection causes overall reduction in the CD4+ and CD8+ T cell counts in peripheral blood (3, 4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:CM)\n",
      "In addition, we observed markers indicating cardiac injury both in hamsters and in clinical COVID-19 samples, suggesting that SARS-CoV-2 infection can cause CM damage.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 42 pandemic\")\n",
      "It is not 35 known whether SARS-CoV-2 antibodies will hinder DV infections by binding to DV 36 particles and reduce dengue incidences in the future or, augment DV infection and severity 37 by deploying antibody dependent enhancement.38 39Since the beginning of 2020, people around the world are confronting the COVID-19 42 pandemic, caused by SARS-CoV-2, a beta coronavirus.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 disease\")\n",
      "With emergence and pandemic spread of new human coronaviruses (HCoVs), SARS-CoV-2 (the cause of the COVID-19 disease), which is the origin of the serious and occasionally fatal respiratory complaints in human, scientists are trying to focus more on different aspects of COVID-19 disease to find some strategies to improve the management and design specific therapeutic programs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 outbreak\")\n",
      "SARS-CoV-2 emerged in 2019 and caused the COVID-19 outbreak.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 outbreak\")\n",
      "The COVID-19 outbreak, caused by Sars-Cov-2, was officially declared a global pandemic in February 2020, after an unexpected increase in hospitalization and mortality.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 outbreak\")\n",
      "To summarize, MP1032 was successfully evaluated in the clinical setting, showing an exceptional safety profile with good tolerability.The World Health Organization declared the COVID-19 outbreak, caused by SARS-CoV-2, a pandemic on 11 March 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 outbreak\")\n",
      "The World Health Organization declared the COVID-19 outbreak, caused by SARS-CoV-2, a pandemic on 11 March 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 pneumonia\")\n",
      "COVID-19 pneumonia has caused by SARS-CoV-2, as the seventh known CoV, that infects humans (Corman et al., 2019a).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 pneumonia\")\n",
      "The SARS-CoV-2 viral infection causes COVID-19 pneumonia.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"COVID-19 pneumonia.Pandemic coronavirus disease 2019\")\n",
      "Tocilizumab and steroid treatment in patients with COVID-19 pneumonia.Pandemic coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 coronavirus has recently emerged [1].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:CS)\n",
      "XREF_BIBR, XREF_BIBR Several antiviral and anti-inflammatory drug therapies have been proposed for treating SARS-CoV-2 mediated CS, in order to decrease both the morbidity and mortality in COVID-19 patients, and a comprehensive review and critical appraisal of these therapies has been made in this review.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:CS)\n",
      "67, 72 Several antiviral and anti-inflammatory drug therapies have been proposed for treating SARS-CoV-2 mediated CS, in order to decrease both the morbidity and mortality in COVID-19 patients, and a comprehensive review and critical appraisal of these therapies has been made in this review.Presently, there are no definitive therapeutic protocols for COVID-19 management.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Cellular413 ImmuneResponsewithoutSeroconversion.414 https://www.medrxiv.org/content/10.1101/2020.06.21.20132449v1 2020415 25\")\n",
      "Intrafamilial Exposure to SARS-CoV-2 Induces Cellular413 ImmuneResponsewithoutSeroconversion.414 https://www.medrxiv.org/content/10.1101/2020.06.21.20132449v1 2020415 25.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Corona Virus Disease\")\n",
      "A global outbreak of the SARS-CoV-2 caused Corona Virus Disease (COVID-19) seems inevitable.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Corona Virus Disease\")\n",
      "A novel Corona Virus Disease (COVID-19) caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has occurred in Wuhan, Hubei, China since December 2019 [1].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Corona Virus Disease\")\n",
      "A novel Corona Virus Disease (COVID-19) caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has occurred in Wuhan, Hubei, China since December 2019 [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Corona Virus Disease\")\n",
      "Severe respiratory syndrome SARS-COV-2 leads to Corona Virus Disease (COVID-19) and has the capability of transmission through human-to-human and surface-to-human transmission leads the world to catastrophic phase.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Corona Virus Disease-2019\")\n",
      "The novel virus, Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) caused the Corona Virus Disease-2019 (COVID-19) outbreak in Wuhan, Hubei province of China.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Coronavirus Disease (COVID) - 19)2\")\n",
      "It is caused by a novel beta coronavirus, the Severe Acute Respiratory Syndrome - Coronavirus - 2 (SARS-CoV-2), which causes Coronavirus Disease (COVID) - 19) 2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Coved-19)\n",
      "CONCLUSIONSIn this pilot study, we have tested five representative conjugated oligomers and polymers from an array of phenylene ethynylene based cationic and anionic conjugated materials against SARS-CoV-2, the virus that causes the Coved-19 disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Covid-19 Pandemic.The Coronavirus disease 2019\")\n",
      "A Case Study of Covid-19 Pandemic.The Coronavirus disease 2019 (Covid-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a highly contagious disease affecting huge numbers of people all over the world.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Covid-19 disease\")\n",
      "SARS-CoV-2, the virus which causes the Covid-19 disease, is a zoonotic pathogen which emerged in Wuhan in late 2019.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Cytopathic Effect\")\n",
      "Chemical Structure and Activity of Clioquinol (CLQ) and Analogues against SARS-CoV-2 induced Cytopathic Effect (CPE) in Vero E6 Cells.Inhibitor ID Screen IDChemical StructureIC50 (microM) Maximum Inhibition at 30microM (%) CLBQ14 MDXC19T00114.69102.96 CLCQMDXC19T00216.3089.78 CLQ MDXC19T00312.6291.78 Table 2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Cytopathic Effect\")\n",
      "Efficacy of Clioquinol (CLQ) and Analogues against SARS-CoV-2 induced Cytopathic Effect (CPE) in Vero E6 cells : A. CLBQ14, B. CLCQ, and C. CLQ.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"Cytopathic Effect\")\n",
      "Efficacy of Reference Inhibitors against SARS-CoV-2 induced Cytopathic Effect (CPE) in Vero E6 cells : A. CalpainInhibitorIV, B. Chloroquine, C Remdesivir, D. Hydroxychloroquine and E. E64d (Aloxistatin).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"Cytopathic Effect\")\n",
      "Chemical Structure and Activity of Reference Inhibitors against SARS-CoV-2 induced Cytopathic Effect (CPE) in Vero E6 Cells.Inhibitor IDScreen ID Chemical StructureIC50 (microM) Maximum Inhibition (%) Concentration at Maximum% Inhibition (microM) CalpainInhibitorIV AB01968659ChloroquineAB000534360.41 95.75 1.10 111.087.17 7.50 RemdesivirAB01952209Hydroxychloroquine AB00053257E64d (Aloxistatin) AB019554114.42 100.275.04 93.1416.7298.5515.00 30.00 30.00 Table 3.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02109 ! DAD)\n",
      "COVID-19 group (n = 6) comprises lung samples from postmortem biopsies of patients whose cause of death was DAD caused by SARS-CoV-2 during the 2020 outbreak.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02109 ! DAD)\n",
      "Therefore, the use of chloroquine and hydroxychloriquein reduces the DAD caused by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF05384 ! DEGs)\n",
      "The list of DEGs related to the coagulation pathway in cells treated with SARS-CoV-2 and H1N1Cell line-virus-MOI A549-SARS-CoV-2-MOI2 Calu3-SARS-CoV-2-MOI2 NHBE-SARS-CoV-2-MOI2 A549-H1N1-MOI5 NHBE-H1N1-MOI5Up-regulated (LogFC) SERPINE1 (4.11) SERPINA5 (1.63) SERPINB2 (1.24) PLAUR (1.33); TFPI (0.92) SERPING1 (1.28) Down-regulated (LogFC) --------- PROC (-1.31) ---- PPI networkThe identification of proteins that interact directly with DEGs involved in the coagulation pathway could help understand the molecular mechanism of hypercoagulation in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF05384 ! DEGs)\n",
      "GO and KEGG pathway analysis of all up-(A) and downregulated (B) genes in response to SARS-COV-2 infection.Pie chart of top miRNAs that target DEGs in response to SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Feb 11\")\n",
      "On Feb 11, 2020, WHO officially announced the disease caused by SARS-CoV-2 as \" Coronavirus Disease 2019 \" (COVID-19).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Feb 11\")\n",
      "On Feb 11, 2020, WHO officially announced the disease caused by SARS-CoV-2 as \" Coronavirus Disease 2019 \" (COVID-19) (4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Feb 11\")\n",
      "On Feb 11, 2020, WHO officially announced the disease caused by SARS-CoV-2 as \" Coronavirus Disease 2019 \" (COVID-19) (4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:GapmeRs)\n",
      "While designing SARS-CoV-2 targeting GapmeRs, we have considered multiple parameters, including RNA secondary structure, accessibility of target region, and avoidance of off-target effects using robust bioinformatics algorithms and in silico analysis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"High-Dose Intravenous Immunoglobulins\")\n",
      "High-Dose Intravenous Immunoglobulins in the Treatment of Severe Acute Viral Pneumonia : The Known Mechanisms and Clinical Effects.The current outbreak of viral pneumonia, caused by novel coronavirus SARS-CoV-2, is the focus of worldwide attention.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:IDO1-AhR-IDO1)\n",
      "SARS-CoV-2 induced IDO1-AhR-IDO1 loop might be exponentially increased in obese COVID-19 patients, and probably inhibited by vitamin D for AhR, and vitamin E for IDO1.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:IFN-I)\n",
      "Severe SARS-CoV-2 infection induces a low IFN-I and III response, providing a potential mechanism for the swift replication of SARS-CoV-2 and viremia in moribund COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:IFN-I)\n",
      "Severe SARS-CoV-2 infection induces a low IFN-I and III response (Blanco-Melo et al., 2020; Bost et al., 2020; Hadjadi et al., 2020), providing a potential mechanism for the swift replication of SARS-CoV-2 and viremia in moribund COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:IFN-III)\n",
      "Transcriptome profiling of certain cell types reveals that SARS-CoV-2 infection elicits exceptionally low INF-I and IFN-III and reduced ISG while inducing pro inflammatory chemokine and cytokine genes (30).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"IMPORTANCE  Syrian hamsters\")\n",
      "IMPORTANCE Syrian hamsters are in use as a model of disease caused by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:INF-I)\n",
      "Transcriptome profiling of certain cell types reveals that SARS-CoV-2 infection elicits exceptionally low INF-I and IFN-III and reduced ISG while inducing pro inflammatory chemokine and cytokine genes (30).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"INTRODUCTION COVID-19\")\n",
      "INTRODUCTION COVID-19, caused by SARS-CoV-2, is one of the most important healthcare concerns worldwide.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:INTRODUCTION)\n",
      "INTRODUCTION The pandemic caused by the SARS-CoV-2 has created new challenges in health-care systems worldwide (1).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:INTRODUCTION)\n",
      "INTRODUCTION The COVID-19 pandemic caused by SARS-CoV-2 threatens global public health, and there is an urgent public health need to assess acquired immunity to SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ISG 511\")\n",
      "However, like MERS-CoV and MHV, SARS-CoV-2 induces limited and late IFN and ISG 511 responses, indicating that proteins antagonizing innate immune responses are likely encoded.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ISG mRNA\")\n",
      "Similarly, SARS-CoV-2 generally induced more ISG mRNA than 275 WT MERS-CoV, and even more OAS2 mRNA than MERS-DNS4ab (Fig 7A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ISG mRNA\")\n",
      "No reuse allowed without permission.416 417 In A549ACE2 cells, SARS-CoV-2 induced low levels of IFNl and IFNb mRNAs and somewhat 418 higher ISG mRNA by 48 hpi, as compared with SINV (Fig 6A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"IgG antibody\")\n",
      "Whether SARS-CoV-2 induced polyclonal IgG antibody responses 504 to SARS-CoV-1 and MERS-CoV spike proteins are retained after affinity maturation, or are 505 cross neutralizing requires further investigation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"IgG antibody\")\n",
      "Whether SARS-CoV-2 induced polyclonal IgG antibody responses 505 to SARS-CoV-1 and MERS-CoV spike proteins are retained after affinity maturation, or are 506 cross neutralizing requires further investigation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Interkeukin-6)\n",
      "Interkeukin-6, which production may be increased by SARS-CoV-2, plays a potential role in this syndrome.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Liu G et al.\")\n",
      "Peng Y, Mentzer AK, Liu G et al. (2020) Broad and strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"M1/M2\")\n",
      "154 155 Macrophage polarization and SARS-CoV-2 infection 156 As SARS-CoV-2 induced an early M1/M2 followed by a late M2 program in macrophages, 157 we investigated the effect of macrophage polarization status on infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"M1/M2\")\n",
      "153 154 Macrophage polarization and SARS-CoV-2 infection 155 As SARS-CoV-2 induced an early M1/M2 followed by a late M2 program in macrophages, 156 we investigated the effect of macrophage polarization status on infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:MIS-C)\n",
      "XREF_BIBR Until now it has yet to be determined that SARS-CoV-2 is the cause of MIS-C or KD.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Martin)\n",
      "Indeed, the immune homeostasis is disrupted by SARS-CoV-2 leading to declined responsiveness of host immune regulatory system, macrophage function, and alterations in lymphocyte subsets (Merad and Martin, 2020; Wan et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "2 : WBC infiltration and subsequent CNS damage mediated by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "If the mechanisms by which SARS-CoV-2 infection primarily causes an immune response or immune cell infiltration can be identified, it may be possible to mitigate excessive immune responses by modulating the infiltration and activation of specific targets, thereby reducing or preventing severe lung damage.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "The increased infiltration of naive B cells activated by spike proteins of SARS-CoV-2 secretes large amounts of IgM to promote humoral immune response [65, 69].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Neoplasm Invasiveness\")\n",
      "If the mechanism by which SARS-CoV-2 infection primarily elicits the immune system response and immune cell infiltration can be found, it would be possible to alleviate the extent of ARDS and lung fibrosis by modulating the infiltration and activation of specific immune cells to attenuate the excessive immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"OC43 S-reactive antibodies\")\n",
      "Further studies are needed to precisely map the footprints of OC43 S-reactive antibodies elicited by SARS-CoV-2 infections.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:OD)\n",
      "This supports the thesis that GD and OD in COVID-19 present two independent symptoms caused by SARS-CoV-2 and that GD is not a secondary consequence of OD.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(IP:IPR004577)\n",
      "Ho, Oxford Immunology Network Covid-19 Response T cell Consortium, 16 ISARIC4C Investigators, R. J. Cornall, C. P. Conlon, P. Klenerman, G. R. Screaton, J. 17 Mongkolsapaya, A. McMichael, J. C. Knight, G. Ogg, T. Dong, Broad and strong memory 18 CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following 19 COVID-19, Nat Immunol 21, 1336-1345 (2020).20 26.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"P1-, P2-,\")\n",
      "P1-, P2-, and P3-pDC were all significantly induced by SARS-CoV-2 and Flu, as compared to medium control (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:P3-pDC)\n",
      "P1-, P2-, and P3-pDC were all significantly induced by SARS-CoV-2 and Flu, as compared to medium control (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"SARS-CoV-2 16\")\n",
      "Calu-3 cells are a particularly attractive model for studying SARS-CoV-2, as they are derived from lung epithelia, which is selectively infected by SARS-CoV-2 in patients 15, are known to support infection of SARS-CoV and SARS-CoV-2 16 and have recently been reported to closely recapitulate gene expression changes that occur in patients 17.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"SARS-CoV-2 neutralizing antibody\")\n",
      "The overwhelming majority of other SARS-CoV-2 vaccines in development target only the S antigen and are expected to elicit SARS-CoV-2 neutralizing antibody responses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"SARS-CoV-2 replication.INTRODUCTIONThe coronavirus disease 2019\")\n",
      "Taken together, our findings demonstrated that the host m6A modification complex interacted with viral proteins to modulate SARS-CoV-2 replication.INTRODUCTIONThe coronavirus disease 2019 (COVID-19) pandemic is caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), which belongs to the genus Betacoronavirus in the Coronavirinaesubfamily of the Coronaviridae family (1-3).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"SARS-CoV-2 replication.INTRODUCTIONThe coronavirus disease 2019\")\n",
      "Taken together, our findings demonstrated that the host m6A modification complex interacted with viral proteins to modulate SARS-CoV-2 replication.INTRODUCTIONThe coronavirus disease 2019 (COVID-19) pandemic is caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), which belongs to the genus Betacoronavirus in the Coronavirinaesubfamily of the Coronaviridae family (1-3).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Spread)\n",
      "Spread mainly through respiratory droplets or close contact, SARS-CoV-2 induced diseases has been growing dramatically in accordance with the published data from the World Health Organization (WHO) XREF_BIBR.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Syndrome Coronavirus 2\")\n",
      "No reuse allowed without permission.105 Introduction 106 The Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) causes 107 the highly communicable disease COVID-19, the ongoing 2019-2020 disease 108 outbreak which has turned into a pandemic.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"T-helper-2 (Th2) cytokines\")\n",
      "However, SARS-CoV-2 is shown to increase the T-helper-2 (Th2) cytokines (IL4 and IL10) that are known to suppress inflammation, which differs from SARs and MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:TMPSS2)\n",
      "50% of studies exclusively targeted 3CLpro of SARS-CoV-2, and only two studies targeted ACE and TMPSS2 for inhibiting host viral interactions.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Topoisomerase 1\")\n",
      "Here, using multidimensional epigenetic, 64 transcriptional, in vitro and in vivo analyses, we report that Topoisomerase 1 (Top1) inhibition 65 suppresses lethal inflammation induced by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02512 ! UK)\n",
      "Broad and strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in 415 UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02512 ! UK)\n",
      "Broad and strong memory CD4 + and CD8 + T cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02512 ! UK)\n",
      "Broad and strong memory CD4+ and CD8+ T 389 cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02512 ! UK)\n",
      "Peng, Y. et al., Broad and strong memory CD4 + and CD8 + T cells induced 314 by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF02512 ! UK)\n",
      "Ho, Oxford Immunology Network Covid-19 Response T cell Consortium, 16 ISARIC4C Investigators, R. J. Cornall, C. P. Conlon, P. Klenerman, G. R. Screaton, J. 17 Mongkolsapaya, A. McMichael, J. C. Knight, G. Ogg, T. Dong, Broad and strong memory 18 CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following 19 COVID-19, Nat Immunol 21, 1336-1345 (2020).20 26.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:aminotransferase)\n",
      "Although SARS-CoV-2 infection can cause aminotransferase elevation per se, 4 of our 5 patients had normal or slightly elevated AST and ALT levels at RDV treatment start, suggesting a direct role of RDV in hepatocellular toxicity.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"angiotensin- [\")\n",
      "For example, recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin converting enzyme-2 (ACE-2) downregulation due to the binding of the viral spike protein to its receptor, then promoting a decrease in the angiotensin- [XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR] production [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"cysLT 166\")\n",
      "Here, we demonstrated that SARS-CoV-2 infection increased LTB4 and cysLT 166 production in comparison with uninfected monocytes (Fig 3D).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:dACE2)\n",
      "dACE2 is induced by SARS-CoV-2 in human cell lines and organoid cultures.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"effector proteins\")\n",
      "Integration of plasma proteomics with nine published scRNAseq datasets shows that SARS-CoV-2 infection upregulates monocyte and macrophage, plasmablast, and T cell effector proteins.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"flu-like symptoms\")\n",
      "Since SARS-CoV-2 infection causes flu like symptoms, the accurate diagnosis of such infections is essential to control the dissemination of the virus within a population and beyond the affected people.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"flu-like symptoms\")\n",
      "Although SARS-COV-2 only produces mild flu like symptoms in the majority of patients affected, the virus may lead to severe or even lethal complications, such as acute respiratory distress syndrome and multiorgan dysfunction.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"flu-like symptoms\")\n",
      "We investigate the epistemological consequences of a positive polymerase chain reaction SARS-CoV test for two relevant hypotheses : (i) V is the hypothesis that an individual has been infected with SARS-CoV-2; (ii) C is the hypothesis that SARS-CoV-2 is the cause of flu like symptoms in a given patient.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"flu-like symptoms\")\n",
      "The Epistemology of a Positive SARS-CoV-2 Test.We investigate the epistemological consequences of a positive polymerase chain reaction SARS-CoV test for two relevant hypotheses : (i) V is the hypothesis that an individual has been infected with SARS-CoV-2; (ii) C is the hypothesis that SARS-CoV-2 is the cause of flu like symptoms in a given patient.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:gRNAsSARS-CoV-2)\n",
      "No reuse allowed without permission.A Cas13b & gRNAs (liposomes) Vero cells48-72 hrsTransfection with Cas13-BFP & various SARS-CoV-2 targeting gRNAsSARS-CoV-2 virus1hr incubation & wash1hr24hrs48hrsSARS-CoV-2 Infection (0.1 & 0.01 MOI) Viral particles in the supernatant collected 1, 24, & 48hrs postSARS-CoV-2 infectionRealtime PCR Infectivity assayB CMOI 0.1 MOI 0.01 MOI 0.116 14 12 108 6 4 2 0NT1hrs20 1824hrs 1648hrs 14 121086420 NCP1NTNCP1106 1hrs 105 24hrs104 48hrs103102101100 MockNTViral load in the supernatant (RT-PCR) D20 18 16 14 12 108 6 4 2 0MOI 0.11 hrs24hrs48hrsViral titer (TCID50 / ml) E106 105 104 103 102 101 1001hrs 24hrs 48hrsMOI 0.1 NCP1107 106 105 104 103 102 101 100 FMOI 0.01 MockNTNCP11.2 MOI 0.1 1.0 (24h post-infection) 0.80.60.4 (P < 0.0001) 0.20.0 Viral load in the supernatant (RT-PCR) SNNNNpSSCiNCPPPkTP871eNT pool pool pool poolMockNCP Spike NSP7 NSP8SNpSiNPkC7ePpppooNooollTol NSP8 poolViral load in the supernatant (RT-PCR) SNNNNpSSCiNCPPkPTP87e1Viral load in the supernatant (RT-PCR) 20 MOI 0.0118 16 14 12 108 6 4 2 0Viral titer (TCID50 / ml) 107 MOI 0.01106 105 104 103 102 101 100Pool gRNAs1.2 MOI 0.01 1.0 (24h post-infection) 0.8 0.6 0.4 (P < 0.0001) 0.2 0.0 NT pool pool pool poolMockNCP Spike NSP7 NSP8SNpSiNPkC7ePpppooNooollTol NSP8 poolPool gRNAs D614G targeting gRNAsRelative expressionNT 5 10 15 20 25 30Figure 5bioRprxeivprpinret p (wrinhticdhowi : ahsttpnso : t//cdeorit.iofiregd/1b0y.1 p1e0e1r/2re0v2ie0w.1) 1i.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:gastrointestinal)\n",
      "However, there is an increasing appreciation of the ability of SARS-CoV-2 to produce a wide range of symptoms, 1 including GI (gastrointestinal) symptoms like diarrhea, nausea, vomiting, and abdominal pain.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:hBMVECs)\n",
      "In conjunction with reduced barrier tightness our finding that SARS-CoV-2 activates hBMVECs strongly indicates the potential for enhanced immune infiltration into the CNS.Endothelial activation also features increased expression of matrix metalloproteinase or MMPs, a family of enzymes involved in the remodeling of extracellular matrix in both normal J o u r n a l P r e -p r o o f physiological and pathological processes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"huge protected receptor-binding domain\")\n",
      "On the behalf of array arrangement and similar design, SARS-CoV and SARS-CoV-2 contribute to a huge protected receptor binding domain (RBD), a domain of S protein, and 76% of array resemblance in their S proteins [XREF_BIBR - XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"immune546 evasionthroughpotentlydownregulatingMHC-1\")\n",
      "The ORF8 protein of SARS-CoV-2 mediates immune546 evasionthroughpotentlydownregulatingMHC-1.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:infiltration)\n",
      "The new SARS-CoV-2 additionally causes serious alveolar mucus infiltration and multiple organ failure.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"interferon-stimulate gene\")\n",
      "SARS-Cov-2 induced a unique transcriptional state within neurons compared to ZIKV -- consistent with the fact that SARS-CoV-2 induces a moderate interferon-stimulate gene (ISG) response in other tissues, and previous reports of specific virus replication being controlled by alternative pathways by neurons.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:mtROS)\n",
      "We also propose that SARS-CoV-2 activates mtROS mediated feedback loops that cause long-term changes in host redox status and endothelial function, promoting cardiovascular disease and lung injury after recovery from COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:mtROS)\n",
      "Furthermore, SARS-CoV-2 infection induced mtROS and HIF-1alpha, subsequently promoted glycolytic genes and cytokines in return.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"multiorgan disease\")\n",
      "The novel coronavirus, SARS-CoV-2, is causing a global pandemic of life threatening multiorgan disease, called COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"neutralizing antibodies 338\")\n",
      "Human neutralizing antibodies 338 elicited by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(PFAM:PF03090 ! replicase)\n",
      "Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the pathogen that causes the disease COVID-19, produces replicase polyproteins 1a and 1ab that contain, respectively, 11 or 16 nonstructural proteins (nsp).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"respiratory syndrome\")\n",
      "SARS-CoV-2 induces a severe acute respiratory syndrome, termed COVID-19, in which immunology is part of the process of clinical evolution consisting of lung tissue damage induced by an inflammatory response, such as a cytokine storm and macrophage and neutrophil activation XREF_BIBR, XREF_BIBR.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"respiratory syndrome\")\n",
      "In the human host, SARS-CoV-2 causes a respiratory syndrome (named COVID-19) which can range from a mild involvement of the upper airways to a severe pneumonia with Acute Respiratory Syndrome (ARS) and the need of mechanical ventilation in an intensive care unit (ICU).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"review Farsalinos et al.) (Farsalinos et al. 2020b\")\n",
      "In fact, up-regulation of ACE2 appears to be protective against tissue damage caused by SARS-CoV-2 (see for review Farsalinos et al.) (Farsalinos et al. 2020b).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"review Farsalinos et al.) (Farsalinos et al. 2020b\")\n",
      "In fact, upregulation of ACE2 appears to be protective against tissue damage caused by SARS-CoV-2 (see for review Farsalinos et al.) (Farsalinos et al. 2020b).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"screen2\f",
      "IntroductionThe coronavirus disease 2019\")\n",
      "These results demonstrate that this comprehensive data set of drug repurposing screen for SARS-CoV-2 is useful for drug repurposing efforts including design of new drug combinations for clinical trials.Keywords : COVID-19, SARS-CoV-2, cytopathic effect, drug repurposing screen2 IntroductionThe coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global health crisis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"threatening infection\")\n",
      "SARS-CoV-2 causes a generally mild, but sometimes severe and even life threatening infection, known as COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:vaccine)\n",
      "The very fact that the discovery and large-scale production of a SARS-CoV-2 targeted vaccine would take a very long time to get into market, makes it important to analyse and research about the existing medicinal drugs which can be utilised for the treatment.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:131170 ! \"(3S)-3-amino-4-[[(2S)-1-[[(2S)-1-[[(2S)-1-[[(2S,3S)-1-[[(2S)-1-[(2S)-2-[[[(1S)-1-carboxy-2-phenylethyl]amino]-oxomethyl]-1-pyrrolidinyl]-3-(1H-imidazol-5-yl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxopentan-2-yl]amino]-3-(4-hydroxyphenyl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxobutan-2-yl]amino]-5-(diaminomethylideneamino)-1-oxopentan-2-yl]amino]-4-oxobutanoic acid\")\n",
      "Furthermore, we also show how the binding of SARS-CoV-2 may trigger a Renin Angiotensin System (RAS) imbalance due to its binding to ACE2, possibly contributing to the pathogenesis of COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:131170 ! \"(3S)-3-amino-4-[[(2S)-1-[[(2S)-1-[[(2S)-1-[[(2S,3S)-1-[[(2S)-1-[(2S)-2-[[[(1S)-1-carboxy-2-phenylethyl]amino]-oxomethyl]-1-pyrrolidinyl]-3-(1H-imidazol-5-yl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxopentan-2-yl]amino]-3-(4-hydroxyphenyl)-1-oxopropan-2-yl]amino]-3-methyl-1-oxobutan-2-yl]amino]-5-(diaminomethylideneamino)-1-oxopentan-2-yl]amino]-4-oxobutanoic acid\")\n",
      "Treatment Options Available for COVID-19 and an Analysis on Possible Role of Combination of rhACE2, Angiotensin (1-7) and Angiotensin (1-9) as Effective Therapeutic Measure.Corona virus disease, caused by SARS-CoV-2, is rapidly spreading all around the world and is posing a threat to mankind.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:140399 ! \"specialised pro-resolving mediator\")\n",
      "In order to determine if SARS-CoV-2 triggered SPM biosynthesis, we measured RvD1 in cell-free supernatants collected from CF MPhi stimulated with S1 protein.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:140399 ! \"specialised pro-resolving mediator\")\n",
      "Moreover, we demonstrate that SARS-CoV-2 initiates the biosynthesis of SPM including RvD1 that, along with and RvD2, control MPhi responses related to NF-kappaB activation and cytokine and chemokine production by targeting miR-16, miR-197, miR-29a, and miR-125a.8 bioRxiv preprint doi : https://doi.org/10.1101/2020.08.28.255463.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15379 ! dioxygen)\n",
      "SARS-CoV-2 can cause CVD by inducing cytokine storms, creating an imbalance in the oxygen supply and demand and disrupting the renin-angiotensin-aldosterone system; SARS-CoV-2 infection can also lead to the development of CVD through the side effects of therapeutic drugs, psychological factors, and aggravation of underlying CVD.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15379 ! dioxygen)\n",
      "Oxygen therapy via high flow nasal cannula in severe respiratory failure caused by Sars-Cov-2 infection : a real-life observational study.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:15379 ! dioxygen)\n",
      "In the severe state, SARS-CoV-2 causes cute Respiratory Distress Syndrome, ie, a lung inflammation that causes fluid accumulation that leads to septic shock due to decreased blood pressure and oxygen starvation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "These results indicate that SARS-CoV-2 induction of the COX-2 and PGE 2 pathway in Calu-3 human lung cells does not regulate viral replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15647 ! \"leukotriene B4\")\n",
      "Here, we demonstrated that SARS-CoV-2 infection increased LTB4 and cysLT 166 production in comparison with uninfected monocytes (Fig 3D).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15843 ! \"arachidonic acid\")\n",
      "AA release is triggered by SARS-CoV-2 and other infections.Figure 2. : Scheme showing potential relationship among AA, PGE2, LXA4 and viral load in a COVID-19 patient who succumbs to the disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:15843 ! \"arachidonic acid\")\n",
      "AA release is triggered by SARS-CoV-2 and other infections.Adipose tissue derived MSCs (ASCs) inhibit secretion of IL-1beta and IL-18 but show increased PGE2 production that, in turn, inhibited inflammasome complex formation and M1 macrophage population and increased regulatory T cells in the DSS induced chronic colitis model suggesting suppression of the inflammatory response [18, 19].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:16750 ! guanosine)\n",
      "BMJ 2020; 369 : m1375 Du, R.H.; Liang, L.R.; Yang, C.Q.; Wang, W.; Cao, T.Z.; Li, M.; Guo, G.Y.; Du, J.; Zheng, C.L.; Zhu, Q.; Hu, M.; Li, X.Y.; Peng, P.; Shi, H.Z. Predictors of mortality for patients with COVID-19 pneumonia caused by SARS-CoV-2 : a prospective cohort study.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:16857 ! L-threonine)\n",
      "In 81 infected male patients of reproductive age , SARS-CoV-2 led to a significant increase in serum LH levels and LH / T ratio , indicating an early stage Leydig cell dysfunction.5 Third , male hypogonadism occurs in 2-15 % of elderly patients ( 40-79 years ) .6 The evaluation of SARS-CoV-2 infection in male untreated hypogonadal patients would be relevant in further confirming the androgen influence on this disease .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:16857 ! L-threonine)\n",
      "In 81 infected male patients of reproductive age , SARS-CoV-2 led to a significant increase in serum LH levels and LH / T ratio , indicating an early stage Leydig cell dysfunction .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:16857 ! L-threonine)\n",
      "In 81 infected male patients of reproductive age , SARS-CoV-2 led to a significant increase in serum LH levels and LH / T ratio , indicating an early stage Leydig cell dysfunction.5 Third , male hypogonadism occurs in 2-15 % of elderly patients ( 40-79 years ) .6 The evaluation of SARS-CoV-2 infection in male untreated hypogonadal patients would be relevant in further confirming the androgen influence on this disease .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:17245 ! \"carbon monoxide\")\n",
      "SARS-CoV-2 causes a disease, better known as COVID-19 (CO for Corona, VI for Virus, D for disease and 19 for the year in which it was identified), with symptoms spanning from mild to serious.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:18361 ! \"diphosphate(4-)\")\n",
      "This creates a new subset of perturbator candidates V PC subset and equalV P which is formed by all those proteins targeted by SARS-CoV-2, which are significantly expressed in the lungs, that can navigate outside this organ via EVs, and that can activate VPs via subdiffusive dynamics in the PPI network.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:24996 ! lactate)\n",
      "Hyperglycemia may increase the expression of ACE2, which mediates SARS-CoV-2 infection.Hyperglycemia may also increase lactate production via HIF-1alpha, which suppresses the innate immune RLR signaling by targeting MAVS, leading to delayed clearance of SARS-CoV-2 and thus severe outcomes in diabetes patients with COVID-19, including ARDS, septic shock, and MODS.This work was partially supported by grants from the National Nature Science Foundation of China (81800758 and 81730022) and the National Key R&D Program of China (2018YFC2000100 and 2019YFA0801903).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:24996 ! lactate)\n",
      "Considering distribution of ACE2 in the muscle system, SARS-CoV-2 could upregulate lactate levels, reduce oxygen and energy supply, and cause myalgia (Kucuk et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:28082 ! \"5alpha-cholestane-3beta,5,6beta-triol\")\n",
      "The most frequently observed CT findings in cases of disease caused by SARS-CoV-2 are ground-glass opacities and consolidations in lungs, with a predominantly peripheral distribution, sometimes associated with fine reticulate (forming the so called crazy paving pattern), vascular thickening and inverted halo signal.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:28082 ! \"5alpha-cholestane-3beta,5,6beta-triol\")\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:28304 ! heparin)\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:30145 ! \"lithium atom\")\n",
      "BMJ 2020; 369 : m1375 Du, R.H.; Liang, L.R.; Yang, C.Q.; Wang, W.; Cao, T.Z.; Li, M.; Guo, G.Y.; Du, J.; Zheng, C.L.; Zhu, Q.; Hu, M.; Li, X.Y.; Peng, P.; Shi, H.Z. Predictors of mortality for patients with COVID-19 pneumonia caused by SARS-CoV-2 : a prospective cohort study.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:33364 ! platinum)\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:3392 ! carbendazim)\n",
      "S2-reactive MBC expansion mediated by SARS-CoV-2 infection could enhance protection against a broad range of coronaviruses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:3392 ! carbendazim)\n",
      "S2-reactive MBC expansion mediated by SARS-CoV-2 infection could enhance protection against a broad range of coronaviruses (25).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:3392 ! carbendazim)\n",
      "This 16 non suppression is in line with the elevated activation indicated by CD71 and ICOSL in memory B 17 cells of CR.21 The lower frequency of IgM- isotype switched MBC in CR than in HD, indicated the 18 induction of isotype switched MBC might be repressed by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:34922 ! picloram)\n",
      "Additionally , in silico studies have indicated that a strong protein-protein interaction exists between the viral S protein and TLR4 , a PPR , suggesting that SARS-CoV-2 directly activates proinflammatory pathways ( 69 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:34922 ! picloram)\n",
      "Subsequently , SARS-CoV-2 directly interacts with MASP2 through a single N-linked glycosylation site in SARS-S and activates the MBL pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:34922 ! picloram)\n",
      "Subsequently , SARS-CoV-2 directly interacts with MASP2 through a single N-linked glycosylation site in SARS-S and activates the MBL pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:34922 ! picloram)\n",
      "Additionally , in silico studies have indicated that a strong protein-protein interaction exists between the viral S protein and TLR4 , a PPR , suggesting that SARS-CoV-2 directly activates proinflammatory pathways ( 69 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:34922 ! picloram)\n",
      "Subsequently , SARS-CoV-2 directly interacts with MASP2 through a single N-linked glycosylation site in SARS-S and activates the MBL pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58-60 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58 ) ( 59 ) ( 60 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:34922 ! picloram)\n",
      "It is possible that the pathway which SARS-CoV-2 amplifies inflammatory response is either absent or diminished in patients with agammaglobulinemia , which results in milder course of COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:34922 ! picloram)\n",
      "It is possible that the pathway which SARS-CoV-2 amplifies inflammatory response is either absent or diminished in patients with agammaglobulinemia , which results in milder course of COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:34922 ! picloram)\n",
      "All these in vitro results suggest that SARS-CoV-2 could inhibit the IFN pathway at one or more of the steps mentioned above .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:41131 ! \"crotonic acid\")\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:41423 ! celecoxib)\n",
      "Celebrex Adjuvant Therapy on Coronavirus Disease 2019 : An Experimental Study.Severe acute respiratory distress syndrome (ARDS) caused by SARS-CoV, MERS-CoV, and SARS-CoV-2 infections is a major factor of mortality.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:52093 ! \"N-methylmorpholine N-oxide\")\n",
      "HIGHLIGHTS COVID-19 has neuro-invasion potential independently of the critical disease process Demyelinating lesions such as neuromyelitis optica may develop in the course of disease Parainfectious NMO induced by SARS-CoV-2 may occur as a result of extended immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:55438 ! \"Ile(5)-angiotensin II (1-7)\")\n",
      "Treatment Options Available for COVID-19 and an Analysis on Possible Role of Combination of rhACE2, Angiotensin (1-7) and Angiotensin (1-9) as Effective Therapeutic Measure.Corona virus disease, caused by SARS-CoV-2, is rapidly spreading all around the world and is posing a threat to mankind.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:65009 ! indol-3-ylmethylamine)\n",
      "Infection due to SARS-CoV-2 has resulted in AMI ( acute myocardial injury ) and chronic damage to the cardiovascular system .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:73832 ! Asp-Val)\n",
      "All these observations confirmed the reliability 182 and robustness of our approach.Schematic diagram depicting why COVID-19 may be less severe in highlyDV-endemic countriesIn highly dengue-prone areas, SARS-CoV-2 infection may stimulate the immunological memory to DV in people with previous DV exposure (s), which could be asymptomatic.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:73907 ! Gly-Ile)\n",
      "Few studies [28, 29] showed that patients with the severe disease more often have GI symptoms and hepatic involvement.While the mechanisms of GI manifestations elicited by SARS-CoV-2 are still unknown, it is clear that ACE2 expression in the esophagus, small and large intestines, plays a role.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:73907 ! Gly-Ile)\n",
      "Though the two routes of SARS-CoV-2 inoculation in this study induced both pneumonia and GI tract dysfunction, there were at least two major differences between them.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:73907 ! Gly-Ile)\n",
      "As per gastroenterologists, human-to-human transmission of SARS-CoV-2 is possible by airborne respiratory droplets and by the fecal-oral route, raising the possibility of GI manifestations of the disease (D'amico et al., 2020; Ding and Liang, 2020; Gu et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:82594 ! Ferritin)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n",
      "In the present work, omics data and computational methods were used to understand the underlying mechanisms of SARS-CoV-2 induced altered host gene regulatory networks, and the potential regulatory mechanism of Vitamin D in suppressing cytokine storm and reducing viral load.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n",
      "In this work, bioinformatics and systems biology approaches were used to understand SARS-CoV-2 induced altered host gene regulatory network, biological pathways of cytokine storm, and the potential regulatory mechanism of Vitamin D in suppressing cytokine storm and reducing viral infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:89324 ! \"24,25-Dihydroxyvitamin D\")\n",
      "In this work, bioinformatics and systems biology approaches were used to understand SARS-CoV-2 induced cytokine pathways and the potential mechanism of Vitamin D in suppressing cytokine storm and enhancing antiviral response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:9937 ! vasopressin)\n",
      "In an individual with severe hyponatraemia, it was shown that SARS-CoV-2 causes a syndrome of inappropriate secretion of antidiuretic hormone and manifestations of hyponatremia [27].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:S1)\n",
      "A Potential Peptide From Soy Cheese Produced Using Lactobacillus delbrueckii WS4 for Effective Inhibition of SARS-CoV-2 Main Protease and S1 Glycoprotein.The COVID-19 pandemic caused by novel SARS-CoV-2 has resulted in an unprecedented loss of lives and economy around the world.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "24,25 By binding respiratory epithelial cells , SARS-CoV-2 activates the cytokines and hypercoagulation pathways in the blood , increasing levels of inflammatory markers such as C-reactive protein , ferritin , IL-1 , IL-6 , TNF-alpha , and D-dimer .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "Cytokines are implicated in the pathogenesis of IPF including initiation of a process called epithelial-mesenchymal transition , and SARS-CoV-2 itself causes further cytokine activation and inflammation thereby also justifying the use of steroids as a therapeutic option in virally driven hyperinflammation , but precisely why COVID-19-linked sepsis may be fatal [ 2,8 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "Cytokines are implicated in the pathogenesis of IPF including initiation of a process called epithelialmesenchymal transition , and SARS-CoV-2 itself causes further cytokine activation and inflammation thereby also justifying the use of steroids as a therapeutic option in virally driven hyperinflammation , but precisely why COVID-19-linked sepsis may be fatal [ 2 , 8] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "It was recently also seen to be expressed in epithelial cells of the middle ear , as well as the stria vascularis and spiral ganglion in mice.14 Furthermore , SARS-CoV-2 causes an inflammatory response and an increase in cytokines such as tumour necrosis factor-alpha , interleukin 1 and interleukin 6.5 Both a direct entry into the cochlea and inflammation leading to cell stress are mechanisms that have been implicated in persistent sensorineural hearing loss and could be occurring in the case of the SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "Current evidence shows that SARS-Cov-2 induces an increase in a small range of cytokines .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "Current evidence shows that SARS-Cov-2 induces an increase in a small range of cytokines .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:cytokine)\n",
      "Current evidence shows that SARS-Cov-2 induces an increase in a small range of cytokines .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:cytokine)\n",
      "Rothan et al. investigated the effect of auranofin on SARS-CoV-2 , finding that auranofin inhibits 50 % replication of SARS-CoV-2 in Huh7 cells at low micro-molar concentrations , and dramatically reduced the expression of SARS-CoV-2 induced cytokines in Huh7 cells , suggesting the drug could be useful in restricting SARS-CoV-2 infection and associated lung injury [ 91 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"neutrophil count\")\n",
      "These results demonstrate that severe COVID-19 patients undergo a functional decline in adaptive immunity , which correlates with their lower T cell population.14 , 15 , 16 SARS-CoV-2 nsp10 Targets NKRF to Facilitate IL-8 Production From the quantitative proteome analysis of PBMCs , it is clear that SARS-CoV-2 triggers IL-8 expression and increases neutrophil count , which may mediate the manifestation of acute respiratory distress syndrome in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"neutrophil count\")\n",
      "14-16 From the quantitative proteome analysis of PBMCs , it is clear that SARS-CoV-2 triggers IL-8 expression and increases neutrophil count which may mediate the manifestation of acute respiratory distress syndrome in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"NOD/ShiLtJ\")\n",
      "SARS-CoV-2 induces IL-1 family members by activating the inflammatory protein complex of NOD like receptor protein 3 inflammasome.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:hypoxemia)\n",
      "According to the limited literature , SARS-CoV-2 utilizes angiotensin converting enzyme 2 as its receptor and causes severe hypoxemia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "TLR4 may also be involved in promoting viral cell entry in other organs , such as the skin where SARS-CoV-2 has caused complications in some patients and which has a relatively low ACE2 expression [ 48 ] ; TLR4 is also expressed on skin fibroblasts [ 105 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "SARS-CoV2 induced pulmonary embolism and complications from anticoagulation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "SARS-CoV2 induced pulmonary embolism and complications from anticoagulation Coronavirus disease ( COVID-19 ) pandemic has rapidly spread around the world .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "'' by Tassorelli and co-workers , in which the authors generate and summarize hypotheses how SARS-CoV-2 may enter the peripheral and central nervous system and cause life-threatening complications [ 1 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:complication)\n",
      "A positive test for SARS-CoV-2 may lead to additional stress , for example that associated with quarantine.8 COVID-19 infection , its complications and its treatment can have direct effects on the brain leading to neuropsychiatric disorders including seizures , delirium , demyelination , encephalopathies and stroke.5 ,9 Current studies on psychiatric presentations of SARS-CoV-2 have several limitations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:event)\n",
      "It is not clear yet whether SARS-CoV-2 can cause all these events , since most of the studies focus on drug discovery and what are the protein ( s ) involved .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:event)\n",
      "It is not clear yet whether SARS-CoV-2 can cause all these events , since most of the studies focus on drug discovery and what are the protein ( s ) involved .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:clinician)\n",
      "Because SARS-CoV-2 causes delirium in a significant proportion of patients in the acute stage , clinicians should also be aware of the possibility of the development of depression , anxiety , fatigue , post-traumatic stress disorder , and rarer neuropsychiatric syndromes [ 41 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:clinician)\n",
      "Because SARS-CoV-2 causes delirium in a significant proportion of patients in the acute stage , clinicians should also be aware of the possibility of the development of depression , anxiety , fatigue , post-traumatic stress disorder , and rarer neuropsychiatric syndromes [ 41 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "Autopsy examinations revealed that SARS-CoV-2 contaminates renal tubules directly and causes severe kidney injury through cytopathogenic action or by infiltration CD68 th macrophages into interstitial compartment , along with tubular accumulation of complement membrane attack complex ( C5b-9 ) [ 39 , 40 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "Autopsy examinations revealed that SARS-CoV-2 contaminates renal tubules directly and causes severe kidney injury through cytopathogenic action or by infiltration CD68 + macrophages into interstitial compartment , along with tubular accumulation of complement membrane attack complex ( C5b-9 ) [ 39 , 40 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "SARS-CoV-2 causes kidney injury through direct and indirect pathogenic pathways [ 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "Autopsy examinations revealed that SARS-CoV-2 contaminates renal tubules directly and causes severe kidney injury through cytopathogenic action or by infiltration CD68 + macrophages into interstitial compartment , along with tubular accumulation of complement membrane attack complex ( C5b-9 ) [ 39 , 40 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "SARS-CoV-2 causes kidney injury through direct and indirect pathogenic pathways [ 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"kidney injury\")\n",
      "SARS-CoV-2 causes kidney injury through direct and indirect pathogenic pathways [ 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:AMPK ! AMPK)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:AMPK ! AMPK)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency (Gassen et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:AMPK ! AMPK)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:AMPK ! AMPK)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency (Gassen et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:\"Acetyl_CoA_synthetase\" ! \"Acetyl_CoA_synthetase\")\n",
      "12 The overlap of ACS and lung disease caused by SARS-CoV-2 is possibly the explanation for more significant radiological findings in this group.When compared with literature, 13 our patients did not differ in terms of genotype, time of hospitalization and clinical outcome.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD3 ! CD3)\n",
      "(7) Two small case series in humans both found that SARS-CoV-2 infected patients on RAS inhibitors had significantly lower inflammatory markers, and increased CD3 and CD8 T cell proliferation than patients on alternative antihypertensive medications.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "This new data indicates that SARS-CoV-2 promotes extreme stimulation and ensuing exhaustive collapse of CD8 + T cells, which is similar in fashion to many malignancies (44) (45) (46).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "(7) Two small case series in humans both found that SARS-CoV-2 infected patients on RAS inhibitors had significantly lower inflammatory markers, and increased CD3 and CD8 T cell proliferation than patients on alternative antihypertensive medications.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Third, the mechanisms by which SARS-CoV-2 infection causes the reduction and functional exhaustion of CD4 + T cells and CD8 + T cells are still unclear.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Broad and strong memory CD4 + and CD8 + T cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Peng, Y. et al., Broad and strong memory CD4 + and CD8 + T cells induced 314 by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Several studies have now shown that the natural immune response to SARS-CoV-2 induces CD8 + and CD4 + T cells specific for epitopes within Spike RBD 34, 46.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Broad and strong memory CD4 + and CD8 + T cells induced by SARS-CoV-2 in UK convalescent COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CD8 ! CD8)\n",
      "Moreover, SARS-CoV-2 hampers the function of CD4 + T cells and promotes excessive activation and possibly subsequent exhaustion of CD8 + T cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:CD8 ! CD8)\n",
      "If SARS-CoV-2 reduces memory CD8 + T cells in obesity, then how effective a vaccine will be for obese individuals?\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:CD8 ! CD8)\n",
      "Urra, J.M., et al., Selective CD8 cell reduction by SARS-CoV-2 is associated with a worseprognosis and systemic inflammation in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:CD8 ! CD8)\n",
      "Selective CD8 cell reduction by SARS-CoV-2 is associated with a worse prognosis and systemic inflammation in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CHRN ! CHRN)\n",
      "Thus , nAChR modulation by SARS-CoV-2 might account for the hyperinflammatory features that are observed in a subgroup of patients with COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CHRN ! CHRN)\n",
      "Thus , nAChR modulation by SARS-CoV-2 might account for the hyperinflammatory features that are observed in a subgroup of patients with COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:CHRN ! CHRN)\n",
      "Thus, nAChR modulation by SARS-CoV-2 might account for the hyperinflammatory features that are observed in a subgroup of patients with COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:ESR ! ESR)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Fibrinogen ! Fibrinogen)\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Fibrinogen ! Fibrinogen)\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Fibrinogen ! Fibrinogen)\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:\"HLA_DR\" ! \"HLA_DR\")\n",
      "199 SARS-CoV-2 increased HLA-DR expression in a dose dependent manner after 24 hours and 200 decreased it after 48 hours .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:\"HLA_DR\" ! \"HLA_DR\")\n",
      "199 SARS-CoV-2 increased HLA-DR expression in a dose dependent manner after 24 hours and 200 decreased it after 48 hours.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:\"HLA_DR\" ! \"HLA_DR\")\n",
      "200 SARS-CoV-2 increased HLA-DR expression in a dose dependent manner after 24 hours and 201 decreased it after 48 hours.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Hemoglobin ! Hemoglobin)\n",
      "This may provide a novel insight into the impact of SARS-CoV-2 on erythropoiesis and hypoxia seen in COVID-19 patients.hypoxia, the impact of SARS-CoV-2 infection on erythropoiesis has received limited attention.Preliminary modeling reports have suggested that SARS-CoV-2 may inhibit heme metabolism and induce hemoglobin denaturation 14.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:IFNA ! IFNA)\n",
      "Here, we demonstrate that SARS-CoV-2 infection of human monocyte derived macrophages (MDMs) and dendritic cells (MDDCs) was abortive, but induced the production of multiple antiviral and pro inflammatory cytokines (IFN-alpha, IFN-beta, TNF, IL-1beta, IL-6 and IL-10) and a chemokine (CXCL10).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:IFNA ! IFNA)\n",
      "Blocking of IFN-alpha production by SARS-CoV-2 might be one reason why COVID-19 patients progress to a severe or critical state .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:IL2R ! IL2R)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:JNK ! JNK)\n",
      "Putative mechanistic involvement of nanoceria in the proposed inhibition of cytokines production stimulated by SARS-CoV-2 : SARS-CoV-2 enters into the host cell by ACE-2 cell surface receptors , and modulates the IKK and JNK activity which potentiates NFkappaB as well as AP1 protein transcription , respectively .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:JNK ! JNK)\n",
      "Putative mechanistic involvement of nanoceria in the proposed inhibition of cytokines production stimulated by SARS-CoV-2 : SARS-CoV-2 enters into the host cell by ACE-2 cell surface receptors , and modulates the IKK and JNK activity which potentiates NFkappaB as well as AP1 protein transcription , respectively .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:JNK ! JNK)\n",
      "Fig. 1 : Putative mechanistic involvement of nanoceria in the proposed inhibition of cytokines production stimulated by SARS-CoV-2 : SARS-CoV-2 enters into the host cell by ACE-2 cell surface receptors , and modulates the IKK and JNK activity which potentiates NFkappaB as well as AP1 protein transcription , respectively .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:LDH ! LDH)\n",
      "It is made available under a CC-BY-NC-ND 4.0 International license.92 different healthy donors, we found that SARS-CoV-2 infection triggers LDH release 93 and activation of NLRP3 inflammasome in monocytes, as shown by NLRP3 puncta 94 formation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "SARS-CoV-2 itself activates nuclear factor kappa B ( NF-kappaB ) via pattern recognition receptors ( PPRs ) , and the accumulated AngII in turn induces the activation of the IL-6 amplifier through enhanced activation of NF-kappaB pathway and IL-6-STAT3 ( signal transducers and activators of transcription family 3 ) axis ( 61 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Thus, SARS-CoV-2 infection in the respiratory tract can activate both NF-kappaB and STAT3 in a feedforward mechanism (IL-6 amplifier or IL-6 Amp) leading to multiple inflammatory and autoimmune diseases (37).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Aggregately, SARS-CoV2 itself also activates NF-kappaB via various 119 pattern recognition receptors (PPRs) [33-40].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "In our study, we found SARS-CoV-2 infection could 187 activate NF-kB signaling pathway and up-regulate some pro inflammatory genes, we further 188 speculate drugs targeting the NF-kB signaling pathway may be one of the candidate drugs for the 189 treatment severe COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Besides, SARS-CoV-2 itself stimulates the NF-kappaB pathway independent of ACE2 through pattern recognition receptors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "In addition, SARS-CoV-2 infection can activate NF-kB, leading to multiple inflammatory and autoimmune diseases, which induces pro inflammatory cytokines and chemokines, including IL-6, and recruits lymphoid cells and myeloid cells, such as activated T cells and macrophages (Hirano and Murakami, 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "In addition, SARS-CoV-2 infection can activate NF-kappaB, leading to multiple inflammatory and autoimmune diseases, which induces pro inflammatory cytokines and chemokines, including IL-6, and recruits lymphoid cells and myeloid cells, such as activated T cells and macrophages (Hirano and Murakami, 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Aggregately , SARS-CoV2 itself also activates NF-kappaB via various 118 pattern recognition receptors ( PPRs ) [ 33-40 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Along the same line, SARS-CoV-2 activates NF-kB, leading to cytokine secretion, including IL-6 and TNF-alpha [85].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Along the same line, SARS-CoV-2 activates NF-kB, leading to cytokine secretion, including IL-6 and TNF-alpha [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "SARS-CoV-2 itself activates nuclear factor kappa B ( NF-kB ) via pattern recognition receptors ( PPRs ) , and the accumulated AngII in turn induces the activation of the IL-6 amplifier through enhanced activation of NF-kB pathway and IL-6-STAT3 ( signal transducers and activators of transcription family 3 ) axis ( 61 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "SARS-CoV-2 itself activates nuclear factor kappa B ( NF-kappaB ) via pattern recognition receptors ( PPRs ) , and the accumulated AngII in turn induces the activation of the IL-6 amplifier through enhanced activation of NF-kappaB pathway and IL-6-STAT3 ( signal transducers and activators of transcription family 3 ) axis ( 61 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Along the same line , SARS-CoV-2 activates NF-kB , leading to cytokine secretion , including IL-6 and TNF-alpha [ 85 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:PI3K ! PI3K)\n",
      "Based on information obtained from studies of other CoVs, we consider it likely that infection with SARS-CoV-2 stimulates PI3K and AKT signaling and that preventing P13K activation can lessen the severity of viral infections.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9030 ! PLA2G1B)\n",
      "PLA2 activated by SARS-CoV-2 and other viruses induce the release of DGLA/AA/EPA/DHA that inactivate enveloped viruses by themselves and through their products and regulate inflammatory process.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Protease ! Protease)\n",
      "A Potential Peptide From Soy Cheese Produced Using Lactobacillus delbrueckii WS4 for Effective Inhibition of SARS-CoV-2 Main Protease and S1 Glycoprotein.The COVID-19 pandemic caused by novel SARS-CoV-2 has resulted in an unprecedented loss of lives and economy around the world.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:Protease ! Protease)\n",
      "The SARS-CoV-2 multibasic cleavage site increases serine protease usage.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:TGFB ! TGFB)\n",
      "Xiong et al. performed transcriptome sequencing of RNAs isolated from bronchoalveolar lavage fluid and PBMC specimens of COVID-19 patients, and found that SARS-CoV-2 infection induced IL-10 and TGF-beta production [124].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:mTORC1 ! mTORC1)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:mTORC1 ! mTORC1)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency (Gassen et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:mTORC1 ! mTORC1)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:mTORC1 ! mTORC1)\n",
      "A very recent study showed that SARS-CoV-2, similar to MERS-CoV, strongly reduced the autophagic flux in infected cell lines downregulating the AMPK and mTORC1 pathway, altered autophagy-relevant signaling, and also reduced autophagosome and lysosome fusion efficiency (Gassen et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:p38 ! p38)\n",
      "In the infected airway epithelium, SARS-CoV-2 can activate the mitogen activated protein kinases (MAPK) pathway by binding to Toll like receptors (TLRs), in particular the p38 MAPK pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:p38 ! p38)\n",
      "Intriguingly, SARS-CoV-2 causes DNA damage, cell growth arrest, cytokine production, and p38 activation during the early stages of infection as a strategy to promote more efficient replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:p38 ! p38)\n",
      "Importantly, Bouhaddou et al. demonstrated that SARS-CoV-2 cell infection promotes the production of several cytokines, cell growth arrest, and p38 MAPK activation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0000077\" ! \"DNA damage checkpoint\")\n",
      "proteins and 24 SARS-CoV-2 targets involving innate immunity and stress response components, and DNA damage response mediators SARS-CoV-2 in vitro modelsBojkova et al. (2020) Caco-212 cells (colon) LC-MS/MS (Q-Exactive HF) Proteins Grenga et al. (2020) Vero cells (renal) LC-MS/MS (Q-Exactive HF) Alteration in membrane trafficking, pre-processing in ER and regulation of mRNA processing and splicing via spliceosome Davidson et al. (2020) Vero cells (renal) RNAseq + LC-MS/MS (Q-Exactive HF) Identification of an 8 amino acid deletion in the S glycoprotein that potentially affects protein cleavage, cell tropism, and infectivity Appelberg et al. (2020) HuH7 cells (hepatic) RNAseq + LC-MS/MS (Orbitrap Fusion Lumos) Proteo-transcriptomic analysis has identified disruptions in ErbB, HIF-1, mTOR, and TNF signaling Covid-19 pathology study and diagnosis Ihling et al. (2020) Gargle LC-MS/MS (Orbitrap-HCD) An MS method that specifically detects unique peptides from SARS-CoV-2 nucleoprotein from gargle samples Bezstarosti et al. (2020) Vero cells (renal) LC-MS/MS (Orbitrap Eclipse Tribrid) A parallel reaction monitoring approach to detect the SARS-CoV-2 nucleocapsid protein in the mid-attomole range Nikolaev et al. (2020) Nasopharynx LC-MS/MS (TIMS-TOF) Confident identification of the SARS-CoV-2 N protein in patients with low viral load Wang et al. (2020) Serum Microarray (n.d.) Antibodies commercially available for SARS-CoV-1 proteins can also target SARS-CoV-2 proteins Zhang et al. (2020) Serum Microarray (GenePix 4300A microarray) Shen et al. (2020) Serum LC-MS/MS (Q-Exactive HF-X) Deregulation of 93 proteins and 204 metabolites revealed metabolic and immune dysregulation in patients with COVID-19 Liang et al. (2020) PBMCs LC-MS/MS (Q-Exactive Orbitrap) ES : conceptualization.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0001525\" ! angiogenesis)\n",
      "Coronavirus disease 2019 (COVID-19) is an infectious disease caused by coronavirus-2 (SARS-CoV-2) that causes a severe acute respiratory syndrome, a characteristic hyperinflammatory response, vascular damage, microangiopathy, angiogenesis and widespread thrombosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0001768\" ! \"establishment of T cell polarity\")\n",
      "However , analysis of immune cell sequencing showed that SARS-CoV-2 enhances T cell polarization from naive to effector cells and that aging promotes the expression of SARS-CoV-2 susceptibility genes , mainly in T cells ( 110 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0001768\" ! \"establishment of T cell polarity\")\n",
      "However , analysis of immune cell sequencing showed that SARS-CoV-2 enhances T cell polarization from naive to effector cells and that aging promotes the expression of SARS-CoV-2 susceptibility genes , mainly in T cells ( 110 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "80 We first piloted our antigen preps for the RBD-S IgG screening assay using serum 81 samples from a PCR-confirmed severe COVID-19 patient ( defined as admission to the Intensive 82 Care Unit , ICU ) who was admitted to the hospital 10 days following symptom onset and based 83 on an early report suggesting that SARS-CoV-2 could trigger antibody responses in this 84 timeframe ( 24 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "We focused our analysis on the genes that played a major role in the adaptation of SARS-CoV [ 24 ] and MERS-CoV [ 28 ] [ 29 ] [ 30 ] to the human host , and those in SARS-CoV-2 that modulate the antibody response [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "We focused our analysis on the genes that played a major role in the adaptation of SARS-CoV [ 24 ] and MERS-CoV [ 28,29,30 ] to the human host , and those in SARS-CoV-2 that modulate the antibody response [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "We focused our analysis on the genes that played a major role in the adaptation of SARS-CoV [ 24 ] and MERS-CoV [ 28,29,30 ] to the human host , and those in SARS-CoV-2 that modulate the antibody response [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "SARS-CoV-2 cause a strong B cell activation , maturation and antibody production .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "SARS-CoV-2 cause a strong B cell activation , maturation and antibody production .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\")\n",
      "16For molecular amplification targets, the CDC, Atlanta, recommends two regions in the N gene, while the WHO recommends two targets, e.g., the E gene followed by a SARS-CoV-2 confirmatory assay targeting RdRP.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Unstable proteins such as GPCRs, like olfactory receptors and taste receptors, may be susceptible to ERAD.Figure 5 : Explosive SARS-CoV-2 replication may induce persistent ER stress, leading to inflammation and apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "The extremely rapid rate of replication of SARS-CoV-2 increases ER stress to levels that can not be accommodated by the UPR, leading to inflammation and apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Therefore, ERphagy and ER stress are activated by SARS-CoV-2 through ORF3 to restrict the appropriate overload of viral proteins in the ER membrane and lumen, recycle overloaded viral proteins or repair ER damage.As a consequence, ERphagy and ER stress induced by ORF3 reprogram ER function and activate inflammatory responses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Studies have shown that SARS-CoV-2 infection in diverse cells induces ER stress and activates proinflammatory responses 40, 41.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Therefore, ERphagy and ER stress are activated by SARS-CoV-2 through ORF3 to restrict the appropriate overload of viral proteins in the ER membrane and lumen, recycle overloaded viral proteins or repair ER damage.As a consequence, ERphagy and ER stress induced by ORF3 reprogram ER function and activate inflammatory responses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0006412\" ! translation)\n",
      "In endotoxin-tolerant (ET) macrophages that mimic conditions which render patients with pre-existing chronic inflammatory diseases vulnerable to severe symptoms, our chemoproteomic approach with a biotinylated inhibitor of G9a identified multiple G9a associated translation regulatory pathways that were upregulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0006412\" ! translation)\n",
      "However, to date, the behavior of ribosomes within the frameshifting element for neither SARS-CoV-2 nor HIV-1 in infected cells has been empirically assessed.In addition to hijacking the host translational machinery, SARS-CoV-2 can modulate host mRNA translation so as to suppress the immune system to ensure efficient propagation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0006412\" ! translation)\n",
      "14 As a consequence of this infectious process, SARS-CoV-2 RNA drives the translation and assembly of viral proteins inside the endoplasmic reticulum and Golgi apparatus.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Upon fusion, the genome is released into the cytoplasm.The SARS-CoV-2 genomic RNA contains a 5′ cap and a 3′ poly(A) tail that allow immediate translation to produce two coterminal replicase polyproteins, pp1a and pp1ab, by utilizing a ribosomal frameshifting mechanism.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Curiously, a previous study failed to show SARS-CoV-2 5 ' UTRmediated evasion of Nsp1 mediated translation inhibition (Schubert et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Blots were imaged via chemiluminescence.NSP1 from SARS-CoV-2 inhibited translation initiation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV-2 Disrupts Splicing , Translation , and Protein Trafficking to Suppress Host Defenses .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "How SARS-CoV-2 simultaneously inhibits host translation to shut down innate immune responses yet also manages to translate its own proteins remains unclear, although recent work points to a functional role for the 5 ' UTR of the viral RNA -- likely together with yet unidentified translation initiation factors -- in promoting efficient translation even in the presence of nsp1 (Schubert et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV-2 Disrupts Splicing, Translation, and Protein Trafficking to Suppress Host Defenses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "In summary, Nsp1 from both, SARS-CoV and SARS-CoV-2 binds 40S and 80S ribosomes and disrupt capdependent translation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV-2 Disrupts Splicing, Translation, and Protein 650 Trafficking to Suppress Host Defenses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV-2 Disrupts Splicing, Translation, and Protein Trafficking to Suppress Host Defenses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV-2 disrupts splicing, translation, and protein trafficking to suppress host defenses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Screening a subset of drugs identified two sets of pharmacological agents effectively reducing SARS-CoV-2 infectivity in Vero-6 cells : inhibitors of mRNA translation and predicted regulators of the sigma-1 and sigma-2 receptors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0006508\" ! proteolysis)\n",
      "This suggests that insufficient vitamin K dependent MGP activation leaves elastic fibres unprotected against SARS-CoV-2 induced proteolysis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007049\" ! \"cell cycle\")\n",
      "Consistent with the concept that coronaviruses can induce cell growth arrest to promote their replication, Bouhaddou et al. recently demonstrated that SARS-CoV-2 causes both DNA damage during the early stage of infection and cell cycle arrest [66].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007049\" ! \"cell cycle\")\n",
      "Consistent with the concept that coronaviruses can induce cell growth arrest to promote their replication, Bouhaddou et al. recently demonstrated that SARS-CoV-2 causes both DNA damage during the early stage of infection and cell cycle arrest [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "In light of the important role for palmitoylation in tricellular tight junction assembly (Oda et al., 2020), these findings suggest that SARS-CoV-2 operates by a similar mechanism to promote adhesion and transcellular interfaces.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007155\" ! \"cell adhesion\")\n",
      "10.20033944 doi: medRxiv preprint SARS-CoV-2 had three FCS (F1-3), where F1 hydrolyzes S protein to S1 and S2 and promotes virus-cell fusion, F2 hydrolyzes S2 and participates in virus pathogenicity after cell entry, while F3 functions through NTD and promotes adhesion between virus and cell surface.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007601\" ! \"visual perception\")\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007601\" ! \"visual perception\")\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0007601\" ! \"visual perception\")\n",
      "SARS-CoV-2 have induced complications in renal functions , cardiac activities , thrombotic mechanism , hepatocellular control , diabetic management , neural system , vision and skin integrity ( Gupta et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "We further assess the challenges to conducting scientifically sound and socially valuable research in the LAC region in order to inform recommendations to encourage clinical trials that are most likely to produce robust evidence during the pandemic.During the SARS-CoV-2 pandemic the urgency to conduct research to find safe and efficacious treatments and vaccines has led to a sizable proliferation of COVID-19 clinical trials.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "The ability of SARSCoV-2 to selectively promote proliferation of memory B cells could explain our findings of a marked increase in an overall immune memory antibody repertoire activity linked to poor clinical trajectories of COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "(7) Two small case series in humans both found that SARS-CoV-2 infected patients on RAS inhibitors had significantly lower inflammatory markers, and increased CD3 and CD8 T cell proliferation than patients on alternative antihypertensive medications.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Thus, we 348 hypothesize that SARS-CoV-2 infection induces basal cell proliferation, which accounts for the 349 observed long lasting infections with recurrent peaks of viral replication, which warrants future 350 investigation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "The marked locations are blocked by SARS-Cov-2 which causes the function of AT2R to be decreased leading to vasodilatation as well as cell apoptosis and increased vasoconstriction and cell proliferation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "SARS-CoV-2 may induce SOX2 to increase goblet cell proliferation, increase mucus hypersecretion, and decrease Clara and specialized epithelial cells, as seen in other allergic and inflammatory diseases.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0016049\" ! \"cell growth\")\n",
      "Thus, SARS-CoV-2 spike protein mediated cell growth signaling may participate in adverse cardiovascular and pulmonary outcomes, and this mechanism may provide new therapeutic targets to combat COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0016049\" ! \"cell growth\")\n",
      "Thus, SARS-CoV-2 spike protein mediated cell growth signaling may participate in adverse cardiovascular and pulmonary outcomes, and this mechanism may provide new therapeutic targets to combat COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0016049\" ! \"cell growth\")\n",
      "Intriguingly, SARS-CoV-2 causes DNA damage, cell growth arrest, cytokine production, and p38 activation during the early stages of infection as a strategy to promote more efficient replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0016049\" ! \"cell growth\")\n",
      "Importantly, Bouhaddou et al. demonstrated that SARS-CoV-2 cell infection promotes the production of several cytokines, cell growth arrest, and p38 MAPK activation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0016049\" ! \"cell growth\")\n",
      "Intriguingly , SARS-CoV-2 causes DNA damage , cell growth arrest , cytokine production , and p38 activation during the early stages of infection as a strategy to promote more efficient replication .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0019031\" ! \"viral envelope\")\n",
      "SARS-CoV-2 necessitates of protein spike ( S ) priming to allow the fusion of viral envelope with the cellular membrane and this process is mediated by host protease such as the type Electronic supplementary material The online version of this article ( https://doi.org/10.1007/s10815-020-01917-0 ) contains supplementary material , which is available to authorized users .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0019031\" ! \"viral envelope\")\n",
      "SARS-CoV-2 necessitates of protein spike ( S ) priming to allow the fusion of viral envelope with the cellular membrane and this process is mediated by host protease such as the type II membrane serine proteases ( TMPRSS ) 2 , 4 , 11A , 11D , and 11E [ 6 , 7 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0030330\" ! \"DNA damage response, signal transduction by p53 class mediator\")\n",
      "Furthermore , Xiong et al. ( 10 ) use the transcriptome dataset approach to discover that SARS-CoV-2 can activate apoptosis and P53 signaling pathway ( one of the pathways responsible for the survival of the cell ) in lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0030330\" ! \"DNA damage response, signal transduction by p53 class mediator\")\n",
      "Furthermore , Xiong et al. ( 10 ) use the transcriptome dataset approach to discover that SARS-CoV-2 can activate apoptosis and P53 signaling pathway ( one of the pathways responsible for the survival of the cell ) in lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0030330\" ! \"DNA damage response, signal transduction by p53 class mediator\")\n",
      "Furthermore , Xiong et al. ( 10 ) use the transcriptome dataset approach to discover that SARS-CoV-2 can activate apoptosis and P53 signaling pathway ( one of the pathways responsible for the survival of the cell ) in lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0042113\" ! \"B cell activation\")\n",
      "SARS-CoV-2 cause a strong B cell activation , maturation and antibody production .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0042113\" ! \"B cell activation\")\n",
      "SARS-CoV-2 cause a strong B cell activation , maturation and antibody production .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "SARS-CoV-2 stimulates an innate immune response via pattern associated molecular patterns (PAMPs) expressed by the virus, which in the case of SARS-CoV-2 consists of viral RNA and its intermediates produced during replication (19).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "The Innate Immune Response Induced by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "In the CNS, SARS-CoV-2 can activate both the adaptive and innate immunity [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Like other neurotropic viruses , SARS-CoV-2 might be able to enter the central nervous system via the olfactory epithelium and induce innate immune responses at the site of entry .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Further clarification of the role and impact of minimal immunosuppressive regimens versus full immunosuppressive regimens of organ transplant recipients on the reactive innate immune response caused by SARS-CoV-2 would shed further light on the topic.Figure 1: Flow diagram of the search strategy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Further clarification of the role and impact of minimal immunosuppressive regimens versus full immunosuppressive regimens of organ transplant recipients on the reactive innate immune response caused by SARS-CoV-2 would shed further light on the topic.LT: liver transplantation; L-T: long-term; S-T: short-term; NR: non-reported.Flow diagram of the search strategy.Study Characteristics\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Once in the brain , SARS-CoV-2 would initiate the innate immune responses at early stages of the COVID-19 resulting in neurological disorders .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Once in the brain , SARS-CoV-2 would initiate the innate immune responses at early stages of the COVID-19 resulting in neurological disorders .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Acute SARS-CoV-2 Infection is Highly Cytopathic, Elicits a Robust Innate Immune Response and is Efficiently Prevented by EIDD-2801.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Several recent studies have revealed seemingly contradictory roles regarding SARS-CoV-2 induced innate immunity [31, 32].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "XREF_BIBR There study identified several other mechanisms, such as induction of endoplasmic reticulum stress, lysosomal damage, and subsequent activation, of the master regulator of the autophagy and lysosome machinery, transcription factor EB by which a SARS-CoV-2 open reading frame activates intracellular stress pathways and targets the innate immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "The earlier SARS virus has shown the robust mechanism of NLRP3 inflammasome activation in macrophage by providing a potent signal for its activation.26 There study identified several other mechanisms, such as induction of endoplasmic reticulum stress, lysosomal damage, and subsequent activation, of the master regulator of the autophagy and lysosome machinery, transcription factor EB by which a SARS-CoV-2 open reading frame activates intracellular stress pathways and targets the innate immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "408 Therefore, SARS-CoV-2 causes the delayed response of host innate immunity, which results in rapid 409 transmission, low mortality and asymptomatic infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "SARS-CoV-2 causes a delay in the innate immune response and it disarranges the immune system leading to an overwhelming inflammatory reaction (the ' cytokine storm ').\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "SARS-CoV-2 mediates viral entry into host cells via the ACE2 , which is expressed at high levels on the surface of pulmonary epithelial cells .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "SARS-CoV-2 mediates viral entry into host cells via the ACE2 , which is expressed at high levels on the surface of pulmonary epithelial cells .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "SARS-CoV-2 mediates viral entry into host cells via the ACE2 , which is expressed at high levels on the surface of pulmonary epithelial cells .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "The Spike protein of severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) is crucial to mediate viral entry into host cells ; however , whether it can be cellularly pathogenic and contribute to pulmonary hyper-inflammations in COVID-19 is not well known .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "The Spike protein of severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) is crucial to mediate viral entry into host cells ; however , whether it can be cellularly pathogenic and contribute to pulmonary hyper-inflammations in COVID-19 is not well known .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0097707\" ! ferroptosis)\n",
      "B A hypothesis of how SARS-CoV-2 may trigger ferroptosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0097707\" ! ferroptosis)\n",
      "B A hypothesis of how SARS-CoV-2 may trigger ferroptosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(GO:\"GO:0097707\" ! ferroptosis)\n",
      "B A hypothesis of how SARS-CoV-2 may trigger ferroptosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10019 ! RIPK1)\n",
      "We can now show that SARS-CoV-2 infection triggers RIPK1 activation in respiratory tract epithelial cells isolated from the throat smears of symptomatic patients who had tested positive for SARS-CoV-2 by PCR.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10050 ! RNASEL)\n",
      "We also found that SARS-CoV-2 activates RNase L and PKR in both 344 cell lines used and PKR in iAT2 cells and iCM, but not in primary nasal cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:10050 ! RNASEL)\n",
      "437 438 We found that SARS-CoV-2 was unable to prevent activation of RNase L and PKR, although to 439 different extents among the cell types, unlike two other betacoronaviruses, MHV and MERS-CoV, 440 which shut down these pathways (Comar et al., 2019; Ye et al., 2007; Zhao et al., 2012).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10424 ! RPS4X)\n",
      "(Some estimates of the R0 of SARS-CoV-2 in different settings are presented in Supplementary Table S3 for comparison, and some examples of evidence supporting efficient indoor transmission are provided in Supplementary Table S4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10499 ! S100A9)\n",
      "We also observed that recombinant S2 and nucleocapsid proteins of SARS-CoV-2 significantly increased pro inflammatory cytokines and chemokines and S100A9 in human primary PBMCs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10637 ! CXCL10)\n",
      "SARS-CoV-2 causes increasing in IL-1B , IFN-gamma , and IP10 levels and so these cytokines cause increasing of hepcidin and Iron storage in macrophages and hepatocytes will be enhanced .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10637 ! CXCL10)\n",
      "SARS-CoV-2 causes increasing in IL-1B, IFN-gamma, and IP10 levels and so these cytokines cause increasing of hepcidin and Iron storage in macrophages and hepatocytes will be enhanced.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:108 ! ACHE)\n",
      "The emerging virus, SARS-CoV-2, causes coughing, fever, muscle ache, and shortness of breath or dyspnea in symptomatic patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10980 ! SLC25A10)\n",
      "These manifestations could be caused by direct injury of vascular endothelium by SARS-CoV-2, which could lead to DIC, antiphospholipid syndrome, and vasculitis mimics.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11048 ! SLC6A2)\n",
      "This increased NET production by SARS-CoV-2 may facilitate thrombus formation ( Kaplan and Radic 2012 ; Allegra et al. 2020b ; Cicco et al. 2020 ) ( Fig. 4 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11291 ! SRF)\n",
      "Thus, severe respiratory failure (SRF)-aggravated pneumonia caused by SARS-CoV-2 might involve the following : a unique immunosuppressive pathway aimed at IL-10 overproduction via the inhibition of HLA-DR activation on APCs; high IL-10 secretion by macrophages and DCs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11362 ! STAT1)\n",
      "SARS-CoV-2 induction of pSTAT1 as well as rapid 280 IFIT1 and OAS2 mRNA induction suggests a similar host response to SARS-CoV-2 as that 281 observed during mutant MERS-CoV-DNS4ab infection, and not that of WT MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11362 ! STAT1)\n",
      "This expression is in opposition to those associated with PARP14 activity : PARP14 is involved in promoting the STAT6 dependent transcription of the Th2 immune response, whereas SARS-CoV-2 promotion of STAT1 transcription is associated with a Th1 response [XREF_BIBR, XREF_BIBR], andthe predominant cytokine of PARP14 activation is IL-4 compared with SARS-CoV infections with IL-6.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11364 ! STAT3)\n",
      "A summary of these events, starting with the downregulation of STAT1 (1, Fig. 4) and compensatory upregulation of STAT3 (2, Fig. 4) induced by NSP1 and ORF6 of SARS-CoV-2, is presented in Fig. 4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11364 ! STAT3)\n",
      "Thus, SARS-CoV-2 infection in the respiratory tract can activate both NF-kappaB and STAT3 in a feedforward mechanism (IL-6 amplifier or IL-6 Amp) leading to multiple inflammatory and autoimmune diseases (37).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11364 ! STAT3)\n",
      "A summary of these events, starting with the downregulation of STAT1 (1, Fig. 4) and compensatory upregulation of STAT3 (2, Fig. 4) induced by NSP1 and ORF6 of SARS-CoV-2, is presented in Fig. 4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11765 ! TGFA)\n",
      "36 It will be interesting to investigate whether intragastric inoculation with SARS-CoV-2 induces TGFalpha to allow viruses to escape from gastric acid.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11849 ! TLR3)\n",
      "17  SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4 , subsequently inducing an inflammatory reaction , inflammasome activation and upregulation of the IL-1beta pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11849 ! TLR3)\n",
      "17 SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4, subsequently inducing an inflammatory reaction, inflammasome activation and upregulation of the IL-1b pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "Hence , a possible model for the interaction of SARS-CoV-2 and TLR4 is outlined in Section 11 and the graphical abstract ( Figure 1 ) in which SARS-CoV-2 may activate TLR4 in the heart and lungs to cause aberrant TLR4 signalling in favour of the proinflammatory MyD88-dependent ( canonical ) pathway rather than the alternative TRIF / TRAM-dependent anti-inflammatory and interferon pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "We can confidently extrapolate the above findings in Sections 8.1 and 8.2 from SARS-CoV-1 to SARS-CoV-2 ; hence , we propose that SARS-CoV-2 would activate TLR4 directly , probably via its spike protein binding to TLR4 ( and / or MD2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "In addition , SARS-CoV-2 may activate TLR4 to increase PI3K / Akt signalling in infected cells , preventing apoptosis and thus increasing time for viral replication .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "A recent in silico study implicated SARS-CoV-2 activation of Toll like receptor 4 (TLR4) as a major contributor to the inflammatory response in COVID-19 [199].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "A recent in silico study implicated SARS-CoV-2 activation of Toll like receptor 4 (TLR4) as a major contributor to the inflammatory response in COVID-19 [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "Although SARS-CoV-2 is unlikely to activate TLR-4 directly , as TLR-4 responds to bacteria [ 12 ] , one hypothesis based on a mouse model of acute lung injury is that oxidized phospholipids can be responsible for activation of TLR-4 and onset of ARDS [ 14 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "17  SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4 , subsequently inducing an inflammatory reaction , inflammasome activation and upregulation of the IL-1beta pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11850 ! TLR4)\n",
      "17 SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4, subsequently inducing an inflammatory reaction, inflammasome activation and upregulation of the IL-1b pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11936 ! FASLG)\n",
      "We hypothesize that SARS-CoV-2 viral induced aPL contributed to severe Covid-19-associated coagulopathy in this patient.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11998 ! TP53)\n",
      "Furthermore, Xiong et al. (10) use the transcriptome dataset approach to discover that SARS-CoV-2 can activate apoptosis and P53 signaling pathway (one of the pathways responsible for the survival of the cell) in lymphocytes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11998 ! TP53)\n",
      "SARS-CoV-2 enhanced the apoptosis and p53 signaling pathways in lymphocytes thereby causing the lymphopenic situation in COVID-19 patients [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11998 ! TP53)\n",
      "Structural homology with SARS-CoV-1 indicates that SARS-CoV-2 may be able to directly impair pRb and p53, which are key gatekeepers with tumor suppressor functions.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11998 ! TP53)\n",
      "SARS-CoV-2 might also inhibit antiviral responses by destroying p53.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11998 ! TP53)\n",
      "Structural homology with SARS-CoV-1 indicates that SARS-CoV-2 may be able to directly impair pRb and p53 , which are key gatekeepers with tumor suppressor functions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:12428 ! TWIST1)\n",
      "SARS-CoV-2 causes COVID-19 and CRS has been a defining feature of severe disease [5, [7] [8] [9].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:12435 ! TXN)\n",
      "However, there is no experimental proof that either SARS-CoV-2 or ACE2 are targets of TRX and TRXR systems and more experimental work is needed to provide a clear understanding of the interplay of these molecules and COVID-19 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1487 ! CAPS)\n",
      "Moreover, since infections, including viral respiratory similar to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), are considered main etiologies for CAPS, it could be possible that SARS-CoV-2 can induce CAPS although no evidence is currently found.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1487 ! CAPS)\n",
      "Moreover , since infections , including viral respiratory similar to severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , are considered main etiologies for CAPS , it could be possible that SARS-CoV-2 can induce CAPS although no evidence is currently found .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1487 ! CAPS)\n",
      "Questioning whether SARS-CoV-2 can induce CAPS , the association of other viral infections with CAPS can give an idea .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1487 ! CAPS)\n",
      "Questioning whether SARS-CoV-2 can induce CAPS , the association of other viral infections with CAPS can give an idea .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1509 ! CASP8)\n",
      "Here we report that SARS-CoV-2 infection activates caspase-8 to trigger cell apoptosis and inflammatory cytokine processing in the lung epithelial cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1509 ! CASP8)\n",
      "Recent study reported that SARS-CoV-2 infection could activate caspase-8 to induce apoptosis and inflammatory cytokine processing in the lung epithelial cells, triggering necroptosis pathway [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:15563 ! IL37)\n",
      "RESULTS Our results demonstrated that SARS-CoV-2 infection causes elevation of plasma IL-37.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16400 ! NLRP3)\n",
      "It is made available under a CC-BY-NC-ND 4.0 International license.92 different healthy donors, we found that SARS-CoV-2 infection triggers LDH release 93 and activation of NLRP3 inflammasome in monocytes, as shown by NLRP3 puncta 94 formation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16400 ! NLRP3)\n",
      "Collectively , these data indicate that SARS-CoV-2 infects human monocytes and triggers NLRP3 activation and a lytic form of cell death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16400 ! NLRP3)\n",
      "Collectively , these data indicate that SARS-CoV-2 infects human monocytes and triggers NLRP3 activation and a lytic form of cell death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Third, the mechanisms by which SARS-CoV-2 infection causes the reduction and functional exhaustion of CD4 + T cells and CD8 + T cells are still unclear.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Having established that SARS-CoV-2 stimulates the production of CD4 Tfh cells, we next sought to understand whether effector Th1 CD4 T cells were induced in the lung.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Based on the results of this study and the close correlation between autoimmune disorders and coagulation abnormalities and thrombosis events in patients, 31 we propose a hypothesis : CD4 cells and D-dimer may trigger in the progression of the older adults with COVID-19; SARS-CoV-2 may act on CD4 cells through ACE2 receptor, and cause CD4 cells count reduction.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Broad and strong memory CD4 + and CD8 + T cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "SARS-CoV-2 induces robust germinal center CD4 T follicular helper cell responses in rhesus macaques.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "CD4 + T lymphocytes activated by SARS-COV-2 become pathogenic T helper (Th) 1 cells and serve as a source of GM-CSF.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Broad and strong memory CD4 + and CD8 + T cells induced by SARS-CoV-2 in UK convalescent COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1678 ! CD4)\n",
      "Based on the results of this study and the close correlation between autoimmune disorders and coagulation abnormalities and thrombosis events in patients, XREF_BIBR we propose a hypothesis : CD4 cells and D-dimer may trigger in the progression of the older adults with COVID-19; SARS-CoV-2 may act on CD4 cells through ACE2 receptor, and cause CD4 cells count reduction.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:1678 ! CD4)\n",
      "Recent studies have reported that SARS-CoV-2 infection can induce decreased CD4 cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:1678 ! CD4)\n",
      "The question arises whether SARS-CoV-2 , like HIV , can directly decrease CD4 + T cell count .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16885 ! SH2D3A)\n",
      "It is expected that this research might be helpful for the development of new antiviral drugs active against SARS-CoV-2 targeting Nsp1.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16885 ! SH2D3A)\n",
      "CONCLUSION It is expected that this research might be helpful for the development of new antiviral drugs active against SARS-CoV-2 targeting Nsp1.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1698 ! CD79A)\n",
      "Because (i) IgM and IgG ratios are not indicative of the timing of disease onset, (ii) IgA is induced by SARS-CoV-2, and (iii) both IgG and IgM isotypes correlated with neutralizing titers, we chose to quantify total (all isotypes) antigen specific antibodies for seroprevalence studies.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1698 ! CD79A)\n",
      "Antibody levels against hCoV-OC43 and hCoV-HKUI in early 180 responder patients did not differ from other patients (data not shown) and antibody 181 levels against circulating coronaviruses did not change during the course of COVID182 19 in both very early and \" normal \" responders while SARS-CoV-2 directed IgA and 183 IgG levels did.184 Two of our patients with severe disease who survived failed to show an IgG response 185 at day 21 after disease onset (although the total antibody assay was positive).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1700 ! CD80)\n",
      "CD80 was induced in a dose dependent manner by SARS-CoV-2 at MOI 0.04 to 1 (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:17381 ! SH2B2)\n",
      "The hypothesis that SARS-CoV-2 induces APS requires demonstration that COVID-19 patients fulfill both the clinical and the laboratory criteria for APS4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:17381 ! SH2B2)\n",
      "The hypothesis that SARS-CoV-2 induces APS requires demonstration that COVID-19 patients fulfill both the clinical and the laboratory criteria for APS 4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1817 ! CEACAM5)\n",
      "Therefore, further studies are needed to investigate the underlying pathophysiological mechanisms.In conclusion, SARS-CoV-2 infection might cause transient and marked CEA elevation especially in non survivors, and this abnormal increment was not correlated with the severity of inflammation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:18643 ! SUGP1)\n",
      "Moreover, regulated RBPs, and especially those stimulated by SARS-CoV-2, include proteins annotated by GO terms and KEGG pathways related to antiviral response and innate immunity.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:18643 ! SUGP1)\n",
      "Other RBPs with prominent roles in virus infection were consistently upregulated by SARS-CoV-2 and SINV, including PPIA (cyclophilin A), PA2G4, ZC3H11A, DDX3, and HSP90AB1 (Dawar et al., 2017; GarciaMoreno et al., 2019; Valiente-Echeverria et al., 2015; Younis et al., 2018).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2367 ! CRP)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:23721 ! ZC3HAV1)\n",
      "Western blot analyses showed that SARS-CoV-2 infection alone enhances expression of the short isoform of ZAP about 2-fold and that this induction was further enhanced by IFN-beta and IFN-gamma treatment.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:23721 ! ZC3HAV1)\n",
      "Western blot analyses showed that SARS-CoV-2 infection alone enhances expression of the short isoform of ZAP about 2-fold and that this induction was further enhanced by IFN-beta and IFN-gamma treatment (XREF_FIG; see also XREF_SUPPLEMENTARY A and B).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2434 ! CSF2)\n",
      "Moreover, the extreme release of cytokines into the CSF mediated by SARS-CoV-2 dysregulates the balance between production, release, and reuptake of neurotransmitters including dopamine, which possibly contributes to the development of PD [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2434 ! CSF2)\n",
      "Moreover, the extreme release of cytokines into the CSF mediated by SARS-CoV-2 dysregulates the balance between production, release, and reuptake of neurotransmitters including dopamine, which possibly contributes to the development of PD [25].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2434 ! CSF2)\n",
      "SARS-Cov2 replication in pulmonary tissues activates production of GM-CSF by endothelial cells and fibroblast.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:257 ! ADK)\n",
      "Peng Y, Mentzer AK, Liu G et al. (2020) Broad and strong memory CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "It is possible that ACE inhibitory components could minimize the ACE dominance caused by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "SARS-CoV-2 causes ACE and ACE2 balance disruption and RAAS activation, which leads ultimately to COVID-19 progression, especially in patients with comorbidities, such as hypertension, diabetes mellitus, and cardiovascular disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "It is still unclear if SARS-CoV-2 can block the entry of DENV by inhibiting the ACE receptors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "SARS-CoV-2 may activate the ACE / AT1 axis by down-regulating ACE2 and thereby promoting the development of ARDS by inducing an inflammatory storm and increasing oxidative stress .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "50% of studies exclusively targeted 3CLpro of SARS-CoV-2, and only two studies targeted ACE and TMPSS2 for inhibiting host viral interactions.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "Actually , SARS-CoV-2 is required to bind to the angiotensin-converting enzyme 2 ( ACE2 ) to enter the cells ( 7 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:2707 ! ACE)\n",
      "Incidentally, the renin-angiotensin-aldosterone system (RAAS) is integral to physiologic control of both ACE and ACE2 expression, and is an essential system utilized by SARS-CoV-2, albeit with varying schools of thought on how it can affect viral entry.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:27962 ! STING1)\n",
      "However, whether STING activation can impact infections from SARS-CoV-2 or other coronaviruses remains largely unknown.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30319 ! MCF2L2)\n",
      "(A) The ability of SARS-CoV-2 to directly target sensory neurons in the DRG depends on the expression of ACE2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30576 ! ADI1)\n",
      "Furthermore , since vitamin D deficiency increases with age , older population is likely to suffer from severe ARDs due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30576 ! ADI1)\n",
      "Furthermore , since vitamin D deficiency increases with age , older population is likely to suffer from severe ARDs due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30576 ! ADI1)\n",
      "Furthermore , since vitamin D deficiency increases with age , older population is likely to suffer from severe ARDs due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30605 ! SEPSECS)\n",
      "SARS-CoV-2 may directly activate the LP of complement through binding to MBL via its glycoprotein envelope spikes, as has been documented for SARS-CoV [XREF_BIBR], and suggested by the demonstration by our group of CoV-2 envelope spike protein binding to MASP2 [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3236 ! EGFR)\n",
      "SARS-CoV-2, like other emerging respiratory viruses, may induce pulmonary fibrosis in survivors of SARS-CoV-2 infection by activating the EGFR pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3325 ! ELK3)\n",
      "A further study proved that SARS-CoV-2 could directly trigger NET formation via an ACE2-serine protease TMPRSS2 dependent pathway and induce lung epithelial cell death in vitro ( 163 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3325 ! ELK3)\n",
      "A further study proved that SARS-CoV-2 could directly trigger NET formation via an ACE2-serine protease TMPRSS2 dependent pathway and induce lung epithelial cell death in vitro ( 163 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "It is theoretically speculated that SARS-CoV-2 reduces the expression of ACE2 after infecting cardiomyocytes through S protein , causing an increase in AngII level , which results in cardiomyocyte damage and apoptosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "In the lungs, SARS-CoV-2 mediated ANGII accumulation leads to alveolar cell death and a reduction in oxygen uptake [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "It is possible that in patients presenting comorbidities associated with a dysregulation of the RAS system , AngII modulation by SARS-CoV-2 reaches a critical level that ultimately induces an inflammatory state causing fatal issues .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "It is possible that in patients presenting comorbidities associated with a dysregulation of the RAS system , AngII modulation by SARS-CoV-2 reaches a critical level that ultimately induces an inflammatory state causing fatal issues .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "It is possible that in patients presenting comorbidities associated with a dysregulation of the RAS system, AngII modulation by SARS-CoV-2 reaches a critical level that ultimately induces an inflammatory state causing fatal issues.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:333 ! AGT)\n",
      "Lastly, SARS-CoV-2 could directly mediate heart injury by entering cardiomyocytes via binding of the viral spike glycoprotein to its extracellular receptor, angiotensin I converting enzyme 2 (ACE2) 9,15,16.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:333 ! AGT)\n",
      "It is theoretically speculated that SARS-CoV-2 reduces the expression of ACE2 after infecting cardiomyocytes through S protein, causing an increase in AngII level, which results in cardiomyocyte damage and apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3383 ! STOM)\n",
      "Therefore, SARS-CoV-2 might trigger an imbalanced immune response resulting in a ' cytokine stom ' and extensive pulmonary inflammation (11).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3535 ! F2)\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3541 ! F3)\n",
      "Chest computed tomography showing diffuse bilateral ground-glass opacities with limited consolidationsArteriography showing normal right (a) and left (b) bronchial arteries risk of thromboembolic eventsAbbreviations SARS-CoV-2 : Severe acute respiratory syndrome coronavirus 2; COVID 19 : Coronavirus disease 2019; CT : Computed tomography; APTT : Activated partial thromboplastin time; PT : Prothrombin time; PEEP : High positive end expiratory pressure; PCR : Polymerase chain reaction; ECMO : Extracorporeal membrane oxygenation; UNFH : Unfractionated heparin; CTA : CT angiography; F : French\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3541 ! F3)\n",
      "Moreover, in the attempt to explain the common thromboembolic complications of COVID-19, it has been proposed that SARS-CoV-2 infection of endothelial cells induces luminal expression of tissue factor (TF) which activate extrinsic clotting.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3768 ! FMN1)\n",
      "SARS-CoV-2 induces an increase of the LD biogenesis in different human 384 pulmonary cell lines.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:3768 ! FMN1)\n",
      "Similarly, pre-treatment with A922500 also blocked LD induced 131 by SARS-CoV-2 in monocytes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:39025 ! ERVK-18)\n",
      "SARS-CoV-2 causes the disease syndrome known as COVID-19 3, now classified as a pandemic with global reach and devastation.CoV host cell entry is mediated by its spike (S) glycoprotein, a large transmembrane protein that decorates the virus particle 4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:39025 ! ERVK-18)\n",
      "SARS-CoV-2 causes the disease syndrome known as COVID-193, now classified as a pandemic with global reach and devastation.CoV host cell entry is mediated by its spike (S) glycoprotein, a large transmembrane protein that decorates the virus particle4.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:4910 ! HIF1A)\n",
      "Furthermore, SARS-CoV-2 infection induced mtROS and HIF-1alpha, subsequently promoted glycolytic genes and cytokines in return.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:4910 ! HIF1A)\n",
      "We suggest that chronic hypoxia under SARS-CoV-2 infection may activate HIF-1alpha pathways, including an increase in ADAM17 transcription that leads to increased processing of TNFalpha-driven signaling as a lever arm of the cytokine storm.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5013 ! HMOX1)\n",
      "Ho, Oxford Immunology Network Covid-19 Response T cell Consortium, 16 ISARIC4C Investigators, R. J. Cornall, C. P. Conlon, P. Klenerman, G. R. Screaton, J. 17 Mongkolsapaya, A. McMichael, J. C. Knight, G. Ogg, T. Dong, Broad and strong memory 18 CD4+ and CD8+ T cells induced by SARS-CoV-2 in UK convalescent individuals following 19 COVID-19, Nat Immunol 21, 1336-1345 (2020).20 26.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5407 ! IFIT1)\n",
      "Levels of ISG mRNAs were 255 variable, with SARS-CoV-2 inducing moderate levels of OAS2 and IFIT1 mRNAs, but only late in 256 infection (48 hpi), similar to those induced by SINV at 24 hpi (Fig 6A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5407 ! IFIT1)\n",
      "SARS-CoV-2 induction of pSTAT1 as well as rapid 280 IFIT1 and OAS2 mRNA induction suggests a similar host response to SARS-CoV-2 as that 281 observed during mutant MERS-CoV-DNS4ab infection, and not that of WT MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5434 ! IFNB1)\n",
      "745 SARS-CoV-2 induced IFNbeta, IFNlambda1 and IFNlambda2,3 release (pg/ml) was measured.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5434 ! IFNB1)\n",
      "No reuse allowed without permission.416 417 In A549ACE2 cells, SARS-CoV-2 induced low levels of IFNl and IFNb mRNAs and somewhat 418 higher ISG mRNA by 48 hpi, as compared with SINV (Fig 6A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:5434 ! IFNB1)\n",
      "However, it has been postulated that SARS-CoV-2, due to the secretion of a novel short protein encoded by orf3b, inhibits the expression of IFNbeta and enhances viral pathogenicity (95).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:5434 ! IFNB1)\n",
      "It was suggested that the SARS-CoV-2 orf3b play a key role in pathogenicity of virus and block the IFNbeta expression, while the functional domain of orf8 still elucidated (7).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:5434 ! IFNB1)\n",
      "Although SARS-CoV-2 inhibits the production of IFNbeta and thus obstructs the innate immune response to this virus, it is sensitive to the antiviral activity of externally administrated IFN-Is.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:5434 ! IFNB1)\n",
      "Although SARS-CoV-2 inhibits the production of IFNb and thus obstructs the innate immune response to this virus, it is sensitive to the antiviral activity of externally administrated IFN-Is.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5438 ! IFNG)\n",
      "Moreover, SARS-CoV-2 was shown to induce a stronger IFN-gamma response in vitro compared with influenza, suggesting that IFN-gamma might play a larger role in the host immune response to COVID-19 [50].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5438 ! IFNG)\n",
      "[4] Moreover, SARS-CoV-2 was shown to induce a stronger IFN-gamma response in vitro compared with influenza, suggesting that IFN-gamma might play a larger role in the host immune response to COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5438 ! IFNG)\n",
      "SARS-CoV-2 increases interferon gamma , tumor necrosis factor-alpha , macrophage inflammatory protein-1 alpha , IL-2 , IL-6 , IL-7 , IL-10 , in patients , that show a form of secondary hemophagocytic lymphohistiocytosis or macrophage activation syndrome ( sHLH / MAS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5438 ! IFNG)\n",
      "SARS-CoV-2 causes increasing in IL-1B , IFN-gamma , and IP10 levels and so these cytokines cause increasing of hepcidin and Iron storage in macrophages and hepatocytes will be enhanced .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5438 ! IFNG)\n",
      "As show our results, SARS-CoV-2 infection induced high IFN-gamma expression in swabbed cells from upper airway; its expression was higher in symptomatic patients in comparison with asymptomatic individuals.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5962 ! IL10)\n",
      "Altogether, our data demonstrate that SARS-CoV-2 infects CD4+ T cells, impairs cell function, leads to increased IL10 expression and compromises cell viability, which in turn dampens immunity against the virus and contributes to disease severity.Impaired innate and adaptive immunity is a hallmark of COVID-19, particularly in patients who progress to the critical stages of the disease34,35.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5962 ! IL10)\n",
      "However, SARS-CoV-2 is shown to increase the T-helper-2 (Th2) cytokines (IL4 and IL10) that are known to suppress inflammation, which differs from SARs and MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5962 ! IL10)\n",
      "Xiong et al. performed transcriptome sequencing of RNAs isolated from bronchoalveolar lavage fluid and PBMC specimens of COVID-19 patients, and found that SARS-CoV-2 infection induced IL-10 and TGF-beta production [124].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5962 ! IL10)\n",
      "These results show that SARS-CoV-2 infection induces IL10 expression in CD4+ T cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6001 ! IL2)\n",
      "SARS-CoV-2 induces the release of inflammatory cytokines IL-2, IL-7, IL-10 which are believed to be involved in the pathology of AKI.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6014 ! IL4)\n",
      "However, SARS-CoV-2 is shown to increase the T-helper-2 (Th2) cytokines (IL4 and IL10) that are known to suppress inflammation, which differs from SARs and MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV-2 itself activates nuclear factor kappa B (NF-kappaB) via pattern recognition receptors (PPRs), and the accumulated AngII in turn induces the activation of the IL-6 amplifier through enhanced activation of NF-kappaB pathway and IL-6-STAT3 (signal transducers and activators of transcription family 3) axis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "A direct mechanistic link between the reduced expression of SELENOS and the production of inflammatory cytokines has been well documented; 14 this may well constitute an underlying mechanism by which SARS-CoV-2 induces marked elevation of IL-6 concentration.Thioredoxin serves as an electron donor for ribonucleotide reductase which catalyzes the conversion of ribonucleotides to deoxyribonucleotides for DNA synthesis.15 Inhibition of the selenoenzyme, thioredoxin reductase (TXNRD), decreases DNA synthesis and increases the ribonucleotide pool for RNA synthesis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "It was also detected that SARS-CoV-2 infects macrophages and triggers secretion of IL-6, which might contribute to lymphocyte apoptosis, suggesting an excessive infiltration and intense pro inflammatory activity of IMM in COVID-19 patients (109).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Inflammatory cytokines IL-6 and IL-8 were comparably induced by SARS-CoV-2 and Flu (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "It has 231 been shown in an in-situ study of post-mortem samples that SARS-CoV-2 induces IL-6 more 232 efficiently than TNF ( 8) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "In turn, IL-6 induced by SARS-CoV-2 in the lung seems to promote and amplify Th17 responses that may worsen the severe lung pathology in susceptible hosts.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "In turn, IL-6 induced by SARS-CoV-2 in the lung seems to promote and amplify Th17 responses that may worsen the severe lung pathology in susceptible hosts (Hotez et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "It has 231 been shown in an in-situ study of post-mortem samples that SARS-CoV-2 induces IL-6 more 232 efficiently than TNF (8).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "It has 232 been shown in an in-situ study of post-mortem samples that SARS-CoV-2 induces IL-6 more 233 efficiently than TNF (8).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV-2 mainly causes a dramatic increase in IL-6 and does not remarkably promote other pro-inflammatory factors , such as IL-1beta and IFN-gamma .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV-2 mainly causes a dramatic increase in IL-6 and does not remarkably promote other pro-inflammatory factors , such as IL-1beta and IFN-gamma .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Our review shows that SARS-Cov-2 selectively induces a high level of IL-6 and results in the exhaustion of lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Our review shows that SARS-Cov-2 selectively induces a high level of IL-6 and results in the exhaustion of lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV-2 mainly causes a dramatic increase in IL-6 and does not remarkably promote other pro-inflammatory factors , such as IL-1beta and IFN-gamma .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "In particular, IL-6 pathway activation by SARS-CoV-2 infection can induce vascular endothelial growth factor (VEGF), fibrinogen, and factor VII (Bikdeli et al. 2020b; Whyte et al. 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Finally, the N protein of SARS-CoV-2 induces the expression of IL-6 in infected airway epithelium by binding to the NF-kappaB regulatory elements of the IL-6 promoter (82).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Our review shows that SARS-Cov-2 selectively induces a high level of IL-6 and results in the exhaustion of lymphocytes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6018 ! IL6)\n",
      "Furthermore, SARS-CoV-2 infection elicited intrinsic expression of type I and type III interferons and IL6 but not IL1 transcripts.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6025 ! CXCL8)\n",
      "337 338 Although the pathophysiology of SARS-CoV-2 has not yet been fully characterized, it has been 339 observed that SARS-CoV-2 infection triggers hyper- activation of pro inflammatory cytokines 340 (IL-6, -Il1b, TNFa) and chemokines (CXCL8, 9, 10, CCL2) (Huang et al., 2020; Lucas et al., 341 2020; Merad and Martin, 2020; Tang et al., 2020; Zhou et al., 2020a).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6025 ! CXCL8)\n",
      "Inflammatory cytokines IL-6 and IL-8 were comparably induced by SARS-CoV-2 and Flu (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6025 ! CXCL8)\n",
      "CRP, ESR, serum ferritin, IL-8, IL-2R, IL-6, and PCT were stimulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6025 ! CXCL8)\n",
      "From the quantitative proteome analysis of PBMCs, it is clear that SARS-CoV-2 triggers IL-8 expression and increases neutrophil count, which may mediate the manifestation of acute respiratory distress syndrome in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6025 ! CXCL8)\n",
      "14-16From the quantitative proteome analysis of PBMCs, it is clear that SARS-CoV-2 triggers IL-8 expression and increases neutrophil count which may mediate the manifestation of acute respiratory distress syndrome in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6081 ! INS)\n",
      "SARS-CoV-2 might direct damage beta-cells via cell surface ACE2 protein and lead to cell injury and apoptosis causing relative insulin deficiency and acute hyperglycemic state, thereby increasing the risk for developing severe COVID-19 or requiring ICU admission (Hussain et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6081 ! INS)\n",
      "Although, this scenario seemed the least likely because the direct damage to the pancreatic islets should result in higher than the reported incidence of COVID-19-induced diabetes, however, recent data presented in Nature News indicate that indead, the SARS-CoV-2 may cause direct damage to the insulin producing beta-cells (Mallapaty 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:6081 ! INS)\n",
      "Thus, a SARS-CoV-2 and ACE2-mediated dysregulation of insulin signaling could further aggravate symptoms in diabetics.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6922 ! MBL2)\n",
      "SARS-CoV-2 may directly activate the LP of complement through binding to MBL via its glycoprotein envelope spikes, as has been documented for SARS-CoV [XREF_BIBR], and suggested by the demonstration by our group of CoV-2 envelope spike protein binding to MASP2 [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6962 ! SMCP)\n",
      "Since IL-1 is toxic, its production from ubiquitous MCs and macrophages activated by SARS-CoV-2 can also provokes both gastrointestinal and brain disorders.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8086 ! OAS1)\n",
      "It is noteworthy that OAS1, involved 19 in the innate immune response to viral infection for RNA degradation, is prominently 20 stimulated by SARS-CoV-2 throughout the infection period for 72 h.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8087 ! OAS2)\n",
      "Levels of ISG mRNAs were 255 variable, with SARS-CoV-2 inducing moderate levels of OAS2 and IFIT1 mRNAs, but only late in 256 infection (48 hpi), similar to those induced by SINV at 24 hpi (Fig 6A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8087 ! OAS2)\n",
      "SARS-CoV-2 induction of pSTAT1 as well as rapid 280 IFIT1 and OAS2 mRNA induction suggests a similar host response to SARS-CoV-2 as that 281 observed during mutant MERS-CoV-DNS4ab infection, and not that of WT MERS-CoV infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8568 ! FURIN)\n",
      "SARS-CoV-2 rapidly restores the furin cleavage site upon passage on TMPRSS2 competent cellsThe low-passage virus growth data in Calu-3 cells suggest that the furin cleavage site confers growth advantage in these cells and may result in selection of SARS-CoV-2 genomes with wildtype furin site.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8568 ! FURIN)\n",
      "Fibrinolysis influences SARS-CoV-2 infection in ciliated cells.Rapid spread of COVID-19 has caused an unprecedented pandemic worldwide, and an inserted furin site 13 in SARS-CoV-2 spike protein (S) may account for increased transmissibility.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8568 ! FURIN)\n",
      "Altogether, these results indicate that reduced FURIN levels mediated by rs4702 result in decreased SARS-CoV-2 infection.Overall, these findings suggest not only that gross manipulations of host genes can impact SARS-CoV-2 infection in lung and brain cells, but also that the more subtle gene expression changes associated with genetic common variants are sufficient to impact viral infection levels.We apply a hiPSC based model to explore the genetics of cell-type-specific host response to SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8568 ! FURIN)\n",
      "Altogether, these results indicate that reduced FURIN levels mediated by rs4702 result in decreased SARS-CoV-2 infection.Overall, these findings suggest not only that gross manipulations of host genes can impact SARS-CoV-2 infection in lung and brain cells, but also that the more subtle gene expression changes associated with genetic common variants are sufficient to impact viral infection levels.DISCUSSIONWe apply a hiPSC based model to explore the genetics of cell-type-specific host response to SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8573 ! PAEP)\n",
      "The temporal sequence suggests that GD could have been triggered by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:8573 ! PAEP)\n",
      "This supports the thesis that GD and OD in COVID-19 present two independent symptoms caused by SARS-CoV-2 and that GD is not a secondary consequence of OD.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9437 ! EIF2AK2)\n",
      "We also found that SARS-CoV-2 activates RNase L and PKR in both 344 cell lines used and PKR in iAT2 cells and iCM, but not in primary nasal cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:9437 ! EIF2AK2)\n",
      "437 438 We found that SARS-CoV-2 was unable to prevent activation of RNase L and PKR, although to 439 different extents among the cell types, unlike two other betacoronaviruses, MHV and MERS-CoV, 440 which shut down these pathways (Comar et al., 2019; Ye et al., 2007; Zhao et al., 2012).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "100 We next assessed whether SARS-CoV-2 induces PTGS2 in a more physiologically 101 relevant cell culture system.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Taken together, these results demonstrate that SARS-CoV-2 infection induces PTGS2 in diverse in vitro and in vivo airway and lung systems, across multiple independent studies.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Aggregating ciliated cells across the three time points, we found that infected ciliated cells expressed higher levels of PTGS2 compared to uninfected bystander ciliated cells (XREF_FIG), indicating that PTGS2 is also induced by SARS-CoV-2 in a cell-intrinsic manner in ciliated cells, a physiologically relevant target cell.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We next assessed whether SARS-CoV-2 induces PTGS2 in a more physiologically relevant cell culture system.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Taken together, these results demonstrate that SARS-CoV-2 infection induces 120 PTGS2 in diverse in vitro and in vivo airway and lung systems, across multiple independent 121 studies.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "107 Aggregating ciliated cells across the three time points, we found that infected ciliated cells 108 expressed higher levels of PTGS2 compared to uninfected bystander ciliated cells, 109 indicating that PTGS2 is also induced by SARS-CoV-2 in a cell-intrinsic manner in ciliated cells, 110 a physiologically relevant target cell.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Our findings suggest that COX-2 signaling driven by SARS-CoV-2 may instead play 35 a role in regulating the lung inflammation and injury observed in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We demonstrate that SARS-CoV-2 infection upregulates COX-2 in diverse human cell culture and mouse systems.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Considering the high identity sequence of S and N proteins between both viruses, SARS-CoV-2 may also elicit upregulation of COX-2, further exacerbating inflammation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Considering the high identity sequence of S and N proteins between both viruses (75 and 90%, respectively [78]), SARS-CoV-2 may also elicit upregulation of COX-2, further exacerbating inflammation.Finally, ACE genotypes may affect the SARS-CoV-2 and RAAS interplay.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 upregulation in diverse human cell culture and mouse systems.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Our findings suggest that COX-2 signaling driven by SARS-CoV-2 may instead play a role in regulating the lung inflammation and injury observed in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We next assessed whether SARS-CoV-2 induces PTGS2 in a more physiologically relevant cell culture system .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Aggregating ciliated cells across the three time points , we found that infected ciliated cells expressed higher levels of PTGS2 compared to uninfected bystander ciliated cells ( Fig. 1E ) , indicating that PTGS2 is also induced by SARS-CoV-2 in a cell-intrinsic manner in ciliated cells , a physiologically relevant target cell .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Any of these potential mechanisms are consistent with our HBEC scRNA-seq results demonstrating that SARS-CoV-2 increases PTGS2 expression in a cell-intrinsic manner .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We next assessed whether SARS-CoV-2 induces PTGS2 in a more physiologically relevant cell culture system .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Aggregating ciliated cells across the three time points , we found that infected ciliated cells expressed higher levels of PTGS2 compared to uninfected bystander ciliated cells ( Fig. 1E ) , indicating that PTGS2 is also induced by SARS-CoV-2 in a cell-intrinsic manner in ciliated cells , a physiologically relevant target cell .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Any of these potential mechanisms are consistent with our HBEC scRNA-seq results demonstrating that SARS-CoV-2 increases PTGS2 expression in a cell-intrinsic manner .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Any of these potential mechanisms are consistent with 180 our HBEC scRNA-seq results demonstrating that SARS-CoV-2 increases PTGS2 expression in 181 a cell-intrinsic manner .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "These results indicate that SARS-CoV-2 induction of the COX-2 and PGE 2 pathway in Calu-3 human lung cells does not regulate viral replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Considering the high identity sequence of S and N proteins between both viruses ( 75 and 90 % , respectively [ 78 ] ) , SARS-CoV-2 may also elicit upregulation of COX-2 , further exacerbating inflammation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 45 expression in both human cell culture systems and mouse models.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "197 In summary, we demonstrated that SARS-CoV-2 infection induces COX-2 expression in 198 diverse systems in vitro and in vivo.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Furthermore, SARS-CoV, the closest relative of SARS-CoV-2 among 83 human coronaviruses and cause of the 2002-2003 epidemic (11), stimulates COX-2 expression 84 via its spike protein and nucleocapsid protein (12, 13), indicating the potential relevance of this 85 pathway for SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induces COX-2 expression in 88 human cells and mice.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 expression in both human cell culture systems and mouse models.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Any of these potential mechanisms are consistent with 181 our HBEC scRNA-seq results demonstrating that SARS-CoV-2 increases PTGS2 expression in 182 a cell-intrinsic manner.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "In summary, we demonstrated that SARS-CoV-2 infection induces COX-2 expression in diverse systems in vitro and in vivo.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Furthermore, SARS-CoV, the closest relative of SARS-CoV-2 among human coronaviruses and cause of the 2002-2003 epidemic, stimulates COX-2 expression via its spike protein and nucleocapsid protein, indicating the potential relevance of this pathway for SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induces COX-2 expression in human cells and mice.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9605 ! PTGS2)\n",
      "Any of these potential mechanisms are consistent with our HBEC scRNA-seq results demonstrating that SARS-CoV-2 increases PTGS2 expression in a cell-intrinsic manner.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9951 ! REG1A)\n",
      "A recent report from Wuhan suggests that SARS-CoV-2 induces a dysregulated immune response in severely ill COVID-19 subjects, characterized by reduced numbers of circulating memory T lymphocytes, as well as reduced helper / suppressor T cell and T Reg subtypes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:9958 ! REN)\n",
      "Furthermore, we also show how the binding of SARS-CoV-2 may trigger a Renin Angiotensin System (RAS) imbalance due to its binding to ACE2, possibly contributing to the pathogenesis of COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Abnormality of mitochondrial metabolism\")\n",
      "SARS-CoV-2 causes mitochondrial dysfunction and downregulates ACE2 , which in turn under-activates the RAS alternative pathway ( ACE2-Ang - ( 1-7 ) - Mas ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Abnormality of mitochondrial metabolism\")\n",
      "SARS-CoV-2 causes mitochondrial dysfunction and downregulates ACE2 , which in turn under-activates the RAS alternative pathway ( ACE2-Ang - ( 1-7 ) - Mas ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Acute)\n",
      "Spike Protein of SARS-CoV-2 Activates Macrophages and Contributes to Induction of Acute Lung Inflammations in Mice .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Peripheral demyelination\")\n",
      "Second , SARS-CoV-2 can cause demyelination in the brain and spinal cord [ 90 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Peripheral demyelination\")\n",
      "Second , SARS-CoV-2 can cause demyelination in the brain and spinal cord [ 90 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Chronic infection\")\n",
      "Furthermore , as SARS-CoV-2 does not cause chronic infection ( seen as a latent stage ) , its reactivation is not virologically possible .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"Nonproductive cough\")\n",
      "26  SARS-CoV-2 typically causes respiratory sickness , the major clinical characteristics observed in infected patients are high fever , dry cough and dyspnea ( shortness of breath or difficulty in breathing ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Hyperfibrinolysis)\n",
      "In contrast , SARS-CoV-2 causes hyper-fibrinolysis without any significant bleeding ( Chen et al ., 2020 ; Han et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C000592401 ! sarilumab)\n",
      "Further efforts should be directed to determine mechanisms of SARS-CoV-2 induced immune dysregulation as well as indications of sarilumab in the patients with COVID-19 to minimize concerns regarding its off-label administration.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C036309 ! \"fibrin fragment D\")\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C036309 ! \"fibrin fragment D\")\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C036309 ! \"fibrin fragment D\")\n",
      "In addition , SARS-CoV-2 induces increases in D-dimer and fibrinogen and further increases the consumption of platelets in damaged lungs .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000069196 ! \"Gastrointestinal Microbiome\")\n",
      "The mentioned aberrant immune response may be a consequence of infection of intestinal mucosa and modulation of the gut microbiome by SARS-CoV-2 , which can trigger neuroinflammation and neurodegeneration [ 24 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000069196 ! \"Gastrointestinal Microbiome\")\n",
      "The mentioned aberrant immune response may be a consequence of infection of intestinal mucosa and modulation of the gut microbiome by SARS-CoV-2 , which can trigger neuroinflammation and neurodegeneration [ 24 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000069338 ! Neuropathology)\n",
      "In conclusion , the SARS-CoV-2 may trigger or exacerbate neuropathologies through direct or indirect ways .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000069338 ! Neuropathology)\n",
      "In conclusion , the SARS-CoV-2 may trigger or exacerbate neuropathologies through direct or indirect ways .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000069338 ! Neuropathology)\n",
      "In conclusion , the SARS-CoV-2 may trigger or exacerbate neuropathologies through direct or indirect ways .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000080424 ! \"Cytokine Release Syndrome\")\n",
      "In addition , the low expression of IFITM3 in immune cells suggested that SARS-CoV-2 might attack lymphocytes and induce the cytokine release syndrome ( CRS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000080424 ! \"Cytokine Release Syndrome\")\n",
      "In addition , the low expression of IFITM3 in immune cells suggested that SARS-CoV-2 might attack lymphocytes and induce the cytokine release syndrome ( CRS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000080424 ! \"Cytokine Release Syndrome\")\n",
      "In addition , the low expression of IFITM3 in immune cells suggested that SARS-CoV-2 might attack lymphocytes and induce the cytokine release syndrome ( CRS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus belonging to the Coronaviridae family , which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 22 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus belonging to the Coronaviridae family , which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 22 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D000375 ! Aging)\n",
      "This study will improve our understanding of the transmission feature of SARS-CoV-2 in different age groups and suggest the most prevention measures should be applied to middle aged and elderly people.Keywords : Transmissibility, SARS-CoV-2, COVID-19, Compartmental model, Age specific dynamicsCoronavirus disease 2019, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, also called 2019-nCoV), has spread around the world.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D000375 ! Aging)\n",
      "Age associated genes are dysregulated by SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001145 ! \"Arrhythmias, Cardiac\")\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3,12,13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001145 ! \"Arrhythmias, Cardiac\")\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3 , 12 , 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001145 ! \"Arrhythmias, Cardiac\")\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3,12,13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001145 ! \"Arrhythmias, Cardiac\")\n",
      "Multiple reports of increased arrhythmias in individuals with COVID-19 without other causes suggest that SARS-CoV-2 itself may cause arrhythmias [ 15 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001145 ! \"Arrhythmias, Cardiac\")\n",
      "Multiple reports of increased arrhythmias in individuals with COVID-19 without other causes suggest that SARS-CoV-2 itself may cause arrhythmias [ 15 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001172 ! \"Arthritis, Rheumatoid\")\n",
      "Besides, SARS-CoV-2 infection also causes RA.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001249 ! Asthma)\n",
      "As with other viruses , SARS-CoV-2 triggers asthma exacerbations , which is why asthma is listed as a risk factor for COVID-19 related morbidity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001249 ! Asthma)\n",
      "Large multicenter cohort studies are needed to confirm these data and explore the reasons why SARS-CoV-2 does not seem to trigger as many asthma exacerbations , as previously seen with other respiratory viruses .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001249 ! Asthma)\n",
      "Large multicenter cohort studies are needed to confirm these data and explore the reasons why SARS-CoV-2 does not seem to trigger as many asthma exacerbations , as previously seen with other respiratory viruses .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001249 ! Asthma)\n",
      "Similarly , like other coronaviruses , SARS-CoV-2 may exacerbate asthma , although the underling molecular mechanisms through which this may occur are still unknown [ 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001249 ! Asthma)\n",
      "The SARS-CoV2 can induce asthma exacerbations which are a source of additional stress for asthma patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001364 ! Awareness)\n",
      "There are numerous reports of negative severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) reverse transcription polymerase chain reaction ( RT-PCR ) test results in participants with known SARS-CoV-2 infection , and increasing awareness that the ability of RT-PCR tests to detect virus depends on the timing of sample retrieval and anatomical sampling site .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001364 ! Awareness)\n",
      ": medRxiv preprint 4 PANEL : RESEARCH IN CONTEXT : There are numerous reports of negative severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) reverse transcription polymerase chain reaction ( RT-PCR ) test results in participants with known SARS-CoV-2 infection , and increasing awareness that the ability of RT-PCR tests to detect virus depends on the timing of sample retrieval and anatomical sampling site .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Moreover , Zhang et al. exhibited that SARS-CoV-2 and its spike protein directly stimulated platelets , resulting in the release of coagulation factors and inflammatory cytokines and enhancing leukocyte-platelet aggregates , which can promote thrombosis and thrombus stability [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "SARS-CoV-2 and its Spike protein directly enhanced platelet activation such as platelet aggregation , PAC-1 binding , CD62P expression , alpha granule secretion , dense granule release , platelet spreading , and clot retraction in vitro , and thereby Spike protein enhanced thrombosis formation in wild-type mice transfused with hACE2 transgenic platelets , but this was not observed in animals transfused with wild-type platelets in vivo .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Direct activation of platelets by SARS-CoV-2 is a likely pathway for the development of thromboembolism .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Direct activation of platelets by SARS-CoV-2 is a likely pathway for the development of thromboembolism .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Here , SARS-CoV-2 was shown to directly induce platelet activation and aggregation and promote thrombosis [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Moreover , Zhang et al. exhibited that SARS-CoV-2 and its spike protein directly stimulated platelets , resulting in the release of coagulation factors and inflammatory cytokines and enhancing leukocyte-platelet aggregates , which can promote thrombosis and thrombus stability [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Here , SARS-CoV-2 was shown to directly induce platelet activation and aggregation and promote thrombosis [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "SARS-CoV-2 and its Spike protein directly enhanced platelet activation such as platelet aggregation , PAC-1 binding , CD62P expression , alpha granule secretion , dense granule release , platelet spreading , and clot retraction in vitro , and thereby Spike protein enhanced thrombosis formation in wild-type mice transfused with hACE2 transgenic platelets , but this was not observed in animals transfused with wild-type platelets in vivo .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Since SARS-CoV-2 , like influenza , is also a SSRNA virus , the possibility exists that SARS-CoV-2 may promote dysregulated platelet activity directly through surface receptor-mediated pathways or indirectly by secreting platelet-derived molecules that regulate the coagulation cascade .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "Since SARS-CoV-2 , like influenza , is also a SSRNA virus , the possibility exists that SARS-CoV-2 may promote dysregulated platelet activity directly through surface receptor-mediated pathways or indirectly by secreting platelet-derived molecules that regulate the coagulation cascade .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "A recent study reports that SARS-CoV-2 may over-activate platelets and thus reduce platelet production ( 46 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001792 ! \"Blood Platelets\")\n",
      "A recent study reports that SARS-CoV-2 may over-activate platelets and thus reduce platelet production ( 46 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D001794 ! \"Blood Pressure\")\n",
      "Furthermore, SARS-CoV-2 by binding to ACE2 may raise BP in the brain, enhance the permeability of BBB, and increase the risk of stroke [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002415 ! Cats)\n",
      "These questions among others will need to be further addressed.Therefore, despite the susceptibility of some animal species revealed in observational or experimental conditions and making the juvenile cat, in particular, a potential model of SARS-CoV-2 replication, our results support evidence for a nil or very low COVID-19 rate of infection in companion dogs and cats, even in a situation of repeated contacts and close proximity to infected humans.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "2 : WBC infiltration and subsequent CNS damage mediated by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "Evidence suggests that SARS-CoV-2 can induce CNS hypoxia that leads to an increase in anaerobic metabolism required to trigger neurological damages, such as demyelinating lesions (33).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "There was no evidence for CNS damage directly caused by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "Although it is too early to elucidate the long-term side effects of SARS-CoV-2 infection, the background obtained with other respiratory viruses suggests that SARS-CoV-2 might induce permanent sequelae in the CNS through damage to the BBB, including dementia in predisposed patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "This indicates that SARS-CoV-2 may induce CNS tissue damage though cytokine pathways and the patients who burdened by severe infection might be more prone to serious neuropathological consequences.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "Suspected CNS infections caused by SARS-CoV-2 should be differentiated from other viruses, bacteria, and fungi.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "Consequently, it is crucial to understand whether SARS-CoV-2 can increase entrance to the CNS and induce neuronal damage, resulting in acute respiratory distress and death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003092 ! Colitis)\n",
      "40 Furthermore , SARS-CoV-2 has been reported to cause colitis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "COVID-19 : Early detection and timely diagnosis in a neurological setup Sir , Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease , coronavirus disease 2019 ( COVID - 19 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "This infectious disease is triggered by severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "This infectious disease is triggered by severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "This infectious disease is triggered by severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "COVID-19 : Early detection and timely diagnosis in a neurological setup Sir , Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease , coronavirus disease 2019 ( COVID - 19 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease and has per July 2020 spread to more than 200 countries and infected more than 15 million people ( WHO , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease and has per July 2020 spread to more than 200 countries and infected more than 15 million people ( WHO , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease ( Covid-19 ) and has per July 2020 spread to more than 200 countries and infected more than 15 million people ( WHO , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease ( Covid-19 ) and has per July 2020 spread to more than 200 countries and infected more than 15 million people ( WHO , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003141 ! \"Communicable Diseases\")\n",
      "Introduction Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) causes an infectious disease ( Covid-19 ) and has per July 2020 spread to more than 200 countries and infected more than 15 million people ( WHO , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Previous reports suggest that SARS-CoV-2 can cause conjunctivitis , either as an early sign of infection or during hospitalization for severe COVID-19 [ 8, 14 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Previous reports suggest that SARS-CoV-2 can cause conjunctivitis , either as an early sign of infection or during hospitalization for severe COVID-19 [ 8, 14 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Previous reports suggest that SARS-CoV-2 can cause conjunctivitis , either as an early sign of infection or during hospitalization for severe COVID-19 [ 8, 14 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Relevant to ophthalmologists , several published reports suggest that SARS-CoV-2 can cause conjunctivitis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Currently , it is still unclear how SARS-CoV-2 can cause conjunctivitis , but theories include : ( i ) conjunctiva can be a direct inoculation site for the virus , ( ii ) the virus can reach the upper respiratory tract through the nasolacrimal duct , or ( iii ) infection can occur via hematogenous through the lacrimal gland ( 123 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Currently , it is still unclear how SARS-CoV-2 can cause conjunctivitis , but theories include : ( i ) conjunctiva can be a direct inoculation site for the virus , ( ii ) the virus can reach the upper respiratory tract through the nasolacrimal duct , or ( iii ) infection can occur via hematogenous through the lacrimal gland ( 123 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Nucleic acid assays have demonstrated that SARS-CoV-2 can infect the ocular surface and cause conjunctivitis , yet the rate of RT-qPCR tests positive for SARS-CoV-2 RNA in ocular specimens appeared to be low ( Table 1 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003231 ! Conjunctivitis)\n",
      "Nucleic acid assays have demonstrated that SARS-CoV-2 can infect the ocular surface and cause conjunctivitis , yet the rate of RT-qPCR tests positive for SARS-CoV-2 RNA in ocular specimens appeared to be low ( Table 1 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003371 ! Cough)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003371 ! Cough)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Five other compounds, trimipramine, ingenol, abemaciclib, osimertinib, and NKH447 also protected against SARS-CoV-2 induced CPE, but to lesser degrees than cepharanthine.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "This was corroborated by a recent study, which found thatcepharanthine was able to block SARS-CoV-2 induced CPE in Vero E6 cells, only when added during the viral entry time period, and not the postentryperiod.27 Cepharanthine is a natural product usedin Japan since the 1950s for treatments of several diseases without majorside effects.28 It has polypharmacology, withanti-inflammatory activity linked to AMPK activation and NFkappaBinhibition.28 Cepharanthine has previously reportedantiviral activities against HIV, SARS-CoV, HCoV-OC43, human T-lymphotropicvirus (HTLV) and hepatitis B virus (HBV).29 We also identified two other approved drugs, abemaciclib and osimertinib asentry inhibitors that showed greater than 50% rescue of the SARS-CoV-2 CPE.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Five othercompounds, trimipramine, ingenol, abemaciclib, osimertinib, and NKH447also protected against SARS-CoV-2 induced CPE, but to lesser degreesthan cepharanthine.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "This was corroborated by a recent study, which found that cepharanthine was able to block SARS-CoV-2 induced CPE in Vero E6 cells, only when added during the viral entry time period, and not the postentry period.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "sgRNAs targeting 167 TMEM106B protected Huh7 cells from CPE caused by SARS-CoV-2, even when exposed to high virus 168 concentrations, but had not effect on cells infected with HCoV-229E.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "175 Immunohistochemistry based staining confirmed the extensive CPE induced by SARS-CoV-2 in 176 both healthy and COPD epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Immunohistochemistry based staining confirmed the extensive CPE induced by SARS-CoV-2 in both healthy and COPD epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "174 Immunohistochemistry based staining confirmed the extensive CPE induced by SARS-CoV-2 in 175 both healthy and COPD epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Plates were monitored daily for the presence of SARS-CoV-2 induced CPE for 4 days using an inverted optical microscope.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Here in, we discovered for the first time that CLQ, CLBQ14 and CLCQ effectively 117 inhibits the novel SARS-CoV-2 infection induced CPE in vitro, inhibited rhACE2 and its 118 interaction with Spike protein and rhACE2 exopeptidase activity in the low micromolar range.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "We found that all 243 three compounds inhibited SARS-CoV-2 infection induced CPE in vitro with 50% Inhibitory 244 Concentration (IC50) values in the low micromolar concentration.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "SARS-CoV2 infection on our HEP-2 cells induced CPE after 120h of infection with high virus multiplication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Also NCI-H2110 cells, in which SARS-CoV-2 causes only a very limited CPE, were protected 205 against SARS-CoV-2-induced cell death by sgRNAs targeting TMEM106B, as determined by MTS 206 assay.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Among the six most potent compounds in the 3CL pro enzyme assay, walrycin B and Z-FA-FMK showed the rescue of SARS-CoV-2 induced CPE with the efficacies of 51.43% and 104.84%, respectively.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "201 202 Vero E6 cells were selected for this study because SARS-COV-2 produces high titers 203 and CPE with a fast replication rate in this cell line.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "The rest of the screening data can be found in Table S1.SCAP is a required and druggable SARS-CoV-2 and pan-coronavirus host factorA central goal of this study was to use a predicted SARS-CoV-2 interactome to identify host factors that are required for virus infection or virus induced CPE by multiple coronaviruses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "We successfully identified and characterized CLQ, a known metal chelator and zinc 328 ionophore, as a novel inhibitor of SARS-CoV-2 infection induced CPE.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "In conclusion, our SARS-CoV-2 interactome screen identified a compendium of host genes that can be disrupted without affecting cell viability in the absence of infection, yet seem to play essential roles during SARS-CoV-2 induced CPE.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Also NCI-H2110 cells , in which SARS-CoV-2 causes only a very limited CPE , were protected 204 against SARS-CoV-2-induced cell death by sgRNAs targeting TMEM106B , as determined by MTS 205 assay ( Figure 4B ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV-2 primarily uses the immune evasion process during infection via the envelope spike glycoprotein , which is followed by a cytokine storm , causing severe acute respiratory disease syndrome and death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Similar to SARS-CoV and Middle Eastern respiratory syndrome ( MERS ) - CoV , SARS-CoV-2 can cause death by triggering acute respiratory failure [ 6 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Similar to SARS-CoV and Middle Eastern respiratory syndrome ( MERS ) - CoV , SARS-CoV-2 can cause death by triggering acute respiratory failure [ 6 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Similar to SARS-CoV and Middle Eastern respiratory syndrome ( MERS ) - CoV , SARS-CoV-2 can cause death by triggering acute respiratory failure [ 6 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Although infrequent, severe respiratory infection and death caused by SARS-CoV-2 is disproportionately high amongst healthcare providers such as craniofacial surgeons who work in the head and neck region.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Severe acute respiratory coronavirus 2 (SARS-CoV-2) has spread and caused death worldwide.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "[3] As of August 23 rd, SARS-CoV-2 has infected over 23 million worldwide and led to the death of 116 over 800,000.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Like SARS-CoV and MERS-CoV, SARS-CoV-2 can also induce excessive and aberrant non effective host immune responses that are associated with severe lung pathology, and lead to death (Hui and Zumla, 2019; Huang et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "We further show that SARS-CoV-2 causes significant neuronal death in human brain organoids.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Previous studies confirmed that SARS-CoV-2 invades host cells through the ACE2 receptor ( Rivellese and Prediletto , 2020 ) , and ACE2 is expressed not only in the lung but also in the gastrointestinal tract , kidney , heart , blood vessels and other organs and tissues , causing SARS-CoV-2 to damage not only the lung but also the heart and kidneys , as well as physiological systems such as the immune system and blood system , causing death in patients due to multiple organ failure and acute respiratory distress syndrome ( Bourgonje et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "As of 27 July 2020, SARS-CoV-2 has infected over 16 million people and led to the death of more than 652,434 individuals as on 27 July 2020 while also causing significant economic losses.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Thromboembolic events as well as severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) are the major causes of death in this continued pandemic .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Thromboembolic events as well as severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) are the major causes of death in this continued pandemic .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has caused massive societal disruption and death globally.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "It has been hypothesised that SARS-CoV-2 may directly infect lymphocytes , particularly T cells , promoting their death , with consequent lymphopenia and impaired antiviral responses [ 16 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Thromboembolic events as well as severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) are the major causes of death in this continued pandemic.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Previous studies confirmed that SARS-CoV-2 invades host cells through the ACE2 receptor ( Rivellese and Prediletto , 2020 ) , and ACE2 is expressed not only in the lung but also in the gastrointestinal tract , kidney , heart , blood vessels and other organs and tissues , causing SARS-CoV-2 to damage not only the lung but also the heart and kidneys , as well as physiological systems such as the immune system and blood system , causing death in patients due to multiple organ failure and acute respiratory distress syndrome ( Bourgonje et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "3 ,  4 It is currently believed that SARS-CoV-2 primarily infects the lungs , and subsequently causes systemic inflammation and immune response disorder , and ultimately leads to multiple organ injury and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "3 , 4 It is currently believed that SARS-CoV-2 primarily infects the lungs , and subsequently causes systemic inflammation and immune response disorder , and ultimately leads to multiple organ injury and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "It is currently believed that SARS-CoV-2 primarily infects the lungs , and subsequently causes systemic inflammation and immune response disorder , and ultimately leads to multiple organ injury and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Recently, Ma et al. [2] disclosed that surge in temperature and humidity conditions have resulted in the decline of death caused by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Like SARS-CoV, SARS-CoV-2 causes a severe interstitial pneumonia that can lead to acute respiratory distress syndrome (ARDS) and death [1].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Unfortunately, SARS-CoV-2 infection might result in more severe complications including severe pneumonia and acute respiratory distress syndrome, and may also lead to death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Severe SARS-CoV-2 infection, also known as coronavirus disease 2019 (COVID-19), can lead to death due to acute respiratory distress syndrome mediated by inflammatory immune cells and cytokines.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Like SARS-CoV and MERS-CoV, SARS-CoV-2 can also induce excessive and aberrant non effective host immune responses that are associated with severe lung pathology, and lead to death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "It is made available under a CC-BY-ND 4.0 International license.INTRODUCTION Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has caused a global pandemic of coronavirus disease 2019 (COVID-19), with over 33.1 million infections and 998,000 death as reported on September 28 of 2020 (Dong et al., 2020; Gorbalenya et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The current fast-evolving coronavirus disease 2019 ( COVID-19 ) pandemic is caused by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which infects lungs and can lead to severe lung injury , multiorgan failure , and death ( Li et al ., 2020 ; Wiersinga et al ., 2020 ; Zhu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Compared to the other members of the family Coronaviridae, which includes the well-known severe acute respiratory syndrome coronavirus and Middle East respiratory syndrome coronavirus, SARS-CoV-2 is transmitted more rapidly and may lead to severe respiratory distress syndrome or even death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) pandemic has caused extensive disease and death , with heavy social and economic losses [ 1 ] [ 2 ] [ 3 ] [ 4 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) pandemic has caused extensive disease and death , with heavy social and economic losses [ 1 ] [ 2 ] [ 3 ] [ 4 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Due to these high transmissibility and extensive community spread, SARS-CoV-2 has already caused nearly 20 million infection and over 700,000 death as of August 2020 (3).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Severe SARS-CoV-2 infection, also known as coronavirus disease 2019, can lead to death due to acute respiratory distress syndrome mediated by inflammatory immune cells and cytokines.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Here, we can present a case of cytokine release syndrome induced by SARS-CoV-2 causing multiorgan failure and death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "[ 1 ] [ 2 ] [ 3 ] The majority of the early investigations focused on how SARS-CoV-2 attacks the respiratory system and elicits typical symptoms in most patients ; nonetheless , new research disturbingly indicates that SARS-CoV-2 can also attack the central nervous system ( CNS ) in a variety of ways and even cause long-term damage or death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "1 - 3 The majority of the early investigations focused on how SARS-CoV-2 attacks the respiratory system and elicits typical symptoms in most patients ; nonetheless , new research disturbingly indicates that SARS-CoV-2 can also attack the central nervous system ( CNS ) in a variety of ways and even cause long-term damage or death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Our current report suggests that SARS-CoV2 does not require its traditional infective property to cause death.12 bioRxiv preprint doi : https://doi.org/10.1101/2020.11.13.378257; this version posted November 16, 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "1 The outbreak in a nursing home in Kirkland , 66 Washington 2 was the first to report death due to SARS-CoV-2 in the United States ( U.S ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "When cells are productively infected, SARS-CoV-2 virions are released at the end of the viral life cycle, causing cell lysis and death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "[ 3 ] As of July 30th , SARS-CoV-2 has infected over 17 million worldwide and led to the death of over 650,000 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Despite global efforts to contain the COVID-19 pandemic, SARS-CoV-2 is now found in over 200 countries and has caused an upward death toll of over 1 million human lives as of November 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Despite global efforts to contain the 28 COVID-19 pandemic, SARS-CoV-2 is now found in over 200 countries and has caused 29 an upward death toll of over 1 million human lives as of November 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Like SARS-CoV and MERS-CoV , SARS-CoV-2 can also induce excessive and aberrant non-effective host immune responses that are associated with severe lung pathology , and lead to death ( Hui and Zumla , 2019 ; Huang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Both infection and death caused by SARS-CoV-2 are still rapidly increasing worldwide.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Like SARS-CoV and MERS-CoV , SARS-CoV-2 can also induce excessive and aberrant non-effective host immune responses that are associated with severe lung pathology , and lead to death ( Hui and Zumla , 2019 ; Huang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Compared to the other members of the family Coronaviridae , which includes the well-known severe acute respiratory syndrome coronavirus and Middle East respiratory syndrome coronavirus , SARS-CoV-2 is transmitted more rapidly and may lead to severe respiratory distress syndrome or even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV-2 enhances ACE2 leading to inflammation and neuronal death with possible development of mood disorders, such as depression and anxiety.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "While the common symptoms are influenza like, SARS-CoV-2 infection may lead to multiorgan dysfunction and death; fragile patients have, as in other viral diseases, a higher risk of adverse events.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "XREF_BIBR - XREF_BIBR The majority of the early investigations focused on how SARS-CoV-2 attacks the respiratory system and elicits typical symptoms in most patients; nonetheless, new research disturbingly indicates that SARS-CoV-2 can also attack the central nervous system (CNS) in a variety of ways and even cause long-term damage or death.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The current fast-evolving coronavirus disease 2019 pandemic is caused by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which infects lungs and can lead to severe lung injury , multiorgan failure , and death Wiersinga et al ., 2020 ; Zhu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The three most pathogenic human coronaviruses , SARS-CoV-2 , SARS-CoV , and MERS-CoV more commonly cause life-threatening pneumonia and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "The three most pathogenic human coronaviruses , SARS-CoV-2 , SARS-CoV , and MERS-CoV more commonly cause life-threatening pneumonia and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infection in humans can cause pneumonia, acute respiratory distress syndrome, acute lung injury, cytokine storm syndrome and death [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "[ 3 ] As of August 23rd , SARS-CoV-2 has infected over 23 million worldwide and led to the death of over 800,000 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "[ 3 ] As of August 23 rd , SARS-CoV-2 has infected over 23 million worldwide and led to the death of 116 over 800,000 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infection in humans can cause pneumonia, acute respiratory distress syndrome, acute lung injury, cytokine storm syndrome and death [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "Furthermore, Zhu et al. observed ten neonates born from mothers infected by COVID-19 and suggested that SARS-COV-2 may cause several harmful effects such as premature labor, respiratory distress, thrombocytopenia or abnormalities in liver function tests or maybe death on neonates however, all their neonate 's COVID-19 RT-PCRs were negative [1].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003643 ! Death)\n",
      "We further show that SARS-CoV-2 causes significant neuronal death in human brain organoids.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV and SARS-CoV-2 cause mild disease and no death was reported in this model .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003681 ! Dehydration)\n",
      "Since the median admission serum sodium levels were well beneath 145 mmol / L , we can postulate that overly restrictive ICU fluid intakes may lead to dehydration , which in our study was present in 52 % of the patients with ARDS due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "Whether SARS-CoV-2 induces efficient pDC activation, or may interfere with various biological pathways in pDC is currently unknown.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "SARS-CoV-2 induces activation and diversification of primary human pDC.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "First, we asked whether SARS-CoV-2 was able to induce pDC activation, and diversification into IFN producing and/or T cell stimulating effectors, as we previously described for influenza virus A (Flu).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "For intracellular cytokine staining, SARS-CoV-2 was used at a MOI of 2 to activate pDC.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "This lack of diversification was associated with a complete absence of IFN-alpha2, IP-10, and IL-6 secretion, establishing a critical role of these molecular nodes in SARS-CoV-2 activation of pDC (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "These data showed that IFN-alpha and TNF-alpha production are features of P1-pDC, as well as P2-pDC, and that SARS-CoV-2 induces a functional specialization of pDC subsets, similar to Flu.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "SARS-CoV-2 induced a marked diversification of tonsillar pDC into all three activated subsets (XREF_SUPPLEMENTARY).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "Thus, in addition to further driving the cytokine storm, SARS-CoV-2 may infect DCs to limit DC maturation, and thus suppress T cell mediated responses (44, 152, 153).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "In SARS-CoV infections , the intestinal inflammation is mainly modulated by ACE2 receptor mediated mechanism and therefore , its disruption by SARS-CoV-2 may result in diarrhea and other GIT related complications .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "Although SARS-CoV-2 RNA has been detected in stool samples from infected individuals ( Bwire et al ., 2020 ; Wu et al ., 2020c ; Xu et al ., 2020 ) , and may cause diarrhea ( Chen et al ., 2020 ; Cholankeril et al ., 2020 ) transmission by ingestion of contaminated food or the fecal-oral route has not been proven .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "SARS-CoV-2 can invade the enterocytes and result in diarrhea .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "SARS-CoV-2 can invade the enterocytes and result in diarrhea .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "Patients with COVID-19 likely have low rates of non - C. difficile enteric infections because SARS-CoV-2 itself is the cause of diarrhea on presentation ; however , over time these patients become more similar to other hospitalized patients and have a similar rate of C. difficile infections compared to patients without COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "difficile enteric infections because SARS-CoV-2 itself is the cause of diarrhea on presentation ; however , over time these patients become more similar to other hospitalized patients and have a similar rate of C. difficile infections compared to patients without COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D003967 ! Diarrhea)\n",
      "Patients with COVID-19 likely have low rates of non - C. difficile enteric infections because SARS-CoV-2 itself is the cause of diarrhea on presentation ; however , over time these patients become more similar to other hospitalized patients and have a similar rate of C. difficile infections compared to patients without COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004211 ! \"Disseminated Intravascular Coagulation\")\n",
      "The implications extend outside the lungs ; SARS-CoV-2 can lead to septic shock , acute kidney injury , and disseminated intravascular coagulation ( DIC ) in some of the most severe cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004211 ! \"Disseminated Intravascular Coagulation\")\n",
      "The implications extend outside the lungs ; SARS-CoV-2 can lead to septic shock , acute kidney injury , and disseminated intravascular coagulation ( DIC ) in some of the most severe cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004249 ! \"DNA Damage\")\n",
      "Intriguingly, SARS-CoV-2 causes DNA damage, cell growth arrest, cytokine production, and p38 activation during the early stages of infection as a strategy to promote more efficient replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004249 ! \"DNA Damage\")\n",
      "Consistent with the concept that coronaviruses can induce cell growth arrest to promote their replication, Bouhaddou et al. recently demonstrated that SARS-CoV-2 causes both DNA damage during the early stage of infection and cell cycle arrest [66].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004249 ! \"DNA Damage\")\n",
      "Consistent with the concept that coronaviruses can induce cell growth arrest to promote their replication, Bouhaddou et al. recently demonstrated that SARS-CoV-2 causes both DNA damage during the early stage of infection and cell cycle arrest [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004487 ! Edema)\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004487 ! Edema)\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004487 ! Edema)\n",
      "Penetration through the Blood-brain - barrier Once in the blood circulation , SARS-CoV-2 could bind to the ACE2 receptors of the endothelium [ 50 , 52 , 55 ] and disrupt the Blood Brain Barrier ( BBB ) , inducing edema , intracranial hypertension and / or penetration of the virus in the CNS [ 64 , 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004826 ! Epiglottitis)\n",
      "In fact , a number of viral agents have previously been implicated as pathogens causing epiglottitis [ 4 ] , and therefore it is plausible for SARS-CoV-2 to be the cause for acute epiglottitis in the above case .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004826 ! Epiglottitis)\n",
      "In fact , a number of viral agents have previously been implicated as pathogens causing epiglottitis [ 4 ] , and therefore it is plausible for SARS-CoV-2 to be the cause for acute epiglottitis in the above case .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004848 ! Epithelium)\n",
      "Third , in contrast to the 219 healthy epithelium , SARS-CoV-2 induced higher squamous metaplasia in the infected COPD 220 epithelium and caused a substantial increase in the height of the epithelium ( Figure 6D ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004890 ! Erythema)\n",
      "Based on our experience and those described in other reports , it is highly possible that SARS-CoV-2 can cause erythema .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004890 ! Erythema)\n",
      "Considering previous studies and the three cases reported here , there is a high probability that SARS-CoV-2 can cause erythema .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004890 ! Erythema)\n",
      "Considering previous studies and the three cases reported here , there is a high probability that SARS-CoV-2 can cause erythema .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004890 ! Erythema)\n",
      "Based on our experience and those described in other reports , it is highly possible that SARS-CoV-2 can cause erythema .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D004906 ! \"Erythrocyte Count\")\n",
      "The CD147 are found abundantly on the red blood cells (RBCs) and SARS-CoV-2 is hypothesized to cause clumping of RBCs impending the blood flow.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D004906 ! \"Erythrocyte Count\")\n",
      "Hence, depletion of RBCs by SARS-CoV-2 may result in stress erythropoiesis as a compensatory mechanism to meet the oxygen supply, resulting in the abundance of erythroid precursors in the blood.Erythroid precursors are defined as CD71 + erythroid cells (CECs) co-expressing CD71 (the transferrin receptor) and CD235a (glycophorin A, erythroid lineage marker) in humans, and CD71 and TER119 in mice [22] [23] [24].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "Abstract The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "Abstract The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005221 ! Fatigue)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005355 ! Fibrosis)\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D005355 ! Fibrosis)\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006261 ! Headache)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006261 ! Headache)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006333 ! \"Heart Failure\")\n",
      "The intricate pathophysiological mechanism involved must be understood to improve the efficiency of therapy for HF induced by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006335 ! \"Heart Injuries\")\n",
      "In this study , they found six children with high CK-MB , which means that SARS-CoV-2 could cause heart injury .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "This has public health officials especially concerned about two respiratory pathogens circulating this influenza season i.e. influenza and SARS-CoV-2 which may result in increased hospitalizations that could overwhelm the healthcare infrastructure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "This has public health officials especially concerned about two respiratory pathogens circulating this influenza season i.e. influenza and SARS-CoV-2 which may result in increased hospitalizations that could overwhelm the healthcare infrastructure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "This has public health officials especially concerned about two respiratory pathogens circulating this influenza season i.e. influenza and SARS-CoV-2 which may result in increased hospitalizations that could overwhelm the healthcare infrastructure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) pandemic of 2019-2020 has resulted in multiple hospitalizations , deaths and economic hardships worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "In March , 2020 , the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the causal agent of coronavirus disease 2019 ( COVID-19 ) , was spreading in the Bay Area , especially in Santa Clara County , causing increases in cases , hospitalizations , and deaths .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006760 ! Hospitalization)\n",
      "In March , 2020 , the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the causal agent of coronavirus disease 2019 ( COVID-19 ) , was spreading in the Bay Area , especially in Santa Clara County , causing increases in cases , hospitalizations , and deaths .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006801 ! Humans)\n",
      "SARS-CoV-2 is highly pathogenic virus , possesses mutative behaviour and causes rapid human to human spread .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006863 ! \"Hydrogen-Ion Concentration\")\n",
      "Interestingly, SARS-CoV-2 has recently been shown to also increase the pH of lysosomes, possibly through the open reading frame protein 3A (ORF3a) (Ghosh et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006943 ! Hyperglycemia)\n",
      "Moreover , obesity causes hyperglycemia via insulin resistance whereas growing evidence demonstrates that SARS-CoV-2 may cause hyperglycemia as well by infecting and killing B-cells ( 66 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006943 ! Hyperglycemia)\n",
      "SARS-CoV-2 may also cause hyperglycemia as well by infecting and killing pancreatic B-cells , leading to a worsen metabolic dysfunction of people with obesity and a poor prognostic of COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006943 ! Hyperglycemia)\n",
      "Moreover , obesity causes hyperglycemia via insulin resistance whereas growing evidence demonstrates that SARS-CoV-2 may cause hyperglycemia as well by infecting and killing B-cells ( 66 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D006943 ! Hyperglycemia)\n",
      "SARS-CoV-2 may also cause hyperglycemia as well by infecting and killing pancreatic B-cells , leading to a worsen metabolic dysfunction of people with obesity and a poor prognostic of COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007070 ! \"Immunoglobulin A\")\n",
      "Because ( i ) IgM and IgG ratios are not indicative of the timing of disease onset ( Hou et al ., 2020 ; Long et al ., 2020b ; Qu et al ., 2020 ) , ( ii ) IgA is induced by SARS-CoV-2 ( Isho et al ., 2020 ; Iyer et al ., 2020 ; Sterlin et al ., 2020 ) , and ( iii ) both IgG and IgM isotypes correlated with neutralizing titers , we chose to quantify total ( all isotypes ) antigen-specific antibodies for seroprevalence studies .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "The idea that circulating HCoVs elicit IgG that cross-reacts with SARS-CoV-2 is supported by the finding that SARS-CoV-2 infection increases IgG titers against the S proteins of multiple HCoVs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "The idea that circulating HCoVs elicit IgG that cross-reacts with SARS-CoV-2 is supported by the finding that SARS-CoV-2 infection increases IgG titers against the S proteins of multiple HCoVs (13).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "SARS-CoV2 is likely to trigger IgM and IgG production in a similar humoral immunity response to SARS-CoV [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "[XREF_BIBR] According to a study from China with limited data, SARS-CoV-2 triggers the formation of both IgM, appearing by day 3, and IgG, appearing by day 8, in approximately 95% of COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007075 ! \"Immunoglobulin M\")\n",
      "[ 28 ] According to a study from China with limited data , SARS-CoV-2 triggers the formation of both IgM , appearing by day 3 , and IgG , appearing by day 8, in approximately 95 % of COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007075 ! \"Immunoglobulin M\")\n",
      "SARS-CoV2 is likely to trigger IgM and IgG production in a similar humoral immunity response to SARS-CoV [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007075 ! \"Immunoglobulin M\")\n",
      "[XREF_BIBR] According to a study from China with limited data, SARS-CoV-2 triggers the formation of both IgM, appearing by day 3, and IgG, appearing by day 8, in approximately 95% of COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007333 ! \"Insulin Resistance\")\n",
      "In addition , from our experience , and reported data [ 29 ] , patients with COVID-19 with ( or without ) diabetes often exhibit hyperglycaemia upon hospital admission , which is likely to be attributed to direct virus-induced decrease in insulin secretion and increase in insulin resistance by SARS-CoV-2 [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007558 ! Italy)\n",
      "It became clear that , without appropriate measures , SARS-CoV-2 would have spread dangerously all over Italy .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007683 ! \"Kidney Tubular Necrosis, Acute\")\n",
      "Recently , Diao et al. [ 10 ] demonstrated that SARS-CoV-2 mainly induced acute tubular necrosis by infecting kidney tubules directly .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D007683 ! \"Kidney Tubular Necrosis, Acute\")\n",
      "Recently , Diao et al. [ 10 ] demonstrated that SARS-CoV-2 mainly induced acute tubular necrosis by infecting kidney tubules directly .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008107 ! \"Liver Diseases\")\n",
      "A preliminary study reported that the ACE2 receptor is abundantly expressed in cholangiocytes ( 36 ) ; suggesting that SARS-CoV-2 might bind to ACE2-positive cholangiocytes and cause liver dysfunction ( 15 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008107 ! \"Liver Diseases\")\n",
      "This suggests that SARS-Cov-2 could potentially infect bile duct cells and cause abnormal liver function tests .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008107 ! \"Liver Diseases\")\n",
      "This suggests that SARS-Cov-2 could potentially infect bile duct cells and cause abnormal liver function tests .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008107 ! \"Liver Diseases\")\n",
      "A preliminary study reported that the ACE2 receptor is abundantly expressed in cholangiocytes ( 36 ) ; suggesting that SARS-CoV-2 might bind to ACE2-positive cholangiocytes and cause liver dysfunction ( 15 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008168 ! Lung)\n",
      "Additionally , SARS-CoV-2 is much more deadly than the chicken pox or influenza viruses and has shown to cause long-lasting effects on the lungs , the heart , and the central nervous system , which are still not fully understood [ 197 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "11 However , when these two pathways were triggered to a hyper-activation status , IAV infection leads to severe lung damage and fatal infection outcome.11 Similar with IAV , the SARS-CoV-2 induced a dual mode of cell death pathways in the lung epithelial cells , which may pose anti-viral responses or immune pathogenesis depending on the activation status .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "We therefore suggest that the mechanism proposed could explain how SARS-CoV-2 causes severe lung damage without requiring a hyperinflammatory reaction resulting from a cytokine storm initiated by activating an innate immunity response .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "We therefore suggest that the mechanism proposed could explain how SARS-CoV-2 causes severe lung damage without requiring a hyperinflammatory reaction resulting from a cytokine storm initiated by activating an innate immunity response .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "We therefore suggest that the mechanism proposed could explain how SARS-CoV-2 causes severe lung damage without requiring a hyperinflammatory reaction resulting from a cytokine storm initiated by activating an innate immunity response .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death.2 Therefore , public health and infection control measures are needed to limit the overall spread of the virus ,3 and all individuals exhibiting symptoms indicative of COVID-19 should adhere to basic prevention measures.4 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "1 This disease is caused by a new strain of coronavirus , the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes severe lung damage , pneumonia , and even death.2 Therefore , public health and infection control measures are needed to limit the overall spread of the virus ,3 and all individuals exhibiting symptoms indicative of COVID-19 should adhere to basic prevention measures.4 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "Previous studies confirmed that SARS-CoV-2 invades host cells through the ACE2 receptor ( Rivellese and Prediletto , 2020 ) , and ACE2 is expressed not only in the lung but also in the gastrointestinal tract , kidney , heart , blood vessels and other organs and tissues , causing SARS-CoV-2 to damage not only the lung but also the heart and kidneys , as well as physiological systems such as the immune system and blood system , causing death in patients due to multiple organ failure and acute respiratory distress syndrome ( Bourgonje et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "Previous studies confirmed that SARS-CoV-2 invades host cells through the ACE2 receptor ( Rivellese and Prediletto , 2020 ) , and ACE2 is expressed not only in the lung but also in the gastrointestinal tract , kidney , heart , blood vessels and other organs and tissues , causing SARS-CoV-2 to damage not only the lung but also the heart and kidneys , as well as physiological systems such as the immune system and blood system , causing death in patients due to multiple organ failure and acute respiratory distress syndrome ( Bourgonje et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "As SARS-COV-2 mainly causes lung damage , we then investigated the influence of immune cells and cytokines on the outcome of pulmonary lesions in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "SARS-CoV-2 could cause severe , even lethal pneumonia and lung failure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "SARS-CoV-2 could cause severe , even lethal pneumonia and lung failure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "As SARS-CoV-2 causes damage to vascular endothelial cells and induces severe lung damage , we evaluated whether DNase-I can protect vascular barrier disruption by observing the transendothelial permeability in the septic mouse model ( Figure S5 , Supporting Information ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "However , SARS-CoV-2 seems to promote lung damage through different mechanisms , such as the scarce participation Th1 / Th17 response and the higher participation of the Th2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "In contrast , SARS-CoV-2 seems to promote lung damage through different mechanisms , such as the scarce participation Th1 / Th17 response , and the higher participation of the Th2 , when combined , might be inefficient for viral clearance .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "In contrast , SARS-CoV-2 seems to promote lung damage through different mechanisms , such as the scarce participation Th1 / Th17 response , and the higher participation of the Th2 , when combined , might be inefficient for viral clearance .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "However , SARS-CoV-2 seems to promote lung damage through different mechanisms , such as the scarce participation Th1 / Th17 response and the higher participation of the Th2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "SARS-CoV-2 infects the host cells via ACE2 receptors and causes damage to lungs , CNS , heart , colon , kidneys , and eyes ; hence , attention must be given when treating COVID-19 patients to its systemic effects .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "Coronaviruses commonly attack the respiratory system and SARS-CoV-2 has been shown to cause lung damage ( 22 , 23 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008168 ! Lung)\n",
      "Coronaviruses commonly attack the respiratory system and SARS-CoV-2 has been shown to cause lung damage ( 22 , 23 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Due to ACE2 expression in mature lymphocytes , SARS-CoV-2 may result in lymphopenia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Due to ACE2 expression in ( Continued ) FIGURE 2 | Continued mature lymphocytes , SARS-CoV-2 may result in lymphopenia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Both SARS-CoV-2 - and immune-mediated mechanisms may contribute to the occurrence of lymphopenia through influencing the lymphocyte production , survival , or tissue re-distribution .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Another study showed that SARS-CoV-2 induces lymphocytopenia which allows the diminished T cells to be more susceptible to SARS-CoV-2 , and this abnormal T-cell expression , and associated mRNA , have been shown to lead to venous thrombo-embolism .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Another study showed that SARS-CoV-2 induces lymphocytopenia which allows the diminished T cells to be more susceptible to SARS-CoV-2 , and this abnormal T-cell expression , and associated mRNA , have been shown to lead to venous thrombo-embolism .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Another study showed that SARS-CoV-2 induces lymphocytopenia which allows the diminished T cells to be more susceptible to SARS-CoV-2 , and this abnormal T-cell expression , and associated mRNA , have been shown to lead to venous thrombo-embolism .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Although SARS-CoV-2 can induce lymphopenia and cause a delay in T cell pathway activation during the first days of infection , after two weeks of symptoms , SARS-CoV-2-specific memory T cell phenotypes start to emerge in the peripheral blood , providing protective immunity ( 28 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "2019-nCoV also targets these circulating immune cells and induces apoptosis of CD3 , CD8 , and CD4 cells , causing lymphocytopenia , overproduction of cytokines , and failure of multiple organs [ 15 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Furthermore , SARS-Cov-2 causes relatively serious lymphocytopenia and lymphocytes exhaustion .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Furthermore , SARS-Cov-2 causes relatively serious lymphocytopenia and lymphocytes exhaustion .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Furthermore , SARS-Cov-2 causes relatively serious lymphocytopenia and lymphocytes exhaustion .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008407 ! \"Mast Cells\")\n",
      "Mast cells ( MCs ) are activated by SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008407 ! \"Mast Cells\")\n",
      "Mast cells ( MCs ) are activated by SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008407 ! \"Mast Cells\")\n",
      "9On the other hand , SARS-CoV-2 can activate mast cells in the submucosa of the 10 respiratory tract to degranulate and release histamine , which may lead to aggravation 11 of respiratory system inflammation ( Kritas et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008407 ! \"Mast Cells\")\n",
      "On the other hand , SARS-CoV-2 can activate mast cells in the submucosa of the respiratory tract to degranulate and release histamine , which may lead to aggravation of respiratory system inflammation ( Kritas et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008587 ! \"Meningitis, Viral\")\n",
      "In this context , SARS-CoV-2 can also cause viral meningitis and encephalitis , as demonstrated by a recent report of a 64-yearold patient with laboratory-confirmed COVID-19 who presented neurologic manifestations during the infection , including lethargy , clonus , and pyramidal signs in the lower limbs as well as stiff neck and Brudzinski sign ( 76 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008587 ! \"Meningitis, Viral\")\n",
      "In this context , SARS-CoV-2 can also cause viral meningitis and encephalitis , as demonstrated by a recent report of a 64-year-old patient with laboratory-confirmed COVID-19 who presented neurologic manifestations during the infection , including lethargy , clonus , and pyramidal signs in the lower limbs as well as stiff neck and Brudzinski sign ( 76 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008587 ! \"Meningitis, Viral\")\n",
      "In this context , SARS-CoV-2 can also cause viral meningitis and encephalitis , as demonstrated by a recent report of a 64-year-old patient with laboratory-confirmed COVID-19 who presented neurologic manifestations during the infection , including lethargy , clonus , and pyramidal signs in the lower limbs as well as stiff neck and Brudzinski sign ( 76 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Bernard and colleagues reported about two ladies who had meningo-encephalitis due to SARS-CoV-2 [ 25 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Bernard and colleagues reported about two ladies who had meningo-encephalitis due to SARS-CoV-2 [ 25 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Bernard and colleagues reported about two ladies who had meningo-encephalitis due to SARS-CoV-2 [ 25 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Recently , meningoencephalitis due to SARS-CoV-2 has been described [ 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Recently , meningoencephalitis due to SARS-CoV-2 has been described [ 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "Recently , meningoencephalitis due to SARS-CoV-2 has been described [ 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "There are few reports of direct involvement of the central nervous system with SARS-CoV-2 causing meningoencephalitis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D008590 ! Meningoencephalitis)\n",
      "To help answer the question of whether SARS-CoV-2 causes meningoencephalitis in encephalopathic COVID-19 patients , we utilized a two-pronged approach .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe Acute Respiratory Syndrome ( SARS-CoV-2 ) has spread rapidly throughout the world , causing high morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "PURPOSE OF REVIEW Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "The spread of SARS-CoV-2 and the COVID-19 pandemic have caused significant morbidity and mortality worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "SARS-CoV-2 has spread globally to cause significant morbidity and mortality , and there still remain unknowns about the biology and pathology of the virus .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "SARS-CoV-2 has spread globally to cause significant morbidity and mortality , and there still remain unknowns about the biology and pathology of the virus .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the virus that causes coronavirus disease 2019 ( COVID - 19 ) , has resulted in substantial morbidity and mortality worldwide 1 , 2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Aptamer BC 007 - Efficient binder of spreading-crucial SARS-CoV-2 proteins Corona virus disease 2019 ( COVID-19 ) is a respiratory disease caused by a new coronavirus ( SARS-CoV-2 ) which causes significant morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the virus responsible for the coronavirus disease ( COVID-19 ) pandemic , has contributed to significant morbidity and mortality , and presents with a myriad of clinical manifestations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the virus responsible for the coronavirus disease ( COVID-19 ) pandemic , has contributed to significant morbidity and mortality , and presents with a myriad of clinical manifestations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Corona virus disease 2019 ( COVID-19 ) is a respiratory disease caused by a new coronavirus ( SARS-CoV-2 ) which causes significant morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Since SARS-CoV-2 causes morbidity and mortality which escalate throughout the world , there might be temptation to progress with the clinical use of novel and unproven therapeutics .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Since SARS-CoV-2 causes morbidity and mortality which escalate throughout the world , there might be temptation to progress with the clinical use of novel and unproven therapeutics .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "The 2019 novel coronavirus has been reported to cause significant morbidity in adults with reportedly a lesser impact on children .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "The continuous increase in morbidities and mortalities due to SARS-CoV-2 leads to international medical health worsen situations day after day .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "The continuous increase in morbidities and mortalities due to SARS-CoV-2 leads to international medical health worsen situations day after day .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009017 ! Morbidity)\n",
      "Deficiencies in SPMs , driven by obesity , its co-morbidities , and chronic pulmonary environmental exposures , could exacerbate the SARS-CoV-2 induced morbidities and mortalities .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009017 ! Morbidity)\n",
      "This commendable - and hopefully permanent - advance in medicine and epidemiology has facilitated more rapid and cross-disciplinary science to rapidly improve understanding around how to mitigate morbidity and mortality due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Cardiopulmonary involvement in COVID-19 ARDS , myocardial injury , disseminated intravascular coagulation ( DIC ) especially of the pulmonary vasculature , are the major contributors to mortality due to SARS-CoV-2 ( Chen C , Yan JT , Zhou N , Zhao JP , 2020 ; Guo et al ., 2020 ; McGonagle et al ., 2020 ; Xie and Chen , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "ARDS , myocardial injury , disseminated intravascular coagulation ( DIC ) especially of the pulmonary vasculature , are the major contributors to mortality due to SARS-CoV-2 ( Chen C , Yan JT , Zhou N , Zhao JP , 2020 ; Guo et al ., 2020 ; McGonagle et al ., 2020 ; Xie and Chen , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has infected millions within just a few months , causing severe respiratory disease and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has infected millions within just a few months , causing severe respiratory disease and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV-2 induces a mortality rate approximately ten times lower 23 than SARS-CoV-1 , but shows much higher effective transmissibility , and thus represents a greater threat 24 to global health ( Petersen et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "With increased mortality due to SARS-CoV-2 , valuable clinical indicators or treatments should be further identified and summarized .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "With increased mortality due to SARS-CoV-2 , valuable clinical indicators or treatments should be further identified and summarized .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe Acute Respiratory Syndrome ( SARS-CoV-2 ) has spread rapidly throughout the world , causing high morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The spread of SARS-CoV-2 and the COVID-19 pandemic have caused significant morbidity and mortality worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has rapidly spread throughout the world causing significant mortality in high risk patients with severe manifestations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "New coronavirus ( SARS-CoV-2 ) has been spreading more than 200 countries , causing huge mortalities .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The global pandemic secondary to the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) is leading to unprecedented global morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The global pandemic secondary to the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) is leading to unprecedented global morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV-2 has spread globally to cause significant morbidity and mortality , and there still remain unknowns about the biology and pathology of the virus .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Though the estimated mortality rate due to SARS-CoV-2 is about 3.6 % in China and about 1.5 % outside China20 , the problem is grave and need to be respond urgently to curb the rapid loss of human life .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Though the estimated mortality rate due to SARS-CoV-2 is about 3.6 % in China and about 1.5 % outside China20 , the problem is grave and need to be respond urgently to curb the rapid loss of human life .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Management principles in patients with COVID-19 : perspectives from a growing global experience with emphasis on cardiovascular surveillance The COVID-19 , due to SARS-CoV-2 , has uncovered many real-world issues when it comes to healthcare management and has led to a widespread mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Background : Severe Acute Respiratory Syndrome Coronavirus 2 ( SARS-CoV-2 ) and its resultant clinical presentation , COVID-19 , is an emergent cause of mortality worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV-2 is progressively inducing a high mortality rate across the globe due to the lack of selective therapeutic interventions or vaccination .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Background :  Severe Acute Respiratory Syndrome Coronavirus 2 ( SARS-CoV-2 ) and its resultant clinical presentation , COVID-19 , is an emergent cause of mortality worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV-2 is progressively inducing a high mortality rate across the globe due to the lack of selective therapeutic interventions or vaccination .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has spread widely , causing coronavirus disease 2019 ( COVID-19 ) and significant mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has spread widely , causing coronavirus disease 2019 ( COVID-19 ) and significant mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "1 A novel betacoronavirus , that is currently named severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , was identified as the pathogen through high-throughput sequencing of respiratory tract samples of patients.2 ,3 Several evidences pointed out that SARS-CoV-2 has caused person-to-person transmission all over the world.4-8 The coronavirus disease 2019 ( COVID-19 ) has already spread rapidly to most parts of China as well as around the world , which has been declared as a pandemic by the World Health Organization ( WHO ) .9 COVID-19 is more likely to develop to a critically ill level and has caused higher mortality especially in patients with metabolic disorders ( diabetes , obesity , hypertension , etc ) .10 -12 The clinical spectrum of COVID-19 ranges from mild to critically ill cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "1 A novel betacoronavirus , that is currently named severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , was identified as the pathogen through high-throughput sequencing of respiratory tract samples of patients.2 ,3 Several evidences pointed out that SARS-CoV-2 has caused person-to-person transmission all over the world.4-8 The coronavirus disease 2019 ( COVID-19 ) has already spread rapidly to most parts of China as well as around the world , which has been declared as a pandemic by the World Health Organization ( WHO ) .9 COVID-19 is more likely to develop to a critically ill level and has caused higher mortality especially in patients with metabolic disorders ( diabetes , obesity , hypertension , etc ) .10 -12 The clinical spectrum of COVID-19 ranges from mild to critically ill cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Aptamer BC 007 - Efficient binder of spreading-crucial SARS-CoV-2 proteins Corona virus disease 2019 ( COVID-19 ) is a respiratory disease caused by a new coronavirus ( SARS-CoV-2 ) which causes significant morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the virus responsible for the coronavirus disease ( COVID-19 ) pandemic , has contributed to significant morbidity and mortality , and presents with a myriad of clinical manifestations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , the virus responsible for the coronavirus disease ( COVID-19 ) pandemic , has contributed to significant morbidity and mortality , and presents with a myriad of clinical manifestations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The newly emerged severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has infected millions of people and caused tremendous morbidity and mortality worldwide .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Corona virus disease 2019 ( COVID-19 ) is a respiratory disease caused by a new coronavirus ( SARS-CoV-2 ) which causes significant morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Since then , SARS-CoV-2 has spread around the world , causing widespread mortality , and with major impacts on societies and economies .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Since then , SARS-CoV-2 has spread around the world , causing widespread mortality , and with major impacts on societies and economies .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , a betacoronavirus that causes the novel coronavirus disease 2019 ( COVID-19 ) , is highly transmissible and pathogenic for humans and may cause life-threatening disease and mortality , especially in individuals with underlying comorbidities .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , a betacoronavirus that causes the novel coronavirus disease 2019 ( COVID-19 ) , is highly transmissible and pathogenic for humans and may cause life-threatening disease and mortality , especially in individuals with underlying comorbidities .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV-2 has spread globally to cause significant morbidity and mortality , and there still remain unknowns about the biology and pathology of the virus .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Deficiencies in SPMs , driven by obesity , its co-morbidities , and chronic pulmonary environmental exposures , could exacerbate the SARS-CoV-2 induced morbidities and mortalities .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Since SARS-CoV-2 causes morbidity and mortality which escalate throughout the world , there might be temptation to progress with the clinical use of novel and unproven therapeutics .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Since SARS-CoV-2 causes morbidity and mortality which escalate throughout the world , there might be temptation to progress with the clinical use of novel and unproven therapeutics .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Projections are even suggesting that the number of deaths due to SARS-CoV-2 in Italy will continue to increase in the near future , leaving us the sad world record of casualties .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The continuous increase in morbidities and mortalities due to SARS-CoV-2 leads to international medical health worsen situations day after day .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "The continuous increase in morbidities and mortalities due to SARS-CoV-2 leads to international medical health worsen situations day after day .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Mortality due to SARS-CoV-2 is increased in older adults , but young adults and children have served to spread infection , and future coronaviruses may have higher mortality in younger populations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "Mortality due to SARS-CoV-2 is increased in older adults , but young adults and children have served to spread infection , and future coronaviruses may have higher mortality in younger populations .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009026 ! Mortality)\n",
      "PURPOSE OF REVIEW Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , the novel coronavirus that causes coronavirus disease 2019 ( COVID-19 ) , has caused substantial morbidity and mortality .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009026 ! Mortality)\n",
      "This commendable - and hopefully permanent - advance in medicine and epidemiology has facilitated more rapid and cross-disciplinary science to rapidly improve understanding around how to mitigate morbidity and mortality due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009077 ! Mucins)\n",
      "In COVID-19 assessed by scRNA-seq, SARS-CoV-2 infection induced IL-1beta and TNF-alpha production may promote mucin secretion from club cells, likely contributing to acute respiratory distress syndrome (ARDS) [50].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009077 ! Mucins)\n",
      "These results indicated that SARS-CoV-2 infection inhibits the turnover of IECs and mucin, and induces apoptosis to disrupt the intestinal barrier, causing further infection in multiple tissues.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009093 ! Mucus)\n",
      "Recent autopsy and lung replacement pathology reports showed that SARS-CoV-2 mainly causes inflammatory reactions characteristically deep in the airway and at the alveolar level , producing extensive inflammatory exudation and mucus that fills the alveolar cavity [ 10 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009093 ! Mucus)\n",
      "Recent autopsy and lung replacement pathology reports showed that SARS-CoV-2 mainly causes inflammatory reactions characteristically deep in the airway and at the alveolar level , producing extensive inflammatory exudation and mucus that fills the alveolar cavity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009393 ! Nephritis)\n",
      "This case highlights that SARS-CoV-2 could be the trigger for IgA nephritis , which calls for a need to determine a specific management for such cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009393 ! Nephritis)\n",
      "This case highlights that SARS-CoV-2 could be the trigger for IgA nephritis , which calls for a need to determine a specific management for such cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009421 ! \"Nervous System Malformations\")\n",
      "It remains to be established whether neurological abnormalities are caused by SARS-CoV-2 itself , by the exaggerated cytokine response it triggers , and / or by the increased formation of blood clots in brain blood vessels .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009421 ! \"Nervous System Malformations\")\n",
      "It remains to be established whether neurological abnormalities are caused by SARS-CoV-2 itself , by the exaggerated cytokine response it triggers , and / or by the increased formation of blood clots in brain blood vessels .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009457 ! Neuroglia)\n",
      "As mentioned earlier , glial cells express ACE2 , We speculate that SARS-CoV-2 , which enters the central nervous system through olfactory nerve , blood-derived , and other pathways , may also activate glial cells and induce a pro-inflammatory state ( 37 , 63 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009457 ! Neuroglia)\n",
      "As mentioned earlier , glial cells express ACE2 , We speculate that SARS-CoV-2 , which enters the central nervous system through olfactory nerve , blood-derived , and other pathways , may also activate glial cells and induce a pro-inflammatory state ( 37 , 63 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009504 ! Neutrophils)\n",
      "Moreover , SARS-CoV-2 stimulates neutrophils from healthy donors into forming NETs , which can cause apoptosis in respiratory epithelial cells in vitro ( Oubihi and Wang , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009504 ! Neutrophils)\n",
      "Moreover , SARS-CoV-2 stimulates neutrophils from healthy donors into forming NETs , which can cause apoptosis in respiratory epithelial cells in vitro ( Oubihi and Wang , 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009504 ! Neutrophils)\n",
      "How neutrophils are activated and the role of NEU in this process 326 remains to be defined in COVID-19 , nevertheless , recent evidence showed that 327 neutrophils could be directly activated by SARS-CoV-2 2 , cytokines 65 , and alarmins 39,42 328 such as calprotectin 39 , a TLR4 ligand 66 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009504 ! Neutrophils)\n",
      "While no study has assessed the precise role of NETs in COVID-19 pathogenesis , certain researchers speculate that SARS-CoV-2 might stimulate neutrophils to produce NETs , similar to several other viral pathogens 56 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D009504 ! Neutrophils)\n",
      "While no study has assessed the precise role of NETs in COVID-19 pathogenesis , certain researchers speculate that SARS-CoV-2 might stimulate neutrophils to produce NETs , similar to several other viral pathogens 56 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010033 ! \"Otitis Media\")\n",
      "CONCLUSION : SARS-CoV-2 is a probable cause of middle ear infections and sensorineural hearing loss , secondary to spread of the novel virus into the middle ear and related neural structures .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010033 ! \"Otitis Media\")\n",
      "Conclusion : SARS-CoV-2 is a probable cause of middle ear infections and sensorineural hearing loss , secondary to spread of the novel virus into the middle ear and related neural structures .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010200 ! Panic)\n",
      "SARS-CoV-2 and the COVID-19 Pandemic Currently , the novel coronavirus , SARS-CoV-2 alternatively termed 2019 nCoV , has caused panic with WHO declaring it as a global health emergency on 30 January 2020 [ 18 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010200 ! Panic)\n",
      "SARS-CoV-2 and the COVID-19 Pandemic Currently , the novel coronavirus , SARS-CoV-2 alternatively termed 2019 nCoV , has caused panic with WHO declaring it as a global health emergency on 30 January 2020 [ 18 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010300 ! \"Parkinson Disease\")\n",
      "Currently , although there is no direct evidence that SARS-CoV-2 causes or accelerates PD [ 82 ] , it should be noted that the wide expression of ACE2 at different areas in the CNS provides a molecular basis for SARS-CoV-2 to mediate or accelerate the occurrence of PD .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010300 ! \"Parkinson Disease\")\n",
      "Currently , although there is no direct evidence that SARS-CoV-2 causes or accelerates PD [ 82 ] , it should be noted that the wide expression of ACE2 at different areas in the CNS provides a molecular basis for SARS-CoV-2 to mediate or accelerate the occurrence of PD .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010300 ! \"Parkinson Disease\")\n",
      "Although, it is early to identify if SARS-CoV-2 led infection enhances PD complications, it is likely that oxidative stress may exacerbate PD progression in COVID-19 affected individuals and/or vice versa.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010300 ! \"Parkinson Disease\")\n",
      "Currently, although there is no direct evidence that SARS-CoV-2 causes or accelerates PD [82], it should be noted that the wide expression of ACE2 at different areas in the CNS provides a molecular basis for SARS-CoV-2 to mediate or accelerate the occurrence of PD.There is currently limited evidence from clinical studies to support an association between the onset of PD as a late complication of SARS-CoV-2 infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010493 ! Pericarditis)\n",
      "SARS-CoV-2 , like other viral infections , can cause pericarditis and resulting pericardial effusion ; however , it is usually entailing an acute course .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010493 ! Pericarditis)\n",
      "SARS-CoV-2 , like other viral infections , can cause pericarditis and resulting pericardial effusion ; however , it is usually entailing an acute course .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D010493 ! Pericarditis)\n",
      "SARS-CoV-2 , like other viral infections , can cause pericarditis and resulting pericardial effusion ; however , it is usually entailing an acute course .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011618 ! \"Psychotic Disorders\")\n",
      "Although heightened stress of a COVID-19 diagnosis or medications ( such as corticosteroids ) may unmask an underlying primary psychotic disorder in a vulnerable individual , the idea that SARS-CoV-2 may itself trigger psychosis through direct neurotoxicity or a heightened immune response is not unreasonable.13 Coronaviruses are neurotropic , for example , and SARS-CoV-2 RNA has recently been isolated from the central nervous system of a patient.4 20 Moreover , the antipsychotics haloperidol and chlorpromazine have displayed antiviral activity against SARS-CoV-2 in vitro and in a mice model , respectively.21 22 In a review of the literature , Troyer et al23 offer direct infection , blood circulation , neuronal involvement , hypoxic injury , immune injury and ACE2 binding as possible culprits of coronavirus nervous system damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011618 ! \"Psychotic Disorders\")\n",
      "Although heightened stress of a COVID-19 diagnosis or medications ( such as corticosteroids ) may unmask an underlying primary psychotic disorder in a vulnerable individual , the idea that SARS-CoV-2 may itself trigger psychosis through direct neurotoxicity or a heightened immune response is not unreasonable .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 can induce pulmonary fibrosis by promoting the upregulation of pro-fibrotic signaling molecules , including transforming growth factor-beta ( TGF-beta ) [ 24,25 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 can induce pulmonary fibrosis by promoting the upregulation of pro-fibrotic signaling molecules , including transforming growth factor-beta ( TGF-beta ) [ 24,25 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 induces pathogenic inflammation , pulmonary fibrosis , acute respiratory distress syndrome ( ARDS ) , and nephropathy which can lead to the death of some infected patients within 2-3 weeks .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 induces pathogenic inflammation , pulmonary fibrosis , acute respiratory distress syndrome ( ARDS ) , and nephropathy which can lead to the death of some infected patients within 2-3 weeks ( Zhou F. et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 , like other emerging respiratory viruses , may induce pulmonary fibrosis in survivors of SARS-CoV-2 infection by activating the EGFR pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 , like other emerging respiratory viruses , may induce pulmonary fibrosis in survivors of SARS-CoV-2 infection by activating the EGFR pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D011658 ! \"Pulmonary Fibrosis\")\n",
      "SARS-CoV-2 , like other emerging respiratory viruses , may induce pulmonary fibrosis in survivors of SARS-CoV-2 infection by activating the EGFR pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012044 ! \"Regression Analysis\")\n",
      "COX regression analysis after adjustment DISCUSSION COVID-19 is caused by SARS-CoV-2 , which currently has no vaccine nor specific antiviral drug .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "SARS-CoV-2 uses the angiotensin-converting enzyme ( ACE ) -2 receptor to enter the host cells and activates the renin-angiotensin system , which can induce a prothrombotic state [ 14 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "COVID-19 virus may cause new-onset diabetes or unmask previously undiagnosed diabetes by injuring pancreatic beta-cells , interfering with the insulin signalling pathways , or activating the renin-angiotensin system .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "SARS-CoV-2 causes ACE and ACE2 balance disruption and RAAS activation, which leads ultimately to COVID-19 progression, especially in patients with comorbidities, such as hypertension, diabetes mellitus, and cardiovascular disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012106 ! Research)\n",
      "In this review , we have discussed and summarized the available multi-factorial information and recent updates on the SARS-CoV-2 which can help support future research and may help in the strategic management of the current COVID-19 pandemic .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The second report, was a single case study, in which a 65 woman in China with worsening conditions due to SARS-CoV-2 induced ARDS, was treated with three doses of 5 x 10 7 UC-MSCs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "By binding TLR4 , SARS-CoV-2 triggers rampant inflammation and ALI , and as it enters the blood circulation , it infects the heart via ACE2 and / or TLR4 ( causing myocarditis ) , as well other organs such as the skin , kidneys , and GI tract where TLR4 is expressed .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "However , unlike MERS-CoV , SARS-CoV-2 is highly infectious , causes ARDS , and has been associated with high rates of lethality ( Vankadari and Wilce , 2020 ; Wu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In the inflammatory course of ARDS due to SARS-CoV-2 , lymphocytes infiltrated into lung tissues , while the host immune system was excessively activated and aggravated tissue injury [ 36 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In the inflammatory course of ARDS due to SARS-CoV-2 , lymphocytes infiltrated into lung tissues , while the host immune system was excessively activated and aggravated tissue injury [ 36 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In the inflammatory course of ARDS due to SARS-CoV-2 , lymphocytes infiltrated into lung tissues , while the host immune system was excessively activated and aggravated tissue injury [ 36 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Whereas ventilator-induced lung injury ( VILI ) has been well studied , there is still no other option for COVID-19 patients with severe respiratory issues.211 Clinical reports show approximately 5-15 % of patients infected with SARS-CoV-2 require a ventilator and intensive care observation.160 There are a variety of in vivo and ex vivo models that have been used to study VILI , and their adaptation to COVID-19 studies can help provide information as to how these severe cases and pathologies are made worse by ventilation.42 ,249 In addition to VILI , SARS-CoV-2 has been shown to lead to edema , fibrosis , and ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Disease heterogeneity was also minimized as these patients all had SARS-COV-2 induced severe ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Disease heterogeneity was also minimized as these patients all had SARS-COV-2 induced severe ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Disease heterogeneity was also minimized as these patients all had SARS-COV-2 induced severe ARDS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Conversely , SARS-CoV-2 can cause severe infections and result in septic shock , acute respiratory distress syndrome , acute cardiac injury , acute kidney injury , and multi-organ failure , which necessitate intensive care unit admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Our results are also in accordance with the preliminary data of Fu et al17 collected from 21 patients who received one or two doses (400 mg/dose) of TCZ for ARDS induced by SARS-CoV2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Conversely , SARS-CoV-2 can cause severe infections and result in septic shock , acute respiratory distress syndrome , acute cardiac injury , acute kidney injury , and multi-organ failure , which necessitate intensive care unit admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "[2, 3] While the exact mechanisms through which SARS-CoV-2 causes ARDS and how certain host factors confer an increased risk of developing severe disease remain unclear, one factor has emerged as a dominant predictor of disease severity and risk of mortality : age.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "While the exact mechanisms through which SARS-CoV-2 causes ARDS and how certain host factors confer an increased risk of developing severe disease remain unclear, one factor has emerged as a dominant predictor of disease severity and risk of mortality : age [4, 5].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In the course of COVID-19 , in addition to affecting the lungs , causing interstitial pneumonia and severe acute respiratory distress syndrome ( ARDS ) , SARS-CoV-2 also damages multiple organs due to a massive increase in inflammatory factors , especially in the cardiovascular system , leading to a variety of cardiac problems such as arrhythmia and myocarditis ( Guzik et al ., 2020 ; Inciardi et al ., 2020 ; Zheng et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 causes acute respiratory distress syndrome , followed by multiple organ failure and death in a small percentage of individuals .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In the course of COVID-19 , in addition to affecting the lungs , causing interstitial pneumonia and severe acute respiratory distress syndrome ( ARDS ) , SARS-CoV-2 also damages multiple organs due to a massive increase in inflammatory factors , especially in the cardiovascular system , leading to a variety of cardiac problems such as arrhythmia and myocarditis ( Guzik et al ., 2020 ; Inciardi et al ., 2020 ; Zheng et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In addition to mild respiratory flu-like symptoms , SARS-CoV-2 may cause severe acute respiratory distress syndrome ( ARDS ) , sepsis , or multiorgan dysfunction [ 1 , 2 ] , as seen in the other two previous beta-coronavirus outbreaks [ severe acute respiratory syndrome ( SARS ) and Middle East respiratory syndrome ( MERS ) ] [ 3 , 4 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In addition to mild respiratory flu-like symptoms , SARS-CoV-2 may cause severe acute respiratory distress syndrome ( ARDS ) , sepsis , or multiorgan dysfunction [ 1 , 2 ] , as seen in the other two previous beta-coronavirus outbreaks [ severe acute respiratory syndrome ( SARS ) and Middle East respiratory syndrome ( MERS ) ] [ 3 , 4 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "All the 150 patients referred for acute respiratory distress syndrome due to SARS-CoV-2 between March 3 and May 5 , 2020 , were included at their admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "All the 150 patients referred for acute respiratory distress syndrome due to SARS-CoV-2 between March 3 and May 5 , 2020 , were included at their admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "All the 150 patients referred for acute respiratory distress syndrome due to SARS-CoV-2 between March 3 and May 5 , 2020 , were included at their admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Between March 3 and May 5 , 2020 , all the patients referred for ARDS [ 19 ] due to SARS-CoV-2 were prospectively included at admission in two ICUs from a French tertiary hospital in Strasbourg .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Between March 3 and May 5 , 2020 , all the patients referred for ARDS [ 19 ] due to SARS-CoV-2 were prospectively included at admission in two ICUs from a French tertiary hospital in Strasbourg .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 induces pathogenic inflammation , pulmonary fibrosis , acute respiratory distress syndrome ( ARDS ) , and nephropathy which can lead to the death of some infected patients within 2-3 weeks .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 induces pathogenic inflammation , pulmonary fibrosis , acute respiratory distress syndrome ( ARDS ) , and nephropathy which can lead to the death of some infected patients within 2-3 weeks ( Zhou F. et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The presence of alveolar wall destruction with detachment of the surface epithelial lining with various degree of type II pneumocyte hyperplasia confirms that impaired gas exchange by loss of alveolar-capillary membrane can still be considered an important pathophysiological mechanism , in which SARS-CoV-2 produces acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The presence of alveolar wall destruction with detachment of the surface epithelial lining with various degree of type II pneumocyte hyperplasia confirms that impaired gas exchange by loss of alveolar-capillary membrane can still be considered an important pathophysiological mechanism , in which SARS-CoV-2 produces acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 causes acute respiratory distress syndrome , followed by multiple organ failure and death in a small percentage of individuals .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "One of the main mechanisms for ARDS induced by SARS-CoV-2 is the cytokine storm, the deadly uncontrolled systemic inflammatory response resulting from the release of large amounts of pro inflammatory cytokines (100).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Published findings from the AAD COVID-19 Registry noted that all patients with retiform purpura were hospitalized and 82 % had ARDS due to SARS-CoV-2 [ 13 * * ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Published findings from the AAD COVID-19 Registry noted that all patients with retiform purpura were hospitalized and 82 % had ARDS due to SARS-CoV-2 [ 13 * * ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Published findings from the AAD COVID-19 Registry noted that all patients with retiform purpura were hospitalized and 82 % had ARDS due to SARS-CoV-2 [ 13 * * ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "While SARS-CoV-2 is known to cause interstitial pneumonia and acute respiratory distress syndrome ( ARDS ) , there is growing evidence of numerous neurological manifestations , including encephalopathy [ 2 ] , limbic and brainstem encephalitis [ 3,4 ] , Guillain-Barr e syndrome [ 5,6 ] , and stroke ( predominantly ischemic , but also hemorrhagic ) [ [ 7 ] , [ 8] , [ 9 ] , [ 10 ] , [ 11 ] ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The figure was created with Biorender.com TP53 and VAMP8 respectively are the key proteins of Cytokines storm and neutrophil degranulation process associated with acute respiratory distress syndrome ( ARDS ) SARS-CoV-2 causes acute lung injury and acute respiratory distress syndrome ( ARDS ) , which can result in severe lethal conditions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "While SARS-CoV-2 is known to cause interstitial pneumonia and acute respiratory distress syndrome ( ARDS ) , there is growing evidence of numerous neurological manifestations , including encephalopathy [ 2 ] , limbic and brainstem encephalitis [ 3,4 ] , Guillain-Barr e syndrome [ 5,6 ] , and stroke ( predominantly ischemic , but also hemorrhagic ) [ [ 7 ] , [ 8] , [ 9 ] , [ 10 ] , [ 11 ] ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Here we report that endothelial KLF2 is significantly reduced in human lung autopsies from COVID-19 patients , which supports that ARDS due to SARS-CoV-2 is a vascular phenotype possibly attributed to KLF2 down-regulation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Here we report that endothelial KLF2 is significantly reduced in human lung autopsies from COVID-19 patients , which supports that ARDS due to SARS-CoV-2 is a vascular phenotype possibly attributed to KLF2 down-regulation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "2 3 4 The novel coronavirus Disease 2019 ( COVID-19 ) is a viral disease induced by severe acute respiratory syndrome-coronavirus-2 ( SARS-CoV-2 ) that may cause an acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "2 3 4 The novel coronavirus Disease 2019 ( COVID-19 ) is a viral disease induced by severe acute respiratory syndrome-coronavirus-2 ( SARS-CoV-2 ) that may cause an acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Autopsy confirmed a severe form of SARS-CoV-2 induced ARDS ( Fig. 5A ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 is believed to cause cytokine storm , which triggers an exaggerated inflammatory response in the body , and results in acute respiratory distress syndrome , shock , and multi-organ failure [ 9,10 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Answering this question has potential to decipher whether ARDS induced by SARS-CoV-2 is mechanistically different from other ARDS etiologies.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "A case of acute respiratory distress syndrome due to SARS-CoV-2 and Influenza A co-infection and a mini-review of the literature is reported .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Acute respiratory distress syndrome due to SARS-CoV-2 and Influenza A co-infection in an Italian patient : mini-review of the literature .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 is believed to cause cytokine storm , which triggers an exaggerated inflammatory response in the body , and results in acute respiratory distress syndrome , shock , and multi-organ failure [ 9 , 10 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 ( CoV-2 ) can cause Acute Respiratory Distress Syndrome ( ARDS ) , and seems to be more contagious than SARS-CoV-1 ( 2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) , which causes coronavirus disease 2019 ( COVID-19 ) , may result in acute respiratory distress syndrome ( ARDS ) , multi-organ failure and death .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "However , unlike MERS-CoV , SARS-CoV-2 is highly infectious , causes ARDS , and has been associated with high rates of lethality ( Vankadari and Wilce , 2020 ; Wu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "However , unlike MERS-CoV , SARS-CoV-2 is highly infectious , causes ARDS , and has been associated with high rates of lethality ( Vankadari and Wilce , 2020 ; Wu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Presently , there is no solid evidence about the benefits and risks of using HFNC for ARDS due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Presently , there is no solid evidence about the benefits and risks of using HFNC for ARDS due to SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Our data confirm that SARS-CoV-2 induced ARDS is characterized by an atypically high respiratory system compliance (C RS), compared to the degree of arterial hypoxemia [XREF_BIBR, XREF_BIBR, XREF_BIBR, XREF_BIBR] Since the hemodynamic effects of PEEP are enhanced when the C RS is relatively high [XREF_BIBR - XREF_BIBR], we speculate that the effect of PEEP on kidney perfusion could have been more pronounced in patients with SARS-CoV-2 ARDS than in those with classical ARDS.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Our data confirm that SARS-CoV-2 induced ARDS is characterized by an atypically high respiratory system compliance (C RS), compared to the degree of arterial hypoxemia [9, 10, 21, 22] Since the hemodynamic effects of PEEP are enhanced when the C RS is relatively high [22] [23] [24], we speculate that the effect of PEEP on kidney perfusion could have been more pronounced in patients with SARS-CoV-2 ARDS than in those with classical ARDS.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV-2 induced acute respiratory distress syndrome and systemic inflammatory response syndrome lead to hypoxemia and shock , which can cause ischemia-reperfusion injury [ 49 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "It is caused by a newly identified viral pathogen , the severe acute respiratory syndrome coronavirus ( SARS-CoV-2 ) , which can cause severe pneumonia and acute respiratory distress syndrome .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012166 ! \"Retinal Hemorrhage\")\n",
      "We hypothesize that SARS-CoV-2 may have played a role in the onset of an acute , transient cold-agglutinin hemolytic anemia , on a backdrop of chronic mild anemia , thus eventually eliciting the retinal bleeding .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012640 ! Seizures)\n",
      "We hypothesise that SARS-CoV-2 could trigger seizures through a neurotropic pathogenic mechanism and subsequently emphasise the importance of considering possible neurological manifestations , including seizure disorder of SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012640 ! Seizures)\n",
      "We hypothesise that SARS-CoV-2 could trigger seizures through a neurotropic pathogenic mechanism and subsequently emphasise the importance of considering possible neurological manifestations , including seizure disorder of SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012640 ! Seizures)\n",
      "The 78-yearold patient was seizure-free for more than two years , and based on a historical correlation of symptoms , it was suggested that SARS-CoV-2 might have triggered the seizures [ 205 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012772 ! \"Shock, Septic\")\n",
      "Conversely , SARS-CoV-2 can cause severe infections and result in septic shock , acute respiratory distress syndrome , acute cardiac injury , acute kidney injury , and multi-organ failure , which necessitate intensive care unit admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012772 ! \"Shock, Septic\")\n",
      "Conversely , SARS-CoV-2 can cause severe infections and result in septic shock , acute respiratory distress syndrome , acute cardiac injury , acute kidney injury , and multi-organ failure , which necessitate intensive care unit admission .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012772 ! \"Shock, Septic\")\n",
      "The implications extend outside the lungs ; SARS-CoV-2 can lead to septic shock , acute kidney injury , and disseminated intravascular coagulation ( DIC ) in some of the most severe cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D012772 ! \"Shock, Septic\")\n",
      "The implications extend outside the lungs ; SARS-CoV-2 can lead to septic shock , acute kidney injury , and disseminated intravascular coagulation ( DIC ) in some of the most severe cases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "Human coronaviruses ( HCoV ) belonging to the genera Alphacoronavirus and Betacoronavirus include HCoV-229E , HCoV-OC43 , HCoV-NL63 , and HCoV-HKU1 , which induce mild pathology in most cases , and SARS-CoV [ 21 ] , MERS-CoV [ 22 ] , and the causative agent of the new pandemic COVID-19 , SARS-CoV-2 [ 23 ] , which can induce severe syndromes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "Human coronaviruses ( HCoV ) belonging to the genera Alphacoronavirus and Betacoronavirus include HCoV-229E , HCoV-OC43 , HCoV-NL63 , and HCoV-HKU1 , which induce mild pathology in most cases , and SARS-CoV [ 21 ] , MERS-CoV [ 22 ] , and the causative agent of the new pandemic COVID-19 , SARS-CoV-2 [ 23 ] , which can induce severe syndromes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "Human coronaviruses ( HCoV ) belonging to the genera Alphacoronavirus and Betacoronavirus include HCoV-229E , HCoV-OC43 , HCoV-NL63 , and HCoV-HKU1 , which induce mild pathology in most cases , and SARS-CoV [ 21 ] , MERS-CoV [ 22 ] , and the causative agent of the new pandemic COVID-19 , SARS-CoV-2 [ 23 ] , which can induce severe syndromes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "National and international registries are being created to collect data on rheumatic patients with CoViD-19.Coronavirus Disease-2019 (CoViD-19) has emerged over the past months as a clinical syndrome caused by a novel beta-Coronavirus, named Severe Acute Respiratory Syndrome (SARS)-CoV2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "We present the first reported case of SARS-COV2 induced syndrome of inappropriate diuretic hormone (SIADH) manifesting as new-onset seizures.A previously healthy 75-year-old man presents with a witnessed episode of new-onset seizures.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "In severe phenotype, resembling a \"cytokine-release syndrome\", SARS-CoV2 causes the lysis of the immune-mediators leading to a cytokine storm able to induce lung epithelium damage and acute respiratory distress syndrome.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013577 ! Syndrome)\n",
      "3 The hypothesis of neurotropism with subsequent neuronal injury, either directly or indirectly (through immune mechanisms), is supported by previous findings from other infections with severe acute respiratory syndrome CoV and Middle East respiratory syndrome CoV.3Pre-existing cardiovascular or pulmonary disease and old age increase the chances of contracting SARS-CoV-2 infection and those risk factors are often present in patients with neurological disorders.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "Currently , the mechanisms by which SARS-CoV-2 causes thrombocytopenia are speculated to involve the following : ( 1 ) an impaired haematopoietic microenvironment caused by systemic inflammation or cytokine storm , for example , elevated IL-6 , which is a common phenomenon in SARS-CoV-2 infection [ 4 ] , could suppress haematopoiesis [ 7 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "Currently , the mechanisms by which SARS-CoV-2 causes thrombocytopenia are speculated to involve the following : ( 1 ) an impaired haematopoietic microenvironment caused by systemic inflammation or cytokine storm , for example , elevated IL-6 , which is a common phenomenon in SARS-CoV-2 infection [ 4 ] , could suppress haematopoiesis [ 7 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "SARS-CoV-2 induce thrombocytopenia and leukopenia , higher D-dimer levels .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "SARS-CoV-2 induce thrombocytopenia and leukopenia , higher D-dimer levels .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "Excluding other potentialities inducing thrombocytopenia , we highly hypothesized the SARS-CoV-2 may cause thrombocytopenia by disturbing the immune system to induce the thrombocytopenia in our report ,, which needs longer time to restore the immune system and platelet count via the glucocorticoid .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "SARS-CoV-2 may cause thrombocytopenia by several mechanisms including bone marrow suppression , consumptive coagulopathy , immune-mediated platelet destruction , or cytokine release syndrome [ 82 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "SARS-CoV-2 induced thrombocytopenia as an important biomarker significantly correlated with abnormal coagulation function , increased intravascular blood clot risk and mortality in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "SARS-CoV-2 induced thrombocytopenia as an important biomarker significantly correlated with abnormal coagulation function , increased intravascular blood clot risk and mortality in COVID-19 patients Background Coronavirus disease 2019 ( COVID-19 ) is a novel infectious viral disease , which lacks well-established diagnostic laboratory parameters that could be used to evaluate disease severity , thromboembolism or cardiovascular events and to predict clinical prognosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D014657 ! Vasculitis)\n",
      "In addition to the endothelial damage , SARS-CoV-2 also can cause vasculitis and presents as a systemic inflammatory vascular disease .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D014657 ! Vasculitis)\n",
      "In addition to the endothelial damage , SARS-CoV-2 also can cause vasculitis and presents as a systemic inflammatory vascular disease .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D014657 ! Vasculitis)\n",
      "In addition to the endothelial damage , SARS-CoV-2 also can cause vasculitis and presents as a systemic inflammatory vascular disease .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015212 ! \"Inflammatory Bowel Diseases\")\n",
      "Our patients presented with a clinical picture characterized by hyperinflammation and gastrointestinal symptoms ; thus , we hypothesized that SARS-CoV-2 could have triggered an inflammatory bowel disease ; consequently , we performed colonoscopy with ileoscopy .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015212 ! \"Inflammatory Bowel Diseases\")\n",
      "Our patients presented with a clinical picture characterized by hyperinflammation and gastrointestinal symptoms ; thus , we hypothesized that SARS-CoV-2 could have triggered an inflammatory bowel disease ; consequently , we performed colonoscopy with ileoscopy .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015726 ! \"Giant Cells\")\n",
      "SARS-CoV-2 infects goblet cells and induces syncytium formation and cell sloughing 450 4 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015726 ! \"Giant Cells\")\n",
      "SARS-CoV-2 induces syncytia and cell sloughing .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015845 ! \"Tonic Pupil\")\n",
      "As tonic pupils are known to be caused by neurotropic viruses and our current understanding of the SARS-CoV-2 is that it does affect the nervous system , we feel that the tonic pupil in our patient may be secondary to COVID 19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015845 ! \"Tonic Pupil\")\n",
      "As tonic pupils are known to be caused by neurotropic viruses and our current understanding of the SARS-CoV-2 is that it does affect the nervous system , we feel that the tonic pupil in our patient may be secondary to COVID 19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015897 ! Comorbidity)\n",
      "Angiotensinconverting enzyme inhibitors ( ACEIs ) / Angiotensin receptor blockers ( ARBs ) regulate the metabolic pathway Factors Associated with Pathogenesis-age , Gender and Co-morbidities Studies have shown that SARS-CoV-2 causes worse outcomes and a higher mortality rate in older people ( more than 60 years of age ) and those with comorbidities such as hypertension , cardiovascular disease , diabetes mellitus , chronic respiratory disease , and chronic kidney disease [ 43 , 44 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015897 ! Comorbidity)\n",
      "Factors Associated with Pathogenesis - age , Gender and Co-morbidities Studies have shown that SARS-CoV-2 causes worse outcomes and a higher mortality rate in older people ( more than 60 years of age ) and those with comorbidities such as hypertension , cardiovascular disease , diabetes mellitus , chronic respiratory disease , and chronic kidney disease [ 43 , 44 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D015897 ! Comorbidity)\n",
      "Studies have shown that SARS-CoV-2 causes worse outcomes and a higher mortality rate in older people ( more than 60 years of age ) and those with comorbidities such as hypertension , cardiovascular disease , diabetes mellitus , chronic respiratory disease , and chronic kidney disease [ 43 , 44 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016676 ! \"Macrophages, Alveolar\")\n",
      "Thus , the persistent presence of SARS-CoV-2 activates alveolar macrophages or epithelial cells to produce various proinflammatory cytokines and chemokines , triggering or recruiting more innate immune cells and thereby amplifying inflammation ( 20 , 29 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016883 ! \"Diabetic Ketoacidosis\")\n",
      "Diagram revealing the mechanism through which SARS-CoV-2 induce diabetic ketoacidosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016883 ! \"Diabetic Ketoacidosis\")\n",
      "The severe acute respiratory distress coronavirus 2 ( SARS-CoV-2 ) has been reported to induce ketoacidosis and diabetic ketoacidosis ( DKA ) even in nondiabetic patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016883 ! \"Diabetic Ketoacidosis\")\n",
      "The severe acute respiratory distress coronavirus 2 ( SARS-CoV-2 ) has been reported to induce ketoacidosis and diabetic ketoacidosis ( DKA ) even in nondiabetic patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016883 ! \"Diabetic Ketoacidosis\")\n",
      "Fig. 2 : Diagram revealing the mechanism through which SARS-CoV-2 induce diabetic ketoacidosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D016883 ! \"Diabetic Ketoacidosis\")\n",
      "SARS-CoV-2 can trigger severe diabetic ketoacidosis at presentation in individuals with new-onset diabetes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D017152 ! \"Antibodies, Antiphospholipid\")\n",
      "Additionally , it is possible that SARS-CoV-2 , as it happens with other viruses , might lead to the formation of antiphospholipid antibodies , potentially contributing to the increased rates of thrombosis seen in COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D017152 ! \"Antibodies, Antiphospholipid\")\n",
      "Additionally , it is possible that SARS-CoV-2 , as it happens with other viruses , might lead to the formation of antiphospholipid antibodies , potentially contributing to the increased rates of thrombosis seen in COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "It has also been indicated that SARS-CoV-2 induced oxidative stress can aggravate DNA methylation defect likely leading to further ACE2 demethylation , thus enhancing the presence of the virus in the blood ( 194 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "Schematic mechanism of lung injury in COVID-19 : The SARS-CoV-2 sensitizes the lung epithelial cells and macrophages , which in response generates inflammatory signals , producing oxidative stress and releases a massive amount of cytokines including TGF-beta .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "Schematic mechanism of lung injury in COVID-19 : The SARS-CoV-2 sensitizes the lung epithelial cells and macrophages , which in response generates inflammatory signals , producing oxidative stress and releases a massive amount of cytokines including TGF-beta .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "It has also been indicated that SARS-CoV-2 induced oxidative stress can aggravate DNA methylation defect likely leading to further ACE2 demethylation , thus enhancing the presence of the virus in the blood ( Huertas et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "Fig. 2 : Schematic mechanism of lung injury in COVID-19 : The SARS-CoV-2 sensitizes the lung epithelial cells and macrophages , which in response generates inflammatory signals , producing oxidative stress and releases a massive amount of cytokines including TGF-beta .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018418 ! \"Th2 Cells\")\n",
      "On the other hand , SARS-CoV-2 is activating the level of Th2 cells .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018562 ! \"Disease Transmission, Infectious\")\n",
      "SARS-CoV-2 has a high potential to cause infection with a higher transmission rate in comparison to SARS-CoV or MERS-CoV , making it difficult to control and thereby increasing its pandemic potential .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018562 ! \"Disease Transmission, Infectious\")\n",
      "SARS-CoV-2 has a high potential to cause infection with a higher transmission rate in comparison to SARS-CoV or MERS-CoV , making it difficult to control and thereby increasing its pandemic potential .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D018562 ! \"Disease Transmission, Infectious\")\n",
      "SARS-CoV-2 has a high potential to cause infection with a higher transmission rate in comparison to SARS-CoV or MERS-CoV , making it difficult to control and thereby increasing its pandemic potential .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Similar to SARS-CoV-1 and MERS-CoV , SARS-CoV-2 might infect FcgammaRII-bearing immune cells ( such as monocytes , macrophages , B-cells as well as some types of dendritic cells ) and promote ADE .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Similar to SARS-CoV-1 and MERS-CoV , SARS-CoV-2 might infect FcgammaRII-bearing immune cells ( such as monocytes , macrophages , B-cells as well as some types of dendritic cells ) and promote ADE .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Similar to SARS-CoV-1 and MERS-CoV , SARS-CoV-2 might infect FcgammaRII-bearing immune cells ( such as monocytes , macrophages , B-cells as well as some types of dendritic cells ) and promote ADE .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "As the usage of vaccine in human have n't been reported in research paper, whether SARS-CoV-2 will induce ADE in patients still needs further verification, like what Dr Jiang (the one contributes in vaccines and treatments for coronaviruses) said, safety testing matters most during the counterattack to the new coronavirus [28].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "As the usage of vaccine in human have not been reported in research paper , whether SARS-CoV-2 will induce ADE in patients still needs further verification , like what Dr Jiang ( the one contributes in vaccines and treatments for coronaviruses ) said , safety testing matters most during the counterattack to the new coronavirus [ 28 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D019586 ! \"Intracranial Hypertension\")\n",
      "Penetration through the Blood-brain - barrier Once in the blood circulation , SARS-CoV-2 could bind to the ACE2 receptors of the endothelium [ 50 , 52 , 55 ] and disrupt the Blood Brain Barrier ( BBB ) , inducing edema , intracranial hypertension and / or penetration of the virus in the CNS [ 64 , 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020275 ! \"Guillain-Barre Syndrome\")\n",
      "For example , SARS-CoV-2 can cause Guillain-Barr e syndrome ( 80 , 81 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020275 ! \"Guillain-Barre Syndrome\")\n",
      "For example , SARS-CoV-2 can cause Guillain-Barr e syndrome ( 80 , 81 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020275 ! \"Guillain-Barre Syndrome\")\n",
      "SARS-CoV-2 can cause GBS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another pathophysiological mechanism through which SARS-CoV-2 can cause acute cerebrovascular accidents is through endothelial injury mediated by depletion of angiotensin-converting enzyme 2 ( ACE2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another pathophysiological mechanism through which SARS-CoV-2 can cause acute cerebrovascular accidents is through endothelial injury mediated by depletion of angiotensin-converting enzyme 2 ( ACE2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another pathophysiological mechanism through which SARS-CoV-2 can cause acute cerebrovascular accidents is through endothelial injury mediated by depletion of angiotensin-converting enzyme 2 ( ACE2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another factor is hypercoagulability caused by SARS-CoV-2 itself , which may cause stroke without any predisposing factors [ 18 , 19 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another factor is hypercoagulability caused by SARS-CoV-2 itself , which may cause stroke without any predisposing factors [ 18,19 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Another factor is hypercoagulability caused by SARS-CoV-2 itself , which may cause stroke without any predisposing factors [ 18,19 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Other mechanisms by which SARS-CoV-2 could cause stroke include coagulation disorders .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Other mechanisms by which SARS-CoV-2 could cause stroke include coagulation disorders .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Other mechanisms by which SARS-CoV-2 could cause stroke include coagulation disorders .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Given the possibility of SARS-CoV-2 causing vascular injury , this rise may be attributed to early stages of the COVID-19 pandemic , i.e. SARS-CoV-2 was propagating in February in the United States possibly causing increase in stroke numbers .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Given the possibility of SARS-CoV-2 causing vascular injury , this rise may be attributed to early stages of the COVID-19 pandemic , i.e. SARS-CoV-2 was propagating in February in the United States possibly causing increase in stroke numbers .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D020521 ! Stroke)\n",
      "Given the possibility of SARS-CoV-2 causing vascular injury , this rise may be attributed to early stages of the COVID-19 pandemic , i.e. SARS-CoV-2 was propagating in February in the United States possibly causing increase in stroke numbers .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Third, in contrast to the healthy epithelium, SARS-CoV-2 induced higher squamous metaplasia in the infected COPD epithelium and caused a substantial increase in the height of the epithelium (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Interestingly, we found that SARS-CoV-2 infection induced \" squamous metaplasia \" in the COPD epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Third, in contrast to the 220 healthy epithelium, SARS-CoV-2 induced higher squamous metaplasia in the infected COPD 221 epithelium and caused a substantial increase in the height of the epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Third, in contrast to the 219 healthy epithelium, SARS-CoV-2 induced higher squamous metaplasia in the infected COPD 220 epithelium and caused a substantial increase in the height of the epithelium.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "The images represent multiple independent SARS-CoV-2 increases squamous metaplasia in the COPD airway epithelia.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "These findings are consistent with SARS-CoV-2 indirectly activating EC programs that regulate thrombosis and endotheliitis in COVID-19 patients and focus strategies on therapeutically targeting epithelial and inflammatory responses that activate the endothelium or initiate limited ACE2 independent EC infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "We can hypothesize that the direct cytopathic effect of SARS-CoV-2 on pericytes may result in endothelial cell and microvascular dysfunction [ 36 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "We can hypothesize that the direct cytopathic effect of SARS-CoV-2 on pericytes may result in endothelial cell and microvascular dysfunction [ 36 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "During the influenza H1N1 pandemic in December 2009 , there was a sharp increase in KD cases.42 Similarly , the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , a cause of the coronavirus pandemic in 2019-2020 ( COVID-19 ) , was associated with sharp increase in the incidence of KD.42-44 It should be noted that SARS-CoV-2 can infect endothelial cells and cause endothelial cell damage and thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D042783 ! \"Endothelial Cells\")\n",
      "[ 42 ] [ 43 ] [ 44 ] It should be noted that SARS-CoV-2 can infect endothelial cells and cause endothelial cell damage and thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Discussion There is a worldwide concern about COVID-19 as a global public health threat as SARS-CoV-2 causes severe acute respiratory syndrome [ 24 , 25 ] , and this has caused the death of many patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "These include the betacoronaviruses that cause severe acute respiratory syndrome ( SARS-CoV ) , the Middle East respiratory syndrome ( MERS-CoV ) [ 3 ] , and SARS-CoV-2 , all of which were responsible for a relatively high number of cases with high mortality rates .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "These include the betacoronaviruses that cause severe acute respiratory syndrome ( SARS-CoV ) , the Middle East respiratory syndrome ( MERS-CoV ) [ 3 ] , and SARS-CoV-2 , all of which were responsible for a relatively high number of cases with high mortality rates .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "As shown in this table , SARS-CoV-2 can cause severe acute respiratory syndrome and has a 4.2 % mortality rate ; similarly , SARS-CoV and MERS-CoV can cause severe acute respiratory syndrome and have mortality rates of 11 % and 34 % , respectively ( Yang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "As shown in this table , SARS-CoV-2 can cause severe acute respiratory syndrome and has a 4.2 % mortality rate ; similarly , SARS-CoV and MERS-CoV can cause severe acute respiratory syndrome and have mortality rates of 11 % and 34 % , respectively ( Yang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "From 2002 to 2019, three deadly human coronaviruses (hCoVs), severe acute respiratory syndrome coronavirus (SARS-CoV), Middle Eastern respiratory syndrome coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) emerged to produce outbreaks of SARS, MERS and coronavirus disease 2019 (Covid-19), respectively.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "From 2002 to 2019, three deadly human coronaviruses (hCoVs), severe acute respiratory syndrome coronavirus (SARS-CoV), Middle Eastern respiratory syndrome coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) emerged to produce outbreaks of SARS, MERS and coronavirus disease 2019 (Covid-19), respectively.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The spike protein of SARS-CoV-2 shares 79.5% sequence identity with the SARS-CoV virus3-5, which caused the SARS pandemic in 2002, resulting in 774 deaths in 8096 confirmed patients in 29 countries6.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "There is a worldwide concern about COVID-19 as a global public health threat as SARS-CoV-2 causes severe acute respiratory syndrome [ 24 , 25 ] , and this has caused the death of many patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "ere is a worldwide concern about COVID-19 as a global public health threat as SARS-CoV-2 causes severe acute respiratory syndrome [ 24 , 25 ] , and this has caused the death of many patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The severe acute respiratory syndrome due to coronavirus 2 ( SARS-CoV-2 ) infection has affected millions of individuals worldwide , causing high mortality rates and severe physical sequelae , with a negative impact on society , economy , health care , lifestyle and personal relationships .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In less than two decades, three deadly zoonotic coronaviruses, severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and SARS-CoV-2, have emerged in humans, causing SARS, MERS, and coronavirus disease 2019 (COVID-19), respectively.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "This is in line with the current proposal that severe acute respiratory syndrome due to SARS-CoV-2 and its associated high mortality rate may be as a result of vitamin D3 deficiency .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "As a possible corollary, application of nasal high-flow has been shown to reduce the work of breathing by approximately 50%.8The use of nasal high-flow in the context of the Severe Acute Respiratory Syndrome caused by the novel Coronavirus-2 (SARS-Cov2) is more nuanced.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The new coronavirus ( SARS-CoV-2 ) that causes a severe acute respiratory syndrome emerges in Wuhan , China , in December 2019 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Both SARS-CoV-1 and SARS-CoV-2 can cause severe acute respiratory syndrome .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "According to genomic sequencing, SARS-CoV-2 shares 79.5% homology with the sequence of SARS-CoV that caused the outbreak of SARS in 2003 [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Both SARS-CoV-1 and SARS-CoV-2 can cause severe acute respiratory syndrome .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV , which caused SARS in 2003 , and SARS-CoV-2 , which is responsible for COVID-19 , are very closely related Sarbecoviruses .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "These include the betacoronaviruses that cause severe acute respiratory syndrome ( SARS-CoV ) , the Middle East respiratory syndrome ( MERS-CoV ) [ 3 ] , and SARS-CoV-2 , all of which were responsible for a relatively high number of cases with high mortality rates .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The new coronavirus ( SARS-CoV-2 ) that causes a severe acute respiratory syndrome emerges in Wuhan , China , in December 2019 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) induces a severe acute respiratory syndrome that is called COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS and SARS-CoV-2 have approximately 79 % genome sequence homology , and SARS-CoV-2 has a higher similarity to coronaviruses found in bats , causing SARS [ 8,11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) induces a severe acute respiratory syndrome that is called COVID-19 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Mild illnesses include some cases of the common cold ( which is also caused by certain other viruses , predominantly rhinoviruses ) , while more lethal varieties can cause SARS , MERS , and COVID-19.1 COVID-19 is caused by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) and has a case-fatality rate of 2:3 % , with higher rates among elderly patients and patients with concurrent medical conditions ( WHO , May.31 During attachment and penetration , the SARS-CoV-2 attaches itself to a host cell ACE2 PD domain through its spike ( S ) protein ( Li et al.15 ,18 Structurally , the coronavirus has the most massive known RNA genome of 26 to 32 kb amongst other known viruses , characterized by non-segmented , positive-sense single-stranded RNA .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Mild illnesses include some cases of the common cold ( which is also caused by certain other viruses , predominantly rhinoviruses ) , while more lethal varieties can cause SARS , MERS , and COVID-19.1 COVID-19 is caused by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) and has a case-fatality rate of 2:3 % , with higher rates among elderly patients and patients with concurrent medical conditions ( WHO , May.31 During attachment and penetration , the SARS-CoV-2 attaches itself to a host cell ACE2 PD domain through its spike ( S ) protein ( Li et al.15 ,18 Structurally , the coronavirus has the most massive known RNA genome of 26 to 32 kb amongst other known viruses , characterized by non-segmented , positive-sense single-stranded RNA .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Coronavirus disease 2019 ( COVID-19 ) is an infectious disease caused by coronavirus-2 ( SARS-CoV-2 ) that causes a severe acute respiratory syndrome , a characteristic hyperinflammatory response , vascular damage , microangiopathy , angiogenesis and widespread thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Coronavirus disease 2019 ( COVID-19 ) is an infectious disease caused by coronavirus-2 ( SARS-CoV-2 ) that causes a severe acute respiratory syndrome , a characteristic hyperinflammatory response , vascular damage , microangiopathy , angiogenesis and widespread thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D047928 ! \"Premature Birth\")\n",
      "However , most of them were reported in China which has a different medical system , and it is not yet clear whether SARS-CoV-2 caused these premature births or whether it was iatrogenic due to maternal or fetal distress or other indications .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D047928 ! \"Premature Birth\")\n",
      "However , most of them were reported in China which has a different medical system , and it is not yet clear whether SARS-CoV-2 caused these premature births or whether it was iatrogenic due to maternal or fetal distress or other indications .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "The figure was created with Biorender.com TP53 and VAMP8 respectively are the key proteins of Cytokines storm and neutrophil degranulation process associated with acute respiratory distress syndrome ( ARDS ) SARS-CoV-2 causes acute lung injury and acute respiratory distress syndrome ( ARDS ) , which can result in severe lethal conditions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) outbreak that began in 2019 and spread rapidly across the globe has been observed to cause acute lung injury and multiorgan system failure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) outbreak that began in 2019 and spread rapidly across the globe has been observed to cause acute lung injury and multiorgan system failure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "SARS-CoV-2 may cause acute lung injury , and secondary infections are thus relevant complications in patients with COVID-19 pneumonia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "OBJECTIVES : SARS-CoV-2 may cause acute lung injury , and secondary infections are thus relevant complications in patients with COVID-19 pneumonia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "Objectives : SARS-CoV-2 may cause acute lung injury , and secondary infections are thus relevant complications in patients with COVID-19 pneumonia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "SARS-CoV-2 damages the respiratory tract and causes acute lung injury ( ALI ) ( 2 , 4 , 5 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D056704 ! \"Adaptive Immunity\")\n",
      "Once in the host cell , SARS-CoV-2 can be detected by macrophages , which orchestrate the production of a pro-inflammatory microenvironment that inhibits viral replication , stimulates adaptive immunity , and recruits other immune cells to the site of infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D056704 ! \"Adaptive Immunity\")\n",
      "Once in the host cell , SARS-CoV-2 can be detected by macrophages , which orchestrate the production of a pro-inflammatory microenvironment that inhibits viral replication , stimulates adaptive immunity , and recruits other immune cells to the site of infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D056704 ! \"Adaptive Immunity\")\n",
      "Once in the host cell , SARS-CoV-2 can be detected by macrophages , which orchestrate the production of a pro-inflammatory microenvironment that inhibits viral replication , stimulates adaptive immunity , and recruits other immune cells to the site of infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D057134 ! \"Antibodies, Neutralizing\")\n",
      "Potent human neutralizing antibodies elicited by SARS-CoV-2 infection.bioRxiv (2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D058345 ! \"Asymptomatic Infections\")\n",
      "Even though the first cases had a contact history with the Huanan Seafood Market the studies have clearly showed that SARS-CoV-2 can be transmitted by person-to-person and frequently cause asymptomatic infections .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D058345 ! \"Asymptomatic Infections\")\n",
      "SARS-CoV-2 may cause asymptomatic infection or a mild viral illness , while more severe COVID-19 cases are characterised by high fever , cough and dyspnoea [ 2-4 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D058847 ! Inflammasomes)\n",
      "Furthermore , SARS-CoV-2 may activate inflammasomes ( Shah , 2020 ) , further putting the elderly at risk of hyperinflammation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "Abstract The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "Abstract The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The 2019-nCoV commonly affects upper and lower respiratory airways and presents with a sore throat , sputum production , sneezing , dry coughs , dyspnea , rhinorrhea , fever , pneumonia and SARS coronavirus 2 ( SARS-CoV-2 ) [ 1-3 ] ; also , the 2019-nCoV can cause myalgia , arthralgia , gastrointestinal presentation ( diarrhea , vomiting , loss of appetite , abdominal pain and liver injury with raised enzymes ) , olfactory and gustatory dysfunctions , CNS involvement and neurological manifestations ( headache , dizziness , change in mental status and meningeal signs ) , metabolic and electrolyte derangements and multiorgan failure [ 3-5 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "In contrast the SARS-CoV-2 is principally causing headache , fever , sore throat , cough , acute respiratory distress syndrome , acute cardiac injury , upper airway congestion , pharyngalgia , myalgia , and diarrhea , anemia , and even the secondary infections [ 37 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The emerging virus , SARS-CoV-2 , causes coughing , fever , muscle ache , and shortness of breath or dyspnea in symptomatic patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D063806 ! Myalgia)\n",
      "The severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) usually causes fever , respiratory symptoms , malaise and myalgia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:D064766 ! \"Influenza A Virus, H7N9 Subtype\")\n",
      "Similar to the H7N9 virus ( 5 , 6 ) , SARS-CoV-2 causes high mortality in elderly patients who have preexisting chronic diseases .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:B4ER10 ! B4ER10)\n",
      "No reuse allowed without permission.416 417 In A549ACE2 cells, SARS-CoV-2 induced low levels of IFNl and IFNb mRNAs and somewhat 418 higher ISG mRNA by 48 hpi, as compared with SINV (Fig 6A).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:O80400 ! O80400)\n",
      "This creates a new subset of perturbator candidates V PC subset and equalV P which is formed by all those proteins targeted by SARS-CoV-2, which are significantly expressed in the lungs, that can navigate outside this organ via EVs, and that can activate VPs via subdiffusive dynamics in the PPI network.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P03126 ! P03126)\n",
      "SARS-CoV-2 infectious particles were quantified in media by standard plaque assay.46 Briefly, Vero E6 cells were treated with serial dilutions of SARS-CoV-2 in DMEM (4% FBS) and incubated at 37degreesC for 1 hr.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P03126 ! P03126)\n",
      "52 Briefly, Vero E6 cells were treated with serial dilutions of SARS-CoV-2 in DMEM (4% FBS) and incubated at 37 degreesC for 1 h.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P03126 ! P03126)\n",
      "52 Briefly, Vero E6 cells were treated with serial dilutions of SARS-CoV-2 in DMEM (4% FBS) and incubated at 37degreesC for 1 h.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P0C6V4 ! P0C6V4)\n",
      "The pathogenesis of SARS-CoV-2 is characterized by a strong inhibition of innate immune sensing and production of type I interferon (IFN-I) responses.2 SARS-CoV-2 shares 79% of its genome with SARS-CoV-1, the agent responsible of the 2003 SARS epidemics, allowing comparative analyses that can address the molecular determinants in pathogenicity.In this context, a study by Shin et al. 1 aimed at identifying similarities and key differences in the activities of PLpro produced by SARS-CoV-1 (SCoV1-PLpro) and by SARS-CoV-2 (SCoV2-PLpro).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P0DTC2 ! P0DTC2)\n",
      "Out of 95 samples obtained from patients diagnosed with dengue before September 2019, SARS-CoV-2 serology targeting the S protein was positive and equivocal in 21 (22%) (sixteen IgA, five IgG) versus four positives and equivocal in 102 controls (4%) (P = 1.6E-4).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(UP:P0DTC2 ! P0DTC2)\n",
      "The SARS-CoV-2 relies on the host cell proteases to cut its S protein in two parts thus forming N-terminal part which recognizes ACE2 receptor and C-terminal part involved in the viral entry which must be further cleaved by furin and / or other furin-like enzymes ( Coutard et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(UP:P0DTC2 ! P0DTC2)\n",
      "The SARS-CoV-2 relies on the host cell proteases to cut its S protein in two parts thus forming N-terminal part which recognizes ACE2 receptor and C-terminal part involved in the viral entry which must be further cleaved by furin and / or other furin-like enzymes ( Coutard et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(UP:P0DTC2 ! P0DTC2)\n",
      "The SARS-CoV-2 relies on the host cell proteases to cut its S protein in two parts thus forming N-terminal part which recognizes ACE2 receptor and C-terminal part involved in the viral entry which must be further cleaved by furin and / or other furin-like enzymes ( Coutard et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P32318 ! P32318)\n",
      "While Y453 of SARS-CoV-2 contributed 2.018 kcal mol-1, F486 214 contributed 3.01 kcal mol-1 to the interaction.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(UP:P84849 ! P84849)\n",
      "SARS-CoV-2 has been shown to activate the lectin (MBL) complement pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"RABV G\")\n",
      "Because the expression of SARS-CoV-2 reduced expression of RABV G on the cell surface and also reduced the incorporation of RABV G into the RABV virus, we performed VNA assays against RABV.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ACE-2 receptors\")\n",
      "Also, SARS-CoV-2 downregulates the expression of ACE-2 receptors, which degrade angiotensin II.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:hACE2)\n",
      "Cellular invasion of SARS-CoV-2 reduces hACE2 expression, thereby promoting the recruitment of neutrophils (Tomar et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:hACE2)\n",
      "Here, we pseudotyped the SARS-CoV-2 Spike glycoprotein (SPG) on a traditional retroviral (MMLV) as well as a third generation lentiviral (pLV) vector and tested the transduction efficiency in several mammalian cell lines expressing SARS-CoV-2 receptor hACE2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:hACE2)\n",
      "Since T lymphocytes display a very low expression level of hACE2, a novel receptor might be involved in the entry of SARS-CoV-2 into T cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:hACE2)\n",
      "Fatal neuroinvasion of SARS-CoV-2 in K18-hACE2 mice is partially dependent on hACE2 expression.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"hoﬆ gene\")\n",
      "(C) A model of how SARS-CoV-2 suppresses hoﬆ gene expression through multi pronged approach : 1.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:17705 ! SELENOF)\n",
      "Of seven ER-resident selenoproteins, SARS-CoV-2 simultaneously suppressed the expression of SELENOF, SELENOM, SELENOK and SELENOS, indicating that the ER is an organelle that is severely adversely affected by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:30394 ! SELENOK)\n",
      "Of seven ER-resident selenoproteins, SARS-CoV-2 simultaneously suppressed the expression of SELENOF, SELENOM, SELENOK and SELENOS, indicating that the ER is an organelle that is severely adversely affected by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:30396 ! SELENOS)\n",
      "Of seven ER-resident selenoproteins, SARS-CoV-2 simultaneously suppressed the expression of SELENOF, SELENOM, SELENOK and SELENOS, indicating that the ER is an organelle that is severely adversely affected by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:30396 ! SELENOS)\n",
      "We found that SARS-CoV-2 suppressed mRNA expression of selenoproteins associated with ferroptosis (GPX4), ER stress (SELENOF, SELENOK, SELENOM and SELENOS) and DNA synthesis (TXNRD3) in Vero cells, providing a deeper insight into the connection between selenium and SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:30397 ! SELENOM)\n",
      "Of seven ER-resident selenoproteins, SARS-CoV-2 simultaneously suppressed the expression of SELENOF, SELENOM, SELENOK and SELENOS, indicating that the ER is an organelle that is severely adversely affected by SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:3621 ! FCGRT)\n",
      "Although SARS-CoV-2 infection did not disrupt FcRn expression in the placenta, maternal transfer of nAb was inhibited by SARS-CoV-2 infection during pregnancy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4556 ! GPX4)\n",
      "Apart from the testis, pulmonary TXNRD3 levels are high according to the Human Protein Atlas.22 Thus, by knockdown of TXNRD3, SARS-CoV-2 would boost virus production in the lung, the major site of SARS-CoV-2 replication.Taken together, Se deficiency, which compromises the production of Se sensitive GPX1, and SARS-CoV-2, which reduces GPX4 expression, could synergistically destroy GPX antioxidant defenses, resulting in concurrent increase of both intracellular ROS and membrane lipid peroxidation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4556 ! GPX4)\n",
      "An in vitro study also showed SARS-CoV-2 suppressed GPX4 , the brake of ferroptosis43 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4556 ! GPX4)\n",
      "An in vitro study also showed SARS-CoV-2 suppressed GPX4 , the brake of ferroptosis43 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4556 ! GPX4)\n",
      "An in vitro study also showed SARS-CoV-2 suppressed GPX4 , the brake of ferroptosis 43 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4556 ! GPX4)\n",
      "An in vitro study also showed SARS-CoV-2 suppressed GPX4, the brake of ferroptosis 43.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "It is not clear that SARS-CoV-2 could downregulate the expression of ACE2 or not as the homology of SARS-CoV-2 with SARS-CoV.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:11362 ! STAT1, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Yang et al. found that the SARS-CoV-2 inhibition of STAT1 phosphorylation leads to the attenuation of the interferon stimulated genes transcription in monocyte derived dendritic cells and macrophages [49].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11362 ! STAT1, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "However, in Calu-3 cells, SARS-CoV-2 infection induced long lasting phosphorylation of STAT1 and STAT3, low virus production, and no prominent apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:1alpha)\n",
      "At the cellular level, once cells are infected with SARS-CoV, a coronavirus that shares 79.5% identity to SARS-CoV-2, accumulation of hypoxia inducible factor 1alpha (HIF-1alpha) may occur due to increased expression as well as inhibited proteasome degradation [20].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:CK)\n",
      "Massive accumulation of SARS-CoV-2 leads to a surge of immune cells and more and more pro inflammatory cytokines, resulting in a rapid increase in CK levels in the blood, or releasing more virus particles into the blood circulation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ISG genes\")\n",
      "On the contrary, SARS-CoV-2 infection induced expression of the ISG genes in positive SARS-CoV-2 samples at baseline compared to negative control subjects, although this increase wanes with time.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"ISG mRNAs\")\n",
      "Furthermore, SARS-CoV-2 induced higher levels of ISG mRNAs than MERS-CoV and, in 426 the case of OAS2, higher than MERS-CoV-DNS4ab as well.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Protease-2)\n",
      "By using publicly available datasets we qualified epithelial ACE2 expression in human intestine, and assessed gene co-expression of ACE2 and SARS-CoV-2 priming Transmembrane Serine Protease 2 (TMPRSS2) with nAChRs in intestinal epithelial cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:Protease-2)\n",
      "By using publicly available datasets we qualified epithelial  ACE2  expression in human intestine, and assessed gene co-expression of  ACE2  and SARS-CoV-2 priming  Transmembrane Serine Protease 2 (TMPRSS2)  with nAChRs in intestinal epithelial cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:SARS-CoV-1)\n",
      "Similar to SARS-CoV-1, cellular entry of SARS-CoV-2 depends on the expression of ACE2, a receptor that is abundantly expressed in the nervous system.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:SARS-CoV-1)\n",
      "Here, we used publicly archived transcriptomic datasets to compute consensus regulatory signatures, or consensomes, that rank human genes based on their rates of differential expression in MERS-CoV (MERS), SARS-CoV-1 (SARS1) and SARS-CoV-2 (SARS2)-infected cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(IP:IPR026113)\n",
      "Moreover, SARS-CoV-2 infection not only increased the expression of methyltransferase like 3 (METTL3) but also altered its distribution.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(TEXT:\"type I and III interferons\")\n",
      "A recent single cell analysis of SARS-CoV-2 infection shows that SARS-CoV-2 induces low levels of type I and III interferons, but an elevated level of inflammatory chemokines (64).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:16335 ! adenosine)\n",
      "Depending on the stage of exposure to and infection by SARS-CoV-2, enhancing adenosine levels by targeting key adenosine regulators such as ADA, ADK and equilibrative nucleoside transporter 1 might find therapeutic use against COVID-19 and warrants further investigation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases a(CHEBI:17234 ! glucose)\n",
      "Together, people with diabetes are more susceptible to SARS-CoV-2 infection, and SARS-CoV-2 infection may increase the level of blood glucose, suggesting that hyperglycemia is a key factor between COVID-19 and diabetes.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) directlyIncreases a(CHEBI:36080 ! protein)\n",
      "The spike protein (S protein) expressed by SARS-CoV-2 contains a receptor binding domain (RBD), which interacts with host cells via the human angiotensin converting enzyme 2 (hACE2).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:IL1 ! IL1)\n",
      "Mast cells activated by SARS-CoV-2 release histamine which increases IL-1 levels causing cytokine storm and inflammatory reaction in COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:10618 ! CCL2)\n",
      "The hearts of SARS-CoV-2 infected hamsters showed downregulation of functional CM associated genes, upregulation of ROS associated genes, and increased CCL2 expression and macrophage infiltration, which was further confirmed using autopsy samples of COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1116 ! BSG)\n",
      "Furthermore, respiratory epithelia were negative for ACE-2 and TMPRSS2 protein expression while positive for BSG and furin, suggesting a possible alternative entry route for SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11892 ! TNF)\n",
      "However, TNF-alpha levels were marginally induced by SARS-CoV-2, when compared with Flu activation (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16028 ! IL33)\n",
      "A preprint XREF_BIBR that recently appeared online supports our model, revealing that SARS-CoV-2 peptide exposure elicits IL-33 expression from patients who are virus seropositive, and IL-33 production is correlated with T-cell activation and lung disease severity.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:16039 ! ANGPTL4)\n",
      "SARS-CoV-2 can induce the transcription of ANGPTL4 by utilizing TCF12 transcription factor which in turn could cause pulmonary tissue damage 66.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:1663 ! CD36)\n",
      "Our results 249 demonstrated that SARS-CoV-2 infection increase the CD36 expression in monocytes, 250 suggesting the increase of lipids uptake can contribute to LD formation, observed after 251 the infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:18363 ! IFNL1)\n",
      "Both SARS-CoV-2 and Flu induced high levels of IFN-alpha2 and IFN-lambda1, both being critical anti-viral effector cytokines (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30048 ! PIR)\n",
      "SARS-CoV-2 disruption of ACE2 is proposed to reduce PIR expression (Fadason et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:30048 ! PIR)\n",
      "SARS-CoV-2 disruption of ACE2 is proposed to reduce PIR expression (Fadason et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:4603 ! CXCL2)\n",
      "Paralleling findings from SARS-CoV infection studies, SARS-CoV-2 infection on CMs downregulated the expression of ACE2 XREF_BIBR and upregulated expression of CXCL2, XREF_BIBR along with other immunomodulatory cytokines.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5423 ! IFNA2)\n",
      "Both SARS-CoV-2 and Flu induced high levels of IFN-alpha2 and IFN-lambda1, both being critical anti-viral effector cytokines (XREF_FIG).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:5423 ! IFNA2)\n",
      "IFN-alpha2 levels following SARS-CoV-2 activation reached up to 80 ng/ml, indicating a very efficient activation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:6383 ! KNG1)\n",
      "This is because SARS-CoV-2 increases the amount of bradykinin in the infected tissues.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "Minimization of spreading of SARS-CoV-2 via household waste produced by subjects affected by COVID-19 or in quarantine.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "The hyperinflammatory response induced by SARS-CoV-2 is pivotal in the pathogenesis of COVID-19 and is accompanied by an upregulated expression of interleukin 6 (IL-6) that correlates with disease severity (3).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) directlyIncreases p(UP:P14107 ! P14107)\n",
      "However, ORF7/8 is 21 only expressed by SARS-CoV-1 and SARS-CoV-2 with little homology to other seasonal 22 coronaviruses,hencetheroleofsuchpeptidecross-reactivecellsispuzzlingandcallsfora 23 moredetailedanalysisoftheeffectofpre-existingimmunityinthecontrolorpathogenesis 10 bioRxiv preprint doi : https://doi.org/10.1101/2020.10.15.341958.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"(S) protein\")\n",
      "Some reports have shown that the neutralizing antibody titers are strongly correlated with RBD binding IgG concentration.16 However, most of these vaccines developed specifically against SARS-CoV-2 infection may not be effective to prevent infection by SARS-CoV-2 with significant mutations in its spike (S) protein and the SARS-CoV and SARSr-CoVs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"(S) protein\")\n",
      "16 However, most of these vaccines developed specifically against SARS-CoV-2 infection may not be effective to prevent infection by SARS-CoV-2 with significant mutations in its spike (S) protein and the SARS-CoV and SARSr-CoVs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"A First Data Release\")\n",
      "Targeting SARS-CoV-2 with AI- and HPC enabled Lead Generation : A First Data Release.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:A.Sharma1)\n",
      "No reuse allowed without permission.Supplementary InformationforStructural stability of SARS-CoV-2 degrades with temperatureAuthors : A.Sharma1, B. Preece1, H, Swann1, X. Fan2, R.J.McKenney2, K.M.Ori-McKenney2, S. Saffarian1,3,4 * and M.D.Vershinin1,3,4 * Affiliations : 1 Department of Physics and Astronomy, University of Utah, Salt Lake City, Utah, USA.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:ACE2R)\n",
      "SARS-CoV-2 attached to host cells through ACE2R and vitamin D might reduce ACE2R.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"Appendix.In WWTP1\")\n",
      "Technical details are included in the Appendix.In WWTP1, SARS-CoV-2 genome copy numbers progressively decreased from April 13 to May 18.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(IP:IPR004724)\n",
      "As in the alveolar cell, SARS-CoV-2 may completely suppress endothelial ENaC activity and thus contribute to the severity of systemic disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:GC)\n",
      "These morphological changes in the testes of COVID-19 patients indicate that SARS-CoV-2 infection may impair male GC development and eventually lead to GC loss.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:IECs)\n",
      "These results indicated that SARS-CoV-2 infection inhibits the turnover of IECs and mucin, and induces apoptosis to disrupt the intestinal barrier, causing further infection in multiple tissues.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:ISG)\n",
      "Transcriptome profiling of certain cell types reveals that SARS-CoV-2 infection elicits exceptionally low INF-I and IFN-III and reduced ISG while inducing pro inflammatory chemokine and cytokine genes (30).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:MHC-I)\n",
      "It has been shown that ORF8 of SARS-CoV-2, but not ORF8 or ORF8a/b of SARS-CoV, downregulates MHC-I in cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:MHC-I)\n",
      "It has been shown that ORF8 of SARS-CoV-2, but not ORF8 or ORF8a/b of SARS-CoV, downregulates MHC-I in cells (16).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"N gene\")\n",
      "Real-time reverse-transcriptase-polymerase-chain-reaction (rRT-PCR) by Roter Gene Q 6000 (Qiagen, Hilden, Germany) for SARS-CoV-2 targeting the N gene and ORF1ab gene (Sansure, China) was positive.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"ORF1ab gene\")\n",
      "Real-time reverse-transcriptase-polymerase-chain-reaction (rRT-PCR) by Roter Gene Q 6000 (Qiagen, Hilden, Germany) for SARS-CoV-2 targeting the N gene and ORF1ab gene (Sansure, China) was positive.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"RANTES mRNA\")\n",
      "However, we could not discard that SARS-CoV-2 disrupt RANTES mRNA as evasion strategy at early stage of infection, trying to block the migration of immune cells to the sites of infection; delaying the immediate immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"SARS-CoV-2 Mpro\")\n",
      "Further, depending on the modeling results, we have proposed novel as well as potent SARS-CoV-2 Mpro inhibitors.Keywords SARS-CoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"SARS-CoV-2 nanoLuc\")\n",
      "Our de novo protein decoys also prevented infection by SARS-CoV-2 expressing the nanoLuc reporter (SARS-CoV-2 nanoLuc, see methods) in the lung epithelial cell line Calu-3, which expresses both ACE2 and the transmembrane protease serine 2 (TMPRSS2), known to augment the infectivity of SARS-CoV-2 (30, 31).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:Sharma1)\n",
      "No reuse allowed without permission.Structural stability of SARS-CoV-2 degrades with temperatureAuthors : A. Sharma1, B. Preece1, H, Swann1, X. Fan2, R.J.McKenney2, K.M.Ori-McKenney2, S. Saffarian1,3,4 * and M.D.Vershinin1,3,4 * Affiliations : 1 Department of Physics and Astronomy, University of Utah, Salt Lake City, Utah, USA.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"Supplementary Figs. 3b-c\")\n",
      "Remarkably, even in the absence of mayor cytopathic effects, SARS-CoV-2 infection rapidly resulted in reduced beating rate, lower depolarization spike amplitude, and decreased electrical conduction velocity (Figs. 3bc and Supplementary Figs. 3b-c).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(PFAM:PF08260 ! kinin)\n",
      "By binding to the ACE-2 receptor on Type II pneumocytes of the lung, SARS-CoV-2 can dysregulate the kallikrein and kinin system and initiate the coagulation cascade [19].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"nanoLuc reporter\")\n",
      "Our de novo protein decoys also prevented infection by SARS-CoV-2 expressing the nanoLuc reporter (SARS-CoV-2 nanoLuc, see methods) in the lung epithelial cell line Calu-3, which expresses both ACE2 and the transmembrane protease serine 2 (TMPRSS2), known to augment the infectivity of SARS-CoV-2 (30, 31).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:15846 ! \"NAD(+)\")\n",
      "SARS-CoV-2 deregulates NAD + synthesis and utilization as well as induces expression of PARP [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:16183 ! methane)\n",
      "(H) SARS-CoV-2 infects AT2 cells and rapidly inhibits sodium reabsorption (red arrow 1).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:16991 ! \"deoxyribonucleic acid\")\n",
      "Intriguingly , SARS-CoV-2 causes DNA damage , cell growth arrest , cytokine production , and p38 activation during the early stages of infection as a strategy to promote more efficient replication .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:16991 ! \"deoxyribonucleic acid\")\n",
      "Consistent with the concept that coronaviruses can induce cell growth arrest to promote their replication , Bouhaddou et al. recently demonstrated that SARS-CoV-2 causes both DNA damage during the early stage of infection and cell cycle arrest [ 66 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:25812 ! ozone)\n",
      "Yao et al. deliberated the influence of local environmental factors on the number of confirmed COVID-19 cases in 31 various regions of China, and the results suggested that the environmental diffusion coefficient of SARS-CoV-2 continued to decrease following the ambient ozone concentration increases (Yao et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:29101 ! \"sodium(1+)\")\n",
      "(H) SARS-CoV-2 infects AT2 cells and rapidly inhibits sodium reabsorption (red arrow 1).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:30413 ! heme)\n",
      "This may provide a novel insight into the impact of SARS-CoV-2 on erythropoiesis and hypoxia seen in COVID-19 patients.hypoxia, the impact of SARS-CoV-2 infection on erythropoiesis has received limited attention.Preliminary modeling reports have suggested that SARS-CoV-2 may inhibit heme metabolism and induce hemoglobin denaturation 14.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:32806 ! \"trans-aconitic acid\")\n",
      "Cardiogenic Shock Caused by SARS-CoV-2 in a Patient with Serial Negative Nucleic Acid Amplification Tests.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:37670 ! \"protease inhibitor\")\n",
      "Thus, the usefulness of a SARS-CoV-2 antiviral regimen even at later times of infection further supports the investigation of therapeutic efficacy of coronavirus specific 3CL pro protease inhibitors for the treatment of COVID-19.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:48433 ! angiotensin)\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ 1 ] [ 2 ] [ 3 ] [ 4 ] [ 5 ] [ 6 ] [ 7 ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:48433 ! angiotensin)\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ [ 1 ] , [ 2 ] , [ 3 ] , [ 4 ] , [ 5 ] , [ 6 ] , [ 7 ] ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:48433 ! angiotensin)\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ [ 1 ] , [ 2 ] , [ 3 ] , [ 4 ] , [ 5 ] , [ 6 ] , [ 7 ] ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:48433 ! angiotensin)\n",
      "Upon binding to ACE2 , SARS-CoV-2 causes its downregulation , thus lowering angiotensin - ( 1-7 ) levels , which can mimic / worsen the vasoconstriction , inflammation , and pro-coagulopathic effects that occur in preeclampsia .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:74566 ! Lys-Met)\n",
      "Characterization of the enzymatic activity of ACE2 demonstrated that SARS-CoV-2 significantly increased substrate affinities by lowering Km values and also boosted efficiency (kcat/Km) on different substrates.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:9584 ! Thyroid)\n",
      "Further studies are required to examine 288 whether SARS-CoV-2 can directly impair the thyroid .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(CHEBI:9584 ! Thyroid)\n",
      "Further studies are required to examine whether SARS-CoV-2 can directly impair the thyroid .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:host)\n",
      "Results Here , we have at first taken the inhibitors of SARS-CoV-2 , which does not allow them to replicate in the host .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:host)\n",
      "However , in COVID-19 , both phases are delayed due to suppression of the host 's gene expression by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) ' s nsp1 protein ( 4 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:host)\n",
      "However , in COVID-19 , both phases are delayed due to suppression of the host 's gene expression by the severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) ' s nsp1 protein ( 4 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"pancreatic cell\")\n",
      "To date , it is not known if SARS-CoV-2 can cause pancreatic cell damage leading to acute pancreatitis , but in many cases serum lipase and amylase levels are elevated [ 8] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"pancreatic cell\")\n",
      "To date , it is not known if SARS-CoV-2 can cause pancreatic cell damage leading to acute pancreatitis , but in many cases serum lipase and amylase levels are elevated [ 8] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:\"G_i\" ! \"G_i\")\n",
      "The mechanism by which SARS-CoV-2 causes GI damage is still unclear .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:\"G_i\" ! \"G_i\")\n",
      "The mechanism by which SARS-CoV-2 causes GI damage is still unclear .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(FPLX:IFNAR ! IFNAR)\n",
      "It will be worth determining exactly how SARS-CoV-2 antagonizes IFN induction and IFNAR signaling [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0001816\" ! \"cytokine production\")\n",
      "Recent studies suggested that SARS-CoV-2 impairs type I Interferon production, hampering the host 's antiviral response while triggering a sustained chemokine storm 10,25,31,32.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0001816\" ! \"cytokine production\")\n",
      "By binding TBK1 and SINTBAD, SARS-CoV-2 NSP13 might 385 inhibit the interferon production by IRF3/IRF7 stimulation (Figure 7).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0001816\" ! \"cytokine production\")\n",
      "As SARS-CoV2 suppresses antiviral cytokine production, virus clearance may also be supported by the substitution of type 1 interferons, which activate their cytokine receptor (IFNAR) and induce anti-viral cellular programs.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In COVID-19 patients , type 1 IFN production is delayed by SARS-CoV-2 , which lowers the adaptive immune response .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In COVID-19 patients, type 1 IFN production is delayed by SARS-CoV-2, which lowers the adaptive immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "Antibody titers to RBD correlate with reduction of 59 plaque formation by SARS-CoV-2 in ACE2-producing Vero cells ( Okba et al ., 2020 ) , and sera 60 with high-titer anti-RBD antibodies neutralize SARS-CoV-2 more potently ( Hansen et al ., 2020 , 61 Yuan et al ., 2020 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "Antibody titers to RBD correlate with reduction of plaque formation by SARS-CoV-2 in ACE2-producing Vero cells ( Okba et al ., 2020 ) , and sera with high-titer anti-RBD antibodies neutralize SARS-CoV-2 more potently ( Hansen et al ., 2020 ; Yuan et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Recently, it has been proved that SARS-CoV-2 blocks the autophagy pathway; however, spermidine reduce SARS-COV-2 infection by alleviating the lysosomal degradation pathway [XREF_BIBR].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "It is also noteworthy that SARS-CoV-2 inhibits autophagy, predisposing infected cells to apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Intriguingly, another recent preprint presents in vitro data showing that SARS-CoV-2 infection restricts autophagy and that, in turn, pro-autophagic compounds - including spermidine - may inhibit viral propagation [85].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "SARS-CoV-2 proteins impede autophagy and phagocytosis to ensure its existence 346 During viral infection one important immune response is autophagy which destroys the virus 347 infected cells and viruses are often found to modulate it for their survival (Ahmad et al., 348 2018).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Similarly , SARS-CoV-2 suppresses autophagy by modulating multiple autophagy regulatory factors ( Gassen et al ., 2020 ) , by blocking the degradation of viral factors , and by increasing the formation of DMVs to promote virus replication .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Moreover, recent studies have found that SARS-CoV-2 infection limits autophagy by interfering with multiple metabolic pathways [7] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Similarly, SARS-CoV-2 suppresses autophagy by modulating multiple autophagy regulatory factors, by blocking the degradation of viral factors, and by increasing the formation of DMVs to promote virus replication.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "SARS-CoV2, similarly to MERS-CoV, is able to reduce autophagy in infected cell lines by reducing the mTORC1-pathway, autophagy related signaling, and the fusion between autophagosome and lysosome.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "SARS-CoV2, similarly to MERS-CoV, is able to reduce autophagy in infected cell lines by reducing the mTORC1-pathway, autophagy related signaling, and the fusion between autophagosome and lysosome (Fecchi et al., 2020).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "25 Here we show that SARS-CoV-2 infection limits autophagy by interfering with multiple metabolic 26 pathways and that compound-driven interventions aimed at autophagy induction reduce SARS-CoV-2 27 propagation in vitro.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "In sum, SARS-CoV-2 infection causally diminishes 34 autophagy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "The strong increase in RNA binding activity of SQSTM1 upon SARS-CoV-2 infection suggests that autophagy is inhibited by SARS-CoV-2 through this pathway.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "198 In summary, our data show that highly pathogenic SARS-CoV-2 reprograms the metabolism of cells 199 and limits AMPK/mTORC1 activation and autophagy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Pharmacological modulation of metabolism-dependent cellular pathways such as autophagy reduced propagation of highly pathogenic Middle East respiratory syndrome (MERS)-CoV.Here we show that SARS-CoV-2 infection limits autophagy by interfering with multiple metabolic pathways and that compound-driven interventions aimed at autophagy induction reduce SARS-CoV-2 propagation in vitro.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "In sum, SARS-CoV-2 infection causally diminishes autophagy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Similarly , SARS-CoV-2 suppresses autophagy by modulating multiple autophagy regulatory factors ( Gassen et al ., 2020 ) , by blocking the degradation of viral factors , and by increasing the formation of DMVs to promote virus replication .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Furthermore , we observed that SARS-CoV-2 fails to induce apoptosis in human bronchial epithelial cells ( i.e ., BEAS2B cells ) and primary human umbilical vein endothelial cells ( HUVECs ) , which are refractory to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "In fact, a recent publication comparing the ability of ORF3a proteins between SARS-CoV and SARS-CoV-2 to induce apoptosis indicates that ORF3a of SARS-CoV-2 decreases levels of apoptosis in infected cells, potentially allowing the virus to spread more widely during the infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our findings reveal a landscape of ligand-receptor interactions in the lung caused by SARS-CoV-2 viral infection and point to potential interventions to reduce the burden of neurogenic inflammation in COVID-19 disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our findings reveal a landscape of ligand-receptor interactions in the lung caused by SARS-CoV-2 viral infection and point to potential interventions to reduce the burden of neurogenic inflammation in COVID-19 pulmonary disease.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "No reuse allowed without permission.suggesting that melatonin 's mechanism of action in the most severe cases of COVID-19 may be targeted to SARS-CoV2 induced inflammation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Thus, exercise may counteract, at least partially, the detrimental effects of the binding of SARS-CoV-2 to ACE2 receptor, reducing the inflammatory response in the lungs.Preliminary evidence suggests that the severity of symptoms associated with Covid-19 and the eventual outcome of being infected with this virus are associated with the health status of individuals prior to infection.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Several abnormalities have been observed including cellular immune deficiency , coagulation activation , myocardial injury , hepatic and kidney injury , and secondary bacterial infection 43.Of note , scientists reported that SARS-COV-2 causes an inflammatory response in the lower airway , which may lead to lung injury 44,45 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Several abnormalities have been observed including cellular immune deficiency , coagulation activation , myocardial injury , hepatic and kidney injury , and secondary bacterial infection 43.Of note , scientists reported that SARS-COV-2 causes an inflammatory response in the lower airway , which may lead to lung injury 44,45 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "1: Potential mechanisms associated with cannabinoid mediated suppression of SARS-CoV-2 induced lung inflammation and fibrosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Potential mechanisms associated with cannabinoid mediated suppression of SARS-CoV-2 induced lung inflammation and fibrosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Despite robust infection efficiency in both the human lung and intestinal epithelial cells, SARS-CoV-2 could attenuate the virus induced pro inflammatory response and IFN response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These findings suggest that the SARS-CoV-2 mainly caused inflammatory response damaging alveolar and deep respiratory tract 10 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These data suggest either lung-specific inflammatory response (21) that is not reflected in the circulation or SARS-CoV-2 mediated mechanisms that may actively suppress the host inflammatory response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our patients with COVID-19 clearly did not have transient ARDS, with a median PaO2:FiO2 ratio of 77 (IQR 68–86) and a positive end-expiry pressure of 10 cm H2O at baseline, which could have further worsened even after orotracheal intubation.We agree with Aouba and colleagues that it is possible that during the initial phases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, even a low dose of subcutaneous anakinra might prevent escalation of inflammation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our patients with COVID-19 clearly did not have transient ARDS, with a median PaO2:FiO2 ratio of 77 (IQR 68–86) and a positive end-expiry pressure of 10 cm H2O at baseline, which could have further worsened even after orotracheal intubation.We agree with Aouba and colleagues that it is possible that during the initial phases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, even a low dose of subcutaneous anakinra might prevent escalation of inflammation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "ACE-2-interacting Domain of SARS-CoV-2 (AIDS) Peptide Suppresses Inflammation to Reduce Fever and Protect Lungs and Heart in Mice : Implications for COVID-19 Therapy.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0007283\" ! spermatogenesis)\n",
      "SARS-CoV-2 has been found to induce impaired spermatogenesis and retarding sperm maturation which may be due to the immune response in testis and epididymis in male patients [ 34 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0007283\" ! spermatogenesis)\n",
      "Therefore, SARS-CoV-2 may directly target ACE2-positive spermatogonia and disrupt spermatogenesis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0009606\" ! tropism)\n",
      "These results demonstrate that SARS-CoV-2 has limited tropism in the lung with AT2 pneumocytes and ciliated airway epithelial cells being the predominant lung cells infected by virus .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0010494\" ! \"cytoplasmic stress granule\")\n",
      "Thereafter , intracellularly , SARS-CoV-2 impairs the stress granule formation and without the stress granules , the virus replicates and damages the cell .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n",
      "SARS-CoV-2 enters human cells via angiotensin-converting enzyme 2 ( ACE2 ) , which is also found in human pancreatic beta-cells , suggesting that SARS-CoV-2 might alter pancreatic beta-cell function and impair insulin secretion [ [ 10 ] , [ 11 ] , [ 12 ] ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0030073\" ! \"insulin secretion\")\n",
      "SARS-CoV-2 enters human cells via angiotensin-converting enzyme 2 ( ACE2 ) , which is also found in human pancreatic beta-cells , suggesting that SARS-CoV-2 might alter pancreatic beta-cell function and impair insulin secretion [ [ 10 ] , [ 11 ] , [ 12 ] ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0030097\" ! hemopoiesis)\n",
      "Based on this phenomenon, it is speculated that SARS-CoV-2 similarly inhibits hematopoiesis in the bone marrow through certain receptors to cause decreased primary platelet formation and lead to thrombocytopenia.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0042098\" ! \"T cell proliferation\")\n",
      "Collazo , Proteins from SARS-CoV-2 reduce T cell proliferation : a mirror image of sepsis , HELIYON , https://doi.and luis.augusto.aguirre@idipaz.es ( L.A.A .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0042098\" ! \"T cell proliferation\")\n",
      "Proteins from SARS-CoV-2 reduce T cell proliferation : a mirror image of sepsis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0042098\" ! \"T cell proliferation\")\n",
      "Proteins from SARS-CoV-2 reduce T cell proliferation : A mirror image of sepsis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(GO:\"GO:0042098\" ! \"T cell proliferation\")\n",
      "Journal Pre-proof Proteins from SARS-CoV-2 reduce T cell proliferation : a mirror image of sepsis Proteins from SARS-CoV-2 reduce T cell proliferation : a mirror image of sepsis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:12680 ! VEGFA)\n",
      "Using vascular endothelial growth factor-A (VEGF-A), a physiological ligand for the b1b2 pocket in NRP-1, we interrogated whether the Spike protein, the major surface antigen of SARS-CoV-2, could block VEGF-A and NRP -1 signaling to affect pain behaviors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:17433 ! CHP1)\n",
      "SARS-CoV-2 has minimal tropism for neurons but disrupts barrier integrity of the ChP We next sought to test whether there could be any conditions in which neurons or other CNS cell types may be infected with SARSCoV-2.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:20667 ! TXNRD3)\n",
      "TXNRD3 is mainly expressed in the testis.17 SARS-CoV-2 has been detected in the semen of patients with COVID19.18 Orchitis was a complication of SARS-CoV infection.19 GPX4 is essential for sperm maturation and is correlated with fertility related parameters.20,21 The present study suggests that TXNRD3 knockdown by SARS-CoV-2 and GPX4 down-regulation owing to SARS-CoV-2 triggered elevation of IL6 together probably deteriorate male fertility.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:20667 ! TXNRD3)\n",
      "According to computational analysis, SARS-CoV-2 targets TXNRD3 by antisense at several sites, with computed interaction energies equivalent to the strongest microRNA interactions.16 Consistent with the computational prediction, our study found that SARS-CoV-2 significantly down-regulated TXNRD3, by 36.9%.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:4853 ! HDAC2)\n",
      "While it is unknown whether SARS-CoV-2 inhibits or enhances HDAC2 activity, this interaction indicates a potential direct modulation of the cytokine storm and antigen presentation.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:6935 ! MCC)\n",
      "( B ) Ciliated cells in the nasal epithelium are heavily infected with SARS-CoV-2 , which disrupts their MCC activity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:6935 ! MCC)\n",
      "( B ) Ciliated cells in the nasal epithelium are heavily infected with SARS-CoV-2 , which disrupts their MCC activity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:6935 ! MCC)\n",
      "( B ) Ciliated cells in the nasal epithelium are heavily infected with SARS-CoV-2 , which disrupts their MCC activity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:8004 ! NRP1)\n",
      "Using vascular endothelial growth factor-A (VEGF-A), a physiological ligand for the b1b2 pocket in NRP-1, we interrogated whether the Spike protein, the major surface antigen of SARS-CoV-2, could block VEGF-A and NRP -1 signaling to affect pain behaviors.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:9884 ! RB1)\n",
      "Structural homology with SARS-CoV-1 indicates that SARS-CoV-2 may be able to directly impair pRb and p53, which are key gatekeepers with tumor suppressor functions.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases p(HGNC:9884 ! RB1)\n",
      "Structural homology with SARS-CoV-1 indicates that SARS-CoV-2 may be able to directly impair pRb and p53 , which are key gatekeepers with tumor suppressor functions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:Peripheral)\n",
      "Thus , since that peripheral parts of the lung strongly express TMPRSS2 , along with ACE2 , the SARS-CoV-2 may be considered to damage the peripheral area at the beginning of infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:Peripheral)\n",
      "Thus , since that peripheral parts of the lung strongly express TMPRSS2 , along with ACE2 , the SARS-CoV-2 may be considered to damage the peripheral area at the beginning of infection .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:C413524 ! \"angiotensin-converting enzyme-2\")\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ [ 1 ] , [ 2 ] , [ 3 ] , [ 4 ] , [ 5 ] , [ 6 ] , [ 7 ] ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:C413524 ! \"angiotensin-converting enzyme-2\")\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ [ 1 ] , [ 2 ] , [ 3 ] , [ 4 ] , [ 5 ] , [ 6 ] , [ 7 ] ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:C413524 ! \"angiotensin-converting enzyme-2\")\n",
      "For example , recent evidence has suggested that SARS-CoV-2 expression may induce angiotensin-converting enzyme-2 ( ACE-2 ) downregulation due to the binding of the viral spike protein to its receptor , then promoting a decrease in the angiotensin - [ 1 ] [ 2 ] [ 3 ] [ 4 ] [ 5 ] [ 6 ] [ 7 ] production [ 21 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:C413524 ! \"angiotensin-converting enzyme-2\")\n",
      "The severe acute respiratory syndrome coronavirus-2 ( SARS-COV-2 ) , a novel coronavirus responsible for the recent infectious pandemic , is known to downregulate angiotensin-converting enzyme-2 ( ACE2 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001402 ! B-Lymphocytes)\n",
      "In addition , recent evidences suggest that SARS-CoV-2 may induce beta-cells damage thus leading to insulin secretion impairment .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001402 ! B-Lymphocytes)\n",
      "In addition , recent evidences suggest that SARS-CoV-2 may induce beta-cells damage thus leading to insulin secretion impairment .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "Additionally , SARS-CoV-2 can bind and engage with the ACE2 receptor in the capillary endothelium to damage the blood-brain barrier .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "Additionally , SARS-CoV-2 can bind and engage with the ACE2 receptor in the capillary endothelium to damage the blood-brain barrier .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "SARs-CoV-2 could also cause damage to the blood-brain barrier and therefore enter the CNS by inflicting injuries to the vascular system.4 Majority of the infected patients suffer only mild , coldlike symptoms .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "Moreover , by binding to ACE2 receptors expressed on the capillary endothelium , the SARS-CoV-2 may disrupt the blood-brain barrier and get access into the CNS ( 113 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "Moreover , by binding to ACE2 receptors expressed on the capillary endothelium , the SARS-CoV-2 may disrupt the blood-brain barrier and get access into the CNS ( 113 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "Penetration through the Blood-brain - barrier Once in the blood circulation , SARS-CoV-2 could bind to the ACE2 receptors of the endothelium [ 50 , 52 , 55 ] and disrupt the Blood Brain Barrier ( BBB ) , inducing edema , intracranial hypertension and / or penetration of the virus in the CNS [ 64 , 65 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "15-17 Furthermore , SARS-CoV-2 can damage the blood-brain barrier , invade the nervous system through the slow cerebral microcirculation , which the facilitates the interaction between the protein S ( spike ) and the ACE-2 receptors expressed on the capillary endothelium , and interact with the ACE-2 receptors expressed in neuronal cells .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D003106 ! Colon)\n",
      "SARS-CoV-2 infects the host cells via ACE2 receptors and causes damage to lungs , CNS , heart , colon , kidneys , and eyes ; hence , attention must be given when treating COVID-19 patients to its systemic effects .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D004548 ! Elasticity)\n",
      "SARS-CoV-2 could induce vascular endothelial injuries in the ascending aorta , reduced its elasticity and then increased diameter , although the exact mechanism could only be revealed via autopsy of the aorta ( 35 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D004730 ! \"Endothelium, Vascular\")\n",
      "31 It is also reported that thrombotic microangiopathy related to direct or indirect damage by SARS-CoV-2 to the vascular endothelium can occur .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D004730 ! \"Endothelium, Vascular\")\n",
      "31 It is also reported that thrombotic microangiopathy related to direct or indirect damage by SARS-CoV-2 to the vascular endothelium can occur .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D005123 ! Eye)\n",
      "SARS-CoV-2 infects the host cells via ACE2 receptors and causes damage to lungs , CNS , heart , colon , kidneys , and eyes ; hence , attention must be given when treating COVID-19 patients to its systemic effects .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007370 ! \"Interferon Type I\")\n",
      "Type I interferons have antiviral properties and may be actively repressed by SARS-CoV-1 and SARS-CoV-2 during infection to escape the host immune response.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Evidence has been accumulating that besides lung injury , SARS-CoV-2 also damages the human heart , kidneys , liver , nervous system and skin ( Edler et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3,12,13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3 , 12 , 13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Except for lung damage , SARS-CoV-2 may infiltrate other organs such as heart , kidney , and brain , causing cardiomyopathy , arrhythmias , kidney failure , and encephalitis , respectively [ 3,12,13 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "However , it is elusive as to whether SARS-CoV-2 can directly damage the kidney or induce acute renal failure .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Most patients had normal renal function , and only 2 ( 3.0 % ) had abnormal renal function indicators , both of whom were severe patients , suggesting that SARS-CoV-2 may not cause significant kidney damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Most patients had normal renal function , and only 2 ( 3.0 % ) had abnormal renal function indicators , both of whom were severe patients , suggesting that SARS-CoV-2 may not cause significant kidney damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Most patients had normal renal function , and only 2 ( 3.0 % ) had abnormal renal function indicators , both of whom were severe patients , suggesting that SARS-CoV-2 may not cause significant kidney damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Therefore , SARS-CoV-2 may cause kidney damage via the ACE2 pathway .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "SARS-CoV-2 infects the host cells via ACE2 receptors and causes damage to lungs , CNS , heart , colon , kidneys , and eyes ; hence , attention must be given when treating COVID-19 patients to its systemic effects .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "It could be speculated that SARS-CoV-2 may directly impair the kidney via proximal renal tubular injury .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "It could be speculated that SARS-CoV-2 may directly impair the kidney via proximal renal tubular injury .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007668 ! Kidney)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007694 ! \"Killer Cells, Natural\")\n",
      "Consistent with a previous study , we found lymphocyte , CD4 + T cell , CD8 + T cell , NK cell , and B cell counts to be negatively correlated to the severity of COVID-19 , suggesting that immune suppression could be more likely found in severe patients and SARS-CoV-2 may directly or indirectly damage the lymphocytes or NK cells and thus further aggravate the disease progression17 ,18 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007694 ! \"Killer Cells, Natural\")\n",
      "Consistent with a previous study , we found lymphocyte , CD4 + T cell , CD8 + T cell , NK cell , and B cell counts to be negatively correlated to the severity of COVID-19 , suggesting that immune suppression could be more likely found in severe patients and SARS-CoV-2 may directly or indirectly damage the lymphocytes or NK cells and thus further aggravate the disease progression17 ,18 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D007694 ! \"Killer Cells, Natural\")\n",
      "Consistent with a previous study , we found lymphocyte , CD4 + T cell , CD8 + T cell , NK cell , and B cell counts to be negatively correlated to the severity of COVID-19 , suggesting that immune suppression could be more likely found in severe patients and SARS-CoV-2 may directly or indirectly damage the lymphocytes or NK cells and thus further aggravate the disease progression 17 , 18 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008099 ! Liver)\n",
      "Evidence has been accumulating that besides lung injury , SARS-CoV-2 also damages the human heart , kidneys , liver , nervous system and skin ( Edler et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008099 ! Liver)\n",
      "3 Chau et al. 4 demonstrated in their study that liver biopsies of SARS patients showed a significant increase in mitotic cells , eosinophils and balloon-like liver cells , which indicated that SARS-CoV-2 might induce liver cell apoptosis and thus lead to liver damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008099 ! Liver)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008099 ! Liver)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008221 ! \"Lymphoid Tissue\")\n",
      "Their results demonstrate that SARS-CoV-2 infected the CD169 + macrophages in spleens and lymph nodes ( LNs ) , and lead to lymphoid tissue damage , such as splenic nodule atrophy and lymph follicle depletion , etc.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008221 ! \"Lymphoid Tissue\")\n",
      "Their results demonstrate that SARS-CoV-2 infected the CD169 + macrophages in spleens and lymph nodes ( LNs ) , and lead to lymphoid tissue damage , such as splenic nodule atrophy and lymph follicle depletion , etc.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D008221 ! \"Lymphoid Tissue\")\n",
      "Their results demonstrate that SARS-CoV-2 infected the CD169 + macrophages in spleens and lymph nodes ( LNs ) , and lead to lymphoid tissue damage , such as splenic nodule atrophy and lymph follicle depletion , etc.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009102 ! \"Multiple Organ Failure\")\n",
      "SARS-CoV-2 is highly contagious and may lead to acute respiratory distress or multiple organ failure [ 7 , 8] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009102 ! \"Multiple Organ Failure\")\n",
      "SARS-CoV-2 is highly contagious and may lead to acute respiratory distress or multiple organ failure [ 7 , 8] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009206 ! Myocardium)\n",
      "Prior studies have reported that SARS-CoV-2 infects host cells through angiotensin-converting enzyme 2 ( ACE2 ) to cause COVID-19 , and also causes damage to the myocardium [ 18 , 19 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009206 ! Myocardium)\n",
      "Prior studies have reported that SARS-CoV-2 infects host cells through angiotensin-converting enzyme 2 ( ACE2 ) to cause COVID-19 , and also causes damage to the myocardium [ 18 , 19 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009206 ! Myocardium)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009206 ! Myocardium)\n",
      "Special attention should be paid to organ protection and supportive treatments as there are reports that SARS-CoV-2 may damage the liver , kidneys and myocardium to some extent .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009361 ! \"Neoplasm Invasiveness\")\n",
      "It is also known that heparin prevents infection by a range of viruses70 and was recently reported to inhibit invasion by SARS-CoV-2 in a cell-based assay71 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009412 ! \"Nerve Fibers\")\n",
      "It is therefore possible that in predisposed individuals , the cardiovascular system is affected before the respiratory system , with electrical dysregulations being caused by circulating levels of pro-inflammatory cytokines , stress hormones , electrolytic imbalances , or drug cardiotoxicity ( Chung et al ., 1990 ; Yokoyama et al ., 1993 ; Hasan , 2013 ; Driggin et al ., 2020 ) , but SARS-CoV-2 might directly damage nerve fibers .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D009412 ! \"Nerve Fibers\")\n",
      "It is therefore possible that in predisposed individuals , the cardiovascular system is affected before the respiratory system , with electrical dysregulations being caused by circulating levels of pro-inflammatory cytokines , stress hormones , electrolytic imbalances , or drug cardiotoxicity ( Chung et al ., 1990 ; Yokoyama et al ., 1993 ; Hasan , 2013 ; Driggin et al ., 2020 ) , but SARS-CoV-2 might directly damage nerve fibers .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D010179 ! Pancreas)\n",
      "5 ) SARS-CoV-2 may damage the pancreas .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D010179 ! Pancreas)\n",
      "5 ) SARS-CoV-2 may damage the pancreas .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D012131 ! \"Respiratory Insufficiency\")\n",
      "Since SARS-CoV-2 may lead to respiratory failure , most of the diagnostic and therapeutic efforts are focused on the consequences of the infection in the respiratory tract .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D012131 ! \"Respiratory Insufficiency\")\n",
      "Since SARS-CoV-2 may lead to respiratory failure , most of the diagnostic and therapeutic efforts are focused on the consequences of the infection in the respiratory tract .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D012141 ! \"Respiratory Tract Infections\")\n",
      "COVID-19 has been suggested to be a disease of the nicotinic cholinergic system ( 7 ) and SARS-CoV-2 most commonly causes a lower respiratory infection or pneumonia ( 8) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D012867 ! Skin)\n",
      "Evidence has been accumulating that besides lung injury , SARS-CoV-2 also damages the human heart , kidneys , liver , nervous system and skin ( Edler et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013116 ! \"Spinal Cord\")\n",
      "Besides involvement of the brain , SARS-CoV-2 can also damage the spinal cord .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013116 ! \"Spinal Cord\")\n",
      "Besides involvement of the brain , SARS-CoV-2 can also damage the spinal cord .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "Some evidences also demonstrated that SARS-CoV-2 dysregulates T lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "Some evidences also demonstrated that SARS-CoV-2 dysregulates T lymphocytes .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "However , SARS-CoV-2 can inhibit T cells by inducing programmed cell death ( apoptosis ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "However , SARS-CoV-2 can inhibit T cells by inducing programmed cell death ( apoptosis ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D015431 ! \"Weight Loss\")\n",
      "SARS-CoV-2 at 10 5 TCID 50 caused weight loss , evoked antibody responses , and developed histological evidence of lung inflammation in a K18-hACE2 transgenedependent manner with interstitial congestion , inflammatory exudate , epithelial damage , etc. [ 4 , 24 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D017951 ! \"Antigen Presentation\")\n",
      "First , SARS-CoV-2 may impair antigen presentation or trigger an acquired immunosuppression with concomitant lymphopenia , thus probably leading to an increased susceptibility to superinfection [ 25 , 26 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018034 ! \"Olfactory Receptor Neurons\")\n",
      "SARS-CoV-2 may also infect and damage olfactory sensory neurons ( OSNs ) in the epithelium lining .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Lymphocyte count and the T cell subgroup counts are decreased after infection by SARS-CoV-2 [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Lymphocyte count and the T cell subgroup counts are decreased after infection by SARS-CoV-2 [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Test results : it is reported that SARS-CoV-2 may cause a decrease in peripheral white blood cell and lymphocyte counts in the early stages of the disease [ 5 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Test results : it is reported that SARS-CoV-2 may cause a decrease in peripheral white blood cell and lymphocyte counts in the early stages of the disease [ 5 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Test results : it is reported that SARS-CoV-2 may cause a decrease in peripheral white blood cell and lymphocyte counts in the early stages of the disease [ 5 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D018655 ! \"Lymphocyte Count\")\n",
      "Lymphocyte count and the T cell subgroup counts are decreased after infection by SARS-CoV-2 [ 11 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D019369 ! Life)\n",
      "The spread of the new severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) has disrupted normal life across the world .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D020490 ! Patent)\n",
      "Our data demonstrated the induction of a specific J o u r n a l P r e - p r o o f monocyte profile ( HLA-DR low PD-L1 high ) due to the presence of the cocktail of proteins from SARS-CoV-2 , which in turn caused a patent T-cells exhaustion .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D020490 ! Patent)\n",
      "Our data demonstrated the induction of a specific monocyte profile ( HLA-DRlowPD-L1high ) due to the presence of the cocktail of proteins from SARS-CoV-2 , which in turn caused a patent T-cells exhaustion .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases bp(MESH:D032342 ! \"Muscle Cells\")\n",
      "The questions remain of whether SARS-CoV-2 can directly infect the myocardium and cause myocyte damage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) decreases a(TEXT:\"Pre-proof L'impact psychologique silencieux de la pandemie de COVID-19 au Soudan.Coronavirus disease-2019\")\n",
      "Journal Pre-proof L'impact psychologique silencieux de la pandemie de COVID-19 au Soudan.Coronavirus disease-2019 (COVID-19), caused by the novel coronavirus SARS-CoV-2, has spread to almost every region of the globe and has affected more than 24,299,923 people worldwide, resulting in 894,476 deaths as of September 7, 2020 [1].\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:MEK ! MEK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Thus, SARS-CoV-2 spike protein mediated cell signaling may occur through a receptor other than ACE2.In contrast to the effects of SARS-CoV-2 spike protein on human cells to activate the MEK phosphorylation, this protein promoted the dephosphorylation of MEK in rat cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:MEK ! MEK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "While we interpreted that the ACE2 antibody blocked and interfered with the binding of the RBD to ACE2, it is possible that ACE2 antibody mediated cell signaling may have desensitized the subsequent spike protein mediated effects to activate MEK.Thus, SARS-CoV-2 spike protein mediated cell signaling may occur through a receptor other than ACE2.In contrast to the effects of SARS-CoV-2 spike protein on human cells to activate the MEK phosphorylation, this protein promoted the dephosphorylation of MEK in rat cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:MEK ! MEK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Thus, SARS-CoV-2 spike protein mediated cell signaling may occur through a receptor other than ACE2.In contrast to the effects of SARS-CoV-2 spike protein on human cells to activate the MEK phosphorylation, this protein promoted the dephosphorylation of MEK in rat cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(FPLX:MEK ! MEK, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "While we interpreted that the ACE2 antibody blocked and interfered with the binding of the RBD to ACE2, it is possible that ACE2 antibody mediated cell signaling may have desensitized the subsequent spike protein mediated effects to activate MEK.Thus, SARS-CoV-2 spike protein mediated cell signaling may occur through a receptor other than ACE2.In contrast to the effects of SARS-CoV-2 spike protein on human cells to activate the MEK phosphorylation, this protein promoted the dephosphorylation of MEK in rat cells.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:11364 ! STAT3, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "However, in Calu-3 cells, SARS-CoV-2 infection induced long lasting phosphorylation of STAT1 and STAT3, low virus production, and no prominent apoptosis.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV-2) increases p(HGNC:18643 ! SUGP1, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "Indeed, we observed that SARS-CoV-2 modulated RBPs were more frequently phosphorylated at multiple sites than their unaltered counterparts.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:eGFP)\n",
      "In agreement with our previous report10, HEK293 219 cells transiently expressing hACE2 proteins showed extensive syncytium formation 220 when coincubated with 293T cells overexpressing eGFP and SARS-CoV or SARS221 CoV-2 S proteins in the presence of trypsin (Fig 4A).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(PFAM:PF03090 ! replicase)\n",
      "The SARS-CoV 3CL pro mediates the proteolytic processing of replicase polypeptides into functional proteins, playing an important role in viral replication.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "The Papainlike protease ( PLpro ) of SARS-CoV activates the ROS / p38 MAPK / STAT3 pathway to ultimately upregulate the production and release of TGF-beta1 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "The Papainlike protease ( PLpro ) of SARS-CoV activates the ROS / p38 MAPK / STAT3 pathway to ultimately upregulate the production and release of TGF-beta1 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n",
      "In an ex vivo human lung explant model , SARS-CoV-2 infection also failed to induce detectable levels of IFNalpha , beta , gamma , or lambda1-3 at any time point for up to 48 hr , while SARS-CoV induced IFNalpha , beta , gamma , or lambda1-3 at late time points ( Chu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n",
      "Coordinate induction of IFN-alpha and - gamma by SARS-CoV also in the absence of virus replication .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n",
      "In this study , we extended the observations to the recently discovered etiologic agent of SARS , obtaining evidence that SARS-CoV has the intrinsic ability to activate both IFN-alpha and - gamma gene expression in PBMC cultures .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n",
      "Concomitant activation of IFN-alpha and - gamma gene expression by SARS-CoV in vivo may be relevant for the pathogenesis of the disease , both for the possible involvement in immunomediated damage of the tissues and for the strong inhibition of SARS-CoV replication as a result of combined cytokine action .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:30212 ! photon)\n",
      "Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:33699 ! \"messenger RNA\")\n",
      "On the basis of the location of putative body TRSs , the synthesis of nine mRNAs by SARS-CoV was expected : the genomic mRNA ( RNA1 ) and eight subgenomic mRNAs with sizes of approximately 8.4 , 4.6 , 3.8 , 3.5 , 3.0 , 2.6 , 2.1 and 1.8 kb ( including 5 ' leader and 3 ' poly ( A ) - tail ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:34342 ! 3-methylcholanthrene)\n",
      "In contrast to the cytopathic infection of the Vero E6 cell line , SARS-CoV did not cause overt cytopathic effects in PTEC or MC .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:36080 ! protein)\n",
      "SARS coronavirus [ 48 ] , produce proteins , which inactivate p53 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:36080 ! protein)\n",
      "While SARS-CoV induces 20 ISG proteins over its 72-h time course , none of these ISG proteins are detected prior to 30 hpi ; similarly , MERS-CoV induces five ISG proteins , with only one ( STAT1 ) detected prior to 18 hpi , thus confirming delayed ISG induction in both CoVs .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:36080 ! protein)\n",
      "A 2005 article by Kuba et al. demonstrated that SARS-CoV ( the viral etiology of the first SARS pandemic ) binds to the ACE2 , elicits endocytosis , and reduces ACE2 protein levels on the cell membrane [ 49 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:48219 ! disinfectant)\n",
      "The spread of SARS coronavirus 2 ( SARS-CoV-2 ) , which causes the disease COVID-19 , has resulted in a surge in disinfectant use to keep the indoor environment safe.1 ,2 Intensified cleaning protocols during the COVID-19 pandemic specifically call for the increased use of disinfectants in homes and high-risk public spaces , such as schools , health and other care facilities , and food service and work spaces .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:50858 ! corticosteroid)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:50858 ! corticosteroid)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:50858 ! corticosteroid)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:60156 ! \"oxidised phospholipid\")\n",
      "For example , SARS-CoV triggers the production of oxidized phospholipids , which then drives IL-6 production via TLR4 ( 80 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:60156 ! \"oxidised phospholipid\")\n",
      "For example , SARS-CoV triggers the production of oxidized phospholipids , which then drives IL-6 production via TLR4 ( 80 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(CHEBI:60156 ! \"oxidised phospholipid\")\n",
      "For example , SARS-CoV triggers the production of oxidized phospholipids , which then drives IL-6 production via TLR4 ( 80 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"Middle East respiratory syndrome\")\n",
      "SARS-CoV-2 belongs to the family of single-stranded RNA viruses known as coronaviruses , including the MERS-CoV and SARS-CoV that cause Middle East respiratory syndrome ( MERS ) and severe acute respiratory syndrome ( SARS ) , respectively .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"acute myocarditis\")\n",
      "Considering this fact , Zheng et al. ( 2020 ) pointed that Middle East respiratory 45 syndrome-related coronavirus ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-Cov ) can cause acute myocarditis and heart failure .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"acute myocarditis\")\n",
      "In December 2019 , started perhaps the most critical outbreak of the recent hundred years : the rapid widespread of the coron - ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-Cov ) can cause acute myocarditis and heart failure ( Zheng et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:cytokine)\n",
      "Furthermore , it was not systematically investigated how the cytokine induction by SARS-CoV compares to other viruses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:cytokine)\n",
      "Here , we have used three control viruses and two different cells lines to elucidate and compare the induction of cytokines by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:cytokine)\n",
      "In contrast to the findings of Kindler et al ., another previous study using well-differentiated normal human bronchial epithelial cells indicated that SARS-CoV induced higher cytokine and chemokine levels than HCoV-229E ( 9 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:cytokine)\n",
      "81 SARS-CoV also causes phenotypic and functional maturation of dendritic cells in vitro , resulting in a moderate production of cytokines and an enhanced T-cell-stimulatory capacity .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:cytokine)\n",
      "With the exception of IP-10 and IL-8 , SARS-CoV is capable to suppress the production of a wide range of cytokines .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:cytokine)\n",
      "On the other hand , evidence from clinical and experimental studies suggests that SARS-CoV can induce cytokine dysregulation .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"fasting blood glucose measurement\")\n",
      "In the earlier studies , SARS-CoV had been shown to cause higher fasting plasma glucose levels [ 30 ] and the damage of the pancreatic beta cells was incriminated as a cause for development of `` acute diabetes '' [ 31 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"adhesion factor\")\n",
      "To study the cellular response in the acute phase of SARS coronavirus ( SARS-CoV ) - host cell interaction , we investigated the induction of chemokines , adhesion molecules , and DC-SIGN ( dendritic cell-specific ICAM-3-grabbing nonintegrin ) by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:complication)\n",
      "SARS-CoV and MERS-CoV were reported to result in severe complications during pregnancy .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:complication)\n",
      "Previously , both severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) created severe complications in pregnant women ( 5 , 6 ) , and the mortality rate of SARS-CoV infection was 25 % in pregnant women ( 4 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:complication)\n",
      "In 2005 , it was proposed that the RAAS system was responsible for complications due to Sars-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:AP1 ! AP1)\n",
      "On the other hand , activation of p38 MAPK , AP-1 and Akt signal transduction pathway by SARS-CoV [ Mizutani et al ., 2004b , c ] may be correlated with inflammatory responses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:AP1 ! AP1)\n",
      "Expression of IL-8 is dependent on the transcription factor AP-1 , and molecular analyses revealed that SARS-CoV and MHV strongly activate AP-1 [ 103 ] , [ 129 ] , [ 135 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "In an ex vivo human lung explant model , SARS-CoV-2 infection also failed to induce detectable levels of IFNalpha , beta , gamma , or lambda1-3 at any time point for up to 48 hr , while SARS-CoV induced IFNalpha , beta , gamma , or lambda1-3 at late time points ( Chu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "In an ex vivo human lung explant model , SARS-CoV-2 infection also failed to induce detectable levels of IFNa , b , g , or l1-3 at any time point for up to 48 hr , while SARS-CoV induced IFNa , b , g , or l1-3 at late time points .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "The results of a representative experiments are shown in Fig. 1 and indicated that SARS-CoV is able to induce both IFN-alpha and - gamma mRNA in a dose-dependent manner , being MOI 10 the most effective ( Fig. 1a ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "In this study , we extended the observations to the recently discovered etiologic agent of SARS , obtaining evidence that SARS-CoV has the intrinsic ability to activate both IFN-alpha and - gamma gene expression in PBMC cultures .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "It was recently demonstrated that MHV-A59 and SARS-CoV produce IFN-a via the activation of intracellular TLR7 in plasmacytoid dendritic cells , thus controlling viral infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "Altogether , the data presented in this study , along with previous results [ 10 , 11 , 12 , 13 , 14 , 15 , 16 , 17 , 18 , 19 ] , demonstrate that SARS-CoV is a weak inducer of IFN-alpha , - beta and - gamma .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IFNA ! IFNA)\n",
      "It was recently demonstrated that MHV-A59 and SARS-CoV produce IFN-alpha via the activation of intracellular TLR7 in plasmacytoid dendritic cells , thus controlling viral infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IL12 ! IL12)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs. 3, 4) [49, 50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IL12 ! IL12)\n",
      "As shown in Fig. 3A , SARS-CoV marginally upregulated the expression of IFN-gamma and did not induce an upregulation of IL-12 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:IL12 ! IL12)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:IL12 ! IL12)\n",
      "IL-12 production following direct stimulation of monocytes was apparently disrupted to a greater extent by SARS-CoV and / or steroid treatment than that induced by PHA-activated T cells via CD40-CD154 interaction [ 27 ] , possibly because of higher levels of inhibitory IL-10 in SAC-stimulated cultures .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:JNK ! JNK)\n",
      "Further , the top 100 genes in the SARS-CoV list of regulators includes all three JNK / p38-specific MAPK phosphatases ( DUSP8 , DUSP10 , and DUSP16 ) , which is significant since SARS-CoV is known to up-regulate JNK [ 44 ] , [ 45 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "The results of the dual-luciferase assay revealed that the SARS-CoV M gene specifically and dramatically upregulated NF-kappaB activity compared with the controls ( Fig. 1G ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "At a late stage of infection , SARS-CoV replicates at a high level and progeny viruses are released from host cells , SARS-CoV activates NF-kB and upregulates COX-2 expression , then triggering antiviral activities of prostanoids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "( G ) The SARS-CoV M gene stimulates NF-kappaB activation .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Together , these results indicate that SARS-CoV M suppresses NF-kappaB activity probably through a direct interaction with IKKbeta , resulting in lower Cox-2 expression .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Together , these results indicate that SARS-CoV M suppresses NF-kB activity probably through a direct interaction with IKKb , resulting in lower Cox-2 expression .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:NFkappaB ! NFkappaB)\n",
      "Earlier studies have reported that SARS-CoV and HCoV-OC43 can impair NF-kappaB activation [ 17 , 25 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:NFkappaB ! NFkappaB)\n",
      "SARS-CoV is also reported to have its membrane protein ( SARS-CoV M ) suppress NF-kappaB activity through interaction with inhibitor of nuclear factor kappa-B kinase subunit beta , which further indicates the need for exogenous NO therapeutics [ 54 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:p38 ! p38)\n",
      "On the other hand , activation of p38 MAPK , AP-1 and Akt signal transduction pathway by SARS-CoV [ Mizutani et al ., 2004b , c ] may be correlated with inflammatory responses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:p38 ! p38)\n",
      "Notably , when the E protein is present , SARS-CoV overstimulates the NF-kappaB inflammatory pathway ( DeDiego et al ., 2014b ) and , through its PDZ-binding motif , interacts with the cellular protein syntenin , triggering p38 MAPK activation .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:p38 ! p38)\n",
      "SARS-CoV can also directly activate p38 or the SARS-CoV E protein can activate syntenin , which in tune activates p38 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(FPLX:p38 ! p38)\n",
      "SARS-CoV can also directly activate p38 or the SARS-CoV E protein can activate syntenin , which in tune activates p38 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "Compared to SARS-CoV-2 S, 127 SARS-CoV S induced weaker cell fusion as recently reported 35 , and this process was 128 also blocked by CH25H expression (Fig. 3C) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "The spike protein of the severe acute respiratory syndrome coronavirus (SARS-CoV) mediates cell fusion by binding to target cell surface receptors.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "Abstract The spike protein of the severe acute respiratory syndrome coronavirus (SARS-CoV) mediates cell fusion by binding to target cell surface receptors.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "It inhibited SARS-CoV-2 and SARS-CoV S-mediated cell–cell fusion with the IC50 of 0.65 and 0.85 μg/mL, respectively [54].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "Fig. 5: a–h Inhibitory activity of EK1C4 in cell–cell fusion mediated by the S proteins of SARS-CoV (a), MERS-CoV (b), HCoV-OC43 (c), HCoV-229E (d), HCoV-NL63 (e), WIV1 (f), Rs3367 (g) and SHC014 (h).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "For SARS-CoV S-, SL-CoV S-, OC43 S- or NL63 S-mediated cell–cell fusion assays, effector cells and target cells were cocultured in DMEM containing trypsin (80 ng/mL) for 4 h, while for SARS-CoV-2 and MERS-CoV S-mediated cell–cell fusion assays, effector cells and target cells were cocultured in DMEM without trypsin but 10% FBS for 2 h.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "This approach identified N-(2-aminoethyl)-1 aziridine-ethamine (NAAE) as a novel ACE2 inhibitor that also is effective in blocking the SARS-CoV spike protein-mediated cell fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "By using a molecular docking approach, an earlier study identified N-(2-aminoethyl)-1 aziridine-ethanamine as a novel ACE2 inhibitor that effectively blocks the SARS-CoV RBD-mediated cell fusion [24].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "An ACE2 inhibitor, N-(2-aminoethyl)-1 aziridine-ethanamine, has been identified as being effective in blocking SARS-CoV spike protein-mediated cell fusion [89].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000747\" ! \"conjugation with cellular fusion\")\n",
      "Identity and similarity in spike protein sequences of SARS-CoV and SL-CoVs and the effect of EK1 on HCoV S protein-mediated cell fusion and virus-cell fusion, as well as the viral loads and histopathological changes in brains of HCoV-infected mice.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0000910\" ! cytokinesis)\n",
      "The nucleocapsid protein of severe acute respiratory syndrome coronavirus (SARS-CoV) could modulate cell cytokinesis and proliferation via interacting with eEF1A [28].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0000910\" ! cytokinesis)\n",
      "It was demonstrated that the N protein of SARS-CoV induces aggregation of EF1α, inhibiting protein translation and cytokinesis by blocking F-actin bundling.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0001171\" ! \"reverse transcription\")\n",
      "However, direct clinical samples of high viral titer and a sensitive PCR amplification are required.Yields of viral RNA should be determined by quantitative RT-PCR, because spectrophotometric determination is prone to error as a result of low RNA quantity and interference by the carrier poly(A) RNA, which contributes to most of the RNA present.A prolonged denaturation step is used to remove secondary RNA structures in the SARS-CoV genome that impede reverse-transcription.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0001171\" ! \"reverse transcription\")\n",
      "A prolonged denaturation step is used to remove secondary RNA structures in the SARS-CoV genome that impede reverse-transcription.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In addition, SARS-CoV inactivated viruses induced adaptive immunity that protected against challenge (49) (50) (51) (52) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In addition, SARS-CoV inactivated viruses induced adaptive immunity that protected against challenge (49–52).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "SARS-CoV infection also actively suppressed the induction of adaptive immunity by impairing macrophage and dendritic cell functions [97] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In addition, SARS-CoV inactivated viruses induced adaptive immunity that protected against challenge (49) (50) (51) (52) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "In addition, SARS-CoV inactivated viruses induced adaptive immunity that protected against challenge (49–52).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "It is noteworthy that the most effective SARS-CoV structural protein that can induce neutralizing antibody production is the S-protein (Buchholz et al. 2004).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "Notably, the SARS-CoV virion itself was able to induce long-term antibody production even without an adjuvant.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "Moreover, the SARS-CoV S protein fragments: residues 318–510 (receptor-binding domain) [9], residues 485–625 [10], and residues 803–828 that fused with thioredoxin (Trx) or glutathione-S-transferase (GST) [11] were found to induce neutralizing antibody production in immunized animals.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "An exposed domain in the S1 subunit of SARS-CoV from amino acid residue 485-625 induced antibody production to inhibit viral infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "An exposed domain in the S1 subunit of SARS-CoV from amino acid residue 485–625 induced antibody production to inhibit viral infection (Zhou et al., 2004).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0002377\" ! \"immunoglobulin production\")\n",
      "A small fragment of the S protein of SARS-CoV linked to a human IgG1 Fc fragment was able to induce high titers of antibody production in mice that neutralized infection caused by SARS-CoV [43] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "However , the elevated transcriptional activation of genes coding for IRF-7 , IFN-beta , IFN-lambdas , and many ISGs detected at 48 hrs in infected 2B4 cells strongly argue for the activation of IRF-3 by SARS-CoV in 2B4 cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "Therefore , further work is needed to determine whether or not SARS-CoV is an activator of antiviral responses in these cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "Parallel to results on other coronaviruses , SARS-CoV has been shown to cause cytopathic effects in Vero cells and the formation of syncytia in lung tissues .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "The suppression of antiviral cytokines in non-immune cells by SARS-CoV is thought to buy time for dissemination of the virus in the host and a similar mechanism may be employed by CRCoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "SARS-CoV reproduces rapidly and causes massive cell damage .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "This finding suggests that SARS-CoV may cause direct damage to those cells , providing useful information for future study of disease pathogenesis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0005623\" ! cell)\n",
      "Inhibition of SARS-CoV mediated CPE in Vero E6 cells was graded into three levels + , + + and + + + ( where + represented least inhibition , and + + + represented most inhibition ) as shown in panels C , D , and E , respectively .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "Initially, SARS-CoV PLpro was shown to interfere with the activation of the transcription factors IRF3 (Devaraj et al., 2007) and NF-κB (Clementz et al., 2010, Frieman et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The transcription factors c-Fos, FosB, CREB-1, and ATF2 were all activated by the addition of SARS-CoV N protein to the sample well, with CREB-1 having the most dramatic increase in activity, about five times higher than the vector control (Fig. 3).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "Partial SARS-CoV cDNAs and antibiotic resistance/reporter gene DNA were generated and assembled in vitro to produce the replicon transcription template, which was then transcribed in vitro to generate the replicon RNA.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "Similarly, SARS-CoV infection stimulates robust ISG transcription and protein production; however, ISGs are delayed 24 to 48 h, permitting SARS-CoV to achieve peak titers prior to ISG production.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "However, SARS-CoV PLPro significantly inhibited IMQ-induced cytokine production through suppressing the activation of transcription factors IRF-3, NF-κB and AP-1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "SARS-CoV initiated viral gene transcription and protein synthesis in macrophages, but replication was abortive and no infectious virus was produced.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "Taken together, our findings did not support the model in which SARS-CoV N protein specifically modulates transcription of the FGL2 gene to cause fibrosis and vascular thrombosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The SARS-CoV N protein can activate AP-1, which is composed of homodimers and heterodimers of Fos, Jun, CREB, and activating transcription factor (ATF) subunits, in Vero and Huh7 cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The SARS-CoV N protein can activate AP-1, which is composed of homodimers and heterodimers of Fos, Jun, CREB, and activating transcription factor (ATF) subunits, in Vero and Huh7 cells (He et al. 2003).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "SARS-CoV 3b protein was shown to induce JNK/c-Jun/AP-1 activation to mediate transcription of MCP-1 [188].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The structural SARS-CoV N protein activates NF-B-driven transcription and upregulates the expression of IL-6 by facilitating the translocation of NF-B from cytosol to nucleus (Liao et al., 2005; Zhang et al., 2007) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "Severe acute respiratory syndrome coronavirus (SARS-CoV) and mouse hepatitis virus (MHV) induce DMVs for targeting their replication and transcription (Reggiori et al., 2010).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "To directly compare transcription induction of immediate early IFNs by MHV and SARS-CoV , we stably transformed murine L cells with a plasmid coding for the SARS-CoV receptor , resulting in a cell line ( L-ACE2 ) that supported infection with both MHV and SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "44 In vitro studies of various human cell types show that SARS-CoV efficiently suppresses transcription of IFN genes but selectively allows expression and translation of other genes .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The SARS-CoV Membrane (M) protein blocks transcription of IFN-I when stimulated by dsRNA or members of the RIG-I signaling pathway including RIG-I, MAVS, IKK-ɛ, and TBK1, but it does not influence the transcriptional activity of the IFN promoter when IRF3 or IRF7 are overexpressed.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "The SARS-CoV Membrane (M) protein blocks transcription of IFN-I when stimulated by dsRNA or members of the RIG-I signaling pathway including RIG-I, MAVS, IKK-, and TBK1, but it does not influence the transcriptional activity of the IFN promoter when IRF3 or IRF7 are overexpressed.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "SARS-CoV also inhibits the nuclear import of the related transcription factor IRF7 (Kuri et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "However, SARS-CoV PLPro significantly inhibited IMQ-induced cytokine production through suppressing the activation of transcription factors IRF-3, NF-κB and AP-1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "In addition, the SARS-CoV accessory protein ORF6 has been shown to antagonize the antiviral activity of the STAT1 transcription factor by sequestering IMPα/β1 on the rough ER/Golgi membrane (Frieman et al., 2007).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "In addition, the SARS-COV accessory protein (ORF6) has been shown to antagonize the antiviral activity of the signal transducer and activator of transcription (STAT) factor by sequestering IMPα/β1 on the rough ER/Golgi membrane (Frieman et al. 2007).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "For example, SARS-CoV infection was reported to disrupt cellular transcription to a larger extent than does HCoV-229E.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "For instance, SARS-CoV nsp1 inhibits mRNA translation and promotes host mRNA degradation by binding to the 40S ribosomal subunits (Kamitani et al., 2009), while nsp1 of MERS-CoV inhibits host mRNA translation and induces host mRNA cleavage without binding to the 40S ribosomal subunits (Lokugamage et al., 2015).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "The severe acute respiratory syndrome coronavirus (SARS-CoV) nsp1 protein has unique biological functions that have not been described in the viral proteins of any RNA viruses; expressed SARS-CoV nsp1 protein has been found to suppress host gene expression by promoting host mRNA degradation and inhibiting translation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "The nsp1 of SARS-CoV is the most studied among CoVs and is known to inhibit host gene expression via a two-pronged strategy )-i.e., translational shutoff through interaction with the 40 S ribosomal subunit, and host mRNA degradation through the recruitment of unidentified host nuclease(s) (Huang et al., 2011a; Kamitani et al., 2009; Lokugamage et al., 2012) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "The nsp1 of SARS-CoV is the most studied among CoVs and is known to inhibit host gene expression via a two-pronged strategy (Kamitani et al., 2009)—i.e., translational shutoff through interaction with the 40 S ribosomal subunit, and host mRNA degradation through the recruitment of unidentified host nuclease(s) (Huang et al., 2011a, Kamitani et al., 2009, Lokugamage et al., 2012).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "In addition to the translation suppression function, SARS-CoV nsp1 promotes host mRNA degradation by inducing endonucleolytic RNA cleavage in the 5′-UTR of host mRNAs (Huang et al., 2011b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "In addition, NSP1 protein of SARS-CoV could suppress host IFN-b gene expression by promoting host mRNA degradation and inhibiting translation (Narayanan et al. 2008) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "In addition to the translation suppression function, SARS-CoV nsp1 promotes host mRNA degradation by inducing endonucleolytic RNA cleavage in the 5 0 -UTR of host mRNAs (Huang et al., 2011b) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006402\" ! \"mRNA catabolic process\")\n",
      "nsp1 protein of severe acute respiratory syndrome coronavirus (SARS-CoV), a group 2b CoV, suppresses host gene expression by promoting host mRNA degradation and translation inhibition.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "44 In vitro studies of various human cell types show that SARS-CoV efficiently suppresses transcription of IFN genes but selectively allows expression and translation of other genes .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Similarly , SARS-CoV enhances its RNA translation via the Nsp1 protein.19 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Similarly, SARS-CoV enhances its RNA translation via the Nsp1 protein.19SARS-CoV-2 gains entry into the human cells by binding its spike protein to a membrane protein, Angiotensin-Converting Enzyme 2 (ACE2).20 However, the spike protein must first be cleaved at the S1/S2 and subsequently at the S2’ sites to enable binding to the ACE2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Similarly, SARS-CoV Nsp1 promotes mRNA downregulation as a consequence of global mRNA degradation in order to maximize viral RNA translation (Huang et al., 2011).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Similarly, SARS-CoV 97 enhances its RNA translation via the Nsp1 protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "It is unclear whether spike protein or 3a protein plays a major role in SARS-CoV induced translation shutoff, as the translational rate is not examined upon overexpression of these proteins.MHV strain A59 infection increases the level of phosphor-eIF2α and attenuates the host protein synthesis, resulting in preferential translation of ATF4.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV [84] [85] [86] [87] , MERS-coronavirus [88] , MHV [36, 89, 90] , transmissible gastroenteritis virus (TEGV) [91, 92] , porcine epidemic diarrhea virus (PEDV) [93] , bat coronaviruses [94] , have been shown to induce host translation shutoff in susceptible cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV initiated viral gene transcription and protein synthesis in macrophages, but replication was abortive and no infectious virus was produced.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV nsp1 protein induces degradation of host mRNA and suppression of translation both in nsp1-expressing cells and in SARS-CoV-infected cells [12].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "The large SARS-CoV RNA genome produces eight 3′-co-terminal, nested subgenomic mRNAs (sg-mRNAs) for the efficient translation of structural and accessory proteins (Masters, 2006, Snijder et al., 2003).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "The final sample of the SAIL SARS-CoV NP CTD was produced by using the Escherichia coli cell-free protein synthesis system, with optimizations for the production of labeled NMR samples.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "The final sample of the SAIL SARS-CoV NP CTD was produced by using the Escherichia coli cell-free protein synthesis system, with optimizations for the production of labeled NMR samples.9The 1H–13C constant time (CT) HSQC spectra of an aliphatic region were compared between uniformly labeled (UL) and SAIL samples under the same conditions.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS-CoV Nsp1 has been shown to induce a near-complete shutdown of host protein translation by a two-pronged strategy: first, it is able to bind the small ribosomal subunit and stall canonical mRNA translation at various stages during initiation (15, 16) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006508\" ! proteolysis)\n",
      "Similarly, Fang et al. reported that the SARS-CoV M protein physically interacted with Iκκ-β, suppressing IκB-α protein degradation and NF-κB activation (Fang et al., 2007).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006508\" ! proteolysis)\n",
      "This result suggested that 8ab might represent another strategy of facilitating viral protein folding to combat unfavorable cellular response during SARS-CoV replication.We found that 8ab could induce the proteolysis of wild-type ATF6 (Fig. 5B) and the Golgi retention of ATF6S1P− (Fig. 6A).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006508\" ! proteolysis)\n",
      "In this study, we demonstrated that 8ab protein of SARS-CoV stimulates the proteolysis of ATF6 (Fig. 5B) and the transition of the cleaved N-terminal domain from the ER to the nucleus (Fig. 5C).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006873\" ! \"cellular ion homeostasis\")\n",
      "Ca 2 þ transport through SARS-CoV E protein channel activates the NLRP3 inflammasome Alteration of cellular ion homeostasis by SARS-CoV E protein IC could have several implications.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006887\" ! exocytosis)\n",
      "In addition, SARS-CoV E protein decreased the levels and activity of ENaC in lung epithelial cells, via the activation of distinct PKC isoforms, decreasing both ENaC exocytosis and endocytosis rates (Ji et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006893\" ! \"Golgi to plasma membrane transport\")\n",
      "Also, the SARS-CoV 3a protein is necessary for Golgi to plasma membrane transport [15].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006900\" ! \"vesicle budding from membrane\")\n",
      "Electron microscopy of infected cells revealed a role for ORF 3a in SARS-CoV induced vesicle formation, a prominent feature of cells from SARS patients.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "In addition, although unclear as whether it is beneficial or detrimental to the host cell, SARS-CoV have been shown to activate host stress response, apoptosis, and autophagy (13).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Chen et al. [143] found that the PL pro +TM of SARS-CoV induces incomplete autophagy by interacting with LC3 and Beclin1 (two key autophagy regulators), thus negatively regulating antiviral immunity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Moreover, SARS-CoV ORF9b could localize to mitochondria and induce autophagy, which caused the MAVS signalosome degradation to further suppress antiviral transcriptional responses [70].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "In addition , although unclear as whether it is beneficial or detrimental to the host cell , SARS-CoV have been shown to activate host stress response , apoptosis , and autophagy ( 13 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Furthermore , there is accumulating evidence that MERS-CoV , SARS-CoV and SARS-CoV-2 inhibit autophagy [ [ 211 ] , [ 212 ] , [ 213 ] , [ 214 ] ] reviewed by [ 215 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006949\" ! \"syncytium formation\")\n",
      "Consistent to our previous report 23 , addition of trypsin triggered syncytium formation on 293/hACE2 cells by SARS-CoV S protein after 4 h incubation (Fig. 4b, c) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006949\" ! \"syncytium formation\")\n",
      "The SARS-CoV S protein caused syncytium formation at neutral pH [98] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Lung inflammation caused by SARS-CoV is likely a complex pathophysiological process involving inflammatory cytokines released from activated macrophages in alveoli, leading to immune system dysregulation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In this study, we investigated the different manifestations of lung inflammation caused by N protein of SARS-CoV in mice.The clinical manifestations, chest radiographic and pathological features of the patients with SARS are similar to those of acute lung injury (ALI) and acute respiratory distress syndrome (ARDS).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "SARS-Cov has been detected in the heart on autopsy 23 ( c ) Hypoxia ( due to SARS-Cov-2-induced ARDS ) can lead to inflammation , cell injury and subsequent cardiac damage 28 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Studies have reported that SARS-CoV can cause myocardial injury and lead to myocardial inflammation [8].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "The SARS coronavirus causes lung injury and inflammation in part through actions on the nonclassical renin angiotensin pathway .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "The finding in NLRP3 inflammasome-mediated activation of IL-1β and CXCL10 up-regulation induced by SARS-CoV SUD might be a valuable target in the development of new anti-SARS agents.In conclusion, SARS-CoV SUD significantly induced the CXCL10-mediated pulmonary inflammation in vitro and in vivo.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Regarding the lung, the differentiated Calu-3 cell line [when cultured at the air-liquid interface (ALI)] is the model of choice: in that set-up, SARS-CoV infection triggered an inflammatory response characterized by increased production of interleukin (IL)-6, IL-8, gamma interferon (IFN-γ), inducible protein 10 (IP-10), and activation of the transcription factor NF-κB (56).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In conclusion, SARS-CoV SUD significantly induced the CXCL10-mediated pulmonary inflammation in vitro and in vivo.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "However, at late times (Ͼ24 h postinfection), SARS-CoV produced robust changes in several host pathways, including cytokine responses, inflammation, and extracellular activity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "28 Such a process has long been known to be sufficient to induce respiratory symptoms (eg, airflow obstruction and increased airway hyperreactivity) and inflammation (eg, expression and secretion of pro-inflammatory cytokines).29 It is apparent, therefore, that the elevated ACE2 expression in lung tissue plays a significant role in the pathogenesis of the acute outcome of infection caused by SARS-CoV and SARS-CoV-2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In contrast to the mild infection of the upper respiratory tract seen in HCOV-229E and HCOV-OC43 infected patients, SARS-COV is pathogenic and may cause severe inflammation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Corticosteroids SARS-CoV and MERS-CoV both cause lung inflammation and can progress into severe respiratory syndrome.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "SARS-CoV and MERS-CoV both cause lung inflammation and can progress into severe respiratory syndrome.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Therefore, SARS-CoV can mediate myocardial inflammation and myocardial ACE2 system downregulation-related damage.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In contrast, the SARS-CoV E protein seems to activate p38 and induces inflammation [69] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Fatal cases of SARS-CoV infection similarly show significant lung damage in response to inflammation, which may very well be driven by necroptosis.The SARS-CoV may have evolved to target Rip3 and the necroptotic pathway for a variety of reasons.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "1 Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) has many similarities to COVID-2019 and has been shown in an animal model to mediate myocardial inflammation and damage through down-regulation of myocardial ACE2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Therefore, due to its poly-pharmacology, fenretinide administration by pulmonary formulations may be expected to be protective against acute lung injury (ALI)/ acute respiratory distress syndrome (ARDS) caused by SARS-CoV infection and could represent a useful tool in a multimodal therapy aimed at establishing a rapid anti-inflammatory and antiviral effect.Pulmonary hyper inflammation is a common feature in COVID-19 patients.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "IFV-A and SARS-CoV infection induce inflammation in vivo117, 118.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "This experiment testified that a low-dose of SARS-CoV specific ssRNA was able to induce excessive inflammatory response and acute lung injury in vivo and lead to a high mortality rate.A great deal of clinical and experimental evidence associated with the hypercytokinemia and systemic immunopathology has proved that a progressive immune-associated injury results in the severe acute respiratory syndrome.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Moreover, an experimental animal model of SARS-CoV infection showed that type I IFNs were induced late and triggered a robust inflammatory response with high TNF, IL-6, and MCP-1 levels causing dramatic lethality (Channappanavar et al, 2016) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Abstract SARS-associated coronavirus (SARS-CoV) causes inflammation and damage to the lungs resulting in severe acute respiratory syndrome.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Interestingly, infection of mice with a set of viruses lacking or displaying E protein IC activity, revealed that the activation of inflammasome pathway, and the exacerbated inflammatory response induced by SARS-CoV was decreased in infections by on channel deficient viruses.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Additionally, elimination of the E gene diminishes inflammation induced by SARS-CoV through the NF-κB pathway [27].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Primary SARS-CoV infection produced inflammation and the appearance of lung immune cells primarily surrounding small-to-medium bronchial airways at 7 days following challenge (Figure 3 B) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Binding of the SARS-CoV to the angiotensin-converting enzyme 2 (ACE-2) receptors in the type II pneumocytes in the lungs triggers a cascade of inflammation in the lower respiratory tract [20].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "As a newly emerging coronavirus in the human, SARS-CoV has led to acute inflammation and a lethal syndrome in patients.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "15 16 Furthermore, SARS-CoV has also been found to mediate myocardial inflammation and damage associated with downregulation of the myocardial ACE2 system.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Interestingly, alveolar macrophages, through a mechanism dependent on TLR3 engagement, inhibit RDC activation during SARS-CoV infections, which in turn results in lymphopenia and prolongation of virus-induced inflammation [19•].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been reported that hepatitis and AIDS patients showed a similar acute inflammatory syndrome [ 28 ] [ 29 ] [ 30 ] in clinic indicate robust expression of COX-2 , thus raising the possibility that SARS-CoV may contribute to inflammation through induction of COX-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our finding may provide clue for the induction of inflammation by SARS-CoV and cast insight into the severity of the SARS epidemic .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In addition, SARS-CoV was able to mediate myocardial inflammation and damage associated with the down-regulation of the myocardial ACE2 system, which may be responsible for myocardial dysfunction and adverse cardiac outcomes in patients with SARS [44].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "The levels of pro-inflammatory cytokines were increased , which indicates that SARS-CoV also results in the inflammation [ 42 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "E protein has a major role in inflammasome activation and the associated exacerbated inflammation elicited by SARS-CoV in the lung parenchyma .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Unlike SARS-CoV-2, SARS-CoV has been shown to enter the brain earlier at 3dpi but does not elicit notable inflammation in this secondary site of infection 31.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "E protein has a major role in inflammasome activation and the associated exacerbated inflammation elicited by SARS-CoV in the lung parenchyma (Nieto-Torres et al., 2014).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Additionally, elimination of the E gene diminishes inflammation induced by SARS-CoV through the NF-kB pathway [27] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "SARS-CoV infection induces an uncontrolled inflammatory response leading to pulmonary damage, edema accumulation, severe hypoxemia and eventually death.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In contrast to the mild infection of the upper respiratory tract seen in HCOV-229E and HCOV-OC43 infected patients , SARS-COV is pathogenic and may cause severe inflammation .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "In fact, E protein is responsible in a significant proportion of the inflammasome activation and the associated inflammation elicited by SARS-CoV in the lung parenchyma.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Unlike other known coronaviruses, both MERS- and SARS-CoV cause inflammation of the upper respiratory tract and spread down to the lower respiratory tract in most cases, resulting in deadly lung damage and death.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "A previous study also demonstrated that SARS-CoV and SARS-CoV-2 induce inflammation ( Malavolta et al ., 2020 ) , and symptoms of the most severe patients can be controlled by low-dose glucocorticoids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "D Significant differences between adult groups and young groups, with p < 0.05.Our goal was to examine the changes in the early stage of lung inflammation caused by N protein of SARS-CoV among different age groups.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our finding may provide clue for the induction of inflammation by SARS-CoV and cast insight into the severity of the SARS epidemic.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Our finding may provide clue for the induction of inflammation by SARS-CoV and cast insight into the severity of the SARS epidemic.SARS coronavirus (SARS-CoV) is the etiological agent of severe acute respiratory syndrome (SARS), an acute pulmonary syndrome that, when it emerged in the winter of 2002-2003, resulted in the death of approximately 800 individuals, close to 10% of those infected [1] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It has been reported that hepatitis and AIDS patients showed a similar acute inflammatory syndrome [28] [29] [30] in clinic indicate robust expression of COX-2, thus raising the possibility that SARS-CoV may contribute to inflammation through induction of COX-2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It was reported that SARS-CoV infection of ferrets caused mild liver inflammation and that the liver damage became much more serious if the ferrets were immunized with vaccinia virus-based SARS vaccines before virus challenge [35].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Baricitinib is now in 702 clinical trials for SARS-CoV in a patient population as it has proven to reduce inflammation and 703 inhibit viral entry (Richardson et al., 2020b).704 5.2 Structure-based drug design 705 Studies to date indicate that drugs that have a similar grouping to chloroquine and 706 hydroxychloroquine, may have some activity against SARS-CoV-2 (Hu and Bajorath, 2012).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Baricitinib is now in clinical trials for SARS-CoV in a patient population as it has proven to reduce inflammation and inhibit viral entry (Richardson et al., 2020b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Animal models and some human data of the closely related SARS-CoV infection have demonstrated early corticosteroid use may reduce initial inflammatory response, but raise the concern of increased viral replication and shedding that could worsen clinical outcomes and increase viral transmission [67, 68] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n",
      "However, we have not found alterations in haptoglobin or the presence of schistocytes to date.A related virus, SARS-CoV, promotes complement activation, increasing C5a levels and contributing to hyperinflammation [11] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n",
      "Another group demonstrated that SARS-CoV could trigger complement activation through the lectin pathway (37).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0006956\" ! \"complement activation\")\n",
      "As a result of defective innate immunity and robust complement activation induced by SARS-CoV, aged cases experience a severe progressing pneumonitis [89,90].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "In the coronavirus family, infectious bronchitis virus (IBV) induces a G2/M-phase arrest in infected cells that favors viral replication (Dove et al., 2006), while mouse hepatitis virus (MHV) replication and some severe acute respiratory syndrome coronavirus (SARS-CoV) proteins induce cell cycle arrest at the G0/G1 phase (Chen and Makino, 2004, Yuan et al., 2005).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Herpesviruses (Flemington, 2001), severe acute respiratory syndrome coronavirus (SARS-CoV) protein (Yuan et al., 2005), influenza A virus and its NS1 protein (Jiang et al., 2013), human respiratory syncytial virus and murine norovirus (MNV) (Davies et al., 2015) can induce cell cycle arrest in the G0/G1 phase.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Infection by the coronaviruses TGEV, MHV, SARS-CoV, and IBV was reported to cause a cell cycle arrest in the G0/G1 phase and a number of cellular proteins (e.g., cyclin D3 and hypophosphorylated restinoblastoma protein) and viral proteins (MHV nsp1, SARS-CoV 3b 7a, and N proteins) have been proposed to be involved in the cell cycle arrest in G0/G1.Many viruses encode proteins that modulate apoptosis and, more generally, cell death, which allows for highly efficient viral replication or the establishment of persistent infections.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Recently, it was reported that the SARS-CoV N protein induced cell cycle arrest in the S phase through down-regulation of the S phase gene products expression (Surjit, 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Infectious bronchitis virus (IBV) induces an S and G2/M-phase arrest to favor viral replication (Dove et al., 2006; Li et al., 2007), and mouse hepatitis virus (MHV) (Chen and Makino, 2004) and some severe acute respiratory syndrome coronavirus (SARS-CoV) proteins induce cell cycle arrest in G1 phase (Yuan et al., 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Infectious bronchitis virus (IBV) induces an S and G2/M-phase arrest to favor viral replication (Dove et al., 2006; Li et al., 2007) , and mouse hepatitis virus (MHV) (Chen and Makino, 2004) and some severe acute respiratory syndrome coronavirus (SARS-CoV) proteins induce cell cycle arrest in G0/G1 phase (Yuan et al., 2005 (Yuan et al., , 2006 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007050\" ! \"cell cycle arrest\")\n",
      "Infectious bronchitis virus (IBV) induces an S and G2/M-phase arrest to favor viral replication (Dove et al., 2006; Li et al., 2007), and mouse hepatitis virus (MHV) (Chen and Makino, 2004) and some severe acute respiratory syndrome coronavirus (SARS-CoV) proteins induce cell cycle arrest in G0/G1 phase (Yuan et al., 2005, 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0007567\" ! parturition)\n",
      "Recent reports suggest that a maternal infection with the severe acute respiratory syndrome ( SARS ) virus can result in preterm birth as well [ 132 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0008217\" ! \"regulation of blood pressure\")\n",
      "However, from genomic and structural analyses, it has been reported that the SARS-CoV-2 has a similar receptor-binding-motif as SARS-CoV, which allows it to infect host cells via the angiotensin-converting-enzyme-2 (ACE2).30 Interestingly, the renin-angiotensin system (RAS), apart from its well-known endocrine role in blood pressure regulation, also has complicated autocrine functions within specific tissues.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Further study to confirm the S2-induced arrest at G1/S phase will be indicative to understand the role of S2 in the pathogenesis of SARS-CoV.It has been shown that SARS-CoV can induce cell proliferation arrest (Mizutani et al., 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "For example, SARS-CoV nsp16 acts as a 2′-O-MTase to prevent innate immune recognition and promote viral proliferation [113,121,123].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "The nucleocapsid protein of severe acute respiratory syndrome coronavirus (SARS-CoV) could modulate cell cytokinesis and proliferation via interacting with eEF1A [28].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Inhibition of cell proliferation by SARS-CoV infection in Vero E6 cells.Severe acute respiratory syndrome (SARS) is a newly discovered infectious disease caused by a novel coronavirus, SARS coronavirus (SARS-CoV) (Marra, 2003; Rota, 2003).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Inhibition of cell proliferation by SARS-CoV infection in Vero E6 cells.Severe acute respiratory syndrome (SARS) is caused by SARS-coronavirus (SARS-CoV).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "Infection of Vero E6 cells with SARS-CoV inhibits cell proliferation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "The nucleocapsid protein of SARS-CoV interacted with EF1A and inhibited cell proliferation (Zhou et al., 2008).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "It has been shown that SARS-CoV can induce cell proliferation arrest ( Mizutani et al ., 2006 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0008283\" ! \"cell population proliferation\")\n",
      "This downregulation of Akt phosphorylation induces inhibition of cell proliferation by SARS-CoV infection and weak activation of Akt cannot induce escape from SARS-CoV-induced apoptosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009293\" ! transduction)\n",
      "Since the SARS-CoV nucleocapsid was shown to activate the AP-1 transduction pathway [26] , it will be of interest to show whether there may be a mechanistic link between PDCD4 suppression and enhancement of AP-1 tranactivation in SARS-CoV-infected cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009293\" ! transduction)\n",
      "A recent study showed that the binding affinity of SARS-CoV-2 to ACE2 is comparable to that of SARS-CoV, as plasma containing antibodies to the SARS-CoV reduced the transduction efficiency of SARS-CoV-2 in cells (5).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009293\" ! transduction)\n",
      "Similarly, the immunization of mice and rabbits with inactivated SARS-CoV induced antibodies that specifically recognize the RBD, block S1-RBD interaction, and prevent cell transduction with S protein-pseudotyped HIV-1 (He et al., 2004b) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "In a study that took the opposite approach to assessing accessory protein function, it was discovered that engineered insertion of gene 6 of SARS-CoV greatly enhanced the virulence of an attenuated variant of MHV (Pewe et al., 2005) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "This newly emerged virus has much greater similarity to SARS-CoV than to MERS-CoV, thus both SARS-CoV and SARS-CoV-2 may cause pathogenesis through similar mechanisms.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "To identify domains within the SARS-CoV nsp1 protein that could contribute to virulence, we compared the sequence to that of the MHV nsp1 ( Fig 8A) , with the goal of identifying conserved regions that could be functionally important.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "CR1 is associated with the pathogenesis caused by SARS-CoV [32], adenoviruses [33] and other viral infections such as HIV and HCV [30].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "SARS-CoV ORF 6 protein can enhance the virulence of attenuated murine coronavirus (MHV) [61], as well as stimulate cellular DNA synthesis [62].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "One is immune pathogenesis, in which the damage of alveolar cells and bronchiolar epithelium could be caused by SARS-CoV specific cytotoxic T-cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Administration of 5H10 significantly suppressed the SARS-CoV–induced pathogenesis in the SARS Rhesus model.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Furthermore, an additional study showed that the pathogenesis caused by the SARS-CoV E protein is connected to its ion channel (IC) activity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "For example, SARS-CoV nsp1 suppresses the expression of host genes including type I IFN, thereby counteracts the host innate immune response and contributes to virulence.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "8 The accessory ORFs likely influence the pathogenesis of the SARS-CoV, as the accessory ORFs of other coronaviruses contribute to in vivo pathogenesis but are not essential for growth in vitro ( Figure 1 ).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "This newly emerged virus has much greater similarity to SARS-CoV than to MERS-CoV , thus both SARS-CoV and SARS-CoV-2 may cause pathogenesis through similar mechanisms .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "SARS-CoV research prompted the generation of novel animal models that have provided insight into (1) genetic changes in the SARS-CoV genome that modulate respiratory pathogenesis [4] ; (2) the impact of SARS-CoV on host innate and adaptive immune responses [5, 6] ; (3) the role of host genes in regulating SARS-CoV pathogenesis in mice [4, 7] ; and (4) novel strategies for the development of vaccines and therapeutic countermeasures [3] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "It was postulated that SARS-CoV may promote severe acute lung injury pathogenesis through increased AngII production and functional alterations of the reninangiotensin system, and the lung failure can be rescued by inhibition of AT1R (Kuba et al., 2005) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "As the extensive outbreak of SARS has not recurred since 2003, the mechanisms by which SARS-CoV infection causes the pathogenesis and host immune responses has been investigated using adequate animal infectious models (Roberts et al., 2005, Roberts et al., 2007, Nagata et al., 2008, Zhao et al., 2009, Zhao and Perlman, 2010).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "As the extensive outbreak of SARS has not recurred since 2003, the mechanisms by which SARS-CoV infection causes the pathogenesis and host immune responses has been investigated using adequate animal infectious models (Roberts et al., 2005 Nagata et al., 2008; Zhao et al., 2009; Zhao and Perlman, 2010) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "3 The public evidence shows that COVID-19 shares similar pathogenesis with the pneumonia induced by SARS-CoV or MERS-CoV.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "C3 activation products (C3 fragments C3a, C3b, iC3b, C3dg, and C3c) were detected by Western blotting in lung tissue of SARS-CoV MA15-infected mice, but not in control mice, as early as 1 day postinfection (dpi) (Fig. 1B), confirming that SARS-CoV MA15 infection activates the complement pathway.To test the importance of the complement signaling pathway in SARS-CoV pathogenesis, we infected C3–/– mice and C57BL/6J controls with SARS-CoV MA15.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "E protein may represent a novel strategy used by SARS-CoV to increase its virulence and may also serve as a potential therapeutic target in outbreaks of SARS-CoV or other coronaviruses.rSARS-CoV and rSARS-CoV-ΔE were rescued from infectious cDNA clones as previously described [26], [101].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "E protein may represent a novel strategy used by SARS-CoV to increase its virulence and may also serve as a potential therapeutic target in outbreaks of SARS-CoV or other coronaviruses.Virus rSARS-CoV and rSARS-CoV-DE were rescued from infectious cDNA clones as previously described [26, 101] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Infection of T cells may play a role in controlling the pathogenesis elicited by both MERS-CoV and SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Thus, the SARS-CoV nsp1 is unique among coronaviruses [7] and could contribute to the exceptional pathogenesis of SARS-CoV in humans [1], although direct demonstration of an nsp1 effect on virulence is yet to be shown.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Previous studies have shown that SARS-CoV mutated over the 2002e2004 epidemic to better bind to its cellular receptor and replication in human cells, enhancing virulence.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "SARS-CoV N protein possesses multifarious activities and may be actively involved in SARS-CoV induced pathogenesis [14].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Infection of T cells may play a role in controlling the pathogenesis elicited by both MERS-CoV and SARS-CoV (Chen et al., 2010).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "In contrast, SARS-CoV D130A mutants were found to have restored virulence in MDA5−/− mice in terms of weight loss, histology, and breathing function (Menachery et al., 2014b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Such a method was recently successfully used to identify critical components of SARS-CoV induced pathogenesis following infection of mice [20••].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "In these experiments, Serpine1, which regulates the deposition of fibrin after lung damage, is shown to have regulatory control over lung pathogenesis in the MA15 mouse model of SARS-CoV.In 2004, Hogan et al. showed that mice deficient in the interferon-activated transcription factor Signal Transducer and Activator of Transcription 1 (STAT1) were much more susceptible to pathogenesis caused by SARS-CoV (Hogan et al., 2004).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "In contrast, SARS-CoV D130A mutants were found to have restored virulence in MDA5−/− mice in terms of weight loss, histology, and breathing function .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Therefore, Egr-1 plays a critical mediator in SARS-CoV PLpro induced pathogenesis, as a potential therapeutic target to prevent the pulmonary fibrosis induced by SARS-CoV.How to cite this article: Li, S.-W.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Mouse models of SARS-CoV and MERS-CoV infection allow us to investigate disease pathogenesis and vaccine applications and to evaluate antiviral drugs and other therapies.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "A human coronavirus called Severe acute respiratory syndrome coronavirus (SARS-CoV) causes severe acute respiratory syndrome (SARS) and can cause unique pathogenesis ranging from lower and upper respiratory tract infections (Cheng et al., 2007) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "In addition, it has been shown that SARS-CoV gene 6 accelerates murine coronavirus infections (Tangudu et al., 2007) and enhances virulence of an attenuated murine hepatitis virus (MHV) (Pewe et al., 2005).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Alanine substitution of the motif I residues (AA-E-D, four 26 nucleotide substitutions) in murine hepatitis virus (MHV) and SARS-CoV yields viable mutants 27 with impaired replication and fitness, increased mutation rates, and attenuated virulence in vivo.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "SARS-CoV with a mutated nsp16 displays reduced virulence that is dependent on MDA5 sensing (Menachery et al., 2014b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Introduction of homologous substitutions in mouse-adapted SARS-CoV conferred resistance to GS-5734 similar to that seen in MHV but also attenuated in vivo pathogenesis of SARS-CoV in a mouse model.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0009405\" ! pathogenesis)\n",
      "Introduction of the MHV resistance mutations into SARS-CoV resulted in the same in vitro resistance phenotype and attenuated SARS-CoV pathogenesis in a mouse model.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0009606\" ! tropism)\n",
      "Also, the SARS-CoV-2 Spike may interact with other receptors such as DC-SIGN and DC-SIGNR as in SARS-CoV to increase tropism 16 and viral spread.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0016246\" ! \"RNA interference\")\n",
      "In addition, we reported for the first time the existence of complemented palindromic small RNAs (RNAs) and proved that one cpsRNA from a 22-bp DNA complemented palindrome in the SARS-CoV genome could induced RNAi (Liu C. et al., 2018).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0016246\" ! \"RNA interference\")\n",
      "Swift detailed characterization of the SARS-CoV genome enabled rapid development of potentially therapeutic anti SARS-CoV RNAi activators (Figure 9.5).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0016246\" ! \"RNA interference\")\n",
      "For example, the accessory proteins 4a and 4b, and possibly also M and accessory protein 5 of MERS-CoV, exhibit interferon antagonist activities, and SARS-CoV N acts as a viral suppressor of RNA silencing and suppresses RNA interference triggered by either short hairpin RNAs or siRNAs [175] [176] [177] [178] (TABLE 2) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0016246\" ! \"RNA interference\")\n",
      "In a similar fashion to SARS-CoV, the N protein of COVID-19 may play an important role in suppressing the RNA interference (RNAi) to overcome the host defense.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0016246\" ! \"RNA interference\")\n",
      "In a similar fashion to SARS-CoV, the N protein of COVID-19 may play an important role in suppressing the RNA interference (RNAi) to overcome the host defense [50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0016570\" ! \"histone modification\")\n",
      "SARS-CoV can inhibit IRF3 nuclear translocation, whereas MERS-CoV can impede histone modification [56] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0019835\" ! cytolysis)\n",
      "During the first week ( phase I ) , SARS-CoV replicates rapidly and induces cytolysis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0019835\" ! cytolysis)\n",
      "During the first week (phase I), SARS-CoV replicates rapidly and induces cytolysis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0030316\" ! \"osteoclast differentiation\")\n",
      "In conclusion, SARS-CoV 3a/X1 potentially promotes osteoclastogenesis by accelerating osteoclast differentiation from precursor monocytes/macrophages, and also indirectly through TNF-α and the RANK/RANKL system.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0030316\" ! \"osteoclast differentiation\")\n",
      "4: Augmentation of osteoclast differentiation by SARS-CoV 3a/X1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0030316\" ! \"osteoclast differentiation\")\n",
      "In the present study, we demonstrate that SARS-CoV 3a/X1 enhances osteoclast differentiation in a murine osteoclast precursor cell line, RAW264.7.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0030316\" ! \"osteoclast differentiation\")\n",
      "In conclusion, SARS-CoV 3a/X1 potentially promotes osteoclastogenesis by accelerating osteoclast differentiation from precursor monocytes/macrophages, and also indirectly through TNF-a and the RANK/RANKL system.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0031982\" ! vesicle)\n",
      "SARS-CoV induces the production of double-membrane vesicles that lack PRRs and can then replicate in these vesicles [ 18 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0031982\" ! vesicle)\n",
      "SARS-CoV induces formation of double-membrane vesicles on an altered endoplasmic reticulum , as revealed by electron tomography images of infected cells [ 45 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0031982\" ! vesicle)\n",
      "SARS-CoV and MERS-CoV trigger the formation of vesicles that lack PRRs , preventing the dsRNA recognition by host PRRs [ 31 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0031982\" ! vesicle)\n",
      "However , SARS-CoV and MERS-CoV can induce the production of double-membrane vesicles that lack PRRs and then replicate in these vesicles , thereby avoiding the host detection of their dsRNA [ 39 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0031982\" ! vesicle)\n",
      "However , SARS-CoV induces the generation of double-membrane vesicles where replication occurs .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0032637\" ! \"interleukin-8 production\")\n",
      "Their sera enhanced SARS-CoV–induced MCP1 and IL-8 production by human monocyte–derived wound-healing macrophages, whereas blockade of FcγR reduced such effects.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0034976\" ! \"response to endoplasmic reticulum stress\")\n",
      "In addition, overexpression of SARS-CoV S was shown to induce a potent ER stress response, and the re-arrangement of DMV production and membrane alteration for the assembly of the virus may contribute to endoplasmic reticulum (ER) stress during CoV infection [40].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "HTVS, SP, and XP step-wise screening protocol were followed to find potent compounds having higher docking scores and binding energy as follows.In a SARS-CoV replicon assay, it has been shown that the specific inhibition of SARS-CoV helicase by SSYA10-001 blocked viral replication [90].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Animal models and some human data of the closely related SARS-CoV infection have demonstrated early corticosteroid use may reduce initial inflammatory response, but raise the concern of increased viral replication and shedding that could worsen clinical outcomes and increase viral transmission [67, 68] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Complete inhibition of the cytopathic effects of SARS-CoV in culture was observed for IFN subtypes β-1b, α-n1, α-n3, and human leukocyte IFN-α.22 In experimentally infected cynomolgus macaques with SARS-CoV, prophylactic treatment with pegylated IFN-α significantly reduced viral replication and excretion, viral antigen expression by type 1 pneumocytes, and pulmonary damage, compared with untreated macaques, whereas postexposure treatment with pegylated IFN-α yielded intermediate results.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Given that GILT plays a crucial role in unfolding internalized viral glycoproteins and affecting the environment in the lysosome [22, 27] , an cellular compartment utilized by SARS-CoV, EBOV and many other viruses to invade into the cytoplasm of host cells and initiate viral replication [26, [28] [29] [30] , it is conceivable that GILT may affect the infectious entry of this group of viruses.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Given that GILT plays a crucial role in unfolding internalized viral glycoproteins and affecting the environment in the lysosome [22,27], an cellular compartment utilized by SARS-CoV, EBOV and many other viruses to invade into the cytoplasm of host cells and initiate viral replication [26,28–30], it is conceivable that GILT may affect the infectious entry of this group of viruses.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Therefore, the efficacy of a SARS vaccine can only be judged by quantitative virology in this model.Hamsters are an excellent model for SARS-CoV infection and vaccine efficacy since they support high levels of viral replication and histopathological changes in respiratory tissues, and demonstrate clinical signs (reduced activity) following intranasal inoculation (6, 11, 15) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Collectively, these data show that ACE2 contributes substantially to the efficiency of SARS-CoV replication.We show here that ACE2 can be immunoprecipitated from Vero E6 cells by the S1 domain of the SARS-CoV S protein, that it mediates fusion with S-protein-expressing cells, and that it promotes viral replication in a cell line otherwise inefficient for SARS-CoV infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "MAb 201 is a severe acute respiratory syndrome-associated coronavirus (SARS-CoV)-specific MAb that prevents establishment of viral replication in vitro and prevents viral replication in vivo when administered prophylactically.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Complete inhibition of the cytopathic effects of SARS-CoV in culture was observed for IFN subtypes β-1b, α-n1, α-n3, and human leukocyte IFN-α.22 In experimentally infected cynomolgus macaques with SARS-CoV, prophylactic treatment with pegylated IFN-α significantly reduced viral replication and excretion, viral antigen expression by type 1 pneumocytes, and pulmonary damage, compared with untreated macaques, whereas postexposure treatment with pegylated IFN-α yielded intermediate results.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "The use of a Dz to target the 5′-untranslated region (5′-UTR) of a highly conserved fragment in the SARS-CoV genome to suppress viral replication has been reported.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "In vitro and in vivo approaches by multiple groups identified mAbs targeting either SARS-CoV or MERS-CoV that inhibited viral replication and ameliorated SARS and MERS disease in animal models [74,78,89,113].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "For SARS-CoV and MERS-CoV, Type I IFNs were effective at decreasing viral replication in vitro and showed additional benefits in in vivo primate models of infection [103,104,107,126,127].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Macaques also serve as a preclinical model for Middle East respiratory syndrome (MERS)-CoV, and similar to SARS-CoV, IFN-α in combination therapy with ribavirin reduces viral replication and severe histopathological changes (64).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0039694\" ! \"viral RNA genome replication\")\n",
      "Pre-exposure of SARS-CoV to UDA was shown to block the viral replication in a dose-dependent manner (Fig. 5A).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0042060\" ! \"wound healing\")\n",
      "SARS-CoV infection induced a range of interferon, cytokine, and pulmonary wound-healing genes, as well as several genes associated with the onset of ARDS.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0042783\" ! \"evasion of host immune response\")\n",
      "Of these miRNAs, miR-17*, -574-5p, and -214 suppress SARS-CoV replication and Ivyspring International Publisher contribute to immune evasion in bronchoalveolar stem cells (BASC) [9] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0042783\" ! \"evasion of host immune response\")\n",
      "The authors conclude that ORF 6 protein-mediated inhibition of the expression of STAT1-transcribed, IFN response genes, which is critical for the host to establish the antiviral state, could contribute to the immune evasion by SARS-CoV (Frieman et al., 2007b).7a and 7b accessory proteins are produced from the bicistronic subgenomic RNA 7 of SARS-CoV (Marra et al., 2003, Rota et al., 2003).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0043491\" ! \"protein kinase B signaling\")\n",
      "On the other hand, activation of p38 MAPK, AP-1 and Akt signal transduction pathway by SARS-CoV [Mizutani et al., 2004b,c] may be correlated with inflammatory responses.In contrast to SARS-CoV, we recognized abundant induction of SOCS3 in RSV infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044350\" ! micropinocytosis)\n",
      "SARS-CoV and MHV-CoV can induce continuous micropinocytosis, but this occurs in the later phase during infection and is not associated with virus entry [31].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "The large number of hosts and identification of the viral strains from different hosts make SARS-CoV an excellent model system for understanding host ranges and cross-species infections of emerging animal viruses.The envelope-anchored trimeric spike protein mediates coronavirus entry into host cells by binding to its host receptor and subsequently fusing host and viral membranes (Bosch et al., 2003, Gallagher and Buchmeier, 2001).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "The spike (S) protein of the severe acute respiratory syndrome coronavirus (SARS-CoV) interacts with cellular receptors to mediate membrane fusion, allowing viral entry into host cells; hence it is recognized as the primary target of neutralizing antibodies, and therefore knowledge of antigenic determinants that can elicit neutralizing antibodies could be beneficial for the development of a protective vaccine.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "Abstract The Spike (S) protein of SARS-coronavirus (SARS-CoV) mediates viral entry into host cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "Biophysical characterization of HRC peptide analogs interaction with heptad repeat regions of the SARS-coronavirus Spike fusion protein core.The Spike (S) protein of SARS-coronavirus (SARS-CoV) mediates viral entry into host cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "Indeed, a recent study showed that SARS-CoV RBD-induced mouse sera bind SARS-CoV-2 RBD with high affinity, but poorly neutralize SARS-CoV-2 pseudovirus entry into host cells; in contrast, the same sera bind SARS-CoV RBD with high affinity and neutralize SARS-CoV pseudovirus entry potently (44).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "Viral entry into host target cells appears to be mediated by SARS-CoV Spike (S) glycoprotein and is dependent on angiotensin-converting enzyme 2 (ACE2) as the functional receptor [6].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044409\" ! \"entry into host\")\n",
      "Viral entry into host target cells appears to 0264 be mediated by SARS-CoV Spike (S) glycoprotein and is dependent on angiotensin-converting enzyme 2 (ACE2) as the functional receptor [6] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044546\" ! \"NLRP3 inflammasome complex assembly\")\n",
      "Together, these data provide evidence that the ion channel activity of the SARS-CoV 3a protein is essential for triggering the NLRP3 inflammasome.Both K + Efflux and ROS Production Are Involved in the IL-1β Release Induced by the SARS-CoV 3a ProteinFinally, we investigated the mechanism by which SARS-CoV 3a triggers NLRP3 inflammasome activation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044546\" ! \"NLRP3 inflammasome complex assembly\")\n",
      "SARS-CoV 3a was sufficient to cause the NLRP3 inflammasome activation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0044546\" ! \"NLRP3 inflammasome complex assembly\")\n",
      "Particularly, ORF3 and ORF8 in SARS-CoV-2 are highly divergent from ORF3a and ORF8b in SARS-CoV that are known to induce NLRP3 inflammasome activation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "On the other hand, SARS-CoV stimulated the host innate immunity by upregulating genes including defensins [24,25] and eosinophil derived neurotoxin [26,27], similar to that of bacterial infections (Fig. 2c).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "It is important to clarify the influence of SARS-CoV upon the IFN system, namely suppression of IFN production and counteraction of IFN signaling by SARS-CoV infection, because many viruses have evolved a variety of strategies to counteract the antiviral effect of IFN.The aim of this study is to analyze the innate immune response induced by SARS-CoV infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Apparently, SARS-CoV protects itself from interferon-triggered innate immune response by interfering with ISG15 or ubiquitin conjugation pathways.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Modulation of innate immune response by SARS-CoV : evasion of innate immune detection Evasion of innate immune responses to SARS-CoV infection requires avoidance of detection by cellular PRRs .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Reminiscent to DENV, Shi and colleagues reported that severe acute respiratory syndrome-coronavirus (SARS-CoV) ORF-9b protein induces mitochondria elongation through the proteasomal degradation of DRP1, and impairs innate immunity by targeting MAVS-dependent signaling .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Nucleocapsid ( N ) protein of severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) is a major pathological determinant in the host that may cause host cell apoptosis , upregulate the proinflammatory cytokine production , and block innate immune responses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Importantly, the SARS-CoV macrodomain suppressed the innate immune response during infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "SARS-CoV and MERS-CoV can suppress the innate immune response through multiple interference strategies with intracellular signaling pathways .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "28 ,  29  In addition , SARS-CoV and possibly SARS-CoV-2 can suppress innate immune responses via counteracting NF-kappaB and Type I IFN signaling without triggering the anti-viral machinery .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Particularly, SARS-CoV nsp1 inhibits innate immune response by translational repression of IFN mRNA transcripts, hence altering IFN production and signaling (Narayanan et al., 2008a; Tanaka et al., 2012) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Recently, the crystal structure of transmissible gastroenteritis virus (TGEV) PL1 has been solved, which was similar to that of SARS-CoV PL2 pro , which may antagonize host innate immunity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "PLP of HCoV-NL63, SARS-CoV, MHV, PEDV, and MERS-CoV dramatically reduced the levels of ubiquitinated STING, RIG-I, TBK1, and IRF-3 to negatively regulate host antiviral innate immunity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0048771\" ! \"tissue remodeling\")\n",
      "The pathologic features of COVID-19 closely resemble those of SARS-CoV infection, which causes massive lung tissue remodeling through the urokinase, coagulation, and wound-healing pathways and through extracellular matrix proteins, including MMPs [31].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051259\" ! \"protein complex oligomerization\")\n",
      "For example, SARS-CoV N protein phosphorylation modulates N protein oligomerization, translation suppression, and localization to stress granules (Peng et al., 2008).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051318\" ! \"G1 phase\")\n",
      "Infectious bronchitis virus (IBV) induces an S and G2/M-phase arrest to favor viral replication (Dove et al., 2006; Li et al., 2007), and mouse hepatitis virus (MHV) (Chen and Makino, 2004) and some severe acute respiratory syndrome coronavirus (SARS-CoV) proteins induce cell cycle arrest in G1 phase (Yuan et al., 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051320\" ! \"S phase\")\n",
      "Recently, it was reported that the SARS-CoV N protein induced cell cycle arrest in the S phase through down-regulation of the S phase gene products expression (Surjit, 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051604\" ! \"protein maturation\")\n",
      "It has been suggested that SARS-CoV and its accessory protein are potent activators of pro-IL-1beta gene transcription and protein maturation , and thus , can activate the NLRP3 inflammasome .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051604\" ! \"protein maturation\")\n",
      "SARS-CoV and its ORF3a protein were found to be potent activators of pro-IL-1beta gene transcription and protein maturation , the 2 signals required for activation of the NLRP3 inflammasome .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051604\" ! \"protein maturation\")\n",
      "It has been suggested that SARS-CoV and its accessory protein are potent activators of pro-IL-1beta gene transcription and protein maturation , and thus , can activate the NLRP3 inflammasome .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051604\" ! \"protein maturation\")\n",
      "It has been suggested that SARS-CoV and its accessory protein are potent activators of pro-IL-1beta gene transcription and protein maturation , and thus , can activate the NLRP3 inflammasome .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051604\" ! \"protein maturation\")\n",
      "It has been suggested that SARS-CoV and its accessory protein are potent activators of pro-IL-1beta gene transcription and protein maturation , and thus , can activate the NLRP3 inflammasome .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0051607\" ! \"defense response to virus\")\n",
      "Microarray studies of host gene expression have shown that SARS-CoV elicits a non-specific innate inflammatory reaction rather than an antiviral response (Reghunathan et al., 2005).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051607\" ! \"defense response to virus\")\n",
      "Here we show that SARS-CoV could inhibit both virus- and interferon (IFN)-dependent signaling, two key steps of the antiviral response.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051607\" ! \"defense response to virus\")\n",
      "The internal ORFs we identified within ORF-N ( Figure 3C ) are homologues of the annotated SARS-CoV ORF9b which was shown to suppress the host antiviral response (Shi et al., 2014 , 72% similarity in amino acids).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The proteolytic cleavage events in SARS-CoV S that lead to membrane fusion occur both at the S1/S2 boundary and adjacent to a fusion peptide in the S2 domain (Belouzard et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "14 Meanwhile, no fusion was observed for 293T/EGFP cells without S-expression or 293T cells without ACE2-expression ( Fig. 1d and Supplementary information, Fig. S1b ), confirming that S-receptor engagement is necessary for the S-mediated viral fusion and entry.X-ray crystallographic analysis of the 6-HB fusion core formed by HR1 and HR2 domains in S2 subunit of SARS-CoV-2 S protein Previously, we identified that the 6-HB formed by HR1 and HR2 domains of the S2 subunit plays a very important role in the membrane fusion process mediated by MERS-CoV or SARS-CoV S protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "ANALYTICAL BIOCHEMISTRY.The membrane fusion process mediated by severe acute respiratory syndrome coronavirus (SARS-CoV) S protein and its cellular receptor angiotensin-converting enzyme 2 (ACE2) had been reconstituted using two Chinese hamster ovary (CHO) cell lines that constitutively express these recombinant proteins separately.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The spike ( S ) protein of severe acute respiratory syndrome-associated CoV ( SARS-CoV ) mediates membrane fusion and viral entry .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The spike (S) protein of the severe acute respiratory syndrome coronavirus (SARS-CoV) interacts with cellular receptors to mediate membrane fusion, allowing viral entry into host cells; hence it is recognized as the primary target of neutralizing antibodies, and therefore knowledge of antigenic determinants that can elicit neutralizing antibodies could be beneficial for the development of a protective vaccine.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Receptor recycling was then tracked after the cells were treated with pseudoviruses or the spike protein, a membrane component of SARS-CoV that mediates membrane fusion and is required for viral entry 11, 12.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The SARS-CoV spike (S) glycoprotein mediates membrane fusion events during virus entry and virus-induced cell-to-cell fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Therefore, it has postulated that membrane fusion process mediated by coronaviruses and SARS-CoV could be different from other type I viral fusion proteins [29], [38], [50], [51], [54], [74].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "of three independent experiments.Moving the trypsin cleavage position by introduction of arginine residues in the S2 region modulates SARS-CoV S-mediated membrane fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The proteolytic cleavage events in SARS-CoV S that lead to membrane fusion occur both at the interface of the receptor binding (S1) and fusion (S2) domains (S1/S2), as well as adjacent to a fusion peptide within S2 (S2′) (11).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      ": Moving the trypsin cleavage position by introduction of arginine residues in the S2′ region modulates SARS-CoV S-mediated membrane fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The proteolytic cleavage events in SARS-CoV S that lead to membrane fusion occur both at the interface of the receptor binding (S1) and fusion (S2) domains (S1/S2), as well as adjacent to a fusion peptide within S2 (S2Ј) (11) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The severe acute respiratory syndrome-coronavirus (SARS-CoV) spike (S) glycoprotein alone can mediate the membrane fusion required for virus entry and cell fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Using cell-cell Virology 393 (2009) 265-271 fusion and pseudovirus assays, we show that this domain is critical for the activation of SARS-CoV S-mediated membrane fusion and virus entry.Bioinformatic analysis of the SARS-CoV S2 domain flanked by cysteine residues C822 and C833A common feature of regions within a viral glycoprotein that are required for entry is that they show a high degree of conservation within a virus family.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Using cell–cell fusion and pseudovirus assays, we show that this domain is critical for the activation of SARS-CoV S-mediated membrane fusion and virus entry.A common feature of regions within a viral glycoprotein that are required for entry is that they show a high degree of conservation within a virus family.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Therefore, determination of S protein six-helix bundle formation is important for studying the mechanism by which SARS-CoV induces membrane fusion and for identification of SARS-CoV fusion effective inhibitors targeting S protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The SARS-CoV spike (S) glycoprotein mediates membrane fusion events during virus entry and virus-induced cell-to-cell fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Therefore, the S-protein of SARS-CoV might be able to mediate membrane fusion in a pH-dependent and independent fashion, and several parameters might control which stimulus is required under what conditions.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "If the model proposed above accurately describes SARS-CoV S-driven membrane fusion, one would expect that peptides mimicking HR1 or HR2 should assemble into a six-helix bundle and that such peptides would inhibit SARS-CoV S-mediated membrane fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The trigger for SARS-CoV S-driven membrane fusion and the possible cleavage by cellular proteases await further investigation.Two functional elements located in the transmembrane subunit of CoV S-proteins are pivotal to membrane fusion: a putative fusion peptide and two HRs (Figure 3).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The nature of pH-induced conformational changes in SARS-CoV S-protein remains to be determined.If the model proposed above accurately describes SARS-CoV S-driven membrane fusion, one would expect that peptides mimicking HR1 or HR2 should assemble into a six-helix bundle and that such peptides would inhibit SARS-CoV S-mediated membrane fusion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Therefore, the S-protein of SARS-CoV might be able to mediate membrane fusion in a pH-dependent and independent fashion, and several parameters might control which stimulus is required.One such parameter could be the association between the S1 and S2 subunits of the SARS-CoV S-protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Therefore, the S-protein of SARS-CoV might be able to mediate membrane fusion in a pH-dependent and independent fashion, and several parameters might control which stimulus is required.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The mechanism of the SARS-CoV and MERS-CoV interspecies transmission appears to be mediated by the S protein, by mediating receptor recognition and membrane fusion, a key factor in host specificity [21] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "This would be relevant for use in humans and deserves further study.The S2 domain of SARS-CoV mediates membrane fusion during viral infection and is hence a potential target for protective antibodies.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "The spike ( S ) protein of the severe acute respiratory syndrome coronavirus ( SARS-CoV ) interacts with cellular receptors to mediate membrane fusion , allowing viral entry into host cells ; hence it is recognized as the primary target of neutralizing antibodies , and therefore knowledge of antigenic determinants that can elicit neutralizing antibodies could be beneficial for the development of a protective vaccine .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "https://doi.org/10.1101/2020.03.09.983247 doi: bioRxiv preprint S2 subunit plays a very important role in the membrane fusion process mediated by 167 MERS-CoV or SARS-CoV S protein 16, 17 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070231\" ! \"T cell apoptotic process\")\n",
      "The SARS-CoV E protein may induce T-cell apoptosis via a pathway antagonistic to the mitochondrion-dependent mechanism of the antiapoptotic protein Bcl-xL [95] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "It has been found that SARS-CoV, a virus that belongs to the same betacoronavirus family as SARS-CoV-2 and is responsible for SARS, causes spermatogenic cell necrosis and apoptosis and induces inflammatory infiltrates in the interstitium, although viral genomic sequences were not detected in testes [15] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "SARS coronavirus which emerged in 2002 has been known to induce necrosis in ciliated airway epithelium of humans in a species-specific manner .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "The presence of ACE2 as a functional receptor for SARS-CoV and the presence of SARS-CoV in enterocytes of the small intestine, combined with the fact that virus is present in patients' stool samples, are consistent with the possibility of faeco-oral transmission.In addition to pulmonary and gastrointestinal problems, SARS-CoV infection also causes massive necrosis of the spleen and lymph nodes.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "It is not clear whether necrosis in these tissues represented secondary necrosis reflecting the degradative changes that apoptotic cells undergo at the later stages of apoptosis, but at least one SARS-CoV protein [open reading frame (ORF) 3b] has been shown to induce necrosis (Khan et al., 2006) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "SARS-CoV is known to cause acute respiratory distress syndrome (ARDS), the leading cause of death due to SARS-CoV infection, as well as a hemophagocytic syndrome, lymphoid depletion, and skeletal muscle fiber necrosis, which all result from elevated levels of proinflammatory cytokines.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "[8] [9] [10] [11] [12] Both SARS-CoV and the influenza virus induce apoptosis and necrosis of various cell types, including T cells, endothelial cells, and epithelial cells, through pro-inflammatory cytokine and chemokine mediated vascular leakage and suboptimal T cell responses.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "For examples , the severe acute respiratory syndrome coronavirus ( SARS-CoV ) ORF-3a was found to induce necrotic cell death , lysosomal damage and caspase-1 activation , which largely contribute to the clinical manifestations of SARS-CoV infection ( Yue et al ., 2018 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "Based on the results, SARS-CoV that caused massive necrosis of pulmonary epithelial cells such as type 1 and 2 pneumocytes, similar to that observed in lungs of PRCV-infected pigs (Gu and Korteweg, 2007, Jung et al., 2007), might also induce •NO production in the respiratory tract.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0070265\" ! \"necrotic cell death\")\n",
      "Moreover, other strains of highly pathogenic viruses such as dengue virus [74] or MERS-CoV [75] appear to induce considerable necrosis in epithelial cells to promote viral replication and propagation, while less virulent viruses like rhinovirus, or SARS-CoV induce limited or no necrosis [75], [76].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070269\" ! pyroptosis)\n",
      "According to recent study it has been found that SARS-CoV Viroporin 3a triggered the activation of the nod like receptor protein-3(NLRP3) inflammasome along with the secretion of IL-1β in bone marrow-derived macrophages, leading to SARS-CoV induced cell pyroptosis [21].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070269\" ! pyroptosis)\n",
      "Notably, SARS-CoV has been shown to cause caspase-1-mediated cell death (pyroptosis, a highly inflammatory form of cell death) via the activation of the Nod-like receptor family, pyrin domain-containing 3 (NLRP3) mediated by the viral 3a protein which acts as a potassium ion channel (viroporin) resulting in inflammasome activation [29].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070269\" ! pyroptosis)\n",
      "For example, subversion of innate immune response is inherent to cytopathic pathogens like SARS-CoV (a related beta coronavirus), which causes an increased release of pyroptosis products (e.g.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070488\" ! \"neutrophil aggregation\")\n",
      "Inhibition of SARS Pseudovirus Cell Entry by Lactoferrin Binding to Heparan Sulfate Proteoglycans.It has been reported that lactoferrin (LF) participates in the host immune response against Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) invasion by enhancing NK cell activity and stimulating neutrophil aggregation and adhesion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070488\" ! \"neutrophil aggregation\")\n",
      "It has been reported that lactoferrin (LF) participates in the host immune response against Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) invasion by enhancing NK cell activity and stimulating neutrophil aggregation and adhesion.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070997\" ! \"neuron death\")\n",
      "Furthermore , animal models showed that SARS-CoV could invade the CNS primarily through the olfactory route [ 265 ] or even after an intra-peritoneal infection [ 284 ] , and induce neuronal cell death [ 265,284 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070997\" ! \"neuron death\")\n",
      "Furthermore, animal models showed that SARS-CoV could invade the CNS primarily through the olfactory route [265] or even after an intra-peritoneal infection [284], and induce neuronal cell death [265,284].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0070997\" ! \"neuron death\")\n",
      "The main theory affirm that SARS-CoV probably uses as the main entry route into the CNS the olfactory nerve fibers [ 4 ] , [ 15 ] , [ 31 ] , [ 32 ] , [ 48 ] , and induces neuronal cell death [ 4 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0097193\" ! \"intrinsic apoptotic signaling pathway\")\n",
      "These data show that N, as well as the commonly used apoptosis inducer staurosporine, can cause de-phosphorylation of Bad.With all these data taken together, we show that SARS-CoV N induces the intrinsic apoptotic pathway through de-phosphorylation of Bad and activation of caspase-9, resulting in processing of N by caspases.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(GO:\"GO:0097753\" ! \"membrane bending\")\n",
      "However, very recently, evidence was presented that the SARS-CoV E protein transmembrane domain can form a palindromic transmembrane helical hairpin and incorporation of the helical hairpin into lipid vesicles increased membrane curvature by deforming lipid bilayers (Arbely et al., 2004), a known function of viral ion channel family proteins (Gonzalez and Carrasco, 2003).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10303 ! RPL13)\n",
      "In an ex vivo human lung explant model , SARS-CoV-2 infection also failed to induce detectable levels of IFNa , b , g , or l1-3 at any time point for up to 48 hr , while SARS-CoV induced IFNa , b , g , or l1-3 at late time points .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10618 ! CCL2)\n",
      "In unproductive infections of DCs , SARS-CoV induced CXCL-10 , CCL-2 , CCL-3 , CCL-5 , and TNF-alpha ( Law et al ., 2005 , Tseng et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10627 ! CCL3)\n",
      "In unproductive infections of DCs , SARS-CoV induced CXCL-10 , CCL-2 , CCL-3 , CCL-5 , and TNF-alpha ( Law et al ., 2005 , Tseng et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10627 ! CCL3)\n",
      "The degree of induction by MERS-CoV and SARS-CoV on IL-8 , MCP-1 , and MIP-1alpha were subtle with comparable levels .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10632 ! CCL5)\n",
      "In unproductive infections of DCs , SARS-CoV induced CXCL-10 , CCL-2 , CCL-3 , CCL-5 , and TNF-alpha ( Law et al ., 2005 , Tseng et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10632 ! CCL5)\n",
      "RANTES expression , by contrast , is never induced by SARS-CoV , although the cells respond normally to other viruses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "Besides , SARS-CoV also upregulated only IP-10 to a similar magnitude as that induced by SARS-CoV-2 in Caco2 ( Fig. 3B , p < 0.01 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "Besides , SARS-CoV also upregulated only IP-10 to a similar magnitude as that induced by SARS-CoV-2 in Caco2 ( Fig. 3B , p < 0.01 ) and IFsITM3 ( Fig. 4A ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "In unproductive infections of DCs , SARS-CoV induced CXCL-10 , CCL-2 , CCL-3 , CCL-5 , and TNF-alpha ( Law et al ., 2005 , Tseng et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "IP-10 and IL-8 , however , can be activated by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "We have previously demonstrated that IP-10 and MCP-1 are up-regulated in primary human macrophage by SARS-CoV [ 28 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10637 ! CXCL10)\n",
      "In both cell lines and primary alveolar type II cells , SARS-CoV induced IFN-beta , IFN-lambda , CXCL10 , CXCL11 , IL-6 , IP-10 , and TNF-alpha ( 18 , 187 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:10638 ! CXCL11)\n",
      "In both cell lines and primary alveolar type II cells , SARS-CoV induced IFN-beta , IFN-lambda , CXCL10 , CXCL11 , IL-6 , IP-10 , and TNF-alpha ( 18 , 187 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11847 ! TLR1)\n",
      "RESULTS : Our results demonstrates that SARS-CoV did not modulate TLR-1 to TLR-10 gene expression but significantly induced the expression of CCR-1 , CCR-3 , and CCR-5 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11849 ! TLR3)\n",
      "17  SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4 , subsequently inducing an inflammatory reaction , inflammasome activation and upregulation of the IL-1beta pathway .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11850 ! TLR4)\n",
      "Reportedly , severe acute respiratory syndrome ( SARS-CoV ) and mouse hepatitis virus ( MHV ) infection can also induce TLR4 expression [ 21 ] , suggesting that TLR4 possibly has a similar effect both in avian and mammalian species as well as during coronavirus infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11850 ! TLR4)\n",
      "17  SARS-CoV and possibly SARS-CoV-2 activate TLR3 and TLR4 , subsequently inducing an inflammatory reaction , inflammasome activation and upregulation of the IL-1beta pathway .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11876 ! TMPRSS2)\n",
      "Upon binding to ACE-2 , both SARS-CoV and SARS-CoV-2 activate the transmembrane serine protease-2 ( TMPRSS2 ) , which is highly expressed in the lungs .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11892 ! TNF)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs. 3, 4) [49, 50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11892 ! TNF)\n",
      "In both cell lines and primary alveolar type II cells , SARS-CoV induced IFN-beta , IFN-lambda , CXCL10 , CXCL11 , IL-6 , IP-10 , and TNF-alpha ( 18 , 187 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11892 ! TNF)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:11896 ! TNFAIP3)\n",
      "In both cell lines , SARS-CoV up-regulated A20 which may protect different cell types against tumour necrosis factor ( TNF ) - mediated programmed cell death and is critical for limiting inflammation by terminating TNF-induced nuclear factor ( NF ) - kB responses in the intestine and other organs [ 20 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:16400 ! NLRP3)\n",
      "There are multiple pathways by which SARS-CoV triggers NLRP3 activation which have yet to be characterized and are likely influenced by cell type and the extracellular microenvironment ( 15 , 84 , 86 , 88 , 107 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:16400 ! NLRP3)\n",
      "Previously SARS-CoV has been shown to induce NLRP3 by its ion channel-forming M protein and ORF8b [ 87 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:16400 ! NLRP3)\n",
      "Dengue virus , influenza , HCV , encephalomyocarditis virus ( EMCV ) , vesicular stomatitis virus ( VSV ) , herpes simplex virus-1 ( HSV-1 ) and SARS-CoV activate NLRP3 , [ 44 ] [ 45 ] [ 46 ] [ 47 ] while HBV suppresses NLRP3 activation by inhibiting NF-jB and the generation of ROS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:1641 ! CD209)\n",
      "To study the cellular response in the acute phase of SARS coronavirus ( SARS-CoV ) - host cell interaction , we investigated the induction of chemokines , adhesion molecules , and DC-SIGN ( dendritic cell-specific ICAM-3-grabbing nonintegrin ) by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:19137 ! CD276)\n",
      "SARS-CoV-2 may induce phenotypic and functional maturation of cells of the mononuclear phagocyte system ( MPS ) as SARS-CoV has been shown to upregulate MHC class II and costimulatory molecules on monocyte-derived DCs ( Tseng et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:195 ! ADAM17)\n",
      "The cellular entry of SARS-CoV-2 is facilitated by ADAM17 activity , which is upregulated by SARS-CoV through the cytoplasmatic domain of ACE2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:195 ! ADAM17)\n",
      "The cellular entry of SARS-CoV-2 is facilitated by ADAM17 activity , which is upregulated by SARS-CoV through the cytoplasmatic domain of ACE2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:195 ! ADAM17)\n",
      "Also, ADAM-17 silencing decreases host cell entry of SARS-CoV [2].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:195 ! ADAM17)\n",
      "In addition , it has been proposed that HCoV-NL63 does not induce ACE2 shedding [ 158 ] , while SARS-CoV activates ADAM17 and promotes the shedding of the ACE2 ectodomain , which may contribute to the differential pathogenicity of these viruses .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:2438 ! CSF3)\n",
      "SARS-CoV did not result in modulation of CSF3 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:5434 ! IFNB1)\n",
      "In both cell lines and primary alveolar type II cells , SARS-CoV induced IFN-beta , IFN-lambda , CXCL10 , CXCL11 , IL-6 , IP-10 , and TNF-alpha ( 18 , 187 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:5438 ! IFNG)\n",
      "As shown in Fig. 3A , SARS-CoV marginally upregulated the expression of IFN-gamma and did not induce an upregulation of IL-12 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs. 3, 4) [49, 50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "Thus , SARS-CoV strongly induces IL-8 , but not IL-6 in a cell-type dependent manner .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "In both cell lines and primary alveolar type II cells , SARS-CoV induced IFN-beta , IFN-lambda , CXCL10 , CXCL11 , IL-6 , IP-10 , and TNF-alpha ( 18 , 187 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV enhances IL-6 production and decreases the expression levels of SOCS3 [ 75 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "SARS-CoV also promoted a moderate increase in the production of IL-6 in DCs [ 29 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6018 ! IL6)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6025 ! CXCL8)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs. 3, 4) [49, 50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6025 ! CXCL8)\n",
      "IL-8 , by contrast , is clearly induced by SARS-CoV in Caco-2 cells , but not in HEK 293 cells ( Fig. 5A and 5B , lower panels ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6025 ! CXCL8)\n",
      "Thus , SARS-CoV strongly induces IL-8 , but not IL-6 in a cell-type dependent manner .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6025 ! CXCL8)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6118 ! IRF3)\n",
      "However , the elevated transcriptional activation of genes coding for IRF-7 , IFN-beta , IFN-lambdas , and many ISGs detected at 48 hrs in infected 2B4 cells strongly argue for the activation of IRF-3 by SARS-CoV in 2B4 cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "Indeed , several studies have demonstrated that PLP of SARS-CoV inhibits both the IRF3 and NF-kappaB pathways ( Devaraj et al ., 2007 , Frieman et al ., 2009 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "SARS-CoV was shown to inhibit IRF3 by preventing its hyperphosphorylation , dimerization , and interaction with the cofactor CBP ( Spiegel et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "SARS-CoV 2 prevents IRF3 / 7 signalling , suppresses Type I Interferon ( IFN-I ) release and inhibits JAK / STAT signalling ( Gordon et al ., 2020 ; Jamilloux et al ., 2020 ; Zumla et al ., 2016 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "Also , severe acute respiratory syndrome coronavirus ( SARS-CoV ) 8a and 8ab bound IRF3 and accelerated IRF3 degradation through a ubiquitin-proteasome pathway , resulting in reduced IFN-beta expression and facilitating virus replication [ 26 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "Indeed , several studies have demonstrated that the PLP of SARS-CoV inhibits both the IRF3 and NF-kappaB pathways ( Devaraj et al. 2007 ; Frieman et al. 2009 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3)\n",
      "Furthermore we show that MHV and SARS-CoV do not actively block IRF3 activation via either RIG-I or MDA-5 as they are unable to inhibit IRF3 translocation upon Sendai virus infection or poly ( I :C ) transfection , respectively .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6122 ! IRF7)\n",
      "Taken together , our results suggest that ( i ) IFN , as it is produced by plasmacytoid dendritic cells , could enable tissue cells to launch a host response to SARS-CoV , ( ii ) IRF-3 and IRF-7 may be active at subdetectable levels , and ( iii ) SARS-CoV does not activate IRF-7 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:939 ! BAG3)\n",
      "During viral infection , the SARS-CoV significantly upregulates BAG3 , but HSV1 does not substantially alter BAG3 within 12 h at the protein level [ 9,10 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9437 ! EIF2AK2)\n",
      "SARS-CoV ( a lineage B beta-CoV ) has been reported to trigger PKR activation but to be resistant to its antiviral activity [ 51 ] , although in another study a strong antiviral effect of PKR was observed [ 52 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "At a late stage of infection , SARS-CoV replicates at a high level and progeny viruses are released from host cells , SARS-CoV activates NF-kB and upregulates COX-2 expression , then triggering antiviral activities of prostanoids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs. 3, 4) [49, 50].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Interestingly, while inhibition of COX-2 expression exerts a suppressive effect on lung inflammation [XREF_BIBR], it has been shown that both S and the nucleoprotein (N) of SARS-CoV upregulate COX-2 [XREF_BIBR] through different molecular mechanisms.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Interestingly, while inhibition of COX-2 expression exerts a suppressive effect on lung inflammation [76], it has been shown that both S and the nucleoprotein (N) of SARS-CoV upregulate COX-2 [77] through different molecular mechanisms.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "The activation of COX-2 by SARS-CoV suggests that similar responses may occur with SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "The activation of COX-2 by SARS-CoV suggests that similar responses may occur with SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "The activation of COX-2 by SARS-CoV suggests that similar responses may occur with SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Furthermore , SARS-CoV , the closest relative of SARS-CoV-2 among human coronaviruses and cause of the 2002-2003 epidemic ( 11 ) , stimulates COX-2 expression via its spike protein and nucleocapsid protein ( 12 , 13 ) , indicating the potential relevance of this pathway for SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Furthermore , SARS-CoV , the closest relative of SARS-CoV-2 among human coronaviruses and cause of the 2002-2003 epidemic ( 11 ) , stimulates COX-2 expression via its spike protein and nucleocapsid protein ( 12 , 13 ) , indicating the potential relevance of this pathway for SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "We have previously demonstrated that nucleocapsid protein ( Yan et al ., 2006 ) and spike protein ( Liu et al ., 2006 ; 2007 ) of SARSassociated coronavirus ( SARS-CoV ) were able to activate COX-2 expression .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "We have previously demonstrated that nucleocapsid protein ( Yan et al ., 2006 ) and spike protein ( Liu et al ., 2006 , Liu et al ., 2007 ) of SARS-associated coronavirus ( SARS-CoV ) were able to activate COX-2 expression .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Furthermore , SARS-CoV , the closest relative of SARS-CoV-2 among 82 human coronaviruses and cause of the 2002-2003 epidemic ( 11 ) , stimulates COX-2 expression 83 via its spike protein and nucleocapsid protein ( 12 , 13 ) , indicating the potential relevance of this 84 pathway for SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Because SARS-CoV is able to delay TLR induced type I interferon responses and activate COX-2, a role for kallikrein-kinin signals that stimulate prostaglandin production may exist in the pathogenesis of SARS-CoV and SARS-CoV-2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "It has been reported that nucleocapsid and spike proteins of SARS-CoV are able to directly up-regulate the promotor of IL-6, IL-8, IL-12, TNF-alpha, COX-2 and probably IL-17 genes and indirectly up-regulate those of IL-1 alpha and beta via NF-kappaB and AP-1 pathways (Figs.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:9605 ! PTGS2)\n",
      "Nucleocapsid protein of SARS-CoV activates the expression of cyclooxygenase-2 by binding directly to regulatory elements for nuclear factor-kappa B and CCAAT and enhancer binding protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"Postural instability\")\n",
      "SARS-CoV reduces ACE2 expression , causing an imbalance between the ACE / Ang II / AT1R axis and the ACE2 / Ang ( 1-7 ) / Mas receptor axis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"Ventricular arrhythmia\")\n",
      "It was shown that in addition to sinus tachycardia , SARS-CoV can also cause supraventricular arrhythmia , ventricular arrhythmia , first-degree atrioventricular block , and ST-T segment changes [ 10 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"Supraventricular arrhythmia\")\n",
      "It was shown that in addition to sinus tachycardia , SARS-CoV can also cause supraventricular arrhythmia , ventricular arrhythmia , first-degree atrioventricular block , and ST-T segment changes [ 10 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:Severe)\n",
      "Patients with defective immune system for example , elderly or individuals with a history of medical cases including hypertension , obesity , diabetes or cancer , might quickly develop a severe or critical status due to SARS-CoV and MERS-CoV infection [ 34 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:\"Chronic infection\")\n",
      "If the SARS-CoV is able to cause a chronic persistent infection , chronic carriers may serve as sources for new SARS outbreaks .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Coronavirus Disease 2019 (COVID-19) and pregnancy: what obstetricians need to know.Severe acute respiratory syndrome (SARS) is caused by the SARS-CoV.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Combating COVID-19 with Chloroquine.Since December 2019, the world is increasingly facing an unprecedented challenge by coronavirus disease 2019 (COVID-19) caused by a virus called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which is structurally related to the human severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV) (Zhu et al., 2020).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is etiologically caused by SARS coronavirus 2 ( SARS-CoV-2 ) , a virus closely related to the SARS virus [ 2 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Angiotensin converting enzyme 2 (ACE2) acts as the receptor for severe acute respiratory syndrome coronavirus (SARS-CoV), which causes COVID-19.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "A series of outbreaks and epidemics of respiratory illnesses have been attributed to various types of these viruses, such as Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS), which were caused by SARS-CoV and MERS-CoV, respectively, in addition to the current Coronavirus Disease 2019 (COVID-19) (1, 2).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "This review summarizes the ongoing researches regarding etiology, epidemiology, transmission dynamics, treatment, and prevention and control strategies of the coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), with comparison to severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV) and pandemic H1N1 virus.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The past 2 decades have witnessed three lethal zoonotic diseases of humans caused by novel coronaviruses: the severe acute respiratory syndrome (SARS) in 2002, the Middle East respiratory syndrome (MERS) in April 2012, and currently, coronavirus disease-19 (COVID-19) caused by SARS-CoV, MERS-CoV and SARS-CoV-2, respectively.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "This is a novel zoonotic CoV related to the SARS-CoV that can cause severe respiratory disease (COVID-19).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "SARS is caused by SARS-CoV, whereas SARS-CoV-2 causes COVID-19.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "So far three deadly human respiratory coronaviruses , severe acute respiratory syndrome coronavirus ( SARS-CoV ) , middle east respiratory syndrome coronavirus ( MERS-CoV ) and coronavirus SARSCoV-2 ( severe acute respiratory syndrome coronavirus-2 ) which causes COVID-19 ( Coronavirus Disease-2019 ) , have been detected .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Comparable to COVID-19 , the SARS-associated coronavirus ( SARS-CoV ) may cause severe acute respiratory syndrome ( SARS ) ( Xu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Angiotensin converting enzyme 2 ( ACE2 ) acts as the receptor for severe acute respiratory syndrome coronavirus ( SARS-CoV ) , which causes COVID-19 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Abstract The worldwide pandemic of COVID-19, caused by the virus SARS-CoV,-2 has continued to progress, and increasing information is becoming available about the incidence of digestive symptoms as well as abnormal liver-associated enzymes in patients who are infected.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "3 The public evidence shows that COVID-19 shares similar pathogenesis with the pneumonia induced by SARS-CoV or MERS-CoV.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Due to its phylogenetic similarity to severe acute respiratory syndrome coronavirus ( SARS-CoV ) , COVID-19 was also named SARS-CoV-2 [ 2 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Rockx et al. have undertaken a comparative study of three human coronaviruses in cynomolgus macaques : severe acute respiratory syndrome-coronavirus ( SARS-CoV ) ( 2002 ) , Middle East respiratory syndrome ( MERS ) - CoV ( 2012 ) , and SARS-CoV-2 ( 2019 ) , which causes COVID-19 ( see the Perspective by Lakdawala and Menachery ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "This includes the 2003 Severe Acute Respiratory Syndrome (SARS) outbreak caused by SARS coronavirus (SARS-CoV) , which originated in the large wet markets of Guangdong province, China (Ksiazek et al., 2003) , and the current COVID-19 outbreak caused by SARS-CoV-2, first discovered in people associated with a wet market in Wuhan Zhu et al., 2020) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The viruses that cause pandemics include SARS-CoV , Middle East respiratory syndrome ( MERS ) - CoV , and SARS-CoV-2 , the virus causing COVID-19 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "2 Angiotensin-converting enzyme type 2 ( ACE2 ) is a protein at the cross-talk between cardiovascular diseases and COVID-19 because ( i ) ACE2 is the functional cellular receptor for the severe acute respiratory syndrome coronavirus ( SARS-CoV ) and SARS-CoV-2 which causes COVID-19 ; and ( ii ) ACE2 is a pivotal cardioprotective protein in the renin-angiotensin-aldosterone system ( RAAS ) where it converts angiotensin II to angiotensin - ( 1-7 ) in order to reduce elevated blood pressure , hypertrophy , and fibrosis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "1 Earlier studies show that COVID-19 risk factors include older age , male sex , hypertension , diabetes , coronary heart disease , and obesity ; and the common outcomes include organ damage ( lungs , kidneys , and heart ) , lymphocytopenia , thrombocytopenia , and death.2 Angiotensin-converting enzyme type 2 ( ACE2 ) is a protein at the cross-talk between cardiovascular diseases and COVID-19 because ( i ) ACE2 is the functional cellular receptor for the severe acute respiratory syndrome coronavirus ( SARS-CoV ) and SARS-CoV-2 which causes COVID-19 ; and ( ii ) ACE2 is a pivotal cardioprotective protein in the renin-angiotensin-aldosterone system ( RAAS ) where it converts angiotensin II to angiotensin - ( 1-7 ) in order to reduce elevated blood pressure , hypertrophy , and fibrosis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "So far three deadly human respiratory coronaviruses , severe acute respiratory syndrome coronavirus ( SARS-CoV ) , middle east respiratory syndrome coronavirus ( MERS-CoV ) and coronavirus SARS-CoV-2 ( severe acute respiratory syndrome coronavirus-2 ) which causes COVID-19 ( Coronavirus Disease-2019 ) , have been detected .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 infection, a highly contagious disease caused by the SARS-CoV virus, and the World Health Organization declared this increasingly spreading disease as a global public health emergency (pandemic).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000022 ! \"Abortion, Spontaneous\")\n",
      "Limited data is available regarding the impact of COVD-19 on pregnancy , however the case of miscarriage due to placental infection caused by SARS-CoV-2 in second trimester has already been reported.2 High rate ( 57 % ) of miscarriage in first trimester due to SARS-CoV has been documented during 2002-2003 epidemic.3 During the MERS-CoV epidemic , alarming rate of adverse pregnancy outcomes were reported previously .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000069078 ! Seroconversion)\n",
      "SARS-CoV induces seroconversion within 4 days from the start of the infection , and specific protective IgG dosages have been found up to 2 years after the infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000072659 ! Hypophysitis)\n",
      "Indeed , SARS-CoV ( a related coronavirus causing the 2002-2004 SARS outbreak ) has also been detected in hypothalamic neurons of SARS autopsies , 7 and hypothalamic-pituitaryadrenal axis dysfunction has been observed in almost 40 % of SARS survivors , suggesting that SARS-CoV could cause hypophysitis and / or hypothalamic damage .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000072659 ! Hypophysitis)\n",
      "Indeed , SARS-CoV ( a related coronavirus causing the 2002-2004 SARS outbreak ) has also been detected in hypothalamic neurons of SARS autopsies ,7 and hypothalamic-pituitary-adrenal axis dysfunction has been observed in almost 40 % of SARS survivors , suggesting that SARS-CoV could cause hypophysitis and / or hypothalamic damage.8 Furthermore , animal models have shown that after intranasal inoculations , SARS-CoV enters the brain through the olfactory bulb and via the olfactory nerve with subsequent trans-synaptic spread , causing selective neuronal infection and death in the absence of inflammation in the amygdala , raphe nuclei , and paramedial hypothalamus , a topographic distribution of lesions similar to that from our cases .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus belonging to the Coronaviridae family , which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 22 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus belonging to the Coronaviridae family , which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 22 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000208 ! \"Acute Disease\")\n",
      "SARS-CoV-2 is a virus from coronaviruses family which causes infection in humans ; SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause acute disease , while NL63 , HKU1 , OC43 , and 229E cause mild symptoms [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000309 ! \"Adrenal Insufficiency\")\n",
      "Leow et al. ( 56 ) showed previously that SARS-CoV could inflict pituitary lesions either directly or indirectly and contribute to secondary thyroid and adrenal insufficiency , which could be treated using levothyroxine and hydrocortisone replacement ( 56 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000936 ! \"Antigen-Antibody Complex\")\n",
      "SARS-CoV may trigger autoimmune antibody or immune complexes formation , but also can directly affect megakaryocytes , hematopoietic stem cells and platelets.13 Very recently , a unique case of Immune Thrombocytopenic Purpura ( ITP ) was reported as a primary finding in a severely ill SARS-CoV-2 infected adult , implying a correlation of the virus and ITP.14 Detection of SARS-CoV-2 RNA in oropharyngeal specimens from the upper respiratory tract with low cycle threshold ( Ct ) values on hospital day 4 ( Fig. 1 ) is suggestive of high viral load and potential for transmission , but provides no information on the clinical impact on the patient according to published data.15 Furthermore , literature is still inconclusive on the time that the RNA of the virus becomes undetectable in respiratory secretions and varies depending on the severity of the disease and the linkage with complications of the disease .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000936 ! \"Antigen-Antibody Complex\")\n",
      "SARS-CoV may trigger autoimmune antibody or immune complexes formation , but also can directly affect megakaryocytes , hematopoietic stem cells and platelets .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D000936 ! \"Antigen-Antibody Complex\")\n",
      "SARS-CoV may trigger autoimmune antibody or immune complexes formation , but also can directly affect megakaryocytes , hematopoietic stem cells and platelets.13 Very recently , a unique case of Immune Thrombocytopenic Purpura ( ITP ) was reported as a primary finding in a severely ill SARS-CoV-2 infected adult , implying a correlation of the virus and ITP.14 Detection of SARS-CoV-2 RNA in oropharyngeal specimens from the upper respiratory tract with low cycle threshold ( Ct ) values on hospital day 4 ( Fig. 1 ) is suggestive of high viral load and potential for transmission , but provides no information on the clinical impact on the patient according to published data.15 Furthermore , literature is still inconclusive on the time that the RNA of the virus becomes undetectable in respiratory secretions and varies depending on the severity of the disease and the linkage with complications of the disease .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D001929 ! \"Brain Edema\")\n",
      "A postmortem analysis has also been shown that SARS-CoV is capable to infect the neural cells in the hypothalamus and cortex and cause brain edema in patients with SARS disease ( Gu et al. 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D001988 ! Bronchiolitis)\n",
      "The other three highly pathogenic viruses , SARS-CoV , MERS-CoV , and SARS-CoV-2 , cause severe respiratory syndrome , pneumonia , bronchiolitis , rhinitis , pharyngitis , sinusitis , and other systemic symptoms such as occasional watery diarrhea in humans of all ages .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D002459 ! \"Cell Fusion\")\n",
      "Abstract The spike protein of the severe acute respiratory syndrome coronavirus ( SARS-CoV ) mediates cell fusion by binding to target cell surface receptors .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D002459 ! \"Cell Fusion\")\n",
      "The spike protein of the severe acute respiratory syndrome coronavirus ( SARS-CoV ) mediates cell fusion by binding to target cell surface receptors .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "A related virus , SARS-CoV , promotes complement activation , increasing C5a levels and contributing to hyperinflammation [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003165 ! \"Complement System Proteins\")\n",
      "Another group demonstrated that SARS-CoV could trigger complement activation through the lectin pathway ( 37 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV could be the principal cause of death or one of the contributing factors in patients with multiple underlying diseases .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS virus causes death in adults , who exhibit hemorrhage from the mouth , nasal cavity , and intestine , hemorrhagic interstitial pneumonia , and a clot in liver , spleen , and kidney .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV is unlike any other human coronavirus to date , causing severe respiratory disease and death in 10 % of infected patients ( WHO , 2003 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "More than one viro-porins were encoded by highly pathogenic RNA viruses , for example , coronaviruses , picornaviruses , influenza A virus , and hepatitis C virus , so that in some viruses especially SARS-CoV resulted in death due to great disruption of the pulmonary epithelium and increasing edema and IL-1beta-mediated proinflammatory response in the lung parenchyma ( Graham , Donaldson , & Baric , 2013 ; Nieto-Torres et al ., 2015 ; Nieva et al ., 2012 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "The direct and indirect effects of SARS-CoV 2 on the respiratory epithelium , and the severity of such process may lead to detrimental anatomical and structural damages , respiratory distress syndrome and death .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "Most human coronaviruses commonly cause relatively mild respiratory illnesses ; however two zoonotic coronaviruses , SARS-CoV and MERS-CoV , can cause severe illness and death .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "Most human coronaviruses commonly cause relatively mild respiratory disease , however two coronaviruses , SARS-CoV ( Rota et al ., 2003 ) and MERS-CoV ( Zaki et al ., 2012 ) can cause severe illness and death .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV and MERS-CoV , which are highly pathogenic to human lives and have caused serious diseases or death , causes about 10 and 36 % mortality respectively .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "In contrast , SARS-CoV and the recently emerged MERS-CoV may cause severe pneumonia with acute respiratory distress syndrome , multi-organ failure , and death in both immunocompetent and immunocompromised hosts [ 4 ] [ 5 ] [ 6 ] [ 7 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "During the severe acute respiratory syndrome ( SARS ) outbreaks , many frontline HCWs had a significantly increased risk of contracting the SARS-coronavirus ( SARS-CoV ) that resulted in severe illness and death [ 2 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "42 43 Main 44 Severe acute respiratory syndrome coronavirus ( SARS-CoV ) , Middle Eastern respiratory 45 syndrome coronavirus ( MERS-CoV ) , and SARS-CoV-2 , the etiological agent of the current 46 COVID-19 pandemic , have caused widespread disease , death , and economic hardship in the 47 past 20 years 1 , highlighting the pandemic potential of the CoV genus .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause respiratory , intestinal , hepatic , and neuronal diseases and may lead to acute respiratory distress syndrome ( ARDS ) , multiple organ failure ( MOF ) , and even death in severe cases [ 13-15 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause respiratory , intestinal , hepatic , and neuronal diseases and may lead to acute respiratory distress syndrome ( ARDS ) , multiple organ failure ( MOF ) , and even death in severe cases [ 13 ] [ 14 ] [ 15 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "Unlike the above four human CoVs , SARS-CoV , MERS-CoV , and 2019-nCoV have caused severe pneumonia and / or failure of other organs , even death , among infected populations ( Nicholls et al ., 2003 ; Zhong et al ., 2003 ; Zaki et al ., 2012 ; Zhu et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "[ 1 ] During an earlier pandemic , it was noted that during the severe acute respiratory syndrome ( SARS ) outbreak , many HCW got infected with SARS-coronavirus ( SARS-CoV ) that resulted in severe illness and death [ 2 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "Seven coronaviruses have been found to infect humans , of which SARS-CoV , SARS-CoV-2 and MERS-CoV can cause severe acute respiratory syndrome and death .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003643 ! Death)\n",
      "Viral infection Many diseases are caused by viral infection , such as influenza , hepatitis , acquired immunodeficient syndrome ( AIDS ) , or emerging viral diseases , such as Ebola , viral pneumonia caused by MERS , SARS-CoV or SARS-CoV-2 infection , which lead to severe illness and death , as well as tremendous economic and social impacts .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV and SARS-CoV-2 cause mild disease and no death was reported in this model .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV and SARS-CoV-2 cause mild disease and no death was reported in this model .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003713 ! \"Dendritic Cells\")\n",
      "103 Excessive proinflammatory cytokine responses are induced because of the activation of dendritic cells and macrophages by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003920 ! \"Diabetes Mellitus\")\n",
      "Moreover , a study showed that SARS-CoV may cause diabetes by ACE2-dependent damage of pancreatic islet cells [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003920 ! \"Diabetes Mellitus\")\n",
      "Moreover , a study showed that SARS-CoV may cause diabetes by ACE2-dependent damage of pancreatic islet cells [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003920 ! \"Diabetes Mellitus\")\n",
      "Moreover , a study showed that SARS-CoV may cause diabetes by ACE2-dependent damage of pancreatic islet cells [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003920 ! \"Diabetes Mellitus\")\n",
      "In another study , SARS-CoV was shown to cause diabetes by ACE2-dependent infection of pancreatic isle cells ( 10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003920 ! \"Diabetes Mellitus\")\n",
      "In the literature , SARS-CoV has been known to cause de novo diabetes by ACE2-dependent uptake on pancreatic isle cells , and furthermore dysregulate lipid autophagy in favor of the viral lifecycle .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003967 ! Diarrhea)\n",
      "Although generally thought to be spread exclusively by respiratory routes , there is evidence that SARS-CoV can replicate in the small and large intestines , causing diarrhea that results in fecal shedding of the virus .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003967 ! Diarrhea)\n",
      "Clinical data indicates that severe acute respiratory syndrome coronavirus ( SARS-CoV ) caused significant diarrhea [ Booth et al ., 2003 ; Choi et al ., 2003 ; Hsu et al ., 2003 ; Peiris et al ., 2003 ; Hsueh et al ., 2004 ; Grant et al ., 2012 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003967 ! Diarrhea)\n",
      "Nonetheless , in view of the limited architectural disturbance , there is still no solid evidence that the SARS-CoV is the cause of diarrhea .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D003967 ! Diarrhea)\n",
      "Besides pneumonia , SARS-CoV causes diarrhea [ 71 ] , lymphopenia [ 72 ] , haematological disorders [ 47 ] , pulmonary vasculitis , and thrombosis [ 73 , 74 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005128 ! \"Eye Diseases\")\n",
      "At the moment , it is unclear whether SARS-CoV 2 can cause more severe eye disease beyond keratoconjunctivitis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "SARS-CoV causes high fever in almost all patients , while SARS-CoV-2 does not .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "SARS-CoV causes high fever in almost all patients , while SARS-CoV-2 does not .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "SARS-CoV causes high fever in almost all patients , while SARS-CoV-2 does not .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "SARS-CoV is a novel zoonotic pathogen , causing typical fever and respiratory symptoms .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "Because COX-2 is normally induced during inflammatory processes and SARS-CoV induces fever , edemar , and diffuse alveolar damage in severely affected individuals [ 22 ] , in this study we investigated the roles of SARS-CoV S protein in the regulation of expression of COX-2 as well as in SARS epidemic .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D005334 ! Fever)\n",
      "SARS-CoV generally caused fever before being transmissible , so fever was used as a marker to track and contain it .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D007006 ! Hypogonadism)\n",
      "For this reason , SARS-CoV might lead to hypogonadism and harm male fertility .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D007006 ! Hypogonadism)\n",
      "For this reason , SARS-CoV might lead to hypogonadism and harm male fertility .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008168 ! Lung)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008168 ! Lung)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008168 ! Lung)\n",
      "Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D008168 ! Lung)\n",
      "The zinc peptidase ACE2 has also expressed in the alveolar type 2 pneumocytes , which explains its role in lung damage due to SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D008168 ! Lung)\n",
      "The zinc peptidase ACE2 has also expressed in the alveolar type 2 pneumocytes , which explains its role in lung damage due to SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D008168 ! Lung)\n",
      "The zinc peptidase ACE2 has also expressed in the alveolar type 2 pneumocytes , which explains its role in lung damage due to SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Both MERS-CoV and SARS-CoV infect human primary T cells and induce massive apoptosis and lymphopenia , although they do not produce infectious virus in these cells ( Chu et al ., 2016 , Gu et al ., 2005 , Zhou et al ., 2015 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Besides pneumonia , SARS-CoV causes diarrhea [ 71 ] , lymphopenia [ 72 ] , haematological disorders [ 47 ] , pulmonary vasculitis , and thrombosis [ 73 , 74 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) causes bronchial epithelial denudation , loss of cilia , and multinucleated in lung tissues [ 1 , 2 ] , and induces lymphopenia , leucopenia , and thrombocytopenia in the SARS patients [ 3 , 4 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D008231 ! Lymphopenia)\n",
      "Furthermore , SARS-CoV causes lymphopenia which involves the depletion of T cells , probably by apoptotic mechanisms that are triggered by direct interactions of the SARS-CoV E protein with the antiapoptotic factor Bcl-xL .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009017 ! Morbidity)\n",
      "However , the epidemics of Middle East respiratory syndrome coronavirus ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-CoV ) caused alarming morbidity and mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009017 ! Morbidity)\n",
      "SARS-CoV moved from civet cats to humans and caused substantial morbidity and mortality before it was brought under control ( Poon et al. 2004 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "The novel coronavirus was named as the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 , 2019-nCoV ) due to its high homology ( ~ 80 % ) to SARS-CoV , which caused acute respiratory distress syndrome ( ARDS ) and high mortality during 2002-2003 [ 1 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "Coronaviruses and disease The human coronaviruses such as HCoVs include severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) , caused high mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "Coronaviruses and disease The human coronaviruses such as HCoVs include severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) , caused high mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "However , the epidemics of Middle East respiratory syndrome coronavirus ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-CoV ) caused alarming morbidity and mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "Background : SARS-CoV 2 , the agent responsible for the current pandemic , is also causing respiratory distress syndrome ( RDS ) , hyperinflammation and high mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "The mortality rate due to SARS-CoV in 2002-2003 was high , and interestingly , this virus also showed higher mortality in males compared to females .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "The mortality rate due to SARS-CoV in 2002-2003 was high , and interestingly , this virus also showed higher mortality in males compared to females .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "The mortality rate due to SARS-CoV in 2002-2003 was high , and interestingly , this virus also showed higher mortality in males compared to females .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "For pathogenesis studies , SARS-CoV need not cause mortality in the animal model , since the vast majority of human SARS cases are not fatal .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "MERS-CoV and SARS-CoV cause high mortality , most often resulting from progressive inflammatory viral pneumonia that clinically culminates in ARDS1 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "18 SARS-CoV causes high mortality ; thus , timely , sensitive , and specific diagnosis is needed to prevent the spread of this contagious disease .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "SARS-CoV moved from civet cats to humans and caused substantial morbidity and mortality before it was brought under control ( Poon et al. 2004 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "Generally , HCoVs cause milder respiratory diseases , whereas SARS-CoV and MERS-CoV , due to their high pathogenicity for humans , result in severe respiratory diseases and high mortality rates as in the case of SARS-CoV-2 ( 8,9 ) In Turkey , the first case of COVID-19 was announced on March 11 , 2020 and the first death was reported on March 17 , 2020 ( 10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "In Syrian hamsters , SARS-CoV replicates extensively in the respiratory tract in the first week after intranasal inoculation with 103 TCID50 [ 9 ] and , depending on the SARS-CoV strain used , can cause mortality in a small subset of infected animals [ 10 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "As a novel coronavirus , SARS-CoV-2 approximates to SARS-COV [ 2 ] and EBOV , which have caused large-scale infection and mortality .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009026 ! Mortality)\n",
      "The main protease ( Mpro ) of SARS-associated coronavirus ( SARS-CoV ) had caused a high rate of mortality in 2003 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009135 ! \"Muscular Diseases\")\n",
      "SARS coronavirus had been reported to cause neuropathies and myopathies ( Tsai et al ., 2004 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009165 ! \"Mycobacterium Infections, Nontuberculous\")\n",
      "In a murine model , it was confirmed that SARS-CoV causes an ACE2-dependent myocardial infection , as well as pulmonary infection ( Oudit et al. 2009a ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009165 ! \"Mycobacterium Infections, Nontuberculous\")\n",
      "In 2002-2003 , SARS-CoV affected 8 096 people , causing severe pulmonary infections and 774 deaths ( case fatality ratio : 10 % ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009165 ! \"Mycobacterium Infections, Nontuberculous\")\n",
      "In 2002-2003 , SARS-CoV affected 8 096 people , causing severe pulmonary infections and 774 deaths ( case fatality ratio : 10 % ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009165 ! \"Mycobacterium Infections, Nontuberculous\")\n",
      "In 2002-2003 , SARS-CoV affected 8 096 people , causing severe pulmonary infections and 774 deaths ( case fatality ratio : 10 % ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009165 ! \"Mycobacterium Infections, Nontuberculous\")\n",
      "In 2002-2003 , SARS-CoV affected 8 096 people , causing severe pulmonary infections and 774 deaths ( case fatality ratio : 10 % ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "ACE2 is also expressed in heart [ 23 ] , and SARS-CoV has been shown in animal models to directly mediate myocardial inflammation and damage by down-regulating myocardial ACE2 and lead to poor cardiac prognosis [ 24 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "SARS-CoV was reported to directly induce myocardial inflammation , and down-regulated myocardial ACE2 contributes to the myocardial abnormalities and cardiac consequences in SARS patients ( Oudit et al. 2009b ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "[ 14 ] Murine models and human autopsy samples demonstrate that SARS-CoV can down-regulate myocardial and pulmonary ACE2 pathways , thereby mediating myocardial inflammation , lung edema , and acute respiratory failure .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "Human autopsy samples show that SARS-CoV can downregulate myocardial and pulmonary ACE2 pathways , thereby mediating myocardial inflammation , pulmonary edema and acute respiratory failure [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "Studies have reported that SARS-CoV can cause myocardial injury and lead to myocardial inflammation [ 8] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009205 ! Myocarditis)\n",
      "SARS-CoV have been previously demonstrated to induce myocarditis in aACE2-dependent manner45 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009420 ! \"Nervous System\")\n",
      "Moreover , SARS-CoV was not observed to spread in nervous system through nerve endings of olfactory bulb .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009421 ! \"Nervous System Malformations\")\n",
      "SARS-CoV was found to trigger many neurological abnormalities such as encephalitis , aortic ischemic stroke , and polyneuropathy ( 86 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "It is important to mention that SARS-CoV ( outbreak in 2002 ) promoted orchitis , probably by inducing inflammatory and immunological host response that damages the gonads [ 18 ] ; considering that ACE2 is the entry receptor of both SARS-CoV and SARS-CoV , it cannot be excluded that the new coronavirus could act in a similar way [ 1 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "It is important to mention that SARS-CoV ( outbreak in 2002 ) promoted orchitis , probably by inducing inflammatory and immunological host response that damages the gonads [ 18 ] ; considering that ACE2 is the entry receptor of both SARS-CoV and SARS-CoV , it cannot be excluded that the new coronavirus could act in a similar way [ 1 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "Although they were unable to isolate a genomic signature from SARS-CoV itself , they postulated that SARS-CoV caused orchitis and reproductive impairment .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "In fact , SARS-CoV had been reported to cause orchitis based on post-mortem histopathological studies .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "In fact , SARS-CoV had been reported to cause orchitis based on post-mortem histopathological studies .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "Although they were unable to isolate a genomic signature from SARS-CoV itself , they postulated that SARS-CoV caused orchitis and reproductive impairment .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "The authors suggested that SARS-CoV may cause orchitis by the autoimmune mechanism in response to viral infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D009920 ! Orchitis)\n",
      "SARS-CoV caused orchitis with widespread destruction of germ cells , such that the number of spermatozoa was severely reduced in the seminiferous tubules of six men who died from the disease [ 18 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D010200 ! Panic)\n",
      "The World Health Organization International Health Regulations Emergency Committee declared COVID-19 outbreak a Global emergency [ 11 ] because SARS-CoV has spread rapidly within and outside China at an alarming pace and has caused considerable consternation and panic among the national , regional , and international public and political communities compounded by news media and social media hype [ 12 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D010612 ! Pharyngitis)\n",
      "The other three highly pathogenic viruses , SARS-CoV , MERS-CoV , and SARS-CoV-2 , cause severe respiratory syndrome , pneumonia , bronchiolitis , rhinitis , pharyngitis , sinusitis , and other systemic symptoms such as occasional watery diarrhea in humans of all ages .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D011024 ! \"Pneumonia, Viral\")\n",
      "At this stage , it is unclear whether the severe acute respiratory syndrome coronavirus ( SARS-CoV ) -2 outbreak will run its course , as SARS-CoV did in 2003 , or will become an endemic cause of viral pneumonia .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D011261 ! \"Pregnancy Trimester, First\")\n",
      "2 High rate ( 57 % ) of miscarriage in first trimester due to SARS-CoV has been documented during 20 02-20 03 epidemic .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D011261 ! \"Pregnancy Trimester, First\")\n",
      "Limited data is available regarding the impact of COVD-19 on pregnancy , however the case of miscarriage due to placental infection caused by SARS-CoV-2 in second trimester has already been reported.2 High rate ( 57 % ) of miscarriage in first trimester due to SARS-CoV has been documented during 2002-2003 epidemic.3 During the MERS-CoV epidemic , alarming rate of adverse pregnancy outcomes were reported previously .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D011654 ! \"Pulmonary Edema\")\n",
      "[ 14 ] Murine models and human autopsy samples demonstrate that SARS-CoV can down-regulate myocardial and pulmonary ACE2 pathways , thereby mediating myocardial inflammation , lung edema , and acute respiratory failure .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D011654 ! \"Pulmonary Edema\")\n",
      "Human autopsy samples show that SARS-CoV can downregulate myocardial and pulmonary ACE2 pathways , thereby mediating myocardial inflammation , pulmonary edema and acute respiratory failure [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012106 ! Research)\n",
      "Indeed , molecular methods have also been invaluable in identifying new respiratory viruses , the majority of which were described after the emergence of the severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) that prompted research into the etiology of ARI .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The novel coronavirus was named as the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 , 2019-nCoV ) due to its high homology ( ~ 80 % ) to SARS-CoV , which caused acute respiratory distress syndrome ( ARDS ) and high mortality during 2002-2003 [ 1 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "Patients included in the trial did not have acute respiratory distress syndrome due to SARS coronavirus 2003 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "More importantly , influenza virus ( H1N1 ( Narasaraju et al ., 2011 ) , H5N1 ( Sun et al ., 2012 ) , H9N2 ) and severe acute respiratory syndrome ( SARS ) virus ( Kuba et al ., 2005 ) might even induce deadly ALI and ARDS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause respiratory , intestinal , hepatic , and neuronal diseases and may lead to acute respiratory distress syndrome ( ARDS ) , multiple organ failure ( MOF ) , and even death in severe cases [ 13-15 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "More importantly , influenza virus ( H1N1 ( Narasaraju et al ., 2011 ) , H5N1 ( Sun et al ., 2012 ) , H9N2 ( Li et al ., 2016 ) ) and severe acute respiratory syndrome ( SARS ) virus ( Kuba et al ., 2005 ) might even induce deadly ALI and ARDS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV , MERS-CoV , and SARS-CoV-2 can cause respiratory , intestinal , hepatic , and neuronal diseases and may lead to acute respiratory distress syndrome ( ARDS ) , multiple organ failure ( MOF ) , and even death in severe cases [ 13 ] [ 14 ] [ 15 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "SARS-CoV and MERS-CoV , can lead to coronaviral pneumonia and ARDS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012220 ! Rhinitis)\n",
      "The other three highly pathogenic viruses , SARS-CoV , MERS-CoV , and SARS-CoV-2 , cause severe respiratory syndrome , pneumonia , bronchiolitis , rhinitis , pharyngitis , sinusitis , and other systemic symptoms such as occasional watery diarrhea in humans of all ages .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012640 ! Seizures)\n",
      "Upon the neural infection , the SARS-CoVs can elicit unregulatable neuroinflammatory responses thereby leading to seizures , convulsions , loss of consciousness , encephalomyelitis and encephalitis [ 7,34,36,37 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D012852 ! Sinusitis)\n",
      "The other three highly pathogenic viruses , SARS-CoV , MERS-CoV , and SARS-CoV-2 , cause severe respiratory syndrome , pneumonia , bronchiolitis , rhinitis , pharyngitis , sinusitis , and other systemic symptoms such as occasional watery diarrhea in humans of all ages .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D013577 ! Syndrome)\n",
      "The SARS-coronavirus ( SARS-CoV ) is proposed to be the cause of this syndrome on the basis of its association with human clinical cases ( 3,4 ) and reproduction of pulmonary lesions in experimentally challenged cynomolgus macaque monkeys ( Macaca fascicularis ) ( 5 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D013812 ! Therapeutics)\n",
      "SARS-CoV causes an acute infection and therefore therapies that offer instantaneous protection are likely to be most effective .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D013921 ! Thrombocytopenia)\n",
      "Severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) causes bronchial epithelial denudation , loss of cilia , and multinucleated in lung tissues [ 1 , 2 ] , and induces lymphopenia , leucopenia , and thrombocytopenia in the SARS patients [ 3 , 4 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D013927 ! Thrombosis)\n",
      "Besides pneumonia , SARS-CoV causes diarrhea [ 71 ] , lymphopenia [ 72 ] , haematological disorders [ 47 ] , pulmonary vasculitis , and thrombosis [ 73 , 74 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D014581 ! Urticaria)\n",
      "Urticaria is known to be caused by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D014774 ! Virulence)\n",
      "Influenza viruses , SARS coronavirus and Ebola virus have all been passaged in mice to enhance virulence ; the resulting host-adapted viruses have been studied biologically and used to evaluate strategies for control and prevention .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D014943 ! \"Global Health\")\n",
      "During the first two decades of the 21 st  century , there have been three coronavirus infection outbreaks raising global health concerns by severe acute respiratory syndrome coronavirus ( SARS-CoV ) , the Middle East respiratory syndrome coronavirus ( MERS-CoV ) , and the SARS-CoV-2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D015431 ! \"Weight Loss\")\n",
      "Remdesivir exhibits in vitro and in vivo antiviral activity against both SARS-CoV and MERS-CoV.52 Treatment of SARS-CoV infected mice with 25 mg / kg remdesivir twice daily mitigated SARS-CoV induced weight loss , reduced SARS-CoV-induced lung pathology , and significantly reduced pulmonary viral antigen levels .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D015726 ! \"Giant Cells\")\n",
      "The SARS-CoV has also been reported to cause the formation of syncytia in vivo , but so far only under the condition of cultured Vero cells [ 6 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D015726 ! \"Giant Cells\")\n",
      "As described before , SARS-CoV spike expression resulted in poor 94 syncytia formation while MERS-CoV spike produced much higher levels of syncytia ( Figure 1B ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D016676 ! \"Macrophages, Alveolar\")\n",
      "SARS-Co-V is an inefficient activator of alveolar macrophages and respiratory DCs , therefore impairing anti-viral responses such as IFNproduction , endocytosis and T cell responses [ 81 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D018149 ! \"Glucose Intolerance\")\n",
      "( 9 ) SARS-CoV was also shown to cause new onset DM or glucose intolerance in patients .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D018384 ! \"Oxidative Stress\")\n",
      "Very likely , SARS-CoV will also induce oxidative stress in the infected host cell ,59 even though its Nsp9 does not seem to form disulfide-linked dimers , at least not in vitro .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D018938 ! \"Caco-2 Cells\")\n",
      "Thus , SARS-CoV induces IP-10 gene expression in Caco-2 cells , but not in HEK 293 cells , again suggesting that the cytokine response is dependent on the host cell type .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D018938 ! \"Caco-2 Cells\")\n",
      "There was little induction of proinflammatory cytokines in Caco-2 cells by SARS-CoV-2 or SARS-CoV , even less so than MERS-CoV , whereas H1N1pdm and H5N1 viruses induced even higher levels of proinflammatory cytokine than MERS-CoV ( with the exception of IP-10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019636 ! \"Neurodegenerative Diseases\")\n",
      "Its ancestor , SARS-CoV has been known to attack the CNS , causing spine demyelinating diseases and neurodegenerative diseases [ 10 ] [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019636 ! \"Neurodegenerative Diseases\")\n",
      "Its ancestor , SARS-CoV has been known to attack the CNS , causing spine demyelinating diseases and neurodegenerative diseases [ 10 ] [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019636 ! \"Neurodegenerative Diseases\")\n",
      "Its ancestor , SARS-CoV has been known to attack the CNS , causing spine demyelinating diseases and neurodegenerative diseases [ 10 ] [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019636 ! \"Neurodegenerative Diseases\")\n",
      "Its ancestor , SARS-CoV has been known to attack the CNS , causing spine demyelinating diseases and neurodegenerative diseases [ 10 ] [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019851 ! Thrombophilia)\n",
      "Murine models suggest that SARS-CoV may interact with urokinase to produce a hyper-coagulable state as previously observed in the SARS-related acute lung injury [ 14 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019851 ! Thrombophilia)\n",
      "Murine models suggest that SARS-CoV may interact with urokinase to produce a hyper-coagulable state as previously observed in the SARS-related acute lung injury [ 14 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D019851 ! Thrombophilia)\n",
      "Murine models suggest that SARS-CoV may interact with urokinase to produce a hyper-coagulable state as previously observed in the SARS-related acute lung injury [ 14 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D020275 ! \"Guillain-Barre Syndrome\")\n",
      "Viral epidemics such as the Zika virus , middle east respiratory syndrome coronavirus ( MERS-CoV ) , SARS-CoV have triggered GBS [ [ 92 ] , [ 93 ] , [ 94 ] ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D021821 ! \"Communicable Diseases, Emerging\")\n",
      "A newly emerging infectious disease , severe acute respiratory syndrome ( SARS ) , is caused by SARS coronavirus ( SARS-CoV ) ( 27 , 46 , 58 ) , a zoonotic virus that most likely originated in its natural reservoir bats , through intermediate transmission such as via palm civets and raccoon dogs , and was finally transmitted to humans ( 21 , 37 , 38 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D037181 ! \"Lectins, C-Type\")\n",
      "SARS-CoV activates the lectin pathway by interacting with the complement component mannose-binding lectin ( MBL ) - - a C-type lectin that activates mannose-binding protein-associated serine protease 2 ( MASP-2 ) - - enhancing the deposition of C4 activation fragments on virus-infected cells140 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "12 Together , these findings prove that SARS-CoV is the primary cause of SARS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV that causes Severe Acute Respiratory Syndrome ( SARS ) was first identified in Asia in 2003 , and then has spread throughout the world .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV that causes Severe Acute Respiratory Syndrome ( SARS ) was first identified in Asia in 2003 , and then has spread throughout the world .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) causes SARS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "However , other coronaviruses cause severe respiratory 28 diseases ; Severe acute respiratory syndrome coronavirus ( SARS-CoV ) was discovered in 2003 29 and causes SARS , while Middle East respiratory syndrome coronavirus ( MERS-CoV ) was 30 discovered in 2012 and causes MERS .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Experimental infection of macaques has confirmed that the SARS-CoV is the cause of SARS ( 6 , 7 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Studies show that , like the severe acute respiratory syndrome coronavirus ( SARS-CoV ) that caused SARS , SARS-CoV-2 binds to human angiotensin-enzyme II ( ACE2 ) , using it as a cell entry receptor to invade respiratory and lung epithelium through the spike ( S ) protein ( Zhou et al ., 2020a ,2020 b ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Studies show that , like the severe acute respiratory syndrome coronavirus ( SARS-CoV ) that caused SARS , SARS-CoV-2 binds to human angiotensin-enzyme II ( ACE2 ) , using it as a cell entry receptor to invade respiratory and lung epithelium through the spike ( S ) protein ( Zhou et al ., 2020a ( Zhou et al ., ,2020 b .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS coronavirus ( SARS-CoV ) emerged in 2002 and caused severe acute respiratory syndrome ( SARS ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS coronavirus ( SARS-CoV ) emerged in 2002 and caused severe acute respiratory syndrome ( SARS ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS coronavirus ( SARS-CoV ) emerged in 2002 and caused severe acute respiratory syndrome ( SARS ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS , due to SARS-CoV was initially identified in 2002 , while MERS emerged in 2012 , which was caused by MERS-CoV infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "2 Both SARS-CoV and MERS-CoV are highly pathogenic and caused the 2003 SARS and 2012 MERS outbreaks .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In this article , we hypothesized the effects of SARS-CoV-2 taking into account what learned from the severe acute respiratory syndrome coronavirus ( SARS-CoV ) that caused SARS in 2003 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Based on the phylogenetic studies , the SARS-CoV-2 is categorized in the same lineage that includes SARS coronavirus ( SARS-CoV ) which causes Severe Acute Respiratory Syndrome ( SARS ) and MERS-CoV which causes Middle East respiratory syndrome ( Lokmana et al ., 2020 ; Raza et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "As shown in this table , SARS-CoV-2 can cause severe acute respiratory syndrome and has a 4.2 % mortality rate ; similarly , SARS-CoV and MERS-CoV can cause severe acute respiratory syndrome and have mortality rates of 11 % and 34 % , respectively ( Yang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "As shown in this table , SARS-CoV-2 can cause severe acute respiratory syndrome and has a 4.2 % mortality rate ; similarly , SARS-CoV and MERS-CoV can cause severe acute respiratory syndrome and have mortality rates of 11 % and 34 % , respectively ( Yang et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV is the cause of the severe acute respiratory syndrome , or ( SARS ) , whereas MERS-CoV was established as the cause of Middle East Respiratory Syndrome ( MERS ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The clinical and pathological features caused by 2019-nCoV resemble those seen in SARS 1-3 , which is caused by SARS coronavirus ( SARS-CoV ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Replication of the pulmonary lesions of SARS in experimentally infected macaques , together with the diagnosis of SARS-CoV infection in the majority of 436 SARS patients examined in detail , provided the final proof that SARS-CoV was the primary cause of SARS 9 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by SARS-CoV , which belongs to the Coronaviridae family .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by SARS-CoV , a previously unrecognized member of the coronavirus family.247 , 248 The incubation period from exposure to the onset of symptoms is typically 2 to 7 days , but can be as long as 10 days and in rare cases even longer .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Caused by a newly discovered SARS-associated coronavirus ( SARS-CoV ) , 4-7 SARS is characterized largely by constitutional symptoms ( fever , chills , myalgia ) and lower respiratory tract symptoms , from cough and dys-pnea , to rapidly progressive respiratory failure requiring intubation and mechanical ventilation .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS starting in 2002 in China due to SARS-CoV ; 2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by SARS-CoV , whereas SARS-CoV-2 causes COVID-19 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "This virus is highly homologous to two other coronaviruses that have emerged over the past two decades : SARS-CoV that causes Severe Acute Respiratory Syndrome ( SARS ) and MERS-CoV that causes Middle East Respiratory Syndrome ( MERS ) , both of which resulted in high mortality and morbidity .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by the SARS coronavirus ( SARS-CoV ) which is detectable in respiratory secretions of patients after infection ( Fouchier et al ., 2003 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS was determined to be caused by SARS-CoV and emerged in a market where civets were sold ( 2 , 3 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV-2 is closely related to SARS-CoV, which caused the 2003 SARS outbreak.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The SARS coronavirus ( SARS-CoV ) causes SARS , and is detectable in the respiratory secretions of patients after infection ( Foucher et al ., 2003 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Between November 2002 and July 2003 , SARS-CoV emerged , swept around the globe via routes of international air travel , and caused 8,098 SARS cases in 26 countries with 774 deaths .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV-2 exhibits 79.5 % sequence identity to SARS-CoV that causes SARS 20 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS ( Severe Acute Respiratory Syndrome ) was found to be due to a previously undescribed coronavirus ( SARS-CoV ) , with nosocomial and household transmission .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In recent decades , epidemics related to emergent coronaviruses have occurred : Severe acute respiratory syndrome due to SARS-CoV in 2002-2003 , Middle East respiratory syndrome caused by MERS-CoV ( 2012 , cases continue till present ) , and the new coronavirus causing COVID-19 , SARS-CoV-2 , which reached pandemic level on March 11 , 2020 [ 12-16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Caused by the SARS-associated coronavirus ( SARS-CoV ) , SARS is a newly emerged infectious disease that may recur at any time in the future [ Leong et al ., 2005 ; Surjit et al ., 2006 ; Chow and Leong , 2009 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS , which is caused by the SARS coronavirus ( SARS-CoV ) , emerged from China and caused nearly 8500 cases and 916 deaths during the outbreak in 2002 and 2003 [ 3-5 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "These findings suggest that SARS is caused by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The other two , SARS-CoV and MERS-CoV can infect the lower respiratory tract and cause severe acute respiratory syndrome in humans ( Paules , Marston , & Fauci , 2020 ; Chen , Liu , & Guo , 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The other two , SARS-CoV and MERS-CoV can infect the lower respiratory tract and cause severe acute respiratory syndrome in humans ( Paules , Marston , & Fauci , 2020 ; Chen , Liu , & Guo , 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by the SARS coronavirus and characterized by airborne transmission .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV is the cause of the severe acute respiratory syndrome , or ( SARS ) , whereas MERS-CoV was established as the cause of Middle East Respiratory Syndrome ( MERS ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Based on the phylogenetic studies , the SARS-CoV-2 is categorized in the same lineage that includes SARS coronavirus ( SARS-CoV ) which causes Severe Acute Respiratory Syndrome ( SARS ) and MERS-CoV which causes Middle East respiratory syndrome ( Lokmana et al ., 2020 ; Raza et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "* SARS is caused by SARS-coronavirus ( SARS-CoV ) , which first emerged in China and gained global notoriety in 2002 to 2003 causing a travel-related global outbreak with 8098 cases and 774 deaths .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Previous studies have shown that some viruses that primarily target the upper respiratory tract also affect the liver , such as SARS-CoV , which causes SARS , and MERS-CoV , which causes Middle East respiratory syndrome [ 8] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV-2 belongs to the family of single-stranded RNA viruses known as coronaviruses , including the MERS-CoV and SARS-CoV that cause Middle East respiratory syndrome ( MERS ) and severe acute respiratory syndrome ( SARS ) , respectively .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS starting in 2002 in China due to SARS-CoV ; 2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "For example , previous studies have shown that SARS-CoV that causes SARS may be transmitted via the fecal-oral route [ 15 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS is caused by the SARS coronavirus , a novel coronavirus .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "ACE2 is the receptor for SARS-COV, the virus that causes SARS, and is also believed to be the receptor for SARS-COV-2, the virus that causes COVID-19 [18].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "These findings suggest that SARS-CoV is the sole cause of SARS , without the involvement of coinfecting or passenger agents .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In April 2003 , WHO announced that a new pathogen , the SARS-coronavirus ( SARS-CoV ) , a member of the coronavirus family never seen before in humans , was the cause of SARS ( 2 , 3 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV which causes SARS , has a unique pathogenesis because it causes both upper and lower respiratory tract infections .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "It has been previously confirmed that SARS-CoV was the cause of SARS in 2003 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045473 ! \"SARS Virus\")\n",
      "At present there still lacks clinical proof that asymptomatic SARS patients could spread SARS coronavirus .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D045473 ! \"SARS Virus\")\n",
      "The result showed that 33G4 mediated SARS-CoV pseudovirus entry into CD32A-expressing cells but blocked SARS-CoV pseudovirus entry into ACE2-expressing cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D045949 ! Viverridae)\n",
      "Extensive epidemiological studies and wildlife surveillance revealed that SARS-CoV had originated in bats and somehow spread to civet cats .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D047928 ! \"Premature Birth\")\n",
      "[ 1 ] [ 2 ] [ 3 ] [ 4 ] [ 5 ] We know that the severe acute respiratory syndrome ( SARS-CoV ) outbreak in 2003 caused a 25 % of mortality rate in pregnant women , as well as an increase in premature delivery , but there was no evidence of vertical transmission .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D047928 ! \"Premature Birth\")\n",
      "We know that the severe acute respiratory syndrome ( SARS-CoV ) outbreak in 2003 caused a 25 % of mortality rate in pregnant women , as well as an increase in premature delivery , but there was no evidence of vertical transmission .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D053609 ! Lethargy)\n",
      "IT and IN inoculation of civets with SARS-CoV results in lethargy , decreased aggressiveness , fever , diarrhea , and conjunctivitis ( Wu et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D053676 ! Syntenins)\n",
      "SARS-CoV can also directly activate p38 or the SARS-CoV E protein can activate syntenin , which in tune activates p38 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D053676 ! Syntenins)\n",
      "SARS-CoV can also directly activate p38 or the SARS-CoV E protein can activate syntenin , which in tune activates p38 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D054875 ! Ubiquitination)\n",
      "Knowledge arising from the studies of other coronaviruses , such a s SARS-C o V , shows that SARS-CoV-2 may affect the expression of some PRRs , since SARS-CoV can craftily promote ubiquitination and thus enhance degradation of RIG-I and MDA5 RNA sensors , as well as TLR3 , TLR7 and TLR8 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "Previous study has shown that SARS-CoV can bind to its receptor ACE2 , resulting in increased angiotensin II ( ANG II ) level in mouse blood samples and induced acute lung injury , via angiotensin II type 2 receptor ( AT2 ) ( Imai et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "Previous study has shown that SARS-CoV can bind to its receptor ACE2 , resulting in increased angiotensin II ( ANG II ) level in mouse blood samples and induced acute lung injury , via angiotensin II type 2 receptor ( AT2 ) ( Imai et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "Previous study has shown that SARS-CoV can bind to its receptor ACE2 , resulting in increased angiotensin II ( ANG II ) level in mouse blood samples and induced acute lung injury , via angiotensin II type 2 receptor ( AT2 ) ( Imai et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "Both Influenza and SARS virus can induce acute lung injury ( ALI ) which is accompanied by high levels of C5a , leading to the influx and activation of innate immune cells ( 199 ) ( Figure 1 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "The induction of acute lung injury by acid aspiration , sepsis , or SARS-CoV resulted in the down-regulation of ACE2 expression in mouse lung tissue and the subsequent upregulation of AngII [ 33,34 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "[ 21 ] SARS-CoV decreases surface expression of ACE2 during infection Decreased ACE2 activity leads to increased Ang II and further downregulation of ACE2 in a vicious cycle , driving acute lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055501 ! \"Macrophage Activation Syndrome\")\n",
      "It is believed that macrophage activation syndrome is induced by SARS-CoV to be lethal .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D055501 ! \"Macrophage Activation Syndrome\")\n",
      "It is believed that macrophage activation syndrome is induced by SARS-CoV to be lethal .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D056647 ! \"Systemic Vasculitis\")\n",
      "SARS-CoV mainly targets the lungs , immune organs , and small systemic blood vessels and causes systemic vasculitis and decrease of immune function [ 99 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D057134 ! \"Antibodies, Neutralizing\")\n",
      "A number of structural proteins comprising spike ( S ) , envelope ( E ) , membrane ( M ) , and nucleocapsid ( N ) are expressed by SARS-CoV that may act as antigens to activate neutralizing antibodies and generate defensive response .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D057134 ! \"Antibodies, Neutralizing\")\n",
      "Reportedly , SARS-CoV and transmissible gastroenteritis virus produce no neutralizing antibodies after immunization of mice with a peptide containing a neutralizing epitope , but enhance immunogenicity and produce neutralizing antibodies when bound to a peptide with Th1 epitopes ( 26 , 27 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D057134 ! \"Antibodies, Neutralizing\")\n",
      "We previously demonstrated that the inactivated SARS-CoV was able to induce neutralizing antibodies in the immunized rabbits ( He et al ., 2004b ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058345 ! \"Asymptomatic Infections\")\n",
      "This finding suggests that SARS-CoV caused a subclinical infection or a very mild disease in mice .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058345 ! \"Asymptomatic Infections\")\n",
      "Epidemiological data have demonstrated that SARS-CoV mainly induces asymptomatic infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "Though the current pandemic is caused by SARS coronavirus 2 ( SARS-CoV-2 ) that is similar to the original SARS CoV , the outcomes have been radically different .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "Two highly pathogenic beta-CoVs , SARS-CoV , and MERS-CoV have caused pandemics in humans since 2002 [ 1,9 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "However , the severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) , two beta coronaviruses , caused pandemics with 37 % mortality rate ( 9 , 10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "However , the severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) , two beta coronaviruses , caused pandemics with 37 % mortality rate ( 9 , 10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "The first pandemic of the 21st century was caused by the SARS-CoV , which emerged in southern China in November 2002 and spread to 29 countries in five continents with 8096 confirmed cases and 774 deaths in 3 months .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "However , the severe acute respiratory syndrome coronavirus ( SARS-CoV ) and Middle East respiratory syndrome coronavirus ( MERS-CoV ) , two beta coronaviruses , caused pandemics with 37 % mortality rate ( 9 , 10 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "HCoV-229E , HCoV-OC43 , HCoV-HKU1 , and HCoV-NL63 are widespread in human populations and known to cause mild respiratory disease , while SARS-CoV and MERS-CoV had led to pandemics ( Channappanavar and Perlman 2017 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "1 The T cell response against the severe acute respiratory syndrome-related coronavirus ( SARS-CoV ) -2 virus , which causes the ongoing pandemic , is presumably initiated by respiratory professional antigen presenting cells ( APC ) that can engulf viral antigens as shown for the 2002/03 SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "1 The T cell response against the severe acute respiratory syndrome-related coronavirus ( SARS-CoV ) -2 virus , which causes the ongoing pandemic , is presumably initiated by respiratory professional antigen presenting cells ( APC ) that can engulf viral antigens as shown for the 2002/03 SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "In this review , we look for the diversity and variability of coronaviruses , including SARS-CoV , MERS-CoV and SARS-CoV-2 , which cause pandemics from 21th century .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "The result of this study may help in identifying significant targets of treatment that can combat the ongoing pandemic due to SARS-CoV - $ 2 $ infection .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D058873 ! Pandemics)\n",
      "SARS-CoV caused the first pandemic of the 21st century , resulting in c. 8400 human cases and an 11 % case fatality rate ( SARS Epidemiology Working Group , 2003 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "While in those cells lacking NLRP3 accumulating ORF8b cytosolic aggregates cause ER stress, mitochondrial dysfunction, and caspase-independent cell death.In 2002-2003 the SARS-CoV caused a severe respiratory illness affecting more than 8000 individuals with a mortality rate near 10% 1,2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "During the replication of SARS-CoV, a large amount of viral proteins are processed in the ER (Stadler et al., 2003); the accumulation of nascent and unfolded SARS-CoV proteins in the ER lumen may cause ER stress (Chan et al., 2006).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "During the replication of SARS-CoV, a large amount of viral proteins are processed in the ER (Stadler et al., 2003) ; the accumulation of nascent and unfolded SARS-CoV proteins in the ER lumen may cause ER stress .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "20 Additionally, overexpression of SARS-CoV ORF8b protein elevates ER stress effector CHOP, which leads to cell apoptosis in HeLa cells.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "Further, it has been shown that SARS-CoV structural proteins S, 6, and 3a [66], [78], [79], [80], and the accessory protein 8ab [81] induce ER stress responses.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "the hemagglutinin protein of the influenza virus and the spike protein of SARS-CoV, causing ER stress and initiating the UPR (Chan et al., 2006; Watowich et al., 1991) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D059865 ! \"Endoplasmic Reticulum Stress\")\n",
      "The SARS-CoV E protein alone was sufficient to reduce the expression of GRP78 and GRP94 (two ER stress inducible proteins) [11], whereas the PEDV E protein could be located in the ER, and then induced ER stress and up-regulated GRP78 alone [20], suggesting that the E proteins from diverse Coronavirus viruses might employ different mechanisms to mediate the stress response.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases bp(MESH:D060085 ! Coinfection)\n",
      "Poliovirus can be transmitted from person to person by close contact , either through upper respiratory tract secretions or the fecal-oral route [ 6 ] , similarly to SARS-CoV [ 7 ] , which could enable co-infection with poliovirus and SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases a(TEXT:Angiotensin-2)\n",
      "[ 23 ] Subsequently and most critically , Kuba et al. demonstrated SARS-CoV spikeprotein augments Ang II increase and ACE2 downregulation , with resultant lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(UP:P0CW71 ! P0CW71)\n",
      "We have previously demonstrated that IP-10 and MCP-1 are up-regulated in primary human macrophage by SARS-CoV [ 28 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(UP:P0CW71 ! P0CW71)\n",
      "The degree of induction by MERS-CoV and SARS-CoV on IL-8 , MCP-1 , and MIP-1alpha were subtle with comparable levels .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "It has been reported that SARS-CoV inhibits the expression of ACE2 in the lung after infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "More interestingly, the spike protein of SARS-CoV has been previously demonstrated to downregulate ACE2 expression, thus resulting in over-production of AngII (the downstream of ACE2) by the related enzyme ACE.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Previous studies have shown that SARS-CoV significantly reduces ACE2 in the lungs of infected mice [ 31 , 33 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "By binding ACE2 , SARS-CoV leads to the downregulation of ACE2 expression and might therefore negate the protective effect of ACE2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Downregulation of ACE-2 by SARS-CoV could accelerate the SARS pathogenesis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "[ 23 ] Subsequently and most critically , Kuba et al. demonstrated SARS-CoV spikeprotein augments Ang II increase and ACE2 downregulation , with resultant lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV reduces ACE2 expression , causing an imbalance between the ACE / Ang II / AT1R axis and the ACE2 / Ang ( 1-7 ) / Mas receptor axis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "The mechanism by which SARS-CoV downregulates ACE2 may be relevant to the development of novel therapeutics , as well .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "[ 23 , 35 ] The mechanism by which SARS-CoV downregulates ACE2 may be relevant to the development of novel therapeutics , as well .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "275 SARS-CoV tends to downregulate ACE2 , thus increasing the expression of cytokines .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Since in SARS-CoV infection, the spike protein of SARS-CoV inhibits ACE2 to cause severe lung injury and acute respiratory failure10,11, it is highly likely that SARS-CoV-2 uses the same mechanism.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "By binding ACE2 , SARS-CoV leads to the downregulation of ACE2 expression and might therefore negate the protective effect of ACE2 145,149 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "By binding ACE2 , SARS-CoV leads to the downregulation of ACE2 expression and might therefore negate the protective effect of ACE2 145,149 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "By binding ACE2 , SARS-CoV leads to the downregulation of ACE2 expression and might therefore negate the protective effect of ACE2 [ 145 , 149 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Angiotensinconverting enzyme 2 ( ACE2 ) , a cell membrane bound protein used by SARS-CoV-2 to enter cells , also inactivates bradykinin degradation products , and expression of ACE2 as well as its enzymatic activity is decreased by SARS-CoV ( 14 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Angiotensin-converting enzyme 2 ( ACE2 ) , a cell membrane bound protein used by SARS-CoV-2 to enter cells , also inactivates bradykinin degradation products , and expression of ACE2 as well as its enzymatic activity is decreased by SARS-CoV ( 14 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV downregulates the activity of ACE2 and causes an increase in the amount of Ang II and lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "The question of whether coronaviruses have evolved to exploit such a network may be relevant to pursue , particularly given that downregulation of ACE2 by SARS-CoV has been reported previously and is associated with poor clinical outcomes 31 , 47 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "The question of whether coronaviruses have evolved to exploit such a network may be relevant to pursue , particularly given that downregulation of ACE2 by SARS-CoV has been reported previously and is associated with poor clinical outcomes ( Glowacka et al ., 2010 ; Kuba et al ., 2010 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Further supporting this concept are the reported findings of downregulation of ACE2 by SARS-CoV , and it has been suggested that this might be similar in SARS-CoV-2 ( Fu et al ., 2020 ; Glowacka et al ., 2010 ; Levi et al ., 2019 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Once replicating within a cell , SARS-CoV downregulates expression of ACE2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "A recent report in Nature Medicine shows that SARS coronavirus downregulates ACE2 [ 2 ] so probably exaggerating the ARDS response by decreasing inactivation of angiotensin II by ACE 2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "However , it is largely unclear how SARS-CoV decreases ACE2 expression and the effect of NL63 on ACE2 levels has not been systematically investigated .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV was found to downregulate ACE2 by binding to it by its spike protein and inflicting severe lung damage ( Kuba et al ., 2005 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV reduced the expression levels of ACE2 in ( A ) Vero E6 cells and ( B ) mice lungs ( * * * P < 0.001 , * P < 0.05 vs. Control ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Notably , it has been shown that SARS-CoV spike could reduce ACE2 expression , resulting in the overproduction of ATII by the related enzyme ACE ( 69 , 70 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Notably , it has been shown that SARS-CoV spike could reduce ACE2 expression , resulting in the overproduction of ATII by the related enzyme ACE ( 69 , 70 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Given that ACE2 downregulation by SARS-CoV is linked to viral replication efficiency ,33 34 it is conceivable that ACE2 augmentation may reduce virulence .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV downregulated ACE2 in the model of respiratory failure , and the viral spike protein induced TNF-alpha-converting enzyme-dependent shedding of the ACE2 ectodomain and led to tissue damage [ 26 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV downregulated ACE2 in the model of respiratory failure , and the viral spike protein induced TNF-alpha-converting enzyme-dependent shedding of the ACE2 ectodomain and led to tissue damage [ 26 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Studies have shown that SARS-Co-V downregulates the ACE2 [ 31 ] and SARS-CoV-2 infection is associated with increased level of angiotensin II [ 32 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Studies have shown that SARS-Co-V downregulates the ACE2 [ 31 ] and SARS-CoV-2 infection is associated with increased level of angiotensin II [ 32 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "Studies have shown that SARS-Co-V downregulates the ACE2 [ 31 ] and SARS-CoV-2 infection is associated with increased level of angiotensin II [ 32 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV has been found to downregulate ACE2 and SARS - CoV-2 infection has been found to be associated with increased level of Angiotensin II. Based on these facts , we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2 , and in absence / reduced activity of ACE2 , level of angiotensin ( 1-7 ) and angiotensin ( 1-9 ) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "54 In a study on acid-induced lung injury in mice , ACE2 downregulation by SARS-CoV worsened lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:13557 ! ACE2)\n",
      "The finding of enhanced RAS signaling and severe acute lung injury as consequences of ACE2 down-regulation by SARS-CoV might possibly implicate that other emerging infectious lung diseases , like H5N1 avian flu , may also utilize the receptor down-regulation system to impair host immune / inflammatory systems and induce severe lung diseases .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:6118 ! IRF3, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "In vitro data have shown that the expression of the membrane anchored PLpro domain (PLpro-TM) of SARS-CoV prevents STING/TBK1/IKKepsilon based stimulation of IFN-I and disrupts the phosphorylation and dimerization of IRF3, which are triggered via STING (stimulator of interferon genes) and TBK1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") increases p(HGNC:6001 ! IL2)\n",
      "In contrast, the Ac-GFP and PBS groups produced no significant amount of IFN-γ or IL-2 producing CD4+ and CD8+ T cells.Since the outbreak of SARS and the identification of SARS-CoV as the causative agent, much effort has been devoted to understanding viral pathogenesis and to developing treatment strategies.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:\"IRF3/7\")\n",
      "Also SARS-CoV inhibits activation of IRF3/7, slowing IFNb production upon infection (70).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:141213 ! isoxaflutole)\n",
      "Moreover , a preclinical model showed that SARS-CoV results in disruption of the fine balance between plasmin and the urokinase pathway , resulting in fibrin accumulation [ 40 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58-60 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58-60 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58 ) ( 59 ) ( 60 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:34922 ! picloram)\n",
      "It has been shown that the IFN pathway is inhibited by SARS-CoV-2 to an extent similar to the inhibition by MERS-CoV and SARS-CoV , thereby impairing both innate T cell antiviral responses ( 58 ) ( 59 ) ( 60 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:63580 ! ribavirin)\n",
      "As shown in XREF_FIG B and D, mizoribine inhibited plaque formation by SARS-CoV more strongly than ribavirin in plaque reduction assay.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:8347 ! Povidone-iodine)\n",
      "We offer suggestions to deal with these uncertainties.Topical povidone-iodine is virucidal with in vitro testing of 0.23% povidone-iodine producing a 10,000-fold reduction in the viral load of SARS-CoV and MERS-CoV within 15 s [17–19].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(CHEBI:8347 ! Povidone-iodine)\n",
      "We offer suggestions to deal with these uncertainties.Topical povidone-iodine is virucidal with in vitro testing of 0.23% povidone-iodine producing a 10,000-fold reduction in the viral load of SARS-CoV and MERS-CoV within 15 s [17] [18] [19] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:host)\n",
      "For example , severe 111 acute respiratory syndrome coronavirus ( SARS-CoV ) and MERS-CoV nsp1 suppresses 112 host gene expression by mediating host mRNA degradation [ 38 ] ; the PLpro nsp3 of 113 SARS-CoV and MERS-CoV harbors deubiquitinase activity and interferes with type I 114 IFN responses [ 39 , 40 ] ; feline infectious peritonitis coronavirus ( FCoV ) and porcine 115 deltacoronavirus ( PDCoV ) nsp5 inhibits type I IFN response by cleaving NEMO [ 41 , the first identified coronavirus in 1930s and infects avian species [ 45 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:host)\n",
      "Furthermore , not only are there many gaps in our understanding of how the few viral proteins actually function and interact with host cell proteins but also there are often multiple mechanisms and alternative molecular pathways used by the virus , for example inhibition of host gene expression and translation by SARS coronavirus ( Lokugamage et al. 2012 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases a(TEXT:host)\n",
      "The nsp1 of severe acute respiratory syndrome coronavirus ( SARS-CoV ) significantly inhibited host gene expression , including that of type I interferon ( IFN ) , by binding to the ribosome , which led to endonucleolytic cleavage and subsequently promoted host endogenous mRNA degradation in vitro [ [ 11 ] , [ 12 ] , [ 13 ] ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:AKT ! AKT)\n",
      "For example , the membrane protein of the severe acute respiratory syndrome coronavirus ( SARS-CoV ) inhibits Akt activation , which causes cytopathic effects in infected cells [ 33 , 34 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:IFNB ! IFNB)\n",
      "SARS-CoV has been also reported to suppress type I IFNbeta production .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:RLR ! RLR)\n",
      "SARS-CoV and other coronaviruses replicate in the interior of double membrane vesicles to prevent RLR activation by dsRNA replication intermediates ( Stertz et al ., 2007 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(FPLX:TLR ! TLR)\n",
      "Indeed, both the induction of IFN-alpha or beta and the ability to restrict both MERS-CoV and SARS-CoV infection requires TLR3 signaling, and SARS-CoV is able to antagonize the TLR signaling pathway via its papain like protease (PLpro) (54, 55).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006413\" ! \"translational initiation\")\n",
      "SARS-CoV nsp1 inhibits cap-dependent and IRES-driven translation by suppressing multiple steps of translation initiation (Lokugamage et al., 2012).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006897\" ! endocytosis)\n",
      "T20 (enfuvirtide) which is approved by USFDA against AIDS may significantly interact with the HR2 region of the SARS-CoV by inhibiting endocytosis into the host cell.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "As ion channel activity has been reported to regulate apoptosis in different pathologic conditions, finding ways to modulate the ion channel activity may offer a new direction toward the inhibition of apoptosis triggered by SARS-CoV.Severe acute respiratory syndrome-coronavirus (SARS-CoV) is the etiological agent causing the global outbreak of atypical pneumonia in 2003.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Overexpression of SARS-CoV nsp15 inhibits the IFN response and 395 MAVS-mediated apoptosis [66].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "In several studies it was shown that SARS-CoV can both induce and inhibit apoptosis in a cell-type specific manner.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "SARS-CoV ORF7a protein activates the intrinsic pathway of apoptosis through an interaction with anti-apoptotic protein Bcl-XL in the endoplasmic reticulum, thereby sequestering a key suppressor of apoptosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "It is conceivable that in addition to their antiviral effects, calpain inhibitors also prevent cellular lysis by inhibition of apoptosis which may be induced by SARS-CoV in infected cells (Yan et al., 2004, Tan et al., 2004b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "In COS-1 cells, SARS-CoV N protein expression down-regulated Bcl-2 levels and induced apoptosis in the absence of growth factors [Surjit et al., 2004a] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "When the NSP15 protein encoded by the SARS-CoV, HKU1 or NL63 genome was coexpressed with MAVS, only SARS-CoV NSP15 inhibited MAVS-mediated apoptosis as measured by Annexin V and 7-AAD staining, while the others showed little effect on cell viability (Fig. 7D).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "It is conceivable that in addition to their antiviral effects , calpain inhibitors also prevent cellular lysis by inhibition of apoptosis which may be induced by SARS-CoV in infected cells ( Yan et al ., 2004 , Tan et al ., 2004b ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "As ion channel activity has been reported to regulate apoptosis in different pathologic conditions, finding ways to modulate the ion channel activity may offer a new direction toward the inhibition of apoptosis triggered by SARS-CoV.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "SARS-CoV N protein modulates the TGF-b signaling pathway to block apoptosis of SARS-CoV-infected host cells (Zhao et al. 2008) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "SARS-CoV N protein modulates the TGF-β signaling pathway to block apoptosis of SARS-CoV-infected host cells (Zhao et al. 2008).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "In several studies it was shown that SARS-CoV can both induce and inhibit apoptosis in a cell-type specific manner .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "C. Apoptosis induced by the nuclear blocked SARS-CoV 9b protein is caspase 3 dependent.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "doi:10.1371/journal.pone.0005466.g006The SARS-CoV NSP15 protein abrogates MAVS-induced apoptosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "The anti-apoptotic SARS-CoV protein NSP15 prevents apoptosis as a result of MAVS expression, but does not mediate apoptosis induced by staurosporine treatment, indicating that it has evolved as a MAVS-specific strategy (Lei et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "The exposure of SARS-CoV infected cells to Z-VAD-FMK completely prevented apoptosis (not shown).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "SARS-CoV, on the other hand, has been shown to attenuate the IRE1 signaling pathway, which generally led to the down‑regulation of stress responses and UPR, and further decreased apoptotic signaling in infected cells as well, to the likely benefit of the virus for better viral progeny production and release [129].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "In addition, SARS-CoV E protein down-regulated the signaling pathway inositol-requiring enzyme 1 (IRE-1) of the unfolded protein response, and limited cell apoptosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "22 Moreover, two other SARS-CoV proteins, namely Orf3b and Orf6, also antagonise IFN synthesis and signalling pathway.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "Recently, it was demonstrated that SARS-CoV does not only prevent the induction of type-I interferons, but that several SARS-CoV proteins also block different steps in the interferon signaling pathway.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "SARS-CoV and MERS-CoV are able to disrupt the interferon signaling pathway.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "In addition, SARS-CoV E protein down-regulated the signaling pathway inositol-requiring enzyme 1 (IRE-1) of the unfolded protein response, but not the PKR-like ER kinase (PERK) or activating transcription factor 6 (ATF-6) pathways, and reduced cell apoptosis.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0007165\" ! \"signal transduction\")\n",
      "One caveat to this work is that SARS-CoV could be inhibiting IFN signaling pathways that are detrimental to its own replication and survival but not inhibiting different pathways inducible by SeV and poly-I :C .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "A recent investigation further showed that SARS-CoV nucleocapsid inhibited Type I interferon production by interfering with tripartite motif protein 25 (TRIM25)-mediated RIG-I ubiquitination [96].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "Type I IFN production induced by poly I·C or Sendai virus (SeV) was suppressed by the SARS-CoV N protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "SARS-CoV nsp1 inhibits type I IFN production, induces host mRNA degradation, and suppresses host protein translation (Narayanan et al., 2008).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "Notably, a recent study using SARS-CoV lacking Nsp16, which acts as a 2′ O-MTase, showed that loss of 2′ O-methylation of the SARS-CoV genome enhanced host type I IFN production [13].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "SARS-CoV ORF3b and ORF6 have been found to antagonize type I IFN production and signaling.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "The induction of IFN-β in these SARS-CoV–infected macaques is surprising, because several reports have shown that SARS-CoV inhibits or delays type I IFN production in a number of cell types [14–18,20,22].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0032606\" ! \"type I interferon production\")\n",
      "The induction of IFN-b in these SARS-CoVinfected macaques is surprising, because several reports have shown that SARS-CoV inhibits or delays type I IFN production in a number of cell types [14] [15] [16] [17] [18] 20, 22] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0036074\" ! \"metaplastic ossification\")\n",
      "SARS-CoV infection induces severe respiratory illnesses, such as bronchial epithelial denudation, loss of cilia, multinucleated syncytial cells, squamous metaplasia and transendothelial migration of monocytes/macrophages and neutrophils into lung tissue 3,4 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "In the proposed model of respiratory failure , SARS-CoV down-regulates its receptor , angiotensin-converting enzyme 2 ( ACE2 ) , but the mechanism involved is unknown .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0040007\" ! growth)\n",
      "Although the development of autoimmune antibodies or immune complexes triggered by viral infection may play a significant role in inducing thrombocytopenia , SARS-CoV may also directly infect haematopoietic stem / progenitor cells , megakaryocytes and platelets , inducing their growth inhibition and apoptosis [ 11 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0043066\" ! \"negative regulation of apoptotic process\")\n",
      "Severe acute respiratory syndrome coronavirus (SARS-CoV) 7a protein has been reported to directly inhibit Bcl-XL pro-survival activity, along with its ability to bind all other pro-survival members (Fig. 1) [7].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051092\" ! \"positive regulation of NF-kappaB transcription factor activity\")\n",
      "Fang X, Gao J, Zheng H, Li B, Kong L, Zhang Y, Wang W, Zeng Y, Ye L: The membrane protein of SARS-CoV suppresses NF-kappaB activation.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV ORF6 blocks STAT1 nuclear translocation by trapping the nuclear import factors in the endoplasmic reticulum and Golgi apparatus .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV ORF6 blocks STAT1 nuclear import [18] , PLP blocks IRF3 activation [51] , NSP7 [51] , NSP15 [51] , ORF3b and N have been shown to be IFN antagonists as well [19] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "In particular, inhibition of nuclear import by SARS-CoV ORF6 has been shown to mediate pleiotropic alterations in host gene expression during infection [50] that enhance viral pathogenesis in vivo [51].\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV also inhibits the nuclear import of the related transcription factor IRF7 (Kuri et al., 2009).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "Frieman M, Yount B, Heise M, Kopecky-Bromberg SA, Palese P, Baric RS: SARS-CoV ORF6 antagonizes STAT1 function by sequestering nuclear import factors on the rER/Golgi membrane.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV ORF6 blocks nuclear import of Stat1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV ORF6 protein effectively disrupts nuclear import of phosphorylated-STAT1 by tethering PY-STAT1-Kapα/Kapβ complex to ER/Golgi membranes.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "SARS-CoV ORF6 blocks STAT1 nuclear translocation by trapping the nuclear import factors in the endoplasmic reticulum and Golgi apparatus (Frieman et al., 2007).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "8: SARS-CoV ORF6 blocks nuclear import of Stat1.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(GO:\"GO:0051170\" ! \"import into nucleus\")\n",
      "Among the extremely pathogenic RNA viruses, SARS-CoV ORF6 protein was shown to block nuclear import by inhibiting karyopherin-α2 in the cytoplasm.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:10489 ! S100A12)\n",
      "For example , SARS-CoV was shown to block STAT1 translocation using p6 [ 82 ] , while MERS-CoV did not inhibit STAT1 translocation after cells were treated with 1000 ng PEG-IFN ml-1 [ 71 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:11362 ! STAT1)\n",
      "Interestingly , SARS-CoV inhibits STAT1 signaling in three different ways .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:11364 ! STAT3)\n",
      "Thus , the mechanism of dysregulation of STAT3 by SARS-CoV is different from that by measles virus in which Measles virus V protein forms complexes with STATs resulting in degradation of STAT3 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:11998 ! TP53)\n",
      "Therefore , SARS-CoV , MERS-CoV , and SARS-CoV-2 PLP are all able to mediate p53 degradation in a process that lead to decreased cell apoptosis favoring viral growth in infected cells as well as to the loss of the antiviral activity of p53 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:11998 ! TP53)\n",
      "Therefore , SARS-CoV , MERS-CoV , and SARS-CoV-2 PLP are all able to mediate p53 degradation in a process that lead to decreased cell apoptosis favoring viral growth in infected cells as well as to the loss of the antiviral activity of p53 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:19391 ! SOCS3)\n",
      "SARS-CoV enhances IL-6 production and decreases the expression levels of SOCS3 [ 75 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:2707 ! ACE)\n",
      "Thus , one could target this interaction site with antibodies or small molecules.Delivering excessive soluble form of ACE2.Kuba et al. [ 10 ] demonstrated in mice that SARS-CoV downregulates ACE2 protein ( but not ACE ) by binding its spike protein , contributing to severe lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:2707 ! ACE)\n",
      "Kuba et al. [ 10 ] demonstrated in mice that SARS-CoV downregulates ACE2 protein ( but not ACE ) by binding its spike protein , contributing to severe lung injury .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:2707 ! ACE)\n",
      "In the proposed model of respiratory failure , SARS-CoV down-regulates its receptor , angiotensin-converting enzyme 2 ( ACE2 ) , but the mechanism involved is unknown .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:2973 ! DNM1L)\n",
      "The open reading frame 9b ( ORF-9b ) of severe acute respiratory syndrome-coronavirus ( SARS-CoV ) activates the ubiquitin pathway and induces Drp1 degradation in a proteasome-dependent manner .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:31603 ! MIR223)\n",
      "Interestingly , bronchoalveolar stem cells ( BASCs ) - - `` microbiome '' that reveal the interaction between microRNAs in stem cells and viral genome has indicated that the miR-223 was inhibited by nucleocapsid and spike protein by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:31603 ! MIR223)\n",
      "Interestingly , bronchoalveolar stem cells ( BASCs ) - - `` microbiome '' that reveal the interaction between microRNAs in stem cells and viral genome has indicated that the miR-223 was inhibited by nucleocapsid and spike protein by SARS-CoV .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases p(HGNC:7097 ! MIF)\n",
      "In Caco-2 cells , SARS-CoV down-regulated MIF .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "A recent study showed that the binding affinity of SARS-CoV-2 to ACE2 is comparable to that of SARS-CoV, as plasma containing antibodies to the SARS-CoV reduced the transduction efficiency of SARS-CoV-2 in cells (5).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Houttuyniae Herba also exhibited significant inhibitory effects against SARS-CoV via suppressing the activities of SARS-CoV 3CLpro and RdRp 92 ., natural compounds such as theaflavin and cepharanthine suppressed SARS-CoV-2 by suppressing RdRp and ACE activities; hirsutenone and tanshinone I-VII showed antiviral action against SARS-CoV via inhibiting the PLpro activity, while celastrol, pristimererin, tingenone, iguesterin, chalcones I-IX and quercetin-3-β-galactoside were able to inhibit SARS-CoV by suppressing the 3CLpro activity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Here, we present data assessing the antiviral activity of 20 FDA approved drugs against SARS-CoV-2 that also inhibit SARS-CoV and MERS-CoV  in vitro  We find 17 of these inhibit SARS-CoV-2, suggesting these may have pan-anti-coronaviral activity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Here, we present data on the antiviral activity of 20 FDA approved drugs against SARS-CoV-2 that also inhibit SARS-CoV and MERS-CoV.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Full-length product formation is generally more pronounced under these conditions, following the order MERS-CoV RdRp Ͼ SARS-CoV RdRp Ͼ EDITORS' PICK: SARS-CoV-2 polymerase inhibition with remdesivir SARS-CoV-2 RdRp.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Similarly, SARS-CoV RBD protein blocked the entry of both SARS-CoV pseudovirus and SARS-CoV-2 pseudovirus into hACE2-expressing 293T cells with IC50 of 4.1 and 11.63 µg/ml, respectively (Fig. 4b).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Since SARS-CoV-2 shares 79% sequence identity with SARS-CoV, several anti-SARS-CoV drugs have shown promise in limiting SARS-CoV-2 replication in vitro and in vivo.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "8 Hoffmann et al utilized this information and found that convalescent SARS-CoV serum, which is known to have a neutralizing antibody against the viral S protein, decreased SARS-CoV-2 entry into the cell in vitro.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Hoffmann et al found that SARS-CoV-2 uses the SARS-CoV angiotensin converting enzyme 2 (ACE2) receptor to enter cells and the serine protease TMPRSS2 for S protein priming.8 Hoffmann et al utilized this information and found that convalescent SARS-CoV serum, which is known to have a neutralizing antibody against the viral S protein, decreased SARS-CoV-2 entry into the cell in vitro.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "In SARS-CoV, E918 in the HR1 domain binds to R1166 in the HR2 domain211224 effectively inhibit SARS-CoV-2 PsV infection.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\")\n",
      "Houttuyniae Herba also exhibited significant inhibitory effects against SARS-CoV via suppressing the activities of SARS-CoV 3CLpro and RdRp 92 ., natural compounds such as theaflavin and cepharanthine suppressed SARS-CoV-2 by suppressing RdRp and ACE activities; hirsutenone and tanshinone I-VII showed antiviral action against SARS-CoV via inhibiting the PLpro activity, while celastrol, pristimererin, tingenone, iguesterin, chalcones I-IX and quercetin-3-β-galactoside were able to inhibit SARS-CoV by suppressing the 3CLpro activity.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D001402 ! B-Lymphocytes)\n",
      "The fact that 76 % of our patients with SARS had low B cell counts could imply that B cells were damaged by SARS-CoV , although the damage was not too severe to affect the production of antibody .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D002415 ! Cats)\n",
      "Experimentally , SARS-CoV has been shown to cause respiratory damage in both cats and ferrets , though only ferrets demonstrated clinical disease ( van den Brand et al ., 2008 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D002415 ! Cats)\n",
      "Experimentally , SARS-CoV has been shown to cause respiratory damage in both cats and ferrets , though only ferrets demonstrated clinical disease ( Brand et al ., 2008 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D002923 ! Cilia)\n",
      "Severe acute respiratory syndrome-associated coronavirus ( SARS-CoV ) causes bronchial epithelial denudation , loss of cilia , and multinucleated in lung tissues [ 1 , 2 ] , and induces lymphopenia , leucopenia , and thrombocytopenia in the SARS patients [ 3 , 4 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D004194 ! Disease)\n",
      "SARS-CoV may disrupt specific cellular pathways to suppress host immune responses , resulting in disease spreading .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D004194 ! Disease)\n",
      "Thus , it is thought that emodin could eliminate SARS-CoV triggered disease conditions by acting as a competitor for the same binding region of S-protein with ACE2.46 Promazine is an anti-psychotic medication that shares a structural similarity with emodin .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005289 ! Ferrets)\n",
      "Experimentally , SARS-CoV has been shown to cause respiratory damage in both cats and ferrets , though only ferrets demonstrated clinical disease ( Brand et al ., 2008 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005289 ! Ferrets)\n",
      "Experimentally , SARS-CoV has been shown to cause respiratory damage in both cats and ferrets , though only ferrets demonstrated clinical disease ( van den Brand et al ., 2008 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005298 ! Fertility)\n",
      "Recent evidence reports that SARS coronavirus could also lead to a fertility damage , despite the absence of a direct mumps-like orchitis .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005854 ! \"Germ Cells\")\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005854 ! \"Germ Cells\")\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005854 ! \"Germ Cells\")\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D005854 ! \"Germ Cells\")\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D006321 ! Heart)\n",
      "Considering this fact , Zheng et al. ( 2020 ) pointed that Middle East respiratory 45 syndrome-related coronavirus ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-Cov ) can cause acute myocarditis and heart failure .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D006321 ! Heart)\n",
      "In addition , SARS-CoV 2 had a stronger interaction with ACE2 than SARS-CoV 26 , and may directly or indirectly cause heart damage through ACE2-related pathways .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D006321 ! Heart)\n",
      "In December 2019 , started perhaps the most critical outbreak of the recent hundred years : the rapid widespread of the coron - ( MERS-CoV ) and severe acute respiratory syndrome coronavirus ( SARS-Cov ) can cause acute myocarditis and heart failure ( Zheng et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007111 ! \"Immunity, Cellular\")\n",
      "Our results indicate that SARS-CoV seem to elicit effective humoral immunity but inhibit cellular immunity .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007111 ! \"Immunity, Cellular\")\n",
      "Conclusion : SARS-CoV seemed to elicit effective humoral immunity but inhibited cellular immunity , especially CD8 + memory T lymphocytes over time .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007422 ! Intestines)\n",
      "It is currently known that in addition to damaging the respiratory tract , SARS-CoV also damages the intestine .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007422 ! Intestines)\n",
      "It is currently known that in addition to damaging the respiratory tract , SARS-CoV also damages the intestine .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007515 ! \"Islets of Langerhans\")\n",
      "A previous study of SARS-CoV patients without pre-existing diabetes ( n = 39 ) has reported that 20 of the 39 SARS-CoV patients developed diabetes during hospitalization because immunostaining of ACE-2 in the pancreatic tissue suggested that SARS-CoV might have damaged pancreatic islets and caused acute Type I diabetes mellitus like condition [ 74,76 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D007515 ! \"Islets of Langerhans\")\n",
      "A previous study of SARS-CoV patients without pre-existing diabetes ( n = 39 ) has reported that 20 of the 39 SARS-CoV patients developed diabetes during hospitalization because immunostaining of ACE-2 in the pancreatic tissue suggested that SARS-CoV might have damaged pancreatic islets and caused acute Type I diabetes mellitus like condition ( Yang et al , 2006 ; Hussain et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D008099 ! Liver)\n",
      "103 Liver damage seems to be directly caused by SARS-CoV rather than hypoxia .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D008099 ! Liver)\n",
      "Chau et al [ 3 ] demonstrated in their study that the liver biopsy of SARS patients showed a significant increase in mitotic cells , as well as eosinophils and balloon-like liver cells , which indicated that SARS-CoV may induce liver cell apoptosis and thus lead to liver damage .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D011770 ! \"Pyruvate Kinase\")\n",
      "Thus , our results suggest that SARS-CoV could reduce pyruvate kinase activity via its nucleocapsid protein , and this may in turn cause disease .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D011770 ! \"Pyruvate Kinase\")\n",
      "Thus , our results suggest that SARS-CoV could reduce pyruvate kinase activity via its nucleocapsid protein , and this may in turn cause abnormal liver function .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "However , persistent SARS-CoV stimulation can deplete T lymphocytes in such a way as to reduce their function and prevent cytokine production [ 16 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "Our hypothesis that T lymphocytes are damaged by the SARS coronavirus ( SARS-CoV ) is based on current pathological findings .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013737 ! Testis)\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013737 ! Testis)\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013737 ! Testis)\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013737 ! Testis)\n",
      "In a study conducted by Xu and colleagues in 2006 , it was said that the coronavirus family ( SARS-CoV ) can cause damage to the testes and germ cells .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013961 ! \"Thyroid Gland\")\n",
      "Severe acute respiratory syndrome coronavirus ( SARS-CoV ) , the virus responsible for severe acute respiratory syndrome ( SARS ) was found to cause extensive damage to the thyroid glands .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D013961 ! \"Thyroid Gland\")\n",
      "Severe acute respiratory syndrome coronavirus ( SARS-CoV ) , the virus responsible for severe acute respiratory syndrome ( SARS ) was found to cause extensive damage to the thyroid glands .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D014024 ! Tissues)\n",
      "Several SARS autopsy research showed that SARS-CoV could damage multiple tissues , however , the major histopathology involves lungs [ 55 , 150 , 151 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D014024 ! Tissues)\n",
      "15 The extent of tissue damage by SARS-CoV is a direct function of the level of expression of tissue ACE2 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D014024 ! Tissues)\n",
      "At least two possibilities exist : ( i ) direct damage to cells and tissues by the SARS-CoV and ( ii ) indirect damage , mediated primarily by the cellular immune response and cytokines .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D018791 ! \"CD4 Lymphocyte Count\")\n",
      "SARS-CoV and HIV may decrease conjunctly CD4 count and lymphocytes , adding high corticosteroids [ 29 ] as a treatment for SARS-CoV may be TB precipitant factors [ 29 ] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D058872 ! Epidemics)\n",
      "One possible scenario is that the SARS-CoV had already infected some people in the spring of 2002 but failed to cause epidemics ; its spread was however suppressed in the summer ( similar to the summer of 2003 ) , and re-emerged around November to cause the epidemic in 2003 .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "It also inhibited other coronaviruses SARS-CoV (EC50 2.4 μM) and HCoV-229E (EC50 5.1 μM) and feline infectious peritonitis (FIPV, EC50 7.0 μM) within a not-toxic range of concentration.An in vitro study of the anti-viral effect of the drug 46 had found to inhibit the early stages of MERS-CoV life cycle in Huh-7 cells through a possible suppression of SFK signaling pathways.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "The SARS-CoV 3CL pro inhibitor CE-5 can block the function of the MERS-CoV 3CL pro (Figure 5(14) ) [53] .\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "These findings offer strong evidence that EPAC1 could act as a novel pharmacological target for diabetes and obesity therapies.Recently, Tao et al.95 found the capacity of compound 14 at a nontoxic concentration (10 µM) to protect Calu-3 cells against MERS-CoV and SARS-CoV infection via inhibiting the replication of viral RNA and expression of MERS-CoV and SARS-CoV protein without affecting virus binding to Calu-3 and expression and localization of EPAC protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "These findings offer strong evidence that EPAC1 could act as a novel pharmacological target for diabetes and obesity therapies.Recently, Tao et al. 95 found the capacity of compound 14 at a nontoxic concentration (10 mM) to protect Calu-3 cells against MERS-CoV and SARS-CoV infection via inhibiting the replication of viral RNA and expression of MERS-CoV and SARS-CoV protein without affecting virus binding to Calu-3 and expression and localization of EPAC protein.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "These findings offer strong evidence that EPAC1 could act as a novel pharmacological target for diabetes and obesity therapies.Recently, Tao et al. 95 found the capacity of compound 14 at a nontoxic concertation (10 μM) to protect Calu-3 cells against MERS-CoV and SARS-CoV infection via inhibiting the replication of viral RNA and expression of MERS-CoV and SARS-CoV protein without affecting virus binding to Calu-3 and expression and localization of EPAC protein.Compared wild-type C57BL/6 mice treated with compound 14 (10 mg/kg, once daily, i.p.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "All SARS-CoV 3CL pro inhibitors were found to inhibit MERS-CoV 3CL pro effectively at low micromolar concentrations.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "(PLpros)= Papain-like proteases MERS-CoV and SARS-CoV -act as a potential anti-viral target for drugs Multiple inhibition assays also support a kinetic mechanism by which disulfiram together with 6TG and/or MPA can synergistically inhibit MERS-CoV PLpro, but not, due to its competitive mode of inhibition, SARS-CoV PLpro.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "In addition, neither SARS-CoV-2 RBD nor SARS-CoV RBD blocked the entry of MERS-CoV pseudovirus into hDPP4-expressing 293T cells (Fig. 4c).\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "Griffithsin, a lectin extract red algae, has been reported to bind to oligosaccharides on the surface of various viral glycoproteins, including HIV glycoprotein 120 and SARS-CoV glycoproteins41.Resveratrol, a natural compound from grape, which is in a clinical phase for heart and other diseases, was also reported to effectively inhibit MERS-CoV in vitro by downregulation of the apoptosis induced by the virus42.\n",
      "\n",
      "bp(MESH:D045473 ! \"SARS Virus\") decreases bp(MESH:D065207 ! \"Middle East Respiratory Syndrome Coronavirus\")\n",
      "Moreover, infection of mice with SARS-CoV reduced MERS-CoV titers 5 days p.i.\n",
      "\n",
      "a(TEXT:\"Architect SARS-CoV-2 IgG\") increases a(TEXT:\"LIAISON SARS-CoV-2 S1/S2 IgG\")\n",
      "Seropositive results were confirmed using the Architect SARS-CoV-2 IgG (Abbott, Wiesbaden, Germany) targeting the viral nucleocapsid and the LIAISON SARS-CoV-2 S1/S2 IgG assay (DiaSorin Deutschland GmbH, Dietzenbach, Germany) targeting the SARS-CoV-2 spike protein.Most samples (2,902/3,186;> 91%) were obtained between 23 March and 22 May 2020.\n",
      "\n",
      "a(TEXT:\"CC-BY-NC-ND 4.0 International license.91 Results92 SARS-CoV-2\") increases bp(GO:\"GO:0008152\" ! \"metabolic process\")\n",
      "It is made available under a CC-BY-NC-ND 4.0 International license.91 Results92 SARS-CoV-2 infection upregulates lipid metabolism, increasing LD biogenesis in 93 human cells.94 Viruses have the ability to modulate cellular metabolism with benefits for viral 95 replication.\n",
      "\n",
      "a(TEXT:\"CC-BY-ND 4.0 International license.25 Summary 26 SARS-CoV-2\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "It is made available under a CC-BY-ND 4.0 International license.25 Summary 26 SARS-CoV-2 is the positive-sense RNA virus that causes COVID-19, a disease that has 27 triggered a major human health and economic crisis.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N protein\") increases p(HGNC:6018 ! IL6)\n",
      "ACE2 expressing CD68 + CD169 + macrophages that contain the SARS-CoV-2 N protein increased IL-6 in spleen and lymph nodes [373], and using nanoparticles that target the inflammatory cells responsible for cytokine storm is a promising approach to developing efficient and safe COVID-19 vaccines.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N protein\") decreases p(HGNC:5434 ! IFNB1)\n",
      "Therefore, we propose that the SARS-CoV-2 N protein represses IFN-beta production by interfering with RIG-I.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N protein\") decreases p(HGNC:5434 ! IFNB1)\n",
      "These results suggested that SARS-CoV-2 N protein suppresses the IFN-beta response through targeting the initial step, potentially the cellular PRR-RNA-recognition step in the innate immune pathway.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N protein\") increases p(HGNC:11362 ! STAT1, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "63 Interestingly, the structural SARS-CoV-2 N protein has also been shown to antagonize IFN signaling by suppressing phosphorylation and translocation of STAT1 and STAT2, representing a potential viral mechanism to evade the innate immune response.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N protein\") increases p(HGNC:11363 ! STAT2, pmod(go:0006468 ! \"protein phosphorylation\"))\n",
      "63 Interestingly, the structural SARS-CoV-2 N protein has also been shown to antagonize IFN signaling by suppressing phosphorylation and translocation of STAT1 and STAT2, representing a potential viral mechanism to evade the innate immune response.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 S\") increases p(HGNC:9906 ! RBFOX2)\n",
      "Our qRT-PCR data showed that transfection of either SARS-CoV-2 S or N vectors significantly increased representative lytic gene expression (Immediate-early gene, RTA; Early gene, ORF59; and Late gene, ORF17) when compared to vector control, regardless of with or without TPA induction.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 S\") increases bp(MESH:D017781 ! \"Genes, Immediate-Early\")\n",
      "Our qRT-PCR data showed that transfection of either SARS-CoV-2 S or N vectors significantly increased representative lytic gene expression (Immediate-early gene, RTA; Early gene, ORF59; and Late gene, ORF17) when compared to vector control, regardless of with or without TPA induction.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 S\") decreases a(TEXT:VSVpp)\n",
      "Thus, specific 177 mutations convert IFITM3 from an inhibitor to an enhancer of SARS-CoV-2 S mediated VSVpp 178 infection in transfected HEK293T cells.182 SARS-CoV-2 at an MOI of 0.05 and measured viral RNA yields in the culture supernatants by qPCR 183 two days later.\n",
      "\n",
      "a(TEXT:VSV-eGFP-SARS-CoV-2) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "To assess whether VSV-eGFP-SARS-CoV-2 limits virus induced 137 inflammation, we measured pro inflammatory cytokine and chemokine mRNA in lung 138 homogenates from vaccinated animals at 4 dpi by RT-qPCR assays (Fig 3A).\n",
      "\n",
      "a(TEXT:ChAd-SARS-CoV-2-S) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Intramuscular dosing of ChAd-SARS-CoV-2-S induces robust systemic humoral and cell mediated immune responses and protects against lung infection, inflammation, and pathology but does not confer sterilizing immunity, as evidenced by detection of viral RNA and induction of anti-nucleoprotein antibodies after SARS-CoV-2 challenge.\n",
      "\n",
      "a(TEXT:\"ChAdOx1 encoded SARS-CoV-2 antigens\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "The ChAdOx1 encoded SARS-CoV-2 antigens vaccine with a non replicating virus may trigger a robust immune response against the virus with the added advantage of safe immunization of geriatric people, children and individuals with comorbidities [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RNA\") increases a(TEXT:\"COVID-19 case numbers\")\n",
      "We found that the prevalence of SARS-CoV-2 RNA on high-touch surfaces in public spaces and essential businesses reflected, and may even lead, local COVID-19 case numbers by one week.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RNA\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Further work is needed to clarify whether the entry of SARS-CoV-2 RNA via EVs is sufficient to induce cell injury and inflammation.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RNA\") decreases a(TEXT:Cas13a)\n",
      "The specific crRNA targeting SARS-CoV-2 RNA activates Cas13a, resulting in the cleavage of the viral genome and blockage of viral protein synthesis.\n",
      "\n",
      "a(TEXT:\"investigation.ParameterValuesSARS-COV-2 ProteomesBinding Threshold HLA allele\") increases a(TEXT:\"HLA haplotype\")\n",
      "Future experimental testing of these epitopes 119 would confirm whether natural infection induces CD8 T-cell responses targeting these regions, 120 but from an in silico perspective they offer a potential target for developing a therapeutic T-cell 121 vaccine that warrants further investigation.ParameterValuesSARS-COV-2 ProteomesBinding Threshold HLA allele contributions HLA haplotype weighting Rank Binding Peptide Conservation threshold (%) Brazil (1), China (13), France (1), Greece (1), Iran (1), South Africa (1), Spain (10), Sweden (1), Taiwan (1), Turkey (1), USA (238), Unknown (16), Vietnam (2) 1% 46 alleles (16 x HLA-A, 19 x HLA-B, 11 x HLA-C) None < 2.02.1% bioRxiv preprint doi : https://doi.org/10.1101/2020.08.15.250647.\n",
      "\n",
      "a(TEXT:INTRODUCTIONSARS-CoV-2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "INTRODUCTIONSARS-CoV-2, a novel coronavirus, was identified by the World Health Organization as originating in the Wuhan province of China in late 2019, 1,2 and causes Corona Virus Disease 2019 (COVID-19).\n",
      "\n",
      "a(TEXT:IntroductionSARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "IntroductionSARS-CoV-2 causes COVID-19, a multidimensional disease characterized predominantly by pneumonia that can progress to respiratory failure and death 1,2.\n",
      "\n",
      "a(TEXT:\"LightMix(R) Modular SARSCoV-2 RT-PCR kit\") increases bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\")\n",
      "The gold standard was the LightMix (R) Modular SARSCoV-2 RT-PCR kit targeting the envelope (E) and RNA dependent RNA polymerase (RdRp) genes.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PL2 pro\") decreases p(HGNC:15631 ! TLR7)\n",
      "In 2016, it was reported that the SARS-CoV PL2 pro can inhibit the Toll like receptor 7 (TLR7)-mediated type-I IFN response and the NF-kappaB pathway by removing the Lys63 linked polyUb chain from TRAF3 and TRAF6.\n",
      "\n",
      "a(TEXT:\"N SARS-CoV-2 LLPS\") increases bp(MESH:D012324 ! \"RNA-Dependent RNA Polymerase\")\n",
      "In agreement with such activity, we showed that N SARS-CoV-2 translocates to stress granules in stressed cells.We demonstrate that N SARS-CoV-2 LLPS promotes cooperative association of the RNA dependent RNA polymerase complex with polyU RNA in vitro.\n",
      "\n",
      "a(TEXT:\"QIAstat-Dx Respiratory SARS-CoV-2 panel\") increases bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\")\n",
      "Examples of multiplex RT-qPCRs include the QIAstat-Dx Respiratory SARS-CoV-2 panel targeting the E and RdRp genes, the TaqPath Covid-19 combo targeting the ORF1b and N and S genes, and the Allplex 2019-nCoV assay targeting the E, N and RdRP genes [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"Results108 Purified SARS-CoV-2 proteins\") increases a(CHEBI:15379 ! dioxygen)\n",
      "These findings implicate the 105 contribution of the viral surface proteins to driving inflammation in a cell type specific manner 106 and highlight their potential for therapeutic intervention.107 Results108 Purified SARS-CoV-2 proteins induce reactive oxygen species generation and 109 proinflammatory chemokine and cytokine production 110 Increased reactive oxidative species generation has been detected in clinical COVID-19 sputum 111 samples (35), although it is unclear to what extent active infection or inflammatory stimulation 112 contribute to this finding.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(CHEBI:30212 ! photon)\n",
      "The force of attraction 81 between liquid molecules generates gamma which acts in all direction in the liquid bulk phase.82 Lung pathology due to SARS-CoV-2 infection : 119 SARS-CoV-2 travels from the nasal cavity to the gaseous exchange units of the lungs.120 Respiratory diseases like influenza and SARS-CoV have been reported to preferentially attack 121 type II pneumocytes of the lungs , and viral release followed by replication directs the cells into 122 apoptosis and eventually death [ 2 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Precursor Plasma Cell Count\")\n",
      "Integration of plasma proteomics with nine published scRNAseq datasets shows that SARS-CoV-2 infection upregulates monocyte / macrophage , plasmablast , and T cell effector proteins .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CD8 ! CD8)\n",
      "SARS-CoV-2 infection induces general lymphopenia and CD8 , VD1 and VD2 activation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CD8 ! CD8)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CD8 ! CD8)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CD8 ! CD8)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CD8 ! CD8)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(FPLX:CD8 ! CD8)\n",
      "But the SARS-CoV-2 infection elicits functionally exhausted NK and CD8 + T cells in which receptor CD94 / NK group 2 member A ( NKG2A ) expression has been surged [ 54 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:CK2 ! CK2)\n",
      "SARS-CoV-2 infection promoted casein kinase II ( CK2 ) and p38 MAPK activation , production of diverse cytokines , and shutdown of mitotic kinases , resulting in cell cycle arrest .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:Chemokine ! Chemokine)\n",
      "SARS-CoV-2 infection caused striking expression of chemokines , as also seen in primary human COVID-19 pulmonary autopsy samples .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:Chemokine ! Chemokine)\n",
      "A number of chemokines are also induced by SARS-CoV-2 infection ( Chen and Subbarao 2007 ; Channappanavar and Perlman 2017 ; Coperchini et al. 2020 ; Li et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "In addition , SARS-CoV-2 infection can activate NF-kB , leading to multiple inflammatory and autoimmune diseases , which induces pro-inflammatory cytokines and chemokines , including IL-6 , and recruits lymphoid cells and myeloid cells , such as activated T cells and macrophages ( Hirano and Murakami , 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "In addition , SARS-CoV-2 infection can activate NF-kappaB , leading to multiple inflammatory and autoimmune diseases , which induces pro-inflammatory cytokines and chemokines , including IL-6 , and recruits lymphoid cells and myeloid cells , such as activated T cells and macrophages ( Hirano and Murakami , 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "COVID-19 activates NFkB 4 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "The SARS-CoV-2 infection triggers an adaptive immune response in which T cell-B cell interactions result in the production of SARS-CoV-2specific antibodies [ 14 ] , but a similarity in viral and ganglioside peptide sequences or structure ( molecular mimicry ) [ 15 ] can result in a loss of self-tolerance [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "The SARS-CoV-2 infection triggers an adaptive immune response in which T cell-B cell interactions result in the production of SARS-CoV-2-specific antibodies [ 14 ] , but a similarity in viral and ganglioside peptide sequences or structure ( molecular mimicry ) [ 15 ] can result in a loss of self-tolerance [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "This study was the first of its kind to suggest that SARS-CoV2 infection induced a broad adaptive immune response that may have facilitated in viral clearance and clinical recovery .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0002250\" ! \"adaptive immune response\")\n",
      "This study was the first of its kind to suggest that SARS-CoV2 infection induced a broad adaptive immune response that may have facilitated in viral clearance and clinical recovery .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "SARS-CoV-2 infection elicits protective immune response and promotes immunoglobulin M and G antibody production against virus antigens such as spike glycoprotein [ 41 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0003823\" ! \"antigen binding\")\n",
      "SARS-CoV-2 infection elicits protective immune response and promotes immunoglobulin M and G antibody production against virus antigens such as spike glycoprotein [ 41 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "These findings suggest that hrsACE2 can considerably prevent SARS-CoV-2 infection by blocking its entry at the early stage of infection.37 Umifenovir ( ArbidolTM ; CAS number : 131707-25-0 ) is a small molecule with an indole core and inhibits viral entry into host cells through inhibition of viral membrane fusion.38 ,39 It is approved in Russia and China as an antiviral treatment for oral prophylaxis and treatment of influenza infections .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0046718\" ! \"viral entry into host cell\")\n",
      "These findings suggest that hrsACE2 can considerably prevent SARS-CoV-2 infection by blocking its entry at the early stage of infection.37 Umifenovir ( ArbidolTM ; CAS number : 131707-25-0 ) is a small molecule with an indole core and inhibits viral entry into host cells through inhibition of viral membrane fusion.38 ,39 It is approved in Russia and China as an antiviral treatment for oral prophylaxis and treatment of influenza infections .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0070254\" ! \"mucus secretion\")\n",
      "189 We also found substantial mucus secretion due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0070266\" ! \"necroptotic process\")\n",
      "The SARS-CoV-2 infection induced a dual mode of cell death pathways , apoptosis , and necroptosis , and inflammatory responses were also observed in the infected HFH4-hACE2 transgenic mouse model and in the postmortem lung sections of fatal COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0070269\" ! pyroptosis)\n",
      "Notably , SARS-CoV-2 infection is suspected to cause pyroptosis in lymphocytes and macrophages .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(GO:\"GO:0070269\" ! pyroptosis)\n",
      "Notably , SARS-CoV-2 infection is suspected to cause pyroptosis in lymphocytes and macrophages .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:10019 ! RIPK1)\n",
      "In this context , after validating the suitability of a commercially available antibody ( Fig. S3 ) , we tested whether SARS-CoV-2 infection triggers RIPK1 activation in respiratory epithelial cells collected from the throat smears of symptomatic patients who had tested positive for SARS-CoV-2 by PCR .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:10019 ! RIPK1)\n",
      "In this context , after validating the suitability of a commercially available antibody ( Fig. S3 ) , we tested whether SARS-CoV-2 infection triggers RIPK1 activation in respiratory epithelial cells collected from the throat smears of symptomatic patients who had tested positive for SARS-CoV-2 by PCR .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:10019 ! RIPK1)\n",
      "We can now show that SARS-CoV-2 infection triggers RIPK1 activation in respiratory tract epithelial cells isolated from the throat smears of symptomatic patients who had tested positive for SARS-CoV-2 by PCR .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:10019 ! RIPK1)\n",
      "We can now show that SARS-CoV-2 infection triggers RIPK1 activation in respiratory tract epithelial cells isolated from the throat smears of symptomatic patients who had tested positive for SARS-CoV-2 by PCR .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1499 ! CASP1)\n",
      "Using monocytes from healthy donors , we found that SARS-CoV-2 infection triggers caspase-1 activation , as measured by the Caspase-Glo 1 luminescent assay that detects caspase-1 activity in the media ( Fig. 1 A ) , and IL-1beta production , quantified using ELISA ( Fig. 1 B ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1499 ! CASP1)\n",
      "Using monocytes from healthy donors , we found that SARS-CoV-2 infection triggers caspase-1 activation , as measured by the Caspase-Glo 1 luminescent assay that detects caspase-1 activity in the media ( Fig. 1 A ) , and IL-1beta production , quantified using ELISA ( Fig. 1 B ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1509 ! CASP8)\n",
      "Recent study reported that SARS-CoV-2 infection could activate caspase-8 to induce apoptosis and inflammatory cytokine processing in the lung epithelial cells , triggering necroptosis pathway [ 10 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1509 ! CASP8)\n",
      "Here we report that SARS-CoV-2 infection activates caspase-8 to trigger cell apoptosis and inflammatory cytokine processing in the lung epithelial cells .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1509 ! CASP8)\n",
      "Here we report that SARS-CoV-2 infection activates caspase-8 to trigger cell apoptosis and inflammatory cytokine processing in the lung epithelial cells .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:1678 ! CD4)\n",
      "SARS-CoV-2 infection induces a strong CD4 + T cell response that favours Th1 polarisation , but Th2 and Th17 responses have also been detected ( 106 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:1678 ! CD4)\n",
      "It has been shown that SARS-CoV-2 infection causes overall reduction in the CD4 + and CD8 + T cell counts in peripheral blood ( 3 , 4 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:2654 ! CCN1)\n",
      "SARS-CoV-2 infection may enhance CCN1 expression , as these viruses do .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:2654 ! CCN1)\n",
      "SARS-CoV-2 infection may enhance CCN1 expression , as these viruses do .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:3349 ! ENG)\n",
      "Our analysis indicated that SARS-CoV-2 infection up-regulates FLT1 and ENG , the main antiangiogenic factors involved in preeclampsia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:348 ! AHR)\n",
      "We find that induction of either interferon ( IFN ) - beta or IFN-gamma upon SARS-CoV-2 infection results in activation of aryl hydrocarbon receptor ( AhR ) signaling through an IDO-Kyn-dependent pathway , leading to transcriptional upregulation of the expression of mucins , both the secreted and membrane-bound , in alveolar epithelial cells .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:3763 ! FLT1)\n",
      "Our analysis indicated that SARS-CoV-2 infection up-regulates FLT1 and ENG , the main antiangiogenic factors involved in preeclampsia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:4983 ! HMGB1)\n",
      "SARS-CoV-2 infection increased HMGB1 protein levels in the nucleus and cytoplasm ( Figure S5A ) and culture media ( Figure S5B - C ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:8537 ! P2RX7)\n",
      "In view of that , we suggest P2X7 receptor as a key mediator of the neuroinflammatory process as a possible consequence of SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:8537 ! P2RX7)\n",
      "In view of that , we suggest P2X7 receptor as a key mediator of the neuroinflammatory process as a possible consequence of SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:8583 ! SERPINE1)\n",
      "In particular , the induction of PAI-1 by SARS-CoV-2 infection seems to be a significant event in the pathogenesis of COVID-19 because PAI-1 can inhibit urokinase-type plasminogen activator ( uPA ) and tissue plasminogen activator ( tPA ) ( Ekholm et al. 2009 ; Whyte et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "Taken together , these results demonstrate that SARS-CoV-2 infection induces PTGS2 in diverse in vitro and in vivo airway and lung systems , across multiple independent studies .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "30 We found that SARS-CoV-2 infection induced COX-2 upregulation in diverse human cell culture 31 and mouse systems .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "Taken together , these results demonstrate that SARS-CoV-2 infection induces PTGS2 in diverse in vitro and in vivo airway and lung systems , across multiple independent studies .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "An important question arising from our findings is how SARS-CoV-2 infection induces COX-2 expression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 upregulation in diverse human cell culture and mouse systems .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "An important question arising from our findings is how SARS-CoV-2 infection induces COX-2 expression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "SARS-CoV-2 infection induces PTGS2 expression in human and mouse systems ( A ) Calu-3 cells were infected with SARS-CoV-2 at a multiplicity of infection ( MOI ) of 0.05 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "168 An important question arising from our findings is how SARS-CoV-2 infection induces 169 COX-2 expression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "SARS-CoV-2 infection induces PTGS2 expression in human and mouse systems 201 ( A ) Calu-3 cells were infected with SARS-CoV-2 at a multiplicity of infection ( MOI ) of 0.05 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 upregulation in diverse human cell culture and mouse systems .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "We found that SARS-CoV-2 infection induced COX-2 upregulation in diverse human cell culture and mouse systems .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(HGNC:9605 ! PTGS2)\n",
      "We demonstrate that SARS-CoV-2 infection upregulates COX-2 in diverse human cell culture and mouse systems .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Ventricular arrhythmia\")\n",
      "As mentioned above , SARS-CoV-2 infection can produce myocardial dysfunction and ventricular arrhythmia through direct myocardial damage and hyper-inflammatory state .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Ventricular arrhythmia\")\n",
      "Furthermore , COVID-19 can also produce fulminant myocarditis , ventricular arrhythmias and pulmonary hypertension in previously healthy children .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Ventricular arrhythmia\")\n",
      "Furthermore , COVID-19 can also produce fulminant myocarditis , ventricular arrhythmias and pulmonary hypertension in previously healthy children .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Ventricular arrhythmia\")\n",
      "As mentioned above , SARS-CoV-2 infection can produce myocardial dysfunction and ventricular arrhythmia through direct myocardial damage and hyperinflammatory state .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Acute necrotizing encephalopathy\")\n",
      "Finally , another case report showed that SARS-CoV-2 infection could cause acute necrotizing encephalopathy ( 81 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Acute necrotizing encephalopathy\")\n",
      "Finally , another case report showed that SARS-CoV-2 infection could cause acute necrotizing encephalopathy ( 81 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases a(TEXT:\"Acute necrotizing encephalopathy\")\n",
      "Finally , another case report showed that SARS-CoV-2 infection could cause acute necrotizing encephalopathy ( 81 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection and characterized by diverse clinical symptoms .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Nevertheless , as Rose said , `` The profound effects of COVID-19 may forever change how future physicians are educated '' ( 4 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection and characterized by diverse clinical symptoms .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection and is primarily spread through contact , respiratory droplets , and feces [ 186 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "In case of New York / Massachusetts , the pollution-reducing effect of COVID-19 is highest at the range of middle to higher quantiles ( 0.4-0.8 ) of COVID-19 , while California / Illinois , Florida , and Texas exhibit a strong negative influence of COVID-19 on air pollution in the grid of lower to higher ( 0.2-0.8 ) quantiles , medium to higher ( 0.6-0.8 ) quantiles , and extreme higher quantiles ( 0.8-0.9 ) , respectively .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "In case of New York / Massachusetts , the pollution-reducing effect of COVID-19 is highest at the range of middle to higher quantiles ( 0.4-0.8 ) of COVID-19 , while California / Illinois , Florida , and Texas exhibit a strong negative influence of COVID-19 on air pollution in the grid of lower to higher ( 0.2-0.8 ) quantiles , medium to higher ( 0.6-0.8 ) quantiles , and extreme higher quantiles ( 0.8-0.9 ) , respectively .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by a SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by a SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 , which is caused by the SARS-CoV-2 infection , was detected in Wuhan , China in December 2019 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 , which is caused by the SARS-CoV-2 infection , was detected in Wuhan , China in December 2019 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The SARS-CoV-2 infection causes the coronavirus disease 2019 ( COVID-19 ) and it is responsible for the pandemic that we are currently living .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "SARS-CoV-2 infection , which causes COVID-19 , has spread rapidly throughout the world since December 2019 when the first outbreak was detected in Wuhan , China .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 , which is caused by the SARS-CoV-2 infection , was detected in Wuhan , China in December 2019 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The initial screening of this study applied the following inclusion criteria : ( 1 ) sample sizes > = 10 participants , ( 2 ) studied prevalence / surveys of psychological or psychosocial problems in the WPA Zone 16 countries , ( 3 ) investigated prevalence / incidence of psychiatric disorders in general population / frontline workers / COVID-19 affected patients as a consequence of COVID-19 pandemic ; ( 4 ) used study population of adults age-ranged between > = 18 years , and ( 5 ) reported significant statistical differences in the studied parameters .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "College students ' use and acceptance of emergency online learning due to COVID-19 Due to COVID-19 , higher education institutions transitioned to online learning .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "SARS-CoV-2 infection is the cause of COVID-19 in humans .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 due to the SARS-CoV-2 infection is a multi-systemic immune syndrome affecting mainly the lungs , oropharyngeal region , and other vascular endothelial beds .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      ") Imagine that a person who does NOT show any symptoms compatible with COVID-19 undergoes a swab test during a screening for the SARS-CoV-2 infection that causes COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Imagine a person who lives in [ Italy / Bergamo / Sassari ] and who , at the moment , is showing some symptoms compatible with the SARS-CoV-2 infection that causes COVID-19 ( the person has a temperature higher than 37.5degreesC and / or mild cough but does not have breathing difficulties ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Severe SX Imagine a person who lives in [ Italy / Bergamo / Sassari ] and who , at the moment , is showing several symptoms compatible with the SARS-CoV-2 infection that causes COVID-19 ( the person has a temperature higher than 37.5degreesC , severe cough , and breathing difficulties ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "In your opinion , given that she  In your opinion , how likely is it that , within the next 2 months , you will contract the SARS-CoV-2 infection that causes COVID-19 ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "_ _ _ _ _ ( Insert a value from 0 = `` impossible '' to 100 = `` certain '' ) Imagine contracting , within the next 2 months , the SARS-CoV-2 infection that causes COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "_ _ _ _ _ ( Insert a value from 0 = `` not worried at all '' to 100 = `` extremely worried '' ) In your opinion , how likely is it that , within the next 2 months , you will contract the SARS-CoV-2 infection that causes COVID-19 ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Imagine a person who lives in [ Italy / Bergamo / Sassari ] and who , at the moment , does NOT show any symptoms compatible with the SARS-CoV-2 infection that causes COVID-19 ( the person does not have a temperature higher than 37.5degreesC , no cough or respiratory difficulties ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Since December 2019 , coronavirus disease 2019 ( COVID-19 ) due to SARS-CoV-2 infection has emerged in Wuhan and rapidly spread throughout China and even to other countries .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "This will result in a further increase in deaths , this time as a consequence of COVID-19 and not as a direct result of COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "This will result in a further increase in deaths , this time as a consequence of COVID-19 and not as a direct result of COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Introduction COVID-19 due to SARS-CoV-2 infection was declared as global pandemic by WHO on 11 March 2020 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 due to SARS-CoV-2 infection was declared as global pandemic by WHO on 11 March 2020 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection and can be transmitted via respiratory droplets and close contact .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 and the Heart Abstract Severe acute respiratory syndrome coronavirus 2 ( SARS-CoV-2 ) which causes coronavirus disease 2019 ( COVID-19 ) has resulted in a global health crisis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Investigation of the Potential Mechanism Governing the Effect of the  Shen Zhu San  on COVID-19 by Network Pharmacology Background Since December 2019 , coronavirus disease 2019 ( COVID-19 ) due to SARS-CoV-2 infection has emerged in Wuhan and rapidly spread throughout China and even to other countries .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Since December 2019 , coronavirus disease 2019 ( COVID-19 ) due to SARS-CoV-2 infection has emerged in Wuhan and rapidly spread throughout China and even to other countries .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "It has been recently confirmed that SARS-COV-2 infection is the definitive cause of COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "It has been recently confirmed that SARS-COV-2 infection is the definitive cause of COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Severe Rhabdomyolysis in a 35-Year-old Woman with COVID-19 due to SARS-CoV-2 Infection : A Case Report .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "COVID-19 is caused by SARS-CoV-2 infection and was initially discovered in Wuhan .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Hence , it needs to validate further that 17beta-estradiol may restrict the SARS-CoV-2 infection by promoting the endolysosomal degradation pathway ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Hence , it needs to validate further that 17beta-estradiol may restrict the SARS-CoV-2 infection by promoting the endolysosomal degradation pathway ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "There is real demand for services related to testing , screening , supply of medicines as well as information needs related to safety , effectiveness , and value of medicines and vaccines related to COVID-19 and the consequences of delayed healthcare due to COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Coronavirus disease 2019 ( COVID-19 ) is caused by SARS-CoV-2 infection and mainly affects the respiratory system , but COVID-19 is also capable of inducing damage to other organs .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Coronavirus disease 2019 ( COVID-19 ) is caused by SARS-CoV-2 infection and mainly affects the respiratory system , but COVID-19 is also capable of inducing damage to other organs .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "As of March 17 , 2020 , the Association of American Medical Colleges ( AAMC ) have provided guidelines suggesting that medical schools support suspending clinical rotations for medical students.1 Rose2 commented that the profound effect of COVID-19 may forever change how future physicians are educated .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D000744 ! \"Anemia, Hemolytic, Autoimmune\")\n",
      "Although the underlying mechanisms are not well known , we confirm that SARS-CoV-2 infection may trigger AIHA , as well as other complications , such as myocardial infarction , possibly related to cytokine storm and hyperinflammation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D000744 ! \"Anemia, Hemolytic, Autoimmune\")\n",
      "Conclusion Although the underlying mechanisms are not well known , we confirm that SARS-CoV-2 infection may trigger AIHA , as well as other complications , such as myocardial infarction , possibly related to cytokine storm and hyperinflammation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D000744 ! \"Anemia, Hemolytic, Autoimmune\")\n",
      "Although the underlying mechanisms are not well known , we confirm that SARS-CoV-2 infection may trigger AIHA , as well as other complications , such as myocardial infarction , possibly related to cytokine storm and hyperinflammation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D001172 ! \"Arthritis, Rheumatoid\")\n",
      "Besides , SARS-CoV-2 infection also causes RA ( Ono et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D001172 ! \"Arthritis, Rheumatoid\")\n",
      "Besides , SARS-CoV-2 infection also causes RA ( Ono et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D001172 ! \"Arthritis, Rheumatoid\")\n",
      "Besides , SARS-CoV-2 infection also causes RA ( Ono et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D001344 ! Autopsy)\n",
      "Autopsy was prohibited due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D002415 ! Cats)\n",
      "An important question remains unanswered : did the cat actually die due to SARS-CoV-2 infection ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D002647 ! Chilblains)\n",
      "They hypothesized that the situation could be due to the media stating that chilblains were caused by SARS-CoV-2 infection and leading to a higher rate of consultation or the lockdown leading to more inactivity and long periods at home barefoot on a cold floor .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D003082 ! Colectomy)\n",
      "Delaying colectomy due to SARS-CoV-2 infection was considered inappropriate .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D003704 ! Dementia)\n",
      "Additionally , damage to the BBB is not the only mechanism where SARS-CoV-2 infection can result in dementia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D003704 ! Dementia)\n",
      "Additionally , damage to the BBB is not the only mechanism where SARS-CoV-2 infection can result in dementia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D003704 ! Dementia)\n",
      "Additionally , damage to the BBB is not the only mechanism where SARS-CoV-2 infection can result in dementia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D004487 ! Edema)\n",
      "Currently , there is neither experimental nor clinical evidence suggesting that SARS-CoV-2 infection could lead to bradykinin-dependent lung inflammation and edema .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D004751 ! Enteritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D004751 ! Enteritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D004751 ! Enteritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D004802 ! Eosinophilia)\n",
      "In our case , eosinophilia was induced by SARS-CoV-2 infection and influenced the clinical course .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005076 ! Exanthema)\n",
      "In the absence of other possible syndromes and diseases , the skin rash was very likely due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005076 ! Exanthema)\n",
      "COVID-19 can cause inflammation reactions of multiple organs , which is similar to the systemic vasculitis of Kawasaki disease , and even COVID-19 can cause skin rash on the extremities of the limbs , which is also similar to Kawasaki disease .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "However , since the patient was confirmed to have COVID-19 later , the fever at that time might have been caused by SARS-CoV-2 infection or underlying pathology during the subclinical period .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Sars-Cov-2 infection causes fever and cough that may rapidly lead to acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 , which causes fever and mild to severe respiratory symptoms , is closely related to other coronaviruses ( CoVs ) , such as SARS ( SARS-CoV-1 ) and MERS-CoV ( Middle East respiratory syndrome CoV ) ( 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 , which causes fever and mild to severe respiratory symptoms , is closely related to other coronaviruses ( CoVs ) , such as SARS ( SARS-CoV-1 ) and MERS-CoV ( Middle East respiratory syndrome CoV ) ( 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "LEARNING POINTS Coronavirus disease ( COVID-19 ) can cause fever , dry cough and acute respiratory failure .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "* Coronavirus disease ( COVID-19 ) can cause fever , dry cough and acute respiratory failure .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Coronavirus disease ( COVID-19 ) can cause fever , dry cough and acute respiratory failure.Cough can result in several complications , including rupture of the diaphragm and abdominal herniation.CT scanning is the gold standard technique to investigate COVID-19 pneumonia and diaphragmatic rupture .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Apart from the persistent fever due to COVID-19 and a high triglyceride level presumably linked to the high-dose propofol required for sedation , there were no reliable signs of macrophage activation syndrome : serum ferritin continued to decrease to 1906 ng / mL , liver function was normal , and there was no leukopenia .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 appears to cause dry cough , high fever and ultimately chest congestion and pain resulting in breathlessness if not controlled .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Clinical aspects may be helpful : COVID-19 seems to cause more frequently fever and cough , while dyspnea occurs later and correlates with severe lung damage .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Initially , COVID-19 infects the respiratory tract of a human and causes fever , cough , shortness in breathing , and many other serious issues like pneumonia [ 1 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Consider whether fever is caused by a wound infection or COVID-19 [ 46 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "However , since the patient was confirmed to have COVID-19 later , the fever at that time might have been caused by SARS-CoV-2 infection or underlying pathology during the subclinical period .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 may increase insulin demand and induce fever , nausea and anorexia with consequent hyperketonemia , which accentuates the gastrointestinal symptoms of infection in a vicious cycle .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Some patients with DFUs may exhibit fever due to COVID-19 ; this fever is generally low-or middle-grade with respiratory symptoms .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Despite evidence that individual symptoms and signs have relatively poor diagnostic properties for COVID-19 in adults ,1 other causes of fever and cough are uncommon in healthy adults .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "1  Coronavirus disease ( COVID-19 ) causes fever , dry cough , dyspnea , loss of smell , and other nonspecific symptoms , meaning it is difficult to differentiate COVID-19 from other infectious diseases .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "On the second day of admission , the patient developed fever and cough due to COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "On the second day of admission , the patient developed fever and cough due to COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Specifically , 2019-nCoV infection can cause systemic inflammation , fever , hypoxia , electrolyte imbalance , acid-base disturbance and even shock .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "On the second day of admission , the patient developed fever and cough due to COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Specifically , 2019-nCoV infection can cause systemic inflammation , fever , hypoxia , electrolyte imbalance , acid-base disturbance and even shock .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "The coronavirus disease 2019 ( Covid-19 ) causes fever , cough , sore throat , and in severe cases shortness of breath and death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 can cause respiratory tract disease , fever and cough , and severe pneumonia in some extreme cases [ 2 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "It is reported that COVID-19 is a single-stranded RNA virus and usually causes respiratory symptoms and fever , as well as death in severe cases [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "It is reported that COVID-19 is a single-stranded RNA virus and usually causes respiratory symptoms and fever , as well as death in severe cases [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "It is reported that COVID-19 is a single-stranded RNA virus and usually causes respiratory symptoms and fever , as well as death in severe cases [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "It is reported that COVID-19 is a single-stranded RNA virus and usually causes respiratory symptoms and fever , as well as death in severe cases [ 4 , 5 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005334 ! Fever)\n",
      "COVID-19 can induce not only mild to severe respiratory diseases , but also inflammation , high fever , cough , acute respiratory tract infection , and dysfunction of internal organs which may result in death [ 4 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005756 ! Gastritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005756 ! Gastritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D005756 ! Gastritis)\n",
      "Based on these data , we can infer that SARS-CoV-2 infection may cause acute gastritis or enteritis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007362 ! \"Intensive Care Units\")\n",
      "SARS-CoV-2 infection can lead to intensive care unit ( ICU ) admission and high mortality [ 1 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007362 ! \"Intensive Care Units\")\n",
      "SARS-CoV-2 infection can lead to intensive care unit ( ICU ) admission and high mortality [ 1 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007362 ! \"Intensive Care Units\")\n",
      "SARS-CoV-2 infection can lead to intensive care unit ( ICU ) admission and high mortality [ 1 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007362 ! \"Intensive Care Units\")\n",
      "As it has been reported elsewhere , comorbidities play an important role in the risk of hospital admission.15 Clinical outcomes are poorer in patients with COVID-19 with a comorbid condition than in those without , and a greater number of comorbidities correlates with poorer clinical outcomes.16 Diabetes is a major comorbidity in COVID-19 , and patient 's history of diabetes is an independent risk factor for morbidity and mortality in this condition.17 18 Diabetes has been associated with admissions to the intensive care unit due to COVID-19 in recent series19 20 and has been shown to increase mortality.6 Therefore , we adjusted for comorbidity in the multivariable analysis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007362 ! \"Intensive Care Units\")\n",
      "17 18 Diabetes has been associated with admissions to Epidemiology the intensive care unit due to COVID-19 in recent series 19 20 and has been shown to increase mortality .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D007676 ! \"Kidney Failure, Chronic\")\n",
      "However , one study demonstrated that SARS-CoV-2 infection does not result in AKI or aggravate CKD in COVID-19 patients [ 10 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D009000 ! Monocytes)\n",
      "Since monocytes and macrophages also produce TF , monocyte , and macrophage upregulation by SARS-CoV-2 infection can also promote TF-initiated coagulation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D009000 ! Monocytes)\n",
      "Recent studies have shown that plasma concentrations of several inflammatory cytokines , such as granulocyte-macrophage colony-stimulating factor ( GM-CSF ) , interleukin ( IL ) -6 , tumour necrosis factor alpha ( TNF-alpha ) , IL-2 , 7 , 10 , and granulocyte colony-stimulating factor ( G-CSF ) , were increased after SARS-CoV-2 infection , which not only decreases the number of monocytes but also perturb peripheral monocytes phenotypes and functions65-67 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "Our results suggest that SARS-CoV-2 infection may induce myocardial injury and consequently exacerbate the clinical course and worsen prognosis .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "Recent reports have indicated that COVID-19 predisposes patients to thrombotic disease , with disseminated coagulation without bleeding , and contributes to poor outcome and prognosis [ 102-104 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "For this reason , LV and RV longitudinal strain are the main factors of prognosis due to COVID-19 early and more pronounced effect compared to other conventional echocardiography parameters .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "Recent reports have indicated that COVID-19 predisposes patients to thrombotic disease , with disseminated coagulation without bleeding , and contributes to poor outcome and prognosis [ 102-104 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "Recent reports have indicated that COVID-19 predisposes patients to thrombotic disease , with disseminated coagulation without bleeding , and contributes to poor outcome and prognosis [ 102 ] [ 103 ] [ 104 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011379 ! Prognosis)\n",
      "First , this population , though relatively young , had a significant burden of risk factors and chronic conditions that are associated with worse prognosis due to COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D011654 ! \"Pulmonary Edema\")\n",
      "Lethality of Covid-19 during the 2020 pandemic , currently in the exponentially-accelerating phase in most countries , is critically driven by disruption of the alveolo-capillary barrier of the lung , leading to lung edema as a direct consequence of SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D011654 ! \"Pulmonary Edema\")\n",
      "Another hypothesis states that SARS-CoV-2 infection blocks ACE2 function when binding to host cells , inhibiting its role of cleaving bradykinin and , as a consequence , bradykinin accumulates in the lung , promoting pulmonary edemas due to vasodilator activity and consequent respiratory failure .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D011654 ! \"Pulmonary Edema\")\n",
      "Another hypothesis states that SARS-CoV-2 infection blocks ACE2 function when binding to host cells , inhibiting its role of cleaving bradykinin and , as a consequence , bradykinin accumulates in the lung , promoting pulmonary edemas due to vasodilator activity and consequent respiratory failure .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "On the other hand direct activation of T cells by the SARS-CoV-2 infection leads to the activation of pathogenic T helper 1 ( Th1 ) cells that secrete GM-CSF and produce inflammatory cytokine IL-17 which recruits macrophages and neutrophils to the site of infection and stimulates IL-1beta and IL-6 cytokine cascades , where IL-6 plays the major role in development of cytokine storm in COVID-19 patients ( Wu and Yang , 2020b ; Y. Zhou et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "On the other hand direct activation of T cells by the SARS-CoV-2 infection leads to the activation of pathogenic T helper 1 ( Th1 ) cells that secrete GM-CSF and produce inflammatory cytokine IL-17 which recruits macrophages and neutrophils to the site of infection and stimulates IL-1beta and IL-6 cytokine cascades , where IL-6 plays the major role in development of cytokine storm in COVID-19 patients ( Wu and Yang , 2020 ; Y. Zhou et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D013601 ! T-Lymphocytes)\n",
      "COVID-19 induces reduction of lymphocytes , mainly CD4 + and CD8 + T cells , and secondary viral sepsis [ 47 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D016178 ! \"Granulocyte-Macrophage Colony-Stimulating Factor\")\n",
      "A cytokine storm is directly or indirectly caused by SARS-CoV-2 infection , which can activate pathogenic T cells and produce IL-6 and granulocyte-macrophage colony stimulating factor ( GM-CSF ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D016178 ! \"Granulocyte-Macrophage Colony-Stimulating Factor\")\n",
      "A cytokine storm is directly or indirectly caused by SARS-CoV-2 infection , which can activate pathogenic T cells and produce IL-6 and granulocyte-macrophage colony stimulating factor ( GM-CSF ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D035683 ! MicroRNAs)\n",
      "In this study , we analyzed publicly available genomic datasets to systematically dissect the crosstalk and dysregulation of the human post-transcriptional regulatory networks governed by RNA-binding proteins ( RBPs ) and micro-RNAs ( miRs ) due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases bp(MESH:D065206 ! \"Extracellular Traps\")\n",
      "SARS-CoV-2 infection induces RAS overactivation , NETs formation , cytokine overproduction , and hyperinflammation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(UP:P0DTC2 ! P0DTC2)\n",
      "The claims that COVID-19 disproportionately affects the individuals of minority groups and aged people are not only supported by reported data but also by our hypothesis that SARS-CoV-2 infection generates spike protein that interacts with ACE2 to either exhaust ACE2 or inhibit ACE2 function or both so that the comorbidities are aggravated ( Figure 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") increases p(UP:P0DTC2 ! P0DTC2)\n",
      "The claims that COVID-19 disproportionately affects the individuals of minority groups and aged people are not only supported by reported data but also by our hypothesis that SARS-CoV-2 infection generates spike protein that interacts with ACE2 to either exhaust ACE2 or inhibit ACE2 function or both so that the comorbidities are aggravated ( Figure 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases a(CHEBI:22587 ! \"antiviral agent\")\n",
      "MEKi may attenuate SARS-CoV-2 infection to allow immune responses and antiviral agents to control disease progression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases a(CHEBI:22587 ! \"antiviral agent\")\n",
      "MEKi may attenuate SARS-CoV-2 infection to allow immune responses and antiviral agents to control disease progression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases a(CHEBI:36080 ! protein)\n",
      "SARS-CoV-2 infection inhibits protein synthesis and enhance the process of protein refolding .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases a(CHEBI:36080 ! protein)\n",
      "SARS-CoV-2 infection inhibits protein synthesis and enhance the process of protein refolding .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0000050\" ! \"urea cycle\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0000050\" ! \"urea cycle\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D  and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0006541\" ! \"glutamine metabolic process\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0006541\" ! \"glutamine metabolic process\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D  and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Not only that , recent studies have also found that SARS-CoV-2 infection can also lead to a reduction in autophagy .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0046655\" ! \"folic acid metabolic process\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(GO:\"GO:0046655\" ! \"folic acid metabolic process\")\n",
      "SARS-CoV-2 infection downregulated urea cycle in airway organoids and downregulated folate metabolism , glutamine metabolism and urea cycle in alveolar organoids ( Fig. S4D  and S4E ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Another interesting aspect that could be explored in patients who have overcome the disease is the possible onset or persistence of the alterations above described in the organs and systems and the evaluation of whether they are transient or permanent ( long-term COVID-19 ) , to assess the extent of ACE2 activity impairment due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "It is believed that a SARS-CoV-2 infection reduces the expression of ACE2 , thereby disrupts the RAS .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "When considering to 76 % identical sequence of S protein between SARS-CoV-2 and SARS-CoV , SARS-CoV-2 infection might downregulate the ACE2 expression through the internalization of the virus to cell [ 18 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "J o u r n a l P r e - p r o o f Journal Pre-proof early stage of the infection , before COVID-19 reduces ACE2 expression and activity [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "When considering to 76 % identical sequence of S protein between SARS-CoV-2 and SARS-CoV , SARS-CoV-2 infection might downregulate the ACE2 expression through the internalization of the virus to cell [ 18 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Thus , chloroquine treatment might be more effective in the early stage of the infection , before COVID-19 reduces ACE2 expression and activity [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Thus , the depletion of ACE2 by SARS-CoV-2 infection results in impaired DABK inactivation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "The continued depletion of ACE2 by SARS-CoV-2 infection increases the extracellular levels of DABK and BK via the kinin-kallikrein pathway , as well as the injury and inflammation in lung tissue [ 19 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "It is also thought that SARS-CoV-2 infection downregulates ACE2 expression , reducing its protective role and explaining the progression of patients into ARDS [ 23 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "It is also thought that SARS-CoV-2 infection downregulates ACE2 expression , reducing its protective role and explaining the progression of patients into ARDS [ 23 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "It is also thought that SARS-CoV-2 infection downregulates ACE2 expression , reducing its protective role and explaining the progression of patients into ARDS [ 23 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "* Chloroquine can inhibit the entry of SARS-CoV-2 and prevent virus-cell fusion by interfering with glycosylation of the ACE2 receptor and its binding with the spike protein , suggesting that chloroquine treatment might be more effective in the early stage of infection before COVID-19 reduces ACE2 expression and activity .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "AAs have a genetic polymorphism predisposing them to angioedema , thus a second-hit mechanism of ACE2 dysregulation by COVID-19 may underlie the angioedema of our cohort .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Thus , chloroquine treatment might be more effective in the early stage of the infection , before COVID-19 reduces ACE2 expression and activity [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Thus , chloroquine treatment might be more effective in the early stage of the infection , before COVID-19 reduces ACE2 expression and activity [ 16 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Figure 2 : SARS-CoV-2 infection down-regulates ACE2 expression and leads to the production of pro-inflammatory mediators , such as IL-6 [ 35 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV-2 infection is shown to down-regulate ACE2 , which may contribute to myocardial dysfunction [ 51 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Another interesting aspect that could be explored in patients who have overcome the disease is the possible onset or persistence of the alterations above described in the organs and systems and the evaluation of whether they are transient or permanent ( long-term COVID - 19 ) , to assess the extent of ACE2 activity impairment due to SARS-CoV-2 infection .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "15 Interestingly , in several Wuhan cohorts cardiac injury and arrythmia were prominent in high-risk Figure 3 Homeostasis of RAS-ACE2 under normal healthy conditions 10 19 ; perturbation of RAS-ACE2 homeostasis in cardiovascular disease , hypertension and diabetes mellitus 22 27 ; COVID-19 may potentially further upregulate RAS in CVD patients and deplete ACE2 33 ; proposition that rhACE2 replacement therapy improves RAS-ACE2 balance by augmenting ACE2 and decreasing RAS activation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Chloroquine can inhibit the entry of SARS-CoV-2 and prevent virus-cell fusion by interfering with glycosylation of ACE2 receptor and its binding with spike protein , suggesting that chloroquine treatment might be more effective in the early stage of infection , before COVID-19 reduces ACE2 expression and activity [ 30 , 38 , 39 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "3  It is proposed that blocking of ACE2 by COVID-19 decreases the conversion of angiotensin II to angiotensin 1 to 7 with a resultant increase in angiotensin II levels .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "SARS-CoV-2 infection is shown to down-regulate ACE2 , which may contribute to myocardial dysfunction [ 51 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "It has been shown that COVID-19 downregulates angiotensin converting enzyme-2, which has protective effects on the endothelium and cardiomyocytes.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Therefore , SARS-CoV infection and S protein on the virion surface reduce the cell-surface expression of ACE2 by both internalization and shedding of ACE2.29 ACE2 is a homologue of ACE and participates in the degradation of DABK by cleaving it into inactive products.30 Partly owing to DABK / B1R axis mediated chemokines release including C-X-C motif chemokine 5 , macrophage inflammatory protein-2 ( MIP2 ) , C-X-C motif chemokine 1 , and TNF-alpha , ACE2 plays an essential role in the development of pulmonary inflammation.30 Thus , the depletion of ACE2 by SARS-CoV-2 infection results in impaired DABK inactivation .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "The continued depletion of ACE2 by SARS-CoV-2 infection increases the extracellular levels of DABK and BK via the kinin-kallikrein pathway , as well as the injury and inflammation in lung tissue.19 Cellular injury and inflammation upregulate the expression of B1R.65 ,66 The excessive generation of BK and DABK , in addition to the upregulation of B1R , contribute to angioedema .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Other conditions that are exacerbated by high ACE activity , such as prostate cancer ( 6 ) , are potentially affected by SARS-CoV-2 infection , which reduces ACE2 since reduced ACE2 implies upregulated ACE activity ( 4 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "However , SARS-CoV-2 infection induces a reduction in ACE2 levels and disrupts its protective function in the respiratory and cardiovascular systems , as well as other important organs .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "However , SARS-CoV-2 infection induces a reduction in ACE2 levels and disrupts its protective function in the respiratory and cardiovascular systems , as well as other important organs .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:13557 ! ACE2)\n",
      "Studies have shown that COVID-19 downregulates ACE2 expression and hinders it 's organoprotective effect .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:1663 ! CD36)\n",
      "SARS-CoV-2 infection increased blood levels of arachidonic acid ( AA ) , 5-HETE , 11-HETE , sCD14 , and ACh but decreased monocyte CD14 and CD36 expression .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases p(HGNC:3621 ! FCGRT)\n",
      "Although SARS-CoV-2 infection did not disrupt FcRn expression in the placenta , maternal transfer of nAb was inhibited by SARS-CoV-2 infection during pregnancy .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:C413524 ! \"ACE2 enzyme\")\n",
      "The imbalance in the RAS resulting from ACE2 loss , and the `` cytokine storm '' Considering the above data from the medical literature , the SARS-CoV-2 infection down-regulates the ACE2 enzyme , leading to an imbalance in the inhibitory function of the ACE2 on the RAS cascade .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:C413524 ! \"ACE2 enzyme\")\n",
      "The imbalance in the RAS resulting from ACE2 loss , and the `` cytokine storm '' Considering the above data from the medical literature , the SARS-CoV-2 infection down-regulates the ACE2 enzyme , leading to an imbalance in the inhibitory function of the ACE2 on the RAS cascade .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:C413524 ! \"ACE2 enzyme\")\n",
      "It has been shown that COVID-19 downregulates angiotensin-converting enzyme-2 , which has protective effects on the endothelium and cardiomyocytes .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "It is believed that a SARS-CoV-2 infection reduces the expression of ACE2 , thereby disrupts the RAS .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "ased on the observation that patients with hypertension appear to have adverse outcomes with coronavirus disease-2019 (COVID-19) infection and the discovery that the severe acute respiratory syndrome-coronavirus-2 (SARS-CoV-2) virus, which causes COVID-19, uses a component of the renin-angiotensin system (RAS) as its route of cellular infection, there has been tremendous interest in determining the net clinical effect of RAS-blocking agents in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "Full determination of the overall net clinical benefit or harm of RAS-blocking agents in COVID-19 requires carefully designed prospective randomized controlled clinical trials.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "The ACE2 enzyme is found in lung alveolar tissue, nasopharyngeal tissue, enterocytes, nervous tissue, vascular tissue, and the kidney (1)—all points of potential viral entry for SARS-CoVID-2.The initial thinking regarding RAS-blocking agents and COVID-19 was that these drugs could have adverse effects due to up-regulation of ACE2 expression on cell surfaces leading to a greater number of potential sites of viral entry.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "Renin-Angiotensin System Blockade in COVID-19.Based on the observation that patients with hypertension appear to have adverse outcomes with coronavirus disease-2019 (COVID-19) infection and the discovery that the severe acute respiratory syndrome-coronavirus-2 (SARS-CoV-2) virus, which causes COVID-19, uses a component of the renin-angiotensin system (RAS) as its route of cellular infection, there has been tremendous interest in determining the net clinical effect of RAS-blocking agents in COVID-19 patients.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "A second mechanism proposed is hypoperfusion due to cerebral blood flow dysautoregulation , secondary to disruption of the renin-angiotensin system ( RAS ) by COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "A second mechanism proposed is hypoperfusion due to cerebral blood flow dysautoregulation , secondary to disruption of the renin-angiotensin system ( RAS ) by COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "The upregulation of ACE2 can be explained as COVID-19 inhibits RAS , which is the main pathway of ACE2 peptide [ 48 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012867 ! Skin)\n",
      "Dermal microvascular injury and thrombosis as described in skin damage by SARS-CoV-2 infection was not observed .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012867 ! Skin)\n",
      "Dermal microvascular injury and thrombosis as described in skin damage by SARS-CoV-2 infection was not observed .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D012867 ! Skin)\n",
      "Prevention and occupational hazards for the skin during COVID-19 pandemic The life of medical specialists worldwide has dramatically changed due to the spread of the coronavirus disease 2019 ( COVID-19 ) pandemic .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D017951 ! \"Antigen Presentation\")\n",
      "Although , SARS-CoV-2 infection might induce impaired antigen presentation and interferon responses , production of INF-a / b may cause a local primary inflammatory response in infected tissues and release of proinflammatory cytokines and chemokines leading to both viricidal and tissue damage [ 5 , 61 , 62 ] ( Fig. 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D017951 ! \"Antigen Presentation\")\n",
      "Although , SARS-CoV-2 infection might induce impaired antigen presentation and interferon responses , production of INF-alpha / beta may cause a local primary inflammatory response in infected tissues and release of proinflammatory cytokines and chemokines leading to both viricidal and tissue damage [ 5 ] , [ 61 ] , [ 62 ] ( Fig. 1 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infection\") decreases bp(MESH:D032383 ! \"Myocytes, Cardiac\")\n",
      "Moreover , as the myocardial interaction network of ACE2 and its own expression is altered in patients with coexisting CVD , SARS-CoV-2 infection may result in greater damage to cardiomyocytes and account for greater disease acuity and poorer survival .\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases a(TEXT:\"S-containing 90 virus-like particles\")\n",
      "89 Furthermore, since SARS-CoV-1 has been proposed to induce the release of S containing 90 virus like particles (VLPs), though through poorly characterized processes, we also aimed at 91 clarifying the minimal requirement for production of S containing VLPs for SARS-CoV-2.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases a(TEXT:\"type I IFNs\")\n",
      "The impaired IFN 224 production is consistent with the reported inhibition of type I IFNs by SARS-CoV-1 and the 225 lack of interferon regulatory factor 3 activation in macrophages and myeloid dendritic cells 226 (5).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases a(TEXT:\"type I IFNs\")\n",
      "The impaired IFN 225 production is consistent with the reported inhibition of type I IFNs by SARS-CoV-1 and the 226 lack of interferon regulatory factor 3 activation in macrophages and myeloid dendritic cells 227 (5).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "These studies have not been replicated in SARS-CoV-2, but if the same mechanism exists, it provides a strong rationale for the early use of anti-TNF in the disease course.There is massive neutrophilia in COVID-19, with elevated blood levels of major neutrophil constituent proteins including S-100A8/9.41 Neutrophil extracellular trap (NET) release, or NETosis, is a mechanism whereby neutrophils release sticky extracellular traps that help limit the spread of infection.42 Levels of NETs are higher in patients with COVID-19 compared with healthy controls, and SARs-CoV-2 virus has been shown to directly promote the release of NETs from neutrophils.43 NETs are prothrombotic and activate platelets and the clotting cascade, leading to the production of thrombi.44 In COVID-19 autopsies, NET containing microthrombi have been found in the lung.45,46 NETs have also been shown to induce pulmonary epithelial cell death, leading to alveolar damage and fibrosis in COVID-19 patients.43 These findings have led authors to suggest that agents targeting NETosis may be effective in COVID-19.43,46 Of relevance, anti-TNF therapy reduces NET formation in vitro as well as in murine models and patients with immune mediated disease.47, 48, 49TNF is involved in the defense against viruses, so a key question is whether anti-TNF interferes with antiviral immunity.50 A number of viral infections have been treated with anti-TNF, including SARS-CoV-1, respiratory syncytial virus (RSV), dengue, and influenza.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Immunity to HCoV-OC43 and HCoV-HKU1 declines within one year, while SARS-CoV-1 can induce longer immunity (Kissler et al., 2020).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Type I interferons have antiviral properties and may be actively repressed by SARS-CoV-1 and SARS-CoV-2 during infection to escape the host immune response.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "However, among older adults (age> = 65 years), male sex was not associated with a more severe illness and worse outcomes.In the previous influenza and novel coronavirus epidemics, SARS-CoV-1 and Middle East respiratory syndrome (MERS), in addition to virus induced cytopathic effects, an excessive and dysregulated host immune response played a crucial role in the pathology and mortality [12,13].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(HGNC:13557 ! ACE2)\n",
      "In prior coronavirus outbreaks, SARS-CoV-1 targeted the angiotensin converting enzyme 2 (ACE2) receptor, whereas MERS-CoV utilized the dipeptidyl peptidase 4 (DPP4) receptor [2, [9] [10].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(HGNC:13557 ! ACE2)\n",
      "The process is similar to the beta-coronaviruses HCoV-NL63 and SARS-CoV-1, which also target ACE2 for cellular entry (19).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(HGNC:13557 ! ACE2)\n",
      "However, SARS-CoV-1 can reduce angiotensin II (Ang II) clearance by reducing ACE2, and Ang II can induce TGF-b expression [25].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(HGNC:13557 ! ACE2)\n",
      "Similar to SARS-CoV-1, cellular entry of SARS-CoV-2 depends on the expression of ACE2, a receptor that is abundantly expressed in the nervous system.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases p(HGNC:13557 ! ACE2)\n",
      "Since SARS-CoV-1 downregulates ACE2, XREF_BIBR SARS-CoV-2 is expected to downregulate ACE2 as well.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases p(HGNC:13557 ! ACE2)\n",
      "XREF_BIBR Interestingly, ACE2 is downregulated by SARS-CoV-1, XREF_BIBR and a similar mechanism is believed to occur in SARS-CoV-2 infection as well.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases p(HGNC:13557 ! ACE2)\n",
      "Since SARS-CoV-1 downregulates ACE2,126 SARS-CoV-2 is expected to downregulate ACE2 as well.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(HGNC:5992 ! IL1B)\n",
      "Furthermore, SARS-CoV-1 triggers the production of IL-1beta through the activation of the inflammasome (31).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV-1 was also shown to directly cause the death of cerebral neurons in ACE2 transgenic mice, but without the pathological changes of encephalitis [35].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Amino acid substitutions in RBD domain of SARS-CoV-1 can cause ADE via neutralizing antibodies that were developed to another virus variant [XREF_BIBR].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Due to the short time-lapse from the discovery of COVID-19, the majority of studies that were appraised in the present review are related to the SARS outbreak caused by SARS-CoV-1.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "2,3 Different interventions have been implemented based on previous experience with other coronavirus diseases such as SARS caused by SARS-CoV-1, and Middle Eastern Respiratory Syndrome (MERS) caused by MERS-CoV.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Specifically, mouse models were previously utilised to investigate diseases caused by other coronaviruses, for example severe acute respiratory syndrome (SARS caused by SARS-CoV-1) and Middle East respiratory syndrome (MERS) [60-62].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "Specifically, mouse models were previously utilised to investigate diseases caused by other coronaviruses, for example severe acute respiratory syndrome (SARS caused by SARS-CoV-1) and Middle East respiratory syndrome (MERS) [60] [61] [62].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In COVID-19, the \" cytokine storm \" bears resemblance to SARS caused by the SARS-CoV-1, where cytokine storm has been associated with disease [100] [101] [102].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "In COVID-19, the \" cytokine storm \" bears resemblance to SARS caused by the SARS-CoV-1, where cytokine storm has been associated with disease [[100], [101], [102]].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The previous SARS epidemic caused by SARS-CoV-1 and its infection mechanisms were widely studied.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The previous SARS epidemic caused by SARS-CoV-1 and its infection mechanisms were widely studied [2].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The surfactant protein D (SP-D) is known to bind to the Spike protein of SARS-CoV-1, the virus that caused the SARS outbreak in 2002.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "For reasons yet to be fully understood the COVID-19 outbreak caused by SARS-CoV-2 has been significantly more widespread than the 2003 SARS epidemic caused by SARS-CoV-1, despite striking similarities between the two viruses.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "For reasons yet to be fully understood the COVID-19 outbreak caused by SARS-CoV-2 has been significantly more widespread than the 2003 SARS epidemic caused by SARS-CoV-1, despite striking similarities between the two viruses.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases a(TEXT:PLpro)\n",
      "In this context, a study by Shin et al.1 aimed at identifying similarities and key differences in the activities of PLpro produced by SARS-CoV-1 (SCoV1-PLpro) and by SARS-CoV-2 (SCoV2-PLpro).\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases p(FPLX:TGFB ! TGFB)\n",
      "The nucleocapsid protein of SARS-CoV-1 can directly promote the upregulation of TGF-b expression [24] [25] [26] [27].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "Similar to SARS-CoV-1, cellular entry of SARS-CoV-2 depends on the expression of ACE2, a receptor that is abundantly expressed in the nervous system.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "Here, we used publicly archived transcriptomic datasets to compute consensus regulatory signatures, or consensomes, that rank human genes based on their rates of differential expression in MERS-CoV (MERS), SARS-CoV-1 (SARS1) and SARS-CoV-2 (SARS2)-infected cells.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) directlyIncreases p(UP:P14107 ! P14107)\n",
      "However, ORF7/8 is 21 only expressed by SARS-CoV-1 and SARS-CoV-2 with little homology to other seasonal 22 coronaviruses,hencetheroleofsuchpeptidecross-reactivecellsispuzzlingandcallsfora 23 moredetailedanalysisoftheeffectofpre-existingimmunityinthecontrolorpathogenesis 10 bioRxiv preprint doi : https://doi.org/10.1101/2020.10.15.341958.\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases a(TEXT:IFN-I)\n",
      "The SARS-CoV-1 and SARS-CoV-2 ORF6 proteins inhibit IFN-I and IFN-III secretion induced by Sendai virus infection, a method often used to elicit IFN-I production, and have profound effects on IRF3 phosphorylation, thus accounting for their potent inhibition of IFN-I production [20, 46].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases a(TEXT:IFN-III)\n",
      "The SARS-CoV-1 and SARS-CoV-2 ORF6 proteins inhibit IFN-I and IFN-III secretion induced by Sendai virus infection, a method often used to elicit IFN-I production, and have profound effects on IRF3 phosphorylation, thus accounting for their potent inhibition of IFN-I production [20, 46].\n",
      "\n",
      "a(TEXT:SARS-CoV-1) decreases bp(MESH:D007370 ! \"Interferon Type I\")\n",
      "Type I interferons have antiviral properties and may be actively repressed by SARS-CoV-1 and SARS-CoV-2 during infection to escape the host immune response.\n",
      "\n",
      "a(TEXT:\"SARS CoV-2\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "SARS CoV-2, the virus that causes COVID-19, has shown the ability to become aerosolized with a potential airborne route of transmission.\n",
      "\n",
      "a(TEXT:\"SARS nsp3\") increases p(HGNC:11998 ! TP53)\n",
      "We discuss the mechanisms by which SARS nsp3 protein targets p53 for degradation and we speculate on the significance for Covid-19 pathogenesis and risk of cancer.\n",
      "\n",
      "a(TEXT:\"SARS-163 CoV-2\") increases a(CHEBI:51079 ! GS26575)\n",
      "The 162 expression of polarization related genes was investigated after 24 and 48 hours of SARS-163 CoV-2 stimulation of M1 and M2 polarized macrophages.\n",
      "\n",
      "a(TEXT:SARS-176CoV-2) increases a(CHEBI:33290 ! food)\n",
      "For this reason, the development of detection tools for SARS-176CoV-2 that can be applied to foods is essential to ensure food safety and prevent the disruption of food supply 177 chains.\n",
      "\n",
      "a(TEXT:SARS-2-S-HDL) increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "Fig. 2 Binding of SARS-2-S-HDL enhances SARS-CoV-2 entry.\n",
      "\n",
      "a(TEXT:\"SARS-280 CoV-2\") increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Interestingly, we found that SARS-280 CoV-2 infection induced \" squamous metaplasia \" in the COPD epithelium.\n",
      "\n",
      "a(TEXT:SARS-COV-19) increases bp(MESH:D012128 ! \"Respiratory Distress Syndrome\")\n",
      "The current data show that liquid surfactant delivery to patients with SARS-COV-19 induced ARDS is feasible and well tolerated.\n",
      "\n",
      "a(TEXT:\"SARS-COV-2 proteins\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Entos is working on developing an optimized payload containing multiple protein epitopes derived from SARS-COV-2 proteins, which will stimulate an immune response in the body to prevent COVID-19 infection.\n",
      "\n",
      "a(TEXT:\"SARS-CoV 9b protein\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "SARS-CoV 9b protein diffuses into nucleus, undergoes active Crm1 mediated nucleocytoplasmic export and triggers apoptosis when retained in the nucleus.\n",
      "\n",
      "a(TEXT:\"SARS-CoV Mpro mutation\") increases p(FPLX:Protease ! Protease)\n",
      "A recent study 55 characterised a SARS-CoV Mpro mutation, S284-T285-I286/A, which dynamically enhanced the protease catalytic activity 56 more than three-fold (16).57 A. ENM Results for Allosteric Control.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") increases p(FPLX:p38 ! p38)\n",
      "SARS-CoV N protein also activates the p38 MAPK pathway to induce the reorganisation of microfilaments in cells devoid of growth factors.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Surjit et al. demonstrated that the SARS-CoV N protein induces apoptosis in COS-1 cells via downregulation of ERK in cells that are free from growth factors [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases p(FPLX:IFNA ! IFNA)\n",
      "The SARS-CoV N protein inhibits IFN-alpha and beta production induced by both SeV and polyI : C whereas, by contrast, it is not able to suppress type I IFN induction upon overexpression of components such as RIG-I-CARD, MAVS, TBK1 and IKK-epsilon, suggesting that the N protein blocks the initial step of dsRNA recognition by RLRs.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases bp(GO:\"GO:0000910\" ! cytokinesis)\n",
      "Furthermore, Zhou and coworkers demonstrated that the SARS-CoV N protein inhibited protein translation and cytokinesis by aggregating EF1alpha.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Furthermore, Zhou and coworkers demonstrated that the SARS-CoV N protein inhibited protein translation and cytokinesis by aggregating EF1alpha.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "It was reported that SARS-CoV N protein could inhibit protein translation, therefore we also used N protein as a control, but we only could observe inhibition at low dosage (XREF_FIG C).\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases p(HGNC:5434 ! IFNB1)\n",
      "Perhaps, SARS-CoV N protein could inhibit IFN-beta production by its peptide or other unclear mechanism.\n",
      "\n",
      "a(TEXT:\"SARS-CoV N protein\") decreases p(UP:P85915 ! P85915)\n",
      "Furthermore, Zhou and coworkers demonstrated that the SARS-CoV N protein inhibited protein translation and cytokinesis by aggregating EF1alpha.\n",
      "\n",
      "a(TEXT:\"SARS-CoV Nsp15\") increases a(TEXT:\"SARS-CoV-2 homolog [\")\n",
      "Based on the structural comparisons, it has been suggested that inhibitors of SARS-CoV Nsp15 may also inhibit the SARS-CoV-2 homolog [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") increases a(CHEBI:3815 ! Collagen)\n",
      "This study indicated that SARS-CoV PLpro stimulated the production of Type I collagen in vitro and in vivo (XREF_FIG, XREF_FIG, XREF_FIG).\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") increases a(CHEBI:3815 ! Collagen)\n",
      "The results showed that SARS-CoV PLpro stimulated TGF-beta1-dependent expression of Type I collagen via activating STAT6 pathway.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") decreases a(CHEBI:3815 ! Collagen)\n",
      "Specific inhibitors for TGF-betaRI, p38 MAPK, MEK, and STAT3 proved that SARS-CoV PLpro induced TGF-beta1-dependent up-regulation of Type I collagen in vitro and in vivo.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") increases p(HGNC:11766 ! TGFB1)\n",
      "To examine the association of SARS-CoV PLpro induced TGF-beta1 production with the collagen up-regulation, A549 lung epithelial cells transiently transfected with pcDNA3.1 and pSARS-PLpro were analyzed the production of TGF-beta1 and type I collagen using Western blot, real-time RT-PCR and Sirius red staining assays (XREF_FIG).\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") increases p(HGNC:11766 ! TGFB1)\n",
      "3.1 Correlation of SARS-CoV PLpro induced TGF-beta1 production with Type I collagen up-regulation.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") decreases p(FPLX:SMAD ! SMAD)\n",
      "The result indicated that SARS-CoV PLpro inactivated SMAD dependent pathways, implying that non SMAD pathways in TGF beta1signaling for the production of Type I collagen would be initiated by SARS-CoV PLpro.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PLpro\") decreases p(HGNC:15631 ! TLR7)\n",
      "These effects were shown to occur in Tolllike receptor 3 and retinoic acid inducible gene 1 pathways.Moreover, it has been demonstrated that SARS-CoV PLpro inhibits the TLR7 pathway via inactivation of TRAF3/6-TBK 1-IRF3/NF-kB/AP1 signalling pathways (Yuan et al., 2015).\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein 8b\") increases p(HGNC:16400 ! NLRP3)\n",
      "XREF_BIBR Finally, SARS-CoV accessory protein 8b interacts with the LRR domain of NLRP3 and also stimulates NLRP3 activation and IL-1beta release along with forming insoluble intracellular aggregates to activate autophagy and ER stress pathways.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein 8b\") increases p(HGNC:5992 ! IL1B)\n",
      "XREF_BIBR Finally, SARS-CoV accessory protein 8b interacts with the LRR domain of NLRP3 and also stimulates NLRP3 activation and IL-1beta release along with forming insoluble intracellular aggregates to activate autophagy and ER stress pathways.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp12\") increases p(PFAM:PF02689 ! helicase)\n",
      "Adedeji and coworkers showed that SARS-CoV nsp12 enhances the helicase activity of nsp13 by two-fold.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp12\") increases p(PFAM:PF06460 ! nsp13)\n",
      "Adedeji and coworkers showed that SARS-CoV nsp12 enhances the helicase activity of nsp13 by two-fold.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nucleocapsid protein\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "A study on SARS-CoV which was responsible for the worldwide outbreak of SARS in 2003 showed that the SARS-CoV nucleocapsid protein (N protein) activates NF-kappaB in Vero E6 cells in a dose dependent manner.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nucleocapsid protein\") increases p(UP:P03126 ! P03126)\n",
      "A study on SARS-CoV which was responsible for the worldwide outbreak of SARS in 2003 showed that the SARS-CoV nucleocapsid protein (N protein) activates NF-kappaB in Vero E6 cells in a dose dependent manner.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nucleocapsid protein\") increases a(TEXT:Ciclooxygenase-2)\n",
      "Data show that SARS-CoV nucleocapsid protein stimulates the expression of Ciclooxygenase-2 (COX-2), an essential enzyme in the catalyses of prostanoids production from fatty acids, as those found at m/z 407.1821 in positive group (25).\n",
      "\n",
      "a(TEXT:\"SARS-CoV open reading frame (ORF) 8b protein\") increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "The SARS-CoV open reading frame (ORF) 8b protein has also been found to undergo aggregation in infected cells, and induce ER stress (Shi et al., 2019).\n",
      "\n",
      "a(TEXT:\"SARS-CoV papain-like protein\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "An interesting observation concerning a SARS-CoV papain like protein, which strongly triggered the ROS and p38 MAPK and STAT3 signaling pathway leading to the activation of the TGF-beta1 promoter in the epithelium cells of the lungs has been made [5].\n",
      "\n",
      "a(TEXT:\"SARS-CoV papain-like protein\") increases p(FPLX:p38 ! p38)\n",
      "An interesting observation concerning a SARS-CoV papain like protein, which strongly triggered the ROS and p38 MAPK and STAT3 signaling pathway leading to the activation of the TGF-beta1 promoter in the epithelium cells of the lungs has been made [5].\n",
      "\n",
      "a(TEXT:\"SARS-CoV papain-like protein\") increases p(HGNC:11364 ! STAT3)\n",
      "An interesting observation concerning a SARS-CoV papain like protein, which strongly triggered the ROS and p38 MAPK and STAT3 signaling pathway leading to the activation of the TGF-beta1 promoter in the epithelium cells of the lungs has been made [5].\n",
      "\n",
      "a(TEXT:\"SARS-CoV proteins\") increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Several SARS-CoV proteins have been shown to induce the ER stress response such as the Spike protein [XREF_BIBR, XREF_BIBR], upregulating GRP78/94 and the PERK pathway, while 3a and 8ab proteins were shown to induce lysosomal damage [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV proteins\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Secondly, other SARS-CoV proteins have been shown to induce apoptosis.\n",
      "\n",
      "a(TEXT:\"SARS-CoV proteins\") increases bp(GO:\"GO:0006950\" ! \"response to stress\")\n",
      "Several SARS-CoV proteins have been shown to induce the ER stress response such as the Spike protein [XREF_BIBR, XREF_BIBR], upregulating GRP78/94 and the PERK pathway, while 3a and 8ab proteins were shown to induce lysosomal damage [XREF_BIBR, XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV proteins\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "This is of utmost importance given that 478 several SARS-CoV proteins have been reported to suppress host translation and that 479 numerous genes involved in innate immunity act to block mRNA translation.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-1 ORF3b\") increases p(HGNC:3796 ! FOS)\n",
      "c-FOS is activated by SARS-CoV-1 ORF3b [82] and TCF-7 is expressed in lung cells [83].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-1 S\") increases bp(MESH:D014780 ! Viruses)\n",
      "Of note, non neutralizing antibody epitopes of SARS-CoV-1 S were previously found to enhance virus infection (Wang et al., 2016) and it was suggested that anti-S IgG from severely ill COVID-19 patients may promote hyper-inflammatory responses (Hoepel et al., 2020).\n",
      "\n",
      "a(TEXT:SARS-CoV-19) increases bp(MESH:C000657245 ! Covid-19)\n",
      "The outbreak of Covid-19 disease caused by SARS-CoV-19, along with the lack of targeted medicaments and vaccines, forced the scientific world to search for new antiviral formulations.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 -a coronavirus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "As of September 2020, SARS-CoV-2 -a coronavirus which likely originated in Wuhan, China- that causes COVID-19 - has been ravaging the world (almost 30 million people infected), causing the deaths of almost 1,000,000 people (at the time of writing) 1.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 154\") increases p(HGNC:11892 ! TNF)\n",
      "In 153 addition, the details of NF-KAPPA B and TNF signaling pathway activated by SARS-CoV-2 154 infection were also shown in Figure S1-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 191\") increases p(HGNC:6116 ! IRF1)\n",
      "190 It is worth mentioning that, IRF1 in respiratory tract cells was up-regulated by SARS-CoV-2 191 infection.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 424\") increases p(FPLX:Interferon ! Interferon)\n",
      "In Calu-3 cells, SARS-CoV-2 424 induced more IFN mRNA than WT MERS-CoV, closer to the level of MERS-CoV-DNS4ab (Fig 425 7A).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 432 miRNAs\") increases a(TEXT:\"Target genes\")\n",
      "Target genes downregulated by SARS-CoV-2 432 miRNAs are found to be involved in Ca2+ signaling pathway which is considered important 433 activators of many signaling pathways (Zhou et al., 2009).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 AP66\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "a CPE induced by SARS-CoV-2 AP66 and SARS-CoV-2 GZ69 in Vero E6 at P0 (72 h p.i.) (Original magnification 10x).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 AP66\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "This result shows that the absence of CPE in Vero E6 cells does not reflect the SARS-CoV-2 GZ69 replicative capacity.A Whole genome sequencing (WGS) approach was applied to evaluate if differences in CPE induced by SARS-CoV-2 GZ69 and SARS-CoV-2 AP66 were mirrored by genetic variations between the two isolates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 AP66\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "A Whole genome sequencing (WGS) approach was applied to evaluate if differences in CPE induced by SARS-CoV-2 GZ69 and SARS-CoV-2 AP66 were mirrored by genetic variations between the two isolates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Assay\") increases a(TEXT:N-gene)\n",
      "To evaluate the automated cartridge based PCR approach ARIES SARS-CoV-2 Assay targeting the ORF-sequence and the N gene of SARS-CoV-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Assay\") increases a(TEXT:N-gene)\n",
      "Introduction To evaluate the automated cartridge based PCR approach ARIES SARS-CoV-2 Assay targeting the ORF-sequence and the N gene of SARS-CoV-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Assay\") increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "To evaluate the automated cartridge based PCR approach ARIES SARS-CoV-2 Assay targeting the ORF-sequence and the N gene of SARS-CoV-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Assay\") increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "Introduction To evaluate the automated cartridge based PCR approach ARIES SARS-CoV-2 Assay targeting the ORF-sequence and the N gene of SARS-CoV-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 CD4+\") increases p(HGNC:5981 ! IL17A)\n",
      "Future 272 studies are warranted to determine if SARS-CoV-2 CD4+ T cells from severely ill patients, 273 particularly those in the ICU or those who succumb to disease, produce IL4, IL17, or IL6.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 CD4+\") increases p(HGNC:6014 ! IL4)\n",
      "Future 272 studies are warranted to determine if SARS-CoV-2 CD4+ T cells from severely ill patients, 273 particularly those in the ICU or those who succumb to disease, produce IL4, IL17, or IL6.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 CD4+\") increases p(HGNC:6018 ! IL6)\n",
      "Future 272 studies are warranted to determine if SARS-CoV-2 CD4+ T cells from severely ill patients, 273 particularly those in the ICU or those who succumb to disease, produce IL4, IL17, or IL6.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 GZ69\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "a CPE induced by SARS-CoV-2 AP66 and SARS-CoV-2 GZ69 in Vero E6 at P0 (72 h p.i.) (Original magnification 10x).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 GZ69\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "This result shows that the absence of CPE in Vero E6 cells does not reflect the SARS-CoV-2 GZ69 replicative capacity.A Whole genome sequencing (WGS) approach was applied to evaluate if differences in CPE induced by SARS-CoV-2 GZ69 and SARS-CoV-2 AP66 were mirrored by genetic variations between the two isolates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 GZ69\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "A Whole genome sequencing (WGS) approach was applied to evaluate if differences in CPE induced by SARS-CoV-2 GZ69 and SARS-CoV-2 AP66 were mirrored by genetic variations between the two isolates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 IgG antibodies\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "In terms of the clinical evidence on hyperimmune plasma rich IgG collected from recovered coronavirus patients, two critically ill patients with COVID-19 were treated with convalescent plasma containing SARS-CoV-2 IgG antibodies from recently recovered donors (12).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 MIRA55\") increases a(TEXT:\"CD8+ TCRs\")\n",
      "(A) Beta-binomial regression coefficient estimates (x-axis) and negative log10 false discovery rates (y-axis) for features developed from CD8+ TCRs activated by SARS-CoV-2 MIRA55 : ORF1ab 5171:5203.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 MIRA55\") increases a(TEXT:\"CD8+\")\n",
      "(A) Beta-binomial regression 850 coefficient estimates (x-axis) and negative log10 false discovery rates (y-axis) for features developed from 851 CD8+ TCRs activated by SARS-CoV-2 MIRA55 : ORF1ab 5171:5203.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 N geneGAACTTCTCCTGCTAGAATGGCDAMP-crRNA\") increases a(TEXT:\"SARS-CoV-2 UAAUUUCUACUAAGUGUAGAUCCCUACUGCUGCCUGGAGUUGN geneAASynthetic SARS-CoV-2 N DNA fragmentGCCATTCGAGCAGGAGAATTTCCCCTACTGCTGCCAGGAGTTGA ATTTCTTGAATTACCGCGACTACGTGnCOV_N1 RT-qPCR forward primer 1 GACCCCAAAATCAGCGAAATnCOV_N1 RT-qPCR reverse primer 1 TCTGGTTACTGCCAGTTGAATCTGnCOV_N1 RT-qPCR probe 1FAM-ACCCCGCATTACGTTTGGTGGACC-BHQ1AIOD-CRISPR\")\n",
      "The list of all used sequences in this studyItem Sequence (5 '-3') DAMP forward outer primer (FO) targeting SARS-CoV-2 N geneGGCTTCTACGCAGAAGGGADAMP reverse outer primer (RO) targeting SARS-CoV-2 N geneAATCTGTCAAGCAGCAGCADAMP forward inner primer (FI) targeting SARS-CoV-2 N geneTTGAACTGTTGCGACTACGTGTTTTAGCCTCTTCTCGTTCCTCATPhosphorothioated FI (PS-FI) T * T * G * A * A * C * T * G * T * T * G * C * G * A * C * T * A * C * G * T * G * TTTTAGCCTCTT CTCGTTCCTCATDAMP reverse inner primer (RI) targeting SARS-CoV-2 N geneGAACTTCTCCTGCTAGAATGGCTTTTGCAAGAGCAGCATCACCGPhosphorothioated RI (PS-RI) G * A * A * C * T * T * C * T * C * C * T * G * C * T * A * G * A * A * T * G * G * C * TTTTGCAAGA GCAGCATCACCGDAMP forward competition primer (FC) targeting SARS-CoV-2 N geneTTGAACTGTTGCGACTACGTGDAMP reverse competition primer (RC) targeting SARS-CoV-2 N geneGAACTTCTCCTGCTAGAATGGCDAMP-crRNA targeting SARS-CoV-2 UAAUUUCUACUAAGUGUAGAUCCCUACUGCUGCCUGGAGUUGN geneAASynthetic SARS-CoV-2 N DNA fragmentGCCATTCGAGCAGGAGAATTTCCCCTACTGCTGCCAGGAGTTGA ATTTCTTGAATTACCGCGACTACGTGnCOV_N1 RT-qPCR forward primer 1 GACCCCAAAATCAGCGAAATnCOV_N1 RT-qPCR reverse primer 1 TCTGGTTACTGCCAGTTGAATCTGnCOV_N1 RT-qPCR probe 1FAM-ACCCCGCATTACGTTTGGTGGACC-BHQ1AIOD-CRISPR forward primer (FP) targeting SARS-CoV-2 N gene 2AGGCAGCAGTAGGGGAACTTCTCCTGCTAGAATAIOD-CRISPR reverse primer (FP) targeting SARS-CoV-2 N gene 2TTGGCCTTTACCAGACATTTTGCTCTCAAGCTGAIOD-CRISPR forward crRNA (crRNA1) targeting SARS-CoV-2 N gene 2UAAUUUCUACUAAGUGUAGAUCAUCACCGCCAUUGCCAGCCAIOD-CRISPR reverse crRNA (crRNA2) targeting SARS-CoV-2 N gene 2UAAUUUCUACUAAGUGUAGAUUUGCUGCUGCUUGACAGAUU * The nucleotides with phosphorothioate modifications.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3\") increases bp(GO:\"GO:0005783\" ! \"endoplasmic reticulum\")\n",
      "Here, we reveal that SARS-CoV-2 ORF3 induces ERphagy and consequently induces ER stress to trigger acute inflammatory responses and enhance sensitivity to ER stress induced apoptosis.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3\") increases p(HGNC:25964 ! RETREG1)\n",
      "By systemic screening, we revealed that SARS-CoV-2 ORF3 localizes to the ER compartment and induces FAM134B mediated ERphagy through the HMGB1-Beclin1 pathway, subsequently contributing to ER stress and activating proinflammatory responses through ERphagy and ER functional reprogramming.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3a\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "Recent study showed that SARS-CoV-2 ORF3a also can efficiently induce apoptosis in cells [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3a\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "This implies that SARS-CoV-2 ORF3a induces apoptosis via the extrinsic apoptotic pathway [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3a\") increases p(HGNC:1509 ! CASP8)\n",
      "SARS-CoV-2 ORF3a can induce the activation and cleavage of caspase-8 without effect on Bcl-2 expression levels.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Orf9b\") increases p(HGNC:11985 ! TOMM70)\n",
      "SARS-CoV-2 Orf9b suppresses type I interferon responses by targeting TOM70.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Orf9b\") increases p(HGNC:11985 ! TOMM70)\n",
      "PLoS One 6 (5), e19436212 Jiang, H.W. et al. (2020) SARS-CoV-2 Orf9b suppresses type I interferon responses by targeting TOM70.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Orf9b\") decreases p(FPLX:Interferon ! Interferon)\n",
      "SARS-CoV-2 Orf9b suppresses type I interferon responses by targeting TOM70.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Orf9b\") decreases p(FPLX:Interferon ! Interferon)\n",
      "PLoS One 6 (5), e19436212 Jiang, H.W. et al. (2020) SARS-CoV-2 Orf9b suppresses type I interferon responses by targeting TOM70.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Proteins\") increases p(FPLX:Interferon ! Interferon)\n",
      "SARS-CoV-2 Proteins Induce IFN Signaling.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RBM\") increases p(HGNC:11892 ! TNF)\n",
      "However, SARS-CoV-2 RBM still markedly elevated the secretion of TNF and GM-CSF in murine macrophage cultures.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RBM\") decreases p(HGNC:2434 ! CSF2)\n",
      "Proposed model for the mAb mediated inhibition of SARS-CoV-2 RBM induced GM-CSF secretion.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RNA elements\") increases p(FPLX:Interferon ! Interferon)\n",
      "It has been hypothesized that strong recognition of SARS-CoV-2 RNA elements by various innate receptors and subsequent target gene transcriptional activation causes IFN, cytokine, and chemokine overproduction.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RT-PCR\") increases bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\")\n",
      "SARS-CoV-2 RT-PCR was performed targeting the N, RdRp, S, and E genes and 14 ORF1ab, using a combination of Emergency Use Authorization laboratory developed tests and 15 commercial assays.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RT-PCR\") increases bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\")\n",
      "SARS-CoV-2 RT-PCR was performed targeting the N, RdRp, S, and E genes and ORF1ab, using a combination of Emergency Use Authorization laboratory developed tests and commercial assays.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RT-ddPCR\") increases bp(GO:\"GO:0003968\" ! \"RNA-directed 5'-3' RNA polymerase activity\")\n",
      "For 10-100 copies and reaction (n = 2), 2.5-5 copies and reaction (n = 4) Two-dimensional scatterplots of the SARS-CoV-2 RT-ddPCR multiplex assay targeting N, E, and RdRp (FAM).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RUO kit\") increases a(TEXT:\"Quant Studio 3 Real-Time PCR System\")\n",
      "Quantitative RT-PCR was carried out using the SARS-CoV-2 RUO kit (Thermofisher Scientific) which targets the N1 and N2 sequences of the virus as well as RP (RNase P) for RNA control (Integrated DNA Technology, Coralville, Iowa), Taqman 4X master mix and Quant Studio 3 Real-Time PCR System (Thermofisher Scientific).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RUO kit\") increases a(TEXT:\"Taqman 4X master mix\")\n",
      "Quantitative RT-PCR was carried out using the SARS-CoV-2 RUO kit (Thermofisher Scientific) which targets the N1 and N2 sequences of the virus as well as RP (RNase P) for RNA control (Integrated DNA Technology, Coralville, Iowa), Taqman 4X master mix and Quant Studio 3 Real-Time PCR System (Thermofisher Scientific).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 RUO kit\") increases p(UP:Q5U5M8 ! Q5U5M8)\n",
      "Quantitative RT-PCR was carried out using the SARS-CoV-2 RUO kit (Thermofisher Scientific) which targets the N1 and N2 sequences of the virus as well as RP (RNase P) for RNA control (Integrated DNA Technology, Coralville, Iowa), Taqman 4X master mix and Quant Studio 3 Real-Time PCR System (Thermofisher Scientific).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 S-glycoprotein\") increases p(FPLX:NFkappaB ! NFkappaB)\n",
      "SARS-CoV-2 S-glycoprotein can be sensed by TLR-4, activating the nuclear factor kappa-B (NF-kB) pathway [35].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 S1/S2 site\") increases p(HGNC:8568 ! FURIN)\n",
      "These predictions have been confirmed experimentally as recombinant furin was shown to process the purified SARS-CoV-2 S protein, generating appropriate cleavage products25, whereas no cleavage products were detected when processing the SARS-CoV S.Notably, the PiTou scores of traditionally accepted furin cleavage sites, such as those found in influenza H5N1 (score : 13.6) and HCoV-HKU1 (score : 14.6), are much higher than for SARS-CoV-2, suggesting that while SARS-CoV-2 S1/S2 site has increased furin recognition when compared to other lineage B-betaCoVs, the site is not optimal for furin cleavage.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 VSV-GFP pseudovirus\") increases a(TEXT:\"ofgreen fluorescent protein\")\n",
      "The SARS-CoV-2 VSV-GFP pseudovirus enables transient expression ofgreen fluorescent protein (GFP) upon entry into the host cell, facilitating directmonitoring of viral uptake following incubation with the nanoparticles and photothermal treatment.In the absence of nanoparticles, HEK293T cells engineered to overexpress ACE2 (ACE2 and HEK293T) were susceptible to pseudovirus infection as demonstrated by GFP expression bioRxiv preprint doi : https://doi.org/10.1101/2020.11.30.404624; this version posted November 30, 2020.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 VSV-GFP pseudovirus\") increases p(PFAM:PF01353 ! \"green fluorescent protein\")\n",
      "The SARS-CoV-2 VSV-GFP pseudovirus enables transient expression of green fluorescent protein (GFP) upon entry into the host cell, facilitating direct monitoring of viral uptake following incubation with the nanoparticles and photothermal treatment.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Vaccine Epitopes\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Identification of SARS-CoV-2 Vaccine Epitopes Predicted to Induce Long-Term Population-Scale Immunity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Vaccine Epitopes\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Identification of SARS-CoV-2 Vaccine Epitopes Predicted to Induce Long-term Population-Scale Immunity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Vaccine Epitopes\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "bioRxiv, 2020.2003.2026.994756, doi :10.1101/2020.03.26.994756 (2020).41 Yarmarkovich, M., Warrington, J. M., Farrel, A. & Maris, J. M. Identification of SARS-CoV-2 Vaccine Epitopes Predicted to Induce Long-Term Population-Scale Immunity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Virus\") increases a(TEXT:\"Clin Case Rep.The emergence\")\n",
      "Clin Case Rep.The emergence of COVID-19, caused by the SARS-CoV-2 Virus, has created a global pandemic and strained our entire healthcare system around the world.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 [\") increases bp(MESH:D002490 ! \"Central Nervous System\")\n",
      "Recent evidence shows that the third phase (also known as SIRS) accounts for the majority of CNS disturbances mediated by SARS-CoV-2 [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 [\") decreases a(TEXT:IFN-I)\n",
      "Compared with SARS-CoV and MERS-CoV, IFN-I signaling is more efficiently inhibited by nsp1 and nsp6 of SARS-CoV-2 [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 a (+) single-stranded RNA virus fasten similar receptor\") increases p(HGNC:3009 ! DPP4)\n",
      "SARS-CoV-2 a (+) single stranded RNA virus fasten similar receptor as SARS-CoV, i.e., angiotensin converting enzyme 2 (ACE-2) to infect the host [XREF_BIBR], which was not the case of MERS-CoV S protein which targets DPP4 [XREF_BIBR, XREF_BIBR], not ACE-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 antibodies\") increases a(TEXT:\"CR3022 epitope\")\n",
      "A few SARS-CoV-2 antibodies from COVID-19 60 patients have also recently been shown to target the CR3022 epitope [12, 17, 21], 61 suggesting that it is an important site of vulnerability for the antibody response in SARS-CoV-2 infection.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 antibodies\") increases a(TEXT:\"CR3022 epitope\")\n",
      "A few SARS-CoV-2 antibodies from COVID-19 61 patients have also recently been shown to target the CR3022 epitope [12, 17, 21], 62 suggesting that it is an important site of vulnerability for the antibody response in SARS63 CoV-2 infection.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 antibodies\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "These results contradict previous reports that suggested a short half-life of SARS-CoV-2 antibodies [27] and support long-term immunity after infection as was shown in other studies [28].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 antibody\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Some governments are suggesting an ' immunity passport ' or ' risk-free certificate ' based on SARS-CoV-2 antibody detection that could allow people to travel or go to work under the assumption that they can not be re-infected; however, there is scant evidence on antibody mediated immunity to SARS-CoV-2 to validate such an ' immunity passport '.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 antibody\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "The concept of an ' immunity passport ' has gained popularity, whereby evidence of SARS-CoV-2 antibody production would signal immunity to reinfection.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 coronavirus\") increases a(TEXT:INTRODUCTION)\n",
      "INTRODUCTION The COVID-19 pandemic caused by the SARS-CoV-2 coronavirus is characterized by a striking age dependent morbidity and mortality, irrespective of ethnicity [1, 2].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 coronavirus\") increases a(TEXT:\"aCC-BY-NC-ND 4.0 International license.1 Introduction2 COVID-19 disease\")\n",
      "It is madeavailable under aCC-BY-NC-ND 4.0 International license.1 Introduction2 COVID-19 disease, caused by the SARS-CoV-2 coronavirus, is currently a 3 worldwide pandemic.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 coronavirus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The disease named COVID-19, caused by the SARS-CoV-2 coronavirus, is currently generating a global pandemic.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 coronavirus\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The COVID-19 pandemic, caused by the SARS-CoV-2 coronavirus, is a global health problem that has radically transformed public and private healthcare organizations around the world.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 encoded 443 miRNAs\") increases p(HGNC:6081 ! INS)\n",
      "We showed that SARS-CoV-2 encoded 443 miRNAs can target insulin signaling pathway and 444 aberration of this pathway might overcomplicate the whole disease condition for COVID-19 445 patients with existing diabetic problems (Shimizu et al., 1980; del Campo et al., 2012).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 encoded miRNAs\") increases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Moreover, several immune 427 response pathways like- TLR signaling, interleukin signaling, TRAF6 signaling, etc. were 428 exclusively found to be targeted by SARS-CoV-2 induced host miRNAs and 429 SARS-CoV-2 encoded miRNAs can target pathways like- autophagy, IFN-I signaling, wnt 430 signaling, mTOR signaling, etc., but SARS-CoV encoded miRNAs ' targets were not found to 431 be enriched in these pathways.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 icELISA\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Altogether, the SARS-CoV-2 icELISA test allows rapid (< 48 h in total, read-out in seconds) and automated quantification of virus infection in cell culture to evaluate the efficacy of NAbs and antiviral drugs using reagents and equipment present in most routine diagnostics departments.Knowledge concerning SARS-CoV-2-neutralizing antibodies (NAbs) is indispensable for COVID-19 convalescent plasma selection, evaluation of vaccines, and immunity certificates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 infection.Main\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These data support a beneficial role for, and elucidate the immunological mechanisms underlying, the use of baricitinib as a frontline treatment for severe inflammation induced by SARS-CoV-2 infection.Main The rapid emergence and dissemination of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and subsequent COVID-19 pandemic has placed an excessive burden on public and private healthcare systems with over 800,000 deaths worldwide.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 infection.SARS-CoV-2\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Figure 1 illustrates the process of SARS-CoV-2 infection.SARS-CoV-2 elicits an exuberant immune response characterized by a dysregulated production of soluble immune mediators.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 merits\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Understanding and managing the aggravated and ineffective immune response elicited by SARS-CoV-2 merits further clarification to improve treatments propose.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 miRNAs\") increases a(TEXT:\"Target genes\")\n",
      "Target genes downregulated by SARS-CoV-2 miRNAs are found to be involved in the Ca 2+ signaling pathway, and these are considered important activators of many signaling pathways (Zhou et al., 2009).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 miRNAs\") increases a(TEXT:\"Target genes\")\n",
      "Target genes downregulated by SARS-CoV-2 miRNAs are found to be involved in the Ca 2+ signaling pathway, and these are considered important activators of many signaling pathways (XREF_FIG).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 miRNAs\") increases p(HGNC:11009 ! SLC2A4)\n",
      "Our results suggest that the SARS-CoV-2 miRNAs target the gene encoding SLC2A4, a 252 glucose transporter responsible for the uptake of extracellular glucose into the cytoplasm.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 neutralizing NbsTo\") increases p(HGNC:7652 ! NBN)\n",
      "Finally, structural characterizations facilitated the bioengineering of multivalent Nbs into multiepitope cocktails that achieved remarkable neutralization potencies of as low as 0.058 ng/ml (1.3 pM) and may block the mutational escape of the virus.ResultsDevelopment of highly potent SARS-CoV-2 neutralizing NbsTo produce high-quality SARS-CoV-2 neutralizing Nbs, we immunized a llama \" Wally \" with the recombinant RBD protein from human 293T cells.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 neutralizing antibody\") increases a(CHEBI:17089 ! glycoprotein)\n",
      "CAR-NK cells were generated using the scFv domain of S309 (henceforward, S309-CAR-NK), a SARS-CoV and SARS-CoV-2 neutralizing antibody that targets the highly conserved region of SARS-CoV-2 spike (S) glycoprotein, therefore would be more likely to recognize different variants of SARS-CoV-2 isolates.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pandemic\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "The SARS-CoV-2 pandemic, which causes COVID-19, has affected every aspect of healthcare.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pathogen\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "We stress that the findings seen on LUS are a consequence of lung injury and inflammation caused by the SARS-CoV-2 pathogen and could be seen in other viral infections and disease processes that cause a similar lung injury pattern.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 peptide ORF1ab 4211:4252\") increases a(TEXT:\"CD8+ TCRs\")\n",
      "TCR features shown here were identified from publicity analysis of CD8+ TCRs activated by an illustrative SARS-CoV-2 peptide ORF1ab 4211:4252 (MIRA55), which is predicted to bind HLA-A * 01.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 peptide ORF1ab 4211:4252\") increases p(FPLX:TCR ! TCR)\n",
      "It is madeavailable under aCC-BY-NC-ND 4.0 International license.Figure S1Figure S1 : Publicity analysis in MIRA participants of CD8+ TCR beta-chain features activated by SARS-CoV-2 peptide ORF1ab 4211:4252 (MIRA55) predicted to bind HLA-A * 01.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 peptide ORF1ab 882 4211:4252\") increases a(TEXT:\"CD8+ TCRs\")\n",
      "TCR features shown here were 881 identified from publicity analysis of CD8+ TCRs activated by an illustrative SARS-CoV-2 peptide ORF1ab 882 4211:4252 (MIRA55), which is predicted to bind HLA-A * 01.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 peptides\") increases a(TEXT:\"CD8+ TCRs\")\n",
      "(A) Framework : sets of CD8+ TCRs activated by SARS-CoV-2 peptides were previously discovered in 61 individuals diagnosed with COVID-19 using a Multiplex Identification of Antigen Specific T Cell Receptors Assay (MIRA) by Nolan et al. (2020).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pneumonia\") increases a(CHEBI:95341 ! \"2-[1-ethylsulfonyl-3-[4-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)-1-pyrazolyl]-3-azetidinyl]acetonitrile\")\n",
      "Baricitinib improves respiratory function in patients treated with corticosteroids for SARS-CoV-2 pneumonia : an observational cohort study.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pneumonia\") decreases bp(MESH:C000657245 ! COVID-19)\n",
      "L/R therapy is one of the exclusion criteria for patients of SARS-CoV-2 pneumonia being enrolled for prazosin therapy in the currently ongoing \" Prazosin to Prevent COVID-19 (PREVENT-COVID) trial \" [96].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 protein NSP12\") increases p(HGNC:6715 ! LTBP2)\n",
      "LTBP3 can promote lung alveolarization 81, but can be modulated by SARS-CoV-2 protein NSP12.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases a(CHEBI:81564 ! \"resolvin D1\")\n",
      "Here were report that SARS-CoV-2 proteins trigger the biosynthesis of RvD1 in peripheral blood and isolated MPhi from volunteers with CF, which is paradigmatic of activation SPM biosynthetic pathways.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases p(FPLX:p38 ! p38)\n",
      "SARS-CoV-2 proteins may directly stimulate p38 activity to promote replication.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases bp(GO:\"GO:0001666\" ! \"response to hypoxia\")\n",
      "Furthermore, hypoxia response through HIF-1 signaling might also be targeted by SARS-CoV-2 proteins.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases bp(GO:\"GO:0009058\" ! \"biosynthetic process\")\n",
      "Here were report that SARS-CoV-2 proteins trigger the biosynthesis of RvD1 in peripheral blood and isolated MPhi from volunteers with CF, which is paradigmatic of activation SPM biosynthetic pathways.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases bp(GO:\"GO:0051179\" ! localization)\n",
      "In addition, other SARS-CoV-2 proteins like NSP13, NSP14, NSP15 and ORF6 have been suggested to antagonize IFN function by suppressing nuclear localization of IRF3.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases p(HGNC:6118 ! IRF3)\n",
      "In addition, other SARS-CoV-2 proteins like NSP13, NSP14, NSP15 and ORF6 have been suggested to antagonize IFN function by suppressing nuclear localization of IRF3.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases p(HGNC:9605 ! PTGS2)\n",
      "Alternatively, SARS-CoV-2 proteins may mediate the induction of COX-2 through their complex effects on host cells.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases bp(MESH:D007370 ! \"Interferon Type I\")\n",
      "This could be either related to the timing of sampling, undersampling cell types producing type I interferons, or inhibition of type I interferon production by SARS-CoV-2 proteins [25,39].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") increases p(HGNC:6018 ! IL6)\n",
      "In the present work, we show that SARS-CoV-2 proteins induce the expression and secretion of IL-6 by human monocytes and macrophages, the first line cells of antiviral immune responses.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases a(TEXT:IFN-I)\n",
      "SARS-CoV-2 proteins that inhibit IFN-I production are indicated in black boxes, and the associated blocked pathways are indicated as dashed lines.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases a(TEXT:IFN-I)\n",
      "The SARS-CoV-2 proteins that inhibit IFN-I signaling are indicated in black boxes, and the associated blocked pathways are indicated by dashed lines.Fig.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases a(TEXT:IFN-I)\n",
      "The SARS-CoV-2 proteins that inhibit IFN-I signaling are indicated in black boxes, and the associated blocked pathways are indicated by dashed lines.STAT1-and STAT3 dependent drug targets in IFN-I signaling.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases a(TEXT:IFN-I)\n",
      "Also, Xia et al. have shown that SARS-CoV-2 proteins suppress the IFN-I response in vitro.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases p(FPLX:Interferon ! Interferon)\n",
      "In addition, other SARS-CoV-2 proteins like NSP13, NSP14, NSP15 and ORF6 have been suggested to antagonize IFN function by suppressing nuclear localization of IRF3.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases p(FPLX:Interferon ! Interferon)\n",
      "Here, we use unbiased screening to identify SARS-CoV-2 proteins that antagonize type I interferon (IFN-I) response.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases bp(GO:\"GO:0001816\" ! \"cytokine production\")\n",
      "This could be either related to the timing of sampling, undersampling cell types producing type I interferons, or inhibition of type I interferon production by SARS-CoV-2 proteins [25,39].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins\") decreases p(HGNC:11362 ! STAT1)\n",
      "After the infection, STAT1 activity is inhibited by the SARS-CoV-2 proteins, NSP1, and ORF6 (2).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pseudovirus\") increases a(TEXT:\"CD4+\")\n",
      "1 x 10 6 CD4+ or CD8+ T cells were activated by purified NA/LE mouse anti-human CD3 antibody (555329, BD, USA, 1 mug/mL) and purified NA/LE mouse anti-human CD28 antibody (555725, BD, USA, 2 mug / mL) for 48 h. Then, the different doses of SARS-CoV-2 pseudovirus (0, 500, 1000, 2000, 4000, and 8000 TCID 50) were added into unactivated or activated CD4+ and CD8+ T cells and incubated for 72 h.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pseudovirus\") increases a(TEXT:\"CD4+\")\n",
      "1 x 106 CD4+ or CD8+ T cells were activated by purified NA/LE mouse anti-human CD3 antibody (555329, BD, USA, 1 mug/mL) and purified NA/LE mouse anti-human CD28 antibody (555725, BD, USA, 2 mug/mL) for 48 h. Then, the different doses of SARS-CoV-2 pseudovirus (0, 500, 1000, 2000, 4000, and 8000 TCID50) were added into unactivated or activated CD4+ and CD8+ T cells and incubated for 72 h.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pseudovirus\") increases a(TEXT:\"CD8+\")\n",
      "1 x 10 6 CD4+ or CD8+ T cells were activated by purified NA/LE mouse anti-human CD3 antibody (555329, BD, USA, 1 mug/mL) and purified NA/LE mouse anti-human CD28 antibody (555725, BD, USA, 2 mug / mL) for 48 h. Then, the different doses of SARS-CoV-2 pseudovirus (0, 500, 1000, 2000, 4000, and 8000 TCID 50) were added into unactivated or activated CD4+ and CD8+ T cells and incubated for 72 h.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pseudovirus\") increases a(TEXT:\"CD8+\")\n",
      "1 x 106 CD4+ or CD8+ T cells were activated by purified NA/LE mouse anti-human CD3 antibody (555329, BD, USA, 1 mug/mL) and purified NA/LE mouse anti-human CD28 antibody (555725, BD, USA, 2 mug/mL) for 48 h. Then, the different doses of SARS-CoV-2 pseudovirus (0, 500, 1000, 2000, 4000, and 8000 TCID50) were added into unactivated or activated CD4+ and CD8+ T cells and incubated for 72 h.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 reactive IgG antibodies\") increases a(TEXT:\"CD4+\")\n",
      "Recovered COVID-19 patients have 123 an increased number of antibody secreting cells, activated CD4+ and CD8+ T cells, and 124 immunoglobulin M (IgM) and SARS-CoV-2 reactive IgG antibodies were detected in blood 125 before full symptomatic recovery26-28.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 reactive IgG antibodies\") increases a(TEXT:\"CD8+\")\n",
      "Recovered COVID-19 patients have 123 an increased number of antibody secreting cells, activated CD4+ and CD8+ T cells, and 124 immunoglobulin M (IgM) and SARS-CoV-2 reactive IgG antibodies were detected in blood 125 before full symptomatic recovery26-28.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 receptors\") increases bp(MESH:D004249 ! \"DNA Damage\")\n",
      "Increased expression of SARS-CoV-2 receptors was blocked by mitoTEMPO, a mitochondrial superoxide scavenger that reduced oxidative stress and DNA damage seen in oxygen exposed mice.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 receptors\") decreases a(TEXT:\"mTOR-S6 kinase\")\n",
      "Subjects with high expression of ACE2 or TMPRSS2 displayed low levels of EIF2AK2, EIF2AK4, mTOR, or RPS6KB1 (XREF_FIG), thereby supporting negative regulation of eIF2alpha kinase and mTOR-S6 kinase signaling by SARS-CoV-2 receptors in the gastrointestinal tract.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 receptors\") decreases p(HGNC:9437 ! EIF2AK2)\n",
      "Subjects with high expression of ACE2 or TMPRSS2 displayed low levels of EIF2AK2, EIF2AK4, mTOR, or RPS6KB1 (XREF_FIG), thereby supporting negative regulation of eIF2alpha kinase and mTOR-S6 kinase signaling by SARS-CoV-2 receptors in the gastrointestinal tract.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 recombinant antigensThe in-planta\") increases a(TEXT:\"N protein antigens\")\n",
      "Cloning and expression of SARS-CoV-2 recombinant antigensThe in-planta production of SARS-CoV-2 RBD and N protein antigens was also assayed in parallel using a similar strategy as described for antibody production.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 specific antibodies\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "In our current understanding, detection of SARS-CoV-2 specific antibodies in serum indicates an exposure to SARS-CoV-2 but does not yet enable certain determination of the level of infectiousness or immunity [1].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 specific protein\") increases bp(GO:\"GO:0006915\" ! \"apoptotic process\")\n",
      "The study by Tan et al. 5 showed that SARS-CoV-2 specific protein 7a could induce cell apoptosis in different organs (including lung, kidney, and liver) through the caspase dependent pathway, further confirming the possibility of SARS-CoV-2 directly attacking liver tissues and causing liver damage.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 specific siRNAs\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These include drugs that can reduce inflammation (such as itolizumab that binds to CD6 receptor and blocks the activation of T lymphocytes and suppress the pro inflammatory cytokines or corticosteroids that decrease the cytokines storm), ACE-2 inhibitor, SARS-CoV-2 specific siRNAs, and immunomodulators.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 surface proteins Spike (S)\") increases p(FPLX:Interferon ! Interferon)\n",
      "Here we demonstrate 28 both in vitro and in vivo that the SARS-CoV-2 surface proteins Spike (S) and Envelope (E) 29 activate the key immune signaling interferon (IFN) pathway in both immune and epithelial cells 30 independent of viral infection and replication.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 transmission\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Increased SARS-CoV-2 transmission might threaten strained health care resources, require extended and more rigorous implementation of public health strategies (4), and increase the percentage of population immunity required for pandemic control.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 transmission\") decreases bp(MESH:C000657245 ! COVID-19)\n",
      "Effective control of SARS-CoV-2 transmission between healthcare workers during a period of diminished community prevalence of COVID-19.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 transmission.Effects\") increases a(CHEBI:15379 ! dioxygen)\n",
      "Currently, the available evidence suggests that the female genital tract is unlikely a route of SARS-CoV-2 transmission.Effects of COVID-19 during pregnancy : fetal, maternal and neonatal risks COVID-19 infection has raised concerns for pregnant woman and their fetus due to physiological changes in immunity during pregnancy, maternal susceptibility to respiratory infections, increased oxygen requirements, and risks associated with treatment during pregnancy.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 viral proteins\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Since our viral-host interactome analysis indicates that SARS-CoV-2 viral proteins may modulate and evade host innate immune response by interacting with key kinases and adaptors of the immune signaling pathway, it is not surprising that COVID-19 patients fail to mount robust antiviral IFN responses.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 viral proteins\") increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Since our viral-host interactome analysis indicates SARS-CoV-2 viral proteins may modulate and evade host innate immune response by interacting with key kinases and adaptors of immune signaling pathway, it is not surprising that COVID-19 patients fail to mount robust anti-viral IFN responses.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 viral proteins\") decreases p(FPLX:Interferon ! Interferon)\n",
      "This may explain why SARS-CoV-2 viral proteins inhibits the interferon response 33.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 virulence factors\") increases p(HGNC:6025 ! CXCL8)\n",
      "Therefore, we hypothesized that SARS-CoV-2 virulence factors may potentiate or regulate the induction of IL-8 during infection, leading to the uncontrolled infiltration and activation of neutrophils.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 virus 19\") increases a(TEXT:\"hCoV-19/Canada/ON-VIDO-01/2020\")\n",
      "17 Importantly, our results also showed that CHPV or suramin can efficiently reduce 18 levels of cytopathic effect caused by SARS-CoV-2 virus 19 (hCoV-19/Canada/ON-VIDO-01/2020) infection in Vero cells.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 virus spike protein\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "A recent study reported that the D614G mutation in the SARS-CoV-2 virus spike protein reduces S1 shedding and increases infectivity of SARS COV-2 virus.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 virus spike protein\") decreases a(TEXT:S1)\n",
      "A recent study reported that the D614G mutation in the SARS-CoV-2 virus spike protein reduces S1 shedding and increases infectivity of SARS COV-2 virus.\n",
      "\n",
      "a(TEXT:SARS-CoV-2.COVID-19) increases bp(MESH:C000657245 ! COVID-19)\n",
      "Furthermore, a unique furin-like cleavage site exists in the S protein of SARS-Cov-S2 [16] ; thus, the theoretical advantage inferred from this cleavage site in disease infection models can be deduced to prevent and combat COVID-19 caused by SARS-CoV-2.COVID-19 is mainly presented with respiratory symptoms.\n",
      "\n",
      "a(TEXT:SARS-CoV-22) increases a(TEXT:\"TEXT COVID-19\")\n",
      "MAIN TEXT COVID-19 is a new form of viral pneumonia1 caused by SARS-CoV-22 and is affecting 67,073,749 patients, with 1,536,072 deaths, world-wide (source : Johns Hopkins University, as of December 07 2020).\n",
      "\n",
      "a(TEXT:SARS-CoV-22) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "No reuse allowed without permission.1 Topoisomerase 1 inhibition therapy protects against SARS-CoV-22 induced inflammation and death in animal models.3 4 Jessica Sook Yuin Ho1, *, Bobo Wing-Yee Mok2, *, Laura Campisi1, Tristan Jordan1, 5 Soner Yildiz1, Sreeja Parameswaran4, Joseph A Wayman9,27, Natasha N Gaudreault10, David 6 A Meekins10, Sabarish V. Indran10, Igor Morozov10, Jessie D Trujillo10, Yesai S Fstkchyan1, 7 Raveen Rathnasinghe1,, Zeyu Zhu1, Simin Zheng1, Nan Zhao1, Kris White1, Helen Ray-Jones3, 8 Valeriya Malysheva3, Michiel J Thiecke26, Siu Ying Lau2, Honglian Liu2, Anna Junxia Zhang2, 9 Andrew Chak-Yiu Lee2, Wen-Chun Liu1, Teresa Aydillo1, Betsaida Salom Melo12,13, Ernesto 10 Guccione11, Robert Sebra12,13,29,30, Elaine Shum5, Jan Bakker14,15,16, David A. Kaufman17, Andre 11 L. Moreira18, Mariano Carossino19, Udeni B R Balasuriya19, Minji Byun20, Emily R Miraldi9,27, 12 Randy A Albrecht1,21, Michael Schotsaert1,21, Adolfo Garcia-Sastre1, 21,22, Sumit K Chanda23, 13 Anand D Jeyasekharan6, Benjamin R TenOever1,21,24, Mikhail Spivakov3, Matthew T 14 Weirauch4,25,28,, Sven Heinz7, Honglin Chen2, Christopher Benner7, Juergen A Richt8,10, Ivan 15 Marazzi1,21.\n",
      "\n",
      "a(TEXT:SARS-CoV-2infection33) increases bp(GO:\"GO:0045087\" ! \"innate immune response\")\n",
      "Given that a strong innate immune response was activated by SARS-CoV-2infection33 and that the resultinginflammation that may affect taste budhomeostasis,2,34 the response of the innateimmune system of the host may be involved in taste loss in COVID-19patients.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-33 2\") increases bp(MESH:D004194 ! Disease)\n",
      "MADVent: A low-cost ventilator for patients with COVID-19.A key challenge in the battle against the disease caused by the novel coronavirus SARS-CoV-33 2, COVID-19, is a potential worldwide shortage of mechanical ventilators.\n",
      "\n",
      "a(TEXT:SARS-CoV-4) increases a(TEXT:\"PLoS Pathog.The ongoing COVID-19 pandemic\")\n",
      "PLoS Pathog.The ongoing COVID-19 pandemic, which is caused by the new coronavirus SARS-CoV-4 and can neutralize SARS-CoV, but not SARS-CoV-2, in a live virus neutralization assay To investigate how P384 and A384 lead to differential binding of CR3022, we compared the RBD structures from SARS-CoV and SAR-CoV-2 when bound with CR3022.\n",
      "\n",
      "a(TEXT:SARS-CoV-ACE2) increases p(HGNC:195 ! ADAM17)\n",
      "On the other hand, the ADAM17 activity is up-regulated by the internalization of SARS-CoV-ACE2 in the ACE2 cytoplasmic dependent manner (74).\n",
      "\n",
      "a(TEXT:SARS-CoV-ST) increases bp(MESH:C000705967 ! \"pediatric multisystem inflammatory disease, COVID-19 related\")\n",
      "Similarities noted between SARS-CoV-2 Spike protein sequences and those of other super antigens has prompted the suggestion that this might be the mechanism by SARS-CoV-ST triggers PIMS-TS.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-g/mL 1 g/mL 2.5 g/mL 5 g/mL 10 g/mL\") increases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "This work demonstrates a proof-of-concept that photothermal inactivation could potentially serve as a preventative approach to inhibit ADE that arises due to incomplete neutralization of the SARS-CoV-2 during SARS-CoV- g/mL 1 g/mL 2.5 g/mL 5 g/mL 10 g/mL be readily adapted to target other strains of SARS-CoV-2 via conjugation to different antibodies or other novel therapeutic proteins and can be extended to other viruses, bacteria 57 and malignant cells.\n",
      "\n",
      "a(TEXT:\"SARS-CoV157 2\") increases p(HGNC:5962 ! IL10)\n",
      "SARS-CoV157 2 infection increased IL-10 and reduced IL-4 production in comparison with uninfected 158 monocytes.\n",
      "\n",
      "a(TEXT:\"SARS-CoV157 2\") decreases p(HGNC:6014 ! IL4)\n",
      "SARS-CoV157 2 infection increased IL-10 and reduced IL-4 production in comparison with uninfected 158 monocytes.\n",
      "\n",
      "a(TEXT:\"SARS-CoV2 Vaccine Epitopes\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "Identification of SARS-CoV2 Vaccine Epitopes Predicted to Induce Long-term Population-Scale Immunity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV2 antibodies\") increases p(HGNC:726 ! ART4)\n",
      "Beyond COVID-19 : DO MS/NMO-SD patients treated with anti-CD20 therapies develop SARS-CoV2 antibodies?\n",
      "\n",
      "a(TEXT:\"SARS-CoV2 antibodies\") increases bp(MESH:C000657245 ! COVID-19)\n",
      "Beyond COVID-19 : DO MS/NMO-SD patients treated with anti-CD20 therapies develop SARS-CoV2 antibodies?\n",
      "\n",
      "a(TEXT:\"SARS-CoV368 2\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "These data indicate that SARS-CoV368 2 elicits an inflammatory response in cardiomyocytes that is at least partially dependent on viral 369 nucleic acid sensing and TBK1 signaling.\n",
      "\n",
      "a(TEXT:\"SARS-CoV368 2\") increases p(HGNC:11584 ! TBK1)\n",
      "These data indicate that SARS-CoV368 2 elicits an inflammatory response in cardiomyocytes that is at least partially dependent on viral 369 nucleic acid sensing and TBK1 signaling.\n",
      "\n",
      "a(TEXT:\"SARS-N mAbs\") increases a(TEXT:\"SARS-CoV N protein\")\n",
      "To identify the epitopes in SARS-CoV N protein that are targeted by SARS-N mAbs, competition ELISA was performed as described previously.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") increases p(HGNC:13557 ! ACE2)\n",
      "Since the downregulation of ACE2 caused by the SARS-S protein was postulated as being correlated with the severe respiratory failure, TACE inhibitors might also attenuate the tissue damage caused by the viral infection.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:13557 ! ACE2)\n",
      "TAPI-2 blocked both the SARS-S protein induced shedding of angiotensin converting enzyme 2 (ACE2), a receptor of SARS-CoV, and TNF-alpha production in lung tissues.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:13557 ! ACE2)\n",
      "Since the downregulation of ACE2 by SARS-S protein was proposed as an etiological event in the severe clinical manifestations, our data suggest that TACE antagonists block SARS-CoV infection and also attenuate its severe clinical outcome.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:13557 ! ACE2)\n",
      "TAPI-2 blocked both the SARS-S protein induced shedding of angiotensin converting enzyme 2 (ACE2), a receptor of SARS-CoV, and TNF-alpha production in lung tissues.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:13557 ! ACE2)\n",
      "Since the downregulation of ACE2 by SARS-S protein was proposed as an etiological event in the severe clinical manifestations, our data suggest that TACE antagonists block SARS-CoV infection and also attenuate its severe clinical outcome.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") increases p(HGNC:195 ! ADAM17)\n",
      "To further show that the SARS-S protein induced TACE activation, we measured the TACE activity in cellular extracts.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") increases p(HGNC:195 ! ADAM17)\n",
      "However, the addition of TAPI-2 blocked the increase in TACE activity upregulated by the SARS-S protein (XREF_FIG A and B; p < 0.01).\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") increases p(HGNC:195 ! ADAM17)\n",
      "Time-course analysis of the TACE activity revealed that the addition of the SARS-S protein gradually increased the TACE activity in the cellular extracts (XREF_FIG A).\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:11892 ! TNF)\n",
      "TAPI-2 blocked both the SARS-S protein induced shedding of angiotensin converting enzyme 2 (ACE2), a receptor of SARS-CoV, and TNF-alpha production in lung tissues.\n",
      "\n",
      "a(TEXT:\"SARS-S protein\") decreases p(HGNC:11892 ! TNF)\n",
      "TAPI-2 blocked both the SARS-S protein induced shedding of angiotensin converting enzyme 2 (ACE2), a receptor of SARS-CoV, and TNF-alpha production in lung tissues.\n",
      "\n",
      "a(TEXT:SARS-S) increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "At a later stage, the SARS-S and ACE2 mediated up-regulation of ADAM17 in pulmonary and cardiac tissues may exacerbate the imbalance in RAS, already present in patients with risk diseases in a looping harmful feedback manner, thus becoming the main responsible for the worst clinical outcome.\n",
      "\n",
      "a(TEXT:SARS-S) increases p(UP:P03126 ! P03126)\n",
      "(c) Representative image montage of Vero E6 and Huh7 cells treated with VSV-G, SARS-S, or delEnv PP for 72 h and immunostained using mouse-antiluciferase antibody (magenta) Schematic of repurposing screen and follow up assays.Concentration response of entry inhibitors.\n",
      "\n",
      "a(TEXT:SARS-S) increases p(UP:P03126 ! P03126)\n",
      "(c) Representative image montage of Vero E6 and 16 Huh7 cells treated with VSV-G, SARS-S, or delEnv PP for 72 hours and immunostained using 17 mouse-anti-luciferase antibody (magenta).\n",
      "\n",
      "a(TEXT:SARS-S) increases p(UP:P03126 ! P03126)\n",
      "(c) Representative imagemontage of Vero E6 and Huh7 cells treated with VSV-G, SARS-S, or delEnv PP for 72 h and immunostained usingmouse-antiluciferase antibody (magenta).\n",
      "\n",
      "a(TEXT:SARS-S) increases p(UP:P0DTC2 ! P0DTC2)\n",
      "TMPRSS2 facilitates infection via two independent mechanisms, cleavage of ACE2, which might promote viral uptake, and cleavage of SARS-S, which activates the S protein for membrane fusion (Heurich et al., 2014).\n",
      "\n",
      "a(TEXT:\"SARS104 CoV-2\") increases bp(GO:\"GO:0008219\" ! \"cell death\")\n",
      "We also tested 103 infection in primary human monocyte derived macrophages and found that SARS104 CoV-2 infection triggers macrophage cell death in a dose dependent effect as shown 105 by the presence of LDH in the supernatants.\n",
      "\n",
      "a(TEXT:\"SARS157 CoV-2\") increases a(CHEBI:15551 ! \"prostaglandin E2\")\n",
      "These results indicate that SARS157 CoV-2 induction of the COX-2 and PGE2 pathway in Calu-3 human lung cells does not regulate viral 158 replication.\n",
      "\n",
      "a(TEXT:\"SARS157 CoV-2\") increases p(HGNC:9605 ! PTGS2)\n",
      "These results indicate that SARS157 CoV-2 induction of the COX-2 and PGE2 pathway in Calu-3 human lung cells does not regulate viral 158 replication.\n",
      "\n",
      "a(TEXT:\"SARS164 CoV-2\") increases a(CHEBI:34826 ! M1)\n",
      "The 163 expression of polarization related genes was investigated after 24 and 48 hours of SARS164 CoV-2 stimulation of M1 and M2 polarized macrophages.\n",
      "\n",
      "a(TEXT:\"SARS175 CoV-2 proteins\") increases p(HGNC:9605 ! PTGS2)\n",
      "Alternatively, SARS175 CoV-2 proteins may mediate the induction of COX-2 through their complex effects on host cells.\n",
      "\n",
      "a(TEXT:\"SARS203 CoV-2\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "TMEM106B sgRNAs protected consistently against CPE caused by SARS203 CoV-2 infection in Hep3B, A549, or NCI-H1975 cells, as determined by crystal violet staining.\n",
      "\n",
      "a(TEXT:\"SARS221 CoV-2 S proteins\") increases a(TEXT:eGFP)\n",
      "In agreement with our previous report10, HEK293 219 cells transiently expressing hACE2 proteins showed extensive syncytium formation 220 when coincubated with 293T cells overexpressing eGFP and SARS-CoV or SARS221 CoV-2 S proteins in the presence of trypsin (Fig 4A).\n",
      "\n",
      "a(TEXT:\"SARS257\t CoV-2\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Notably, this is the first 256 report to our knowledge, revealing that CLQ and its analogues effectively inhibit the novel SARS257 CoV-2 infection induced CPE.\n",
      "\n",
      "a(TEXT:\"SARS281 CoV-2\") increases bp(MESH:D029424 ! \"Pulmonary Disease, Chronic Obstructive\")\n",
      "Interestingly, we found that SARS281 CoV-2 infection induced \" squamous metaplasia \" in the COPD epithelium.\n",
      "\n",
      "a(TEXT:\"SARS288 CoV-2\") increases p(HGNC:10050 ! RNASEL)\n",
      "Evaluation of RNase L activation in 287 SARS-CoV-2, WT MERS-CoV, and MERS-CoV-DNS4ab infected Calu-3 cells showed SARS288 CoV-2 activation of RNase L to a similar extent as MERS-CoV-DNS4ab, which lacks both a 289 dsRNA binding protein and a RNase L specific antagonist (Comar et al., 2019; Siu et al., 2014) 290 (Fig 8B).\n",
      "\n",
      "a(TEXT:\"SARS61 CoV-2\") increases bp(MESH:C000656484 ! \"Severe Acute Respiratory Syndrome Coronavirus 2\")\n",
      "No reuse allowed without permission.58 SUMMARY 59 60 The ongoing pandemic caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS61 CoV-2) is currently affecting millions of lives worldwide.\n",
      "\n",
      "a(TEXT:\"SARS9 CoV-2\") increases bp(MESH:D003588 ! \"Cytopathogenic Effect, Viral\")\n",
      "Trimipramine and abemaciclib also protected against SARS9 CoV-2 induced CPE to lesser degrees, with 48.1% and 41.7% CPE rescue, respectively.1112 Discussion13 Viruses rely on host cells for replication, and cell entry is the first step of the viral 14 infection life cycle, and a prime target for drug intervention.\n",
      "\n",
      "a(TEXT:\"SARSCoV-2 pseudovirus\") increases p(HGNC:13557 ! ACE2)\n",
      "a) Live cell imaging snapshots of HEK293 (ACE2) cells treated with GFP expressing SARSCoV-2 pseudovirus with or without HDL NP (50 nM) co-treatment.\n",
      "\n",
      "a(TEXT:SARSr-Rp-BatCoV) increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "Interestingly, two novel SARSr-Rp-BatCoV strains ZXC21 and ZC45 isolated from least horseshoe bat in Zhoushan, Zhejiang province in eastern China were shown to be able to infect suckling rats, causing inflammation in the brain tissue and histological changes in the lung and intestine despite failed viral isolation in VeroE6 cells.\n",
      "\n",
      "a(TEXT:\"SARS‐CoV‐2)\") increases bp(GO:\"GO:0016032\" ! \"viral process\")\n",
      "This virus is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) (Sheeren et al., 2020).\n",
      "\n",
      "a(TEXT:\"SARS‐CoV‐2)\") increases bp(MESH:D004194 ! Disease)\n",
      "Coronavirus disease 2019 (COVID‐19), caused by the severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2), started in the city of Wuhan late in 2019.\n",
      "\n",
      "a(TEXT:\"Spike protein of SARS-CoV-1\") increases bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\")\n",
      "The surfactant protein D (SP-D) is known to bind to the Spike protein of SARS-CoV-1 (Leth-Larsen, Zhong, Chow, Holmskov, & Lu, 2007), the virus that caused the SARS outbreak in 2002.\n",
      "\n",
      "a(TEXT:VSV-SARS-CoV-2-S) increases p(PFAM:PF01353 ! \"green fluorescent protein\")\n",
      "A technical challenge in generating replication-competent VSV particles (Case et al., 2020b; Dieterle et al., 2020), the authors demonstrate that VSV-SARS-CoV-2-S D21 expressing enhanced green fluorescent protein (eGFP) as a reporter transgene, bearing SARS-CoV-2 S on the surface, can be used as a serological tool to measure NAbs to SARS-CoV-2 in a manner comparable to neutralization assays performed at BSL-3 with SARS-CoV-2 virus.\n",
      "\n",
      "a(TEXT:VSV-SARS-CoV-2-S) decreases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "This model was used to determine that VSV-SARS-CoV-2-S D21 could protect from pneumonia and reduce SARS-CoV-2 replication in the lung and limit the induction of pro inflammatory, antiviral immune responses.\n",
      "\n",
      "a(TEXT:Zanamivir.SARS-CoV-2) increases p(UP:Q940U6 ! Q940U6)\n",
      "Patients with Flu positive were treated with Zanamivir.SARS-CoV-2 has been the seventh identified coronavirus, and ranking the third potentially lethal one, after SARS-CoV and MERS-CoV [1, [10], [11], [12], [13]].\n",
      "\n",
      "a(TEXT:\"aCC-BY-NC-ND 4.0 International license.43 Author Summary 44 SARS-CoV-2\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "It is madeavailable under aCC-BY-NC-ND 4.0 International license.43 Author Summary 44 SARS-CoV-2 robustly activates widespread inflammation, but we do not understand 45 mechanistically how the virus engages the immune system.\n",
      "\n",
      "a(TEXT:\"anti-SARS-CoV-2 non-neutralizing antibodies\") increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Concerns have also been raised that anti-SARS-CoV-2 non neutralizing antibodies, or even declining NAb titers over time, could lead to ADE and enhanced disease after such vaccinations, antibody based drug therapies, or treatment with convalescent plasma from recovered patients.\n",
      "\n",
      "a(TEXT:\"anti-SARS-CoV-2 non-neutralizing antibodies\") increases bp(MESH:D019067 ! \"Antibody-Dependent Enhancement\")\n",
      "Concerns have also been raised that anti-SARS-CoV-2 non neutralizing antibodies, or even declining NAb titers over time, could lead to ADE and enhanced disease after such vaccinations, antibody based drug therapies, or treatment with convalescent plasma from recovered patients (139, 140).\n",
      "\n",
      "a(TEXT:\"coronavirus SARS-CoV-2\") increases bp(MESH:D011946 ! \"Receptors, Antigen\")\n",
      "Little is known about the B cells and antigen receptors stimulated by the novel human coronavirus SARS-CoV-2.\n",
      "\n",
      "a(TEXT:died.SARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "As of 5/9/2020, 70,387 confirmed cases were reported in Poland, of which 2113 people have died.SARS-CoV-2, which causes COVID-19, is a member of Coronaviridae family of enveloped, positive-sense RNA viruses type beta, which also encompasses SARS-CoV-1 and MERS-CoV (Harapan et al. 2020; Shaw et al. 2020; Feng et al. 2020).\n",
      "\n",
      "a(TEXT:\"human-to-human transmission ofSARS-CoV-2\") increases a(CHEBI:73907 ! Gly-Ile)\n",
      "As per gastroenterologists, human-to-human transmission ofSARS-CoV-2 is possible by airborne respiratory droplets and by the fecal-oral route, raising the possibility of GI manifestations of the disease.\n",
      "\n",
      "a(TEXT:immunocytochemistry.SARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "The present study shows that two compounds, GRL-0920 and GRL-0820, exert potent activity against SARS-CoV-2, while a number of previously known compounds such as favipiravir, nelfinavir, chloroquine, and hydroxychloroquine (except the nucleotide analogue remdesivir) failed to show significant activity in examinations of their antiviral effect using RNA-quantitative PCR (RNA-qPCR), cytopathicity, and immunocytochemistry.SARS-CoV-2, which causes COVID-19, belongs to the family of betacoronaviruses that includes SARS-CoV and MERS-CoV.\n",
      "\n",
      "a(TEXT:n-SARS-CoV-2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "This biosensor is simple to manufacture and highly sensitive, revealing immense potential for the detection of IgG and IgM in patients treated with n-SARS-CoV-2 for appropriate health monitoring.Trinh et al. developed a fully integrated foldable biosensor encapsulated in agarose for long-term reagent storage and multiple fluorescence detection [132].\n",
      "\n",
      "a(TEXT:n-SARS-CoV-2) increases bp(MESH:D007075 ! \"Immunoglobulin M\")\n",
      "This biosensor is simple to manufacture and highly sensitive, revealing immense potential for the detection of IgG and IgM in patients treated with n-SARS-CoV-2 for appropriate health monitoring.Trinh et al. developed a fully integrated foldable biosensor encapsulated in agarose for long-term reagent storage and multiple fluorescence detection [132].\n",
      "\n",
      "a(TEXT:\"new SARS-CoV\") increases bp(MESH:D003643 ! Death)\n",
      "Currently, the apparition of new SARS-CoV, known as SARS-CoV-2, affected more than 34 million people and causing high death rates worldwide.\n",
      "\n",
      "a(TEXT:\"permission.Figure 2 SARS-CoV-2 proteins\") increases a(CHEBI:81564 ! \"resolvin D1\")\n",
      "No reuse allowed without permission.Figure 2 SARS-CoV-2 proteins stimulate RvD1 biosynthesis.\n",
      "\n",
      "a(TEXT:\"permission.Figure S2.Fig.S2 SARS-Cov2 RBD-Fc Vacc\") increases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "No reuse allowed without permission.Figure S2.Fig.S2 SARS-Cov2 RBD-Fc Vacc elicit strong humoral immunity in mice.\n",
      "\n",
      "a(TEXT:permission.SARS-CoV-2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "No reuse allowed without permission.SARS-CoV-2 causing COVID-19 is an RNA virus and studies have confirmed its presence in respiratory tract samples, gastro-intestinal tract samples, feces (25-30%) and blood (1-15%) 3-4.\n",
      "\n",
      "a(CHEBI:147285 ! \"EC 3.4.22.69 (SARS coronavirus main proteinase) inhibitor\") increases bp(GO:\"GO:0019058\" ! \"viral life cycle\")\n",
      "Mpro inhibitors bind to Mpro and prevent this viral replication and block the proteolytic cleavage of protein precursors essential for the infection 's inception .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:ARF)\n",
      "COVID19 = Severe Acute Coronavirus (SARS-CoV2) induced Disease; ARF = Acute Hypoxic Respiratory Failure; DNI = Do not Intubate Order; NRS = Noninvasive respiratory support; HFNC = High Flow Nasal Cannula; CPAP = Continuous Positive Airway Pressure; NIV = Noninvasive Mechanical Ventilation; ETI = Endotracheal Intubation; SB = Spontaneous Breathing.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "Corona Virus Disease 2019 (COVID-19) is triggered by the Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV2) and has rapidly developed into a worldwide pandemic.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV-2) caused Corona Virus Disease 2019 (COVID-19) in more than ~ 77 million individuals with ~ 1.7 million fatalities globally.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "SARS-CoV-2 causes the Corona Virus Disease 2019 (CoViD-19), designated a global pandemic by the World Health Organization (WHO).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "Corona Virus Disease 2019 (COVID-19) is caused by SARS-CoV-2, a new type of coronavirus, which has high contagion and mainly causes respiratory symptoms.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Corona Virus Disease 2019\")\n",
      "The ongoing Corona Virus Disease 2019 (COVID-19) pandemic, caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), has emphasized the urgent need for antiviral therapeutics.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"bioRxiv preprint doi\")\n",
      "101 Preparation to launch SRP, FSP, and TRP for the 2020 summer started in Fall 2019 102, and application review, interviews, and candidate selection were almost complete 103 just before the COVID-19 pandemic caused by the SARS-CoV2 novel coronavirus [16] forced a6 bioRxiv preprint doi : https://doi.org/10.1101/2020.11.23.393967; this version posted November 23, 2020.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:\"Coronavirus infectious disease\")\n",
      "SARS-CoV2 is a source of coronavirus infectious disease ( COVID-19 ) , this is considered as a fatal disease to universal communal health apprehension .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases p(HGNC:11892 ! TNF)\n",
      "SARS-CoV2 induces rapid Th1-type lymphocyte activation entailing the production of pro inflammatory cytokines (TNF, IL6, but also IL1, vascular endothelial growth factor (VEGF), monocyte chemoattractant protein-1 (MCP-1), IL-8,...).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases p(HGNC:11892 ! TNF)\n",
      "Moreover , SARS-CoV2 inhibits tumor necrosis factor ( TNF ) receptor-associated factors ( TRAF ) 3 and 6 , which are crucial for the induction of IRF-3 / 7 in response to TLR3 / 7 and NFkappaB signaling pathways .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases p(HGNC:11892 ! TNF)\n",
      "Moreover , SARS-CoV2 inhibits tumor necrosis factor ( TNF ) receptor-associated factors ( TRAF ) 3 and 6 , which are crucial for the induction of IRF-3 / 7 in response to TLR3 / 7 and NFkB signaling pathways .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases p(HGNC:11892 ! TNF)\n",
      "Moreover, SARS-CoV2 inhibits tumor necrosis factor (TNF) receptor associated factors (TRAF) 3 and 6, which are crucial for the induction of IRF-3/7 in response to TLR3/7 and NFkappaB signaling pathways.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases p(HGNC:11892 ! TNF)\n",
      "Moreover, SARS-CoV2 inhibits tumor necrosis factor (TNF) receptor associated factors (TRAF) 3 and 6, which are crucial for the induction of IRF-3/7 in response to TLR3/7 and NFkB signaling pathways.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:Hyposmia)\n",
      "Following an accumulation of publications in the recent weeks , it has become widely accepted that anosmia or hyposmia is induced by SARS-CoV2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases a(TEXT:Hyposmia)\n",
      "Following an accumulation of publications in the recent weeks , it has become widely accepted that anosmia or hyposmia is induced by SARS-CoV2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D004751 ! Enteritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D004751 ! Enteritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D004751 ! Enteritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D005756 ! Gastritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D005756 ! Gastritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D005756 ! Gastritis)\n",
      "As such , our findings add to the evolving evidence that SARS-CoV2 might cause acute gastritis and enteritis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D011655 ! \"Pulmonary Embolism\")\n",
      "SARS-CoV2 induced pulmonary embolism and complications from anticoagulation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D011655 ! \"Pulmonary Embolism\")\n",
      "SARS-CoV2 induced pulmonary embolism and complications from anticoagulation Coronavirus disease ( COVID-19 ) pandemic has rapidly spread around the world .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D012306 ! Risk)\n",
      "Aside from the direct invasion of CNS , SARS-CoV2 might increase the risk for PD because of the induction of a systemic inflammatory state [ 22 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D013927 ! Thrombosis)\n",
      "18 It has also been suggested that there is an association between the inflammatory cascade and the coagulation cascade.19 Taken together , it has been postulated that SARS-CoV2 can cause thrombosis formation .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D013927 ! Thrombosis)\n",
      "[ 42 ] [ 43 ] [ 44 ] It should be noted that SARS-CoV-2 can infect endothelial cells and cause endothelial cell damage and thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D013927 ! Thrombosis)\n",
      "During the influenza H1N1 pandemic in December 2009 , there was a sharp increase in KD cases.42 Similarly , the severe acute respiratory syndrome coronavirus-2 ( SARS-CoV-2 ) , a cause of the coronavirus pandemic in 2019-2020 ( COVID-19 ) , was associated with sharp increase in the incidence of KD.42-44 It should be noted that SARS-CoV-2 can infect endothelial cells and cause endothelial cell damage and thrombosis .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D013927 ! Thrombosis)\n",
      "2 Herein , we report a flare of APLS following COVID-19 , suggesting that SARS-CoV-2 may also trigger thrombosis in pre-existing conditions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D013927 ! Thrombosis)\n",
      "2 Herein , we report a flare of APLS following COVID-19 , suggesting that SARS-CoV-2 may also trigger thrombosis in pre-existing conditions .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "It is unknown if SARS-CoV2 triggers autoimmunity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "It is unknown if SARS-CoV2 triggers autoimmunity .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "Severe acute respiratory syndrome corona virus 2 ( SARS-CoV-2 ) , the cause of COVID-19 disease , has the potential to elicit autoimmunity because mimicry of human molecular chaperones by viral proteins .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "Severe acute respiratory syndrome corona virus 2 ( SARS-CoV-2 ) , the cause of COVID-19 disease , has the potential to elicit autoimmunity because mimicry of human molecular chaperones by viral proteins .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "There have been reports in the literature of COVID-19 shortly preceding a de novo diagnosis of SLE , presenting concomitantly or mimicking SLE , raising the possibility of SARS-CoV-2 being a trigger of autoimmunity , as it has been postulated for other viruses .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "ASSOCIATION OF COVID-19 WITH SLE FLARES AND de novo SLE There have been reports in the literature of COVID-19 shortly preceding a de novo diagnosis of SLE , presenting concomitantly or mimicking SLE , raising the possibility of SARS-CoV-2 being a trigger of autoimmunity , as it has been postulated for other viruses.19 , 111 , 211 However , we need to use caution when making a diagnosis of SLE in patients with COVID-19 for various reasons .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) increases bp(MESH:D015551 ! Autoimmunity)\n",
      "There have been reports in the literature of COVID-19 shortly preceding a de novo diagnosis of SLE , presenting concomitantly or mimicking SLE , raising the possibility of SARS-CoV-2 being a trigger of autoimmunity , as it has been postulated for other viruses.19 , 111 , 211 However , we need to use caution when making a diagnosis of SLE in patients with COVID-19 for various reasons .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "SARS-Cov and SARS-CoV2 bind ACE2 receptors , leading to membrane fusion and virus penetration into the cell , thereby causing the downregulation of these receptors [ 98 , 103 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-CoV2) decreases bp(GO:\"GO:0038023\" ! \"signaling receptor activity\")\n",
      "Upon the entrance into the cell via ACE2 receptor , SARS-CoV-2 leads to the downregulation of these receptors ( Zhang H. et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"demyelinating polyneuropathy\")\n",
      "NCS shows , as common elements , the presence of a partial or complete conduction block on several nerves , slightly prolonged latency of the tibial nerve , and rare or absent F-waves , all suggesting a demyelinating polyneuropathy due to SARS-COV-2 .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "SARS-COV-2 induces endothelial dysfunction [ 23 ] and infects vascular beds in multiple tissues [ 24 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "SARS-COV-2 induces endothelial dysfunction [ 23 ] and infects vascular beds in multiple tissues [ 24 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "SARS-CoV-2 causes endothelial dysfunction , diffuse microangiopathy , and microthrombosis , leading to ventilation-perfusion mismatch ; and intrapulmonary right-to-left shunting aggravates further hypoxia and a vicious cycle leading to remodeling of the pulmonary vasculature .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "SARS-CoV-2 causes endothelial dysfunction , diffuse microangiopathy , and microthrombosis , leading to ventilation-perfusion mismatch ; and intrapulmonary right-to-left shunting aggravates further hypoxia and a vicious cycle leading to remodeling of the pulmonary vasculature .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "They have highlighted that SARS-CoV-2 causes endothelial dysfunction in various district [ 67 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases a(TEXT:\"endothelial dysfunction\")\n",
      "These findings appear to support the hypothesis that SARS-CoV-2 induces endothelial dysfunction and diffuse inflammation that could explain the circulatory impairment and the high risk of rethrombosis despite anticoagulation.17 , 18 In fact , except in one case , arterial occlusions occurred in previously healthy arteries .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "In addition , it is possible to raise delirium as a primary encephalopathy and harbinger of invasion of the CNS by SARS-COV-2 ( Beach et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "In addition , it is possible to raise delirium as a primary encephalopathy and harbinger of invasion of the CNS by SARS-COV-2 ( Beach et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "In addition , it is possible to raise delirium as a primary encephalopathy and harbinger of invasion of the CNS by SARS-COV-2 ( Beach et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "Because SARS-CoV-2 causes delirium in a significant proportion of patients in the acute stage , clinicians should also be aware of the possibility of the development of depression , anxiety , fatigue , post-traumatic stress disorder , and rarer neuropsychiatric syndromes [ 41 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "Because SARS-CoV-2 causes delirium in a significant proportion of patients in the acute stage , clinicians should also be aware of the possibility of the development of depression , anxiety , fatigue , post-traumatic stress disorder , and rarer neuropsychiatric syndromes [ 41 ] .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "This analysis suggests that SARS-CoV-2 might cause delirium in a significant proportion of patients in the acute stage and clinicians should be aware of the possibility of depression , anxiety , fatigue and post-traumatic stress disorder in the longer term .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "[ 4 ] Because of the similarities in the pathogenicity of the SARS-CoV and SARS-CoV-2 , it is plausible that SARS-CoV-2 has a similar ability to cause delirium .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "[ 4 ] Because of the similarities in the pathogenicity of the SARS-CoV and SARS-CoV-2 , it is plausible that SARS-CoV-2 has a similar ability to cause delirium .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "This analysis suggests that SARS-CoV-2 might cause delirium in a significant proportion of patients in the acute stage and clinicians should be aware of the possibility of depression , anxiety , fatigue and post-traumatic stress disorder in the longer term .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "[ 4 ] Because of the similarities in the pathogenicity of the SARS-CoV and SARS-CoV-2 , it is plausible that SARS-CoV-2 has a similar ability to cause delirium .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D003693 ! Delirium)\n",
      "SARS-CoV-2 might cause delirium in a significant proportion of patients in the acute stage .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D005121 ! Extremities)\n",
      "Smaller case series report a purpuric rash often on extremities due to SARS-COV-2 mediated complement-associated vascular injury .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D016638 ! \"Critical Illness\")\n",
      "Our patient with critical illness due to SARS-COV-2 , unfortunately , has achieved poor functional outcome .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases bp(MESH:D016638 ! \"Critical Illness\")\n",
      "Our patient with critical illness due to SARS-COV-2 , unfortunately , has achieved poor functional outcome .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) increases p(UP:Q940U6 ! Q940U6)\n",
      "It is made available under a CC-BY-ND 4.0 International license.53 Introduction54 A novel human coronavirus named by the World Health Organization as Severe Acute Respiratory 55 Syndrome coronavirus 2 (SARS-COV-2) has been causing a pandemic of flu like disease from the end 56 of 2019 (1).\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) decreases a(TEXT:\"bradykinin 2-receptor\")\n",
      "SARS-COV-2 also decreases bradykinin 2-receptor effects as it affects bradykinin synthesis by inhibiting cathepsin L, a kininogenase present at the site of infection and involved in bradykinin production.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) decreases p(HGNC:6383 ! KNG1)\n",
      "SARS-COV-2 also decreases bradykinin 2-receptor effects as it affects bradykinin synthesis by inhibiting cathepsin L, a kininogenase present at the site of infection and involved in bradykinin production.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) decreases a(TEXT:Chronic)\n",
      "Since some research showed that SARS-COV-2 might also cause myocardial injury and chronic damage to the cardiovascular system through angiotensinconverting enzyme 2 ( ACE2 ) receptors [ 49 ] , the significant association between CK-MB and RadScore may indicate that RadScore could reveal the COVID-19-related myocardial lesion in the late-phase group .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-COV-2) decreases a(TEXT:Chronic)\n",
      "Since some research showed that SARS-COV-2 might also cause myocardial injury and chronic damage to the cardiovascular system through angiotensin-converting enzyme 2 ( ACE2 ) receptors 49 , the significant association between CK-MB and RadScore may indicate that RadScore could reveal the COVID-19-related myocardial lesion in the late-phase group .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) increases a(TEXT:\"covid19 outbreak\")\n",
      "We illustrate the interest to analyse the receptor binding domain (RBD) or the virus SARS-Cov-2 causing the covid19 outbreak.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) increases a(TEXT:\"Abnormal renal physiology\")\n",
      "Therefore , SARS-Cov-2 could enter the renal tubular cell by binding to ACE2 and induce cytotoxicity and abnormal renal function , which is a potential mechanism how abnormal renal function happens in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) increases a(TEXT:\"Abnormal renal physiology\")\n",
      "Therefore , SARS-Cov-2 could enter the renal tubular cell by binding to ACE2 and induce cytotoxicity and abnormal renal function , which is a potential mechanism how abnormal renal function happens in COVID-19 patients .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) decreases a(CHEBI:138560 ! \"5-amino-1-(5-phospho-beta-D-ribosyl)imidazole\")\n",
      "Reduction of Contagion Risks by SARS-Cov-2 (COVID-19) in Air Conditioned Work Environments.\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) decreases bp(MESH:D001814 ! \"Blood-Testis Barrier\")\n",
      "SARS-Cov-2 could infect Sertoli cells and damage the blood-testis barrier by binding ACE2 , other than directly entering cells or testis tissues , resulting in leukocyte infiltration , which also happened in SARS .\n",
      "\n",
      "bp(MESH:C000656484 ! SARS-Cov-2) decreases bp(MESH:D001814 ! \"Blood-Testis Barrier\")\n",
      "SARS-Cov-2 could infect Sertoli cells and damage the blood-testis barrier by binding ACE2 , other than directly entering cells or testis tissues , resulting in leukocyte infiltration , which also happened in SARS .\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 5' UTR\") increases bp(GO:\"GO:0006412\" ! translation)\n",
      "Consistent with this hypothesis, increasing the linker length between Nsp1-NT and Nsp1-CT greatly reduced SARS-CoV-2 5 ' UTR mediated translation in the presence of Nsp1.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 5' UTR\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Interestingly, compared with the 20-residue linker, the 40-residue linker was more effective in suppressing SARS-CoV-2 5 ' UTR mediated evasion of translation inhibition.\n",
      "\n",
      "a(TEXT:anti-SARS-CoV-2) decreases a(TEXT:rAOaV-1-SARS-CoV-2)\n",
      "In conjunction to this hypothesis, the anti-295 AOaV-1 antiserum as well as anti-SARS-CoV-2 could substantially block the entry of296 the rAOaV-1-SARS-CoV-2.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Mpro\") decreases p(UP:P0C6V4 ! P0C6V4)\n",
      "Our results show that ATV could dock in the active site of SARS-CoV-2 Mpro, with greater strength than LPV, blocking Mpro activity.\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "SARS-CoV-2 spike protein binds to the ACE-2 receptor which may cause activation of p38 mitogen-activated protein kinase ( MAPK ) , upregulate ADAM-17 , and stimulate ROS formation [ 69 , 70 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "SARS-CoV-2 spike protein binds to the ACE-2 receptor which may cause activation of p38 mitogen-activated protein kinase ( MAPK ) , upregulate ADAM-17 , and stimulate ROS formation [ 69 , 70 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases a(TEXT:S1)\n",
      "SARS-CoV-2 spike protein ( S ) is cleaved by the human furin enzyme to generate S1 , which binds to the host receptor , ACE-2 .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases a(TEXT:S1)\n",
      "SARS-CoV-2 spike protein ( S ) is cleaved by the human furin enzyme to generate S1 , which binds to the host receptor , ACE-2 .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "SARS-CoV-2 Spike Protein Elicits Cell Signaling in Human Host Cells : Implications for Possible Consequences of COVID-19 Vaccines .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "The major finding of this study is that the SARS-CoV-2 spike protein without the rest of the virus can elicit cell signaling , specifically the activation of the MEK / ERK pathway , in human host lung vascular smooth muscle and endothelial cells .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "Thus , SARS-CoV-2 spike protein promotes the MEK phosphorylation in human cells , but not in rat cells .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "The major finding of this study is that the SARS-CoV-2 spike protein without the rest of the virus can elicit cell signaling , specifically the activation of the MEK / ERK pathway , in human host lung vascular smooth muscle and endothelial cells .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0005623\" ! cell)\n",
      "Thus , SARS-CoV-2 spike protein promotes the MEK phosphorylation in human cells , but not in rat cells .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "A structure model analysis shows that SARS-CoV-2 S protein binds to ACE2 with higher affinity than SARS-CoV S leading to rapid viral replication and then inducing an inflammatory response and provoking an accumulation of a large amounts of pro-inflammatory cytokines ( Wrapp et al. 2020b ) ( Fig. 1 ; adapted from de la Rica et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "A structure model analysis shows that SARS-CoV-2 S protein binds to ACE2 with higher affinity than SARS-CoV S leading to rapid viral replication and then inducing an inflammatory response and provoking an accumulation of a large amounts of pro-inflammatory cytokines ( Wrapp et al. 2020b ) ( Fig. 1 ; adapted from de la Rica et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0006954\" ! \"inflammatory response\")\n",
      "A structure model analysis shows that SARS-CoV-2 S protein binds to ACE2 with higher affinity than SARS-CoV S leading to rapid viral replication and then inducing an inflammatory response and provoking an accumulation of a large amounts of pro-inflammatory cytokines ( Wrapp et al. 2020b ) ( Fig. 1 ; adapted from de la Rica et al. 2020 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "SARS-CoV-2 spike protein can recognize ACE2 in the open conformation and initiate membrane fusion and viral entry .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "SARS-CoV-2 spike protein can recognize ACE2 in the open conformation and initiate membrane fusion and viral entry .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases bp(GO:\"GO:0061025\" ! \"membrane fusion\")\n",
      "Internalized 25HC accumulates in the late endosomes and potentially restricts SARS-CoV-2 spike protein catalyzed membrane fusion via blockade of cholesterol export .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:195 ! ADAM17)\n",
      "SARS-CoV-2 spike protein binds to the ACE-2 receptor which may cause activation of p38 mitogen-activated protein kinase ( MAPK ) , upregulate ADAM-17 , and stimulate ROS formation [ 69 , 70 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:195 ! ADAM17)\n",
      "SARS-CoV-2 spike protein binds to the ACE-2 receptor which may cause activation of p38 mitogen-activated protein kinase ( MAPK ) , upregulate ADAM-17 , and stimulate ROS formation [ 69 , 70 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:6018 ! IL6)\n",
      "While the SARS-CoV-1 S protein upregulated IL-6 , IL-8 , and TNF , the SARS-CoV-2 S protein upregulated IL-6 but not IL-8 or TNF ( Figures 1 and 2 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:6018 ! IL6)\n",
      "While the SARS-CoV-1 S protein upregulated IL-6 , IL-8 , and TNF , the SARS-CoV-2 S protein upregulated IL-6 but not IL-8 or TNF ( Figure 1 and Figure 2 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:6025 ! CXCL8)\n",
      "While the SARS-CoV-1 S protein upregulated IL-6 , IL-8 , and TNF , the SARS-CoV-2 S protein upregulated IL-6 but not IL-8 or TNF ( Figures 1 and 2 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases p(HGNC:6025 ! CXCL8)\n",
      "While the SARS-CoV-1 S protein upregulated IL-6 , IL-8 , and TNF , the SARS-CoV-2 S protein upregulated IL-6 but not IL-8 or TNF ( Figure 1 and Figure 2 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(MESH:D009619 ! Nociceptors)\n",
      "Khanna et al. recently reported that the SARS-CoV-2 spike protein interacts with NRP1 to induce signaling in nociceptors that then causes antinociceptive effects.64 Neuropilin 1 is strongly expressed in hDRG71 ,90 and is found in rodent nociceptors.64 Importantly , NRP1 was recently reported as a host entry receptor for SARS-CoV-2.13 ,19 This happens because when the spike protein is cleaved by the protease furin , which is also expressed in hDRG ,76 the processed protein has a high affinity for NRP1 and NRP2 .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(MESH:D009619 ! Nociceptors)\n",
      "77 Khanna et al. recently reported that the SARS-CoV-2 spike protein interacts with NRP1 to induce signaling in nociceptors that then causes antinociceptive effects .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(MESH:D012084 ! \"Renin-Angiotensin System\")\n",
      "In line , it was proposed that the SARS-CoV-2 spike protein activates RAS in the lung , thereby promoting platelet adhesion and aggregation and increasing the risk for pulmonary embolism , hypertension and fibrosis .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") increases bp(MESH:D057134 ! \"Antibodies, Neutralizing\")\n",
      "Ad5 encoding SARS-CoV-2 S protein that was designed to produce neutralizing antibodies upon intramuscular injection generated toxicity at a high dose and a low dose was further evaluated in Phase II clinical trials [ 309,312 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases p(HGNC:13557 ! ACE2)\n",
      "Moreover , ACE2 downregulation by SARS-CoV-2 spike protein and related deleterious effects of elevated angiotensin II and decreased angiotensin 1-7 have been proposed as an explanation of the cytokine storm and the multi-organs dysfunction seen in COVID-19 patients ( Banu et al ., 2020 ; Hoiland et al ., 2020 ) .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases p(HGNC:13557 ! ACE2)\n",
      "In contrast , Kuba et al. reported that the SARS-CoV-2 spike protein downregulated the expression of ACE2 [ 5 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases p(HGNC:13557 ! ACE2)\n",
      "In contrast , Kuba et al. reported that the SARS-CoV-2 spike protein downregulated the expression of ACE2 [ 5 ] .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases p(HGNC:13557 ! ACE2)\n",
      "While ACE2 downregulation by SARS-CoV-2 spike protein leads to an overactivation of Angiotensin ( Ang ) II / AT1R axis and the deleterious effects of Ang II may explain the multiorgan dysfunction seen in patients .\n",
      "\n",
      "bp(MESH:C000657845 ! \"spike protein, SARS-CoV-2\") decreases bp(MESH:D001812 ! \"Blood-Brain Barrier\")\n",
      "A recent study by Buzhdygan et al. found that the SARS-CoV-2 spike protein can destabilize the blood-brain barrier ( BBB ) and trigger a proinflammatory state at the brain endothelium [ 9 ] .\n",
      "\n",
      "p(HGNC:10537 ! SARS1) increases p(HGNC:11784 ! THBD)\n",
      "Further, we explored the potent antiviral mechanism of EK1C4 and found that the complexes of EK1C4/SARS-HR1, EK1C4/MERS-HR1, and EK1C4/SARS-2-HR1 harbor higher stability and increased Tm values than those of the complexes formed by EK1 and HR1s (Supplementary information, Fig. S4).\n",
      "\n",
      "p(HGNC:10537 ! SARS1) increases p(HGNC:39025 ! ERVK-18)\n",
      "Further, we explored the potent antiviral mechanism of EK1C4 and found that the complexes of EK1C4 and SARS-HR 1, EK1C4 and MERS-HR 1, and EK1C4 and SARS-2-HR1 harbor higher stability and increased Tm values than those of the complexes formed by EK1 and HR1s (Supplementary information, Fig.S4).\n",
      "\n",
      "p(HGNC:10537 ! SARS1) increases bp(MESH:D014615 ! Vaccinia)\n",
      "A panel of five contiguous cDNA fragments (SARS-1 to SARS-5) spanning the entire viral genome of the SARS-CoV HKU-39849 strain was cloned in vaccinia virus DNA (V) by in vitro ligation to produce two recombinant vaccinia viruses containing nucleotides 1–20,288 (vSARS-CoV 5′) and 20,272–29,727 (vSARS-CoV 3′) of the SARS-CoV genome, respectively.\n",
      "\n",
      "p(HGNC:10537 ! SARS1) increases bp(MESH:D015994 ! Incidence)\n",
      "All these features were seen in a group of post-SARS survivors following the 2003 SARS-1 outbreak , 8  and it is well accepted that SARS-1 led to a significant incidence of CFSME .\n",
      "\n",
      "p(HGNC:10537 ! SARS1) increases p(HGNC:6018 ! IL6)\n",
      "SARS-1 notably produced severe inflammation, with high levels of IL-6, TNF-ɑ, interferons, and multi-organ system failure, and case reports suggest patients with this level of inflammation are benefitting from anti-IL6 monoclonal antibody therapy, prompting randomized clinical trials testing tocilizumab and sarilumab.MERS spread rapidly among travelers in 2012, particularly those making the Hajj to Mecca, and had a case mortality rate over 34%.\n",
      "\n",
      "p(HGNC:10537 ! SARS1) decreases p(FPLX:Interferon ! Interferon)\n",
      "Some reports have indicated that the SARS- and MERS- CoVs are able to inhibit the production of the IFNs.\n",
      "\n",
      "p(HGNC:10537 ! SARS1) decreases p(FPLX:Interferon ! Interferon)\n",
      "Both SARS-1 and MERS inhibit Type I IFN production by preventing nuclear transport of the transcription factor , IFN regulatory factor 3 ( IRF3 ) , and thereby inhibit activation of the IFN-beta promoter by IRF3 [ 45 ] , although this has not been established for SARS CoV-2 .\n",
      "\n",
      "a(TEXT:\"SARS-CoV S\") increases a(TEXT:hACE2)\n",
      "In addition, SARS-CoV S and NL63 S may cause differential downregulation of hACE2 level by induced shedding of its ectodomain from the cell surface.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(GO:\"GO:0001837\" ! \"epithelial to mesenchymal transition\")\n",
      "Interestingly, several members of the group of SARS2 induced EMT genes have been associated with signature pulmonary comorbidities of CoV infection, including ADAR, CLDN1 and SOD2.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases p(HGNC:4819 ! HAS2)\n",
      "In that, it's natural to testify whether HIS-SARS2 could activate HAS2.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "Now, in the present times, it emerges as a COVID-19, which is caused by the SARS2 Coronavirus and poses a significant risk to the human race.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(MESH:C000657245 ! COVID-19)\n",
      "The present COVID-19 epidemic caused by SARS-2 was declared pandemic in March, 2020.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "In conclusion, we confirmed that multimerization of RBDs on nanoparticles enhances immunogenicity compared with soluble antigen and further showed that homotypic SARS-2 nanoparticle immunization produced IgG responses that bound zoonotic RBDs and neutralized heterologous coronaviruses after boosting.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(MESH:D011014 ! Pneumonia)\n",
      "Temporal Clinical and Laboratory Response to Interleukin-6 Receptor Blockade With Tocilizumab in 89 Hospitalized Patients With COVID-19 Pneumonia Background : Pandemic COVID-19 pneumonia due to SARS-2 is an important cause of morbidity and mortality .\n",
      "\n",
      "p(HGNC:17697 ! SARS2) increases bp(MESH:D011014 ! Pneumonia)\n",
      "BACKGROUND : Pandemic COVID-19 pneumonia due to SARS-2 is an important cause of morbidity and mortality .\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:13557 ! ACE2)\n",
      "It has been posited that patients with baseline ACE2 deficiency would be particularly vulnerable to severe outcomes because of unchecked ACE activity with SARS2 virus downregulating ACE2 expression.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:13557 ! ACE2)\n",
      "It has been posited that patients with baseline ACE2 deficiency would be particularly vulnerable to severe outcomes because of unchecked ACE activity with SARS2 virus downregulating ACE2 expression.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:18157 ! TAB1)\n",
      "Combined with the results from in vitro assay, we found the proteases of SARS2 (PLpro and 3CLpro) can degrade IRF3, TAB1, and NLRP12 probably through their protease activity, leading to the imbalance responses of host innate immunityExperimental screens identify accessible cleavage sites in proteins and more importantly, detects noncanonical sequences.In this report, we show that the viral proteases PLpro and 3CLpro of SARS-CoV2 lead to the in-vitro proteolytic cleavage of three important proteins of the host immune response: IRF3, TAB1 and NLRP12 (Fig. 1B) .\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:22938 ! NLRP12)\n",
      "Combined with the results from in vitro assay, we found the proteases of SARS2 (PLpro and 3CLpro) can degrade IRF3, TAB1, and NLRP12 probably through their protease activity, leading to the imbalance responses of host innate immunityExperimental screens identify accessible cleavage sites in proteins and more importantly, detects noncanonical sequences.In this report, we show that the viral proteases PLpro and 3CLpro of SARS-CoV2 lead to the in-vitro proteolytic cleavage of three important proteins of the host immune response: IRF3, TAB1 and NLRP12 (Fig. 1B) .\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:6118 ! IRF3)\n",
      "SARS M protein inhibits the activation of IRF3 mediated by both TBK1 and IKKepsilon.\n",
      "\n",
      "p(HGNC:17697 ! SARS2) decreases p(HGNC:6118 ! IRF3)\n",
      "Combined with the results from in vitro assay, we found the proteases of SARS2 (PLpro and 3CLpro) can degrade IRF3, TAB1, and NLRP12 probably through their protease activity, leading to the imbalance responses of host innate immunityExperimental screens identify accessible cleavage sites in proteins and more importantly, detects noncanonical sequences.In this report, we show that the viral proteases PLpro and 3CLpro of SARS-CoV2 lead to the in-vitro proteolytic cleavage of three important proteins of the host immune response: IRF3, TAB1 and NLRP12 (Fig. 1B) .\n",
      "\n",
      "a(TEXT:SARS-CoV-2-RBD) decreases p(HGNC:7652 ! NBN)\n",
      "Identification and characterization of SARS-CoV-2-RBD and ACE2 blocking Nbs.\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "SARs-COV2 infection may have triggered relapse in this patient , as was recently reported in a Spanish study 9 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "SARs-COV2 infection may have triggered relapse in this patient , as was recently reported in a Spanish study9 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "SARs-COV2 infection may have triggered relapse in this patient , as was recently reported in a Spanish study9 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "In our results ( Figure 2 ) , a positive correlation was found between age with the following : feelings of lack of energy ( r =0 .33 ; p =0 .01 ) , concern for the further course of the disease ( r =0 .32 ; p =0 .01 ) , and feeling of the possibility of relapse due to COVID-19 ( r =0 .34 ; p =0 .03 ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "23 However , beyond the direct risk of infection , could COVID-19 indirectly lead to relapse among schizophrenia patients ?\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "But 30.8 % of them believed that `` COVID-19 may cause relapse '' , and 29.3 % selected the answer `` COVID-19 infection may worsen MS symptoms '' .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARs-COV2 infection\") increases a(TEXT:\"disease recurrence\")\n",
      "But 30.8 % of them believed that `` COVID-19 may cause relapse '' , and 29.3 % selected the answer `` COVID-19 infection may worsen MS symptoms '' .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Sars-Cov-2 infection causes fever and cough that may rapidly lead to acute respiratory distress syndrome ( ARDS ) .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Clinical aspects may be helpful : COVID-19 seems to cause more frequently fever and cough , while dyspnea occurs later and correlates with severe lung damage .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Initially , COVID-19 infects the respiratory tract of a human and causes fever , cough , shortness in breathing , and many other serious issues like pneumonia [ 1 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "COVID-19 has been reported to cause fever , cough , shortness of breath , and even death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Despite evidence that individual symptoms and signs have relatively poor diagnostic properties for COVID-19 in adults ,1 other causes of fever and cough are uncommon in healthy adults .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "The coronavirus disease 2019 ( Covid-19 ) causes fever , cough , sore throat , and in severe cases shortness of breath and death .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "COVID-19 can cause respiratory tract disease , fever and cough , and severe pneumonia in some extreme cases [ 2 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "Fever , chills , colds , cough , sore throat , difficulty of breathing , or diarrhea , which may occur either due to their rheumatic disease or COVID-19 , occurred in less than 10 % .\n",
      "\n",
      "bp(MESH:C000657245 ! \"Sars-Cov-2 infection\") increases bp(MESH:D003371 ! Cough)\n",
      "COVID-19 can induce not only mild to severe respiratory diseases , but also inflammation , high fever , cough , acute respiratory tract infection , and dysfunction of internal organs which may result in death [ 4 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-COV-2 infection\") increases bp(MESH:D012640 ! Seizures)\n",
      "It is suspected that SARS-COV-2 infection may aggravate seizures in a patient with a history of epilepsy or being monitored for epilepsy .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-COV-2 infection\") increases bp(MESH:D012640 ! Seizures)\n",
      "It is suspected that SARS-COV-2 infection may aggravate seizures in a patient with a history of epilepsy or being monitored for epilepsy .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-COV-2 infection\") increases bp(MESH:D012640 ! Seizures)\n",
      "However , available studies are consistent with the general concept that seizures and epilepsy are infrequently worsened by COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-COV-2 infection\") increases bp(MESH:D012640 ! Seizures)\n",
      "However , available studies are consistent with the general concept that seizures and epilepsy are infrequently worsened by COVID-19 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-COV-2 infection\") increases bp(MESH:D012640 ! Seizures)\n",
      "Dizziness ( 16.8 % ) , headache ( 13.1 % ) , skeletal muscle injury ( 10.7 % ) are more frequently seen , but COVID-19 can also lead to impaired consciousness ( 7.5 % ) , acute cerebrovascular disease ( 2.8 % ) , ataxia ( 0.5 % ) , seizures ( 0.5 % ) , meningoencephalitis and the Guillain-Barr e syndrome13 .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 Infection\") increases bp(MESH:D019590 ! \"Nucleocapsid Proteins\")\n",
      "SARS-CoV-2 Infection induced GC T fh  cells specific for the SARS-CoV-2 spike and nucleocapsid proteins , and a corresponding early appearance of antiviral serum IgG antibodies .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 Infection\") increases bp(MESH:D019590 ! \"Nucleocapsid Proteins\")\n",
      "SARS-CoV-2 Infection induced GC Tfh cells specific for the SARS-CoV-2 spike and nucleocapsid proteins , and a corresponding early appearance of antiviral serum IgG antibodies .\n",
      "\n",
      "bp(MESH:C000669933 ! \"SARS-CoV-2 vaccines\") increases bp(MESH:D018417 ! \"Th1 Cells\")\n",
      "Optimal SARS-CoV-2 vaccines must also produce a T helper type 1 (Th1) versus type 2 (Th2) balance favoring Th1 responses.Previous SARS 17 and MERS 18 vaccine candidates raised preclinical safety concerns regarding exacerbating lung disease from Th2 responses or as a result of antibody dependent enhancement (ADE) of viral entry 19.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp14\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "In this report, we showed that SARS-CoV nsp14 but not the MTase defective D331A mutant could inhibit the in vitro translation similarly as m7GTP (XREF_FIG), suggesting that nsp14 could efficiently transform GTP into m7GTP in the translation system and consequently inhibit the mRNA translation.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp14\") directlyIncreases a(CHEBI:15996 ! GTP)\n",
      "In this study, we found that GTP, dGTP as well as cap analogs GpppG, GpppA and m7GpppG could be methylated by SARS-CoV nsp14.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp14\") directlyIncreases a(CHEBI:15996 ! GTP)\n",
      "3.2 Deoxy GTP and dinucleotide cap analogs can also be methylated by SARS-CoV nsp14.\n",
      "\n",
      "a(TEXT:\"SARS-CoV nsp14\") directlyIncreases a(CHEBI:16750 ! guanosine)\n",
      "Taken together, these results showed that SARS-CoV nsp14 could methylate guanosine with three phosphates at N7-position in either oxy or deoxy form.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2's Spike protein\") decreases a(TEXT:SNI)\n",
      "SARS-CoV-2 's Spike protein and NRP-1 antagonism reverses SNI induced 15 mechanical allodynia.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 like SARS-CoV\") decreases p(HGNC:13557 ! ACE2)\n",
      "Based on these facts, we presume that SARS-CoV-2 like SARS-CoV downregulates ACE2, and in absence and reduced activity of ACE2, level of angiotensin (1-7) and angiotensin (1-9) is decreased while that of angiotensin II is increased and increased level of angiotensin II has been found to correlate with lung injury and viral load.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 PLpro\") decreases p(HGNC:4053 ! ISG15)\n",
      "Further addition of purified SARS-CoV-2 PLpro (100nM) to cell lysate efficiently reduced ISG15 conjugated proteins.\n",
      "\n",
      "a(TEXT:\"SARS-CoV PL2\") directlyDecreases p(HGNC:17479 ! RCHY1, pmod(go:0016567 ! \"protein ubiquitination\"))\n",
      "While the HCoV-NL63 PL2 pro stabilizes MDM2 by deubiquitinating it, the SARS-CoV PL2 pro surprisingly does not deubiquitinate RCHY1.\n",
      "\n",
      "a(TEXT:\"COVID-19pos patients.SARS-CoV2\") increases p(HGNC:11127 ! SIGLEC1)\n",
      "This broad and unbiased approach showed that immune markers, in particular related to monocytes, segregated COVID19neg and COVID-19pos patients.SARS-CoV2 induces CD169 expressing monocyte subsets To investigate circulating monocyte heterogeneity and define consistent phenotypes, we used the FlowSOM algorithm.\n",
      "\n",
      "a(TEXT:\"SARS-114 CoV-2\") increases p(HGNC:13557 ! ACE2)\n",
      "Similar to those of SARS and MERS, the SARS-114 CoV-2 infection also increased the expression level of ACE2 in A549 cells (P < 0.05) (Smith et al. 2020).\n",
      "\n",
      "a(TEXT:\"SARS-CoV 3b protein\") increases p(HGNC:6204 ! JUN)\n",
      "Furthermore, the SARS-CoV 3b protein stimulates the expression of the transcription factor AP-1 by activating JNK and ERK signaling pathways in Huh7 cells.\n",
      "\n",
      "a(TEXT:\"SARS-CoV encoding proteins\") increases a(TEXT:\"hfgl2 gene\")\n",
      "It was concluded that these recombinant eukaryotic expression plasmids were constructed successfully, and SARS-CoV encoding proteins could activate transcription and expression of hfgl2 gene.\n",
      "\n",
      "a(TEXT:\"SARS-CoV encoding proteins\") increases a(TEXT:\"hfgl2 gene\")\n",
      "It was concluded that these recombinant eukaryotic expression plasmids were constructed successfully, and SARS-CoV encoding proteins could activate transcription and expression of hfgl2 gene.\n",
      "\n",
      "a(TEXT:\"SARS-CoV unique domain\") increases p(HGNC:5992 ! IL1B)\n",
      "Recently, the SARS-CoV unique domain (SUD), that include three macrodomains (N, M, and C), was shown to significantly up-regulate the expression of IL-1beta and cytokines through NLRP3 activation, in human lung epithelial cells and in mice lung tissues [XREF_BIBR].\n",
      "\n",
      "a(TEXT:\"SARS-CoV unique domain\") increases p(HGNC:5992 ! IL1B)\n",
      "Recently, the SARS-CoV unique domain (SUD), that include three macrodomains (N, M, and C), was shown to significantly up-regulate the expression of IL-1beta and cytokines through NLRP3 activation, in human lung epithelial cells and in mice lung tissues [15].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 COVID-19 4 5\") increases p(HGNC:8568 ! FURIN)\n",
      "Persistently increased systemic ACE2 activity and Furin levels are 2 associated with increased inflammatory response in smokers with SARS-CoV-2 COVID-19 4 5.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 MNA vaccines\") increases bp(MESH:D007074 ! \"Immunoglobulin G\")\n",
      "When tested in mice, both MERS-CoV and SARS-CoV-2 MNA vaccines were able to induce significant levels of serum IgG.\n",
      "\n",
      "a(TEXT:\"SARS-CoV2 (1*106\") increases p(HGNC:13557 ! ACE2)\n",
      "In our current manuscript, we demonstrated that the intranasal administration of SARS-CoV2 (1 * 106) strongly induced the expression of ACE-2, promoted the entry of virions into the lung and kidney cells, caused acute histopathological toxicities, and mortality (28%).\n",
      "\n",
      "a(TEXT:\"SARS-Cov-2 antigens\") increases p(UP:P51526 ! P51526)\n",
      "For example, a SARS-CoV-2-specific cellular 184 response was associated with severity of disease : recovered patients with mild COVID-19 185 illnesses demonstrated high levels of IFN- induced by SARS-Cov-2 antigens, while severe 186 pneumonia patients showed significantly lower level of this cytokine45.\n",
      "\n",
      "a(TEXT:\"SARS26 CoV-2\") increases p(HGNC:13557 ! ACE2)\n",
      "Moreover, such enhancement of SARS26 CoV-2 infectivity was not seen with several other viruses probably due to a unique IAV 27 segment as an inducer to elevate ACE2 expression.\n",
      "\n",
      "a(TEXT:\"SARS435 CoV\") increases p(HGNC:9605 ! PTGS2)\n",
      "Nucleocapsid protein of SARS435 CoV activates the expression of cyclooxygenase-2 by binding directly to regulatory elements 436 for nuclear factor-kappa B and CCAAT and enhancer binding protein.\n",
      "\n",
      "a(TEXT:\"COVID-19 virus SARS-CoV-2\") decreases bp(GO:\"GO:0006914\" ! autophagy)\n",
      "Here, we demonstrated that ORF3a of the COVID-19 virus SARS-CoV-2 inhibits autophagy activity by blocking fusion of autophagosomes and amphisomes with lysosomes.\n",
      "\n",
      "a(TEXT:\"SARS HRC\") decreases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "The [SARS HRC -PEG 4] 2 -chol lipopeptide inhibited all SARS-CoV-2 strains with S mutations at comparable potency and showed considerable potency against both SARS-CoV and MERSCoV.\n",
      "\n",
      "a(TEXT:\"SARS-CoV ORF6\") decreases bp(GO:\"GO:0006810\" ! transport)\n",
      "SARS-CoV ORF6 was shown to inhibit NPCmediated transport by tethering the importin proteins KPNA2 and KPNB2 and TNPO1 to ER/Golgi membranes 32, which effectively blocks the import of immune signaling proteins such as STAT1 into the nucleus.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases bp(GO:\"GO:0006351\" ! \"transcription, DNA-templated\")\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane (Frieman et al. 2007).\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases p(HGNC:11362 ! STAT1)\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases p(HGNC:11362 ! STAT1)\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane (Frieman et al. 2007).\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases bp(MESH:C048760 ! \"isopropyl methylphosphonic acid\")\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases bp(MESH:C048760 ! \"isopropyl methylphosphonic acid\")\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane (Frieman et al. 2007).\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases p(UP:Q1QXE1 ! Q1QXE1)\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane.\n",
      "\n",
      "a(TEXT:\"SARS-CoV accessory protein ORF6\") decreases p(UP:Q1QXE1 ! Q1QXE1)\n",
      "SARS-CoV accessory protein ORF6 antagonizes the antiviral activity of the STAT1 transcription factor by sequestering IMPalpha and beta1 on the rough ER/Golgi membrane (Frieman et al. 2007).\n",
      "\n",
      "a(TEXT:\"SARS-CoV field isolate Frankfurt-1\") decreases a(TEXT:EC)\n",
      "The SARS-CoV field isolate Frankfurt-1 was inhibited with an EC 50 of 8.3 muM and for the mouse adapted SARS-CoV strain MA-15 (MOI 0.05; kindly provided by prof. Luis Enjuanes, National Center of Biotechnology (CNB-CSIC), Madrid, Spain) an EC 50 of 1.3 muM was determined.\n",
      "\n",
      "a(TEXT:\"SARS-CoV protein 16\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "As predicted based on the SARS-CoV protein 16, the NSP1 (RK/AA) mutant inhibited translation similar to the wild-type protein (IC50 = ~ 420 +/- 11 nM, 95% C.I., R 2 = 0.89), while the NSP1 (KH/AA) mutant failed to inhibit translation.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-1 proteins\") decreases p(IP:IPR007702)\n",
      "Several SARS-CoV-1 proteins antagonize the antiviral activities of IFNs and the downstream JAK (Janus kinase)-STAT signaling pathways they activate.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 (2\") decreases p(HGNC:11362 ! STAT1)\n",
      "Once STAT1 function is impaired by the NSP1 or ORF6 protein of SARS-CoV-2 (2, Fig. 3), a concomitant and compensatory shift to signaling via STAT1 independent pathways may occur, and a STAT3 dependent transcriptional profile becomes dominant in many situations (3, Fig. 3) [55] [56] [57].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 (2\") decreases p(HGNC:11362 ! STAT1)\n",
      "Once STAT1 function is impaired by the NSP1 or ORF6 protein of SARS-CoV-2 (2, Fig. 3), a concomitant and compensatory shift to signaling via STAT1 independent pathways may occur, and a STAT3 dependent transcriptional profile becomes dominant in many situations (3, Fig. 3) [55-57].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 -ACE2 complex2,33\") decreases p(HGNC:13557 ! ACE2)\n",
      "Functional ACE2 deficiency occurs due to internalization of the SARS-CoV-2 -ACE2 complex2,33, which restricts ACE2 from performing its physiological functions, including its role as carboxypeptidase for angiotensin II and desArg9-bradykinin and other peptide hormones.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 122 transmission\") decreases bp(GO:\"GO:0006955\" ! \"immune response\")\n",
      "It is reasonably anticipated that population immunity will wane over time, 120 requiring robust memory responses21 to prevent reinfection or modify the clinical course22.121 Our findings support emerging evidence that population heterogeneity of SARS-CoV-2 122 transmission, attributable to a range of biological and sociological factors, reduces the herd 123 immunity threshold16,23,24.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Mpro activity.Lopinavir\") decreases a(TEXT:\"SARS-CoV Mpro\")\n",
      "Here, we examined the inhibitory effect of lopinavir-ritonavir on SARS-CoV-2 Mpro protease activity and found that lopinavir-ritonavir is not effective to inhibit SARS-CoV-2 Mpro activity.Lopinavir was reported to inhibit SARS-CoV Mpro with an IC50 of 25 ~ 50 muM [2].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Mpro inhibitor cinanserin\") decreases a(CHEBI:37670 ! \"protease inhibitor\")\n",
      "Using the optimization guidelines, the SARS-CoV-2 Mpro inhibitor cinanserin was optimized resulting in an increase in binding affinity of 4.59 -log10 (Kd) and increased protease inhibitor bioactivity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF3b protein\") decreases p(FPLX:Interferon ! Interferon)\n",
      "91 Finally, the SARS-CoV-2 ORF3b protein, a potent interferon antagonist, suppresses production of type I interferon and this antagonism is reportedly elevated in severe COVID-19 cases.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF6 protein\") decreases p(HGNC:5434 ! IFNB1)\n",
      "Although SARS-CoV-2 ORF6 shares only 69% sequence homology with SARS-CoV-1 ORF6, the SARS-CoV-2 ORF6 protein reduced the IFN-beta promoter activity by 100-fold and suppressed IFN signaling to levels comparable to those imposed by SARS-CoV-1 ORF6.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF6 proteins\") decreases a(TEXT:IFN-I)\n",
      "The SARS-CoV-1 and SARS-CoV-2 ORF6 proteins inhibit IFN-I and IFN-III secretion induced by Sendai virus infection, a method often used to elicit IFN-I production, and have profound effects on IRF3 phosphorylation, thus accounting for their potent inhibition of IFN-I production [20, 46].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF6 proteins\") decreases a(TEXT:IFN-III)\n",
      "The SARS-CoV-1 and SARS-CoV-2 ORF6 proteins inhibit IFN-I and IFN-III secretion induced by Sendai virus infection, a method often used to elicit IFN-I production, and have profound effects on IRF3 phosphorylation, thus accounting for their potent inhibition of IFN-I production [20, 46].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 ORF6\") decreases bp(GO:\"GO:0006810\" ! transport)\n",
      "SARS-CoV-2 ORF6 disrupts nucleocytoplasmic transport through 661 interactions with Running title : SARS-CoV-2 ORF6 disrupts mRNA nuclear export.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 PLP\") decreases p(HGNC:11998 ! TP53)\n",
      "Therefore, SARS-CoV, MERS-CoV, and SARS-CoV-2 PLP are all able to mediate p53 degradation in a process that lead to decreased cell apoptosis favoring viral growth in infected cells as well as to the loss of the antiviral activity of p53.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 PLP\") decreases p(HGNC:11998 ! TP53)\n",
      "Therefore, SARS-CoV, MERS-CoV, and SARS-CoV-2 PLP are all able to mediate p53 degradation in a process that lead to decreased cell apoptosis favoring viral growth in infected cells as well as to the loss of the antiviral activity of p53.In this regard, p53 is a pleiotropic molecule deeply analyzed for its several functions and involvement in different pathways, including those related to antiviral innate immune responses carried out particularly by inducing apoptosis of infected cells and mediating type I interferon (IFN) production and signaling (Munoz-Fontela et al., 2008).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 Receptor Binding Domain\") decreases bp(MESH:C000656484 ! SARS-CoV-2)\n",
      "These changes can lead to decreased binding between ACE2 receptor and SARS-CoV-2 Receptor Binding Domain (RBD) due to steric hindrance by the addition of a large sized PO 4 3- molecule, thus reducing SARS-CoV-2 infectivity.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 accessory proteins\") decreases a(TEXT:IFN-I)\n",
      "70 For example, one of the SARS-CoV-2 accessory proteins, ORF6, was suggested to disrupt 71 nucleocytoplasmic transport leading to inhibition of gene expression 29 and several viral proteins 72 have been demonstrated to antagonize IFN-I production and signaling 24,30.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 accessory proteins\") decreases bp(GO:\"GO:0006810\" ! transport)\n",
      "70 For example, one of the SARS-CoV-2 accessory proteins, ORF6, was suggested to disrupt 71 nucleocytoplasmic transport leading to inhibition of gene expression 29 and several viral proteins 72 have been demonstrated to antagonize IFN-I production and signaling 24,30.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 cases\") decreases p(HGNC:17661 ! PLA1A)\n",
      "Services provided by neuromuscular centers, such as rehabilitation, home care nursing, psychological support and prenatal diagnosis, faced a reduction in 57% of centers, showing higher reduction rates in geographical areas with higher numbers of SARS-CoV-2 cases (aggregate services reduction in 66% of Northern NMD centers, 43% of Central centers and 44% of Southern centers) (XREF_FIG).\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 gene\") decreases p(FPLX:Interferon ! Interferon)\n",
      "Orf6 was also the only SARS-CoV-2 gene suppressing the activity of an interferon stimulated response element (ISRE) promoter in both studies.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 nonstructural protein 1\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "The SARS-CoV-2 nonstructural protein 1 (Nsp1) is known to inhibit host protein translation and could be a target for antiviral therapy against COVID-19.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 nonstructural protein 1\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "The SARS-CoV-2 nonstructural protein 1 (Nsp1) is known to inhibit host protein translation and could be a target for antiviral therapy against COVID-19.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 pathobiology.ACE2\") decreases p(HGNC:333 ! AGT)\n",
      "Robust replication of the entered SARS-CoV-2 through ACE2 receptors triggers release of large numbers of cytokines, especially IL-6, causing acute fatal \" cytokine storms \"; that is, efficient entry followed by replication of the virus is essential for SARS-CoV-2 pathobiology.ACE2 reduces angiotensin II (ANGII), the major effector of the renin-angiotensin-aldosterone system that promotes HTN, by converting ANGII to Ang-(1-7) which increases blood pressure.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins ORF6\") decreases a(TEXT:IFN-I)\n",
      "The influenza and SARS-CoV genomes code for the nonstructural protein 1 (NS1), which antagonises IFN-I and -III signalling; similarly, the SARS-CoV-2 proteins ORF6, ORF8 and its nucleocapsid also inhibit IFN-I signalling in vitro [52-54].\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 proteins ORF6\") decreases a(TEXT:IFN-I)\n",
      "[50, 51] The influenza and SARS-CoV genomes code for the nonstructural protein 1 (NS1), which antagonizes IFN-I and -III signaling; similarly, the SARS-CoV-2 proteins ORF6, ORF8 and its nucleocapsid also inhibit IFN-I signaling in vitro.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 replicon\") decreases a(TEXT:\"SARS-CoV-2 viral infection17\")\n",
      "116 Sensitivity of the SARS-CoV-2 replicon to antiviral agents 117 We tested the sensitivity of SARS-CoV-2 replicon to remdesivir, which has been 118 demonstrated to inhibit SARS-CoV-2 viral infection17.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2, b) control proin-flammatory cytokines\") decreases a(CHEBI:26523 ! \"reactive oxygen species\")\n",
      "In another study, Luo et al., has also discussed the medicinal properties of glycyrrhizin against COVID-19 with the following mechanisms; a) Binding S-RBD and ACE2 to inhibit SARS-CoV-2, b) control proin-flammatory cytokines, c) preventing the intracellular ROS buildup of, d) controlling thrombi,.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-217\f",
      "bioRxiv preprint doi\") decreases p(HGNC:3287 ! EIF4E)\n",
      "Blocking these upstream eIF4E kinases resulted in reduced SARS-CoV-217 bioRxiv preprint doi : https://doi.org/10.1101/2020.11.29.401984; this version posted November 30, 2020.\n",
      "\n",
      "a(TEXT:SARS-CoV-2nsp1) decreases a(TEXT:IFN-I)\n",
      "Remarkably, SARS-CoV-2nsp1 and nsp6 suppress IFN-I signaling more efficiently than SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV).\n",
      "\n",
      "a(TEXT:\"SARS486 CoV-2\") decreases p(FPLX:CD8 ! CD8)\n",
      "Urra JM, Cabrera CM, Porras L, Rodenas I. Selective CD8 cell reduction by SARS486 CoV-2 is associated with a worse prognosis and systemic inflammation in COVID487 19 patients.\n",
      "\n",
      "a(TEXT:\"SARS78 CoV-2\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "SARS78 CoV-2 infection induces global translation inhibition but surprisingly the translation of viral 79 transcripts is not preferred over their cellular counterparts.\n",
      "\n",
      "a(TEXT:\"SARSCoV-2 virus isolate SARS-CoV-2-Jan\") decreases p(UP:P03126 ! P03126)\n",
      "Very similar values were also observed in an ELISA based SARS-CoV-2 spike S1 protein neutralization for all ACE2-IgG4-Fc and ACE2-IgG1-Fc constructs.All ACE2-IgG4-Fc and ACE2-IgG1-Fc fusion proteins inhibited Vero E6 cell infection by the SARSCoV-2 virus isolate SARS-CoV-2-Jan, which was obtained from the earliest documented COVID19 cases in Germany from January 2020 and is closely related to the original Wuhan strain [38], with IC50 values of around 10 nM.\n",
      "\n",
      "a(TEXT:\"VSV260 eGFP-SARS-CoV-2\") decreases a(TEXT:\"https://doi.org/10.1101/2020.07.09.196386\")\n",
      "Should substantial reactogenicity or neuronal infection be observed with VSV260 eGFP-SARS-CoV-2, the vaccine could be attenuated further by introducing mutations into the11 bioRxiv preprint doi : https://doi.org/10.1101/2020.07.09.196386.\n",
      "\n",
      "a(TEXT:ppSARS-CoV-2) decreases p(HGNC:13557 ! ACE2)\n",
      "Besides, we have validated that ppSARS-CoV-2 entry in vitrocan be effectively blocked using a novel, soluble, ACE2, whose IC50 was determined as 0.032 mug/mL, indicating that soluble ACE2 can be a promising alternative strategy for prevention and treatment of 22].\n",
      "\n",
      "a(TEXT:ppSARS-CoV-2) decreases p(HGNC:13557 ! ACE2)\n",
      "Besides, we have validated that ppSARS-CoV-2 entry in vitro can be effectively blocked using a novel, soluble, ACE2, whose IC50 was determined as 0.032 mug/mL, indicating that soluble ACE2 can be a promising alternative strategy for prevention and treatment of COVID-19 [21,22].\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV2 infection\") decreases bp(MESH:D005298 ! Fertility)\n",
      "These260 findings together with our study suggest that the SARS-CoV2 infection is unlikely to damage 261 fertility .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infections\") decreases bp(MESH:D016131 ! \"Lymphocyte Subsets\")\n",
      "Previous studies indicated that SARS-CoV-2 infections could lead to dysregulation of the levels of lymphocyte subsets , by showing that the absolute counts of CD3 + T cells , CD4 + T cells , CD8 + T cells , CD19 + B cells , and CD56 + NK cells were reduced in patients with COVID-19 [ 2 , 5 ] .\n",
      "\n",
      "bp(MESH:C000657245 ! \"SARS-CoV-2 infections\") decreases bp(MESH:D016131 ! \"Lymphocyte Subsets\")\n",
      "Previous studies indicated that SARS-CoV-2 infections could lead to dysregulation of the levels of lymphocyte subsets , by showing that the absolute counts of CD3 + T cells , CD4 + T cells , CD8 + T cells , CD19 + B cells , and CD56 + NK cells were reduced in patients with COVID-19 [ 2 , 5 ] .\n",
      "\n",
      "bp(MESH:C000669933 ! \"SARS-CoV-2 vaccine\") decreases bp(MESH:D000906 ! Antibodies)\n",
      "Theoretically, a SARS-CoV-2 vaccine could skew the immune system toward production of less effective or even harmful cross-reactive hCoV Abs via these processes of OAS and ADE, rather than induce highly targeted SARS-CoV-2-specific Abs as intended (162).\n",
      "\n",
      "bp(MESH:C000669933 ! \"SARS-CoV-2 vaccine\") decreases bp(MESH:D003643 ! Death)\n",
      "SARS-CoV-2 vaccine prevents severe cases and thus death.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 DeltaSL1 5' UTR mScarlet reporter\") decreases bp(GO:\"GO:0006412\" ! translation)\n",
      "Compared with mScarlet translation in control cells co-transfected with MBP, the translation of SARS-CoV-2 DeltaSL1 5 ' UTR mScarlet reporter was markedly reduced in MBP-Nsp1 transfected cells (p < 0.001), suggesting that SL1 is required for evasion of Nsp1 mediated translation suppression.\n",
      "\n",
      "a(TEXT:\"SARS-CoV-2 nsp16-nsp10 2'-O-methyltransferase complex\") directlyIncreases a(TEXT:\"Cap-0 viral mRNAs\")\n",
      "To facilitate structure based drug design, we conducted an x-ray crystallographic study of the SARS-CoV-2 nsp16-nsp10 2 '-O and methyltransferase complex, which methylates Cap-0 viral mRNAs to improve viral protein translation and to avoid host immune detection.\n"
     ]
    }
   ],
   "source": [
    "for u, v, data in emmaa_covid_graph.edges(sars2, data=True):\n",
    "        if 'evidence' in data:\n",
    "            print()\n",
    "            print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "            print(data['evidence'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get all ACE2-angiotensin edges\n",
    "for u, v, data in emmaa_covid_graph.edges(ace2s, data=True):\n",
    "    if v == angiotensin:\n",
    "        print()\n",
    "        print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "        print(data['evidence'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "p(hgnc:13557 ! ACE2) increases a(chebi:48433 ! angiotensin)\n",
      "Upon its discovery , ACE-2 was shown to generate angiotensin 1-7 ( Ang ( 1-7 ) ) from Ang II , and , to a lesser extent , angiotensin 1-9 ( Ang ( 1-9 ) ) from Ang I [ 23 , 24 , 28 ] .\n",
      "\n",
      "p(hgnc:13557 ! ACE2) increases a(chebi:48433 ! angiotensin)\n",
      "Down-regulation of ACE2 causes excessive production of angiotensin (ANG) II by the related enzyme ACE with stimulation of ANG type 1a receptor (AT1R) and enhanced lung vascular permeability [18].\n",
      "\n",
      "p(hgnc:13557 ! ACE2) increases a(chebi:48433 ! angiotensin)\n",
      "In this study, the purified ACE2 was able to convert Ang II into Ang 1-7 (XREF_FIG A and B) (K m = 2.87 +/-0.76 muM, XREF_FIG), suggesting that ACE2 prevents Ang II accumulation in the glomeruli, by regulating the degradation of angiotensin peptides and supporting the concept that ACE2 acts in a conter-regulatory manner to ACE and may play an important role to modulate the balance between vasoconstrictors and vasodilators in the kidney.\n",
      "\n",
      "p(hgnc:13557 ! ACE2) decreases a(chebi:48433 ! angiotensin)\n",
      "In 2000, ACE2, a homolog of the angiotensin converting enzyme (ACE) gene was reported to negatively regulate RAS by converting angiotensin (Ang)-II to Ang-1-7 XREF_BIBR.\n",
      "\n",
      "p(hgnc:13557 ! ACE2) decreases a(chebi:48433 ! angiotensin)\n",
      "ACE2 is not only a receptor for the SARS virus, but also inhibits the Angiotensin cascade contributing to control excessive AT1R activity.\n",
      "\n",
      "p(hgnc:13557 ! ACE2) decreases a(chebi:48433 ! angiotensin)\n",
      "Angiotensin converting enzyme 2 (ACE2) degrades angiotensin (Ang) II to Ang-(1-7), and protects against diabetic renal injury.\n"
     ]
    }
   ],
   "source": [
    "# Get all ACE2-angiotensin edges\n",
    "for u, v, data in emmaa_covid_graph.edges(ace2s, data=True):\n",
    "    if v == angiotensin:\n",
    "        print()\n",
    "        print(emmaa_covid_graph.edge_to_bel(u, v, data))\n",
    "        print(data['evidence'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---------------------  ----------\n",
      "Name\n",
      "Version\n",
      "Number of Nodes        1970\n",
      "Number of Namespaces     11\n",
      "Number of Edges        7253\n",
      "Number of Annotations     7\n",
      "Number of Citations    1599\n",
      "Number of Authors         0\n",
      "Network Density           0.00187\n",
      "Number of Components      1\n",
      "Number of Warnings        0\n",
      "---------------------  ----------\n",
      "\n",
      "Type (5)             Count  Example\n",
      "-----------------  -------  --------------------------------------------------------------------------------\n",
      "Complex                709  complex(p(HGNC:13557 ! ACE2), p(HGNC:483 ! ANG))\n",
      "Abundance              547  a(CHEBI:74795 ! Pro-Phe)\n",
      "Protein                391  p(HGNC:5962 ! IL10)\n",
      "BiologicalProcess      312  bp(MESH:D000544 ! \"Alzheimer Disease\")\n",
      "Reaction                11  rxn(reactants(a(CHEBI:80131 ! Apelin-13)), products(a(CHEBI:80131 ! Apelin-13)))\n",
      "\n",
      "Namespace (11)      Count  Example\n",
      "----------------  -------  -------------------------------------------\n",
      "HGNC                 1216  p(HGNC:5039 ! HNRNPF)\n",
      "TEXT                  481  a(TEXT:cytokine)\n",
      "CHEBI                 409  a(CHEBI:85045 ! H3K27Me3)\n",
      "MESH                  330  bp(MESH:D003469 ! Culture)\n",
      "UP                    176  p(UP:O29445 ! O29445)\n",
      "GO                     92  bp(GO:\"GO:0050900\" ! \"leukocyte migration\")\n",
      "FPLX                   86  p(FPLX:PKC ! PKC)\n",
      "PFAM                   69  p(PFAM:PF03145 ! Sina)\n",
      "go                     21\n",
      "IP                      6  p(IP:IPR006053)\n",
      "PUBCHEM                 2  a(PUBCHEM:21881944 ! 21881944)\n",
      "\n",
      "Edge Type (22)                                 Count  Example\n",
      "-------------------------------------------  -------  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
      "Protein increases Protein                       1178  p(HGNC:95 ! SLC33A1) increases p(HGNC:13557 ! ACE2)\n",
      "Protein decreases Protein                       1081  p(HGNC:5438 ! IFNG) decreases p(HGNC:13557 ! ACE2)\n",
      "Abundance increases Protein                      815  a(CHEBI:16412 ! lipopolysaccharide) increases p(HGNC:13557 ! ACE2)\n",
      "Protein increases BiologicalProcess              700  p(HGNC:13557 ! ACE2) increases bp(MESH:D003092 ! Colitis)\n",
      "Protein partOf Complex                           663  p(HGNC:13557 ! ACE2) partOf complex(p(HGNC:13557 ! ACE2), p(HGNC:6561 ! LGALS1))\n",
      "Abundance decreases Protein                      618  a(CHEBI:16234 ! hydroxide) decreases p(HGNC:13557 ! ACE2)\n",
      "Protein decreases BiologicalProcess              504  p(HGNC:13557 ! ACE2) decreases bp(MESH:D055371 ! \"Acute Lung Injury\")\n",
      "BiologicalProcess increases Protein              466  bp(MESH:D045169 ! \"Severe Acute Respiratory Syndrome\") increases p(HGNC:13557 ! ACE2)\n",
      "Protein increases Abundance                      448  p(HGNC:13557 ! ACE2) increases a(CHEBI:6801 ! metformin)\n",
      "BiologicalProcess decreases Protein              370  bp(MESH:D014764 ! \"Viral Proteins\") decreases p(HGNC:13557 ! ACE2)\n",
      "Protein decreases Abundance                      195  p(HGNC:13557 ! ACE2) decreases a(TEXT:Bi)\n",
      "BiologicalProcess partOf Complex                  40  bp(MESH:C413524 ! ACE2) partOf complex(bp(GO:\"GO:0008233\" ! \"peptidase activity\"), bp(MESH:C413524 ! ACE2))\n",
      "Abundance directlyIncreases Protein               34  a(CHEBI:5801 ! hydroxychloroquine) directlyIncreases p(HGNC:13557 ! ACE2, pmod(go:0006486 ! \"protein glycosylation\"))\n",
      "Protein directlyIncreases Protein                 33  p(FPLX:AMPK ! AMPK) directlyIncreases p(HGNC:13557 ! ACE2, pmod(go:0006468 ! \"protein phosphorylation\", Ser, 680))\n",
      "Complex decreases Protein                         26  complex(a(CHEBI:77260 ! \"5-methoxy-2-\\{[(4-methoxy-3,5-dimethylpyridin-2-yl)methyl]sulfinyl\\}-1H-benzimidazole\"), p(HGNC:13557 ! ACE2)) decreases p(HGNC:13557 ! ACE2)\n",
      "Complex increases Protein                         22  complex(bp(MESH:C000656484 ! \"severe acute respiratory syndrome coronavirus 2\"), p(HGNC:13557 ! ACE2)) increases p(HGNC:13557 ! ACE2)\n",
      "Protein directlyIncreases Reaction                21  p(HGNC:13557 ! ACE2) directlyIncreases rxn(reactants(a(CHEBI:80131 ! Apelin-13)), products(a(CHEBI:80131 ! Apelin-13)))\n",
      "Abundance directlyDecreases Protein               13  a(CHEBI:27584 ! aldosterone) directlyDecreases p(HGNC:13557 ! ACE2)\n",
      "BiologicalProcess directlyIncreases Protein       12  bp(MESH:D000375 ! Aging) directlyIncreases p(HGNC:13557 ! ACE2)\n",
      "Protein hasVariant Protein                        11  p(HGNC:13557 ! ACE2) hasVariant p(HGNC:13557 ! ACE2, var(\"p.?\"))\n",
      "Abundance increases BiologicalProcess              2  a(CHEBI:50114 ! estrogen) increases bp(MESH:C413524 ! ACE2)\n",
      "BiologicalProcess decreases Abundance              1  bp(MESH:C413524 ! ACE2) decreases a(TEXT:Angiotensin-2)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "ace2_graph = pybel.struct.get_subgraph_by_neighborhood(emmaa_covid_graph, ace2s)\n",
    "\n",
    "# Make sure all the edges between are added\n",
    "#pybel_tools.mutation.expand_internal_causal(emmaa_covid_graph, ace2_graph)\n",
    "\n",
    "ace2_graph.summarize()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Counter({'Protein': 391,\n",
       "         'Abundance': 547,\n",
       "         'BiologicalProcess': 312,\n",
       "         'Complex': 709,\n",
       "         'Reaction': 11})"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.struct.count_functions(ace2_graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['EDGE_HIGHLIGHT',\n",
       " 'NODE_HIGHLIGHT',\n",
       " '__builtins__',\n",
       " '__cached__',\n",
       " '__doc__',\n",
       " '__file__',\n",
       " '__loader__',\n",
       " '__name__',\n",
       " '__package__',\n",
       " '__path__',\n",
       " '__spec__',\n",
       " 'bound',\n",
       " 'build_delete_node_by_hash',\n",
       " 'build_expand_node_neighborhood_by_hash',\n",
       " 'collapse',\n",
       " 'collapse_consistent_edges',\n",
       " 'collapse_equivalencies_by_namespace',\n",
       " 'collapse_gene_variants',\n",
       " 'collapse_nodes',\n",
       " 'collapse_nodes_with_same_names',\n",
       " 'collapse_orthologies_by_namespace',\n",
       " 'collapse_protein_variants',\n",
       " 'collapse_to_protein_interactions',\n",
       " 'count_peripheral_predecessors',\n",
       " 'count_peripheral_successors',\n",
       " 'count_sources',\n",
       " 'count_targets',\n",
       " 'deletion',\n",
       " 'enrich_complexes',\n",
       " 'enrich_composites',\n",
       " 'enrich_internal_unqualified_edges',\n",
       " 'enrich_protein_and_rna_origins',\n",
       " 'enrich_reactions',\n",
       " 'enrich_unqualified',\n",
       " 'enrich_variants',\n",
       " 'expand_internal_causal',\n",
       " 'expand_periphery',\n",
       " 'expansion',\n",
       " 'get_peripheral_predecessor_edges',\n",
       " 'get_peripheral_successor_edges',\n",
       " 'get_subgraph_edges',\n",
       " 'get_subgraph_peripheral_nodes',\n",
       " 'highlight',\n",
       " 'highlight_edges',\n",
       " 'highlight_nodes',\n",
       " 'highlight_subgraph',\n",
       " 'infer_missing_backwards_edge',\n",
       " 'infer_missing_two_way_edges',\n",
       " 'inference',\n",
       " 'is_edge_highlighted',\n",
       " 'is_node_highlighted',\n",
       " 'random',\n",
       " 'random_by_edges',\n",
       " 'random_by_nodes',\n",
       " 'remove_highlight_edges',\n",
       " 'remove_highlight_nodes',\n",
       " 'remove_highlight_subgraph',\n",
       " 'remove_inconsistent_edges',\n",
       " 'rewire_variants_to_genes',\n",
       " 'shuffle_node_data',\n",
       " 'shuffle_relations']"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dir(pybel_tools.mutation)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Counter({'HGNC': 1216,\n",
       "         'go': 21,\n",
       "         'TEXT': 481,\n",
       "         'CHEBI': 409,\n",
       "         'MESH': 330,\n",
       "         'PFAM': 69,\n",
       "         'UP': 176,\n",
       "         'FPLX': 86,\n",
       "         'PUBCHEM': 2,\n",
       "         'IP': 6,\n",
       "         'GO': 92})"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.struct.count_namespaces(ace2_graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "import networkx as nx\n",
    "G = nx.DiGraph()\n",
    "G.add_edge(\"x\", \"a\", capacity=3.0)\n",
    "G.add_edge(\"x\", \"b\", capacity=1.0)\n",
    "G.add_edge(\"a\", \"c\", capacity=3.0)\n",
    "G.add_edge(\"b\", \"c\", capacity=5.0)\n",
    "G.add_edge(\"b\", \"d\", capacity=4.0)\n",
    "G.add_edge(\"d\", \"e\", capacity=2.0)\n",
    "G.add_edge(\"c\", \"y\", capacity=2.0)\n",
    "G.add_edge(\"e\", \"y\", capacity=3.0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...</td>\n",
       "      <td>p(HGNC:11892 ! TNF)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...</td>\n",
       "      <td>p(HGNC:6025 ! CXCL8)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...</td>\n",
       "      <td>p(HGNC:11362 ! STAT1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>p(FPLX:Histone ! Histone)</td>\n",
       "      <td>p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>p(FPLX:Histone ! Histone)</td>\n",
       "      <td>p(FPLX:PPP1 ! PPP1, pmod(go:0006473 ! \"protein...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                   0  \\\n",
       "0  p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...   \n",
       "1  p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...   \n",
       "2  p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...   \n",
       "3                          p(FPLX:Histone ! Histone)   \n",
       "8                          p(FPLX:Histone ! Histone)   \n",
       "\n",
       "                                                   1  \n",
       "0                                p(HGNC:11892 ! TNF)  \n",
       "1                               p(HGNC:6025 ! CXCL8)  \n",
       "2                              p(HGNC:11362 ! STAT1)  \n",
       "3  p(FPLX:Histone ! Histone, pmod(go:0006473 ! \"p...  \n",
       "8  p(FPLX:PPP1 ! PPP1, pmod(go:0006473 ! \"protein...  "
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.DataFrame([\n",
    "    (str(u), str(v))\n",
    "    for u, v, data in emmaa_covid_graph.edges(data=True)\n",
    "    if isinstance(u, pybel.dsl.Protein) and isinstance(v, pybel.dsl.Protein)\n",
    "])\n",
    "df.drop_duplicates().head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "cut_value, partition = nx.minimum_cut(G, \"x\", \"y\")\n",
    "reachable, non_reachable = partition"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[('c', 'y'), ('x', 'b')]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cutset = set()\n",
    "for u, nbrs in ((n, G[n]) for n in reachable):\n",
    "    cutset.update((u, v) for v in nbrs if v in non_reachable)\n",
    "print(sorted(cutset))\n",
    "\n",
    "cut_value == sum(G.edges[u, v][\"capacity\"] for (u, v) in cutset)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from networkx.algorithms.flow import shortest_augmenting_path\n",
    "cut_value == nx.minimum_cut(G, \"x\", \"y\", flow_func=shortest_augmenting_path)[0]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['__builtins__',\n",
       " '__cached__',\n",
       " '__doc__',\n",
       " '__file__',\n",
       " '__loader__',\n",
       " '__name__',\n",
       " '__package__',\n",
       " '__path__',\n",
       " '__spec__',\n",
       " 'boykov_kolmogorov',\n",
       " 'boykovkolmogorov',\n",
       " 'build_flow_dict',\n",
       " 'build_residual_network',\n",
       " 'capacity_scaling',\n",
       " 'capacityscaling',\n",
       " 'cost_of_flow',\n",
       " 'dinitz',\n",
       " 'dinitz_alg',\n",
       " 'edmonds_karp',\n",
       " 'edmondskarp',\n",
       " 'gomory_hu',\n",
       " 'gomory_hu_tree',\n",
       " 'max_flow_min_cost',\n",
       " 'maxflow',\n",
       " 'maximum_flow',\n",
       " 'maximum_flow_value',\n",
       " 'min_cost_flow',\n",
       " 'min_cost_flow_cost',\n",
       " 'mincost',\n",
       " 'minimum_cut',\n",
       " 'minimum_cut_value',\n",
       " 'network_simplex',\n",
       " 'networksimplex',\n",
       " 'preflow_push',\n",
       " 'preflowpush',\n",
       " 'shortest_augmenting_path',\n",
       " 'shortestaugmentingpath',\n",
       " 'utils']"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dir(nx.algorithms.flow)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "anaconda3-2020.11",
   "language": "python",
   "name": "anaconda3-2020.11"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": false,
   "autoclose": false,
   "autocomplete": false,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": true,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
